<?xml version="1.0" encoding="UTF-8"?>
<html xmlns="http://www.w3.org/1999/xhtml">
 <head charset="UTF-8">
  <title>classificationP</title>
  <link rel="stylesheet" type="text/css" href="http://ajax.aspnetcdn.com/ajax/jquery.dataTables/1.9.4/css/jquery.dataTables.css"/>
  <script src="http://ajax.aspnetcdn.com/ajax/jQuery/jquery-1.8.2.min.js" charset="UTF-8" type="text/javascript"> </script>
  <script src="http://ajax.aspnetcdn.com/ajax/jquery.dataTables/1.9.4/jquery.dataTables.min.js" charset="UTF-8" type="text/javascript"> </script>
  <script charset="UTF-8" type="text/javascript">$(function() { $("#results").dataTable(); }) </script>
 </head>
 <body>
  <div class="bs-example table-responsive">
   <table class="table table-striped table-bordered table-hover" id="results">
    <thead>
     <tr>
      <th>articles</th>
      <th>bibliography</th>
      <th>dic:funder</th>
      <th>word:frequencies</th>
     </tr>
    </thead>
    <tbody>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4018281" target="separate" title="remote PDF on server">PMC4018281</a>
       <a href="PMC4018281/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Seroepidemiology and molecular epidemiology of enterovirus 71 in Russia.">
         <i>titl: </i>Seroepidemiology and molecular ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Enterovirus 71 (EV71) is an emerging human pathogen causing massive epidemics of hand, foot and mouth disease with severe neurological complications in Asia. EV71 also circulates in Europe, however it does not cause large outbreaks. The reason for distinct epidemiological patterns of EV71 infection in Europe and Asia and the risk of EV71 epidemic in Europe and Russia remain unknown. Seroepidemiology of EV71 and molecular epidemiology of occasional EV71 isolates were studied to explore circulation of EV71 in Russia. In six regions of Russian Federation, seroprevalence of EV71 in sera collected in 2008 ranged from 5% to 20% in children aged 1-2 years and from 19% to 83% in children aged 3-5 years. The seroprevalence among elder children was significantly higher (41-83% vs. 19-27%) in Asian regions of Russia. EV71 strains identified in Russia in 2001-2011 belonged to subtypes C1 and C2, while genotype C4 that was causing epidemics in Asia since 1998 emerged in 2009 and became dominant in 2013.">
         <i>abst: </i>Enterovirus 71 (EV71) is an em ...
        </span>
       </div>
      </td>
      <td>
       <div title="2">
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/borders" target="separate">borders</a>
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="14">
        <span>
         <a href="http://en.wikipedia.org/wiki/from" target="separate">from x 32</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/{background" target="separate">{background x 29</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/The" target="separate">The x 26</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/seroprevalence" target="separate">seroprevalence x 27</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/children" target="separate">children x 26</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4063664" target="separate" title="remote PDF on server">PMC4063664</a>
       <a href="PMC4063664/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="An HIV epidemic model based on viral load dynamics: value in assessing empirical trends in HIV virulence and community viral load.">
         <i>titl: </i>An HIV epidemic model based on ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Trends in HIV virulence have been monitored since the start of the AIDS pandemic, as studying HIV virulence informs our understanding of HIV epidemiology and pathogenesis. Here, we model changes in HIV virulence as a strictly evolutionary process, using set point viral load (SPVL) as a proxy, to make inferences about empirical SPVL trends from longitudinal HIV cohorts. We develop an agent-based epidemic model based on HIV viral load dynamics. The model contains functions for viral load and transmission, SPVL and disease progression, viral load trajectories in multiple stages of infection, and the heritability of SPVL across transmissions. We find that HIV virulence evolves to an intermediate level that balances infectiousness with longer infected lifespans, resulting in an optimal SPVL∼4.75 log10 viral RNA copies/mL. Adaptive viral evolution may explain observed HIV virulence trends: our model produces SPVL trends with magnitudes that are broadly similar to empirical trends. With regard to variation among studies in empirical SPVL trends, results from our model suggest that variation may be explained by the specific epidemic context, e.g. the mean SPVL of the founding lineage or the age of the epidemic; or improvements in HIV screening and diagnosis that results in sampling biases. We also use our model to examine trends in community viral load, a population-level measure of HIV viral load that is thought to reflect a population's overall transmission potential. We find that community viral load evolves in association with SPVL, in the absence of prevention programs such as antiretroviral therapy, and that the mean community viral load is not necessarily a strong predictor of HIV incidence.">
         <i>abst: </i>Trends in HIV virulence have b ...
        </span>
       </div>
      </td>
      <td>
       <div title="7">
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a> x 5
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/strong" target="separate">strong</a> x 2
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="56">
        <span>
         <a href="http://en.wikipedia.org/wiki/set" target="separate">set x 29</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/and" target="separate">and x 237</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/log" target="separate">log x 46</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/HIV" target="separate">HIV x 120</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/virulence" target="separate">virulence x 41</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4318579" target="separate" title="remote PDF on server">PMC4318579</a>
       <a href="PMC4318579/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="HIV competition dynamics over sexual networks: first comer advantage conserves founder effects.">
         <i>titl: </i>HIV competition dynamics over ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Outside Africa, the global phylogeography of HIV is characterized by compartmentalized local epidemics that are typically dominated by a single subtype, which indicates strong founder effects. We hypothesized that the competition of viral strains at the epidemic level may involve an advantage of the resident strain that was the first to colonize a population. Such an effect would slow down the invasion of new strains, and thus also the diversification of the epidemic. We developed a stochastic modelling framework to simulate HIV epidemics over dynamic contact networks. We simulated epidemics in which the second strain was introduced into a population where the first strain had established a steady-state epidemic, and assessed whether, and on what time scale, the second strain was able to spread in the population. Simulations were parameterized based on empirical data; we tested scenarios with varying levels of overall prevalence. The spread of the second strain occurred on a much slower time scale compared with the initial expansion of the first strain. With strains of equal transmission efficiency, the second strain was unable to invade on a time scale relevant for the history of the HIV pandemic. To become dominant over a time scale of decades, the second strain needed considerable (&gt;25%) advantage in transmission efficiency over the resident strain. The inhibition effect was weaker if the second strain was introduced while the first strain was still in its growth phase. We also tested how possible mechanisms of interference (inhibition of superinfection, depletion of highly connected hubs in the network, one-time acute peak of infectiousness) contribute to the inhibition effect. Our simulations confirmed a strong first comer advantage in the competition dynamics of HIV at the population level, which may explain the global phylogeography of the virus and may influence the future evolution of the pandemic.">
         <i>abst: </i>Outside Africa, the global phy ...
        </span>
       </div>
      </td>
      <td>
       <div title="18">
        <span>
         <a href="http://en.wikipedia.org/wiki/strong" target="separate">strong</a> x 12
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/orange" target="separate">orange</a> x 3
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/borders" target="separate">borders</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/project" target="separate">project</a>
        </span>
       </div>
      </td>
      <td>
       <div title="66">
        <span>
         <a href="http://en.wikipedia.org/wiki/epidemics" target="separate">epidemics x 23</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/was" target="separate">was x 97</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/also" target="separate">also x 31</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/were" target="separate">were x 54</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/its" target="separate">its x 25</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4355742" target="separate" title="remote PDF on server">PMC4355742</a>
       <a href="PMC4355742/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Multi-scale modeling for the transmission of influenza and the evaluation of interventions toward it.">
         <i>titl: </i>Multi-scale modeling for the t ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Mathematical modeling of influenza epidemic is important for analyzing the main cause of the epidemic and finding effective interventions towards it. The epidemic is a dynamic process. In this process, daily infections are caused by people's contacts, and the frequency of contacts can be mainly influenced by their cognition to the disease. The cognition is in turn influenced by daily illness attack rate, climate, and other environment factors. Few existing methods considered the dynamic process in their models. Therefore, their prediction results can hardly be explained by the mechanisms of epidemic spreading. In this paper, we developed a heterogeneous graph modeling approach (HGM) to describe the dynamic process of influenza virus transmission by taking advantage of our unique clinical data. We built social network of studied region and embedded an Agent-Based Model (ABM) in the HGM to describe the dynamic change of an epidemic. Our simulations have a good agreement with clinical data. Parameter sensitivity analysis showed that temperature influences the dynamic of epidemic significantly and system behavior analysis showed social network degree is a critical factor determining the size of an epidemic. Finally, multiple scenarios for vaccination and school closure strategies were simulated and their performance was analyzed.">
         <i>abst: </i>Mathematical modeling of influ ...
        </span>
       </div>
      </td>
      <td>
       <div title="11">
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a> x 9
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/strong" target="separate">strong</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/project" target="separate">project</a>
        </span>
        <br/>
       </div>
      </td>
      <td/>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4372574" target="separate" title="remote PDF on server">PMC4372574</a>
       <a href="PMC4372574/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Superinfection between influenza and RSV alternating patterns in San Luis Potosí State, México.">
         <i>titl: </i>Superinfection between influen ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="The objective of this paper is to explain through the ecological hypothesis superinfection and competitive interaction between two viral populations and niche (host) availability, the alternating patterns of Respiratory Syncytial Virus (RSV) and influenza observed in a regional hospital in San Luis Potosí State, México using a mathematical model as a methodological tool. The data analyzed consists of community-based and hospital-based Acute Respiratory Infections (ARI) consultations provided by health-care institutions reported to the State Health Service Epidemiology Department from 2003 through 2009.">
         <i>abst: </i>The objective of this paper is ...
        </span>
       </div>
      </td>
      <td>
       <div title="12">
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a> x 8
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/strong" target="separate">strong</a> x 2
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/orange" target="separate">orange</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/k" target="separate">k</a>
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="24">
        <span>
         <a href="http://en.wikipedia.org/wiki/cases" target="separate">cases x 25</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/epidemic" target="separate">epidemic x 21</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/for" target="separate">for x 120</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/time" target="separate">time x 22</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/{background" target="separate">{background x 29</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4395159" target="separate" title="remote PDF on server">PMC4395159</a>
       <a href="PMC4395159/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="SNPer: an R library for quantitative variant analysis on single nucleotide polymorphisms among influenza virus populations.">
         <i>titl: </i>SNPer: an R library for quanti ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Influenza virus (IFV) can evolve rapidly leading to genetic drifts and shifts resulting in human and animal influenza epidemics and pandemics. The genetic shift that gave rise to the 2009 influenza A/H1N1 pandemic originated from a triple gene reassortment of avian, swine and human IFVs. More minor genetic alterations in genetic drift can lead to influenza drug resistance such as the H274Y mutation associated with oseltamivir resistance. Hence, a rapid tool to detect IFV mutations and the potential emergence of new virulent strains can better prepare us for seasonal influenza outbreaks as well as potential pandemics. Furthermore, identification of specific mutations by closely examining single nucleotide polymorphisms (SNPs) in IFV sequences is essential to classify potential genetic markers associated with potentially dangerous IFV phenotypes. In this study, we developed a novel R library called &quot;SNPer&quot; to analyze quantitative variants in SNPs among IFV subpopulations. The computational SNPer program was applied to three different subpopulations of published IFV genomic information. SNPer queried SNPs data and grouped the SNPs into (1) universal SNPs, (2) likely common SNPs, and (3) unique SNPs. SNPer outperformed manual visualization in terms of time and labor. SNPer took only three seconds with no errors in SNP comparison events compared with 40 hours with errors using manual visualization. The SNPer tool can accelerate the capacity to capture new and potentially dangerous IFV strains to mitigate future influenza outbreaks.">
         <i>abst: </i>Influenza virus (IFV) can evol ...
        </span>
       </div>
      </td>
      <td>
       <div title="2">
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/interface" target="separate">interface</a>
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="13">
        <span>
         <a href="http://en.wikipedia.org/wiki/the" target="separate">the x 140</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/and" target="separate">and x 102</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/data" target="separate">data x 28</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/The" target="separate">The x 37</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/{background" target="separate">{background x 29</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4405370" target="separate" title="remote PDF on server">PMC4405370</a>
       <a href="PMC4405370/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Spatial structure, transmission modes and the evolution of viral exploitation strategies.">
         <i>titl: </i>Spatial structure, transmissio ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Spatial structure and local migration are predicted to promote the evolution of less aggressive host exploitation strategies in horizontally transmitted pathogens. Here we explore the effect of spatial structure on the evolution of pathogens that can use both horizontal and vertical routes of transmission. First, we analyse theoretically how vertical transmission can alter evolutionary trajectories and confirm that space can impede the spread of virulent pathogens. Second, we test this prediction using the latent phage λ which transmits horizontally and vertically in Escherichia coli populations. We show that the latent phage λ wins competition against the virulent mutant λcI857 in spatially structured epidemics, but loses when spatial structure is eroded. The vertical transmission of phage λ immunizes its local host pool against superinfection and prevents the spread of the virulent λcI857. This effect breaks down when mixing facilitates horizontal transmission to uninfected hosts. We thus confirm the importance of spatial structure for the evolutionary maintenance of prudent infection strategies in latent viruses.">
         <i>abst: </i>Spatial structure and local mi ...
        </span>
       </div>
      </td>
      <td>
       <div title="2">
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/strong" target="separate">strong</a>
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="28">
        <span>
         <a href="http://en.wikipedia.org/wiki/transmission" target="separate">transmission x 57</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/infected" target="separate">infected x 27</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/host" target="separate">host x 33</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/can" target="separate">can x 21</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/virulent" target="separate">virulent x 22</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4510129" target="separate" title="remote PDF on server">PMC4510129</a>
       <a href="PMC4510129/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Genetic Characterization of a Novel HIV-1 Circulating Recombinant Form (CRF74_01B) Identified among Intravenous Drug Users in Malaysia: Recombination History and Phylogenetic Linkage with Previously Defined Recombinant Lineages.">
         <i>titl: </i>Genetic Characterization of a ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="In many parts of Southeast Asia, the HIV-1 epidemic has been driven by the sharing of needles and equipment among intravenous drug users (IDUs). Over the last few decades, many studies have proven time and again that the diversity of HIV-1 epidemics can often be linked to the route of infection transmission. That said, the diversity and complexity of HIV-1 molecular epidemics in the region have been increasing at an alarming rate, due in part to the high tendency of the viral RNA to recombine. This scenario was exemplified by the discovery of numerous circulating recombinant forms (CRFs), especially in Thailand and Malaysia. In this study, we characterized a novel CRF designated CRF74_01B, which was identified in six epidemiologically unlinked IDUs in Kuala Lumpur, Malaysia. The near-full length genomes were composed of CRF01_AE and subtype B', with eight breakpoints dispersed in the gag-pol and nef regions. Remarkably, this CRF shared four and two recombination hotspots with the previously described CRF33_01B and the less prevalent CRF53_01B, respectively. Genealogy-based Bayesian phylogenetic analysis of CRF74_01B genomic regions showed that it is closely related to both CRF33_01B and CRF53_01B. This observation suggests that CRF74_01B was probably a direct descendent from specific lineages of CRF33_01B, CRF53_01B and subtype B' that could have emerged in the mid-1990s. Additionally, it illustrated the active recombination processes between prevalent HIV-1 subtypes and recombinants in Malaysia. In summary, we report a novel HIV-1 genotype designated CRF74_01B among IDUs in Kuala Lumpur, Malaysia. The characterization of the novel CRF74_01B is of considerable significance towards the understanding of the genetic diversity and population dynamics of HIV-1 circulating in the region.">
         <i>abst: </i>In many parts of Southeast Asi ...
        </span>
       </div>
      </td>
      <td>
       <div title="2">
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a> x 2
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="12">
        <span>
         <a href="http://en.wikipedia.org/wiki/HIV-" target="separate">HIV- x 37</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/and" target="separate">and x 106</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/{background" target="separate">{background x 29</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/from" target="separate">from x 24</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/with" target="separate">with x 26</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4520691" target="separate" title="remote PDF on server">PMC4520691</a>
       <a href="PMC4520691/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Forecasting Influenza Epidemics in Hong Kong.">
         <i>titl: </i>Forecasting Influenza Epidemic ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Recent advances in mathematical modeling and inference methodologies have enabled development of systems capable of forecasting seasonal influenza epidemics in temperate regions in real-time. However, in subtropical and tropical regions, influenza epidemics can occur throughout the year, making routine forecast of influenza more challenging. Here we develop and report forecast systems that are able to predict irregular non-seasonal influenza epidemics, using either the ensemble adjustment Kalman filter or a modified particle filter in conjunction with a susceptible-infected-recovered (SIR) model. We applied these model-filter systems to retrospectively forecast influenza epidemics in Hong Kong from January 1998 to December 2013, including the 2009 pandemic. The forecast systems were able to forecast both the peak timing and peak magnitude for 44 epidemics in 16 years caused by individual influenza strains (i.e., seasonal influenza A(H1N1), pandemic A(H1N1), A(H3N2), and B), as well as 19 aggregate epidemics caused by one or more of these influenza strains. Average forecast accuracies were 37% (for both peak timing and magnitude) at 1-3 week leads, and 51% (peak timing) and 50% (peak magnitude) at 0 lead. Forecast accuracy increased as the spread of a given forecast ensemble decreased; the forecast accuracy for peak timing (peak magnitude) increased up to 43% (45%) for H1N1, 93% (89%) for H3N2, and 53% (68%) for influenza B at 1-3 week leads. These findings suggest that accurate forecasts can be made at least 3 weeks in advance for subtropical and tropical regions.">
         <i>abst: </i>Recent advances in mathematica ...
        </span>
       </div>
      </td>
      <td>
       <div title="1">
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a>
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="39">
        <span>
         <a href="http://en.wikipedia.org/wiki/for" target="separate">for x 166</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/The" target="separate">The x 39</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/more" target="separate">more x 29</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/and" target="separate">and x 214</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/model" target="separate">model x 29</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4608293" target="separate" title="remote PDF on server">PMC4608293</a>
       <a href="PMC4608293/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Strategic information is everyone's business: perspectives from an international stakeholder meeting to enhance strategic information data along the HIV Cascade for people who inject drugs.">
         <i>titl: </i>Strategic information is every ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="People who inject drugs (PWID) are at increased HIV transmission risk because of unsafe injecting practices and a host of other individual, network, and structural factors. Thus, PWID have a great need for services within the Cascade of HIV prevention, diagnosis, care, and treatment (HIV Cascade). Yet the systems that monitor their progress through the Cascade are often lacking. Subsequently, fewer reliable data are available to guide programs targeting this key population (KP). Programmatic data, which are helpful in tracking PWID through the Cascade, also are limited because not all countries have harm reduction programming from which to estimate Cascade indicators. Also, due to stigma and the illegal nature of drug use, PWID may not disclose their drug use behavior or HIV status when accessing services. Consequently, PWID appear to have low HIV testing rates and, for those living with HIV, lower access to health services and lower viral suppression rates than do other KP groups. This commentary, based on outcomes from an international stakeholder meeting, identifies data gaps and proposes solutions to strengthen strategic information (SI), the systematic collection, analysis, and dissemination of information, to optimize HIV prevention, care, and treatment programming for PWID.">
         <i>abst: </i>People who inject drugs (PWID) ...
        </span>
       </div>
      </td>
      <td>
       <div title="3">
        <span>
         <a href="http://en.wikipedia.org/wiki/project" target="separate">project</a> x 2
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/strong" target="separate">strong</a>
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="10">
        <span>
         <a href="http://en.wikipedia.org/wiki/HIV" target="separate">HIV x 65</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/{background" target="separate">{background x 29</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/for" target="separate">for x 70</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/and" target="separate">and x 194</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/are" target="separate">are x 42</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4634195" target="separate" title="remote PDF on server">PMC4634195</a>
       <a href="PMC4634195/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="HIV Transmission Networks in the San Diego-Tijuana Border Region.">
         <i>titl: </i>HIV Transmission Networks in t ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="HIV sequence data can be used to reconstruct local transmission networks. Along international borders, like the San Diego-Tijuana region, understanding the dynamics of HIV transmission across reported risks, racial/ethnic groups, and geography can help direct effective prevention efforts on both sides of the border.We gathered sociodemographic, geographic, clinical, and viral sequence data from HIV infected individuals participating in ten studies in the San Diego-Tijuana border region. Phylogenetic and network analysis was performed to infer putative relationships between HIV sequences. Correlates of identified clusters were evaluated and spatiotemporal relationships were explored using Bayesian phylogeographic analysis.After quality filtering, 843 HIV sequences with associated demographic data and 263 background sequences from the region were analyzed, and 138 clusters were inferred (2-23 individuals). Overall, the rate of clustering did not differ by ethnicity, residence, or sex, but bisexuals were less likely to cluster than heterosexuals or men who have sex with men (p = 0.043), and individuals identifying as white (p ≤ 0.01) were more likely to cluster than other races. Clustering individuals were also 3.5 years younger than non-clustering individuals (p &lt; 0.001). Although the sampled San Diego and Tijuana epidemics were phylogenetically compartmentalized, five clusters contained individuals residing on both sides of the border.This study sampled ~ 7% of HIV infected individuals in the border region, and although the sampled networks on each side of the border were largely separate, there was evidence of persistent bidirectional cross-border transmissions that linked risk groups, thus highlighting the importance of the border region as a &quot;melting pot&quot; of risk groups.NIH, VA, and Pendleton Foundation.">
         <i>abst: </i>HIV sequence data can be used ...
        </span>
       </div>
      </td>
      <td>
       <div title="4">
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a> x 2
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/borders" target="separate">borders</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/project" target="separate">project</a>
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="27">
        <span>
         <a href="http://en.wikipedia.org/wiki/and" target="separate">and x 178</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/(.%)" target="separate">(.%) x 43</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/from" target="separate">from x 64</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/San" target="separate">San x 71</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/risk" target="separate">risk x 27</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4642928" target="separate" title="remote PDF on server">PMC4642928</a>
       <a href="PMC4642928/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Strain Interactions as a Mechanism for Dominant Strain Alternation and Incidence Oscillation in Infectious Diseases: Seasonal Influenza as a Case Study.">
         <i>titl: </i>Strain Interactions as a Mecha ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="BACKGROUND:Many human infectious diseases are caused by pathogens that have multiple strains and show oscillation in infection incidence and alternation of dominant strains which together are referred to as epidemic cycling. Understanding the underlying mechanisms of epidemic cycling is essential for forecasting outbreaks of epidemics and therefore important for public health planning. Current theoretical effort is mainly focused on the factors that are extrinsic to the pathogens themselves (&quot;extrinsic factors&quot;) such as environmental variation and seasonal change in human behaviours and susceptibility. Nevertheless, co-circulation of different strains of a pathogen was usually observed and thus strains interact with one another within concurrent infection and during sequential infection. The existence of these intrinsic factors is common and may be involved in the generation of epidemic cycling of multi-strain pathogens. METHODS AND FINDINGS:To explore the mechanisms that are intrinsic to the pathogens themselves (&quot;intrinsic factors&quot;) for epidemic cycling, we consider a multi-strain SIRS model including cross-immunity and infectivity enhancement and use seasonal influenza as an example to parameterize the model. The Kullback-Leibler information distance was calculated to measure the match between the model outputs and the typical features of seasonal flu (an outbreak duration of 11 weeks and an annual attack rate of 15%). Results show that interactions among strains can generate seasonal influenza with these characteristic features, provided that: the infectivity of a single strain within concurrent infection is enhanced 2-7 times that within a single infection; cross-immunity as a result of past infection is 0.5-0.8 and lasts 2-9 years; while other parameters are within their widely accepted ranges (such as a 2-3 day infectious period and the basic reproductive number of 1.8-3.0). Moreover, the observed alternation of the dominant strain among epidemics emerges naturally from the best fit model. Alternative modelling that also includes seasonal forcing in transmissibility shows that both external mechanisms (i.e. seasonal forcing) and the intrinsic mechanisms (i.e., strain interactions) are equally able to generate the observed time-series in seasonal flu. CONCLUSIONS:The intrinsic mechanism of strain interactions alone can generate the observed patterns of seasonal flu epidemics, but according to Kullback-Leibler information distance the importance of extrinsic mechanisms cannot be excluded. The intrinsic mechanism illustrated here to explain seasonal flu may also apply to other infectious diseases caused by polymorphic pathogens.">
         <i>abst: </i>BACKGROUND:Many human infectio ...
        </span>
       </div>
      </td>
      <td>
       <div title="14">
        <span>
         <a href="http://en.wikipedia.org/wiki/strong" target="separate">strong</a> x 9
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a> x 5
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="56">
        <span>
         <a href="http://en.wikipedia.org/wiki/The" target="separate">The x 57</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/transmission" target="separate">transmission x 33</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/during" target="separate">during x 27</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/this" target="separate">this x 23</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/mean" target="separate">mean x 28</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4666598" target="separate" title="remote PDF on server">PMC4666598</a>
       <a href="PMC4666598/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Socio-economic and Climate Factors Associated with Dengue Fever Spatial Heterogeneity: A Worked Example in New Caledonia.">
         <i>titl: </i>Socio-economic and Climate Fac ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="BACKGROUND/OBJECTIVES:Understanding the factors underlying the spatio-temporal distribution of infectious diseases provides useful information regarding their prevention and control. Dengue fever spatio-temporal patterns result from complex interactions between the virus, the host, and the vector. These interactions can be influenced by environmental conditions. Our objectives were to analyse dengue fever spatial distribution over New Caledonia during epidemic years, to identify some of the main underlying factors, and to predict the spatial evolution of dengue fever under changing climatic conditions, at the 2100 horizon. METHODS:We used principal component analysis and support vector machines to analyse and model the influence of climate and socio-economic variables on the mean spatial distribution of 24,272 dengue cases reported from 1995 to 2012 in thirty-three communes of New Caledonia. We then modelled and estimated the future evolution of dengue incidence rates using a regional downscaling of future climate projections. RESULTS:The spatial distribution of dengue fever cases is highly heterogeneous. The variables most associated with this observed heterogeneity are the mean temperature, the mean number of people per premise, and the mean percentage of unemployed people, a variable highly correlated with people's way of life. Rainfall does not seem to play an important role in the spatial distribution of dengue cases during epidemics. By the end of the 21st century, if temperature increases by approximately 3 °C, mean incidence rates during epidemics could double. CONCLUSION:In New Caledonia, a subtropical insular environment, both temperature and socio-economic conditions are influencing the spatial spread of dengue fever. Extension of this study to other countries worldwide should improve the knowledge about climate influence on dengue burden and about the complex interplay between different factors. This study presents a methodology that can be used as a step by step guide to model dengue spatial heterogeneity in other countries.">
         <i>abst: </i>BACKGROUND/OBJECTIVES:Understa ...
        </span>
       </div>
      </td>
      <td>
       <div title="10">
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a> x 4
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/strong" target="separate">strong</a> x 3
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/orange" target="separate">orange</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/borders" target="separate">borders</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/project" target="separate">project</a>
        </span>
       </div>
      </td>
      <td>
       <div title="74">
        <span>
         <a href="http://en.wikipedia.org/wiki/number" target="separate">number x 36</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/one" target="separate">one x 22</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/unemployed" target="separate">unemployed x 24</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/year" target="separate">year x 22</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/temperature" target="separate">temperature x 99</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4684369" target="separate" title="remote PDF on server">PMC4684369</a>
       <a href="PMC4684369/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Human Social Behavior and Demography Drive Patterns of Fine-Scale Dengue Transmission in Endemic Areas of Colombia.">
         <i>titl: </i>Human Social Behavior and Demo ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Dengue is known to transmit between humans and A. aegypti mosquitoes living in neighboring houses. Although transmission is thought to be highly heterogeneous in both space and time, little is known about the patterns and drivers of transmission in groups of houses in endemic settings. We carried out surveys of PCR positivity in children residing in 2-block patches of highly endemic cities of Colombia. We found high levels of heterogeneity in PCR positivity, varying from less than 30% in 8 of the 10 patches to 56 and 96%, with the latter patch containing 22 children simultaneously PCR positive (PCR22) for DEN2. We then used an agent-based model to assess the likely eco-epidemiological context of this observation. Our model, simulating daily dengue dynamics over a 20 year period in a single two block patch, suggests that the observed heterogeneity most likely derived from variation in the density of susceptible people. Two aspects of human adaptive behavior were critical to determining this density: external social relationships favoring viral introduction (by susceptible residents or infectious visitors) and immigration of households from non-endemic areas. External social relationships generating frequent viral introduction constituted a particularly strong constraint on susceptible densities, thereby limiting the potential for explosive outbreaks and dampening the impact of heightened vectorial capacity. Dengue transmission can be highly explosive locally, even in neighborhoods with significant immunity in the human population. Variation among neighborhoods in the density of local social networks and rural-to-urban migration is likely to produce significant fine-scale heterogeneity in dengue dynamics, constraining or amplifying the impacts of changes in mosquito populations and cross immunity between serotypes.">
         <i>abst: </i>Dengue is known to transmit be ...
        </span>
       </div>
      </td>
      <td>
       <div title="12">
        <span>
         <a href="http://en.wikipedia.org/wiki/strong" target="separate">strong</a> x 5
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a> x 4
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/sparks" target="separate">sparks</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/project" target="separate">project</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/bp" target="separate">bp</a>
        </span>
       </div>
      </td>
      <td>
       <div title="39">
        <span>
         <a href="http://en.wikipedia.org/wiki/was" target="separate">was x 31</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/model" target="separate">model x 25</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/The" target="separate">The x 34</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/each" target="separate">each x 37</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/this" target="separate">this x 22</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4704709" target="separate" title="remote PDF on server">PMC4704709</a>
       <a href="PMC4704709/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="The Genomic Signature of Population Reconnection Following Isolation: From Theory to HIV.">
         <i>titl: </i>The Genomic Signature of Popul ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Ease of worldwide travel provides increased opportunities for organisms not only to colonize new environments but also to encounter related but diverged populations. Such events of reconnection and secondary contact of previously isolated populations are widely observed at different time scales. For example, during the quaternary glaciation, sea water level fluctuations caused temporal isolation of populations, often to be followed by secondary contact. At shorter time scales, population isolation and reconnection of viruses are commonly observed, and such events are often associated with epidemics and pandemics. Here, using coalescent theory and simulations, we describe the temporal impact of population reconnection after isolation on nucleotide differences and the site frequency spectrum, as well as common summary statistics of DNA variation. We identify robust genomic signatures of population reconnection after isolation. We utilize our development to infer the recent evolutionary history of human immunodeficiency virus 1 (HIV-1) in Asia and South America, successfully retrieving the successive HIV subtype colonization events in these regions. Our analysis reveals that divergent HIV-1 subtype populations are currently admixing in these regions, suggesting that HIV-1 may be undergoing a process of homogenization, contrary to popular belief.">
         <i>abst: </i>Ease of worldwide travel provi ...
        </span>
       </div>
      </td>
      <td>
       <div title="92">
        <span>
         <a href="http://en.wikipedia.org/wiki/k" target="separate">k</a> x 64
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a> x 19
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/strong" target="separate">strong</a> x 7
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/project" target="separate">project</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/bp" target="separate">bp</a>
        </span>
       </div>
      </td>
      <td>
       <div title="47">
        <span>
         <a href="http://en.wikipedia.org/wiki/model" target="separate">model x 23</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/and" target="separate">and x 376</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/from" target="separate">from x 62</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/can" target="separate">can x 25</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/pairwise" target="separate">pairwise x 42</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4707738" target="separate" title="remote PDF on server">PMC4707738</a>
       <a href="PMC4707738/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Virus evolution and transmission in an ever more connected world.">
         <i>titl: </i>Virus evolution and transmissi ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="The frequency and global impact of infectious disease outbreaks, particularly those caused by emerging viruses, demonstrate the need for a better understanding of how spatial ecology and pathogen evolution jointly shape epidemic dynamics. Advances in computational techniques and the increasing availability of genetic and geospatial data are helping to address this problem, particularly when both information sources are combined. Here, we review research at the intersection of evolutionary biology, human geography and epidemiology that is working towards an integrated view of spatial incidence, host mobility and viral genetic diversity. We first discuss how empirical studies have combined viral spatial and genetic data, focusing particularly on the contribution of evolutionary analyses to epidemiology and disease control. Second, we explore the interplay between virus evolution and global dispersal in more depth for two pathogens: human influenza A virus and chikungunya virus. We discuss the opportunities for future research arising from new analyses of human transportation and trade networks, as well as the associated challenges in accessing and sharing relevant spatial and genetic data.">
         <i>abst: </i>The frequency and global impac ...
        </span>
       </div>
      </td>
      <td>
       <div title="2">
        <span>
         <a href="http://en.wikipedia.org/wiki/strong" target="separate">strong</a> x 2
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="28">
        <span>
         <a href="http://en.wikipedia.org/wiki/has" target="separate">has x 21</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/transmission" target="separate">transmission x 30</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/influenza" target="separate">influenza x 48</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/can" target="separate">can x 25</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/from" target="separate">from x 44</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4735166" target="separate" title="remote PDF on server">PMC4735166</a>
       <a href="PMC4735166/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Assessing herd immunity against rubella in Japan: a retrospective seroepidemiological analysis of age-dependent transmission dynamics.">
         <i>titl: </i>Assessing herd immunity agains ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="OBJECTIVE:We aimed to epidemiologically assess rubella herd immunity as a function of time, age and gender in Japan, with reference to the recent 2012-2014 rubella epidemic. DESIGN:This study is a retrospective seroepidemiological analysis. MAIN OUTCOME MEASURES:The susceptible fraction of the population was examined as a function of age and time. The age at infection was assessed using reported case data. RESULTS:Whereas 30 years ago rubella cases were seen only among children, the median (25-75th centiles) age of cases in 2014 was elevated to 32.0 (17.0-42.0) years among males and 27.0 (7.0-37.0) years among females. Susceptible pockets among male birth cohorts 1989-1993 and 1974-1978 were identified, with seropositive proportions of 70.0% and 68.0%, respectively. The majority of female age groups had greater seropositive proportions than the herd immunity threshold, with a minor susceptible pocket for those born from 1989 to 1993 (78.3% seropositive). The age-standardised seronegative proportion decreased to 18.3% (95% CI 16.8% to 19.8%) among males and 15.6% (95% CI 10.0% to 21.2%) among females in 2013, and the immune fraction was not sufficiently below the herd immunity threshold. While the number of live births born to susceptible mothers in 1983 was estimated at 171,876 across Japan, in 2013 it was reduced to 23,698. CONCLUSIONS:An elevated age at rubella virus infection and the presence of susceptible pockets among adults were observed in Japan. Although, overall, the absolute number of rubella cases has steadily declined in Japan, the elevated age of rubella cases, along with increased numbers of susceptible adults, contributed to the observation of as many as 45 congenital rubella syndrome (CRS) cases, which calls for supplementary vaccination among susceptible adults. Assessing herd immunity is considered essential for routinely monitoring the risk of future rubella epidemics and CRS cases.">
         <i>abst: </i>OBJECTIVE:We aimed to epidemio ...
        </span>
       </div>
      </td>
      <td>
       <div title="1">
        <span>
         <a href="http://en.wikipedia.org/wiki/nationwide" target="separate">nationwide</a>
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="22">
        <span>
         <a href="http://en.wikipedia.org/wiki/age" target="separate">age x 42</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/data" target="separate">data x 24</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/cases" target="separate">cases x 37</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/immunity" target="separate">immunity x 34</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/that" target="separate">that x 27</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4749335" target="separate" title="remote PDF on server">PMC4749335</a>
       <a href="PMC4749335/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Assessment of Overlap of Phylogenetic Transmission Clusters and Communities in Simple Sexual Contact Networks: Applications to HIV-1.">
         <i>titl: </i>Assessment of Overlap of Phylo ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="BACKGROUND:Transmission patterns of sexually-transmitted infections (STIs) could relate to the structure of the underlying sexual contact network, whose features are therefore of interest to clinicians. Conventionally, we represent sexual contacts in a population with a graph, that can reveal the existence of communities. Phylogenetic methods help infer the history of an epidemic and incidentally, may help detecting communities. In particular, phylogenetic analyses of HIV-1 epidemics among men who have sex with men (MSM) have revealed the existence of large transmission clusters, possibly resulting from within-community transmissions. Past studies have explored the association between contact networks and phylogenies, including transmission clusters, producing conflicting conclusions about whether network features significantly affect observed transmission history. As far as we know however, none of them thoroughly investigated the role of communities, defined with respect to the network graph, in the observation of clusters. METHODS:The present study investigates, through simulations, community detection from phylogenies. We simulate a large number of epidemics over both unweighted and weighted, undirected random interconnected-islands networks, with islands corresponding to communities. We use weighting to modulate distance between islands. We translate each epidemic into a phylogeny, that lets us partition our samples of infected subjects into transmission clusters, based on several common definitions from the literature. We measure similarity between subjects' island membership indices and transmission cluster membership indices with the adjusted Rand index. RESULTS AND CONCLUSION:Analyses reveal modest mean correspondence between communities in graphs and phylogenetic transmission clusters. We conclude that common methods often have limited success in detecting contact network communities from phylogenies. The rarely-fulfilled requirement that network communities correspond to clades in the phylogeny is their main drawback. Understanding the link between transmission clusters and communities in sexual contact networks could help inform policymaking to curb HIV incidence in MSMs.">
         <i>abst: </i>BACKGROUND:Transmission patter ...
        </span>
       </div>
      </td>
      <td>
       <div title="14">
        <span>
         <a href="http://en.wikipedia.org/wiki/path" target="separate">path</a> x 6
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/orange" target="separate">orange</a> x 2
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a> x 2
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/project" target="separate">project</a> x 2
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/strong" target="separate">strong</a>
        </span>
       </div>
      </td>
      <td>
       <div title="38">
        <span>
         <a href="http://en.wikipedia.org/wiki/islands" target="separate">islands x 34</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/transmission" target="separate">transmission x 73</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/from" target="separate">from x 45</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/which" target="separate">which x 26</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/this" target="separate">this x 23</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4764515" target="separate" title="remote PDF on server">PMC4764515</a>
       <a href="PMC4764515/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Developing a Time Series Predictive Model for Dengue in Zhongshan, China Based on Weather and Guangzhou Dengue Surveillance Data.">
         <i>titl: </i>Developing a Time Series Predi ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="BACKGROUND:Dengue is a re-emerging infectious disease of humans, rapidly growing from endemic areas to dengue-free regions due to favorable conditions. In recent decades, Guangzhou has again suffered from several big outbreaks of dengue; as have its neighboring cities. This study aims to examine the impact of dengue epidemics in Guangzhou, China, and to develop a predictive model for Zhongshan based on local weather conditions and Guangzhou dengue surveillance information. METHODS:We obtained weekly dengue case data from 1st January, 2005 to 31st December, 2014 for Guangzhou and Zhongshan city from the Chinese National Disease Surveillance Reporting System. Meteorological data was collected from the Zhongshan Weather Bureau and demographic data was collected from the Zhongshan Statistical Bureau. A negative binomial regression model with a log link function was used to analyze the relationship between weekly dengue cases in Guangzhou and Zhongshan, controlling for meteorological factors. Cross-correlation functions were applied to identify the time lags of the effect of each weather factor on weekly dengue cases. Models were validated using receiver operating characteristic (ROC) curves and k-fold cross-validation. RESULTS:Our results showed that weekly dengue cases in Zhongshan were significantly associated with dengue cases in Guangzhou after the treatment of a 5 weeks prior moving average (Relative Risk (RR) = 2.016, 95% Confidence Interval (CI): 1.845-2.203), controlling for weather factors including minimum temperature, relative humidity, and rainfall. ROC curve analysis indicated our forecasting model performed well at different prediction thresholds, with 0.969 area under the receiver operating characteristic curve (AUC) for a threshold of 3 cases per week, 0.957 AUC for a threshold of 2 cases per week, and 0.938 AUC for a threshold of 1 case per week. Models established during k-fold cross-validation also had considerable AUC (average 0.938-0.967). The sensitivity and specificity obtained from k-fold cross-validation was 78.83% and 92.48% respectively, with a forecasting threshold of 3 cases per week; 91.17% and 91.39%, with a threshold of 2 cases; and 85.16% and 87.25% with a threshold of 1 case. The out-of-sample prediction for the epidemics in 2014 also showed satisfactory performance. CONCLUSION:Our study findings suggest that the occurrence of dengue outbreaks in Guangzhou could impact dengue outbreaks in Zhongshan under suitable weather conditions. Future studies should focus on developing integrated early warning systems for dengue transmission including local weather and human movement.">
         <i>abst: </i>BACKGROUND:Dengue is a re-emer ...
        </span>
       </div>
      </td>
      <td>
       <div title="8">
        <span>
         <a href="http://en.wikipedia.org/wiki/k" target="separate">k</a> x 6
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/resist" target="separate">resist</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/nationwide" target="separate">nationwide</a>
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="31">
        <span>
         <a href="http://en.wikipedia.org/wiki/from" target="separate">from x 43</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/data" target="separate">data x 29</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/that" target="separate">that x 34</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/the" target="separate">the x 247</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/Guangzhou," target="separate">Guangzhou, x 22</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4767226" target="separate" title="remote PDF on server">PMC4767226</a>
       <a href="PMC4767226/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Provider-Initiated HIV Testing for Migrants in Spain: A Qualitative Study with Health Care Workers and Foreign-Born Sexual Minorities.">
         <i>titl: </i>Provider-Initiated HIV Testing ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="INTRODUCTION:Provider-initiated HIV testing (PITC) is increasingly adopted in Europe. The success of the approach at identifying new HIV cases relies on its effectiveness at testing individuals most at risk. However, its suitability to reach populations facing overlapping vulnerabilities is under researched. This qualitative study examined HIV testing experiences and perceptions amongst Latin-American migrant men who have sex with men and transgender females in Spain, as well as health professionals' experiences offering HIV tests to migrants in Barcelona and Madrid. METHODS:We conducted 32 in-depth interviews and 8 discussion groups with 38 Latin-American migrants and 21 health professionals. We imported verbatim transcripts and detailed field work notes into the qualitative software package Nvivo-10 and applied to all data a coding framework to examine systematically different HIV testing dimensions and modalities. The dimensions analysed were based on the World Health Organization &quot;5 Cs&quot; principles: Consent, Counselling, Connection to treatment, Correctness of results and Confidentiality. RESULTS:Health professionals reported that PITC was conceptually acceptable for them, although their perceived inability to adequately communicate HIV+ results and resulting bottle necks in the flow of care were recurrent concerns. Endorsement and adherence to the principles underpinning the rights-based response to HIV varied widely across health settings. The offer of an HIV test during routine consultations was generally appreciated by users as a way of avoiding the embarrassment of asking for it. Several participants deemed compulsory testing as acceptable on public health grounds. In spite of--and sometimes because of--partial endorsement of rights-based approaches, PITC was acceptable in a population with high levels of internalised stigma. CONCLUSION:PITC is a promising approach to reach sexual minority migrants who hold high levels of internalised stigma but explicit extra efforts are needed to safeguard the rights of the most vulnerable.">
         <i>abst: </i>INTRODUCTION:Provider-initiate ...
        </span>
       </div>
      </td>
      <td>
       <div title="2">
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/ministry" target="separate">ministry of health</a>
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="33">
        <span>
         <a href="http://en.wikipedia.org/wiki/and" target="separate">and x 312</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/for" target="separate">for x 105</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/they" target="separate">they x 36</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/offer" target="separate">offer x 22</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/test" target="separate">test x 46</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4806993" target="separate" title="remote PDF on server">PMC4806993</a>
       <a href="PMC4806993/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Cyclical Patterns of Hand, Foot and Mouth Disease Caused by Enterovirus A71 in Malaysia.">
         <i>titl: </i>Cyclical Patterns of Hand, Foo ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Enterovirus A71 (EV-A71) is an important emerging pathogen causing large epidemics of hand, foot and mouth disease (HFMD) in children. In Malaysia, since the first EV-A71 epidemic in 1997, recurrent cyclical epidemics have occurred every 2-3 years for reasons that remain unclear. We hypothesize that this cyclical pattern is due to changes in population immunity in children (measured as seroprevalence). Neutralizing antibody titers against EV-A71 were measured in 2,141 residual serum samples collected from children ≤12 years old between 1995 and 2012 to determine the seroprevalence of EV-A71. Reported national HFMD incidence was highest in children &lt;2 years, and decreased with age; in support of this, EV-A71 seroprevalence was significantly associated with age, indicating greater susceptibility in younger children. EV-A71 epidemics are also characterized by peaks of increased genetic diversity, often with genotype changes. Cross-sectional time series analysis was used to model the association between EV-A71 epidemic periods and EV-A71 seroprevalence adjusting for age and climatic variables (temperature, rainfall, rain days and ultraviolet radiance). A 10% increase in absolute monthly EV-A71 seroprevalence was associated with a 45% higher odds of an epidemic (adjusted odds ratio, aOR1.45; 95% CI 1.24-1.69; P&lt;0.001). Every 10% decrease in seroprevalence between preceding and current months was associated with a 16% higher odds of an epidemic (aOR = 1.16; CI 1.01-1.34 P&lt;0.034). In summary, the 2-3 year cyclical pattern of EV-A71 epidemics in Malaysia is mainly due to the fall of population immunity accompanying the accumulation of susceptible children between epidemics. This study will impact the future planning, timing and target populations for vaccine programs.">
         <i>abst: </i>Enterovirus A71 (EV-A71) is an ...
        </span>
       </div>
      </td>
      <td>
       <div title="7">
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a> x 4
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/strong" target="separate">strong</a> x 2
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/nationwide" target="separate">nationwide</a>
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="27">
        <span>
         <a href="http://en.wikipedia.org/wiki/population" target="separate">population x 23</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/epidemics" target="separate">epidemics x 43</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/was" target="separate">was x 52</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/HFMD" target="separate">HFMD x 42</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/were" target="separate">were x 54</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4817087" target="separate" title="remote PDF on server">PMC4817087</a>
       <a href="PMC4817087/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Absence of Active Hepatitis C Virus Infection in Human Immunodeficiency Virus Clinics in Zambia and Mozambique.">
         <i>titl: </i>Absence of Active Hepatitis C ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Few studies have evaluated the prevalence of replicating hepatitis C virus (HCV) infection in sub-Saharan Africa. Among 1812 individuals infected with human immunodeficiency virus, no patient in rural Mozambique and 4 patients in urban Zambia were positive for anti-HCV antibodies. Of these, none had confirmed HCV replication.">
         <i>abst: </i>Few studies have evaluated the ...
        </span>
       </div>
      </td>
      <td/>
      <td>
       <div title="7">
        <span>
         <a href="http://en.wikipedia.org/wiki/the" target="separate">the x 84</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/HCV" target="separate">HCV x 36</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/for" target="separate">for x 30</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/and" target="separate">and x 60</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/{background" target="separate">{background x 29</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4821482" target="separate" title="remote PDF on server">PMC4821482</a>
       <a href="PMC4821482/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Detecting Presymptomatic Infection Is Necessary to Forecast Major Epidemics in the Earliest Stages of Infectious Disease Outbreaks.">
         <i>titl: </i>Detecting Presymptomatic Infec ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="We assess how presymptomatic infection affects predictability of infectious disease epidemics. We focus on whether or not a major outbreak (i.e. an epidemic that will go on to infect a large number of individuals) can be predicted reliably soon after initial cases of disease have appeared within a population. For emerging epidemics, significant time and effort is spent recording symptomatic cases. Scientific attention has often focused on improving statistical methodologies to estimate disease transmission parameters from these data. Here we show that, even if symptomatic cases are recorded perfectly, and disease spread parameters are estimated exactly, it is impossible to estimate the probability of a major outbreak without ambiguity. Our results therefore provide an upper bound on the accuracy of forecasts of major outbreaks that are constructed using data on symptomatic cases alone. Accurate prediction of whether or not an epidemic will occur requires records of symptomatic individuals to be supplemented with data concerning the true infection status of apparently uninfected individuals. To forecast likely future behavior in the earliest stages of an emerging outbreak, it is therefore vital to develop and deploy accurate diagnostic tests that can determine whether asymptomatic individuals are actually uninfected, or instead are infected but just do not yet show detectable symptoms.">
         <i>abst: </i>We assess how presymptomatic i ...
        </span>
       </div>
      </td>
      <td>
       <div title="11">
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a> x 11
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="46">
        <span>
         <a href="http://en.wikipedia.org/wiki/which" target="separate">which x 40</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/SEIR" target="separate">SEIR x 23</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/model" target="separate">model x 28</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/The" target="separate">The x 34</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/individual" target="separate">individual x 22</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4836353" target="separate" title="remote PDF on server">PMC4836353</a>
       <a href="PMC4836353/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="The Proteolytic Activation of (H3N2) Influenza A Virus Hemagglutinin Is Facilitated by Different Type II Transmembrane Serine Proteases.">
         <i>titl: </i>The Proteolytic Activation of ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Cleavage of influenza virus hemagglutinin (HA) by host cell proteases is necessary for viral activation and infectivity. In humans and mice, members of the type II transmembrane protease family (TTSP), e.g., TMPRSS2, TMPRSS4, and TMPRSS11d (HAT), have been shown to cleave influenza virus HA for viral activation and infectivity in vitro Recently, we reported that inactivation of a single HA-activating protease gene,Tmprss2, in knockout mice inhibits the spread of H1N1 influenza viruses. However, after infection of Tmprss2 knockout mice with an H3N2 influenza virus, only a slight increase in survival was observed, and mice still lost body weight. In this study, we investigated an additional trypsin-like protease, TMPRSS4. Both TMPRSS2 and TMPRSS4 are expressed in the same cell types of the mouse lung. Deletion of Tmprss4 alone in knockout mice does not protect them from body weight loss and death upon infection with H3N2 influenza virus. In contrast,Tmprss2(-/-)Tmprss4(-/-)double-knockout mice showed a remarkably reduced virus spread and lung pathology, in addition to reduced body weight loss and mortality. Thus, our results identified TMPRSS4 as a second host cell protease that, in addition to TMPRSS2, is able to activate the HA of H3N2 influenza virus in vivoInfluenza epidemics and recurring pandemics are responsible for significant global morbidity and mortality. Due to high variability of the virus genome, resistance to available antiviral drugs is frequently observed, and new targets for treatment of influenza are needed. Host cell factors essential for processing of the virus hemagglutinin represent very suitable drug targets because the virus is dependent on these host factors for replication. We reported previously that Tmprss2-deficient mice are protected against H1N1 virus infections, but only marginal protection against H3N2 virus infections was observed. Here we show that deletion of two host protease genes,Tmprss2 and Tmprss4, strongly reduced viral spread as well as lung pathology and resulted in increased survival after H3N2 virus infection. Thus, TMPRSS4 represents another host cell factor that is involved in cleavage activation of H3N2 influenza viruses in vivo.">
         <i>abst: </i>Cleavage of influenza virus he ...
        </span>
       </div>
      </td>
      <td>
       <div title="5">
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a> x 5
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="33">
        <span>
         <a href="http://en.wikipedia.org/wiki/that" target="separate">that x 37</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/alveolar" target="separate">alveolar x 26</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/days" target="separate">days x 25</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/cleavage" target="separate">cleavage x 25</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/were" target="separate">were x 97</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4844159" target="separate" title="remote PDF on server">PMC4844159</a>
       <a href="PMC4844159/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Zika Virus Outbreak in Haiti in 2014: Molecular and Clinical Data.">
         <i>titl: </i>Zika Virus Outbreak in Haiti i ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="BACKGROUND:Zika virus (ZIKV), first isolated in Uganda in 1947, is currently spreading rapidly through South America and the Caribbean. In Brazil, infection has been linked with microcephaly and other serious complications, leading to declaration of a public health emergency of international concern; however, there currently are only limited data on the virus (and its possible sources and manifestations) in the Caribbean. METHODS:From May, 2014-February, 2015, in conjunction with studies of chikungunya (CHIKV) and dengue (DENV) virus infections, blood samples were collected from children in the Gressier/Leogane region of Haiti who presented to a school clinic with undifferentiated febrile illness. Samples were initially screened by RT-PCR for CHIKV and DENV, with samples negative in these assays further screened by viral culture. FINDINGS:Of 177 samples screened, three were positive for ZIKV, confirmed by viral sequencing; DENV-1 was also identified in culture from one of the three positive case patients. Patients were from two different schools and 3 different towns, with all three cases occurring within a single week, consistent with the occurrence of an outbreak in the region. Phylogenetic analysis of known full genome viral sequences demonstrated a close relationship with ZIKV from Brazil; additional analysis of the NS5 gene, for which more sequences are currently available, showed the Haitian strains clustering within a monophyletic clade distinct from Brazilian, Puerto Rican and Guatemalan sequences, with all part of a larger clade including isolates from Easter Island. Phylogeography also clarified that at least three major African sub-lineages exist, and confirmed that the South American epidemic is most likely to have originated from an initial ZIKV introduction from French Polynesia into Easter Island, and then to the remainder of the Americas. CONCLUSIONS:ZIKV epidemics in South America, as well as in Africa, show complex dissemination patterns. The virus appears to have been circulating in Haiti prior to the first reported cases in Brazil. Factors contributing to transmission and the possible linkage of this early Haitian outbreak with microcephaly remain to be determined.">
         <i>abst: </i>BACKGROUND:Zika virus (ZIKV), ...
        </span>
       </div>
      </td>
      <td>
       <div title="4">
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a> x 2
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/path" target="separate">path</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/bp" target="separate">bp</a>
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="17">
        <span>
         <a href="http://en.wikipedia.org/wiki/data" target="separate">data x 24</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/from" target="separate">from x 51</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/Zika" target="separate">Zika x 41</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/{background" target="separate">{background x 29</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/Biology" target="separate">Biology x 26</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4850254" target="separate" title="remote PDF on server">PMC4850254</a>
       <a href="PMC4850254/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Chemical Genomics Identifies the PERK-Mediated Unfolded Protein Stress Response as a Cellular Target for Influenza Virus Inhibition.">
         <i>titl: </i>Chemical Genomics Identifies t ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="UNLABELLED:Influenza A viruses generate annual epidemics and occasional pandemics of respiratory disease with important consequences for human health and the economy. Therefore, a large effort has been devoted to the development of new anti-influenza virus drugs directed to viral targets, as well as to the identification of cellular targets amenable to anti-influenza virus therapy. Here we have addressed the identification of such potential cellular targets by screening collections of drugs approved for human use. We reasoned that screening with a green fluorescent protein-based recombinant replicon system would identify cellular targets involved in virus transcription/replication and/or gene expression and hence address an early stage of virus infection. By using such a strategy, we identified Montelukast (MK) as an inhibitor of virus multiplication. MK inhibited virus gene expression but did not alter viral RNA synthesis in vitro or viral RNA accumulation in vivo The low selectivity index of MK prevented its use as an antiviral, but it was sufficient to identify a new cellular pathway suitable for anti-influenza virus intervention. By deep sequencing of RNA isolated from mock- and virus-infected human cells, treated with MK or left untreated, we showed that it stimulates the PERK-mediated unfolded protein stress response. The phosphorylation of PERK was partly inhibited in virus-infected cells but stimulated in MK-treated cells. Accordingly, pharmacological inhibition of PERK phosphorylation led to increased viral gene expression, while inhibition of PERK phosphatase reduced viral protein synthesis. These results suggest the PERK-mediated unfolded protein response as a potential cellular target to modulate influenza virus infection. IMPORTANCE:Influenza A viruses are responsible for annual epidemics and occasional pandemics with important consequences for human health and the economy. The unfolded protein response is a defense mechanism fired by cells when the demand of protein synthesis and folding is excessive, for instance, during an acute virus infection. In this report, we show that influenza virus downregulates the unfolded protein response mediated by the PERK sensor, while Montelukast, a drug used to treat asthma in humans, specifically stimulated this response and downregulated viral protein synthesis and multiplication. Accordingly, we show that PERK phosphorylation was reduced in virus-infected cells and increased in cells treated with Montelukast. Hence, our studies suggest that modulation of the PERK-mediated unfolded protein response is a target for influenza virus inhibition.">
         <i>abst: </i>UNLABELLED:Influenza A viruses ...
        </span>
       </div>
      </td>
      <td>
       <div title="20">
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a> x 17
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/strong" target="separate">strong</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/resist" target="separate">resist</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/project" target="separate">project</a>
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="37">
        <span>
         <a href="http://en.wikipedia.org/wiki/and" target="separate">and x 312</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/was" target="separate">was x 98</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/were" target="separate">were x 93</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/infected" target="separate">infected x 28</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/for" target="separate">for x 105</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4851397" target="separate" title="remote PDF on server">PMC4851397</a>
       <a href="PMC4851397/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Predicting Dengue Fever Outbreaks in French Guiana Using Climate Indicators.">
         <i>titl: </i>Predicting Dengue Fever Outbre ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="BACKGROUND:Dengue fever epidemic dynamics are driven by complex interactions between hosts, vectors and viruses. Associations between climate and dengue have been studied around the world, but the results have shown that the impact of the climate can vary widely from one study site to another. In French Guiana, climate-based models are not available to assist in developing an early warning system. This study aims to evaluate the potential of using oceanic and atmospheric conditions to help predict dengue fever outbreaks in French Guiana. METHODOLOGY/PRINCIPAL FINDINGS:Lagged correlations and composite analyses were performed to identify the climatic conditions that characterized a typical epidemic year and to define the best indices for predicting dengue fever outbreaks during the period 1991-2013. A logistic regression was then performed to build a forecast model. We demonstrate that a model based on summer Equatorial Pacific Ocean sea surface temperatures and Azores High sea-level pressure had predictive value and was able to predict 80% of the outbreaks while incorrectly predicting only 15% of the non-epidemic years. Predictions for 2014-2015 were consistent with the observed non-epidemic conditions, and an outbreak in early 2016 was predicted. CONCLUSIONS/SIGNIFICANCE:These findings indicate that outbreak resurgence can be modeled using a simple combination of climate indicators. This might be useful for anticipating public health actions to mitigate the effects of major outbreaks, particularly in areas where resources are limited and medical infrastructures are generally insufficient.">
         <i>abst: </i>BACKGROUND:Dengue fever epidem ...
        </span>
       </div>
      </td>
      <td>
       <div title="10">
        <span>
         <a href="http://en.wikipedia.org/wiki/strong" target="separate">strong</a> x 4
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/k" target="separate">k</a> x 4
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/resist" target="separate">resist</a>
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="23">
        <span>
         <a href="http://en.wikipedia.org/wiki/epidemic" target="separate">epidemic x 43</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/dengue" target="separate">dengue x 54</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/was" target="separate">was x 54</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/data" target="separate">data x 25</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/and" target="separate">and x 222</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4852896" target="separate" title="remote PDF on server">PMC4852896</a>
       <a href="PMC4852896/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Potential Impact of Sexual Transmission on Ebola Virus Epidemiology: Sierra Leone as a Case Study.">
         <i>titl: </i>Potential Impact of Sexual Tra ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="BACKGROUND:Sexual transmission of Ebola virus disease (EVD) 6 months after onset of symptoms has been recently documented, and Ebola virus RNA has been detected in semen of survivors up to 9 months after onset of symptoms. As countries affected by the 2013-2015 epidemic in West Africa, by far the largest to date, are declared free of Ebola virus disease (EVD), it remains unclear what threat is posed by rare sexual transmission events that could arise from survivors. METHODOLOGY/PRINCIPAL FINDINGS:We devised a compartmental mathematical model that includes sexual transmission from convalescent survivors: a SEICR (susceptible-exposed-infectious-convalescent-recovered) transmission model. We fitted the model to weekly incidence of EVD cases from the 2014-2015 epidemic in Sierra Leone. Sensitivity analyses and Monte Carlo simulations showed that a 0.1% per sex act transmission probability and a 3-month convalescent period (the two key unknown parameters of sexual transmission) create very few additional cases, but would extend the epidemic by 83 days [95% CI: 68-98 days] (p &lt; 0.0001) on average. Strikingly, a 6-month convalescent period extended the average epidemic by 540 days (95% CI: 508-572 days), doubling the current length, despite an insignificant rise in the number of new cases generated. CONCLUSIONS/SIGNIFICANCE:Our results show that reductions in the per sex act transmission probability via abstinence and condom use should reduce the number of sporadic sexual transmission events, but will not significantly reduce the epidemic size and may only minimally shorten the length of time the public health community must maintain response preparedness. While the number of infectious survivors is expected to greatly decline over the coming months, our results show that transmission events may still be expected for quite some time as each event results in a new potential cluster of non-sexual transmission. Precise measurement of the convalescent period is thus important for planning ongoing surveillance efforts.">
         <i>abst: </i>BACKGROUND:Sexual transmission ...
        </span>
       </div>
      </td>
      <td>
       <div title="11">
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a> x 8
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/k" target="separate">k</a> x 2
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/strong" target="separate">strong</a>
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="27">
        <span>
         <a href="http://en.wikipedia.org/wiki/from" target="separate">from x 60</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/model" target="separate">model x 26</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/that" target="separate">that x 54</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/The" target="separate">The x 35</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/period" target="separate">period x 35</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4854472" target="separate" title="remote PDF on server">PMC4854472</a>
       <a href="PMC4854472/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="On Temporal Patterns and Circulation of Influenza Virus Strains in Taiwan, 2008-2014: Implications of 2009 pH1N1 Pandemic.">
         <i>titl: </i>On Temporal Patterns and Circu ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="BACKGROUND:It has been observed that, historically, strains of pandemic influenza led to succeeding seasonal waves, albeit with decidedly different patterns. Recent studies suggest that the 2009 A(H1N1)pdm09 pandemic has had an impact on the circulation patterns of seasonal influenza strains in the post-pandemic years. In this work we aim to investigate this issue and also to compare the relative transmissibility of these waves of differing strains using Taiwan influenza surveillance data before, during and after the pandemic. METHODS:We make use of the Taiwan Center for Disease Control and Prevention influenza surveillance data on laboratory-confirmed subtyping of samples and a mathematical model to determine the waves of circulating (and co-circulating) H1, H3 and B virus strains in Taiwan during 2008-2014; or namely, short before, during and after the 2009 pandemic. We further pinpoint the turning points and relative transmissibility of each wave, in order to ascertain whether any temporal pattern exists. RESULTS/FINDINGS:For two consecutive years following the 2009 pandemic, A(H1N1)pdm09 circulated in Taiwan (as in most of Northern Hemisphere), sometimes co-circulating with AH3. From the evolution point of view, A(H1N1)pdm09 and AH3 were able to sustain their circulation patterns to the end of 2010. In fact, A(H1N1)pdm09 virus circulated in six separate waves in Taiwan between summer of 2009 and spring of 2014. Since 2009, a wave of A(H1N1)pmd09 occurred every fall/winter influenza season during our study period except 2011-2012 season, when mainly influenza strain B circulated. In comparing transmissibility, while the estimated per capita weekly growth rates for cumulative case numbers (and the reproduction number) seem to be lower for most of the influenza B waves (0.06~0.26; range of 95% CIs: 0.05~0.32) when compared to those of influenza A, the wave of influenza B from week 8 to week 38 of 2010 immediately following the fall/winter wave of 2009 A(H1N1) pdm09 was substantially higher at r = 0.89 (95% CI: 0.49, 1.28), in fact highest among all the waves detected in this study. Moreover, when AH3 or A(H1N1)pdm09 exhibit high incidence, reported cases of subtype B decreases and vice versa. Further modeling analysis indicated that during the study period, Taiwan nearly experienced at least one wave of influenza epidemic of some strain every summer except in 2012. DISCUSSION:Estimates of R for seasonal influenza are consistent with that of temperate and tropical-subtropical regions, while estimate of R for A(H1N1)pdm09 is comparatively less than countries in Europe and North America, but similar to that of tropical-subtropical regions. This offers indication of regional differences in transmissibility of influenza virus that exists only for pandemic influenza. Despite obvious limitations in the data used, this study, designed to qualitatively compare the temporal patterns and transmissibility of the waves of different strains, illustrates how influenza subtyping data can be utilized to explore the mechanism for various influenza strains to compete or to circulate, to possibly provide predictors of future trends in the evolution of influenza viruses of various subtypes, and perhaps more importantly, to be of use to future annual seasonal influenza vaccine design.">
         <i>abst: </i>BACKGROUND:It has been observe ...
        </span>
       </div>
      </td>
      <td>
       <div title="1">
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a>
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="25">
        <span>
         <a href="http://en.wikipedia.org/wiki/the" target="separate">the x 240</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/influenza" target="separate">influenza x 108</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/that" target="separate">that x 39</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/and" target="separate">and x 177</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/(.,.)" target="separate">(.,.) x 62</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4892648" target="separate" title="remote PDF on server">PMC4892648</a>
       <a href="PMC4892648/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Epidemiological and Virological Characterizations of the 2014 Dengue Outbreak in Guangzhou, China.">
         <i>titl: </i>Epidemiological and Virologica ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Dengue used to be recognized as an imported and sporadic disease in China. Since June 2014, an unexpected large dengue outbreak has attacked Guangzhou, China, resulting in more than 40,000 cases. Among the 1,942 laboratory-confirmed hospitalized dengue cases, 121 were diagnosed as severe dengue according to the 2009 WHO guideline, and 2 patients finally died. Laboratory diagnosis and virus isolation demonstrated that the majority (96%) cases were caused by dengue virus serotype 1 (DENV-1), and the others by serotype 2 (DENV-2). 14 DENV strains were isolated from the sera of acute-phase dengue patients during this outbreak, and the complete envelope (E) gene of 12 DENV-1 strains and two DENV-2 strains were determined using RT-PCR assay. Phylogenetic analysis based on the E gene revealed the DENV-1 strains isolated during the outbreak belonged to genotype I and V, respectively. These isolates formed three clades. DENV-2 isolates were assigned to the same clade belonging to genotype cosmopolitan. These strains isolated in 2014 were closely related to the isolates obtained from the same province, Guangdong, in 2013. No amino acid mutations known to increase virulence were identified throughout the E protein of isolates in 2014. These results indicate that dengue is turning into endemic in Guangdong, China, and extensive seroepidemiological investigation and mosquito control measures are critically needed in the future.">
         <i>abst: </i>Dengue used to be recognized a ...
        </span>
       </div>
      </td>
      <td>
       <div title="3">
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a> x 2
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/nationwide" target="separate">nationwide</a>
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="16">
        <span>
         <a href="http://en.wikipedia.org/wiki/DENV-" target="separate">DENV- x 26</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/{background" target="separate">{background x 29</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/were" target="separate">were x 45</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/with" target="separate">with x 23</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/the" target="separate">the x 164</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4909611" target="separate" title="remote PDF on server">PMC4909611</a>
       <a href="PMC4909611/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Climate Change and Aedes Vectors: 21st Century Projections for Dengue Transmission in Europe.">
         <i>titl: </i>Climate Change and Aedes Vecto ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Warming temperatures may increase the geographic spread of vector-borne diseases into temperate areas. Although a tropical mosquito-borne viral disease, a dengue outbreak occurred in Madeira, Portugal, in 2012; the first in Europe since 1920s. This outbreak emphasizes the potential for dengue re-emergence in Europe given changing climates. We present estimates of dengue epidemic potential using vectorial capacity (VC) based on historic and projected temperature (1901-2099). VC indicates the vectors' ability to spread disease among humans. We calculated temperature-dependent VC for Europe, highlighting 10 European cities and three non-European reference cities. Compared with the tropics, Europe shows pronounced seasonality and geographical heterogeneity. Although low, VC during summer is currently sufficient for dengue outbreaks in Southern Europe to commence-if sufficient vector populations (either Ae. aegypti and Ae. albopictus) were active and virus were introduced. Under various climate change scenarios, the seasonal peak and time window for dengue epidemic potential increases during the 21st century. Our study maps dengue epidemic potential in Europe and identifies seasonal time windows when major cities are most conducive for dengue transmission from 1901 to 2099. Our findings illustrate, that besides vector control, mitigating greenhouse gas emissions crucially reduces the future epidemic potential of dengue in Europe.">
         <i>abst: </i>Warming temperatures may incre ...
        </span>
       </div>
      </td>
      <td>
       <div title="13">
        <span>
         <a href="http://en.wikipedia.org/wiki/strong" target="separate">strong</a> x 6
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/project" target="separate">project</a> x 5
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a> x 2
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="46">
        <span>
         <a href="http://en.wikipedia.org/wiki/were" target="separate">were x 51</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/cities" target="separate">cities x 54</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/transmission" target="separate">transmission x 35</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/from" target="separate">from x 44</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/that" target="separate">that x 35</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4965063" target="separate" title="remote PDF on server">PMC4965063</a>
       <a href="PMC4965063/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Preventing Zika Virus Infection during Pregnancy Using a Seasonal Window of Opportunity for Conception.">
         <i>titl: </i>Preventing Zika Virus Infectio ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="It has come to light that Zika virus (ZIKV) infection during pregnancy can result in trans-placental transmission to the fetus along with fetal death, congenital microcephaly, and/or Central Nervous System (CNS) malformations. There are projected to be &gt;9,200,000 births annually in countries with ongoing ZIKV transmission. In response to the ZIKV threat, the World Health Organization (WHO) is strategically targeting prevention of infection in pregnant women and funding contraception in epidemic regions. I propose that the damaging effects of ZIKV can be reduced using a seasonal window of opportunity for conception that may minimize maternal exposure. Like other acute viral infections-including the related flavivirus, dengue virus (DENV)-the transmission of ZIKV is anticipated to be seasonal. By seasonally planning pregnancy, this aspect of pathogen ecology can be leveraged to align sensitive periods of gestation with the low-transmission season.">
         <i>abst: </i>It has come to light that Zika ...
        </span>
       </div>
      </td>
      <td>
       <div title="2">
        <span>
         <a href="http://en.wikipedia.org/wiki/strong" target="separate">strong</a> x 2
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="22">
        <span>
         <a href="http://en.wikipedia.org/wiki/transmission" target="separate">transmission x 43</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/that" target="separate">that x 26</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/from" target="separate">from x 30</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/Zika" target="separate">Zika x 30</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/data" target="separate">data x 29</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4978396" target="separate" title="remote PDF on server">PMC4978396</a>
       <a href="PMC4978396/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Zika Virus, a New Threat for Europe?">
         <i>titl: </i>Zika Virus, a New Threat for E ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="BACKGROUND:Since its emergence in 2007 in Micronesia and Polynesia, the arthropod-borne flavivirus Zika virus (ZIKV) has spread in the Americas and the Caribbean, following first detection in Brazil in May 2015. The risk of ZIKV emergence in Europe increases as imported cases are repeatedly reported. Together with chikungunya virus (CHIKV) and dengue virus (DENV), ZIKV is transmitted by Aedes mosquitoes. Any countries where these mosquitoes are present could be potential sites for future ZIKV outbreak. We assessed the vector competence of European Aedes mosquitoes (Aedes aegypti and Aedes albopictus) for the currently circulating Asian genotype of ZIKV. METHODOLOGY/PRINCIPAL FINDINGS:Two populations of Ae. aegypti from the island of Madeira (Funchal and Paul do Mar) and two populations of Ae. albopictus from France (Nice and Bar-sur-Loup) were challenged with an Asian genotype of ZIKV isolated from a patient in April 2014 in New Caledonia. Fully engorged mosquitoes were then maintained in insectary conditions (28°±1°C, 16h:8h light:dark cycle and 80% humidity). 16-24 mosquitoes from each population were examined at 3, 6, 9 and 14 days post-infection to estimate the infection rate, disseminated infection rate and transmission efficiency. Based on these experimental infections, we demonstrated that Ae. albopictus from France were not very susceptible to ZIKV. CONCLUSIONS/SIGNIFICANCE:In combination with the restricted distribution of European Ae. albopictus, our results on vector competence corroborate the low risk for ZIKV to expand into most parts of Europe with the possible exception of the warmest regions bordering the Mediterranean coastline.">
         <i>abst: </i>BACKGROUND:Since its emergence ...
        </span>
       </div>
      </td>
      <td>
       <div title="9">
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a> x 7
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/path" target="separate">path</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/strong" target="separate">strong</a>
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="14">
        <span>
         <a href="http://en.wikipedia.org/wiki/and" target="separate">and x 136</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/Ae." target="separate">Ae. x 52</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/ZIKV" target="separate">ZIKV x 35</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/{background" target="separate">{background x 29</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/the" target="separate">the x 114</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4989503" target="separate" title="remote PDF on server">PMC4989503</a>
       <a href="PMC4989503/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Epidemiology and virologic investigation of human enterovirus 71 infection in the Republic of Korea from 2007 to 2012: a nationwide cross-sectional study.">
         <i>titl: </i>Epidemiology and virologic inv ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Enterovirus (EV) 71 is the main pathogen associated with hand, foot and mouth disease (HFMD) or herpangina. Outbreaks of HFMD caused by EV71 infection are associated severe neurological disease and high mortality rates in children. Several sporadic cases of EV71 infection occurred in the Republic of Korea (ROK) in 2000, and EV71 infections were not reported thereafter until 2006. In this prospective study, we report the epidemic and virologic characteristics of the EV71 endemic from 2007 to 2012 in the Republic of Korea.We analyzed characteristics of the EV71 infection-associated epidemic from collected specimens and clinical information from 9987 patients with suspected EV infection from the National EV Surveillance System in ROK. To identify the EV71 subgenotype, the homology of viral protein 1 sequences obtained using reverse transcription polymerase chain reaction was compared with the sequences on other countries available from GenBank database.EV71 was detected in 585 (16.7 %) specimens (cerebrospinal fluid, stool or rectal swabs, throat swabs and blood) during study period and was most frequently observed during epidemic seasons in 2009-2012. Major manifestations due to EV71 infection were HFMD (62.2 %) and HFMD with severe neurological complications (28.4 %). Five deaths (0.9 %) due to EV71 infection occurred, with an increased mortality rate during the period after 2009. Most patients (476; 81.4 %) were less than 5 years of age. Analysis of the monthly distribution showed that there was an obvious seasonal pattern to the epidemics, with infections appearing from June to August. The major subgenotype of EV71 isolates circulating in ROK was the C4a strain, which has also appeared in China, Japan and Vietnam.This surveillance provided valuable data on the epidemic characteristics of EV71 infections in ROK during a 6-year period. Our findings provide data to assist during future outbreaks of EV71 and associated acute neurologic disease.">
         <i>abst: </i>Enterovirus (EV) 71 is the mai ...
        </span>
       </div>
      </td>
      <td>
       <div title="3">
        <span>
         <a href="http://en.wikipedia.org/wiki/nationwide" target="separate">nationwide</a> x 2
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a>
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="11">
        <span>
         <a href="http://en.wikipedia.org/wiki/was" target="separate">was x 33</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/with" target="separate">with x 40</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/{background" target="separate">{background x 29</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/enterovirus" target="separate">enterovirus x 35</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/from" target="separate">from x 36</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC5014447" target="separate" title="remote PDF on server">PMC5014447</a>
       <a href="PMC5014447/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="High Prevalence and Onward Transmission of Non-Pandemic HIV-1 Subtype B Clades in Northern and Northeastern Brazilian Regions.">
         <i>titl: </i>High Prevalence and Onward Tra ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="The Human immunodeficiency virus type-1 (HIV-1) epidemic in Brazil is mainly driven by the subtype B pandemic lineage (BPANDEMIC), while Caribbean non-pandemic subtype B clades (BCAR) seem to account for a very low fraction of HIV-infections in this country. The molecular characteristics of the HIV-1 subtype B strains disseminated in the Northern and Northeastern Brazilian regions, however, have not been explored so far. In this study, we estimate the prevalence of the HIV-1 BPANDEMIC and BCAR clades across different Brazilian regions and we reconstruct the spatiotemporal dynamics of dissemination of the major Brazilian BCAR clades. A total of 2,682 HIV-1 subtype B pol sequences collected from 21 different Brazilian states from the five country regions between 1998 and 2013 were analyzed. Maximum Likelihood phylogenetic analyses revealed that the BCAR strains reached 16 out 21 Brazilian states here analyzed. The BCAR clades comprise a low fraction (&lt;10%) of subtype B infections in most Brazilian states analyzed, with exception of Roraima (41%), Amazonas (14%) and Maranhão (14%). Bayesian phylogeographic analyses indicate that BCAR strains originally from the Hispaniola and Trinidad and Tobago were introduced at multiple times into different states from all Brazilian regions and a few of those strains, probably introduced into Roraima, Maranhão and São Paulo between the late 1970s and the early 1980s, established secondary outbreaks in the Brazilian population. These results support that the HIV-1 subtype B epidemics in some Brazilian states from the Northern and Northeastern regions display a unique molecular pattern characterized by the high prevalence of BCAR lineages, which probably reflects a strong epidemiological link with the HIV-1 epidemics in the Caribbean region.">
         <i>abst: </i>The Human immunodeficiency vir ...
        </span>
       </div>
      </td>
      <td>
       <div title="6">
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a> x 3
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/strong" target="separate">strong</a> x 3
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="25">
        <span>
         <a href="http://en.wikipedia.org/wiki/Caribbean" target="separate">Caribbean x 24</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/that" target="separate">that x 38</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/subtype" target="separate">subtype x 51</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/Biology" target="separate">Biology x 26</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/The" target="separate">The x 50</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC5027562" target="separate" title="remote PDF on server">PMC5027562</a>
       <a href="PMC5027562/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="A Direct Comparison of Two Densely Sampled HIV Epidemics: The UK and Switzerland.">
         <i>titl: </i>A Direct Comparison of Two Den ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Phylogenetic clustering approaches can elucidate HIV transmission dynamics. Comparisons across countries are essential for evaluating public health policies. Here, we used a standardised approach to compare the UK HIV Drug Resistance Database and the Swiss HIV Cohort Study while maintaining data-protection requirements. Clusters were identified in subtype A1, B and C pol phylogenies. We generated degree distributions for each risk group and compared distributions between countries using Kolmogorov-Smirnov (KS) tests, Degree Distribution Quantification and Comparison (DDQC) and bootstrapping. We used logistic regression to predict cluster membership based on country, sampling date, risk group, ethnicity and sex. We analysed &gt;8,000 Swiss and &gt;30,000 UK subtype B sequences. At 4.5% genetic distance, the UK was more clustered and MSM and heterosexual degree distributions differed significantly by the KS test. The KS test is sensitive to variation in network scale, and jackknifing the UK MSM dataset to the size of the Swiss dataset removed the difference. Only heterosexuals varied based on the DDQC, due to UK male heterosexuals who clustered exclusively with MSM. Their removal eliminated this difference. In conclusion, the UK and Swiss HIV epidemics have similar underlying dynamics and observed differences in clustering are mainly due to different population sizes.">
         <i>abst: </i>Phylogenetic clustering approa ...
        </span>
       </div>
      </td>
      <td>
       <div title="3">
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/strong" target="separate">strong</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/nationwide" target="separate">nationwide</a>
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="28">
        <span>
         <a href="http://en.wikipedia.org/wiki/the" target="separate">the x 361</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/were" target="separate">were x 55</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/between" target="separate">between x 38</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/two" target="separate">two x 23</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/The" target="separate">The x 51</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC5028067" target="separate" title="remote PDF on server">PMC5028067</a>
       <a href="PMC5028067/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Assessment of Local Mosquito Species Incriminates Aedes aegypti as the Potential Vector of Zika Virus in Australia.">
         <i>titl: </i>Assessment of Local Mosquito S ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="BACKGROUND:Within the last 10 years Zika virus (ZIKV) has caused unprecedented epidemics of human disease in the nations and territories of the western Pacific and South America, and continues to escalate in both endemic and non-endemic regions. We evaluated the vector competence of Australian mosquitoes for ZIKV to assess their potential role in virus transmission. METHODOLOGY/PRINCIPAL FINDINGS:Mosquitoes were exposed to infectious blood meals containing the prototype African ZIKV strain. After 14 days incubation at 28°C and high relative humidity, infection, dissemination and transmission rates were assessed. Infection in Culex annulirostris and Cx. sitiens could not be detected. 8% of Cx. quinquefasciatus were infected, but the virus did not disseminate in this species. Despite having infection rates &gt; 50%, Aedes notoscriptus and Ae. vigilax did not transmit ZIKV. In contrast, Ae. aegypti had infection and transmission rates of 57% and 27%, respectively. In susceptibility trials, the virus dose required to infect 50% (ID50) of Ae. aegypti was106.4 tissue culture infectious dose50 (TCID50)/mL. Additionally, a threshold viral load within the mosquito of at least 105.1 TCID50 equivalents/mL had to be reached before virus transmission occurred. CONCLUSIONS/SIGNIFICANCE:We confirmed Ae. aegypti to be the most likely mosquito vector of ZIKV in Australia, although the restricted distribution of this species will limit the receptive zone to northern Queensland where this species occurs. Importantly, the role in ZIKV transmission of Culex and other Aedes spp. tested will be negligible. Despite being the implicated vector, the relatively high ID50 and need for a high titer disseminated infection in Ae. aegypti suggest that high mosquito population densities will be required to facilitate epidemic ZIKV transmission among the currently immunologically naïve human population in Australia.">
         <i>abst: </i>BACKGROUND:Within the last 10 ...
        </span>
       </div>
      </td>
      <td>
       <div title="3">
        <span>
         <a href="http://en.wikipedia.org/wiki/research" target="separate">research and development</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/project" target="separate">project</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/bp" target="separate">bp</a>
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="25">
        <span>
         <a href="http://en.wikipedia.org/wiki/The" target="separate">The x 39</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/that" target="separate">that x 38</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/Australian" target="separate">Australian x 21</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/the" target="separate">the x 204</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/were" target="separate">were x 56</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC5028185" target="separate" title="remote PDF on server">PMC5028185</a>
       <a href="PMC5028185/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Isolation of Zika Virus Imported from Tonga into Australia.">
         <i>titl: </i>Isolation of Zika Virus Import ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="The globally emergent Zika virus (ZIKV) is a threat to Australia, given the number of imported cases from epidemic regions and the presence of competent mosquito vectors. We report the isolation of ZIKV from a female traveler who recently returned from Tonga to Brisbane, Queensland, Australia in 2016.A specific TaqMan real-time reverse transcriptase polymerase chain reaction assay (RT-PCR) assay was used to detect ZIKV in serum and urine samples. Conventional cell culture techniques and suckling mice were employed in an attempt to isolate ZIKV from serum and urine.A ZIKV isolate (TS17-2016) was recovered from the serum sample after one passage in suckling mouse brains and harvested 11 days post inoculation. Phylogenetic analysis of complete envelope (E) gene sequences demonstrated TS17-2016 shared 99.9% nucleotide identity with other contemporary sequences from Tonga 2016, Brazil 2015 and French Polynesia 2013 within the Asian lineage.This is the first known report of successful isolation of ZIKV from a human clinical sample in Australia and the first from a traveler from Tonga. This study highlights the potential difficulties in isolating ZIKV from acute clinical samples using conventional cell culture techniques, particularly in non-endemic countries like Australia where access to samples of sufficient viral load is limited. The successful isolation of TS17-2016 will be essential for continued investigations of ZIKV transmission and pathogenicity and will enable the advancement of new preventative control measures extremely relevant to the Australian and Pacific region.">
         <i>abst: </i>The globally emergent Zika vir ...
        </span>
       </div>
      </td>
      <td>
       <div title="1">
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a>
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="7">
        <span>
         <a href="http://en.wikipedia.org/wiki/from" target="separate">from x 30</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/{background" target="separate">{background x 29</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/and" target="separate">and x 90</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/margin" target="separate">margin x 21</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/the" target="separate">the x 56</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC5031435" target="separate" title="remote PDF on server">PMC5031435</a>
       <a href="PMC5031435/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="DNA Microarray Platform for Detection and Surveillance of Viruses Transmitted by Small Mammals and Arthropods.">
         <i>titl: </i>DNA Microarray Platform for De ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Viruses transmitted by small mammals and arthropods serve as global threats to humans. Most emergent and re-emergent viral agents are transmitted by these groups; therefore, the development of high-throughput screening methods for the detection and surveillance of such viruses is of great interest. In this study, we describe a DNA microarray platform that can be used for screening all viruses transmitted by small mammals and arthropods (SMAvirusChip) with nucleotide sequences that have been deposited in the GenBank. SMAvirusChip was designed with more than 15,000 oligonucleotide probes (60-mers), including viral and control probes. Two SMAvirusChip versions were designed: SMAvirusChip v1 contains 4209 viral probes for the detection of 409 viruses, while SMAvirusChip v2 contains 4943 probes for the detection of 416 viruses. SMAvirusChip was evaluated with 20 laboratory reference-strain viruses. These viruses could be specifically detected when alone in a sample or when artificially mixed within a single sample. The sensitivity of SMAvirusChip was evaluated using 10-fold serial dilutions of dengue virus (DENV). The results showed a detection limit as low as 2.6E3 RNA copies/mL. Additionally, the sensitivity was one log10 lower (2.6E2 RNA copies/mL) than quantitative real-time RT-PCR and sufficient to detect viral genomes in clinical samples. The detection of DENV in serum samples of DENV-infected patients (n = 6) and in a whole blood sample spiked with DENV confirmed the applicability of SMAvirusChip for the detection of viruses in clinical samples. In addition, in a pool of mosquito samples spiked with DENV, the virus was also detectable. SMAvirusChip was able to specifically detect viruses in cell cultures, serum samples, total blood samples and a pool of mosquitoes, confirming that cellular RNA/DNA did not interfere with the assay. Therefore, SMAvirusChip may represent an innovative surveillance method for the rapid identification of viruses transmitted by small mammals and arthropods.">
         <i>abst: </i>Viruses transmitted by small m ...
        </span>
       </div>
      </td>
      <td>
       <div title="9">
        <span>
         <a href="http://en.wikipedia.org/wiki/total" target="separate">total</a> x 9
        </span>
        <br/>
       </div>
      </td>
      <td>
       <div title="35">
        <span>
         <a href="http://en.wikipedia.org/wiki/from" target="separate">from x 41</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/signal" target="separate">signal x 23</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/virus" target="separate">virus x 88</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/viruses" target="separate">viruses x 90</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/transmitted" target="separate">transmitted x 32</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC5033240" target="separate" title="remote PDF on server">PMC5033240</a>
       <a href="PMC5033240/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Beyond Risk Compensation: Clusters of Antiretroviral Treatment (ART) Users in Sexual Networks Can Modify the Impact of ART on HIV Incidence.">
         <i>titl: </i>Beyond Risk Compensation: Clus ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="INTRODUCTION:Concerns about risk compensation-increased risk behaviours in response to a perception of reduced HIV transmission risk-after the initiation of ART have largely been dispelled in empirical studies, but other changes in sexual networking patterns may still modify the effects of ART on HIV incidence. METHODS:We developed an exploratory mathematical model of HIV transmission that incorporates the possibility of ART clusters, i.e. subsets of the sexual network in which the density of ART patients is much higher than in the rest of the network. Such clusters may emerge as a result of ART homophily-a tendency for ART patients to preferentially form and maintain relationships with other ART patients. We assessed whether ART clusters may affect the impact of ART on HIV incidence, and how the influence of this effect-modifying variable depends on contextual variables such as HIV prevalence, HIV serosorting, coverage of HIV testing and ART, and adherence to ART. RESULTS:ART homophily can modify the impact of ART on HIV incidence in both directions. In concentrated epidemics and generalized epidemics with moderate HIV prevalence (≈ 10%), ART clusters can enhance the impact of ART on HIV incidence, especially when adherence to ART is poor. In hyperendemic settings (≈ 35% HIV prevalence), ART clusters can reduce the impact of ART on HIV incidence when adherence to ART is high but few people living with HIV (PLWH) have been diagnosed. In all contexts, the effects of ART clusters on HIV epidemic dynamics are distinct from those of HIV serosorting. CONCLUSIONS:Depending on the programmatic and epidemiological context, ART clusters may enhance or reduce the impact of ART on HIV incidence, in contrast to serosorting, which always leads to a lower impact of ART on HIV incidence. ART homophily and the emergence of ART clusters should be measured empirically and incorporated into more refined models used to plan and evaluate ART programmes.">
         <i>abst: </i>INTRODUCTION:Concerns about ri ...
        </span>
       </div>
      </td>
      <td/>
      <td>
       <div title="33">
        <span>
         <a href="http://en.wikipedia.org/wiki/that" target="separate">that x 37</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/The" target="separate">The x 38</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/serosorting" target="separate">serosorting x 27</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/HIV" target="separate">HIV x 222</a>
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/risk" target="separate">risk x 25</a>
        </span>
       </div>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC6211219" target="separate" title="remote PDF on server">PMC6211219</a>
       <a href="PMC6211219/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span>no bibliographic metadata supplied</span>
       </div>
      </td>
      <td/>
      <td>
       <div title="988">
        <span>
         <a href="http://en.wikipedia.org/wiki/[]." target="separate">[]. x 22</a> x 4
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/Aedes" target="separate">Aedes x 58</a> x 3
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/transmission" target="separate">transmission x 24</a> x 3
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/virus" target="separate">virus x 35</a> x 3
        </span>
        <br/>
        <span>
         <a href="http://en.wikipedia.org/wiki/data" target="separate">data x 34</a> x 3
        </span>
       </div>
      </td>
     </tr>
    </tbody>
    <tfoot/>
   </table>
  </div>
 </body>
</html>
