<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/03/12/2020.03.11.987222.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.03.11.987222v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Discovery of a 382-nt deletion during the early evolution of SARS-CoV-2"/>
  <meta name="DC.Identifier" content="10.1101/2020.03.11.987222"/>
  <meta name="DC.Date" content="2020-03-12"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="To date, the SARS-CoV-2 genome has been considered genetically more stable than SARS-CoV or MERS-CoV. Here we report a 382-nt deletion covering almost the entire open reading frame 8 (ORF8) of SARS-CoV-2 obtained from eight hospitalized patients in Singapore. The deletion also removes the ORF8 transcription-regulatory sequence (TRS), which in turn enhances the downstream transcription of the N gene. We also found that viruses with the deletion have been circulating for at least four weeks. During the SARS-CoV outbreak in 2003, a number of genetic variants were observed in the human population [[1][1]], and similar variation has since been observed across SARS-related CoVs in humans and bats. Overwhelmingly these viruses had mutations or deletions in ORF8, that have been associated with reduced replicative fitness of the virus [[2][2]]. This is also consistent with the observation that towards the end of the outbreak sequences obtained from human SARS cases possessed an ORF8 deletion that may be associated with host adaptation [[1][1]]. We therefore hypothesise that the major deletion revealed in this study may lead to an attenuated phenotype of SARS-CoV-2. [1]: #ref-1 [2]: #ref-2"/>
  <meta name="DC.Contributor" content="Yvonne CF Su"/>
  <meta name="DC.Contributor" content="Danielle E Anderson"/>
  <meta name="DC.Contributor" content="Barnaby E Young"/>
  <meta name="DC.Contributor" content="Feng Zhu"/>
  <meta name="DC.Contributor" content="Martin Linster"/>
  <meta name="DC.Contributor" content="Shirin Kalimuddin"/>
  <meta name="DC.Contributor" content="Jenny GH Low"/>
  <meta name="DC.Contributor" content="Zhuang Yan"/>
  <meta name="DC.Contributor" content="Jayanthi Jayakumar"/>
  <meta name="DC.Contributor" content="Louisa Sun"/>
  <meta name="DC.Contributor" content="Gabriel Z Yan"/>
  <meta name="DC.Contributor" content="Ian H Mendenhall"/>
  <meta name="DC.Contributor" content="Yee-Sin Leo"/>
  <meta name="DC.Contributor" content="David Chien Lye"/>
  <meta name="DC.Contributor" content="Lin-Fa Wang"/>
  <meta name="DC.Contributor" content="Gavin JD Smith"/>
  <meta name="article:published_time" content="2020-03-12"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Discovery of a 382-nt deletion during the early evolution of SARS-CoV-2"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;To date, the SARS-CoV-2 genome has been considered genetically more stable than SARS-CoV or MERS-CoV. Here we report a 382-nt deletion covering almost the entire open reading frame 8 (ORF8) of SARS-CoV-2 obtained from eight hospitalized patients in Singapore. The deletion also removes the ORF8 transcription-regulatory sequence (TRS), which in turn enhances the downstream transcription of the N gene. We also found that viruses with the deletion have been circulating for at least four weeks. During the SARS-CoV outbreak in 2003, a number of genetic variants were observed in the human population, and similar variation has since been observed across SARS-related CoVs in humans and bats. Overwhelmingly these viruses had mutations or deletions in ORF8, that have been associated with reduced replicative fitness of the virus. This is also consistent with the observation that towards the end of the outbreak sequences obtained from human SARS cases possessed an ORF8 deletion that may be associated with host adaptation. We therefore hypothesise that the major deletion revealed in this study may lead to an attenuated phenotype of SARS-CoV-2.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.03.11.987222v1"/>
  <meta name="citation_id" content="2020.03.11.987222v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.11.987222v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.11.987222v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.11.987222v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987222.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.03.11.987222"/>
  <meta name="citation_num_pages" content="23"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.03.11.987222"/>
  <meta name="citation_author" content="Yvonne CF Su"/>
  <meta name="citation_author_institution" content="Programme in Emerging Infectious Diseases"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4349-9211"/>
  <meta name="citation_author" content="Danielle E Anderson"/>
  <meta name="citation_author_institution" content="Programme in Emerging Infectious Diseases"/>
  <meta name="citation_author" content="Barnaby E Young"/>
  <meta name="citation_author_institution" content="National Centre for Infectious Diseases"/>
  <meta name="citation_author_institution" content="Tan Tock Seng Hospital"/>
  <meta name="citation_author_institution" content="Lee Kong Chian School of Medicine"/>
  <meta name="citation_author" content="Feng Zhu"/>
  <meta name="citation_author_institution" content="Programme in Emerging Infectious Diseases"/>
  <meta name="citation_author" content="Martin Linster"/>
  <meta name="citation_author_institution" content="Programme in Emerging Infectious Diseases"/>
  <meta name="citation_author" content="Shirin Kalimuddin"/>
  <meta name="citation_author_institution" content="Programme in Emerging Infectious Diseases"/>
  <meta name="citation_author_institution" content="Singapore General Hospital"/>
  <meta name="citation_author" content="Jenny GH Low"/>
  <meta name="citation_author_institution" content="Programme in Emerging Infectious Diseases"/>
  <meta name="citation_author_institution" content="Singapore General Hospital"/>
  <meta name="citation_author" content="Zhuang Yan"/>
  <meta name="citation_author_institution" content="Programme in Emerging Infectious Diseases"/>
  <meta name="citation_author" content="Jayanthi Jayakumar"/>
  <meta name="citation_author_institution" content="Programme in Emerging Infectious Diseases"/>
  <meta name="citation_author" content="Louisa Sun"/>
  <meta name="citation_author_institution" content="Alexandra Hospital"/>
  <meta name="citation_author" content="Gabriel Z Yan"/>
  <meta name="citation_author_institution" content="National University Hospital"/>
  <meta name="citation_author" content="Ian H Mendenhall"/>
  <meta name="citation_author_institution" content="Programme in Emerging Infectious Diseases"/>
  <meta name="citation_author" content="Yee-Sin Leo"/>
  <meta name="citation_author_institution" content="National Centre for Infectious Diseases"/>
  <meta name="citation_author_institution" content="Tan Tock Seng Hospital"/>
  <meta name="citation_author_institution" content="Saw Swee Hock School of Public Health"/>
  <meta name="citation_author_institution" content="Yong Loo Lin School of Medicine"/>
  <meta name="citation_author_institution" content="Lee Kong Chian School of Medicine"/>
  <meta name="citation_author" content="David Chien Lye"/>
  <meta name="citation_author_institution" content="National Centre for Infectious Diseases"/>
  <meta name="citation_author_institution" content="Tan Tock Seng Hospital"/>
  <meta name="citation_author_institution" content="Yong Loo Lin School of Medicine"/>
  <meta name="citation_author_institution" content="Lee Kong Chian School of Medicine"/>
  <meta name="citation_author" content="Lin-Fa Wang"/>
  <meta name="citation_author_institution" content="Programme in Emerging Infectious Diseases"/>
  <meta name="citation_author_institution" content="SingHealth Duke-NUS Global Health Institute"/>
  <meta name="citation_author_email" content="linfa.wang@duke-nus.edu.sg"/>
  <meta name="citation_author_email" content="gavin.smith@duke-nus.edu.sg"/>
  <meta name="citation_author" content="Gavin JD Smith"/>
  <meta name="citation_author_institution" content="Programme in Emerging Infectious Diseases"/>
  <meta name="citation_author_institution" content="SingHealth Duke-NUS Global Health Institute"/>
  <meta name="citation_author_email" content="linfa.wang@duke-nus.edu.sg"/>
  <meta name="citation_author_email" content="gavin.smith@duke-nus.edu.sg"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5031-468X"/>
  <meta name="citation_reference" content="Chinese, S.M.E.C., Molecular evolution of the SARS coronavirus during the course of the SARS epidemic in China. Science, 2004. 303(5664): p. 1666–9."/>
  <meta name="citation_reference" content="Muth, D., et al., Attenuation of replication by a 29 nucleotide deletion in SARS-coronavirus acquired during the early stages of human-to-human transmission. Sci Rep, 2018. 8(1): p. 15177."/>
  <meta name="citation_reference" content="Lu, R., et al., Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding. The Lancet, 2020. 395(10224): p. 565–574."/>
  <meta name="citation_reference" content="World_Health_Organization. Coronavirus disease (COVID-2019) situation reports. 2020 [cited 2020 Mar 2]; Available from: https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports."/>
  <meta name="citation_reference" content="Chinese, S.M.E.C., Molecular evolution of the SARS coronavirus during the course of the SARS epidemic in China. Science (New York, N.Y.), 2004. 303(5664): p. 1666–1669."/>
  <meta name="citation_reference" content="Wang, L.-F., et al., Review of bats and SARS. Emerging infectious diseases, 2006. 12(12): p. 1834–1840."/>
  <meta name="citation_reference" content="Lau, S.K., et al., Severe Acute Respiratory Syndrome (SARS) Coronavirus ORF8 Protein Is Acquired from SARS-Related Coronavirus from Greater Horseshoe Bats through Recombination. J Virol, 2015. 89(20): p. 10532–47."/>
  <meta name="citation_reference" content="Xu, L., et al., Detection and characterization of diverse alpha- and betacoronaviruses from bats in China. Virol Sin, 2016. 31(1): p. 69–77."/>
  <meta name="citation_reference" content="Hussain, S., et al., Identification of novel subgenomic RNAs and noncanonical transcription initiation signals of severe acute respiratory syndrome coronavirus. J Virol, 2005. 79(9): p. 5288–95."/>
  <meta name="citation_reference" content="Su, Y.C.F., et al., Phylodynamics of H1N1/2009 influenza reveals the transition from host adaptation to immune-driven selection. Nature Communications, 2015. 6: p. 7952."/>
  <meta name="citation_reference" content="Tang, X., et al., On the origin and continuing evolution of SARS-CoV-2. National Science Review, 2020."/>
  <meta name="citation_reference" content="Zhao, Z., et al., Moderate mutation rate in the SARS coronavirus genome and its implications. BMC Evolutionary Biology, 2004. 4(1): p. 21."/>
  <meta name="citation_reference" content="Cotten, M., et al., Spread, circulation, and evolution of the Middle East respiratory syndrome coronavirus. mBio, 2014. 5(1): p. e01062–13."/>
  <meta name="citation_reference" content="Vijaykrishna, D., et al., The contrasting phylodynamics of human influenza B viruses. eLife, 2015. 4: p. e05055."/>
  <meta name="citation_reference" content="Virk, R.K., et al., Divergent evolutionary trajectories of influenza B viruses underlie their contemporaneous epidemic activity. Proceedings of the National Academy of Sciences, 2020. 117(1): p. 619–628."/>
  <meta name="citation_reference" content="Rambaut, A., et al., The genomic and epidemiological dynamics of human influenza A virus. Nature, 2008. 453(7195): p. 615–619."/>
  <meta name="citation_reference" content="Vijgen, L., et al., Complete Genomic Sequence of Human Coronavirus OC43: Molecular Clock Analysis Suggests a Relatively Recent Zoonotic Coronavirus Transmission Event. Journal of Virology, 2005. 79(3): p. 1595–1604."/>
  <meta name="citation_reference" content="Corman, V.M., et al., Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR. Eurosurveillance, 2020. 25(3): p. 2000045."/>
  <meta name="citation_reference" content="Bolger, A.M., M. Lohse, and B. Usadel, Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics, 2014. 30(15): p. 2114–2120."/>
  <meta name="citation_reference" content="Suchard, M.A., et al., Bayesian phylogenetic and phylodynamic data integration using BEAST 1.10. Virus Evol, 2018. 4(1): p. vey016."/>
  <meta name="citation_reference" content="Rambaut, A., et al., Posterior Summarization in Bayesian Phylogenetics Using Tracer 1.7. Syst Biol, 2018. 67(5): p. 901–904."/>
  <meta name="twitter:title" content="Discovery of a 382-nt deletion during the early evolution of SARS-CoV-2"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="To date, the SARS-CoV-2 genome has been considered genetically more stable than SARS-CoV or MERS-CoV. Here we report a 382-nt deletion covering almost the entire open reading frame 8 (ORF8) of SARS-CoV-2 obtained from eight hospitalized patients in Singapore. The deletion also removes the ORF8 transcription-regulatory sequence (TRS), which in turn enhances the downstream transcription of the N gene. We also found that viruses with the deletion have been circulating for at least four weeks. During the SARS-CoV outbreak in 2003, a number of genetic variants were observed in the human population [[1][1]], and similar variation has since been observed across SARS-related CoVs in humans and bats. Overwhelmingly these viruses had mutations or deletions in ORF8, that have been associated with reduced replicative fitness of the virus [[2][2]]. This is also consistent with the observation that towards the end of the outbreak sequences obtained from human SARS cases possessed an ORF8 deletion that may be associated with host adaptation [[1][1]]. We therefore hypothesise that the major deletion revealed in this study may lead to an attenuated phenotype of SARS-CoV-2. [1]: #ref-1 [2]: #ref-2"/>
  <meta name="og-title" property="og:title" content="Discovery of a 382-nt deletion during the early evolution of SARS-CoV-2"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.11.987222v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="To date, the SARS-CoV-2 genome has been considered genetically more stable than SARS-CoV or MERS-CoV. Here we report a 382-nt deletion covering almost the entire open reading frame 8 (ORF8) of SARS-CoV-2 obtained from eight hospitalized patients in Singapore. The deletion also removes the ORF8 transcription-regulatory sequence (TRS), which in turn enhances the downstream transcription of the N gene. We also found that viruses with the deletion have been circulating for at least four weeks. During the SARS-CoV outbreak in 2003, a number of genetic variants were observed in the human population [[1][1]], and similar variation has since been observed across SARS-related CoVs in humans and bats. Overwhelmingly these viruses had mutations or deletions in ORF8, that have been associated with reduced replicative fitness of the virus [[2][2]]. This is also consistent with the observation that towards the end of the outbreak sequences obtained from human SARS cases possessed an ORF8 deletion that may be associated with host adaptation [[1][1]]. We therefore hypothesise that the major deletion revealed in this study may lead to an attenuated phenotype of SARS-CoV-2. [1]: #ref-1 [2]: #ref-2"/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-03-12"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Discovery of a 382-nt deletion during the early evolution of SARS-CoV-2 | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1193407 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2>Abstract</h2>
    <p id="p-4">To date, the SARS-CoV-2 genome has been considered genetically more stable than SARS-CoV or MERS-CoV. Here we report a 382-nt deletion covering almost the entire open reading frame 8 (ORF8) of SARS-CoV-2 obtained from eight hospitalized patients in Singapore. The deletion also removes the ORF8 transcription-regulatory sequence (TRS), which in turn enhances the downstream transcription of the N gene. We also found that viruses with the deletion have been circulating for at least four weeks. During the SARS-CoV outbreak in 2003, a number of genetic variants were observed in the human population [
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>], and similar variation has since been observed across SARS-related CoVs in humans and bats. Overwhelmingly these viruses had mutations or deletions in ORF8, that have been associated with reduced replicative fitness of the virus [
     <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]. This is also consistent with the observation that towards the end of the outbreak sequences obtained from human SARS cases possessed an ORF8 deletion that may be associated with host adaptation [
     <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>]. We therefore hypothesise that the major deletion revealed in this study may lead to an attenuated phenotype of SARS-CoV-2.
    </p>
   </div>
   <p id="p-5">On 1 December 2019, a novel coronavirus emerged from Hubei province in China and infected people visiting the Huanan seafood market in Wuhan [
    <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>]. The virus demonstrated efficient human-to-human transmission within mainland China and subsequently spread across many countries. The virus was soon identified as 2019-nCoV, more recently designated SARS-CoV-2, while the disease is referred to as COVID-19. On 30 January 2020, the World Health Organization declared a Public Health Emergency of International Concern. As of 10 March 2020, the COVID-19 outbreak has led to 113,702 confirmed cases and 4,012 deaths globally [
    <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>]. Zoonotic viruses have crossed the species barrier from animals to humans and the success of an interspecies transmission is the ability of a novel virus to adapt, via genetic mutation events, to a new host and cause sustained transmission that can lead to a significant outbreak.
   </p>
   <p id="p-6">Nasopharyngeal swabs collected from hospitalized patients positive for SARS-CoV-2 in Singapore were subjected to next generation sequencing (NGS) analysis, with and without passaging in Vero-E6 cells. The NGS data revealed a 382-nt deletion towards the 3’ end of the viral genomes obtained from multiple patients (
    <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Suppl. Table 1</a>). To confirm this observation, specific PCR primers were designed flanking the deleted region and Sanger sequencing performed (
    <a id="xref-fig-4-1" class="xref-fig" href="#F4">Suppl. Fig. 1</a>). This verified the deletion at positions 27,848 to 28,229 of the SARS-CoV-2 genome. Interrogation of the NGS assemblies of these 382-nt deletion variants (referred to hereafter as Δ382) indicated that the virus populations were homogenous.
   </p>
   <div id="F4" class="fig pos-float type-figure odd">
    <div class="highwire-figure">
     <div class="fig-inline-img-wrapper">
      <div class="fig-inline-img">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987222/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Sanger sequences of BetaCoV/Singapore/12/2020 [[#x394]]382 mapped to Wuhan-Hu-1 showing the position of the 382-nt deletion in the SARS-CoV-2 genome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-479570057" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Sanger sequences of BetaCoV/Singapore/12/2020 [[#x394]]382 mapped to Wuhan-Hu-1 showing the position of the 382-nt deletion in the SARS-CoV-2 genome.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
        <span class="hw-responsive-img">
         <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987222/F4.medium.gif" width="440" height="173"/>
         <noscript>
          <img class="highwire-fragment fragment-image" alt="Supplementary Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987222/F4.medium.gif" width="440" height="173"/>
         </noscript>
        </span>
       </a>
      </div>
     </div>
     <ul class="highwire-figure-links inline">
      <li class="download-fig first">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987222/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
      </li>
      <li class="new-tab last">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987222/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
      </li>
     </ul>
    </div>
    <div class="fig-caption">
     <span class="fig-label">Supplementary Figure 1.</span>
     <p id="p-26" class="first-child">Sanger sequences of BetaCoV/Singapore/12/2020 Δ382 mapped to Wuhan-Hu-1 showing the position of the 382-nt deletion in the SARS-CoV-2 genome.</p>
     <div class="sb-div caption-clear"/>
    </div>
   </div>
   <div id="T2" class="table pos-float">
    <div class="table-inline table-callout-links">
     <div class="callout">
      <span>View this table:</span>
      <ul class="callout-links">
       <li class="view-inline first">
        <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1198532/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
       </li>
       <li class="view-popup last">
        <a href="/highwire/markup/1198532/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
       </li>
      </ul>
     </div>
    </div>
    <div class="table-caption">
     <span class="table-label">Supplementary Table 1.</span>
     <div class="sb-div caption-clear"/>
    </div>
   </div>
   <p id="p-7">Apart from the 382-nt deletion, the genome organization of the Δ382 viruses is identical to that of other SARS-CoV-2 (
    <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1A</a>). Closer examination of the deletion indicated that it spans an area of the ORF7b and ORF8 that includes the ORF8 transcriptional regulator sequence (TRS), eliminating ORF8 transcription (
    <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1B</a>). The ORF8 region has been identified as an evolutionary hotspot of SARSr-CoVs. For instance, sequences of human SARS-CoV TOR2 and LC2 exhibit 29-nt and 415-nt deletions, whereas those of bat SARS-CoV JTMC15 have discontinuous deletions in ORF7/8 (
    <a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1B</a>) [
    <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>-
    <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>]. The ORF8 region of SARS-CoVs has been shown to play a significant role in adaptation to the human host following interspecies transmission [
    <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>] and virus replicative efficiency [
    <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>]. Similarly the disruption of ORF8 region in Δ382 viruses may be a result of human adaptation after the emergence of SARS-CoV-2.
   </p>
   <div id="F1" class="fig pos-float type-figure odd">
    <div class="highwire-figure">
     <div class="fig-inline-img-wrapper">
      <div class="fig-inline-img">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987222/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Schematic comparison of SARS-2-CoV, SARS-CoV and SARSr-CoV genomes. A. Full genome sequences of SARS-2-CoV isolates Wuhan-Hu-1 (GenBank: MN908947), BetaCoV/Singapore/2/2020 (GISAID: 407987), and BetaCoV/Singapore/12/2020 [[#x394]]382 (GISAID: EPI_ISL_414378). B. Magnification of genomic region (dashed box in Figure 1A), numbers on horizontal axes indicate the nucleotide position relative to Wuhan-Hu-1, grey boxes indicate sequence coverage at a certain position, black horizontal lines represent a deletion at a certain position, open reading frames (ORFs) are indicated by colored arrows. A red-lined box indicates the 382-nt deletion. Transcription-regulatory sequences (TRSs) are indicated by yellow arrows." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-479570057" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Schematic comparison of SARS-2-CoV, SARS-CoV and SARSr-CoV genomes.&lt;;/span&gt;; &lt;;strong&gt;;A.&lt;;/strong&gt;; Full genome sequences of SARS-2-CoV isolates Wuhan-Hu-1 (GenBank: MN908947), BetaCoV/Singapore/2/2020 (GISAID: 407987), and BetaCoV/Singapore/12/2020 [[#x394]]382 (GISAID: EPI_ISL_414378). &lt;;strong&gt;;B.&lt;;/strong&gt;; Magnification of genomic region (dashed box in Figure 1A), numbers on horizontal axes indicate the nucleotide position relative to Wuhan-Hu-1, grey boxes indicate sequence coverage at a certain position, black horizontal lines represent a deletion at a certain position, open reading frames (ORFs) are indicated by colored arrows. A red-lined box indicates the 382-nt deletion. Transcription-regulatory sequences (TRSs) are indicated by yellow arrows.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
        <span class="hw-responsive-img">
         <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987222/F1.medium.gif" width="440" height="335"/>
         <noscript>
          <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987222/F1.medium.gif" width="440" height="335"/>
         </noscript>
        </span>
       </a>
      </div>
     </div>
     <ul class="highwire-figure-links inline">
      <li class="download-fig first">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987222/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
      </li>
      <li class="new-tab last">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987222/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
      </li>
     </ul>
    </div>
    <div class="fig-caption">
     <span class="fig-label">Figure 1.</span>
     <span class="caption-title">Schematic comparison of SARS-2-CoV, SARS-CoV and SARSr-CoV genomes.</span>
     <p id="p-8" class="first-child">
      <strong>A.</strong> Full genome sequences of SARS-2-CoV isolates Wuhan-Hu-1 (GenBank: 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=MN908947&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.11.987222.atom">MN908947</a>), BetaCoV/Singapore/2/2020 (GISAID: 407987), and BetaCoV/Singapore/12/2020 Δ382 (GISAID: EPI_ISL_414378). 
      <strong>B.</strong> Magnification of genomic region (dashed box in 
      <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A</a>), numbers on horizontal axes indicate the nucleotide position relative to Wuhan-Hu-1, grey boxes indicate sequence coverage at a certain position, black horizontal lines represent a deletion at a certain position, open reading frames (ORFs) are indicated by colored arrows. A red-lined box indicates the 382-nt deletion. Transcription-regulatory sequences (TRSs) are indicated by yellow arrows.
     </p>
     <div class="sb-div caption-clear"/>
    </div>
   </div>
   <p id="p-9">To investigate the possible effects of the TRS deletion, we calculated and compared the total number of transcripts per million (TPM) of each gene between wild-type (WT) and Δ382 viruses. For each gene, we counted unambiguous reads that uniquely mapped to the gene-specific transcripts that include the joint leader and TRS sequences [
    <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>]. We analysed three WT and two Δ382 sequences for which two independent NGS library preparations were available. Our results indicate differential patterns of TPM level between WT and Δ382 viruses (
    <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2A</a>). The Δ382 sequences displayed a greater level of TPMs in the ORF6, ORF7a and N genes compared to WT. More specifically, the Δ382 N gene, downstream of ORF8, showed a significant increase in the TPM level. To further determine the impact of the 382-nt deletion on the transcription of the N gene, we conducted qPCR to measure the relative abundance of the RNAs from 2 genes (E and N) located upstream and downstream of the deleted region, respectively. Results shown in 
    <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a> confirm the enhancement of N gene transcription in Δ382 viruses. This enhancement was consistently observed regardless of the sample type, whether directly from swab samples or passaged viruses.
   </p>
   <div id="F2" class="fig pos-float type-figure odd">
    <div class="highwire-figure">
     <div class="fig-inline-img-wrapper">
      <div class="fig-inline-img">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987222/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Comparison of transcription of the transcription of the N gene in wild-type (WT) versus [[#x394]]382 viruses. A. Abundance of mapped reads relative to transcriptional regulatory sequence (TRS) positions across the genome. 70 bp of leader sequence and 230 of each TRS-downstream sequence were merged in a splice-junction in the annotation. Transcripts per million reads (TPM) was calculated from reads mapped specifically to each leader-TRS region and a whisker and scatter plot was drawn for each gene. A Wilcoxon test was applied to the TPM for each gene of [[#x394]]382 to WT (*: p [[#x2264]] 0.05 **: p [[#x2264]] 0.01). B. Relative abundance of the RNAs of the E gene in comparison to the N gene of WT versus [[#x394]]382 SARS-CoV-2 (p = 0.002) using qPCR. Horizontal bars indicate the mean of each group." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-479570057" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Comparison of transcription of the transcription of the N gene in wild-type (WT) versus [[#x394]]382 viruses.&lt;;/span&gt;; &lt;;strong&gt;;A.&lt;;/strong&gt;; Abundance of mapped reads relative to transcriptional regulatory sequence (TRS) positions across the genome. 70 bp of leader sequence and 230 of each TRS-downstream sequence were merged in a splice-junction in the annotation. Transcripts per million reads (TPM) was calculated from reads mapped specifically to each leader-TRS region and a whisker and scatter plot was drawn for each gene. A Wilcoxon test was applied to the TPM for each gene of [[#x394]]382 to WT (*: p [[#x2264]] 0.05 **: p [[#x2264]] 0.01). &lt;;strong&gt;;B.&lt;;/strong&gt;; Relative abundance of the RNAs of the E gene in comparison to the N gene of WT versus [[#x394]]382 SARS-CoV-2 (p = 0.002) using qPCR. Horizontal bars indicate the mean of each group.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
        <span class="hw-responsive-img">
         <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987222/F2.medium.gif" width="440" height="129"/>
         <noscript>
          <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987222/F2.medium.gif" width="440" height="129"/>
         </noscript>
        </span>
       </a>
      </div>
     </div>
     <ul class="highwire-figure-links inline">
      <li class="download-fig first">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987222/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
      </li>
      <li class="new-tab last">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987222/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
      </li>
     </ul>
    </div>
    <div class="fig-caption">
     <span class="fig-label">Figure 2.</span>
     <span class="caption-title">Comparison of transcription of the transcription of the N gene in wild-type (WT) versus Δ382 viruses.</span>
     <p id="p-10" class="first-child">
      <strong>A.</strong> Abundance of mapped reads relative to transcriptional regulatory sequence (TRS) positions across the genome. 70 bp of leader sequence and 230 of each TRS-downstream sequence were merged in a splice-junction in the annotation. Transcripts per million reads (TPM) was calculated from reads mapped specifically to each leader-TRS region and a whisker and scatter plot was drawn for each gene. A Wilcoxon test was applied to the TPM for each gene of Δ382 to WT (*: p ≤ 0.05 **: p ≤ 0.01). 
      <strong>B.</strong> Relative abundance of the RNAs of the E gene in comparison to the N gene of WT versus Δ382 SARS-CoV-2 (p = 0.002) using qPCR. Horizontal bars indicate the mean of each group.
     </p>
     <div class="sb-div caption-clear"/>
    </div>
   </div>
   <p id="p-11">The global phylogeny of SARS-CoV-2 (
    <a id="xref-fig-5-1" class="xref-fig" href="#F5">Suppl. Fig. 2</a>) revealed a comb-like appearance in the phylogenetic tree and a general lack of phylogenetic resolution, reflecting the high similarity of the virus genomes. This tree topology is a typical characteristic of a novel interspecies transmission, wherein the viruses are infecting immunologically naïve populations with sustained human-to-human transmission, as previously shown for the early stages of the 2009 H1N1 pandemic [
    <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>]. Notably, we observe an emerging virus lineage (tentatively designated as “Lineage 1” in 
    <a id="xref-fig-5-2" class="xref-fig" href="#F5">Suppl. Fig. 2</a>) that consists of sequences from various newly affected countries with recent virus introductions. All viruses from this recent lineage possess an amino acid substitution at residue 250 (from lysine to serine) on the ORF8 gene [
    <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>], which is known to undergo strong positive selection during animal-to-human transmission [
    <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>].
   </p>
   <div id="F5" class="fig pos-float type-figure odd">
    <div class="highwire-figure">
     <div class="fig-inline-img-wrapper">
      <div class="fig-inline-img">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987222/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Maximum-likelihood tree of SARS-CoV-2 genomes reconstructed using RAxML with 1,000 bootstrap replicates. [[#x394]]382 viruses were exlcluded from this analysis as the L250S mutation is located in the ORF8 deletion." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-479570057" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Maximum-likelihood tree of SARS-CoV-2 genomes reconstructed using RAxML with 1,000 bootstrap replicates. [[#x394]]382 viruses were exlcluded from this analysis as the L250S mutation is located in the ORF8 deletion.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
        <span class="hw-responsive-img">
         <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987222/F5.medium.gif" width="287" height="440"/>
         <noscript>
          <img class="highwire-fragment fragment-image" alt="Supplementary Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987222/F5.medium.gif" width="287" height="440"/>
         </noscript>
        </span>
       </a>
      </div>
     </div>
     <ul class="highwire-figure-links inline">
      <li class="download-fig first">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987222/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
      </li>
      <li class="new-tab last">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987222/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
      </li>
     </ul>
    </div>
    <div class="fig-caption">
     <span class="fig-label">Supplementary Figure 2.</span>
     <p id="p-27" class="first-child">Maximum-likelihood tree of SARS-CoV-2 genomes reconstructed using RAxML with 1,000 bootstrap replicates. Δ382 viruses were exlcluded from this analysis as the L250S mutation is located in the ORF8 deletion.</p>
     <div class="sb-div caption-clear"/>
    </div>
   </div>
   <p id="p-12">To estimate the divergence times among lineages of SARS-CoV-2, we reconstructed a dated phylogeny based on 137 complete genomes, including data from this study. The estimated rate of nucleotide substitutions among SARS-CoV-2 viruses is approximately at 8.68×10
    <sup>−4</sup> substitutions per site per year (95% HPD: 5.44×10
    <sup>−4</sup> –1.22×10
    <sup>−3</sup>), which is moderately lower than SARS-CoV ([
    <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>]: 0.80–2.38×10
    <sup>−3</sup>) and MERS-CoV (with a mean rate [
    <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>] of: 1.12×10
    <sup>−3</sup> and 95% HPD: 58.76×10
    <sup>−4</sup> –1.37×10
    <sup>−3</sup>) as well as human seasonal influenza viruses (ranging 1.0–5.5×10
    <sup>−3</sup> depending on individual gene segments [
    <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>, 
    <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>-
    <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>]. In contrast, the estimated rate of SARS-CoV-2 viruses is greater than estimates for human coronaviruses by a magnitude order of 4×10
    <sup>−4</sup> [
    <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>].
   </p>
   <p id="p-13">The mean TMRCA estimates indicate the introduction of SARS-CoV-2 into humans (
    <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a>. node A) occurred in the middle of November 2019 (95% HPD: 2019.77–2019.94) (
    <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>), suggesting that the viruses were present in human hosts approximately one month before the outbreak was detected. A single amino-acid mutation was fixed on the ORF8 region of all full-genome Lineage 1 viruses (
    <a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3</a>, node B), with the TMRCA estimate of 22 December 2019 (95% HPD: 2019.9–2020.0). The Δ382 viruses form a monophyletic group within Lineage 1, and dating estimates indicate that they may have arisen in the human population in Singapore around 07 February 2020 (95% HPD: 2020.12–2020.06), consistent with the date of collection of the Δ382 positive samples (17–19 February 2020). The three Δ382 viruses shared a high level of nucleotide similarities (99.9%) with only 5 nucleotide differences between them. Although the dated phylogeny indicates that Δ382 viruses clustered together and are possibly derived from a single source, there is a general lack of statistical support in SARS-CoV-2 phylogenies due to its recent emergence, and evolutionary inferences should be made with caution.
   </p>
   <div id="T1" class="table pos-float">
    <div class="table-inline table-callout-links">
     <div class="callout">
      <span>View this table:</span>
      <ul class="callout-links">
       <li class="view-inline first">
        <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1198533/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
       </li>
       <li class="view-popup">
        <a href="/highwire/markup/1198533/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
       </li>
       <li class="download-ppt last">
        <a href="/highwire/powerpoint/1198533" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
       </li>
      </ul>
     </div>
    </div>
    <div class="table-caption">
     <span class="table-label">Table 1.</span>
     <p id="p-14" class="first-child">TMRCA estimates of major SARS-CoV-2 nodes.</p>
     <div class="sb-div caption-clear"/>
    </div>
   </div>
   <div id="F3" class="fig pos-float type-figure odd">
    <div class="highwire-figure">
     <div class="fig-inline-img-wrapper">
      <div class="fig-inline-img">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987222/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Temporal phylogeny of the complete genomes of SARS-CoV-2 viruses reconstructed using an uncorrelated lognormal relaxed clock model in BEAST. Isolate names in red indicate [[#x394]]382 viruses with a large deletion in the ORF7/8 region as described in this study. *Indicates virus genomes generated in this study. Colored branches denote different geographical locations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-479570057" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Temporal phylogeny of the complete genomes of SARS-CoV-2 viruses reconstructed using an uncorrelated lognormal relaxed clock model in BEAST. Isolate names in red indicate [[#x394]]382 viruses with a large deletion in the ORF7/8 region as described in this study. *Indicates virus genomes generated in this study. Colored branches denote different geographical locations.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
        <span class="hw-responsive-img">
         <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987222/F3.medium.gif" width="256" height="440"/>
         <noscript>
          <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987222/F3.medium.gif" width="256" height="440"/>
         </noscript>
        </span>
       </a>
      </div>
     </div>
     <ul class="highwire-figure-links inline">
      <li class="download-fig first">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987222/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
      </li>
      <li class="new-tab last">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987222/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
      </li>
     </ul>
    </div>
    <div class="fig-caption">
     <span class="fig-label">Figure 3.</span>
     <p id="p-15" class="first-child">Temporal phylogeny of the complete genomes of SARS-CoV-2 viruses reconstructed using an uncorrelated lognormal relaxed clock model in BEAST. Isolate names in red indicate Δ382 viruses with a large deletion in the ORF7/8 region as described in this study. *Indicates virus genomes generated in this study. Colored branches denote different geographical locations.</p>
     <div class="sb-div caption-clear"/>
    </div>
   </div>
   <p id="p-16">In this report we describe the first major evolutionary event of the SARS-CoV-2 virus following its emergence into the human population. Although the biological consequences of this deletion remain unknown, the alteration of the N gene transcription would suggest this may have an impact on virus phenotype. Recent work has indicated that ORF8 of SARS-CoV plays a functional role in virus replicative fitness and may be associated with attenuation during the early stages of human-to-human transmission [
    <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">2</a>]. Given the prevalence of a variety of deletions in the ORF8 of SARSr-CoVs, it is likely that we will see further deletion variants emerge with the sustained transmission of SARS-CoV-2 in humans. Future research should focus on the phenotypic consequences of Δ382 viruses in the transmission dynamics of the current epidemics and the immediate application of this genomic marker for molecular epidemiological investigation.
   </p>
   <div class="section" id="sec-1">
    <h2 class="">Online content</h2>
    <p id="p-17">Any methods, additional references, Nature Research reporting summaries, source data, extended data, supplementary information, acknowledgements, peer review information; details of author contributions and competing interests; and statements of data availability are available at [Article DOI].</p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Data availability</h2>
    <p id="p-18">The new sequences generated in this study have been deposited in GISAID database under accession numbers EPI_ISL_407987, EPI_ISL_407988, EPI_ISL_410535 to EPI_ISL_410537 for WT viruses and EPI_ISL_414378 to EPI_ISL_414380 for Δ382 viruses.</p>
   </div>
   <div class="section" id="sec-3">
    <h2 class="">Author contributions</h2>
    <p id="p-20">YCFS, DEA, LFW, GJDS designed and supervised research. BEY, SK, JGHL, LS, GZY, YSL, DCL collected and provided samples. DEA, ML, YZ conducted experiments. YCFS, ZF, JJ, IHM, GJDS performed analyses. YCFS, ZF, ML, LFW, GJDS wrote the paper. All authors reviewed and approved the manuscript.</p>
   </div>
   <div class="section" id="sec-4">
    <h2 class="">Methods</h2>
    <div class="section" id="sec-5">
     <h3>Ethics statement</h3>
     <p id="p-21">This study was undertaken as part of the national disease outbreak and the response and the protocols were approved by the ethics committee of the National Healthcare Group. Patient samples were collected under PROTECT (2012/00917), a multi-centred Prospective Study to Detect Novel Pathogens and Characterize Emerging Infections. Work undertaken at the Duke-NUS Medical School ABSL3 laboratory was approved by the Duke-NUS ABSL3 Biosafety Committee, National University of Singapore and Ministry of Health Singapore.</p>
    </div>
    <div class="section" id="sec-6">
     <h3>Virus culture, RNA extraction and sequencing</h3>
     <p id="p-22">Clinical samples from infected patients were collected from public hospitals in Singapore from January–February 2020. Material from clinical samples was used to inoculate Vero-E6 cells (ATCC
      <sup>®</sup>CRL-1586™). Total RNA was extracted using E.Z.N.A. Total RNA Kit I (Omega Bio-tek) according to manufacturer’s instructions and samples analysed by real-time quantitative reverse transcription-PCR (RT-qPCR) for the detection of SARS-CoV-2 as previously described [
      <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>]. Whole genome sequencing was performed using next-generation sequencing (NGS) methodology. The cDNA libraries were constructed using TruSeq RNA Library Prep Kit (Illumina) according to the manufacturer’s instructions and sequenced on an Illumina MiSeq System. Raw NGS reads were trimmed by Trimmomatic v0.39 [
      <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>] to remove adaptors and low-quality bases. Genome sequences were assembled and consensus sequences obtained using the BWA algorithm in UGENE v.33. To verify the presence of the deletion in the SARS-CoV-2 genome, we designed two specific PCR primers (F primer: 5’-TGTTAGAGGTACAACAGTACTTT-3’ and R primer: 5’-GGTAGTAGAAATACCATCTTGGA-3’) targeting the ORF7–8 regions. For samples with low Ct values, a hemi-nested PCR was then performed with primers (5’-TGTTTATAACACTTTGCTTCACA-3’) and (5’-GGTAGTAGAAATACCATCTTGGA-3’). The PCR mixture contained the cDNA, primers (10µM each), 10x Pfu reaction buffer (Promega), Pfu DNA polymerase (Promega) and dNTP mix (10mM, Thermo Scientific). The PCR reaction was carried out with the following conditions: 95°C for 2 min, 35 cycles at 95°C for 1 min, 52°C for 30 sec, 72°C for 1 min and a final extension at 72°C for 10 min in a thermal cycler (Applied Biosystems Veriti). Deletions in the PCR products were visualized by gel electrophoresis and confirmed by Sanger sequencing. Three full Δ382 genomes were generated and designated BetaCoV/Singapore/12/2020, BetaCoV/Singapore/13/2020 and BetaCoV/Singapore/14/2020 along with five WT SARS-CoV-2 genomes previously generated in our laboratories and deposited in GISAID (see 
      <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Suppl. Table 1</a>).
     </p>
    </div>
    <div class="section" id="sec-7">
     <h3>Genomic characterization</h3>
     <p id="p-23">To characterize and map the deletion regions of SARS-CoV-2 viruses, we compared with available SARS-CoV-2 and SARs-CoV related genomes from bat SARs-CoV-JTMC15 (accession number: 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=KY182964&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.11.987222.atom">KY182964</a>) and humans SARs-CoV-Tor 2 (accession number: 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=AY274119&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.11.987222.atom">AY274119</a>) and LC2 (accession number: 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=AY394999&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.11.987222.atom">AY394999</a>). The viral genome organizations of Wuhan-Hu-1 (accession number: 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=MN908947&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.11.987222.atom">MN908947</a>) and Singapore SARS-CoV-2 (Singapore/2 /2020: EPI_ISL_407987) comprise the following gene order and lengths: ORF1ab (open-reading frame) replicase (21291 nt), spike (S: 3822 nt), ORF3 (828 nt), envelope (E: 228 nt), membrane (M: 669 nt), ORF6 (186 nt), ORF7ab (498 nt), ORF8 (366 nt) and nucleocapsid (N: 1260 nt).
     </p>
     <p id="p-24">To understand possible effects of the TRS deletion on the ORF8 region, we analysed the total number of transcripts per million (TPM) of each gene between wildtype and Δ382 variants. For each gene, unambiguous reads which uniquely mapped to the specific joint leader and TRS sequences are counted. We analysed three patients of SARS-CoV-2 wild type and two patients of SARS-CoV-2 variants and independent NGS library preparations were performed twice for each sample. The Wuhan-Hu-1 was used as the reference sequence for genomic coordinates. Leader RNA sequence is situated at the 5’ UTR region of the genome and the core sequence consists of 26 nt: UCU
      <strong>CUAAACGAA</strong>CUUUAAAAUCUGUG. Each of the coding genes is preceded by a core transcription regulatory sequence (TRS) which is highly conserved (i.e. ACGAAC), resembling the leader core sequence. The leader RNA is joined to the TRS regions of different genes and the joining is required for driving subgenomic transcription. Coding sequence (CDS), leader and TRS sequences annotations were generated in Geneious and followed published SARS-CoV studies [
      <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>]. To characterize the differential levels of TRS for each gene, 70 nt leader sequence and 230 nt downstream of each TRS sequence were annotated individually to form a 300bp leader-TRS transcript for the splicing-aware aligners in R package rtracklayer (v 1.44). NGS raw fastq reads were then mapped to the reference genome by Geneious RNA-Seq mapper with the annotation of splice-junctions for the leader-TRS. For each gene, the TPM of leader-TRS was then calculated in Geneious by excluding ambiguous reads which may come from other TRS. The resulting TPM data was generated in Geneious and graphs plotted using ggplot 2 (v3.2.0) in R v3.6.1. Wilcoxon one-sided tests were performed in R to test the significant differences between WT and Δ382 samples. We also compared individual Ct values of E and N genes between WT and Δ382 viruses using qRT-PCR assays as previously described [
      <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>]. Samples used in this analysis included nasopharyngeal swabs from three WT patient samples and both nasopharyngeal swabs and passaged samples from two Δ382 patients (n=7). Significance between groups was assessed using a one-way analysis of variance (ANOVA) in PRISM v8.3.1.
     </p>
    </div>
    <div class="section" id="sec-8">
     <h3>Phylogenetic analyses</h3>
     <p id="p-25">All available genomes of SARS-CoV-2 with associated virus sampling dates were downloaded from the GISAID database. Genome sequence alignment was performed and preliminary maximum likelihood phylogenies of complete genome were reconstructed using RAxML with 1,000 bootstrap replicates in Geneious R9.0.3 software (Biomatters Ltd). Any sequence outliers were removed from subsequent analyses. To reconstruct a time-scaled phylogeny, an uncorrelated lognormal relaxed-clock model with an exponential growth coalescent prior and the HKY85+Γ substitution model was used in the program BEAST v1.10.4 [
      <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>] to simultaneously estimate phylogeny, divergence times and rates of nucleotide substitution. Four independent Markov Chain Monte Carlo (MCMC) runs of 100 million generations were performed and sampled every 10,000 generations. The runs were checked for convergence in Tracer v1.7 [
      <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>] and that effective sampling size (ESS) values of all parameters was &gt;;200. The resulting log and tree files were combined after removing appropriate burn-in values using LogCombiner [
      <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a>], and the maximum clade credibility (MCC) tree was subsequently generated using TreeAnnontator [
      <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">20</a>].
     </p>
    </div>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgements</h2>
    <p id="p-19">This study was supported by the Duke-NUS Signature Research Programme funded by the Ministry of Health, Singapore, the National Medical Research Council under its COVID-19 Research Fund (NMRC Project No. COVID19RF-001) and by National Research Foundation Singapore grant NRF2016NRFNSFC002-013 (Combating the Next SARS-or MERS-Like Emerging Infectious Disease Outbreak by Improving Active Surveillance). We thank Viji Vijayan, Benson Ng and Velraj Sivalingam of the Duke-NUS Medical School ABSL3 facility for logistics management and assistance. We thank all scientific staff who assisted with processing clinical samples, especially Velraj Sivalingam, Adrian Kang, Randy Foo, Wan Ni Chia and Akshamal Gamage. We thank Su Ting Tay, Ming Hui Lee and Angie Tan from the Duke-NUS Medical School Genome Biology Facility for expert and rapid technical assistance.</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987222v1.1" data-doi="10.1126/science.1092002">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chinese</span>, 
          <span class="cit-name-given-names">S.M.E.C.</span>
         </span>, 
         <span class="cit-article-title">Molecular evolution of the SARS coronavirus during the course of the SARS epidemic in China</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>, 
         <span class="cit-pub-date">2004</span>. 
         <span class="cit-vol">303</span>(
         <span class="cit-issue">5664</span>): p. 
         <span class="cit-fpage">1666</span>–
         <span class="cit-lpage">9</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1092002%26rft_id%253Dinfo%253Apmid%252F14752165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDMvNTY2NC8xNjY2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTIvMjAyMC4wMy4xMS45ODcyMjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987222v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Muth</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Attenuation of replication by a 29 nucleotide deletion in SARS-coronavirus acquired during the early stages of human-to-human transmission</span>. 
         <abbr class="cit-jnl-abbrev">Sci Rep</abbr>, 
         <span class="cit-pub-date">2018</span>. 
         <span class="cit-vol">8</span>(
         <span class="cit-issue">1</span>): p. 
         <span class="cit-fpage">15177</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D8%26rft.spage%253D15177%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987222v1.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</span>. 
         <abbr class="cit-jnl-abbrev">The Lancet</abbr>, 
         <span class="cit-pub-date">2020</span>. 
         <span class="cit-vol">395</span>(
         <span class="cit-issue">10224</span>): p. 
         <span class="cit-fpage">565</span>–
         <span class="cit-lpage">574</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BLancet%26rft.volume%253D395%26rft.spage%253D565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.03.11.987222v1.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">World_Health_Organization</span>. 
         <span class="cit-source">Coronavirus disease (COVID-2019) situation reports</span>. 
         <span class="cit-pub-date">2020</span> [cited 2020 Mar 2]; Available from: 
         <a href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports">https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987222v1.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chinese</span>, 
          <span class="cit-name-given-names">S.M.E.C.</span>
         </span>, 
         <span class="cit-article-title">Molecular evolution of the SARS coronavirus during the course of the SARS epidemic in China</span>. 
         <abbr class="cit-jnl-abbrev">Science (New York, N.Y</abbr>.), 
         <span class="cit-pub-date">2004</span>. 
         <span class="cit-vol">303</span>(
         <span class="cit-issue">5664</span>): p. 
         <span class="cit-fpage">1666</span>–
         <span class="cit-lpage">1669</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%2B%2528New%2BYork%252C%2BN.Y%26rft.volume%253D303%26rft.spage%253D1666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.11.987222v1.6" data-doi="10.3201/eid1212.060401">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">L.-F.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Review of bats and SARS</span>. 
         <abbr class="cit-jnl-abbrev">Emerging infectious diseases</abbr>, 
         <span class="cit-pub-date">2006</span>. 
         <span class="cit-vol">12</span>(
         <span class="cit-issue">12</span>): p. 
         <span class="cit-fpage">1834</span>–
         <span class="cit-lpage">1840</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEmerging%2Binfectious%2Bdiseases%26rft.stitle%253DEmerg%2BInfect%2BDis%26rft.aulast%253DWang%26rft.auinit1%253DL.%26rft.volume%253D12%26rft.issue%253D12%26rft.spage%253D1834%26rft.epage%253D1840%26rft.atitle%253DReview%2Bof%2Bbats%2Band%2BSARS.%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid1212.060401%26rft_id%253Dinfo%253Apmid%252F17326933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.3201/eid1212.060401&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17326933&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.11.987222.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000242301900005&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987222v1.7" data-doi="10.1128/JVI.01048-15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lau</span>, 
          <span class="cit-name-given-names">S.K.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Severe Acute Respiratory Syndrome (SARS) Coronavirus ORF8 Protein Is Acquired from SARS-Related Coronavirus from Greater Horseshoe Bats through Recombination</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>, 
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-vol">89</span>(
         <span class="cit-issue">20</span>): p. 
         <span class="cit-fpage">10532</span>–
         <span class="cit-lpage">47</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01048-15%26rft_id%253Dinfo%253Apmid%252F26269185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4OS8yMC8xMDUzMiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzEyLzIwMjAuMDMuMTEuOTg3MjIyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987222v1.8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Detection and characterization of diverse alpha- and betacoronaviruses from bats in China</span>. 
         <abbr class="cit-jnl-abbrev">Virol Sin</abbr>, 
         <span class="cit-pub-date">2016</span>. 
         <span class="cit-vol">31</span>(
         <span class="cit-issue">1</span>): p. 
         <span class="cit-fpage">69</span>–
         <span class="cit-lpage">77</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVirol%2BSin%26rft.volume%253D31%26rft.spage%253D69%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987222v1.9" data-doi="10.1128/JVI.79.9.5288-5295.2005">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hussain</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Identification of novel subgenomic RNAs and noncanonical transcription initiation signals of severe acute respiratory syndrome coronavirus</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>, 
         <span class="cit-pub-date">2005</span>. 
         <span class="cit-vol">79</span>(
         <span class="cit-issue">9</span>): p. 
         <span class="cit-fpage">5288</span>–
         <span class="cit-lpage">95</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DHussain%26rft.auinit1%253DS.%26rft.volume%253D79%26rft.issue%253D9%26rft.spage%253D5288%26rft.epage%253D5295%26rft.atitle%253DIdentification%2Bof%2BNovel%2BSubgenomic%2BRNAs%2Band%2BNoncanonical%2BTranscription%2BInitiation%2BSignals%2Bof%2BSevere%2BAcute%2BRespiratory%2BSyndrome%2BCoronavirus%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.79.9.5288-5295.2005%26rft_id%253Dinfo%253Apmid%252F15827143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijc5LzkvNTI4OCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzEyLzIwMjAuMDMuMTEuOTg3MjIyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987222v1.10">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Su</span>, 
          <span class="cit-name-given-names">Y.C.F.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Phylodynamics of H1N1/2009 influenza reveals the transition from host adaptation to immune-driven selection</span>. 
         <abbr class="cit-jnl-abbrev">Nature Communications</abbr>, 
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-vol">6</span>: p. 
         <span class="cit-fpage">7952</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D6%26rft.spage%253D7952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987222v1.11">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Tang</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">On the origin and continuing evolution of SARS-CoV-2</span>. 
         <abbr class="cit-jnl-abbrev">National Science Review</abbr>, 
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987222v1.12">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Moderate mutation rate in the SARS coronavirus genome and its implications</span>. 
         <abbr class="cit-jnl-abbrev">BMC Evolutionary Biology</abbr>, 
         <span class="cit-pub-date">2004</span>. 
         <span class="cit-vol">4</span>(
         <span class="cit-issue">1</span>): p. 
         <span class="cit-fpage">21</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2BEvolutionary%2BBiology%26rft.volume%253D4%26rft.spage%253D21%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987222v1.13" data-doi="10.1128/mBio.01062-13">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Cotten</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">
          <em>Spread, circulation</em>, and evolution of the Middle East respiratory syndrome coronavirus
         </span>. 
         <abbr class="cit-jnl-abbrev">mBio</abbr>, 
         <span class="cit-pub-date">2014</span>. 
         <span class="cit-vol">5</span>(
         <span class="cit-issue">1</span>): p. 
         <span class="cit-fpage">e01062</span>–
         <span class="cit-lpage">13</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DmBio%26rft.volume%253D5%26rft.spage%253De01062%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.01062-13%26rft_id%253Dinfo%253Apmid%252F24549846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1128/mBio.01062-13&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24549846&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.11.987222.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987222v1.14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Vijaykrishna</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">The contrasting phylodynamics of human influenza B viruses</span>. 
         <abbr class="cit-jnl-abbrev">eLife</abbr>, 
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-vol">4</span>: p. 
         <span class="cit-fpage">e05055</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DeLife%26rft.stitle%253Delife%26rft.volume%253D4%26rft.issue%253D0%26rft.spage%253De05055%26rft.epage%253De05055%26rft.atitle%253DThe%2Bcontrasting%2Bphylodynamics%2Bof%2Bhuman%2Binfluenza%2BB%2Bviruses%26rft_id%253Dinfo%253Apmid%252F25594904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=25594904&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.11.987222.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.11.987222v1.15" data-doi="10.1073/pnas.1916585116">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Virk</span>, 
          <span class="cit-name-given-names">R.K.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Divergent evolutionary trajectories of influenza B viruses underlie their contemporaneous epidemic activity</span>. 
         <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr>, 
         <span class="cit-pub-date">2020</span>. 
         <span class="cit-vol">117</span>(
         <span class="cit-issue">1</span>): p. 
         <span class="cit-fpage">619</span>–
         <span class="cit-lpage">628</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1916585116%26rft_id%253Dinfo%253Apmid%252F31843889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMTcvMS82MTkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8xMi8yMDIwLjAzLjExLjk4NzIyMi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987222v1.16" data-doi="10.1038/nature06945">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rambaut</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">The genomic and epidemiological dynamics of human influenza A virus</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>, 
         <span class="cit-pub-date">2008</span>. 
         <span class="cit-vol">453</span>(
         <span class="cit-issue">7195</span>): p. 
         <span class="cit-fpage">615</span>–
         <span class="cit-lpage">619</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRambaut%26rft.auinit1%253DA.%26rft.volume%253D453%26rft.issue%253D7195%26rft.spage%253D615%26rft.epage%253D619%26rft.atitle%253DThe%2Bgenomic%2Band%2Bepidemiological%2Bdynamics%2Bof%2Bhuman%2Binfluenza%2BA%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06945%26rft_id%253Dinfo%253Apmid%252F18418375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature06945&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18418375&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.11.987222.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000256185200035&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">17.</span>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987222v1.17" data-doi="10.1128/JVI.79.3.1595-1604.2005">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Vijgen</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Complete Genomic Sequence of Human Coronavirus OC43: Molecular Clock Analysis Suggests a Relatively Recent Zoonotic Coronavirus Transmission Event</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>, 
         <span class="cit-pub-date">2005</span>. 
         <span class="cit-vol">79</span>(
         <span class="cit-issue">3</span>): p. 
         <span class="cit-fpage">1595</span>–
         <span class="cit-lpage">1604</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DVijgen%26rft.auinit1%253DL.%26rft.volume%253D79%26rft.issue%253D3%26rft.spage%253D1595%26rft.epage%253D1604%26rft.atitle%253DComplete%2BGenomic%2BSequence%2Bof%2BHuman%2BCoronavirus%2BOC43%253A%2BMolecular%2BClock%2BAnalysis%2BSuggests%2Ba%2BRelatively%2BRecent%2BZoonotic%2BCoronavirus%2BTransmission%2BEvent%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.79.3.1595-1604.2005%26rft_id%253Dinfo%253Apmid%252F15650185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijc5LzMvMTU5NSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzEyLzIwMjAuMDMuMTEuOTg3MjIyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">18.</span>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987222v1.18">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Corman</span>, 
          <span class="cit-name-given-names">V.M.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR</span>. 
         <abbr class="cit-jnl-abbrev">Eurosurveillance</abbr>, 
         <span class="cit-pub-date">2020</span>. 
         <span class="cit-vol">25</span>(
         <span class="cit-issue">3</span>): p. 
         <span class="cit-fpage">2000045</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEurosurveillance%26rft.volume%253D25%26rft.spage%253D2000045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">19.</span>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987222v1.19" data-doi="10.1093/bioinformatics/btu170">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bolger</span>, 
          <span class="cit-name-given-names">A.M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Lohse</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">B.</span>
          <span class="cit-name-surname">Usadel</span>
         </span>, 
         <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, 
         <span class="cit-pub-date">2014</span>. 
         <span class="cit-vol">30</span>(
         <span class="cit-issue">15</span>): p. 
         <span class="cit-fpage">2114</span>–
         <span class="cit-lpage">2120</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24695404&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.11.987222.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000340049100004&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">20.</span>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987222v1.20" data-doi="10.1093/ve/vey016">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Suchard</span>, 
          <span class="cit-name-given-names">M.A.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Bayesian phylogenetic and phylodynamic data integration using BEAST 1.10</span>. 
         <abbr class="cit-jnl-abbrev">Virus Evol</abbr>, 
         <span class="cit-pub-date">2018</span>. 
         <span class="cit-vol">4</span>(
         <span class="cit-issue">1</span>): p. 
         <span class="cit-fpage">vey016</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVirus%2BEvol%26rft.volume%253D4%26rft.spage%253Dvey016%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fve%252Fvey016%26rft_id%253Dinfo%253Apmid%252F29942656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/ve/vey016&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=29942656&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.11.987222.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">21.</span>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987222v1.21" data-doi="10.1093/sysbio/syy032">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rambaut</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Posterior Summarization in Bayesian Phylogenetics Using Tracer 1.7</span>. 
         <abbr class="cit-jnl-abbrev">Syst Biol</abbr>, 
         <span class="cit-pub-date">2018</span>. 
         <span class="cit-vol">67</span>(
         <span class="cit-issue">5</span>): p. 
         <span class="cit-fpage">901</span>–
         <span class="cit-lpage">904</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSyst%2BBiol%26rft.volume%253D67%26rft.spage%253D901%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyy032%26rft_id%253Dinfo%253Apmid%252F29718447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/sysbio/syy032&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=29718447&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.11.987222.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
