<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2018/12/10/492835.atom"/>
  <meta name="HW.pisa" content="biorxiv;492835v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Metagenomic Sequencing for Combined Detection of RNA and DNA Viruses in Respiratory Samples from Paediatric Patients"/>
  <meta name="DC.Identifier" content="10.1101/492835"/>
  <meta name="DC.Date" content="2018-12-10"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2018, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Introduction Viruses are the main cause of respiratory tract infections. Metagenomic next-generation sequencing (mNGS) enables the unbiased detection of all potential pathogens in a clinical sample, including variants and even unknown pathogens. To apply mNGS in viral diagnostics, there is a need for sensitive and simultaneous detection of RNA and DNA viruses. In this study, the performance of an in-house mNGS protocol for routine diagnostics of viral respiratory infections, with single tube DNA and RNA sample-pre-treatment and potential for automated pan-pathogen detection was studied. Materials and Methods The sequencing protocol and bioinformatics analysis was designed and optimized including the optimal concentration of the spike-in internal controls equine arteritis virus (EAV) and phocine-herpes virus-1 (PhHV-1).The whole genome of PhHV-1 was sequenced and added to the NCBI database. Subsequently, the protocol was retrospectively validated using a selection of 25 respiratory samples with in total 29 positive and 346 negative PCR results, previously sent to the lab for routine diagnostics. Results The results demonstrated that our protocol using Illumina Nextseq 500 sequencing with 10 million reads showed high repeatability. The NCBI RefSeq database as opposed to the NCBI nucleotide database led to enhanced specificity of virus classification. A correlation was established between read counts and PCR cycle threshold value, demonstrating the semi-quantitative nature of viral detection by mNGS. The results as obtained by mNGS appeared condordant with PCR based diagnostics in 25 out of the 29 (86%) respiratory viruses positive by PCR and in 315 of 346 (91%) PCR-negative results. Viral pathogens only detected by mNGS, not present in the routine diagnostic workflow were influenza C, KI polyomavirus, and cytomegalovirus. Conclusions Sensitivity and analytical specificity of this mNGS protocol was comparable with PCR and higher when considering off-PCR target viral pathogens. All potential viral pathogens were detected in one single test, while it simultaneously obtained detailed information on detected viruses."/>
  <meta name="DC.Contributor" content="Sander van Boheemen"/>
  <meta name="DC.Contributor" content="Anneloes L. van Rijn-Klink"/>
  <meta name="DC.Contributor" content="Nikos Pappas"/>
  <meta name="DC.Contributor" content="Ellen C. Carbo"/>
  <meta name="DC.Contributor" content="Ruben H.P. Vorderman"/>
  <meta name="DC.Contributor" content="Peter J. van `t Hof"/>
  <meta name="DC.Contributor" content="Hailiang Mei"/>
  <meta name="DC.Contributor" content="Eric C.J. Claas"/>
  <meta name="DC.Contributor" content="Aloys C.M. Kroes"/>
  <meta name="DC.Contributor" content="Jutte J.C. de Vries"/>
  <meta name="article:published_time" content="2018-12-10"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Metagenomic Sequencing for Combined Detection of RNA and DNA Viruses in Respiratory Samples from Paediatric Patients"/>
  <meta name="citation_abstract" lang="en" content="&lt;;h3&gt;;ABSTRACT&lt;;/h3&gt;; &lt;;h3&gt;;Introduction&lt;;/h3&gt;; &lt;;p&gt;;Viruses are the main cause of respiratory tract infections. Metagenomic next-generation sequencing (mNGS) enables the unbiased detection of all potential pathogens in a clinical sample, including variants and even unknown pathogens. To apply mNGS in viral diagnostics, there is a need for sensitive and simultaneous detection of RNA and DNA viruses. In this study, the performance of an in-house mNGS protocol for routine diagnostics of viral respiratory infections, with single tube DNA and RNA sample-pre-treatment and potential for automated pan-pathogen detection was studied.&lt;;/p&gt;;&lt;;h3&gt;;Materials and Methods&lt;;/h3&gt;; &lt;;p&gt;;The sequencing protocol and bioinformatics analysis was designed and optimized including the optimal concentration of the spike-in internal controls equine arteritis virus (EAV) and phocine-herpes virus-1 (PhHV-1).The whole genome of PhHV-1 was sequenced and added to the NCBI database. Subsequently, the protocol was retrospectively validated using a selection of 25 respiratory samples with in total 29 positive and 346 negative PCR results, previously sent to the lab for routine diagnostics.&lt;;/p&gt;;&lt;;h3&gt;;Results&lt;;/h3&gt;; &lt;;p&gt;;The results demonstrated that our protocol using Illumina Nextseq 500 sequencing with 10 million reads showed high repeatability. The NCBI RefSeq database as opposed to the NCBI nucleotide database led to enhanced specificity of virus classification. A correlation was established between read counts and PCR cycle threshold value, demonstrating the semi-quantitative nature of viral detection by mNGS. The results as obtained by mNGS appeared condordant with PCR based diagnostics in 25 out of the 29 (86%) respiratory viruses positive by PCR and in 315 of 346 (91%) PCR-negative results. Viral pathogens only detected by mNGS, not present in the routine diagnostic workflow were influenza C, KI polyomavirus, and cytomegalovirus.&lt;;/p&gt;;&lt;;h3&gt;;Conclusions&lt;;/h3&gt;; &lt;;p&gt;;Sensitivity and analytical specificity of this mNGS protocol was comparable with PCR and higher when considering off-PCR target viral pathogens. All potential viral pathogens were detected in one single test, while it simultaneously obtained detailed information on detected viruses.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2018/01/01"/>
  <meta name="citation_mjid" content="biorxiv;492835v1"/>
  <meta name="citation_id" content="492835v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/492835v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/492835v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/492835v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835.full.pdf"/>
  <meta name="citation_doi" content="10.1101/492835"/>
  <meta name="citation_num_pages" content="39"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="492835"/>
  <meta name="citation_author" content="Sander van Boheemen"/>
  <meta name="citation_author_institution" content="Department of Medical Microbiology, Leiden University Medical Center"/>
  <meta name="citation_author" content="Anneloes L. van Rijn-Klink"/>
  <meta name="citation_author_institution" content="Department of Medical Microbiology, Leiden University Medical Center"/>
  <meta name="citation_author" content="Nikos Pappas"/>
  <meta name="citation_author_institution" content="Sequencing Analysis Support Core, Department of Biomedical Data Sciences, Leiden University Medical Center"/>
  <meta name="citation_author" content="Ellen C. Carbo"/>
  <meta name="citation_author_institution" content="Department of Medical Microbiology, Leiden University Medical Center"/>
  <meta name="citation_author" content="Ruben H.P. Vorderman"/>
  <meta name="citation_author_institution" content="Sequencing Analysis Support Core, Department of Biomedical Data Sciences, Leiden University Medical Center"/>
  <meta name="citation_author" content="Peter J. van `t Hof"/>
  <meta name="citation_author_institution" content="Sequencing Analysis Support Core, Department of Biomedical Data Sciences, Leiden University Medical Center"/>
  <meta name="citation_author" content="Hailiang Mei"/>
  <meta name="citation_author_institution" content="Sequencing Analysis Support Core, Department of Biomedical Data Sciences, Leiden University Medical Center"/>
  <meta name="citation_author" content="Eric C.J. Claas"/>
  <meta name="citation_author_institution" content="Department of Medical Microbiology, Leiden University Medical Center"/>
  <meta name="citation_author" content="Aloys C.M. Kroes"/>
  <meta name="citation_author_institution" content="Department of Medical Microbiology, Leiden University Medical Center"/>
  <meta name="citation_author" content="Jutte J.C. de Vries"/>
  <meta name="citation_author_institution" content="Department of Medical Microbiology, Leiden University Medical Center"/>
  <meta name="citation_author_email" content="jjcdevries@lumc.nl"/>
  <meta name="citation_reference" content="Lozano R, Naghavi M, Foreman K, Lim S, Shibuya K, Aboyans V, Abraham J, Adair T, Aggarwal R, Ahn SY et al: Global and regional mortality from 235 causes of death for 20 age groups in 1990 and 2010: a systematic analysis for the Global Burden of Disease Study 2010. Lancet (London, England) 2012, 380(9859):2095–2128."/>
  <meta name="citation_reference" content="Nair H, Simoes EA, Rudan I, Gessner BD, Azziz-Baumgartner E, Zhang JS, Feikin DR, Mackenzie GA, Moisi JC, Roca A et al: Global and regional burden of hospital admissions for severe acute lower respiratory infections in young children in 2010: a systematic analysis. Lancet (London, England) 2013, 381(9875):1380–1390."/>
  <meta name="citation_reference" content="Bates M, Mudenda V, Mwaba P, Zumla A: Deaths due to respiratory tract infections in Africa: a review of autopsy studies. Current opinion in pulmonary medicine 2013, 19(3):229–237."/>
  <meta name="citation_reference" content="Jain S, Self WH, Wunderink RG, Fakhran S, Balk R, Bramley AM, Reed C, Grijalva CG, Anderson EJ, Courtney DM et al: Community-Acquired Pneumonia Requiring Hospitalization among U.S. Adults. N Engl J Med 2015, 373(5):415–427."/>
  <meta name="citation_reference" content="Heikkinen T, Jarvinen A: The common cold. Lancet (London, England) 2003, 361 (9351):51–59."/>
  <meta name="citation_reference" content="Ieven M, Coenen S, Loens K, Lammens C, Coenjaerts F, Vanderstraeten A, Henriques-Normark B, Crook D, Huygen K, Butler CC et al: Aetiology of lower respiratory tract infection in adults in primary care: a prospective study in 11 European countries. Clinical microbiology and infection : the official publication of the European Society of Clinical Microbiology and Infectious Diseases 2018."/>
  <meta name="citation_reference" content="Zaki AM, van Boheemen S, Bestebroer TM, Osterhaus AD, Fouchier RA: Isolation of a novel coronavirus from a man with pneumonia in Saudi Arabia. N Engl J Med 2012, 367(19):1814–1820."/>
  <meta name="citation_reference" content="Prachayangprecha S, Schapendonk CM, Koopmans MP, Osterhaus AD, Schurch AC, Pas SD, van der Eijk AA, Poovorawan Y, Haagmans BL, Smits SL: Exploring the potential of next-generation sequencing in detection of respiratory viruses. Journal of clinical microbiology 2014, 52(10):3722–3730."/>
  <meta name="citation_reference" content="Wooley JC, Godzik A, Friedberg I: A primer on metagenomics. PLoS computational biology 2010, 6(2):e1000667."/>
  <meta name="citation_reference" content="Hoffmann B, Scheuch M, Hoper D, Jungblut R, Holsteg M, Schirrmeier H, Eschbaumer M, Goller KV, Wernike K, Fischer M et al: Novel orthobunyavirus in Cattle, Europe, 2011. Emerging infectious diseases 2012, 18(3):469–472."/>
  <meta name="citation_reference" content="Mongkolrattanothai K, Naccache SN, Bender JM, Samayoa E, Pham E, Yu G, Dien Bard J, Miller S, Aldrovandi G, Chiu CY: Neurobrucellosis: Unexpected Answer From Metagenomic Next-Generation Sequencing. Journal of the Pediatric Infectious Diseases Society 2017, 6(4):393–398."/>
  <meta name="citation_reference" content="van Boheemen S, de Graaf M, Lauber C, Bestebroer TM, Raj VS, Zaki AM, Osterhaus AD, Haagmans BL, Gorbalenya AE, Snijder EJ et al: Genomic characterization of a newly discovered coronavirus associated with acute respiratory distress syndrome in humans. mBio 2012, 3(6)."/>
  <meta name="citation_reference" content="Kohl C, Brinkmann A, Dabrowski PW, Radonic A, Nitsche A, Kurth A: Protocol for metagenomic virus detection in clinical specimens. Emerging infectious diseases 2015, 21(1):48–57."/>
  <meta name="citation_reference" content="Parker J, Chen J: Application of next generation sequencing for the detection of human viral pathogens in clinical specimens. Journal of clinical virology: the official publication of the Pan American Society for Clinical Virology 2017, 86:20–26."/>
  <meta name="citation_reference" content="Zou X, Tang G, Zhao X, Huang Y, Chen T, Lei M, Chen W, Yang L, Zhu W, Zhuang L et al: Simultaneous virus identification and characterization of severe unexplained pneumonia cases using a metagenomics sequencing technique. Science China Life sciences 2017, 60(3):279–286."/>
  <meta name="citation_reference" content="Wylie KM, Mihindukulasuriya KA, Sodergren E, Weinstock GM, Storch GA: Sequence analysis of the human virome in febrile and afebrile children. PloS one 2012, 7(6):e27735."/>
  <meta name="citation_reference" content="Loens K, van Loon AM, Coenjaerts F, van Aarle Y, Goossens H, Wallace P, Claas EJ, Ieven M: Performance of different mono- and multiplex nucleic acid amplification tests on a multipathogen external quality assessment panel. Journal of clinical microbiology 2012, 50(3):977–987."/>
  <meta name="citation_reference" content="Welsh J, McClelland M: Fingerprinting genomes using PCR with arbitrary primers. Nucleic acids research 1990, 18(24):7213–7218."/>
  <meta name="citation_reference" content="Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ: Basic local alignment search tool. Journal of molecular biology 1990, 215(3):403–410."/>
  <meta name="citation_reference" content="Wery M, Descrimes M, Thermes C, Gautheret D, Morillon A: Zinc-mediated RNA fragmentation allows robust transcript reassembly upon whole transcriptome RNA-Seq. Methods (San Diego, Calif) 2013, 63(1):25–31."/>
  <meta name="citation_reference" content="New England BioLabs Inc. [https://www.neb.com/products/e7420-nebnext-ultra-directional-rna-library-prep-kit-for-illumina#Product%20Information] Version 8.0."/>
  <meta name="citation_reference" content="[http://biopet-docs.readthedocs.io/en/stable/]"/>
  <meta name="citation_reference" content="[https://www.bioinformatics.babraham.ac.uk/projects/fastqc/]"/>
  <meta name="citation_reference" content="Joshi NA, Fass JN: Sickle: A sliding-window, adaptive, quality-based trimming tool for FastQ files. 2011."/>
  <meta name="citation_reference" content="Martin M: Cutadept removes adapter sequences from high-throughput sequencing reads. EMBnetjournal 2011, 17(10)."/>
  <meta name="citation_reference" content="Kim D, Song L, Breitwieser FP, Salzberg SL: Centrifuge: rapid and sensitive classification of metagenomic sequences. Genome research 2016, 26(12):1721–1729."/>
  <meta name="citation_reference" content="O’Leary NA, Wright MW, Brister JR, Ciufo S, Haddad D, McVeigh R, Rajput B, Robbertse B, Smith-White B, Ako-Adjei D et al: Reference sequence (RefSeq) database at NCBI: current status, taxonomic expansion, and functional annotation. Nucleic acids research 2016, 44(D1):D733–745."/>
  <meta name="citation_reference" content="Ondov BD, Bergman NH, Phillippy AM: Interactive metagenomic visualization in a Web browser. BMC bioinformatics 2011, 12:385."/>
  <meta name="citation_reference" content="https://github.com/biowdl/virus-assembly"/>
  <meta name="citation_reference" content="https://github.com/lh3/seqtk"/>
  <meta name="citation_reference" content="Bankevich A, Nurk S, Antipov D, Gurevich AA, Dvorkin M, Kulikov AS, Lesin VM, Nikolenko SI, Pham S, Prjibelski AD et al: SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. Journal of computational biology: a journal of computational molecular cell biology 2012, 19(5):455–477."/>
  <meta name="citation_reference" content="Li H: Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM, vol. 1303; 2013."/>
  <meta name="citation_reference" content="Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, Marth G, Abecasis G, Durbin R: The Sequence Alignment/Map format and SAMtools. Bioinformatics (Oxford, England) 2009, 25(16):2078–2079."/>
  <meta name="citation_reference" content="Sinha R, Stanley G, Gulati GS, Ezran C, Travaglini KJ, Wei E, Chan CKF, Nabhan AN, Su T, Morganti RM et al: Index Switching Causes “Spreading-Of-Signal” Among Multiplexed Samples In Illumina HiSeq 4000 DNA Sequencing. bioRxiv 2017."/>
  <meta name="citation_reference" content="Orozovic G, Orozovic K, Lennerstrand J, Olsen B: Detection of resistance mutations to antivirals oseltamivir and zanamivir in avian influenza A viruses isolated from wild birds. PloS one 2011, 6(1):e16028."/>
  <meta name="citation_reference" content="Hsieh NH. Assessing the oseltamivir-induced resistance risk and implications for influenza infection control strategies. Infect Drug Resist. 2017;10:215–226"/>
  <meta name="citation_reference" content="Hasan MR, Rawat A, Tang P, Jithesh PV, Thomas E, Tan R, Tilley P: Depletion of Human DNA in Spiked Clinical Specimens for Improvement of Sensitivity of Pathogen Detection by Next-Generation Sequencing. Journal of clinical microbiology 2016, 54(4):919–927."/>
  <meta name="citation_reference" content="Ninove L, Nougairede A, Gazin C, Thirion L, Delogu I, Zandotti C, Charrel RN, De Lamballerie X: RNA and DNA bacteriophages as molecular diagnosis controls in clinical virology: a comprehensive study of more than 45,000 routine PCR tests. PloS one 2011, 6(2):e16142."/>
  <meta name="citation_reference" content="Schlaberg R, Chiu CY, Miller S, Procop GW, Weinstock G: Validation of Metagenomic Next-Generation Sequencing Tests for Universal Pathogen Detection. Archives of pathology &amp;; laboratory medicine 2017, 141(6):776–786."/>
  <meta name="citation_reference" content="van der Valk T, Vezzi F, Ormestad M, Dalen L, Guschanski K: Estimating the rate of index hopping on the Illumina HiSeq X platform. bioRxiv 2018."/>
  <meta name="citation_reference" content="Naccache SN, Hackett J, Jr., Delwart EL, Chiu CY: Concerns over the origin of NIH-CQV, a novel virus discovered in Chinese patients with seronegative hepatitis. Proceedings of the National Academy of Sciences of the United States of America 2014, 111(11):E976."/>
  <meta name="citation_reference" content="Bal A, Pichon M, Picard C, Casalegno JS, Valette M, Schuffenecker I, Billard L, Vallet S, Vilchez G, Cheynet V, Oriol G, Trouillet-Assant S, Gillet Y, Lina B, Brengel-Pesce K, Morfin F, Josset L: Quality control implementation for universal characterization of DNA and RNA viruses in clinical respiratory samples using single metagenomics next-generation sequencing workflow. BMC Infect Dis 2018: 18(1):537"/>
  <meta name="twitter:title" content="Metagenomic Sequencing for Combined Detection of RNA and DNA Viruses in Respiratory Samples from Paediatric Patients"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Introduction Viruses are the main cause of respiratory tract infections. Metagenomic next-generation sequencing (mNGS) enables the unbiased detection of all potential pathogens in a clinical sample, including variants and even unknown pathogens. To apply mNGS in viral diagnostics, there is a need for sensitive and simultaneous detection of RNA and DNA viruses. In this study, the performance of an in-house mNGS protocol for routine diagnostics of viral respiratory infections, with single tube DNA and RNA sample-pre-treatment and potential for automated pan-pathogen detection was studied. Materials and Methods The sequencing protocol and bioinformatics analysis was designed and optimized including the optimal concentration of the spike-in internal controls equine arteritis virus (EAV) and phocine-herpes virus-1 (PhHV-1).The whole genome of PhHV-1 was sequenced and added to the NCBI database. Subsequently, the protocol was retrospectively validated using a selection of 25 respiratory samples with in total 29 positive and 346 negative PCR results, previously sent to the lab for routine diagnostics. Results The results demonstrated that our protocol using Illumina Nextseq 500 sequencing with 10 million reads showed high repeatability. The NCBI RefSeq database as opposed to the NCBI nucleotide database led to enhanced specificity of virus classification. A correlation was established between read counts and PCR cycle threshold value, demonstrating the semi-quantitative nature of viral detection by mNGS. The results as obtained by mNGS appeared condordant with PCR based diagnostics in 25 out of the 29 (86%) respiratory viruses positive by PCR and in 315 of 346 (91%) PCR-negative results. Viral pathogens only detected by mNGS, not present in the routine diagnostic workflow were influenza C, KI polyomavirus, and cytomegalovirus. Conclusions Sensitivity and analytical specificity of this mNGS protocol was comparable with PCR and higher when considering off-PCR target viral pathogens. All potential viral pathogens were detected in one single test, while it simultaneously obtained detailed information on detected viruses."/>
  <meta name="og-title" property="og:title" content="Metagenomic Sequencing for Combined Detection of RNA and DNA Viruses in Respiratory Samples from Paediatric Patients"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/492835v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Introduction Viruses are the main cause of respiratory tract infections. Metagenomic next-generation sequencing (mNGS) enables the unbiased detection of all potential pathogens in a clinical sample, including variants and even unknown pathogens. To apply mNGS in viral diagnostics, there is a need for sensitive and simultaneous detection of RNA and DNA viruses. In this study, the performance of an in-house mNGS protocol for routine diagnostics of viral respiratory infections, with single tube DNA and RNA sample-pre-treatment and potential for automated pan-pathogen detection was studied. Materials and Methods The sequencing protocol and bioinformatics analysis was designed and optimized including the optimal concentration of the spike-in internal controls equine arteritis virus (EAV) and phocine-herpes virus-1 (PhHV-1).The whole genome of PhHV-1 was sequenced and added to the NCBI database. Subsequently, the protocol was retrospectively validated using a selection of 25 respiratory samples with in total 29 positive and 346 negative PCR results, previously sent to the lab for routine diagnostics. Results The results demonstrated that our protocol using Illumina Nextseq 500 sequencing with 10 million reads showed high repeatability. The NCBI RefSeq database as opposed to the NCBI nucleotide database led to enhanced specificity of virus classification. A correlation was established between read counts and PCR cycle threshold value, demonstrating the semi-quantitative nature of viral detection by mNGS. The results as obtained by mNGS appeared condordant with PCR based diagnostics in 25 out of the 29 (86%) respiratory viruses positive by PCR and in 315 of 346 (91%) PCR-negative results. Viral pathogens only detected by mNGS, not present in the routine diagnostic workflow were influenza C, KI polyomavirus, and cytomegalovirus. Conclusions Sensitivity and analytical specificity of this mNGS protocol was comparable with PCR and higher when considering off-PCR target viral pathogens. All potential viral pathogens were detected in one single test, while it simultaneously obtained detailed information on detected viruses."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2018-12-10"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Metagenomic Sequencing for Combined Detection of RNA and DNA Viruses in Respiratory Samples from Paediatric Patients | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-146468 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">ABSTRACT</h2>
    <div id="sec-1" class="subsection">
     <p id="p-5">
      <strong>Introduction</strong> Viruses are the main cause of respiratory tract infections. Metagenomic next-generation sequencing (mNGS) enables the unbiased detection of all potential pathogens in a clinical sample, including variants and even unknown pathogens. To apply mNGS in viral diagnostics, there is a need for sensitive and simultaneous detection of RNA and DNA viruses. In this study, the performance of an in-house mNGS protocol for routine diagnostics of viral respiratory infections, with single tube DNA and RNA sample-pre-treatment and potential for automated pan-pathogen detection was studied.
     </p>
    </div>
    <div id="sec-2" class="subsection">
     <p id="p-6">
      <strong>Materials and Methods</strong> The sequencing protocol and bioinformatics analysis was designed and optimized including the optimal concentration of the spike-in internal controls equine arteritis virus (EAV) and phocine-herpes virus-1 (PhHV-1).The whole genome of PhHV-1 was sequenced and added to the NCBI database. Subsequently, the protocol was retrospectively validated using a selection of 25 respiratory samples with in total 29 positive and 346 negative PCR results, previously sent to the lab for routine diagnostics.
     </p>
    </div>
    <div id="sec-3" class="subsection">
     <p id="p-7">
      <strong>Results</strong> The results demonstrated that our protocol using Illumina Nextseq 500 sequencing with 10 million reads showed high repeatability. The NCBI RefSeq database as opposed to the NCBI nucleotide database led to enhanced specificity of virus classification. A correlation was established between read counts and PCR cycle threshold value, demonstrating the semi-quantitative nature of viral detection by mNGS. The results as obtained by mNGS appeared condordant with PCR based diagnostics in 25 out of the 29 (86%) respiratory viruses positive by PCR and in 315 of 346 (91%) PCR-negative results. Viral pathogens only detected by mNGS, not present in the routine diagnostic workflow were influenza C, KI polyomavirus, and cytomegalovirus.
     </p>
    </div>
    <div id="sec-4" class="subsection">
     <p id="p-8">
      <strong>Conclusions</strong> Sensitivity and analytical specificity of this mNGS protocol was comparable with PCR and higher when considering off-PCR target viral pathogens. All potential viral pathogens were detected in one single test, while it simultaneously obtained detailed information on detected viruses.
     </p>
    </div>
   </div>
   <div class="section" id="sec-5">
    <h2 class="">INTRODUCTION</h2>
    <p id="p-9">Respiratory tract infections pose a great burden on public health, causing extensive morbidity and mortality among patients worldwide [
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>–
     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>]. The majority of acute respiratory infections is caused by viruses, such as rhinovirus (RV), influenza (INF) A and B viruses, metapneumovirus (MPV), and respiratory syncytial virus (RSV) [
     <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>]. However, in 20-62% of the patients, no pathogen is detected [
     <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>–
     <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>]. This might be the result of diagnostic failures or even infection by unknown pathogens, such as the Middle East respiratory syndrome coronavirus (MERS-CoV), that was first recognized in 2012 [
     <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>].
    </p>
    <p id="p-10">Rapid identification of the respiratory pathogen is critical to determine downstream decision making such as isolation measures or treatment, including cessation of antibiotic therapy. Current diagnostic amplification methods as real-time polymerase chain reaction (qPCR) are very sensitive and specific, but are aiming at particular virus species or types. Genetic diversity within the virus genome and the sheer number of potential pathogens in many clinical conditions pose limitations to predefined primer and probe based approaches, leading to false negative results [
     <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>]. These limitations, combined with the potential emergence of new or unusual pathogens highlight the need for less restricted approaches that could improve the diagnosis and subsequent outbreak management of infectious diseases.
    </p>
    <p id="p-11">Metagenomics relates to the study of the complete genomic content in a complex mixture of (micro)organisms [
     <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>]. Unlike bacteria, viruses do not display a common gene in all virus families, and therefore pan-virus detection relies on catch-all analytic methods. Metagenomics or untargeted next-generation sequencing (mNGS) offers a culture and nucleotide-sequence-independent method that eliminates the need to define the targets for diagnosis beforehand. Besides primary detection, mNGS immediately offers additional information, on virulence markers, epidemiology, genotyping, and evolution of pathogens [
     <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>, 
     <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>–
     <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>]. Furthermore, quantitative assessment of the presence of virus copies in the sample is enabled by the number reads [
     <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>].
    </p>
    <p id="p-12">While original mNGS studies typically aim at analysis of (shifts in) population diversity of abundant DNA microbes, detection of viral pathogens in patient samples requires a different technical approach because of 1) the very low abundancy of viral pathogens (&lt;;1%) in clinical samples and 2) the requisite of detection of both DNA and RNA viruses. Hence, a low limit of detection for RNA and DNA in one single assay is essential for implementation of mNGS for routine pathogen detection in clinical diagnostic laboratories. Current viral mNGS protocols are optimized for either RNA or DNA detection [
     <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>, 
     <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>–
     <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>]. Consequently, detection of both RNA and DNA viruses requires parallel work-up of both RNA and DNA pre-treatment methods. Additionally, to increase the relative concentration of viral sequences, viral particle enrichment techniques are often applied [
     <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">8</a>, 
     <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>]. These techniques are laborious and not easily automated for routine clinical diagnostic use. Moreover, during enrichment directed at viral particles, intracellular viral nucleic acids as genomes and mRNAs are being discarded. Following sequencing, the bioinformatics classification and interpretation of the results remain a major challenge. Bioinformatic classifiers are often developed for usage in either microbiome studies or classification of high abundant reads whereas extensive validation for clinical diagnostic usage in settings of very low abundancy is very limited. After bioinformatics classification, the challenge remains to discriminate between viruses that play a role in aetiology and bona fide viruses [
     <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>]. Before mNGS might be considered in routine diagnostics, there is a need for critical evaluation and validation of every step in the procedure.
    </p>
    <p id="p-13">In this study, we evaluated a metagenomic protocol for NGS-based pathogen detection with sample pre-treatment for DNA and RNA in a single tube. The method was validated using a selection of 25 respiratory paediatric samples with in total 29 positive and 346 negative viral PCR results. The main study objective was to define a sensitive and specific method for mNGS to be used as a broad diagnostic tool for viral respiratory diseases with the potency for automated pan-pathogen detection.</p>
   </div>
   <div class="section" id="sec-6">
    <h2 class="">MATERIAL AND METHODS</h2>
    <div id="sec-7" class="subsection">
     <h3>Sample selection</h3>
     <p id="p-14">Twenty-five stored clinical respiratory samples (−80 °C) from paediatric patients, sent to the microbiological laboratory for routine viral diagnostics in 2016, were selected from the laboratory database (GLIMS, MIPS, Belgium) at the Leiden University Medical Center (LUMC). The selection was based on respiratory virus PCR test results: 21 out of the 25 samples had one or more positive PCR result, with a variety of respiratory viruses and a wide range of quantification cycle (Cq) values. The lab-developed real-time multiplex PCR method used was an updated version of the assay previously described by Loens et al [
      <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>]. The sample types represented the routine diagnostic samples from paediatric patients sent to our laboratory: predominantly nasopharyngeal washings (n=17), sputum (n=2) and broncho-alveolar lavages (BAL, n=2). The patient selection (age range 1.2 months – 15 years) represented the paediatric population with respiratory diagnostics in our university hospital in terms of (underlying) illness.
     </p>
    </div>
    <div id="sec-8" class="subsection">
     <h3>Sample pre-treatment</h3>
     <p id="p-15">A sample pre-treatment protocol was designed with 1) potential for automation, 2) potential for pan-pathogen detection and 3) detection of intracellular viral nucleic acids. Consequently, any type of viral enrichment steps were excluded (filtration, centrifugation, nucleases, rRNA removal). Total nucleid acids (NA) were extracted directly from 200 ul of clinical material, using the MagNAPure 96 DNA and Viral NA Small Volume Kit (Roche Diagnostics, Almere, the Netherlands) with 100 μL output eluate.</p>
    </div>
    <div id="sec-9" class="subsection">
     <h3>Internal controls</h3>
     <p id="p-16">Clinical material was spiked with equine arteritis virus (EAV) and phocine herpesvirus 1 (PhHV1, kindly provided by prof. dr. H.G.M. Niesters, the Netherlands) as internal controls for RNA and DNA virus detection respectively. To determine the optimal concentration of the internal controls a dilution series was added to a mix of two pooled influenza A positive throat swabs (Cq 25) (PhHV1/EAV 1:100,000 1:10,000 1:1,000 1:100). Concentration was based on the number of mNGS reads (Centrifuge output as well as BLAST) in order to serve as control [
      <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>].
     </p>
    </div>
    <div id="sec-10" class="subsection">
     <h3>Quality control</h3>
     <p id="p-17">Before sequencing the DNA input concentration was measured with the Qubit (ThermoFisher Scientific, Waltham, USA), to determine whether there was sufficient DNA in the sample to obtain sequencing results. The range of DNA input for library preparation was 0.5 ng/μl for throat swabs (see reproducibility experiment) up to 300 ng/μl for bronchoalveolar lavages and sputa.</p>
    </div>
    <div id="sec-11" class="subsection">
     <h3>Fragmentation</h3>
     <p id="p-18">To compare the effect of different DNA fragmentation techniques, ten samples were 1) chemically fragmented using zinc (10 min.) and 2) physically fragmented using sonication with the Bioruptor
      <sup>®</sup> pico (Diagenode, Seraing, Belgium, on/off time: 18/30s, 5 cycli) [
      <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>]. Three samples were also tested with the 3) high intensity settings of the Bioruptor
      <sup>®</sup> pico (on/off time: 30/40s, 14 cycli).
     </p>
    </div>
    <div id="sec-12" class="subsection">
     <h3>Library preparation</h3>
     <p id="p-19">Libraries were constructed with 7μL extracted nucleic acids using the NEBNext
      <sup>®</sup> Ultra™ Directional RNA Library Prep Kit for Illumina
      <sup>®</sup> [
      <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>] using single, unique adaptors. This kit has been developed for transcriptome analyses. We made several adaptations to the manufacturers protocol in order to enable simultaneous detection of both DNA and RNA viruses: the following steps were omitted: Poly A mRNA capture isolation (Instruction manual NEB #E7420S/L, version 8.0, Chapter 1), rRNA depletion and DNase step (Chapter 2.1-2.4, 2.5B, 2.11A).
     </p>
     <p id="p-20">The size of fragments in the library was 300-700 bp. Adaptors were diluted 30 fold given the low RNA/DNA input and 21 PCR cycli were run post-adaptor ligation.</p>
    </div>
    <div id="sec-13" class="subsection">
     <h3>Nucleotide Sequence Analysis</h3>
     <p id="p-21">Sequencing was performed on Illumina HiSeq 4000 and NextSeq 500 sequencing systems (Illumina, San Diego, CA, USA), obtaining 10 million 150 bp paired-end reads per sample.</p>
    </div>
    <div id="sec-14" class="subsection">
     <h3>Detection limit</h3>
     <p id="p-22">To determine the detection limit of mNGS, serial dilutions (undiluted, 10
      <sup>−1</sup>, 10
      <sup>−2</sup>, 10
      <sup>−3</sup>, 10
      <sup>−4</sup>) of an influenza A positive sample was tested with both lab developed real-time PCR and mNGS.
     </p>
    </div>
    <div id="sec-15" class="subsection">
     <h3>Repeatability (within run precision)</h3>
     <p id="p-23">To determine the reproducibility of metagenomic sequencing an influenza A positive clinical sample (throat swab) was tested in quadruple. This sample was divided into separate aliquots, nucleic acids were extracted, library preparation and subsequent sequence analysis on the Illumina HiSeq 4000 was performed in one run.</p>
    </div>
    <div id="sec-16" class="subsection">
     <h3>Bioinformatics: taxonomic classification</h3>
     <p id="p-24">All FASTQ files were processed using the BIOPET Gears pipeline version 0.9.0 developed at the LUMC [
      <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>]. This pipeline performs FASTQ pre-processing (including quality control, quality trimming and adapter clipping) and taxonomic classification of sequencing reads. In this project, FastQC version 0.11.2 [
      <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>] was used for checking the quality of the raw reads. Low quality read trimming was done using Sickle [
      <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>] version 1.33 with default settings. Adapter clipping was performed using Cutadapt [
      <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>] version 1.10 with default settings. Taxonomic classification of reads was performed with Centrifuge [
      <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>] version 1.0.1-beta. The pre-built NT index, which contains all sequences from NCBI’s nucleotide database, provided by the Centrifuge developers was used (
      <a href="ftp://ftp.ccb.jhu.edu/pub/infphilo/centrifuge/data/nt.tar.gz">ftp://ftp.ccb.jhu.edu/pub/infphilo/centrifuge/data/nt.tar.gz</a>) as the reference database.
     </p>
     <p id="p-25">In addition, a customized reference centrifuge index with sequence information obtained from the NCBI’s RefSeq [
      <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>] (accessed November 2017) database was built. RefSeq genomic sequences for the domains of bacteria, viruses, archaea, fungi, protozoa, as well as the human reference, along with the taxonomy identifiers, were downloaded with the Centrifuge-download utility and were used as input for centrifuge-build.
     </p>
     <p id="p-26">Centrifuge settings were evaluated to increase the sensitivity and specificity. The default setting, with which a read can be assigned to up to five different taxonomic categories, was compared to one unique assignment per read [
      <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a>] where a read is assigned to a single taxonomic category, corresponding to the lowest common ancestor of all matching species.
     </p>
     <p id="p-27">Kraken-style reports with taxonomical information were produced by the Centrifuge-kreport utility for all (default) options. Both unique and non-unique assignments can be reported, and these settings were compared. The resulting tree-like structured, Kraken-style reports were visualized with Krona [
      <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>] version 2.0.
     </p>
     <p id="p-28">In silico simulated EAV reads were analysed in different databases (NCBI nucleotide vs RefSeq), classification algorithms (max 5 labels per sequence, vs unique (common ancestor)) and reporting (non-unique vs unique) to determine the most sensitive an specific bioinformatic analyses using Centrifuge.</p>
     <p id="p-29">To determine the amount of reads needed, results of 1 and 10 million reads were compared. 1 million reads were randomly selected of the 10 million reads of one FASTQ file and analysed.</p>
    </div>
    <div id="sec-17" class="subsection">
     <h3>Bioinformatics: assembly of PhHV1 sequences</h3>
     <p id="p-30">Assembly of PhHV1 was done using the biowdl virus-assembly pipeline 0.1 [
      <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>]. The QC part of the biowdl pipeline determines which adapters need to be clipped by using FastQC version 0.11.7 [
      <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>] and cutadapt version 1.16 [
      <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>], with minimum length setting “1”. The resulting reads were downsampled within bowdl to 250 000 reads using seqtk 1.2 [
      <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>] after which SPADES version 3.11.1 [
      <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>] was run to get the first proposed genome contigs.
     </p>
     <p id="p-31">To retrieve longer assembly contigs a reiterative assembly approach was used by processing the proposed contigs by the biowdl reAssembly pipeline 0.1. This preassembly pipeline aligns reads to contigs of a previous assembly, then selects the aligned reads, downsamples them and runs a new assembly using SPADES. Subtools used for this consisted of BWA 0.7.17 [
      <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>] for indexing and mapping, SAMtools 1.6 [
      <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>] for creating bam files, SAMtools view (version 1.7) for filtering out unmapped reads using the setting “-G 12”, Picard SamToFastq (version 2.18.4) and seqtk for creating fastq files with 250 000 reads. The contigs from the reAssembly pipeline were then processed for a second using SPADES, with setting the ‘cov-cutoff to 5. The resulting contigs were then processed with the reAssembly pipeline for the third and last time setting the ‘cov-cutoff in SPADES to 20.
     </p>
     <p id="p-32">The contigs from the last reAssembly step were then run against the blast NT database using blastn 2.7.1 [
      <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a>] Out of 23 contigs only 5 contigs, that showed the lowest % in identity matches with any other possible non herpes virus species, were selected. The final 5 contigs contained sequence lengths of 97893, 8170 3710, 3294 and1279 nucleotides, the average coverage was 206, 131, 211, 285 and 154, respectively. These five contigs were published as partial genome under accession number (NCBI accession number: 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=MH509440&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F10%2F492835.atom">MH509440</a>)
     </p>
    </div>
    <div id="sec-18" class="subsection">
     <h3>Retrospective validation</h3>
     <p id="p-33">Sensitivity and specificity of the metagenomic NGS procedure was compared with the lab developed multiplex qPCR [
      <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>] using a selection of 21 samples positive for at least one respiratory PCR target and 4 negative samples.. The routine multiplex PCR panel consisted of 15 respiratory target pathogens: influenza virus A/ B, respiratory syncytial virus (RSV), metapneumovirus (MPV), adenovirus (ADV), human bocavirus (HBoV), parainfluenza viruses (PIV) 1/ 2/ 3/ 4, rhinovirus (RV), and the coronaviruses HKU1, NL63, 227E and OC43. Thus, in total 375 PCR results were available (15 targets × 25 samples) of which 29 PCR positive and 346 PCR negative for comparison with mNGS. Validation samples were tested with mNGS, using the total NA extraction protocol, the adapted NEBNext library preparation protocol, and sequencing 10 million reads on an Illumina NextSeq 500. Bioinformatics analyses was performed using Centrifuge with the RefSeq database and unique assignment of sequence reads.
     </p>
    </div>
    <div id="sec-19" class="subsection">
     <h3>Ethical approval of patient studies</h3>
     <p id="p-34">The study design was approved by the medical ethics review committee of the Leiden University Medical Center.</p>
    </div>
   </div>
   <div class="section" id="sec-20">
    <h2 class="">RESULTS</h2>
    <div id="sec-21" class="subsection">
     <h3>Internal controls</h3>
     <p id="p-35">Serial dilutions of EAV and PhHV1 were added to an influenza A PCR positive sample. Serial dilution 1:10,000 detected EAV with a substantial read count in the presence of a viral infection and without a significant decline in target virus family reads (
      <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Based on these results we determined the concentration of internal controls for further experiments.
     </p>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/691417/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/691417/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/691417" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 1.</span>
       <p id="p-36" class="first-child">Internal controls EAV/PhHV-1: serial dilutions against a clinical sample background and within-run precision (INFA)</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-37">The EAV Cq value of the dilutions correlated with the number of EAV reads from both BLAST alignment and the Centrifuge analysis (
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>).
     </p>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Correlation of Cq value and the number of EAV reads (serial dilutions)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-175190024" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Correlation of Cq value and the number of EAV reads (serial dilutions).&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F1.medium.gif" width="440" height="342"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F1.medium.gif" width="440" height="342"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 1.</span>
       <p id="p-38" class="first-child">Correlation of Cq value and the number of EAV reads (serial dilutions).</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-39">Since the NCBI database was lacking a complete PhHV1 genome sequence, PhHV1 was sequenced and based on the gained sequence reads the genome was build using SPAdes [
      <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a>]. The proposed almost complete genome of PhHV1 was added to the NCBI genbank database (submission ID 2124975, GenBank 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=MH509440&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F10%2F492835.atom">MH509440</a>, release date 4 Dec 2018) and used for BLAST alignment.
     </p>
    </div>
    <div id="sec-22" class="subsection">
     <h3>Fragmentation</h3>
     <p id="p-40">The comparison of fragmentation methods for a selection of the samples with relevant target reads, is shown in 
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. Root reads were comparable among the three protocols. The protocol with fragmentation with Zinc had higher yield in target virus reads for all RNA viruses tested and adenovirus.
     </p>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Comparison of fragmentation methods on target reads (species level, log scale)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-175190024" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Comparison of fragmentation methods on target reads (species level, log scale).&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F2.medium.gif" width="440" height="257"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F2.medium.gif" width="440" height="257"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 2.</span>
       <p id="p-41" class="first-child">Comparison of fragmentation methods on target reads (species level, log scale).</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-23" class="subsection">
     <h3>Index hopping</h3>
     <p id="p-42">Sequence analysis by Illumina HiSeq 4000 with single, unique indexes resulted in HRV-C sequences (22-159 reads), in all samples run on the same lane, in contrast to samples run on another lane. Comparison of HRV-C sequences between these samples resulted in an exact match. Retesting of these samples with Illumina Nextseq 500 resulted in disappearance of HRV reads in the samples, with the exception of a few HRV PCR positive samples (
      <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Combined, this was highly suggestive for index hopping at the Illumina HiSeq 4000 [
      <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>].
     </p>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Decline in index hopping (percentages Rhinovirus C of root reads) with Illumina NextSeq 500 as compared to HiSeq 4000. Each line represents one sample." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-175190024" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Decline in index hopping (percentages Rhinovirus C of root reads) with Illumina NextSeq 500 as compared to HiSeq 4000. Each line represents one sample.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F3.medium.gif" width="349" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F3.medium.gif" width="349" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 3.</span>
       <p id="p-43" class="first-child">Decline in index hopping (percentages Rhinovirus C of root reads) with Illumina NextSeq 500 as compared to HiSeq 4000. Each line represents one sample.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-24" class="subsection">
     <h3>Detection limit</h3>
     <p id="p-44">The detection limit, deduced from serial dilutions of EAV (
      <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>) and influenza A (
      <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>) was comparable with a real time PCR Cq value of approximately 35.
     </p>
     <div id="F4" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Serial dilutions of an influenza A positive clinical sample." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-175190024" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Serial dilutions of an influenza A positive clinical sample.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F4.medium.gif" width="440" height="346"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F4.medium.gif" width="440" height="346"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 4.</span>
       <p id="p-45" class="first-child">Serial dilutions of an influenza A positive clinical sample.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-25" class="subsection">
     <h3>Repeatability: within run precision</h3>
     <p id="p-46">The mNGS results of an influenza A positive sample tested in quadruple could be reproduced with only minor differences (
      <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">table 1</a>): coefficient of variation of 1.2%: 0.05 log SD/ 4.0 log average.
     </p>
    </div>
    <div id="sec-26" class="subsection">
     <h3>Bioinformatics: taxonomic classification</h3>
     <p id="p-47">The Centrifuge default settings, with the NCBI nucleotide database and maximum 5 labels per sequence, resulted in various spurious classifications (
      <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>), for example Lassa virus (
      <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>), evidently highly unlikely to be present in patient samples from the Netherlands with respiratory complaints. The specificity could be increased by using the NCBI RefSeq database instead of the nucleotide database. The classification was further improved by changing the Centrifuge tool settings to limit the assignment of homologous reads to the lowest common ancestor (maximum 1 label per sequence). Classification with maximum 5 labels per read resulted in two different outcomes using the report with all mappings and the report with unique mappings, with the latter missing the reads assigned to multiple organisms.
     </p>
     <div id="F5" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Analysis of in silico simulated EAV reads with the different bioinformatic settings of the Centrifuge pipeline." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-175190024" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Analysis of in silico simulated EAV reads with the different bioinformatic settings of the Centrifuge pipeline.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F5.medium.gif" width="440" height="414"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F5.medium.gif" width="440" height="414"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 5.</span>
       <p id="p-48" class="first-child">Analysis of in silico simulated EAV reads with the different bioinformatic settings of the Centrifuge pipeline.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F6" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Spurious Lassa virus reads with the NCBI Nucleotide (NT) database versus the RefSeq database. k5; up to 5 labels per sequence." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-175190024" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Spurious Lassa virus reads with the NCBI Nucleotide (NT) database versus the RefSeq database. k5; up to 5 labels per sequence.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F6.medium.gif" width="330" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F6.medium.gif" width="330" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 6.</span>
       <p id="p-49" class="first-child">Spurious Lassa virus reads with the NCBI Nucleotide (NT) database versus the RefSeq database. k5; up to 5 labels per sequence.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-50">Comparison of classification using these different settings shows the highest sensitivity and specificity using the RefSeq database with one label (lowest common ancestor) assignment, both with in silico prepared datasets containing solely EAV sequence fragments (
      <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>) and with clinical datasets (with highly abundant background) (
      <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>).
     </p>
     <p id="p-51">To determine the effect of the total number of sequencing reads obtained per sample on sensitivity, one million and 10 million reads were compared by means of in silico analysis (
      <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).
     </p>
     <div id="T2" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/690827/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/690827/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/690827" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 2.</span>
       <p id="p-52" class="first-child">Comparison of analysis of 1 million vs 10 million reads.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-27" class="subsection">
     <h3>Retrospective validation</h3>
     <div id="sec-28" class="subsection">
      <h4>Clinical sensitivity based on PCR target pathogens</h4>
      <p id="p-53">The sample collection consisted of 21 clinical specimens positive for at least one of the following PCR target viruses: rhinovirus, influenza A&amp;;B, parainfluenza 1 &amp;;4 (PIV), metapneumovirus, respiratory syncytial virus, coronaviruses NL63 and HKU1 (CoV), human bocavirus (hBoV), and adenovirus (ADV). Fourteen samples were positive for one virus, six samples for two and one sample for three viruses with the lab-developed respiratory multiplex qPCR. Cq values ranged from Cq 17 to Cq 35, with a median of 23.</p>
      <p id="p-54">With mNGS 25 of the 29 viruses demonstrated in routine diagnostics were detected (
       <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>), resulting in a sensitivity of 86% for PCR targets. If a cut-off of 15 reads was handled, sensitivity declined to 67% (
       <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 4</a>).
      </p>
      <p id="p-55">mNGS target read count showed a correlation with the Cq values of the qPCR (
       <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>).
      </p>
      <div id="T3" class="table pos-float">
       <div class="table-inline table-callout-links">
        <div class="callout">
         <span>View this table:</span>
         <ul class="callout-links">
          <li class="view-inline first">
           <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/691073/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
          </li>
          <li class="view-popup">
           <a href="/highwire/markup/691073/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
          </li>
          <li class="download-ppt last">
           <a href="/highwire/powerpoint/691073" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
          </li>
         </ul>
        </div>
       </div>
       <div class="table-caption">
        <span class="table-label">Table 3.</span>
        <p id="p-56" class="first-child">Detection of qPCR viruses positive respiratory samples with mNGS</p>
        <div class="sb-div caption-clear"/>
       </div>
      </div>
      <div id="T4" class="table pos-float">
       <div class="table-inline table-callout-links">
        <div class="callout">
         <span>View this table:</span>
         <ul class="callout-links">
          <li class="view-inline first">
           <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/690859/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
          </li>
          <li class="view-popup">
           <a href="/highwire/markup/690859/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
          </li>
          <li class="download-ppt last">
           <a href="/highwire/powerpoint/690859" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
          </li>
         </ul>
        </div>
       </div>
      </div>
      <div id="T5" class="table pos-float">
       <div class="table-inline table-callout-links">
        <div class="callout">
         <span>View this table:</span>
         <ul class="callout-links">
          <li class="view-inline first">
           <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/690837/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
          </li>
          <li class="view-popup">
           <a href="/highwire/markup/690837/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
          </li>
          <li class="download-ppt last">
           <a href="/highwire/powerpoint/690837" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
          </li>
         </ul>
        </div>
       </div>
       <div class="table-caption">
        <span class="table-label">Table 4.</span>
        <p id="p-57" class="first-child">Sensitivity and specificity of the mNGS protocol tested, based on PCR target virus, and different cut-off levels for defining a positive result.</p>
        <div class="sb-div caption-clear"/>
       </div>
      </div>
      <div id="F7" class="fig pos-float type-figure odd">
       <div class="highwire-figure">
        <div class="fig-inline-img-wrapper">
         <div class="fig-inline-img">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F7.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Semi-quantification of the mNGS assay for target virus detection in clinical samples with qPCR confirmed human respiratory viruses." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-175190024" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Semi-quantification of the mNGS assay for target virus detection in clinical samples with qPCR confirmed human respiratory viruses.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
           <span class="hw-responsive-img">
            <img class="highwire-fragment fragment-image lazyload" alt="Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F7.medium.gif" width="440" height="251"/>
            <noscript>
             <img class="highwire-fragment fragment-image" alt="Figure 7." src="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F7.medium.gif" width="440" height="251"/>
            </noscript>
           </span>
          </a>
         </div>
        </div>
        <ul class="highwire-figure-links inline">
         <li class="download-fig first">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7." data-icon-position="" data-hide-link-title="0">Download figure</a>
         </li>
         <li class="new-tab last">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/10/492835/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
         </li>
        </ul>
       </div>
       <div class="fig-caption">
        <span class="fig-label">Figure 7.</span>
        <p id="p-58" class="first-child">Semi-quantification of the mNGS assay for target virus detection in clinical samples with qPCR confirmed human respiratory viruses.</p>
        <div class="sb-div caption-clear"/>
       </div>
      </div>
     </div>
     <div id="sec-29" class="subsection">
      <h4>Detection of additional viral pathogens by mNGS: off-PCR target viruses</h4>
      <p id="p-59">Next to the viral pathogens tested by PCR, mNGS also detected other pathogenic viruses, indicating additional viral sequences uncovered by mNGS but not included in the routine diagnostics, with influenza C virus being the most prominent. A high amount,4800 reads, of influenza virus C reads (C/Ann Arbor/1/50) (88% of all viral reads and 0.02 of the root reads) was found in one sample. Other potential respiratory pathogens detected by mNGS and not included in PCR analysis were KI polyomavirus (2 samples: 159 and 30 reads respectively), cytomegalovirus (human betaherpesvirus 5) (1704 and 132 reads). All of these viruses are not included routinely in the diagnostic multiplex qPCRs.</p>
     </div>
     <div id="sec-30" class="subsection">
      <h4>Internal controls</h4>
      <p id="p-60">The spiked-in internal controls were detected by mNGS in all samples. EAV sequence reads ranged from 18-34660 (median 538) and herpesviridae reads as indicator of PhHV1 ranged from 1-1707 (median 23).</p>
     </div>
     <div id="sec-31" class="subsection">
      <h4>Analytical specificity based on PCR target viruses</h4>
      <p id="p-61">In total 25 paediatric respiratory samples were available for analysis of analytical specificity of mNGS: 4 samples were negative for all 15 viral pathogens in the multiplex PCR panel (influenza A/B, RSV, HMPV, ADV, HBoV, PIV1/2/3/4, RV, HKU1, NL63, 227E, OC43) and 21 samples were negative for 12-14 of these PCR target pathogens.</p>
      <p id="p-62">Out of in total 346 negative target PCR results of these 25 samples, 315 results corresponded with the finding of 0 target specific reads by mNGS. If a cut-off of 15 reads was used 343 of the 346 negative PCR targets were negative with mNGS. The 3 samples positive by mNGS and negative by PCR were human parainfluenzavirus 1 (27 reads), 3 (31 reads), and 4 (27 reads). Though no conclusive proof for neither true positive or false mNGS results could be found, specificity of mNGS was 91% (315/346) when encountering all reads and ≥99% (343/346) with a 15 reads cut-off (Table 6).</p>
     </div>
     <div id="sec-32" class="subsection">
      <h4>Drug resistance data</h4>
      <p id="p-63">Using the sequence data we identified several mutations on the genome of two clinical samples tested positive for influenza A virus. None of these mutations conferred resistance to either oseltamivir or zanamivir. For the positions where resistance associated mutations occur, amino acids I117, E119, D198, I222, H274, R292, N294 and I314 showed susceptibility to oseltamivir and V116, R118, E119, Q136, D151, R152, R224, E276, R292 and R371 revealed susceptibility to zanamivir [
       <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>, 
       <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>].
      </p>
     </div>
    </div>
   </div>
   <div class="section" id="sec-33">
    <h2 class="">DISCUSSION</h2>
    <p id="p-64">Metagenomic sequencing has not yet been implemented as routine diagnostic tool in clinical diagnostics of viral infections. Such application would require the careful definition and validation of several parameters to enable the accurate assessment of a clinical sample with regard to the presence or absence of a pathogen, in order to fulfil current accreditation guidelines. For this purpose, this study has initiated the optimization of several steps throughout the pre- and post-sequencing workflow, which are considered essential for sensitive and specific mNGS based virus detection. Many virus discovery or virus diagnostic protocols have focussed on the enrichment of viral particles [
     <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>] with the intention to increase the relative amount of virus reads. However, these methods are laborious and intrinsically exclude viral nucleic acid located in host cells. The current protocol enabled high throughput sample pre-treatment by means of automated NA extraction and without depletion of bacterial nor human genome, with potential for pan-pathogen detection. Several adaptations in the bioinformatic script resulted in more accurate classification output reporting.
    </p>
    <p id="p-65">The addition of an internal control to a PCR reaction is widely used as an quality control of a qPCR [
     <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>]. While the addition of internal controls in mNGS is not yet an accepted standard procedure, we employed EAV and PhHV1 as an RNA and DNA controls, respectively, as for diagnostic application such precautions are required. The amount of internal control reads and target virus reads have been reported to be dependent of the amount of background reads (negative correlation) [
     <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>]. In our protocol, the internal controls were used as qualitative controls but may be used as indicator of the amount of background. Since the NCBI database was lacking a complete PhHV1 genome, the Centrifuge index building and classification was limited to classification on a higher taxonomic rank. In order to achieve classification of PhHV1 at species level, the whole genome of PhHV1 was sequenced, and based on the gained sequence reads the genome was build [
     <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">31</a>]. The proposed almost complete genome of PhHV1 was added to the NCBI GenBank database.
    </p>
    <p id="p-66">Sensitivity of the mNGS procotol was 86% (25/29) based on PCR target viruses. Four, all RNA viruses, that were not recovered by mNGS had high Cq values, over 31, i.e. a relatively low viral load. This may be a drawback of the retrospective nature of this clinical evaluation as RNA viruses may be degraded due to storage and freeze-thaw steps, resulting in lower sensitivity of mNGS. A correlation was found between read counts and PCR Cq value, demonstrating the quantitative nature of viral detection by mNGS. Discrepancies between the Cq values and the number of mNGS reads may be explained by 1) unrepresentative Cq values, e.g. by primer mismatch for highly divergent viruses like rhino/enteroviruses and 2) differences in sensitivity of mNGS for several groups of viruses, as has been reported by others [
     <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>]. Additionally, several viral pathogens were detected that were not targeted by the routine PCR assays, including influenza C virus, which is typical of the unbiased nature of the method. In addition, though not within the scope of this study, bacterial pathogens, including Bordetella pertussis (qPCR positive), were also detected. In the current study only viruses were targeted since these could be well compared to qPCR results, bacterial targets remain to be studied in clinical sample types more suitable for bacterial detection than nasopharyngeal washings. The analytical specificity of mNGS appeared to be high, especially with a cut-off of 15 reads. However, the clinical specificity, the relevance of the lower read numbers, still needs further investigation in clinical studies.
    </p>
    <p id="p-67">Sequencing using Illumina HiSeq 4000 chemistry resulted in frequent false positive rhinovirus reads in numerous samples. This problem could be attributed to ‘index hopping’ (index miss-assignment) as earlier described [
     <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a>]. Although the percentage of reads which contributed to the index hopping was very low, this is critical for clinical viral diagnostics, as this is aimed specifically at low abundancy targets [
     <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">34</a>, 
     <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>].
    </p>
    <p id="p-68">Bioinformatics classification of metagenomic sequence data with the pipeline Centrifuge required identification of the optimal parameters in order to minimize miss- and unclassified reads. Default settings of this pipeline resulted in higher rates of both false positive and false negative results. The nucleotide database includes a wide variety of unannotated viral sequences, such as partial sequences and (chimeric) constructs, in contrast to the curated and well-annotated sequences in the NCBI Refseq database, which resulted in a higher specificity. In addition to the database, settings for the assignment algorithm were adapted as well. The assignment settings were adjusted to unique assignment in the case of homology to the lowest common ancestor. This modification resulted in higher sensitivity and specificity than the default settings, however the ability to further subtyping diminished. This is likely to be attributed to the limited representation/availability of strain types within the RefSeq database. In consequence, this leads to a more accurate estimation of the common ancestor for particular viruses, but limited typing results in case of highly variable ones. To obtain optimal typing results, additional annotated sequences may be added or a new database should be build, with a high variety of well-defined and frequently updated virus strain types.</p>
    <p id="p-69">To conclude, this study contributes to the increasing evidence that metagenomic NGS can effectively be used for a wide variety of diagnostic assays in virology, such as unbiased virus detection, resistance mutations, virulence markers, and epidemiology, as shown by the ability to detect SNPs in influenza virus.</p>
    <p id="p-70">These findings support the feasibility of moving this promising field forward to a role in the routine detection of pathogens by the use of mNGS. Further optimization should include the parallel evaluation of adult samples, the inclusion of additional annotated strain sequences to the database, and further elaboration of the classification algorithm and reporting for clinical diagnostics. The importance of both negative non-template control samples [
     <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>] and healthy control cases may support the critical discrimination of contaminants and viral ‘colonization’ from clinically relevant pathogens.
    </p>
   </div>
   <div class="section" id="sec-34">
    <h2 class="">CONCLUSIONS</h2>
    <p id="p-71">Optimal sample preparation and bioinformatics analysis are essential for sensitive and specific mNGS based virus detection.</p>
    <p id="p-72">Using a high-throughput genome extraction method without viral enrichment, both RNA and DNA viruses could be detected with a sensitivity comparable to PCR.</p>
    <p id="p-73">Using mNGS, all potential pathogens can be detected in one single test, while simultaneously obtaining additional detailed information on detected viruses. Interpretation of clinical relevance is an important issue but essentially not different from the use of PCR based assays and supported by the available information on typing and relative quantities. These findings support the feasibility of a role of mNGS in the routine detection of pathogens.</p>
   </div>
   <div class="section" id="sec-35">
    <h2 class="">AUTHOR CONTRIBUTIONS</h2>
    <p id="p-75">SB, ALR, ECJC, ACMK, and JJCV participated in the study design. SB performed the prelibrary preparation experiments. SB, NP, ECC, RHPV, PH, and HM carried out bioinformatic analyses. SB, ALR and ECC analyzed the data. SB and ALR wrote the first version of the manuscript. All authors contributed and revised the manuscript and approved the final manuscript.</p>
   </div>
   <div class="section" id="sec-36">
    <h2 class="">DISCLOSURE DECLARATION</h2>
    <p id="p-76">This research is partially funded by GENERADE Centre of Expertise Genomics in Leiden.</p>
    <p id="p-77">Competing interests: none declared.</p>
   </div>
   <div class="section" id="sec-37">
    <h2 class="">DATA ACCESS</h2>
    <p id="p-78">The raw datasets of this study are not publicly made available given the confidential character of human sequences.</p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">ACKNOWLEDGEMENTS</h2>
    <p id="p-74">We thank our project partners Floyd Wittink, Wouter Suring (Hogeschool Leiden), Danny Duijsings (BaseClear) and Christiaan Henkel (Leiden University). We also like to thank thank Tom Vreeswijk, Lopje Höcker and Mario van Bussel (KML, LUMC) for help with the pre-sequencing experiments and Jeroen Laros (Human Genetics, LUMC) for help with the bioinformatics.</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">REFERENCES</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.1" data-doi="10.1016/s0140-6736(12)61728-0">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lozano</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Naghavi</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Foreman</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lim</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shibuya</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Aboyans</span>
          <span class="cit-name-given-names">V</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Abraham</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Adair</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Aggarwal</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ahn</span>
          <span class="cit-name-given-names">SY</span>
         </span>
         <span class="cit-etal">et al</span>: 
         <span class="cit-article-title">Global and regional mortality from 235 causes of death for 20 age groups in 1990 and 2010: a systematic analysis for the Global Burden of Disease Study 2010</span>. 
         <abbr class="cit-jnl-abbrev">Lancet (London, England)</abbr>
         <span class="cit-pub-date">2012</span>, 
         <span class="cit-vol">380</span>(
         <span class="cit-issue">9859</span>):
         <span class="cit-fpage">2095</span>–
         <span class="cit-lpage">2128</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DLancet%2B%2528London%252C%2BEngland%2529%26rft.volume%253D380%26rft.spage%253D2095%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fs0140-6736%252812%252961728-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/s0140-6736(12)61728-0&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-492835v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Nair</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Simoes</span>
          <span class="cit-name-given-names">EA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rudan</span>
          <span class="cit-name-given-names">I</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gessner</span>
          <span class="cit-name-given-names">BD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Azziz-Baumgartner</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">JS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Feikin</span>
          <span class="cit-name-given-names">DR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mackenzie</span>
          <span class="cit-name-given-names">GA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Moisi</span>
          <span class="cit-name-given-names">JC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Roca</span>
          <span class="cit-name-given-names">A</span>
         </span>
         <span class="cit-etal">et al</span>: 
         <span class="cit-article-title">Global and regional burden of hospital admissions for severe acute lower respiratory infections in young children in 2010: a systematic analysis</span>. 
         <abbr class="cit-jnl-abbrev">Lancet (London, England)</abbr>
         <span class="cit-pub-date">2013</span>, 
         <span class="cit-vol">381</span>(
         <span class="cit-issue">9875</span>):
         <span class="cit-fpage">1380</span>–
         <span class="cit-lpage">1390</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DLancet%2B%2528London%252C%2BEngland%2529%26rft.volume%253D381%26rft.spage%253D1380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.3" data-doi="10.1097/MCP.0b013e32835f4fe4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bates</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mudenda</span>
          <span class="cit-name-given-names">V</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mwaba</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zumla</span>
          <span class="cit-name-given-names">A</span>
         </span>: 
         <span class="cit-article-title">Deaths due to respiratory tract infections in Africa: a review of autopsy studies</span>. 
         <abbr class="cit-jnl-abbrev">Current opinion in pulmonary medicine</abbr>
         <span class="cit-pub-date">2013</span>, 
         <span class="cit-vol">19</span>(
         <span class="cit-issue">3</span>):
         <span class="cit-fpage">229</span>–
         <span class="cit-lpage">237</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bpulmonary%2Bmedicine%26rft.volume%253D19%26rft.spage%253D229%26rft_id%253Dinfo%253Adoi%252F10.1097%252FMCP.0b013e32835f4fe4%26rft_id%253Dinfo%253Apmid%252F23429099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1097/MCP.0b013e32835f4fe4&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23429099&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F10%2F492835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.4" data-doi="10.1056/NEJMoa1500245">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jain</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Self</span>
          <span class="cit-name-given-names">WH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wunderink</span>
          <span class="cit-name-given-names">RG</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fakhran</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Balk</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bramley</span>
          <span class="cit-name-given-names">AM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Reed</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Grijalva</span>
          <span class="cit-name-given-names">CG</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Anderson</span>
          <span class="cit-name-given-names">EJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Courtney</span>
          <span class="cit-name-given-names">DM</span>
         </span>
         <span class="cit-etal">et al</span>: 
         <span class="cit-article-title">Community-Acquired Pneumonia Requiring Hospitalization among U.S. Adults</span>. 
         <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>
         <span class="cit-pub-date">2015</span>, 
         <span class="cit-vol">373</span>(
         <span class="cit-issue">5</span>):
         <span class="cit-fpage">415</span>–
         <span class="cit-lpage">427</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft.volume%253D373%26rft.spage%253D415%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1500245%26rft_id%253Dinfo%253Apmid%252F26172429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1056/NEJMoa1500245&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26172429&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F10%2F492835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-492835v1.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Heikkinen</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jarvinen</span>
          <span class="cit-name-given-names">A</span>
         </span>: 
         <span class="cit-article-title">The common cold</span>. 
         <abbr class="cit-jnl-abbrev">Lancet (London, England)</abbr>
         <span class="cit-pub-date">2003</span>, 
         <span class="cit-vol">361</span> (
         <span class="cit-issue">9351</span>):
         <span class="cit-fpage">51</span>–
         <span class="cit-lpage">59</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DLancet%2B%2528London%252C%2BEngland%2529%26rft.volume%253D361%26rft.spage%253D51%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ieven</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Coenen</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Loens</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lammens</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Coenjaerts</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vanderstraeten</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Henriques-Normark</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Crook</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huygen</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Butler</span>
          <span class="cit-name-given-names">CC</span>
         </span>
         <span class="cit-etal">et al</span>: 
         <span class="cit-article-title">Aetiology of lower respiratory tract infection in adults in primary care: a prospective study in 11 European countries</span>. 
         <abbr class="cit-jnl-abbrev">Clinical microbiology and infection : the official publication of the European Society of Clinical Microbiology and Infectious Diseases</abbr>
         <span class="cit-pub-date">2018</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.7" data-doi="10.1056/NEJMoa1211721">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zaki</span>
          <span class="cit-name-given-names">AM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">van Boheemen</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bestebroer</span>
          <span class="cit-name-given-names">TM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Osterhaus</span>
          <span class="cit-name-given-names">AD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fouchier</span>
          <span class="cit-name-given-names">RA</span>
         </span>: 
         <span class="cit-article-title">Isolation of a novel coronavirus from a man with pneumonia in Saudi Arabia</span>. 
         <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>
         <span class="cit-pub-date">2012</span>, 
         <span class="cit-vol">367</span>(
         <span class="cit-issue">19</span>):
         <span class="cit-fpage">1814</span>–
         <span class="cit-lpage">1820</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.stitle%253DNEJM%26rft.aulast%253DZaki%26rft.auinit1%253DA.%2BM.%26rft.volume%253D367%26rft.issue%253D19%26rft.spage%253D1814%26rft.epage%253D1820%26rft.atitle%253DIsolation%2Bof%2Ba%2Bnovel%2Bcoronavirus%2Bfrom%2Ba%2Bman%2Bwith%2Bpneumonia%2Bin%2BSaudi%2BArabia.%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1211721%26rft_id%253Dinfo%253Apmid%252F23075143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1056/NEJMoa1211721&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23075143&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F10%2F492835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000310773200008&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.8" data-doi="10.1128/JCM.01641-14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Prachayangprecha</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schapendonk</span>
          <span class="cit-name-given-names">CM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Koopmans</span>
          <span class="cit-name-given-names">MP</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Osterhaus</span>
          <span class="cit-name-given-names">AD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schurch</span>
          <span class="cit-name-given-names">AC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pas</span>
          <span class="cit-name-given-names">SD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">van der Eijk</span>
          <span class="cit-name-given-names">AA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Poovorawan</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Haagmans</span>
          <span class="cit-name-given-names">BL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Smits</span>
          <span class="cit-name-given-names">SL</span>
         </span>: 
         <span class="cit-article-title">Exploring the potential of next-generation sequencing in detection of respiratory viruses</span>. 
         <abbr class="cit-jnl-abbrev">Journal of clinical microbiology</abbr>
         <span class="cit-pub-date">2014</span>, 
         <span class="cit-vol">52</span>(
         <span class="cit-issue">10</span>):
         <span class="cit-fpage">3722</span>–
         <span class="cit-lpage">3730</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bclinical%2Bmicrobiology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.01641-14%26rft_id%253Dinfo%253Apmid%252F25100822%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjEwOiI1Mi8xMC8zNzIyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMTIvMTAvNDkyODM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wooley</span>
          <span class="cit-name-given-names">JC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Godzik</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Friedberg</span>
          <span class="cit-name-given-names">I</span>
         </span>: 
         <span class="cit-article-title">A primer on metagenomics</span>. 
         <abbr class="cit-jnl-abbrev">PLoS computational biology</abbr>
         <span class="cit-pub-date">2010</span>, 
         <span class="cit-vol">6</span>(
         <span class="cit-issue">2</span>):
         <span class="cit-fpage">e1000667</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2Bcomputational%2Bbiology%26rft.volume%253D6%26rft.spage%253De1000667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.10" data-doi="10.3201/eid1803.111905">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hoffmann</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Scheuch</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hoper</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jungblut</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Holsteg</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schirrmeier</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Eschbaumer</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Goller</span>
          <span class="cit-name-given-names">KV</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wernike</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fischer</span>
          <span class="cit-name-given-names">M</span>
         </span>
         <span class="cit-etal">et al</span>: 
         <span class="cit-article-title">Novel orthobunyavirus in Cattle, Europe, 2011</span>. 
         <abbr class="cit-jnl-abbrev">Emerging infectious diseases</abbr>
         <span class="cit-pub-date">2012</span>, 
         <span class="cit-vol">18</span>(
         <span class="cit-issue">3</span>):
         <span class="cit-fpage">469</span>–
         <span class="cit-lpage">472</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEmerging%2Binfectious%2Bdiseases%26rft.stitle%253DEmerg%2BInfect%2BDis%26rft.aulast%253DHoffmann%26rft.auinit1%253DB.%26rft.volume%253D18%26rft.issue%253D3%26rft.spage%253D469%26rft.epage%253D472%26rft.atitle%253DNovel%2Borthobunyavirus%2Bin%2BCattle%252C%2BEurope%252C%2B2011.%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid1803.111905%26rft_id%253Dinfo%253Apmid%252F22376991%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.3201/eid1803.111905&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22376991&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F10%2F492835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.11">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Mongkolrattanothai</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Naccache</span>
          <span class="cit-name-given-names">SN</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bender</span>
          <span class="cit-name-given-names">JM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Samayoa</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pham</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yu</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dien Bard</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Miller</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Aldrovandi</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chiu</span>
          <span class="cit-name-given-names">CY</span>
         </span>: 
         <span class="cit-article-title">Neurobrucellosis: Unexpected Answer From Metagenomic Next-Generation Sequencing</span>. 
         <abbr class="cit-jnl-abbrev">Journal of the Pediatric Infectious Diseases Society</abbr>
         <span class="cit-pub-date">2017</span>, 
         <span class="cit-vol">6</span>(
         <span class="cit-issue">4</span>):
         <span class="cit-fpage">393</span>–
         <span class="cit-lpage">398</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BPediatric%2BInfectious%2BDiseases%2BSociety%26rft.volume%253D6%26rft.spage%253D393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.12">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">van Boheemen</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">de Graaf</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lauber</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bestebroer</span>
          <span class="cit-name-given-names">TM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Raj</span>
          <span class="cit-name-given-names">VS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zaki</span>
          <span class="cit-name-given-names">AM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Osterhaus</span>
          <span class="cit-name-given-names">AD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Haagmans</span>
          <span class="cit-name-given-names">BL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gorbalenya</span>
          <span class="cit-name-given-names">AE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Snijder</span>
          <span class="cit-name-given-names">EJ</span>
         </span>
         <span class="cit-etal">et al</span>: 
         <span class="cit-article-title">Genomic characterization of a newly discovered coronavirus associated with acute respiratory distress syndrome in humans</span>. 
         <abbr class="cit-jnl-abbrev">mBio</abbr>
         <span class="cit-pub-date">2012</span>, 
         <span class="cit-vol">3</span>(
         <span class="cit-fpage">6</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DmBio%26rft.volume%253D3%26rft.spage%253D6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.13" data-doi="10.3201/eid2101.140766">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kohl</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Brinkmann</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dabrowski</span>
          <span class="cit-name-given-names">PW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Radonic</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nitsche</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kurth</span>
          <span class="cit-name-given-names">A</span>
         </span>: 
         <span class="cit-article-title">Protocol for metagenomic virus detection in clinical specimens</span>. 
         <abbr class="cit-jnl-abbrev">Emerging infectious diseases</abbr>
         <span class="cit-pub-date">2015</span>, 
         <span class="cit-vol">21</span>(
         <span class="cit-issue">1</span>):
         <span class="cit-fpage">48</span>–
         <span class="cit-lpage">57</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEmerging%2Binfectious%2Bdiseases%26rft.volume%253D21%26rft.spage%253D48%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid2101.140766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.3201/eid2101.140766&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-492835v1.14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Parker</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">J</span>
         </span>: 
         <span class="cit-article-title">Application of next generation sequencing for the detection of human viral pathogens in clinical specimens</span>. 
         <abbr class="cit-jnl-abbrev">Journal of clinical virology: the official publication of the Pan American Society for Clinical Virology</abbr>
         <span class="cit-pub-date">2017</span>, 
         <span class="cit-vol">86</span>:
         <span class="cit-fpage">20</span>–
         <span class="cit-lpage">26</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bclinical%2Bvirology%253A%2Bthe%2Bofficial%2Bpublication%2Bof%2Bthe%2BPan%2BAmerican%2BSociety%2Bfor%2BClinical%2BVirology%26rft.volume%253D86%26rft.spage%253D20%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zou</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tang</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lei</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhuang</span>
          <span class="cit-name-given-names">L</span>
         </span>
         <span class="cit-etal">et al</span>: 
         <span class="cit-article-title">Simultaneous virus identification and characterization of severe unexplained pneumonia cases using a metagenomics sequencing technique</span>. 
         <abbr class="cit-jnl-abbrev">Science China Life sciences</abbr>
         <span class="cit-pub-date">2017</span>, 
         <span class="cit-vol">60</span>(
         <span class="cit-issue">3</span>):
         <span class="cit-fpage">279</span>–
         <span class="cit-lpage">286</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%2BChina%2BLife%2Bsciences%26rft.volume%253D60%26rft.spage%253D279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.16" data-doi="10.1371/journal.pone.0027735">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wylie</span>
          <span class="cit-name-given-names">KM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mihindukulasuriya</span>
          <span class="cit-name-given-names">KA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sodergren</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Weinstock</span>
          <span class="cit-name-given-names">GM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Storch</span>
          <span class="cit-name-given-names">GA</span>
         </span>: 
         <span class="cit-article-title">Sequence analysis of the human virome in febrile and afebrile children</span>. 
         <abbr class="cit-jnl-abbrev">PloS one</abbr>
         <span class="cit-pub-date">2012</span>, 
         <span class="cit-vol">7</span>(
         <span class="cit-issue">6</span>):
         <span class="cit-fpage">e27735</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DWylie%26rft.auinit1%253DK.%2BM.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253De27735%26rft.epage%253De27735%26rft.atitle%253DSequence%2Banalysis%2Bof%2Bthe%2Bhuman%2Bvirome%2Bin%2Bfebrile%2Band%2Bafebrile%2Bchildren.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0027735%26rft_id%253Dinfo%253Apmid%252F22719819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0027735&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22719819&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F10%2F492835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">17.</span>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.17" data-doi="10.1128/JCM.00200-11">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Loens</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">van Loon</span>
          <span class="cit-name-given-names">AM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Coenjaerts</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">van Aarle</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Goossens</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wallace</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Claas</span>
          <span class="cit-name-given-names">EJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ieven</span>
          <span class="cit-name-given-names">M</span>
         </span>: 
         <span class="cit-article-title">Performance of different mono- and multiplex nucleic acid amplification tests on a multipathogen external quality assessment panel</span>. 
         <abbr class="cit-jnl-abbrev">Journal of clinical microbiology</abbr>
         <span class="cit-pub-date">2012</span>, 
         <span class="cit-vol">50</span>(
         <span class="cit-issue">3</span>):
         <span class="cit-fpage">977</span>–
         <span class="cit-lpage">987</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bclinical%2Bmicrobiology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.00200-11%26rft_id%253Dinfo%253Apmid%252F22170925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjg6IjUwLzMvOTc3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMTIvMTAvNDkyODM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">18.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-492835v1.18" data-doi="10.1093/nar/18.24.7213">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Welsh</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">McClelland</span>
          <span class="cit-name-given-names">M</span>
         </span>: 
         <span class="cit-article-title">Fingerprinting genomes using PCR with arbitrary primers</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr>
         <span class="cit-pub-date">1990</span>, 
         <span class="cit-vol">18</span>(
         <span class="cit-issue">24</span>):
         <span class="cit-fpage">7213</span>–
         <span class="cit-lpage">7218</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DWelsh%26rft.auinit1%253DJ.%26rft.volume%253D18%26rft.issue%253D24%26rft.spage%253D7213%26rft.epage%253D7218%26rft.atitle%253DFingerprinting%2Bgenomes%2Busing%2BPCR%2Bwith%2Barbitrary%2Bprimers%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F18.24.7213%26rft_id%253Dinfo%253Apmid%252F2259619%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/18.24.7213&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=2259619&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F10%2F492835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1990EQ47900001&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">19.</span>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.19" data-doi="10.1006/jmbi.1990.9999">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Altschul</span>
          <span class="cit-name-given-names">SF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gish</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Miller</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Myers</span>
          <span class="cit-name-given-names">EW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lipman</span>
          <span class="cit-name-given-names">DJ</span>
         </span>: 
         <span class="cit-article-title">Basic local alignment search tool</span>. 
         <abbr class="cit-jnl-abbrev">Journal of molecular biology</abbr>
         <span class="cit-pub-date">1990</span>, 
         <span class="cit-vol">215</span>(
         <span class="cit-issue">3</span>):
         <span class="cit-fpage">403</span>–
         <span class="cit-lpage">410</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=2231712&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F10%2F492835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1990ED16700008&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">20.</span>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.20" data-doi="10.1016/j.ymeth.2013.03.009">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wery</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Descrimes</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Thermes</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gautheret</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Morillon</span>
          <span class="cit-name-given-names">A</span>
         </span>: 
         <span class="cit-article-title">Zinc-mediated RNA fragmentation allows robust transcript reassembly upon whole transcriptome RNA-Seq</span>. 
         <abbr class="cit-jnl-abbrev">Methods (San Diego, Calif)</abbr>
         <span class="cit-pub-date">2013</span>, 
         <span class="cit-vol">63</span>(
         <span class="cit-issue">1</span>):
         <span class="cit-fpage">25</span>–
         <span class="cit-lpage">31</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMethods%2B%2528San%2BDiego%252C%2BCalif%2529%26rft.volume%253D63%26rft.spage%253D25%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ymeth.2013.03.009%26rft_id%253Dinfo%253Apmid%252F23523657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.ymeth.2013.03.009&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23523657&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F10%2F492835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">21.</span>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-web" id="cit-492835v1.21">
       <div class="cit-metadata">
        <cite>
         <span class="cit-article-title">New England BioLabs Inc</span>. [
         <a href="https://www.neb.com/products/e7420-nebnext-ultra-directional-rna-library-prep-kit-for-illumina#Product%20Information">https://www.neb.com/products/e7420-nebnext-ultra-directional-rna-library-prep-kit-for-illumina#Product%20Information</a>] Version 8.0.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">22.</span>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-web" id="cit-492835v1.22">
       <div class="cit-metadata unstructured">[
        <a href="http://biopet-docs.readthedocs.io/en/stable/">http://biopet-docs.readthedocs.io/en/stable/</a>]
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">23.</span>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-web" id="cit-492835v1.23">
       <div class="cit-metadata unstructured">[
        <a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>]
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">24.</span>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.24">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Joshi</span>
          <span class="cit-name-given-names">NA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fass</span>
          <span class="cit-name-given-names">JN</span>
         </span>: 
         <span class="cit-article-title">Sickle: A sliding-window, adaptive, quality-based trimming tool for FastQ files</span>. 
         <span class="cit-pub-date">2011</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">25.</span>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.25">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Martin</span>
          <span class="cit-name-given-names">M</span>
         </span>: 
         <span class="cit-article-title">Cutadept removes adapter sequences from high-throughput sequencing reads</span>. 
         <abbr class="cit-jnl-abbrev">EMBnetjournal</abbr>
         <span class="cit-pub-date">2011</span>, 
         <span class="cit-vol">17</span>(
         <span class="cit-issue">10</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">26.</span>
      <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.26" data-doi="10.1101/gr.210641.116">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kim</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Song</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Breitwieser</span>
          <span class="cit-name-given-names">FP</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Salzberg</span>
          <span class="cit-name-given-names">SL</span>
         </span>: 
         <span class="cit-article-title">Centrifuge: rapid and sensitive classification of metagenomic sequences</span>. 
         <abbr class="cit-jnl-abbrev">Genome research</abbr>
         <span class="cit-pub-date">2016</span>, 
         <span class="cit-vol">26</span>(
         <span class="cit-issue">12</span>):
         <span class="cit-fpage">1721</span>–
         <span class="cit-lpage">1729</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.210641.116%26rft_id%253Dinfo%253Apmid%252F27852649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8xMi8xNzIxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMTIvMTAvNDkyODM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">27.</span>
      <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.27" data-doi="10.1093/nar/gkv1189">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">O’Leary</span>
          <span class="cit-name-given-names">NA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wright</span>
          <span class="cit-name-given-names">MW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Brister</span>
          <span class="cit-name-given-names">JR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ciufo</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Haddad</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">McVeigh</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rajput</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Robbertse</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Smith-White</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ako-Adjei</span>
          <span class="cit-name-given-names">D</span>
         </span>
         <span class="cit-etal">et al</span>: 
         <span class="cit-article-title">Reference sequence (RefSeq) database at NCBI: current status, taxonomic expansion, and functional annotation</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr>
         <span class="cit-pub-date">2016</span>, 
         <span class="cit-vol">44</span>(
         <span class="cit-issue">D1</span>):
         <span class="cit-fpage">D733</span>–
         <span class="cit-lpage">745</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1189%26rft_id%253Dinfo%253Apmid%252F26553804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkv1189&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26553804&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F10%2F492835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">28.</span>
      <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.28" data-doi="10.1186/1471-2105-12-385">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ondov</span>
          <span class="cit-name-given-names">BD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bergman</span>
          <span class="cit-name-given-names">NH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Phillippy</span>
          <span class="cit-name-given-names">AM</span>
         </span>: 
         <span class="cit-article-title">Interactive metagenomic visualization in a Web browser</span>. 
         <abbr class="cit-jnl-abbrev">BMC bioinformatics</abbr>
         <span class="cit-pub-date">2011</span>, 
         <span class="cit-vol">12</span>:
         <span class="cit-fpage">385</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DOndov%26rft.auinit1%253DB.%2BD.%26rft.volume%253D12%26rft.spage%253D385%26rft.epage%253D385%26rft.atitle%253DInteractive%2Bmetagenomic%2Bvisualization%2Bin%2Ba%2BWeb%2Bbrowser.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-385%26rft_id%253Dinfo%253Apmid%252F21961884%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-385&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21961884&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F10%2F492835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">29.</span>
      <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a>
      <div class="cit ref-cit ref-web" id="cit-492835v1.29">
       <div class="cit-metadata unstructured">
        <a href="https://github.com/biowdl/virus-assembly">https://github.com/biowdl/virus-assembly</a>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">30.</span>
      <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a>
      <div class="cit ref-cit ref-web" id="cit-492835v1.30">
       <div class="cit-metadata unstructured">
        <a href="https://github.com/lh3/seqtk">https://github.com/lh3/seqtk</a>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">31.</span>
      <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.31">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bankevich</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nurk</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Antipov</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gurevich</span>
          <span class="cit-name-given-names">AA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dvorkin</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kulikov</span>
          <span class="cit-name-given-names">AS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lesin</span>
          <span class="cit-name-given-names">VM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nikolenko</span>
          <span class="cit-name-given-names">SI</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pham</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Prjibelski</span>
          <span class="cit-name-given-names">AD</span>
         </span>
         <span class="cit-etal">et al</span>: 
         <span class="cit-article-title">SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing</span>. 
         <abbr class="cit-jnl-abbrev">Journal of computational biology: a journal of computational molecular cell biology</abbr>
         <span class="cit-pub-date">2012</span>, 
         <span class="cit-vol">19</span>(
         <span class="cit-issue">5</span>):
         <span class="cit-fpage">455</span>–
         <span class="cit-lpage">477</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%253A%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.volume%253D19%26rft.spage%253D455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">32.</span>
      <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.32">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">H</span>
         </span>: 
         <span class="cit-article-title">Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM</span>, vol. 
         <span class="cit-vol">1303</span>; 
         <span class="cit-pub-date">2013</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">33.</span>
      <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.33" data-doi="10.1093/bioinformatics/btp352">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Handsaker</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wysoker</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fennell</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ruan</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Homer</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Marth</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Abecasis</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Durbin</span>
          <span class="cit-name-given-names">R</span>
         </span>: 
         <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics (Oxford, England)</abbr>
         <span class="cit-pub-date">2009</span>, 
         <span class="cit-vol">25</span>(
         <span class="cit-issue">16</span>):
         <span class="cit-fpage">2078</span>–
         <span class="cit-lpage">2079</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%2B%2528Oxford%252C%2BEngland%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19505943&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F10%2F492835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000268808600014&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">34.</span>
      <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.34">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Sinha</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Stanley</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gulati</span>
          <span class="cit-name-given-names">GS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ezran</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Travaglini</span>
          <span class="cit-name-given-names">KJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wei</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chan</span>
          <span class="cit-name-given-names">CKF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nabhan</span>
          <span class="cit-name-given-names">AN</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Su</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Morganti</span>
          <span class="cit-name-given-names">RM</span>
         </span>
         <span class="cit-etal">et al</span>: 
         <span class="cit-article-title">Index Switching Causes “Spreading-Of-Signal” Among Multiplexed Samples In Illumina HiSeq 4000 DNA Sequencing</span>. 
         <abbr class="cit-jnl-abbrev">bioRxiv</abbr>
         <span class="cit-pub-date">2017</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">35.</span>
      <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.35" data-doi="10.1371/journal.pone.0016028">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Orozovic</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Orozovic</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lennerstrand</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Olsen</span>
          <span class="cit-name-given-names">B</span>
         </span>: 
         <span class="cit-article-title">Detection of resistance mutations to antivirals oseltamivir and zanamivir in avian influenza A viruses isolated from wild birds</span>. 
         <abbr class="cit-jnl-abbrev">PloS one</abbr>
         <span class="cit-pub-date">2011</span>, 
         <span class="cit-vol">6</span>(
         <span class="cit-issue">1</span>):
         <span class="cit-fpage">e16028</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DOrozovic%26rft.auinit1%253DG.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253De16028%26rft.epage%253De16028%26rft.atitle%253DDetection%2Bof%2Bresistance%2Bmutations%2Bto%2Bantivirals%2Boseltamivir%2Band%2Bzanamivir%2Bin%2Bavian%2Binfluenza%2BA%2Bviruses%2Bisolated%2Bfrom%2Bwild%2Bbirds.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0016028%26rft_id%253Dinfo%253Apmid%252F21253602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0016028&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21253602&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F10%2F492835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">36.</span>
      <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.36">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hsieh</span>
          <span class="cit-name-given-names">NH</span>
         </span>. 
         <span class="cit-article-title">Assessing the oseltamivir-induced resistance risk and implications for influenza infection control strategies</span>. 
         <abbr class="cit-jnl-abbrev">Infect Drug Resist</abbr>. 
         <span class="cit-pub-date">2017</span>;
         <span class="cit-vol">10</span>:
         <span class="cit-fpage">215</span>–
         <span class="cit-lpage">226</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DInfect%2BDrug%2BResist%26rft.volume%253D10%26rft.spage%253D215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">37.</span>
      <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.37" data-doi="10.1128/JCM.03050-15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hasan</span>
          <span class="cit-name-given-names">MR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rawat</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tang</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jithesh</span>
          <span class="cit-name-given-names">PV</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Thomas</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tan</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tilley</span>
          <span class="cit-name-given-names">P</span>
         </span>: 
         <span class="cit-article-title">Depletion of Human DNA in Spiked Clinical Specimens for Improvement of Sensitivity of Pathogen Detection by Next-Generation Sequencing</span>. 
         <abbr class="cit-jnl-abbrev">Journal of clinical microbiology</abbr>
         <span class="cit-pub-date">2016</span>, 
         <span class="cit-vol">54</span>(
         <span class="cit-issue">4</span>):
         <span class="cit-fpage">919</span>–
         <span class="cit-lpage">927</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bclinical%2Bmicrobiology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.03050-15%26rft_id%253Dinfo%253Apmid%252F26763966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjg6IjU0LzQvOTE5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMTIvMTAvNDkyODM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">38.</span>
      <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.38" data-doi="10.1371/journal.pone.0016142">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ninove</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nougairede</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gazin</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Thirion</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Delogu</span>
          <span class="cit-name-given-names">I</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zandotti</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Charrel</span>
          <span class="cit-name-given-names">RN</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">De Lamballerie</span>
          <span class="cit-name-given-names">X</span>
         </span>: 
         <span class="cit-article-title">RNA and DNA bacteriophages as molecular diagnosis controls in clinical virology: a comprehensive study of more than 45,000 routine PCR tests</span>. 
         <abbr class="cit-jnl-abbrev">PloS one</abbr>
         <span class="cit-pub-date">2011</span>, 
         <span class="cit-vol">6</span>(
         <span class="cit-issue">2</span>):
         <span class="cit-fpage">e16142</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DNinove%26rft.auinit1%253DL.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253De16142%26rft.epage%253De16142%26rft.atitle%253DRNA%2Band%2BDNA%2Bbacteriophages%2Bas%2Bmolecular%2Bdiagnosis%2Bcontrols%2Bin%2Bclinical%2Bvirology%253A%2Ba%2Bcomprehensive%2Bstudy%2Bof%2Bmore%2Bthan%2B45%252C000%2Broutine%2BPCR%2Btests.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0016142%26rft_id%253Dinfo%253Apmid%252F21347398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0016142&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21347398&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F10%2F492835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">39.</span>
      <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.39">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Schlaberg</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chiu</span>
          <span class="cit-name-given-names">CY</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Miller</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Procop</span>
          <span class="cit-name-given-names">GW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Weinstock</span>
          <span class="cit-name-given-names">G</span>
         </span>: 
         <span class="cit-article-title">Validation of Metagenomic Next-Generation Sequencing Tests for Universal Pathogen Detection</span>. 
         <abbr class="cit-jnl-abbrev">Archives of pathology &amp;; laboratory medicine</abbr>
         <span class="cit-pub-date">2017</span>, 
         <span class="cit-vol">141</span>(
         <span class="cit-issue">6</span>):
         <span class="cit-fpage">776</span>–
         <span class="cit-lpage">786</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DArchives%2Bof%2Bpathology%2B%2526%2Blaboratory%2Bmedicine%26rft.volume%253D141%26rft.spage%253D776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">40.</span>
      <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.40">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">van der Valk</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vezzi</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ormestad</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dalen</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Guschanski</span>
          <span class="cit-name-given-names">K</span>
         </span>: 
         <span class="cit-article-title">Estimating the rate of index hopping on the Illumina HiSeq X platform</span>. 
         <abbr class="cit-jnl-abbrev">bioRxiv</abbr>
         <span class="cit-pub-date">2018</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">41.</span>
      <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a>
      <div class="cit ref-cit ref-confproc" id="cit-492835v1.41">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Naccache</span>
          <span class="cit-name-given-names">SN</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hackett</span>
          <span class="cit-name-given-names">J</span>, 
          <span class="cit-name-suffix">Jr.</span>.
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Delwart</span>
          <span class="cit-name-given-names">EL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chiu</span>
          <span class="cit-name-given-names">CY</span>
         </span>: 
         <span class="cit-article-title">Concerns over the origin of NIH-CQV, a novel virus discovered in Chinese patients with seronegative hepatitis</span>. 
         <span class="cit-conf-name">Proceedings of the National Academy of Sciences of the United States of America</span>
         <span class="cit-conf-date">2014</span>, 
         <span class="cit-vol">111</span>(
         <span class="cit-issue">11</span>):
         <span class="cit-fpage">E976</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">42.</span>
      <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-492835v1.42" data-doi="10.1186/s12879-018-3446-5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bal</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pichon</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Picard</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Casalegno</span>
          <span class="cit-name-given-names">JS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Valette</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schuffenecker</span>
          <span class="cit-name-given-names">I</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Billard</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vallet</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vilchez</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cheynet</span>
          <span class="cit-name-given-names">V</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Oriol</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Trouillet-Assant</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gillet</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lina</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Brengel-Pesce</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Morfin</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Josset</span>
          <span class="cit-name-given-names">L</span>
         </span>: 
         <span class="cit-article-title">Quality control implementation for universal characterization of DNA and RNA viruses in clinical respiratory samples using single metagenomics next-generation sequencing workflow</span>. 
         <abbr class="cit-jnl-abbrev">BMC Infect Dis</abbr>
         <span class="cit-pub-date">2018</span>: 
         <span class="cit-vol">18</span>(
         <span class="cit-issue">1</span>):
         <span class="cit-fpage">537</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2BInfect%2BDis%26rft.volume%253D18%26rft.spage%253D537%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12879-018-3446-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/s12879-018-3446-5&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
