<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">ACS Cent Sci</span><span class="iso-abbrev" title="iso-abbrev">ACS Cent Sci</span><span class="publisher-id" title="publisher-id">oc</span><span class="coden" title="coden">acscii</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">ACS Central Science</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">2374-7943</span><span class="issn-epub" tagx="issn" title="issn-epub">2374-7951</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">American Chemical Society</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7048371">7048371</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1021/acscentsci.9b01071">10.1021/acscentsci.9b01071</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Mesoscale All-Atom Influenza Virus Simulations Suggest New Substrate Binding Mechanism</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Durrant</span><span class="given-names" tagx="given-names" title="given-names">Jacob D.</span></span><a href="#aff1">†</a><a href="#notes2">§</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kochanek</span><span class="given-names" tagx="given-names" title="given-names">Sarah E.</span></span><a href="#aff2">‡</a><a href="#notes2">§</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Casalino</span><span class="given-names" tagx="given-names" title="given-names">Lorenzo</span></span><a href="#aff2">‡</a><a href="#notes2">§</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ieong</span><span class="given-names" tagx="given-names" title="given-names">Pek U.</span></span><a href="#aff2">‡</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dommer</span><span class="given-names" tagx="given-names" title="given-names">Abigail C.</span></span><a href="#aff2">‡</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amaro</span><span class="given-names" tagx="given-names" title="given-names">Rommie E.</span></span><a href="#cor1">*</a><a href="#aff2">‡</a></span><span class="citation_author_institution" id="aff1">[†], <span class="institution" title="institution">University of Pittsburgh</span><span class="country" tagx="country" title="country">United States</span></span><span class="citation_author_institution" id="aff2">[‡], <span class="institution" title="institution">University of California, San Diego</span><span class="country" tagx="country" title="country">United States</span></span></div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">*</span>E-mail: <span class="email" tagx="email" title="email">ramaro@ucsd.edu</span>.</div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-02-02</span></span><span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2020-02-02</span></span><span class="volume" tagx="volume" title="volume">6</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">189</span><span class="lpage" tagx="lpage" title="lpage">196</span><span class="history" title="history"><span class="received" title="received">received: 2019-10-19</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">American Chemical Society</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article published under an ACS AuthorChoice <a href="http://pubs.acs.org/page/policy/authorchoice_termsofuse.html">License</a>, which permits copying and redistribution of the article or any adaptations for non-commercial purposes.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p content-type="toc-graphic"> <div class="graphic_UNKNOWN" id="ab-tgr1" /> </p> <p>Influenza virus circulates in human, avian, and swine hosts, causing seasonal epidemic and occasional pandemic outbreaks. Influenza neuraminidase, a viral surface glycoprotein, has two sialic acid binding sites. The catalytic (primary) site, which also binds inhibitors such as oseltamivir carboxylate, is responsible for cleaving the sialic acid linkages that bind viral progeny to the host cell. In contrast, the functional annotation of the secondary site remains unclear. Here, we better characterize these two sites through the development of an all-atom, explicitly solvated, and experimentally based integrative model of the pandemic influenza A H1N1 2009 viral envelope, containing ∼160 million atoms and spanning ∼115 nm in diameter. Molecular dynamics simulations of this crowded subcellular environment, coupled with Markov state model theory, provide a novel framework for studying realistic molecular systems at the mesoscale and allow us to quantify the kinetics of the neuraminidase 150-loop transition between the open and closed states. An analysis of chloride ion occupancy along the neuraminidase surface implies a potential new role for the neuraminidase secondary site, wherein the terminal sialic acid residues of the linkages may bind before transfer to the primary site where enzymatic cleavage occurs. Altogether, our work breaks new ground for molecular simulation in terms of size, complexity, and methodological analyses of the components. It also provides fundamental insights into the understanding of substrate recognition processes for this vital influenza drug target, suggesting a new strategy for the development of anti-influenza therapeutics.</p> </div> <div class="abstract" title="abstract"> <p>Molecular dynamics simulations and chloride ion analyses provide fundamental insights into the understanding of substrate recognition processes for two sialic binding sites of influenza neuraminidase.</p> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="document-id-old-9" value="oc9b01071" /> <meta name="document-id-new-14" value="oc9b01071" /> <meta name="ccc-price" value="" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Influenza virus infection is responsible for millions of deaths worldwide each year. The Center for Disease Control estimates that pandemic influenza A H1N1 2009 (pH1N1) affected 60.8 million people, resulting in 12468 casualties in the United States alone.<sup><a href="#ref1">1</a>,<a href="#ref2">2</a></sup> Along with others, this strain dramatically contributes to yearly epidemics, continuously fueling concerns about the emergence of a new pandemic strain. In addition, the increasingly widespread resistance to antiviral medications is compounding this threat,<sup><a href="#ref3">3</a></sup> thus requiring the development of novel approaches for the prevention and treatment of influenza virus infection. One such strategy is to target the viral surface glycoprotein neuraminidase (NA), which promotes viral progeny release from the host cell by cleaving terminal sialic acid residues.<sup><a href="#ref4">4</a>−<a href="#ref6">6</a></sup> Previous work has identified the importance of characterizing the dynamics of the NA catalytic site for drug design,<sup><a href="#ref7">7</a>−<a href="#ref12">12</a></sup> understanding mechanisms of antiviral resistance,<sup><a href="#ref13">13</a></sup> and deciphering the mechanisms underlying substrate binding.<sup><a href="#ref14">14</a>−<a href="#ref18">18</a></sup></p> <p>The catalytic (primary, 1°) site of NA is highly flexible, in part due to the adjacent 150- and 430-loops (residues 147–152 and 429–433, respectively, N2 numbering).<sup><a href="#ref11">11</a>,<a href="#ref14">14</a>,<a href="#ref19">19</a></sup> The significance of this flexibility is highlighted by the structural comparison of the phylogenetically distinct group-1 (N1, N4, N5, and N8) and group-2 (N2, N3, N6, N7, and N9) NAs, which illustrates that the opening of the 150-loop in the group-1 structures leads to the formation of the so-called 150-cavity<sup><a href="#ref12">12</a></sup> that can bind compounds with increased specificity and potency.<sup><a href="#ref10">10</a></sup> However, crystal structures of pH1N1 NA (pN1) reveal that, unlike all other group-1 NAs, its 150-loop is closed, and therefore no 150-cavity is present.<sup><a href="#ref20">20</a></sup> In contrast, previous investigations utilizing molecular dynamics (MD) simulations have found that the 150-loop of pN1 is in the open state ∼60–65% of the time.<sup><a href="#ref13">13</a>,<a href="#ref19">19</a>,<a href="#ref21">21</a></sup></p> <p>NA also contains a secondary (2°) sialic acid binding site adjacent to the catalytic site. This site was first identified as a hemadsorption site in avian-origin influenza NAs<sup><a href="#ref22">22</a>−<a href="#ref26">26</a></sup> and was not initially believed to be present in swine-origin strains due to nonconservation of critical residues at this site.<sup><a href="#ref24">24</a>,<a href="#ref27">27</a></sup> However, more recent studies provide support for the presence of a 2° site in swine-origin influenza NAs, including pN1.<sup><a href="#ref16">16</a>,<a href="#ref17">17</a></sup> The precise mechanism by which this 2° site functions remains unclear; however, a number of studies have demonstrated its role in receptor binding<sup><a href="#ref28">28</a>−<a href="#ref32">32</a></sup> and catalytic efficiency.<sup><a href="#ref28">28</a>,<a href="#ref29">29</a></sup> In addition, previous Brownian dynamics (BD) simulations of single glycoproteins and various ligands suggested that both endogenous substrates and the drug oseltamivir carboxylate bind faster to the 2° site than the 1° site (i.e., the <i>k</i><sub>on</sub> rate is 2- and 7-fold higher for the N1 and N2 2° site, respectively, vs the corresponding 1° site).<sup><a href="#ref17">17</a></sup> Finally, the 2° site has recently been identified as a target for a novel influenza virus inhibitor,<sup><a href="#ref33">33</a>,<a href="#ref34">34</a></sup> further highlighting the need to understand its role in viral infectivity.</p> <p>To study the 1° and 2° sites in the context of the viral surface, we used integrative modeling to construct a fully atomistic model of the pH1N1 viral envelope (<a href="#fig1">Figure </a><a href="#fig1">1</a>A, and <a href="http://pubs.acs.org/doi/suppl/10.1021/acscentsci.9b01071/suppl_file/oc9b01071_si_001.pdf">Figure S1</a> of the <a href="http://pubs.acs.org/doi/suppl/10.1021/acscentsci.9b01071/suppl_file/oc9b01071_si_001.pdf">Supporting Information</a> (SI)). The model was built using high-resolution crystallographic structures of individual glycoproteins (∼1.9–2.6 Å resolution)<sup><a href="#ref20">20</a>,<a href="#ref35">35</a></sup> that were spatially positioned according to a lower-resolution cryo-electron tomography (cryo-ET, ∼16–20 Å resolution) map of a viral particle.<sup><a href="#ref36">36</a></sup> Our viral envelope construct includes 30 NA tetramers (120 monomers) and 236 hemagglutinin (HA) trimers (708 monomers) embedded in a phospholipid bilayer, with realistic density and patterning taken from the cryo-ET. The entire pH1N1 all-atom system modeled here amounts to ∼160 million atoms (fully solvated) and is ∼115 nm in diameter. As such, it is among the largest biophysical systems yet studied with all-atom molecular dynamics.<sup><a href="#ref37">37</a>,<a href="#ref38">38</a></sup> A complete description of the integrative modeling and computational approaches used to build the viral envelope is provided in the <a href="http://pubs.acs.org/doi/suppl/10.1021/acscentsci.9b01071/suppl_file/oc9b01071_si_001.pdf">SI</a> (<a href="http://pubs.acs.org/doi/suppl/10.1021/acscentsci.9b01071/suppl_file/oc9b01071_si_001.pdf">section S1.3</a>). Additional details can also be found in Amaro et al.<sup><a href="#ref39">39</a></sup></p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Mesoscale simulations enhance conformational sampling of the viral glycoproteins. (A) A fully intact all-atom model of the influenza A H1N1 2009 (pH1N1) viral envelope, containing over 160 million atoms, shown without explicit water molecules, was simulated with all-atom MD simulations. Hemagglutinin (HA) glycoproteins shown in royal (dark) blue, neuraminidase (NA) glycoproteins shown in ice (light) blue. (B) Top view of a single NA monomer in surface representation with the catalytic site (white), secondary site (yellow), 150-loop (green), and 430-loop (red) highlighted. (C–E) Principal component analysis (PCA) was performed by considering the motions of the C<sub>α</sub> atoms of 19 1° pocket residues. PCA histograms were independently normalized so the bins containing the minimum and maximum number of points were blue and red, respectively. (C) PCA of the four monomers sampled during a single-NA-tetramer simulation ("single-glycoprotein�?). (D) PCA of the 120 monomeric trajectories extracted during the last 8.33 ns of the viral envelope simulation ("terminal-envelope�?). (E) PCA of all 120 monomeric trajectories extracted from the full simulation of the viral envelope ("complete-envelope�?).</p>   </div> <p>Over the past decade, studies of viruses at the molecular and coarse-grained (CG) levels have given unique insights into these systems, complementing and extending available experimental data by providing highly detailed models at never-before-seen scales as well as suggesting testable biological hypotheses (predictions).<sup><a href="#ref40">40</a>,<a href="#ref41">41</a></sup> Work by Schulten and co-workers established the first explicitly solvated atomic MD simulation of an intact virus, the satellite tobacco mosaic virus (∼17 nm diameter, ∼1 million atoms, 50 ns dynamics), in 2006.<sup><a href="#ref42">42</a></sup> Zink and Grubmuller in 2009 used steered MD to explore the dynamics of the explicitly solvated icosahedral shell of the southern bean mosaic virus (∼36 nm diameter, ∼4.5 million atoms, 100 ns).<sup><a href="#ref43">43</a></sup> In 2010, Ayton and Voth developed and simulated an implicitly solvated CG representation of the immature HIV-1 virion (∼125 nm diameter, 280,000 CG particles).<sup><a href="#ref44">44</a></sup> In 2012, Larsson and co-workers simulated with explicitly solvated all-atom MD the satellite tobacco necrosis virus (∼17 nm diameter, ∼1.2 million atoms, ∼1 us),<sup><a href="#ref44">44</a></sup> and Roberts et al. developed a fully atomic poliovirus (∼30 nm diameter, ∼2.8–4 million atoms, 50 ns).<sup><a href="#ref45">45</a></sup> In 2013, Schulten and co-workers built and simulated a fully atomic representation of the HIV capsid (∼70 nm diameter, ∼64 million atoms, ∼100 ns),<sup><a href="#ref46">46</a></sup> and Andoh et al. in 2014 simulated an all-atom poliovirus capsid (∼30 nm diameter, ∼6.5 million atoms, ∼200 ns).<sup><a href="#ref47">47</a></sup> Sansom and colleagues in 2015 reported an explicitly solvated CG influenza virus simulation (∼80 nm diameter, 5 million particles).<sup><a href="#ref48">48</a></sup> In two separate studies in 2016, Reddy and Sansom<sup><a href="#ref49">49</a></sup> and Bond, Verma, and co-workers,<sup><a href="#ref50">50</a></sup> reported CG simulations of the Dengue viral membrane (∼50 nm diameter, ∼1 million particles). In addition to the rich structural, dynamical, and biophysical insights that these studies each provided, the investigations have collectively pushed the capabilities of molecular simulation, often relying on the world’s fastest and most advanced computing architectures.</p> <p>As the first explicitly solvated atomic-scale simulation of a viral lipid envelope (∼115 nm diameter, ∼160 million particles, ∼121 ns), the work reported here breaks new ground in molecular simulation. To further characterize the structural dynamics of the viral envelope and its glycoproteins, we combined our mesoscale all-atom MD simulations with Markov state model (MSM) theory,<sup><a href="#ref51">51</a>−<a href="#ref53">53</a></sup> thus enabling the extraction of long-time-scale (e.g., microseconds) individual glycoprotein dynamics in a crowded environment from the short-time-scale MD (e.g., nanoseconds) of the fully intact viral surface. The accuracy and utility of MSMs have been demonstrated by experimental validation for many use cases, including protein–protein binding, small-molecule binding kinetics, and protein-folding rate prediction.<sup><a href="#ref54">54</a>−<a href="#ref56">56</a></sup> Correspondingly, the approach reported here, which relies on the many copies of single glycoproteins present within a biologically accurate environment, provides a novel methodological advancement for extracting long-time-scale dynamics from short simulations through the powerful MSM theoretical framework at subcellular and cellular scales.</p> <p>Here, we quantitatively compare how calculated protein dynamics differ when simulating many proteins in a single subcellular environment versus simulating single proteins in isolation. By exploiting the whole pH1N1 viral envelope treated entirely with atomic resolution, this study provides unique insights into the two sialic acid binding sites of NA (e.g., 1° and 2°). Our mesoscale atomic simulations suggest that the NA 1° site is even more flexible than previously appreciated and provide the first rigorous kinetic characterization of the 150-loop dynamics. Furthermore, our work suggests that the 2° site, which is more solvent exposed and, in some strains, has a higher <i>k</i><sub>on</sub> rate than the 1° site,<sup><a href="#ref17">17</a>,<a href="#ref39">39</a></sup> may be responsible for initially capturing sialic acid residues, which are then electrostatically guided to the 1° site for enzymatic cleavage. Within this context, our mesoscale simulations unveil an unprecedented cooperative interplay between the two sites that further illuminates the process of sialic acid/oseltamivir carboxylate recognition and 2° site functional annotation. This fundamental discovery may be used as a rationale for the development of novel anti-influenza small-molecule therapeutics targeting NA.</p> </div> <div class="influenzavirusall-atomsimulations" title="sec"> <div class="title" tagx="title" title="title">
Influenza Virus All-Atom Simulations</div> <p>All-atom MD simulations of the pH1N1 viral envelope were performed using NAMD2.10<sup><a href="#ref57">57</a></sup> and CHARMM36 all-atom additive force fields.<sup><a href="#ref58">58</a></sup> The system was fully solvated with explicit water molecules (TIP3P force field<sup><a href="#ref59">59</a></sup>), while ions were described using Beglov and Roux force fields.<sup><a href="#ref60">60</a></sup> To broaden conformational sampling and more efficiently use supercomputer resources, the initial simulation was forked twice, generating two additional shorter daughter simulations (schematic representation in <a href="http://pubs.acs.org/doi/suppl/10.1021/acscentsci.9b01071/suppl_file/oc9b01071_si_001.pdf">Figure S2</a>). Taken together, these simulations achieved a comprehensive simulation time of ∼121 ns. The complete viral envelope simulation included 30 NA tetramers, yielding 14.5 μs of monomeric simulation (121 ns × 30 tetramers × 4 monomers/tetramer), and 236 HA tetramers, accounting for 85.6 μs of monomeric dynamics (121 ns × 236 tetramers × 3 monomers/trimer). Each glycoprotein structure used to build the initial viral envelope system was taken from fully equilibrated single-glycoprotein MD simulations (see <a href="http://pubs.acs.org/doi/suppl/10.1021/acscentsci.9b01071/suppl_file/oc9b01071_si_001.pdf">sections S1.1–S1.2</a> for computational details relative to these sets of simulations). The viral envelope simulations were run on the Blue Waters petascale supercomputer using 114688 processors, equivalent to 16384 Blue Waters nodes or 4096 physical nodes. The simulation averaged 25.57 steps/sec. Frames were written every 10,000 steps (20 ps), ultimately occupying 11.66 terabytes of disk space. Data analysis drew upon conformations extracted at equally spaced time points from these trajectories. The adopted MD protocol for the viral envelope simulations is fully described in <a href="http://pubs.acs.org/doi/suppl/10.1021/acscentsci.9b01071/suppl_file/oc9b01071_si_001.pdf">section S1.4</a>, including the NAMD input file. The physical properties of the virus (RMSD, RMSF analyses) and its lipid bilayer (curvature, motions) per the simulations are reported in <a href="http://pubs.acs.org/doi/suppl/10.1021/acscentsci.9b01071/suppl_file/oc9b01071_si_001.pdf">sections S1.5–S1.6</a> and shown in <a href="http://pubs.acs.org/doi/suppl/10.1021/acscentsci.9b01071/suppl_file/oc9b01071_si_001.pdf">Figures S3–S9</a>).</p> <p>To explore the flexibility of the 1° pocket (shown in <a href="http://pubs.acs.org/doi/suppl/10.1021/acscentsci.9b01071/suppl_file/oc9b01071_si_001.pdf">Figure S10</a>), we concatenated the MD trajectories of all 120 NA monomers and performed principal component analysis (PCA) of 19 catalytic-pocket-lining NA residues by considering their C<sub>α</sub> atoms (heatmap, <a href="#fig1">Figure </a><a href="#fig1">1</a>C–E). We selected these 19 residues because they are homologous to those within 5 Å of the crystallographic oseltamivir carboxylate from the 2HU4 structure.<sup><a href="#ref12">12</a></sup> PCA details, including indices of the catalytic and active site residues used in the analysis, are provided in <a href="http://pubs.acs.org/doi/suppl/10.1021/acscentsci.9b01071/suppl_file/oc9b01071_si_001.pdf">section S1.7</a>.</p> <p>To judge whether mesoscale simulations enhance conformational sampling, we compared the viral envelope full simulation (referred to as "complete-envelope�?, <a href="#fig1">Figure </a><a href="#fig1">1</a>E) to five long time scale simulations of isolated NA tetramers embedded in small lipid-bilayer patches, described in a previous work ("single-glycoprotein�?, <a href="#fig1">Figure </a><a href="#fig1">1</a>C).<sup><a href="#ref61">61</a></sup> A fair comparison requires that the sampling of the two systems be consistent in terms of the actual simulation length. The viral envelope full simulation sampled 14.5 μs of monomeric dynamics, but the five simulations of isolated NA tetramers sampled only 1.0 μs of monomeric dynamics (5 simulations × 50 ns/simulation × 1 NA tetramer × 4 monomers/tetramer). Thus, to improve the comparison, we considered only the final 8.33 ns of the viral envelope simulation, which is equivalent to 1.0 μs of monomeric dynamics (1 simulation × 8.33 ns/simulation × 30 NA tetramers × 4 monomers/tetramer) (<a href="#fig1">Figure </a><a href="#fig1">1</a>D). We refer to this truncated segment of the full simulation as the "terminal-envelope�? simulation. In all cases, the motions of the C<sub>α</sub> atoms of the same 19 residues were projected onto the first two principal components of the viral envelope NA trajectories, and the resulting heatmaps were compared (<a href="#fig1">Figure </a><a href="#fig1">1</a>C–E). Strikingly, the PCA of the NA catalytic site residues indicates that the viral envelope simulation more thoroughly explored the conformational landscape, even after controlling for total simulation time.</p> <p>To better study the 1° site conformations sampled by the viral envelope simulation, we applied <i>k</i>-means clustering to the PCA points of <a href="#fig1">Figure </a><a href="#fig1">1</a>E. A visual inspection of cluster centroids corresponding to four representative 1° site conformations (shown in <a href="http://pubs.acs.org/doi/suppl/10.1021/acscentsci.9b01071/suppl_file/oc9b01071_si_001.pdf">Figure S10</a>) reveals that R292 and R371, two key residues known to interact with the sialic acid carboxylate group, are the most flexible. In contrast, the carboxylate-stabilizing R152 residue moves outward in only one of the four representative conformations. Other pocket-lining residues such as R118 and D151, which previous works suggest may play a role in the molecular mechanisms of oseltamivir resistance,<sup><a href="#ref13">13</a></sup> are also relatively flexible in the apo state.</p> <p>The PCA analysis demonstrates that the viral envelope simulation more thoroughly sampled distinct 1° pocket states (<a href="#fig1">Figure </a><a href="#fig1">1</a>E) that are scarcely populated in the single glycoprotein (<a href="#fig1">Figure </a><a href="#fig1">1</a>C). This holds true even when comparing the terminal-envelope (<a href="#fig1">Figure </a><a href="#fig1">1</a>D) simulation to the single-glycoprotein simulations (where the total sampling time is equal). The enhanced conformational sampling may simply be a product of the large number of NA copies blanketed across the viral surface; however, we do expect some effects from the viral surface environment, including long-range electrostatic forces and glycoprotein–glycoprotein interactions that only the viral envelope simulation can capture.</p> </div> <div class="pn1catalyticsitestructureanddynamics" title="sec"> <div class="title" tagx="title" title="title">
pN1 Catalytic Site Structure and Dynamics</div> <p>To explore the dynamics of the catalytic site, we analyzed the volumes of the 1° pocket and adjacent regions over the course of the entire viral envelope simulation (120 NA monomers, <a href="#fig2">Figure </a><a href="#fig2">2</a>).<sup><a href="#ref62">62</a></sup> The volumes ranged from 450 to 4440 Å<sup>3</sup>, with an average of 1536 Å<sup>3</sup> (<a href="http://pubs.acs.org/doi/suppl/10.1021/acscentsci.9b01071/suppl_file/oc9b01071_si_001.pdf">Table S1</a> and <a href="http://pubs.acs.org/doi/suppl/10.1021/acscentsci.9b01071/suppl_file/oc9b01071_si_001.pdf">section S1.8</a>). By comparison, the starting crystal structure pN1 (PDB ID: 3NSS(<a href="#ref20">20</a>)) with a closed 150-cavity has a volume of 800 Å<sup>3</sup>, and the structure of a nonpandemic N1 (PDB ID: 2HTY(<a href="#ref12">12</a>)) with an open 150-cavity has a volume of 1088 Å<sup>3</sup>. This indicates that the volume and depth of the catalytic pocket and adjacent regions can increase remarkably over what has been observed in crystal structures (<a href="#fig2">Figure </a><a href="#fig2">2</a>C). Contributing to this additional cavity volume and depth are two novel subpockets near residues G351 and E227, buried deep inside, but contiguous with, the 1° site (<a href="#fig2">Figure </a><a href="#fig2">2</a>B). An analysis with FTMap, a server for mapping ligand binding hot spots in macromolecules,<sup><a href="#ref63">63</a></sup> suggests that the G351 subpocket can accommodate small-molecule ligands. Similar to the 150-cavity and 430-cavity, the G351- and E227-adjacent subpockets may provide new ligand-binding opportunities.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>Volumetric and ligand binding "hot spot�? analyses of the 1° catalytic site and adjacent regions. (A) NA is shown in ice blue ribbon, and the pocket volume is filled with semitransparent gel. The 1° active site, 430-loop, and 150-loop are visible. (B) NA is shown as a solid gradient, and ligand-binding hotspots are metallic. A portion of the surface-rendered protein was removed to facilitate visualization of internal cavities. This NA conformation has a notably open G351 pocket, which has a high propensity to bind ligands. (C) A histogram of the NA catalytic-site volumes sampled during the MD simulations. As reference, the volumes of the same active-site cavity from two crystal structures are indicated with black circled stars. The 3NSS(<a href="#ref20">20</a>) structure (pH1N1 with a closed 150-cavity) has a volume of 800 Å<sup>3</sup>, and the 2HTY(<a href="#ref12">12</a>) structure (H5N1 with an open 150-cavity) has a pocket volume of 1088 Å<sup>3</sup>. The simulated-pocket volumes range from ∼450 to 4440 Å<sup>3</sup> (intervals 3500–4500 not shown), as reported in <a href="http://pubs.acs.org/doi/suppl/10.1021/acscentsci.9b01071/suppl_file/oc9b01071_si_001.pdf">Table S1</a>; the average pocket volume is 1536 Å<sup>3</sup>.</p>   </div> <p>The volumetric and dynamical properties of the 1° site revealed in our simulations suggest that NA can bind many structurally distinct and complex sialoglycan receptors as part of the host-cell recognition process. Indeed, human glycans are vastly diverse in both their sugar composition and configuration (e.g., long, short, biantennary, triantennary, etc.).<sup><a href="#ref64">64</a></sup> As such, the transient deepening and broadening of the 1° NA site may allow the glycoprotein to accommodate bulkier (e.g., long, bi/triantennary) and longer glycan receptors. Given that our simulations model the entire viral envelope, it may be that full-pocket opening only occurs in a crowded viral-surface environment. Alternatively, our simulations may capture full-pocket opening because the viral coat includes many replicates of individual glycoproteins, enabling extensive conformational sampling.</p> <p>Using the intramolecular distance between the 150- and 430-loops as a metric for 150-cavity formation, we constructed a two-state MSM from the conformations sampled by the viral envelope simulation to estimate the time scales of 150-loop opening and closing motions. Ultimately, we find the stationary distribution (equilibrium probabilities) of the open and closed states to be similar (0.53 and 0.47, respectively). Correspondingly, the time to transition between the two states (i.e., the mean first-passage time (MFPT)) is also roughly equal (39 ± 15 ns for open to closed and 29 ± 11 ns from closed to open), indicating that loop opening and closing occur at similar rates (<a href="#fig3">Figure </a><a href="#fig3">3</a>). MSM calculations are detailed in <a href="http://pubs.acs.org/doi/suppl/10.1021/acscentsci.9b01071/suppl_file/oc9b01071_si_001.pdf">section S1.9</a> and depicted in <a href="http://pubs.acs.org/doi/suppl/10.1021/acscentsci.9b01071/suppl_file/oc9b01071_si_001.pdf">Figures S11–S14</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>A two-state MSM with representative structures from the viral envelope simulation. The equilibrium populations of the open and closed states are approximately equal in both the viral envelope and single-glycoprotein simulations. Correspondingly, the mean first-passage times between the states are approximately equal. The 150-loop and 430-loop are represented as green and red ribbons, respectively.</p>   </div> <p>To understand the impact of a crowded viral environment on loop sampling, we used the same protocol to construct an MSM from structures extracted from simulations of isolated NAs embedded in planar bilayer patches.<sup><a href="#ref61">61</a></sup> The 150-loop dynamics of both the viral envelope and single-glycoprotein simulations are nearly equivalent; however, the error associated with the viral-envelope MSM is much smaller, likely due to the increased simulation time. The single-glycoprotein equilibrium probabilities of the open and closed states were 0.61 and 0.39, respectively, and the MFPT ranges (open to closed and closed to open after 50 ± 96 ns and 72 ± 44 ns, respectively) overlap with those calculated using the viral envelope simulations. This comparison suggests that 150-cavity dynamics are not influenced by the crowded environment of the viral envelope, an expected result given that this pocket is oriented inward (toward the neighboring three monomers of the same tetramer) rather than outward (toward the environment).</p> </div> <div class="secondarybindingsite:functionalannotation" title="sec"> <div class="title" tagx="title" title="title">
Secondary Binding Site: Functional Annotation</div> <p>We note that all FDA-approved NA inhibitors, as well as the endogenous ligand sialic acid, contain negatively charged carboxylate groups. Considering the hypothesis that the 2° site contributes to catalytic efficiency by recruiting and keeping substrates within close proximity to the catalytic site<sup><a href="#ref28">28</a>,<a href="#ref29">29</a></sup> and given that prior BD simulations indicate that substrates bind faster to the 2° site than the 1° site,<sup><a href="#ref17">17</a>,<a href="#ref39">39</a></sup> we postulate that sialic acid first binds to the more solvent-exposed 2° site. Subsequently, the electrostatics of the NA surface guides the substrates to the 1° enzymatic site.</p> <p>Although sialic acid substrates were not included in the viral envelope simulation, we propose that the negatively charged chloride anions in the bulk solvent surrounding the NA monomers serve as a rough surrogate for negatively charged ligand moieties that may associate with the glycoprotein surface. To identify regions favorable to chloride occupancy, we concatenated the 120 monomeric NA simulations and aligned them by the alpha carbons of the 1° site. The chloride atoms were binned into 3375000 voxels (0.67 Å × 0.67 Å × 0.67 Å each). We focused on voxels containing chloride counts greater than three standard deviations above the mean. Notably, our simulations reveal that a volume of high chloride occupancy connects the 1° and 2° sialic acid binding sites (<a href="#fig4">Figure </a><a href="#fig4">4</a>A). This path is wide enough to allow negatively charged small molecules such as sialic acid or oseltamivir carboxylate to move from the 2° site to the 1° catalytic site. Additional regions of high chloride density are also depicted in <a href="#fig4">Figure </a><a href="#fig4">4</a>A.</p> <p>To further explore the role of electrostatics in the in this transfer mechanism, we calculated the electrostatic potential of the 120 NA monomers at the end of MD simulations using the adaptive Poisson–Boltzmann solver (APBS 1.4) software.<sup><a href="#ref65">65</a></sup> When projected onto the NA surface, the electrostatic potential ranging from −1 <i>k</i><sub>b</sub><i>T</i>/e<sub><i>c</i></sub> to +1 <i>k</i><sub>b</sub><i>T</i>/<i>e</i><sub>c</sub> shows a positive region connecting the two sites (<a href="#fig4">Figure </a><a href="#fig4">4</a>B). Positively charged residues such as R118, R368, R430, K432, and P431 (N2 numbering scheme) largely determine this profile. Interestingly, the same analysis performed on the representative NA structures with open and closed 150-loop pockets (extracted with MSM and shown in <a href="#fig3">Figure </a><a href="#fig3">3</a>) reveals that these residues are less exposed in the closed state (<a href="http://pubs.acs.org/doi/suppl/10.1021/acscentsci.9b01071/suppl_file/oc9b01071_si_001.pdf">Figure S15</a>). These results provide evidence that the two sites may act cooperatively, supporting the work of Lai et al.,<sup><a href="#ref16">16</a></sup> which confirmed that pN1 has a 2° site that can bind sialic acid. It also supports the work of Le et al.,<sup><a href="#ref66">66</a></sup> which suggested electrostatic funneling as being the main driving force for oseltamivir carboxylate association to the active site. Chloride anion and electrostatic analyses are detailed in <a href="http://pubs.acs.org/doi/suppl/10.1021/acscentsci.9b01071/suppl_file/oc9b01071_si_001.pdf">section S1.10</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>Chlorine anion distribution within the NA binding sites. The chlorine anion distribution (A) and the projection of the electrostatic potential onto the NA surface (B) show the pathway between the 1° and 2° sites. In panel A, NA is drawn as ice blue cartoon. Regions of high chloride occupancy are illustrated as dotted silver bubbles. Two sialic acids (PDB ID: 1MWE(<a href="#ref24">24</a>)) are superimposed in the catalytic (center) and 2° (upper right) sites for reference.<sup><a href="#ref24">24</a></sup> In panel B the NA surface is colored with a palette varying from red (negative) to royal blue (positive), representing electrostatic potential values of −1 <i>k</i><sub>b</sub><i>T</i>/<i>e</i><sub>c</sub> and +1 <i>k</i><sub>b</sub><i>T</i>/<i>e</i><sub>c</sub>, respectively. The path connecting the 2° site with the catalytic site is shown as a dashed arrow between circles fading from yellow (2° site) to white (catalytic site).</p>   </div> <p>Taken together, these results suggest a biophysical mechanism for the previously uncharacterized 2° site. Sialic acid receptors may first bind the 2° site before being transferred to the 1° sialidase site (<a href="#fig5">Figure </a><a href="#fig5">5</a>). We propose that our chloride distribution analysis is well suited for studying these possible mechanisms of molecular transfer. In contrast to a simple electrostatic map, our simulation-based analysis accounts for both electrostatic and steric factors, as well as for the conformational dynamics sampled over all 14.5 μs of the monomeric simulation in the context of the whole viral-envelope environment. In addition, the proposed "bind and transfer�? mechanism is in good agreement with prior experimental results and proposed mechanisms.<sup><a href="#ref28">28</a></sup></p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>The predicted sialic acid "bind and transfer�? mechanism. Yellow stars represent a sialic-acid-containing glycan receptor. Ice blue half circles represent NA. The 1° catalytic site and 2° site are also labeled.</p>   </div> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>Our work suggests a novel NA binding mechanism wherein a sialic acid-containing substrate (e.g., a glycan receptor) first binds the 2° site, as predicted by earlier BD simulations.<sup><a href="#ref17">17</a></sup> After binding, the substrate is transferred to the catalytic site via electrostatic interactions. Finally, the catalytic site cleaves the terminal sialic acid substrate. In other words, a budding viral particle might use the 2° site to first attract the sialic acid-tipped receptors before these are cleaved within the catalytic site, ultimately allowing viral escape from the infected host-cell surface. MSM and volumetric analyses also further expand the functional annotation of the 1° site and surrounding regions, disclosing exceptionally deep and broad catalytic-pocket conformations. These findings can be exploited to design novel multipronged inhibitors capable of reaching the additional NA cavities unveiled in our multiscale simulations.</p> <p>Taken together, this information provides fundamental insights into our understanding of sialic acid/oseltamivir carboxylate recognition, suggesting new strategies for the development of NA inhibitors. Our work also provides a novel methodological advancement for extracting long-time-scale dynamics from short-time-scale simulations by applying the powerful MSM theoretical framework at subcellular and cellular scales.</p> </div> </div> <div class="back" title="back"> <div class="title" tagx="title" title="title">
Supporting Information Available</div> <p>Supporting Information available online. The Supporting Information is available free of charge at <a href="https://pubs.acs.org/doi/10.1021/acscentsci.9b01071?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acscentsci.9b01071</a>. </p><ul> <li> <p>A NAMD input script used for the production run and the Jupyter notebook used to create the MSM are provided as <a href="http://pubs.acs.org/doi/suppl/10.1021/acscentsci.9b01071/suppl_file/oc9b01071_si_001.pdf">Supporting Information</a>. Additional analysis scripts are available upon request. In addition, the <a href="http://pubs.acs.org/doi/suppl/10.1021/acscentsci.9b01071/suppl_file/oc9b01071_si_001.pdf">Supporting Information</a> includes the following: supplementary materials and methods, information regarding building single-glycoprotein HA and NA structural models, details of the single-glycoprotein HA and NA MD simulations, information regarding building the viral envelope model from a simplified cryoelectron-tomography "point model�?, complete viral envelope all-atom MD simulations, RMSD and RMSF analyses of the viral envelope MD simulations, viral envelope lipid and curvature analysis, NA catalytic-site PCA, volumetric and FTMap analysis, MSMs, and chloride anion and electrostatic analyses (<a href="http://pubs.acs.org/doi/suppl/10.1021/acscentsci.9b01071/suppl_file/oc9b01071_si_001.pdf">PDF</a>)</p> </li> </ul> <p /> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="oc9b01071_si_001.pdf">LINK</a> <p>oc9b01071_si_001.pdf</p>  </div> </div> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p><sup>§</sup> J.D.D., S.E.K., and L.C. contributed equally. R.E.A. designed and oversaw the research project. J.D.D. built the structures and performed the MD simulations. R.E.A., J.D.D., S.E.K., L.C., P.U.L., and A.D. performed the simulation analyses. J.D.D, L.C., P.U.L., and A.D. created the figures. R.E.A., J.D.D., S.E.K., L.C., P.U.L., and A.D. wrote the paper.</p> <p>The authors declare the following competing financial interest(s): R.E.A. is a cofounder and on the scientific advisory board of, and has equity interest in, Actavalon, Inc.</p> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>This work was funded in part by the Director’s New Innovator Award Program NIH DP2 OD007237 and the National Biomedical Computation Resource (NBCR) through NIH P41 GM103426 to R.E.A. Computing support on the NSF Blue Waters Petascale Computer was provided through NSF OAC-1811685. We thank the NSF for access to the TACC Stampede supercomputer through CHE060073N to R.E.A. and the Center for Research Computing at the University of Pittsburgh for a computing allocation to J.D.D.. The authors also thank Dr. Robert Malmstrom for providing sample MSM-building scripts and useful discussions. This work is dedicated to the memory of Klaus Schulten.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="ref1" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="collab'">collab: Centers for Disease Control and Prevention</span></span>. <span class="source" tagx="source" title="source">Summary of the 2017–2018 Influenza Season</span>; United States Centers for Disease Control and Prevention: <span class="year" tagx="year" title="year">2018</span>; Vol. <span class="volume" tagx="volume" title="volume">11</span>, pp. <span class="fpage" tagx="fpage" title="fpage">7</span>–<span class="lpage" tagx="lpage" title="lpage">10</span>.</span></li> <li tag="ref"><a name="ref2" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>; et al. <span class="mixed-article-title" title="mixed-article-title">Epidemiology of avian influenza A H7N9 virus in human beings across five epidemics in mainland China, 2013–17: an epidemiological study of laboratory-confirmed case series</span>. <span class="source" tagx="source" title="source">Lancet Infect. Dis.</span><span class="year" tagx="year" title="year">2017</span>, <span class="volume" tagx="volume" title="volume">17</span>, <span class="fpage" tagx="fpage" title="fpage">822</span>–<span class="lpage" tagx="lpage" title="lpage">832</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S1473-3099(17)30323-7">10.1016/S1473-3099(17)30323-7</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28583578">28583578</a></span></span></li> <li tag="ref"><a name="ref3" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Clercq</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="mixed-article-title" title="mixed-article-title">Antiviral agents active against influenza A viruses</span>. <span class="source" tagx="source" title="source">Nat. Rev. Drug Discovery</span><span class="year" tagx="year" title="year">2006</span>, <span class="volume" tagx="volume" title="volume">5</span>, <span class="fpage" tagx="fpage" title="fpage">1015</span>–<span class="lpage" tagx="lpage" title="lpage">1025</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nrd2175">10.1038/nrd2175</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17139286">17139286</a></span></span></li> <li tag="ref"><a name="ref4" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hamilton</span><span class="given-names" tagx="given-names" title="given-names">B. S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Whittaker</span><span class="given-names" tagx="given-names" title="given-names">G. R.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daniel</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="mixed-article-title" title="mixed-article-title">Influenza virus-mediated membrane fusion: determinants of hemagglutinin fusogenic activity and experimental approaches for assessing virus fusion</span>. <span class="source" tagx="source" title="source">Viruses</span><span class="year" tagx="year" title="year">2012</span>, <span class="volume" tagx="volume" title="volume">4</span>, <span class="fpage" tagx="fpage" title="fpage">1144</span>–<span class="lpage" tagx="lpage" title="lpage">1168</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3390/v4071144">10.3390/v4071144</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22852045">22852045</a></span></span></li> <li tag="ref"><a name="ref5" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">von Itzstein</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="mixed-article-title" title="mixed-article-title">The war against influenza: discovery and development of sialidase inhibitors</span>. <span class="source" tagx="source" title="source">Nat. Rev. Drug Discovery</span><span class="year" tagx="year" title="year">2007</span>, <span class="volume" tagx="volume" title="volume">6</span>, <span class="fpage" tagx="fpage" title="fpage">967</span>–<span class="lpage" tagx="lpage" title="lpage">974</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nrd2400">10.1038/nrd2400</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18049471">18049471</a></span></span></li> <li tag="ref"><a name="ref6" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gamblin</span><span class="given-names" tagx="given-names" title="given-names">S. J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skehel</span><span class="given-names" tagx="given-names" title="given-names">J. J.</span></span><span class="mixed-article-title" title="mixed-article-title">Influenza hemagglutinin and neuraminidase membrane glycoproteins</span>. <span class="source" tagx="source" title="source">J. Biol. Chem.</span><span class="year" tagx="year" title="year">2010</span>, <span class="volume" tagx="volume" title="volume">285</span>, <span class="fpage" tagx="fpage" title="fpage">28403</span>–<span class="lpage" tagx="lpage" title="lpage">28409</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.R110.129809">10.1074/jbc.R110.129809</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20538598">20538598</a></span></span></li> <li tag="ref"><a name="ref7" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">An</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>; et al. <span class="mixed-article-title" title="mixed-article-title">A novel small-molecule inhibitor of the avian influenza H5N1 virus determined through computational screening against the neuraminidase</span>. <span class="source" tagx="source" title="source">J. Med. Chem.</span><span class="year" tagx="year" title="year">2009</span>, <span class="volume" tagx="volume" title="volume">52</span>, <span class="fpage" tagx="fpage" title="fpage">2667</span>–<span class="lpage" tagx="lpage" title="lpage">2672</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/jm800455g">10.1021/jm800455g</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19419201">19419201</a></span></span></li> <li tag="ref"><a name="ref8" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>; et al. <span class="mixed-article-title" title="mixed-article-title">Structure-based design and synthesis of C-1- and C-4-modified analogs of zanamivir as neuraminidase inhibitors</span>. <span class="source" tagx="source" title="source">J. Med. Chem.</span><span class="year" tagx="year" title="year">2013</span>, <span class="volume" tagx="volume" title="volume">56</span>, <span class="fpage" tagx="fpage" title="fpage">671</span>–<span class="lpage" tagx="lpage" title="lpage">684</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/jm3009713">10.1021/jm3009713</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23330908">23330908</a></span></span></li> <li tag="ref"><a name="ref9" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wen</span><span class="given-names" tagx="given-names" title="given-names">W. H.</span></span>; et al. <span class="mixed-article-title" title="mixed-article-title">Analogs of zanamivir with modified C4-substituents as the inhibitors against the group-1 neuraminidases of influenza viruses</span>. <span class="source" tagx="source" title="source">Bioorg. Med. Chem.</span><span class="year" tagx="year" title="year">2010</span>, <span class="volume" tagx="volume" title="volume">18</span>, <span class="fpage" tagx="fpage" title="fpage">4074</span>–<span class="lpage" tagx="lpage" title="lpage">4084</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bmc.2010.04.010">10.1016/j.bmc.2010.04.010</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20452227">20452227</a></span></span></li> <li tag="ref"><a name="ref10" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rudrawar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dyason</span><span class="given-names" tagx="given-names" title="given-names">J. C.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rameix-Welti</span><span class="given-names" tagx="given-names" title="given-names">M.-A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rose</span><span class="given-names" tagx="given-names" title="given-names">F. J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kerry</span><span class="given-names" tagx="given-names" title="given-names">P. S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russell</span><span class="given-names" tagx="given-names" title="given-names">R. J. M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van der Werf</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thomson</span><span class="given-names" tagx="given-names" title="given-names">R. J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naffakh</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">von Itzstein</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="mixed-article-title" title="mixed-article-title">Novel sialic acid derivatives lock open the 150-loop of an influenza A virus group-1 sialidase</span>. <span class="source" tagx="source" title="source">Nat. Commun.</span><span class="year" tagx="year" title="year">2010</span>, <span class="volume" tagx="volume" title="volume">1</span>, <span class="fpage" tagx="fpage" title="fpage">113</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ncomms1114">10.1038/ncomms1114</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21081911">21081911</a></span></span></li> <li tag="ref"><a name="ref11" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amaro</span><span class="given-names" tagx="given-names" title="given-names">R. E.</span></span>; et al. <span class="mixed-article-title" title="mixed-article-title">Remarkable Loop Flexibility in Avian Influenza N1 and Its Implications for Antiviral Drug Design</span>. <span class="source" tagx="source" title="source">J. Am. Chem. Soc.</span><span class="year" tagx="year" title="year">2007</span>, <span class="volume" tagx="volume" title="volume">129</span>, <span class="fpage" tagx="fpage" title="fpage">7764</span>–<span class="lpage" tagx="lpage" title="lpage">7765</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/ja0723535">10.1021/ja0723535</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17539643">17539643</a></span></span></li> <li tag="ref"><a name="ref12" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russell</span><span class="given-names" tagx="given-names" title="given-names">R. J.</span></span>; et al. <span class="mixed-article-title" title="mixed-article-title">The structure of H5N1 avian influenza neuraminidase suggests new opportunities for drug design</span>. <span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2006</span>, <span class="volume" tagx="volume" title="volume">443</span>, <span class="fpage" tagx="fpage" title="fpage">45</span>–<span class="lpage" tagx="lpage" title="lpage">49</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nature05114">10.1038/nature05114</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16915235">16915235</a></span></span></li> <li tag="ref"><a name="ref13" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woods</span><span class="given-names" tagx="given-names" title="given-names">C. J.</span></span>; et al. <span class="mixed-article-title" title="mixed-article-title">Long time scale GPU dynamics reveal the mechanism of drug resistance of the dual mutant I223R/H275Y neuraminidase from H1N1–2009 influenza virus</span>. <span class="source" tagx="source" title="source">Biochemistry</span><span class="year" tagx="year" title="year">2012</span>, <span class="volume" tagx="volume" title="volume">51</span>, <span class="fpage" tagx="fpage" title="fpage">4364</span>–<span class="lpage" tagx="lpage" title="lpage">4375</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/bi300561n">10.1021/bi300561n</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22574858">22574858</a></span></span></li> <li tag="ref"><a name="ref14" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amaro</span><span class="given-names" tagx="given-names" title="given-names">R. E.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ivanov</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCammon</span><span class="given-names" tagx="given-names" title="given-names">J. A. A.</span></span><span class="mixed-article-title" title="mixed-article-title">Characterizing Loop Dynamics and Ligand Recognition in Human- and Avian-Type Influenza Neuraminidases via Generalized Born Molecular Dynamics and End-Point Free Energy Calculations</span>. <span class="source" tagx="source" title="source">J. Am. Chem. Soc.</span><span class="year" tagx="year" title="year">2009</span>, <span class="volume" tagx="volume" title="volume">131</span>, <span class="fpage" tagx="fpage" title="fpage">4702</span>–<span class="lpage" tagx="lpage" title="lpage">4709</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/ja8085643">10.1021/ja8085643</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19296611">19296611</a></span></span></li> <li tag="ref"><a name="ref15" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hausmann</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kretzschmar</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garten</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klenk</span><span class="given-names" tagx="given-names" title="given-names">H.-D.</span></span><span class="mixed-article-title" title="mixed-article-title">N1 neuraminidase of influenza virus A/FPV/Rostock/34 has haemadsorbing activity</span>. <span class="source" tagx="source" title="source">J. Gen. Virol.</span><span class="year" tagx="year" title="year">1995</span>, <span class="volume" tagx="volume" title="volume">76</span>, <span class="fpage" tagx="fpage" title="fpage">1719</span>–<span class="lpage" tagx="lpage" title="lpage">1728</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1099/0022-1317-76-7-1719">10.1099/0022-1317-76-7-1719</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9049377">9049377</a></span></span></li> <li tag="ref"><a name="ref16" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lai</span><span class="given-names" tagx="given-names" title="given-names">J. C. C.</span></span>; et al. <span class="mixed-article-title" title="mixed-article-title">A secondary sialic acid binding site on influenza virus neuraminidase: Fact or fiction?</span>. <span class="source" tagx="source" title="source">Angew. Chem., Int. Ed.</span><span class="year" tagx="year" title="year">2012</span>, <span class="volume" tagx="volume" title="volume">51</span>, <span class="fpage" tagx="fpage" title="fpage">2221</span>–<span class="lpage" tagx="lpage" title="lpage">2224</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/anie.201108245">10.1002/anie.201108245</a></span>.</span></li> <li tag="ref"><a name="ref17" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sung</span><span class="given-names" tagx="given-names" title="given-names">J. C.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wynsberghe</span><span class="given-names" tagx="given-names" title="given-names">A. W. V.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amaro</span><span class="given-names" tagx="given-names" title="given-names">R. E.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W. W.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCammon</span><span class="given-names" tagx="given-names" title="given-names">J. A.</span></span><span class="mixed-article-title" title="mixed-article-title">Role of Secondary Sialic Acid Binding Sites in Influenza N1 Neuraminidase</span>. <span class="source" tagx="source" title="source">J. Am. Chem. Soc.</span><span class="year" tagx="year" title="year">2010</span>, <span class="volume" tagx="volume" title="volume">132</span>, <span class="fpage" tagx="fpage" title="fpage">2883</span>–<span class="lpage" tagx="lpage" title="lpage">2885</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/ja9073672">10.1021/ja9073672</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20155919">20155919</a></span></span></li> <li tag="ref"><a name="ref18" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wade</span><span class="given-names" tagx="given-names" title="given-names">R. C.</span></span><span class="mixed-article-title" title="mixed-article-title">Comparative Binding Energy (COMBINE) Analysis of Influenza Neuraminidase-Inhibitor Complexes</span>. <span class="source" tagx="source" title="source">J. Med. Chem.</span><span class="year" tagx="year" title="year">2001</span>, <span class="volume" tagx="volume" title="volume">44</span>, <span class="fpage" tagx="fpage" title="fpage">961</span>–<span class="lpage" tagx="lpage" title="lpage">971</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/jm001070j">10.1021/jm001070j</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11300878">11300878</a></span></span></li> <li tag="ref"><a name="ref19" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amaro</span><span class="given-names" tagx="given-names" title="given-names">R. E.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Swift</span><span class="given-names" tagx="given-names" title="given-names">R. V.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Votapka</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W. W.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walker</span><span class="given-names" tagx="given-names" title="given-names">R. C.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bush</span><span class="given-names" tagx="given-names" title="given-names">R. M.</span></span><span class="mixed-article-title" title="mixed-article-title">Mechanism of 150-cavity formation in influenza neuraminidase</span>. <span class="source" tagx="source" title="source">Nat. Commun.</span><span class="year" tagx="year" title="year">2011</span>, <span class="volume" tagx="volume" title="volume">2</span>, <span class="fpage" tagx="fpage" title="fpage">388</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ncomms1390">10.1038/ncomms1390</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21750542">21750542</a></span></span></li> <li tag="ref"><a name="ref20" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span>; et al. <span class="mixed-article-title" title="mixed-article-title">The 2009 pandemic H1N1 neuraminidase N1 lacks the 150-cavity in its active site</span>. <span class="source" tagx="source" title="source">Nat. Struct. Mol. Biol.</span><span class="year" tagx="year" title="year">2010</span>, <span class="volume" tagx="volume" title="volume">17</span>, <span class="fpage" tagx="fpage" title="fpage">1266</span>–<span class="lpage" tagx="lpage" title="lpage">1268</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nsmb.1909">10.1038/nsmb.1909</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20852645">20852645</a></span></span></li> <li tag="ref"><a name="ref21" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="mixed-article-title" title="mixed-article-title">Plasticity of 150-Loop in Influenza Neuraminidase Explored by Hamiltonian Replica Exchange Molecular Dynamics Simulations</span>. <span class="source" tagx="source" title="source">PLoS One</span><span class="year" tagx="year" title="year">2013</span>, <span class="volume" tagx="volume" title="volume">8</span>, <span class="elocation-id" tagx="elocation-id" title="elocation-id">e60995</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0060995">10.1371/journal.pone.0060995</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23593372">23593372</a></span></span></li> <li tag="ref"><a name="ref22" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Laver</span><span class="given-names" tagx="given-names" title="given-names">W. G.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colman</span><span class="given-names" tagx="given-names" title="given-names">P. M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webster</span><span class="given-names" tagx="given-names" title="given-names">R. G.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hinshaw</span><span class="given-names" tagx="given-names" title="given-names">V. S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Air</span><span class="given-names" tagx="given-names" title="given-names">G. M.</span></span><span class="mixed-article-title" title="mixed-article-title">Influenza virus neuraminidase with hemagglutinin activity</span>. <span class="source" tagx="source" title="source">Virology</span><span class="year" tagx="year" title="year">1984</span>, <span class="volume" tagx="volume" title="volume">137</span>, <span class="fpage" tagx="fpage" title="fpage">314</span>–<span class="lpage" tagx="lpage" title="lpage">323</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/0042-6822(84)90223-X">10.1016/0042-6822(84)90223-X</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6485252">6485252</a></span></span></li> <li tag="ref"><a name="ref23" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webster</span><span class="given-names" tagx="given-names" title="given-names">R. G.</span></span>; et al. <span class="mixed-article-title" title="mixed-article-title">Antigenic structure and variation in an influenza virus N9 neuraminidase</span>. <span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">1987</span>, <span class="volume" tagx="volume" title="volume">61</span>, <span class="fpage" tagx="fpage" title="fpage">2910</span>–<span class="lpage" tagx="lpage" title="lpage">6</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.61.9.2910-2916.1987">10.1128/JVI.61.9.2910-2916.1987</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3612957">3612957</a></span></span></li> <li tag="ref"><a name="ref24" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Varghese</span><span class="given-names" tagx="given-names" title="given-names">J. N.</span></span>; et al. <span class="mixed-article-title" title="mixed-article-title">Structural evidence for a second sialic acid binding site in avian influenza virus neuraminidases</span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U. S. A.</span><span class="year" tagx="year" title="year">1997</span>, <span class="volume" tagx="volume" title="volume">94</span>, <span class="fpage" tagx="fpage" title="fpage">11808</span>–<span class="lpage" tagx="lpage" title="lpage">11812</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.94.22.11808">10.1073/pnas.94.22.11808</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9342319">9342319</a></span></span></li> <li tag="ref"><a name="ref25" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Air</span><span class="given-names" tagx="given-names" title="given-names">G. M.</span></span><span class="mixed-article-title" title="mixed-article-title">Influenza neuraminidase</span>. <span class="source" tagx="source" title="source">Influenza Other Respir. Viruses</span><span class="year" tagx="year" title="year">2012</span>, <span class="volume" tagx="volume" title="volume">6</span>, <span class="fpage" tagx="fpage" title="fpage">245</span>–<span class="lpage" tagx="lpage" title="lpage">256</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1750-2659.2011.00304.x">10.1111/j.1750-2659.2011.00304.x</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22085243">22085243</a></span></span></li> <li tag="ref"><a name="ref26" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Streltsov</span><span class="given-names" tagx="given-names" title="given-names">V. A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pilling</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barrett</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McKimm-Breschkin</span><span class="given-names" tagx="given-names" title="given-names">J. L.</span></span><span class="mixed-article-title" title="mixed-article-title">Catalytic mechanism and novel receptor binding sites of human parainfluenza virus type 3 hemagglutinin-neuraminidase (hPIV3 HN)</span>. <span class="source" tagx="source" title="source">Antiviral Res.</span><span class="year" tagx="year" title="year">2015</span>, <span class="volume" tagx="volume" title="volume">123</span>, <span class="fpage" tagx="fpage" title="fpage">216</span>–<span class="lpage" tagx="lpage" title="lpage">223</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.antiviral.2015.08.014">10.1016/j.antiviral.2015.08.014</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26364554">26364554</a></span></span></li> <li tag="ref"><a name="ref27" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kobasa</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodgers</span><span class="given-names" tagx="given-names" title="given-names">M. E.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wells</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawaoka</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="mixed-article-title" title="mixed-article-title">Neuraminidase hemadsorption activity, conserved in avian influenza A viruses, does not influence viral replication in ducks</span>. <span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">1997</span>, <span class="volume" tagx="volume" title="volume">71</span>, <span class="fpage" tagx="fpage" title="fpage">6706</span>–<span class="lpage" tagx="lpage" title="lpage">6713</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.71.9.6706-6713.1997">10.1128/JVI.71.9.6706-6713.1997</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9261394">9261394</a></span></span></li> <li tag="ref"><a name="ref28" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uhlendorff</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matrosovich</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klenk</span><span class="given-names" tagx="given-names" title="given-names">H.-D.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matrosovich</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="mixed-article-title" title="mixed-article-title">Functional significance of the hemadsorption activity of influenza virus neuraminidase and its alteration in pandemic viruses</span>. <span class="source" tagx="source" title="source">Arch. Virol.</span><span class="year" tagx="year" title="year">2009</span>, <span class="volume" tagx="volume" title="volume">154</span>, <span class="fpage" tagx="fpage" title="fpage">945</span>–<span class="lpage" tagx="lpage" title="lpage">957</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s00705-009-0393-x">10.1007/s00705-009-0393-x</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19458903">19458903</a></span></span></li> <li tag="ref"><a name="ref29" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dai</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McBride</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dortmans</span><span class="given-names" tagx="given-names" title="given-names">J. C. F. M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peng</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bakkers</span><span class="given-names" tagx="given-names" title="given-names">M. J. G.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Groot</span><span class="given-names" tagx="given-names" title="given-names">R. J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Kuppeveld</span><span class="given-names" tagx="given-names" title="given-names">F. J. M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paulson</span><span class="given-names" tagx="given-names" title="given-names">J. C.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Vries</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Haan</span><span class="given-names" tagx="given-names" title="given-names">C. A. M.</span></span><span class="mixed-article-title" title="mixed-article-title">Mutation of the Second Sialic Acid-Binding Site, Resulting in Reduced Neuraminidase Activity, Preceded the Emergence of H7N9 Influenza A Virus</span>. <span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2017</span>, <span class="volume" tagx="volume" title="volume">91</span>, <span class="fpage" tagx="fpage" title="fpage">91</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00049-17">10.1128/JVI.00049-17</a></span>.</span></li> <li tag="ref"><a name="ref30" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Du</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dai</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boons</span><span class="given-names" tagx="given-names" title="given-names">G. J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peeters</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Kuppeveld</span><span class="given-names" tagx="given-names" title="given-names">F. J. M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Vries</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Haan</span><span class="given-names" tagx="given-names" title="given-names">C. A. M.</span></span><span class="mixed-article-title" title="mixed-article-title">Substrate Binding by the Second Sialic Acid-Binding Site of Influenza A Virus N1 Neuraminidase Contributes to Enzymatic Activity</span>. <span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2018</span>, <span class="volume" tagx="volume" title="volume">92</span>, <span class="fpage" tagx="fpage" title="fpage">e01243</span>–<span class="lpage" tagx="lpage" title="lpage">18</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.01243-18">10.1128/JVI.01243-18</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30089692">30089692</a></span></span></li> <li tag="ref"><a name="ref31" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Du</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>; et al. <span class="mixed-article-title" title="mixed-article-title">The 2nd sialic acid-binding site of influenza A virus neuraminidase is an important determinant of the hemagglutinin-neuraminidase-receptor balance</span>. <span class="source" tagx="source" title="source">PLoS Pathog.</span><span class="year" tagx="year" title="year">2019</span>, <span class="volume" tagx="volume" title="volume">15</span>, <span class="elocation-id" tagx="elocation-id" title="elocation-id">e1007860</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1007860">10.1371/journal.ppat.1007860</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31181126">31181126</a></span></span></li> <li tag="ref"><a name="ref32" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zeller</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luitz</span><span class="given-names" tagx="given-names" title="given-names">M. P.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bomblies</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zacharias</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="mixed-article-title" title="mixed-article-title">Multiscale Simulation of Receptor–Drug Association Kinetics: Application to Neuraminidase Inhibitors</span>. <span class="source" tagx="source" title="source">J. Chem. Theory Comput.</span><span class="year" tagx="year" title="year">2017</span>, <span class="volume" tagx="volume" title="volume">13</span>, <span class="fpage" tagx="fpage" title="fpage">5097</span>–<span class="lpage" tagx="lpage" title="lpage">5105</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/acs.jctc.7b00631">10.1021/acs.jctc.7b00631</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28820938">28820938</a></span></span></li> <li tag="ref"><a name="ref33" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ivinson</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deliyannis</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McNabb</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grollo</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gilbertson</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jackson</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">L. E.</span></span><span class="mixed-article-title" title="mixed-article-title">Salivary Blockade Protects the Lower Respiratory Tract of Mice from Lethal Influenza Virus Infection</span>. <span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2017</span>, <span class="volume" tagx="volume" title="volume">91</span>, <span class="fpage" tagx="fpage" title="fpage">e00624</span>–<span class="lpage" tagx="lpage" title="lpage">17</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00624-17">10.1128/JVI.00624-17</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28446669">28446669</a></span></span></li> <li tag="ref"><a name="ref34" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gilbertson</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ng</span><span class="given-names" tagx="given-names" title="given-names">W. C.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crawford</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McKimm-Breschkin</span><span class="given-names" tagx="given-names" title="given-names">J. L.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">L. E.</span></span><span class="mixed-article-title" title="mixed-article-title">Mouse Saliva Inhibits Transit of Influenza Virus to the Lower Respiratory Tract by Efficiently Blocking Influenza Virus Neuraminidase Activity</span>. <span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2017</span>, <span class="volume" tagx="volume" title="volume">91</span>, <span class="fpage" tagx="fpage" title="fpage">e00145</span>–<span class="lpage" tagx="lpage" title="lpage">17</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00145-17">10.1128/JVI.00145-17</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28446666">28446666</a></span></span></li> <li tag="ref"><a name="ref35" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>; et al. <span class="mixed-article-title" title="mixed-article-title">Structural Basis of Preexisting Immunity to the 2009 H1N1 Pandemic Influenza Virus</span>. <span class="source" tagx="source" title="source">Science (Washington, DC, U. S.)</span><span class="year" tagx="year" title="year">2010</span>, <span class="volume" tagx="volume" title="volume">328</span>, <span class="fpage" tagx="fpage" title="fpage">357</span>–<span class="lpage" tagx="lpage" title="lpage">360</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1186430">10.1126/science.1186430</a></span>.</span></li> <li tag="ref"><a name="ref36" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; et al. <span class="mixed-article-title" title="mixed-article-title">Influenza virus pleiomorphy characterized by cryoelectron tomography</span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U. S. A.</span><span class="year" tagx="year" title="year">2006</span>, <span class="volume" tagx="volume" title="volume">103</span>, <span class="fpage" tagx="fpage" title="fpage">19123</span>–<span class="lpage" tagx="lpage" title="lpage">19127</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0607614103">10.1073/pnas.0607614103</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17146053">17146053</a></span></span></li> <li tag="ref"><a name="ref37" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singharoy</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; et al. <span class="mixed-article-title" title="mixed-article-title">Atoms to Phenotypes: Molecular Design Principles of Cellular Energy Metabolism</span>. <span class="source" tagx="source" title="source">Cell</span><span class="year" tagx="year" title="year">2019</span>, <span class="volume" tagx="volume" title="volume">179</span>, <span class="fpage" tagx="fpage" title="fpage">1098</span>–<span class="lpage" tagx="lpage" title="lpage">1111</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cell.2019.10.021">10.1016/j.cell.2019.10.021</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31730852">31730852</a></span></span></li> <li tag="ref"><a name="ref38" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karplus</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="mixed-article-title" title="mixed-article-title">Development of Multiscale Models for Complex Chemical Systems: From H+H 2 to Biomolecules (Nobel Lecture)</span>. <span class="source" tagx="source" title="source">Angew. Chem., Int. Ed.</span><span class="year" tagx="year" title="year">2014</span>, <span class="volume" tagx="volume" title="volume">53</span>, <span class="fpage" tagx="fpage" title="fpage">9992</span>–<span class="lpage" tagx="lpage" title="lpage">10005</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/anie.201403924">10.1002/anie.201403924</a></span>.</span></li> <li tag="ref"><a name="ref39" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amaro</span><span class="given-names" tagx="given-names" title="given-names">R. E.</span></span>; et al. <span class="mixed-article-title" title="mixed-article-title">A Computational Assay that Explores the Hemagglutinin/Neuraminidase Functional Balance Reveals the Neuraminidase Secondary Site as a Novel Anti-Influenza Target</span>. <span class="source" tagx="source" title="source">ACS Cent. Sci.</span><span class="year" tagx="year" title="year">2018</span>, <span class="volume" tagx="volume" title="volume">4</span>, <span class="fpage" tagx="fpage" title="fpage">1570</span>–<span class="lpage" tagx="lpage" title="lpage">1577</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/acscentsci.8b00666">10.1021/acscentsci.8b00666</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30555910">30555910</a></span></span></li> <li tag="ref"><a name="ref40" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hadden</span><span class="given-names" tagx="given-names" title="given-names">J. A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perilla</span><span class="given-names" tagx="given-names" title="given-names">J. R.</span></span><span class="mixed-article-title" title="mixed-article-title">All-atom virus simulations</span>. <span class="source" tagx="source" title="source">Curr. Opin. Virol.</span><span class="year" tagx="year" title="year">2018</span>, <span class="volume" tagx="volume" title="volume">31</span>, <span class="fpage" tagx="fpage" title="fpage">82</span>–<span class="lpage" tagx="lpage" title="lpage">91</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.coviro.2018.08.007">10.1016/j.coviro.2018.08.007</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30181049">30181049</a></span></span></li> <li tag="ref"><a name="ref41" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huber</span><span class="given-names" tagx="given-names" title="given-names">R. G.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marzinek</span><span class="given-names" tagx="given-names" title="given-names">J. K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holdbrook</span><span class="given-names" tagx="given-names" title="given-names">D. A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bond</span><span class="given-names" tagx="given-names" title="given-names">P. J.</span></span><span class="mixed-article-title" title="mixed-article-title">Multiscale molecular dynamics simulation approaches to the structure and dynamics of viruses</span>. <span class="source" tagx="source" title="source">Prog. Biophys. Mol. Biol.</span><span class="year" tagx="year" title="year">2017</span>, <span class="volume" tagx="volume" title="volume">128</span>, <span class="fpage" tagx="fpage" title="fpage">121</span>–<span class="lpage" tagx="lpage" title="lpage">132</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.pbiomolbio.2016.09.010">10.1016/j.pbiomolbio.2016.09.010</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27765545">27765545</a></span></span></li> <li tag="ref"><a name="ref42" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Freddolino</span><span class="given-names" tagx="given-names" title="given-names">P. L.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arkhipov</span><span class="given-names" tagx="given-names" title="given-names">A. S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larson</span><span class="given-names" tagx="given-names" title="given-names">S. B.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McPherson</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schulten</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="mixed-article-title" title="mixed-article-title">Molecular dynamics simulations of the complete satellite tobacco mosaic virus</span>. <span class="source" tagx="source" title="source">Structure</span><span class="year" tagx="year" title="year">2006</span>, <span class="volume" tagx="volume" title="volume">14</span>, <span class="fpage" tagx="fpage" title="fpage">437</span>–<span class="lpage" tagx="lpage" title="lpage">449</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.str.2005.11.014">10.1016/j.str.2005.11.014</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16531228">16531228</a></span></span></li> <li tag="ref"><a name="ref43" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zink</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grubmüller</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="mixed-article-title" title="mixed-article-title">Mechanical properties of the icosahedral shell of southern bean mosaic virus: A molecular dynamics study</span>. <span class="source" tagx="source" title="source">Biophys. J.</span><span class="year" tagx="year" title="year">2009</span>, <span class="volume" tagx="volume" title="volume">96</span>, <span class="fpage" tagx="fpage" title="fpage">1350</span>–<span class="lpage" tagx="lpage" title="lpage">1363</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bpj.2008.11.028">10.1016/j.bpj.2008.11.028</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19217853">19217853</a></span></span></li> <li tag="ref"><a name="ref44" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ayton</span><span class="given-names" tagx="given-names" title="given-names">G. S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Voth</span><span class="given-names" tagx="given-names" title="given-names">G. A.</span></span><span class="mixed-article-title" title="mixed-article-title">Multiscale computer simulation of the immature HIV-1 virion</span>. <span class="source" tagx="source" title="source">Biophys. J.</span><span class="year" tagx="year" title="year">2010</span>, <span class="volume" tagx="volume" title="volume">99</span>, <span class="fpage" tagx="fpage" title="fpage">2757</span>–<span class="lpage" tagx="lpage" title="lpage">2765</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bpj.2010.08.018">10.1016/j.bpj.2010.08.018</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21044572">21044572</a></span></span></li> <li tag="ref"><a name="ref45" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roberts</span><span class="given-names" tagx="given-names" title="given-names">J. A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuiper</span><span class="given-names" tagx="given-names" title="given-names">M. J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thorley</span><span class="given-names" tagx="given-names" title="given-names">B. R.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smooker</span><span class="given-names" tagx="given-names" title="given-names">P. M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hung</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="mixed-article-title" title="mixed-article-title">Investigation of a predicted N-terminal amphipathic α-helix using atomistic molecular dynamics simulation of a complete prototype poliovirus virion</span>. <span class="source" tagx="source" title="source">J. Mol. Graphics Modell.</span><span class="year" tagx="year" title="year">2012</span>, <span class="volume" tagx="volume" title="volume">38</span>, <span class="fpage" tagx="fpage" title="fpage">165</span>–<span class="lpage" tagx="lpage" title="lpage">173</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jmgm.2012.06.009">10.1016/j.jmgm.2012.06.009</a></span>.</span></li> <li tag="ref"><a name="ref46" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>; et al. <span class="mixed-article-title" title="mixed-article-title">Mature HIV-1 capsid structure by cryo-electron microscopy and all-atom molecular dynamics</span>. <span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2013</span>, <span class="volume" tagx="volume" title="volume">497</span>, <span class="fpage" tagx="fpage" title="fpage">643</span>–<span class="lpage" tagx="lpage" title="lpage">646</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nature12162">10.1038/nature12162</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23719463">23719463</a></span></span></li> <li tag="ref"><a name="ref47" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andoh</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>; et al. <span class="mixed-article-title" title="mixed-article-title">All-atom molecular dynamics calculation study of entire poliovirus empty capsids in solution</span>. <span class="source" tagx="source" title="source">J. Chem. Phys.</span><span class="year" tagx="year" title="year">2014</span>, <span class="volume" tagx="volume" title="volume">141</span>, <span class="fpage" tagx="fpage" title="fpage">165101</span><span class="pub-id"><a href="https://dx.doi.org/10.1063/1.4897557">10.1063/1.4897557</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25362342">25362342</a></span></span></li> <li tag="ref"><a name="ref48" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reddy</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>; et al. <span class="mixed-article-title" title="mixed-article-title">Nothing to Sneeze At: A Dynamic and Integrative Computational Model of an Influenza A Virion</span>. <span class="source" tagx="source" title="source">Structure</span><span class="year" tagx="year" title="year">2015</span>, <span class="volume" tagx="volume" title="volume">23</span>, <span class="fpage" tagx="fpage" title="fpage">584</span>–<span class="lpage" tagx="lpage" title="lpage">597</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.str.2014.12.019">10.1016/j.str.2014.12.019</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25703376">25703376</a></span></span></li> <li tag="ref"><a name="ref49" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reddy</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sansom</span><span class="given-names" tagx="given-names" title="given-names">M. S. P.</span></span><span class="mixed-article-title" title="mixed-article-title">The Role of the Membrane in the Structure and Biophysical Robustness of the Dengue Virion Envelope</span>. <span class="source" tagx="source" title="source">Structure</span><span class="year" tagx="year" title="year">2016</span>, <span class="volume" tagx="volume" title="volume">24</span>, <span class="fpage" tagx="fpage" title="fpage">375</span>–<span class="lpage" tagx="lpage" title="lpage">382</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.str.2015.12.011">10.1016/j.str.2015.12.011</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26833387">26833387</a></span></span></li> <li tag="ref"><a name="ref50" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marzinek</span><span class="given-names" tagx="given-names" title="given-names">J. K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holdbrook</span><span class="given-names" tagx="given-names" title="given-names">D. A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huber</span><span class="given-names" tagx="given-names" title="given-names">R. G.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Verma</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bond</span><span class="given-names" tagx="given-names" title="given-names">P. J.</span></span><span class="mixed-article-title" title="mixed-article-title">Pushing the Envelope: Dengue Viral Membrane Coaxed into Shape by Molecular Simulations</span>. <span class="source" tagx="source" title="source">Structure</span><span class="year" tagx="year" title="year">2016</span>, <span class="volume" tagx="volume" title="volume">24</span>, <span class="fpage" tagx="fpage" title="fpage">1410</span>–<span class="lpage" tagx="lpage" title="lpage">1420</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.str.2016.05.014">10.1016/j.str.2016.05.014</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27396828">27396828</a></span></span></li> <li tag="ref"><a name="ref51" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pande</span><span class="given-names" tagx="given-names" title="given-names">V. S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beauchamp</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bowman</span><span class="given-names" tagx="given-names" title="given-names">G. R.</span></span><span class="mixed-article-title" title="mixed-article-title">Everything you wanted to know about Markov State Models but were afraid to ask</span>. <span class="source" tagx="source" title="source">Methods</span><span class="year" tagx="year" title="year">2010</span>, <span class="volume" tagx="volume" title="volume">52</span>, <span class="fpage" tagx="fpage" title="fpage">99</span>–<span class="lpage" tagx="lpage" title="lpage">105</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ymeth.2010.06.002">10.1016/j.ymeth.2010.06.002</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20570730">20570730</a></span></span></li> <li tag="ref"><a name="ref52" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prinz</span><span class="given-names" tagx="given-names" title="given-names">J. H.</span></span>; et al. <span class="mixed-article-title" title="mixed-article-title">Markov models of molecular kinetics: Generation and validation</span>. <span class="source" tagx="source" title="source">J. Chem. Phys.</span><span class="year" tagx="year" title="year">2011</span>, <span class="volume" tagx="volume" title="volume">134</span>, <span class="fpage" tagx="fpage" title="fpage">174105</span><span class="pub-id"><a href="https://dx.doi.org/10.1063/1.3565032">10.1063/1.3565032</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21548671">21548671</a></span></span></li> <li tag="ref"><a name="ref53" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chodera</span><span class="given-names" tagx="given-names" title="given-names">J. D.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Noé</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="mixed-article-title" title="mixed-article-title">Markov state models of biomolecular conformational dynamics</span>. <span class="source" tagx="source" title="source">Curr. Opin. Struct. Biol.</span><span class="year" tagx="year" title="year">2014</span>, <span class="volume" tagx="volume" title="volume">25</span>, <span class="fpage" tagx="fpage" title="fpage">135</span>–<span class="lpage" tagx="lpage" title="lpage">144</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.sbi.2014.04.002">10.1016/j.sbi.2014.04.002</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24836551">24836551</a></span></span></li> <li tag="ref"><a name="ref54" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plattner</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Noé</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="mixed-article-title" title="mixed-article-title">Protein conformational plasticity and complex ligand-binding kinetics explored by atomistic simulations and Markov models</span>. <span class="source" tagx="source" title="source">Nat. Commun.</span><span class="year" tagx="year" title="year">2015</span>, <span class="volume" tagx="volume" title="volume">6</span>, <span class="fpage" tagx="fpage" title="fpage">7653</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ncomms8653">10.1038/ncomms8653</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26134632">26134632</a></span></span></li> <li tag="ref"><a name="ref55" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plattner</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doerr</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Fabritiis</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Noé</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="mixed-article-title" title="mixed-article-title">Complete protein-protein association kinetics in atomic detail revealed by molecular dynamics simulations and Markov modelling</span>. <span class="source" tagx="source" title="source">Nat. Chem.</span><span class="year" tagx="year" title="year">2017</span>, <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">1005</span>–<span class="lpage" tagx="lpage" title="lpage">1011</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nchem.2785">10.1038/nchem.2785</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28937668">28937668</a></span></span></li> <li tag="ref"><a name="ref56" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Voelz</span><span class="given-names" tagx="given-names" title="given-names">V. A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bowman</span><span class="given-names" tagx="given-names" title="given-names">G. R.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beauchamp</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pande</span><span class="given-names" tagx="given-names" title="given-names">V. S.</span></span><span class="mixed-article-title" title="mixed-article-title">Molecular Simulation of ab Initio Protein Folding for a Millisecond Folder NTL9(1–39)</span>. <span class="source" tagx="source" title="source">J. Am. Chem. Soc.</span><span class="year" tagx="year" title="year">2010</span>, <span class="volume" tagx="volume" title="volume">132</span>, <span class="fpage" tagx="fpage" title="fpage">1526</span>–<span class="lpage" tagx="lpage" title="lpage">1528</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/ja9090353">10.1021/ja9090353</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20070076">20070076</a></span></span></li> <li tag="ref"><a name="ref57" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Phillips</span><span class="given-names" tagx="given-names" title="given-names">J. C.</span></span>; et al. <span class="mixed-article-title" title="mixed-article-title">Scalable molecular dynamics with NAMD</span>. <span class="source" tagx="source" title="source">J. Comput. Chem.</span><span class="year" tagx="year" title="year">2005</span>, <span class="volume" tagx="volume" title="volume">26</span>, <span class="fpage" tagx="fpage" title="fpage">1781</span>–<span class="lpage" tagx="lpage" title="lpage">1802</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/jcc.20289">10.1002/jcc.20289</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16222654">16222654</a></span></span></li> <li tag="ref"><a name="ref58" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MacKerell</span><span class="given-names" tagx="given-names" title="given-names">A. D.</span></span><span class="mixed-article-title" title="mixed-article-title">CHARMM36 all-atom additive protein force field: Validation based on comparison to NMR data</span>. <span class="source" tagx="source" title="source">J. Comput. Chem.</span><span class="year" tagx="year" title="year">2013</span>, <span class="volume" tagx="volume" title="volume">34</span>, <span class="fpage" tagx="fpage" title="fpage">2135</span>–<span class="lpage" tagx="lpage" title="lpage">2145</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/jcc.23354">10.1002/jcc.23354</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23832629">23832629</a></span></span></li> <li tag="ref"><a name="ref59" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jorgensen</span><span class="given-names" tagx="given-names" title="given-names">W. L.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chandrasekhar</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madura</span><span class="given-names" tagx="given-names" title="given-names">J. D.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Impey</span><span class="given-names" tagx="given-names" title="given-names">R. W.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klein</span><span class="given-names" tagx="given-names" title="given-names">M. L.</span></span><span class="mixed-article-title" title="mixed-article-title">Comparison of simple potential functions for simulating liquid water</span>. <span class="source" tagx="source" title="source">J. Chem. Phys.</span><span class="year" tagx="year" title="year">1983</span>, <span class="volume" tagx="volume" title="volume">79</span>, <span class="fpage" tagx="fpage" title="fpage">926</span><span class="pub-id"><a href="https://dx.doi.org/10.1063/1.445869">10.1063/1.445869</a></span>.</span></li> <li tag="ref"><a name="ref60" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beglov</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roux</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="mixed-article-title" title="mixed-article-title">Finite representation of an infinite bulk system: Solvent boundary potential for computer simulations</span>. <span class="source" tagx="source" title="source">J. Chem. Phys.</span><span class="year" tagx="year" title="year">1994</span>, <span class="volume" tagx="volume" title="volume">100</span>, <span class="fpage" tagx="fpage" title="fpage">9050</span>–<span class="lpage" tagx="lpage" title="lpage">9063</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1063/1.466711">10.1063/1.466711</a></span>.</span></li> <li tag="ref"><a name="ref61" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Durrant</span><span class="given-names" tagx="given-names" title="given-names">J. D.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bush</span><span class="given-names" tagx="given-names" title="given-names">R. M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amaro</span><span class="given-names" tagx="given-names" title="given-names">R. E.</span></span><span class="mixed-article-title" title="mixed-article-title">Microsecond Molecular Dynamics Simulations of Influenza Neuraminidase Suggest a Mechanism for the Increased Virulence of Stalk-Deletion Mutants</span>. <span class="source" tagx="source" title="source">J. Phys. Chem. B</span><span class="year" tagx="year" title="year">2016</span>, <span class="volume" tagx="volume" title="volume">120</span>, <span class="fpage" tagx="fpage" title="fpage">8590</span>–<span class="lpage" tagx="lpage" title="lpage">8599</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/acs.jpcb.6b02655">10.1021/acs.jpcb.6b02655</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27141956">27141956</a></span></span></li> <li tag="ref"><a name="ref62" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Durrant</span><span class="given-names" tagx="given-names" title="given-names">J. D.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Votapka</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sørensen</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amaro</span><span class="given-names" tagx="given-names" title="given-names">R. E.</span></span><span class="mixed-article-title" title="mixed-article-title">POVME 2.0: An Enhanced Tool for Determining Pocket Shape and Volume Characteristics</span>. <span class="source" tagx="source" title="source">J. Chem. Theory Comput.</span><span class="year" tagx="year" title="year">2014</span>, <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">5047</span>–<span class="lpage" tagx="lpage" title="lpage">5056</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/ct500381c">10.1021/ct500381c</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25400521">25400521</a></span></span></li> <li tag="ref"><a name="ref63" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ngan</span><span class="given-names" tagx="given-names" title="given-names">C. H.</span></span>; et al. <span class="mixed-article-title" title="mixed-article-title">FTMAP: extended protein mapping with user-selected probe molecules</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2012</span>, <span class="volume" tagx="volume" title="volume">40</span>, <span class="fpage" tagx="fpage" title="fpage">W271</span>–<span class="lpage" tagx="lpage" title="lpage">5</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gks441">10.1093/nar/gks441</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22589414">22589414</a></span></span></li> <li tag="ref"><a name="ref64" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marino</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bones</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kattla</span><span class="given-names" tagx="given-names" title="given-names">J. J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rudd</span><span class="given-names" tagx="given-names" title="given-names">P. M.</span></span><span class="mixed-article-title" title="mixed-article-title">A systematic approach to protein glycosylation analysis: a path through the maze</span>. <span class="source" tagx="source" title="source">Nat. Chem. Biol.</span><span class="year" tagx="year" title="year">2010</span>, <span class="volume" tagx="volume" title="volume">6</span>, <span class="fpage" tagx="fpage" title="fpage">713</span>–<span class="lpage" tagx="lpage" title="lpage">723</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nchembio.437">10.1038/nchembio.437</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20852609">20852609</a></span></span></li> <li tag="ref"><a name="ref65" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baker</span><span class="given-names" tagx="given-names" title="given-names">N. A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sept</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joseph</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holst</span><span class="given-names" tagx="given-names" title="given-names">M. J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCammon</span><span class="given-names" tagx="given-names" title="given-names">J. A.</span></span><span class="mixed-article-title" title="mixed-article-title">Electrostatics of nanosystems: application to microtubules and the ribosome</span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U. S. A.</span><span class="year" tagx="year" title="year">2001</span>, <span class="volume" tagx="volume" title="volume">98</span>, <span class="fpage" tagx="fpage" title="fpage">10037</span>–<span class="lpage" tagx="lpage" title="lpage">41</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.181342398">10.1073/pnas.181342398</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11517324">11517324</a></span></span></li> <li tag="ref"><a name="ref66" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Le</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">E. H.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hardy</span><span class="given-names" tagx="given-names" title="given-names">D. J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Truong</span><span class="given-names" tagx="given-names" title="given-names">T. N.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schulten</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="mixed-article-title" title="mixed-article-title">Molecular Dynamics Simulations Suggest that Electrostatic Funnel Directs Binding of Tamiflu to Influenza N1 Neuraminidases</span>. <span class="source" tagx="source" title="source">PLoS Comput. Biol.</span><span class="year" tagx="year" title="year">2010</span>, <span class="volume" tagx="volume" title="volume">6</span>, <span class="elocation-id" tagx="elocation-id" title="elocation-id">e1000939</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pcbi.1000939">10.1371/journal.pcbi.1000939</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20885781">20885781</a></span></span></li> </ul> </div> </div>  </body></html>