<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/03/31/2020.03.29.014290.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.03.29.014290v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Sequence variation among SARS-CoV-2 isolates in Taiwan"/>
  <meta name="DC.Identifier" content="10.1101/2020.03.29.014290"/>
  <meta name="DC.Date" content="2020-03-31"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Taiwan experienced two waves of imported cases of coronavirus disease 2019 (COVID-19), first from China in January to late February, followed by those from other countries starting in early March. Additionally, several cases could not be traced to any imported cases and were suspected as sporadic local transmission. Twelve full viral genomes were determined in this study by Illumina sequencing either from virus isolates or directly from specimens, among which 5 originated from clustered infections. Phylogenetic tree analysis revealed that these sequences were in different clades, indicating that no major strain has been circulating in Taiwan. A deletion in open reading frame 8 was found in one isolate. Only a 4-nucleotide difference was observed among the 5 genomes from clustered infections."/>
  <meta name="DC.Contributor" content="Yu-Nong Gong"/>
  <meta name="DC.Contributor" content="Kuo-Chien Tsao"/>
  <meta name="DC.Contributor" content="Mei-Jen Hsiao"/>
  <meta name="DC.Contributor" content="Chung-Guei Huang"/>
  <meta name="DC.Contributor" content="Peng-Nien Huang"/>
  <meta name="DC.Contributor" content="Po-Wei Huang"/>
  <meta name="DC.Contributor" content="Kuo-Ming Lee"/>
  <meta name="DC.Contributor" content="Yi-Chun Liu"/>
  <meta name="DC.Contributor" content="Shu-Li Yang"/>
  <meta name="DC.Contributor" content="Rei-Lin Kuo"/>
  <meta name="DC.Contributor" content="Ming-Tsan Liu"/>
  <meta name="DC.Contributor" content="Ji-Rong Yang"/>
  <meta name="DC.Contributor" content="Cheng-Hsun Chiu"/>
  <meta name="DC.Contributor" content="Cheng-Ta Yang"/>
  <meta name="DC.Contributor" content="Shin-Ru Shih"/>
  <meta name="DC.Contributor" content="Guang-Wu Chen"/>
  <meta name="article:published_time" content="2020-03-31"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Sequence variation among SARS-CoV-2 isolates in Taiwan"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;Taiwan experienced two waves of imported cases of coronavirus disease 2019 (COVID-19), first from China in January to late February, followed by those from other countries starting in early March. Additionally, several cases could not be traced to any imported cases and were suspected as sporadic local transmission. Twelve full viral genomes were determined in this study by Illumina sequencing either from virus isolates or directly from specimens, among which 5 originated from clustered infections. Phylogenetic tree analysis revealed that these sequences were in different clades, indicating that no major strain has been circulating in Taiwan. A deletion in open reading frame 8 was found in one isolate. Only a 4-nucleotide difference was observed among the 5 genomes from clustered infections.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.03.29.014290v1"/>
  <meta name="citation_id" content="2020.03.29.014290v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.29.014290v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.29.014290v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.29.014290v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.29.014290.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.03.29.014290"/>
  <meta name="citation_num_pages" content="19"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.03.29.014290"/>
  <meta name="citation_author" content="Yu-Nong Gong"/>
  <meta name="citation_author_institution" content="Research Center for Emerging Viral Infections, College of Medicine, Chang Gung University"/>
  <meta name="citation_author_institution" content="Department of Laboratory Medicine, Linkou Chang Gung Memorial Hospital"/>
  <meta name="citation_author" content="Kuo-Chien Tsao"/>
  <meta name="citation_author_institution" content="Research Center for Emerging Viral Infections, College of Medicine, Chang Gung University"/>
  <meta name="citation_author_institution" content="Department of Laboratory Medicine, Linkou Chang Gung Memorial Hospital"/>
  <meta name="citation_author_institution" content="Department of Medical Biotechnology and Laboratory Science, College of Medicine, Chang Gung University"/>
  <meta name="citation_author" content="Mei-Jen Hsiao"/>
  <meta name="citation_author_institution" content="Department of Laboratory Medicine, Linkou Chang Gung Memorial Hospital"/>
  <meta name="citation_author" content="Chung-Guei Huang"/>
  <meta name="citation_author_institution" content="Department of Laboratory Medicine, Linkou Chang Gung Memorial Hospital"/>
  <meta name="citation_author_institution" content="Department of Medical Biotechnology and Laboratory Science, College of Medicine, Chang Gung University"/>
  <meta name="citation_author" content="Peng-Nien Huang"/>
  <meta name="citation_author_institution" content="Research Center for Emerging Viral Infections, College of Medicine, Chang Gung University"/>
  <meta name="citation_author" content="Po-Wei Huang"/>
  <meta name="citation_author_institution" content="Department of Laboratory Medicine, Linkou Chang Gung Memorial Hospital"/>
  <meta name="citation_author" content="Kuo-Ming Lee"/>
  <meta name="citation_author_institution" content="Research Center for Emerging Viral Infections, College of Medicine, Chang Gung University"/>
  <meta name="citation_author" content="Yi-Chun Liu"/>
  <meta name="citation_author_institution" content="Department of Laboratory Medicine, Linkou Chang Gung Memorial Hospital"/>
  <meta name="citation_author" content="Shu-Li Yang"/>
  <meta name="citation_author_institution" content="Department of Laboratory Medicine, Linkou Chang Gung Memorial Hospital"/>
  <meta name="citation_author_institution" content="Department of Medical Biotechnology and Laboratory Science, College of Medicine, Chang Gung University"/>
  <meta name="citation_author" content="Rei-Lin Kuo"/>
  <meta name="citation_author_institution" content="Research Center for Emerging Viral Infections, College of Medicine, Chang Gung University"/>
  <meta name="citation_author_institution" content="Department of Medical Biotechnology and Laboratory Science, College of Medicine, Chang Gung University"/>
  <meta name="citation_author_institution" content="Division of Asthma, Allergy, and Rheumatology, Department of Pediatrics, Linkou Chang Gung Memorial Hospital"/>
  <meta name="citation_author_institution" content="Graduate Institute of Biomedical Sciences, College of Medicine, Chang Gung University"/>
  <meta name="citation_author" content="Ming-Tsan Liu"/>
  <meta name="citation_author_institution" content="Centers for Disease Control"/>
  <meta name="citation_author" content="Ji-Rong Yang"/>
  <meta name="citation_author_institution" content="Centers for Disease Control"/>
  <meta name="citation_author" content="Cheng-Hsun Chiu"/>
  <meta name="citation_author_institution" content="Division of Pediatric Infectious Diseases, Department of Pediatrics, Chang Gung Memorial Hospital, Chang Gung University College of Medicine"/>
  <meta name="citation_author_institution" content="Molecular Infectious Disease Research Center, Chang Gung Memorial Hospital, Chang Gung University College of Medicine"/>
  <meta name="citation_author" content="Cheng-Ta Yang"/>
  <meta name="citation_author_institution" content="Department of Respiratory Therapy, College of Medicine, Chang Gung University"/>
  <meta name="citation_author_institution" content="Department of Thoracic Medicine, Linkou Chang Gung Memorial Hospital"/>
  <meta name="citation_author" content="Shin-Ru Shih"/>
  <meta name="citation_author_institution" content="Research Center for Emerging Viral Infections, College of Medicine, Chang Gung University"/>
  <meta name="citation_author_institution" content="Department of Laboratory Medicine, Linkou Chang Gung Memorial Hospital"/>
  <meta name="citation_author_institution" content="Department of Medical Biotechnology and Laboratory Science, College of Medicine, Chang Gung University"/>
  <meta name="citation_author_institution" content="Research Center for Chinese Herbal Medicine, Research Center for Food and Cosmetic Safety, and Graduate Institute of Health Industry Technology, College of Human Ecology, Chang Gung University of Science and Technology"/>
  <meta name="citation_author_email" content="gwchen@mail.cgu.edu.tw"/>
  <meta name="citation_author_email" content="srshih@mail.cgu.edu.tw"/>
  <meta name="citation_author" content="Guang-Wu Chen"/>
  <meta name="citation_author_institution" content="Research Center for Emerging Viral Infections, College of Medicine, Chang Gung University"/>
  <meta name="citation_author_institution" content="Department of Laboratory Medicine, Linkou Chang Gung Memorial Hospital"/>
  <meta name="citation_author_institution" content="Department of Computer Science and Information Engineering, School of Electrical and Computer Engineering, College of Engineering, Chang Gung University"/>
  <meta name="citation_author_email" content="gwchen@mail.cgu.edu.tw"/>
  <meta name="citation_author_email" content="srshih@mail.cgu.edu.tw"/>
  <meta name="citation_reference" content="Zhu N, Zhang D, Wang W, Li X, Yang B, Song J, et al. A Novel Coronavirus from Patients with Pneumonia in China, 2019. N Engl J Med. 2020 Feb 20;382(8):727–33."/>
  <meta name="citation_reference" content="Coronavirus disease (COVID-2019) situation reports. [cited 2020 March 26]; https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports"/>
  <meta name="citation_reference" content="Number of Confirmed Cases of COVID-19 in Taiwan. [cited 2020 March 26]; https://sites.google.com/cdc.gov.tw/2019-ncov/taiwan"/>
  <meta name="citation_reference" content="Bolger AM, Lohse M, Usadel B. Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics. 2014 Aug 1;30(15):2114–20."/>
  <meta name="citation_reference" content="Kim D, Langmead B, Salzberg SL. HISAT: a fast spliced aligner with low memory requirements. Nat Methods. 2015 Apr;12(4):357–60."/>
  <meta name="citation_reference" content="Bankevich A, Nurk S, Antipov D, Gurevich AA, Dvorkin M, Kulikov AS, et al. SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J Comput Biol. 2012 May;19(5):455–77."/>
  <meta name="citation_reference" content="Gong YN, Chen GW, Yang SL, Lee CJ, Shih SR, Tsao KC. A Next-Generation Sequencing Data Analysis Pipeline for Detecting Unknown Pathogens from Mixed Clinical Samples and Revealing Their Genetic Diversity. PLoS One. 2016;11(3):e0151495."/>
  <meta name="citation_reference" content="Kuraku S, Zmasek CM, Nishimura O, Katoh K. aLeaves facilitates on-demand exploration of metazoan gene family trees on MAFFT sequence alignment server with enhanced interactivity. Nucleic Acids Res. 2013 Jul;41(Web Server issue):W22–8."/>
  <meta name="citation_reference" content="Stamatakis A. RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. Bioinformatics. 2014 May 1;30(9):1312–3."/>
  <meta name="citation_reference" content="Su YC AD, Young BE, Zhu F, Linster M, Kalimuddin S. Discovery of a 382-nt deletion during the early evolution of SARS-CoV-2. biorxiv. 2020."/>
  <meta name="citation_reference" content="Chinese SMEC. Molecular evolution of the SARS coronavirus during the course of the SARS epidemic in China. Science. 2004 Mar 12;303(5664):1666–9."/>
  <meta name="citation_reference" content="Hanada K, Suzuki Y, Gojobori T. A large variation in the rates of synonymous substitution for RNA viruses and its relationship to a diversity of viral infection and transmission modes. Mol Biol Evol. 2004 Jun;21(6):1074–80."/>
  <meta name="citation_reference" content="Eckerle LD, Lu X, Sperry SM, Choi L, Denison MR. High fidelity of murine hepatitis virus replication is decreased in nsp14 exoribonuclease mutants. J Virol. 2007 Nov;81(22):12135–44."/>
  <meta name="citation_reference" content="Denison MR, Graham RL, Donaldson EF, Eckerle LD, Baric RS. Coronaviruses: an RNA proofreading machine regulates replication fidelity and diversity. RNA Biol. 2011 Mar-Apr;8(2):270–9."/>
  <meta name="twitter:title" content="Sequence variation among SARS-CoV-2 isolates in Taiwan"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Taiwan experienced two waves of imported cases of coronavirus disease 2019 (COVID-19), first from China in January to late February, followed by those from other countries starting in early March. Additionally, several cases could not be traced to any imported cases and were suspected as sporadic local transmission. Twelve full viral genomes were determined in this study by Illumina sequencing either from virus isolates or directly from specimens, among which 5 originated from clustered infections. Phylogenetic tree analysis revealed that these sequences were in different clades, indicating that no major strain has been circulating in Taiwan. A deletion in open reading frame 8 was found in one isolate. Only a 4-nucleotide difference was observed among the 5 genomes from clustered infections."/>
  <meta name="og-title" property="og:title" content="Sequence variation among SARS-CoV-2 isolates in Taiwan"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.29.014290v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Taiwan experienced two waves of imported cases of coronavirus disease 2019 (COVID-19), first from China in January to late February, followed by those from other countries starting in early March. Additionally, several cases could not be traced to any imported cases and were suspected as sporadic local transmission. Twelve full viral genomes were determined in this study by Illumina sequencing either from virus isolates or directly from specimens, among which 5 originated from clustered infections. Phylogenetic tree analysis revealed that these sequences were in different clades, indicating that no major strain has been circulating in Taiwan. A deletion in open reading frame 8 was found in one isolate. Only a 4-nucleotide difference was observed among the 5 genomes from clustered infections."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-03-31"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Sequence variation among SARS-CoV-2 isolates in Taiwan | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1218993 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-3">Taiwan experienced two waves of imported cases of coronavirus disease 2019 (COVID-19), first from China in January to late February, followed by those from other countries starting in early March. Additionally, several cases could not be traced to any imported cases and were suspected as sporadic local transmission. Twelve full viral genomes were determined in this study by Illumina sequencing either from virus isolates or directly from specimens, among which 5 originated from clustered infections. Phylogenetic tree analysis revealed that these sequences were in different clades, indicating that no major strain has been circulating in Taiwan. A deletion in open reading frame 8 was found in one isolate. Only a 4-nucleotide difference was observed among the 5 genomes from clustered infections.</p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Introduction</h2>
    <p id="p-4">A novel coronavirus emerged from Wuhan, Hubei province in China in December 2019 (
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>). This virus has been designated as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), and the disease is named as coronavirus disease 2019 (COVID-19). The World Health Organization declared this disease a Public Health Emergency of International Concern on January 30, 2020. As of March 26, 2020, the outbreak of COVID-19 has resulted in 462,684 confirmed cases and 20,834 deaths worldwide (
     <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>), and 252 confirmed cases and two deaths were reported in Taiwan (
     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>).
    </p>
    <p id="p-5">There have been two waves of COVID-19 cases in Taiwan. The first occurred from late January to the end of February, with most cases imported from China, either by Chinese tourists or Taiwanese businessmen returning for Chinese New Year. This wave was smaller than the second wave. The second wave started in early March, during which the disease occurred largely in Taiwanese tourists, business travelers, or students returning from other countries. Although most of these cases were traced to their foreign origins, some small and clustered infections were suspected to have been acquired by local transmission.</p>
    <p id="p-6">In this study, we performed virus culture and full-genome sequencing of isolates or clinical specimens of SARS-CoV-2. We compared the genomes obtained from Taiwanese samples to those of other strains in a database to understand their evolutionary trajectory. An open reading frame 8 (ORF8) deletion was found in one strain. Moreover, we assessed the number of nucleotide substitutions that may have accumulated in clustered infections during a short period of time.</p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Methods</h2>
    <div id="sec-3" class="subsection">
     <h3>Specimen Collection</h3>
     <p id="p-7">Infection of patients by COVID-19 was confirmed by real-time reverse-transcriptase polymerase chain-reaction (RT-PCR) according to the guidelines of the Taiwan Centers for Disease Control (CDC; 
      <a href="https://www.cdc.gov.tw/En">https://www.cdc.gov.tw/En</a>), and all nasopharyngeal (NP), throat (TH) swab, and sputum (SP) samples were maintained in universal transport medium for further analysis.
     </p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>Cell Culture and Virus Isolation</h3>
     <p id="p-8">Vero-E6 (ATCC, Manassas, VA, USA) and MK-2 (ATCC) cells were maintained in Modified Eagle Medium (MEM, Thermo Fisher Scientific, Waltham, MA, USA) supplemented with 10% fetal bovine serum and 1x penicillin-streptomycin at 37°C in the presence of 5% CO
      <sub>2</sub>. To isolate the virus, all procedures following the laboratory biosafety guidelines of the Taiwan CDC were conducted in a biosafety Level-3 facility. Cells grown to 80–90% confluency in a T-25 flask were inoculated with 500 μL of virus solution, which was prepared by diluting 100 μL of specimen samples with 1.5 mL of sample pretreatment medium consisting of MEM and 2× penicillin-streptomycin solution, followed by incubation at 37°C for 1 h. The absorption was performed at 37°C for 1 h, then cells were refreshed with 5 mL virus culture medium composed of MEM, 2% fetal bovine serum, and 1× penicillin-streptomycin solution and maintained at 37°C. Infected cells were observed daily to determine their cytopathic effect. Additionally, RT-PCR analysis using the RNA extracted from part of the culture supernatant every two days after inoculation was performed to monitor viral growth. We continuously observed the infected cells until cytopathic effects occurred in more than 75% of the cells, after which the culture supernatant was harvested.
     </p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Whole-Genome Sequencing</h3>
     <p id="p-9">RNA was extracted either from the culture supernatant or directly from the specimens using a QIAmp viral RNA mini Kit (Qiagen, Hilden, Germany) according to the manufacturer’s instructions, except that the carrier RNA was replaced with linear acrylamide (Thermo Fisher Scientific) as the co-precipitant. The amount of viral RNA was evaluated by quantitative RT-PCR to examine the Ct value of the viral E gene. For RNAs showing a high Ct value, we used the Ovation RNA-Seq System V2 (Nugen Technologies, San Carlos, CA, USA) to synthesize cDNA which was further processed into a library using the Celero DNA-Seq System (Nugen Technologies). Other samples with lower Ct values were used for library preparation by using the Trio RNA-Seq kit (Nugen Technologies). Sequencing was performed on an Illumina MiSeq System (San Diego, CA, USA) with paired-end reads. More than 0.75 and 2.5 Gb of raw data were generated for samples from viral isolates and clinical specimens, respectively.</p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Next-generation Sequencing Data Analysis Pipeline</h3>
     <p id="p-10">We first trimmed the raw data by removing low-quality and short reads using Trimmomatic (version 0.39) (
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>). Next, quality reads were mapped to the human reference genome to remove host sequences using HISAT2 (version 2.1.0) (
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>). SPAdes (version 3.14.0) (
      <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>) was used to perform 
      <em>de novo</em> assembly for constructing contig sequences. Fourth, the BLASTN tool was used to search the assembled contigs against the nucleotide sequence (NT) database of the National Center for Biotechnology Information (NCBI). Viral candidates were identified using the reported top BLASTN hits for each of the queried contig sequences. Finally, we used an iterative mapping approach (
      <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>) to increase the read depth and coverage of quality contigs to obtain the whole genome.
     </p>
    </div>
    <div id="sec-7" class="subsection">
     <h3>Phylogenetic and Sequence Analysis</h3>
     <p id="p-11">Twelve whole genomes were assembled by using our pipeline, including three genomes from specimens and nine genomes from isolates, which were deposited in the Global Initiative on Sharing All Influenza Data (GISAID, 
      <a href="https://www.gisaid.org/">https://www.gisaid.org/</a>) with accession numbers EPI_ISL_411915, EPI_ISL_417518, EPI_ISL_415741–3, and EPI_ISL_417519–25, according to CGMH-CGU No. 1–12. We further downloaded all complete and high-coverage genomes from GISAID as of March 14, 2020, and obtained 335 sequences after removing those with sequences gaps or ambiguous nucleotides. One reference strain ( accession number 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=MN908947.3&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.29.014290.atom">MN908947.3</a>) was downloaded from GenBank (NCBI). In total 348 sequences were aligned using MAFFT (version 7.427) (
      <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>) for further analyses. The phylogenetic tree was inferred using RAxML (version 8.2.12) (
      <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>) under the GTRGAMMA model with a bootstrap value of 1000 to investigate the genomic relationships.
     </p>
    </div>
   </div>
   <div class="section" id="sec-8">
    <h2 class="">Results</h2>
    <div id="sec-9" class="subsection">
     <h3>Phylogenetic Tree of Taiwanese and Global Strains</h3>
     <p id="p-12">Twelve complete genomes from three specimens (CGMH-CGU No. 1, 7, and 8) and nine isolates (No. 2–6 and 9–12) were uploaded to GISAID. 
      <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> shows their next-generation sequencing (NGS) coverage and depth. All average depths were greater than 10,000, except for CGMH-CGU-04 and −08 which showed values of 446.0 and 53.0, respectively. 
      <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> also includes two earlier strains, hCoV-19/Taiwan/2/2020 and hCoV-19/Taiwan/3/2020, previously submitted by Taiwan CDC.
     </p>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1232051/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1232051/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1232051" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 1.</span>
       <span class="caption-title">Specimen collection, culture, and sequencing</span>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-13">The phylogenetic tree revealed that the SARS-CoV-2 viral genomes from Taiwan (highlighted) were in different clades (
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Viral genomes of No. 3–7 were from clustered infections, together with No. 8 (a case originating from the United Kingdom), and some from Australia (AUS) and New Zealand (NZ) in the yellow clade. Three patients with AUS/NZ infections had a travel history to Iran. This figure also shows eight additional Taiwanese isolates (highlighted), which appeared in distinct lineages, indicating that no single dominant strain has been circulating in Taiwan.
     </p>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.29.014290/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Phylogenetic tree of Taiwanese and global strains. Phylogeny was inferred using a maximum likelihood approach. Taiwanese strains are highlighted. Strains isolated from different locations and clades with specific variations are marked in different colors. Significant bootstrap support values greater than 70% are shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1083087760" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Phylogenetic tree of Taiwanese and global strains.&lt;;/span&gt;; Phylogeny was inferred using a maximum likelihood approach. Taiwanese strains are highlighted. Strains isolated from different locations and clades with specific variations are marked in different colors. Significant bootstrap support values greater than 70% are shown.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.29.014290/F1.medium.gif" width="440" height="311"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.29.014290/F1.medium.gif" width="440" height="311"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.29.014290/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.29.014290/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 1.</span>
       <span class="caption-title">Phylogenetic tree of Taiwanese and global strains.</span>
       <p id="p-14" class="first-child">Phylogeny was inferred using a maximum likelihood approach. Taiwanese strains are highlighted. Strains isolated from different locations and clades with specific variations are marked in different colors. Significant bootstrap support values greater than 70% are shown.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-15">The two earliest sequences in this yellow clade were dated to mid-January from Wuhan and Shangdong, which may have been the origin of the yellow clade. CGMH-CGU-03 had no travel history and the specimen was collected nearly 6 weeks after the two Chinese isolates were collected. All other viruses in this clade were also dated after February 26. Separated by this long duration from the two Chinese strains in mid-January, it is unlikely the later strains were directly linked to the Wuhan strains. Although some AUS/NZ cases in this clade had a travel history to Iran, the transmission route of these five Taiwanese cases remains unclear.</p>
    </div>
    <div id="sec-10" class="subsection">
     <h3>ORF8 Deletion Revealed by NGS Data Analysis</h3>
     <p id="p-16">
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a> shows the NGS coverage and depth of CGMH-CGU-01. This strain was identical to the WuHan-1 strain (accession number 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=MN908947.3&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.29.014290.atom">MN908947.3</a>). The most divergent strain among the 14 Taiwanese sequences was CGMH-CGU-04 which showed nine nucleotide changes (resulting in five amino acid changes) in the coding region compared to CGMH-CGU-01. Notably, we detected a deletion in a 382-nucleotide (nt) sequence at genomic positions 27,848– 28,229 in CGMH-CGU-02. 
      <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a> shows the coverage and depth of this strain. According to the reference strain (WuHan-1), the genomic position of ORF8 was 27,894–28,259 (
      <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>). This 382-nt deletion begins upstream of ORF8 to nearly the end of ORF8. We further performed NGS using a specimen isolated from the same patient. Reads yielding this 382-nt deletion were confirmed in original specimen, although only the partial genome was assembled (
      <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>).
     </p>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.29.014290/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="ORF8 deletion in SARS-CoV-2 genome. A and B) NGS depths of CGMH-CGU-01 and CGMH-CGU-02 and C) genomic regions of ORF8 and ORF8 deletion according to the reference strain are shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1083087760" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;ORF8 deletion in SARS-CoV-2 genome.&lt;;/span&gt;; A and B) NGS depths of CGMH-CGU-01 and CGMH-CGU-02 and C) genomic regions of ORF8 and ORF8 deletion according to the reference strain are shown.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.29.014290/F2.medium.gif" width="440" height="267"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.29.014290/F2.medium.gif" width="440" height="267"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.29.014290/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.29.014290/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 2.</span>
       <span class="caption-title">ORF8 deletion in SARS-CoV-2 genome.</span>
       <p id="p-17" class="first-child">A and B) NGS depths of CGMH-CGU-01 and CGMH-CGU-02 and C) genomic regions of ORF8 and ORF8 deletion according to the reference strain are shown.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-11" class="subsection">
     <h3>Within Four Nucleotide Changes among Virus Isolates from Clustered Patients</h3>
     <p id="p-18">COVID-19 has been reported to be transmitted through close contact among confirmed cases. Regardless of whether individuals are symptomatic, their family members and co-workers are at risk of becoming infected. Viral genomes No. 4–7 were from patients who had contact with an index patient (CGMH-CGU-03). To identify the number of nucleotides changed in the viral genome during clustered infections, we determined the viral full genomes either from viral isolates (No. 3–6) or specimens (No. 7) of these 5 cases. Although the genomes of samples No. 3, 5, and 6 were identical, they differed from that of No. 4 at 3 ORF1ab nucleotide positions A4788G, C10809T, and G21055A; the third position showed a synonymous change with a G7019S amino acid substitution (
      <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Number 7 showed only one nucleotide difference from No. 3, 5, and 6. These results suggest that only 4 nucleotide changes occurred in the viral genome among cases in clustered infections.
     </p>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.29.014290/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Nucleotide and amino acid variations in SARS-CoV-2 genomes. Compared to CGMH-CGU-01 (identical to the reference strain), nucleotide and amino acid variations in the SARS-CoV-2 genomes from Taiwanese strains are shown. Synonymous and nonsynonymous mutations were marked by blue and red color, respectively. Amino acid changes were annotated in parentheses. ORF8 deletion was marked in gray." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1083087760" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Nucleotide and amino acid variations in SARS-CoV-2 genomes.&lt;;/span&gt;; Compared to CGMH-CGU-01 (identical to the reference strain), nucleotide and amino acid variations in the SARS-CoV-2 genomes from Taiwanese strains are shown. Synonymous and nonsynonymous mutations were marked by blue and red color, respectively. Amino acid changes were annotated in parentheses. ORF8 deletion was marked in gray.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.29.014290/F3.medium.gif" width="356" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.29.014290/F3.medium.gif" width="356" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.29.014290/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.29.014290/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 3.</span>
       <span class="caption-title">Nucleotide and amino acid variations in SARS-CoV-2 genomes.</span>
       <p id="p-19" class="first-child">Compared to CGMH-CGU-01 (identical to the reference strain), nucleotide and amino acid variations in the SARS-CoV-2 genomes from Taiwanese strains are shown. Synonymous and nonsynonymous mutations were marked by blue and red color, respectively. Amino acid changes were annotated in parentheses. ORF8 deletion was marked in gray.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
   </div>
   <div class="section" id="sec-12">
    <h2 class="">Discussion</h2>
    <p id="p-20">Twelve full viral genomes were resolved in this study either from virus isolates or directly from specimens. Phylogenetic tree analysis showed that these sequences were in different clades, indicating that no major strain is currently circulating in Taiwan. A deletion in ORF8 was found in one isolate, which has also been detected in patients in Singapore (
     <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>). Four or fewer nucleotide differences were observed in the 5 genomes from clustered infections.
    </p>
    <p id="p-21">We detected a 382-nt deletion covering nearly the entire open reading frame 8 of the CGMH-CGU-2 isolate obtained from a patient who returned from Wuhan in January. A similar observation was reported for eight hospitalized patients in Singapore. During the SARS-CoV outbreak in 2003, deletions in ORF8 were observed, which were associated with a reduced ability for virus replication in human cells (
     <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>).
    </p>
    <p id="p-22">RNA viruses show variations in their genomes due to nucleotide substitutions generated by the low fidelity of RNA-dependent RNA polymerase during replication. The genome variation of these viruses is thought to facilitate successful adaption to the environment of various hosts. However, previous studies showed that the mutation rates of RNA viruses vary in different viruses and depend on the viral transmission modes (
     <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>). Sequence analysis of SARS-CoV-2 isolated from 5 patients from February 26 to March 9, 2020 in CGMH Taiwan revealed only 4 mutations in their 29,903-nt genomic RNA. This suggests that the nucleotide substitution rate is controlled during viral RNA replication. The nsp14 exoribonuclease encoded by several coronaviruses plays a role in proofreading during genome replication (
     <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>, 
     <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>); further studies are required to investigate the function of SARS-CoV-2 nsp14 in replication fidelity.
    </p>
    <p id="p-23">Timely sharing full genomes of SASR-CoV-2 from different locations is important for monitoring genetic changes in the virus which may be associated with viral spreading and clinical manifestations. We determined the sequences of SARS-CoV-2 in Taiwan in different clades. Moreover, four or fewer nucleotide changes in viral genomes from five cases in clustered infections indicated that sequencing is a useful tool for tracing the source of infection for this type of RNA virus.</p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgments</h2>
    <p id="p-24">This work was financially supported by the Research Center for Emerging Viral Infections from The Featured Areas Research Center Program within the framework of the Higher Education Sprout Project by the Ministry of Education (MOE) in Taiwan, the Ministry of Science and Technology (MOST), Taiwan (MOST 108-3017-F-182-001, MOST 107-2221-E-182-064-MY2, and MOST 106-2320-B-182A-013-MY3), and Linkou Chang Gung Memorial Hospital, Taiwan (No. CLRPG3B0048 and CMRPD1H0231–3).</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.29.014290v1.1" data-doi="10.1056/NEJMoa2001017">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Song</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A Novel Coronavirus from Patients with Pneumonia in China, 2019</span>. 
         <abbr class="cit-jnl-abbrev">N Engl J Med.</abbr>
         <span class="cit-pub-date">2020</span>
         <span class="cit-month">Feb</span>
         <span class="cit-day">20</span>;
         <span class="cit-vol">382</span>(
         <span class="cit-issue">8</span>):
         <span class="cit-fpage">727</span>–
         <span class="cit-lpage">33</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed.%26rft.volume%253D382%26rft.spage%253D727%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa2001017%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1056/NEJMoa2001017&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=http://www.n&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.29.014290.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.03.29.014290v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-source">Coronavirus disease (COVID-2019) situation reports</span>. [cited 
         <span class="cit-date-in-citation">2020 March 26</span>]; 
         <a href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports">https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports</a>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.03.29.014290v1.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-source">Number of Confirmed Cases of COVID-19 in Taiwan</span>. [cited 
         <span class="cit-date-in-citation">2020 March 26</span>]; 
         <a href="https://sites.google.com/cdc.gov.tw/2019-ncov/taiwan">https://sites.google.com/cdc.gov.tw/2019-ncov/taiwan</a>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.29.014290v1.4" data-doi="10.1093/bioinformatics/btu170">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bolger</span>
          <span class="cit-name-given-names">AM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lohse</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Usadel</span>
          <span class="cit-name-given-names">B</span>
         </span>. 
         <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics.</abbr>
         <span class="cit-pub-date">2014</span>
         <span class="cit-month">Aug</span>
         <span class="cit-day">1</span>;
         <span class="cit-vol">30</span>(
         <span class="cit-issue">15</span>):
         <span class="cit-fpage">2114</span>–
         <span class="cit-lpage">20</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24695404&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.29.014290.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000340049100004&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.29.014290v1.5" data-doi="10.1038/nmeth.3317">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kim</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Langmead</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Salzberg</span>
          <span class="cit-name-given-names">SL</span>
         </span>. 
         <span class="cit-article-title">HISAT: a fast spliced aligner with low memory requirements</span>. 
         <abbr class="cit-jnl-abbrev">Nat Methods</abbr>. 
         <span class="cit-pub-date">2015</span>
         <span class="cit-month">Apr</span>;
         <span class="cit-vol">12</span>(
         <span class="cit-issue">4</span>):
         <span class="cit-fpage">357</span>–
         <span class="cit-lpage">60</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D12%26rft.spage%253D357%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3317%26rft_id%253Dinfo%253Apmid%252F25751142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nmeth.3317&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25751142&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.29.014290.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.29.014290v1.6" data-doi="10.1089/cmb.2012.0021">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bankevich</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nurk</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Antipov</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gurevich</span>
          <span class="cit-name-given-names">AA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dvorkin</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kulikov</span>
          <span class="cit-name-given-names">AS</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing</span>. 
         <abbr class="cit-jnl-abbrev">J Comput Biol.</abbr>
         <span class="cit-pub-date">2012</span>
         <span class="cit-month">May</span>;
         <span class="cit-vol">19</span>(
         <span class="cit-issue">5</span>):
         <span class="cit-fpage">455</span>–
         <span class="cit-lpage">77</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DBankevich%26rft.auinit1%253DA.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D455%26rft.epage%253D477%26rft.atitle%253DSPAdes%253A%2Ba%2Bnew%2Bgenome%2Bassembly%2Balgorithm%2Band%2Bits%2Bapplications%2Bto%2Bsingle-cell%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2012.0021%26rft_id%253Dinfo%253Apmid%252F22506599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1089/cmb.2012.0021&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22506599&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.29.014290.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.29.014290v1.7" data-doi="10.1371/journal.pone.0151495">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gong</span>
          <span class="cit-name-given-names">YN</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">GW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">SL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lee</span>
          <span class="cit-name-given-names">CJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shih</span>
          <span class="cit-name-given-names">SR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tsao</span>
          <span class="cit-name-given-names">KC</span>
         </span>. 
         <span class="cit-article-title">A Next-Generation Sequencing Data Analysis Pipeline for Detecting Unknown Pathogens from Mixed Clinical Samples and Revealing Their Genetic Diversity</span>. 
         <abbr class="cit-jnl-abbrev">PLoS One</abbr>. 
         <span class="cit-pub-date">2016</span>;
         <span class="cit-vol">11</span>(
         <span class="cit-issue">3</span>):
         <span class="cit-fpage">e0151495</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0151495%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0151495%26rft_id%253Dinfo%253Apmid%252F26986479%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0151495&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26986479&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.29.014290.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.29.014290v1.8" data-doi="10.1093/nar/gkt389">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kuraku</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zmasek</span>
          <span class="cit-name-given-names">CM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nishimura</span>
          <span class="cit-name-given-names">O</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Katoh</span>
          <span class="cit-name-given-names">K</span>
         </span>. 
         <span class="cit-article-title">aLeaves facilitates on-demand exploration of metazoan gene family trees on MAFFT sequence alignment server with enhanced interactivity</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>
         <span class="cit-pub-date">2013</span>
         <span class="cit-month">Jul</span>;
         <span class="cit-vol">41</span>(
         <span class="cit-issue">Web Server issue</span>):
         <span class="cit-fpage">W22</span>–
         <span class="cit-lpage">8</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt389%26rft_id%253Dinfo%253Apmid%252F23677614%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkt389&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23677614&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.29.014290.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.29.014290v1.9" data-doi="10.1093/bioinformatics/btu033">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Stamatakis</span>
          <span class="cit-name-given-names">A</span>
         </span>. 
         <span class="cit-article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. 
         <span class="cit-pub-date">2014</span>
         <span class="cit-month">May</span>
         <span class="cit-day">1</span>;
         <span class="cit-vol">30</span>(
         <span class="cit-issue">9</span>):
         <span class="cit-fpage">1312</span>–
         <span class="cit-lpage">3</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu033%26rft_id%253Dinfo%253Apmid%252F24451623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu033&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24451623&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.29.014290.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000336095100024&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.29.014290v1.10">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Su YC</span>
          <span class="cit-name-given-names">AD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Young</span>
          <span class="cit-name-given-names">BE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Linster</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kalimuddin</span>
          <span class="cit-name-given-names">S</span>
         </span>. 
         <span class="cit-article-title">Discovery of a 382-nt deletion during the early evolution of SARS-CoV-2</span>. 
         <abbr class="cit-jnl-abbrev">biorxiv</abbr>. 
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.29.014290v1.11" data-doi="10.1126/science.1092002">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chinese</span>
          <span class="cit-name-given-names">SMEC</span>
         </span>. 
         <span class="cit-article-title">Molecular evolution of the SARS coronavirus during the course of the SARS epidemic in China</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>. 
         <span class="cit-pub-date">2004</span>
         <span class="cit-month">Mar</span>
         <span class="cit-day">12</span>;
         <span class="cit-vol">303</span>(
         <span class="cit-issue">5664</span>):
         <span class="cit-fpage">1666</span>–
         <span class="cit-lpage">9</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1092002%26rft_id%253Dinfo%253Apmid%252F14752165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDMvNTY2NC8xNjY2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMzEvMjAyMC4wMy4yOS4wMTQyOTAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.29.014290v1.12" data-doi="10.1093/molbev/msh109">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hanada</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Suzuki</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gojobori</span>
          <span class="cit-name-given-names">T</span>
         </span>. 
         <span class="cit-article-title">A large variation in the rates of synonymous substitution for RNA viruses and its relationship to a diversity of viral infection and transmission modes</span>. 
         <abbr class="cit-jnl-abbrev">Mol Biol Evol.</abbr>
         <span class="cit-pub-date">2004</span>
         <span class="cit-month">Jun</span>;
         <span class="cit-vol">21</span>(
         <span class="cit-issue">6</span>):
         <span class="cit-fpage">1074</span>–
         <span class="cit-lpage">80</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsh109%26rft_id%253Dinfo%253Apmid%252F15014142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/molbev/msh109&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15014142&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.29.014290.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000221599300012&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.29.014290v1.13" data-doi="10.1128/JVI.01296-07">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Eckerle</span>
          <span class="cit-name-given-names">LD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sperry</span>
          <span class="cit-name-given-names">SM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Choi</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Denison</span>
          <span class="cit-name-given-names">MR</span>
         </span>. 
         <span class="cit-article-title">High fidelity of murine hepatitis virus replication is decreased in nsp14 exoribonuclease mutants</span>. 
         <abbr class="cit-jnl-abbrev">J Virol.</abbr>
         <span class="cit-pub-date">2007</span>
         <span class="cit-month">Nov</span>;
         <span class="cit-vol">81</span>(
         <span class="cit-issue">22</span>):
         <span class="cit-fpage">12135</span>–
         <span class="cit-lpage">44</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01296-07%26rft_id%253Dinfo%253Apmid%252F17804504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4MS8yMi8xMjEzNSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzMxLzIwMjAuMDMuMjkuMDE0MjkwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.29.014290v1.14" data-doi="10.4161/rna.8.2.15013">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Denison</span>
          <span class="cit-name-given-names">MR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Graham</span>
          <span class="cit-name-given-names">RL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Donaldson</span>
          <span class="cit-name-given-names">EF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Eckerle</span>
          <span class="cit-name-given-names">LD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Baric</span>
          <span class="cit-name-given-names">RS</span>
         </span>. 
         <span class="cit-article-title">Coronaviruses: an RNA proofreading machine regulates replication fidelity and diversity</span>. 
         <abbr class="cit-jnl-abbrev">RNA Biol.</abbr>
         <span class="cit-pub-date">2011</span>
         <span class="cit-month">Mar-Apr</span>;
         <span class="cit-vol">8</span>(
         <span class="cit-issue">2</span>):
         <span class="cit-fpage">270</span>–
         <span class="cit-lpage">9</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DRNA%2BBiol%26rft.aulast%253DDenison%26rft.auinit1%253DM.%2BR.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253D270%26rft.epage%253D279%26rft.atitle%253DCoronaviruses%253A%2Ban%2BRNA%2Bproofreading%2Bmachine%2Bregulates%2Breplication%2Bfidelity%2Band%2Bdiversity.%26rft_id%253Dinfo%253Adoi%252F10.4161%252Frna.8.2.15013%26rft_id%253Dinfo%253Apmid%252F21593585%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.4161/rna.8.2.15013&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21593585&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.29.014290.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000289934300013&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
