<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/03/08/571455.atom" />
<meta name="HW.pisa" content="biorxiv;571455v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Noroviruses subvert the core stress granule component G3BP1 to promote viral VPg-dependent translation" />
<meta name="DC.Identifier" content="10.1101/571455" />
<meta name="DC.Date" content="2019-03-08" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Knowledge of the host factors required for norovirus replication has been hindered by the challenges associated with culturing human noroviruses. We have combined proteomic analysis of the viral translation and replication complexes with a CRISPR screen, to identify host factors required for norovirus infection. The core stress granule component G3BP1 was identified as a host factor essential for efficient human and murine norovirus infection, demonstrating a conserved function across the Norovirus genus. Furthermore, we show that G3BP1 functions in the novel paradigm of viral VPg-dependent translation initiation, contributing to the assembly of translation complexes on the VPg-linked viral positive sense RNA genome by facilitating 40S recruitment. Our data suggest that G3BP1 functions by providing viral RNA a competitive advantage over capped cellular RNAs, uncovering a novel function for G3BP1 in the life cycle of positive sense RNA viruses and identifying the first host factor with pan-norovirus pro-viral activity." />
<meta name="DC.Contributor" content="Myra Hosmillo" />
<meta name="DC.Contributor" content="Jia Lu" />
<meta name="DC.Contributor" content="Michael R. McAllaster" />
<meta name="DC.Contributor" content="James B. Eaglesham" />
<meta name="DC.Contributor" content="Xinjie Wang" />
<meta name="DC.Contributor" content="Edward Emmott" />
<meta name="DC.Contributor" content="Patricia Domingues" />
<meta name="DC.Contributor" content="Yasmin Chaudhry" />
<meta name="DC.Contributor" content="Timothy J Fitzmaurice" />
<meta name="DC.Contributor" content="Matthew K.H. Tung" />
<meta name="DC.Contributor" content="Marc Panas" />
<meta name="DC.Contributor" content="Gerald McInerney" />
<meta name="DC.Contributor" content="Nicholas Locker" />
<meta name="DC.Contributor" content="Craig B. Willen" />
<meta name="DC.Contributor" content="Ian Goodfellow" />
<meta name="article:published_time" content="2019-03-08" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Noroviruses subvert the core stress granule component G3BP1 to promote viral VPg-dependent translation" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Knowledge of the host factors required for norovirus replication has been hindered by the challenges associated with culturing human noroviruses. We have combined proteomic analysis of the viral translation and replication complexes with a CRISPR screen, to identify host factors required for norovirus infection. The core stress granule component G3BP1 was identified as a host factor essential for efficient human and murine norovirus infection, demonstrating a conserved function across the &lt;i&gt;Norovirus&lt;/i&gt; genus. Furthermore, we show that G3BP1 functions in the novel paradigm of viral VPg-dependent translation initiation, contributing to the assembly of translation complexes on the VPg-linked viral positive sense RNA genome by facilitating 40S recruitment. Our data suggest that G3BP1 functions by providing viral RNA a competitive advantage over capped cellular RNAs, uncovering a novel function for G3BP1 in the life cycle of positive sense RNA viruses and identifying the first host factor with pan-norovirus pro-viral activity.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;571455v1" />
<meta name="citation_id" content="571455v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/571455v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/571455v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/571455v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455.full.pdf" />
<meta name="citation_doi" content="10.1101/571455" />
<meta name="citation_num_pages" content="63" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="571455" />
<meta name="citation_author" content="Myra Hosmillo" />
<meta name="citation_author_institution" content="Division of Virology, Department of Pathology, University of Cambridge" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3514-7681" />
<meta name="citation_author" content="Jia Lu" />
<meta name="citation_author_institution" content="Division of Virology, Department of Pathology, University of Cambridge" />
<meta name="citation_author" content="Michael R. McAllaster" />
<meta name="citation_author_institution" content="Washington University School of Medicine, Department of Pathology and Immunology" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9729-7290" />
<meta name="citation_author" content="James B. Eaglesham" />
<meta name="citation_author_institution" content="Division of Virology, Department of Pathology, University of Cambridge" />
<meta name="citation_author_institution" content="Department of Microbiology, Harvard Medical School" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5725-0743" />
<meta name="citation_author" content="Xinjie Wang" />
<meta name="citation_author_institution" content="Division of Virology, Department of Pathology, University of Cambridge" />
<meta name="citation_author_institution" content="Institute for Brain Research and Rehabilitation, South China Normal University" />
<meta name="citation_author" content="Edward Emmott" />
<meta name="citation_author_institution" content="Division of Virology, Department of Pathology, University of Cambridge" />
<meta name="citation_author_institution" content="Department of Bioengineering &amp; Barnett Institute for Chemical and Biological Analyses, Northeastern University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3239-8178" />
<meta name="citation_author" content="Patricia Domingues" />
<meta name="citation_author_institution" content="Division of Virology, Department of Pathology, University of Cambridge" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1158-2926" />
<meta name="citation_author" content="Yasmin Chaudhry" />
<meta name="citation_author_institution" content="Division of Virology, Department of Pathology, University of Cambridge" />
<meta name="citation_author" content="Timothy J Fitzmaurice" />
<meta name="citation_author_institution" content="Division of Virology, Department of Pathology, University of Cambridge" />
<meta name="citation_author" content="Matthew K.H. Tung" />
<meta name="citation_author_institution" content="Division of Virology, Department of Pathology, University of Cambridge" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7088-226X" />
<meta name="citation_author" content="Marc Panas" />
<meta name="citation_author_institution" content="Department of Microbiology, Tumor and Cell Biology, Karolinska Institute" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7373-0341" />
<meta name="citation_author" content="Gerald McInerney" />
<meta name="citation_author_institution" content="Department of Microbiology, Tumor and Cell Biology, Karolinska Institute" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2257-7241" />
<meta name="citation_author" content="Nicholas Locker" />
<meta name="citation_author_institution" content="School of Biosciences and Medicine, University of Surrey" />
<meta name="citation_author" content="Craig B. Willen" />
<meta name="citation_author_institution" content="Yale School of Medicine, Departments of Laboratory Medicine and Immunobiology" />
<meta name="citation_author_email" content="craig.wilen@yale.edu" />
<meta name="citation_author_email" content="ig299@cam.ac.uk" />
<meta name="citation_author" content="Ian Goodfellow" />
<meta name="citation_author_institution" content="Division of Virology, Department of Pathology, University of Cambridge" />
<meta name="citation_author_email" content="craig.wilen@yale.edu" />
<meta name="citation_author_email" content="ig299@cam.ac.uk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9483-510X" />
<meta name="citation_reference" content="Alam, U., and Kennedy, D. (2019). Rasputin a decade on and more promiscuous than ever? A review of G3BPs. Biochim Biophys Acta Mol Cell Res 1866, 360–370." />
<meta name="citation_reference" content="Altan-Bonnet, N. (2017). Lipid Tales of Viral Replication and Transmission. Trends Cell Biol. 27, 201–213." />
<meta name="citation_reference" content="Bailey, D., Carrara, G., Benson, A., Chaudhry, Y., Heeney, J., Yarovinsky, F., Simmonds, P., Goodfellow, I., McFadden, N., Shortland, A., et al. (2011). Norovirus regulation of the innate immune response and apoptosis occurs via the product of the alternative open reading frame 4. PLoS Pathog. 7, e1002413." />
<meta name="citation_reference" content="Bailey, D., Karakasiliotis, I., Vashist, S., Chung, L.M.W., Rees, J., Reese, J., McFadden, N., Benson, A., Yarovinsky, F., Simmonds, P., et al. (2010). Functional analysis of RNA structures present at the 3’ extremity of the murine norovirus genome: the variable polypyrimidine tract plays a role in viral virulence. J Virol 84, 2859–2870." />
<meta name="citation_reference" content="Bartsch, S.M., Lopman, B.A., Ozawa, S., Hall, A.J., and Lee, B.Y. (2016). Global Economic Burden of Norovirus Gastroenteritis. PLoS ONE 11, e0151219." />
<meta name="citation_reference" content="Blasi, E., Barluzzi, R., Bocchini, V., Mazzolla, R., and Bistoni, F. (1990). Immortalization of murine microglial cells by a v-raf/v-myc carrying retrovirus. J. Neuroimmunol. 27, 229–237." />
<meta name="citation_reference" content="Brocard, M., Iadevaia, V., Klein, P., Hall, B., Lewis, G., Lu, J., Burke, J., Parker, R., Ruggieri, A., Goodfellow, I., et al. (2018). Norovirus infection results in assembly of virus-specific G3BP1 granules and evasion of eIF2α signaling. bioRxiv 490318." />
<meta name="citation_reference" content="Chang, K.-O., Sosnovtsev, S.V., Belliot, G., King, A.D., and Green, K.Y. (2006). Stable expression of a Norwalk virus RNA replicon in a human hepatoma cell line. Virology 353, 463–473." />
<meta name="citation_reference" content="Chaudhry, Y., Nayak, A., Bordeleau, M.-E., Tanaka, J., Pelletier, J., Belsham, G.J., Roberts, L.O., and Goodfellow, I.G. (2006). Caliciviruses differ in their functional requirements for eIF4F components. J. Biol. Chem. 281, 25315–25325." />
<meta name="citation_reference" content="Chaudhry, Y., Skinner, M.A., and Goodfellow, I.G. (2007). Recovery of genetically defined murine norovirus in tissue culture by using a fowlpox virus expressing T7 RNA polymerase. J. Gen. Virol. 88, 2091–2100." />
<meta name="citation_reference" content="Chung, L., Bailey, D., Leen, E.N., Emmott, E.P., Chaudhry, Y., Roberts, L.O., Curry, S., Locker, N., and Goodfellow, I.G. (2014). Norovirus translation requires an interaction between the C Terminus of the genome-linked viral protein VPg and eukaryotic translation initiation factor 4G. J. Biol. Chem. 289, 21738–21750." />
<meta name="citation_reference" content="Cotton, B.T., Hyde, J.L., Sarvestani, S.T., Sosnovtsev, S.V., Green, K.Y., White, P.A., and Mackenzie, J.M. (2017). The Norovirus NS3 Protein Is a Dynamic Lipid-and Microtubule-Associated Protein Involved in Viral RNA Replication. J Virol 91, e02138–16." />
<meta name="citation_reference" content="Cristea, I.M., Rozjabek, H., Molloy, K.R., Karki, S., White, L.L., Rice, C.M., Rout, M.P., Chait, B.T., and MacDonald, M.R. (2010). Host factors associated with the Sindbis virus RNA-dependent RNA polymerase: role for G3BP1 and G3BP2 in virus replication. J Virol 84, 6720–6732." />
<meta name="citation_reference" content="Doench, J.G., Fusi, N., Sullender, M., Hegde, M., Vaimberg, E.W., Donovan, K.F., Smith, I., Tothova, Z., Wilen, C., Orchard, R., et al. (2016). Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR-Cas9. Nat. Biotechnol. 34, 184–191." />
<meta name="citation_reference" content="Emmott, E., and Goodfellow, I. (2014). Identification of protein interaction partners in mammalian cells using SILAC-immunoprecipitation quantitative proteomics. J Vis Exp e51656–e51656." />
<meta name="citation_reference" content="Emmott, E., de Rougemont, A., Hosmillo, M., Lu, J., Fitzmaurice, T.J., Haas, J., and Goodfellow, I. (2019). Polyprotein processing and intermolecular interactions within the viral replication complex spatially and temporally control norovirus protease activity. J. Biol. Chem. jbc.RA118.006780." />
<meta name="citation_reference" content="Emmott, E., Sorgeloos, F., Caddy, S.L., Vashist, S., Sosnovtsev, S., Lloyd, R., Heesom, K., Locker, N., and Goodfellow, I. (2017). Norovirus-mediated modification of the translational landscape via virus and host-induced cleavage of translation initiation factors. Mol. Cell Proteomics mcp.M116.062448." />
<meta name="citation_reference" content="Ettayebi, K., Crawford, S.E., Murakami, K., Broughman, J.R., Karandikar, U., Tenge, V.R., Neill, F.H., Blutt, S.E., Zeng, X.-L., Qu, L., et al. (2016). Replication of human noroviruses in stem cell-derived human enteroids. Science 353, 1387–1393." />
<meta name="citation_reference" content="Favre, D., and Trepo, C. (2001). Translational extracts active biologically in vitro obtained from eukaryotic monolayer cells: a versatile method for viral RNA studies. Virol. Methods 92, 177–181." />
<meta name="citation_reference" content="Firth, A.E., and Brierley, I. (2012). Non-canonical translation in RNA viruses. J. Gen. Virol. 93, 1385–1409." />
<meta name="citation_reference" content="Fritzlar, S., Aktepe, T., Chao, Y.-W., McAllaster, M., Wilen, C., White, P., and Mackenzie, J. (2019). Mouse Norovirus infection arrests host cell translation uncoupled from the stress granule-PKR-eIF2α axis. bioRxiv 536052." />
<meta name="citation_reference" content="Goodfellow, I. (2011). The genome-linked protein VPg of vertebrate viruses — a multifaceted protein. Curr Opin Virol 1, 355–362." />
<meta name="citation_reference" content="Goodfellow, I., Chaudhry, Y., Gioldasi, I., Gerondopoulos, A., Natoni, A., Labrie, L., Laliberté, J.-F., and Roberts, L. (2005). Calicivirus translation initiation requires an interaction between VPg and eIF 4 E. EMBO Rep. 6, 968–972." />
<meta name="citation_reference" content="Haga, K., Fujimoto, A., Takai-Todaka, R., Miki, M., Doan, Y.H., Murakami, K., Yokoyama, M., Murata, K., Nakanishi, A., and Katayama, K. (2016). Functional receptor molecules CD300lf and CD300ld within the CD300 family enable murine noroviruses to infect cells. Proc. Natl. Acad. Sci. U.S.a. 113, E6248–E6255." />
<meta name="citation_reference" content="Hosmillo, M., Chaudhry, Y., Kim, D.-S., Goodfellow, I., and Cho, K.-O. (2014). Sapovirus translation requires an interaction between VPg and the cap binding protein eIF4E. J Virol 88, 12213–12221." />
<meta name="citation_reference" content="Humoud, M.N., Doyle, N., Royall, E., Willcocks, M.M., Sorgeloos, F., van Kuppeveld, F., Roberts, L.O., Goodfellow, I.G., Langereis, M.A., and Locker, N. (2016). Feline Calicivirus Infection Disrupts Assembly of Cytoplasmic Stress Granules and Induces G3BP1 Cleavage. J Virol 90, 6489–6501." />
<meta name="citation_reference" content="Hyde, J.L., and Mackenzie, J.M. (2010). Subcellular localization of the MNV-1 ORF1 proteins and their potential roles in the formation of the MNV-1 replication complex. Virology 406, 138–148." />
<meta name="citation_reference" content="Hyde, J.L., Sosnovtsev, S.V., Green, K.Y., Wobus, C., Virgin, H.W., and Mackenzie, J.M. (2009). Mouse norovirus replication is associated with virus-induced vesicle clusters originating from membranes derived from the secretory pathway. J Virol 83, 9709–9719." />
<meta name="citation_reference" content="Jaafar, Z.A., and Kieft, J.S. (2019). Viral RNA structure-based strategies to manipulate translation. Nat. Rev. Microbiol. 17, 110–123." />
<meta name="citation_reference" content="Jones, M.K., Watanabe, M., Zhu, S., Graves, C.L., Keyes, L.R., Grau, K.R., Gonzalez-Hernandez, M.B., Iovine, N.M., Wobus, C.E., Vinjé, J., et al. (2014). Enteric bacteria promote human and mouse norovirus infection of B cells. Science 346, 755–759." />
<meta name="citation_reference" content="Karst, S.M., Wobus, C.E., Lay, M., Davidson, J., and Virgin, H.W. (2003). STAT1-dependent innate immunity to a Norwalk-like virus. Science 299, 1575–1578." />
<meta name="citation_reference" content="Kedersha, N., Panas, M.D., Achorn, C.A., Lyons, S., Tisdale, S., Hickman, T., Thomas, M., Lieberman, J., McInerney, G.M., Ivanov, P., et al. (2016). G3BP-Caprin1-USP10 complexes mediate stress granule condensation and associate with 40S subunits. J. Cell Biol. 212, 845–860." />
<meta name="citation_reference" content="Kim, D.Y., Reynaud, J.M., Rasalouskaya, A., Akhrymuk, I., Mobley, J.A., Frolov, I., and Frolova, E.I. (2016). New World and Old World Alphaviruses Have Evolved to Exploit Different Components of Stress Granules, FXR and G3BP Proteins, for Assembly of Viral Replication Complexes. PLoS Pathog. 12, e1005810." />
<meta name="citation_reference" content="Kitano, M., Hosmillo, M., Emmott, E., Lu, J., and Goodfellow, I. (2018). Selection and Characterization of Rupintrivir-Resistant Norwalk Virus Replicon Cells In Vitro. Antimicrob. Agents Chemother. 62, 725." />
<meta name="citation_reference" content="Kumar, P., Hellen, C.U.T., and Pestova, T.V. (2016). Toward the mechanism of eIF4F-mediated ribosomal attachment to mammalian capped mRNAs. Genes Dev. 30, 1573–1588." />
<meta name="citation_reference" content="Leen, E.N., Sorgeloos, F., Correia, S., Chaudhry, Y., Cannac, F., Pastore, C., Xu, Y., Graham, S.C., Matthews, S.J., Goodfellow, I.G., et al. (2016). A Conserved Interaction between a C-Terminal Motif in Norovirus VPg and the HEAT-1 Domain of eIF4G Is Essential for Translation Initiation. PLoS Pathog. 12, e1005379." />
<meta name="citation_reference" content="Li, T.-F., Hosmillo, M., Schwanke, H., Shu, T., Wang, Z., Yin, L., Curry, S., Goodfellow, I.G., and Zhou, X. (2018). Human Norovirus NS3 Has RNA Helicase and Chaperoning Activities. J Virol 92, 312." />
<meta name="citation_reference" content="López-Manríquez, E., Vashist, S., Ureña, L., Goodfellow, I., Chavez, P., Mora-Heredia, J.E., Cancio-Lonches, C., Garrido, E., and Gutiérrez-Escolano, A.L. (2013). Norovirus Genome Circularisation and Efficient Replication is Facilitated by Binding of PCBP2 and hnRNP A1. J Virol 87, 11371–11387." />
<meta name="citation_reference" content="McCormick, C., and Khaperskyy, D.A. (2017). Translation inhibition and stress granules in the antiviral immune response. Nat. Rev. Immunol. 17, 647–660." />
<meta name="citation_reference" content="McCune, B.T., Tang, W., Lu, J., Eaglesham, J.B., Thorne, L., Mayer, A.E., Condiff, E., Nice, T.J., Goodfellow, I., Krezel, A.M., et al. (2017). Noroviruses Co-opt the Function of Host Proteins VAPA and VAPB for Replication via a Phenylalanine-Phenylalanine-Acidic-Tract-Motif Mimic in Nonstructural Viral Protein NS1/2. MBio 8, e00668–17." />
<meta name="citation_reference" content="Nice, T.J., Strong, D.W., McCune, B.T., Pohl, C.S., and Virgin, H.W. (2012). A single amino acid change in murine norovirus NS1/2 is sufficient for colonic tropism and persistence. J Virol." />
<meta name="citation_reference" content="Orchard, R.C., Wilen, C.B., Doench, J.G., Baldridge, M.T., McCune, B.T., Lee, Y.-C.J., Lee, S., Pruett-Miller, S.M., Nelson, C.A., Fremont, D.H., et al. (2016). Discovery of a proteinaceous cellular receptor for a norovirus. Science 353, 933–936." />
<meta name="citation_reference" content="Panas, M.D., Ahola, T., and McInerney, G.M. (2014). The C-terminal repeat domains of nsP3 from the Old World alphaviruses bind directly to G3BP. J Virol 88, 5888–5893." />
<meta name="citation_reference" content="Panas, M.D., Schulte, T., Thaa, B., Sandalova, T., Kedersha, N., Achour, A., and McInerney, G.M. (2015). Viral and cellular proteins containing FGDF motifs bind G3BP to block stress granule formation. PLoS Pathog. 11, e1004659." />
<meta name="citation_reference" content="Panas, M.D., Varjak, M., Lulla, A., Eng, K.E., Merits, A., Karlsson Hedestam, G.B., and McInerney, G.M. (2012). Sequestration of G3BP coupled with efficient translation inhibits stress granules in Semliki Forest virus infection. Mol Biol Cell 23, 4701–4712." />
<meta name="citation_reference" content="Protter, D.S.W., and Parker, R. (2016). Principles and Properties of Stress Granules. Trends Cell Biol. 26, 668–679." />
<meta name="citation_reference" content="Rakotondrafara, A.M., and Hentze, M.W. (2011). An efficient factor-depleted mammalian in vitro translation system. Nat Protoc 6, 563–571." />
<meta name="citation_reference" content="Rocha-Pereira, J., Jochmans, D., Dallmeier, K., Leyssen, P., Cunha, R., Costa, I., Nascimento, M.S.J., and Neyts, J. (2012). Inhibition of norovirus replication by the nucleoside analogue 2’-C-methylcytidine. Biochem. Biophys. Res. Commun. 427, 796–800." />
<meta name="citation_reference" content="Rocha-Pereira, J., Jochmans, D., Debing, Y., Verbeken, E., Nascimento, M.S.J., and Neyts, J. (2013). The viral polymerase inhibitor 2’-C-methylcytidine inhibits Norwalk virus replication and protects against norovirus-induced diarrhea and mortality in a mouse model. J Virol 87, 11798–11805." />
<meta name="citation_reference" content="Sanjana, N.E., Shalem, O., and Zhang, F. (2014). Improved vectors and genome-wide libraries for CRISPR screening. Nat. Methods 11, 783–784." />
<meta name="citation_reference" content="Shannon, P., Markiel, A., Ozier, O., Baliga, N.S., Wang, J.T., Ramage, D., Amin, N., Schwikowski, B., and Ideker, T. (2003). Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome Res. 13, 2498–2504." />
<meta name="citation_reference" content="Thackray, L.B., Wobus, C.E., Chachu, K.A., Liu, B., Alegre, E.R., Henderson, K.S., Kelley, S.T., and Virgin, H.W. (2007). Murine noroviruses comprising a single genogroup exhibit biological diversity despite limited sequence divergence. J Virol 81, 10460–10473." />
<meta name="citation_reference" content="Thandapani, P., O’Connor, T.R., Bailey, T.L., and Richard, S. (2013). Defining the RGG/RG motif. Mol. Cell 50, 613–623." />
<meta name="citation_reference" content="Thorne, L.G., and Goodfellow, I.G. (2014). Norovirus gene expression and replication. J. Gen. Virol. 95, 278–291." />
<meta name="citation_reference" content="Thorne, L., Bailey, D., and Goodfellow, I. (2012). High-Resolution functional profiling of the norovirus genome. J Virol 86, 11441–11456." />
<meta name="citation_reference" content="V’kovski, P., Gerber, M., Kelly, J., Pfaender, S., Ebert, N., Braga Lagache, S., Simillion, C., Portmann, J., Stalder, H., Gaschen, V., et al. (2019). Determination of host proteins composing the microenvironment of coronavirus replicase complexes by proximity-labeling. Elife 8, 25." />
<meta name="citation_reference" content="van Beek, J., van der Eijk, A.A., Fraaij, P.L.A., Caliskan, K., Cransberg, K., Dalinghaus, M., Hoek, R.A.S., Metselaar, H.J., Roodnat, J., Vennema, H., et al. (2016). Chronic norovirus infection among solid organ recipients in a tertiary care hospital, the Netherlands, 2006-2014. Clin. Microbiol. Infect." />
<meta name="citation_reference" content="Van, J.D., Ny, A., Conceição-Neto, N., Maes, J., Hosmillo, M., Cuvry, A., Goodfellow, I., de, T.A.N., Verbeken, E., Matthijnssens, J., et al. (2019). A robust human norovirus replication model in zebrafish larvae: Supplementary data file. bioRxiv 528364." />
<meta name="citation_reference" content="Vashist, S., Ureña, L., and Goodfellow, I. (2012a). Development of a strand specific real-time RT-qPCR assay for the detection and quantitation of murine norovirus RNA. J. Virol. Methods." />
<meta name="citation_reference" content="Vashist, S., Ureña, L., Chaudhry, Y., and Goodfellow, I. (2012b). Identification of RNA-protein interaction networks involved in the norovirus life cycle. J Virol 86, 11977–11990." />
<meta name="citation_reference" content="Villa, N., Do, A., Hershey, J.W.B., and Fraser, C.S. (2013). Human Eukaryotic Initiation Factor 4G (eIF4G) Protein Binds to eIF3c, -d, and -e to Promote mRNA Recruitment to the Ribosome. J. Biol. Chem. 288, 32932–32940." />
<meta name="citation_reference" content="Walsh, D., Mathews, M.B., and Mohr, I. (2013). Tinkering with translation: protein synthesis in virus-infected cells. Cold Spring Harb Perspect Biol 5, a012351–a012351." />
<meta name="citation_reference" content="White, J.P., Cardenas, A.M., Marissen, W.E., and Lloyd, R.E. (2007). Inhibition of cytoplasmic mRNA stress granule formation by a viral proteinase. Cell Host Microbe 2, 295–305." />
<meta name="citation_reference" content="Wobus, C.E., Karst, S.M., Thackray, L.B., Chang, K.-O., Sosnovtsev, S.V., Belliot, G., Krug, A., Mackenzie, J.M., Green, K.Y., and Virgin, H.W. (2004). Replication of Norovirus in cell culture reveals a tropism for dendritic cells and macrophages. PLoS Biol. 2, e432." />
<meta name="citation_reference" content="Yunus, M.A., Chung, L.M.W., Chaudhry, Y., Bailey, D., and Goodfellow, I. (2010). Development of an optimized RNA-based murine norovirus reverse genetics system. J. Virol. Methods 169, 112–118." />
<meta name="citation_reference" content="(2006). Differential Cleavage of eIF4GI and eIF4GII in Mammalian Cells. 1–12." />
<meta name="twitter:title" content="Noroviruses subvert the core stress granule component G3BP1 to promote viral VPg-dependent translation" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Knowledge of the host factors required for norovirus replication has been hindered by the challenges associated with culturing human noroviruses. We have combined proteomic analysis of the viral translation and replication complexes with a CRISPR screen, to identify host factors required for norovirus infection. The core stress granule component G3BP1 was identified as a host factor essential for efficient human and murine norovirus infection, demonstrating a conserved function across the Norovirus genus. Furthermore, we show that G3BP1 functions in the novel paradigm of viral VPg-dependent translation initiation, contributing to the assembly of translation complexes on the VPg-linked viral positive sense RNA genome by facilitating 40S recruitment. Our data suggest that G3BP1 functions by providing viral RNA a competitive advantage over capped cellular RNAs, uncovering a novel function for G3BP1 in the life cycle of positive sense RNA viruses and identifying the first host factor with pan-norovirus pro-viral activity." />
<meta name="og-title" property="og:title" content="Noroviruses subvert the core stress granule component G3BP1 to promote viral VPg-dependent translation" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/571455v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Knowledge of the host factors required for norovirus replication has been hindered by the challenges associated with culturing human noroviruses. We have combined proteomic analysis of the viral translation and replication complexes with a CRISPR screen, to identify host factors required for norovirus infection. The core stress granule component G3BP1 was identified as a host factor essential for efficient human and murine norovirus infection, demonstrating a conserved function across the Norovirus genus. Furthermore, we show that G3BP1 functions in the novel paradigm of viral VPg-dependent translation initiation, contributing to the assembly of translation complexes on the VPg-linked viral positive sense RNA genome by facilitating 40S recruitment. Our data suggest that G3BP1 functions by providing viral RNA a competitive advantage over capped cellular RNAs, uncovering a novel function for G3BP1 in the life cycle of positive sense RNA viruses and identifying the first host factor with pan-norovirus pro-viral activity." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-03-08" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/571455v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/571455v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/571455v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/571455v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/225768" />
    <title>Noroviruses subvert the core stress granule component G3BP1 to promote viral VPg-dependent translation | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"UVE6dS26_S0uULgQoz2dmJKJCorknZL-DLXOe2Iq3zc","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"225768","apath":"\/biorxiv\/early\/2019\/03\/08\/571455.atom","pisa":"biorxiv;571455v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;571455v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/571455v1","title":"Noroviruses subvert the core stress granule component G3BP1 to promote viral VPg-dependent translation","identifier":"node\/225768"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022eNnzuQfut3RDkzv4+pRLujU\\\\\\\/t8BrvQK9J28jaM3inZ\\\\\\\/5RAtzpR4fHVlECBlobOR\\\\\\\/CzkX6s9Ds8KRrHBUN0G8ZRcWpblh+mPVc5EqbdSll+amZ6kfraO50eBqDdWEalHnuucqq6l7MfwameBjNE0kk1fcaLpALqpjWkIyClpAjELq9EuegnkqZUWTDTmc1sTBOIs4HfH0f7\\\\\\\/cl10pnMhQFJ5+8zub9jyUjW\\\\\\\/B2MPhwkaUV7TQY97DM0oh1R3DHCb5kQsK9QECL54I3JISnYQFxxIYyXZfOCajhWDpxyTNEq0XDe++kC+WhmYB+b7ZOQcvJpL5JGHN03mgA+4zyEDDzJJFMncIwJx53GszVeOTgYReCdpSwbWernIEOPdH34SDSCriF3JIZdt9x4dTuGxqR5GHGJILNf8Yx2g0DC9iS9pBssso0no915kmkzm+OqS0eL\\\\\\\/+v+2ZV8Q4YRaw0GUxNWtPZ2SeYnoqFhIjpI4ByQy1tGmeVSykm5aMvmF2VOOI1vNVL2c6FD0DUDthFNv2i2lGJHhuYUmDumIjl17Qjgv\\\\\\\/k9N1rENzmQL8y4frPVB7WGD7GmLEkmwFOX+rnF2y9TpbOjU1lxV2qn5sYIrcsx4xoORtpA6PJFiQpuYo1dWAh3Hj0zrwGJRcZGX3BGeg4U9yrWkD0uj7ODo0Eu8Coo27CtSfSAWWZtWR6ORS0LzxPg\\\\\\\/wT7PfXEkYP22AeGs2IHd9xqD+YaQ3hAgRQDikG94+jA5MvbNrGJz+UsVNIe2I68E+ha2Caa5X0vhhvzoky5KRABVRVjKkS52TozGx3SbTm7noSzG2UPSWil7WPJX4WO\\\\\\\/vF7VjsP82xcCF1bPSip7WzZEfzWEjNr3iwCDhiSfjkHKX3TGRRbFX1Kn0XEX7JPbBxgxxWOAIMvU8R7svxqK2VKkYzojxfimuz4R+hMd5hMNCqp1pUMe89KryDPvg2avGkBvbUhGoiYzelHsN1e9cE6fDnZ4XAyk3IZ626pgp2RH0Kme\\\\\\\/\\\\\\\/7McJUozSh9OpjO3TE\\\\\\\/JWcMv7bh6hJLGrPkgjzcGjDxAoZnzLCe1BycOrMutodbqULcv4FXaX2yidGNa3uEQmpkazflKAYAImOwMcwT9yTOfHSwjsKVesMCNIMC0TEHjuSM6uwDDgU16L1d5qd\\\\\\\/RNIZaxGS4nfuufBrre894CatCJR5d0QuBEm\\\\\\\/RKuskYYQWx5Z7Ew4NnjUy9i7c55FE6CMmPPuAg\\\\\\\/JFvY7nrkx4W4nYEFiBujqc1wSHkZQYN3p5\\\\\\\/gS5lmu\\\\\\\/JoxADMBwYqgQpPSbAb7nGzuonuStiauhqqgcycQ7Nw7pCm4=\\\u0022,\\\u0022iv\\\u0022:\\\u0022e8xx+CSEZDH8zcuds\\\\\\\/dNqw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022341a7778b250dd83c73f071c4cc7526dbe171dfc687fc53f59395c0197dee546\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/571455v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-225768 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/571455v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Ppu1jsurX1gQAYEAvaeqHGbHw1GTnsez0AtuOBOOs68" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="225768" id="node225768" data-pisa="biorxiv;571455v1" data-pisa-master="biorxiv;571455" data-apath="/biorxiv/early/2019/03/08/571455.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Noroviruses subvert the core stress granule component G3BP1 to promote viral VPg-dependent translation</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3514-7681" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Myra</span> <span class="nlm-surname">Hosmillo</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Jia</span> <span class="nlm-surname">Lu</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9729-7290" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Michael R.</span> <span class="nlm-surname">McAllaster</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-5725-0743" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">James B.</span> <span class="nlm-surname">Eaglesham</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Xinjie</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-3239-8178" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Edward</span> <span class="nlm-surname">Emmott</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-1158-2926" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Patricia</span> <span class="nlm-surname">Domingues</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Yasmin</span> <span class="nlm-surname">Chaudhry</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Timothy J</span> <span class="nlm-surname">Fitzmaurice</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-7088-226X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Matthew K.H.</span> <span class="nlm-surname">Tung</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0002-7373-0341" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Marc</span> <span class="nlm-surname">Panas</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11"><a href="http://orcid.org/0000-0003-2257-7241" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Gerald</span> <span class="nlm-surname">McInerney</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Nicholas</span> <span class="nlm-surname">Locker</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Craig B.</span> <span class="nlm-surname">Willen</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="14"><a href="http://orcid.org/0000-0002-9483-510X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Ian</span> <span class="nlm-surname">Goodfellow</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/571455 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node225768" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Myra Hosmillo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Division of Virology, Department of Pathology, University of Cambridge</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Myra%2BHosmillo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hosmillo%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMyra%2BHosmillo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3514-7681" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Myra Hosmillo</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jia Lu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Division of Virology, Department of Pathology, University of Cambridge</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jia%2BLu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJia%2BLu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Michael R. McAllaster </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Washington University School of Medicine, Department of Pathology and Immunology</span>, St. Louis, MO 63110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BR.%2BMcAllaster%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McAllaster%20MR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BR.%2BMcAllaster%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9729-7290" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael R. McAllaster</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">James B. Eaglesham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Division of Virology, Department of Pathology, University of Cambridge</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Microbiology, Harvard Medical School</span>, Boston, MA 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BB.%2BEaglesham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eaglesham%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJames%2BB.%2BEaglesham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5725-0743" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for James B. Eaglesham</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Xinjie Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Division of Virology, Department of Pathology, University of Cambridge</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Institute for Brain Research and Rehabilitation, South China Normal University</span>, Guangzhou 510631, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xinjie%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXinjie%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Edward Emmott </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Division of Virology, Department of Pathology, University of Cambridge</span></div><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Department of Bioengineering &#38; Barnett Institute for Chemical and Biological Analyses, Northeastern University</span>, Boston, MA 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edward%2BEmmott%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Emmott%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEdward%2BEmmott%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3239-8178" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Edward Emmott</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Patricia Domingues </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Division of Virology, Department of Pathology, University of Cambridge</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricia%2BDomingues%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Domingues%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatricia%2BDomingues%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1158-2926" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patricia Domingues</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Yasmin Chaudhry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Division of Virology, Department of Pathology, University of Cambridge</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yasmin%2BChaudhry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chaudhry%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYasmin%2BChaudhry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Timothy J Fitzmaurice </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Division of Virology, Department of Pathology, University of Cambridge</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BJ%2BFitzmaurice%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fitzmaurice%20TJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATimothy%2BJ%2BFitzmaurice%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Matthew K.H. Tung </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Division of Virology, Department of Pathology, University of Cambridge</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BK.H.%2BTung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tung%20MK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthew%2BK.H.%2BTung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7088-226X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Matthew K.H. Tung</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Marc Panas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Department of Microbiology, Tumor and Cell Biology, Karolinska Institute</span>, Stockholm, <span class='nlm-country'>Sweden</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marc%2BPanas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Panas%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarc%2BPanas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7373-0341" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marc Panas</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Gerald McInerney </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Department of Microbiology, Tumor and Cell Biology, Karolinska Institute</span>, Stockholm, <span class='nlm-country'>Sweden</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gerald%2BMcInerney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McInerney%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGerald%2BMcInerney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2257-7241" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gerald McInerney</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Nicholas Locker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>School of Biosciences and Medicine, University of Surrey</span>, Guildford <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BLocker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Locker%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicholas%2BLocker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Craig B. Willen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>8</span><span class='nlm-institution'>Yale School of Medicine, Departments of Laboratory Medicine and Immunobiology</span>, New Haven, CT 06520</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Craig%2BB.%2BWillen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Willen%20CB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACraig%2BB.%2BWillen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:craig.wilen@yale.edu" class="" data-icon-position="" data-hide-link-title="0">craig.wilen@yale.edu</a>
<a href="mailto:ig299@cam.ac.uk" class="" data-icon-position="" data-hide-link-title="0">ig299@cam.ac.uk</a></span></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Ian Goodfellow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Division of Virology, Department of Pathology, University of Cambridge</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ian%2BGoodfellow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goodfellow%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIan%2BGoodfellow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9483-510X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ian Goodfellow</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:craig.wilen@yale.edu" class="" data-icon-position="" data-hide-link-title="0">craig.wilen@yale.edu</a>
<a href="mailto:ig299@cam.ac.uk" class="" data-icon-position="" data-hide-link-title="0">ig299@cam.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/571455v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:225768" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:225768/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/571455v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:225768" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:225768/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/571455v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:225768" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:225768/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/571455v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:225768" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:225768/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/571455v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:225768" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:225768/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/571455v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:225768" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:225768/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">Knowledge of the host factors required for norovirus replication has been hindered by the challenges associated with culturing human noroviruses. We have combined proteomic analysis of the viral translation and replication complexes with a CRISPR screen, to identify host factors required for norovirus infection. The core stress granule component G3BP1 was identified as a host factor essential for efficient human and murine norovirus infection, demonstrating a conserved function across the <em>Norovirus</em> genus. Furthermore, we show that G3BP1 functions in the novel paradigm of viral VPg-dependent translation initiation, contributing to the assembly of translation complexes on the VPg-linked viral positive sense RNA genome by facilitating 40S recruitment. Our data suggest that G3BP1 functions by providing viral RNA a competitive advantage over capped cellular RNAs, uncovering a novel function for G3BP1 in the life cycle of positive sense RNA viruses and identifying the first host factor with pan-norovirus pro-viral activity.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-4">Positive sense RNA viruses rely heavily on host cell factors for all aspects of their life cycle. They replicate on host derived membranous vesicles that are induced following viral infection, the formation of which requires the activity of key membrane bound viral enzymes (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Altan-Bonnet, 2017</a>). Within the membrane bound viral replication complex, translation of the viral genome and the synthesis of new viral RNA occurs in a highly coordinated process. Positive sense RNA viruses have evolved novel gene expression mechanisms that enable them to overcome the genome size limitations that accompany error-prone replication and which might restrict their overall coding capacity (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Firth and Brierley, 2012</a>). In addition, viral modification of the host cell translation machinery often provides a competitive advantage allowing for the efficient translation of viral RNA in an environment where competing cellular RNAs are in abundance (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">McCormick and Khaperskyy, 2017</a>). This ability to compete with cellular RNAs is particularly important for the initiation of infection where the incoming viral genome may be present at only a single copy per cell.</p><p id="p-5">We have previously described a novel paradigm of viral translation that relies on the interaction of host translation initiation factors with a virus-encoded protein (VPg), covalently linked to the 5’ end of the genome of members of the <em>Caliciviridae</em> family of positive sense RNA viruses (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chaudhry et al., 2006</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chung et al., 2014</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Goodfellow et al., 2005</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hosmillo et al., 2014</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Leen et al., 2016</a>). Unlike the 22-amino acid VPg peptides from picornaviruses, the VPg protein linked to the genomes of caliciviruses is significantly larger and is essential for the translation of viral RNA and viral RNA infectivity (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Goodfellow, 2011</a>).</p><p id="p-6">Human noroviruses (HuNoV) and sapoviruses (HuSaV) are enteropathogenic members of the <em>Caliciviridae</em> family of positive sense RNA viruses, and together cause &gt;20% of all cases of gastroenteritis (GE). They are also a significant cause of morbidity and mortality in the immunocompromised; individuals with genetic immune-deficiencies, cancer patients undergoing treatment and transplant recipients often experience chronic norovirus infections lasting months to years (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">van Beek et al., 2016</a>). The economic impact of HuNoV is estimated to be at least ~$4.2 billion in direct health care costs, with wider societal costs of ~$60 billion (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bartsch et al., 2016</a>). Despite their socioeconomic impact, we have, until very recently lacked a detailed understanding of much of the norovirus life cycle and many significant questions remain unanswered. HuNoV replicons (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chang et al., 2006</a>), a murine norovirus that replicates in cell culture (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Karst et al., 2003</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Wobus et al., 2004</a>) and the recent B cell (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Jones et al., 2014</a>), stem-cell derived organoid (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Ettayebi et al., 2016</a>) and zebrafish larvae infection models (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Van et al., 2019</a>), have all provided invaluable tools to dissect the norovirus life cycle. However, due to the technical limitations associated with many of these experimental systems, in comparison to other positive sense RNA viruses, our knowledge of the intracellular life of noroviruses is significantly lacking (reviewed in <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Thorne and Goodfellow, 2014</a>).</p><p id="p-7">In the current study, we have combined three independent unbiased approaches to identify host factors involved in the norovirus life cycle. Combining experimental systems that incorporated both murine and human noroviruses, allowed the identification of cellular factors for which the function is likely conserved across the <em>Norovirus</em> genus. By combining three complimentary approaches, we identify the host protein G3BP1 as a critical host factor required for norovirus VPg-dependent translation, identifying a new role for G3BP1 in virus-specific translation.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3>Comparative analysis of the norovirus translation initiation complex</h3><p id="p-8">The MNV and the prototype HuNoV Norwalk virus (NV) VPg proteins contain a highly conserved C-terminal domain (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig 1A</a>) which we have previously shown to be necessary and sufficient for binding to the translation initiation factor eIF4G (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Chung et al., 2014</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Leen et al., 2016</a>). Using affinity purification on m7-GTP sepharose, we confirmed that the NV VPg protein, as produced during authentic virus replication in a NV replicon bearing cell line, interacts with the cap-binding complex eIF4F (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig 1B</a>). Components of the eIF4F complex, namely the eIF4E cap-binding protein, the eIF4A helicase and the eIF4GI scaffold protein, along with poly-A binding protein (PABP) and eIF3 subunits, were readily purified on m7-GTP sepharose, whereas GAPDH was not. In NV-replicon containing cells, mature VPg was also enriched on m7-GTP sepharose but the NS3 protein, known to have RNA binding and helicase activity (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Li et al., 2018</a>), was not. Furthermore, we demonstrated that transfection of GFP-tagged versions of either the MNV or NV VPg proteins into 293T cells allowed for the affinity purification of eIF4F components and that mutations in the eIF4G binding domain of VPg reduced this association (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig 1C</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The norovirus VPg proteins interacts with ribosome associated translation initiation factors. A) Amino acid sequence alignment of the GV murine norovirus VPg sequences with VPg from representative human noroviruses from GI Norwalk virus (NV), GII, and GIV. The position of the site of RNA linkage to the highly conserved tyrosine residue is highlighted in green. The eIF4G binding motif is boxed and the position of the C-terminal single amino acid change known to interfere with eIF4G binding highlighted in orange. B) m7-GTP sepharose was used to affinity purify eIF4F containing complexes from either wild-type BHK cells (BHK) or BHK cells containing the Norwalk virus (NV) replicon (BHK-NV). Samples of the lysate (L) or the affinity purified complexes (m7) were separated by SDS-PAGE then analysed by western blot for the indicate proteins. Molecular mass shown on the left of the gels represent the positions of molecular weight markers. C) GFP fusion proteins to either the wild type (WT) or C-terminal eIF4G binding domain mutants of the MNV and NV VPg proteins (F123A, F137A) were transfected into human 293T cells and subjected to immunoaffinity purification using anti-GFP. Samples of the input lysates (Input) and the purified complexes (GFP-IP) were then separated by SDS-PAGE and analysed by western blot analysis for the indicated proteins. Mock transfected cells served as a specificity control. The approximate expected molecular mass of each protein is shown to the left. D) Quantitative proteomics was used as described in the text to identify host factors that were affinity purified following transfection of GFP-tagged derivative of either the NV or MNV VPg proteins. Proteins specifically enriched in comparison to the GFP control are shown. Data visualisation was performed using Cystoscape (Shannon et al., 2003). E) Western blot analysis of cell extracts (Input) or immunoprecipitated (GFP IP) complexes isolated from cells transfected as described in panel C. For clarity, the molecular masses shown in this panel refer to the expected mass of the protein being examined." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1550667250" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;The norovirus VPg proteins interacts with ribosome associated translation initiation factors.&lt;/span&gt; A) Amino acid sequence alignment of the GV murine norovirus VPg sequences with VPg from representative human noroviruses from GI Norwalk virus (NV), GII, and GIV. The position of the site of RNA linkage to the highly conserved tyrosine residue is highlighted in green. The eIF4G binding motif is boxed and the position of the C-terminal single amino acid change known to interfere with eIF4G binding highlighted in orange. B) m7-GTP sepharose was used to affinity purify eIF4F containing complexes from either wild-type BHK cells (BHK) or BHK cells containing the Norwalk virus (NV) replicon (BHK-NV). Samples of the lysate (L) or the affinity purified complexes (m7) were separated by SDS-PAGE then analysed by western blot for the indicate proteins. Molecular mass shown on the left of the gels represent the positions of molecular weight markers. C) GFP fusion proteins to either the wild type (WT) or C-terminal eIF4G binding domain mutants of the MNV and NV VPg proteins (F123A, F137A) were transfected into human 293T cells and subjected to immunoaffinity purification using anti-GFP. Samples of the input lysates (Input) and the purified complexes (GFP-IP) were then separated by SDS-PAGE and analysed by western blot analysis for the indicated proteins. Mock transfected cells served as a specificity control. The approximate expected molecular mass of each protein is shown to the left. D) Quantitative proteomics was used as described in the text to identify host factors that were affinity purified following transfection of GFP-tagged derivative of either the NV or MNV VPg proteins. Proteins specifically enriched in comparison to the GFP control are shown. Data visualisation was performed using Cystoscape (Shannon et al., 2003). E) Western blot analysis of cell extracts (Input) or immunoprecipitated (GFP IP) complexes isolated from cells transfected as described in panel C. For clarity, the molecular masses shown in this panel refer to the expected mass of the protein being examined.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F1.medium.gif" width="425" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F1.medium.gif" width="425" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">The norovirus VPg proteins interacts with ribosome associated translation initiation factors.</span><p id="p-9" class="first-child">A) Amino acid sequence alignment of the GV murine norovirus VPg sequences with VPg from representative human noroviruses from GI Norwalk virus (NV), GII, and GIV. The position of the site of RNA linkage to the highly conserved tyrosine residue is highlighted in green. The eIF4G binding motif is boxed and the position of the C-terminal single amino acid change known to interfere with eIF4G binding highlighted in orange. B) m7-GTP sepharose was used to affinity purify eIF4F containing complexes from either wild-type BHK cells (BHK) or BHK cells containing the Norwalk virus (NV) replicon (BHK-NV). Samples of the lysate (L) or the affinity purified complexes (m7) were separated by SDS-PAGE then analysed by western blot for the indicate proteins. Molecular mass shown on the left of the gels represent the positions of molecular weight markers. C) GFP fusion proteins to either the wild type (WT) or C-terminal eIF4G binding domain mutants of the MNV and NV VPg proteins (F123A, F137A) were transfected into human 293T cells and subjected to immunoaffinity purification using anti-GFP. Samples of the input lysates (Input) and the purified complexes (GFP-IP) were then separated by SDS-PAGE and analysed by western blot analysis for the indicated proteins. Mock transfected cells served as a specificity control. The approximate expected molecular mass of each protein is shown to the left. D) Quantitative proteomics was used as described in the text to identify host factors that were affinity purified following transfection of GFP-tagged derivative of either the NV or MNV VPg proteins. Proteins specifically enriched in comparison to the GFP control are shown. Data visualisation was performed using Cystoscape (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Shannon et al., 2003</a>). E) Western blot analysis of cell extracts (Input) or immunoprecipitated (GFP IP) complexes isolated from cells transfected as described in panel C. For clarity, the molecular masses shown in this panel refer to the expected mass of the protein being examined.</p><div class="sb-div caption-clear"></div></div></div><p id="p-10">We next used quantitative mass spectrometry of the affinity purified complexes isolated from cells transfected with the GFP-Tagged VPg proteins to identify host factors specifically enriched on the norovirus VPg protein (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig 1D</a>, <a id="xref-fig-12-1" class="xref-fig" href="#F12">Fig S1</a> and Table S1). Most of the proteins identified were components of the host cell translation complex including ribosomal proteins, translation initiation factors and host RNA binding proteins. These data agrees with but significantly extend our previous observations using a less sensitive multi-step affinity purification approach to characterise host factors associated with the MNV VPg protein only (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Chaudhry et al., 2006</a>; <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Chung et al., 2014</a>). In addition, we identified hnRNPA1 which we have previously shown to act in norovirus genome circularization (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">López-Manríquez et al., 2013</a>). YBX1, DDX3 and several other proteins that we have previously found to interact with the 5’ end of the viral RNA (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Vashist et al., 2012b</a>) were also enriched on VPg (<a id="xref-fig-12-2" class="xref-fig" href="#F12">Fig S1</a>). To validate a select number of these interactions and to assess whether the interaction of VPg with eIF4G is required for their association with VPg, we performed western blot analysis of complexes purified from cells transfected with either the WT or eIF4G-binding mutants (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig 1E</a>). Except for YBX1, the association of all proteins tested were reduced by the introduction of eIF4G-binding site mutations into the MNV VPg protein. Together, these data extend our previous observations and confirm that the norovirus VPg proteins interact with a complex network of host factors, many of which have been implicated in the host cell translation initiation process.</p></div><div id="sec-4" class="subsection"><h3>Determination of the norovirus replication complex proteome</h3><p id="p-11">To further identify the components of the norovirus translation and replication complex, as formed during authentic viral replication in highly permissive cells, we utilised two recombinant infectious MNV strains that carried epitope purification tags within the NS1/2 or NS4 proteins (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">McCune et al., 2017</a>) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig 2A</a>). The insertion positions were previously identified using a transposon based mutagenesis screen as sites that tolerate insertions, without compromising virus viability (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Thorne et al., 2012</a>). Our approach was somewhat analogous to that recently published for coronaviruses (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">V’kovski et al., 2019</a>) but instead used stable isotope labelling of permissive cells and the FLAG affinity purification tag rather than proximity labelling. Unlabelled or stable isotope labelled highly permissive BV-2 microglial cells were infected with either wild type MNV or the equivalent virus carrying the FLAG epitope purification tag in either NS1/2 or NS4, and the viral replication complex was purified. The experiment was performed three times by swapping the labelled derivatives of arginine and lysine as described in the materials and methods. Silver stain of the purified complexes confirmed the presence of the bait proteins, with both the uncleaved and cleaved forms of NS1/2 and NS2 being highly enriched (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig 2B</a>). As expected, complexes purified from NS1/2-Flag virus infected cells co-purified untagged NS4 and vice versa (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig 2B</a>), as we have previously shown these proteins to interact to form a complex (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Thorne et al., 2012</a>). Western blot analysis of the purified complexes confirmed that viral non-structural and structural proteins were specifically enriched in the purified complexes, including NS5 (VPg)-containing precursors (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig 2C</a>). We noted that anti-NS4 monoclonal antibody was unable to detect protein in the extracts prior to enrichment, which most likely reflected the limited sensitivity of the antibody. Quantitative mass spectrometry of the purified complexes allowed the identified of viral and cellular proteins enriched in the complex (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig 2D</a> and Table S2).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proteomic characterisation of the norovirus replication complex using infectious epitope tagged MNV. A) Schematic representation of NS1/2-FLAG and NS4-FLAG viruses contain insertions of nucleotide sequences encoding the FLAG peptide DYKDDDDK (in yellow) in their coding sequences. The NS1/2-FLAG virus FLAG peptide was inserted between 2 of the 3 caspase-3 cleavage sites present in NS1/2 (underlined). B) BV-2 cells labelled with stable derivatives of arginine and lysine were infected with either wild type MNV (WT) or recombinant epitope-tagged MNV as described in the materials and methods. 12 hours post infection samples were lysed, samples pooled and immunoaffinity purifications performed as described in the text. Samples of the cell lysates (Input) and the affinity purified complexes (IP:Flag) were analysed by SDS-PAGE on a 4-12% gradient gel prior to silver staining. The positions of the NS1/2, NS2 and NS4 proteins is shown. C) Western blot analysis of lysates purified from cells infected as described in panel B, for various viral proteins, confirming the specific enrichment of viral replicase components. Plot showing detailing overlapping proteins identified in pulldowns from cells infected with FLAG-tagged NS1/2 or NS4 expressing viruses. All MNV proteins were identified in association with NS1/2 and NS4 (light blue) including the viral polymerase NS7, demonstrating enrichment of the MNV replication complex. Proteins previously identified as host factors potentially involved in some aspect of the norovirus life cycle through various biochemical or genetic screens are shown in red. Selected highly enriched proteins are highlighted in black. The NS1/2 binding partner VapA (McCune et al., 2017) and paralog VapB were both enriched by NS1/2 and NS4." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1550667250" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Proteomic characterisation of the norovirus replication complex using infectious epitope tagged MNV.&lt;/span&gt; A) Schematic representation of NS1/2-FLAG and NS4-FLAG viruses contain insertions of nucleotide sequences encoding the FLAG peptide DYKDDDDK (in yellow) in their coding sequences. The NS1/2-FLAG virus FLAG peptide was inserted between 2 of the 3 caspase-3 cleavage sites present in NS1/2 (underlined). B) BV-2 cells labelled with stable derivatives of arginine and lysine were infected with either wild type MNV (WT) or recombinant epitope-tagged MNV as described in the materials and methods. 12 hours post infection samples were lysed, samples pooled and immunoaffinity purifications performed as described in the text. Samples of the cell lysates (Input) and the affinity purified complexes (IP:Flag) were analysed by SDS-PAGE on a 4-12% gradient gel prior to silver staining. The positions of the NS1/2, NS2 and NS4 proteins is shown. C) Western blot analysis of lysates purified from cells infected as described in panel B, for various viral proteins, confirming the specific enrichment of viral replicase components. Plot showing detailing overlapping proteins identified in pulldowns from cells infected with FLAG-tagged NS1/2 or NS4 expressing viruses. All MNV proteins were identified in association with NS1/2 and NS4 (light blue) including the viral polymerase NS7, demonstrating enrichment of the MNV replication complex. Proteins previously identified as host factors potentially involved in some aspect of the norovirus life cycle through various biochemical or genetic screens are shown in red. Selected highly enriched proteins are highlighted in black. The NS1/2 binding partner VapA (McCune et al., 2017) and paralog VapB were both enriched by NS1/2 and NS4.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F2.medium.gif" width="440" height="426"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F2.medium.gif" width="440" height="426"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <span class="caption-title">Proteomic characterisation of the norovirus replication complex using infectious epitope tagged MNV.</span><p id="p-12" class="first-child">A) Schematic representation of NS1/2-FLAG and NS4-FLAG viruses contain insertions of nucleotide sequences encoding the FLAG peptide DYKDDDDK (in yellow) in their coding sequences. The NS1/2-FLAG virus FLAG peptide was inserted between 2 of the 3 caspase-3 cleavage sites present in NS1/2 (underlined). B) BV-2 cells labelled with stable derivatives of arginine and lysine were infected with either wild type MNV (WT) or recombinant epitope-tagged MNV as described in the materials and methods. 12 hours post infection samples were lysed, samples pooled and immunoaffinity purifications performed as described in the text. Samples of the cell lysates (Input) and the affinity purified complexes (IP:Flag) were analysed by SDS-PAGE on a 4-12% gradient gel prior to silver staining. The positions of the NS1/2, NS2 and NS4 proteins is shown. C) Western blot analysis of lysates purified from cells infected as described in panel B, for various viral proteins, confirming the specific enrichment of viral replicase components. Plot showing detailing overlapping proteins identified in pulldowns from cells infected with FLAG-tagged NS1/2 or NS4 expressing viruses. All MNV proteins were identified in association with NS1/2 and NS4 (light blue) including the viral polymerase NS7, demonstrating enrichment of the MNV replication complex. Proteins previously identified as host factors potentially involved in some aspect of the norovirus life cycle through various biochemical or genetic screens are shown in red. Selected highly enriched proteins are highlighted in black. The NS1/2 binding partner VapA (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">McCune et al., 2017</a>) and paralog VapB were both enriched by NS1/2 and NS4.</p><div class="sb-div caption-clear"></div></div></div><p id="p-13">As expected, all viral proteins, including the VF1 protein product of ORF4, an innate immune antagonist (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bailey et al., 2011</a>), were enriched in the viral replication complex. There was a significant correlation between the relative enrichment of proteins identified using NS1/2 and NS4 (Spearman correlation of 0.8832), fitting with our prior knowledge that both proteins form a complex during viral replication (<a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Thorne et al., 2012</a>). Ontology analysis indicated that proteins involved in vesicle transport and fatty acid metabolism were significantly enriched (<a id="xref-fig-13-1" class="xref-fig" href="#F13">Fig S2</a> and Table S3), fitting with previous observations that the viral replication complex is associated with cytoplasmic membranous structures (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cotton et al., 2017</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hyde and Mackenzie, 2010</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hyde et al., 2009</a>). Several host proteins previously identified in a variety of biochemical and genetic screens were enriched (<a id="xref-fig-13-2" class="xref-fig" href="#F13">Fig S2</a> and Table S3) providing additional confidence that the approach identified biologically relevant interactions. We noted that the VapA and the paralogue VapB, which we have recently identified as binding to the NS1/2 protein (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">McCune et al., 2017</a>), were both highly enriched. Comparison with the data obtained using VPg as a bait protein (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig 1</a>) showed some degree of overlap, however it is worth noting that most of the factors that were identified using VPg were enriched by &gt;2 fold using only the NS1/2 tagged virus and not the NS4 tagged virus (<a id="xref-fig-13-3" class="xref-fig" href="#F13">Fig S2</a>). One of the exceptions to this was the core stress granule protein G3BP1, which was enriched by both MNV and NV VPg proteins, and was also enriched in complexes purified using both NS1/2 and NS4-tagged viruses.</p></div><div id="sec-5" class="subsection"><h3>Identification of host factors required for norovirus infection using a CRISPR-knockout screen</h3><p id="p-14">A high density CRISPR library screen was undertaken to identify genes that contribute to the norovirus life cycle. The Brie library (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Doench et al., 2016</a>) was selected due to the reduced off-target effects relative to previously described CRISPR libraries used for norovirus studies (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Haga et al., 2016</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Orchard et al., 2016</a>). In addition, to minimise the impact of gRNAs that may have deleterious effects on long term cell viability and to increase our ability to detect genes that may be important, but not essential, for norovirus-induced cell death, the infection was reduced to 24 hours as compared to 2-10 days post infection in previous studies. BV-2-Cas9 expressing cells were infected with lentiviruses carrying the Brie gRNA library carrying 78,637 independent guide RNAs to 19,674 genes (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Doench et al., 2016</a>). The transduced cells were then infected with two MNV strains, CW3 and CR6, which cause acute and persistent infections in immunocompetent mice respectively (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Nice et al., 2012</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Thackray et al., 2007</a>), and guide RNA abundance compared to mock infected cells at 24 hours post infection as illustrated in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig 3A</a>. Genes that were enriched by STARS analysis following MNV infection represent putative pro-viral factors which when disrupted result in slower cell death, whereas those with a negative STARS value represent putative anti-viral factors where virus-induced cell death has occurred quicker, resulting in their underrepresentation in the final pool of cells. MNV-CR6 infection resulted in 212 genes being enriched and 43 being negatively selected (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig 3B</a>), whereas for MNV-CW3 279 and 19 genes were positively and negatively selected respectively (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig 3B</a>). In most cases, there was a clear correlation between the datasets obtained using either strain (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig 3C</a>). STARS analysis was used to ranks genes with positive and negative values (Table S4). In both screens, the MNV receptor Cd300lf was the most highly positively selected gene identified, in agreement with previous reports (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Haga et al., 2016</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Orchard et al., 2016</a>). The second most highly enriched gene was G3BP1, a gene also identified in one of the two previous CRISPR screens performed on norovirus infected cells (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Orchard et al., 2016</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="CRISPR screen identifies host genes positively and negatively selected upon MNV infection. A) Schematic overview of the infection CRISPR screen workflow. B) Volcano plot identifying candidate genes enriched upon MNV-CW3 (red) or MNV-CR6 (blue); red or blue labelled genes correspond to the top-ten positive or negatively selected genes ranked by the STARS algorithm." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1550667250" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;CRISPR screen identifies host genes positively and negatively selected upon MNV infection.&lt;/span&gt; A) Schematic overview of the infection CRISPR screen workflow. B) Volcano plot identifying candidate genes enriched upon MNV-CW3 (red) or MNV-CR6 (blue); red or blue labelled genes correspond to the top-ten positive or negatively selected genes ranked by the STARS algorithm.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F3.medium.gif" width="316" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F3.medium.gif" width="316" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <span class="caption-title">CRISPR screen identifies host genes positively and negatively selected upon MNV infection.</span><p id="p-15" class="first-child">A) Schematic overview of the infection CRISPR screen workflow. B) Volcano plot identifying candidate genes enriched upon MNV-CW3 (red) or MNV-CR6 (blue); red or blue labelled genes correspond to the top-ten positive or negatively selected genes ranked by the STARS algorithm.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">A comparison of the data obtained from all three approaches allowed us to identify several host proteins that were common to all screens (Table S5). G3BP1, the core stress granule component was identified in all three screens as a potential host factor essential for norovirus infection. G3BP1 was found to be associated with the MNV and NV VPg proteins (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Fig 1D</a>), enriched in viral replication complexes purified using either NS1/2 or NS4 flag tagged viruses (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig 2D</a>) and identified in a CRISPR screen using two different MNV strains as a putative pro-viral factor involved in the norovirus life cycle (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig 3C</a>).</p></div><div id="sec-6" class="subsection"><h3>G3BP1 is essential for murine norovirus replication</h3><p id="p-17">To validate the importance of G3BP1 in the norovirus life cycle we generated G3BP1 deficient BV-2 cell lines (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig 4A</a>) and examined the impact of G3BP1 ablation on MNV infection. Western blotting confirmed the loss of G3BP1 in the three lines tested and we noted that at in some cases, a concomitant increase in G3BP2 expression was observed as has been previously noted (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kedersha et al., 2016</a>). A clear defect was observed in the ability to replicate to produce infectious virus in three independently selected ΔG3BP1 cell lines (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig 4B</a>). This effect was mirrored by an inability to induce cytopathic effect leading to virus-induced cell death (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig 4C</a>). In contrast, the ability of encephalomyocarditis virus (EMCV) to infect and cause cell death was unaffected by the deletion of G3BP1(<a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig 4C</a>). These data confirm that cells lacking G3BP1 are highly resistant to norovirus infection.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="CRISPR knockout of G3BP1 renders cells non permissive for MNV replication. A) Western blot analysis of three independent &#x394;G3BP1 clones for GAPDH, G3BP1 and G3BP2. B) High multiplicity, single cycle growth curve analysis of the impact of G3BP1 ablation on MNV replication. BV-2 &#x394;G3BP1 clone C cells were infected at a MOI of 10 TCID50/cell, samples were collected at the time points illustrated, the samples then processed and titrated by TCID50 as described in the text. The error bars represent standard errors of three biological repeats and the data are representative of at least three independent experiments. C) Wild type (WT) or &#x394;G3BP1 clone C BV-2 cells were plated in a 96 well plate and subsequently infected using a serial dilution of either EMCV or MNV. Cells were fixed in paraformaldehyde and stained with crystal violet 5 days post infection. D) Light micrographs of WT or &#x394;G3BP1 cells either mock infected (-) or infected with EMCV or MNV and visualised 5 days post infection." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1550667250" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;CRISPR knockout of G3BP1 renders cells non permissive for MNV replication.&lt;/span&gt; A) Western blot analysis of three independent &#x394;G3BP1 clones for GAPDH, G3BP1 and G3BP2. B) High multiplicity, single cycle growth curve analysis of the impact of G3BP1 ablation on MNV replication. BV-2 &#x394;G3BP1 clone C cells were infected at a MOI of 10 TCID50/cell, samples were collected at the time points illustrated, the samples then processed and titrated by TCID50 as described in the text. The error bars represent standard errors of three biological repeats and the data are representative of at least three independent experiments. C) Wild type (WT) or &#x394;G3BP1 clone C BV-2 cells were plated in a 96 well plate and subsequently infected using a serial dilution of either EMCV or MNV. Cells were fixed in paraformaldehyde and stained with crystal violet 5 days post infection. D) Light micrographs of WT or &#x394;G3BP1 cells either mock infected (-) or infected with EMCV or MNV and visualised 5 days post infection.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F4.medium.gif" width="440" height="232"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F4.medium.gif" width="440" height="232"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4:</span> <span class="caption-title">CRISPR knockout of G3BP1 renders cells non permissive for MNV replication.</span><p id="p-18" class="first-child">A) Western blot analysis of three independent ΔG3BP1 clones for GAPDH, G3BP1 and G3BP2. B) High multiplicity, single cycle growth curve analysis of the impact of G3BP1 ablation on MNV replication. BV-2 ΔG3BP1 clone C cells were infected at a MOI of 10 TCID50/cell, samples were collected at the time points illustrated, the samples then processed and titrated by TCID50 as described in the text. The error bars represent standard errors of three biological repeats and the data are representative of at least three independent experiments. C) Wild type (WT) or ΔG3BP1 clone C BV-2 cells were plated in a 96 well plate and subsequently infected using a serial dilution of either EMCV or MNV. Cells were fixed in paraformaldehyde and stained with crystal violet 5 days post infection. D) Light micrographs of WT or ΔG3BP1 cells either mock infected (-) or infected with EMCV or MNV and visualised 5 days post infection.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>G3BP1 is essential for human norovirus replication in cell culture</h3><p id="p-19">To determine if the G3BP1 was also essential for HuNoV, we examined the impact of loss of G3BP1 on human norovirus replication in cell culture using the Norwalk virus replicon. To establish the experimental system, we first confirmed that the presence of VPg on the 5’ end of the Norwalk RNA was essential for the replication of the replicon RNA and for the capacity to form G418 resistant colonies. Transfection of replicon RNA, purified from replicon containing cells, into BHK cells readily resulted in the formation of antibiotic resistant cell colonies (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig 5A</a>). In contrast, RNA that was proteinase K treated prior to transfection was unable to produce replicon containing colonies. Transfection of replicon RNA into wild type U2OS osteosarcoma cells allowed the formation of replicon-containing colonies, although the efficiency of formation was significantly less than that seen in BHK cells (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig 5B</a>). CRISPR modified U2OS cells that lacked G3BP1 (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Kedersha et al., 2016</a>) were unable to support NV replication, as evident by the lack of antibiotic resistant colonies (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig 5B</a>). To further examine the role of G3BP1 in human Norwalk virus replication, WT or G3BP1 deficient U2OS cells were transfected with NV replicon VPg-linked RNA, and RNA synthesis monitored overtime following the addition of G418. While a significant increase in NV viral RNA levels was seen in WT U2OS cells, those lacking G3BP1 were completely unable to support NV RNA synthesis (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Fig 5C</a>). These data indicate that like for MNV, G3BP1 is essential for human Norwalk virus replication.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="G3BP1 is required for human Norwalk virus replication in cell culture. A) Colony formation ability of human norovirus VPg-linked RNA isolated from BHK-NV replicon containing cells is dependent on the presence of VPg. NV VPg-linked RNA isolated from BHK-NV cells was either mock treated or treated with proteinase K prior to transfection into BHK cells. Wells were transfected with either 1.5&#x3BC;g or 0.75&#x3BC;g of total RNA isolated from NV replicon containing BHK cells. Following 2 weeks of antibiotic selection with G418, surviving replicon containing colonies were fixed and stained with crystal violet in paraformaldehyde. B) NV replicon colony forming assay in WT and G3BP1-/-U2OS cells performed as described in panel A, with the exception that colonies were stained 12 days post transfection. C) Quantification of NV replication in WT or &#x394;G3BP1 U2OS cells following transfection of viral VPg-linked RNA. Viral RNA was quantified by RT-qPCR following transfection and antibiotic selection. The error bars represent the standard error of three biological repeats and are representative of three independent experiments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1550667250" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;G3BP1 is required for human Norwalk virus replication in cell culture.&lt;/span&gt; A) Colony formation ability of human norovirus VPg-linked RNA isolated from BHK-NV replicon containing cells is dependent on the presence of VPg. NV VPg-linked RNA isolated from BHK-NV cells was either mock treated or treated with proteinase K prior to transfection into BHK cells. Wells were transfected with either 1.5&#x3BC;g or 0.75&#x3BC;g of total RNA isolated from NV replicon containing BHK cells. Following 2 weeks of antibiotic selection with G418, surviving replicon containing colonies were fixed and stained with crystal violet in paraformaldehyde. B) NV replicon colony forming assay in WT and G3BP1-/-U2OS cells performed as described in panel A, with the exception that colonies were stained 12 days post transfection. C) Quantification of NV replication in WT or &#x394;G3BP1 U2OS cells following transfection of viral VPg-linked RNA. Viral RNA was quantified by RT-qPCR following transfection and antibiotic selection. The error bars represent the standard error of three biological repeats and are representative of three independent experiments.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F5.medium.gif" width="297" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F5.medium.gif" width="297" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <span class="caption-title">G3BP1 is required for human Norwalk virus replication in cell culture.</span><p id="p-20" class="first-child">A) Colony formation ability of human norovirus VPg-linked RNA isolated from BHK-NV replicon containing cells is dependent on the presence of VPg. NV VPg-linked RNA isolated from BHK-NV cells was either mock treated or treated with proteinase K prior to transfection into BHK cells. Wells were transfected with either 1.5μg or 0.75μg of total RNA isolated from NV replicon containing BHK cells. Following 2 weeks of antibiotic selection with G418, surviving replicon containing colonies were fixed and stained with crystal violet in paraformaldehyde. B) NV replicon colony forming assay in WT and G3BP1-/-U2OS cells performed as described in panel A, with the exception that colonies were stained 12 days post transfection. C) Quantification of NV replication in WT or ΔG3BP1 U2OS cells following transfection of viral VPg-linked RNA. Viral RNA was quantified by RT-qPCR following transfection and antibiotic selection. The error bars represent the standard error of three biological repeats and are representative of three independent experiments.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>The RNA-binding domain of G3BP1 is required for its function in the norovirus life cycle</h3><p id="p-21">To confirm the role of G3BP1 in the norovirus life cycle we examined the ability of full length and truncated versions of G3BP1 to restore norovirus replication in G3BP1 knockout cells. A mouse BV-2 G3BP1 knockout cell line was complemented with either full length G3BP1 or variants lacking the RGG or both the RGG and RRM binding domains (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig 6A</a>) and the impact on viral replication assessed. Complementation with full length murine G3BP1 restored the ability of MNV to induce cell death (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig 6B</a>) and to produce infectious virus (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Fig 6C</a>) back to near wild type levels. In contrast, complementation with a variant carrying a deletion of the RGG domain resulted in limited complementation, and deletion of both the RGG and RRM domains together resulted in complete loss of complementation capacity (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Fig 6B</a>&amp;<a id="xref-fig-6-5" class="xref-fig" href="#F6">C</a>). These data confirm that the RNA binding domains of G3BP1 are essential for its function in the norovirus life cycle.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="MNV replication in BV-2 cells requires the RNA binding activity of G3BP1. Western blot analysis of wild type BV-2 cells (WT) or a &#x394;G3BP1 cells (clone 1B2) and the respective complemented lines expressing either WT or the various G3BP1 truncations. Cells were lysed prior to separation by 12% SDS-PAGE. B) WT or &#x394;G3BP1 cells complemented with the indicated constructs were plated in a 96 well plate then infected with a serial dilution of MNV, before being fixed and stained 5 days post infection as described in the text. C) WT or &#x394;G3BP1 cells complemented with the indicated constructs were infected with MNV at an MOI of 10 TCID50 per cell. After 24 hours the virus yield was determined by TCID50. The error bars represent the standard error of three independent repeats. The data are representative of at least two independently repeated experiments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1550667250" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;MNV replication in BV-2 cells requires the RNA binding activity of G3BP1.&lt;/span&gt; Western blot analysis of wild type BV-2 cells (WT) or a &#x394;G3BP1 cells (clone 1B2) and the respective complemented lines expressing either WT or the various G3BP1 truncations. Cells were lysed prior to separation by 12% SDS-PAGE. B) WT or &#x394;G3BP1 cells complemented with the indicated constructs were plated in a 96 well plate then infected with a serial dilution of MNV, before being fixed and stained 5 days post infection as described in the text. C) WT or &#x394;G3BP1 cells complemented with the indicated constructs were infected with MNV at an MOI of 10 TCID50 per cell. After 24 hours the virus yield was determined by TCID50. The error bars represent the standard error of three independent repeats. The data are representative of at least two independently repeated experiments.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F6.medium.gif" width="440" height="310"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6:" src="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F6.medium.gif" width="440" height="310"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6:</span> <span class="caption-title">MNV replication in BV-2 cells requires the RNA binding activity of G3BP1.</span><p id="p-22" class="first-child">Western blot analysis of wild type BV-2 cells (WT) or a ΔG3BP1 cells (clone 1B2) and the respective complemented lines expressing either WT or the various G3BP1 truncations. Cells were lysed prior to separation by 12% SDS-PAGE. B) WT or ΔG3BP1 cells complemented with the indicated constructs were plated in a 96 well plate then infected with a serial dilution of MNV, before being fixed and stained 5 days post infection as described in the text. C) WT or ΔG3BP1 cells complemented with the indicated constructs were infected with MNV at an MOI of 10 TCID50 per cell. After 24 hours the virus yield was determined by TCID50. The error bars represent the standard error of three independent repeats. The data are representative of at least two independently repeated experiments.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>G3BP1 is required for a post entry step in the norovirus life cycle</h3><p id="p-23">To further define the role of G3BP1 in the norovirus life cycle we examined whether G3BP1 functioned at the level of viral entry or post-entry. We therefore bypassed the entry phase of the infection process and transfected viral VPg-linked RNA into WT and two independently generated BV-2 ΔG3BP1 cell lines and examined the impact on norovirus replication. Transfection of MNV viral VPg-linked RNA into WT cells resulted in high yields of infectious virus (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Fig 7A</a>) and viral proteins (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Fig 7C</a>). The levels of infectivity obtained following transfection of ΔG3BP1 cell lines with MNV viral RNA was comparable to that obtained in WT cells in the presence of the nucleoside analogue 2’-C-methylcytidin (2CMC), a known inhibitor of the norovirus RNA polymerase (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Rocha-Pereira et al., 2012</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">2013</a>) (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Fig 7A</a>). No viral proteins were detected in either of the ΔG3BP1 cell lines suggesting a defect at a very early stage in the viral life cycle (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Fig 7C</a>). Transfection of VPg-linked RNA into the ΔG3BP1 cell lines reconstituted with WT G3BP1 restored the ability to produce infectious virus (<a id="xref-fig-7-5" class="xref-fig" href="#F7">Fig 7B</a>) and the production of viral proteins (<a id="xref-fig-7-6" class="xref-fig" href="#F7">Fig 7D</a>). A minor increase in viral infectivity was observed in the ΔG3BP1 cell line reconstituted with the ΔRGG construct producing viral titres that were higher than those obtained from the WT complemented line in the presence of 2CMC, suggesting low levels of viral replication (<a id="xref-fig-7-7" class="xref-fig" href="#F7">Fig 7B</a>). However, the levels of viral proteins produced in this line was below the limit of detection by western blot (<a id="xref-fig-7-8" class="xref-fig" href="#F7">Fig 7D</a>). These data confirm that G3BP1 is required for a post entry stage of the norovirus life cycle and that in the absence of G3BP1 only residual norovirus replication is observed.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Loss of G3BP1 results in a defect following transfection of viral VPg-linked RNA into &#x394;G3BP1 cells. A) The indicated cell lines were transfected with MNV viral RNA and harvested at 9 hours post transfection for TCID50 to assess the virus yield. In some instances, the nucleoside analogue 2CMC was included to inhibit viral replication. The dotted line indicates the limit of detection (LOD) and the error bars represent the standard error from three biological repeats. B) Infectious virus yield from &#x394;G3BP1 and reconstituted cell lines performed as described in panel A. C) and D) illustrate the accompanying western blots for samples prepared in panel A and B respectively. Samples were prepared at 24 hours post transfection, prior to harvesting, separation by SDS-PAGE on a 4-12% gradient gel prior to western blotting for the indicated proteins." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1550667250" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Loss of G3BP1 results in a defect following transfection of viral VPg-linked RNA into &#x394;G3BP1 cells.&lt;/span&gt; A) The indicated cell lines were transfected with MNV viral RNA and harvested at 9 hours post transfection for TCID50 to assess the virus yield. In some instances, the nucleoside analogue 2CMC was included to inhibit viral replication. The dotted line indicates the limit of detection (LOD) and the error bars represent the standard error from three biological repeats. B) Infectious virus yield from &#x394;G3BP1 and reconstituted cell lines performed as described in panel A. C) and D) illustrate the accompanying western blots for samples prepared in panel A and B respectively. Samples were prepared at 24 hours post transfection, prior to harvesting, separation by SDS-PAGE on a 4-12% gradient gel prior to western blotting for the indicated proteins.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F7.medium.gif" width="440" height="336"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7:" src="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F7.medium.gif" width="440" height="336"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7:</span> <span class="caption-title">Loss of G3BP1 results in a defect following transfection of viral VPg-linked RNA into ΔG3BP1 cells.</span><p id="p-24" class="first-child">A) The indicated cell lines were transfected with MNV viral RNA and harvested at 9 hours post transfection for TCID50 to assess the virus yield. In some instances, the nucleoside analogue 2CMC was included to inhibit viral replication. The dotted line indicates the limit of detection (LOD) and the error bars represent the standard error from three biological repeats. B) Infectious virus yield from ΔG3BP1 and reconstituted cell lines performed as described in panel A. C) and D) illustrate the accompanying western blots for samples prepared in panel A and B respectively. Samples were prepared at 24 hours post transfection, prior to harvesting, separation by SDS-PAGE on a 4-12% gradient gel prior to western blotting for the indicated proteins.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>G3BP1 is required for viral negative sense RNA synthesis</h3><p id="p-25">To define the precise role of G3BP1 in the early stages of the virus life cycle, we used strand-specific RT-qPCR to quantify the levels of viral positive and negative sense RNA in WT and ΔG3BP1 cell lines following infection with MNV. As a control, 2CMC was included following virus inoculation as illustrated in the experimental time line (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Fig 8A</a>). The production of viral positive sense RNA was reduced to background levels in the absence of G3BP1, comparable to levels observed when the 2CMC was present during the infection (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Fig 8B</a>). Viral negative sense RNA synthesis was also reduced to below the detection limit of the assay in ΔG3BP1 cell lines (<a id="xref-fig-8-3" class="xref-fig" href="#F8">Fig 8C</a>). Surprisingly, we were able to detect an ~5 fold increase in viral negative sense RNA production at 6 hours post infection of WT cells in the presence of 2CMC, which, given that 2CMC was added after the inoculation phase (<a id="xref-fig-8-4" class="xref-fig" href="#F8">Fig 8B</a>), likely represents the first round of viral negative sense RNA synthesis, confirming the sensitivity of the assay. Addition of 2CMC during the inoculation phase reduced this background levels (data not shown).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The Lack of G3BP1 results in a failure to produce viral negative sense RNA. The experimental design is illustrated in A. Wild type or &#x394;G3BP1 (1B2) cells were infected prior to the addition of the nucleoside analogue 2CMC to prevent viral RNA synthesis. Samples were harvest at the indicated time post infection and viral positive (B) and negative sense RNA quantified by stand specific RT-qPCR (C). Error bars represent standard error of three biological repeats. LOD refers to the limit of detection of the assay. D) and E) Viral RNA synthesis following transfection of viral VPg-linked RNA into WT or two &#x394;G3BP1 cell lines. F) and G) Viral RNA synthesis following transfection of viral VPg-linked RNA into &#x394;G3BP1 (1B2) complemented with full length G3BP1 or truncated derivatives." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1550667250" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;The Lack of G3BP1 results in a failure to produce viral negative sense RNA.&lt;/span&gt; The experimental design is illustrated in A. Wild type or &#x394;G3BP1 (1B2) cells were infected prior to the addition of the nucleoside analogue 2CMC to prevent viral RNA synthesis. Samples were harvest at the indicated time post infection and viral positive (B) and negative sense RNA quantified by stand specific RT-qPCR (C). Error bars represent standard error of three biological repeats. LOD refers to the limit of detection of the assay. D) and E) Viral RNA synthesis following transfection of viral VPg-linked RNA into WT or two &#x394;G3BP1 cell lines. F) and G) Viral RNA synthesis following transfection of viral VPg-linked RNA into &#x394;G3BP1 (1B2) complemented with full length G3BP1 or truncated derivatives.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F8.medium.gif" width="378" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8:" src="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F8.medium.gif" width="378" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8:</span> <span class="caption-title">The Lack of G3BP1 results in a failure to produce viral negative sense RNA.</span><p id="p-26" class="first-child">The experimental design is illustrated in A. Wild type or ΔG3BP1 (1B2) cells were infected prior to the addition of the nucleoside analogue 2CMC to prevent viral RNA synthesis. Samples were harvest at the indicated time post infection and viral positive (B) and negative sense RNA quantified by stand specific RT-qPCR (C). Error bars represent standard error of three biological repeats. LOD refers to the limit of detection of the assay. D) and E) Viral RNA synthesis following transfection of viral VPg-linked RNA into WT or two ΔG3BP1 cell lines. F) and G) Viral RNA synthesis following transfection of viral VPg-linked RNA into ΔG3BP1 (1B2) complemented with full length G3BP1 or truncated derivatives.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">Similar results were obtained following transfection of viral RNA into cells to bypass the entry phase; viral positive and negative sense RNA synthesis was near (or below) the sensitivity of the assay following transfection of viral VPg-linked RNA into two independent ΔG3BP1 cell lines (<a id="xref-fig-8-5" class="xref-fig" href="#F8">Fig 8D</a> &amp; <a id="xref-fig-8-6" class="xref-fig" href="#F8">E</a>). Complementation with WT G3BP1, but not the mutant forms lacking the RNA binding domains, also restored viral positive and negative sense RNA synthesis (<a id="xref-fig-8-7" class="xref-fig" href="#F8">Fig 8F</a> &amp; <a id="xref-fig-8-8" class="xref-fig" href="#F8">G</a>). We did not detect viral positive or negative sense RNAs in the ΔRGG complemented cell line, despite the presence of low levels of viral infectivity (<a id="xref-fig-7-9" class="xref-fig" href="#F7">Fig 7B</a>). This discrepancy likely reflects the relative sensitivities of the assays and the nature of the strand specific qPCR assay which requires low levels of RNA input to maintain strand specificity. Together these data suggest that the function of G3BP1 is prior to, or at the level of viral negative sense RNA synthesis, with the most logical steps being either viral RNA translation or the formation of viral replication complexes.</p></div><div id="sec-11" class="subsection"><h3>G3BP1 is required for the association of VPg with 40S ribosomal subunits</h3><p id="p-28">We have previously shown that norovirus VPg interacts with eIF4G to recruit ribosomal subunits and direct viral translation (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Chaudhry et al., 2006</a>; <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Chung et al., 2014</a>). The interaction between VPg and eIF4G occurs via a direct interaction between the highly conserved C-terminal region in VPg and the central HEAT domain of eIF4G (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Leen et al., 2016</a>) and does not require any additional cellular cofactors, at least <em>in vitro</em>. The interaction between the eIF4G HEAT domain and the eIF3 complex plays a central role in the recruitment of the 40S ribosomal subunit for translation initiation (Marcotrigiano:2001uq; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kumar et al., 2016</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Villa et al., 2013</a>). Our proteomics analysis also confirms that the norovirus VPg proteins form a complex that contains multiple components of the 40S subunit (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Fig 1D</a>) and it has been established previously that G3BP1 associates with 40S subunits (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Kedersha et al., 2016</a>). To assess a potential role for G3BP1 in the formation of VPg-driven translation complexes in cells, we examined the ability of GFP tagged version of MNV VPg to pull down 40S subunits in the presence and absence of G3BP1. GFP-tagged WT MNV VPg was readily able to pull down eIF4G, G3BP1 and RpS6, a component of the 40S subunit (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Fig 9A</a>). However, in the absence of G3BP1, the ability to pull down RpS6 was lost (<a id="xref-fig-9-2" class="xref-fig" href="#F9">Fig 9A</a>). Furthermore, we found that disruption of the VPg-eIF4G interaction by the introduction of the F123A mutation into the eIF4G binding domain, also significantly reduced the ability to pull down RpS6 (<a id="xref-fig-9-3" class="xref-fig" href="#F9">Fig 9B</a>). These data suggest that the interaction of VPg with eIF4G is important for complex formation with ribosomal proteins and that that G3BP1 contributes in some manner to the formation of this complex.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="G3BP1 is required for the association of VPg with 40S ribosomal subunits. A) GFP-Trap immunoprecipitation of complexes isolated on with GFP alone or GFP tagged wild type MNV-VPg demonstrating the pull down of eIF4G1, G3BP1 and 40S subunits (Rps6). BVv2 cells were transfected with the relevant constructs, lysates prepared and GFP-Trap pull downs performed as detailed in the text. Samples were separated by SDS-PAGE and western blotted for the proteins as shown. B) Mutations in the eIF4G binding domain ablate the association of VPg with G3BP1 and 40S subunits. GFP-Trap pull downs were performed as described in panel A with the addition of the MNV VPG Fq123A mutation known to reduce the association with eIF4G." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1550667250" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;G3BP1 is required for the association of VPg with 40S ribosomal subunits.&lt;/span&gt; A) GFP-Trap immunoprecipitation of complexes isolated on with GFP alone or GFP tagged wild type MNV-VPg demonstrating the pull down of eIF4G1, G3BP1 and 40S subunits (Rps6). BVv2 cells were transfected with the relevant constructs, lysates prepared and GFP-Trap pull downs performed as detailed in the text. Samples were separated by SDS-PAGE and western blotted for the proteins as shown. B) Mutations in the eIF4G binding domain ablate the association of VPg with G3BP1 and 40S subunits. GFP-Trap pull downs were performed as described in panel A with the addition of the MNV VPG Fq123A mutation known to reduce the association with eIF4G.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F9.medium.gif" width="440" height="434"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9:" src="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F9.medium.gif" width="440" height="434"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9:</span> <span class="caption-title">G3BP1 is required for the association of VPg with 40S ribosomal subunits.</span><p id="p-29" class="first-child">A) GFP-Trap immunoprecipitation of complexes isolated on with GFP alone or GFP tagged wild type MNV-VPg demonstrating the pull down of eIF4G1, G3BP1 and 40S subunits (Rps6). BVv2 cells were transfected with the relevant constructs, lysates prepared and GFP-Trap pull downs performed as detailed in the text. Samples were separated by SDS-PAGE and western blotted for the proteins as shown. B) Mutations in the eIF4G binding domain ablate the association of VPg with G3BP1 and 40S subunits. GFP-Trap pull downs were performed as described in panel A with the addition of the MNV VPG Fq123A mutation known to reduce the association with eIF4G.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>G3BP1 is require for efficient polysome loading of norovirus VPg-linked RNA</h3><p id="p-30">To assess the impact of G3BP on the selective translation of viral VPg-linked RNA following viral infection, we evaluated the impact of loss of G3BP1 on the recruitment of viral RNA to polysomes under conditions where viral RNA synthesis was inhibited, namely in the presence of 2CMC. This approach enabled us to assess only the capacity of the incoming parental viral RNA to assemble into translationally active complexes, a stage often referred to as the “maiden round” of RNA virus genome translation. To this aim, cells were infected with MNV in the presence of 2CMC and polysomes profiling on extracts prepared from cells at 4 and 9 hours post infection performed (<a id="xref-fig-10-1" class="xref-fig" href="#F10">Fig 10A</a>). Quantification of the viral RNA levels in cells in the presence of 2CMC confirmed that the absence of G3BP1 has no impact on the overall levels present at the time points examined (data not shown). We noted that even in the presence of 2CMC, which inhibits viral RNA synthesis, there was a small but measurable increase in free 80S ribosomes over time in WT cells but not in cells lacking G3BP1 (<a id="xref-fig-10-2" class="xref-fig" href="#F10">Fig 10A</a>). We have previously found that MNV infection results in translation shut off and that this effect is at least partuialkly due to the activity of the NS6 protease (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Emmott et al., 2017</a>). The fact we observed 80S accumulation in WT cells, even in the absence of viral RNA synthesis, but not in cells lacking G3BP1, indirectly lead us to suspect that translation of viral RNA had occurred in WT cells, but was much less efficient in cells lacking G3BP1. Further analyses indicated that while most ribosome-associated norovirus RNA in WT cells was found in polysomes containing fractions, less viral RNA was found in ribosome-containing fractions (1-12 in <a id="xref-fig-10-3" class="xref-fig" href="#F10">Fig 10A</a> &amp; <a id="xref-fig-10-4" class="xref-fig" href="#F10">B</a>) in the absence of G3BP1 cells and, in comparison to WT cells, very little viral RNA was found in fractions containing polysomes (<a id="xref-fig-10-5" class="xref-fig" href="#F10">Fig 10B</a>). Extending the fractionation to include the free RNA and ribonucleoprotein complexes at the top of each gradient confirmed that in the absence of G3BP1 norovirus RNA is less efficient at assembling into polysomal fractions, suggesting a defect at the level of viral protein synthesis (<a id="xref-fig-10-6" class="xref-fig" href="#F10">Fig 10C</a>). Together these data support the hypothesis that G3BP1 functions to promote the translation of norovirus VPg-linked RNA, by facilitating the association with 40S subunits.</p><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="G3BP1 is required for polysome association of viral RNA association. A) Polysome profiles of the ribosome containing fractions from mock or MNV infected wild type (WT) or &#x394;G3BP1 (1B2) BV-2 cells at 4 and 9 hours post infection (moi 3 TCID50/cell). B) Relative viral RNA levels present in ribosome containing fractions expressed relative to WT infected BV-2 cells. C) Extended gradient fractionation of WT or &#x394;G3BP1 cells infected with MNV and harvested 9 hours post infection. Viral RNA levels across the gradient are expressed as described in panel B." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1550667250" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;G3BP1 is required for polysome association of viral RNA association.&lt;/span&gt; A) Polysome profiles of the ribosome containing fractions from mock or MNV infected wild type (WT) or &#x394;G3BP1 (1B2) BV-2 cells at 4 and 9 hours post infection (moi 3 TCID50/cell). B) Relative viral RNA levels present in ribosome containing fractions expressed relative to WT infected BV-2 cells. C) Extended gradient fractionation of WT or &#x394;G3BP1 cells infected with MNV and harvested 9 hours post infection. Viral RNA levels across the gradient are expressed as described in panel B.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 10:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F10.medium.gif" width="440" height="413"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 10:" src="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F10.medium.gif" width="440" height="413"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 10:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 10:</span> <span class="caption-title">G3BP1 is required for polysome association of viral RNA association.</span><p id="p-31" class="first-child">A) Polysome profiles of the ribosome containing fractions from mock or MNV infected wild type (WT) or ΔG3BP1 (1B2) BV-2 cells at 4 and 9 hours post infection (moi 3 TCID50/cell). B) Relative viral RNA levels present in ribosome containing fractions expressed relative to WT infected BV-2 cells. C) Extended gradient fractionation of WT or ΔG3BP1 cells infected with MNV and harvested 9 hours post infection. Viral RNA levels across the gradient are expressed as described in panel B.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>G3BP1 is require for efficient norovirus VPg-dependent translation in the presence of cellular capped RNAs</h3><p id="p-32">To further examine a potential role of G3BP1 in norovirus VPg-dependent translation, cytoplasmic translationally competent extracts were prepared from WT and ΔG3BP1 cell lines and the translation of highly purified viral VPg-linked viral RNA (<a id="xref-fig-14-1" class="xref-fig" href="#F14">Fig S11A</a>&amp;<a id="xref-fig-14-2" class="xref-fig" href="#F14">B</a>) examined. Cap-dependent and cricket paralysis virus IRES (CrPV)-dependent translation were comparable in nuclease treated extracts prepared from WT and ΔG3BP1 cell lines (<a id="xref-fig-11-1" class="xref-fig" href="#F11">Fig 11A</a>), whereas norovirus VPg-dependent translation was reduced (<a id="xref-fig-11-2" class="xref-fig" href="#F11">Fig 11B</a>). Quantification of multiple experiments indicated that translation in nuclease treated extracts was on average reduced by ~40-50% because of G3BP1 ablation (<a id="xref-fig-11-3" class="xref-fig" href="#F11">Fig 11C</a>). A similar reduction in <em>in vitro</em> translation was observed across multiple time points (<a id="xref-fig-14-3" class="xref-fig" href="#F14">Fig S11C</a>).</p><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="G3BP1 is required for efficient norovirus VPg-dependent translation. Translation competent extracts from &#x394;G3BP1 cells are fully competent for cap-dependent and CrPV IRES dependent translation (A). B) Translation of MNV VPg-linked viral RNA is diminished in extracts prepared from &#x394;G3BP1 cells. Translation of viral RNA in rabbit reticulocyte lysates (RRL) are used as a side by side comparison. The positions of the viral proteins are indicated (1-5). C) Quantification of protein products and total translation levels across multiple experiments. The error bars represent the standard error of three independent experiments. D) Norovirus VPg-dependent translation in extracts from WT or &#x394;G3BP1 cells in the presence of increasing concentrations of total cellular RNA isolated from uninfected cells. Translation of viral RNA in rabbit reticulocyte lysates (RRL) are used as a side by side comparison. E) Quantification of viral proteins produced in panel (D)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1550667250" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;G3BP1 is required for efficient norovirus VPg-dependent translation.&lt;/span&gt; Translation competent extracts from &#x394;G3BP1 cells are fully competent for cap-dependent and CrPV IRES dependent translation (A). B) Translation of MNV VPg-linked viral RNA is diminished in extracts prepared from &#x394;G3BP1 cells. Translation of viral RNA in rabbit reticulocyte lysates (RRL) are used as a side by side comparison. The positions of the viral proteins are indicated (1-5). C) Quantification of protein products and total translation levels across multiple experiments. The error bars represent the standard error of three independent experiments. D) Norovirus VPg-dependent translation in extracts from WT or &#x394;G3BP1 cells in the presence of increasing concentrations of total cellular RNA isolated from uninfected cells. Translation of viral RNA in rabbit reticulocyte lysates (RRL) are used as a side by side comparison. E) Quantification of viral proteins produced in panel (D).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 11:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F11.medium.gif" width="393" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 11:" src="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F11.medium.gif" width="393" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 11:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/08/571455/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 11:</span> <span class="caption-title">G3BP1 is required for efficient norovirus VPg-dependent translation.</span><p id="p-33" class="first-child">Translation competent extracts from ΔG3BP1 cells are fully competent for cap-dependent and CrPV IRES dependent translation (A). B) Translation of MNV VPg-linked viral RNA is diminished in extracts prepared from ΔG3BP1 cells. Translation of viral RNA in rabbit reticulocyte lysates (RRL) are used as a side by side comparison. The positions of the viral proteins are indicated (1-5). C) Quantification of protein products and total translation levels across multiple experiments. The error bars represent the standard error of three independent experiments. D) Norovirus VPg-dependent translation in extracts from WT or ΔG3BP1 cells in the presence of increasing concentrations of total cellular RNA isolated from uninfected cells. Translation of viral RNA in rabbit reticulocyte lysates (RRL) are used as a side by side comparison. E) Quantification of viral proteins produced in panel (D).</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">The ability of VPg-linked norovirus RNA to be translated in the presence of increasing amounts of total cellular RNAs was then examined to assess the relative role of G3BP1 under conditions where cellular RNA are present. Total RNA isolated from uninfected cells was titrated into the nuclease treated extracts and the impact norovirus VPg-dependent translation examined (<a id="xref-fig-11-4" class="xref-fig" href="#F11">Fig 11D</a>). We found that in the presence of cellular RNAs, the translation of norovirus VPg-linked RNA in extracts from ΔG3BP1 cells is reduced by up to 80% in comparison to extracts from WT cells. In agreement with our data using polysome profiling, these data suggest that G3BP1 functions to provide a competitive advantage for norovirus VPg-linked RNA under conditions where cellular RNAs are present.</p></div></div><div class="section" id="sec-14"><h2 class="">Discussion</h2><p id="p-35">In this study, we have used a combination of biochemical and genetic approaches to identify host factors involved in the norovirus life cycle. Our combined approaches resulted in the identification of the core stress granule component G3BP1 as a host protein critical for the replication of both murine and human noroviruses in cell culture. Furthermore, we determined that G3BP1 plays a key role in the processes of norovirus VPg-dependent protein synthesis, uncovering a new function for G3BP1 in facilitating RNA virus genome translation.</p><p id="p-36">The orthogonal approaches used in the current study provide an unprecedented insight into the identity of host factors with potential roles in the norovirus life cycle. The detailed proteomic analysis of the viral replication and translation complexes formed during MNV infection (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Fig 2</a>) resulted in the identification of several host factors with previously identified roles in the MNV life cycle. We focused our efforts on G3BP1 as it was identified in all three approaches and was also identified in a CRISPR screen published during this study (<a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Orchard et al., 2016</a>). Furthermore, we have previously shown that feline calicivirus (FCV), a relative of noroviruses within the <em>Vesivirus</em> genus, cleaves G3BP1 to inhibit stress granule formation (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Humoud et al., 2016</a>). In contrast, MNV infection does not result in G3BP1 cleavage and instead forms cytoplasmic foci the composition of which is distinct from canonical stress granules (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Brocard et al., 2018</a>).</p><p id="p-37">G3BP1 is one member of a group of G3BP proteins (Ras-GTPase-activating protein (SH3 domain)-binding proteins), referred to as Rasputin in insects, that possess RNA binding activity and have multiple cellular functions including the regulation of RNA stability and translation in response to stress. Originally identified as a protein that interacted with Ras-GTPase activating protein (RasGAP), more than two decades of research have significantly expanded our knowledge of the multifunctional role in cellular processes. It is now well accepted that G3BPs play a role in cancer cell survival, cancer metastasis and invasion, processing of specific miRNAs and stress granule formation (Reviewed in (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alam and Kennedy, 2019</a>). Stress granules are dynamic cytoplasmic ribonucleoprotein complexes that form rapidly under stress conditions and within which cellular RNAs are stored in stalled translation complexes (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Protter and Parker, 2016</a>). In the context of viral infection, numerous studies have suggested that many, if not all, viruses must interact in some manner with stress granules as there is growing evidence that the formation of cytoplasmic stress granules is part of the antiviral defence mechanism (Reviewed in (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">McCormick and Khaperskyy, 2017</a>). Some viruses interact with stress granules to promote viral replication (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cristea et al., 2010</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kim et al., 2016</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Panas et al., 2014</a>; 2012) whereas some do so to counteract the inhibitory effect of stress granules on the translation of viral RNA (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Panas et al., 2015</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">White et al., 2007</a>).</p><p id="p-38">Our data suggests that G3BP1 plays a key role in promoting the translation of norovirus VPg-linked viral RNA. Positive sense RNA viruses have evolved mechanisms to ensure the efficient translation of their viral genomic RNAs in the presence of high concentrations of competing cellular RNAs. These mechanisms include the use of internal ribosome entry site elements (IRES), modified cap-dependent mechanisms (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Firth and Brierley, 2012</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Jaafar and Kieft, 2019</a>) and the ability to target the host cell translation machinery to generate an environment where viral RNA translation is favoured over cellular capped RNAs (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Walsh et al., 2013</a>).</p><p id="p-39">G3BP1 is known to associate primarily with free 40S subunits and not 80S monosomes (<a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Kedersha et al., 2016</a>). Our data supports a hypothesis whereby the association of G3BP1 with 40S ribosomal subunits provides a selective advantage for norovirus VPg-dependent translation, thereby uncovering a new function in virus specific translation. The mechanism by which G3BP1 contributes to this process has yet to be fully explored but our data supports the hypothesis that G3BP1 directly or indirectly promotes the recruitment of 40S ribosomal subunits to VPg-driven translation complexes. The RGG motif of G3BP1 is known to be essential for the association between G3BP1 and 40S subunits as well as the ability to from stress granules, whereas data would suggest that the RRM may play a regulatory role (<a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">Kedersha et al., 2016</a>). These domains were also required for the function of G3BP1 in the norovirus life cycle (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Fig 6</a>) confirming that the G3BP1 association with 40S is important for its role in promoting norovirus VPg-dependent translation. Importantly, RGG domains are known to have many functions (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Thandapani et al., 2013</a>) and therefore in the context of G3BP1 function in the norovirus life cycle, may also contribute to unknown interactions that promote norovirus translation. Previous work on alphaviruses have shown that G3BP1 is sequestered by binding to the nsP3 protein (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Panas et al., 2012</a>; 2014; 2015). Furthermore, this interaction occurs via an FGDF motif also found in other viral proteins including the ICP8 protein of herpes simplex virus (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Panas et al., 2015</a>). While the MNV VPg protein has a similar motif FGDGF (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Fig 1A</a>), this motif is not conserved in the GI Norwalk virus VPg protein. Therefore our data suggest that the interaction of VPg with G3BP1 is not direct, fitting with our observation that this interaction is reduced by mutations in the eIF4G binding domain (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Fig 1A</a> and <a id="xref-fig-9-4" class="xref-fig" href="#F9">Fig 9B</a>.) While our data fit with a primary role for G3BP1 in norovirus translation, we are unable to exclude the possibility that G3BP1 plays other roles in the viral life cycle. Recent studies have confirmed that G3BP1 is enriched at sites of viral RNA synthesis (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Brocard et al., 2018</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Fritzlar et al., 2019</a>) so it is possible that G3BP1 makes multiple contacts between the 40S subunit and viral RNA genome directly.</p><p id="p-40">The technical challenges associated with studying human norovirus replication in cell culture have limited the experimental approaches we could use to validate the role of G3BP1 in human norovirus translation. However, our results have clearly demonstrated that in the absence of G3BP1, human Norwalk virus is unable to replicate or form replicon-containing colonies. The presence of G3BP1 in the NV VPg-containing complexes again fits with our hypothesis that G3BP1 plays a role in promoting viral VPg-dependent protein synthesis.</p><p id="p-41">We have previously found that norovirus infection leads to a translation bias whereby cellular mRNAs induced in response to infection are inefficiently translated (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Emmott et al., 2017</a>). Our data suggested that this modification of host cell translation was at least partially driven by the ability of the viral NS6 protease to cleave PABP and the induction of apoptosis which results in cleavage of cellular translation initiation factors (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Emmott et al., 2017</a>). Furthermore, we have more recently shown that the ability of the protease to cleave PABP and other substrates is controlled by polyprotein processing and interactions with other viral proteins (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Emmott et al., 2019</a>). Recent work would agree with our observations and confirms that the translational bias is not driven by phosphorylation of eIF2α and that activation of GCN2 leads to the phosphorylation of eIF2α (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Brocard et al., 2018</a>). We note however that others have suggested that NS3 may contribute to translational shut off (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Fritzlar et al., 2019</a>), with the caveat that this observation was made outside of the context of infected cells and using overexpressed tagged proteins. We suspect that noroviruses use multiple mechanisms that work co-operatively to enable the control of host gene expression and the subsequent translation of the cellular mRNAs. The relative contribution of these processes in any given cell type may also differ depending on the degree to which the cells can sense and respond to viral infection through the induction of innate and apototic responses.</p><p id="p-42">The observation that many of the factors enriched using the VPg protein were only enriched on complexes purified with NS1/2 tagged infectious MNV, could suggest that the viral proteins present in the viral translation complex are distinct from those present in complexes active for viral RNA synthesis. However, we cannot formally rule out other possible explanations including the possibility that the specific enrichment of translation factors on NS1/2 occurs because NS1/2 is the first protein to be translated from ORF1, therefore unprocessed NS1/2 at the N-terminus of the ORF1 polyprotein being actively translated could function as an anchor, facilitating the enrichment of ribosomes and the associated factors. In addition, we have previously seen that VPg-containing precursors may bind the translation initiation factor eIF4G less well (<a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Leen et al., 2016</a>), which could prevent some VPg (NS5) containing precursors associating with translation initiation complexes.</p><p id="p-43">In conclusion, our data adds significantly to the growing body of literature on the role of G3BP proteins in the life cycle of viruses and further extends the functional roles of G3BP1 to include the promotion of viral translation processes. We identify G3BP1 as a host protein that has a critical role in the life cycle of murine and human noroviruses, identifying the first cellular pro-viral protein with pan-norovirus activity. Furthermore, given the apparent importance of G3BP1 to an early stage of the norovirus life cycle, this work suggests that targeting G3BP1 may hold future therapeutic potential.</p><div id="sec-15" class="subsection"><h3>Grants</h3><p id="p-44">CBW was supported by NIH K08 AI128043 and Burroughs Wellcome Fund. MAM was supported by NIH U19 AI109725. IG is a Wellcome Senior Fellow. This work was supported by funding from the Wellcome Trust (Refs: 207498/Z/17/Z and 104914/Z/14/Z) and the Biotechnology and Biological Sciences Research Council UK (Refs: BB/N001176/1and BB/000943N/1). JBE was supported by a Churchill Scholarship.</p></div></div><div class="section" id="sec-17"><h2 class="">Materials and methods</h2><div class="section" id="sec-18"><h3>Cells</h3><p id="p-46">The murine microglial BV-2 cell line (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Blasi et al., 1990</a>) was provided by Jennifer Pocock (University College London). BV-2 cells were maintained in DMEM supplemented with 10% FCS (Biosera), 2 mM L-glutamine, 0.075 % sodium bicarbonate (Gibco) and the antibiotics penicillin and streptomycin. BHK cells engineered to express T7 RNA polymerase (BSR-T7 cells, obtained from Karl-Klaus Conzelmann, Ludwid Maximillians University, Munich, Germany) were maintained in DMEM containing 10 % FCS, penicillin (100 SI units/ml) and streptomycin (100 μg/ml), and 0.5 mg/ml G418.</p></div><div class="section" id="sec-19"><h3>Generation of G3BP1 KO cells</h3><p id="p-47">BV-2 cells were cultured in DMEM containing 10% FBS and 1% HEPES. G3BP1 knockout BV-2 cells were generated using two approaches. The clone 1B2 was generated by transiently transfected with Cas9 and a sgRNA (5TTCCCCGGCCCCGGCTGATGNGG) targeting exon 7 of G3BP1. BV-2 cells were then single cell cloned and G3BP1 was sequenced by Illumina HiSeq. BV-2 cells are polyploid at the G3BP1 locus as described previously (<a id="xref-ref-42-5" class="xref-bibr" href="#ref-42">Orchard et al., 2016</a>). Clone 1B2 also had three independent deletions at the sgRNA binding site resulting in deletions of 1, 2, and 5 base pairs respectively. The mutations introduced into the IB2 BV-2 cell clone resulted in frame shifts at nucleotide positions 253, 254 and 244 and the absence of detectable G3BP1 protein as measured by western blot. G3BP1 knockout BV-2 cell clones A, C and F were generated using an independent approach that relied on first generating a pool of three lentiviruses carrying guide RNAs TGTGCAACATGTCCGGGGCC, CAAACTCCCGCCCGACCAGC and TAGTCCCCTGCTGGTCGGGC targeting the first 100bp of the coding sequence, cloned into pLentiCRISPRv2 (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Sanjana et al., 2014</a>). BV-2 cells were then transduced with the pool of 3 lentiviruses, selected by puromycin treatment for 72 hours, prior to cloning by limiting dilution. Individual clones were then screened by western blot for the absence of G3BP1.</p></div><div class="section" id="sec-20"><h3>G3BP1 complementation</h3><p id="p-48">Mouse G3BP1 cDNAs were subcloned into pCDH-MCS-T2A-puro-MSCV lentiviral vector (System Biosciences) by NEBuilder HiFi DNA assembly (New England Biolabs). Mouse G3BP1 was subcloned from pCM6-G3BP1 (MR207441; Origene). Mouse G3BP1 lentiviral constructs deficient in the C-terminal RGG domain (mG3BP1ΔRGG) and the RGG and RRM domains (mG3BP1ΔRGGRRM) were generated by Gibson cloning from the pCMV6-G3BP1 vector. Lentivirus was generated by co-transfecting pCDH-G3BP1-T2A-puro-MSCV with pCMV-VSV-G and pSPAX2 into 293T cells with Trans-IT LT1 (Mirus Biosciences) per manufacture instructions. Two days post-transfection, supernatants were harvested, filtered through a 0.22 micron filter, and stored at −80C. Lentivirus encoding G3BP1 or an empty control was then used to transduce G3BP1 KO 1B2 BV-2 cells. Two days post-transduction BV-2 cells were selected with puromycin (2.5ug/ml) for six days.</p></div><div class="section" id="sec-21"><h3>MNV growth curves</h3><p id="p-49">To determine the effects of G3BP1 disruption on MNV replication G3BP1 WT, KO, or complemented cells were plated in each well of a flat bottom 96-well plate and then infected with either MNV strains CW1, CW3, or CR6 as described in the text. Infected cells were flash frozen at −80°C at the times post infection indicated in the text. Viral replication was then assessed by plaque assay or TCID50 in BV-2 cells as described in the text. In cases where the appearance of virus-induced cytopathic effect was examined, infected monolayers were either visualized by light microscopy directly or fixed with crystal violet in formalin, prior to washing and imaging.</p></div><div class="section" id="sec-22"><h3>CRISPR screens</h3><p id="p-50">The CRISPR screen was performed similarly to that described previously (<a id="xref-ref-42-6" class="xref-bibr" href="#ref-42">Orchard et al., 2016</a>) with a number of modifications that included the use of the Brie gRNA library to reduce off target effects (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Doench et al., 2016</a>) and shorter infection times to improve the recovery of gRNAs that may also compromise cell viability. BV-2 cells stably expressing Cas9 nuclease (<a id="xref-ref-42-7" class="xref-bibr" href="#ref-42">Orchard et al., 2016</a>) were transduced with the Brie library using previously described protocols (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Doench et al., 2016</a>). MNV strains CW3 and CR6 were used to infect BV-2 CRISPR library at MOI 5 pfu/cell and cells were isolated 24 hours post infection and preparation of gDNA for sequencing as described previously (<a id="xref-ref-42-8" class="xref-bibr" href="#ref-42">Orchard et al., 2016</a>). The screen relies on the premise that guide RNAs targeting genes that are overrepresented following infection represent genes that when disrupted are protected against infection and therefore likely represent factors with pro-viral activity. Likewise, genes for which guide RNA are underrepresented suggest that infection had proceeded faster and the gene is antiviral. Following sequencing, the data was analyzed by STARS method as previously described (<a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">Doench et al., 2016</a>; <a id="xref-ref-42-9" class="xref-bibr" href="#ref-42">Orchard et al., 2016</a>). Visualization of candidate genes was accomplished using R (RStudio, Inc., Boston, MA).</p></div><div class="section" id="sec-23"><h3>Maintenance of SILAC cell lines</h3><p id="p-51">Stable isotope labelling of amino acids in cell culture of BV-2 cells (SILAC, Ong et al., 2002), was carried out in high-glucose DMEM lacking arginine and lysine (Sigma-Aldrich), supplemented with dialyzed fetal bovine serum, 1% L-glutamine, 1X nonessential amino acids, 10 mM HEPES, and 1X penicillin/streptomycin. SILAC media were supplemented with Light (R0K0), Medium (R6K4) or Heavy (R10K8) Arginine and Lysine (Cambridge Isotope Laboratories). BV-2 cells were maintained in SILAC medium for 2 weeks to ensure complete metabolic labelling of proteins. Labelling of HEK-293T cells was performed essentially as described for BV-2 cells, with the omission of 10 mM HEPES and 1X non-essential amino acids from the cell culture media.</p></div><div class="section" id="sec-24"><h3>DNA based recovery of murine norovirus</h3><p id="p-52">Experiments were performed according to previously published protocols (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chaudhry et al., 2007</a>). Briefly, BSRT7 cells were infected to an MOI of 0.5-1 PFU/cell with fowlpox virus expressing T7 RNA polymerase. Cells were then transfected with a plasmid encoding the MNV full length clone, or a derivative thereof (e.g. pT7 MNV 383FLAG 3’Rz or pT7 MNV 2600FLAG 3’Rz, our FLAG-tagged virus constructs containing FLAG tags in either NS1/2 or NS4 respectively). MNV was harvested by freeze-thaw at 24h post-transfection.</p><p id="p-53">To generate higher titre stocks, WT MNV, NS1/2-FLAG MNV, and NS4-FLAG MNV (<a id="xref-ref-55-4" class="xref-bibr" href="#ref-55">Thorne et al., 2012</a>) generated using the DNA based recovery method described above were passaged once in BV-2 cells. After 80-90% of cells displayed visible cytopathic effects (CPE) of viral infection, flasks containing infected cells were frozen at −80<sup>°</sup>C. Flasks were frozen and thawed twice before cell debris was removed by centrifugation at 4000 rpm for 10 minutes in a benchtop centrifuge. Viruses were pelleted by centrifuging over a 30% sucrose cushion at 76,755xg in a SW32ti rotor for 2 hours at 4<sup>°</sup>C. Virus pellets were resuspended overnight in PBS to achieve 100-fold concentration. Concentrated virus was then passed through a 23-gauge blunt needle 15 times, and clarified by centrifugation at maximum speed in a benchtop microcentrifuge for 10 minutes. Supernatant aliquoted, and titrated prior to use.</p></div><div class="section" id="sec-25"><h3>Infection of SILAC labelled BV-2 cells with FLAG-tagged viruses</h3><p id="p-54">SILAC-labelled BV-2 cells were infected with WT MNV, NS1/2-FLAG or NS4-FLAG viruses at an MOI of 10 TCID50 cell. Infections were performed in triplicate, using different combinations of SILAC-labeled BV-2 cells each time to control for any impact of the SILAC labelling. Infected cells were then plated in the appropriate SILAC media. At 10 hours post-infection, cells were harvested by scraping, and pelleted at 500xg for 5 minutes. Cells were then washed 3 times with ice-cold PBS, and were lysed in (0.5% Nonidet-P40 substitute, 10 mM Tris-HCl pH 7.5, 150 mM NaCl, 0.5 mM EDTA, 2 mM MgCl). Benzonase nuclease (Sigma-Aldrich) was added to lysis buffer to a concentration of 5 μl/ml to prevent nonspecific interactions mediated by RNA or DNA.</p></div><div class="section" id="sec-26"><h3>Transfection of SILAC labelled HEK-293T cells with GFP-tagged VPg</h3><p id="p-55">SILAC-labelled HEK-293T cells were transfected with pEGFP-C1 (control) or derivatives thereof containing either human or murine norovirus VPg protein as described in <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Emmott &amp; Goodfellow, (2014)</a>. GFP fusions of both wild-type protein or mutant VPg containing a mutation to inhibit initiation factor binding (MNV: F123A, HuNoV: F137A) were used. Cells were transfected using Lipofectamine 2000 (Life Technologies) according to the manufacturers protocol, using antibiotic-free SILAC media in place of Opti-mem. The experiment was performed in triplicate and SILAC labels switched in one of the replicates.</p></div><div class="section" id="sec-27"><h3>FLAG and GFP-TRAP immunoprecipitation</h3><p id="p-56">FLAG immunoprecipitations were performed following the manufacturer’s protocol (FLAG M2 beads, Sigma Aldrich) as described (<a id="xref-ref-55-5" class="xref-bibr" href="#ref-55">Thorne et al., 2012</a>). In brief, protein concentration in lysates was normalized using BCA. Lysates were then diluted with 1 volume of wash buffer (10 mM Tris-Cl pH 7.5, 150 mM NaCl, 0.5 mM EDTA). Equal volumes of anti-FLAG affinity gel were dispensed into either WT infected cell lysates, or lysates of cells infected with NS1/2-FLAG or NS4-FLAG. Binding was carried out overnight at 4<sup>°</sup>C with rotation. After binding, beads were washed 3 times with wash buffer. All liquid was carefully removed from each tube, before boiling in SDS-PAGE loading buffer for 10 minutes. GFP-trap immunoprecipitation of GFP-tagged VPg was accomplished using GFP-trap beads (Chromotek) per the manufacturer’s protocol, as described (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Emmott and Goodfellow, 2014</a>). RNase cocktail (Ambion) was also included in the lysis buffer at a concentration of 5 μl/ml to prevent non-specific interactions mediated by RNA. In all cases, light, medium, and heavy-labelled proteins eluted from the beads for each experimental replicate were pooled together in a ratio of 1:1:1 before submission for mass spectrometry analysis at the University of Bristol Proteomics Facility.</p></div><div class="section" id="sec-28"><h3>Mass spectrometry analysis</h3><p id="p-57">Mass spectrometry analysis was performed at the University of Bristol Proteomics Facility. In brief, samples were run into precast SDS-PAGE gels for 5 minutes, the entire sample cut from the gel as a single band, and then subjected to in-gel tryptic digestion including reduction and alkylation using a ProGest automated digestion unit. The resulting peptides were fractionated using a Dionex Ultimate 3000 nanoHPLC system in line with an LTQ-Orbitrap Velos or Orbitrap Tribrid Fusion mass spectrometer.</p></div><div class="section" id="sec-29"><h3>Interpretation of SILAC Proteomics data</h3><p id="p-58">Raw data files were processed and quantified using Maxquant v1.5.5.1 or 1.6.0.16 (Tyanova <em>et. al.</em> 2016). The GFP-VPg experiments were searched against the Uniprot human database (70,550 entries, downloaded September 19<sup>th</sup> 2016) plus a custom FASTA file containing the wild-type and mutant VPg sequences. The raw data, search results and FASTA files can be found as part of PRIDE submission PXD007585 (Reviewer username: <span class="em-link"><span class="em-addr">reviewer75984{at}ebi.ac.uk</span></span>, Password: BH2pTctW). The FLAG-virus experiments were searched against the Uniprot mouse database (Swiss-prot only, 16,966 entries, downloaded May 19<sup>th</sup> 2018) plus a custom FASTA file containing the various Murine norovirus proteins. The raw data, search results and FASTA files can be found as part of PRIDE submission PXD011779 (Reviewer username: <span class="em-link"><span class="em-addr">reviewer49419{at}ebi.ac.uk</span></span>, Password: eLYwivNP). Data were searched with default Maxquant parameters including upto 2 missed tryptic cleavages, oxidation of methionine and N-terminal acetylation as variable modifications, and carbamidomethylation of cysteine as a fixed modification. The data were searched against a reverse database and PSM and Protein FDR were set to 0.01. The requantify option was not selected.</p><p id="p-59">GFP-VPg data were analysed as described previously (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Emmott and Goodfellow, 2014</a>). FLAG-virus experiments were analysed by computing the pairwise ratios of samples infected with NS1/2-FLAG or NS4-FLAG relative to WT MNV-infected controls. Log<sub>2</sub> SILAC ratios for proteins identified in at least 2/3 replicates were averaged, and ratios for NS1/2-FLAG:WT and NS4-FLAG:WT were plotted for comparison of host cell proteins by viral replication complex-associated proteins.</p></div><div class="section" id="sec-30"><h3>Assessment of virus-induced cytopathic effect</h3><p id="p-60">BV-2 WT, KO G3BP1 or respective G3BP1 complemented cells as described in the text, were seeded onto 96 well plates and infected with serial 10-fold dilutions (starting at MOI=10 TCID50/cell) of MNV (CW1) or EMCV. At 48h post-infection, cells were fixed in ice-cold methanol and stained with toluidine blue prior to washing and imaging.</p></div><div class="section" id="sec-31"><h3>Cap-Sepharose purification for eIF4F complex</h3><p id="p-61">Cell lysates were prepared from BHK parental cells or BHK containing GI Norwalk virus (BHK-NV) replicon cells in cap-Sepharose lysis buffer (100 mM KCl, 0.1 mM EDTA, 10% glycerol, 2 mM MgCl<sub>2</sub>, 20 mM HEPES, pH 7.6 in KOH) with 1% TX-100, proteinase and phosphatase inhibitor cocktails (Calbiochem). Cytoplasmic extracts were centrifuged and RNase treated for 15 min at room temperature. At least 1000 µg of the cell lysates were incubated with Sepharose beads coupled to 7-methylguanosine (m<sup>7</sup>GTP, Jena Biosciences). Input cell lysates were collected for western blot analysis while the remaining were incubated overnight with continuous rotation at 4°C. The eIF4F-enriched complex was precipitated and washed 2 times with cap-sepharose lysis buffer. Bound proteins were eluted in 2x reducing SDS-PAGE samples buffer and resolved by SDS-PAGE prior to western blot.</p></div><div class="section" id="sec-32"><h3>Human Norwalk virus colony formation assay</h3><p id="p-62">Total RNAs extracts from BHK or BHK-NV replicon-containing cells (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Kitano et al., 2018</a>) were pretreated with and without proteinase K (10 µg/ml) in 10 mM Tris, pH 8.0, 1.0 mM EDTA, 0.1 M NaCl, and 0.5% SDS. Pretreated RNAs were immediately purified using GenElute RNA purification columns (Sigma). Serial 10-fold dilutions of mock or proteinase K-treated RNAs were transfected in BHK cells and 24 h post transfection, cells were passaged and maintained in growth media containing 0.5 mg/ml G418. Colonies began to form after 5 d and were allowed to grow until 14 d. All plates were harvested at day 14 and well-formed colonies were fixed in 10% formaldehyde and stained with toluidine blue. A similar protocol was followed to assess colony formation in U2OS cells with the exception that selections were maintained for up to 12 days post transfection. Where indicated, cell aliquots from each time point were collected for qRT-PCR analysis to assess viral RNA synthesis over time.</p></div><div class="section" id="sec-33"><h3>Polyribosome fractionation analysis</h3><p id="p-63">BV2 WT and BV2 ∆G3BP1 cells were seeded at a density of 7.5 x 10<sup>6</sup> cells per T-75 flask, and then either mock infected or infected with MNV1 (CW1) at MOI 3 TCID50 per cell in the presence of 2-CMC (400µM) for each set of infection. After 1h, the inoculum was then removed; the cells were washed and maintained in growth media containing 2-CMC accordingly until the cells were harvested at 4h and 9h p.i. Prior to harvesting, cells were treated with cycloheximide (CHX) for 10 mins at 37°C (Sigma-Aldrich; 100 µg/ml) and were rinsed with 5 ml of ice-cold PBS supplemented with CHX 100 µg/ml. Polysome lysis buffer [20 mM Tris-HCl pH 7.5, 150 mM NaCl, 5mM MgCl, 1 mM DTT, 1% Triton X-100, 100 µg/ml cycloheximide, 25 U/ml TURBO DNase (Life Technologies)] was used to lyse the cells. Lysates were clarified by centrifugation for 20 min at 13,000 <em>g</em> at 4<sup>°</sup>C. Aliquots of the lysates were collected for BSA assay and qPCR analysis against MNV1 RNA before proceeding with fractionation. Input lysates were normalized to total protein concentration and RT-qPCR was used to confirm the levels of viral RNA in samples were comparable. Lysates were subjected next to 10-50% sucrose gradient centrifugation for 90 mins SW41Ti rotor at 190,000 <em>x g</em> at 4<sup>°</sup>C.</p><p id="p-64">The gradients were fractionated at 0.5 ml/min and the levels of RNA in each sample measured using an in line-254 nm spectrophotometer connected to a chart recorder. RNAs were extracted from each fraction, converted to cDNA and immediately used for qPCR. The distribution of viral RNA across the gradient was then calculated as percentage (%) of the viral RNA seen in WT BV-2 cells using the reference gene (GAPDH) to obtain normalized values across the gradient. Samples were performed in duplicates on the same qPCR plate, and the observations were robust across three independent experiments. Data were collected using a ViiA 7 Real-Time PCR System (Applied Biosystems).</p></div><div class="section" id="sec-34"><h3>Transfection of VPg-linked MNV RNA into BV-2 cells</h3><p id="p-65">VPg-linked RNA purified from MNV-1 virus particles was transfected in BV-2 cells using NEON<sup>™</sup> as previously described (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Yunus et al., 2010</a>). Total cell lysates were harvested at 3 and 9 hours post transfection with RIPA buffer. 10μg total lysates were analysed by 4-12% SDS-PAGE (Invitrogen) and antibodies against MNV, VPg, G3BP1 and GAPDH were used for detection using LI-COR<sup>®</sup> Odyssey<sup>®</sup> CLx. Virus yield was determined by TCID<sub>50</sub>. For strand-specific qPCR detection of MNV RNA, total cellular RNA was extracted using GeneElute Mammalian Total RNA Miniprep kit (Sigma). RT-qPCR was performed as described previously (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Vashist et al., 2012a</a>).</p></div><div class="section" id="sec-35"><h3>Purification of MNV VPg-linked RNA</h3><p id="p-66">BV-2 cells were infected at an MOI=0.01 TCID<sub>50</sub> per cell and harvested after ~30 post infection. Cell debris was removed by low speed centrifugation for 10 minutes and supernatant loaded onto 5 ml of 30% sucrose solution in PBS. MNV particles were pelleted using a SW32Ti rotor at 25,000 RPM for 4 hours at 4 ℃. Virus was then resuspended in PBS and total RNA extracted from soluble fraction. Where detailed, the authenticity of the viral RNA was examined by nuclease digestion. 500 ng of viral RNA or plasmid DNA was treated with DNase I (10U, Roche), XrnI+RppH (1U XrnI + 5U RppH, both from NEB) or RNase cocktail (0.5U RNase A + 20U RNase T1, ThermoFisher) at 37 ℃ for 10 minutes. Then analysed on 1% agarose gel.</p></div><div class="section" id="sec-36"><h3>Preparation of BV2 S10 cytoplasmic extracts</h3><p id="p-67">Preparation of BV-2 S10 extracts was based on a previously published protocol (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Rakotondrafara and Hentze, 2011</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">2006</a>). BV-2 cells were harvested, washed with PBS, and lysed with 1x packed volume of hypotonic buffer containing 10 mM HEPES pH7.6, 10 mM potassium acetate, 0.5 mM magnesium acetate, 5 mM DTT, 1x protease inhibitors cocktail (EDTA-free, Roche). Cells were lysed on ice for 45 minutes, then passed through 25G and 27G needles to achieve &gt;95% lysis. Cell lysates were then centrifuged at 10,000 x g for 10 minutes at 4 ℃ twice and the supernatant collected. The total protein concentration was measured by Bradford assay and normalised to 20 mg/ml before freezing at −80 ℃ until use. For micrococcal nuclease treatment, S10 extracts were thawed on ice, 1 mM calcium chloride and 200 unit/ml final concentrations of micrococcal nuclease (NEB). Cell lysates were incubated at 25 ℃ for 15 minutes before adding 3 mM final concentration of EGTA was added.</p></div><div class="section" id="sec-37"><h3><em>In vitro</em> translation of BV2 S10 lysates</h3><p id="p-68"><em>In vitro</em> translation assays were set up based on a previous protocol (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Favre and Trepo, 2001</a>). Translation reactions were set up in 12.5 μl total volume containing 5 μl BV2 S10 lysate, 2.5 μl 5X translation buffer, 0.25 μl of 5 mg/ml creatine kinase, 1.25 μl RRL, 0.225 μl of 5 M potassium acetate, 0.25 mM of 100 mM magnesium acetate, 5.13 μCi <sup>35</sup>S-labelled methionine (PerkinElmer) and 10-100 ng/μl RNA as detailed in the text. 5X translation buffer contains 350 mM HEPES, 75 mM creatine phosphate, 10 mM ATP, 3.75 mM GTP, 100 μM amino acid minus methionine, 3.75 mM spermidine and 0.375 mM S-adenosyl-methionine. For control experiments using RRL (Promega), the reactions were set up according to manufacturer’s instructions using 0.5-1 ng/μl RNA. Reactions were incubated at 30 <sup>°</sup>C for 90 minutes before addition of 12.5 μl trans-stop buffer containing 10 mM EDTA and 0.1 mg/ml RNase A and incubated at room temperature for 10 minutes, then 25 μl 5X loading buffer was added to the reaction and heated at 95 <sup>°</sup>C for 5 minutes. 10 μl lysates were resolved in 15% SDS-PAGE and exposed to a phosphorimager screen and visualised using a TyphoonFLA7000 machine. For non-radioactive translation, 1.25 μl of 1 mM methionine was used instead of <sup>35</sup>S-labelled methionine, and the reactions were stopped with 100 μl 1x passive lysis buffer (Promega) and the luminescence read using a GloMax luminometer (Promega).</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-45">The authors would like to thank Skip Virgin (Washington University in St. Louis) for intellectual input and the provision of reagents and resources. Kate Heesom of the University of Bristol Proteomics facility for support with the mass spectrometry analysis and Nerea Irigoyen (University of Cambridge) for help with polysome profiling.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Alam</span>,  <span class="cit-name-given-names">U.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kennedy</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Rasputin a decade on and more promiscuous than ever? A review of G3BPs</span>. <abbr class="cit-jnl-abbrev">Biochim Biophys Acta Mol Cell Res</abbr> <span class="cit-vol">1866</span>, <span class="cit-fpage">360</span>–<span class="cit-lpage">370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochim%2BBiophys%2BActa%2BMol%2BCell%2BRes%26rft.volume%253D1866%26rft.spage%253D360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.2" data-doi="10.1016/j.tcb.2016.09.011"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Altan-Bonnet</span>,  <span class="cit-name-given-names">N.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Lipid Tales of Viral Replication and Transmission</span>. <abbr class="cit-jnl-abbrev">Trends Cell Biol.</abbr> <span class="cit-vol">27</span>, <span class="cit-fpage">201</span>–<span class="cit-lpage">213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BCell%2BBiol.%26rft.volume%253D27%26rft.spage%253D201%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tcb.2016.09.011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tcb.2016.09.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.3" data-doi="10.1371/journal.ppat.1002413"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bailey</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carrara</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Benson</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chaudhry</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heeney</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yarovinsky</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simmonds</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goodfellow</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McFadden</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shortland</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Norovirus regulation of the innate immune response and apoptosis occurs via the product of the alternative open reading frame 4</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>. <span class="cit-vol">7</span>, <span class="cit-fpage">e1002413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DMcFadden%26rft.auinit1%253DN.%26rft.volume%253D7%26rft.issue%253D12%26rft.spage%253De1002413%26rft.epage%253De1002413%26rft.atitle%253DNorovirus%2Bregulation%2Bof%2Bthe%2Binnate%2Bimmune%2Bresponse%2Band%2Bapoptosis%2Boccurs%2Bvia%2Bthe%2Bproduct%2Bof%2Bthe%2Balternative%2Bopen%2Breading%2Bframe%2B4.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002413%26rft_id%253Dinfo%253Apmid%252F22174679%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002413&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22174679&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F08%2F571455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.4" data-doi="10.1128/JVI.02053-09"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bailey</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Karakasiliotis</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vashist</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chung</span>,  <span class="cit-name-given-names">L.M.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rees</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reese</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McFadden</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Benson</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yarovinsky</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simmonds</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Functional analysis of RNA structures present at the 3’ extremity of the murine norovirus genome: the variable polypyrimidine tract plays a role in viral virulence</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">84</span>, <span class="cit-fpage">2859</span>–<span class="cit-lpage">2870</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02053-09%26rft_id%253Dinfo%253Apmid%252F20053745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg0LzYvMjg1OSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAzLzA4LzU3MTQ1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.5" data-doi="10.1371/journal.pone.0151219"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bartsch</span>,  <span class="cit-name-given-names">S.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lopman</span>,  <span class="cit-name-given-names">B.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ozawa</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hall</span>,  <span class="cit-name-given-names">A.J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">B.Y.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Global Economic Burden of Norovirus Gastroenteritis</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">e0151219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D11%26rft.spage%253De0151219%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0151219%26rft_id%253Dinfo%253Apmid%252F27115736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0151219&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27115736&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F08%2F571455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.6" data-doi="10.1016/0165-5728(90)90073-V"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Blasi</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barluzzi</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bocchini</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mazzolla</span>,  <span class="cit-name-given-names">R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Bistoni</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">1990</span>). <span class="cit-article-title">Immortalization of murine microglial cells by a v-raf/v-myc carrying retrovirus</span>. <abbr class="cit-jnl-abbrev">J. Neuroimmunol</abbr>. <span class="cit-vol">27</span>, <span class="cit-fpage">229</span>–<span class="cit-lpage">237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bneuroimmunology%26rft.stitle%253DJ%2BNeuroimmunol%26rft.aulast%253DBlasi%26rft.auinit1%253DE.%26rft.volume%253D27%26rft.issue%253D2-3%26rft.spage%253D229%26rft.epage%253D237%26rft.atitle%253DImmortalization%2Bof%2Bmurine%2Bmicroglial%2Bcells%2Bby%2Ba%2Bv-raf%252Fv-myc%2Bcarrying%2Bretrovirus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0165-5728%252890%252990073-V%26rft_id%253Dinfo%253Apmid%252F2110186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0165-5728(90)90073-V&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2110186&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F08%2F571455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990DB67400016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brocard</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Iadevaia</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klein</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hall</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lewis</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burke</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Parker</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ruggieri</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goodfellow</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Norovirus infection results in assembly of virus-specific G3BP1 granules and evasion of eIF2α signaling</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-fpage">490318</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.8" data-doi="10.1016/j.virol.2006.06.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">K.-O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sosnovtsev</span>,  <span class="cit-name-given-names">S.V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Belliot</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">King</span>,  <span class="cit-name-given-names">A.D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Green</span>,  <span class="cit-name-given-names">K.Y.</span></span> (<span class="cit-pub-date">2006</span>). <span class="cit-article-title">Stable expression of a Norwalk virus RNA replicon in a human hepatoma cell line</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">353</span>, <span class="cit-fpage">463</span>–<span class="cit-lpage">473</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DChang%26rft.auinit1%253DK.%2BO.%26rft.volume%253D353%26rft.issue%253D2%26rft.spage%253D463%26rft.epage%253D473%26rft.atitle%253DStable%2Bexpression%2Bof%2Ba%2BNorwalk%2Bvirus%2BRNA%2Breplicon%2Bin%2Ba%2Bhuman%2Bhepatoma%2Bcell%2Bline.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2006.06.006%26rft_id%253Dinfo%253Apmid%252F16843517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2006.06.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16843517&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F08%2F571455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240930900021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.9" data-doi="10.1074/jbc.M602230200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chaudhry</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nayak</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bordeleau</span>,  <span class="cit-name-given-names">M.-E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tanaka</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pelletier</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Belsham</span>,  <span class="cit-name-given-names">G.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roberts</span>,  <span class="cit-name-given-names">L.O.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Goodfellow</span>,  <span class="cit-name-given-names">I.G.</span></span> (<span class="cit-pub-date">2006</span>). <span class="cit-article-title">Caliciviruses differ in their functional requirements for eIF4F components</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem</abbr>. <span class="cit-vol">281</span>, <span class="cit-fpage">25315</span>–<span class="cit-lpage">25325</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M602230200%26rft_id%253Dinfo%253Apmid%252F16835235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODEvMzUvMjUzMTUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wMy8wOC81NzE0NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.10" data-doi="10.1099/vir.0.82940-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chaudhry</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Skinner</span>,  <span class="cit-name-given-names">M.A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Goodfellow</span>,  <span class="cit-name-given-names">I.G.</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">Recovery of genetically defined murine norovirus in tissue culture by using a fowlpox virus expressing T7 RNA polymerase</span>. <abbr class="cit-jnl-abbrev">J. Gen. Virol</abbr>. <span class="cit-vol">88</span>, <span class="cit-fpage">2091</span>–<span class="cit-lpage">2100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DChaudhry%26rft.auinit1%253DY.%26rft.volume%253D88%26rft.issue%253D8%26rft.spage%253D2091%26rft.epage%253D2100%26rft.atitle%253DRecovery%2Bof%2Bgenetically%2Bdefined%2Bmurine%2Bnorovirus%2Bin%2Btissue%2Bculture%2Bby%2Busing%2Ba%2Bfowlpox%2Bvirus%2Bexpressing%2BT7%2BRNA%2Bpolymerase%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.82940-0%26rft_id%253Dinfo%253Apmid%252F17622609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.82940-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17622609&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F08%2F571455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248828600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.11" data-doi="10.1074/jbc.M114.550657"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chung</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bailey</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leen</span>,  <span class="cit-name-given-names">E.N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Emmott</span>,  <span class="cit-name-given-names">E.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chaudhry</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roberts</span>,  <span class="cit-name-given-names">L.O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Curry</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Locker</span>,  <span class="cit-name-given-names">N.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Goodfellow</span>,  <span class="cit-name-given-names">I.G.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Norovirus translation requires an interaction between the C Terminus of the genome-linked viral protein VPg and eukaryotic translation initiation factor 4G</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem</abbr>. <span class="cit-vol">289</span>, <span class="cit-fpage">21738</span>–<span class="cit-lpage">21750</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M114.550657%26rft_id%253Dinfo%253Apmid%252F24928504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODkvMzEvMjE3MzgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wMy8wOC81NzE0NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cotton</span>,  <span class="cit-name-given-names">B.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hyde</span>,  <span class="cit-name-given-names">J.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sarvestani</span>,  <span class="cit-name-given-names">S.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sosnovtsev</span>,  <span class="cit-name-given-names">S.V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Green</span>,  <span class="cit-name-given-names">K.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">White</span>,  <span class="cit-name-given-names">P.A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Mackenzie</span>,  <span class="cit-name-given-names">J.M.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">The Norovirus NS3 Protein Is a Dynamic Lipid-and Microtubule-Associated Protein Involved in Viral RNA Replication</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">91</span>, <span class="cit-fpage">e02138</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft.volume%253D91%26rft.spage%253De02138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.13" data-doi="10.1128/JVI.01983-09"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cristea</span>,  <span class="cit-name-given-names">I.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rozjabek</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Molloy</span>,  <span class="cit-name-given-names">K.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Karki</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">White</span>,  <span class="cit-name-given-names">L.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rice</span>,  <span class="cit-name-given-names">C.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rout</span>,  <span class="cit-name-given-names">M.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chait</span>,  <span class="cit-name-given-names">B.T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">MacDonald</span>,  <span class="cit-name-given-names">M.R.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Host factors associated with the Sindbis virus RNA-dependent RNA polymerase: role for G3BP1 and G3BP2 in virus replication</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">84</span>, <span class="cit-fpage">6720</span>–<span class="cit-lpage">6732</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01983-09%26rft_id%253Dinfo%253Apmid%252F20392851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4NC8xMy82NzIwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDMvMDgvNTcxNDU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.14" data-doi="10.1038/nbt.3437"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Doench</span>,  <span class="cit-name-given-names">J.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fusi</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sullender</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hegde</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vaimberg</span>,  <span class="cit-name-given-names">E.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Donovan</span>,  <span class="cit-name-given-names">K.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tothova</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilen</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Orchard</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR-Cas9</span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol</abbr>. <span class="cit-vol">34</span>, <span class="cit-fpage">184</span>–<span class="cit-lpage">191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol%26rft.volume%253D34%26rft.spage%253D184%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3437%26rft_id%253Dinfo%253Apmid%252F26780180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3437&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26780180&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F08%2F571455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Emmott</span>,  <span class="cit-name-given-names">E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Goodfellow</span>,  <span class="cit-name-given-names">I.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Identification of protein interaction partners in mammalian cells using SILAC-immunoprecipitation quantitative proteomics</span>. <abbr class="cit-jnl-abbrev">J Vis Exp</abbr> <span class="cit-fpage">e51656</span>–<span class="cit-lpage">e51656</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Emmott</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Rougemont</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hosmillo</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fitzmaurice</span>,  <span class="cit-name-given-names">T.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haas</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Goodfellow</span>,  <span class="cit-name-given-names">I.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Polyprotein processing and intermolecular interactions within the viral replication complex spatially and temporally control norovirus protease activity</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> jbc.RA118.006780.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Emmott</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sorgeloos</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Caddy</span>,  <span class="cit-name-given-names">S.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vashist</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sosnovtsev</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lloyd</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heesom</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Locker</span>,  <span class="cit-name-given-names">N.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Goodfellow</span>,  <span class="cit-name-given-names">I.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Norovirus-mediated modification of the translational landscape via virus and host-induced cleavage of translation initiation factors</span>. <abbr class="cit-jnl-abbrev">Mol. Cell Proteomics</abbr> mcp.M116.062448.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.18" data-doi="10.1126/science.aaf5211"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ettayebi</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crawford</span>,  <span class="cit-name-given-names">S.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murakami</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Broughman</span>,  <span class="cit-name-given-names">J.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Karandikar</span>,  <span class="cit-name-given-names">U.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tenge</span>,  <span class="cit-name-given-names">V.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neill</span>,  <span class="cit-name-given-names">F.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blutt</span>,  <span class="cit-name-given-names">S.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zeng</span>,  <span class="cit-name-given-names">X.-L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qu</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Replication of human noroviruses in stem cell-derived human enteroids</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">353</span>, <span class="cit-fpage">1387</span>–<span class="cit-lpage">1393</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaf5211%26rft_id%253Dinfo%253Apmid%252F27562956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNTMvNjMwNi8xMzg3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDMvMDgvNTcxNDU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Favre</span>,  <span class="cit-name-given-names">D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Trepo</span>,  <span class="cit-name-given-names">C.</span></span> (<span class="cit-pub-date">2001</span>). <span class="cit-article-title">Translational extracts active biologically in vitro obtained from eukaryotic monolayer cells: a versatile method for viral RNA studies</span>. <abbr class="cit-jnl-abbrev">Virol. Methods</abbr> <span class="cit-vol">92</span>, <span class="cit-fpage">177</span>–<span class="cit-lpage">181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirol.%2BMethods%26rft.volume%253D92%26rft.spage%253D177%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.20" data-doi="10.1099/vir.0.042499-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Firth</span>,  <span class="cit-name-given-names">A.E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Brierley</span>,  <span class="cit-name-given-names">I.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Non-canonical translation in RNA viruses</span>. <abbr class="cit-jnl-abbrev">J. Gen. Virol</abbr>. <span class="cit-vol">93</span>, <span class="cit-fpage">1385</span>–<span class="cit-lpage">1409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGen.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.042499-0%26rft_id%253Dinfo%253Apmid%252F22535777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.042499-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22535777&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F08%2F571455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306348900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fritzlar</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aktepe</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chao</span>,  <span class="cit-name-given-names">Y.-W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McAllaster</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilen</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">White</span>,  <span class="cit-name-given-names">P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Mackenzie</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Mouse Norovirus infection arrests host cell translation uncoupled from the stress granule-PKR-eIF2α axis</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-fpage">536052</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.22" data-doi="10.1016/j.coviro.2011.09.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goodfellow</span>,  <span class="cit-name-given-names">I.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">The genome-linked protein VPg of vertebrate viruses — a multifaceted protein</span>. <abbr class="cit-jnl-abbrev">Curr Opin Virol</abbr> <span class="cit-vol">1</span>, <span class="cit-fpage">355</span>–<span class="cit-lpage">362</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCurr%2BOpin%2BVirol%26rft.aulast%253DGoodfellow%26rft.auinit1%253DI.%26rft.volume%253D1%26rft.issue%253D5%26rft.spage%253D355%26rft.epage%253D362%26rft.atitle%253DThe%2Bgenome-linked%2Bprotein%2BVPg%2Bof%2Bvertebrate%2Bviruses%2B-%2Ba%2Bmultifaceted%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.coviro.2011.09.003%26rft_id%253Dinfo%253Apmid%252F22440837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.coviro.2011.09.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22440837&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F08%2F571455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.23" data-doi="10.1038/sj.embor.7400510"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goodfellow</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chaudhry</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gioldasi</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gerondopoulos</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Natoni</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Labrie</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Laliberté</span>,  <span class="cit-name-given-names">J.-F.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Roberts</span>,  <span class="cit-name-given-names">L.</span></span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">Calicivirus translation initiation requires an interaction between VPg and eIF 4 E</span>. <abbr class="cit-jnl-abbrev">EMBO Rep</abbr>. <span class="cit-vol">6</span>, <span class="cit-fpage">968</span>–<span class="cit-lpage">972</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BRep%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.embor.7400510%26rft_id%253Dinfo%253Apmid%252F16142217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiZW1ib3IiO3M6NToicmVzaWQiO3M6ODoiNi8xMC85NjgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wMy8wOC81NzE0NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.24" data-doi="10.1073/pnas.1605575113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Haga</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fujimoto</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Takai-Todaka</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miki</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Doan</span>,  <span class="cit-name-given-names">Y.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murakami</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yokoyama</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murata</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nakanishi</span>,  <span class="cit-name-given-names">A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Katayama</span>,  <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Functional receptor molecules CD300lf and CD300ld within the CD300 family enable murine noroviruses to infect cells</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U.S.a.</abbr> <span class="cit-vol">113</span>, <span class="cit-fpage">E6248</span>–<span class="cit-lpage">E6255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.S.a.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1605575113%26rft_id%253Dinfo%253Apmid%252F27681626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEzLzQxL0U2MjQ4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDMvMDgvNTcxNDU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.25" data-doi="10.1128/JVI.01650-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hosmillo</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chaudhry</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">D.-S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goodfellow</span>,  <span class="cit-name-given-names">I.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Cho</span>,  <span class="cit-name-given-names">K.-O.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Sapovirus translation requires an interaction between VPg and the cap binding protein eIF4E</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">88</span>, <span class="cit-fpage">12213</span>–<span class="cit-lpage">12221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01650-14%26rft_id%253Dinfo%253Apmid%252F25142584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4OC8yMS8xMjIxMyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAzLzA4LzU3MTQ1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.26" data-doi="10.1128/JVI.00647-16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Humoud</span>,  <span class="cit-name-given-names">M.N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Doyle</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Royall</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Willcocks</span>,  <span class="cit-name-given-names">M.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sorgeloos</span>,  <span class="cit-name-given-names">F.</span></span>, van <span class="cit-auth"><span class="cit-name-surname">Kuppeveld</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roberts</span>,  <span class="cit-name-given-names">L.O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goodfellow</span>,  <span class="cit-name-given-names">I.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Langereis</span>,  <span class="cit-name-given-names">M.A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Locker</span>,  <span class="cit-name-given-names">N.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Feline Calicivirus Infection Disrupts Assembly of Cytoplasmic Stress Granules and Induces G3BP1 Cleavage</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">90</span>, <span class="cit-fpage">6489</span>–<span class="cit-lpage">6501</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00647-16%26rft_id%253Dinfo%253Apmid%252F27147742%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI5MC8xNC82NDg5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDMvMDgvNTcxNDU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hyde</span>,  <span class="cit-name-given-names">J.L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Mackenzie</span>,  <span class="cit-name-given-names">J.M.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Subcellular localization of the MNV-1 ORF1 proteins and their potential roles in the formation of the MNV-1 replication complex</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">406</span>, <span class="cit-fpage">138</span>–<span class="cit-lpage">148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D406%26rft.spage%253D138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.28" data-doi="10.1128/JVI.00600-09"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hyde</span>,  <span class="cit-name-given-names">J.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sosnovtsev</span>,  <span class="cit-name-given-names">S.V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Green</span>,  <span class="cit-name-given-names">K.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wobus</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Virgin</span>,  <span class="cit-name-given-names">H.W.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Mackenzie</span>,  <span class="cit-name-given-names">J.M.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Mouse norovirus replication is associated with virus-induced vesicle clusters originating from membranes derived from the secretory pathway</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">83</span>, <span class="cit-fpage">9709</span>–<span class="cit-lpage">9719</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00600-09%26rft_id%253Dinfo%253Apmid%252F19587041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4My8xOS85NzA5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDMvMDgvNTcxNDU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jaafar</span>,  <span class="cit-name-given-names">Z.A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kieft</span>,  <span class="cit-name-given-names">J.S.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Viral RNA structure-based strategies to manipulate translation</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol.</abbr> <span class="cit-vol">17</span>, <span class="cit-fpage">110</span>–<span class="cit-lpage">123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BMicrobiol.%26rft.volume%253D17%26rft.spage%253D110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.30" data-doi="10.1126/science.1257147"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">M.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Watanabe</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graves</span>,  <span class="cit-name-given-names">C.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keyes</span>,  <span class="cit-name-given-names">L.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grau</span>,  <span class="cit-name-given-names">K.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gonzalez-Hernandez</span>,  <span class="cit-name-given-names">M.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Iovine</span>,  <span class="cit-name-given-names">N.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wobus</span>,  <span class="cit-name-given-names">C.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vinjé</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Enteric bacteria promote human and mouse norovirus infection of B cells</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">346</span>, <span class="cit-fpage">755</span>–<span class="cit-lpage">759</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DJones%26rft.auinit1%253DM.%2BK.%26rft.volume%253D346%26rft.issue%253D6210%26rft.spage%253D755%26rft.epage%253D759%26rft.atitle%253DEnteric%2Bbacteria%2Bpromote%2Bhuman%2Band%2Bmouse%2Bnorovirus%2Binfection%2Bof%2BB%2Bcells%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1257147%26rft_id%253Dinfo%253Apmid%252F25378626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDYvNjIxMC83NTUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wMy8wOC81NzE0NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.31" data-doi="10.1126/science.1077905"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Karst</span>,  <span class="cit-name-given-names">S.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wobus</span>,  <span class="cit-name-given-names">C.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lay</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davidson</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Virgin</span>,  <span class="cit-name-given-names">H.W.</span></span> (<span class="cit-pub-date">2003</span>). <span class="cit-article-title">STAT1-dependent innate immunity to a Norwalk-like virus</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">299</span>, <span class="cit-fpage">1575</span>–<span class="cit-lpage">1578</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKarst%26rft.auinit1%253DS.%2BM.%26rft.volume%253D299%26rft.issue%253D5612%26rft.spage%253D1575%26rft.epage%253D1578%26rft.atitle%253DSTAT1-Dependent%2BInnate%2BImmunity%2Bto%2Ba%2BNorwalk-Like%2BVirus%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1077905%26rft_id%253Dinfo%253Apmid%252F12624267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTkvNTYxMi8xNTc1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDMvMDgvNTcxNDU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.32" data-doi="10.1083/jcb.201508028"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kedersha</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Panas</span>,  <span class="cit-name-given-names">M.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Achorn</span>,  <span class="cit-name-given-names">C.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lyons</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tisdale</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hickman</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thomas</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lieberman</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McInerney</span>,  <span class="cit-name-given-names">G.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ivanov</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">G3BP-Caprin1-USP10 complexes mediate stress granule condensation and associate with 40S subunits</span>. <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">212</span>, <span class="cit-fpage">845</span>–<span class="cit-lpage">860</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.201508028%26rft_id%253Dinfo%253Apmid%252F27022092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjIxMi83Lzg0NSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAzLzA4LzU3MTQ1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.33" data-doi="10.1371/journal.ppat.1005810"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">D.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reynaud</span>,  <span class="cit-name-given-names">J.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rasalouskaya</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Akhrymuk</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mobley</span>,  <span class="cit-name-given-names">J.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frolov</span>,  <span class="cit-name-given-names">I.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Frolova</span>,  <span class="cit-name-given-names">E.I.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">New World and Old World Alphaviruses Have Evolved to Exploit Different Components of Stress Granules, FXR and G3BP Proteins, for Assembly of Viral Replication Complexes</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">e1005810</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D12%26rft.spage%253De1005810%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1005810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1005810&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kitano</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hosmillo</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Emmott</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Goodfellow</span>,  <span class="cit-name-given-names">I.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Selection and Characterization of Rupintrivir-Resistant Norwalk Virus Replicon Cells In Vitro</span>. <abbr class="cit-jnl-abbrev">Antimicrob. Agents Chemother.</abbr> <span class="cit-vol">62</span>, <span class="cit-fpage">725</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntimicrob.%2BAgents%2BChemother.%26rft.volume%253D62%26rft.spage%253D725%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.35" data-doi="10.1101/gad.282418.116"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kumar</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hellen</span>,  <span class="cit-name-given-names">C.U.T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Pestova</span>,  <span class="cit-name-given-names">T.V.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Toward the mechanism of eIF4F-mediated ribosomal attachment to mammalian capped mRNAs</span>. <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">30</span>, <span class="cit-fpage">1573</span>–<span class="cit-lpage">1588</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DKumar%26rft.auinit1%253DP.%26rft.volume%253D30%26rft.issue%253D13%26rft.spage%253D1573%26rft.epage%253D1588%26rft.atitle%253DToward%2Bthe%2Bmechanism%2Bof%2BeIF4F-mediated%2Bribosomal%2Battachment%2Bto%2Bmammalian%2Bcapped%2BmRNAs%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.282418.116%26rft_id%253Dinfo%253Apmid%252F27401559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjMwLzEzLzE1NzMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wMy8wOC81NzE0NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.36" data-doi="10.1371/journal.ppat.1005379"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Leen</span>,  <span class="cit-name-given-names">E.N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sorgeloos</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Correia</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chaudhry</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cannac</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pastore</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>,  <span class="cit-name-given-names">S.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matthews</span>,  <span class="cit-name-given-names">S.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goodfellow</span>,  <span class="cit-name-given-names">I.G.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">A Conserved Interaction between a C-Terminal Motif in Norovirus VPg and the HEAT-1 Domain of eIF4G Is Essential for Translation Initiation</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">e1005379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D12%26rft.spage%253De1005379%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1005379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1005379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">T.-F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hosmillo</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schwanke</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shu</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yin</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Curry</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goodfellow</span>,  <span class="cit-name-given-names">I.G.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">X.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Human Norovirus NS3 Has RNA Helicase and Chaperoning Activities</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">92</span>, <span class="cit-fpage">312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft.volume%253D92%26rft.spage%253D312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.38" data-doi="10.1128/JVI.03433-12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">López-Manríquez</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vashist</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ureña</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goodfellow</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chavez</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mora-Heredia</span>,  <span class="cit-name-given-names">J.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cancio-Lonches</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garrido</span>,  <span class="cit-name-given-names">E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Gutiérrez-Escolano</span>,  <span class="cit-name-given-names">A.L.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Norovirus Genome Circularisation and Efficient Replication is Facilitated by Binding of PCBP2 and hnRNP A1</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">87</span>, <span class="cit-fpage">11371</span>–<span class="cit-lpage">11387</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.03433-12%26rft_id%253Dinfo%253Apmid%252F23946460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4Ny8yMS8xMTM3MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAzLzA4LzU3MTQ1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.39" data-doi="10.1038/nri.2017.63"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McCormick</span>,  <span class="cit-name-given-names">C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Khaperskyy</span>,  <span class="cit-name-given-names">D.A.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Translation inhibition and stress granules in the antiviral immune response</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Immunol.</abbr> <span class="cit-vol">17</span>, <span class="cit-fpage">647</span>–<span class="cit-lpage">660</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BImmunol.%26rft.volume%253D17%26rft.spage%253D647%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnri.2017.63%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nri.2017.63&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.40" data-doi="10.1128/mBio.00668-17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McCune</span>,  <span class="cit-name-given-names">B.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eaglesham</span>,  <span class="cit-name-given-names">J.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thorne</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mayer</span>,  <span class="cit-name-given-names">A.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Condiff</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nice</span>,  <span class="cit-name-given-names">T.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goodfellow</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krezel</span>,  <span class="cit-name-given-names">A.M.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Noroviruses Co-opt the Function of Host Proteins VAPA and VAPB for Replication via a Phenylalanine-Phenylalanine-Acidic-Tract-Motif Mimic in Nonstructural Viral Protein NS1/2</span>. <abbr class="cit-jnl-abbrev">MBio</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">e00668</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio%26rft.volume%253D8%26rft.spage%253De00668%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.00668-17%26rft_id%253Dinfo%253Apmid%252F28698274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/mBio.00668-17&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28698274&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F08%2F571455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nice</span>,  <span class="cit-name-given-names">T.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Strong</span>,  <span class="cit-name-given-names">D.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McCune</span>,  <span class="cit-name-given-names">B.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pohl</span>,  <span class="cit-name-given-names">C.S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Virgin</span>,  <span class="cit-name-given-names">H.W.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">A single amino acid change in murine norovirus NS1/2 is sufficient for colonic tropism and persistence</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.42" data-doi="10.1126/science.aaf1220"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Orchard</span>,  <span class="cit-name-given-names">R.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilen</span>,  <span class="cit-name-given-names">C.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Doench</span>,  <span class="cit-name-given-names">J.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baldridge</span>,  <span class="cit-name-given-names">M.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McCune</span>,  <span class="cit-name-given-names">B.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">Y.-C.</span></span>J., <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pruett-Miller</span>,  <span class="cit-name-given-names">S.</span></span>M., <span class="cit-auth"><span class="cit-name-surname">Nelson</span>,  <span class="cit-name-given-names">C.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fremont</span>,  <span class="cit-name-given-names">D.H.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Discovery of a proteinaceous cellular receptor for a norovirus</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">353</span>, <span class="cit-fpage">933</span>–<span class="cit-lpage">936</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaf1220%26rft_id%253Dinfo%253Apmid%252F27540007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNTMvNjMwMi85MzMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wMy8wOC81NzE0NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.43" data-doi="10.1128/JVI.00439-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Panas</span>,  <span class="cit-name-given-names">M.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ahola</span>,  <span class="cit-name-given-names">T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">McInerney</span>,  <span class="cit-name-given-names">G.M.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">The C-terminal repeat domains of nsP3 from the Old World alphaviruses bind directly to G3BP</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">88</span>, <span class="cit-fpage">5888</span>–<span class="cit-lpage">5893</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00439-14%26rft_id%253Dinfo%253Apmid%252F24623412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4OC8xMC81ODg4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDMvMDgvNTcxNDU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.44" data-doi="10.1371/journal.ppat.1004659"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Panas</span>,  <span class="cit-name-given-names">M.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schulte</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thaa</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sandalova</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kedersha</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Achour</span>,  <span class="cit-name-given-names">A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">McInerney</span>,  <span class="cit-name-given-names">G.M.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Viral and cellular proteins containing FGDF motifs bind G3BP to block stress granule formation</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">e1004659</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D11%26rft.spage%253De1004659%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1004659%26rft_id%253Dinfo%253Apmid%252F25658430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1004659&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25658430&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F08%2F571455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.45" data-doi="10.1091/mbc.E12-08-0619"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Panas</span>,  <span class="cit-name-given-names">M.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Varjak</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lulla</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eng</span>,  <span class="cit-name-given-names">K.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Merits</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Karlsson Hedestam</span>,  <span class="cit-name-given-names">G.B.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">McInerney</span>,  <span class="cit-name-given-names">G.M.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Sequestration of G3BP coupled with efficient translation inhibits stress granules in Semliki Forest virus infection</span>. <abbr class="cit-jnl-abbrev">Mol Biol Cell</abbr> <span class="cit-vol">23</span>, <span class="cit-fpage">4701</span>–<span class="cit-lpage">4712</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E12-08-0619%26rft_id%253Dinfo%253Apmid%252F23087212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyMy8yNC80NzAxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDMvMDgvNTcxNDU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.46" data-doi="10.1016/j.tcb.2016.05.004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Protter</span>,  <span class="cit-name-given-names">D.S.W.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Parker</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Principles and Properties of Stress Granules</span>. <abbr class="cit-jnl-abbrev">Trends Cell Biol.</abbr> <span class="cit-vol">26</span>, <span class="cit-fpage">668</span>–<span class="cit-lpage">679</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BCell%2BBiol.%26rft.volume%253D26%26rft.spage%253D668%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tcb.2016.05.004%26rft_id%253Dinfo%253Apmid%252F27289443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tcb.2016.05.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27289443&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F08%2F571455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.47" data-doi="10.1038/nprot.2011.314"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rakotondrafara</span>,  <span class="cit-name-given-names">A.M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Hentze</span>,  <span class="cit-name-given-names">M.W.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">An efficient factor-depleted mammalian in vitro translation system</span>. <abbr class="cit-jnl-abbrev">Nat Protoc</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">563</span>–<span class="cit-lpage">571</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DRakotondrafara%26rft.auinit1%253DA.%2BM.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253D563%26rft.epage%253D571%26rft.atitle%253DAn%2Befficient%2Bfactor-depleted%2Bmammalian%2Bin%2Bvitro%2Btranslation%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2011.314%26rft_id%253Dinfo%253Apmid%252F21527914%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2011.314&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21527914&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F08%2F571455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.48" data-doi="10.1016/j.bbrc.2012.10.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rocha-Pereira</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jochmans</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dallmeier</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leyssen</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cunha</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Costa</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nascimento</span>,  <span class="cit-name-given-names">M.S.J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Neyts</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Inhibition of norovirus replication by the nucleoside analogue 2’-C-methylcytidine</span>. <abbr class="cit-jnl-abbrev">Biochem. Biophys. Res. Commun.</abbr> <span class="cit-vol">427</span>, <span class="cit-fpage">796</span>–<span class="cit-lpage">800</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2Band%2Bbiophysical%2Bresearch%2Bcommunications%26rft.stitle%253DBiochem%2BBiophys%2BRes%2BCommun%26rft.aulast%253DRocha-Pereira%26rft.auinit1%253DJ.%26rft.volume%253D427%26rft.issue%253D4%26rft.spage%253D796%26rft.epage%253D800%26rft.atitle%253DInhibition%2Bof%2Bnorovirus%2Breplication%2Bby%2Bthe%2Bnucleoside%2Banalogue%2B2%2527-C-methylcytidine.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbrc.2012.10.003%26rft_id%253Dinfo%253Apmid%252F23063849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbrc.2012.10.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23063849&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F08%2F571455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.49" data-doi="10.1128/JVI.02064-13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rocha-Pereira</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jochmans</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Debing</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Verbeken</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nascimento</span>,  <span class="cit-name-given-names">M.S.J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Neyts</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">The viral polymerase inhibitor 2’-C-methylcytidine inhibits Norwalk virus replication and protects against norovirus-induced diarrhea and mortality in a mouse model</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">87</span>, <span class="cit-fpage">11798</span>–<span class="cit-lpage">11805</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02064-13%26rft_id%253Dinfo%253Apmid%252F23986582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4Ny8yMS8xMTc5OCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAzLzA4LzU3MTQ1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.50" data-doi="10.1038/nmeth.3047"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sanjana</span>,  <span class="cit-name-given-names">N.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shalem</span>,  <span class="cit-name-given-names">O.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Improved vectors and genome-wide libraries for CRISPR screening</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">783</span>–<span class="cit-lpage">784</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D11%26rft.spage%253D783%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3047%26rft_id%253Dinfo%253Apmid%252F25075903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3047&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25075903&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F08%2F571455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340075600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.51" data-doi="10.1101/gr.1239303"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shannon</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Markiel</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ozier</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baliga</span>,  <span class="cit-name-given-names">N.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">J.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ramage</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Amin</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schwikowski</span>,  <span class="cit-name-given-names">B.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ideker</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2003</span>). <span class="cit-article-title">Cytoscape: a software environment for integrated models of biomolecular interaction networks</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>, <span class="cit-fpage">2498</span>–<span class="cit-lpage">2504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DShannon%26rft.auinit1%253DP.%26rft.volume%253D13%26rft.issue%253D11%26rft.spage%253D2498%26rft.epage%253D2504%26rft.atitle%253DCytoscape%253A%2BA%2BSoftware%2BEnvironment%2Bfor%2BIntegrated%2BModels%2Bof%2BBiomolecular%2BInteraction%2BNetworks%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1239303%26rft_id%253Dinfo%253Apmid%252F14597658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMy8xMS8yNDk4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDMvMDgvNTcxNDU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.52" data-doi="10.1128/JVI.00783-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thackray</span>,  <span class="cit-name-given-names">L.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wobus</span>,  <span class="cit-name-given-names">C.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chachu</span>,  <span class="cit-name-given-names">K.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alegre</span>,  <span class="cit-name-given-names">E.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Henderson</span>,  <span class="cit-name-given-names">K.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kelley</span>,  <span class="cit-name-given-names">S.T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Virgin</span>,  <span class="cit-name-given-names">H.W.</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">Murine noroviruses comprising a single genogroup exhibit biological diversity despite limited sequence divergence</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">81</span>, <span class="cit-fpage">10460</span>–<span class="cit-lpage">10473</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00783-07%26rft_id%253Dinfo%253Apmid%252F17652401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4MS8xOS8xMDQ2MCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAzLzA4LzU3MTQ1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.53" data-doi="10.1016/j.molcel.2013.05.021"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thandapani</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Connor</span>,  <span class="cit-name-given-names">T.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bailey</span>,  <span class="cit-name-given-names">T.L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Richard</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Defining the RGG/RG motif</span>. <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">50</span>, <span class="cit-fpage">613</span>–<span class="cit-lpage">623</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D50%26rft.spage%253D613%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2013.05.021%26rft_id%253Dinfo%253Apmid%252F23746349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2013.05.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23746349&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F08%2F571455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000320678500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.54" data-doi="10.1099/vir.0.059634-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thorne</span>,  <span class="cit-name-given-names">L.G.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Goodfellow</span>,  <span class="cit-name-given-names">I.G.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Norovirus gene expression and replication</span>. <abbr class="cit-jnl-abbrev">J. Gen. Virol.</abbr> <span class="cit-vol">95</span>, <span class="cit-fpage">278</span>–<span class="cit-lpage">291</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGen.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.059634-0%26rft_id%253Dinfo%253Apmid%252F24243731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.059634-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24243731&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F08%2F571455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000338177400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.55" data-doi="10.1128/JVI.00439-12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thorne</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bailey</span>,  <span class="cit-name-given-names">D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Goodfellow</span>,  <span class="cit-name-given-names">I.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">High-Resolution functional profiling of the norovirus genome</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">86</span>, <span class="cit-fpage">11441</span>–<span class="cit-lpage">11456</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00439-12%26rft_id%253Dinfo%253Apmid%252F22915807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4Ni8yMS8xMTQ0MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAzLzA4LzU3MTQ1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.56"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">V’kovski</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gerber</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kelly</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pfaender</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ebert</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Braga Lagache</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simillion</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Portmann</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stalder</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gaschen</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Determination of host proteins composing the microenvironment of coronavirus replicase complexes by proximity-labeling</span>. <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElife%26rft.volume%253D8%26rft.spage%253D25%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.57"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van Beek</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van der Eijk</span>,  <span class="cit-name-given-names">A.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fraaij</span>,  <span class="cit-name-given-names">P.L.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Caliskan</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cransberg</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dalinghaus</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoek</span>,  <span class="cit-name-given-names">R.A.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Metselaar</span>,  <span class="cit-name-given-names">H.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roodnat</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vennema</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Chronic norovirus infection among solid organ recipients in a tertiary care hospital, the Netherlands, 2006-2014</span>. <abbr class="cit-jnl-abbrev">Clin. Microbiol. Infect.</abbr></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.58"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Van</span>,  <span class="cit-name-given-names">J.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ny</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Conceição-Neto</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maes</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hosmillo</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cuvry</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goodfellow</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">de</span>,  <span class="cit-name-given-names">T.A.N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Verbeken</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matthijnssens</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">A robust human norovirus replication model in zebrafish larvae: Supplementary data file</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-fpage">528364</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.59"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vashist</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ureña</span>,  <span class="cit-name-given-names">L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Goodfellow</span>,  <span class="cit-name-given-names">I.</span></span> (<span class="cit-pub-date">2012a</span>). <span class="cit-article-title">Development of a strand specific real-time RT-qPCR assay for the detection and quantitation of murine norovirus RNA</span>. <abbr class="cit-jnl-abbrev">J. Virol. Methods.</abbr></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.60" data-doi="10.1128/JVI.00432-12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vashist</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ureña</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chaudhry</span>,  <span class="cit-name-given-names">Y.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Goodfellow</span>,  <span class="cit-name-given-names">I.</span></span> (<span class="cit-pub-date">2012b</span>). <span class="cit-article-title">Identification of RNA-protein interaction networks involved in the norovirus life cycle</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">86</span>, <span class="cit-fpage">11977</span>–<span class="cit-lpage">11990</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00432-12%26rft_id%253Dinfo%253Apmid%252F22933270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4Ni8yMi8xMTk3NyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAzLzA4LzU3MTQ1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.61" data-doi="10.1074/jbc.M113.517011"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Villa</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Do</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hershey</span>,  <span class="cit-name-given-names">J.W.B.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Fraser</span>,  <span class="cit-name-given-names">C.S.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Human Eukaryotic Initiation Factor 4G (eIF4G) Protein Binds to eIF3c, -d, and -e to Promote mRNA Recruitment to the Ribosome</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">288</span>, <span class="cit-fpage">32932</span>–<span class="cit-lpage">32940</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M113.517011%26rft_id%253Dinfo%253Apmid%252F24092755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODgvNDYvMzI5MzIiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wMy8wOC81NzE0NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.62" data-doi="10.1101/cshperspect.a012351"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Walsh</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mathews</span>,  <span class="cit-name-given-names">M.B.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Mohr</span>,  <span class="cit-name-given-names">I.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Tinkering with translation: protein synthesis in virus-infected cells</span>. <abbr class="cit-jnl-abbrev">Cold Spring Harb Perspect Biol</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">a012351</span>–<span class="cit-lpage">a012351</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarb%2BPerspect%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fcshperspect.a012351%26rft_id%253Dinfo%253Apmid%252F23209131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6ImNzaHBlcnNwZWN0IjtzOjU6InJlc2lkIjtzOjExOiI1LzEvYTAxMjM1MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAzLzA4LzU3MTQ1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.63" data-doi="10.1016/j.chom.2007.08.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">White</span>,  <span class="cit-name-given-names">J.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cardenas</span>,  <span class="cit-name-given-names">A.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marissen</span>,  <span class="cit-name-given-names">W.E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Lloyd</span>,  <span class="cit-name-given-names">R.E.</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">Inhibition of cytoplasmic mRNA stress granule formation by a viral proteinase</span>. <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr> <span class="cit-vol">2</span>, <span class="cit-fpage">295</span>–<span class="cit-lpage">305</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCell%2BHost%2BMicrobe%26rft.aulast%253DWhite%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D2%26rft.issue%253D5%26rft.spage%253D295%26rft.epage%253D305%26rft.atitle%253DInhibition%2Bof%2Bcytoplasmic%2BmRNA%2Bstress%2Bgranule%2Bformation%2Bby%2Ba%2Bviral%2Bproteinase.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2007.08.006%26rft_id%253Dinfo%253Apmid%252F18005751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2007.08.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18005751&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F08%2F571455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251124300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.64" data-doi="10.1371/journal.pbio.0020432"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wobus</span>,  <span class="cit-name-given-names">C.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Karst</span>,  <span class="cit-name-given-names">S.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thackray</span>,  <span class="cit-name-given-names">L.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">K.-O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sosnovtsev</span>,  <span class="cit-name-given-names">S.V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Belliot</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krug</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mackenzie</span>,  <span class="cit-name-given-names">J.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Green</span>,  <span class="cit-name-given-names">K.Y.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Virgin</span>,  <span class="cit-name-given-names">H.W.</span></span> (<span class="cit-pub-date">2004</span>). <span class="cit-article-title">Replication of Norovirus in cell culture reveals a tropism for dendritic cells and macrophages</span>. <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">2</span>, <span class="cit-fpage">e432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DWobus%26rft.auinit1%253DC.%2BE.%26rft.volume%253D2%26rft.issue%253D12%26rft.spage%253De432%26rft.epage%253De432%26rft.atitle%253DReplication%2Bof%2BNorovirus%2Bin%2Bcell%2Bculture%2Breveals%2Ba%2Btropism%2Bfor%2Bdendritic%2Bcells%2Band%2Bmacrophages.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0020432%26rft_id%253Dinfo%253Apmid%252F15562321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0020432&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15562321&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F08%2F571455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.65" data-doi="10.1016/j.jviromet.2010.07.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yunus</span>,  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chung</span>,  <span class="cit-name-given-names">L.M.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chaudhry</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bailey</span>,  <span class="cit-name-given-names">D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Goodfellow</span>,  <span class="cit-name-given-names">I.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Development of an optimized RNA-based murine norovirus reverse genetics system</span>. <abbr class="cit-jnl-abbrev">J. Virol. Methods</abbr> <span class="cit-vol">169</span>, <span class="cit-fpage">112</span>–<span class="cit-lpage">118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirological%2Bmethods%26rft.stitle%253DJ%2BVirol%2BMethods%26rft.aulast%253DYunus%26rft.auinit1%253DA.%2BM.%26rft.volume%253D169%26rft.issue%253D1%26rft.spage%253D112%26rft.epage%253D118%26rft.atitle%253DDevelopment%2Bof%2Ban%2Boptimized%2BRNA-based%2Bmurine%2Bnorovirus%2Breverse%2Bgenetics%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jviromet.2010.07.006%26rft_id%253Dinfo%253Apmid%252F20637238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jviromet.2010.07.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20637238&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F08%2F571455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282398600017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-571455v1.66"><div class="cit-metadata"><cite>(<span class="cit-pub-date">2006</span>). <abbr class="cit-jnl-abbrev">Differential Cleavage of eIF4GI and eIF4GII in Mammalian Cells.</abbr> <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/571455v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F571455v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/571570v1" title="Nuclear Genome Size is Positively Correlated with Mean LTR Insertion Date in Fern and Lycophyte Genomes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 08, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/571455v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/571455v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/571455v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/571455v1" class="active" data-icon-position="" data-hide-link-title="0">Noroviruses subvert the core stress granule component G3BP1 to promote viral VPg-dependent translation</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/225768" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-O3cP3xmk0GbkTiIyqKtwc1DH9-PnMda007EDzvm1W7k" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="225768" id="node225768--2" data-pisa="biorxiv;571455v1" data-pisa-master="biorxiv;571455" data-seqnum="225768" data-apath="/biorxiv/early/2019/03/08/571455.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Noroviruses subvert the core stress granule component G3BP1 to promote viral VPg-dependent translation</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Myra</span> <span class="nlm-surname">Hosmillo</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Jia</span> <span class="nlm-surname">Lu</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Michael R.</span> <span class="nlm-surname">McAllaster</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">James B.</span> <span class="nlm-surname">Eaglesham</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Xinjie</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Edward</span> <span class="nlm-surname">Emmott</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Patricia</span> <span class="nlm-surname">Domingues</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Yasmin</span> <span class="nlm-surname">Chaudhry</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Timothy J</span> <span class="nlm-surname">Fitzmaurice</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Matthew K.H.</span> <span class="nlm-surname">Tung</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Marc</span> <span class="nlm-surname">Panas</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Gerald</span> <span class="nlm-surname">McInerney</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Nicholas</span> <span class="nlm-surname">Locker</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Craig B.</span> <span class="nlm-surname">Willen</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Ian</span> <span class="nlm-surname">Goodfellow</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">571455; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/571455 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/571455v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/571455v1&amp;title=Noroviruses%20subvert%20the%20core%20stress%20granule%20component%20G3BP1%20to%20promote%20viral%20VPg-dependent%20translation" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/571455v1&amp;title=Noroviruses%20subvert%20the%20core%20stress%20granule%20component%20G3BP1%20to%20promote%20viral%20VPg-dependent%20translation" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/571455v1&amp;text=Noroviruses%20subvert%20the%20core%20stress%20granule%20component%20G3BP1%20to%20promote%20viral%20VPg-dependent%20translation" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/571455v1&amp;title=Noroviruses%20subvert%20the%20core%20stress%20granule%20component%20G3BP1%20to%20promote%20viral%20VPg-dependent%20translation" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/571455v1&amp;t=Noroviruses%20subvert%20the%20core%20stress%20granule%20component%20G3BP1%20to%20promote%20viral%20VPg-dependent%20translation" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/571455v1&amp;title=Noroviruses%20subvert%20the%20core%20stress%20granule%20component%20G3BP1%20to%20promote%20viral%20VPg-dependent%20translation" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/571455v1&amp;title=Noroviruses%20subvert%20the%20core%20stress%20granule%20component%20G3BP1%20to%20promote%20viral%20VPg-dependent%20translation" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="225768" id="node225768--3" data-pisa="biorxiv;571455v1" data-pisa-master="biorxiv;571455" data-seqnum="225768" data-apath="/biorxiv/early/2019/03/08/571455.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Noroviruses subvert the core stress granule component G3BP1 to promote viral VPg-dependent translation</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Myra</span> <span class="nlm-surname">Hosmillo</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Jia</span> <span class="nlm-surname">Lu</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Michael R.</span> <span class="nlm-surname">McAllaster</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">James B.</span> <span class="nlm-surname">Eaglesham</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Xinjie</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Edward</span> <span class="nlm-surname">Emmott</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Patricia</span> <span class="nlm-surname">Domingues</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Yasmin</span> <span class="nlm-surname">Chaudhry</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Timothy J</span> <span class="nlm-surname">Fitzmaurice</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Matthew K.H.</span> <span class="nlm-surname">Tung</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Marc</span> <span class="nlm-surname">Panas</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Gerald</span> <span class="nlm-surname">McInerney</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Nicholas</span> <span class="nlm-surname">Locker</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Craig B.</span> <span class="nlm-surname">Willen</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Ian</span> <span class="nlm-surname">Goodfellow</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">571455; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/571455 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Noroviruses%20subvert%20the%20core%20stress%20granule%20component%20G3BP1%20to%20promote%20viral%20VPg-dependent%20translation&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Hosmillo%2C+Myra&amp;rft.au=Lu%2C+Jia&amp;rft.au=McAllaster%2C+Michael+R.&amp;rft.au=Eaglesham%2C+James+B.&amp;rft.au=Wang%2C+Xinjie&amp;rft.au=Emmott%2C+Edward&amp;rft.au=Domingues%2C+Patricia&amp;rft.au=Chaudhry%2C+Yasmin&amp;rft.au=Fitzmaurice%2C+Timothy+J&amp;rft.au=Tung%2C+Matthew+K.H.&amp;rft.au=Panas%2C+Marc&amp;rft.au=McInerney%2C+Gerald&amp;rft.au=Locker%2C+Nicholas&amp;rft.au=Willen%2C+Craig+B.&amp;rft.au=Goodfellow%2C+Ian"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/225768/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/225768/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/225768/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/225768/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/225768/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/225768/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/225768/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/225768/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/225768/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/225768/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/225768/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/225768/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/571455v1&amp;count=horizontal&amp;via=&amp;text=Noroviruses%20subvert%20the%20core%20stress%20granule%20component%20G3BP1%20to%20promote%20viral%20VPg-dependent%20translation&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/571455v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/571455v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/571455v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">Microbiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20277)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
