<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/05/14/565218.atom" />
<meta name="HW.pisa" content="biorxiv;565218v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A nematode-specific gene underlies bleomycin-response variation in Caenorhabditis elegans" />
<meta name="DC.Identifier" content="10.1101/565218" />
<meta name="DC.Date" content="2019-05-14" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Bleomycin is a powerful chemotherapeutic drug used to treat a variety of cancers. However, individual patients vary in their responses to bleomycin. The identification of genetic differences that underlie this response variation could improve treatment outcomes by tailoring bleomycin dosages to each patient. We used the model organism Caenorhabditis elegans to identify genetic determinants of bleomycin-response differences by performing linkage mapping on recombinants derived from a cross between the laboratory strain (N2) and a wild strain (CB4856). This approach identified a small genomic region on chromosome V that underlies bleomycin-response variation. Using near-isogenic lines and strains with CRISPR-Cas9 mediated deletions and allele replacements, we discovered that a novel nematode-specific gene ( scb-1 ) is required for bleomycin resistance. Although the mechanism by which this gene causes variation in bleomycin responses is unknown, we suggest that a rare variant present in the CB4856 strain might cause differences in the potential stress-response function of scb-1 between the N2 and CB4856 strains, thereby leading to differences in bleomycin resistance.

Article summary We performed linkage mapping on a panel of recombinant lines generated between two genetically divergent strains of Caenorhabditis elegans and identified a bleomycin-response QTL. We generated CRISPR-Cas9 deletions and reciprocal allele-replacement strains for all six candidate genes across the QTL confidence interval. Deletions of one gene, H19N07.3 , caused increased bleomycin sensitivity in both divergent genetic backgrounds. This gene might act in stress responses and detoxification in nematodes. We further compared our linkage mapping to a genome-wide association mapping and showed that a rare expression variant in the CB4856 strain likely underlies bleomycin-response differences." />
<meta name="DC.Contributor" content="Shannon C. Brady" />
<meta name="DC.Contributor" content="Stefan Zdraljevic" />
<meta name="DC.Contributor" content="Karol W. Bisaga" />
<meta name="DC.Contributor" content="Robyn E. Tanny" />
<meta name="DC.Contributor" content="Daniel E. Cook" />
<meta name="DC.Contributor" content="Daehan Lee" />
<meta name="DC.Contributor" content="Ye Wang" />
<meta name="DC.Contributor" content="Erik C. Andersen" />
<meta name="article:published_time" content="2019-05-14" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="A nematode-specific gene underlies bleomycin-response variation in Caenorhabditis elegans" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;ABSTRACT&lt;/h3&gt;
&lt;p&gt;Bleomycin is a powerful chemotherapeutic drug used to treat a variety of cancers. However, individual patients vary in their responses to bleomycin. The identification of genetic differences that underlie this response variation could improve treatment outcomes by tailoring bleomycin dosages to each patient. We used the model organism &lt;i&gt;Caenorhabditis elegans&lt;/i&gt; to identify genetic determinants of bleomycin-response differences by performing linkage mapping on recombinants derived from a cross between the laboratory strain (N2) and a wild strain (CB4856). This approach identified a small genomic region on chromosome V that underlies bleomycin-response variation. Using near-isogenic lines and strains with CRISPR-Cas9 mediated deletions and allele replacements, we discovered that a novel nematode-specific gene (&lt;i&gt;scb-1&lt;/i&gt;) is required for bleomycin resistance. Although the mechanism by which this gene causes variation in bleomycin responses is unknown, we suggest that a rare variant present in the CB4856 strain might cause differences in the potential stress-response function of &lt;i&gt;scb-1&lt;/i&gt; between the N2 and CB4856 strains, thereby leading to differences in bleomycin resistance.&lt;/p&gt;&lt;h3&gt;Article summary&lt;/h3&gt;
&lt;p&gt;We performed linkage mapping on a panel of recombinant lines generated between two genetically divergent strains of &lt;i&gt;Caenorhabditis elegans&lt;/i&gt; and identified a bleomycin-response QTL. We generated CRISPR-Cas9 deletions and reciprocal allele-replacement strains for all six candidate genes across the QTL confidence interval. Deletions of one gene, &lt;i&gt;H19N07.3&lt;/i&gt;, caused increased bleomycin sensitivity in both divergent genetic backgrounds. This gene might act in stress responses and detoxification in nematodes. We further compared our linkage mapping to a genome-wide association mapping and showed that a rare expression variant in the CB4856 strain likely underlies bleomycin-response differences.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;565218v2" />
<meta name="citation_id" content="565218v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/565218v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/565218v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/565218v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218.full.pdf" />
<meta name="citation_doi" content="10.1101/565218" />
<meta name="citation_num_pages" content="55" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="565218" />
<meta name="citation_author" content="Shannon C. Brady" />
<meta name="citation_author_institution" content="Molecular Biosciences, Northwestern University" />
<meta name="citation_author_institution" content="Interdisciplinary Biological Sciences Program, Northwestern University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3043-1544" />
<meta name="citation_author" content="Stefan Zdraljevic" />
<meta name="citation_author_institution" content="Molecular Biosciences, Northwestern University" />
<meta name="citation_author_institution" content="Interdisciplinary Biological Sciences Program, Northwestern University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2883-4616" />
<meta name="citation_author" content="Karol W. Bisaga" />
<meta name="citation_author_institution" content="Weinberg College of Arts and Sciences, Northwestern University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7680-6022" />
<meta name="citation_author" content="Robyn E. Tanny" />
<meta name="citation_author_institution" content="Molecular Biosciences, Northwestern University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0611-3909" />
<meta name="citation_author" content="Daniel E. Cook" />
<meta name="citation_author_institution" content="The Francis Crick Institute" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3347-562X" />
<meta name="citation_author" content="Daehan Lee" />
<meta name="citation_author_institution" content="Molecular Biosciences, Northwestern University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0546-8484" />
<meta name="citation_author" content="Ye Wang" />
<meta name="citation_author_institution" content="Molecular Biosciences, Northwestern University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5423-6196" />
<meta name="citation_author" content="Erik C. Andersen" />
<meta name="citation_author_institution" content="Molecular Biosciences, Northwestern University" />
<meta name="citation_author_institution" content="Interdisciplinary Biological Sciences Program, Northwestern University" />
<meta name="citation_author_institution" content="Robert H. Lurie Comprehensive Cancer Center, Northwestern University" />
<meta name="citation_author_email" content="erik.andersen@northwestern.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0229-9651" />
<meta name="citation_reference" content="A. K. Corsi, B. Wightman, M. A. Chalfie, 2015 A transparent window into biology: A primer on Caenorhabditis elegans, in WormBook, edited by The C. elegans Research Community." />
<meta name="citation_reference" content="Altés A., L. Paré, A. Esquirol, B. Xicoy, E. Rámila, et al., 2013 Pharmacogenetic analysis in the treatment of Hodgkin lymphoma. Leuk. Lymphoma 54: 1706–1712." />
<meta name="citation_reference" content="Anders S., and W. Huber, 2010 Differential expression analysis for sequence count data. Genome Biol. 11: R106." />
<meta name="citation_reference" content="Andersen E. C., J. P. Gerke, J. A. Shapiro, J. R. Crissman, R. Ghosh, et al., 2012 Chromosome-scale selective sweeps shape Caenorhabditis elegans genomic diversity. Nat. Genet. 44: 285–290." />
<meta name="citation_reference" content="Andersen E. C., J. S. Bloom, J. P. Gerke, and L. Kruglyak, 2014 A variant in the neuropeptide receptor npr-1 is a major determinant of Caenorhabditis elegans growth and physiology. PLoS Genet. 10: e1004156." />
<meta name="citation_reference" content="Andersen E. C., T. C. Shimko, J. R. Crissman, R. Ghosh, J. S. Bloom, et al., 2015 A Powerful New Quantitative Genetics Platform, Combining Caenorhabditis elegans High-Throughput Fitness Assays with a Large Collection of Recombinant Strains. G3 5: 911–920." />
<meta name="citation_reference" content="Bendesky A., M. Tsunozaki, M. V. Rockman, L. Kruglyak, and C. I. Bargmann, 2011 Catecholamine receptor polymorphisms affect decision-making in C. elegans. Nature 472: 313–318." />
<meta name="citation_reference" content="Bendesky A., J. Pitts, M. V. Rockman, W. C. Chen, M.-W. Tan, et al., 2012 Long-range regulatory polymorphisms affecting a GABA receptor constitute a quantitative trait locus (QTL) for social behavior in Caenorhabditis elegans. PLoS Genet. 8: e1003157." />
<meta name="citation_reference" content="Bloom J. S., I. M. Ehrenreich, W. T. Loo, T.-L. V. Lite, and L. Kruglyak, 2013 Finding the sources of missing heritability in a yeast cross. Nature 494: 234–237." />
<meta name="citation_reference" content="Blum R. H., S. K. Carter, and K. Agre, 1973 A clinical review of bleomycin--a new antineoplastic agent. Cancer 31: 903–914." />
<meta name="citation_reference" content="Boyd W. A., M. V. Smith, and J. H. Freedman, 2012 Caenorhabditis elegans as a model in developmental toxicology. Methods Mol. Biol. 889: 15–24." />
<meta name="citation_reference" content="Bray N. L., H. Pimentel, P. Melsted, and L. Pachter, 2016 Near-optimal probabilistic RNA-seq quantification. Nat. Biotechnol. 34: 525–527." />
<meta name="citation_reference" content="Brenner S., 1974 The genetics of Caenorhabditis elegans. Genetics 77: 71–94." />
<meta name="citation_reference" content="Calcutt M. J., and F. J. Schmidt, 1994 Gene organization in the bleomycin-resistance region of the producer organism Streptomyces verticillus. Gene 151: 17–21." />
<meta name="citation_reference" content="C. elegans Sequencing Consortium, 1998 Genome sequence of the nematode C. elegans: a platform for investigating biology. Science 282: 2012–2018." />
<meta name="citation_reference" content="Chang C. C., C. C. Chow, L. C. Tellier, S. Vattikuti, S. M. Purcell, et al., 2015 Second-generation PLINK: rising to the challenge of larger and richer datasets. Gigascience 4: 7." />
<meta name="citation_reference" content="Chen J., and J. Stubbe, 2005 Bleomycins: towards better therapeutics. Nat. Rev. Cancer 5: 102– 112." />
<meta name="citation_reference" content="Cloos J., E. J. Nieuwenhuis, D. I. Boomsma, D. J. Kuik, M. L. van der Sterre, et al., 1999 Inherited susceptibility to bleomycin-induced chromatid breaks in cultured peripheral blood lymphocytes. J. Natl. Cancer Inst. 91: 1125–1130." />
<meta name="citation_reference" content="Cook D. E., S. Zdraljevic, R. E. Tanny, B. Seo, D. D. Riccardi, et al., 2016 The Genetic Basis of Natural Variation in Caenorhabditis elegans Telomere Length. Genetics 204: 371–383." />
<meta name="citation_reference" content="Cook D. E., S. Zdraljevic, J. P. Roberts, and E. C. Andersen, 2017 CeNDR, the Caenorhabditis elegans natural diversity resource. Nucleic Acids Res. 45: D650–D657." />
<meta name="citation_reference" content="Cook D. E., and E. C. Andersen, 2017 VCF-kit: assorted utilities for the variant call format. Bioinformatics 33: 1581–1582." />
<meta name="citation_reference" content="Covarrubias-Pazaran G., 2016 Genome-Assisted Prediction of Quantitative Traits Using the R Package sommer. PLoS One 11: e0156744." />
<meta name="citation_reference" content="Dagogo-Jack I., and A. T. Shaw, 2018 Tumour heterogeneity and resistance to cancer therapies. Nat. Rev. Clin. Oncol. 15: 81–94." />
<meta name="citation_reference" content="Doench J. G., N. Fusi, M. Sullender, M. Hegde, E. W. Vaimberg, et al., 2016 Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR-Cas9. Nat. Biotechnol. 34: 184–191." />
<meta name="citation_reference" content="Du L., C. Sánchez, M. Chen, D. J. Edwards, and B. Shen, 2000 The biosynthetic gene cluster for the antitumor drug bleomycin from Streptomyces verticillus ATCC15003 supporting functional interactions between nonribosomal peptide synthetases and a polyketide synthase. Chem. Biol. 7: 623–642." />
<meta name="citation_reference" content="Edgar R. C., 2004a MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res. 32: 1792–1797." />
<meta name="citation_reference" content="Edgar R. C., 2004b MUSCLE: a multiple sequence alignment method with reduced time and space complexity. BMC Bioinformatics 5: 113." />
<meta name="citation_reference" content="Eilbeck K., S. E. Lewis, C. J. Mungall, M. Yandell, L. Stein, et al., 2005 The Sequence Ontology: a tool for the unification of genome annotations. Genome Biol. 6: R44." />
<meta name="citation_reference" content="Endelman J. B., 2011 Ridge Regression and Other Kernels for Genomic Selection with R Package rrBLUP. Plant Genome 4: 250–255." />
<meta name="citation_reference" content="Endelman J. B., and J.-L. Jannink, 2012 Shrinkage estimation of the realized relationship matrix. G3 2: 1405–1413." />
<meta name="citation_reference" content="Evans K. S., S. C. Brady, J. S. Bloom, R. E. Tanny, D. E. Cook, et al., 2018 Shared Genomic Regions Underlie Natural Variation in Diverse Toxin Responses. Genetics 210: 1509–1525." />
<meta name="citation_reference" content="García-González A. P., A. D. Ritter, S. Shrestha, E. C. Andersen, L. S. Yilmaz, et al., 2017 Bacterial Metabolism Affects the C. elegans Response to Cancer Chemotherapeutics. Cell 169: 431–441.e8." />
<meta name="citation_reference" content="Ghosh R., E. C. Andersen, J. A. Shapiro, J. P. Gerke, and L. Kruglyak, 2012 Natural variation in a chloride channel subunit confers avermectin resistance in C. elegans. Science 335: 574– 578." />
<meta name="citation_reference" content="Gu J., Y. Ye, M. R. Spitz, J. Lin, L. A. Kiemeney, et al., 2011 A genetic variant near the PMAIP1/Noxa gene is associated with increased bleomycin sensitivity. Hum. Mol. Genet. 20: 820–826." />
<meta name="citation_reference" content="Haas E. C. de, N. Zwart, C. Meijer, J. Nuver, H. M. Boezen, et al., 2008 Variation in bleomycin hydrolase gene is associated with reduced survival after chemotherapy for testicular germ cell cancer. J. Clin. Oncol. 26: 1817–1823." />
<meta name="citation_reference" content="Hahnel S. R., S. Zdraljevic, B. C. Rodriguez, Y. Zhao, P. T. McGrath, et al., 2018 Extreme allelic heterogeneity at a Caenorhabditis elegans beta-tubulin locus explains natural resistance to benzimidazoles. PLoS Pathog. 14: e1007226." />
<meta name="citation_reference" content="Hillier L. W., A. Coulson, J. I. Murray, Z. Bao, J. E. Sulston, et al., 2005 Genomics in C. elegans: so many genes, such a little worm. Genome Res. 15: 1651–1660." />
<meta name="citation_reference" content="Hunter D. J., 2005 Gene-environment interactions in human diseases. Nat. Rev. Genet. 6: 287– 298." />
<meta name="citation_reference" content="Kammenga J. E., A. Doroszuk, J. A. G. Riksen, E. Hazendonk, L. Spiridon, et al., 2007 A Caenorhabditis elegans wild type defies the temperature-size rule owing to a single nucleotide polymorphism in tra-3. PLoS Genet. 3: e34." />
<meta name="citation_reference" content="Kang H. M., N. A. Zaitlen, C. M. Wade, A. Kirby, D. Heckerman, et al., 2008 Efficient control of population structure in model organism association mapping. Genetics 178: 1709–1723." />
<meta name="citation_reference" content="Kelley L. A., S. Mezulis, C. M. Yates, M. N. Wass, and M. J. E. Sternberg, 2015 The Phyre2 web portal for protein modeling, prediction and analysis. Nat. Protoc. 10: 845–858." />
<meta name="citation_reference" content="Lazo J. S., and C. J. Humphreys, 1983 Lack of metabolism as the biochemical basis of bleomycin-induced pulmonary toxicity. Proc. Natl. Acad. Sci. U. S. A. 80: 3064–3068." />
<meta name="citation_reference" content="Lee D., H. Yang, J. Kim, S. Brady, S. Zdraljevic, et al., 2017 The genetic basis of natural variation in a phoretic behavior. Nat. Commun. 8: 273." />
<meta name="citation_reference" content="Li H., 2011 A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data. Bioinformatics 27: 2987–2993." />
<meta name="citation_reference" content="Liu J., J. Huang, Y. Zhang, Q. Lan, N. Rothman, et al., 2013 Identification of gene-environment interactions in cancer studies using penalization. Genomics 102: 189–194." />
<meta name="citation_reference" content="Low S.-K., S. Chung, A. Takahashi, H. Zembutsu, T. Mushiroda, et al., 2013 Genome-wide association study of chemotherapeutic agent-induced severe neutropenia/leucopenia for patients in Biobank Japan. Cancer Sci. 104: 1074–1082." />
<meta name="citation_reference" content="McClellan J., and M.-C. King, 2010 Genetic heterogeneity in human disease. Cell 141: 210–217." />
<meta name="citation_reference" content="McGrath P. T., M. V. Rockman, M. Zimmer, H. Jang, E. Z. Macosko, et al., 2009 Quantitative mapping of a digenic behavioral trait implicates globin variation in C. elegans sensory behaviors. Neuron 61: 692–699." />
<meta name="citation_reference" content="Nuver J., M. F. Lutke Holzik, M. van Zweeden, H. J. Hoekstra, C. Meijer, et al., 2005 Genetic variation in the bleomycin hydrolase gene and bleomycin-induced pulmonary toxicity in germ cell cancer patients. Pharmacogenet. Genomics 15: 399–405." />
<meta name="citation_reference" content="Palopoli M. F., M. V. Rockman, A. TinMaung, C. Ramsay, S. Curwen, et al., 2008 Molecular basis of the copulatory plug polymorphism in Caenorhabditis elegans. Nature 454: 1019– 1022." />
<meta name="citation_reference" content="Pimentel H., N. L. Bray, S. Puente, P. Melsted, and L. Pachter, 2017 Differential analysis of RNA-seq incorporating quantification uncertainty. Nat. Methods 14: 687–690." />
<meta name="citation_reference" content="Purcell S., B. Neale, K. Todd-Brown, L. Thomas, M. A. R. Ferreira, et al., 2007 PLINK: a tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet. 81: 559–575." />
<meta name="citation_reference" content="R Core Team, R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing." />
<meta name="citation_reference" content="Reddy K. C., E. C. Andersen, L. Kruglyak, and D. H. Kim, 2009 A polymorphism in npr-1 is a behavioral determinant of pathogen susceptibility in C. elegans. Science 323: 382–384." />
<meta name="citation_reference" content="Relling M. V., and T. Dervieux, 2001 Pharmacogenetics and cancer therapy. Nat. Rev. Cancer 1: 99–108." />
<meta name="citation_reference" content="Riedel C. G., R. H. Dowen, G. F. Lourenco, N. V. Kirienko, T. Heimbucher, et al., 2013 DAF-16 employs the chromatin remodeller SWI/SNF to promote stress resistance and longevity. Nat. Cell Biol. 15: 491–501." />
<meta name="citation_reference" content="Rockman M. V., and L. Kruglyak, 2009 Recombinational landscape and population genomics of Caenorhabditis elegans. PLoS Genet. 5: e1000419." />
<meta name="citation_reference" content="Rockman M. V., S. S. Skrovanek, and L. Kruglyak, 2010 Selection at linked sites shapes heritable phenotypic variation in C. elegans. Science 330: 372–376." />
<meta name="citation_reference" content="Samuel B. S., H. Rowedder, C. Braendle, M.-A. Félix, and G. Ruvkun, 2016 Caenorhabditis elegans responses to bacteria from its natural habitats. Proc. Natl. Acad. Sci. U. S. A. 113: E3941–9." />
<meta name="citation_reference" content="Santos R., O. Ursu, A. Gaulton, A. P. Bento, R. S. Donadi, et al., 2017 A comprehensive map of molecular drug targets. Nat. Rev. Drug Discov. 16: 19–34." />
<meta name="citation_reference" content="Schmid T., L. B. Snoek, E. Fröhli, M. L. van der Bent, J. Kammenga, et al., 2015 Systemic Regulation of RAS/MAPK Signaling by the Serotonin Metabolite 5-HIAA. PLoS Genet. 11: e1005236." />
<meta name="citation_reference" content="Seidel H. S., M. V. Rockman, and L. Kruglyak, 2008 Widespread genetic incompatibility in C. elegans maintained by balancing selection. Science 319: 589–594." />
<meta name="citation_reference" content="Seidel H. S., M. Ailion, J. Li, A. van Oudenaarden, M. V. Rockman, et al., 2011 A novel sperm-delivered toxin causes late-stage embryo lethality and transmission ratio distortion in C. elegans. PLoS Biol. 9: e1001115." />
<meta name="citation_reference" content="Sham P. C., and S. M. Purcell, 2014 Statistical power and significance testing in large-scale genetic studies. Nat. Rev. Genet. 15: 335–346." />
<meta name="citation_reference" content="Shen B., L. Du, C. Sanchez, D. J. Edwards, M. Chen, et al., 2002 Cloning and characterization of the bleomycin biosynthetic gene cluster from Streptomyces verticillus ATCC15003. J. Nat. Prod. 65: 422–431." />
<meta name="citation_reference" content="Shimko T. C., and E. C. Andersen, 2014 COPASutils: an R package for reading, processing, and visualizing data from COPAS large-particle flow cytometers. PLoS One 9: e111090." />
<meta name="citation_reference" content="Snoek L. B., H. E. Orbidans, J. J. Stastna, A. Aartse, M. Rodriguez, et al., 2014 Widespread genomic incompatibilities in Caenorhabditis elegans. G3 4: 1813–1823." />
<meta name="citation_reference" content="Stein L., P. Sternberg, R. Durbin, J. Thierry-Mieg, and J. Spieth, 2001 WormBase: network access to the genome and biology of Caenorhabditis elegans. Nucleic Acids Res. 29: 82– 86." />
<meta name="citation_reference" content="Stubbe J., and J. W. Kozarich, 1987 Mechanisms of bleomycin-induced DNA degradation. Chem. Rev. 87: 1107–1136." />
<meta name="citation_reference" content="Swan K. A., D. E. Curtis, K. B. McKusick, A. V. Voinov, F. A. Mapa, et al., 2002 High-throughput gene mapping in Caenorhabditis elegans. Genome Res. 12: 1100–1105." />
<meta name="citation_reference" content="Taylor E. M., and A. R. Lehmann, 1998 Conservation of eukaryotic DNA repair mechanisms. Int. J. Radiat. Biol. 74: 277–286." />
<meta name="citation_reference" content="Thompson O. A., L. B. Snoek, H. Nijveen, M. G. Sterken, R. J. M. Volkers, et al., 2015 Remarkably Divergent Regions Punctuate the Genome Assembly of the Caenorhabditis elegans Hawaiian Strain CB4856. Genetics 200: 975–989." />
<meta name="citation_reference" content="Turek M., J. Besseling, J.-P. Spies, S. König, and H. Bringmann, 2016 Sleep-active neuron specification and sleep induction require FLP-11 neuropeptides to systemically induce sleep. Elife 5. https://doi.org/10.7554/eLife.12499" />
<meta name="citation_reference" content="Wicks S. R., R. T. Yeh, W. R. Gish, R. H. Waterston, and R. H. Plasterk, 2001 Rapid gene mapping in Caenorhabditis elegans using a high density polymorphism map. Nat. Genet. 28: 160–164." />
<meta name="citation_reference" content="World Health Organization, 2018" />
<meta name="citation_reference" content="Zdraljevic S., C. Strand, H. S. Seidel, D. E. Cook, J. G. Doench, et al., 2017 Natural variation in a single amino acid substitution underlies physiological responses to topoisomerase II poisons. PLoS Genet. 13: e1006891." />
<meta name="citation_reference" content="Zdraljevic S., and E. C. Andersen, 2017 Natural diversity facilitates the discovery of conserved chemotherapeutic response mechanisms. Curr. Opin. Genet. Dev. 47: 41–47." />
<meta name="citation_reference" content="Zdraljevic S., B. W. Fox, C. Strand, O. Panda, F. J. Tenjo, et al., 2019 Natural variation in C. elegans arsenic toxicity is explained by differences in branched chain amino acid metabolism. Elife 8. https://doi.org/10.7554/eLife.40260" />
<meta name="citation_reference" content="Zhang L., L. Li, L. Yan, Z. Ming, Z. Jia, et al., 2018 Structural and Biochemical Characterization of Endoribonuclease Nsp15 Encoded by Middle East Respiratory Syndrome Coronavirus. J. Virol. 92. https://doi.org/10.1128/JVI.00893-18" />
<meta name="twitter:title" content="A nematode-specific gene underlies bleomycin-response variation in Caenorhabditis elegans" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Bleomycin is a powerful chemotherapeutic drug used to treat a variety of cancers. However, individual patients vary in their responses to bleomycin. The identification of genetic differences that underlie this response variation could improve treatment outcomes by tailoring bleomycin dosages to each patient. We used the model organism Caenorhabditis elegans to identify genetic determinants of bleomycin-response differences by performing linkage mapping on recombinants derived from a cross between the laboratory strain (N2) and a wild strain (CB4856). This approach identified a small genomic region on chromosome V that underlies bleomycin-response variation. Using near-isogenic lines and strains with CRISPR-Cas9 mediated deletions and allele replacements, we discovered that a novel nematode-specific gene ( scb-1 ) is required for bleomycin resistance. Although the mechanism by which this gene causes variation in bleomycin responses is unknown, we suggest that a rare variant present in the CB4856 strain might cause differences in the potential stress-response function of scb-1 between the N2 and CB4856 strains, thereby leading to differences in bleomycin resistance.

Article summary We performed linkage mapping on a panel of recombinant lines generated between two genetically divergent strains of Caenorhabditis elegans and identified a bleomycin-response QTL. We generated CRISPR-Cas9 deletions and reciprocal allele-replacement strains for all six candidate genes across the QTL confidence interval. Deletions of one gene, H19N07.3 , caused increased bleomycin sensitivity in both divergent genetic backgrounds. This gene might act in stress responses and detoxification in nematodes. We further compared our linkage mapping to a genome-wide association mapping and showed that a rare expression variant in the CB4856 strain likely underlies bleomycin-response differences." />
<meta name="og-title" property="og:title" content="A nematode-specific gene underlies bleomycin-response variation in Caenorhabditis elegans" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/565218v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Bleomycin is a powerful chemotherapeutic drug used to treat a variety of cancers. However, individual patients vary in their responses to bleomycin. The identification of genetic differences that underlie this response variation could improve treatment outcomes by tailoring bleomycin dosages to each patient. We used the model organism Caenorhabditis elegans to identify genetic determinants of bleomycin-response differences by performing linkage mapping on recombinants derived from a cross between the laboratory strain (N2) and a wild strain (CB4856). This approach identified a small genomic region on chromosome V that underlies bleomycin-response variation. Using near-isogenic lines and strains with CRISPR-Cas9 mediated deletions and allele replacements, we discovered that a novel nematode-specific gene ( scb-1 ) is required for bleomycin resistance. Although the mechanism by which this gene causes variation in bleomycin responses is unknown, we suggest that a rare variant present in the CB4856 strain might cause differences in the potential stress-response function of scb-1 between the N2 and CB4856 strains, thereby leading to differences in bleomycin resistance.

Article summary We performed linkage mapping on a panel of recombinant lines generated between two genetically divergent strains of Caenorhabditis elegans and identified a bleomycin-response QTL. We generated CRISPR-Cas9 deletions and reciprocal allele-replacement strains for all six candidate genes across the QTL confidence interval. Deletions of one gene, H19N07.3 , caused increased bleomycin sensitivity in both divergent genetic backgrounds. This gene might act in stress responses and detoxification in nematodes. We further compared our linkage mapping to a genome-wide association mapping and showed that a rare expression variant in the CB4856 strain likely underlies bleomycin-response differences." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-05-14" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/565218v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/565218v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/565218v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/565218v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/732964" />
    <title>A nematode-specific gene underlies bleomycin-response variation in Caenorhabditis elegans | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"G-mw0UpRMdOwCgUPsknJ5YJBFub6w-92UL9Aji5sWuo","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"732964","apath":"\/biorxiv\/early\/2019\/05\/14\/565218.atom","pisa":"biorxiv;565218v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;565218v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/565218v2","title":"A nematode-specific gene underlies bleomycin-response variation in Caenorhabditis elegans","identifier":"node\/732964"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022XxJ4CogKGCps\\\\\\\/2sQPU4nsfRdQa7stJUSVNXCzMVJAq00S8RL2TwTvZqov0Ha85A2zfmx5lO8PSCnRhmiiv+hAqdS4gn6Ea2S5BLJYpwWra8\\\\\\\/bHpaeEtcVUUzgTIU5XRiYqcWEs0nhaT081O4ZqFby2unCQq+cA4gFGElfeKy0kVxWtdBunz59K8x47quHpOs5ZLsmUYh87ZfFBURXtRtEG8UTVfChYKiCyCAcJ9F9EXjkQCGWmyqsOLAoSLYbShWwB3LjHGXGQvjJkDJFuUMpIfJIoQFQbJbSFQJMFpeAuetwgMuBo5NCms1OTxSLz21C95lWVou3gAW+A4An4CdfWoH\\\\\\\/01DDkBjbIXQpfigO5nvL+pPSjdxGKdQE4292vTuC5M\\\\\\\/I0ZORYQHTuxJtUzdhzU7zUeba0wmjLz3Urh38UON76ybOv5fh9w77GO2lng7bC+UvHmMj5KI35nzrr40tchuaQ2s8Iuu9Szrz8AT4qt2tMuG7FmA5h\\\\\\\/ZHB10hQi6EgPuHBqWQxi+f00PDtZejPEYM6fZjmo\\\\\\\/ivONlcbb+a9REB5W3hlRAPrNkWrv+WeFZuuOeeKk6F62iVu338Rfxl9NAnGoPneR9JO7LdliBqxhFfHuul4nBQAfm9VN15UHsWyC9OP3E7B8D4fTZqBNq3J3vPLUiyBShVnmmp0y1PNXrSra\\\\\\\/fjsjmoe6CRHrmntyZEK+foY61cAneM6eVloBzyDdzMxRdf61sWkKpf4jeWYqqofGlNSJc1MRT1+CUZ02kd2aYNwmdKB8654OuTnLaTtkoAbrG7Hv3ec9Dzx5viJ3q1g3TPRHi73mx2XwUAJLTVwd3R2k+VcBaVKKmcTa0ltv63PhVn1Tnjx4T81Aj+FF8NGJqe9vv0F2k5j5SHIqWkONOsl3n7RKG+V3qMlJ4qoCCdOs53+yuMjV1szyy+hYt7+afqEc+SO7g6Yj0IObL99iepGSr\\\\\\\/fgCT29itlJ\\\\\\\/0RJe6epY23h+T3AXFYJ1MP8VLO0nKNtq2c6gMP52FqDDDFKhy7jV2iDWTJ2GWgVFHtdOI0wbnDmnJvWQoP5VVxxkeddF8OFLT+kD9FTM4x+s7qy8TrcTDi074DNdbnWrth8nn1WhOncfA63+5u5cNd3VdCNnJdId7ZgQbeWRUDlqMJxvCGJRbXo9t3EfNmjG8nUj64InQMp0BBte5uCn6jDl3wiYq3PCHyC9OrtmlueFokAZnGwYz9MFTKyQMitsvuh6RgvhqwQoVZ95jyBhhTKDfzuYbO88+qI\\\\\\\/8\\\\\\\/m0dNtMFynvmjUiHT9585bzqyKw\\\\\\\/dLr0PA7qlmPb2mYkiWes=\\\u0022,\\\u0022iv\\\u0022:\\\u0022Li8hp94noJGuEGcPUMIwvg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00222d18aadf9a4afdee3e0ecc7164fc7643dd20e308634ef635021089a34c60406d\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/565218v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-732964 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/565218v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-9INrPR-4x2nbYPpbJZFvNqMK_Oc2oqhFXJmVcpPo_ZM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="732964" id="node732964" data-pisa="biorxiv;565218v2" data-pisa-master="biorxiv;565218" data-apath="/biorxiv/early/2019/05/14/565218.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">A nematode-specific gene underlies bleomycin-response variation in <em>Caenorhabditis elegans</em></h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3043-1544" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Shannon C.</span> <span class="nlm-surname">Brady</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-2883-4616" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Stefan</span> <span class="nlm-surname">Zdraljevic</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7680-6022" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Karol W.</span> <span class="nlm-surname">Bisaga</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-0611-3909" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Robyn E.</span> <span class="nlm-surname">Tanny</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-3347-562X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Daniel E.</span> <span class="nlm-surname">Cook</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-0546-8484" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Daehan</span> <span class="nlm-surname">Lee</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-5423-6196" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Ye</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-0229-9651" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Erik C.</span> <span class="nlm-surname">Andersen</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/565218 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node732964" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Shannon C. Brady </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span><span class='nlm-institution'>Molecular Biosciences, Northwestern University</span>, Evanston, IL 60208</div><div class='author-affiliation'><span class='nlm-sup'>†</span><span class='nlm-institution'>Interdisciplinary Biological Sciences Program, Northwestern University</span>, Evanston, IL 60208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shannon%2BC.%2BBrady%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brady%20SC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShannon%2BC.%2BBrady%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3043-1544" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Shannon C. Brady</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Stefan Zdraljevic </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span><span class='nlm-institution'>Molecular Biosciences, Northwestern University</span>, Evanston, IL 60208</div><div class='author-affiliation'><span class='nlm-sup'>†</span><span class='nlm-institution'>Interdisciplinary Biological Sciences Program, Northwestern University</span>, Evanston, IL 60208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BZdraljevic%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zdraljevic%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStefan%2BZdraljevic%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2883-4616" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Stefan Zdraljevic</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Karol W. Bisaga </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span><span class='nlm-institution'>Weinberg College of Arts and Sciences, Northwestern University</span>, Evanston, IL 60208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karol%2BW.%2BBisaga%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bisaga%20KW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKarol%2BW.%2BBisaga%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7680-6022" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Karol W. Bisaga</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Robyn E. Tanny </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span><span class='nlm-institution'>Molecular Biosciences, Northwestern University</span>, Evanston, IL 60208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robyn%2BE.%2BTanny%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tanny%20RE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobyn%2BE.%2BTanny%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0611-3909" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Robyn E. Tanny</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Daniel E. Cook </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span><span class='nlm-institution'>The Francis Crick Institute</span>, London NW1 1ST, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BE.%2BCook%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cook%20DE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BE.%2BCook%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3347-562X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel E. Cook</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Daehan Lee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span><span class='nlm-institution'>Molecular Biosciences, Northwestern University</span>, Evanston, IL 60208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daehan%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lee%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaehan%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0546-8484" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daehan Lee</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Ye Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span><span class='nlm-institution'>Molecular Biosciences, Northwestern University</span>, Evanston, IL 60208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ye%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYe%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5423-6196" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ye Wang</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Erik C. Andersen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span><span class='nlm-institution'>Molecular Biosciences, Northwestern University</span>, Evanston, IL 60208</div><div class='author-affiliation'><span class='nlm-sup'>†</span><span class='nlm-institution'>Interdisciplinary Biological Sciences Program, Northwestern University</span>, Evanston, IL 60208</div><div class='author-affiliation'><span class='nlm-sup'>**</span><span class='nlm-institution'>Robert H. Lurie Comprehensive Cancer Center, Northwestern University</span>, Chicago, IL 60611</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erik%2BC.%2BAndersen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Andersen%20EC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AErik%2BC.%2BAndersen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0229-9651" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Erik C. Andersen</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:erik.andersen@northwestern.edu" class="" data-icon-position="" data-hide-link-title="0">erik.andersen@northwestern.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/565218v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:732964" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:732964/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/565218v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:732964" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:732964/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/565218v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:732964" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:732964/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/565218v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:732964" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:732964/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/565218v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:732964" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:732964/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/565218v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:732964" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:732964/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">ABSTRACT</h2><p id="p-2">Bleomycin is a powerful chemotherapeutic drug used to treat a variety of cancers. However, individual patients vary in their responses to bleomycin. The identification of genetic differences that underlie this response variation could improve treatment outcomes by tailoring bleomycin dosages to each patient. We used the model organism <em>Caenorhabditis elegans</em> to identify genetic determinants of bleomycin-response differences by performing linkage mapping on recombinants derived from a cross between the laboratory strain (N2) and a wild strain (CB4856). This approach identified a small genomic region on chromosome V that underlies bleomycin-response variation. Using near-isogenic lines and strains with CRISPR-Cas9 mediated deletions and allele replacements, we discovered that a novel nematode-specific gene (<em>scb-1</em>) is required for bleomycin resistance. Although the mechanism by which this gene causes variation in bleomycin responses is unknown, we suggest that a rare variant present in the CB4856 strain might cause differences in the potential stress-response function of <em>scb-1</em> between the N2 and CB4856 strains, thereby leading to differences in bleomycin resistance.</p><div id="sec-1" class="subsection"><p id="p-3"><strong>Article summary</strong> We performed linkage mapping on a panel of recombinant lines generated between two genetically divergent strains of <em>Caenorhabditis elegans</em> and identified a bleomycin-response QTL. We generated CRISPR-Cas9 deletions and reciprocal allele-replacement strains for all six candidate genes across the QTL confidence interval. Deletions of one gene, <em>H19N07.3</em>, caused increased bleomycin sensitivity in both divergent genetic backgrounds. This gene might act in stress responses and detoxification in nematodes. We further compared our linkage mapping to a genome-wide association mapping and showed that a rare expression variant in the CB4856 strain likely underlies bleomycin-response differences.</p></div></div><div class="section" id="sec-2"><h2 class="">INTRODUCTION</h2><p id="p-5">Cancer is the second leading cause of death worldwide (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">World Health Organization, 2018</a>), which has led to extensive research for treatments, including the identification of over 100 effective chemotherapeutic drugs (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Santos <em>et al</em>. 2017</a>). One of these drugs is bleomycin, an anti-tumor antibiotic that interacts with oxygen and transition metals to cause double-stranded DNA breaks (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Chen and Stubbe 2005</a>). Although the cytotoxicity of bleomycin can reliably induce cell death in tumor cells, off-target effects can lead to a range of harmful consequences from mild gastrointestinal irritation to severe bleomycin-induced pulmonary fibrosis (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Blum <em>et al</em>. 1973</a>). The tradeoff between efficacy and toxicity varies across individuals, and understanding the genetic variants that affect bleomycin response might yield opportunities to broaden the therapeutic range (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Relling and Dervieux 2001</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">de Haas <em>et al</em>. 2008</a>).</p><p id="p-6">Bleomycin sensitivity has been shown to be heritable, suggesting that genetic markers can be used to predict bleomycin responses (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Cloos <em>et al</em>. 1999</a>). Many studies have attempted to identify the genetic variant(s) that underlie bleomycin-response differences across cancer patients, and some have identified potential connections between the metabolic enzyme bleomycin hydrolase (BLMH) and patient outcomes. However, none of these studies established a causal connection between genetic differences in BLMH and variation in bleomycin responses (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lazo and Humphreys 1983</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Nuver <em>et al</em>. 2005</a>; de Haas <em>et al</em>. 2008; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Gu <em>et al</em>. 2011</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Altés <em>et al</em>. 2013</a>). The inability to identify a human genetic variant that causes differences in bleomycin responses might be attributed to limited sample sizes (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Sham and Purcell 2014</a>), confounding environmental factors (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Hunter 2005</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Liu <em>et al</em>. 2013</a>), variation in drug regimens across patients (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Low <em>et al</em>. 2013</a>), or tumor complexity and progression (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">McClellan and King 2010</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Dagogo-Jack and Shaw 2018</a>). However, the DNA-damage pathways that might be implicated in bleomycin responses are evolutionarily conserved across eukaryotes (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Taylor and Lehmann 1998</a>). Therefore, studying bleomycin responses in a model organism with natural genetic variation can offer insights into how bleomycin response differs across individuals and can potentially be applied to the clinic (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Zdraljevic and Andersen 2017</a>).</p><p id="p-7"><em>Caenorhabditis elegans</em> is a soil-associated microscopic roundworm that is an excellent model for basic cellular and organismal processes (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">A. K. Corsi, B. Wightman, M. A. Chalfie 2015</a>). Not only does <em>C. elegans</em> have a well annotated reference genome (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">C. elegans Sequencing Consortium 1998</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Stein <em>et al</em>. 2001</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Hillier <em>et al</em>. 2005</a>); <a href="http://www.wormbase.org">www.wormbase.org</a> WS268), but this species also has broad genomic diversity across global populations (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Cook <em>et al</em>. 2017</a>). Notably, the N2 strain and the CB4856 strain are well characterized and genetically divergent with approximately one single nucleotide variant per 850 bp (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Wicks <em>et al</em>. 2001</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Swan <em>et al</em>. 2002</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Thompson <em>et al</em>. 2015</a>). These two strains were used to generate a panel of recombinant inbred advanced intercross lines (RIAILs) (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Rockman and Kruglyak 2009</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Andersen <em>et al</em>. 2015</a>), which has been used to correlate genetic variants with differences in quantitative traits (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Kammenga <em>et al</em>. 2007</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Seidel <em>et al</em>. 2008</a>, <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">2011</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Palopoli <em>et al</em>. 2008</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Reddy <em>et al</em>. 2009</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">McGrath <em>et al</em>. 2009</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bendesky <em>et al</em>. 2011</a>, <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">2012</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Andersen <em>et al</em>. 2014</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Schmid <em>et al</em>. 2015</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Zdraljevic <em>et al</em>. 2017</a>, <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">2019</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Lee <em>et al</em>. 2017</a>).</p><p id="p-8">Here, we used a high-throughput fitness assay to measure bleomycin responses across a panel of 249 RIAILs (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Andersen <em>et al</em>. 2015</a>) and then performed linkage mapping to identify quantitative trait loci (QTL) that underlie bleomycin-response variation. We used near-isogenic lines (NILs) to validate the largest effect QTL on chromosome V. Our results from the NIL assays suggested that epistatic loci underlie bleomycin responses, but a two-factor genome scan was unable to detect significant genetic interactions. Next, we created and tested CRISPR-Cas9 mediated deletion alleles to investigate all six candidate genes in the QTL region. We identified a nematode-specific gene, <em>H19N07.3</em>, that underlies this QTL. Although this gene does not contain a protein-coding variant between the N2 and CB4856 strains, its gene expression varies across the RIAIL panel. Interestingly, a genome-wide association (GWA) approach identifies different QTL than the linkage mapping approach, suggesting that both common and rare variants underlie bleomycin response variation. Given the genetic complexity underlying the bleomycin response phenotype, this study highlights the power of the <em>C. elegans</em> model system to identify elusive causal genes.</p></div><div class="section" id="sec-3"><h2 class="">MATERIALS AND METHODS</h2><div id="sec-4" class="subsection"><h3>Strains</h3><p id="p-9">Animals were grown at 20° on 6 cm plates of modified nematode growth medium (NGMA), which contained 1% agar and 0.7% agarose, spotted with OP50 bacteria (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Ghosh <em>et al</em>. 2012</a>). The two parental strains used in this study were N2 and CB4856. N2 is the canonical laboratory strain of <em>C. elegans</em> that has been extensively studied (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Brenner 1974</a>). CB4856 is a well studied Hawaiian wild isolate that is genetically divergent from N2 and has a characterized genome (<a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Wicks <em>et al</em>. 2001</a>; <a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Swan <em>et al</em>. 2002</a>; <a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Thompson <em>et al</em>. 2015</a>). The N2 and CB4856 strains were crossed for several generations to create a panel of recombinant inbred advanced intercross lines (RIAILs) that contain regions of the genome derived from each parental strain. These RIAILs were constructed previously (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Rockman and Kruglyak 2009</a>; <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Andersen <em>et al</em>. 2015</a>) and have well characterized genotypes and allele frequencies, and we used this panel of RIAILs in our study to identify regions of the genome correlated with drug response. The construction of near-isogenic lines (NILs), as well as CRISPR-Cas9 mediated deletion and allele-replacement strains, is detailed below. All strains and reagents used in strain constructions are listed in the <strong>Supplementary Information</strong>.</p></div><div id="sec-5" class="subsection"><h3>High-throughput fitness assays</h3><p id="p-10">We used the high-throughput assay (HTA) described previously (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Evans <em>et al</em>. 2018</a>) and following is a summary of that assay (<strong>Figure S1</strong>). Populations of each strain were passaged on 6 cm plates for four generations to amplify animal numbers and reduce the effects of starvation (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Andersen <em>et al</em>. 2014</a>). Gravid adults were bleached for stage synchronization, and approximately 25 embryos from each strain were aliquoted into 96-well plates at a final volume of 50 µL of K medium (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Boyd <em>et al</em>. 2012</a>). The following day, arrested L1 larvae were fed 5 mg/mL HB101 bacterial lysate in K medium (Pennsylvania State University Shared Fermentation Facility, State College, PA; (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">García-González <em>et al</em>. 2017</a>) and were grown for 48 hours at 20° with constant shaking. A large-particle flow cytometer (COPAS BIOSORT, Union Biometrica, Holliston, MA) was used to sort three L4 larvae into each well of a 96-well plate that contained 50 µL K medium plus HB101 lysate at 10 mg/mL, 50 µM kanamycin, and either 1% distilled water (control) or 1% distilled water and bleomycin (drug). The sorted L4 larvae were grown and propagated for 96 hours at 20° with constant shaking. The population of parents and progeny were treated with sodium azide (50 mM in M9) and quantified by the BIOSORT for several fitness parameters. Because bleomycin exposure can affect animal proliferation (brood size), animal growth (length), and animal development (optical density), the fitness parameters we measured with the BIOSORT included brood size (n), animal length (time of flight, TOF), and optical density (extinction time, EXT).</p></div><div id="sec-6" class="subsection"><h3>Bleomycin-response trait measurements and processing</h3><p id="p-11">Phenotypic measurements collected by the BIOSORT were processed using the R package <em>easysorter</em> (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Shimko and Andersen 2014</a>). Using this package, <em>read_data</em> imported measurements from the BIOSORT and <em>remove_contamination</em> was used to remove contaminated wells from analysis. The <em>sumplate</em> function then calculated normalized measurements (norm.n -- brood size normalized to number of animals sorted, norm.EXT -- EXT normalized by TOF measurements) and summary statistics (mean, median, 10<sup>th,</sup> 25<sup>th</sup>, 75<sup>th</sup>, 90<sup>th</sup> percentile, interquartile range, covariance, and variance) of each trait for the population of animals. A total of 26 HTA traits were measured. When strains were phenotyped across multiple days, the <em>regress(assay=TRUE)</em> function was used to fit a linear model with the phenotype as the dependent variable and assay as the independent variable (<em>phenotype ∼ assay</em>) to account for variation among assay days. Next, the <em>prune_outliers()</em> function removed phenotypic values that were beyond two standard deviations of the mean (unless at least 5% of the strains were outside this range in the case of RIAIL assays). Finally, bleomycin-specific effects were calculated using the <em>regress(assay=FALSE)</em> function from <em>easysorter</em>, which fits a linear model with the phenotype as the dependent variable and control phenotype as the independent variable (<em>phenotype ∼ control phenotype</em>). The residual phenotypic values account for differences among strains that were present in control conditions.</p></div><div id="sec-7" class="subsection"><h3>Bleomycin dose response</h3><p id="p-12">A dose-response high-throughput assay was performed using quadruplicates of four genetically divergent strains (N2, CB4856, JU258, and DL238) tested in various concentrations of bleomycin (<strong>File S1</strong>). The broad-sense heritability at each concentration was calculated using the <em>lmer</em> function within the <em>lme4</em> R package with the phenotype as the dependent variable and strain as a random effect <em>phenotype ∼ 1 + (1|strain)</em> (<strong>File S2</strong>). The concentration of bleomycin that provided the highest mean broad-sense heritability across the 26 HTA traits was selected for linkage mapping experiments (50 µM, mean <em>H</em><sup>2</sup> = 0.58). Bleomycin sulfate was purchased from Biotang Inc via Fisher Scientific (Catalog No. 50-148-546).</p></div><div id="sec-8" class="subsection"><h3>Whole-genome sequence library prep and analysis</h3><p id="p-13">Whole-genome sequencing was performed on recombinant advanced intercross lines (RIAILs) and near-isogenic lines (NILs) using low-coverage sequencing. DNA was isolated from 100-300 µL of packed worms using Omega BioTek’s EZ 96 Tissue DNA Kit (catalog no. D1196-01). All samples were diluted to 0.2 ng/µL and incubated with diluted Illumina transposome (catalog no. FC-121-1031). Tagmented samples were amplified with barcoded primers. Unique libraries (192) were pooled by adding 8 µL of each library. The pooled material was size-selected by separating the material on a 2% agarose gel and excising the fragments ranging from 400-600 bp. The sample was purified using Qiagen’s Gel Extraction Kit (catalog no. 28706) and eluted in 30 µL of buffer EB. The concentration of the purified sample was determined using the Qubit dsDNA HS Assay Kit (catalog no. Q32851). RIAILs and NILs were sequenced at low coverage (mean = 2.13x) using the Illumina HiSeq 2500 platform with a paired-end 100 bp reaction lane. RIAIL and NIL genotypes were imputed using VCF-kit (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Cook and Andersen 2017</a>). To determine genotypes, a list of filtered, high-quality sites (n = 196,565) where parental strains possess different genotypes was extracted from a previously established variant dataset (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Cook <em>et al</em>. 2016</a>). All RIAIL genotypes can be accessed by downloading the <em>linkagemapping</em> R package at github.com/AndersenLab/linkagemapping and by running <em>load_cross_obj(“N2xCB4856cross_full”)</em>. NIL genotypes are described below and are available in <strong>File S5</strong> and <strong>File S6</strong>.</p></div><div id="sec-9" class="subsection"><h3>Linkage mapping</h3><p id="p-14">Linkage mapping was performed on each of the 26 HTA traits described above using the R package <em>linkagemapping</em> (<a href="http://www.github.com/AndersenLab/linkagemapping">www.github.com/AndersenLab/linkagemapping</a>, <strong>Figure S4, Figure S5, File S4</strong>). The function <em>load_cross_obj</em>(“N2xCB4856cross_full”) was executed to load a cross object containing 13,003 SNPs that describe locations of genetic recombination in the RIAIL panel (out of the 195,565 high-quality SNPs at which genotypes were called). The genotypic data and residual phenotypic data (after control-condition regression) were merged into a cross object using the <em>merge_pheno</em> function with <em>set</em> = 2 to include strains with a reduced allele-frequency skew on chromosome I. The <em>fsearch</em> function was used to calculate logarithm of odds (LOD) scores for each marker and each trait as -n(ln(1 - R<sup>2</sup>)/2ln(10)), where <em>R</em> is the Pearson correlation coefficient between RIAIL genotypes at the marker and trait values (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bloom <em>et al</em>. 2013</a>; <a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Zdraljevic <em>et al</em>. 2017</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Lee <em>et al</em>. 2017</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Evans <em>et al</em>. 2018</a>). A 5% genome-wide error rate was calculated by permuting phenotype and genotype data 1,000 times. Mappings were repeated iteratively, each time using the marker with the highest LOD score as a cofactor until no significant QTL were detected. The <em>annotate_lods</em> function was used to calculate the percent of variance in RIAIL phenotypes explained by each QTL and a 95% confidence interval around each peak marker, defined by any marker on the same chromosome within a 1.5-LOD drop from the peak marker.</p></div><div id="sec-10" class="subsection"><h3>Generation of near-isogenic lines (NILs)</h3><p id="p-15">A near-isogenic line (NIL) is genetically identical to another strain aside from a small genomic region that is derived from an alternate strain. NILs are used to test the effect of modifications to particular genomic regions in a consistent genetic background. To make each NIL, males and hermaphrodites of the desired RIAIL (QX131 for ECA230 and QX450 for ECA232) and parental background (CB4856 for ECA230 and N2 for ECA232) were crossed in bulk, then male progeny were crossed to the parental strain in bulk for another generation. For each NIL, eight single-parent crosses were performed followed by six generations of propagating isogenic lines to ensure homozygosity of the genome. For each cross, PCR was used to select non-recombinant progeny genotypes within the introgressed region by amplifying insertion-deletion (indel) variants between the N2 and CB4856 genotypes on the left and right side of the introgressed region. NIL strains ECA411 and ECA528 were generated by crossing ECA230 and CB4856 in bulk. Heterozygous F1 hermaphrodites were crossed to CB4856 males and the F2 L4 hermaphrodites were placed into individual wells of a 96-well plate with K medium and 5 mg/mL bacterial lysate and grown to starvation. After starvation, each well of the 96-well plate was genotyped to identify recombinants in the desired region. Recombinant strains were plated onto 6 cm plates and individual hermaphrodites were propagated for several generations to ensure homozygosity across the genome. NIL strains were whole-genome sequenced as described above to confirm their genotypes (<strong>File S5, File S6</strong>). Reagents used to generate all NIL strains and a summary of each introgressed region are detailed in the <strong>Supplementary Information</strong>.</p></div><div id="sec-11" class="subsection"><h3>Two-factor genome scan</h3><p id="p-16">We performed a two-factor genome scan to identify potentially epistatic loci that might contribute to traits of interest (either bleomycin responses or gene-expression levels). We used the <em>scantwo</em> function in the R <em>qtl</em> package to perform this analysis. Each pairwise combination of loci was tested for correlations with trait variation in the RIAILs. The summary of each scantwo output includes the maximum interactive LOD score for each pair of chromosomes. To determine a threshold for significant interactions, we performed 1000 permutations of the scantwo analysis and selected the 95<sup>th</sup> percentile from these permutations. For the bleomycin-response variation scantwo, the significant interaction threshold was a LOD score of 4.08. The significant interaction threshold for the <em>scb-1</em> expression variation scantwo was 4.05. The bleomycin-response variation scantwo summary is available as <strong>File S8</strong>, and the scantwo summary for <em>scb-1</em> expression variation is available as <strong>File S17</strong>.</p></div><div id="sec-12" class="subsection"><h3>Identification of genes with non-synonymous variants</h3><p id="p-17">The <em>get_db</em> function within the <em>cegwas</em> R package was used to query WormBase build WS245 for genes in the QTL confidence interval (V:11,042,745-11,189,364). Our initial linkage mappings used the 1,454 marker set (<a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Rockman and Kruglyak 2009</a>) and had a QTL confidence interval larger than the interval found using the whole-genome marker set (described above). Because this confidence interval was larger and more conservative, we kept it for subsequent testing of candidate genes. This expanded interval contained an additional 20 kb on either side of the whole-genome marker set confidence interval (<strong>File S9</strong>). The <em>snpeff</em> function within the <em>cegwas</em> R package was used to identify variants within the region of interest (<strong>File S11</strong>). We identified variants predicted to have MODERATE (coding variant, inframe insertion/deletion, missense variant, regulatory region ablation, or splice region variant) or HIGH (chromosome number variant, exon loss, frameshift variant, rare amino acid variant, splice donor/acceptor variant, start loss, stop loss/gain, or transcript ablation) phenotypic effects according to Sequence Ontology (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Eilbeck <em>et al</em>. 2005</a>) and selected variants at which the CB4856 strain contains the alternate allele. This search found five candidate genes in the interval: <em>C45B11.8</em>, <em>C45B11.6</em>, <em>jmjd-5</em>, <em>srg-42</em>, and <em>cnc-10</em>, which each contain one non-synonymous variant between the N2 and CB4856 strains.</p></div><div id="sec-13" class="subsection"><h3>Generation of deletion strains</h3><p id="p-18">Deletion alleles for genes of interest were generated to test the effects of loss-of-function variants on bleomycin responses. For each desired deletion, we designed a 5’ and a 3’ guide RNA with the highest possible on-target and off-target scores, calculated using the Doench algorithm (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Doench <em>et al</em>. 2016</a>). The N2 and CB4856 L4 hermaphrodite larvae were picked to 6 cm agar plates seeded with OP50 <em>E. coli</em> 24 hours before injection. The CRISPR injection mix was assembled by first incubating 0.88 µL of 200 µM AltR® CRISPR-Cas9 tracrRNA (IDT, catalog no. 1072532), 0.82 µL of each of the 5’ and 3’ AltR® CRISPR-Cas9 crRNAs at a stock concentration of 100 µM in Duplex Buffer (IDT), and 0.12 µL of 100 µM <em>dpy-10</em> co-injection crRNA at 95° for five minutes. 2.52 µL of 69 µM AltR® <em>S. pyogenes</em> Cas9 Nuclease (IDT, catalog no. 1081058) was added to the tracrRNA/crRNA complex mixture and incubated at room temperature for five minutes. Finally, 0.5 µL of 10 µM <em>dpy-10</em> repair construct and distilled water were added to a final volume of 10 µL. The injection mixture was loaded into the injection capillary using a mouth pipet to avoid bubbles in the injection solution. Young adult animals were mounted onto agar injection pads, injected in either the anterior or posterior arm of the gonad, and allowed to recover on 6 cm plates in bulk. Twelve hours after injection, survivors were transferred to individual 6 cm plates. Broods with successful edits were easily observed because of the <em>dpy-10</em> co-injection marker, which created animals with an obvious locomotive defect, roller (Rol), or morphological phenotype, dumpy (Dpy). Three to four days after injections, plates were checked for the presence of Rol or Dpy F1 progeny. These Rol F1 progeny were transferred to individual 6 cm plates, allowed to lay offspring, and genotyped for the desired edit 24 hours later. Genotyping was performed with PCR amplicons designed around the desired site of the deletion. Plates with heterozygous or homozygous deletions were propagated and genotyped for at least two more generations to ensure homozygosity and to cross out the Rol mutation. Deletion amplicons were Sanger sequenced to identify the exact location of the deletion. Reagents used to generate deletion alleles are detailed in <strong>Supplemental Information.</strong></p></div><div id="sec-14" class="subsection"><h3>Generation of CRISPR-mediated <em>jmjd-5</em> allele replacements</h3><p id="p-19">Allele replacement strains were created to test the effect of a particular amino acid substitution on bleomycin responses. A guide RNA was designed to cut two bp downstream of the natural variant, with an on-target score of 31 and off-target score of 47 (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Doench <em>et al</em>. 2016</a>). Two repair constructs were designed, one for the N2 to CB4856 replacement and <em>vice versa</em>. Repair oligonucleotides were homologous to the background strain except for the nucleotide variant, a silent mutation in the PAM site (A339A) to eliminate repair construct cleavage, and a silent mutation that introduces a <em>BsaA</em>I restriction enzyme cut site (T336T). Repair constructs contained a 35-bp homology arm on the PAM-proximal side of the edit and a 91-bp homology arm on the PAM-distal side of the edit. Injection mixes were made as above, with 0.6 µL of 100 µM <em>jmjd-5</em> repair construct added with the <em>dpy-10</em> repair construct in the last step of the protocol. Animals were injected as above and Rol F1 progeny were genotyped using PCR and restriction enzyme digestion. As with the deletion alleles, edited strains were homozygosed and their genotypes were confirmed with Sanger sequencing. Reagents used to generate these allele replacement strains are detailed in the <strong>Supplemental Information</strong>.</p></div><div id="sec-15" class="subsection"><h3>Linkage mapping of expression QTL</h3><p id="p-20">Microarray data for 13,107 probes were collected for synchronized young adult populations of 209 recombinant lines previously (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Rockman <em>et al</em>. 2010</a>). We performed linkage mapping as described above on these expression data and identified significant peaks for 3,298 probes (<strong>File S13</strong>). For the <em>H19N07.3</em> probe (A_12_P104350), the full annotated-LOD data frame is provided (<strong>File S14</strong>).</p></div><div id="sec-16" class="subsection"><h3>Hemizygosity high-throughput assay</h3><p id="p-21">Hemizygosity assays were used to test the effect of zero (two deletion alleles), one (one deletion allele and one wild-type allele), or two (two wild-type alleles) functional copies of a gene product on bleomycin responses. If the phenotype is affected by gene function, one would expect to see bleomycin responses scale with the number of functional alleles of the gene present in each strain. For each heterozygous/hemizygous genotype, 30 hermaphrodites and 60 males were placed onto each of four 6 cm plates and allowed to mate for 48 hours. The same process was completed for homozygous strains to remove biases introduced by the presence of male progeny in the assay. Mated hermaphrodites were transferred to a clean 6 cm plate and allowed to lay embryos for eight hours. After the egg lay period, adults were manually removed from egg lay plates, and embryos were washed into 15 mL conicals using M9 and a combination of pasteur pipette rinsing and scraping with plastic inoculation loops. Embryos were rinsed and centrifuged four times with M9 before being resuspended in K medium and 50 µM kanamycin. Embryos hatched and arrested in the L1 larval stage in conicals overnight at 20° with shaking. The next morning, 50 L1 larvae were sorted into each well of a 96-well plate containing K medium, 10 mg/mL bacterial lysate, 50 µM kanamycin, and either 1% distilled water or 1% distilled water plus bleomycin using the BIOSORT (dose-response assay, <strong>Figure S10, File S1</strong>) or by titering (hemizygosity assays, <strong><a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>, Figure S11, Figure S12, File S1</strong>). Animals were incubated in these plates for 48 hours at 20° with shaking and were paralyzed with 50 µM sodium azide in M9 before being scored for phenotypic parameters using the BIOSORT.</p></div><div id="sec-17" class="subsection"><h3>Statistical analysis</h3><p id="p-22">All statistical tests of phenotypic differences in the NIL, deletion strain, and allele-replacement strain assays were performed in R (version 3.3.1) using the <em>TukeyHSD</em> function (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">R Core Team</a>) and an ANOVA model with phenotype as the dependent variable and strain as the independent variable (<em>phenotype ∼ strain</em>). The p-values of individual pairwise strain comparisons were reported, and p-values less than 0.05 were deemed significant. For each figure (with the exception of hemizygosity tests), phenotypes of NILs, deletion strains, and allele-replacement strains were compared to phenotypes of their respective background strains (either N2 or CB4856), and statistical significance is denoted by an asterisk above the boxplot for each strain. P-values of all pairwise comparisons are reported in <strong>File S7</strong>. Correlation between RIAIL <em>H19N07.3</em> expression and median optical density in bleomycin was tested using a Spearman’s correlation test. Statistical difference between N2 and CB4856 expression of <em>H19N07.3</em> measured by RNA-seq was tested using a likelihood-ratio and a Wald test with a threshold of <em>p</em> &lt; 0.05.</p></div><div id="sec-18" class="subsection"><h3>RNA-seq</h3><p id="p-23">Three independent replicates of RNA were sampled as follows. Bleach-synchronized embryos (∼2,000) of the N2 and CB4856 strains were grown on 10 cm plates of NGMA for 66-69 hours. When F1 embryos appeared on the plate, synchronized young adults were collected by washing each plate twice with M9 buffer and incubating for 30 minutes in M9 to remove remaining bacteria. Then, samples were washed twice again with M9 buffer, and then washed with sterile water. Animals were pelleted and homogenized in Trizol (Ambion) by repeating freezing-thawing with liquid nitrogen five times. To extract RNA from each sample, chloroform, isopropanol, and ethanol were used for phase separation, precipitation and washing steps, respectively. RNA pellets were resuspended in TE buffer, and RNA quality was measured with a 2100 Bioanalyzer (Agilent). Library preparation and RNA sequencing (HiSeq4000, Illumina) were performed by the Genomics Facility at the University of Chicago. RNA-seq data were quantified with kallisto and then within-sample and between-sample normalization was performed using sleuth, which is based on DESeq (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Anders and Huber 2010</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Bray <em>et al</em>. 2016</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Pimentel <em>et al</em>. 2017</a>). Significant differences between samples were determined by likelihood-ratio and Wald tests. RNA-seq data are reported in <strong>File S16</strong>.</p></div><div id="sec-19" class="subsection"><h3>Genome-wide association mapping</h3><p id="p-24">Bleomycin responses were measured for 83 <em>C. elegans</em> isotypes using the high-throughput fitness assay (<strong>File S18</strong>). Genome-wide association mapping was performed as described previously (<a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Zdraljevic <em>et al</em>. 2019</a>) using genotype data from CeNDR Release 20180527 (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Cook <em>et al</em>. 2017</a>). In short, BCFtools was used to remove variants with missing genotype calls and variants with a minor allele frequency below 5% (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Li 2011</a>), and PLINK v1.9 was used to prune the genotypes at a linkage disequilibrium threshold of r<sup>2</sup> &lt; 0.8 (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Purcell <em>et al</em>. 2007</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Chang <em>et al</em>. 2015</a>), for a total of 59,241 pruned markers. A kinship matrix was generated using the <em>A.mat</em> function in the <em>rrBLUP</em> R package (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Endelman 2011</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Endelman and Jannink 2012</a>). The <em>GWAS</em> function in the <em>rrBLUP</em> package was used to perform genome-wide association mapping with EMMA algorithm to correct for kinship (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Kang <em>et al</em>. 2008</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Covarrubias-Pazaran 2016</a>). The relatedness among these wild isolates was described previously (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Andersen <em>et al</em>. 2012</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Hahnel <em>et al</em>. 2018</a>; <a id="xref-ref-78-3" class="xref-bibr" href="#ref-78">Zdraljevic <em>et al</em>. 2019</a>).</p></div><div id="sec-20" class="subsection"><h3>Identification of rare variants</h3><p id="p-25">VCF release 20180527 was downloaded from elegansvariation.org (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Cook <em>et al</em>. 2017</a>). The VCF was filtered to select all variants within the linkage mapping confidence interval (V:11042745-11189364) where CB4856 contains the alternate allele (<strong>File S20</strong>). Variants with a minor allele frequency less than 0.05 within the 83 wild isolates that have a bleomycin median optical density measurement were deemed “rare”.</p></div><div id="sec-21" class="subsection"><h3>Creation of neighbor-joining tree</h3><p id="p-26">Protein sequences for homologs of the <em>C. elegans</em> H19N07.3 protein (<strong>File S21</strong>) were input to MUSCLE (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Edgar 2004a</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">b</a>) to generate a multiple-sequence alignment. CLUSTALW was then used to generate a neighbor-joining tree and output as a Newick formatted file (<strong>File S22</strong>).</p></div><div id="sec-22" class="subsection"><h3>Data availability</h3><p id="p-27">All data are available on Figshare. <strong>File S1</strong> contains all pruned data from the high-throughput bleomycin assays. <strong>File S2</strong> contains the broad-sense heritability estimates calculated for each drug concentration for all 26 HTA traits for the HTA dose response as well as for the 24 HTA traits in the modified HTA dose response. <strong>File S3</strong> contains all control-regressed data for the 26 HTA traits for all assays. <strong>File S4</strong> contains the annotated linkage mapping data for the 26 control-regressed HTA traits. <strong>File S5</strong> is a VCF that reports the genotype from whole-genome sequence for all NILs in the manuscript. <strong>File S6</strong> is a simplified version of <strong>File S5</strong> that contains information on recombination locations for all NILs and can be used for more user-friendly visualization of NIL genotypes. <strong>File S7</strong> contains all statistical information for HTA phenotypic differences reported in the manuscript. <strong>File S8</strong> is a summary of the scantwo analysis for bleomycin responses in the RIAILs and reports the maximum interaction LOD score for each chromosome pair. <strong>File S9</strong> contains information on all genes in the QTL confidence interval plus 20 kb on either side. <strong>File S10</strong> contains locations of the exons, introns, and transcription start and stop sites for all genes in the region. <strong>File S11</strong> reports predicted non-synonymous variants between the N2 and CB4856 strains in the region. <strong>File S12</strong> is derived from the <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Rockman <em>et al</em>. 2010</a> RIAIL microarray expression data, and reports the expression measurements for each of the 13,107 microarray probes across 209 RIAILs. <strong>File S13</strong> contains all significant QTL identified by linkage mapping of <strong>File S12</strong> data. <strong>File S14</strong> contains the annotated linkage mapping of the <em>H19N07.3</em> expression data. <strong>File S15</strong> reports the <em>H19N07.3</em> expression and residual median optical density for strains of the RIAIL panel that were assayed for both of those traits. <strong>File S16</strong> contains <em>H19N07.3</em> RNA-seq expression data for populations of young adults of N2 and CB4856. <strong>File S17</strong> is a summary of the scantwo analysis for <em>H19N07.3</em> expression in the RIAILs and reports the maximum interaction LOD score for each chromosome pair. <strong>File S18</strong> contains control-regressed phenotypic data for all wild isolates assayed in response to bleomycin. <strong>File S19</strong> contains genome-wide association mapping for the phenotypes in <strong>File S18</strong>. <strong>File S20</strong> contains genotype information for each strain measured in <strong>File S18</strong> across all variants within the linkage mapping confidence interval around the QTL for which CB4856 contains the alternate allele. <strong>File S21</strong> is a FASTA file containing the protein sequences for all <em>H19N07.3</em> homologs. <strong>File S22</strong> is a neighbor-joining tree derived from a multiple sequence alignment of all sequences from <strong>File S21</strong> in Newick tree format.</p></div></div><div class="section" id="sec-23"><h2 class="">RESULTS</h2><div id="sec-24" class="subsection"><h3>Genetic differences underlie bleomycin-response variation</h3><p id="p-28">Bleomycin causes double-stranded DNA breaks, which ultimately lead to cytotoxicity of rapidly dividing cell populations. Therefore, exposure to bleomycin can affect the development of <em>C. elegans</em> larvae as well as germ-cell production of adult animals. We used a high-throughput assay (HTA) to measure the effects of bleomycin on development and brood size (<strong>Figure S1,</strong> Materials and Methods). To determine the concentration of bleomycin that would maximize among-strain while minimizing within-strain phenotypic variation, we used the HTA to perform a dose-response assay. We assessed bleomycin responses for four divergent strains (N2, CB4856, JU258, and DL238) across each of 26 HTA traits (<strong>File S1, Figure S2</strong>). For each concentration of bleomycin, we calculated the broad-sense heritability of the traits (Materials and Methods) and found that heritability was maximized at 50 µM bleomycin (mean <em>H</em><sup>2</sup> across all traits = 0.58, <strong>File S2</strong>). Given these results, we exposed animals to 50 µM bleomycin for all future HTA experiments.</p><p id="p-29">Two of the strains used in the dose response assay, N2 and CB4856, have been extensively characterized at the genome level (<a id="xref-ref-74-3" class="xref-bibr" href="#ref-74">Wicks <em>et al</em>. 2001</a>; <a id="xref-ref-70-3" class="xref-bibr" href="#ref-70">Swan <em>et al</em>. 2002</a>; <a id="xref-ref-72-3" class="xref-bibr" href="#ref-72">Thompson <em>et al</em>. 2015</a>) and displayed divergent bleomycin responses (<strong>Figure S2</strong>). Recombinant inbred advanced intercross lines (RIAILs) were previously constructed between these two strains (<a id="xref-ref-57-4" class="xref-bibr" href="#ref-57">Rockman and Kruglyak 2009</a>; <a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Andersen <em>et al</em>. 2015</a>), and these RIAILs have been leveraged to identify genetic variants that cause phenotypic differences between the N2 and CB4856 strains (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Kammenga <em>et al</em>. 2007</a>; <a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Seidel <em>et al</em>. 2008</a>, <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">2011</a>; <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Palopoli <em>et al</em>. 2008</a>; <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Reddy <em>et al</em>. 2009</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">McGrath <em>et al</em>. 2009</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Bendesky <em>et al</em>. 2011</a>, <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">2012</a>; <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Andersen <em>et al</em>. 2014</a>; <a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Schmid <em>et al</em>. 2015</a>; <a id="xref-ref-76-3" class="xref-bibr" href="#ref-76">Zdraljevic <em>et al</em>. 2017</a>; <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Lee <em>et al</em>. 2017</a>). We used these RIAILs to identify genetic variants that contribute to differential bleomycin responses between the N2 and CB4856 strains. Using our HTA, we measured each of the 26 fitness parameters for 249 RIAILs (Materials and Methods, <strong>File S3</strong>). Correlations between each pairwise combination of the 26 HTA measurements revealed several clusters of highly correlated traits (<strong>Figure S3</strong>). Therefore, the summary statistics measured by the BIOSORT should not be considered independent traits for linkage mapping. We selected median optical density (median.EXT) for future analyses, which is related to both animal length and optical extinction, because this trait was highly correlated with many of the 26 HTA traits and was highly heritable (<em>H<sup>2</sup></em> = 0.73, <strong>File S2</strong>).</p></div><div id="sec-25" class="subsection"><h3>The QTL on the center of chromosome V strongly impacts bleomycin response</h3><p id="p-30">We performed linkage mapping on the residual median optical density measurements in bleomycin (<strong>Figure S4, Figure S5</strong>, <strong>File S4,</strong> Materials and Methods) and identified four significant quantitative trait loci (QTL, <strong><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a></strong>). The QTL on the center of chromosome V was highly significant (explained 43.58% of the total variation and 55.60% of the genetic variation) with a LOD score of 32.57, and it was detected for 25 of the 26 HTA traits (<strong>Figure S5, File S4</strong>). The QTL 95% confidence interval was approximately 147 kb. Strains with the N2 allele at the peak marker had a lower median optical density in bleomycin and were interpreted to be more sensitive than those RIAILs with the CB4856 allele (<strong><a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a></strong>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Linkage-mapping analysis of bleomycin-response variation is shown for residual median optical density in bleomycin. A. On the x-axis, each of 13,001 genomic markers, split by chromosome, were tested for correlation with phenotypic variation across the RIAIL panel. The log of the odds (LOD) score for each marker is reported on the y-axis. Each significant quantitative trait locus (QTL) is indicated by a red triangle at the peak marker, and a blue ribbon shows the 95% confidence interval around the peak marker. The total amount of phenotypic variance across the RIAIL panel explained by the genotype at each peak marker is shown as a percentage. B. Residual median optical density phenotypes (y-axis), split by allele at each QTL peak marker (x-axis), are shown. For each significant QTL, phenotypes of RIAILs that contain the N2 allele (orange) are compared to RIAILs that contain the CB4856 allele (blue). Phenotypes are shown as Tukey box plots with the phenotypes of each individual strain shown as points behind the box plots." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-598015653" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Linkage-mapping analysis of bleomycin-response variation is shown for residual median optical density in bleomycin. &lt;strong&gt;A.&lt;/strong&gt; On the x-axis, each of 13,001 genomic markers, split by chromosome, were tested for correlation with phenotypic variation across the RIAIL panel. The log of the odds (LOD) score for each marker is reported on the y-axis. Each significant quantitative trait locus (QTL) is indicated by a red triangle at the peak marker, and a blue ribbon shows the 95% confidence interval around the peak marker. The total amount of phenotypic variance across the RIAIL panel explained by the genotype at each peak marker is shown as a percentage. &lt;strong&gt;B.&lt;/strong&gt; Residual median optical density phenotypes (y-axis), split by allele at each QTL peak marker (x-axis), are shown. For each significant QTL, phenotypes of RIAILs that contain the N2 allele (orange) are compared to RIAILs that contain the CB4856 allele (blue). Phenotypes are shown as Tukey box plots with the phenotypes of each individual strain shown as points behind the box plots.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F1.medium.gif" width="440" height="341"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F1.medium.gif" width="440" height="341"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <p id="p-31" class="first-child">Linkage-mapping analysis of bleomycin-response variation is shown for residual median optical density in bleomycin. <strong>A.</strong> On the x-axis, each of 13,001 genomic markers, split by chromosome, were tested for correlation with phenotypic variation across the RIAIL panel. The log of the odds (LOD) score for each marker is reported on the y-axis. Each significant quantitative trait locus (QTL) is indicated by a red triangle at the peak marker, and a blue ribbon shows the 95% confidence interval around the peak marker. The total amount of phenotypic variance across the RIAIL panel explained by the genotype at each peak marker is shown as a percentage. <strong>B.</strong> Residual median optical density phenotypes (y-axis), split by allele at each QTL peak marker (x-axis), are shown. For each significant QTL, phenotypes of RIAILs that contain the N2 allele (orange) are compared to RIAILs that contain the CB4856 allele (blue). Phenotypes are shown as Tukey box plots with the phenotypes of each individual strain shown as points behind the box plots.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">We isolated this QTL in a controlled genetic background by generating near-isogenic lines (NILs, <strong>File S5</strong>, <strong>File S6</strong>) that each contain a genetic background derived from either the N2 or CB4856 strain and a region of chromosome V from the opposite parental genotype. We used the HTA to test these strains in response to bleomycin (<strong>File S3</strong>). The NIL with the N2 genotype across this QTL introgressed into the CB4856 background (ECA230) was statistically more sensitive to bleomycin than CB4856 (<strong>Figure S6, File S7</strong>, <em>p</em> = 1.3e-14, Tukey HSD). This phenotype indicated that the N2 genotype within the introgressed region (which includes the QTL confidence interval) confers sensitivity to bleomycin. However, the reciprocal NIL with the CB4856 locus introgressed into the N2 background (ECA232) had a bleomycin-response phenotype that was not significantly different from the N2 strain (<strong>Figure S6, File S7,</strong> <em>p</em> = 0.053, Tukey HSD), suggesting that interacting loci could underlie bleomycin responses in a background-dependent manner. We performed a two-factor genome scan to map potential epistatic loci but did not identify a significant interaction between the QTL on chromosome V and other loci (<strong>Figure S7, File S8</strong>). However, the failure to detect significant interacting QTL could be because we have too few recombinant strains or because too few replicates of each RIAIL were phenotyped. Alternatively, more than two loci might underlie the transgressive phenotype of ECA230 and a two-factor genome scan might not be able to capture this complexity.</p><p id="p-33">Nonetheless, because ECA230 recapitulated the expected QTL phenotype, we generated two NILs (ECA411 and ECA528) that narrowed this introgressed region to more precisely locate the causal variant (<strong>File S5, File S6</strong>). In addition, the N2 region on the left of chromosome V was removed from both NIL strains to ensure that this region of introgression did not underlie the phenotypic difference between ECA230 and CB4856. The genotypes of ECA411 and ECA528 differ in a small region of chromosome V that includes the QTL confidence interval (<strong><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, File S5, File S6</strong>). Both of these strains were more sensitive to bleomycin than the background parental strain, CB4856. This result could suggest that the introgressed region shared between these strains, which does not include the QTL, conferred some bleomycin-response variation between the N2 and CB4856 strains (<strong><a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a></strong>). Alternatively, the hypersensitivity of these NILs could suggest the presence of Dobzhansky-Muller incompatibilities between the N2 and CB4856 genotypes (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Snoek <em>et al</em>. 2014</a>) that might affect stress responses of the NILs. However, ECA528 was much more sensitive to bleomycin than ECA411 (<strong><a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>, File S7</strong>). Because ECA528 has the N2 genotype across the QTL region and ECA411 has the CB4856 genotype, these results suggest that the QTL genotype strongly affects bleomycin sensitivity (<strong><a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a></strong>, ECA528 vs. each other strain <em>p</em> &lt; 1e-14, Tukey HSD). The empirically defined region lies between 10,339,727 and 11,345,443 bp on chromosome V and fully encompasses the linkage mapping confidence interval (from 11,042,745 to 11,189,364 bp on chromosome V).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phenotypes and genotypes of near-isogenic lines (NILs) are shown. (A) Phenotypes for each strain are shown as Tukey box plots, with strain name on the y-axis and residual bleomycin median optical density on the x-axis. Each point is a biological replicate. Parental strain box plots are colored by their genetic background, with orange indicating an N2 background and blue indicating a CB4856 genetic background. NILs are shown as grey box plots. A red asterisk indicates a significant difference between the phenotype of a given strain and the phenotype of the corresponding parental strain (p &lt; 0.05, Tukey HSD). (B) Chromosomal representations of chromosome V are shown for each of the strains in A. Strain names are reported on the y-axis, and genomic position (Mb) is shown on the x-axis. Blocks of color indicate genotypes of genomic regions with orange indicating the N2 genotype and blue indicating the CB4856 genotype. Vertical red lines mark the confidence interval of the QTL from linkage mapping. (C) Background genotypes are represented as rectangles with colors indicating N2 (orange) or CB4856 (blue) genetic backgrounds." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-598015653" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phenotypes and genotypes of near-isogenic lines (NILs) are shown. (&lt;strong&gt;A&lt;/strong&gt;) Phenotypes for each strain are shown as Tukey box plots, with strain name on the y-axis and residual bleomycin median optical density on the x-axis. Each point is a biological replicate. Parental strain box plots are colored by their genetic background, with orange indicating an N2 background and blue indicating a CB4856 genetic background. NILs are shown as grey box plots. A red asterisk indicates a significant difference between the phenotype of a given strain and the phenotype of the corresponding parental strain (&lt;em&gt;p&lt;/em&gt; &lt; 0.05, Tukey HSD). (&lt;strong&gt;B&lt;/strong&gt;) Chromosomal representations of chromosome V are shown for each of the strains in &lt;strong&gt;A&lt;/strong&gt;. Strain names are reported on the y-axis, and genomic position (Mb) is shown on the x-axis. Blocks of color indicate genotypes of genomic regions with orange indicating the N2 genotype and blue indicating the CB4856 genotype. Vertical red lines mark the confidence interval of the QTL from linkage mapping. (&lt;strong&gt;C&lt;/strong&gt;) Background genotypes are represented as rectangles with colors indicating N2 (orange) or CB4856 (blue) genetic backgrounds.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F2.medium.gif" width="440" height="271"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F2.medium.gif" width="440" height="271"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <p id="p-34" class="first-child">Phenotypes and genotypes of near-isogenic lines (NILs) are shown. (<strong>A</strong>) Phenotypes for each strain are shown as Tukey box plots, with strain name on the y-axis and residual bleomycin median optical density on the x-axis. Each point is a biological replicate. Parental strain box plots are colored by their genetic background, with orange indicating an N2 background and blue indicating a CB4856 genetic background. NILs are shown as grey box plots. A red asterisk indicates a significant difference between the phenotype of a given strain and the phenotype of the corresponding parental strain (<em>p</em> &lt; 0.05, Tukey HSD). (<strong>B</strong>) Chromosomal representations of chromosome V are shown for each of the strains in <strong>A</strong>. Strain names are reported on the y-axis, and genomic position (Mb) is shown on the x-axis. Blocks of color indicate genotypes of genomic regions with orange indicating the N2 genotype and blue indicating the CB4856 genotype. Vertical red lines mark the confidence interval of the QTL from linkage mapping. (<strong>C</strong>) Background genotypes are represented as rectangles with colors indicating N2 (orange) or CB4856 (blue) genetic backgrounds.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-26" class="subsection"><h3>Genes with non-synonymous variants in the QTL region do not impact bleomycin responses</h3><p id="p-35">Because the recombination rate in the centers of <em>C. elegans</em> chromosomes is lower than chromosome arms (<a id="xref-ref-57-5" class="xref-bibr" href="#ref-57">Rockman and Kruglyak 2009</a>), it was difficult to generate additional NILs to narrow the QTL region further. Therefore, we took a targeted approach and created CRISPR-Cas9 directed modifications of candidate genes in the region. The 147 kb confidence interval on chromosome V contains 93 genes, including pseudogenes, piRNA, miRNA, ncRNA, and protein-coding genes (<strong>File S9</strong>). Given the narrow confidence interval, we expanded our search to include an additional 20 kb on each side of the 147 kb interval (Materials and Methods). Of the 118 genes included in the wider region, five genes, <em>C45B11.8</em>, <em>C45B11.6</em>, <em>jmjd-5</em>, <em>srg-42</em>, and <em>cnc-10</em>, contain predicted non-synonymous variants between the N2 and CB4856 strains (<strong><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a></strong>, <strong>File S10</strong>, <strong>File S11</strong>). These variants could cause differential bleomycin sensitivity between the N2 and CB4856 strains.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bleomycin responses of the deletion alleles for each candidate gene are shown. A. The linkage mapping QTL confidence interval (light blue) with 20 kb on the left and the right is displayed. Each protein-coding gene in the region is indicated by an arrow that points in the direction of transcription. Genes with non-synonymous variants between the N2 and CB4856 strains are shown as red arrows and are labeled with their gene name. B. Deletion alleles for each of these genes were tested in response to bleomycin. Bleomycin responses are shown as Tukey box plots, with the strain name on the x-axis, split by gene, and residual median optical density on the y-axis. Each point is a biological replicate. Strains are colored by their background genotype (orange indicates an N2 genetic background, and blue indicates a CB4856 genetic background). For each gene, two independent deletion alleles in each background were created and tested. Red asterisks indicate a significant difference (p &lt; 0.05, Tukey HSD) between a strain with a deletion and the parental strain that has the same genetic background. Depictions of each deletion allele are shown below the phenotype for each candidate gene. White rectangles indicate exons and diagonal lines indicate introns. The 5&#x2019; and 3&#x2019; UTRs are shown by grey rectangles and triangles, respectively. The region of the gene that was deleted by CRISPR-Cas9 directed genome editing is shown as a red bar beneath each gene model." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-598015653" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Bleomycin responses of the deletion alleles for each candidate gene are shown. &lt;strong&gt;A.&lt;/strong&gt; The linkage mapping QTL confidence interval (light blue) with 20 kb on the left and the right is displayed. Each protein-coding gene in the region is indicated by an arrow that points in the direction of transcription. Genes with non-synonymous variants between the N2 and CB4856 strains are shown as red arrows and are labeled with their gene name. &lt;strong&gt;B&lt;/strong&gt;. Deletion alleles for each of these genes were tested in response to bleomycin. Bleomycin responses are shown as Tukey box plots, with the strain name on the x-axis, split by gene, and residual median optical density on the y-axis. Each point is a biological replicate. Strains are colored by their background genotype (orange indicates an N2 genetic background, and blue indicates a CB4856 genetic background). For each gene, two independent deletion alleles in each background were created and tested. Red asterisks indicate a significant difference (&lt;em&gt;p&lt;/em&gt; &lt; 0.05, Tukey HSD) between a strain with a deletion and the parental strain that has the same genetic background. Depictions of each deletion allele are shown below the phenotype for each candidate gene. White rectangles indicate exons and diagonal lines indicate introns. The 5&#x2019; and 3&#x2019; UTRs are shown by grey rectangles and triangles, respectively. The region of the gene that was deleted by CRISPR-Cas9 directed genome editing is shown as a red bar beneath each gene model.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F3.medium.gif" width="440" height="337"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F3.medium.gif" width="440" height="337"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <p id="p-36" class="first-child">Bleomycin responses of the deletion alleles for each candidate gene are shown. <strong>A.</strong> The linkage mapping QTL confidence interval (light blue) with 20 kb on the left and the right is displayed. Each protein-coding gene in the region is indicated by an arrow that points in the direction of transcription. Genes with non-synonymous variants between the N2 and CB4856 strains are shown as red arrows and are labeled with their gene name. <strong>B</strong>. Deletion alleles for each of these genes were tested in response to bleomycin. Bleomycin responses are shown as Tukey box plots, with the strain name on the x-axis, split by gene, and residual median optical density on the y-axis. Each point is a biological replicate. Strains are colored by their background genotype (orange indicates an N2 genetic background, and blue indicates a CB4856 genetic background). For each gene, two independent deletion alleles in each background were created and tested. Red asterisks indicate a significant difference (<em>p</em> &lt; 0.05, Tukey HSD) between a strain with a deletion and the parental strain that has the same genetic background. Depictions of each deletion allele are shown below the phenotype for each candidate gene. White rectangles indicate exons and diagonal lines indicate introns. The 5’ and 3’ UTRs are shown by grey rectangles and triangles, respectively. The region of the gene that was deleted by CRISPR-Cas9 directed genome editing is shown as a red bar beneath each gene model.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">To test these genes in bleomycin-response variation, we systematically deleted each of the candidate genes in both the N2 and CB4856 backgrounds. We used CRISPR-Cas9 mediated genome editing to generate two independent deletion alleles of each gene in each genetic background to reduce the possibility that off-target mutations could cause phenotypic differences (Materials and Methods, Supplemental Information). We tested the bleomycin response of each deletion allele in comparison to the N2 and CB4856 parental strains (<strong><a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>, File S3</strong>). The deletion alleles of <em>C45B11.8</em>, <em>C45B11.6</em>, <em>srg-42</em>, and <em>cnc-10</em> each had a bleomycin response similar to the respective parent genetic background, which suggested that the functions of each of these genes did not affect bleomycin responses (<strong><a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a></strong>, <strong>File S7</strong>, <em>p</em> &gt; 0.05, Tukey HSD). By contrast, the <em>jmjd-5</em> deletion alleles in the N2 and the CB4856 backgrounds were each more resistant to bleomycin than their respective parental strains (<strong><a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>, File S7</strong>, ECA1047 vs. CB4856 <em>p</em> = 3.8e-10, ECA1048 vs. CB4856 <em>p</em> = 0.026, ECA1051 vs. N2 <em>p</em> = 7.4e-4, ECA1052 vs. N2 <em>p</em> = 2.9e-6, Tukey HSD). However, we also noted that these strains were more sensitive in the control condition than other deletion strains (<strong>Figure S8, File S1, File S7</strong>). Therefore, the relative increased bleomycin resistance observed in the <em>jmjd-5</em> deletion strains could be caused by their increased sensitivity in the control condition.</p><p id="p-38">We tested if the non-synonymous variant in <em>jmjd-5</em> between the N2 and CB4856 strains caused bleomycin-response differences. At residue 338 of JMJD-5, the N2 strain has a proline, whereas the CB4856 strain has a serine (S338P, <strong>Figure S9A</strong>). We used CRISPR-Cas9 to generate reciprocal allele replacements of the <em>jmjd-5</em> single-nucleotide polymorphism that encodes the putative amino-acid change in the N2 background <em>jmjd-5(N2 to CB4856)</em> and in the CB4856 background <em>jmjd-5(CB4856 to N2)</em> (Materials and Methods, Supplemental Information). We created two independent allele replacements in each genetic background and measured each strain for bleomycin-response differences as compared to the parental strains (<strong>Figure S9B, File S3</strong>). Although the allele-replacement strains with the CB4856 allele in the N2 genetic background <em>jmjd-5(N2 to CB4856)</em> were significantly different from the N2 parental strain, these strains were more sensitive to bleomycin than N2 (<strong>Figure S9B, File S7</strong>, ECA576 vs. N2 <em>p</em> = 0.006, ECA577 vs. N2 <em>p</em> = 1.6e-6, Tukey HSD). This increased sensitivity was unexpected, because the CB4856 allele at the <em>jmjd-5</em> locus should confer resistance. However, the NIL with the CB4856 genotype across the QTL was not different from the N2 parental strain (ECA232 in <strong>Figure S6, File S7</strong>), suggesting that the QTL might only confer increased sensitivity when the N2 allele is in the CB4856 background. Therefore, it remained unclear whether an allele replacement of <em>jmjd-5</em> in the N2 parental background could confer resistance. Neither of the two strains with the N2 allele in the CB4856 background, <em>jmjd-5(CB4856 to N2)</em>, conferred a significantly more sensitive phenotype than the CB4856 parental strain (<strong>Figure S9B, File S7</strong>). Given that the QTL explained 43.58% of phenotypic variation among the RIAILs, the causal variant should have a clear impact on bleomycin response. Additionally, the NILs with the N2 allele at the QTL introgressed into the CB4856 background displayed a significant increase in bleomycin sensitivity compared to the parental CB4856 strain (<strong>Figure S6, File S7</strong>). Taken together, the phenotypes of the reciprocal allele-replacement strains showed that the amino-acid change in JMJD-5 likely does not underlie bleomycin-response variation between the N2 and CB4856 strains, although deletion of this gene did cause resistance to bleomycin regardless of the genetic background. We performed a reciprocal hemizygosity assay to test if natural variation in <em>jmjd-5</em> function affected bleomycin responses (<strong>Figure S10, Figure S11, Figure S12</strong>). The results of this assay supported the previously identified increase in bleomycin resistance of homozygous <em>jmjd-5</em> deletions in both parental backgrounds (<strong>Figure S11, File S7,</strong> <em>p</em> &lt; 0.05, Tukey HSD), which again might be caused by an increased sensitivity in the control condition (<strong>Figure S12</strong>). However, the increases in bleomycin resistance between each <em>jmjd-5</em> deletion strain and the strain with the same genetic background were similar, and the reciprocal hemizygous strains show equivalent bleomycin responses (<strong>Figure S11</strong>). Taken together, these results suggest that natural variation in <em>jmjd-5</em> function does not underlie this QTL.</p></div><div id="sec-27" class="subsection"><h3>The nematode-specific gene <em>H19N07.3</em> impacts bleomycin variation</h3><p id="p-39">Because none of the genes with a non-synonymous variant between the N2 and CB4856 strains explained the QTL, we explored other ways in which natural variation could impact bleomycin responses. We used the 13,001 SNPs to perform linkage mapping on the gene expression data of the RIAILs and identified 4,326 expression QTL (eQTL) across the genome (<a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">Rockman <em>et al</em>. 2010</a>, <strong>File S12, File S13</strong>, Materials and Methods). Of the 118 genes in the 187 kb surrounding the bleomycin-response QTL, expression for 50 genes were measured in the previous microarray study. We identified a significant eQTL for eight of these 50 genes, four of which mapped to chromosome V (<strong>File S13</strong>). eQTL for two of those four genes, <em>H19N07.3</em> and <em>cnc-10,</em> mapped to the center of chromosome V and overlapped with the bleomycin-response QTL. Because <em>cnc-10</em> did not underlie bleomycin response variation (<strong><a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3B</a></strong>), we hypothesized that <em>H19N07.3</em> might underlie the bleomycin-response QTL. The <em>H19N07.3</em> eQTL explains 45.70% of the variation in <em>H19N07.3</em> expression among the RIAILs (<strong><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A and B</a>, File S14</strong>). The length of animals and expression of <em>scb-1</em> was correlated in the RIAIL strains (<strong><a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4C</a>, File S15</strong>, <em>r</em><sup>2</sup>= 0.61, <em>p</em> &lt; 9.5e-13 Spearman’s correlation). Although this gene does not have a non-synonymous variant between the N2 and CB4856 strains, natural variation in gene expression could impact bleomycin responses.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Linkage mapping of the H19N07.3 expression difference among RIAILs is shown. A. On the x-axis, each of 13,001 genomic markers, split by chromosome, were tested for correlation with H19N07.3 expression variation across the RIAIL panel. The log of the odds (LOD) score for each marker is reported on the y-axis. The significant quantitative trait locus (QTL) is indicated by a red triangle at the peak marker, and a blue ribbon shows the 95% confidence interval around the peak marker. The total amount of expression variance across the RIAIL panel explained by the genotype at the peak marker is printed as a percentage. B. RIAIL gene expression (y-axis), split by allele at the QTL peak marker (x-axis) is shown. Phenotypes of RIAILs containing the N2 allele (orange) are compared to RIAILs containing the CB4856 allele (blue). Phenotypes are shown as Tukey box plots, and each point is the H19N07.3 expression of an individual strain. C. The correlation between animal size in bleomycin and H19N07.3 expression is shown as a scatterplot, with each RIAIL shown as a point. Each axis was scaled to have a mean of zero and a standard deviation of one. The line of best fit is shown in blue. The identity line is shown in black for reference." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-598015653" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Linkage mapping of the &lt;em&gt;H19N07.3&lt;/em&gt; expression difference among RIAILs is shown. &lt;strong&gt;A.&lt;/strong&gt; On the x-axis, each of 13,001 genomic markers, split by chromosome, were tested for correlation with &lt;em&gt;H19N07.3&lt;/em&gt; expression variation across the RIAIL panel. The log of the odds (LOD) score for each marker is reported on the y-axis. The significant quantitative trait locus (QTL) is indicated by a red triangle at the peak marker, and a blue ribbon shows the 95% confidence interval around the peak marker. The total amount of expression variance across the RIAIL panel explained by the genotype at the peak marker is printed as a percentage. &lt;strong&gt;B.&lt;/strong&gt; RIAIL gene expression (y-axis), split by allele at the QTL peak marker (x-axis) is shown. Phenotypes of RIAILs containing the N2 allele (orange) are compared to RIAILs containing the CB4856 allele (blue). Phenotypes are shown as Tukey box plots, and each point is the &lt;em&gt;H19N07.3&lt;/em&gt; expression of an individual strain. &lt;strong&gt;C.&lt;/strong&gt; The correlation between animal size in bleomycin and &lt;em&gt;H19N07.3&lt;/em&gt; expression is shown as a scatterplot, with each RIAIL shown as a point. Each axis was scaled to have a mean of zero and a standard deviation of one. The line of best fit is shown in blue. The identity line is shown in black for reference.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F4.medium.gif" width="440" height="327"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F4.medium.gif" width="440" height="327"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4:</span> <p id="p-40" class="first-child">Linkage mapping of the <em>H19N07.3</em> expression difference among RIAILs is shown. <strong>A.</strong> On the x-axis, each of 13,001 genomic markers, split by chromosome, were tested for correlation with <em>H19N07.3</em> expression variation across the RIAIL panel. The log of the odds (LOD) score for each marker is reported on the y-axis. The significant quantitative trait locus (QTL) is indicated by a red triangle at the peak marker, and a blue ribbon shows the 95% confidence interval around the peak marker. The total amount of expression variance across the RIAIL panel explained by the genotype at the peak marker is printed as a percentage. <strong>B.</strong> RIAIL gene expression (y-axis), split by allele at the QTL peak marker (x-axis) is shown. Phenotypes of RIAILs containing the N2 allele (orange) are compared to RIAILs containing the CB4856 allele (blue). Phenotypes are shown as Tukey box plots, and each point is the <em>H19N07.3</em> expression of an individual strain. <strong>C.</strong> The correlation between animal size in bleomycin and <em>H19N07.3</em> expression is shown as a scatterplot, with each RIAIL shown as a point. Each axis was scaled to have a mean of zero and a standard deviation of one. The line of best fit is shown in blue. The identity line is shown in black for reference.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">We created two independent CRISPR-Cas9 mediated deletion alleles of <em>H19N07.3</em> in the N2 and the CB4856 backgrounds and measured the bleomycin responses of these strains compared to the parental strains (<strong><a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, File S3, File S7,</strong> Supplemental Information, Materials and Methods). Each <em>H19N07.3</em> deletion strain was more sensitive to bleomycin than the respective parental strain (<strong><a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>, File S3, File S7,</strong> ECA1133 vs. CB4856 <em>p</em> &lt; 1.4e-14, ECA1134 vs. CB4856, <em>p</em> &lt; 1.4e-14, ECA1132 vs. N2, <em>p</em> = 6.9e-5, ECA1135 vs. N2, <em>p</em> = 0.006, Tukey HSD). These results suggest that <em>H19N07.3</em> function is required for resistance to bleomycin. Therefore, we renamed this gene <em>scb-1</em> for sensitivity to the chemotherapeutic bleomycin. Unlike with the <em>jmjd-5</em> deletion strains, the <em>scb-1</em> deletion strains had no significant differences in the control condition (<strong>File S7</strong>). Therefore, the bleomycin sensitivity of the <em>scb-1</em> deletion strains were not caused by control-condition phenotypes.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bleomycin responses of H19N07.3 deletion alleles are shown as Tukey box plots, with the strain name on the x-axis, split by genotype, and residual median optical density on the y-axis. Each point is a biological replicate. Strains are colored by their background genotype (orange indicates an N2 genetic background, and blue indicates a CB4856 genetic background). Two independent deletion alleles in each genetic background were created and tested. Red asterisks indicate a significant difference (p &lt; 0.05, Tukey HSD) between a strain with a deletion and the parental strain that has the same genetic background. A depiction of the deletion allele is shown below the box plots. White rectangles indicate exons, and diagonal lines indicate introns. The 5&#x2019; and 3&#x2019; UTRs are shown by grey rectangles and triangles, respectively. The region of the gene that was deleted by CRISPR-Cas9 directed genome editing is shown as a red bar beneath the gene model." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-598015653" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Bleomycin responses of &lt;em&gt;H19N07.3&lt;/em&gt; deletion alleles are shown as Tukey box plots, with the strain name on the x-axis, split by genotype, and residual median optical density on the y-axis. Each point is a biological replicate. Strains are colored by their background genotype (orange indicates an N2 genetic background, and blue indicates a CB4856 genetic background). Two independent deletion alleles in each genetic background were created and tested. Red asterisks indicate a significant difference (&lt;em&gt;p&lt;/em&gt; &lt; 0.05, Tukey HSD) between a strain with a deletion and the parental strain that has the same genetic background. A depiction of the deletion allele is shown below the box plots. White rectangles indicate exons, and diagonal lines indicate introns. The 5&#x2019; and 3&#x2019; UTRs are shown by grey rectangles and triangles, respectively. The region of the gene that was deleted by CRISPR-Cas9 directed genome editing is shown as a red bar beneath the gene model.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F5.medium.gif" width="440" height="220"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F5.medium.gif" width="440" height="220"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5:</span> <p id="p-42" class="first-child">Bleomycin responses of <em>H19N07.3</em> deletion alleles are shown as Tukey box plots, with the strain name on the x-axis, split by genotype, and residual median optical density on the y-axis. Each point is a biological replicate. Strains are colored by their background genotype (orange indicates an N2 genetic background, and blue indicates a CB4856 genetic background). Two independent deletion alleles in each genetic background were created and tested. Red asterisks indicate a significant difference (<em>p</em> &lt; 0.05, Tukey HSD) between a strain with a deletion and the parental strain that has the same genetic background. A depiction of the deletion allele is shown below the box plots. White rectangles indicate exons, and diagonal lines indicate introns. The 5’ and 3’ UTRs are shown by grey rectangles and triangles, respectively. The region of the gene that was deleted by CRISPR-Cas9 directed genome editing is shown as a red bar beneath the gene model.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">Because an <em>scb-1</em> non-synonymous variant has not been identified between the N2 and CB4856 strains, changes to protein function likely do not cause bleomycin response differences. RIAILs with the CB4856 allele at the QTL peak marker have increased expression of <em>scb-1</em> and increased bleomycin resistance compared to RIAILs with the N2 allele (<strong><a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a></strong>, <strong><a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a></strong>). Therefore, <em>scb-1</em> expression differences might cause the bleomycin-response variation between the parental strains. We performed RNA-seq of the N2 and CB4856 strains to assess <em>scb-1</em> expression differences between the parental strains and did not identify a significant increase in expression in the CB4856 strain (<strong>Figure S13, File S16,</strong> <em>p</em> = 0.20, Wald test; <em>p</em> = 0.17, likelihood ratio test). This result could be caused by the low sample size (n = 3) in the RNA-seq experiment, or the RIAIL strains could have a novel variant that arose during strain construction that causes <em>scb-1</em> expression variation. Alternatively, the expression difference observed in the RIAIL strains could be attributed to epistatic loci. We performed a two-factor genome scan to identify epistatic loci that underlie <em>scb-1</em> expression variation in the RIAILs and identified two significant interactions: one between loci on chromosomes IV and X and another between loci on chromosomes II and V (<strong>Figure S14, File S17</strong>). This result might suggest that epistatic loci underlie <em>scb-1</em> expression variation in the RIAILs and could explain why <em>scb-1</em> expression is not variable in the parental strains.</p><p id="p-44">To test the role of natural variation in <em>scb-1</em> function, we performed a reciprocal hemizygosity test in control and bleomycin conditions (<strong><a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>, File S3</strong>). These results matched the increase in sensitivity of homozygous deletions in both parental backgrounds observed previously. The hemizygous strain with the CB4856 allele of <em>scb-1</em> had a bleomycin phenotype similar to the heterozygous strain, whereas the hemizygous strain with the N2 allele of <em>scb-1</em> was more sensitive to bleomycin than the heterozygous strain. Taken together, these results suggest that natural variation in <em>scb-1</em> function underlies the bleomycin-response difference between the N2 and CB4856 strains.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results of the scb-1 reciprocal hemizygosity assay are shown. The y-axis shows the residual median optical density for each strain reported along the x-axis. Bleomycin responses are reported as Tukey box plots where each point is a biological replicate. The genotypes of each strain are shown as colored rectangles beneath each box plot, where each rectangle represents a homolog (orange rectangles are an N2 genotype, and blue rectangles are a CB4856 genotype). Maternal homologs are shown on top and paternal homologs are shown on bottom. Grey triangles indicate a deletion of scb-1, placed on the rectangle showing the background into which the deletion was introduced. The box plots for the parental strains (N2 and CB4856, on the left) are colored according to genotype." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-598015653" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Results of the &lt;em&gt;scb-1&lt;/em&gt; reciprocal hemizygosity assay are shown. The y-axis shows the residual median optical density for each strain reported along the x-axis. Bleomycin responses are reported as Tukey box plots where each point is a biological replicate. The genotypes of each strain are shown as colored rectangles beneath each box plot, where each rectangle represents a homolog (orange rectangles are an N2 genotype, and blue rectangles are a CB4856 genotype). Maternal homologs are shown on top and paternal homologs are shown on bottom. Grey triangles indicate a deletion of &lt;em&gt;scb-1,&lt;/em&gt; placed on the rectangle showing the background into which the deletion was introduced. The box plots for the parental strains (N2 and CB4856, on the left) are colored according to genotype.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F6.medium.gif" width="440" height="216"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6:" src="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F6.medium.gif" width="440" height="216"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6:</span> <p id="p-45" class="first-child">Results of the <em>scb-1</em> reciprocal hemizygosity assay are shown. The y-axis shows the residual median optical density for each strain reported along the x-axis. Bleomycin responses are reported as Tukey box plots where each point is a biological replicate. The genotypes of each strain are shown as colored rectangles beneath each box plot, where each rectangle represents a homolog (orange rectangles are an N2 genotype, and blue rectangles are a CB4856 genotype). Maternal homologs are shown on top and paternal homologs are shown on bottom. Grey triangles indicate a deletion of <em>scb-1,</em> placed on the rectangle showing the background into which the deletion was introduced. The box plots for the parental strains (N2 and CB4856, on the left) are colored according to genotype.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-28" class="subsection"><h3>Differences in <em>scb-1</em> function might be regulated by a rare variant</h3><p id="p-46">The <em>scb-1</em> natural variant that underlies the bleomycin-response differences remains unknown. Because this gene does not have a predicted non-synonymous variant between the N2 and CB4856 strains, <em>scb-1</em> gene expression might underlie bleomycin response differences. Potential candidate variants that could cause this expression difference include one variant two kilobases upstream of the gene and one variant in the third intron of <em>scb-1</em>. However, gene expression can be regulated by distant loci, so the identification of the specific variant is difficult. To understand whether natural variation of <em>scb-1</em> underlies bleomycin-response differences in other strains, we compared the bleomycin-response linkage mapping to a genome-wide association mapping (GWA). We used the HTA to measure median optical density in bleomycin for 83 divergent wild isolates and performed GWA mapping (<strong><a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>, File S18, File S19</strong>). Six QTL were identified from the GWA, but none of these QTL regions overlapped the QTL from linkage mapping (<strong><a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>, File S19</strong>). Therefore, the CB4856 strain might have a rare variant that underlies its increase in bleomycin resistance compared to the N2 strain.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A genome-wide association study for residual median optical density in bleomycin for 83 wild isolates is shown. On the x-axis, each genomic marker, split by chromosome, was tested for correlation with phenotypic variation across the wild isolates. The log10(p) value of these correlations are reported on the y-axis. Each marker that reached a significance threshold determined by eigenvalue decomposition of the SNP correlation matrix is colored in red. QTL regions of interest are indicated by blue regions surrounding the significant markers." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-598015653" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A genome-wide association study for residual median optical density in bleomycin for 83 wild isolates is shown. On the x-axis, each genomic marker, split by chromosome, was tested for correlation with phenotypic variation across the wild isolates. The log&lt;sub&gt;10&lt;/sub&gt;(&lt;em&gt;p&lt;/em&gt;) value of these correlations are reported on the y-axis. Each marker that reached a significance threshold determined by eigenvalue decomposition of the SNP correlation matrix is colored in red. QTL regions of interest are indicated by blue regions surrounding the significant markers.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F7.medium.gif" width="440" height="146"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7:" src="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F7.medium.gif" width="440" height="146"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/14/565218/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7:</span> <p id="p-47" class="first-child">A genome-wide association study for residual median optical density in bleomycin for 83 wild isolates is shown. On the x-axis, each genomic marker, split by chromosome, was tested for correlation with phenotypic variation across the wild isolates. The log<sub>10</sub>(<em>p</em>) value of these correlations are reported on the y-axis. Each marker that reached a significance threshold determined by eigenvalue decomposition of the SNP correlation matrix is colored in red. QTL regions of interest are indicated by blue regions surrounding the significant markers.</p><div class="sb-div caption-clear"></div></div></div><p id="p-48">We identified all single nucleotide variants (SNVs), small insertion/deletions (indels), and structural variants (SVs) present in these 83 strains for which the CB4856 strain contains the alternate allele compared to the N2 reference strain. We found 105 variants within the QTL confidence interval (79 SNVs, 26 indels, 0 SVs) for which the CB4856 strain contains the alternate allele (<strong>Figure S15, File S20)</strong>. We then identified SNVs and indels with a minor-allele frequency less than 5% within the 83 strains, because these low-frequency variants are likely to have insufficient power to map by GWA. Seventy-two of the 105 variants in the region were identified as rare variants that might underlie the bleomycin-response difference between the N2 and CB4856 strains (<strong>Figure S15, File S20</strong>). Twenty-eight of these rare variants were not unique to CB4856, and other strains in the wild isolate panel shared these variants. However, none of these variants showed phenotypic trends consistent with an alternate allele conferring resistance to bleomycin (<strong>Figure S16</strong>). Forty-four of the 72 rare variants were unique to CB4856 within this set of 83 strains. One or more of these 44 variants could underlie the bleomycin-response QTL, but further work must be performed to identify which, if any, of these variants underlies the <em>scb-1</em> bleomycin-response difference between N2 and CB4856.</p><p id="p-49">We searched for homologs of <em>scb-1</em> in other species using a BLASTp search (<a href="http://www.wormbase.org">www.wormbase.org</a>, Release WS268) and identified homologs in nine other <em>Caenorhabditis</em> species but none outside of the Nematoda phylum (<strong>Figure S17</strong>, <strong>File S21, File S22</strong>) (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Edgar 2004a</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">b</a>). None of the homologs of SCB-1 have previously identified functions. We used Phyre2 to predict protein domains within the SCB-1 protein and were unable to detect any functional domains by sequence homology. Twenty-three percent of the SCB-1 protein sequence matched a hydrolase of a Middle East respiratory syndrome-related coronavirus (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Zhang <em>et al</em>. 2018</a>). However, the low confidence of the model (21.5%) should be considered before making conclusions about the function of <em>scb-1</em> based on these results.</p></div></div><div class="section" id="sec-29"><h2 class="">DISCUSSION</h2><p id="p-50">Here, we performed linkage mapping of bleomycin-response variation and identified a highly significant QTL on chromosome V. We tested all six candidate genes in the QTL region to identify a causal gene that underlies bleomycin-response variation between two divergent strains. Deletions of four of these genes, <em>C45B11.8</em>, <em>C45B11.6</em>, <em>srg-42</em>, and <em>cnc-10</em> did not impact bleomycin responses. Deletions in one gene, <em>jmjd-5</em>, showed increased bleomycin resistance in both parental backgrounds. However, we concluded that the QTL cannot be explained by differences in <em>jmjd-5</em> after further analysis of allele-replacement strains and hemizygosity tests. Deletions in a gene with an expression difference, <em>scb-1</em> (previously named <em>H19N07.3</em>), caused an increase in bleomycin sensitivity in both the N2 and the CB4856 strains. Results from a reciprocal hemizygosity assay indicated that natural variation in <em>scb-1</em> function caused differences in bleomycin responses between the N2 and CB4856 strains. Because loss-of-function alleles in <em>scb-1</em> caused increased bleomycin sensitivity (<strong><a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a></strong>) and the RIAILs with lower <em>scb-1</em> expression levels show increased bleomycin sensitivity (<strong><a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a></strong>), natural differences in <em>scb-1</em> expression might cause the bleomycin-response variation between the N2 and CB4856 strains.</p><p id="p-51">The function of <em>scb-1</em>, and particularly how it regulates bleomycin response, remains unknown. A previous study found that RNAi of <em>scb-1</em> impaired the DAF-16/FOXO-induced lifespan extension of <em>daf-2(e1370ts)</em> mutants, which suggests that <em>scb-1</em> might play a role in stress response (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Riedel <em>et al</em>. 2013</a>). Because bleomycin causes double-stranded DNA breaks and introduces oxidative stress to cells (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Stubbe and Kozarich 1987</a>), reduction of <em>scb-1</em> function might inhibit the ability of an animal to respond to bleomycin. This model is in agreement with our observation that <em>scb-1</em> deletions and RIAILs with lower <em>scb-1</em> expression are sensitive to bleomycin (<strong><a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a></strong>, <strong><a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a></strong>). We used the amino acid sequence of SCB-1 to query the Phyre2 database and found weak homology to a viral hydrolase (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Kelley <em>et al</em>. 2015</a>; <a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">Zhang <em>et al</em>. 2018</a>). This result could suggest that SCB-1 might function as a hydrolase, which could be the mechanism by which <em>scb-1</em> regulates cellular stress. This finding would be similar to clinical studies that have suggested a role of bleomycin hydrolase (BLMH) in bleomycin-response variation (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Lazo and Humphreys 1983</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Nuver <em>et al</em>. 2005</a>; de Haas <em>et al</em>. 2008; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Gu <em>et al</em>. 2011</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Altés <em>et al</em>. 2013</a>). Because <em>scb-1</em> is expressed in the nucleus of all somatic cells, this gene might impact the ability of bleomycin to cause DNA damage within the cell nucleus (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Turek <em>et al</em>. 2016</a>). Alternatively, <em>scb-1</em> could impact bleomycin import, export, or another mechanism. If the mechanism of <em>scb-1</em> is conserved in humans, this discovery could offer insights into the clinical applications of bleomycin. Our results also suggested the presence of genes that interact with <em>scb-1</em> to cause bleomycin-response differences. These interacting genes could be conserved in humans and therefore inform the use of bleomycin in the clinic.</p><p id="p-52">Despite its lack of conservation in humans, the SCB-1 protein is homologous to other proteins in other nematode species. Bleomycin is produced by the soil bacterium, <em>Streptomyces verticillus (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Calcutt and Schmidt 1994</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Du et al. 2000</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Shen et al. 2002</a>)</em>, which might be found in association with nematodes such as <em>C. elegans</em> in the wild (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Samuel <em>et al</em>. 2016</a>). A shared niche between <em>C. elegans</em> and <em>S. verticillus</em> could cause bleomycin resistance to be selected. Additionally, the CB4856 wild isolate is more resistant to bleomycin than the laboratory-adapted strain, N2. In fact, the N2 strain is the most sensitive to bleomycin across all strains tested in our HTA (<strong>Figure S16</strong>), which could indicate that bleomycin resistance is beneficial for wild isolates. Given its potential role in the highly conserved insulin-like pathway, <em>scb-1</em> could be beneficial in responses to multiple toxins. Interestingly, the <em>scb-1</em> gene lies within a toxin-response QTL hotspot on chromosome V (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Evans <em>et al</em>. 2018</a>). Understanding the mechanism of the role of <em>scb-1</em> in toxin responses might offer insights into evolutionary processes that shaped the genomic diversity of <em>C. elegans</em> and other nematode species.</p><p id="p-53">Previous studies have leveraged both linkage mapping and GWA in <em>C. elegans</em> to identify genetic variants that underlie drug-response differences (<a id="xref-ref-76-4" class="xref-bibr" href="#ref-76">Zdraljevic <em>et al</em>. 2017</a>, <a id="xref-ref-78-4" class="xref-bibr" href="#ref-78">2019</a>). In each of these studies, drug-response QTL overlap between linkage mapping and GWA, and variants in common between both mapping strain sets have been shown to underlie drug-response QTL. In the case of the bleomycin response, the linkage-mapping QTL did not overlap with the QTL identified through GWA. Therefore, the variant that underlies the QTL likely is not present at an allele frequency above 5% in the panel of wild isolates used for the bleomycin GWA. The difference between linkage mapping and GWA results indicates that both rare and common natural variants underlie bleomycin-response variation.</p><p id="p-54">This study emphasizes the power of the <em>C. elegans</em> model system to dissect complex traits. Although linkage mapping detected a highly significant QTL, the manner in which genetic components affect bleomycin responses is not simple. Certain near-isogenic lines showed transgressive phenotypes (<strong><a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>, Figure S6</strong>), which indicates that multiple loci interact to create extreme bleomycin sensitivity in particular strains with mixed genetic backgrounds. Our attempts to identify epistatic loci that underlie bleomycin responses in the RIAILs were unsuccessful, potentially because of the complexity of these epistatic interactions. Despite this complexity, <em>scb-1</em> deletions showed increased bleomycin sensitivity in both parental backgrounds, and expression variation among the RIAIL panel mapped to the same locus as the bleomycin response QTL. Interestingly, the parental strains do not seem to vary in <em>scb-1</em> expression, as measured by RNA-seq (<strong>Figure S13</strong>). We found evidence of epistatic loci that underlie <em>scb-1</em> expression variation in the RIAILs, which might explain why the parental strains do not differ in <em>scb-1</em> expression (<strong>Figure S14</strong>). Additional complexities of this trait include the lack of overlap between GWA and linkage mapping QTL and the potential effect of <em>jmjd-5</em> loss-of-function on bleomycin responses. Despite the complicated manner in which genetic variants seem to affect bleomycin responses, we leveraged the powerful model of <em>C. elegans</em> to identify a single gene that underlies this complex trait.</p></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGEMENTS</h2><p id="p-55">This work was supported by an American Cancer Society Research Scholar Grant to ECA (127313-RSG-15-135-01-DD) to ECA. Additionally, SCB received support from the Biotechnology Training Program training grant (T32GM008449) and the Dr. John N. Nicholson Fellowship. SZ received support from the Cell and Molecular Basis of Disease training grant (T32GM008061) and The Bernard and Martha Rappaport Fellowship. KWB received support from a Northwestern Undergraduate Research Grant. DEC was supported by the National Science Foundation Graduate Research Fellowship (DGE-1324585). YW was supported as a joint PhD student by China Scholarship Council (No. 201706910052). We would like to thank members of the Andersen Lab for helpful comments on the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-update fn-group-summary-of-updates" id="fn-1"><p id="p-4">We added a two-factor genomic scan to identify epistatic loci that underlie bleomycin responses and scb-1 expression variation. We edited the discussion to highlight the power of our study and its implications for understanding evolutionary processes. Additionally, we made edits to improve clarity and flow of the manuscript.</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">REFERENCES</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-565218v2.1"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A. K.</span> <span class="cit-name-surname">Corsi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Wightman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Chalfie</span></span>, <span class="cit-pub-date">2015</span> <span class="cit-chapter-title">A transparent window into biology: A primer on Caenorhabditis elegans</span>, in <span class="cit-source">WormBook</span>, edited by <span class="cit-publ-name">The C. elegans Research Community</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Altés</span>  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Paré</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Esquirol</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Xicoy</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Rámila</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Pharmacogenetic analysis in the treatment of Hodgkin lymphoma</span>. <abbr class="cit-jnl-abbrev">Leuk. Lymphoma</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">1706</span>–<span class="cit-lpage">1712</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLeuk.%2BLymphoma%26rft.volume%253D54%26rft.spage%253D1706%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.3" data-doi="10.1186/gb-2010-11-10-r106"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Huber</span></span>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Differential expression analysis for sequence count data</span>. <abbr class="cit-jnl-abbrev">Genome Biol</abbr>. <span class="cit-vol">11</span>: <span class="cit-fpage">R106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DAnders%26rft.auinit1%253DS.%26rft.volume%253D11%26rft.issue%253D10%26rft.spage%253DR106%26rft.epage%253DR106%26rft.atitle%253DDifferential%2Bexpression%2Banalysis%2Bfor%2Bsequence%2Bcount%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-10-r106%26rft_id%253Dinfo%253Apmid%252F20979621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-10-r106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20979621&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.4" data-doi="10.1038/ng.1050"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">E. C.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Gerke</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Shapiro</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Crissman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Ghosh</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Chromosome-scale selective sweeps shape <em>Caenorhabditis elegans</em> genomic diversity</span>. <abbr class="cit-jnl-abbrev">Nat. Genet</abbr>. <span class="cit-vol">44</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">290</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAndersen%26rft.auinit1%253DE.%2BC.%26rft.volume%253D44%26rft.issue%253D3%26rft.spage%253D285%26rft.epage%253D290%26rft.atitle%253DChromosome-scale%2Bselective%2Bsweeps%2Bshape%2BCaenorhabditis%2Belegans%2Bgenomic%2Bdiversity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.1050%26rft_id%253Dinfo%253Apmid%252F22286215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.1050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22286215&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.5" data-doi="10.1371/journal.pgen.1004156"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">E. C.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Bloom</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Gerke</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Kruglyak</span></span>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A variant in the neuropeptide receptor npr-1 is a major determinant of <em>Caenorhabditis elegans</em> growth and physiology</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr>. <span class="cit-vol">10</span>: <span class="cit-fpage">e1004156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet%26rft.volume%253D10%26rft.spage%253De1004156%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004156%26rft_id%253Dinfo%253Apmid%252F24586193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24586193&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">E. C.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. C.</span> <span class="cit-name-surname">Shimko</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Crissman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Ghosh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Bloom</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A Powerful New Quantitative Genetics Platform</span>, <abbr class="cit-jnl-abbrev">Combining Caenorhabditis elegans High-Throughput Fitness Assays with a Large Collection of Recombinant Strains</abbr>. G3 <span class="cit-vol">5</span>: <span class="cit-fpage">911</span>–<span class="cit-lpage">920</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCombining%2BCaenorhabditis%2Belegans%2BHigh-Throughput%2BFitness%2BAssays%2Bwith%2Ba%2BLarge%2BCollection%2Bof%2BRecombinant%2BStrains%26rft.volume%253D5%26rft.spage%253D911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.7" data-doi="10.1038/nature09821"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bendesky</span>  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Tsunozaki</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. V.</span> <span class="cit-name-surname">Rockman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Kruglyak</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">C. I.</span> <span class="cit-name-surname">Bargmann</span></span>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Catecholamine receptor polymorphisms affect decision-making in <em>C. elegans</em></span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">472</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBendesky%26rft.auinit1%253DA.%26rft.volume%253D472%26rft.issue%253D7343%26rft.spage%253D313%26rft.epage%253D318%26rft.atitle%253DCatecholamine%2Breceptor%2Bpolymorphisms%2Baffect%2Bdecision-making%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09821%26rft_id%253Dinfo%253Apmid%252F21412235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09821&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21412235&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289724600032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.8" data-doi="10.1371/journal.pgen.1003157"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bendesky</span>  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Pitts</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. V.</span> <span class="cit-name-surname">Rockman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W. C.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.-W.</span> <span class="cit-name-surname">Tan</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Long-range regulatory polymorphisms affecting a GABA receptor constitute a quantitative trait locus (QTL) for social behavior in <em>Caenorhabditis elegans</em></span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr>. <span class="cit-vol">8</span>: <span class="cit-fpage">e1003157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet%26rft.volume%253D8%26rft.spage%253De1003157%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003157%26rft_id%253Dinfo%253Apmid%252F23284308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003157&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23284308&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.9" data-doi="10.1038/nature11867"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">J. S.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I. M.</span> <span class="cit-name-surname">Ehrenreich</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W. T.</span> <span class="cit-name-surname">Loo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.-L. V.</span> <span class="cit-name-surname">Lite</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Kruglyak</span></span>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Finding the sources of missing heritability in a yeast cross</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">494</span>: <span class="cit-fpage">234</span>–<span class="cit-lpage">237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D494%26rft.spage%253D234%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11867%26rft_id%253Dinfo%253Apmid%252F23376951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11867&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23376951&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000315137700039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.10" data-doi="10.1002/1097-0142(197304)31:4&lt;903::AID-CNCR2820310422&gt;3.0.CO;2-N"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Blum</span>  <span class="cit-name-given-names">R. H.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. K.</span> <span class="cit-name-surname">Carter</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Agre</span></span>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">A clinical review of bleomycin--a new antineoplastic agent</span>. <abbr class="cit-jnl-abbrev">Cancer</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">903</span>–<span class="cit-lpage">914</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%26rft.stitle%253DCancer%26rft.aulast%253DBlum%26rft.auinit1%253DR.%2BH.%26rft.volume%253D31%26rft.issue%253D4%26rft.spage%253D903%26rft.epage%253D914%26rft.atitle%253DA%2Bclinical%2Breview%2Bof%2Bbleomycin--a%2Bnew%2Bantineoplastic%2Bagent.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F1097-0142%2528197304%252931%253A4%253C903%253A%253AAID-CNCR2820310422%253E3.0.CO%253B2-N%26rft_id%253Dinfo%253Apmid%252F4122362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/1097-0142(197304)31:4&lt;903::AID-CNCR2820310422&gt;3.0.CO;2-N&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4122362&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1973P452800022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.11" data-doi="10.1007/978-1-61779-867-2_3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Boyd</span>  <span class="cit-name-given-names">W. A.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. V.</span> <span class="cit-name-surname">Smith</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Freedman</span></span>, <span class="cit-pub-date">2012</span> <span class="cit-article-title"><em>Caenorhabditis elegans</em> as a model in developmental toxicology</span>. <abbr class="cit-jnl-abbrev">Methods Mol. Biol</abbr>. <span class="cit-vol">889</span>: <span class="cit-fpage">15</span>–<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DBoyd%26rft.auinit1%253DW.%2BA.%26rft.volume%253D889%26rft.spage%253D15%26rft.epage%253D24%26rft.atitle%253DCaenorhabditis%2Belegans%2Bas%2Ba%2Bmodel%2Bin%2Bdevelopmental%2Btoxicology.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-61779-867-2_3%26rft_id%253Dinfo%253Apmid%252F22669657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-61779-867-2_3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22669657&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.12" data-doi="10.1038/nbt.3519"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bray</span>  <span class="cit-name-given-names">N. L.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Pimentel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Melsted</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Pachter</span></span>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Near-optimal probabilistic RNA-seq quantification</span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol</abbr>. <span class="cit-vol">34</span>: <span class="cit-fpage">525</span>–<span class="cit-lpage">527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol%26rft.volume%253D34%26rft.spage%253D525%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3519%26rft_id%253Dinfo%253Apmid%252F27043002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3519&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27043002&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.13" data-doi="10.1051/medsci/20031967725"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brenner</span>  <span class="cit-name-given-names">S</span></span>., <span class="cit-pub-date">1974</span> <span class="cit-article-title">The genetics of <em>Caenorhabditis elegans</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBrenner%26rft.auinit1%253DS.%26rft.volume%253D77%26rft.issue%253D1%26rft.spage%253D71%26rft.epage%253D94%26rft.atitle%253DTHE%2BGENETICS%2BOF%2BCAENORHABDITIS%2BELEGANS%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fmedsci%252F20031967725%26rft_id%253Dinfo%253Apmid%252F4366476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6NzoiNzcvMS83MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA1LzE0LzU2NTIxOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.14" data-doi="10.1016/0378-1119(94)90627-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Calcutt</span>  <span class="cit-name-given-names">M. J.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">F. J.</span> <span class="cit-name-surname">Schmidt</span></span>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Gene organization in the bleomycin-resistance region of the producer organism <em>Streptomyces verticillus</em></span>. <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">17</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DCalcutt%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D151%26rft.issue%253D1-2%26rft.spage%253D17%26rft.epage%253D21%26rft.atitle%253DGene%2Borganization%2Bin%2Bthe%2Bbleomycin-resistance%2Bregion%2Bof%2Bthe%2Bproducer%2Borganism%2BStreptomyces%2Bverticillus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1119%252894%252990627-0%26rft_id%253Dinfo%253Apmid%252F7530225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1119(94)90627-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7530225&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994QC64800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.15" data-doi="10.1126/science.282.5396.2012"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">C. elegans Sequencing Consortium</span>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Genome sequence of the nematode <em>C. elegans</em>: a platform for investigating biology</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">2012</span>–<span class="cit-lpage">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DThe%2BC.%2Belegans%2BSequencing%2BConsortium%26rft.auinit1%253D%2B%26rft.volume%253D282%26rft.issue%253D5396%26rft.spage%253D2012%26rft.epage%253D2018%26rft.atitle%253DGenome%2BSequence%2Bof%2Bthe%2BNematode%2BC.%2Belegans%253A%2BA%2BPlatform%2Bfor%2BInvestigating%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.282.5396.2012%26rft_id%253Dinfo%253Apmid%252F9851916%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODIvNTM5Ni8yMDEyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDUvMTQvNTY1MjE4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.16" data-doi="10.1186/s13742-015-0047-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">C. C.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Chow</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. C.</span> <span class="cit-name-surname">Tellier</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Vattikuti</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Purcell</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Second-generation PLINK: rising to the challenge of larger and richer datasets</span>. <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D4%26rft.spage%253D7%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13742-015-0047-8%26rft_id%253Dinfo%253Apmid%252F25722852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13742-015-0047-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25722852&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.17" data-doi="10.1038/nrc1547"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Stubbe</span></span>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Bleomycins: towards better therapeutics</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Cancer</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">102</span>– <span class="cit-lpage">112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BCancer%26rft.stitle%253DNat%2BRev%2BCancer%26rft.aulast%253DChen%26rft.auinit1%253DJ.%26rft.volume%253D5%26rft.issue%253D2%26rft.spage%253D102%26rft.epage%253D112%26rft.atitle%253DBleomycins%253A%2Btowards%2Bbetter%2Btherapeutics.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrc1547%26rft_id%253Dinfo%253Apmid%252F15685195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrc1547&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15685195&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226721000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.18" data-doi="10.1093/jnci/91.13.1125"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cloos</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. J.</span> <span class="cit-name-surname">Nieuwenhuis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. I.</span> <span class="cit-name-surname">Boomsma</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Kuik</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">van der Sterre</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Inherited susceptibility to bleomycin-induced chromatid breaks in cultured peripheral blood lymphocytes</span>. <abbr class="cit-jnl-abbrev">J. Natl. Cancer Inst</abbr>. <span class="cit-vol">91</span>: <span class="cit-fpage">1125</span>–<span class="cit-lpage">1130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJNCI%2BJournal%2Bof%2Bthe%2BNational%2BCancer%2BInstitute%26rft.stitle%253DJNCI%2BJ%2BNatl%2BCancer%2BInst%26rft.aulast%253DCloos%26rft.auinit1%253DJ.%26rft.volume%253D91%26rft.issue%253D13%26rft.spage%253D1125%26rft.epage%253D1130%26rft.atitle%253DInherited%2BSusceptibility%2Bto%2BBleomycin-Induced%2BChromatid%2BBreaks%2Bin%2BCultured%2BPeripheral%2BBlood%2BLymphocytes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjnci%252F91.13.1125%26rft_id%253Dinfo%253Apmid%252F10393720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jnci/91.13.1125&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10393720&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081416500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.19" data-doi="10.1534/genetics.116.191148"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">D. E.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Zdraljevic</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. E.</span> <span class="cit-name-surname">Tanny</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Seo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. D.</span> <span class="cit-name-surname">Riccardi</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The Genetic Basis of Natural Variation in <em>Caenorhabditis elegans</em> Telomere Length</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">204</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">383</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.191148%26rft_id%253Dinfo%253Apmid%252F27449056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA0LzEvMzcxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDUvMTQvNTY1MjE4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.20" data-doi="10.1093/nar/gkw893"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">D. E.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Zdraljevic</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Roberts</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">E. C.</span> <span class="cit-name-surname">Andersen</span></span>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">CeNDR, the <em>Caenorhabditis elegans</em> natural diversity resource</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">45</span>: <span class="cit-fpage">D650</span>–<span class="cit-lpage">D657</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw893%26rft_id%253Dinfo%253Apmid%252F27701074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw893&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27701074&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">D. E.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">E. C.</span> <span class="cit-name-surname">Andersen</span></span>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">VCF-kit: assorted utilities for the variant call format</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1581</span>–<span class="cit-lpage">1582</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D33%26rft.spage%253D1581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.22" data-doi="10.1371/journal.pone.0156744"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Covarrubias-Pazaran</span>  <span class="cit-name-given-names">G</span></span>., <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-Assisted Prediction of Quantitative Traits Using the R Package sommer</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0156744</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0156744%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0156744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0156744&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dagogo-Jack</span>  <span class="cit-name-given-names">I.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A. T.</span> <span class="cit-name-surname">Shaw</span></span>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Tumour heterogeneity and resistance to cancer therapies</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Clin. Oncol</abbr>. <span class="cit-vol">15</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BClin.%2BOncol%26rft.volume%253D15%26rft.spage%253D81%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.24" data-doi="10.1038/nbt.3437"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Doench</span>  <span class="cit-name-given-names">J. G.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Fusi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Sullender</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hegde</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. W.</span> <span class="cit-name-surname">Vaimberg</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR-Cas9</span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol</abbr>. <span class="cit-vol">34</span>: <span class="cit-fpage">184</span>–<span class="cit-lpage">191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol%26rft.volume%253D34%26rft.spage%253D184%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3437%26rft_id%253Dinfo%253Apmid%252F26780180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3437&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26780180&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.25" data-doi="10.1016/S1074-5521(00)00011-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Sánchez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Edwards</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Shen</span></span>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The biosynthetic gene cluster for the antitumor drug bleomycin from <em>Streptomyces verticillus</em> ATCC15003 supporting functional interactions between nonribosomal peptide synthetases and a polyketide synthase</span>. <abbr class="cit-jnl-abbrev">Chem. Biol</abbr>. <span class="cit-vol">7</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">642</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChemistry%2B%2526%2Bbiology%26rft.stitle%253DChem%2BBiol%26rft.aulast%253DDu%26rft.auinit1%253DL.%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253D623%26rft.epage%253D642%26rft.atitle%253DThe%2Bbiosynthetic%2Bgene%2Bcluster%2Bfor%2Bthe%2Bantitumor%2Bdrug%2Bbleomycin%2Bfrom%2BStreptomyces%2Bverticillus%2BATCC15003%2Bsupporting%2Bfunctional%2Binteractions%2Bbetween%2Bnonribosomal%2Bpeptide%2Bsynthetases%2Band%2Ba%2Bpolyketide%2Bsynthase.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1074-5521%252800%252900011-9%26rft_id%253Dinfo%253Apmid%252F11048953%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1074-5521(00)00011-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11048953&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089866200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.26" data-doi="10.1093/nar/gkh340"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">R. C</span></span>., <span class="cit-pub-date">2004a</span> <span class="cit-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">32</span>: <span class="cit-fpage">1792</span>–<span class="cit-lpage">1797</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkh340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15034147&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220487200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.27" data-doi="10.1186/1471-2105-5-113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">R. C</span></span>., <span class="cit-pub-date">2004b</span> <span class="cit-article-title">MUSCLE: a multiple sequence alignment method with reduced time and space complexity</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D5%26rft.spage%253D113%26rft.epage%253D113%26rft.atitle%253DMUSCLE%253A%2Ba%2Bmultiple%2Bsequence%2Balignment%2Bmethod%2Bwith%2Breduced%2Btime%2Band%2Bspace%2Bcomplexity.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-5-113%26rft_id%253Dinfo%253Apmid%252F15318951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-5-113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15318951&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.28" data-doi="10.1186/gb-2005-6-5-r44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eilbeck</span>  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Lewis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Mungall</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Yandell</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Stein</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The Sequence Ontology: a tool for the unification of genome annotations</span>. <abbr class="cit-jnl-abbrev">Genome Biol</abbr>. <span class="cit-vol">6</span>: <span class="cit-fpage">R44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DEilbeck%26rft.auinit1%253DK.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253DR44%26rft.epage%253DR44%26rft.atitle%253DThe%2BSequence%2BOntology%253A%2Ba%2Btool%2Bfor%2Bthe%2Bunification%2Bof%2Bgenome%2Bannotations.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2005-6-5-r44%26rft_id%253Dinfo%253Apmid%252F15892872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2005-6-5-r44&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15892872&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B</span></span>., <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge Regression and Other Kernels for Genomic Selection with R Package rrBLUP</span>. <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Shrinkage estimation of the realized relationship matrix</span>. G3 <span class="cit-vol">2</span>: <span class="cit-fpage">1405</span>–<span class="cit-lpage">1413</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.31" data-doi="10.1534/genetics.118.301311"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">K. S.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. C.</span> <span class="cit-name-surname">Brady</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Bloom</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. E.</span> <span class="cit-name-surname">Tanny</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. E.</span> <span class="cit-name-surname">Cook</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Shared Genomic Regions Underlie Natural Variation in Diverse Toxin Responses</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">210</span>: <span class="cit-fpage">1509</span>–<span class="cit-lpage">1525</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.301311%26rft_id%253Dinfo%253Apmid%252F30341085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIxMC80LzE1MDkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNS8xNC81NjUyMTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.32" data-doi="10.1016/j.cell.2017.03.046"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">García-González</span>  <span class="cit-name-given-names">A. P.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Ritter</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Shrestha</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. C.</span> <span class="cit-name-surname">Andersen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. S.</span> <span class="cit-name-surname">Yilmaz</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Bacterial Metabolism Affects the <em>C. elegans</em> Response to Cancer Chemotherapeutics</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">431</span>–<span class="cit-lpage">441</span>.e8.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D169%26rft.spage%253D431%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2017.03.046%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2017.03.046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.33" data-doi="10.1126/science.1214318"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ghosh</span>  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. C.</span> <span class="cit-name-surname">Andersen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Shapiro</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Gerke</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Kruglyak</span></span>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Natural variation in a chloride channel subunit confers avermectin resistance in <em>C. elegans</em></span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">335</span>: <span class="cit-fpage">574</span>– <span class="cit-lpage">578</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGhosh%26rft.auinit1%253DR.%26rft.volume%253D335%26rft.issue%253D6068%26rft.spage%253D574%26rft.epage%253D578%26rft.atitle%253DNatural%2BVariation%2Bin%2Ba%2BChloride%2BChannel%2BSubunit%2BConfers%2BAvermectin%2BResistance%2Bin%2BC.%2Belegans%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1214318%26rft_id%253Dinfo%253Apmid%252F22301316%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzUvNjA2OC81NzQiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNS8xNC81NjUyMTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Ye</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Spitz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Lin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. A.</span> <span class="cit-name-surname">Kiemeney</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A genetic variant near the PMAIP1/Noxa gene is associated with increased bleomycin sensitivity</span>. <abbr class="cit-jnl-abbrev">Hum. Mol. Genet</abbr>. <span class="cit-vol">20</span>: <span class="cit-fpage">820</span>–<span class="cit-lpage">826</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2BMolecular%2BGenetics%26rft.stitle%253DHum%2BMol%2BGenet%26rft.aulast%253DGu%26rft.auinit1%253DJ.%26rft.volume%253D20%26rft.issue%253D4%26rft.spage%253D820%26rft.epage%253D826%26rft.atitle%253DA%2Bgenetic%2Bvariant%2Bnear%2Bthe%2BPMAIP1%252FNoxa%2Bgene%2Bis%2Bassociated%2Bwith%2Bincreased%2Bbleomycin%2Bsensitivity.%26rft_id%253Dinfo%253Apmid%252F21106707%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21106707&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.35" data-doi="10.1200/JCO.2007.14.1606"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Haas E. C.</span> <span class="cit-name-surname">de</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Zwart</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Meijer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Nuver</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H. M.</span> <span class="cit-name-surname">Boezen</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Variation in bleomycin hydrolase gene is associated with reduced survival after chemotherapy for testicular germ cell cancer</span>. <abbr class="cit-jnl-abbrev">J. Clin. Oncol</abbr>. <span class="cit-vol">26</span>: <span class="cit-fpage">1817</span>–<span class="cit-lpage">1823</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BOncology%26rft.stitle%253DJCO%26rft.aulast%253Dde%2BHaas%26rft.auinit1%253DE.%2BC.%26rft.volume%253D26%26rft.issue%253D11%26rft.spage%253D1817%26rft.epage%253D1823%26rft.atitle%253DVariation%2Bin%2BBleomycin%2BHydrolase%2BGene%2BIs%2BAssociated%2BWith%2BReduced%2BSurvival%2BAfter%2BChemotherapy%2Bfor%2BTesticular%2BGerm%2BCell%2BCancer%26rft_id%253Dinfo%253Adoi%252F10.1200%252FJCO.2007.14.1606%26rft_id%253Dinfo%253Apmid%252F18398146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNvIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8xMS8xODE3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDUvMTQvNTY1MjE4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hahnel</span>  <span class="cit-name-given-names">S. R.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Zdraljevic</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. C.</span> <span class="cit-name-surname">Rodriguez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. T.</span> <span class="cit-name-surname">McGrath</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Extreme allelic heterogeneity at a <em>Caenorhabditis elegans</em> beta-tubulin locus explains natural resistance to benzimidazoles</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>. <span class="cit-vol">14</span>: <span class="cit-fpage">e1007226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D14%26rft.spage%253De1007226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.37" data-doi="10.1101/gr.3729105"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hillier</span>  <span class="cit-name-given-names">L. W.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Coulson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. I.</span> <span class="cit-name-surname">Murray</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Bao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Sulston</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genomics in <em>C. elegans:</em> so many genes, such a little worm</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr>. <span class="cit-vol">15</span>: <span class="cit-fpage">1651</span>–<span class="cit-lpage">1660</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DHillier%26rft.auinit1%253DL.%2BW.%26rft.volume%253D15%26rft.issue%253D12%26rft.spage%253D1651%26rft.epage%253D1660%26rft.atitle%253DGenomics%2Bin%2BC.%2Belegans%253A%2BSo%2Bmany%2Bgenes%252C%2Bsuch%2Ba%2Blittle%2Bworm%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3729105%26rft_id%253Dinfo%253Apmid%252F16339362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNS8xMi8xNjUxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDUvMTQvNTY1MjE4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.38" data-doi="10.1038/nrg1578"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">D. J</span></span>., <span class="cit-pub-date">2005</span> <span class="cit-article-title">Gene-environment interactions in human diseases</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet</abbr>. <span class="cit-vol">6</span>: <span class="cit-fpage">287</span>– <span class="cit-lpage">298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DHunter%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253D287%26rft.epage%253D298%26rft.atitle%253DGene-environment%2Binteractions%2Bin%2Bhuman%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1578%26rft_id%253Dinfo%253Apmid%252F15803198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1578&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15803198&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228093000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.39" data-doi="10.1371/journal.pgen.0030034"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kammenga</span>  <span class="cit-name-given-names">J. E.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Doroszuk</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. A. G.</span> <span class="cit-name-surname">Riksen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Hazendonk</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Spiridon</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A <em>Caenorhabditis elegans</em> wild type defies the temperature-size rule owing to a single nucleotide polymorphism in tra-3</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr>. <span class="cit-vol">3</span>: <span class="cit-fpage">e34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKammenga%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D3%26rft.issue%253D3%26rft.spage%253De34%26rft.epage%253De34%26rft.atitle%253DA%2BCaenorhabditis%2Belegans%2Bwild%2Btype%2Bdefies%2Bthe%2Btemperature-size%2Brule%2Bowing%2Bto%2Ba%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bin%2Btra-3.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0030034%26rft_id%253Dinfo%253Apmid%252F17335351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0030034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17335351&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.40" data-doi="10.1534/genetics.107.080101"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Zaitlen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Wade</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kirby</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Heckerman</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient control of population structure in model organism association mapping</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">1709</span>–<span class="cit-lpage">1723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D178%26rft.issue%253D3%26rft.spage%253D1709%26rft.epage%253D1723%26rft.atitle%253DEfficient%2BControl%2Bof%2BPopulation%2BStructure%2Bin%2BModel%2BOrganism%2BAssociation%2BMapping%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080101%26rft_id%253Dinfo%253Apmid%252F18385116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC8zLzE3MDkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNS8xNC81NjUyMTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.41" data-doi="10.1038/nprot.2015.053"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">L. A.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Mezulis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Yates</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. N.</span> <span class="cit-name-surname">Wass</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M. J. E.</span> <span class="cit-name-surname">Sternberg</span></span>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The Phyre2 web portal for protein modeling, prediction and analysis</span>. <abbr class="cit-jnl-abbrev">Nat. Protoc</abbr>. <span class="cit-vol">10</span>: <span class="cit-fpage">845</span>–<span class="cit-lpage">858</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BProtoc%26rft.volume%253D10%26rft.spage%253D845%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2015.053%26rft_id%253Dinfo%253Apmid%252F25950237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2015.053&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25950237&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.42" data-doi="10.1073/pnas.80.10.3064"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lazo</span>  <span class="cit-name-given-names">J. S.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Humphreys</span></span>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Lack of metabolism as the biochemical basis of bleomycin-induced pulmonary toxicity</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>. <span class="cit-vol">80</span>: <span class="cit-fpage">3064</span>–<span class="cit-lpage">3068</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DLazo%26rft.auinit1%253DJ.%2BS.%26rft.volume%253D80%26rft.issue%253D10%26rft.spage%253D3064%26rft.epage%253D3068%26rft.atitle%253DLack%2Bof%2Bmetabolism%2Bas%2Bthe%2Bbiochemical%2Bbasis%2Bof%2Bbleomycin-induced%2Bpulmonary%2Btoxicity.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.80.10.3064%26rft_id%253Dinfo%253Apmid%252F6190169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODAvMTAvMzA2NCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA1LzE0LzU2NTIxOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.43" data-doi="10.1038/s41467-017-00386-x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Yang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Kim</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Brady</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Zdraljevic</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The genetic basis of natural variation in a phoretic behavior</span>. <abbr class="cit-jnl-abbrev">Nat. Commun</abbr>. <span class="cit-vol">8</span>: <span class="cit-fpage">273</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun%26rft.volume%253D8%26rft.spage%253D273%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-017-00386-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-017-00386-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.44" data-doi="10.1093/bioinformatics/btr509"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H</span></span>., <span class="cit-pub-date">2011</span> <span class="cit-article-title">A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2987</span>–<span class="cit-lpage">2993</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr509%26rft_id%253Dinfo%253Apmid%252F21903627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr509&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21903627&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296099300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.45" data-doi="10.1016/j.ygeno.2013.08.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Huang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Lan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Rothman</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Identification of gene-environment interactions in cancer studies using penalization</span>. <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">189</span>–<span class="cit-lpage">194</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.volume%253D102%26rft.spage%253D189%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2013.08.006%26rft_id%253Dinfo%253Apmid%252F23994599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2013.08.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23994599&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.46" data-doi="10.1111/cas.12186"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Low</span>  <span class="cit-name-given-names">S.-K.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Chung</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Takahashi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Zembutsu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Mushiroda</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide association study of chemotherapeutic agent-induced severe neutropenia/leucopenia for patients in Biobank Japan</span>. <abbr class="cit-jnl-abbrev">Cancer Sci</abbr>. <span class="cit-vol">104</span>: <span class="cit-fpage">1074</span>–<span class="cit-lpage">1082</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BSci%26rft.volume%253D104%26rft.spage%253D1074%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fcas.12186%26rft_id%253Dinfo%253Apmid%252F23648065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/cas.12186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23648065&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.47" data-doi="10.1016/j.cell.2010.03.032"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McClellan</span>  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M.-C.</span> <span class="cit-name-surname">King</span></span>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic heterogeneity in human disease</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">210</span>–<span class="cit-lpage">217</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMcClellan%26rft.auinit1%253DJ.%26rft.volume%253D141%26rft.issue%253D2%26rft.spage%253D210%26rft.epage%253D217%26rft.atitle%253DGenetic%2Bheterogeneity%2Bin%2Bhuman%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2010.03.032%26rft_id%253Dinfo%253Apmid%252F20403315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2010.03.032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20403315&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276738400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.48" data-doi="10.1016/j.neuron.2009.02.012"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McGrath</span>  <span class="cit-name-given-names">P. T.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. V.</span> <span class="cit-name-surname">Rockman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Zimmer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Jang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. Z.</span> <span class="cit-name-surname">Macosko</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Quantitative mapping of a digenic behavioral trait implicates globin variation in <em>C. elegans</em> sensory behaviors</span>. <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">692</span>–<span class="cit-lpage">699</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D61%26rft.spage%253D692%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2009.02.012%26rft_id%253Dinfo%253Apmid%252F19285466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2009.02.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19285466&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264366900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nuver</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. F.</span> <span class="cit-name-surname">Lutke Holzik</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">van Zweeden</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H. J.</span> <span class="cit-name-surname">Hoekstra</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Meijer</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genetic variation in the bleomycin hydrolase gene and bleomycin-induced pulmonary toxicity in germ cell cancer patients</span>. <abbr class="cit-jnl-abbrev">Pharmacogenet. Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">405</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPharmacogenet%2BGenomics%26rft.aulast%253DNuver%26rft.auinit1%253DJ.%26rft.volume%253D15%26rft.issue%253D6%26rft.spage%253D399%26rft.epage%253D405%26rft.atitle%253DGenetic%2Bvariation%2Bin%2Bthe%2Bbleomycin%2Bhydrolase%2Bgene%2Band%2Bbleomycin-induced%2Bpulmonary%2Btoxicity%2Bin%2Bgerm%2Bcell%2Bcancer%2Bpatients.%26rft_id%253Dinfo%253Apmid%252F15900213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15900213&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230102500005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.50" data-doi="10.1038/nature07171"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Palopoli</span>  <span class="cit-name-given-names">M. F.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. V.</span> <span class="cit-name-surname">Rockman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">TinMaung</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Ramsay</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Curwen</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Molecular basis of the copulatory plug polymorphism in <em>Caenorhabditis elegans</em></span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">454</span>: <span class="cit-fpage">1019</span>– <span class="cit-lpage">1022</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPalopoli%26rft.auinit1%253DM.%2BF.%26rft.volume%253D454%26rft.issue%253D7207%26rft.spage%253D1019%26rft.epage%253D1022%26rft.atitle%253DMolecular%2Bbasis%2Bof%2Bthe%2Bcopulatory%2Bplug%2Bpolymorphism%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07171%26rft_id%253Dinfo%253Apmid%252F18633349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07171&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18633349&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258591000044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.51" data-doi="10.1038/nmeth.4324"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pimentel</span>  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N. L.</span> <span class="cit-name-surname">Bray</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Puente</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Melsted</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Pachter</span></span>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Differential analysis of RNA-seq incorporating quantification uncertainty</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">687</span>–<span class="cit-lpage">690</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D14%26rft.spage%253D687%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.4324%26rft_id%253Dinfo%253Apmid%252F28581496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.4324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28581496&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.52" data-doi="10.1086/519795"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Neale</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Todd-Brown</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Thomas</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. A. R.</span> <span class="cit-name-surname">Ferreira</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PLINK: a tool set for whole-genome association and population-based linkage analyses</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet</abbr>. <span class="cit-vol">81</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D575%26rft.atitle%253DPLINK%253A%2Ba%2Btool%2Bset%2Bfor%2Bwhole-genome%2Bassociation%2Band%2Bpopulation-based%2Blinkage%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F519795%26rft_id%253Dinfo%253Apmid%252F17701901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/519795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17701901&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.53"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">R Core Team</span>, <span class="cit-article-title">R: A Language and Environment for Statistical Computing</span>. <abbr class="cit-jnl-abbrev">R Foundation for Statistical Computing</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.54" data-doi="10.1126/science.1166527"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reddy</span>  <span class="cit-name-given-names">K. C.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. C.</span> <span class="cit-name-surname">Andersen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Kruglyak</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D. H.</span> <span class="cit-name-surname">Kim</span></span>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A polymorphism in <em>npr-1</em> is a behavioral determinant of pathogen susceptibility in <em>C. elegans</em></span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">323</span>: <span class="cit-fpage">382</span>–<span class="cit-lpage">384</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DReddy%26rft.auinit1%253DK.%2BC.%26rft.volume%253D323%26rft.issue%253D5912%26rft.spage%253D382%26rft.epage%253D384%26rft.atitle%253DA%2BPolymorphism%2Bin%2Bnpr-1%2BIs%2Ba%2BBehavioral%2BDeterminant%2Bof%2BPathogen%2BSusceptibility%2Bin%2BC.%2Belegans%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1166527%26rft_id%253Dinfo%253Apmid%252F19150845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjMvNTkxMi8zODIiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNS8xNC81NjUyMTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.55" data-doi="10.1038/35101056"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Relling</span>  <span class="cit-name-given-names">M. V.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Dervieux</span></span>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Pharmacogenetics and cancer therapy</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Cancer</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">99</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BCancer%26rft.stitle%253DNat%2BRev%2BCancer%26rft.aulast%253DRelling%26rft.auinit1%253DM.%2BV.%26rft.volume%253D1%26rft.issue%253D2%26rft.spage%253D99%26rft.epage%253D108%26rft.atitle%253DPharmacogenetics%2Band%2Bcancer%2Btherapy.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35101056%26rft_id%253Dinfo%253Apmid%252F11905809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35101056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11905809&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.56" data-doi="10.1038/ncb2720"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Riedel</span>  <span class="cit-name-given-names">C. G.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. H.</span> <span class="cit-name-surname">Dowen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. F.</span> <span class="cit-name-surname">Lourenco</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N. V.</span> <span class="cit-name-surname">Kirienko</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Heimbucher</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">DAF-16 employs the chromatin remodeller SWI/SNF to promote stress resistance and longevity</span>. <abbr class="cit-jnl-abbrev">Nat. Cell Biol</abbr>. <span class="cit-vol">15</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">501</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCell%2BBiol%26rft.volume%253D15%26rft.spage%253D491%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncb2720%26rft_id%253Dinfo%253Apmid%252F23604319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncb2720&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23604319&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318265300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.57" data-doi="10.1371/journal.pgen.1000419"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Kruglyak</span></span>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Recombinational landscape and population genomics of <em>Caenorhabditis elegans</em></span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr>. <span class="cit-vol">5</span>: <span class="cit-fpage">e1000419</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DRockman%26rft.auinit1%253DM.%2BV.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253De1000419%26rft.epage%253De1000419%26rft.atitle%253DRecombinational%2Blandscape%2Band%2Bpopulation%2Bgenomics%2Bof%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000419%26rft_id%253Dinfo%253Apmid%252F19283065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000419&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19283065&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.58" data-doi="10.1126/science.1194208"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. S.</span> <span class="cit-name-surname">Skrovanek</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Kruglyak</span></span>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Selection at linked sites shapes heritable phenotypic variation in <em>C. elegans</em></span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">330</span>: <span class="cit-fpage">372</span>–<span class="cit-lpage">376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DRockman%26rft.auinit1%253DM.%2BV.%26rft.volume%253D330%26rft.issue%253D6002%26rft.spage%253D372%26rft.epage%253D376%26rft.atitle%253DSelection%2Bat%2BLinked%2BSites%2BShapes%2BHeritable%2BPhenotypic%2BVariation%2Bin%2BC.%2Belegans%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1194208%26rft_id%253Dinfo%253Apmid%252F20947766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzAvNjAwMi8zNzIiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNS8xNC81NjUyMTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.59" data-doi="10.1073/pnas.1607183113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Samuel</span>  <span class="cit-name-given-names">B. S.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Rowedder</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Braendle</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.-A.</span> <span class="cit-name-surname">Félix</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Ruvkun</span></span>, <span class="cit-pub-date">2016</span> <span class="cit-article-title"><em>Caenorhabditis elegans</em> responses to bacteria from its natural habitats</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>. <span class="cit-vol">113</span>: <span class="cit-fpage">E3941</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1607183113%26rft_id%253Dinfo%253Apmid%252F27317746%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEzLzI3L0UzOTQxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDUvMTQvNTY1MjE4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.60" data-doi="10.1038/nrd.2016.230"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Santos</span>  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Ursu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Gaulton</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Bento</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. S.</span> <span class="cit-name-surname">Donadi</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A comprehensive map of molecular drug targets</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Drug Discov</abbr>. <span class="cit-vol">16</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BDrug%2BDiscov%26rft.volume%253D16%26rft.spage%253D19%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrd.2016.230%26rft_id%253Dinfo%253Apmid%252F27910877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrd.2016.230&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27910877&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.61" data-doi="10.1371/journal.pgen.1005236"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schmid</span>  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. B.</span> <span class="cit-name-surname">Snoek</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Fröhli</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">van der Bent</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Kammenga</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Systemic Regulation of RAS/MAPK Signaling by the Serotonin Metabolite 5-HIAA</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr>. <span class="cit-vol">11</span>: <span class="cit-fpage">e1005236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet%26rft.volume%253D11%26rft.spage%253De1005236%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005236%26rft_id%253Dinfo%253Apmid%252F25978500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005236&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25978500&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.62" data-doi="10.1126/science.1151107"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Seidel</span>  <span class="cit-name-given-names">H. S.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. V.</span> <span class="cit-name-surname">Rockman</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Kruglyak</span></span>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Widespread genetic incompatibility in <em>C. elegans</em> maintained by balancing selection</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">319</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">594</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1151107%26rft_id%253Dinfo%253Apmid%252F18187622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTkvNTg2My81ODkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNS8xNC81NjUyMTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.63" data-doi="10.1371/journal.pbio.1001115"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Seidel</span>  <span class="cit-name-given-names">H. S.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Ailion</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">van Oudenaarden</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. V.</span> <span class="cit-name-surname">Rockman</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A novel sperm-delivered toxin causes late-stage embryo lethality and transmission ratio distortion in <em>C. elegans</em></span>. <abbr class="cit-jnl-abbrev">PLoS Biol</abbr>. <span class="cit-vol">9</span>: <span class="cit-fpage">e1001115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DSeidel%26rft.auinit1%253DH.%2BS.%26rft.volume%253D9%26rft.issue%253D7%26rft.spage%253De1001115%26rft.epage%253De1001115%26rft.atitle%253DA%2Bnovel%2Bsperm-delivered%2Btoxin%2Bcauses%2Blate-stage%2Bembryo%2Blethality%2Band%2Btransmission%2Bratio%2Bdistortion%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001115%26rft_id%253Dinfo%253Apmid%252F21814493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001115&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21814493&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.64" data-doi="10.1038/nrg3706"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sham</span>  <span class="cit-name-given-names">P. C.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Purcell</span></span>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Statistical power and significance testing in large-scale genetic studies</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet</abbr>. <span class="cit-vol">15</span>: <span class="cit-fpage">335</span>–<span class="cit-lpage">346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet%26rft.volume%253D15%26rft.spage%253D335%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3706%26rft_id%253Dinfo%253Apmid%252F24739678%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3706&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24739678&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.65" data-doi="10.1021/np010550q"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Du</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Sanchez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Edwards</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Cloning and characterization of the bleomycin biosynthetic gene cluster from <em>Streptomyces verticillus</em> ATCC15003</span>. <abbr class="cit-jnl-abbrev">J. Nat. Prod</abbr>. <span class="cit-vol">65</span>: <span class="cit-fpage">422</span>–<span class="cit-lpage">431</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNatural%2BProducts%2B%2528Print%2529%26rft.stitle%253DJournal%2Bof%2BNatural%2BProducts%2B%2528Print%2529%26rft.aulast%253DShen%26rft.auinit1%253DB.%26rft.volume%253D65%26rft.issue%253D3%26rft.spage%253D422%26rft.epage%253D431%26rft.atitle%253DCloning%2Band%2Bcharacterization%2Bof%2Bthe%2Bbleomycin%2Bbiosynthetic%2Bgene%2Bcluster%2Bfrom%2BStreptomyces%2Bverticillus%2BATCC15003.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fnp010550q%26rft_id%253Dinfo%253Apmid%252F11908996%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/np010550q&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11908996&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.66" data-doi="10.1371/journal.pone.0111090"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shimko</span>  <span class="cit-name-given-names">T. C.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">E. C.</span> <span class="cit-name-surname">Andersen</span></span>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">COPASutils: an R package for reading, processing, and visualizing data from COPAS large-particle flow cytometers</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e111090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De111090%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0111090%26rft_id%253Dinfo%253Apmid%252F25329171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0111090&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25329171&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.67"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Snoek</span>  <span class="cit-name-given-names">L. B.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H. E.</span> <span class="cit-name-surname">Orbidans</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Stastna</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Aartse</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Rodriguez</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Widespread genomic incompatibilities in <em>Caenorhabditis elegans</em></span>. G3 <span class="cit-vol">4</span>: <span class="cit-fpage">1813</span>–<span class="cit-lpage">1823</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.68" data-doi="10.1093/nar/29.1.82"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Sternberg</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Durbin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Thierry-Mieg</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Spieth</span></span>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">WormBase: network access to the genome and biology of <em>Caenorhabditis elegans</em></span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">29</span>: <span class="cit-fpage">82</span>– <span class="cit-lpage">86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DStein%26rft.auinit1%253DL.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D82%26rft.epage%253D86%26rft.atitle%253DWormBase%253A%2Bnetwork%2Baccess%2Bto%2Bthe%2Bgenome%2Band%2Bbiology%2Bof%2BCaenorhabditis%2Belegans%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F29.1.82%26rft_id%253Dinfo%253Apmid%252F11125056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/29.1.82&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11125056&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166360300020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.69" data-doi="10.1021/cr00081a011"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stubbe</span>  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">Kozarich</span></span>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Mechanisms of bleomycin-induced DNA degradation</span>. <abbr class="cit-jnl-abbrev">Chem. Rev</abbr>. <span class="cit-vol">87</span>: <span class="cit-fpage">1107</span>–<span class="cit-lpage">1136</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChem.%2BRev%26rft.volume%253D87%26rft.spage%253D1107%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fcr00081a011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/cr00081a011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1987K412400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.70"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Swan</span>  <span class="cit-name-given-names">K. A.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. E.</span> <span class="cit-name-surname">Curtis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. B.</span> <span class="cit-name-surname">McKusick</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. V.</span> <span class="cit-name-surname">Voinov</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F. A.</span> <span class="cit-name-surname">Mapa</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">High-throughput gene mapping in <em>Caenorhabditis elegans</em></span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr>. <span class="cit-vol">12</span>: <span class="cit-fpage">1100</span>–<span class="cit-lpage">1105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DSwan%26rft.auinit1%253DK.%2BA.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D1100%26rft.epage%253D1105%26rft.atitle%253DHigh-Throughput%2BGene%2BMapping%2Bin%2BCaenorhabditis%2Belegans%26rft_id%253Dinfo%253Apmid%252F12097347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjEyLzcvMTEwMCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA1LzE0LzU2NTIxOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.71" data-doi="10.1080/095530098141429"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">E. M.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Lehmann</span></span>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Conservation of eukaryotic DNA repair mechanisms</span>. <abbr class="cit-jnl-abbrev">Int. J. Radiat. Biol</abbr>. <span class="cit-vol">74</span>: <span class="cit-fpage">277</span>–<span class="cit-lpage">286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2Bjournal%2Bof%2Bradiation%2Bbiology%26rft.stitle%253DInt%2BJ%2BRadiat%2BBiol%26rft.aulast%253DTaylor%26rft.auinit1%253DE.%2BM.%26rft.volume%253D74%26rft.issue%253D3%26rft.spage%253D277%26rft.epage%253D286%26rft.atitle%253DConservation%2Bof%2Beukaryotic%2BDNA%2Brepair%2Bmechanisms.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F095530098141429%26rft_id%253Dinfo%253Apmid%252F9737531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/095530098141429&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9737531&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000075721900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.72" data-doi="10.1534/genetics.115.175950"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">O. A.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. B.</span> <span class="cit-name-surname">Snoek</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Nijveen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. G.</span> <span class="cit-name-surname">Sterken</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. J. M.</span> <span class="cit-name-surname">Volkers</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Remarkably Divergent Regions Punctuate the Genome Assembly of the <em>Caenorhabditis elegans</em> Hawaiian Strain CB4856</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">200</span>: <span class="cit-fpage">975</span>–<span class="cit-lpage">989</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.175950%26rft_id%253Dinfo%253Apmid%252F25995208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAwLzMvOTc1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDUvMTQvNTY1MjE4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.73"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Turek</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Besseling</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.-P.</span> <span class="cit-name-surname">Spies</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">König</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Bringmann</span></span>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Sleep-active neuron specification and sleep induction require FLP-11 neuropeptides to systemically induce sleep</span>. <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">5</span>. <a href="https://doi.org/10.7554/eLife.12499">https://doi.org/10.7554/eLife.12499</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.74" data-doi="10.1038/88878"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wicks</span>  <span class="cit-name-given-names">S. R.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. T.</span> <span class="cit-name-surname">Yeh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W. R.</span> <span class="cit-name-surname">Gish</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. H.</span> <span class="cit-name-surname">Waterston</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">R. H.</span> <span class="cit-name-surname">Plasterk</span></span>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Rapid gene mapping in <em>Caenorhabditis elegans</em> using a high density polymorphism map</span>. <abbr class="cit-jnl-abbrev">Nat. Genet</abbr>. <span class="cit-vol">28</span>: <span class="cit-fpage">160</span>–<span class="cit-lpage">164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DWicks%26rft.auinit1%253DS.%2BR.%26rft.volume%253D28%26rft.issue%253D2%26rft.spage%253D160%26rft.epage%253D164%26rft.atitle%253DRapid%2Bgene%2Bmapping%2Bin%2BCaenorhabditis%2Belegans%2Busing%2Ba%2Bhigh%2Bdensity%2Bpolymorphism%2Bmap.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F88878%26rft_id%253Dinfo%253Apmid%252F11381264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/88878&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11381264&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F14%2F565218.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169011800019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.75"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">World Health Organization</span>, <span class="cit-pub-date">2018</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.76" data-doi="10.1371/journal.pgen.1006891"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zdraljevic</span>  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Strand</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H. S.</span> <span class="cit-name-surname">Seidel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. E.</span> <span class="cit-name-surname">Cook</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. G.</span> <span class="cit-name-surname">Doench</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Natural variation in a single amino acid substitution underlies physiological responses to topoisomerase II poisons</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr>. <span class="cit-vol">13</span>: <span class="cit-fpage">e1006891</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet%26rft.volume%253D13%26rft.spage%253De1006891%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1006891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1006891&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.77" data-doi="10.1016/j.gde.2017.08.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zdraljevic</span>  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">E. C.</span> <span class="cit-name-surname">Andersen</span></span>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Natural diversity facilitates the discovery of conserved chemotherapeutic response mechanisms</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev</abbr>. <span class="cit-vol">47</span>: <span class="cit-fpage">41</span>–<span class="cit-lpage">47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BGenet.%2BDev%26rft.volume%253D47%26rft.spage%253D41%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2017.08.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gde.2017.08.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.78"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zdraljevic</span>  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. W.</span> <span class="cit-name-surname">Fox</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Strand</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Panda</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F. J.</span> <span class="cit-name-surname">Tenjo</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Natural variation in <em>C. elegans</em> arsenic toxicity is explained by differences in branched chain amino acid metabolism</span>. <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">8</span>. <a href="https://doi.org/10.7554/eLife.40260">https://doi.org/10.7554/eLife.40260</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-565218v2.79"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Yan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Ming</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Jia</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Structural and Biochemical Characterization of Endoribonuclease Nsp15 Encoded by Middle East Respiratory Syndrome Coronavirus</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">92</span>. <a href="https://doi.org/10.1128/JVI.00893-18">https://doi.org/10.1128/JVI.00893-18</a></cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/565218v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F565218v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/636894v2" title="Developmental changes in connectivity between the amygdala subnuclei and occipitotemporal cortex" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;May 14, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/565218v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/565218v2.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/565218v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/565218v2" class="active" data-icon-position="" data-hide-link-title="0">A nematode-specific gene underlies bleomycin-response variation in Caenorhabditis elegans</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/732964" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-tbVYxh6-pBOeDwaEi-CPXYjSP3ma80wdb0DW3nI1KK0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="732964" id="node732964--2" data-pisa="biorxiv;565218v2" data-pisa-master="biorxiv;565218" data-seqnum="732964" data-apath="/biorxiv/early/2019/05/14/565218.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">A nematode-specific gene underlies bleomycin-response variation in <em>Caenorhabditis elegans</em></div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Shannon C.</span> <span class="nlm-surname">Brady</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Stefan</span> <span class="nlm-surname">Zdraljevic</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Karol W.</span> <span class="nlm-surname">Bisaga</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Robyn E.</span> <span class="nlm-surname">Tanny</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Daniel E.</span> <span class="nlm-surname">Cook</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Daehan</span> <span class="nlm-surname">Lee</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Ye</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Erik C.</span> <span class="nlm-surname">Andersen</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">565218; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/565218 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/565218v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/565218v2&amp;title=A%20nematode-specific%20gene%20underlies%20bleomycin-response%20variation%20in%20Caenorhabditis%20elegans" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/565218v2&amp;title=A%20nematode-specific%20gene%20underlies%20bleomycin-response%20variation%20in%20Caenorhabditis%20elegans" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/565218v2&amp;text=A%20nematode-specific%20gene%20underlies%20bleomycin-response%20variation%20in%20Caenorhabditis%20elegans" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/565218v2&amp;title=A%20nematode-specific%20gene%20underlies%20bleomycin-response%20variation%20in%20Caenorhabditis%20elegans" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/565218v2&amp;t=A%20nematode-specific%20gene%20underlies%20bleomycin-response%20variation%20in%20Caenorhabditis%20elegans" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/565218v2&amp;title=A%20nematode-specific%20gene%20underlies%20bleomycin-response%20variation%20in%20Caenorhabditis%20elegans" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/565218v2&amp;title=A%20nematode-specific%20gene%20underlies%20bleomycin-response%20variation%20in%20Caenorhabditis%20elegans" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="732964" id="node732964--3" data-pisa="biorxiv;565218v2" data-pisa-master="biorxiv;565218" data-seqnum="732964" data-apath="/biorxiv/early/2019/05/14/565218.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">A nematode-specific gene underlies bleomycin-response variation in <em>Caenorhabditis elegans</em></div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Shannon C.</span> <span class="nlm-surname">Brady</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Stefan</span> <span class="nlm-surname">Zdraljevic</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Karol W.</span> <span class="nlm-surname">Bisaga</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Robyn E.</span> <span class="nlm-surname">Tanny</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Daniel E.</span> <span class="nlm-surname">Cook</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Daehan</span> <span class="nlm-surname">Lee</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Ye</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Erik C.</span> <span class="nlm-surname">Andersen</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">565218; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/565218 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=A%20nematode-specific%20gene%20underlies%20bleomycin-response%20variation%20in%20Caenorhabditis%20elegans&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Brady%2C+Shannon+C.&amp;rft.au=Zdraljevic%2C+Stefan&amp;rft.au=Bisaga%2C+Karol+W.&amp;rft.au=Tanny%2C+Robyn+E.&amp;rft.au=Cook%2C+Daniel+E.&amp;rft.au=Lee%2C+Daehan&amp;rft.au=Wang%2C+Ye&amp;rft.au=Andersen%2C+Erik+C."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/732964/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/732964/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/732964/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/732964/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/732964/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/732964/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/732964/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/732964/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/732964/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/732964/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/732964/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/732964/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/565218v2&amp;count=horizontal&amp;via=&amp;text=A%20nematode-specific%20gene%20underlies%20bleomycin-response%20variation%20in%20Caenorhabditis%20elegans&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/565218v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/565218v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/565218v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/genetics" class="highlight" data-icon-position="" data-hide-link-title="0">Genetics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
