<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/05/03/627158.atom" />
<meta name="HW.pisa" content="biorxiv;627158v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Extensive genetic diversity of bat-borne polyomaviruses reveals inter-family host-switching events" />
<meta name="DC.Identifier" content="10.1101/627158" />
<meta name="DC.Date" content="2019-05-03" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Polyomaviruses (PyVs) are small, double-stranded DNA tumor viruses carried by diverse vertebrates. PyVs have previously been considered highly host restricted in mammalian hosts, with host-switching events thought rare or nonexistent. Prior investigations have revealed short-range host-switching events of PyVs in two different African bat species within the horseshoe bat genus Rhinolophus . Herein, we have conducted a systematic investigation of PyVs in 1,083 archived bat samples collected from five provinces across China, and identified 192 PyVs from 186 bats from 15 host species within 6 families (Rhinolophidae, Vespertilionidae, Hipposideridae, Emballonuridae, Miniopteridae and Pteropodidae) representing 28 newly-described PyVs, indicative of extensive genetic diversity of bat PyVs. Surprisingly, two PyVs were identified in multiple bat species from different families, and another PyV clustered phylogenetically with PyVs carried by bats from a different host family, indicative of three inter-family PyV host-switching events. The time to most recent common ancestor (tMRCA) of the three events was estimated at 0.02-11.6 million years ago (MYA), which is inconsistent with the estimated tMRCA of their respective bat hosts (36.3-66.7 MYA), and is most parsimoniously explained by host-switching events. PyVs identified from geographically separated Chinese horseshoe bat species in the present study showed close genetic identities, and clustered with each other and with PyVs from African horseshoe bats, allowing assessment of the effects of positive selection in VP1 within the horseshoe bat family Rhinolophidae. Correlation analysis indicated that co-evolution with their hosts contributed much more to evolutionary divergence of PyV than geographic distance. In conclusion, our findings provide the first evidence of inter-family host-switching events of PyV in mammals and challenge the prevailing evolutionary paradigm for strict host restriction of mammalian PyVs.

Author summary Since the discovery of murine polyomavirus in the 1950s, polyomaviruses (PyVs) have been considered both genetically stable and highly host-restricted in their mammalian hosts. In this study, we have identified multiple cases of host-switching events of PyVs by large scale surveillance in diverse bat species collected in China. These host-switching events occurred between bat families living in the same colony, indicating that a large population with frequent contacts between different bat species may represent an ecological niche facilitating PyV host-switching. The cases studied involved members of bats from several families, including horseshoe bats, which were previously found to harbor a number of highly virulent viruses to both humans and domestic animals. Our findings have provided evidence that even highly host-specific DNA viruses can transmit between bats of different species and indicate an increased propensity for spillover events involving horseshoe bats. We propose an evolutionary scheme for bat-borne PyVs in which intra-host divergence and host-switching has generated the diverse PyVs in present day bats. This scheme provides a useful model to study the evolution of PyVs in other hosts and, potentially, the modeling of bat zoonoses and the transmission of other DNA viruses in other mammals, including humans." />
<meta name="DC.Contributor" content="Zhizhou Tan" />
<meta name="DC.Contributor" content="Gabriel Gonzalez" />
<meta name="DC.Contributor" content="Jinliang Sheng" />
<meta name="DC.Contributor" content="Jianmin Wu" />
<meta name="DC.Contributor" content="Fuqiang Zhang" />
<meta name="DC.Contributor" content="Lin Xu" />
<meta name="DC.Contributor" content="Peisheng Zhang" />
<meta name="DC.Contributor" content="Aiwei Zhu" />
<meta name="DC.Contributor" content="Yonggang Qu" />
<meta name="DC.Contributor" content="Changchun Tu" />
<meta name="DC.Contributor" content="Michael J. Carr" />
<meta name="DC.Contributor" content="Biao He" />
<meta name="article:published_time" content="2019-05-03" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Extensive genetic diversity of bat-borne polyomaviruses reveals inter-family host-switching events" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Polyomaviruses (PyVs) are small, double-stranded DNA tumor viruses carried by diverse vertebrates. PyVs have previously been considered highly host restricted in mammalian hosts, with host-switching events thought rare or nonexistent. Prior investigations have revealed short-range host-switching events of PyVs in two different African bat species within the horseshoe bat genus &lt;i&gt;Rhinolophus&lt;/i&gt;. Herein, we have conducted a systematic investigation of PyVs in 1,083 archived bat samples collected from five provinces across China, and identified 192 PyVs from 186 bats from 15 host species within 6 families (Rhinolophidae, Vespertilionidae, Hipposideridae, Emballonuridae, Miniopteridae and Pteropodidae) representing 28 newly-described PyVs, indicative of extensive genetic diversity of bat PyVs. Surprisingly, two PyVs were identified in multiple bat species from different families, and another PyV clustered phylogenetically with PyVs carried by bats from a different host family, indicative of three inter-family PyV host-switching events. The time to most recent common ancestor (tMRCA) of the three events was estimated at 0.02-11.6 million years ago (MYA), which is inconsistent with the estimated tMRCA of their respective bat hosts (36.3-66.7 MYA), and is most parsimoniously explained by host-switching events. PyVs identified from geographically separated Chinese horseshoe bat species in the present study showed close genetic identities, and clustered with each other and with PyVs from African horseshoe bats, allowing assessment of the effects of positive selection in VP1 within the horseshoe bat family Rhinolophidae. Correlation analysis indicated that co-evolution with their hosts contributed much more to evolutionary divergence of PyV than geographic distance. In conclusion, our findings provide the first evidence of inter-family host-switching events of PyV in mammals and challenge the prevailing evolutionary paradigm for strict host restriction of mammalian PyVs.&lt;/p&gt;&lt;h3&gt;Author summary&lt;/h3&gt;
&lt;p&gt;Since the discovery of murine polyomavirus in the 1950s, polyomaviruses (PyVs) have been considered both genetically stable and highly host-restricted in their mammalian hosts. In this study, we have identified multiple cases of host-switching events of PyVs by large scale surveillance in diverse bat species collected in China. These host-switching events occurred between bat families living in the same colony, indicating that a large population with frequent contacts between different bat species may represent an ecological niche facilitating PyV host-switching. The cases studied involved members of bats from several families, including horseshoe bats, which were previously found to harbor a number of highly virulent viruses to both humans and domestic animals. Our findings have provided evidence that even highly host-specific DNA viruses can transmit between bats of different species and indicate an increased propensity for spillover events involving horseshoe bats. We propose an evolutionary scheme for bat-borne PyVs in which intra-host divergence and host-switching has generated the diverse PyVs in present day bats. This scheme provides a useful model to study the evolution of PyVs in other hosts and, potentially, the modeling of bat zoonoses and the transmission of other DNA viruses in other mammals, including humans.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;627158v1" />
<meta name="citation_id" content="627158v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/627158v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/627158v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/627158v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158.full.pdf" />
<meta name="citation_doi" content="10.1101/627158" />
<meta name="citation_num_pages" content="61" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="627158" />
<meta name="citation_author" content="Zhizhou Tan" />
<meta name="citation_author_institution" content="Key Laboratory of Jilin Province for Zoonosis Prevention and Control, Institute of Military Veterinary Medicine, Academy of Military Medical Sciences, Changchun" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2844-9673" />
<meta name="citation_author" content="Gabriel Gonzalez" />
<meta name="citation_author_institution" content="National Advanced Computing Collaboratory, National Center for High Technology" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2180-2120" />
<meta name="citation_author" content="Jinliang Sheng" />
<meta name="citation_author_institution" content="College of Animal Science and Technology, Shihezi University, Shihezi, Xinjiang Uyghur Autonomous Region" />
<meta name="citation_author" content="Jianmin Wu" />
<meta name="citation_author_institution" content="Guangxi Key Laboratory of Veterinary Biotechnology, Guangxi Veterinary Research Institute, Nanning" />
<meta name="citation_author" content="Fuqiang Zhang" />
<meta name="citation_author_institution" content="Center for Disease Control and Prevention of Southern Theater Command" />
<meta name="citation_author" content="Lin Xu" />
<meta name="citation_author_institution" content="Key Laboratory of Jilin Province for Zoonosis Prevention and Control, Institute of Military Veterinary Medicine, Academy of Military Medical Sciences, Changchun" />
<meta name="citation_author" content="Peisheng Zhang" />
<meta name="citation_author_institution" content="Key Laboratory of Jilin Province for Zoonosis Prevention and Control, Institute of Military Veterinary Medicine, Academy of Military Medical Sciences, Changchun" />
<meta name="citation_author_institution" content="College of Animal Science and Technology, Shihezi University, Shihezi, Xinjiang Uyghur Autonomous Region" />
<meta name="citation_author" content="Aiwei Zhu" />
<meta name="citation_author_institution" content="Key Laboratory of Jilin Province for Zoonosis Prevention and Control, Institute of Military Veterinary Medicine, Academy of Military Medical Sciences, Changchun" />
<meta name="citation_author" content="Yonggang Qu" />
<meta name="citation_author_institution" content="College of Animal Science and Technology, Shihezi University, Shihezi, Xinjiang Uyghur Autonomous Region" />
<meta name="citation_author" content="Changchun Tu" />
<meta name="citation_author_institution" content="Key Laboratory of Jilin Province for Zoonosis Prevention and Control, Institute of Military Veterinary Medicine, Academy of Military Medical Sciences, Changchun" />
<meta name="citation_author_institution" content="Jiangsu Co-innovation Center for Prevention and Control of Important Animal Infectious Diseases and Zoonosis, Yangzhou" />
<meta name="citation_author_email" content="heb-001001@163.com" />
<meta name="citation_author_email" content="michael.carr@ucd.ie" />
<meta name="citation_author_email" content="changchun_tu@hotmail.com" />
<meta name="citation_author" content="Michael J. Carr" />
<meta name="citation_author_institution" content="National Virus Reference Laboratory, School of Medicine, University College Dublin" />
<meta name="citation_author_email" content="heb-001001@163.com" />
<meta name="citation_author_email" content="michael.carr@ucd.ie" />
<meta name="citation_author_email" content="changchun_tu@hotmail.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0011-6209" />
<meta name="citation_author" content="Biao He" />
<meta name="citation_author_institution" content="Key Laboratory of Jilin Province for Zoonosis Prevention and Control, Institute of Military Veterinary Medicine, Academy of Military Medical Sciences, Changchun" />
<meta name="citation_author_institution" content="Jiangsu Co-innovation Center for Prevention and Control of Important Animal Infectious Diseases and Zoonosis, Yangzhou" />
<meta name="citation_author_email" content="heb-001001@163.com" />
<meta name="citation_author_email" content="michael.carr@ucd.ie" />
<meta name="citation_author_email" content="changchun_tu@hotmail.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7587-152X" />
<meta name="citation_reference" content="Wilson DE, Reeder DM. Mammal species of the world: a taxonomic and geographic reference, 3rd ed, Order Chiroptera. Baltimore, MD: Johns Hopkins University Press; 2005." />
<meta name="citation_reference" content="Streicker DG, Turmelle AS, Vonhof MJ, Kuzmin IV, McCracken GF, Rupprecht CE. Host phylogeny constrains cross-species emergence and establishment of rabies virus in bats. Science. 2010;329(5992):676. DOI:10.1126/science.1188836." />
<meta name="citation_reference" content="Towner JS, Pourrut X, Albariño CG, Nkogue CN, Bird BH, Grard G, et al. Marburg virus infection detected in a common African bat. PLoS One. 2007;2(8):e764. DOI:10.1371/journal.pone.0000764." />
<meta name="citation_reference" content="Leroy EM, Kumulungui B, Pourrut X, Rouquet P, Hassanin A, Yaba P, et al. Fruit bats as reservoirs of Ebola virus. Nature. 2005;438(7068):575. DOI:10.1038/438575a." />
<meta name="citation_reference" content="Chua KB, Lek Koh C, Hooi PS, Wee KF, Khong JH, Chua BH, et al. Isolation of Nipah virus from Malaysian Island flying-foxes. Microb Infect. 2002;4(2):145–51. DOI:10.1016/S1286-4579(01)01522-2." />
<meta name="citation_reference" content="Li W, Shi Z, Yu M, Ren W, Smith C, Epstein JH, et al. Bats are natural reservoirs of SARS-like coronaviruses. Science. 2005;310(5748):676–9. DOI:10.1126/science.1118391." />
<meta name="citation_reference" content="Lau SK, Woo PC, Li KS, Huang Y, Tsoi H-W, Wong BH, et al. Severe acute respiratory syndrome coronavirus-like virus in Chinese horseshoe bats. Proc Natl Acad Sci U S A. 2005;102(39):14040–5. DOI:10.1073/pnas.0506735102." />
<meta name="citation_reference" content="He B, Zhang Y, Xu L, Yang W, Yang F, Feng Y, et al. Identification of diverse alphacoronaviruses and genomic characterization of a novel severe acute respiratory syndrome-like coronavirus from bats in China. J Virol. 2014;88(12):7070–82. DOI:10.1128/JVI.00631-14." />
<meta name="citation_reference" content="Zhou P, Fan H, Lan T, Yang X-L, Shi W-F, Zhang W, et al. Fatal swine acute diarrhoea syndrome caused by an HKU2-related coronavirus of bat origin. Nature. 2018;556:255–8. DOI:10.1038/s41586-018-0010-9." />
<meta name="citation_reference" content="Wolfe ND, Dunavan CP, Diamond J. Origins of major human infectious diseases. Nature. 2007;447(7142):279–83. DOI:10.1038/nature05775." />
<meta name="citation_reference" content="DeCaprio JA, Imperiale MJ, Major EO. Polyomaviridae, Fields virology. 6th ed. Philadelphia: WoltersKluwer; 2013." />
<meta name="citation_reference" content="Moens U, Krumbholz A, Ehlers B, Zell R, Johne R, Calvignac-Spencer S, et al. Biology, evolution, and medical importance of polyomaviruses: An update. Infect Genet Evol. 2017;54:18–38. DOI:10.1016/j.meegid.2017.06.011." />
<meta name="citation_reference" content="Feng H, Shuda M, Chang Y, Moore PS. Clonal integration of a polyomavirus in human Merkel cell carcinoma. Science. 2008;319(5866):1096. DOI:10.1126/science.1152586." />
<meta name="citation_reference" content="Dalianis T, Hirsch HH. Human polyomaviruses in disease and cancer. Virology. 2013;437(2):63–72. DOI:10.1016/j.virol.2012.12.015." />
<meta name="citation_reference" content="Dela Cruz FN, Giannitti F, Li L, Woods LW, Del Valle L, Delwart E, et al. Novel polyomavirus associated with brain tumors in free-ranging raccoons, western United States. Emerg Infect Dis. 2013;19(1):77–84. DOI:10.3201/eid1901.121078." />
<meta name="citation_reference" content="Moens U, Calvignac-Spencer S, Lauber C, Ramqvist T, Feltkamp MCW, Daugherty MD, et al. ICTV virus taxonomy profile: Polyomaviridae. J Gen Virol. 2017;98(6):1159–60. DOI:10.1099/jgv.0.000839." />
<meta name="citation_reference" content="Buck CB, Van Doorslaer K, Peretti A, Geoghegan EM, Tisza MJ, An P, et al. The ancient evolutionary history of polyomaviruses. PLoS Pathog. 2016;12(4):e1005574. DOI:10.1371/journal.ppat.1005574." />
<meta name="citation_reference" content="Perez-Losada M, Christensen RG, McClellan DA, Adams BJ, Viscidi RP, Demma JC, et al. Comparing phylogenetic codivergence between polyomaviruses and their hosts. J Virol. 2006;80(12):5663–9. DOI:10.1128/jvi.00056-06." />
<meta name="citation_reference" content="Misra V, Dumonceaux T, Dubois J, Willis C, Nadin-Davis S, Severini A, et al. Detection of polyoma and corona viruses in bats of Canada. J Gen Virol. 2009;90(8):2015–22. DOI:10.1099/vir.0.010694-0." />
<meta name="citation_reference" content="Fagrouch Z, Sarwari R, Lavergne A, Delaval M, De Thoisy B, Lacoste V, et al. Novel polyomaviruses in South American bats and their relationship to other members of the family Polyomaviridae. J Gen Virol. 2012;93(12):2652–7. DOI:10.1099/vir.0.044149-0." />
<meta name="citation_reference" content="Tao Y, Shi M, Conrardy C, Kuzmin IV, Recuenco S, Agwanda B, et al. Discovery of diverse polyomaviruses in bats and the evolutionary history of the Polyomaviridae. J Gen Virol. 2013;94(4):738–48. DOI:10.1099/vir.0.047928-0." />
<meta name="citation_reference" content="Lima FES, Cibulski SP, Witt AA, Franco AC, Roehe PM. Genomic characterization of two novel polyomaviruses in Brazilian insectivorous bats. Arch Virol. 2015;160(7):1831–6. DOI:10.1007/s00705-015-2447-6." />
<meta name="citation_reference" content="Kobayashi S, Sasaki M, Nakao R, Setiyono A, Handharyani E, Orba Y, et al. Detection of novel polyomaviruses in fruit bats in Indonesia. Arch Virol. 2015;160(4):1075–82. DOI:10.1007/s00705-015-2349-7." />
<meta name="citation_reference" content="Wang J, Moore NE, Murray ZL, McInnes K, White DJ, Tompkins DM, et al. Discovery of novel virus sequences in an isolated and threatened bat species, the New Zealand lesser short-tailed bat (Mystacina tuberculata). J Gen Virol. 2015;96(8):2442–52. DOI:10.1099/vir.0.000158." />
<meta name="citation_reference" content="Carr M, Gonzalez G, Sasaki M, Ito K, Ishii A, Hang’ombe BM, et al. Discovery of African bat polyomaviruses and infrequent recombination in the large T antigen in the Polyomaviridae. J Gen Virol. 2017;98(4):726–38. DOI:10.1099/jgv.0.000737." />
<meta name="citation_reference" content="Carr M, Gonzalez G, Sasaki M, Dool SE, Ito K, Ishii A, et al. Identification of the same polyomavirus species in different African horseshoe bat species is indicative of short-range host-switching events. J Gen Virol. 2017;98(11):2771–85. DOI:10.1099/jgv.0.000935." />
<meta name="citation_reference" content="Johne R, Enderlein D, Nieper H, Müller H. Novel polyomavirus detected in the feces of a chimpanzee by nested broad-spectrum PCR. J Virol. 2005;79(6):3883–7. DOI:10.1128/JVI.79.6.3883-3887.2005." />
<meta name="citation_reference" content="Wu Z, Yang L, Ren X, He G, Zhang J, Yang J, et al. Deciphering the bat virome catalog to better understand tha ecological diversity of bat viruses and the bat origin of emerging infectious diseases. The ISME Journal. 2015;2015:1–12. DOI:10.1038/ismej.2015.138." />
<meta name="citation_reference" content="Cantalupo PG, Buck CB, Pipas JM. Complete genome sequence of a polyomavirus recovered from a pomona leaf-nosed bat (Hipposideros pomona) metagenome data set. Genome Announc. 2017;5(3):e01053–16. DOI:10.1128/genomeA.01053-16." />
<meta name="citation_reference" content="Mishra N, Fagbo SF, Alagaili AN, Nitido A, Williams SH, Ng J, et al. A viral metagenomic survey identifies known and novel mammalian viruses in bats from Saudi Arabia. PLoS One. 2019;14(4):e0214227. DOI:10.1371/journal.pone.0214227." />
<meta name="citation_reference" content="Hsieh H-M, Chiang H-L, Tsai L-C, Lai S-Y, Huang N-E, Linacre A, et al. Cytochrome bgene for species identification of the conservation animals. Forensic Sci Int. 2001;122(1):7–18." />
<meta name="citation_reference" content="He B, Huang X, Zhang F, Tan W, Matthijnssens J, Qin S, et al. Group A rotaviruses in Chinese bats: Genetic composition, serology, and evidence for bat-to-human transmission and reassortment. J Virol. 2017;91(12). DOI:10.1128/jvi.02493-16." />
<meta name="citation_reference" content="Xu L, Wu J, Li Q, Wei Y, Tan Z, Cai J, et al. Seroprevalence, cross antigenicity and circulation sphere of bat-borne hantaviruses revealed by serological and antigenic analyses. PLoS Pathog. 2019;15(1):e1007545. DOI:10.1371/journal.ppat.1007545." />
<meta name="citation_reference" content="He B, Li Z, Yang F, Zheng J, Feng Y, Guo H, et al. Virome profiling of bats from Myanmar by metagenomic analysis of tissue samples reveals more novel mammalian viruses. PLoS One. 2013;8(4):e61950. DOI:10.1371/journal.pone.0061950." />
<meta name="citation_reference" content="Ehlers B, Richter D, Matuschka FR, Ulrich RG. Genome sequences of a rat polyomavirus related to murine polyomavirus, Rattus norvegicus polyomavirus 1. Genome Announc. 2015;3(5). DOI:10.1128/genomeA.00997-15." />
<meta name="citation_reference" content="Calvignac-Spencer S, Feltkamp MC, Daugherty MD, Moens U, Ramqvist T, Johne R, et al. A taxonomy update for the family Polyomaviridae. Arch Virol. 2016;161(6):1739–50." />
<meta name="citation_reference" content="Sun K, Feng J, Zhang Z, Xu L, Liu Y. Cryptic diversity in Chinese rhinolophids and hipposiderids (Chiroptera: Rhinolophidae and Hipposideridae). Mammalia. 2009;73(2):135–41. DOI:10.1515/MAMM.2009.022." />
<meta name="citation_reference" content="Tan Z, Yu H, Xu L, Zhao Z, Zhang P, Qu Y, et al. Virome profiling of rodents in Xinjiang Uygur Autonomous Region, China: Isolation and characterization of a new strain of Wenzhou virus. Virology. 2019;529:122–34. DOI:10.1016/j.virol.2019.01.010." />
<meta name="citation_reference" content="Dool SE, Puechmaille SJ, Foley NM, Allegrini B, Bastian A, Mutumi GL, et al. Nuclear introns outperform mitochondrial DNA in inter-specific phylogenetic reconstruction: lessons from horseshoe bats (Rhinolophidae: Chiroptera). Mol Phylogenet Evol. 2016;97:196–212. DOI:10.1016/j.ympev.2016.01.003." />
<meta name="citation_reference" content="Agnarsson I, Zambrana-Torrelio CM, Flores-Saldana NP, May-Collado LJ. A time-calibrated species-level phylogeny of bats (Chiroptera, Mammalia). PLoS currents. 2011;3:Rrn1212. DOI:10.1371/currents.RRN1212." />
<meta name="citation_reference" content="Meredith RW, Janecka JE, Gatesy J, Ryder OA, Fisher CA, Teeling EC, et al. Impacts of the Cretaceous Terrestrial Revolution and KPg extinction on mammal diversification. Science. 2011;334(6055):521. DOI:10.1126/science.1211028." />
<meta name="citation_reference" content="Stehle T, Yan Y, Benjamin TL, Harrison SC. Structure of murine polyomavirus complexed with an oligosaccharide receptor fragment. Nature. 1994;369(6476):160–3. DOI:10.1038/369160a0." />
<meta name="citation_reference" content="Neu U, Woellner K, Gauglitz G, Stehle T. Structural basis of GM1 ganglioside recognition by simian virus 40. Proc Natl Acad Sci U S A. 2008;105(13):5219–24. DOI:10.1073/pnas.0710301105." />
<meta name="citation_reference" content="Neu U, Maginnis MS, Palma AS, Stroh LJ, Nelson CD, Feizi T, et al. Structure-function analysis of the human JC polyomavirus establishes the LSTc pentasaccharide as a functional receptor motif. Cell Host Microbe. 2010;8(4):309–19. DOI:10.1016/j.chom.2010.09.004." />
<meta name="citation_reference" content="Neu U, Hengel H, Blaum BS, Schowalter RM, Macejak D, Gilbert M, et al. Structures of Merkel cell polyomavirus VP1 complexes define a sialic acid binding site required for infection. PLoS Pathog. 2012;8(7):e1002738. DOI:10.1371/journal.ppat.1002738." />
<meta name="citation_reference" content="Hurdiss DL, Frank M, Snowden JS, Macdonald A, Ranson NA. The structure of an infectious human polyomavirus and its interactions with cellular receptors. Structure. 2018;26(6):839-47.e3. DOI:10.1016/j.str.2018.03.019." />
<meta name="citation_reference" content="Warden CD, Lacey SF. Updated phylogenetic analysis of polyomavirus-host co-evolution. J Bioinfo Res. 2012;1:46–9." />
<meta name="citation_reference" content="Krumbholz A, Bininda-Emonds OR, Wutzler P, Zell R. Evolution of four BK virus subtypes. Infect Genet Evol. 2008;8(5):632–43. DOI:10.1016/j.meegid.2008.05.006." />
<meta name="citation_reference" content="Johne R, Muller H. Polyomaviruses of birds: etiologic agents of inflammatory diseases in a tumor virus family. J Virol. 2007;81(21):11554–9. DOI:10.1128/jvi.01178-07." />
<meta name="citation_reference" content="Nie FY, Lin XD, Hao ZY, Chen XN, Wang ZX, Wang MR, et al. Extensive diversity and evolution of hepadnaviruses in bats in China. Virology. 2018;514:88–97. DOI:10.1016/j.virol.2017.11.005." />
<meta name="citation_reference" content="Eckerle I, Ehlen L, Kallies R, Wollny R, Corman VM, Cottontail VM, et al. Bat airway epithelial cells: a novel tool for the study of zoonotic viruses. PLoS One. 2014;9(1):e84679. DOI:10.1371/journal.pone.0084679." />
<meta name="citation_reference" content="Banerjee A, Rapin N, Miller M, Griebel P, Zhou Y, Munster V, et al. Generation and characterization of Eptesicus fuscus (Big brown bat) kidney cell lines immortalized using the Myotis polyomavirus large T-antigen. J Virol Methods. 2016;237:166–73. DOI:10.1016/j.jviromet.2016.09.008." />
<meta name="citation_reference" content="García-Pérez R, Ibáñez C, Godínez JM, Aréchiga N, Garin I, Pérez-Suárez G, et al. Novel papillomaviruses in free-ranging Iberian bats: no virus–host co-evolution, no strict host specificity, and hints for recombination. Genome Biol Evol. 2014;6(1):94–104. DOI:10.1093/gbe/evt211" />
<meta name="citation_reference" content="Corman VM, Ithete NL, Richards LR, Schoeman MC, Preiser W, Drosten C, et al. Rooting the phylogenetic tree of middle East respiratory syndrome coronavirus by characterization of a conspecific virus from an African bat. J Virol. 2014;88(19):11297–303. DOI:10.1128/jvi.01498-14." />
<meta name="citation_reference" content="Peretti A, FitzGerald PC, Bliskovsky V, Buck CB, Pastrana DV. Hamburger polyomaviruses. J Gen Virol. 2015;96(Pt 4):833–9. DOI:10.1099/vir.0.000033." />
<meta name="citation_reference" content="Cook L. Polyomaviruses, Diagnostic microbiology of the immunocompromised host. 2 ed. Washington, DC: ASM Press; 2016." />
<meta name="citation_reference" content="Beckervordersandforth J, Pujari S, Rennspiess D, Speel EJ, Winnepenninckx V, Diaz C, et al. Frequent detection of human polyomavirus 6 in keratoacanthomas. Diagn Pathol. 2016;11(1):58. DOI:10.1186/s13000-016-0509-z." />
<meta name="citation_reference" content="Hausen Hz. Red meat consumption and cancer: reasons to suspect involvement of bovine infectious factors in colorectal cancer. Int J Cancer. 2012;130(11):2475–83. DOI:10.1002/ijc.27413." />
<meta name="citation_reference" content="Sauerbrei A, Wutzler P. Testing thermal resistance of viruses. Arch Virol. 2009;154(1):115–9. DOI:10.1007/s00705-008-0264-x." />
<meta name="citation_reference" content="Anthony SJ, Islam A, Johnson C, Navarrete-Macias I, Liang E, Jain K, et al. Non-random patterns in viral diversity. Nat Commun. 2015;6:8147. DOI:10.1038/ncomms9147." />
<meta name="citation_reference" content="Campanero-Rhodes MA, Smith A, Chai W, Sonnino S, Mauri L, Childs RA, et al. N-glycolyl GM1 ganglioside as a receptor for simian virus 40. J Virol. 2007;81(23):12846–58. DOI:10.1128/jvi.01311-07." />
<meta name="citation_reference" content="O’Hara SD, Stehle T, Garcea R. Glycan receptors of the Polyomaviridae: Structure, function, and pathogenesis. Curr Opin Virol. 2014;7:73–8. DOI:10.1016/j.coviro.2014.05.004." />
<meta name="citation_reference" content="Jin L, Gibson PE, Knowles WA, Clewley JP. BK virus antigenic variants: sequence analysis within the capsid VP1 epitope. J Med Virol. 1993;39(1):50–6. DOI:10.1002/jmv.1890390110" />
<meta name="citation_reference" content="Erickson KD, Garcea RL, Tsai B. Ganglioside GT1b is a putative host cell receptor for the Merkel cell polyomavirus. J Virol. 2009;83(19):10275–9. DOI:10.1128/jvi.00949-09." />
<meta name="citation_reference" content="Murata H, Teferedegne B, Sheng L, Lewis AM, Peden K. Identification of a neutralization epitope in the VP1 capsid protein of SV40. Virology. 2008;381(1):116–22. DOI:10.1016/j.virol.2008.07.032." />
<meta name="citation_reference" content="Fleury MJ, Nicol JT, Samimi M, Arnold F, Cazal R, Ballaire R, et al. Identification of the neutralizing epitopes of Merkel cell polyomavirus major capsid protein within the BC and EF surface loops. PLoS One. 2015;10(3):e0121751. DOI:10.1371/journal.pone.0121751." />
<meta name="citation_reference" content="Ströh LJ, Neu U, Blaum BS, Buch MHC, Garcea RL, Stehle T. Structure analysis of the major capsid proteins of human polyomaviruses 6 and 7 reveals an obstructed sialic acid binding site. J Virol. 2014;88(18):10831. DOI:10.1128/JVI.01084-14." />
<meta name="citation_reference" content="Teeling EC, Vernes SC, Davalos LM, Ray DA, Gilbert MTP, Myers E. Bat biology, genomes, and the Bat1K project: To generate chromosome-level genomes for all Living bat species. Annu Rev Anim Biosci. 2018;6:23–46. DOI:10.1146/annurev-animal-022516-022811." />
<meta name="citation_reference" content="Orba Y, Kobayashi S, Nakamura I, Ishii A, Hang, Apos, et al. Detection and characterization of a novel polyomavirus in wild rodents. J Gen Virol. 2011;92(4):789–95. DOI:DOI:10.1099/vir.0.027854-0." />
<meta name="citation_reference" content="Muhire BM, Varsani A, Martin DP. SDT: A virus classification tool based on pairwise sequence alignment and identity calculation. PLoS One. 2014;9(9):e108277. DOI:10.1371/journal.pone.0108277." />
<meta name="citation_reference" content="Katoh K, Standley DM. MAFFT multiple sequence alignment software version 7: Improvements in performance and usability. Mol Biol Evol. 2013;30(4):772–80. DOI:10.1093/molbev/mst010." />
<meta name="citation_reference" content="Ronquist F, Teslenko M, van der Mark P, Ayres DL, Darling A, Höhna S, et al. MrBayes 3.2: Efficient Bayesian phylogenetic inference and model choice across a large model space. Syst Biol. 2012;61(3):539–42. DOI:10.1093/sysbio/sys029." />
<meta name="citation_reference" content="Carr M, Kawaguchi A, Sasaki M, Gonzalez G, Ito K, Thomas Y, et al. Isolation of a simian immunodeficiency virus from a malbrouck (Chlorocebus cynosuros). Arch Virol. 2017;162(2):543-8. Epub 2016/11/03. DOI:10.1007/s00705-016-3129-8." />
<meta name="citation_reference" content="Kumar S, Stecher G, Tamura K. MEGA7: molecular evolutionary genetics analysis version 7.0 for bigger datasets. Mol Biol Evol. 2016;33(7):1870–4. DOI:10.1093/molbev/msw054." />
<meta name="citation_reference" content="Drummond AJ, Suchard MA, Xie D, Rambaut A. Bayesian phylogenetics with BEAUti and the BEAST 1.7. Mol Biol Evol. 2012;29(8):1969–73. DOI:10.1093/molbev/mss075." />
<meta name="citation_reference" content="Perelman P, Johnson WE, Roos C, Seuánez HN, Horvath JE, Moreira MAM, et al. A molecular phylogeny of living primates. PLoS Genet. 2011;7(3):e1001342. DOI:10.1371/journal.pgen.1001342." />
<meta name="citation_reference" content="Conow C, Fielder D, Ovadia Y, Libeskind-Hadas R. Jane: a new tool for the cophylogeny reconstruction problem. Algorithms Mol Biol. 2010;5(1):16. DOI:10.1186/1748-7188-5-16." />
<meta name="citation_reference" content="Waterhouse A, Bertoni M, Bienert S, Studer G, Tauriello G, Gumienny R, et al. SWISS-MODEL: homology modelling of protein structures and complexes. Nucleic Acids Res. 2018;46(W1):W296-W303. DOI:10.1093/nar/gky427." />
<meta name="twitter:title" content="Extensive genetic diversity of bat-borne polyomaviruses reveals inter-family host-switching events" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Polyomaviruses (PyVs) are small, double-stranded DNA tumor viruses carried by diverse vertebrates. PyVs have previously been considered highly host restricted in mammalian hosts, with host-switching events thought rare or nonexistent. Prior investigations have revealed short-range host-switching events of PyVs in two different African bat species within the horseshoe bat genus Rhinolophus . Herein, we have conducted a systematic investigation of PyVs in 1,083 archived bat samples collected from five provinces across China, and identified 192 PyVs from 186 bats from 15 host species within 6 families (Rhinolophidae, Vespertilionidae, Hipposideridae, Emballonuridae, Miniopteridae and Pteropodidae) representing 28 newly-described PyVs, indicative of extensive genetic diversity of bat PyVs. Surprisingly, two PyVs were identified in multiple bat species from different families, and another PyV clustered phylogenetically with PyVs carried by bats from a different host family, indicative of three inter-family PyV host-switching events. The time to most recent common ancestor (tMRCA) of the three events was estimated at 0.02-11.6 million years ago (MYA), which is inconsistent with the estimated tMRCA of their respective bat hosts (36.3-66.7 MYA), and is most parsimoniously explained by host-switching events. PyVs identified from geographically separated Chinese horseshoe bat species in the present study showed close genetic identities, and clustered with each other and with PyVs from African horseshoe bats, allowing assessment of the effects of positive selection in VP1 within the horseshoe bat family Rhinolophidae. Correlation analysis indicated that co-evolution with their hosts contributed much more to evolutionary divergence of PyV than geographic distance. In conclusion, our findings provide the first evidence of inter-family host-switching events of PyV in mammals and challenge the prevailing evolutionary paradigm for strict host restriction of mammalian PyVs.

Author summary Since the discovery of murine polyomavirus in the 1950s, polyomaviruses (PyVs) have been considered both genetically stable and highly host-restricted in their mammalian hosts. In this study, we have identified multiple cases of host-switching events of PyVs by large scale surveillance in diverse bat species collected in China. These host-switching events occurred between bat families living in the same colony, indicating that a large population with frequent contacts between different bat species may represent an ecological niche facilitating PyV host-switching. The cases studied involved members of bats from several families, including horseshoe bats, which were previously found to harbor a number of highly virulent viruses to both humans and domestic animals. Our findings have provided evidence that even highly host-specific DNA viruses can transmit between bats of different species and indicate an increased propensity for spillover events involving horseshoe bats. We propose an evolutionary scheme for bat-borne PyVs in which intra-host divergence and host-switching has generated the diverse PyVs in present day bats. This scheme provides a useful model to study the evolution of PyVs in other hosts and, potentially, the modeling of bat zoonoses and the transmission of other DNA viruses in other mammals, including humans." />
<meta name="og-title" property="og:title" content="Extensive genetic diversity of bat-borne polyomaviruses reveals inter-family host-switching events" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/627158v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Polyomaviruses (PyVs) are small, double-stranded DNA tumor viruses carried by diverse vertebrates. PyVs have previously been considered highly host restricted in mammalian hosts, with host-switching events thought rare or nonexistent. Prior investigations have revealed short-range host-switching events of PyVs in two different African bat species within the horseshoe bat genus Rhinolophus . Herein, we have conducted a systematic investigation of PyVs in 1,083 archived bat samples collected from five provinces across China, and identified 192 PyVs from 186 bats from 15 host species within 6 families (Rhinolophidae, Vespertilionidae, Hipposideridae, Emballonuridae, Miniopteridae and Pteropodidae) representing 28 newly-described PyVs, indicative of extensive genetic diversity of bat PyVs. Surprisingly, two PyVs were identified in multiple bat species from different families, and another PyV clustered phylogenetically with PyVs carried by bats from a different host family, indicative of three inter-family PyV host-switching events. The time to most recent common ancestor (tMRCA) of the three events was estimated at 0.02-11.6 million years ago (MYA), which is inconsistent with the estimated tMRCA of their respective bat hosts (36.3-66.7 MYA), and is most parsimoniously explained by host-switching events. PyVs identified from geographically separated Chinese horseshoe bat species in the present study showed close genetic identities, and clustered with each other and with PyVs from African horseshoe bats, allowing assessment of the effects of positive selection in VP1 within the horseshoe bat family Rhinolophidae. Correlation analysis indicated that co-evolution with their hosts contributed much more to evolutionary divergence of PyV than geographic distance. In conclusion, our findings provide the first evidence of inter-family host-switching events of PyV in mammals and challenge the prevailing evolutionary paradigm for strict host restriction of mammalian PyVs.

Author summary Since the discovery of murine polyomavirus in the 1950s, polyomaviruses (PyVs) have been considered both genetically stable and highly host-restricted in their mammalian hosts. In this study, we have identified multiple cases of host-switching events of PyVs by large scale surveillance in diverse bat species collected in China. These host-switching events occurred between bat families living in the same colony, indicating that a large population with frequent contacts between different bat species may represent an ecological niche facilitating PyV host-switching. The cases studied involved members of bats from several families, including horseshoe bats, which were previously found to harbor a number of highly virulent viruses to both humans and domestic animals. Our findings have provided evidence that even highly host-specific DNA viruses can transmit between bats of different species and indicate an increased propensity for spillover events involving horseshoe bats. We propose an evolutionary scheme for bat-borne PyVs in which intra-host divergence and host-switching has generated the diverse PyVs in present day bats. This scheme provides a useful model to study the evolution of PyVs in other hosts and, potentially, the modeling of bat zoonoses and the transmission of other DNA viruses in other mammals, including humans." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-05-03" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/627158v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/627158v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/627158v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/627158v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/682555" />
    <title>Extensive genetic diversity of bat-borne polyomaviruses reveals inter-family host-switching events | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ks4vmPEKmboSZrY0qvC3JaKgXiM2Wmc_uxFt08UxBWk","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"682555","apath":"\/biorxiv\/early\/2019\/05\/03\/627158.atom","pisa":"biorxiv;627158v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;627158v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/627158v1","title":"Extensive genetic diversity of bat-borne polyomaviruses reveals inter-family host-switching events","identifier":"node\/682555"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022op8nlMgjFIg8YICbUYkxt7mUjIz5pdMismCCqPxtfuk+ipVQbpVTHwlf8pAzxfMzrCoe88xfiBSo162WQD2XNcoWJbz2wDxnSn3A9hDKmQDZt1Yy7a5QDRXbkqyREkje6nNLjhBKhH27aPSlblHFFqoh4w2zZWY1EqlfdQ4yO4BrArv9DF4R+RNK3duhVS+0sq7gjHx79ap6S6AqLrqkFNCWhjgJCKyQwiJs3VUNf1r0vkvt9Smk2RaVjc7wWcUEdFL+DW\\\\\\\/ZfeFDRFVwA5bqgmJNXhSinfnu1+LNKW1jIaL3sPcZQVDNbWHa7kNvk0p9piRwQDoKJCNlZ4\\\\\\\/A1okC5MmI6ttBCwqJxePqJmiEf8A\\\\\\\/1uH7r3I8QbbeUCMjoebw\\\\\\\/dYGMQrKBlJDNJORzzpDzrgWEqXcrkLZBgbiQ\\\\\\\/3Wd+vaMDYoB239njZk+Oe7hVpjmCyU739aLzkJaIEZ2TAcyD7E1+CMTcMZbXoHB6jYuPlKPA8Zb8693JWzU6CWVw1Ejbn55E1F9SFFfXILGLB8PfY+pXH58cmisxB1afZ0yU9CWr4tZXG3cVQ3irP+IRSvyKbGC3pV4jmtbtraDXbHFMCskeHlAbcxdZiLQu5iFoZz9\\\\\\\/GMaZITPrmn\\\\\\\/z1hx7iF4G6BAi8tJqbQXUUx0Dm6k4B0Y5\\\\\\\/ZNWXLds+pcSDBwNOW86nKnKy5VCzc+99jpfJUEgHgKOkACAhCM1zwDpW5DVAhintPp+DcgFkENEs51xOZAr8qAZXvaxFLi\\\\\\\/+vJvI+CJiwLhKdgS8wJCAiNwNe6UqmmRI4QaUL2PyTuyNLamTpBT\\\\\\\/pBBrSSIxbAyaOA\\\\\\\/LM4asfr5yzGOkKz+oy1BMyg3Bl2kwniY1IM7ob\\\\\\\/hCVj0vThnmvFWWRmokc2MTXWt\\\\\\\/p5MKRLnXJPiJMJY7x8pB4yu0zAj9A5iwB8sRR+428TU\\\\\\\/nav+PyozdHPHuFQqtWuqaXrTkFnAjMVRiM\\\\\\\/ZiOtbndMvDctvpdrhEAICGyb5Q2rchO7goNyBLuZ0Y7dYrAASq3jRM5X0O\\\\\\\/gDOycLAi3I3+DMLnFLmDSszk7MHnd5dYgF7Ljtzj6rayJnqn4HcXkbUAycRgf4rZOaIPf+w2T6RxV9rez60V\\\\\\\/RY5pMrh7mgpzGyHKqrzuGwUoXKJ4MuTJaaTuZUv6ZvGF3kZz5IRMrgwMaAVrAJhC5lmJKY8wARl0MsLa\\\\\\\/Ou5qetkKNo5z9m6iQ4qBANqkGt5+mL7Dnk7YcceMGDWMW4ixQ8eCfzI4REs5fvOv3gNBF0lK4TtJArpbO01H+Q1YiXmNspgDKrGjjXxd+iW6uRua9LW8l7\\\\\\\/hXG6w=\\\u0022,\\\u0022iv\\\u0022:\\\u0022yiiAzJXYc8HuEk3ufL30RA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022f1bceaa7339e148acf557821fcc310bd847acfbfd92a8032d30a2f2c834eceb9\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/627158v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-682555 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/627158v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-rVsX1ogThSTP4SPf1NZ8MMMBhxfvvirhhE08kUhubLE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="682555" id="node682555" data-pisa="biorxiv;627158v1" data-pisa-master="biorxiv;627158" data-apath="/biorxiv/early/2019/05/03/627158.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Extensive genetic diversity of bat-borne polyomaviruses reveals inter-family host-switching events</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2844-9673" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Zhizhou</span> <span class="nlm-surname">Tan</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2180-2120" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Gabriel</span> <span class="nlm-surname">Gonzalez</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jinliang</span> <span class="nlm-surname">Sheng</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Jianmin</span> <span class="nlm-surname">Wu</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Fuqiang</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Lin</span> <span class="nlm-surname">Xu</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Peisheng</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Aiwei</span> <span class="nlm-surname">Zhu</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Yonggang</span> <span class="nlm-surname">Qu</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Changchun</span> <span class="nlm-surname">Tu</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0003-0011-6209" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Michael J.</span> <span class="nlm-surname">Carr</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11"><a href="http://orcid.org/0000-0001-7587-152X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Biao</span> <span class="nlm-surname">He</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/627158 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node682555" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhizhou Tan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Key Laboratory of Jilin Province for Zoonosis Prevention and Control, Institute of Military Veterinary Medicine, Academy of Military Medical Sciences, Changchun</span>, Jilin Province, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhizhou%2BTan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tan%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhizhou%2BTan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2844-9673" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Zhizhou Tan</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gabriel Gonzalez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>National Advanced Computing Collaboratory, National Center for High Technology</span>, San Jose, Costa Rica</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gabriel%2BGonzalez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gonzalez%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGabriel%2BGonzalez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2180-2120" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gabriel Gonzalez</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jinliang Sheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>College of Animal Science and Technology, Shihezi University, Shihezi, Xinjiang Uyghur Autonomous Region</span>, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jinliang%2BSheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sheng%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJinliang%2BSheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jianmin Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Guangxi Key Laboratory of Veterinary Biotechnology, Guangxi Veterinary Research Institute, Nanning</span>, Guangxi Province, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jianmin%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJianmin%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Fuqiang Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Center for Disease Control and Prevention of Southern Theater Command</span>, Kunming, Yunnan Province, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fuqiang%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFuqiang%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Lin Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Key Laboratory of Jilin Province for Zoonosis Prevention and Control, Institute of Military Veterinary Medicine, Academy of Military Medical Sciences, Changchun</span>, Jilin Province, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lin%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALin%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Peisheng Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Key Laboratory of Jilin Province for Zoonosis Prevention and Control, Institute of Military Veterinary Medicine, Academy of Military Medical Sciences, Changchun</span>, Jilin Province, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>College of Animal Science and Technology, Shihezi University, Shihezi, Xinjiang Uyghur Autonomous Region</span>, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peisheng%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeisheng%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Aiwei Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Key Laboratory of Jilin Province for Zoonosis Prevention and Control, Institute of Military Veterinary Medicine, Academy of Military Medical Sciences, Changchun</span>, Jilin Province, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aiwei%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAiwei%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Yonggang Qu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>College of Animal Science and Technology, Shihezi University, Shihezi, Xinjiang Uyghur Autonomous Region</span>, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yonggang%2BQu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Qu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYonggang%2BQu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Changchun Tu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Key Laboratory of Jilin Province for Zoonosis Prevention and Control, Institute of Military Veterinary Medicine, Academy of Military Medical Sciences, Changchun</span>, Jilin Province, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Jiangsu Co-innovation Center for Prevention and Control of Important Animal Infectious Diseases and Zoonosis, Yangzhou</span>, Jiangsu Province, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Changchun%2BTu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tu%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChangchun%2BTu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:heb-001001@163.com" class="" data-icon-position="" data-hide-link-title="0">heb-001001@163.com</a>
<a href="mailto:michael.carr@ucd.ie" class="" data-icon-position="" data-hide-link-title="0">michael.carr@ucd.ie</a>
<a href="mailto:changchun_tu@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">changchun_tu@hotmail.com</a></span></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Michael J. Carr </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>National Virus Reference Laboratory, School of Medicine, University College Dublin</span>, Dublin 4, <span class='nlm-country'>Ireland</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BJ.%2BCarr%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carr%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BJ.%2BCarr%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0011-6209" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael J. Carr</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:heb-001001@163.com" class="" data-icon-position="" data-hide-link-title="0">heb-001001@163.com</a>
<a href="mailto:michael.carr@ucd.ie" class="" data-icon-position="" data-hide-link-title="0">michael.carr@ucd.ie</a>
<a href="mailto:changchun_tu@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">changchun_tu@hotmail.com</a></span></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Biao He </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Key Laboratory of Jilin Province for Zoonosis Prevention and Control, Institute of Military Veterinary Medicine, Academy of Military Medical Sciences, Changchun</span>, Jilin Province, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Jiangsu Co-innovation Center for Prevention and Control of Important Animal Infectious Diseases and Zoonosis, Yangzhou</span>, Jiangsu Province, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Biao%2BHe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=He%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABiao%2BHe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7587-152X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Biao He</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:heb-001001@163.com" class="" data-icon-position="" data-hide-link-title="0">heb-001001@163.com</a>
<a href="mailto:michael.carr@ucd.ie" class="" data-icon-position="" data-hide-link-title="0">michael.carr@ucd.ie</a>
<a href="mailto:changchun_tu@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">changchun_tu@hotmail.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/627158v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:682555" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:682555/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/627158v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:682555" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:682555/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/627158v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:682555" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:682555/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/627158v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:682555" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:682555/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/627158v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:682555" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:682555/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">Polyomaviruses (PyVs) are small, double-stranded DNA tumor viruses carried by diverse vertebrates. PyVs have previously been considered highly host restricted in mammalian hosts, with host-switching events thought rare or nonexistent. Prior investigations have revealed short-range host-switching events of PyVs in two different African bat species within the horseshoe bat genus <em>Rhinolophus</em>. Herein, we have conducted a systematic investigation of PyVs in 1,083 archived bat samples collected from five provinces across China, and identified 192 PyVs from 186 bats from 15 host species within 6 families (Rhinolophidae, Vespertilionidae, Hipposideridae, Emballonuridae, Miniopteridae and Pteropodidae) representing 28 newly-described PyVs, indicative of extensive genetic diversity of bat PyVs. Surprisingly, two PyVs were identified in multiple bat species from different families, and another PyV clustered phylogenetically with PyVs carried by bats from a different host family, indicative of three inter-family PyV host-switching events. The time to most recent common ancestor (tMRCA) of the three events was estimated at 0.02-11.6 million years ago (MYA), which is inconsistent with the estimated tMRCA of their respective bat hosts (36.3-66.7 MYA), and is most parsimoniously explained by host-switching events. PyVs identified from geographically separated Chinese horseshoe bat species in the present study showed close genetic identities, and clustered with each other and with PyVs from African horseshoe bats, allowing assessment of the effects of positive selection in VP1 within the horseshoe bat family Rhinolophidae. Correlation analysis indicated that co-evolution with their hosts contributed much more to evolutionary divergence of PyV than geographic distance. In conclusion, our findings provide the first evidence of inter-family host-switching events of PyV in mammals and challenge the prevailing evolutionary paradigm for strict host restriction of mammalian PyVs.</p><div id="sec-1" class="subsection"><p id="p-4"><strong>Author summary</strong> Since the discovery of murine polyomavirus in the 1950s, polyomaviruses (PyVs) have been considered both genetically stable and highly host-restricted in their mammalian hosts. In this study, we have identified multiple cases of host-switching events of PyVs by large scale surveillance in diverse bat species collected in China. These host-switching events occurred between bat families living in the same colony, indicating that a large population with frequent contacts between different bat species may represent an ecological niche facilitating PyV host-switching. The cases studied involved members of bats from several families, including horseshoe bats, which were previously found to harbor a number of highly virulent viruses to both humans and domestic animals. Our findings have provided evidence that even highly host-specific DNA viruses can transmit between bats of different species and indicate an increased propensity for spillover events involving horseshoe bats. We propose an evolutionary scheme for bat-borne PyVs in which intra-host divergence and host-switching has generated the diverse PyVs in present day bats. This scheme provides a useful model to study the evolution of PyVs in other hosts and, potentially, the modeling of bat zoonoses and the transmission of other DNA viruses in other mammals, including humans.</p></div></div><div class="section" id="sec-2"><h2 class="">Introduction</h2><p id="p-5">Bats (order Chiroptera) account for more than 20% of mammalian species with &gt; 1,100 known species in &gt; 200 genera worldwide [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>]. These diverse mammals harbor numerous highly virulent viral pathogens, including rabies virus [<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>], Marburg virus [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>], Ebola virus [<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>], Nipah virus [<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>], severe acute respiratory syndrome (SARS)-related coronavirus [<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>-<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>] and the newly identified swine acute diarrhea syndrome coronavirus [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>]. Spillover of viruses from bats to humans or domestic animals can give rise to outbreaks of emerging infectious diseases [<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>].</p><p id="p-6">Polyomaviruses (PyVs) are non-enveloped icosahedral DNA viruses containing a circular and highly stable double-stranded DNA genome [<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>]. PyVs can induce neoplastic transformation in cell culture [<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>], and are associated with malignancy in humans [<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>] and raccoons [<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>]. Currently, 98 species carried by various mammals, birds and fish are recognized within the family <em>Polyomaviridae</em> by the International Committee on Taxonomy of Viruses (ICTV) and classified into four separate genera (<em>Alpha-, Beta-, Gamma-</em> and <em>Deltapolyomavirus</em>) with nine floating species [<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>]. Recent identification of diverse PyV-like sequences in different fish species [<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>] and invertebrates (including arachnids and insects) has revealed a deep co-evolutionary history of PyVs with their metazoan hosts [<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>]. PyVs are highly host-specific in different mammalian species, and long-range host jumps leading to productive infection and transmission within mammalian genera are considered rare or nonexistent [<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>]. Recently, an intra-host divergence model with contributions from viral lineage duplication, recombination and, potentially, host-switching at evolutionary timescales has been proposed to explain the extant diversity based on the available genetic data [<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">17</a>].</p><p id="p-7">Currently, 15 alphapolyomaviruses and 10 betapolyomaviruses have been formally recognized in insectivorous and frugivorous bat species within the Americas, Africa, Southeast Asia (Indonesia) and the Pacific (New Zealand) [<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>-<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>]. These studies identified significant diversity and high positive rates (10-20 %) using PCR-based approaches [<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>]. PyVs with complete or partial genomes has also been identified by metagenomic analyses in bats collected in China and recently from Saudi Arabia [<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>-<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>]. Investigation of PyVs in African insectivorous and frugivorous bats [<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>, <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a>] has revealed evidence for short-range host-switching events of PyVs in horseshoe bats (family Rhinolophidae, genus <em>Rhinolophus</em>) following identification of nearly identical viruses (99.9% at the genomic level) in <em>Rh. blasii</em> and <em>Rh. simulator</em> [<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">26</a>].</p><p id="p-8">In the present study, we have characterized the diversity and genetic relationships of novel PyVs from a large number of insectivorous and frugivorous bat samples collected in diverse ecological environments in China, resulting in the identification of inter-family host-switching events in these bat-borne PyVs. The findings challenge the prevailing paradigm of intra-host PyV diversification in their mammalian hosts and extend the knowledge of PyV genetic diversity, evolution, host restriction and host-switching.</p></div><div class="section" id="sec-3"><h2 class="">Results</h2><div id="sec-4" class="subsection"><h3>Sampling information and detection of polyomaviruses</h3><p id="p-9">In 2015-2016, a total of 1,083 bats belonging to 20 species in 6 families (Rhinolophidae, Vespertilionidae, Hipposideridae, Emballonuridae, Miniopteridae and Pteropodidae) were collected from 29 different colonies (colonies 1-29 in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig 1</a> and S1 Table) from five provinces in southwest (Yunnan and Guangxi), southeast (Fujian and Zhejiang) and northwest (Xinjiang) provinces in China. The bats were identified morphologically by field-trained experts during sampling and confirmed by partial mitochondrial <em>cytochrome b</em> (<em>cytb</em>) gene sequencing [<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>] of representative individuals in every colony (S1 Table). The bats were insectivorous except for one fruit bat species <em>Rousettus leschenaultii</em> in the family Pteropodidae, and were captured in diverse habitats, including natural and artificial caves, orchards and human-made structures (S1 Table). Archived tissues from individual bats had previously been sampled for zoonotic virus surveillance, including for group A rotaviruses (RVA) [<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>] and hantaviruses [<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>].</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sampling locations and species composition of animals collected in different provinces. The filled circles represent the locations of sampled colonies encoded by numbers (see details in S1 File) with the red ones representing PyV-positive colonies or mammalian species and the grey ones representing the PyV-negative colonies or mammalian species (species names of PyV negative bats are not shown). The composition of collected bats or rodents in each province is presented by the colored rings with total numbers in the rings. The color region in the rings represents each positive bat families or rodent species as indicated in the top right-hand corner. Every filled circle around the colored rings represents one mammalian species. The numbers of PyV positive bats in each species are shown in parentheses. Abbreviations of provinces: XJ, Xinjiang Uyghur Autonomous Region; YN, Yunnan Province; GX, Guangxi Zhuang Autonomous Region; FJ, Fujian Province; ZJ, Zhejiang Province. Abbreviations of bat genera: H., Hipposideros; A., Aselliscus; Rh., Rhinolophus; S., Scotophilus; Mi., Miniopterus; My., Myotis; T., Taphozous; Ro., Rousettus." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1269924037" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Sampling locations and species composition of animals collected in different provinces.&lt;/span&gt; The filled circles represent the locations of sampled colonies encoded by numbers (see details in S1 File) with the red ones representing PyV-positive colonies or mammalian species and the grey ones representing the PyV-negative colonies or mammalian species (species names of PyV negative bats are not shown). The composition of collected bats or rodents in each province is presented by the colored rings with total numbers in the rings. The color region in the rings represents each positive bat families or rodent species as indicated in the top right-hand corner. Every filled circle around the colored rings represents one mammalian species. The numbers of PyV positive bats in each species are shown in parentheses. Abbreviations of provinces: XJ, Xinjiang Uyghur Autonomous Region; YN, Yunnan Province; GX, Guangxi Zhuang Autonomous Region; FJ, Fujian Province; ZJ, Zhejiang Province. Abbreviations of bat genera: &lt;em&gt;H&lt;/em&gt;., &lt;em&gt;Hipposideros; A&lt;/em&gt;., &lt;em&gt;Aselliscus; Rh&lt;/em&gt;., &lt;em&gt;Rhinolophus; S&lt;/em&gt;., &lt;em&gt;Scotophilus; Mi&lt;/em&gt;., &lt;em&gt;Miniopterus; My&lt;/em&gt;., &lt;em&gt;Myotis; T&lt;/em&gt;., &lt;em&gt;Taphozous; Ro&lt;/em&gt;., &lt;em&gt;Rousettus&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F1.medium.gif" width="440" height="335"/><noscript><img class="highwire-fragment fragment-image" alt="Fig 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F1.medium.gif" width="440" height="335"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Fig 1.</span> <span class="caption-title">Sampling locations and species composition of animals collected in different provinces.</span><p id="p-10" class="first-child">The filled circles represent the locations of sampled colonies encoded by numbers (see details in S1 File) with the red ones representing PyV-positive colonies or mammalian species and the grey ones representing the PyV-negative colonies or mammalian species (species names of PyV negative bats are not shown). The composition of collected bats or rodents in each province is presented by the colored rings with total numbers in the rings. The color region in the rings represents each positive bat families or rodent species as indicated in the top right-hand corner. Every filled circle around the colored rings represents one mammalian species. The numbers of PyV positive bats in each species are shown in parentheses. Abbreviations of provinces: XJ, Xinjiang Uyghur Autonomous Region; YN, Yunnan Province; GX, Guangxi Zhuang Autonomous Region; FJ, Fujian Province; ZJ, Zhejiang Province. Abbreviations of bat genera: <em>H</em>., <em>Hipposideros; A</em>., <em>Aselliscus; Rh</em>., <em>Rhinolophus; S</em>., <em>Scotophilus; Mi</em>., <em>Miniopterus; My</em>., <em>Myotis; T</em>., <em>Taphozous; Ro</em>., <em>Rousettus</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">As part of the regional viral surveillance of a bat-borne zoonosis program, the lungs and rectal tissue of 208 bats of 9 bat species from colonies 1-4 (S1 Table) collected in Yunnan province were pooled and subjected to viral metagenomic analysis, as previously described [<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>]. Nucleic acids were amplified by sequence-independent single primer amplification prior to Illumina sequencing. A total of 1.76×10<sup>7</sup> sequence reads with an average length of 125 nucleotides (nt) were obtained, of which 694 PyV-related reads with average length of 167 nt were identified from 38,468 virus-related sequences. Preliminary analyses of these PyV sequences indicated a high level of diversity with previously unrecognized members from both <em>Alpha</em>- and <em>Betapolyomavirus</em> genera.</p><p id="p-12">To determine the prevalence of PyVs in bats collected not only from Yunnan but also from other provinces in China, a broad-spectrum PyV nested PCR targeting the VP1 gene of known members within the <em>Alpha-</em> and <em>Betapolyomavirus</em> genera [<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a>] was performed using genomic DNAs extracted from archived rectal tissues of all 1,083 bats. This yielded 186 (17.2%) PyV positive samples from 15 bat species within 6 families. These PyV positive bats were from 21 of 29 bat colonies in the survey area. As shown in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig 1</a> and <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, all five investigated provinces had PyV positive bats with detection rates of 9.6% (20/208) in Yunnan, 11.9% (59/495) in Guangxi, 10.1% (16/158) in Xinjiang, 42.5% (74/174) in Fujian and 35.4% (17/48) in Zhejiang. With respect to the bat families (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig 1</a> and <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), 103 from 6 species of Rhinolophidae, 60 from 4 species of Vespertilionidae, 7 from 2 species of Hipposideridae, and 11, 2 and 3 respectively from 1 species each of Emballonuridae, Miniopteridae and Pteropodidae were PyV positive. At the species level, <em>Rh. affinis</em>, in the Rhinolophidae, widely distributed in southern China, yielded the highest number of positive bats (n=64), while <em>Myotis horsfieldii</em> in the Vespertilionidae collected in Fujian, showed the highest positivity rate: 66.6% (24/36).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/696040/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/696040/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/696040" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <span class="caption-title">Sampling of bats and positive rate (%) of PyVs in Chinese provinces</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Extensive genetic diversity of polyomaviruses among bats</h3><p id="p-13">VP1 genes of PyV were amplified using broad-spectrum primers from 186 positive samples and then cloned into vectors. Four clones derived from each amplicon were sequenced to detect potential co-infection of multiple PyVs in each positive sample. As a result, 180 of 186 bats were confirmed to harbor one PyV, while the remaining 6 had a co-infection with two distinct PyVs (detailed in S1 Table). A total of 192 non-redundant partial VP1 sequences were obtained and phylogenetically compared with VP1 sequences of known PyVs. Results showed that the 192 VP1 sequences had extensive diversity and could be classified into 44 clusters within the <em>Alpha-</em> and <em>Betapolyomavirus</em> genera (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig 2</a> and S1 Fig), with 46.6-90.1% nt identities between clusters and 95.1-100% within clusters. Of these 44 PyVs clusters, 42 showed exclusive host specificity within a single bat species while, surprisingly, the remaining 2 clusters included viruses detected in two bat species within different families (<em>Rh. affinis</em> and <em>Hipposideros larvatus; Rh. affinis</em> and <em>Myotis horsfieldii</em>, respectively). This finding provided preliminary evidence of inter-family transmission of PyVs (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig 2</a> and S1 Fig). The majority (n= 191) of these partial VP1 shared less than 90% nt identity with the closest known PyVs. Interestingly, one VP1 sequence (marked by cluster 10 in S1 Fig), named here as Pipistrellus pipistrellus polyomavirus XJPp02 (strain PyV10-YXAC25) shared 98% nt identity with Rattus norvegicus polyomavirus 1, previously identified in brown rats in Germany [<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>].</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bayesian phylogenetic tree of partial VP1 nucleotide sequences from PyV positive bats. The tree was constructed with 46 partial VP1 sequences (non-black-colored) of 44 new PyVs in present study, and 93 PyV references (including 38 reference bat PyV species) in Alpha- and Betapolyomavirus genera available in GenBank. The names of putative novel PyV strains are differentiated using colors according to the Chiropteran family of bat hosts following the legend in the top right-hand corner. The PyV positive rate of each putative PyV species from different colonies is shown in red in parentheses next to the tip names. Each parenthesis represents one colony. Host-switching events are marked in the clusters showing the presence of closely related PyVs with different bat host species (HS1-HS4). The posterior probability supporting the branching is shown next to the branch. Scale: 0.2 nucleotide mutations per site." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1269924037" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Bayesian phylogenetic tree of partial VP1 nucleotide sequences from PyV positive bats.&lt;/span&gt; The tree was constructed with 46 partial VP1 sequences (non-black-colored) of 44 new PyVs in present study, and 93 PyV references (including 38 reference bat PyV species) in &lt;em&gt;Alpha-&lt;/em&gt; and &lt;em&gt;Betapolyomavirus&lt;/em&gt; genera available in GenBank. The names of putative novel PyV strains are differentiated using colors according to the Chiropteran family of bat hosts following the legend in the top right-hand corner. The PyV positive rate of each putative PyV species from different colonies is shown in red in parentheses next to the tip names. Each parenthesis represents one colony. Host-switching events are marked in the clusters showing the presence of closely related PyVs with different bat host species (HS1-HS4). The posterior probability supporting the branching is shown next to the branch. Scale: 0.2 nucleotide mutations per site.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F2.medium.gif" width="424" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F2.medium.gif" width="424" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig 2.</span> <span class="caption-title">Bayesian phylogenetic tree of partial VP1 nucleotide sequences from PyV positive bats.</span><p id="p-14" class="first-child">The tree was constructed with 46 partial VP1 sequences (non-black-colored) of 44 new PyVs in present study, and 93 PyV references (including 38 reference bat PyV species) in <em>Alpha-</em> and <em>Betapolyomavirus</em> genera available in GenBank. The names of putative novel PyV strains are differentiated using colors according to the Chiropteran family of bat hosts following the legend in the top right-hand corner. The PyV positive rate of each putative PyV species from different colonies is shown in red in parentheses next to the tip names. Each parenthesis represents one colony. Host-switching events are marked in the clusters showing the presence of closely related PyVs with different bat host species (HS1-HS4). The posterior probability supporting the branching is shown next to the branch. Scale: 0.2 nucleotide mutations per site.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">To further characterize the genetic diversity of the PyVs in these bats, inverse PCR based on the partial VP1 sequences was employed to amplify the remaining genomes. A total of 42 full genomes were obtained from 40 individual bats. As shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, the genome sizes of the novel bat-borne PyV were 4,869–5,529 bp with 39.49–44.69% G+C content. All PyV genomes displayed an archetypal genomic organization, including an early region encoding two nonstructural proteins [large tumor antigen (LTAg), and short tumor antigen (STAg)], and a late region encoding three capsid proteins (VP1, VP2 and VP3) on opposite strands. Early and late regions were separated by a non-coding control region harboring regulatory elements. For all samples from which PyVs genomes were obtained, a ∼1,800 bp fragment of the <em>cytb</em> gene was amplified to corroborate the bat host species (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> and S2 Fig).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/696037/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/696037/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/696037" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2.</span> <span class="caption-title">Novel bat PyV genomes identified in Chinese bat hosts</span><div class="sb-div caption-clear"></div></div></div><p id="p-16">The PyV strains sharing ≥ 97% nt identity within the LTAg coding sequence were defined as the same PyV, while those sharing ≤ 90% identities were defined as different PyV [<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>]. Consequently, as shown in <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a> and <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig 3</a>, the 42 PyV strains were identified as 28 novel bat-borne PyVs based on genomic identities. Of these novel bat-borne PyVs, 26 were identified in a single bat species, indicative of a high degree of host specificity. They included one PyV identified in a cryptic bat species (provisionally named <em>Rhinolophus sp. CN2016</em>) the permanent identification of which awaits formal binomial classification of the host species as the <em>cytb</em> gene showed only 90% nt identity with that of known bat species [<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>]. The PyVs named in the present study were based on the binomial names of their host bat species and the chronological order of discovery, as per ICTV guidelines. For example, Aselliscus stoliczkanus polyomavirus 1 was the first PyV discovered in <em>Aselliscus stoliczkanus</em> bats, and Rhinolophus affinis polyomavirus 2 was the second PyV discovered in <em>Rh. affinis</em> bats. Interestingly, two remaining PyVs were identified in more than one bat species. In detail, a PyV, herein provisionally named as Rhinolophus affinis-Hipposideros larvatus polyomavirus (Ra-Hl PyV), was identified in both <em>Rh. affinis</em> and <em>H. larvatus</em> bats. Similarly, another PyV named as Rhinolophus affinis-Myotis horsfieldii polyomavirus (Ra-Mh PyV) was identified in <em>Rh. affinis</em> and <em>My. horsfieldii</em> bats. For clarity and according to the naming tradition of PyV, the Ra-Hl PyV identified in <em>Rh. affinis</em> was named Rhinolophus affinis polyomavirus 5 (the fifth PyV discovered in <em>Rh. affinis</em>), and the Ra-Hl PyV identified in <em>H. larvatus</em> was named Hipposideros larvatus polyomavirus 1. Similarly, the Ra-Mh PyV identified in <em>Rh. affinis</em> was named Rhinolophus affinis polyomavirus 3, and the Ra-Mh PyV identified in <em>My. horsfieldii</em> named Myotis horsfieldii polyomavirus 3.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pairwise identity comparison of the LTAg genes of bat-borne PyVs identified in the present study. The LTAg coding sequences of 42 bat-borne PyV strains identified in the present study and 15 PyV references (row names) are compared with those of 30 PyVs representing 28 newly proposed PyVs (column names). Cells are colored according to their nt sequence identity in red (&gt; 0.90), orange (0.85-0.90), yellow tones (0.80-0.84) and white (&lt; 0.80). Yellow and light yellow: nt identity 0.80-0.84. Rhinolophus affinis-Hipposideros larvatus polyomavirus (Ra-Hl PyV) are identified by triangles and Rhinolophus affinis-Myotis horsfieldii polyomavirus (Ra-Mh PyV) by circles." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1269924037" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Pairwise identity comparison of the LTAg genes of bat-borne PyVs identified in the present study.&lt;/span&gt; The LTAg coding sequences of 42 bat-borne PyV strains identified in the present study and 15 PyV references (row names) are compared with those of 30 PyVs representing 28 newly proposed PyVs (column names). Cells are colored according to their nt sequence identity in red (&gt; 0.90), orange (0.85-0.90), yellow tones (0.80-0.84) and white (&lt; 0.80). Yellow and light yellow: nt identity 0.80-0.84. Rhinolophus affinis-Hipposideros larvatus polyomavirus (Ra-Hl PyV) are identified by triangles and Rhinolophus affinis-Myotis horsfieldii polyomavirus (Ra-Mh PyV) by circles.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F3.medium.gif" width="408" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig 3." src="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F3.medium.gif" width="408" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig 3.</span> <span class="caption-title">Pairwise identity comparison of the LTAg genes of bat-borne PyVs identified in the present study.</span><p id="p-17" class="first-child">The LTAg coding sequences of 42 bat-borne PyV strains identified in the present study and 15 PyV references (row names) are compared with those of 30 PyVs representing 28 newly proposed PyVs (column names). Cells are colored according to their nt sequence identity in red (&gt; 0.90), orange (0.85-0.90), yellow tones (0.80-0.84) and white (&lt; 0.80). Yellow and light yellow: nt identity 0.80-0.84. Rhinolophus affinis-Hipposideros larvatus polyomavirus (Ra-Hl PyV) are identified by triangles and Rhinolophus affinis-Myotis horsfieldii polyomavirus (Ra-Mh PyV) by circles.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">Among the newly identified single-host PyVs (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig 3</a>), 19 PyVs shared 68-83% nt sequence identities with the closest-known species or with each other in the LTAg coding sequences, therefore were considered to be novel PyV species according to a demarcation standard based on ≥ 15% genetic divergence in the LTAg gene [<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">36</a>]. Seven PyVs shared 86-90% identity with the closest known species or with each other in the LTAg gene, therefore showing insufficient divergence; however, all were identified for the first time in their specific bat hosts. Furthermore, these PyVs were exclusively detected in bat species separated by considerable distances (≥ 564 ± 328 km) (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig 3</a> and S2 Table). They could therefore be considered novel PyV species, a proposal further reinforced by additional correlation provided by the genetic selection analysis described below [<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">36</a>]. Ra-Hl PyV and Ra-Mh PyV, exhibited 66% and 77% nt sequence identities in their LTAg coding sequences with the closest known PyV, could therefore be proposed as novel PyV species. However, the nomenclature for these PyVs requires further discussion as inter-family host-switching is unprecedented for mammalian PyVs [<a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">36</a>].</p></div><div id="sec-6" class="subsection"><h3>Phylogenetic analysis and identification of host-switching events</h3><p id="p-19">In order to verify the host-switching events, an LTAg amino acid tree was constructed to compare the 28 new PyVs with 126 known <em>Alpha</em>- and <em>Betapolyomavirus</em> species (including 38 bat-borne PyV reference species). As shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig 4</a>, 28 new bat PyVs (colored in the figure) clustered within the <em>Alphapolyomavirus</em> (n=19) and <em>Betapolyomavirus</em> (n=9) genera along with 38 known bat PyVs. Within each viral genus, bat PyV sequences grouped into 7 clusters, referred to here as Alpha 1-7 and Beta 1-7, respectively. Among these novel PyVs, two viruses, Ra-Hl PyV in the <em>Alphapolyomavirus</em> and Ra-Mh PyV in the <em>Betapolyomavirus</em> genera, were identified in bat hosts of different families living in the same colony, suggestive of two viral host-switching events, labeled as host-switching event 1 and 2 (HS1 and 2 in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig 4</a>). Furthermore, a third PyV clustered in the phylogenetic tree with PyVs carried by bats of different families (Hipposideridae and Rhinolophidae), indicative of another inter-family host-switching event (HS3 in <a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig 4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bayesian phylogenetic tree of the PyV LTAg protein sequences. The tree was constructed with 156 amino acid sequences of 28 new PyVs identified in the present study, and 126 PyV references (including 38 bat PyV species references) in Alpha- and Betapolyomavirus genera available in GenBank. Names of new PyVs with their strain names in parentheses at the branch tips are colored differently according to their host species within the Chiropteran family, as shown in the top left-hand corner. Three host-switching events are marked as HS1 to HS3 by opposing arrows. Host switching lineages identified by correlation analyses are also shown with black circles. Silhouettes of the mammalian hosts of PyVs are shown alongside each clade. Bayesian posterior probability supporting the branching is shown next to the branches. Abbreviations of PyVs: Ra-Hl PyV, Rhinolophus affinis-Hipposideros larvatus polyomavirus; Ra-Mh PyV, Rhinolophus affinis-Myotis horsfieldii polyomavirus." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1269924037" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Bayesian phylogenetic tree of the PyV LTAg protein sequences.&lt;/span&gt; The tree was constructed with 156 amino acid sequences of 28 new PyVs identified in the present study, and 126 PyV references (including 38 bat PyV species references) in &lt;em&gt;Alpha-&lt;/em&gt; and &lt;em&gt;Betapolyomavirus&lt;/em&gt; genera available in GenBank. Names of new PyVs with their strain names in parentheses at the branch tips are colored differently according to their host species within the Chiropteran family, as shown in the top left-hand corner. Three host-switching events are marked as HS1 to HS3 by opposing arrows. Host switching lineages identified by correlation analyses are also shown with black circles. Silhouettes of the mammalian hosts of PyVs are shown alongside each clade. Bayesian posterior probability supporting the branching is shown next to the branches. Abbreviations of PyVs: Ra-Hl PyV, Rhinolophus affinis-Hipposideros larvatus polyomavirus; Ra-Mh PyV, Rhinolophus affinis-Myotis horsfieldii polyomavirus.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F4.medium.gif" width="440" height="320"/><noscript><img class="highwire-fragment fragment-image" alt="Fig 4." src="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F4.medium.gif" width="440" height="320"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig 4.</span> <span class="caption-title">Bayesian phylogenetic tree of the PyV LTAg protein sequences.</span><p id="p-20" class="first-child">The tree was constructed with 156 amino acid sequences of 28 new PyVs identified in the present study, and 126 PyV references (including 38 bat PyV species references) in <em>Alpha-</em> and <em>Betapolyomavirus</em> genera available in GenBank. Names of new PyVs with their strain names in parentheses at the branch tips are colored differently according to their host species within the Chiropteran family, as shown in the top left-hand corner. Three host-switching events are marked as HS1 to HS3 by opposing arrows. Host switching lineages identified by correlation analyses are also shown with black circles. Silhouettes of the mammalian hosts of PyVs are shown alongside each clade. Bayesian posterior probability supporting the branching is shown next to the branches. Abbreviations of PyVs: Ra-Hl PyV, Rhinolophus affinis-Hipposideros larvatus polyomavirus; Ra-Mh PyV, Rhinolophus affinis-Myotis horsfieldii polyomavirus.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">The first host-switching event HS1 in the Alpha 3 cluster (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig 4</a>) occurred between <em>Rh. affinis</em> (family Rhinolophidae) and <em>H. larvatus</em> (family Hipposideridae). Strikingly, Ra-Hl PyV, characterized in <em>Rh. affinis</em> (Rhinolophus affinis polyomavirus 5) and <em>H. larvatus</em> (Hipposideros larvatus polyomavirus 1), exhibited 100% nt identity between both genomes. Importantly, they were detected in multiple animals and, also, from different colonies. The Rhinolophus affinis polyomavirus 5 was detected in two colonies (colonies 16 and 17 separated 10 km apart in south Guangxi province in southern China) with detection rates of 1/22 and 8/28, respectively. Notably, the Hipposideros larvatus polyomavirus 1 was also detected in colonies 16 and 17 in <em>H. larvatus</em>, also in Guangxi province, with positive rates of 3/19 and 2/10, respectively.</p><p id="p-22">The host-switching event HS2 in the Beta 5 cluster (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig 4</a>) occurred between <em>Rh. affinis</em> (family Rhinolophidae) and <em>My. horsfieldii</em> (family Vespertilionidae), involving Ra-Mh PyV characterized in <em>Rh. affinis</em> (Rhinolophus affinis polyomavirus 3) and in <em>My. horsfieldii</em> (Myotis horsfieldii polyomavirus 3). These PyV strains were also extremely close genetically but not identical, sharing 97-99% nt genomic identity. Ra-Mh PyV was frequently detected in both hosts: Rhinolophus affinis polyomavirus 3 was detected in five colonies of <em>Rh. affinis</em> bats in Yunnan, Guangxi and Fujian provinces (23%; 43/188), in which 7 full genomes shared 97-100% nt identity in their LTAg coding sequences (marked with black circles in <a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig 3</a>). The Myotis horsfieldii polyomavirus 3 had a high positive rate 63.8% (23/36) in <em>My. horsfieldii</em> bats within a single colony (colony 24 <a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig 1</a> and S1 Table) where, notably, <em>Rh. affinis</em> and <em>My. horsfieldii</em> species roosted together. In this cave, <em>Rh. affinis</em> also harbored Rhinolophus affinis polyomavirus 3 with a 57% (12/21) detection rate. Geographically, Rhinolophus affinis polyomavirus 3 was widely distributed in <em>Rh. affinis</em> hosts.</p><p id="p-23">The third host-switching event, HS3, in the Alpha 7 cluster (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Fig 4</a>) occurred between <em>A. stoliczkanus</em> (family Hipposideridae) and <em>Rhinolophus</em> spp. (family Rhinolophidae). The Aselliscus stoliczkanus polyomavirus 1 exhibited 87% and 86% nt identity with Rhinolophus affinis polyomavirus 2 and Rhinolophus pearsonii polyomavirus 2, respectively (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig 3</a>). The two sampling locations of the <em>Rhinolophus</em> species were from caves located 731 km and 86 km respectively from the cave in which the single Aselliscus stoliczkanus polyomavirus 1 positive sample was detected.</p><p id="p-24">Besides these three host-switching events, a PyV-positive rectal tissue sample, named Pipistrellus pipistrellus polyomavirus XJPp02 [strain PyV10-YXAC25], collected from a <em>Pipistrellus pipistrellus</em> (family Vespertilionidae) bat in colony 28 in northwestern Xinjiang exhibited 98% nt identity in a partial VP1 fragment (∼270 nt) with Rattus norvegicus polyomavirus 1 (detected in German rats) [<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">35</a>] in the <em>Alphapolyomavirus</em> genus (marked as host-switching event HS4 in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig 2</a>). A qPCR protocol designed to detect both the previously described rat PyV and the <em>P. pipistrellus</em> PyV was employed to screen samples comprising spleen, kidney, lung, liver, rectal and brain tissues in all <em>P. pipistrellus</em> bats (n=158) collected in three locations in Xinjiang in 2016. Also screened were rectal and kidney tissues from brown rats (n=51) available from collections in two other locations in Xinjiang during 2015-2016 [<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>]. The original PyV10-YXAC25 isolate yielded CT values of 36-38 upon repeated testing, indicative of very low copy numbers of target. An additional lung tissue sample derived from another <em>P. pipistrellus</em> bat from the same colony in which the original PyV10 was detected was also positive, with a higher viral load (CT: 34) by qPCR. No rat samples were positive. Despite repeated attempts to amplify by inverse PCR, no genomes were recovered from the bat samples.</p></div><div id="sec-7" class="subsection"><h3>Correlation of genetic divergences between PyVs and bat hosts</h3><p id="p-25">As shown in <a id="xref-fig-3-6" class="xref-fig" href="#F3">Fig 3</a> and <a id="xref-fig-4-7" class="xref-fig" href="#F4">4</a>, although seven new PyVs showed high LTAg nt identities (86-90%) with each other or the closest known PyVs, the phylogenetic clustering supported codivergence events rather than host-switching events. For instance, Rhinolophus pearsonii polyomavirus 1 and Rhinolophus affinis polyomavirus 1 shared 88% nt identity (Alpha 6); Rhinolophus affinis polyomavirus 1 and Rhinolophus sp. CN 2016 polyomavirus 1 shared 85% identity (Alpha 6); Rhinolophus ferrumequinum polyomavirus 2 shared 89% identity with Rhinolophus simulator polyomavirus 1 (LC269978) previously identified in Zambian bats (Alpha 7); and Rhinolophus ferrumequinum polyomavirus 1 shared 90% identity with Rhinolophus simulator polyomavirus 3 (LC269980) also identified in Zambian bats (Alpha 6). These last two instances showed that PyVs from Chinese and African horseshoe bats were remarkably similar despite the geographical separation.</p><p id="p-26">To investigate further the factors potentially influencing the divergence between PyVs with high nt sequence identities, Mantel’s correlation tests were performed to compare the divergence among PyV genome sequences with those among <em>cytb</em> genes of their corresponding hosts and to control the putative effects of the geographic location on the bat collection. As shown in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>, correlation analysis was performed using three groups of data: 1) a total of 42 PyVs with full genomes from the present study; 2) a total of 48 PyVs including the 42 PyV genomes described here and 6 PyV genomes we previously identified in Zambian horseshoe bats [<a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">26</a>]; 3) a total of 37 PyVs obtained by removing 11 PyVs (marked in <a id="xref-fig-4-8" class="xref-fig" href="#F4">Fig 4</a> which potentially arose by host-switching events) from the 48 PyVs identified in both China and Zambia. Analysis of each of these three groups of data showed that there was negligible correlation between geographical distances and divergence for either hosts or viruses. Moreover, the geographical proximity between the colonies had little effect on the co-evolution of viruses with their respective hosts.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/696039/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/696039/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/696039" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3.</span> <span class="caption-title">Correlation analyses of genetic divergences between PyVs and bat hosts.</span><div class="sb-div caption-clear"></div></div></div><p id="p-27">Analysis using the first group of data showed that a significant correlation existed between host and PyV divergence per genus considering the 42 genomes identified in China. The correlation between bat hosts and PyVs in the <em>Alpha-</em> and <em>Betapolyomavirus</em> genera was 0.47 (<em>P</em> &lt; 2×10<sup>-4</sup>) and 0.65 (<em>P</em> &lt; 1.5×10<sup>-3</sup>), respectively. This first dataset supported the intra-host divergence model of PyV and hosts, with multiple viral lineage duplications, in support of a previous model of PyV evolution [<a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">17</a>]. The addition of PyV sequences sampled from African <em>Rhinolophus</em> spp. [<a id="xref-ref-26-5" class="xref-bibr" href="#ref-26">26</a>] increased the correlation index between host and viral divergence as the number of considered sequences increased. Altogether, these findings support an intra-host divergence model for the radiation of PyV species within the order Chiroptera. In the third group, by removing 8 sequences (also marked as host switching lineages in <a id="xref-fig-4-9" class="xref-fig" href="#F4">Fig 4</a>) in the <em>Alpha-</em> and 3 in the <em>Betapolyomavirus</em> genera, which, notably, included host-switching events (HS1-HS3) from both PyV genera, the correlation indices increased 40% and 30%, respectively.</p></div><div id="sec-8" class="subsection"><h3>Bayesian estimation of tMRCA of bat hosts and PyVs</h3><p id="p-28">To test whether the host-switching events of the PyV species occurred after the divergence of host species, the times to most recent common ancestor (tMRCA) of both bat hosts and the PyVs were estimated employing a Bayesian approach. As shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig 5A</a>, the tMRCA of bat hosts were estimated based on the <em>cytb</em> sequences (∼1800 bp) of bats that harbored the 28 novel PyVs. The results showed that divergence among <em>Rhinolophus</em> species happened 14.2 to 17.3 million years ago (MYA) with a 95% highest posterior density (HPD), which is consistent with previously reported divergence times (16–17 MYA) of the bat hosts based on mitochondrial DNA [<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>] and other sampled bat families [<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>]. The Bayesian results for PyV divergence times (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig 5B</a>) also showed that the tMRCAs for both <em>Alpha</em>- and <em>Betapolyomavirus</em> genera were 115-182 MYA with 95% HPD, overlapping with the divergence times of the tMRCA of placental mammals (92 to 116 MYA) [<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>]. However, when compared with the Bayesian results for the bat hosts, the time scale of PyV divergence was not always in agreement: specifically, 1) the divergence time between <em>Rhinolophus</em> and <em>Hipposideros</em> hosts was estimated with a median 36.3 MYA in the 95% HPD range (34, 38) MYA, while the tMRCAs of Rhinolophus affinis polyomavirus 5 and Hipposideros larvatus polyomavirus 1 were estimated with a median of 0.02 MYA in the 95% HPD range (0.0012, 0.12) MYA; 2) The tMRCAs of Myotis horsefieldii polyomavirus 3 and Rhinolophus affinis polyomavirus 3 had a median of 1.05 MYA in the 95% HPD range (0.4, 1.9) MYA, while the median tMRCA of both corresponding host species was 66.76 MYA in the 95% HPD range (60.9, 73.5) MYA; 3) The median tMRCA of Rhinolophus affinis polyomavirus 2 and Aselliscus stoliczkanus polyomavirus 1 was estimated as 11.61 MYA in the 95% HPD range (6.6, 17.8) MYA, however, the tMRCA between both hosts had a median of 36.3 MYA in the 95% HPD range (34.3, 38.2) MYA. Finally, 4) the estimated time of divergence between Scotophillus kuhlii polyomavirus 2 and Pipistrellus pipistrellus polyomavirus 1, with a median of 37 MYA in the 95% HPD range (23, 54) MYA, was less than the predicted divergence time between the hosts, with a median of 44.42 MYA in the 95% HPD range (40, 49) MYA. In the four examples, the most parsimonious explanation for the lesser median ages to divergence of the PyVs compared to the longer tMRCAs for the bat hosts is that these represent examples of inter-family viral host-switching events.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dating the tMRCA of host bat species employing mitochondrial cytb sequences (A) and PyV species using the LTAg gene (B). Cytb and PyV sequences from this study are colored according to the Chiropteran family of the host species and following the legend in the top right-hand corner. Calibration points of cytb are marked on the branches with black squares (A). Calibration points using PyVs identified from non-human primates and other bat species are shown in the upper section of the phylogeny and marked with black squares (B). The timescale at the bottom of both figures and the tMRCAs shown next to the branches are in millions of years (MYA) before the present. Median age of the tMRCAs of cytb (A) and PyVs (B) are shown next to the branches. In Fig 5B, branches that matched the divergence between host species in Fig 5A are identified with red circles." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1269924037" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Dating the tMRCA of host bat species employing mitochondrial &lt;em&gt;cytb&lt;/em&gt; sequences (A) and PyV species using the LTAg gene (B).&lt;/span&gt; &lt;em&gt;Cytb&lt;/em&gt; and PyV sequences from this study are colored according to the Chiropteran family of the host species and following the legend in the top right-hand corner. Calibration points of &lt;em&gt;cytb&lt;/em&gt; are marked on the branches with black squares (A). Calibration points using PyVs identified from non-human primates and other bat species are shown in the upper section of the phylogeny and marked with black squares (B). The timescale at the bottom of both figures and the tMRCAs shown next to the branches are in millions of years (MYA) before the present. Median age of the tMRCAs of &lt;em&gt;cytb&lt;/em&gt; (A) and PyVs (B) are shown next to the branches. In Fig 5B, branches that matched the divergence between host species in Fig 5A are identified with red circles.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F5.medium.gif" width="347" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig 5." src="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F5.medium.gif" width="347" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig 5.</span> <span class="caption-title">Dating the tMRCA of host bat species employing mitochondrial <em>cytb</em> sequences (A) and PyV species using the LTAg gene (B).</span><p id="p-29" class="first-child"><em>Cytb</em> and PyV sequences from this study are colored according to the Chiropteran family of the host species and following the legend in the top right-hand corner. Calibration points of <em>cytb</em> are marked on the branches with black squares (A). Calibration points using PyVs identified from non-human primates and other bat species are shown in the upper section of the phylogeny and marked with black squares (B). The timescale at the bottom of both figures and the tMRCAs shown next to the branches are in millions of years (MYA) before the present. Median age of the tMRCAs of <em>cytb</em> (A) and PyVs (B) are shown next to the branches. In <a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig 5B</a>, branches that matched the divergence between host species in <a id="xref-fig-5-4" class="xref-fig" href="#F5">Fig 5A</a> are identified with red circles.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Intra-lineage host duplication and host-switching events during bat PyV evolution</h3><p id="p-30">The findings of the Mantel test and other studies [<a id="xref-ref-17-6" class="xref-bibr" href="#ref-17">17</a>] supporting intra-host divergence for PyVs, required reconciliation with the compelling evidence of similar viral species infecting evolutionarily distant bat species. Co-phylogenetic reconstruction analyses using JANE 4 were therefore performed separately for both viral genera. The best-supported solution to explain the divergence in the genus <em>Alphapolyomavirus</em> for the novel Chinese PyVs and African samples was modeled with 11 co-speciations, 4 lineage duplications, 13 lineage losses and 9 host-switching events (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig 6A</a>). Among the 9 inferred host-switching events, in agreement with the tMRCA estimations, it was implied that Rhinolophus affinis polyomavirus 5 originated in the lineage which gave rise to Hipposideros larvatus polyomavirus 1, Rhinolophus sinicus polyomavirus 1 and 2, and Scotophilus kuhli polyomavirus 2 indicating that Rh-Hl PyV was originally a PyV harbored by <em>Rh. affinis</em>, which was subsequently transmitted to <em>H. larvatus</em>. The best-supported solution also suggested that the lineage of Rhinolophus affinis polyomavirus 2 had given rise to Aselliscus stoliczkanus polyomavirus 1. Other host-switching events inferred earlier in the divergence among PyVs were more difficult to corroborate with the available tMRCA data. Divergence in the genus <em>Betapolyomavirus</em> involved 6 co-speciations, 2 lineage duplications, 7 lineage losses and 7 host-switching events (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig 6B</a>). Similar to the <em>Alphapolyomavirus</em> genus, the solution for the <em>Betapolyomavirus</em> genus suggested, among the 7 host-switching events, that the origin of Myotis horsfieldii polyomavirus 3 was as a descendant of the lineage giving rise to Rhinolophus affinis polyomavirus 3 (i.e. that Rh-Mh PyV was originally a PyV harbored by <em>Rh. affinis</em> and transmitted to <em>My. horsfieldii</em>), in agreement with the tMRCA results. In both solutions, although the presence of duplication events could explain the existence of more than one different viral species in the same host species, in multiple instances the dissimilarity between viral species from the same host species was more easily explained by host-switching events following the divergence between host species.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Co-phylogenetic models for bat PyVs in Alpha- and Betapolyomavirus genera with their hosts. The blue phylogenies show the inferred host divergence. The red phylogenies are annotated to reconstruct the divergence among viral species according to JANE 4 in the Alphapolyomavirus (A) and Betapolyomavirus (B) genera. Symbols for the annotations are shown in the solution key legend and percentage support for the events are shown next to their annotations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1269924037" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Co-phylogenetic models for bat PyVs in &lt;em&gt;Alpha-&lt;/em&gt; and &lt;em&gt;Betapolyomavirus&lt;/em&gt; genera with their hosts.&lt;/span&gt; The blue phylogenies show the inferred host divergence. The red phylogenies are annotated to reconstruct the divergence among viral species according to JANE 4 in the &lt;em&gt;Alphapolyomavirus&lt;/em&gt; (A) and &lt;em&gt;Betapolyomavirus&lt;/em&gt; (B) genera. Symbols for the annotations are shown in the solution key legend and percentage support for the events are shown next to their annotations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F6.medium.gif" width="440" height="414"/><noscript><img class="highwire-fragment fragment-image" alt="Fig 6." src="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F6.medium.gif" width="440" height="414"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig 6.</span> <span class="caption-title">Co-phylogenetic models for bat PyVs in <em>Alpha-</em> and <em>Betapolyomavirus</em> genera with their hosts.</span><p id="p-31" class="first-child">The blue phylogenies show the inferred host divergence. The red phylogenies are annotated to reconstruct the divergence among viral species according to JANE 4 in the <em>Alphapolyomavirus</em> (A) and <em>Betapolyomavirus</em> (B) genera. Symbols for the annotations are shown in the solution key legend and percentage support for the events are shown next to their annotations.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Positive selection in divergence of Chinese and African <em>Rhinolophus</em> PyVs are focused on the external surface of the VP1</h3><p id="p-32">Given the diversity of viral species identified in the genus <em>Rhinolophus</em>, viral proteins under positive selection during the evolution of the different PyV lineages were of interest as possible speciation determinants. We performed a Bayesian analysis to calculate the ω rate (dN/dS) to identify genomic regions of the <em>Alpha</em>- and <em>Betapolyomavirus</em> genera in <em>Rhinolophus</em> PyV clusters, which were likely subject to positive selection (S3 Fig). As expected, the majority of the horseshoe bat PyV genomes were found to be under neutral or negative selection (ω values ≤ 1); however, a number of residues in VP2, VP1, LTAg and STAg proteins did show evidence of positive selection (ω values &gt; 1) in both viral genera (shown in red in S3 Fig) and amino acid residues relative to human BK virus (species of <em>Human polyomavirus 1</em>) and simian virus 40 (species of <em>Macaca mulatta polyomavirus 1</em>) are detailed in S5 Table. We focused on the major capsid protein VP1 as this determines antigenicity and the engagement of sialylated glycans present on host glycoproteins and glycolipids during cellular attachment and entry in mammalian (murine, human and monkey) viruses [<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>-<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>].</p><p id="p-33">In the <em>Rhinolophus</em> alphapolyomavirus cluster, 11 residues were found under positive selection, located in four regions of VP1 (S5 Table): in the BC-loop (D77, N79 and V82), the DE-loop (L135, A139 and T143), the HI loop (N274, N275 and S279) and the C-terminal residues I367 and N369. In the <em>Rhinolophus</em> betapolyomavirus cluster, 9 residues had ω values &gt; 1 at the N-terminus (K17), in the BC loop (A75, A77, T80, V82 and P84), the DE loop (N134 and Q149) and the EF loop (S178). No positively selected amino acids were identified in the interior of the major capsid protein.</p></div></div><div class="section" id="sec-11"><h2 class="">Discussion</h2><p id="p-34">While mammalian PyVs are highly host specific [<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">12</a>] there are data suggesting cross-species jumps of PyVs between closely related mammalian hosts. Previously, short-range intra-genus host-switching events of PyVs were identified in African horseshoe bats, occurring between <em>Rh. simulator</em> and <em>Rh. blasii</em> in a single Zambian cave [<a id="xref-ref-26-6" class="xref-bibr" href="#ref-26">26</a>]. Strikingly, the genomes of PyVs characterized in respective bat species were 99.9% identical and contained only four virus species-specific polymorphisms. Importantly, these polymorphisms were found to be present in both spleen and kidney samples of each bat host species, indicating a systemic and productive infection. The genetic stability and restricted host-specificity of mammalian PyVs had previously suggested a model of strict codivergence for PyV evolution [<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>]. However, this is inconsistent with family-wide patterns of diversity, where an important contribution of intra-host lineage duplication, recombination and, possibly, host-switching events is evident. A model of intra-host divergence, with these additional evolutionary processes, occurring over the ancient timescales for association of PyVs with multicellular life, has been proposed to explain the extant diversity within the family <em>Polyomaviridae</em> [<a id="xref-ref-17-7" class="xref-bibr" href="#ref-17">17</a>]. The findings in the present study extend these observations to PyVs in Chinese horseshoe bats and other bat hosts (see model in <a id="xref-fig-7-1" class="xref-fig" href="#F7">Fig 7</a>) and provide the first definitive evidence of inter-family host-switching of PyVs in mammals occurring among the bat families Rhinolophidae, Hipposideridae and Vespertilionidae.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Model of PyV evolution in mammals by intra-host divergence with infrequent host-switching events. The panel illustrates the models for: A) intra-host divergence and B) intra-host divergence with infrequent host-switching events. In both panels, the bifurcating fork in blue represents the divergence from a common ancestor of two mammalian host species, with the overlapping co-divergence of viral species in color. Divergence events are marked A-D representing: (A) viral duplication before the divergence of host species, (B) co-divergence with the host species, (C) viral duplication in the same host species and (D) duplication and host-switching to a diverged (susceptible) host species. The phylogenies close to the cartoons are the expected phylogenetic relationship for each respective case with the events marked at the nodes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1269924037" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Model of PyV evolution in mammals by intra-host divergence with infrequent host-switching events.&lt;/span&gt; The panel illustrates the models for: A) intra-host divergence and B) intra-host divergence with infrequent host-switching events. In both panels, the bifurcating fork in blue represents the divergence from a common ancestor of two mammalian host species, with the overlapping co-divergence of viral species in color. Divergence events are marked A-D representing: (A) viral duplication before the divergence of host species, (B) co-divergence with the host species, (C) viral duplication in the same host species and (D) duplication and host-switching to a diverged (susceptible) host species. The phylogenies close to the cartoons are the expected phylogenetic relationship for each respective case with the events marked at the nodes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F7.medium.gif" width="440" height="224"/><noscript><img class="highwire-fragment fragment-image" alt="Fig 7." src="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F7.medium.gif" width="440" height="224"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 7." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig 7.</span> <span class="caption-title">Model of PyV evolution in mammals by intra-host divergence with infrequent host-switching events.</span><p id="p-35" class="first-child">The panel illustrates the models for: A) intra-host divergence and B) intra-host divergence with infrequent host-switching events. In both panels, the bifurcating fork in blue represents the divergence from a common ancestor of two mammalian host species, with the overlapping co-divergence of viral species in color. Divergence events are marked A-D representing: (A) viral duplication before the divergence of host species, (B) co-divergence with the host species, (C) viral duplication in the same host species and (D) duplication and host-switching to a diverged (susceptible) host species. The phylogenies close to the cartoons are the expected phylogenetic relationship for each respective case with the events marked at the nodes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">The host-switching events of PyVs identified in the present study indicate that transmission of PyVs between different bat species has been an ongoing process during the evolutionary history of PyVs. HS1 and HS2 occurred between bats of different families and strains of PyVs sharing 100% or 97-100% nt identity were characterized in their respective hosts. The tMRCA analysis estimated that HS1 and HS2 occurred between 0.02-1.05 MYA. Considering the “slow” evolutionary rate of PyVs (8 × 10<sup>-9</sup> substitutions per site per year as previously estimated [<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>]), the transmission events of Rh-Hl PyV and Rh-Mh PyV in bats happened recently, with a small number of mutations accumulating in the viral genomes during viral adaptation to the permissive hosts. Compared with HS1 and HS2, HS3 was estimated to have occurred much earlier, between 6.6-17.8 MYA, with the Aselliscus stoliczkanus polyomavirus 1 showing 87% nt identity with the closest PyV (Rhinolophus affinis polyomavirus 2), indicating that the former had been transmitted from Rhinolophus bats to A. stoliczkanus bats for a long period of time, had co-evolved with A. stoliczkanus and, finally, had formed a novel PyV species. In addition to these observed host switching events, the tMRCA estimation and correlation analysis showed strikingly that numerous PyVs had also been derived from host switching, thereby shaping the evolutionary history of PyVs and providing evidence that inter-species transmission of PyVs, at least in bats, is not as rare as previously thought.</p><p id="p-37">An issue raised by the present study is how to define and name a PyV species when inter-family host-switching events of PyV are identified. The ICTV criteria for the formal designation of PyV species in mammals must include the binomial name of the host species [<a id="xref-ref-36-5" class="xref-bibr" href="#ref-36">36</a>]; nevertheless, the inter-family host-switching events of PyVs discovered in the present study as well as the inter-species host-switching event also identified in <em>Rhinolophus</em> hosts in a previous study [<a id="xref-ref-26-7" class="xref-bibr" href="#ref-26">26</a>] which has challenged this nomenclature system for bat PyVs. Furthermore, numerous host-switching lineages have also been characterized here by correlation analyses and dating estimation. All these findings suggest that host-switching of PyVs has occurred not only in different bat species but also at different time points in different geographical locations. The ICTV nomenclature criteria should therefore be updated to include recognition of PyVs derived from two different mammalian host species.</p><p id="p-38">Inter-species transmission has already been observed in bird PyVs such as budgerigar fledgling disease PyV (BFDV), which can infect multiple avian species with pronounced pathogenicity [<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>]. Nevertheless, the ICTV nomenclature has utilized only the order level of the host species in the naming scheme: for instance, species belonging to BFDV have been classified as <em>Aves polyomavirus 1</em>. This naming scheme could also be expanded using the more descriptive host species, as with the mammalian PyV species. So far, PyVs with host-switching events have only involved two host species, illustrating the limited ability of PyVs for transmission between different mammalian hosts. Therefore, the improved naming scheme of novel PyVs with host-switching events proposed here would include the binomial names of both host species; e.g., Rhinolophus affinis-Hipposideros larvatus polyomavirus (Rh-Hl PyV), identified in the present study, would become the prototype of a new species named <em>Rhinolophus affinis-Hipposideros larvatus polyomavirus 1</em>.</p><p id="p-39">Another ICTV criterion is the allowable genetic distance of the LTAg coding sequence (≥ 15%) to demarcate PyV species. This cut-off value is not strict, as exemplified by two PyVs identified in two different species of squirrel monkeys which shared 89% sequence identity but were recognized as two separated PyV species [<a id="xref-ref-36-6" class="xref-bibr" href="#ref-36">36</a>]. In the present study, we have discovered numerous novel PyVs carried by diverse bat species, all exhibiting 86-90% sequence identities with their closest known species or with each other. These should be considered new PyV species since all were detected exclusively in distinct bat species in different areas and exhibiting strict host specificity. Examples include the PyVs identified in different <em>Rhinolophus</em> bat species from China and Africa sharing 89-90% nt identify. The correlation and genetic selection analysis reported here has shown that the high sequence identity of different PyVs carried by different host species was due to the co-divergence with strong negative selection of PyVs in respective hosts, indicating that the ≥ 15% cut-off in LTAg gene identity for species demarcation is perhaps no longer adequate and should be updated to ≥10%. With regard to the relationship between Chinese and African bat PyVs referred to above, it is of interest to note that hepadnaviruses (pararetroviruses with DNA genomes) identified in Chinese <em>Rhinolophus</em> bats in another study [<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">49</a>] were also found to exhibit the closest genetic relation (79.6–79.7% nt identify) with viruses carried by African <em>Rhinolophus</em> bats, [<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>] thereby suggesting that co-divergence of DNA viruses within their <em>Rhinolophus</em> bat hosts is relatively common.</p><p id="p-40">In the present study we have identified 28 novel PyVs from 15 bat species in 6 families [<a id="xref-ref-36-7" class="xref-bibr" href="#ref-36">36</a>], a finding that has considerably increased our knowledge of both the genomic diversity and the evolution of PyVs. Additionally, since there is a dearth of commercially available, standardized bat cell lines for isolation and identification of bat-borne viruses there are now added options to establish immortalized bat cell lines from many bat species by using LTAg genes cloned from divergent PyVs identified in specific Chinese bat species. At present, the Tb1 Lu cell line derived from the free-tailed bat (<em>Tardarida brasiliensis</em>, family Molossidae) is currently the only bat cell line available in the American Type Culture Collection (ATCC Nr. CCL-88) [<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>]. Recently, however, Banerjee <em>et al</em>. [<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>] have reported immortalization of primary bat kidney cell lines from the big brown bat (<em>Eptesicus fuscus</em>, family Vespertilionidae) which were stably transformed by a plasmid harboring a little brown bat (<em>Myotis lucifugus</em>, family Vespertilionidae) PyV LTAg. The ability to establish cell lines from different tissues of diverse bat species and to make these available to researchers is a prerequisite for replication studies, virus-host interactions and new virus discoveries.</p><p id="p-41">Papillomaviruses (PVs), which are closely related to PyVs and share similar properties such as, high genetic stability and strict host specificity, have been shown to cause transient and potentially cross-species infections. García-Pérez <em>et al</em>. [<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>] reported that PVs do not follow strict host restrictions in free-ranging Iberian bats (<em>Eptesicus serotinus</em>, family Vespertilionidae), as they found evidence of host-switching events between two different bat species (<em>E. serotinus</em> and <em>E. isabellinus</em>). They therefore recommended a revised paradigm of strict host specificity in PVs [<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">53</a>].</p><p id="p-42">The findings presented here have possible implications for the study of zoonotic transmissions involving highly virulent viral pathogens in bats. For example, horseshoe bats within the family Rhinolophidae have been found to harbor genetically diverse severe acute respiratory syndrome (SARS)-like corona viruses, which acted as major reservoirs for spillover of SARS coronavirus to humans [<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>]. The Middle Eastern respiratory syndrome (MERS) coronavirus has similarly been associated with severe disease and lethal respiratory infections, mostly in Saudi Arabia. Likely transmitted via dromedary camels, it has been shown to be closely related to several bat coronaviruses, including those sequenced from species within the family Vespertilionidae (<em>Neoromicia capensis, Pipistrellus abramus and Vespertilios uperans</em>) [<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>]. Recently, swine acute diarrhea syndrome (SADS) coronavirus has been associated with high morbidity and mortality in pig populations in Guangdong province, China in close proximity to the origin of the SARS pandemic, and striking similarities between the SADS and SARS outbreaks in geographical, temporal, ecological and etiological settings exist [<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>]. We consider noteworthy that in the present study we identified host-switching events of the normally highly host-restricted PyVs within the bat families Rhinolophidae and Vespertilionidae. Whether this reflects a greater propensity for host-switching in these bat families when compared with others and what mechanisms are involved require further functional studies. Conceivably, given the high prevalence we and others have detected of bat PyVs, co-infection of bat species within the families Rhinolophidae and Vespertilionidae may affect the prevalence and/or transmission of CoVs and other high virulent pathogens in bats. This suggestion warrants further study.</p><p id="p-43">Our findings that inter-family transmission of PyVs can occur in mammals may also have implications for studies of viral oncogenesis proposed previously for the <em>Polyomaviridae</em>. For example, three bovine PyVs (BoPyVs) have been identified in cattle [<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>] with more likely given the 13 so far identified as infectious agents in humans [<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>]. Intriguingly, BoPyV2, occupies the same clade as Merkel cell PyV which is causally implicated in oncogenic transformation in a neuroendocrine carcinoma of the elderly. BoPyV3, is also related to human polyomavirus 6 which has been reported in non-melanoma skin cancers, specifically keratocanthomas [<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>]. Colorectal cancer has been suggested to have an infectious disease etiology [<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>] and the relatively high thermoresistance of PyVs may preclude inactivation by lowered cooking temperatures [<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>]. Similarly, in field studies, sequences of BK virus were identified by metagenomics in the feces of non-human primates [<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a>]. Our evidence that inter-family transmission of PyVs can occur in bats provides a precedent for studies of longer-range transmission of PyVs in mammals.</p><p id="p-44">In addition to three bat inter-family host-switching events of PyVs, we also identified a partial VP1 fragment in rectal tissue of a <em>P. pipistrellus</em> bat collected in northwestern Xinjiang that showed 98% nt identity with the VP1 of a previously described Rattus norvegicus polyomavirus 1 identified in Germany [<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">35</a>]. Further qPCR assay also detected the virus in lung tissue of another <em>P. pipistrellus</em> bat but not from any brown rat (n=51) collected in the same Xinjiang area. This preliminary observation provides an impetus to investigate potential PyV host-switching between mammal species beyond families.</p><p id="p-45">The host-switching events identified in this study were strongly associated with the ecological environment of the hosts, with host-switching only identified between different bat species in one to two colonies. Our study suggests that PyVs can transmit between different mammalian hosts between two different animal families by frequent contact of hosts within specific environments, such as the densely packed bat populations that occur in large hibernacula in caves. The host-switching PyVs were identified from rectal tissues of multiple individuals, indicating that shedding via feces is a key factor for viral transmission by the fecal-oral route between different host species.</p><p id="p-46">PyV attachment to target cells is mediated by cell surface receptors. MHC class I acts as a co-receptor and SV40 employs the ganglioside GM1 as receptor [<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a>]. The finding that gangliosides are also primary receptors for human PyVs via VP1 [<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a>] suggests that this may be a general property for primate PyVs although whether this is applicable to other mammalian PyVs, including bats, is unknown. In the present study, amino acid residues in VP1 were found to be under positive selection in <em>Rhinolophus</em> PyVs in the BC, DE and HI loops (<em>Alphapolyomavirus</em>) and in the BC, DE and EF loops (<em>Betapolyomavirus</em>). Interestingly, both <em>Rhinolophus</em> PyV genera showed evidence of positive selection in the external surface BC and DE loops. In BK virus, serological subtypes are assigned based on mutations present within the BC loop [<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a>]. It has also been previously demonstrated that sialylated glycans can act as primary receptors for PyVs [<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a>] and binding to sialic acid moieties was attributed to amino acids residues within the BC, DE and HI loops, suggesting that these VP1 domains play a key role in virus attachment and cellular entry events [<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>]. The surface loops of VP1, i.e. the BC, DE, HI and EF loops, are responsible for viral antigenicity and escape mutants cluster in these regions, suggesting they represent conformational epitopes recognized by host neutralizing antibodies [<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a>, <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">66</a>]. VP1 surface loops form a unique virus-host interaction conformation that determines the host range, cellular tropism and pathogenicity which may explain the enrichment of positive selected residues within these regions [<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">67</a>]. The availability of annotated bat genomes in the future, as proposed by the genome consortium Bat1K [<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">68</a>], and annotation of the genes encoding putative entry receptors (such as, for example, the cell surface ganglioside and other sialylated glycan receptors) from within the bat families Rhinolophidae, Hipposideridae and Vespertilionidae from species previously found to harbor identical or closely related viruses would facilitate the functional characterization of the cellular receptors/attachment factors and of which amino acid residues in VP1 are required for binding and entry. Availability of annotated bat genomes would also help determine whether certain families of bats are more susceptible than others to virus-host switching events.</p></div><div class="section" id="sec-12"><h2 class="">Materials and methods</h2><div id="sec-13" class="subsection"><h3>Ethics statement</h3><p id="p-47">The procedures for sampling bats and brown rats in this study were reviewed and approved by the Administrative Committee on Animal Welfare of the Institute of Military Veterinary Medicine (Laboratory Animal Care and Use Committee Authorization permits JSY-DW-2010-02 and JSY-DW-2016-02). All live animals were maintained and handled according to the Principles and Guidelines for Laboratory Animal Medicine (2006), Ministry of Science and Technology, China.</p></div><div id="sec-14" class="subsection"><h3>Sample information</h3><p id="p-48">The tissues of 1,083 bats in the present study were archived and sub-packed samples were stored at −80°C following collection between 2015-2016 from 29 colonies distributed across the provinces of Yunnan, Fujian and Zhejiang, Guangxi and Xinjiang in China. Archived samples of rectal and kidney tissues from brown rats (n=51) were collected in Xinjiang between 2015-2016 [<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">38</a>]. Detailed information of sampled bats and rats is shown in S1 Table.</p></div><div id="sec-15" class="subsection"><h3>Polyomavirus screening and complete genome sequencing</h3><p id="p-49">Approximately 50 mg samples of rectal and lung tissues from the 208 bats in colonies 1-4 collected in Yunnan province were pooled and subjected to viral metagenomic analysis, as per our previously published method [<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a>]. Due to the complexity of the PyV-related reads detected by the viral metagenomic analyses, the PyV-specific nested PCR protocol [<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">69</a>] based on the VP1 gene developed by Johne <em>et al</em>. in 2005 [<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">27</a>] was employed to screen all metagenomic samples, and partial VP1 amplicons were cloned and bidirectionally sequenced by standard methods. In addition, the PCR was expanded to screen genomic DNA extracts from rectal tissues of bats collected from other Chinese provinces.</p><p id="p-50">Genomic DNA from each bat tissue sample was extracted manually using the TIANamp Genomic DNA Kit (Tiangen), according to the manufacturer’s protocol. PCR screening employed the PCR master mix (Tiangen) with the following thermocycling conditions: 45 cycles for both outer and inner PCRs with denaturation at 94°C for 30 s, annealing at 46°C (outer PCR) or 56°C (inner PCR) for 30 s or 1 min, and extension at 72°C for 1 min, with ddH2O as a negative control. Positive PCR amplicons were purified by gel extraction (Axygen) and sequenced directly on a 3730xl DNA Analyzer (Genewiz) to confirm homology to PyVs and then ligated into vector pMD-18T (TaKaRa) and used to transform <em>E. coli</em> DH5α competent cells (Tiangen). Four clones from each VP1 amplicon were picked for Sanger sequencing. GenBank accession numbers of 192 partial PyV VP1 sequences are showed in S3 Table.</p><p id="p-51">Oligonucleotide primers for inverse PCRs for amplification of the 5 kb PyV genomes were design based on the partial VP1 sequences. Multiple pairs of hemi-nested degenerate primers were subsequently designed for VP1 clusters which showed close phylogenetic relationships (see S6 Table). All ∼5 kb products were amplified from bat genomic DNA by inverse PCR employing the PrimeSTAR MAX DNA polymerase (Takara), purified by gel extraction (Axygen), cloned into pCR-Blunt II-TOPO vector (Invitrogen) and bidirectionally sequenced by primer walking to Phred quality scores &gt; 30 by 3730xl DNA Analyzer (Genewiz). After removal of vector sequences, the full genomes of each PyV were assembled by Geneious software using the partial VP1 and 5 kb products.</p><p id="p-52">A real-time PCR (qPCR) method (S6 Table) was established based on a 59 nt fragment of the VP1 genes of Pipistrellus pipistrellus polyomavirus XJPp02 and Rattus norvegicus polyomavirus 1 was designed to screen spleen, kidney, lung, liver, rectal and brain tissues in all the <em>P. pipistrellus</em> bats (n=158) collected in three locations in Xinjiang in 2016 and also rectal and kidney tissues from brown rats (n=51) available from collections in two other locations in Xinjiang during 2015-2016 (S1 Table). The extracted DNAs from the tissues were amplified using the Probe qPCR kit (TaKaRa) as per the manufacturer’s protocol on a Stratagene Mx3000P.</p></div><div id="sec-16" class="subsection"><h3>Phylogenetic analysis</h3><p id="p-53">The pairwise identity of the LTAg coding sequences was calculated with Sequence Demarcation Tool (SDT) v1.2 [<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">70</a>]. The nucleotide sequence of host <em>cytb</em>, the sequences of PyV genes and proteins were aligned using MAFFT under the algorithm FFT-NS-i [<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">71</a>]. Phylogenetic trees were inferred using MrBayes v3.2, with 10<sup>6</sup> generations chains to ensure &lt; 0.01 standard deviations between the split frequencies and 25% of generations discarded as burn-in [<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">72</a>], as described previously [<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">73</a>]. For the LTAg multiple-sequence alignment, a mixed substitution model was used to explore the best substitution model when inferring the topology. On the other hand, the best fitting nucleotide substitution models for host <em>cytb</em> and the PyV LTAg coding sequence alignments were chosen with MEGA7 [<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">74</a>] by choosing the lowest Bayesian information criterion (BIC).</p><p id="p-54">The tMRCA for hosts and viral samples were estimated with Bayesian analyses conducted as 4 independent MCMC chains of 10 million generations each, sampled every 1,000 generations (BEAST v2.4.6 [<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">75</a>]). The <em>cytb</em> and LTAg gene multiple sequence alignments were used to infer the divergence times among hosts and PyVs, respectively. The general time-reversible substitution model with gamma-distributed rate variation across sites and a proportion of invariable sites (GTR+G+I) were assumed for both models. Other settings were: uncorrelated lognormal relaxed molecular clocks for host and PyV models with a Bayesian skyline population and exponential population models, respectively. These models chosen had the highest likelihood when compared to other models with the PathSampler application in BEAST v2.4.6 package. The host model was calibrated with divergence times between genera as inferred by Agnarsson [<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">40</a>] and the PyV model was calibrated following Buck <em>et al</em>. [<a id="xref-ref-17-8" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">76</a>].</p></div><div id="sec-17" class="subsection"><h3>Correlation analysis</h3><p id="p-55">The divergence correlation between the hosts and PyVs in the <em>Alpha-</em> and <em>Betapolyomavirus</em> genera was assessed by applying Mantel’s correlation tests to the pairwise evolutionary distance matrices for hosts and viral samples in both genera with 10<sup>4</sup> repetitions. Additionally, the matrices were also analysed against the geographical distance between sampling points, and partial correlations between host and polyomavirus distance matrices were estimated controlling for the correlation with the geographical distance. The correlation analysis was performed considering three sets of data per genus: 1) considering only samples identified in China, 2) considering samples identified in China and samples identified in Africa [<a id="xref-ref-26-8" class="xref-bibr" href="#ref-26">26</a>], and 3) removing samples from 2) such as the correlation index increased. Removed samples in 3) were chosen by estimating the correlation coefficient without each sample and then removing samples whose absence maximized the correlation between hosts and PyVs.</p></div><div id="sec-18" class="subsection"><h3>Co-phylogenetic analysis</h3><p id="p-56">Mapping the divergence of PyV species to the divergence among hosts was performed with JANE 4 [<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">77</a>]. The solutions for <em>Alpha-</em> and <em>Betapolyomavirus</em> genera were generated with costs for co-speciation, duplication, loss and host switch events as 0, 1, 1 and 3, respectively. The input was the phylogenetic tree for the host species inferred with mitochondrial <em>cytb</em> sequences, the phylogenetic trees for the <em>Alpha-</em> and <em>Betapolyomavirus</em> genera inferred with amino acid sequences of the LTAg protein and the table indicating the association between hosts and viruses.</p></div><div id="sec-19" class="subsection"><h3>Analysis of natural selection on the evolution of proteins in <em>Rhinolophus</em> PyV species</h3><p id="p-57">The ω ratio (dN/dS) was estimated for each coding region in clades of <em>Rhinolophus</em> PyV species in <em>Alpha-</em> and <em>Betapolyomavirus</em> genera with MrBayes v3.2.6 [<a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">72</a>] with NY98 as the codon model for 5×10<sup>4</sup> generations sampling every 250 and assuring the effective sample size was &gt; 100 for estimated parameters. Tridimensional structure of the VP1 protein was modeled by homology with SWISS-MODEL [<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">78</a>].</p></div></div><div class="section" id="sec-20"><h2 class="">Supporting information</h2><p id="p-58"><strong>S1 Fig</strong>. <strong>Maximum-likelihood phylogenetic tree of partial VP1 nucleotide sequences from PyV positive samples</strong>. The tree was constructed with 192 partial VP1 sequences of PyVs in present study (filled circles), and 74 PyV references (including 38 bat PyV species references) in <em>Alpha-</em> and <em>Betapolyomavirus</em> genera downloaded from GenBank. These 192 partial VP1 sequences were classified into 44 clusters representing host-switching events (green circles; clusters 7, 10, 17 and 31; red circles: no host switching). GenBank accession numbers of 192 partial PyV VP1 sequences in this tree are listed in S3 Table. The tree was generated in MEGA7 by the maximum-likelihood method based on the TN93 model and evaluated with 250 bootstrap replicates.</p><p id="p-59"><strong>S2 Fig</strong>. <strong>Bayesian phylogenetic tree of the <em>cytb</em> sequences of bats</strong>. The tree was constructed with 37 reference bat <em>cytb</em> genes and 42 bat <em>cytb</em> genes in the present study (colored differently according to the bat families) for confirmation of the species of bats from which PyV full genomes were obtained. The posterior probability supporting the branching is shown next to the branch. GenBank accession numbers of <em>cytb</em> sequences in this tree are listed in S4 Table.</p><p id="p-60"><strong>S3 Fig</strong>. <strong>Analysis of positive selection in PyVs infecting the horseshoe bat <em>Rhinolophus</em> in <em>Alpha-</em> and <em>Betapolyomavirus</em> genera</strong>. The top panels show the ω rate (dN/dS) estimated with a Bayesian approach for clusters of <em>Rhinolophus</em> PyVs in both <em>Alpha-</em> and <em>Betapolyomavirus</em> genera with positions adjusted to Rhinolophus affinis polyomavirus 1 [PyVB-FN04C] and Rhinolophus affinis polyomavirus 3 [PyV19-SWAC9], respectively. Tridimensional models for the VP1 proteins corresponding to both genera were inferred by structure homology modelling. The top and bottom views of both modelled proteins are shown in gray with sites under positive selection colored in red.</p><p id="p-61"><strong>S1 Table</strong>. <strong>Summary of sampled colonies and detailed information of PyV positive bats following a geographic approach of the collection sites</strong>.</p><p id="p-62"><strong>S2 Table</strong>. <strong>Summary of detected PyVs and their positive rates in colonies</strong>.</p><p id="p-63"><strong>S3 Table</strong>. <strong>GenBank accession numbers of partial PyV VP1 sequences</strong>.</p><p id="p-64"><strong>S4 Table</strong>. <strong>GenBank accession numbers of 1</strong>.<strong>8 kb <em>cytb</em> sequences</strong>.</p><p id="p-65"><strong>S5 Table</strong>. <strong>VP1 amino acid residues under positive selection in Rhinolophus affinis polyomavirus 1 in the <em>Alphapolyomavirus</em> genus and Rhinolophus affinis polyomavirus 3 in the <em>Betapolyomavirus</em> genus with homologous positions relative to human BK virus (species of <em>Human polyomavirus 1</em>) and Simian virus 40 (species of <em>Macaca mulatta polyomavirus 1</em>)</strong>.</p><p id="p-66"><strong>S6 Table</strong>. <strong>List of oligonucleotides used in the present study</strong>.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-627158v1.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">DE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reeder</span>  <span class="cit-name-given-names">DM</span></span>. <span class="cit-source">Mammal species of the world: a taxonomic and geographic reference, 3rd ed, Order Chiroptera</span>. <span class="cit-publ-name">Baltimore, MD: Johns Hopkins University Press</span>; <span class="cit-pub-date">2005</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.2" data-doi="10.1126/science.1188836"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Streicker</span>  <span class="cit-name-given-names">DG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Turmelle</span>  <span class="cit-name-given-names">AS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vonhof</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuzmin</span>  <span class="cit-name-given-names">IV</span></span>, <span class="cit-auth"><span class="cit-name-surname">McCracken</span>  <span class="cit-name-given-names">GF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rupprecht</span>  <span class="cit-name-given-names">CE</span></span>. <span class="cit-article-title">Host phylogeny constrains cross-species emergence and establishment of rabies virus in bats</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-pub-date">2010</span>;<span class="cit-vol">329</span>(<span class="cit-issue">5992</span>):<span class="cit-fpage">676</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1188836</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DStreicker%26rft.auinit1%253DD.%2BG.%26rft.volume%253D329%26rft.issue%253D5992%26rft.spage%253D676%26rft.epage%253D679%26rft.atitle%253DHost%2BPhylogeny%2BConstrains%2BCross-Species%2BEmergence%2Band%2BEstablishment%2Bof%2BRabies%2BVirus%2Bin%2BBats%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1188836%26rft_id%253Dinfo%253Apmid%252F20689015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjkvNTk5Mi82NzYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNS8wMy82MjcxNTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.3" data-doi="10.1371/journal.pone.0000764"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Towner</span>  <span class="cit-name-given-names">JS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pourrut</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Albariño</span>  <span class="cit-name-given-names">CG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nkogue</span>  <span class="cit-name-given-names">CN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bird</span>  <span class="cit-name-given-names">BH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grard</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Marburg virus infection detected in a common African bat</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>. <span class="cit-pub-date">2007</span>;<span class="cit-vol">2</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">e764</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0000764</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DTowner%26rft.auinit1%253DJ.%2BS.%26rft.volume%253D2%26rft.issue%253D8%26rft.spage%253De764%26rft.epage%253De764%26rft.atitle%253DMarburg%2Bvirus%2Binfection%2Bdetected%2Bin%2Ba%2Bcommon%2BAfrican%2Bbat.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000764%26rft_id%253Dinfo%253Apmid%252F17712412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0000764&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17712412&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.4" data-doi="10.1038/438575a"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Leroy</span>  <span class="cit-name-given-names">EM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kumulungui</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pourrut</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rouquet</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hassanin</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yaba</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Fruit bats as reservoirs of Ebola virus</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-pub-date">2005</span>;<span class="cit-vol">438</span>(<span class="cit-issue">7068</span>):<span class="cit-fpage">575</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/438575a</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLeroy%26rft.auinit1%253DE.%2BM.%26rft.volume%253D438%26rft.issue%253D7068%26rft.spage%253D575%26rft.epage%253D576%26rft.atitle%253DFruit%2Bbats%2Bas%2Breservoirs%2Bof%2BEbola%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F438575a%26rft_id%253Dinfo%253Apmid%252F16319873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/438575a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16319873&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233593100030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.5" data-doi="10.1016/S1286-4579(01)01522-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chua</span>  <span class="cit-name-given-names">KB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lek Koh</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hooi</span>  <span class="cit-name-given-names">PS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wee</span>  <span class="cit-name-given-names">KF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Khong</span>  <span class="cit-name-given-names">JH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chua</span>  <span class="cit-name-given-names">BH</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Isolation of Nipah virus from Malaysian Island flying-foxes</span>. <abbr class="cit-jnl-abbrev">Microb Infect</abbr>. <span class="cit-pub-date">2002</span>;<span class="cit-vol">4</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">145</span>–<span class="cit-lpage">51</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/S1286-4579(01)01522-2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobes%2Band%2Binfection%2B%252F%2BInstitut%2BPasteur%26rft.stitle%253DMicrobes%2BInfect%26rft.aulast%253DChua%26rft.auinit1%253DK.%2BB.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253D145%26rft.epage%253D151%26rft.atitle%253DIsolation%2Bof%2BNipah%2Bvirus%2Bfrom%2BMalaysian%2BIsland%2Bflying-foxes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1286-4579%252801%252901522-2%26rft_id%253Dinfo%253Apmid%252F11880045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1286-4579(01)01522-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11880045&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174578300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.6" data-doi="10.1126/science.1118391"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Epstein</span>  <span class="cit-name-given-names">JH</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Bats are natural reservoirs of SARS-like coronaviruses</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-pub-date">2005</span>;<span class="cit-vol">310</span>(<span class="cit-issue">5748</span>):<span class="cit-fpage">676</span>–<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1118391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1118391%26rft_id%253Dinfo%253Apmid%252F16195424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTAvNTc0OC82NzYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNS8wMy82MjcxNTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.7" data-doi="10.1073/pnas.0506735102"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">SK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Woo</span>  <span class="cit-name-given-names">PC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">KS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tsoi</span>  <span class="cit-name-given-names">H-W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">BH</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Severe acute respiratory syndrome coronavirus-like virus in Chinese horseshoe bats</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>. <span class="cit-pub-date">2005</span>;<span class="cit-vol">102</span>(<span class="cit-issue">39</span>):<span class="cit-fpage">14040</span>–<span class="cit-lpage">5</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.0506735102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0506735102%26rft_id%253Dinfo%253Apmid%252F16169905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAyLzM5LzE0MDQwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDUvMDMvNjI3MTU4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.8" data-doi="10.1128/JVI.00631-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Identification of diverse alphacoronaviruses and genomic characterization of a novel severe acute respiratory syndrome-like coronavirus from bats in China</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">2014</span>;<span class="cit-vol">88</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">7070</span>–<span class="cit-lpage">82</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.00631-14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00631-14%26rft_id%253Dinfo%253Apmid%252F24719429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4OC8xMi83MDcwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDUvMDMvNjI3MTU4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.9" data-doi="10.1038/s41586-018-0010-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lan</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">X-L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">W-F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Fatal swine acute diarrhoea syndrome caused by an HKU2-related coronavirus of bat origin</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-pub-date">2018</span>;<span class="cit-vol">556</span>:<span class="cit-fpage">255</span>–<span class="cit-lpage">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41586-018-0010-9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D556%26rft.spage%253D255%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-018-0010-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-018-0010-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.10" data-doi="10.1038/nature05775"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">ND</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dunavan</span>  <span class="cit-name-given-names">CP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Diamond</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Origins of major human infectious diseases</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-pub-date">2007</span>;<span class="cit-vol">447</span>(<span class="cit-issue">7142</span>):<span class="cit-fpage">279</span>–<span class="cit-lpage">83</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature05775</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWolfe%26rft.auinit1%253DN.%2BD.%26rft.volume%253D447%26rft.issue%253D7142%26rft.spage%253D279%26rft.epage%253D283%26rft.atitle%253DOrigins%2Bof%2Bmajor%2Bhuman%2Binfectious%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05775%26rft_id%253Dinfo%253Apmid%252F17507975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05775&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17507975&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246520300036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-book" id="cit-627158v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">DeCaprio</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Imperiale</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Major</span>  <span class="cit-name-given-names">EO</span></span>. <span class="cit-source">Polyomaviridae, Fields virology</span>. <span class="cit-edition">6th ed</span>. <span class="cit-publ-loc">Philadelphia</span>: <span class="cit-publ-name">WoltersKluwer</span>; <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.12" data-doi="10.1016/j.meegid.2017.06.011"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Moens</span>  <span class="cit-name-given-names">U</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krumbholz</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ehlers</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zell</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Johne</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Calvignac-Spencer</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Biology, evolution, and medical importance of polyomaviruses: An update</span>. <abbr class="cit-jnl-abbrev">Infect Genet Evol</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">54</span>:<span class="cit-fpage">18</span>–<span class="cit-lpage">38</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.meegid.2017.06.011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfect%2BGenet%2BEvol%26rft.volume%253D54%26rft.spage%253D18%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.meegid.2017.06.011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.meegid.2017.06.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.13" data-doi="10.1126/science.1152586"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shuda</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">PS</span></span>. <span class="cit-article-title">Clonal integration of a polyomavirus in human Merkel cell carcinoma</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-pub-date">2008</span>;<span class="cit-vol">319</span>(<span class="cit-issue">5866</span>):<span class="cit-fpage">1096</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1152586</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1152586%26rft_id%253Dinfo%253Apmid%252F18202256%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTkvNTg2Ni8xMDk2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDUvMDMvNjI3MTU4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.14" data-doi="10.1016/j.virol.2012.12.015"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dalianis</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hirsch</span>  <span class="cit-name-given-names">HH</span></span>. <span class="cit-article-title">Human polyomaviruses in disease and cancer</span>. <abbr class="cit-jnl-abbrev">Virology</abbr>. <span class="cit-pub-date">2013</span>;<span class="cit-vol">437</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">63</span>–<span class="cit-lpage">72</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virol.2012.12.015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D437%26rft.spage%253D63%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2012.12.015%26rft_id%253Dinfo%253Apmid%252F23357733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2012.12.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23357733&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000315615100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.15" data-doi="10.3201/eid1901.121078"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dela Cruz</span>  <span class="cit-name-given-names">FN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Giannitti</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Woods</span>  <span class="cit-name-given-names">LW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Del Valle</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Delwart</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Novel polyomavirus associated with brain tumors in free-ranging raccoons, western United States</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr>. <span class="cit-pub-date">2013</span>;<span class="cit-vol">19</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">77</span>–<span class="cit-lpage">84</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.3201/eid1901.121078</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg%2BInfect%2BDis%26rft.volume%253D19%26rft.spage%253D77%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid1901.121078%26rft_id%253Dinfo%253Apmid%252F23260029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid1901.121078&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23260029&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.16" data-doi="10.1099/jgv.0.000839"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Moens</span>  <span class="cit-name-given-names">U</span></span>, <span class="cit-auth"><span class="cit-name-surname">Calvignac-Spencer</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lauber</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ramqvist</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feltkamp</span>  <span class="cit-name-given-names">MCW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Daugherty</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">ICTV virus taxonomy profile: <em>Polyomaviridae</em></span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">98</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">1159</span>–<span class="cit-lpage">60</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1099/jgv.0.000839</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BGen%2BVirol%26rft.volume%253D98%26rft.spage%253D1159%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fjgv.0.000839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/jgv.0.000839&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.17" data-doi="10.1371/journal.ppat.1005574"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Buck</span>  <span class="cit-name-given-names">CB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van Doorslaer</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peretti</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Geoghegan</span>  <span class="cit-name-given-names">EM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tisza</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">An</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">The ancient evolutionary history of polyomaviruses</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">12</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">e1005574</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.ppat.1005574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D12%26rft.spage%253De1005574%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1005574%26rft_id%253Dinfo%253Apmid%252F27093155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1005574&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27093155&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.18" data-doi="10.1128/JVI.00056-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Perez-Losada</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">RG</span></span>, <span class="cit-auth"><span class="cit-name-surname">McClellan</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">BJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Viscidi</span>  <span class="cit-name-given-names">RP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Demma</span>  <span class="cit-name-given-names">JC</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Comparing phylogenetic codivergence between polyomaviruses and their hosts</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">2006</span>;<span class="cit-vol">80</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">5663</span>–<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.00056-06</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DPerez-Losada%26rft.auinit1%253DM.%26rft.volume%253D80%26rft.issue%253D12%26rft.spage%253D5663%26rft.epage%253D5669%26rft.atitle%253DComparing%2BPhylogenetic%2BCodivergence%2Bbetween%2BPolyomaviruses%2Band%2BTheir%2BHosts%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00056-06%26rft_id%253Dinfo%253Apmid%252F16731904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4MC8xMi81NjYzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDUvMDMvNjI3MTU4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.19" data-doi="10.1099/vir.0.010694-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Misra</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dumonceaux</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dubois</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nadin-Davis</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Severini</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Detection of polyoma and corona viruses in bats of Canada</span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr>. <span class="cit-pub-date">2009</span>;<span class="cit-vol">90</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">2015</span>–<span class="cit-lpage">22</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1099/vir.0.010694-0</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BGen%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.010694-0%26rft_id%253Dinfo%253Apmid%252F19357225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.010694-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19357225&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268699400026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">20.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-627158v1.20" data-doi="10.1099/vir.0.044149-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fagrouch</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sarwari</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lavergne</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Delaval</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">De Thoisy</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lacoste</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Novel polyomaviruses in South American bats and their relationship to other members of the family <em>Polyomaviridae</em></span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">93</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">2652</span>–<span class="cit-lpage">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1099/vir.0.044149-0</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BGen%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.044149-0%26rft_id%253Dinfo%253Apmid%252F22971823%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.044149-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22971823&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000313027500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">21.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-627158v1.21" data-doi="10.1099/vir.0.047928-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tao</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Conrardy</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuzmin</span>  <span class="cit-name-given-names">IV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Recuenco</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Agwanda</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Discovery of diverse polyomaviruses in bats and the evolutionary history of the <em>Polyomaviridae</em></span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr>. <span class="cit-pub-date">2013</span>;<span class="cit-vol">94</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">738</span>–<span class="cit-lpage">48</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1099/vir.0.047928-0</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BGen%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.047928-0%26rft_id%253Dinfo%253Apmid%252F23239573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.047928-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23239573&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000317995200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">22.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-627158v1.22" data-doi="10.1007/s00705-015-2447-6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lima</span>  <span class="cit-name-given-names">FES</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cibulski</span>  <span class="cit-name-given-names">SP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Witt</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Franco</span>  <span class="cit-name-given-names">AC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roehe</span>  <span class="cit-name-given-names">PM</span></span>. <span class="cit-article-title">Genomic characterization of two novel polyomaviruses in Brazilian insectivorous bats</span>. <abbr class="cit-jnl-abbrev">Arch Virol</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">160</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">1831</span>–<span class="cit-lpage">6</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/s00705-015-2447-6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArch%2BVirol%26rft.volume%253D160%26rft.spage%253D1831%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00705-015-2447-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00705-015-2447-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">23.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-627158v1.23" data-doi="10.1007/s00705-015-2349-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kobayashi</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nakao</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Setiyono</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Handharyani</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Orba</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Detection of novel polyomaviruses in fruit bats in Indonesia</span>. <abbr class="cit-jnl-abbrev">Arch Virol</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">160</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">1075</span>–<span class="cit-lpage">82</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/s00705-015-2349-7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArch%2BVirol%26rft.volume%253D160%26rft.spage%253D1075%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00705-015-2349-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00705-015-2349-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">24.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-627158v1.24" data-doi="10.1099/vir.0.000158"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">NE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">ZL</span></span>, <span class="cit-auth"><span class="cit-name-surname">McInnes</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">DJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tompkins</span>  <span class="cit-name-given-names">DM</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Discovery of novel virus sequences in an isolated and threatened bat species, the New Zealand lesser short-tailed bat (<em>Mystacina tuberculata</em>)</span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">96</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">2442</span>–<span class="cit-lpage">52</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1099/vir.0.000158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BGen%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.000158%26rft_id%253Dinfo%253Apmid%252F25900137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.000158&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25900137&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.25" data-doi="10.1099/jgv.0.000737"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carr</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gonzalez</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ito</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ishii</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hang’ombe</span>  <span class="cit-name-given-names">BM</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Discovery of African bat polyomaviruses and infrequent recombination in the large T antigen in the <em>Polyomaviridae</em></span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">98</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">726</span>–<span class="cit-lpage">38</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1099/jgv.0.000737</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BGen%2BVirol%26rft.volume%253D98%26rft.spage%253D726%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fjgv.0.000737%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/jgv.0.000737&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.26" data-doi="10.1099/jgv.0.000935"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carr</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gonzalez</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dool</span>  <span class="cit-name-given-names">SE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ito</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ishii</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Identification of the same polyomavirus species in different African horseshoe bat species is indicative of short-range host-switching events</span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">98</span>(<span class="cit-issue">11</span>):<span class="cit-fpage">2771</span>–<span class="cit-lpage">85</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1099/jgv.0.000935</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BGen%2BVirol%26rft.volume%253D98%26rft.spage%253D2771%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fjgv.0.000935%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/jgv.0.000935&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.27" data-doi="10.1128/JVI.79.6.3883-3887.2005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Johne</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Enderlein</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nieper</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">H.</span></span> <span class="cit-article-title">Novel polyomavirus detected in the feces of a chimpanzee by nested broad-spectrum PCR</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">2005</span>;<span class="cit-vol">79</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">3883</span>–<span class="cit-lpage">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.79.6.3883-3887.2005</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DJohne%26rft.auinit1%253DR.%26rft.volume%253D79%26rft.issue%253D6%26rft.spage%253D3883%26rft.epage%253D3887%26rft.atitle%253DNovel%2BPolyomavirus%2BDetected%2Bin%2Bthe%2BFeces%2Bof%2Ba%2BChimpanzee%2Bby%2BNested%2BBroad-Spectrum%2BPCR%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.79.6.3883-3887.2005%26rft_id%253Dinfo%253Apmid%252F15731285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijc5LzYvMzg4MyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA1LzAzLzYyNzE1OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.28" data-doi="10.1038/ismej.2015.138"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Deciphering the bat virome catalog to better understand tha ecological diversity of bat viruses and the bat origin of emerging infectious diseases</span>. <abbr class="cit-jnl-abbrev">The ISME Journal</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">2015</span>:<span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/ismej.2015.138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BISME%2BJournal%26rft.volume%253D2015%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fismej.2015.138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ismej.2015.138&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">29.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-627158v1.29" data-doi="10.1128/genomeA.01053-16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cantalupo</span>  <span class="cit-name-given-names">PG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buck</span>  <span class="cit-name-given-names">CB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pipas</span>  <span class="cit-name-given-names">JM</span></span>. <span class="cit-article-title">Complete genome sequence of a polyomavirus recovered from a pomona leaf-nosed bat (<em>Hipposideros pomona</em>) metagenome data set</span>. <abbr class="cit-jnl-abbrev">Genome Announc</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">5</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">e01053</span>–<span class="cit-lpage">16</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/genomeA.01053-16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BAnnounc%26rft.volume%253D5%26rft.spage%253D01053e%26rft_id%253Dinfo%253Adoi%252F10.1128%252FgenomeA.01053-16%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/genomeA.01053-16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.30" data-doi="10.1371/journal.pone.0214227"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mishra</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fagbo</span>  <span class="cit-name-given-names">SF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alagaili</span>  <span class="cit-name-given-names">AN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nitido</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">SH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">A viral metagenomic survey identifies known and novel mammalian viruses in bats from Saudi Arabia</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>. <span class="cit-pub-date">2019</span>;<span class="cit-vol">14</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">e0214227</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0214227</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D14%26rft.spage%253De0214227%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0214227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0214227&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.31" data-doi="10.1016/S0379-0738(01)00403-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hsieh</span>  <span class="cit-name-given-names">H-M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chiang</span>  <span class="cit-name-given-names">H-L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">L-C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">S-Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">N-E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Linacre</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Cytochrome <em>b</em>gene for species identification of the conservation animals</span>. <abbr class="cit-jnl-abbrev">Forensic Sci Int</abbr>. <span class="cit-pub-date">2001</span>;<span class="cit-vol">122</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">7</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DForensic%2BScience%2BInternational%26rft.stitle%253DForensic%2BScience%2BInternational%26rft.aulast%253DHsieh%26rft.auinit1%253DH.%2BM.%26rft.volume%253D122%26rft.issue%253D1%26rft.spage%253D7%26rft.epage%253D18%26rft.atitle%253DCytochrome%2Bb%2Bgene%2Bfor%2Bspecies%2Bidentification%2Bof%2Bthe%2Bconservation%2Banimals.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0379-0738%252801%252900403-0%26rft_id%253Dinfo%253Apmid%252F11587860%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0379-0738(01)00403-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11587860&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171906700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.32" data-doi="10.1128/jvi.02493-16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matthijnssens</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qin</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Group A rotaviruses in Chinese bats: Genetic composition, serology, and evidence for bat-to-human transmission and reassortment</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">91</span>(<span class="cit-issue">12</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.02493-16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjvi.02493-16%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/jvi.02493-16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.33" data-doi="10.1371/journal.ppat.1007545"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Seroprevalence, cross antigenicity and circulation sphere of bat-borne hantaviruses revealed by serological and antigenic analyses</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>. <span class="cit-pub-date">2019</span>;<span class="cit-vol">15</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">e1007545</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.ppat.1007545</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D15%26rft.spage%253De1007545%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1007545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1007545&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.34" data-doi="10.1371/journal.pone.0061950"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Virome profiling of bats from Myanmar by metagenomic analysis of tissue samples reveals more novel mammalian viruses</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>. <span class="cit-pub-date">2013</span>;<span class="cit-vol">8</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">e61950</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0061950</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De61950%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0061950%26rft_id%253Dinfo%253Apmid%252F23630620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0061950&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23630620&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.35" data-doi="10.1128/genomeA.00997-15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ehlers</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Richter</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matuschka</span>  <span class="cit-name-given-names">FR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ulrich</span>  <span class="cit-name-given-names">RG</span></span>. <span class="cit-article-title">Genome sequences of a rat polyomavirus related to murine polyomavirus, Rattus norvegicus polyomavirus 1</span>. <abbr class="cit-jnl-abbrev">Genome Announc</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">3</span>(<span class="cit-issue">5</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/genomeA.00997-15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2BResource%2BAnnouncements%26rft.stitle%253DMicrobiol%2BResour%2BAnnounc%26rft.aulast%253DEhlers%26rft.auinit1%253DB.%26rft.volume%253D3%26rft.issue%253D5%26rft.spage%253De00997-15%26rft.epage%253De00997-15%26rft.atitle%253DGenome%2BSequences%2Bof%2Ba%2BRat%2BPolyomavirus%2BRelated%2Bto%2BMurine%2BPolyomavirus%252C%2BRattus%2Bnorvegicus%2BPolyomavirus%2B1%26rft_id%253Dinfo%253Adoi%252F10.1128%252FgenomeA.00997-15%26rft_id%253Dinfo%253Apmid%252F26337891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiZ2EiO3M6NToicmVzaWQiO3M6MTM6IjMvNS9lMDA5OTctMTUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNS8wMy82MjcxNTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.36" data-doi="10.1007/s00705-016-2794-y"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Calvignac-Spencer</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feltkamp</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Daugherty</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moens</span>  <span class="cit-name-given-names">U</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ramqvist</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Johne</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">A taxonomy update for the family <em>Polyomaviridae</em></span>. <abbr class="cit-jnl-abbrev">Arch Virol</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">161</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">1739</span>–<span class="cit-lpage">50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArch%2BVirol%26rft.volume%253D161%26rft.spage%253D1739%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00705-016-2794-y%26rft_id%253Dinfo%253Apmid%252F26923930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00705-016-2794-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26923930&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.37" data-doi="10.1515/MAMM.2009.022"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span> <span class="cit-article-title">Cryptic diversity in Chinese rhinolophids and hipposiderids (Chiroptera: Rhinolophidae and Hipposideridae)</span>. <abbr class="cit-jnl-abbrev">Mammalia</abbr>. <span class="cit-pub-date">2009</span>;<span class="cit-vol">73</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">135</span>–<span class="cit-lpage">41</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1515/MAMM.2009.022</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalia%26rft.volume%253D73%26rft.spage%253D135%26rft_id%253Dinfo%253Adoi%252F10.1515%252FMAMM.2009.022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1515/MAMM.2009.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.38" data-doi="10.1016/j.virol.2019.01.010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Virome profiling of rodents in Xinjiang Uygur Autonomous Region, China: Isolation and characterization of a new strain of Wenzhou virus</span>. <abbr class="cit-jnl-abbrev">Virology</abbr>. <span class="cit-pub-date">2019</span>;<span class="cit-vol">529</span>:<span class="cit-fpage">122</span>–<span class="cit-lpage">34</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virol.2019.01.010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D529%26rft.spage%253D122%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2019.01.010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2019.01.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.39" data-doi="10.1016/j.ympev.2016.01.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dool</span>  <span class="cit-name-given-names">SE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Puechmaille</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Foley</span>  <span class="cit-name-given-names">NM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Allegrini</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bastian</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mutumi</span>  <span class="cit-name-given-names">GL</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Nuclear introns outperform mitochondrial DNA in inter-specific phylogenetic reconstruction: lessons from horseshoe bats (Rhinolophidae: Chiroptera)</span>. <abbr class="cit-jnl-abbrev">Mol Phylogenet Evol</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">97</span>:<span class="cit-fpage">196</span>–<span class="cit-lpage">212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.ympev.2016.01.003</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BPhylogenet%2BEvol%26rft.volume%253D97%26rft.spage%253D196%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2016.01.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2016.01.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.40" data-doi="10.1371/currents.RRN1212"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Agnarsson</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zambrana-Torrelio</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Flores-Saldana</span>  <span class="cit-name-given-names">NP</span></span>, <span class="cit-auth"><span class="cit-name-surname">May-Collado</span>  <span class="cit-name-given-names">LJ</span></span>. <span class="cit-article-title">A time-calibrated species-level phylogeny of bats (Chiroptera, Mammalia)</span>. <abbr class="cit-jnl-abbrev">PLoS currents</abbr>. <span class="cit-pub-date">2011</span>;<span class="cit-vol">3</span>:<span class="cit-fpage">Rrn1212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/currents.RRN1212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bcurrents%26rft.volume%253D3%26rft.spage%253D1212Rrn%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fcurrents.RRN1212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/currents.RRN1212&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.41" data-doi="10.1126/science.1211028"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Meredith</span>  <span class="cit-name-given-names">RW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Janecka</span>  <span class="cit-name-given-names">JE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gatesy</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ryder</span>  <span class="cit-name-given-names">OA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Teeling</span>  <span class="cit-name-given-names">EC</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Impacts of the Cretaceous Terrestrial Revolution and KPg extinction on mammal diversification</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-pub-date">2011</span>;<span class="cit-vol">334</span>(<span class="cit-issue">6055</span>):<span class="cit-fpage">521</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1211028</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1211028%26rft_id%253Dinfo%253Apmid%252F21940861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzQvNjA1NS81MjEiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNS8wMy82MjcxNTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.42" data-doi="10.1038/369160a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stehle</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Benjamin</span>  <span class="cit-name-given-names">TL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harrison</span>  <span class="cit-name-given-names">SC</span></span>. <span class="cit-article-title">Structure of murine polyomavirus complexed with an oligosaccharide receptor fragment</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-pub-date">1994</span>;<span class="cit-vol">369</span>(<span class="cit-issue">6476</span>):<span class="cit-fpage">160</span>–<span class="cit-lpage">3</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/369160a0</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DStehle%26rft.auinit1%253DT.%26rft.volume%253D369%26rft.issue%253D6476%26rft.spage%253D160%26rft.epage%253D163%26rft.atitle%253DStructure%2Bof%2Bmurine%2Bpolyomavirus%2Bcomplexed%2Bwith%2Ban%2Boligosaccharide%2Breceptor%2Bfragment.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F369160a0%26rft_id%253Dinfo%253Apmid%252F8177322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/369160a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8177322&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994NK97100053&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">43.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-627158v1.43" data-doi="10.1073/pnas.0710301105"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Neu</span>  <span class="cit-name-given-names">U</span></span>, <span class="cit-auth"><span class="cit-name-surname">Woellner</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gauglitz</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stehle</span>  <span class="cit-name-given-names">T.</span></span> <span class="cit-article-title">Structural basis of GM1 ganglioside recognition by simian virus 40</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>. <span class="cit-pub-date">2008</span>;<span class="cit-vol">105</span>(<span class="cit-issue">13</span>):<span class="cit-fpage">5219</span>–<span class="cit-lpage">24</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.0710301105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0710301105%26rft_id%253Dinfo%253Apmid%252F18353982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA1LzEzLzUyMTkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNS8wMy82MjcxNTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">44.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-627158v1.44" data-doi="10.1016/j.chom.2010.09.004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Neu</span>  <span class="cit-name-given-names">U</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maginnis</span>  <span class="cit-name-given-names">MS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Palma</span>  <span class="cit-name-given-names">AS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stroh</span>  <span class="cit-name-given-names">LJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">CD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feizi</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Structure-function analysis of the human JC polyomavirus establishes the LSTc pentasaccharide as a functional receptor motif</span>. <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr>. <span class="cit-pub-date">2010</span>;<span class="cit-vol">8</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">309</span>–<span class="cit-lpage">19</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.chom.2010.09.004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCell%2BHost%2BMicrobe%26rft.aulast%253DNeu%26rft.auinit1%253DU.%26rft.volume%253D8%26rft.issue%253D4%26rft.spage%253D309%26rft.epage%253D319%26rft.atitle%253DStructure-function%2Banalysis%2Bof%2Bthe%2Bhuman%2BJC%2Bpolyomavirus%2Bestablishes%2Bthe%2BLSTc%2Bpentasaccharide%2Bas%2Ba%2Bfunctional%2Breceptor%2Bmotif.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2010.09.004%26rft_id%253Dinfo%253Apmid%252F20951965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2010.09.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20951965&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283806000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.45" data-doi="10.1371/journal.ppat.1002738"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Neu</span>  <span class="cit-name-given-names">U</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hengel</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blaum</span>  <span class="cit-name-given-names">BS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schowalter</span>  <span class="cit-name-given-names">RM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Macejak</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Structures of Merkel cell polyomavirus VP1 complexes define a sialic acid binding site required for infection</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">8</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">e1002738</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.ppat.1002738</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DNeu%26rft.auinit1%253DU.%26rft.volume%253D8%26rft.issue%253D7%26rft.spage%253De1002738%26rft.epage%253De1002738%26rft.atitle%253DStructures%2Bof%2BMerkel%2Bcell%2Bpolyomavirus%2BVP1%2Bcomplexes%2Bdefine%2Ba%2Bsialic%2Bacid%2Bbinding%2Bsite%2Brequired%2Bfor%2Binfection.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002738%26rft_id%253Dinfo%253Apmid%252F22910713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002738&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22910713&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.46" data-doi="10.1016/j.str.2018.03.019"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hurdiss</span>  <span class="cit-name-given-names">DL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frank</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snowden</span>  <span class="cit-name-given-names">JS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Macdonald</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ranson</span>  <span class="cit-name-given-names">NA</span></span>. <span class="cit-article-title">The structure of an infectious human polyomavirus and its interactions with cellular receptors</span>. <abbr class="cit-jnl-abbrev">Structure</abbr>. <span class="cit-pub-date">2018</span>;<span class="cit-vol">26</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">839</span>-<span class="cit-lpage">47</span>.e3.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.str.2018.03.019</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStructure%26rft.volume%253D26%26rft.spage%253D839%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.str.2018.03.019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.str.2018.03.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Warden</span>  <span class="cit-name-given-names">CD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lacey</span>  <span class="cit-name-given-names">SF</span></span>. <span class="cit-article-title">Updated phylogenetic analysis of polyomavirus-host co-evolution</span>. <abbr class="cit-jnl-abbrev">J Bioinfo Res</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">1</span>:<span class="cit-fpage">46</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBioinfo%2BRes%26rft.volume%253D1%26rft.spage%253D46%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.48" data-doi="10.1016/j.meegid.2008.05.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Krumbholz</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bininda-Emonds</span>  <span class="cit-name-given-names">OR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wutzler</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zell</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Evolution of four BK virus subtypes</span>. <abbr class="cit-jnl-abbrev">Infect Genet Evol</abbr>. <span class="cit-pub-date">2008</span>;<span class="cit-vol">8</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">632</span>–<span class="cit-lpage">43</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.meegid.2008.05.006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%252C%2Bgenetics%2Band%2Bevolution%2B%253A%2B%2Bjournal%2Bof%2Bmolecular%2Bepidemiology%2Band%2Bevolutionary%2Bgenetics%2Bin%2Binfectious%2Bdiseases%26rft.stitle%253DInfect%2BGenet%2BEvol%26rft.aulast%253DKrumbholz%26rft.auinit1%253DA.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253D632%26rft.epage%253D643%26rft.atitle%253DEvolution%2Bof%2Bfour%2BBK%2Bvirus%2Bsubtypes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.meegid.2008.05.006%26rft_id%253Dinfo%253Apmid%252F18582602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.meegid.2008.05.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18582602&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260160900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.49" data-doi="10.1128/JVI.01178-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Johne</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">H.</span></span> <span class="cit-article-title">Polyomaviruses of birds: etiologic agents of inflammatory diseases in a tumor virus family</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">2007</span>;<span class="cit-vol">81</span>(<span class="cit-issue">21</span>):<span class="cit-fpage">11554</span>–<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.01178-07</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01178-07%26rft_id%253Dinfo%253Apmid%252F17715213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4MS8yMS8xMTU1NCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA1LzAzLzYyNzE1OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.50" data-doi="10.1016/j.virol.2017.11.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nie</span>  <span class="cit-name-given-names">FY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">XD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">ZY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">XN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">ZX</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Extensive diversity and evolution of hepadnaviruses in bats in China</span>. <abbr class="cit-jnl-abbrev">Virology</abbr>. <span class="cit-pub-date">2018</span>;<span class="cit-vol">514</span>:<span class="cit-fpage">88</span>–<span class="cit-lpage">97</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virol.2017.11.005</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D514%26rft.spage%253D88%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2017.11.005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2017.11.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.51" data-doi="10.1371/journal.pone.0084679"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eckerle</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ehlen</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kallies</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wollny</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Corman</span>  <span class="cit-name-given-names">VM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cottontail</span>  <span class="cit-name-given-names">VM</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Bat airway epithelial cells: a novel tool for the study of zoonotic viruses</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>. <span class="cit-pub-date">2014</span>;<span class="cit-vol">9</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">e84679</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0084679</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De84679%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0084679%26rft_id%253Dinfo%253Apmid%252F24454736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0084679&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24454736&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.52" data-doi="10.1016/j.jviromet.2016.09.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Banerjee</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rapin</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Griebel</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Munster</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Generation and characterization of Eptesicus fuscus (Big brown bat) kidney cell lines immortalized using the <em>Myotis</em> polyomavirus large T-antigen</span>. <abbr class="cit-jnl-abbrev">J Virol Methods</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">237</span>:<span class="cit-fpage">166</span>–<span class="cit-lpage">73</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.jviromet.2016.09.008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%2BMethods%26rft.volume%253D237%26rft.spage%253D166%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jviromet.2016.09.008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jviromet.2016.09.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.53" data-doi="10.1093/gbe/evt211"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">García-Pérez</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ibáñez</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Godínez</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aréchiga</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garin</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pérez-Suárez</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Novel papillomaviruses in free-ranging Iberian bats: no virus–host co-evolution, no strict host specificity, and hints for recombination</span>. <abbr class="cit-jnl-abbrev">Genome Biol Evol</abbr>. <span class="cit-pub-date">2014</span>;<span class="cit-vol">6</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">94</span>–<span class="cit-lpage">104</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/gbe/evt211</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevt211%26rft_id%253Dinfo%253Apmid%252F24391150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evt211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24391150&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.54" data-doi="10.1128/JVI.01498-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Corman</span>  <span class="cit-name-given-names">VM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ithete</span>  <span class="cit-name-given-names">NL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">LR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schoeman</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Preiser</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drosten</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Rooting the phylogenetic tree of middle East respiratory syndrome coronavirus by characterization of a conspecific virus from an African bat</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">2014</span>;<span class="cit-vol">88</span>(<span class="cit-issue">19</span>):<span class="cit-fpage">11297</span>–<span class="cit-lpage">303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.01498-14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01498-14%26rft_id%253Dinfo%253Apmid%252F25031349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4OC8xOS8xMTI5NyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA1LzAzLzYyNzE1OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.55" data-doi="10.1099/vir.0.000033"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peretti</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">FitzGerald</span>  <span class="cit-name-given-names">PC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bliskovsky</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buck</span>  <span class="cit-name-given-names">CB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pastrana</span>  <span class="cit-name-given-names">DV</span></span>. <span class="cit-article-title">Hamburger polyomaviruses</span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">96</span>(<span class="cit-issue">Pt 4</span>):<span class="cit-fpage">833</span>–<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1099/vir.0.000033</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BGen%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.000033%26rft_id%253Dinfo%253Apmid%252F25568187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoidmlyIjtzOjU6InJlc2lkIjtzOjExOiI5Ni9QdF80LzgzMyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA1LzAzLzYyNzE1OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-book" id="cit-627158v1.56"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-source">Polyomaviruses, Diagnostic microbiology of the immunocompromised host</span>. <span class="cit-edition">2 ed</span>. <span class="cit-publ-loc">Washington, DC</span>: <span class="cit-publ-name">ASM Press</span>; <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.57" data-doi="10.1186/s13000-016-0509-z"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Beckervordersandforth</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pujari</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rennspiess</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Speel</span>  <span class="cit-name-given-names">EJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Winnepenninckx</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Diaz</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Frequent detection of human polyomavirus 6 in keratoacanthomas</span>. <abbr class="cit-jnl-abbrev">Diagn Pathol</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">11</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">58</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/s13000-016-0509-z</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDiagn%2BPathol%26rft.volume%253D11%26rft.spage%253D58%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13000-016-0509-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13000-016-0509-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.58" data-doi="10.1002/ijc.27413"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hausen</span>  <span class="cit-name-given-names">Hz</span></span>. <span class="cit-article-title">Red meat consumption and cancer: reasons to suspect involvement of bovine infectious factors in colorectal cancer</span>. <abbr class="cit-jnl-abbrev">Int J Cancer</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">130</span>(<span class="cit-issue">11</span>):<span class="cit-fpage">2475</span>–<span class="cit-lpage">83</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1002/ijc.27413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2Bjournal%2Bof%2Bcancer.%2B%2BJournal%2Binternational%2Bdu%2Bcancer%26rft.stitle%253DInt%2BJ%2BCancer%26rft.aulast%253DHausen%26rft.auinit1%253DH.%2BZ.%26rft.volume%253D130%26rft.issue%253D11%26rft.spage%253D2475%26rft.epage%253D2483%26rft.atitle%253DRed%2Bmeat%2Bconsumption%2Band%2Bcancer%253A%2Breasons%2Bto%2Bsuspect%2Binvolvement%2Bof%2Bbovine%2Binfectious%2Bfactors%2Bin%2Bcolorectal%2Bcancer.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fijc.27413%26rft_id%253Dinfo%253Apmid%252F22212999%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/ijc.27413&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22212999&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.59" data-doi="10.1007/s00705-008-0264-x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sauerbrei</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wutzler</span>  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">Testing thermal resistance of viruses</span>. <abbr class="cit-jnl-abbrev">Arch Virol</abbr>. <span class="cit-pub-date">2009</span>;<span class="cit-vol">154</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">115</span>–<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/s00705-008-0264-x</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArchives%2Bof%2Bvirology%26rft.stitle%253DArch%2BVirol%26rft.aulast%253DSauerbrei%26rft.auinit1%253DA.%26rft.volume%253D154%26rft.issue%253D1%26rft.spage%253D115%26rft.epage%253D119%26rft.atitle%253DTesting%2Bthermal%2Bresistance%2Bof%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00705-008-0264-x%26rft_id%253Dinfo%253Apmid%252F19039515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00705-008-0264-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19039515&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.60" data-doi="10.1038/ncomms9147"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anthony</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Islam</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Navarrete-Macias</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Non-random patterns in viral diversity</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">6</span>:<span class="cit-fpage">8147</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/ncomms9147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D6%26rft.spage%253D8147%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms9147%26rft_id%253Dinfo%253Apmid%252F26391192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms9147&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26391192&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.61" data-doi="10.1128/JVI.01311-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Campanero-Rhodes</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chai</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sonnino</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mauri</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Childs</span>  <span class="cit-name-given-names">RA</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">N-glycolyl GM1 ganglioside as a receptor for simian virus 40</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">2007</span>;<span class="cit-vol">81</span>(<span class="cit-issue">23</span>):<span class="cit-fpage">12846</span>–<span class="cit-lpage">58</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.01311-07</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01311-07%26rft_id%253Dinfo%253Apmid%252F17855525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4MS8yMy8xMjg0NiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA1LzAzLzYyNzE1OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.62" data-doi="10.1016/j.coviro.2014.05.004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">O’Hara</span>  <span class="cit-name-given-names">SD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stehle</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garcea</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Glycan receptors of the <em>Polyomaviridae</em>: Structure, function, and pathogenesis</span>. <abbr class="cit-jnl-abbrev">Curr Opin Virol</abbr>. <span class="cit-pub-date">2014</span>;<span class="cit-vol">7</span>:<span class="cit-fpage">73</span>–<span class="cit-lpage">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.coviro.2014.05.004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr%2BOpin%2BVirol%26rft.volume%253D7%26rft.spage%253D73%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.coviro.2014.05.004%26rft_id%253Dinfo%253Apmid%252F24983512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.coviro.2014.05.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24983512&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.63" data-doi="10.1002/jmv.1890390110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">PE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knowles</span>  <span class="cit-name-given-names">WA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clewley</span>  <span class="cit-name-given-names">JP</span></span>. <span class="cit-article-title">BK virus antigenic variants: sequence analysis within the capsid VP1 epitope</span>. <abbr class="cit-jnl-abbrev">J Med Virol</abbr>. <span class="cit-pub-date">1993</span>;<span class="cit-vol">39</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">50</span>–<span class="cit-lpage">6</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1002/jmv.1890390110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmedical%2Bvirology%26rft.stitle%253DJ%2BMed%2BVirol%26rft.aulast%253DJin%26rft.auinit1%253DL.%26rft.volume%253D39%26rft.issue%253D1%26rft.spage%253D50%26rft.epage%253D56%26rft.atitle%253DBK%2Bvirus%2Bantigenic%2Bvariants%253A%2Bsequence%2Banalysis%2Bwithin%2Bthe%2Bcapsid%2BVP1%2Bepitope.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjmv.1890390110%26rft_id%253Dinfo%253Apmid%252F7678637%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jmv.1890390110&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7678637&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.64" data-doi="10.1128/JVI.00949-09"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Erickson</span>  <span class="cit-name-given-names">KD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garcea</span>  <span class="cit-name-given-names">RL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">Ganglioside GT1b is a putative host cell receptor for the Merkel cell polyomavirus</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">2009</span>;<span class="cit-vol">83</span>(<span class="cit-issue">19</span>):<span class="cit-fpage">10275</span>–<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.00949-09</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00949-09%26rft_id%253Dinfo%253Apmid%252F19605473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4My8xOS8xMDI3NSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA1LzAzLzYyNzE1OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">65.</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.65" data-doi="10.1016/j.virol.2008.07.032"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Murata</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Teferedegne</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sheng</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peden</span>  <span class="cit-name-given-names">K.</span></span> <span class="cit-article-title">Identification of a neutralization epitope in the VP1 capsid protein of SV40</span>. <abbr class="cit-jnl-abbrev">Virology</abbr>. <span class="cit-pub-date">2008</span>;<span class="cit-vol">381</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">116</span>–<span class="cit-lpage">22</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virol.2008.07.032</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DVirology%26rft.aulast%253DMurata%26rft.auinit1%253DH.%26rft.volume%253D381%26rft.issue%253D1%26rft.spage%253D116%26rft.epage%253D122%26rft.atitle%253DIdentification%2Bof%2Ba%2Bneutralization%2Bepitope%2Bin%2Bthe%2BVP1%2Bcapsid%2Bprotein%2Bof%2BSV40.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2008.07.032%26rft_id%253Dinfo%253Apmid%252F18789470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2008.07.032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18789470&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">66.</span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66. in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.66" data-doi="10.1371/journal.pone.0121751"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fleury</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nicol</span>  <span class="cit-name-given-names">JT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Samimi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cazal</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ballaire</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Identification of the neutralizing epitopes of Merkel cell polyomavirus major capsid protein within the BC and EF surface loops</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">10</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">e0121751</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0121751</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0121751%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0121751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0121751&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">67.</span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference 67. in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.67" data-doi="10.1128/JVI.01084-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ströh</span>  <span class="cit-name-given-names">LJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neu</span>  <span class="cit-name-given-names">U</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blaum</span>  <span class="cit-name-given-names">BS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buch</span>  <span class="cit-name-given-names">MHC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garcea</span>  <span class="cit-name-given-names">RL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stehle</span>  <span class="cit-name-given-names">T.</span></span> <span class="cit-article-title">Structure analysis of the major capsid proteins of human polyomaviruses 6 and 7 reveals an obstructed sialic acid binding site</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">2014</span>;<span class="cit-vol">88</span>(<span class="cit-issue">18</span>):<span class="cit-fpage">10831</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.01084-14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01084-14%26rft_id%253Dinfo%253Apmid%252F25008942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4OC8xOC8xMDgzMSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA1LzAzLzYyNzE1OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">68.</span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference 68. in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.68" data-doi="10.1146/annurev-animal-022516-022811"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Teeling</span>  <span class="cit-name-given-names">EC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vernes</span>  <span class="cit-name-given-names">SC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davalos</span>  <span class="cit-name-given-names">LM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ray</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">MTP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E.</span></span> <span class="cit-article-title">Bat biology, genomes, and the Bat1K project: To generate chromosome-level genomes for all Living bat species</span>. <abbr class="cit-jnl-abbrev">Annu Rev Anim Biosci</abbr>. <span class="cit-pub-date">2018</span>;<span class="cit-vol">6</span>:<span class="cit-fpage">23</span>–<span class="cit-lpage">46</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1146/annurev-animal-022516-022811</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BAnim%2BBiosci%26rft.volume%253D6%26rft.spage%253D23%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-animal-022516-022811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-animal-022516-022811&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">69.</span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference 69. in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.69" data-doi="10.1099/vir.0.027854-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Orba</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kobayashi</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nakamura</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ishii</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hang</span>,  <span class="cit-name-given-names">Apos</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Detection and characterization of a novel polyomavirus in wild rodents</span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr>. <span class="cit-pub-date">2011</span>;<span class="cit-vol">92</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">789</span>–<span class="cit-lpage">95</span>. DOI<span class="cit-pub-id-sep cit-pub-id-doi-sep">:</span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1099/vir.0.027854-0</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DOrba%26rft.auinit1%253DY.%26rft.volume%253D92%26rft.issue%253DPt%2B4%26rft.spage%253D789%26rft.epage%253D795%26rft.atitle%253DDetection%2Band%2Bcharacterization%2Bof%2Ba%2Bnovel%2Bpolyomavirus%2Bin%2Bwild%2Brodents.%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.027854-0%26rft_id%253Dinfo%253Apmid%252F21177925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.027854-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21177925&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">70.</span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference 70. in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.70" data-doi="10.1371/journal.pone.0108277"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Muhire</span>  <span class="cit-name-given-names">BM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Varsani</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">DP</span></span>. <span class="cit-article-title">SDT: A virus classification tool based on pairwise sequence alignment and identity calculation</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>. <span class="cit-pub-date">2014</span>;<span class="cit-vol">9</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">e108277</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0108277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De108277%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0108277%26rft_id%253Dinfo%253Apmid%252F25259891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0108277&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25259891&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">71.</span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference 71. in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.71" data-doi="10.1093/molbev/mst010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Standley</span>  <span class="cit-name-given-names">DM</span></span>. <span class="cit-article-title">MAFFT multiple sequence alignment software version 7: Improvements in performance and usability</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr>. <span class="cit-pub-date">2013</span>;<span class="cit-vol">30</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">772</span>–<span class="cit-lpage">80</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/molbev/mst010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst010%26rft_id%253Dinfo%253Apmid%252F23329690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23329690&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000317002300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">72.</span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference 72. in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.72" data-doi="10.1093/sysbio/sys029"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ronquist</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Teslenko</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">van der Mark</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ayres</span>  <span class="cit-name-given-names">DL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Darling</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Höhna</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">MrBayes 3.2: Efficient Bayesian phylogenetic inference and model choice across a large model space</span>. <abbr class="cit-jnl-abbrev">Syst Biol</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">61</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">539</span>–<span class="cit-lpage">42</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/sysbio/sys029</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsys029%26rft_id%253Dinfo%253Apmid%252F22357727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/sys029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22357727&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">73.</span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference 73. in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.73" data-doi="10.1007/s00705-016-3129-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carr</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kawaguchi</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gonzalez</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ito</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Isolation of a simian immunodeficiency virus from a malbrouck (<em>Chlorocebus cynosuros</em>)</span>. <abbr class="cit-jnl-abbrev">Arch Virol</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">162</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">543</span>-<span class="cit-lpage">8</span>. Epub 2016/11/03.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/s00705-016-3129-8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArch%2BVirol%26rft.volume%253D162%26rft.spage%253D543%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00705-016-3129-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00705-016-3129-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">74.</span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference 74. in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.74" data-doi="10.1093/molbev/msw054"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stecher</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K.</span></span> <span class="cit-article-title">MEGA7: molecular evolutionary genetics analysis version 7.0 for bigger datasets</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">33</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">1870</span>–<span class="cit-lpage">4</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/molbev/msw054</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw054%26rft_id%253Dinfo%253Apmid%252F27004904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27004904&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">75.</span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference 75. in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.75" data-doi="10.1093/molbev/mss075"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suchard</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Bayesian phylogenetics with BEAUti and the BEAST 1.7</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">29</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">1969</span>–<span class="cit-lpage">73</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/molbev/mss075</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss075%26rft_id%253Dinfo%253Apmid%252F22367748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mss075&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22367748&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000307171300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">76.</span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference 76. in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.76" data-doi="10.1371/journal.pgen.1001342"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Perelman</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">WE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roos</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Seuánez</span>  <span class="cit-name-given-names">HN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horvath</span>  <span class="cit-name-given-names">JE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moreira</span>  <span class="cit-name-given-names">MAM</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">A molecular phylogeny of living primates</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr>. <span class="cit-pub-date">2011</span>;<span class="cit-vol">7</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">e1001342</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pgen.1001342</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DPerelman%26rft.auinit1%253DP.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253De1001342%26rft.epage%253De1001342%26rft.atitle%253DA%2Bmolecular%2Bphylogeny%2Bof%2Bliving%2Bprimates.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001342%26rft_id%253Dinfo%253Apmid%252F21436896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001342&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21436896&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">77.</span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference 77. in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.77" data-doi="10.1186/1748-7188-5-16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Conow</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fielder</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ovadia</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Libeskind-Hadas</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Jane: a new tool for the cophylogeny reconstruction problem</span>. <abbr class="cit-jnl-abbrev">Algorithms Mol Biol</abbr>. <span class="cit-pub-date">2010</span>;<span class="cit-vol">5</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">16</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/1748-7188-5-16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAlgorithms%2BMol%2BBiol%26rft.aulast%253DConow%26rft.auinit1%253DC.%26rft.volume%253D5%26rft.spage%253D16%26rft.epage%253D16%26rft.atitle%253DJane%253A%2Ba%2Bnew%2Btool%2Bfor%2Bthe%2Bcophylogeny%2Breconstruction%2Bproblem.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1748-7188-5-16%26rft_id%253Dinfo%253Apmid%252F20181081%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1748-7188-5-16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20181081&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">78.</span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference 78. in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-627158v1.78" data-doi="10.1093/nar/gky427"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bertoni</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bienert</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Studer</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tauriello</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gumienny</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">SWISS-MODEL: homology modelling of protein structures and complexes</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-pub-date">2018</span>;<span class="cit-vol">46</span>(<span class="cit-issue">W1</span>):<span class="cit-fpage">W296</span>-<span class="cit-lpage">W303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gky427</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D46%26rft.spage%253DW296%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky427%26rft_id%253Dinfo%253Apmid%252F29788355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gky427&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29788355&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F03%2F627158.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/627158v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F627158v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/627133v1" title="Surprise response as a probe for compressed memory states" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;May 03, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/627158v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/627158v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/627158v1" class="active" data-icon-position="" data-hide-link-title="0">Extensive genetic diversity of bat-borne polyomaviruses reveals inter-family host-switching events</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/682555" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-95SonkRFTarF4pN8-YfijVuorrU2mdNfbFPVjfs6KaA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="682555" id="node682555--2" data-pisa="biorxiv;627158v1" data-pisa-master="biorxiv;627158" data-seqnum="682555" data-apath="/biorxiv/early/2019/05/03/627158.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Extensive genetic diversity of bat-borne polyomaviruses reveals inter-family host-switching events</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Zhizhou</span> <span class="nlm-surname">Tan</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Gabriel</span> <span class="nlm-surname">Gonzalez</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jinliang</span> <span class="nlm-surname">Sheng</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Jianmin</span> <span class="nlm-surname">Wu</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Fuqiang</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Lin</span> <span class="nlm-surname">Xu</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Peisheng</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Aiwei</span> <span class="nlm-surname">Zhu</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Yonggang</span> <span class="nlm-surname">Qu</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Changchun</span> <span class="nlm-surname">Tu</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Michael J.</span> <span class="nlm-surname">Carr</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Biao</span> <span class="nlm-surname">He</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">627158; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/627158 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/627158v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/627158v1&amp;title=Extensive%20genetic%20diversity%20of%20bat-borne%20polyomaviruses%20reveals%20inter-family%20host-switching%20events" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/627158v1&amp;title=Extensive%20genetic%20diversity%20of%20bat-borne%20polyomaviruses%20reveals%20inter-family%20host-switching%20events" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/627158v1&amp;text=Extensive%20genetic%20diversity%20of%20bat-borne%20polyomaviruses%20reveals%20inter-family%20host-switching%20events" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/627158v1&amp;title=Extensive%20genetic%20diversity%20of%20bat-borne%20polyomaviruses%20reveals%20inter-family%20host-switching%20events" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/627158v1&amp;t=Extensive%20genetic%20diversity%20of%20bat-borne%20polyomaviruses%20reveals%20inter-family%20host-switching%20events" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/627158v1&amp;title=Extensive%20genetic%20diversity%20of%20bat-borne%20polyomaviruses%20reveals%20inter-family%20host-switching%20events" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/627158v1&amp;title=Extensive%20genetic%20diversity%20of%20bat-borne%20polyomaviruses%20reveals%20inter-family%20host-switching%20events" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="682555" id="node682555--3" data-pisa="biorxiv;627158v1" data-pisa-master="biorxiv;627158" data-seqnum="682555" data-apath="/biorxiv/early/2019/05/03/627158.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Extensive genetic diversity of bat-borne polyomaviruses reveals inter-family host-switching events</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Zhizhou</span> <span class="nlm-surname">Tan</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Gabriel</span> <span class="nlm-surname">Gonzalez</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jinliang</span> <span class="nlm-surname">Sheng</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Jianmin</span> <span class="nlm-surname">Wu</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Fuqiang</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Lin</span> <span class="nlm-surname">Xu</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Peisheng</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Aiwei</span> <span class="nlm-surname">Zhu</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Yonggang</span> <span class="nlm-surname">Qu</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Changchun</span> <span class="nlm-surname">Tu</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Michael J.</span> <span class="nlm-surname">Carr</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Biao</span> <span class="nlm-surname">He</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">627158; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/627158 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Extensive%20genetic%20diversity%20of%20bat-borne%20polyomaviruses%20reveals%20inter-family%20host-switching%20events&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Tan%2C+Zhizhou&amp;rft.au=Gonzalez%2C+Gabriel&amp;rft.au=Sheng%2C+Jinliang&amp;rft.au=Wu%2C+Jianmin&amp;rft.au=Zhang%2C+Fuqiang&amp;rft.au=Xu%2C+Lin&amp;rft.au=Zhang%2C+Peisheng&amp;rft.au=Zhu%2C+Aiwei&amp;rft.au=Qu%2C+Yonggang&amp;rft.au=Tu%2C+Changchun&amp;rft.au=Carr%2C+Michael+J.&amp;rft.au=He%2C+Biao"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/682555/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/682555/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/682555/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/682555/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/682555/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/682555/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/682555/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/682555/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/682555/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/682555/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/682555/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/682555/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/627158v1&amp;count=horizontal&amp;via=&amp;text=Extensive%20genetic%20diversity%20of%20bat-borne%20polyomaviruses%20reveals%20inter-family%20host-switching%20events&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/627158v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/627158v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/627158v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">Microbiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
