<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/20/2020.03.18.996975.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.18.996975v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Single-cell transcriptome profiling an adult human cell atlas of 15 major organs" />
<meta name="DC.Identifier" content="10.1101/2020.03.18.996975" />
<meta name="DC.Date" content="2020-03-20" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Background As core units of organ tissues, cells of various types play their harmonious rhythms to maintain the homeostasis of human body. It is essential to identify characteristics of the cells in human organs and their regulatory networks for understanding biological mechanisms related to health and disease. However, a systematic and comprehensive single-cell transcriptional profile across multiple organs of normal human adults has been pending.

Results we performed single-cell transcriptomes of 88,622 cells derived from 15 tissue organs of one adult donor and generated an adult human cell atlas (AHCA). The AHCA depicted 234 subtypes of cells, including major cell types such as T, B, myeloid, epithelial, and stromal cells, as well as novel cell types in skin, each of which was distinguished by multiple marker genes and transcriptional profiles and collectively contributed to the heterogeneity of major human organs. Moreover, TCR and BCR repertoire comparison and trajectory analyses revealed direct clonal sharing of T and B cells with various developmental states among different tissues. Furthermore, novel cell markers, transcription factors and ligand-receptor pairs were identified with potential functional regulations on maintaining the homeostasis of human cells among tissues.

Conclusions The AHCA reveals the inter- and intra-organ heterogeneity of cell characteristics and provides a useful resource to uncover key events during the development of human diseases such as the recent outbreak of coronavirus disease 2019 (COVID-19) in the context of heterogeneity of cells and organs." />
<meta name="DC.Contributor" content="Shuai He" />
<meta name="DC.Contributor" content="Lin-he Wang" />
<meta name="DC.Contributor" content="Yang Liu" />
<meta name="DC.Contributor" content="Yi-qi Li" />
<meta name="DC.Contributor" content="Haitian Chen" />
<meta name="DC.Contributor" content="Jinghong Xu" />
<meta name="DC.Contributor" content="Wan Peng" />
<meta name="DC.Contributor" content="Guo-Wang Lin" />
<meta name="DC.Contributor" content="Pan-Pan Wei" />
<meta name="DC.Contributor" content="Bo Li" />
<meta name="DC.Contributor" content="Xiaojun Xia" />
<meta name="DC.Contributor" content="Dan Wang" />
<meta name="DC.Contributor" content="Jin-Xin Bei" />
<meta name="DC.Contributor" content="Xiaoshun He" />
<meta name="DC.Contributor" content="Zhiyong Guo" />
<meta name="article:published_time" content="2020-03-20" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Single-cell transcriptome profiling an adult human cell atlas of 15 major organs" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Background: As core units of organ tissues, cells of various types play their harmonious rhythms to maintain the homeostasis of human body. It is essential to identify characteristics of the cells in human organs and their regulatory networks for understanding biological mechanisms related to health and disease. However, a systematic and comprehensive single-cell transcriptional profile across multiple organs of normal human adults has been pending. Results: we performed single-cell transcriptomes of 88,622 cells derived from 15 tissue organs of one adult donor and generated an adult human cell atlas (AHCA). The AHCA depicted 234 subtypes of cells, including major cell types such as T, B, myeloid, epithelial, and stromal cells, as well as novel cell types in skin, each of which was distinguished by multiple marker genes and transcriptional profiles and collectively contributed to the heterogeneity of major human organs. Moreover, TCR and BCR repertoire comparison and trajectory analyses revealed direct clonal sharing of T and B cells with various developmental states among different tissues. Furthermore, novel cell markers, transcription factors and ligand-receptor pairs were identified with potential functional regulations on maintaining the homeostasis of human cells among tissues. Conclusions: The AHCA reveals the inter- and intra-organ heterogeneity of cell characteristics and provides a useful resource to uncover key events during the development of human diseases such as the recent outbreak of coronavirus disease 2019 (COVID-19) in the context of heterogeneity of cells and organs.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.18.996975v1" />
<meta name="citation_id" content="2020.03.18.996975v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.18.996975v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.18.996975v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.18.996975v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.18.996975" />
<meta name="citation_num_pages" content="55" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.18.996975" />
<meta name="citation_author" content="Shuai He" />
<meta name="citation_author_institution" content="Sun Yat-sen University Cancer Centre, State Key Laboratory of Oncology in South China, Collaborative Innovation Centre for Cancer Medicine" />
<meta name="citation_author_institution" content="Organ Transplant Center, The First Affiliated Hospital, Sun Yat-sen University" />
<meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Organ Donation and Transplant Immunology" />
<meta name="citation_author_institution" content="Guangdong Provincial International Cooperation Base of Science and Technology (Organ Transplantation)" />
<meta name="citation_author" content="Lin-he Wang" />
<meta name="citation_author_institution" content="Organ Transplant Center, The First Affiliated Hospital, Sun Yat-sen University" />
<meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Organ Donation and Transplant Immunology" />
<meta name="citation_author_institution" content="Guangdong Provincial International Cooperation Base of Science and Technology (Organ Transplantation)" />
<meta name="citation_author" content="Yang Liu" />
<meta name="citation_author_institution" content="Sun Yat-sen University Cancer Centre, State Key Laboratory of Oncology in South China, Collaborative Innovation Centre for Cancer Medicine" />
<meta name="citation_author" content="Yi-qi Li" />
<meta name="citation_author_institution" content="Sun Yat-sen University Cancer Centre, State Key Laboratory of Oncology in South China, Collaborative Innovation Centre for Cancer Medicine" />
<meta name="citation_author" content="Haitian Chen" />
<meta name="citation_author_institution" content="Organ Transplant Center, The First Affiliated Hospital, Sun Yat-sen University" />
<meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Organ Donation and Transplant Immunology" />
<meta name="citation_author_institution" content="Guangdong Provincial International Cooperation Base of Science and Technology (Organ Transplantation)" />
<meta name="citation_author" content="Jinghong Xu" />
<meta name="citation_author_institution" content="Organ Transplant Center, The First Affiliated Hospital, Sun Yat-sen University" />
<meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Organ Donation and Transplant Immunology" />
<meta name="citation_author_institution" content="Guangdong Provincial International Cooperation Base of Science and Technology (Organ Transplantation)" />
<meta name="citation_author" content="Wan Peng" />
<meta name="citation_author_institution" content="Sun Yat-sen University Cancer Centre, State Key Laboratory of Oncology in South China, Collaborative Innovation Centre for Cancer Medicine" />
<meta name="citation_author" content="Guo-Wang Lin" />
<meta name="citation_author_institution" content="Sun Yat-sen University Cancer Centre, State Key Laboratory of Oncology in South China, Collaborative Innovation Centre for Cancer Medicine" />
<meta name="citation_author" content="Pan-Pan Wei" />
<meta name="citation_author_institution" content="Sun Yat-sen University Cancer Centre, State Key Laboratory of Oncology in South China, Collaborative Innovation Centre for Cancer Medicine" />
<meta name="citation_author" content="Bo Li" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, Zhongshan School of Medicine, Sun Yat-sen University" />
<meta name="citation_author_institution" content="RNA Biomedical Institute, Sun Yat-sen Memorial Hospital, Sun Yat-sen University" />
<meta name="citation_author" content="Xiaojun Xia" />
<meta name="citation_author_institution" content="Sun Yat-sen University Cancer Centre, State Key Laboratory of Oncology in South China, Collaborative Innovation Centre for Cancer Medicine" />
<meta name="citation_author" content="Dan Wang" />
<meta name="citation_author_institution" content="Sun Yat-sen University Cancer Centre, State Key Laboratory of Oncology in South China, Collaborative Innovation Centre for Cancer Medicine" />
<meta name="citation_author" content="Jin-Xin Bei" />
<meta name="citation_author_institution" content="Sun Yat-sen University Cancer Centre, State Key Laboratory of Oncology in South China, Collaborative Innovation Centre for Cancer Medicine" />
<meta name="citation_author_institution" content="Centre for Precision Medicine, Sun Yat-sen University" />
<meta name="citation_author_email" content="beijx@sysucc.org.cn" />
<meta name="citation_author_email" content="gdtrc@163.com" />
<meta name="citation_author_email" content="rockyucsf1981@126.com" />
<meta name="citation_author" content="Xiaoshun He" />
<meta name="citation_author_institution" content="Organ Transplant Center, The First Affiliated Hospital, Sun Yat-sen University" />
<meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Organ Donation and Transplant Immunology" />
<meta name="citation_author_institution" content="Guangdong Provincial International Cooperation Base of Science and Technology (Organ Transplantation)" />
<meta name="citation_author_email" content="beijx@sysucc.org.cn" />
<meta name="citation_author_email" content="gdtrc@163.com" />
<meta name="citation_author_email" content="rockyucsf1981@126.com" />
<meta name="citation_author" content="Zhiyong Guo" />
<meta name="citation_author_institution" content="Organ Transplant Center, The First Affiliated Hospital, Sun Yat-sen University" />
<meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Organ Donation and Transplant Immunology" />
<meta name="citation_author_institution" content="Guangdong Provincial International Cooperation Base of Science and Technology (Organ Transplantation)" />
<meta name="citation_author_email" content="beijx@sysucc.org.cn" />
<meta name="citation_author_email" content="gdtrc@163.com" />
<meta name="citation_author_email" content="rockyucsf1981@126.com" />
<meta name="citation_reference" content="Dalerba P, Kalisky T, Sahoo D, Rajendran PS, Rothenberg ME, Leyrat AA, Sim S, Okamoto J, Johnston DM, Qian D, et al: Single-cell dissection of transcriptional heterogeneity in human colon tumors. Nat Biotechnol 2011, 29:1120–1127." />
<meta name="citation_reference" content="Klein AM, Mazutis L, Akartuna I, Tallapragada N, Veres A, Li V, Peshkin L, Weitz DA, Kirschner MW: Droplet barcoding for single-cell transcriptomics applied to embryonic stem cells. Cell 2015, 161:1187–1201." />
<meta name="citation_reference" content="Wu AR, Neff NF, Kalisky T, Dalerba P, Treutlein B, Rothenberg ME, Mburu FM, Mantalas GL, Sim S, Clarke MF, Quake SR: Quantitative assessment of single-cell RNA-sequencing methods. Nat Methods 2014, 11:41–46." />
<meta name="citation_reference" content="Qiu X, Hill A, Packer J, Lin D, Ma YA, Trapnell C: Single-cell mRNA quantification and differential analysis with Census. Nat Methods 2017, 14:309–315." />
<meta name="citation_reference" content="Fabre PJ, Leleu M, Mascrez B, Lo Giudice Q, Cobb J, Duboule D: Heterogeneous combinatorial expression of Hoxd genes in single cells during limb development. BMC Biol 2018, 16:101." />
<meta name="citation_reference" content="Butler A, Hoffman P, Smibert P, Papalexi E, Satija R: Integrating single-cell transcríptomic data across different conditions, technologies, and species. Nat Biotechnol 2018, 36:411–420." />
<meta name="citation_reference" content="Haghverdi L, Lun ATL, Morgan MD, Marioni JC: Batch effects in single-cell RNA-sequencing data are corrected by matching mutual nearest neighbors. Nat Biotechnol 2018, 36:421–427." />
<meta name="citation_reference" content="Tabula Muris C, Overall c, Logistical c, Organ c, processing, Library p, sequencing, Computational data a, Cell type a, Writing g, et al: Single-cell transcriptomics of 20 mouse organs creates a Tabula Muris. Nature 2018, 562:367–372." />
<meta name="citation_reference" content="Han X, Wang R, Zhou Y, Fei L, Sun H, Lai S, Saadatpour A, Zhou Z, Chen H, Ye F, et al: Mapping the Mouse Cell Atlas by Microwell-Seq. Cell 2018, 173:1307." />
<meta name="citation_reference" content="Wagner DE, Weinreb C, Collins ZM, Briggs JA, Megason SG, Klein AM: Single-cell mapping of gene expression landscapes and lineage in the zebrafish embryo. Science 2018, 360:981–987." />
<meta name="citation_reference" content="Briggs JA, Weinreb C, Wagner DE, Megason S, Peshkin L, Kirschner MW, Klein AM: The dynamics of gene expression in vertebrate embryogenesis at single-cell resolution. Science 2018,360." />
<meta name="citation_reference" content="Rosenberg AB, Roco CM, Muscat RA, Kuchina A, Sample P, Yao Z, Graybuck LT, Peeler DJ, Mukherjee S, Chen W, et al: Single-cell profiling of the developing mouse brain and spinal cord with split-pool barcoding. Science 2018, 360:176–182." />
<meta name="citation_reference" content="Young MD, Mitchell TJ, Vieira Braga FA, Tran MGB, Stewart BJ, Ferdinand JR, Collord G, Botting RA, Popescu DM, Loudon KW, et al: Single-cell transcriptomes from human kidneys reveal the cellular identity of renal tumors. Science 2018, 361:594–599." />
<meta name="citation_reference" content="Schiller HB, Montoro DT, Simon LM, Rawlins EL, Meyer KB, Strunz M, Vieira Braga FA, Timens W, Koppelman GH, Budinger GRS, et al: The Human Lung Cell Atlas: A High-Resolution Reference Map of the Human Lung in Health and Disease. Am J Respir Cell MoiBiol 2019, 61:31–41." />
<meta name="citation_reference" content="Cheng JB, Sedgewick AJ, Finnegan Al, Harirchian P, Lee J, Kwon S, Fassett MS, Golovato J, Gray M, Ghadially R, et al: Transcriptional Programming of Normal and Inflamed Human Epidermis at Single-Cell Resolution. Cell Rep 2018, 25:871–883." />
<meta name="citation_reference" content="Plasschaert LW, Zilionis R, Choo-Wing R, Savova V, Knehr J, Roma G, Klein AM, Jaffe AB: A single-cell atlas of the airway epithelium reveals the CFTR-rich pulmonary ionocyte. Nature 2018, 560:377–381." />
<meta name="citation_reference" content="Giladi A, Paul F, Herzog Y, Lubling Y, Weiner A, Yofe I, Jaitin D, Cabezas-Wallscheid N, Dress R, Ginhoux F, et al: Single-cell characterization of haematopoietic progenitors and their trajectories in homeostasis and perturbed haematopoiesis. Nat Cell Biol 2018, 20:836–846." />
<meta name="citation_reference" content="Vento-Tormo R, Efremova M, Botting RA, Turco MY, Vento-Tormo M, Meyer KB, Park JE, Stephenson E, Polanski K, Goncalves A, et al: Single-cell reconstruction of the early matemal-fetal interface in humans. Nature 2018, 563:347–353." />
<meta name="citation_reference" content="Cao J, Spielmann M, Qiu X, Huang X, Ibrahim DM, Hill AJ, Zhang F, Mundlos S, Christiansen L, Steemers FJ, et al: The single-cell transcriptional landscape of mammalian organogenesis. Nature 2019, 566:496–502." />
<meta name="citation_reference" content="Zhang X, Lan Y, Xu J, Quan F, Zhao E, Deng C, Luo T, Xu L, Liao G, Yan M, et al: CellMarker: a manually curated resource of cell markers in human and mouse. Nucleic Acids Res 2019, 47:D721–d728." />
<meta name="citation_reference" content="Mizumoto N, Takashima A: CD1a and langerin: acting as more than Langerhans cell markers. J Clin Invest 2004, 113:658–660." />
<meta name="citation_reference" content="Lacroix M: Significance, detection and markers of disseminated breast cancer cells. Endocr Relat Cancer 2006, 13:1033–1067." />
<meta name="citation_reference" content="Wong MT, Ong DE, Lim FS, Teng KW, McGovern N, Narayanan S, Ho WQ, Cerny D, Tan HK, Anicete R, et al: A High-Dimensional Atlas of Human T Cell Diversity Reveals Tissue-Specific Trafficking and Cytokine Signatures. Immunity 2016, 45:442–456." />
<meta name="citation_reference" content="Zhang L, Yu X, Zheng L, Zhang Y, Li Y, Fang Q, Gao R, Kang B, Zhang Q, Huang JY, et al: Lineage tracking reveals dynamic relationships of T cells in colorectal cancer. Nature 2018, 564:268–272." />
<meta name="citation_reference" content="Rothenberg EV: The chromatin landscape and transcription factors in T cell programming. Trends Immunol 2014, 35:195–204." />
<meta name="citation_reference" content="Vaeth M, Gogishvili T, Bopp T, Klein M, Berberich-Siebelt F, Gattenloehner S, Avots A, Sparwasser T, Grebe N, Schmitt E, et al: Regulatory T cells facilitate the nuclear accumulation of inducible cAMP early repressor (ICER) and suppress nuclear factor of activated T cell c1 (NFATc1). Proc Natl Acad Sci USA 2011, 108:2480–2485." />
<meta name="citation_reference" content="Willinger T, Freeman T, Herbert M, Hasegawa H, McMichael AJ, Callan MF: Human naive CD8 T cells down-regulate expression of the WNT pathway transcription factors lymphoid enhancer binding factor 1 and transcription factor 7 (T cell factor-1) following antigen encounter in vitro and in vivo. J Immunol 2006, 176:1439–1446." />
<meta name="citation_reference" content="Sebzda E, Zou Z, Lee JS, Wang T, Kahn ML: Transcription factor KLF2 regulates the migration of naive T cells by restricting chemokine receptor expression patterns. Nat Immunol 2008, 9:292–300." />
<meta name="citation_reference" content="Karwacz K, Miraldi ER, Pokrovskii M, Madi A, Yosef N, Wortman I, Chen X, Watters A, Carriero N, Awasthi A, et al: Critical role of IRF1 and BATF in forming chromatin landscape during type 1 regulatory cell differentiation. Nat Immunol 2017, 18:412–421." />
<meta name="citation_reference" content="Wang D, Diao H, Getzler AJ, Rogal W, Frederick MA, Milner J, Yu B, Crotty S, Goldrath AW, Pipkin ME: The Transcription Factor Runx3 Establishes Chromatin Accessibility of cis-Regulatory Landscapes that Drive Memory Cytotoxic T Lymphocyte Formation. Immunity 2018, 48:659–674.e656." />
<meta name="citation_reference" content="Agematsu K, Hokibara S, Nagumo H, Komiyama A: CD27: a memory B-cell marker. Immunol Today 2000, 21:204–206." />
<meta name="citation_reference" content="Herling M, Patel KA, Weit N, Lilienthal N, Hallek M, Keating MJ, Jones D: High TCL1 levels are a marker of B-cell receptor pathway responsiveness and adverse outcome in chronic lymphocytic leukemia. Blood 2009, 114:4675–4686." />
<meta name="citation_reference" content="Yoon HS, Scharer CD, Majumder P, Davis CW, Butler R, Zinzow-Kramer W, Skountzou I, Koutsonanos DG, Ahmed R, Boss JM: ZBTB32 is an early repressor of the CIITA and MHC class II gene expression during B cell differentiation to plasma cells. Journal of immunology (Baltimore, Md: 1950) 2012, 189:2393–2403." />
<meta name="citation_reference" content="Shinnakasu R, Kurosaki T: Regulation of memory B and plasma cell differentiation. Curr Opin Immunol 2017, 45:126–131." />
<meta name="citation_reference" content="Fernandez D, Ortiz M, Rodriguez L, Garcia A, Martinez D, Moreno de Alboran I: The proto-oncogene c-myc regulates antibody secretion and Ig class switch recombination. J Immunol 2013, 190:6135–6144." />
<meta name="citation_reference" content="Roy K, Mitchell S, Liu Y, Ohta S, Lin YS, Metzig MO, Nutt SL, Hoffmann A: A Regulatory Circuit Controlling the Dynamics of NFkappaB cRel Transitions B Cells from Proliferation to Plasma Cell Differentiation. Immunity 2019, 50:616–628 e616." />
<meta name="citation_reference" content="Valledor AF, Borràs FE, Cullell-Young M, Celada A: Transcription factors that regulate monocyte/macrophage differentiation. J Leukoc Biol 1998, 63:405–417." />
<meta name="citation_reference" content="Kueh HY, Champhekar A, Champhekhar A, Nutt SL, Elowitz MB, Rothenberg EV: Positive feedback between PU.1 and the cell cycle controls myeloid differentiation. Science 2013, 341:670–673." />
<meta name="citation_reference" content="Davies LC, Jenkins SJ, Allen JE, Taylor PR: Tissue-resident macrophages. Nature Immunology 2013, 14:986." />
<meta name="citation_reference" content="Roche PA, Furuta K: The ins and outs of MHC class ll-mediated antigen processing and presentation. Nat Rev lmmunol 2015, 15:203–216." />
<meta name="citation_reference" content="Amatschek S, Kriehuber E, Bauer W, Reininger B, Meraner P, Wolpl A, Schweifer N, Haslinger C, Stingl G, Maurer D: Blood and lymphatic endothelial cell-specific differentiation programs are stringently controlled by the tissue environment. Blood 2007, 109:4777–4785." />
<meta name="citation_reference" content="Mueller SN, Germain RN: Stromal cell contributions to the homeostasis and functionality of the immune system. Nat Rev Immunol 2009, 9:618–629." />
<meta name="citation_reference" content="Xu X, Chen P, Wang J, Feng J, Zhou H, Li X, Zhong W, Hao P: Evolution of the novel coronavirus from the ongoing Wuhan outbreak and modeling of its spike protein for risk of human transmission. Sci China Life Sci 2020." />
<meta name="citation_reference" content="Lu R, Zhao X, Li J, Niu P, Yang B, Wu H, Wang W, Song H, Huang B, Zhu N, et al: Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding. Lancet 2020." />
<meta name="citation_reference" content="Xin Zou KC, Jiawei Zou, Peiyi Han, Jie Hao, Zeguang Han: The single-cell RNA-seq data analysis on the receptor ACE2 expression reveals the potential risk of different human organs vulnerable to Wuhan 2019-nCoV infection. Front Med:O-." />
<meta name="citation_reference" content="Hamming I, Timens W, Bulthuis ML, Lely AT, Navis G, van Goor H: Tissue distribution of ACE2 protein, the functional receptor for SARS coronavirus. A first step in understanding SARS pathogenesis. J Pathol 2004, 203:631–637." />
<meta name="citation_reference" content="Crackower MA, Sarao R, Oudit GY, Yagil C, Kozieradzki I, Scanga SE, Oliveira-dos-Santos AJ, da Costa J, Zhang L, Pei Y, et al: Angiotensin-converting enzyme 2 is an essential regulator of heart function. Nature 2002, 417:822–828." />
<meta name="citation_reference" content="Kowalczuk S, Broer A, Tietze N, Vanslambrouck JM, Rasko JE, Broer S: A protein complex in the brush-border membrane explains a Hartnup disorder allele. FASEB J 2008, 22:2880–2887." />
<meta name="citation_reference" content="Harmer D, Gilbert M, Borman R, Clark KL: Quantitative mRNA expression profiling of ACE 2, a novel homologue of angiotensin converting enzyme. FEBS Lett 2002, 532:107–110." />
<meta name="citation_reference" content="Bach JP, Deuster O, Balzer-Geldsetzer M, Meyer B, Dodel R, Bacher M: The role of macrophage inhibitory factor in tumorigenesis and central nervous system tumors. Cancer 2009, 115:2031–2040." />
<meta name="citation_reference" content="Liu YH, Lin CY, Lin WC, Tang SW, Lai MK, Lin JY: Up-regulation of vascular endothelial growth factor-D expression in clear cell renal cell carcinoma by CD74: a critical role in cancer cell tumorigenesis. J Immunol 2008, 181:6584–6594." />
<meta name="citation_reference" content="Calame DG, Mueller-Ortiz SL, Morales JE, Wetsel RA: The C5a anaphylatoxin receptor (C5aR1) protects against Listeria monocytogenes infection by inhibiting type 1 IFN expression. J lmmunol 2014, 193:5099–5107." />
<meta name="citation_reference" content="Stein R, Mattes MJ, Cardillo TM, Hansen HJ, Chang CH, Burton J, Govindan S, Goldenberg DM: CD74: a new candidate target for the immunotherapy of B-cell neoplasms. Clin Cancer Res 2007, 13:5556s–5563s." />
<meta name="citation_reference" content="Long EO: ICAM-1: getting a grip on leukocyte adhesion. J Immunol 2011, 186:5021–5023." />
<meta name="citation_reference" content="Osei-Owusu P, Charlton TM, Kim HK, Missiakas D, Schneewind O: FPR1 is the plague receptor on host immune cells. Nature 2019, 574:57–62." />
<meta name="citation_reference" content="Contento RL, Molon B, Boularan C, Pozzan T, Manes S, Marullo S, Viola A: CXCR4-CCR5: a couple modulating T cell functions. Proc Natl Acad Sci U S A 2008, 105:10101–10106." />
<meta name="citation_reference" content="Thome JJ, Yudanin N, Ohmura Y, Kubota M, Grinshpun B, Sathaliyawala T, Kato T, Lerner H, Shen Y, Farber DL: Spatial map of human T cell compartmentalization and maintenance over decades of life. Cell 2014, 159:814–828." />
<meta name="citation_reference" content="Schenkel JM, Masopust D: Tissue-resident memory T cells. Immunity 2014, 41:886–897." />
<meta name="citation_reference" content="Qian BZ, Li J, Zhang H, Kitamura T, Zhang J, Campion LR, Kaiser EA, Snyder LA, Pollard JW: CCL2 recruits inflammatory monocytes to facilitate breast-tumour metastasis. Nature 2011, 475:222–225." />
<meta name="citation_reference" content="Yona S, Kim KW, Wolf Y, Mildner A, Varol D, Breker M, Strauss-Ayali D, Viukov S, Guilliams M, Misharin A, et al: Fate mapping reveals origins and dynamics of monocytes and tissue macrophages under homeostasis. Immunity 2013, 38:79–91." />
<meta name="citation_reference" content="Hume DA, Ross IL, Himes SR, Sasmono RT, Wells CA, Ravasi T: The mononuclear phagocyte system revisited. J Leukoc Biol 2002, 72:621–627." />
<meta name="citation_reference" content="Hume DA: The mononuclear phagocyte system. Curr Opin lmmunol 2006, 18:49–53." />
<meta name="citation_reference" content="Jakubzick C, Gautier EL, Gibbings SL, Sojka DK, Schlitzer A, Johnson TE, Ivanov S, Duan Q, Bala S, Condon T, et al: Minimal differentiation of classical monocytes as they survey steady-state tissues and transport antigen to lymph nodes. Immunity 2013, 39:599–610." />
<meta name="citation_reference" content="Hashimoto D, Chow A, Noizat C, Teo P, Beasley MB, Leboeuf M, Becker CD, See P, Price J, Lucas D, et al: Tissue-resident macrophages self-maintain locally throughout adult life with minimal contribution from circulating monocytes. Immunity 2013, 38:792–804." />
<meta name="citation_reference" content="Bryant DM, Mostov KE: From cells to organs: building polarized tissue. Nat Rev Mol Cell Biol 2008, 9:887–901." />
<meta name="citation_reference" content="Macara IG, Guyer R, Richardson G, Huo Y, Ahmed SM: Epithelial homeostasis. Curr Biol 2014, 24:R815–825." />
<meta name="citation_reference" content="Lynch MD, Watt FM: Fibroblast heterogeneity: implications for human disease. J clin Invest 2018, 128:26–35." />
<meta name="citation_reference" content="Eckers A, Haendeler J: Endothelial cells in health and disease. Antioxid Redox Signal 2015, 22:1209–1211." />
<meta name="citation_reference" content="Haber AL, Biton M, Rogel N, Herbst RH, Shekhar K, Smillie C, Burgin G, Delorey TM, Howitt MR, Katz Y, et al: A single-cell survey of the small intestinal epithelium. Nature 2017, 551:333–339." />
<meta name="citation_reference" content="Nguyen QH, Pervolarakis N, Blake K, Ma D, Davis RT, James N, Phung AT, Willey E, Kumar R, Jabart E, et al: Profiling human breast epithelial cells using single cell RNA sequencing identifies cell diversity. Nature Communications 2018, 9:2028." />
<meta name="citation_reference" content="Dong J, Hu Y, Fan X, Wu X, Mao Y, Hu B, Guo H, Wen L, Tang F: Single-cell RNA-seq analysis unveils a prevalent epithelial/mesenchymal hybrid state during mouse organogenesis. Genome Biol 2018, 19:31." />
<meta name="citation_reference" content="Lee TI, Young RA: Transcriptional regulation and its misregulation in disease. Cell 2013, 152:1237–1251." />
<meta name="citation_reference" content="Singh H, Khan AA, Dinner AR: Gene regulatory networks in the immune system. Trends Immunol 2014, 35:211–218." />
<meta name="citation_reference" content="Jiang S, Shi Z, Shu Y, Song J, Gao GF, Tan W, Guo D: A distinct name is needed for the new coronavirus. Lancet 2020." />
<meta name="citation_reference" content="Wang D, Hu B, Hu C, Zhu F, Liu X, Zhang J, Wang B, Xiang H, Cheng Z, Xiong Y, et al: Clinical Characteristics of 138 Hospitalized Patients With 2019 Novel Coronavirus-lnfected Pneumonia in Wuhan, China. JAMA 2020." />
<meta name="citation_reference" content="Chen N, Zhou M, Dong X, Qu J, Gong F, Han Y, Qiu Y, Wang J, Liu Y, Wei Y, et al: Epidemiological and clinical characteristics of 99 cases of 2019 novel coronavirus pneumonia in Wuhan, China: a descriptive study. Lancet 2020, 395:507–513." />
<meta name="citation_reference" content="Dobin A, Davis CA, Schlesinger F, Drenkow J, Zaleski C, Jha S, Batut P, Chaisson M, Gingeras TR: STAR: ultrafast universal RNA-seq aligner. Bioinformatics 2013, 29:15–21." />
<meta name="citation_reference" content="Whitfield ML, George LK, Grant GD, Perou CM: Common markers of proliferation. Nat Rev Cancer 2006, 6:99–106." />
<meta name="citation_reference" content="Zhou Y, Zhou B, Pache L, Chang M, Khodabakhshi AH, Tanaseichuk O, Benner C, Chanda SK: Metascape provides a biologist-oriented resource for the analysis of systems-level datasets. Nat Commun 2019, 10:1523." />
<meta name="citation_reference" content="Aibar S, Gonzalez-Blas CB, Moerman T, Huynh-Thu VA, Imrichova H, Hulselmans G, Rambow F, Marine JC, Geurts P, Aerts J, et al: SCENIC: single-cell regulatory network inference and clustering. Nat Methods 2017, 14:1083–1086." />
<meta name="twitter:title" content="Single-cell transcriptome profiling an adult human cell atlas of 15 major organs" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Background As core units of organ tissues, cells of various types play their harmonious rhythms to maintain the homeostasis of human body. It is essential to identify characteristics of the cells in human organs and their regulatory networks for understanding biological mechanisms related to health and disease. However, a systematic and comprehensive single-cell transcriptional profile across multiple organs of normal human adults has been pending.

Results we performed single-cell transcriptomes of 88,622 cells derived from 15 tissue organs of one adult donor and generated an adult human cell atlas (AHCA). The AHCA depicted 234 subtypes of cells, including major cell types such as T, B, myeloid, epithelial, and stromal cells, as well as novel cell types in skin, each of which was distinguished by multiple marker genes and transcriptional profiles and collectively contributed to the heterogeneity of major human organs. Moreover, TCR and BCR repertoire comparison and trajectory analyses revealed direct clonal sharing of T and B cells with various developmental states among different tissues. Furthermore, novel cell markers, transcription factors and ligand-receptor pairs were identified with potential functional regulations on maintaining the homeostasis of human cells among tissues.

Conclusions The AHCA reveals the inter- and intra-organ heterogeneity of cell characteristics and provides a useful resource to uncover key events during the development of human diseases such as the recent outbreak of coronavirus disease 2019 (COVID-19) in the context of heterogeneity of cells and organs." />
<meta name="og-title" property="og:title" content="Single-cell transcriptome profiling an adult human cell atlas of 15 major organs" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.18.996975v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Background As core units of organ tissues, cells of various types play their harmonious rhythms to maintain the homeostasis of human body. It is essential to identify characteristics of the cells in human organs and their regulatory networks for understanding biological mechanisms related to health and disease. However, a systematic and comprehensive single-cell transcriptional profile across multiple organs of normal human adults has been pending.

Results we performed single-cell transcriptomes of 88,622 cells derived from 15 tissue organs of one adult donor and generated an adult human cell atlas (AHCA). The AHCA depicted 234 subtypes of cells, including major cell types such as T, B, myeloid, epithelial, and stromal cells, as well as novel cell types in skin, each of which was distinguished by multiple marker genes and transcriptional profiles and collectively contributed to the heterogeneity of major human organs. Moreover, TCR and BCR repertoire comparison and trajectory analyses revealed direct clonal sharing of T and B cells with various developmental states among different tissues. Furthermore, novel cell markers, transcription factors and ligand-receptor pairs were identified with potential functional regulations on maintaining the homeostasis of human cells among tissues.

Conclusions The AHCA reveals the inter- and intra-organ heterogeneity of cell characteristics and provides a useful resource to uncover key events during the development of human diseases such as the recent outbreak of coronavirus disease 2019 (COVID-19) in the context of heterogeneity of cells and organs." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-20" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.18.996975v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.18.996975v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.18.996975v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.18.996975v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1203872" />
    <title>Single-cell transcriptome profiling an adult human cell atlas of 15 major organs | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"wFLvxeqsXOcJAkZ_8FW9j7ULuET0xqCwC8fKq4bPC0M","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1203872","apath":"\/biorxiv\/early\/2020\/03\/20\/2020.03.18.996975.atom","pisa":"biorxiv;2020.03.18.996975v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.18.996975v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.18.996975v1","title":"Single-cell transcriptome profiling an adult human cell atlas of 15 major organs","identifier":"node\/1203872"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022HI7HxYsnVAYIYh0cMZpVdzw85qC9hsNCdhuprAJse87\\\\\\\/kNsJ06g841ONeBfgyR5hewqwkPuGkE\\\\\\\/wjp3lnHp7\\\\\\\/pfUFSzYkSZ9mXLJJpTF2gTpfag7+J8MXbO3JE65Z3ZSIeoz8\\\\\\\/IpG5ue3LgA\\\\\\\/LKaUKYK5\\\\\\\/JG1V8oFtN\\\\\\\/B7anFacWcOcMLUe0ePQddy6V\\\\\\\/sR4Af0u64Z8EafmZ1kVMEFd0aYrQ5pZwW4XwY9OFLiTSCUVn+pF0mpx4rtAU+sd1CbspKYLNneqhB+1xZfM3Tgm+qr4Bu+Qtpek\\\\\\\/2ralYOoJDkL1yZbd744msa8EE50IY+QJ6e5\\\\\\\/cffjigVLgBUbeJ22hje2zoKD0yCvfiybSsuqHAPhx5wL0z7JA8tZBd5CS+8v49AUK2XKJTKvHV6q88+8\\\\\\\/Y8yMv4KG4vNu8p7yB0H6T5eMVf8rqfwOEJf9ZlND93MGY7GUFQhT4LQo4WLXc2c7Vvpf\\\\\\\/LvLZqQNNQKUEylSXFrCVQNzFqoWPEE\\\\\\\/rNJOoNZLCQ2ZNvG0VbLTeCpofKsCJZn5skIV2NnDSicbMiS10n2coqfypHKAAkDcT9p\\\\\\\/25BA7ESk7vPE3Hw\\\\\\\/NHrNcd0JTsAXPIqcdFFvDTQB9CN9k3AcCE4kSHdsCYdHh6w4QrsHLFUvEHkFTMqgxVTj49RdbFvFFv5Ugpyq9wwCMxb+AHthx8qgAeaMq0qDtPOMMWvFoXj1ZanNrvKJy00AUEFb5GhG32sta0uO4ysEp5Rh5T0+5fwBY\\\\\\\/FVtrFMbObpI4lg7J1nPsVZKvcxO4zE88BwzUhyE485botnPLkO9Gpq3Efzrex+aDmgpb+kawTv3WcnqWhMkWAsD7wfco7GQBkElReS0FfCuIAkN+2qA4vjPYKCIXV5Gg6x5OxTtDEI1lpqnV+APUu0eFOcAe9iWXkBYR5OZzH+zrDiSstCssi9o3A2p1yErR8YAAF9T1HcoqY7LIYCLDxk8ihcph3\\\\\\\/LAPNkY96cMaZbiORMhi1XufNvRCs4su5vr5q2WQPVNNxiuQbR4WsnQBwPu4sVF9dBf9pjAU9boblhhNNDdSvQ5X7MbPYuWUnhURnFofYexezNFvhRBApqPmkpg3sv5XENodf4YJbAIlC791W\\\\\\\/RmKYMszTq5H2P+YHXs3e5chKswWnXfOwPOigUr8IAFLAKka3smNnSgqwje0jhTwSDT0nsr1LbmBSVU1n6emMMJX5F1ti7OOZSCocskAqzjMnbrwkRxK5dNp2VQd+bS3VJzv5rCHOJ\\\\\\\/WIxD4HveOzsEJY\\\\\\\/+nXgmK5ZyerOlWTz6B51coqZNL3hhjpw5Z2caRAi8GQ9h3BpsuieqGRcdv0G\\\\\\\/oYKHD7p\\\u0022,\\\u0022iv\\\u0022:\\\u0022gknwEHVZfhnfU\\\\\\\/uMTdTGUg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022e977216b60ce0bd01f3b03a1047acb97050bcc1a73a395b0f2b91a41c4bb5b50\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.18.996975v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1203872 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.18.996975v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Ggwf48HnoRMB1HqukYjoiELQ4fzkAZFPTFhTwmaxYzw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1203872" id="node1203872" data-pisa="biorxiv;2020.03.18.996975v1" data-pisa-master="biorxiv;2020.03.18.996975" data-apath="/biorxiv/early/2020/03/20/2020.03.18.996975.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Single-cell transcriptome profiling an adult human cell atlas of 15 major organs</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Shuai</span> <span class="nlm-surname">He</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Lin-he</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Yang</span> <span class="nlm-surname">Liu</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Yi-qi</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Haitian</span> <span class="nlm-surname">Chen</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Jinghong</span> <span class="nlm-surname">Xu</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Wan</span> <span class="nlm-surname">Peng</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Guo-Wang</span> <span class="nlm-surname">Lin</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Pan-Pan</span> <span class="nlm-surname">Wei</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Bo</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Xiaojun</span> <span class="nlm-surname">Xia</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Dan</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Jin-Xin</span> <span class="nlm-surname">Bei</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Xiaoshun</span> <span class="nlm-surname">He</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Zhiyong</span> <span class="nlm-surname">Guo</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.18.996975 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1203872" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Shuai He </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Sun Yat-sen University Cancer Centre, State Key Laboratory of Oncology in South China, Collaborative Innovation Centre for Cancer Medicine</span>, Guangzhou 510060, <span class='nlm-country'>P. R. China</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Organ Transplant Center, The First Affiliated Hospital, Sun Yat-sen University</span>, Guangzhou 510080, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Guangdong Provincial Key Laboratory of Organ Donation and Transplant Immunology</span>, Guangzhou 510080, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Guangdong Provincial International Cooperation Base of Science and Technology (Organ Transplantation)</span>, Guangzhou 510080, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shuai%2BHe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=He%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShuai%2BHe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lin-he Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Organ Transplant Center, The First Affiliated Hospital, Sun Yat-sen University</span>, Guangzhou 510080, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Guangdong Provincial Key Laboratory of Organ Donation and Transplant Immunology</span>, Guangzhou 510080, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Guangdong Provincial International Cooperation Base of Science and Technology (Organ Transplantation)</span>, Guangzhou 510080, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lin-he%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALin-he%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yang Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Sun Yat-sen University Cancer Centre, State Key Laboratory of Oncology in South China, Collaborative Innovation Centre for Cancer Medicine</span>, Guangzhou 510060, <span class='nlm-country'>P. R. China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yang%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYang%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Yi-qi Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Sun Yat-sen University Cancer Centre, State Key Laboratory of Oncology in South China, Collaborative Innovation Centre for Cancer Medicine</span>, Guangzhou 510060, <span class='nlm-country'>P. R. China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yi-qi%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYi-qi%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Haitian Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Organ Transplant Center, The First Affiliated Hospital, Sun Yat-sen University</span>, Guangzhou 510080, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Guangdong Provincial Key Laboratory of Organ Donation and Transplant Immunology</span>, Guangzhou 510080, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Guangdong Provincial International Cooperation Base of Science and Technology (Organ Transplantation)</span>, Guangzhou 510080, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haitian%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHaitian%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jinghong Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Organ Transplant Center, The First Affiliated Hospital, Sun Yat-sen University</span>, Guangzhou 510080, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Guangdong Provincial Key Laboratory of Organ Donation and Transplant Immunology</span>, Guangzhou 510080, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Guangdong Provincial International Cooperation Base of Science and Technology (Organ Transplantation)</span>, Guangzhou 510080, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jinghong%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJinghong%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Wan Peng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Sun Yat-sen University Cancer Centre, State Key Laboratory of Oncology in South China, Collaborative Innovation Centre for Cancer Medicine</span>, Guangzhou 510060, <span class='nlm-country'>P. R. China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wan%2BPeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peng%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWan%2BPeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Guo-Wang Lin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Sun Yat-sen University Cancer Centre, State Key Laboratory of Oncology in South China, Collaborative Innovation Centre for Cancer Medicine</span>, Guangzhou 510060, <span class='nlm-country'>P. R. China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guo-Wang%2BLin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lin%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuo-Wang%2BLin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Pan-Pan Wei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Sun Yat-sen University Cancer Centre, State Key Laboratory of Oncology in South China, Collaborative Innovation Centre for Cancer Medicine</span>, Guangzhou 510060, <span class='nlm-country'>P. R. China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pan-Pan%2BWei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wei%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APan-Pan%2BWei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Bo Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Department of Biochemistry and Molecular Biology, Zhongshan School of Medicine, Sun Yat-sen University</span>, Guangzhou 510080, <span class='nlm-country'>P. R. China</span></div><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>RNA Biomedical Institute, Sun Yat-sen Memorial Hospital, Sun Yat-sen University</span>, Guangzhou 510120, <span class='nlm-country'>P. R. China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bo%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABo%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Xiaojun Xia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Sun Yat-sen University Cancer Centre, State Key Laboratory of Oncology in South China, Collaborative Innovation Centre for Cancer Medicine</span>, Guangzhou 510060, <span class='nlm-country'>P. R. China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaojun%2BXia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xia%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaojun%2BXia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Dan Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Sun Yat-sen University Cancer Centre, State Key Laboratory of Oncology in South China, Collaborative Innovation Centre for Cancer Medicine</span>, Guangzhou 510060, <span class='nlm-country'>P. R. China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dan%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADan%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Jin-Xin Bei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Sun Yat-sen University Cancer Centre, State Key Laboratory of Oncology in South China, Collaborative Innovation Centre for Cancer Medicine</span>, Guangzhou 510060, <span class='nlm-country'>P. R. China</span></div><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>Centre for Precision Medicine, Sun Yat-sen University</span>, Guangzhou 510080, <span class='nlm-country'>P. R. China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jin-Xin%2BBei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bei%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJin-Xin%2BBei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:beijx@sysucc.org.cn" class="" data-icon-position="" data-hide-link-title="0">beijx@sysucc.org.cn</a>
<a href="mailto:gdtrc@163.com" class="" data-icon-position="" data-hide-link-title="0">gdtrc@163.com</a>
<a href="mailto:rockyucsf1981@126.com" class="" data-icon-position="" data-hide-link-title="0">rockyucsf1981@126.com</a></span></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Xiaoshun He </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Organ Transplant Center, The First Affiliated Hospital, Sun Yat-sen University</span>, Guangzhou 510080, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Guangdong Provincial Key Laboratory of Organ Donation and Transplant Immunology</span>, Guangzhou 510080, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Guangdong Provincial International Cooperation Base of Science and Technology (Organ Transplantation)</span>, Guangzhou 510080, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaoshun%2BHe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=He%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXiaoshun%2BHe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:beijx@sysucc.org.cn" class="" data-icon-position="" data-hide-link-title="0">beijx@sysucc.org.cn</a>
<a href="mailto:gdtrc@163.com" class="" data-icon-position="" data-hide-link-title="0">gdtrc@163.com</a>
<a href="mailto:rockyucsf1981@126.com" class="" data-icon-position="" data-hide-link-title="0">rockyucsf1981@126.com</a></span></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Zhiyong Guo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Organ Transplant Center, The First Affiliated Hospital, Sun Yat-sen University</span>, Guangzhou 510080, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Guangdong Provincial Key Laboratory of Organ Donation and Transplant Immunology</span>, Guangzhou 510080, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Guangdong Provincial International Cooperation Base of Science and Technology (Organ Transplantation)</span>, Guangzhou 510080, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhiyong%2BGuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guo%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhiyong%2BGuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:beijx@sysucc.org.cn" class="" data-icon-position="" data-hide-link-title="0">beijx@sysucc.org.cn</a>
<a href="mailto:gdtrc@163.com" class="" data-icon-position="" data-hide-link-title="0">gdtrc@163.com</a>
<a href="mailto:rockyucsf1981@126.com" class="" data-icon-position="" data-hide-link-title="0">rockyucsf1981@126.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.18.996975v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1203872" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1203872/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.18.996975v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1203872" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1203872/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.18.996975v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1203872" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1203872/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.18.996975v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1203872" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1203872/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.18.996975v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1203872" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1203872/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.18.996975v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1203872" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1203872/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">ABSTRACT</h2><div id="sec-1" class="subsection"><p id="p-4"><strong>Background</strong> As core units of organ tissues, cells of various types play their harmonious rhythms to maintain the homeostasis of human body. It is essential to identify characteristics of the cells in human organs and their regulatory networks for understanding biological mechanisms related to health and disease. However, a systematic and comprehensive single-cell transcriptional profile across multiple organs of normal human adults has been pending.</p></div><div id="sec-2" class="subsection"><p id="p-5"><strong>Results</strong> we performed single-cell transcriptomes of 88,622 cells derived from 15 tissue organs of one adult donor and generated an adult human cell atlas (AHCA). The AHCA depicted 234 subtypes of cells, including major cell types such as T, B, myeloid, epithelial, and stromal cells, as well as novel cell types in skin, each of which was distinguished by multiple marker genes and transcriptional profiles and collectively contributed to the heterogeneity of major human organs. Moreover, TCR and BCR repertoire comparison and trajectory analyses revealed direct clonal sharing of T and B cells with various developmental states among different tissues. Furthermore, novel cell markers, transcription factors and ligand-receptor pairs were identified with potential functional regulations on maintaining the homeostasis of human cells among tissues.</p></div><div id="sec-3" class="subsection"><p id="p-6"><strong>Conclusions</strong> The AHCA reveals the inter- and intra-organ heterogeneity of cell characteristics and provides a useful resource to uncover key events during the development of human diseases such as the recent outbreak of coronavirus disease 2019 (COVID-19) in the context of heterogeneity of cells and organs.</p></div></div><div class="section" id="sec-4"><h2 class="">INTRODUCTION</h2><p id="p-7">Human body consists of multiple organs, where cells of different types are the core units of structure and function. Like instruments from different families in a symphony orchestra, cells and organs play their harmonious rhythms to maintain the homeostasis of human body. Perturbation of the homeostasis leads to various pathological conditions. Therefore, it is essential to identify characteristics of the cells in human organs and their regulatory networks for understanding biological mechanisms related to health and disease.</p><p id="p-8">Recent technological innovations in the single-cell transcriptional profiling using single-cell RNA sequencing (scRNA-seq) provide a promising strategy to quantify gene expression at the genome-wide level in thousands of individual cells simultaneously[<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>–<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>], which have expanded our knowledge regarding cellular heterogeneity and networks as well as development in human tissues and organs at single-cell resolution[<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>–<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>]. Previous studies have demonstrated cell composition for many tissues of human and mouse, such as brain[<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>], kidney[<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>], lung[<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>], skin[<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>]. The strategy empowers the identification of novel cell types. A cell type marked by cystic fibrosis transmembrane conductance regulator (<em>CFTR</em>) was identified in the lung of human and mouse, which has ability to regulate luminal pH and was implicated in the pathogenesis of cystic fibrosis[<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>]. Non-genetic cellular heterogeneity has been revealed in hematopoietic progenitor cells and keratinocytes, which play important roles in maintaining hematopoiesis[<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>] and compartmentalizing crucial molecular activities in human epidermis[<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>], respectively. For the development of human embryo, transcriptome analysis of about 70,000 single cells from the first-trimester placentas with matched maternal blood and decidual cells uncover the cellular organization of decidua and placenta, as well as distinctive immunomodulatory and chemokine profiles of decidual nature killer (NK) cells[<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>]. In addition, the single-cell transcriptional profiles of embryonic and adult organs in mice have been reported, which reveal the landscape of organogenesis and cellular heterogeneity in organs[<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>]. However, a systematic and comprehensive single-cell transcriptional profile across multiple organs of normal human adults has been pending. Previous studies with scRNA-seq on human samples were mostly restricted to a few specific organs with disease conditions and did not attempt to characterize the heterogeneity and connexity among multiple organs in the same individuals.</p><p id="p-9">Here we aimed to investigate the transcriptional heterogeneity and interactions of cells from human adult organs at single-cell resolution. Using single-cell RNA sequencing, we profiled the transcriptomes of more than 88,000 cells of 15 organs from one individual donor. Comprehensive comparisons within and across tissues for distinct cell types were performed to reveal the complexity of inter-cells in gene profiles, active transcription factors and potential biological functions. The resulting high-resolution adult human cell atlas (AHCA) provides a global view of various cell populations and connexity in human body, and also a useful resource to investigate the biology of normal human cells and the development of diseases affecting different organs such as the distribution of potential viral receptor for the most recent outbreak of coronavirus disease 2019 (COVID-19).</p></div><div class="section" id="sec-5"><h2 class="">RESULTS</h2><div id="sec-6" class="subsection"><h3>Global view of single cell RNA sequencing of 15 organ samples</h3><p id="p-10">Viable single cells were prepared from tissue samples of 15 different organs of a research-consented adult donor (<strong><a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1A</a> and Additional file 19: Table S1</strong>). mRNA transcripts from each sample were ligated with barcoded indexes at 5’-end and reverse transcribed into cDNA, using GemCode technology (10X Genomics, USA). cDNA libraries including the enriched fragments spanning the full-length V(D)J segments of T cell receptors (TCR) or B cell receptor (BCR), and 5’-end fragments for gene expression were separately constructed, which were subsequently subjected for high-throughput sequencing. In average, we obtained more than 400 million sequencing reads for each organ sample, which resulted in the median of sequencing saturation (covering the fraction of library complexity) at 88% (61.6%-97%) for each sample (<strong>Additional file 20: Table S2</strong>). After primary quality control (QC) filters, 91,393 cells were identified (<strong>Additional file 20: Table S2</strong>). Higher unique molecular indices (UMIs) and genes were detected in the skin and trachea samples (with median UMIs of 4,022.5 and 4,100.5 and genes of 1,528 and 1,653, respectively) compared to the other organs, indicating their higher transcriptional activation (<strong><a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1B</a>, Additional file 1: Figure S1, and Additional file 20: Table S2</strong>). The cells in each organ were classified using unsupervised clustering and cell types were assigned based on canonical marker genes[<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>] (<strong>Additional file 21: Table S3</strong>). Next, visualization of the cells by t-distributed stochastic neighbor embedding (t-SNE) revealed multiple subpopulations of cells in each organ, with numbers of clusters ranging from 9 in the blood to 23 in the skin (<strong>Additional file 2: Figure S2, Additional file 3: Figure S2 continued, and Additional file 21: Table S3</strong>). Clusters due to cell doublets or unassigned cell types were identified and excluded manually in each organ, which resulted in a total of 88,622 cells for the downstream analysis (<strong>Additional file 21: Table S3</strong>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overview of single-cell RNA sequencing of 15 organ tissues from a male adult donor. A. An experiment schematic diagram highlights the sites of organs for tissue collection and sample processing. Live cells were collected using flow cytometry sorting (FACS) and subjected for cell barcoding. cDNA libraries for TCR, BCR and 5&#x2019;-mRNA expression were constructed independently, followed by high throughput sequencing and downstream analyses. B. Bar plot showing the numbers of high-quality cells (UMI &#x2265; 1000, Gene number &#x2265; 500 and the proportion of mitochondrial gene UMI &lt; 0.25) in each organ (colored). C. t-SNE visualization of all cells (88,622) in organs (colored as in B). The labeled cell types were the predominant cell types in each cluster." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-124545678" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Overview of single-cell RNA sequencing of 15 organ tissues from a male adult donor.&lt;/span&gt; &lt;strong&gt;A.&lt;/strong&gt; An experiment schematic diagram highlights the sites of organs for tissue collection and sample processing. Live cells were collected using flow cytometry sorting (FACS) and subjected for cell barcoding. cDNA libraries for TCR, BCR and 5&#x2019;-mRNA expression were constructed independently, followed by high throughput sequencing and downstream analyses. &lt;strong&gt;B.&lt;/strong&gt; Bar plot showing the numbers of high-quality cells (UMI &#x2265; 1000, Gene number &#x2265; 500 and the proportion of mitochondrial gene UMI &lt; 0.25) in each organ (colored). &lt;strong&gt;C.&lt;/strong&gt; t-SNE visualization of all cells (88,622) in organs (colored as in &lt;strong&gt;B&lt;/strong&gt;). The labeled cell types were the predominant cell types in each cluster.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975/F1.medium.gif" width="440" height="437"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975/F1.medium.gif" width="440" height="437"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">Overview of single-cell RNA sequencing of 15 organ tissues from a male adult donor.</span><p id="p-11" class="first-child"><strong>A.</strong> An experiment schematic diagram highlights the sites of organs for tissue collection and sample processing. Live cells were collected using flow cytometry sorting (FACS) and subjected for cell barcoding. cDNA libraries for TCR, BCR and 5’-mRNA expression were constructed independently, followed by high throughput sequencing and downstream analyses.</p><p id="p-12"><strong>B.</strong> Bar plot showing the numbers of high-quality cells (UMI ≥ 1000, Gene number ≥ 500 and the proportion of mitochondrial gene UMI &lt; 0.25) in each organ (colored).</p><p id="p-13"><strong>C.</strong> t-SNE visualization of all cells (88,622) in organs (colored as in <strong>B</strong>). The labeled cell types were the predominant cell types in each cluster.</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">With transcriptional profiles of such large number of cells, we identified some rare and novel cell populations. For instance, a group of Langerhans cells was observed in the skin sample (1% of all cells in the sample), which specifically expressed <em>CD207</em> and <em>CD1A</em>[<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>]. A fibroblast subtype with high expression of <em>COCH</em> gene (fold change, FC &gt; 116), and a very small group of 26 epithelial cells (0.31%) that specifically highly expressed <em>SCGB2A2, SCGB1B2P</em> and <em>PIP</em> genes (FC &gt; 546, 392 and 179, respectively, compared to all other cells; <strong>Additional file 4: Figure S3A</strong>) were identified in the skin sample. Both <em>SCGB2A2</em> and <em>PIP</em> genes have been found specifically expressed in breast tissue and over-expressed in breast cancer[<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>]. However, the functions of these genes in skin epithelial cells await further investigations.</p><p id="p-15">We combined all the 88,622 cells in clustering analysis and identified 43 clusters in 15 organs (<strong>Additional file 4: Figure S3B</strong>). We observed close clustering of cells from different organs (more than seven organs) for major cell types, including T cells, B cells, NK cells, fibroblasts, endothelial cells, smooth muscle cells, macrophages and monocytes (<strong><a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1C</a>, Additional file 4: Figure S3B, and Additional file 5: Figure S4</strong>). This is consistent with the understanding that cells derived from a same linage distribute across human body, especially for the circulating immune cells. Moreover, multiple clusters were further identified for several major cell types (T cells, B cells, fibroblasts, myeloid cells and endothelial cells), reflecting their heterogenous transcriptional profiles (<strong>Additional file 6: Figure S5A, B</strong>).</p></div><div id="sec-7" class="subsection"><h3>The heterogeneity of T cells in development state and clonalities over the body</h3><p id="p-16">In this study, we identified a total of 20,494 T cells that prevailing in the immune cells of most tissues including bladder, common bile duct, liver, marrow, muscle, rectum, small intestine, spleen and stomach (<strong>Additional file 22: Table S4</strong>), which is consistent with a previous finding[<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>]. The cells were divided into CD4<sup>+</sup> (7,122) and CD8<sup>+</sup> (13,372) T cells according their gene profiles and further grouped into 11 and 22 major unsupervised clusters, respectively (<strong><a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2A, B</a></strong>), including naive T (T<sub>N</sub>), central memory T (T<sub>CM</sub>), effector memory T (T<sub>EM</sub>), regulatory T (T<sub>reg</sub>), tissue-resident memory T (T<sub>RM</sub>), effector T (T<sub>EFF</sub>), intraepithelial lymphocytes (IEL) T, mucosal-associated invariant T (MAIT) and yδ T cells, based on known markers[<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>]. Both CD4<sup>+</sup> and CD8<sup>+</sup> T cell clusters showed a distribution pattern in organ-specific manner (<strong>Additional file 7: Figure S6A, B</strong>) and each of them had differentially expressed genes (<strong>Additional file 7: S6C, D and Additional file 22: Table S4</strong>). Overlapping of clusters was also observed between organs (such as blood and marrow), suggesting shared T cell subtypes (<strong>Additional file 7: Figure S6A, B</strong>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The heterogeneity, development and clonality of T cells in human organs. A, B. t-SNE plots of 7,122 CD4+ T (11 clusters) and 13,372 CD8+ T cells (22 clusters) from 15 organ tissues, respectively. Each dot represents a cell. Each color-coded region represents one cluster, which is indicated at the right panel of the plot. C. Pseudo-time trajectory analysis of all CD4+ (top panel) and CD8+ T cells (bottom panel) with high variable genes. Each dot represents a single cell and is colored according to their clustering at a for CD4+ and b for CD8+. The inlet t-SNE plot at each plot shows each cell with a pseudo-time score from dark blue to yellow, indicating early and terminal states, respectively. D,E. Heatmaps of the active scores of each T cell cluster as numbered on top, of which expression was regulated by transcription factors (TFs), as estimated using SCENIC analysis. Shown only the top 15 TFs for CD4+ T cells (D) and the top 10 for CD8+ T cells (E), which had the highest difference in expression regulation estimates between each cluster and all other cells, under a Wilcoxon rank-sum test. F. Sharing intensity of TCR clones in CD4+ (top panel) and CD8+ (bottom panel) T cells between different organ samples. Each line represents a sharing of TCR between two organs at the ends and the thickness of the line represents a migration-index score between paired organs calculated by STARTRAC. The sizes of the dots were shown according to the size of T cell clones in organs with different colors. G. Migration- (top panel) and expansion-index (bottom panel) scores of CD4+ and CD8+ T cells of each tissue calculated by STARTRAC with a paired Student&#x2019;s t-test. H. Expansion- (top panel) and transition-index (bottom panel) scores of each CD4+ and CD8+ T cell cluster calculated by STARTRAC. I. Alluvial diagram showing the CD8+ T cells with same TCRs detected in at least six tissues and different cell types. Each line represents a sharing of a cell, colored by tissue." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-124545678" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;The heterogeneity, development and clonality of T cells in human organs.&lt;/span&gt; &lt;strong&gt;A, B.&lt;/strong&gt; t-SNE plots of 7,122 CD4&lt;sup&gt;+&lt;/sup&gt; T (11 clusters) and 13,372 CD8&lt;sup&gt;+&lt;/sup&gt; T cells (22 clusters) from 15 organ tissues, respectively. Each dot represents a cell. Each color-coded region represents one cluster, which is indicated at the right panel of the plot. &lt;strong&gt;C.&lt;/strong&gt; Pseudo-time trajectory analysis of all CD4&lt;sup&gt;+&lt;/sup&gt; (top panel) and CD8&lt;sup&gt;+&lt;/sup&gt; T cells (bottom panel) with high variable genes. Each dot represents a single cell and is colored according to their clustering at a for CD4&lt;sup&gt;+&lt;/sup&gt; and b for CD8&lt;sup&gt;+&lt;/sup&gt;. The inlet t-SNE plot at each plot shows each cell with a pseudo-time score from dark blue to yellow, indicating early and terminal states, respectively. &lt;strong&gt;D,E.&lt;/strong&gt; Heatmaps of the active scores of each T cell cluster as numbered on top, of which expression was regulated by transcription factors (TFs), as estimated using SCENIC analysis. Shown only the top 15 TFs for CD4&lt;sup&gt;+&lt;/sup&gt; T cells (D) and the top 10 for CD8&lt;sup&gt;+&lt;/sup&gt; T cells (E), which had the highest difference in expression regulation estimates between each cluster and all other cells, under a Wilcoxon rank-sum test. &lt;strong&gt;F.&lt;/strong&gt; Sharing intensity of TCR clones in CD4&lt;sup&gt;+&lt;/sup&gt; (top panel) and CD8&lt;sup&gt;+&lt;/sup&gt; (bottom panel) T cells between different organ samples. Each line represents a sharing of TCR between two organs at the ends and the thickness of the line represents a migration-index score between paired organs calculated by STARTRAC. The sizes of the dots were shown according to the size of T cell clones in organs with different colors. &lt;strong&gt;G.&lt;/strong&gt; Migration- (top panel) and expansion-index (bottom panel) scores of CD4&lt;sup&gt;+&lt;/sup&gt; and CD8&lt;sup&gt;+&lt;/sup&gt; T cells of each tissue calculated by STARTRAC with a paired Student&#x2019;s t-test. &lt;strong&gt;H.&lt;/strong&gt; Expansion- (top panel) and transition-index (bottom panel) scores of each CD4&lt;sup&gt;+&lt;/sup&gt; and CD8&lt;sup&gt;+&lt;/sup&gt; T cell cluster calculated by STARTRAC. &lt;strong&gt;I.&lt;/strong&gt; Alluvial diagram showing the CD8&lt;sup&gt;+&lt;/sup&gt; T cells with same TCRs detected in at least six tissues and different cell types. Each line represents a sharing of a cell, colored by tissue.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975/F2.medium.gif" width="310" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975/F2.medium.gif" width="310" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">The heterogeneity, development and clonality of T cells in human organs.</span><p id="p-17" class="first-child"><strong>A, B.</strong> t-SNE plots of 7,122 CD4<sup>+</sup> T (11 clusters) and 13,372 CD8<sup>+</sup> T cells (22 clusters) from 15 organ tissues, respectively. Each dot represents a cell. Each color-coded region represents one cluster, which is indicated at the right panel of the plot.</p><p id="p-18"><strong>C.</strong> Pseudo-time trajectory analysis of all CD4<sup>+</sup> (top panel) and CD8<sup>+</sup> T cells (bottom panel) with high variable genes. Each dot represents a single cell and is colored according to their clustering at a for CD4<sup>+</sup> and b for CD8<sup>+</sup>. The inlet t-SNE plot at each plot shows each cell with a pseudo-time score from dark blue to yellow, indicating early and terminal states, respectively.</p><p id="p-19"><strong>D,E.</strong> Heatmaps of the active scores of each T cell cluster as numbered on top, of which expression was regulated by transcription factors (TFs), as estimated using SCENIC analysis. Shown only the top 15 TFs for CD4<sup>+</sup> T cells (D) and the top 10 for CD8<sup>+</sup> T cells (E), which had the highest difference in expression regulation estimates between each cluster and all other cells, under a Wilcoxon rank-sum test.</p><p id="p-20"><strong>F.</strong> Sharing intensity of TCR clones in CD4<sup>+</sup> (top panel) and CD8<sup>+</sup> (bottom panel) T cells between different organ samples. Each line represents a sharing of TCR between two organs at the ends and the thickness of the line represents a migration-index score between paired organs calculated by STARTRAC. The sizes of the dots were shown according to the size of T cell clones in organs with different colors.</p><p id="p-21"><strong>G.</strong> Migration- (top panel) and expansion-index (bottom panel) scores of CD4<sup>+</sup> and CD8<sup>+</sup> T cells of each tissue calculated by STARTRAC with a paired Student’s t-test.</p><p id="p-22"><strong>H.</strong> Expansion- (top panel) and transition-index (bottom panel) scores of each CD4<sup>+</sup> and CD8<sup>+</sup> T cell cluster calculated by STARTRAC.</p><p id="p-23"><strong>I.</strong> Alluvial diagram showing the CD8<sup>+</sup> T cells with same TCRs detected in at least six tissues and different cell types. Each line represents a sharing of a cell, colored by tissue.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">To better understand the development state of T cells, we performed trajectory analyses of CD4<sup>+</sup> and CD8<sup>+</sup> T cells. We observed that the trajectory tree rooted from T<sub>N</sub> cells, sprouting into T<sub>EFF</sub>, T<sub>CM</sub> and T<sub>RM</sub> branches for CD4<sup>+</sup> T cells and T<sub>EFF</sub>, T<sub>CM</sub>, IEL, γδ and T<sub>RM</sub> branches for CD8<sup>+</sup> T cell (<strong><a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2C</a> and Additional file 7: Figure S6E, F</strong>). T<sub>RM</sub> cells and IEL cells with higher pseudo-time scores were found in CD4<sup>+</sup> and CD8<sup>+</sup> T cells, respectively, suggesting their terminal development state (<strong><a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2C</a> and Additional file 7: Figure S6E, F</strong>). Moreover, several T<sub>RM</sub> clusters at the end of other branches with mediate scores for both CD4<sup>+</sup> and CD8<sup>+</sup> T cells, indicating the middle state of these clusters. In addition, the clusters with different states showed an organ specific pattern (<strong>Additional file 7: Figure S6E, F</strong>). These observations reveal the existence of heterogeneity in the developmental states of both CD4<sup>+</sup> and CD8<sup>+</sup> cells in human organs.</p><p id="p-25">Transcription factors (TFs) have been demonstrated as important regulators of gene expression and to shape different phenotype of T cells[<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>]. We therefore performed Single-Cell Regulatory Network Inference and Clustering (SCENIC) analysis to assess TFs underlying differential gene expression in T cells. We identified well-defined and cell subtype-specific TFs for CD4<sup>+</sup> (<strong><a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2D</a></strong>) and CD8<sup>+</sup> (<strong><a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig. 2E</a></strong>) clusters (<strong>Additional file 22: Table S4</strong>), such as higher activity of <em>FOXP3[<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>]</em> and <em>NFATC1</em>[<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>] in T<sub>reg</sub> cells, upregulation of <em>LEF1, MYC, TCF7[<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>]</em> and <em>KLF2[<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>]</em> in T<sub>N</sub> cells, <em>TBX21, STAT1 and IRF1</em> in T<sub>EFF</sub> cells[<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">25</a>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>] (<strong><a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig. 2D, E</a></strong>). In addition, many other poorly investigated TFs were also observed in CD4<sup>+</sup> and CD8<sup>+</sup> clusters, such as upregulation of <em>FOSB, JUN, ID2, ATF4, FOSL2, JUND</em> in CD4<sup>+</sup> T<sub>RM</sub> cells, and higher activity of <em>FOS, JUND, ATF3, JUN, JUNB, GTF2B</em> and <em>CEBPD in</em> CD8<sup>+</sup> T<sub>RM</sub> <em>cells.</em> Collectively, these results indicate that the multiple TFs combinations regulate T cells development to maintain heterogeneous states of CD4<sup>+</sup> and CD8<sup>+</sup> T cells.</p><p id="p-26">To better investigate the clonalities and dynamic relationships among T cell subtypes across tissues, we preformed TCR clonal typing accompanied with transcriptome analysis (<strong><a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1E</a></strong>). After stringent QC filters, we identified 4,150 TCR clonotypes with unique heterodimer α and β chains among 6,203 T cells (30.3% of the whole 20,494 T cells), including 2,301 CD4<sup>+</sup> T cells and 3,902 CD8<sup>+</sup> T cells. Among them, 3,728 cells (1,641 CD4<sup>+</sup> and 2,087 CD8<sup>+</sup> T cells) had a unique TCR clonotype for each, while the remaining 2,475 cells (214 CD4<sup>+</sup> and 2,261 CD8<sup>+</sup> T cells) shared two or more of the 422 TCR clonotypes (<strong>Additional file 22: Table S4</strong>). We observed similar numbers of V and J segments for TCR α chain in both CD4<sup>+</sup> and CD8<sup>+</sup> T cells, which shared 50% and 40% of the top 10 frequent V <em>(TRAV12-1, TRAV21, TRAV29/DV5, TARV13-1</em> and <em>TRAV12-2)</em> and J <em>(TRAVJ49, TRAVJ48, TRAVJ9</em>, and <em>TRAVJ39)</em> segments, respectively. By contrast, the diversity of V segment was much higher than that of J segment for β chain in both CD4<sup>+</sup> and CD8<sup>+</sup> T cells, which shared 40 % and 80% of the top 10 frequent V (<em>TRBV6-5, TRBV20-1, TRBV7-2</em>, and <em>TRBV2)</em> and J <em>(TRBJ2-1, TRBJ2-7, TRBJ1-5, TRBJ1-1, TRBJ2-3, TRBJ1-2, TRBJ2-5</em>, and <em>TRBJ2-2</em>) segments, respectively (<strong>Additional file 8: Figure S7A, B</strong>). Although more CD8<sup>+</sup> T cells were detected than CD4<sup>+</sup> T cells among all tissues, no significant difference in clone sizes (numbers of unique clonotypes) was observed between the two cell populations (<strong>Additional file 8: Figure S7C</strong>). Singular cells of unique TCR clonotypes were prevalent for CD4<sup>+</sup> T cells among tissues, except that higher proportion of multiple cells with identical TCR clonotypes or clonal expansions were observed in the stomach and marrow (<strong>Additional file 8: Figure S7D top panel</strong>). Clonal expansions were much commonly found for CD8<sup>+</sup> T cells in all tissues (<strong>Additional file 8: Figure S7D bottom panel</strong>).</p><p id="p-27">To investigate clonotype distribution of T cells across tissues, we evaluated the ability of sharing TCR for each tissue with others. We observed more intensive and broader sharing of TCR clonotype for CD8<sup>+</sup> than CD4<sup>+</sup> T cells across tissues (<strong><a id="xref-fig-2-7" class="xref-fig" href="#F2">Fig. 2F</a></strong>). Higher migration capacity as reflected by migration-index score of tissues was found in CD8<sup>+</sup> T cells than CD4<sup>+</sup> T cells (<strong><a id="xref-fig-2-8" class="xref-fig" href="#F2">Fig. 2G</a> top panel</strong>). Moreover, higher expansion ability but lower diversity was observed in CD8<sup>+</sup> T cells in each tissue compared to CD4<sup>+</sup> T cells (<strong><a id="xref-fig-2-9" class="xref-fig" href="#F2">Fig. 2G</a> bottom panel and Additional file 8: Figure S7E</strong>). Cells with clonal expansion had considerable proportions in T<sub>EM</sub>, T<sub>EFF</sub>, T<sub>RM</sub> and IEL clusters of CD8<sup>+</sup> T cells (<strong>Additional file 8: Figure S7F bottom panel</strong>), and T<sub>EFF</sub> and <em>TRBV12-3</em>_T<sub>RM</sub> of CD4<sup>+</sup> T cells (<strong>Additional file 8: Figure S7F top pane</strong>l). We further evaluated the clonal expansion and transition (clonotype sharing ability of each subpopulation) of each T cell cluster, which revealed significantly stronger expansion and transition ability of CD8<sup>+</sup> compared with CD4<sup>+</sup> T cells (<strong><a id="xref-fig-2-10" class="xref-fig" href="#F2">Fig. 2H</a> and Additional file 8: Figure S7G</strong>). This is consistent with the finding of stronger links between subtypes of CD8<sup>+</sup> than CD4<sup>+</sup> T cells (<strong><a id="xref-fig-2-11" class="xref-fig" href="#F2">Fig. 2F</a> and Additional file 8: Figure S7H, I</strong>). Moreover, we observed that the clonal expansion of certain CD8<sup>+</sup> T cells (T<sub>EFF</sub>, T<sub>RM</sub>, T<sub>EM</sub>, γδ and IEL T cells) distributed across multiple tissues with different development states (<strong><a id="xref-fig-2-12" class="xref-fig" href="#F2">Fig. 2I</a> and Additional file 8: Figure S7I</strong>), suggesting that most of these cell types recognizing the same antigens propagated and migrated more intensively than other cell types. In addition, we observed consistent results of TCR diversity using β chains (<strong>Additional file 8: Figure S7J-I</strong>).</p></div><div id="sec-8" class="subsection"><h3>The heterogeneity of B cells and plasma cells</h3><p id="p-28">Clustering analysis revealed 15 distinct cell clusters among 10,655 B and plasma cells from 11 organ samples, including nine B cell <em>(CD20)</em> and six plasma cell <em>(SDC1)</em> clusters, respectively (<strong><a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3A</a></strong>). We observed the predominance of B cells over plasma cells in all tested organs except for esophagus and rectum (<strong><a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3B</a></strong>). Differential gene expression (DEG) analysis revealed that B cells exhibited distinct gene profiles from plasma cells (<strong>Additional file 9: Figure S8A and Additional file 23: Table S5</strong>). Although <em>CD27</em> is as a canonical marker of memory B cells[<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>], we observed low transcription level of this gene in memory B cells, but higher transcription level in plasma cells (<strong><a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3C</a></strong>). <em>TCL1A</em> was significantly expressed in two naive B cell clusters with distinct gene expression profiles, one from lymph nodes (TCL1A_ly_naive_B) and the other (TCL1A_naive_B) from multiple tissues, compared to the other B and plasma cell clusters (<strong><a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3A</a> and Additional file 9: Figure S8A</strong>). Moreover, <em>TCL1A</em> was exclusively expressed in non-CD27-expressing B cell clusters and was highly reversely correlated with <em>CD27</em> transcription (Person’s R = −0.83, <em>P</em> = 0; <strong><a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig. 3C</a> and Additional file 9: Figure S8B</strong>). <em>TCL1A</em> has been reported as an important gene in B cells lymphoma[<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>]. These suggest that <em>TCL1A</em> might be a novel marker of naive B cells.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The heterogeneity and clonality of B cells in human organs. A. t-SNE plots showed 15 clusters (10,655 cells) of B and plasma cells colored by tissues (top panel) and cell subtypes (bottom panel). Each dot represents a cell. B. Distribution of B and plasma cells in each organ. Pie charts on top illustrate the composition of B and plasma cells in each organ. The stacked bars represent the percentage of each cluster in the indicated organ. C. Violin plots showed the normalized expression of maker genes for B (MS4A1), plasma cells (SDC1), na&#xEF;ve B cell (TCL1A) and memory B cells (CD27). D. Gene Ontology enrichment analysis results of B and plasma cell clusters. Cell clustered as numbered below are colored according to their &#x2212;log10P values. Only the top 20 significant terms (p-value &lt; 0.05) were shown. E. Heatmap of the active scores of each B and plasma cell subtype as numbered on top, of which expression was regulated by transcription factors (TFs), as estimated using SCENIC analysis. Shown are the top 10 TFs having the highest difference in expression regulation estimated between each cluster and all other cells, tested with a Wilcoxon rank-sum test. F. Sharing intensity of BCR clones between different organs. Each line represents a sharing of BCR between two organs at the ends and the thickness of the line represents a migration-index score between paired organs calculated by STARTRAC. The size of the dot is shown as the size of B and plasma cell clones, colored by organ. G. Expansion- (top panel) and transition-index (bottom panel) scores of each B and plasma cell cluster calculated by STARTRAC." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-124545678" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;The heterogeneity and clonality of B cells in human organs.&lt;/span&gt; &lt;strong&gt;A.&lt;/strong&gt; t-SNE plots showed 15 clusters (10,655 cells) of B and plasma cells colored by tissues (top panel) and cell subtypes (bottom panel). Each dot represents a cell. &lt;strong&gt;B.&lt;/strong&gt; Distribution of B and plasma cells in each organ. Pie charts on top illustrate the composition of B and plasma cells in each organ. The stacked bars represent the percentage of each cluster in the indicated organ. &lt;strong&gt;C.&lt;/strong&gt; Violin plots showed the normalized expression of maker genes for B (&lt;em&gt;MS4A1&lt;/em&gt;), plasma cells (&lt;em&gt;SDC1&lt;/em&gt;), na&#xEF;ve B cell (&lt;em&gt;TCL1A&lt;/em&gt;) and memory B cells (&lt;em&gt;CD27&lt;/em&gt;). &lt;strong&gt;D.&lt;/strong&gt; Gene Ontology enrichment analysis results of B and plasma cell clusters. Cell clustered as numbered below are colored according to their &#x2212;log&lt;sub&gt;10&lt;/sub&gt;&lt;em&gt;P&lt;/em&gt; values. Only the top 20 significant terms (p-value &lt; 0.05) were shown. &lt;strong&gt;E.&lt;/strong&gt; Heatmap of the active scores of each B and plasma cell subtype as numbered on top, of which expression was regulated by transcription factors (TFs), as estimated using SCENIC analysis. Shown are the top 10 TFs having the highest difference in expression regulation estimated between each cluster and all other cells, tested with a Wilcoxon rank-sum test. &lt;strong&gt;F.&lt;/strong&gt; Sharing intensity of BCR clones between different organs. Each line represents a sharing of BCR between two organs at the ends and the thickness of the line represents a migration-index score between paired organs calculated by STARTRAC. The size of the dot is shown as the size of B and plasma cell clones, colored by organ. &lt;strong&gt;G.&lt;/strong&gt; Expansion- (top panel) and transition-index (bottom panel) scores of each B and plasma cell cluster calculated by STARTRAC.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975/F3.medium.gif" width="301" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975/F3.medium.gif" width="301" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">The heterogeneity and clonality of B cells in human organs.</span><p id="p-29" class="first-child"><strong>A.</strong> t-SNE plots showed 15 clusters (10,655 cells) of B and plasma cells colored by tissues (top panel) and cell subtypes (bottom panel). Each dot represents a cell.</p><p id="p-30"><strong>B.</strong> Distribution of B and plasma cells in each organ. Pie charts on top illustrate the composition of B and plasma cells in each organ. The stacked bars represent the percentage of each cluster in the indicated organ.</p><p id="p-31"><strong>C.</strong> Violin plots showed the normalized expression of maker genes for B (<em>MS4A1</em>), plasma cells (<em>SDC1</em>), naïve B cell (<em>TCL1A</em>) and memory B cells (<em>CD27</em>).</p><p id="p-32"><strong>D.</strong> Gene Ontology enrichment analysis results of B and plasma cell clusters. Cell clustered as numbered below are colored according to their −log<sub>10</sub><em>P</em> values. Only the top 20 significant terms (p-value &lt; 0.05) were shown.</p><p id="p-33"><strong>E.</strong> Heatmap of the active scores of each B and plasma cell subtype as numbered on top, of which expression was regulated by transcription factors (TFs), as estimated using SCENIC analysis. Shown are the top 10 TFs having the highest difference in expression regulation estimated between each cluster and all other cells, tested with a Wilcoxon rank-sum test.</p><p id="p-34"><strong>F.</strong> Sharing intensity of BCR clones between different organs. Each line represents a sharing of BCR between two organs at the ends and the thickness of the line represents a migration-index score between paired organs calculated by STARTRAC. The size of the dot is shown as the size of B and plasma cell clones, colored by organ.</p><p id="p-35"><strong>G.</strong> Expansion- (top panel) and transition-index (bottom panel) scores of each B and plasma cell cluster calculated by STARTRAC.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">B cells are professional antigen-presenting cells (APCs) with high expression of <em>CIITA</em> and <em>MHC</em> class II genes, which are silenced during the differentiation to plasma cells[<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>]. Consistently, plasma cells had much lower presenting-antigen score (PAS) compared to B cell clusters (<strong>Additional file 9: Figure S8C</strong>), suggesting different abilities to present antigen between the two cell types. We also investigated the potential biological function of different cell clusters using GO analysis. The potential biological functions of B cells were found be enriched in immune response (for example, <em>responding to activation of immune</em>) and that of plasma cells were associated with protein synthesis (including <em>signal peptide processing</em>, <em>protein localization to endoplasmic reticulum</em> and <em>protein folding</em>; <strong><a id="xref-fig-3-6" class="xref-fig" href="#F3">Fig. 3D</a></strong>). In addition, gene set variation analysis (GSEA) showed that TCL1A_naive_B was enriched in <em>oxidative phosphorylation</em> and <em>fatty acid metabolism</em> (<strong>Additional file 9: Figure S8D</strong>).</p><p id="p-37">TFs play a critical role in the differentiation of B cells to plasma cells, engaging B cells with effector or memory functions[<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>]. SCENIC analysis revealed that TFs exhibited similar activities within B and plasma cells but with distinct patterns between the two populations (<strong>Additional file 23: Table S5</strong>). TFs with higher activity were found in B cells, including <em>MYC[<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>]</em> and <em>REL</em>[<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>], which have been known to modulate B cell development, as well as other TFs, such as EGF receptors (<em>EGFR1/2/3</em>), which has not been characterized in B cells. Likewise, TFs enriched in plasma cells included <em>PRDM1</em>, <em>XBP1</em>, <em>FOS</em> and <em>IRF4</em>, which play important roles in the development of plasma cells[<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a>]. In addition, many TFs with unknown roles were found, including <em>CREB3L2</em>, <em>ATF4</em>, <em>ATF3</em> and <em>FOXO3</em>. Taken together, these results suggest that various TFs regulate the development of B cells to plasma cells.</p><p id="p-38">To explore the clonalities of B and plasma cell clusters across organ tissues, we performed a single-cell BCR sequencing analysis. After stringent QC filters, 4,001 of 10,655 cells were assigned with 3,844 clonotypes, among which 3,654 clonotypes were presented on singular cells and 190 were presented by multiple cells (<strong>Additional file 23: Table S5</strong>). We observed various usage of V and J gene segments for both heavy and light chains of immunoglobulin genes, with preferred usage of some particular V and J segments (<strong>Additional file 9: Figure S8E-G</strong>). Unlike T cells, clonal diversity was common for B cells among all organs, while clonal expansion of B cells was limited and restricted in the spleen and stomach (<strong>Additional file 9: Figure S8H</strong>). Moreover, less sharing of BCR clonotypes between B and plasma cells was observed across organs compared to T cells (<strong><a id="xref-fig-2-13" class="xref-fig" href="#F2">Fig. 2F</a> and <a id="xref-fig-3-7" class="xref-fig" href="#F3">Fig. 3F</a></strong>). In addition, BCR analysis showed lower expansion and transition abilities of B cells compared to plasma cells and T cells (<strong><a id="xref-fig-2-14" class="xref-fig" href="#F2">Fig. 2H</a>, <a id="xref-fig-3-8" class="xref-fig" href="#F3">Fig. 3G</a>, and Additional file 9: Figure S8I</strong>).</p></div><div id="sec-9" class="subsection"><h3>The heterogeneity of myeloid cells and the origin of macrophages</h3><p id="p-39">We obtained 5,605 myeloid cells from 14 organ tissues, which were grouped into 17 distinct clusters (<strong><a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4A, B</a></strong>). Based on the differential expressions of marker genes, we further identified seven monocyte clusters, nine macrophage clusters, and one small dendritic cell (DC) cluster (<strong><a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4A, B</a>, and Additional file 10: Figure S9A</strong>). The hierarchical clustering analysis revealed that all classical monocytes were closely related as a branch node, while the intermediate and non-classical monocytes were closely related to macrophages (<strong><a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4C</a></strong>). DEG analysis showed that each myeloid cell cluster had a specific gene signature, suggesting the inter-cell heterogeneity among monocytes, macrophages and DC cells (<strong><a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4D</a> and Additional file 24: Table S6</strong>). All clusters contained cells from multiple tissues, except for FN1_Intermediate_Mon (C5) and CCL20_Intermediate_Mon (C10) in the rectum and Langerhans clusters (C16) in the skin, suggesting similar transcriptional profiles and origins of these myeloid cells <strong>(<a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig. 4A, B</a></strong>). Monocytes showed predominance in 10 of the 14 tested tissues, except for the esophagus, heart, small intestine and skin, where macrophages accounted for more than 50% of all myeloid cells (<strong><a id="xref-fig-4-6" class="xref-fig" href="#F4">Fig. 4B</a></strong>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heterogeneity and developmental stages of myeloid cells. A. t-SNE plots showed 5,605 myeloid cells, colored according to their origins of tissues (top panel) or cell clusters (bottom panel). Each dot represents a cell. B. Distribution of different myeloid cell subpopulations in each tissue. Pie charts on top illustrate the composition of macrophages, monocytes and DC cells in each tissue. The stacked bars represent the percentage of each cluster in the indicated organ. C. Dendrogram of 15 clusters based on their normalized mean expression values (correlation distance metric, average linkage). Only genes with ln(fold-change) above 0.25, p.adjust &lt; 0.05 and pct.1 &#x2265; 0.2 in each cluster were included in the calculations. D. Heatmap of top 20 differentially expressed genes (239 genes ranked descending according to fold change) in each cell cluster as numbered on top. E. Heatmap of the active scores of each myeloid cell subtype as numbered on top, of which expression was regulated by transcription factors (TFs), as estimated using SCENIC analysis. Shown are the top 10 TFs having the highest difference in expression regulation estimates between each cluster and all other cells, tested with a Wilcoxon rank-sum test. F. Pseudo-time trajectory analysis of all myeloid cells with high variable genes. Each dot represents a single cell and is colored according to their clustering in A. The inlet t-SNE plot at each plot shows each cell with a pseudo-time score from dark blue to yellow, indicating early and terminal states, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-124545678" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Heterogeneity and developmental stages of myeloid cells.&lt;/span&gt; &lt;strong&gt;A.&lt;/strong&gt; t-SNE plots showed 5,605 myeloid cells, colored according to their origins of tissues (top panel) or cell clusters (bottom panel). Each dot represents a cell. &lt;strong&gt;B.&lt;/strong&gt; Distribution of different myeloid cell subpopulations in each tissue. Pie charts on top illustrate the composition of macrophages, monocytes and DC cells in each tissue. The stacked bars represent the percentage of each cluster in the indicated organ. &lt;strong&gt;C.&lt;/strong&gt; Dendrogram of 15 clusters based on their normalized mean expression values (correlation distance metric, average linkage). Only genes with ln(fold-change) above 0.25, p.adjust &lt; 0.05 and pct.1 &#x2265; 0.2 in each cluster were included in the calculations. &lt;strong&gt;D.&lt;/strong&gt; Heatmap of top 20 differentially expressed genes (239 genes ranked descending according to fold change) in each cell cluster as numbered on top. &lt;strong&gt;E.&lt;/strong&gt; Heatmap of the active scores of each myeloid cell subtype as numbered on top, of which expression was regulated by transcription factors (TFs), as estimated using SCENIC analysis. Shown are the top 10 TFs having the highest difference in expression regulation estimates between each cluster and all other cells, tested with a Wilcoxon rank-sum test. &lt;strong&gt;F.&lt;/strong&gt; Pseudo-time trajectory analysis of all myeloid cells with high variable genes. Each dot represents a single cell and is colored according to their clustering in &lt;strong&gt;A&lt;/strong&gt;. The inlet t-SNE plot at each plot shows each cell with a pseudo-time score from dark blue to yellow, indicating early and terminal states, respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975/F4.medium.gif" width="310" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975/F4.medium.gif" width="310" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">Heterogeneity and developmental stages of myeloid cells.</span><p id="p-40" class="first-child"><strong>A.</strong> t-SNE plots showed 5,605 myeloid cells, colored according to their origins of tissues (top panel) or cell clusters (bottom panel). Each dot represents a cell.</p><p id="p-41"><strong>B.</strong> Distribution of different myeloid cell subpopulations in each tissue. Pie charts on top illustrate the composition of macrophages, monocytes and DC cells in each tissue. The stacked bars represent the percentage of each cluster in the indicated organ.</p><p id="p-42"><strong>C.</strong> Dendrogram of 15 clusters based on their normalized mean expression values (correlation distance metric, average linkage). Only genes with ln(fold-change) above 0.25, p.adjust &lt; 0.05 and pct.1 ≥ 0.2 in each cluster were included in the calculations.</p><p id="p-43"><strong>D.</strong> Heatmap of top 20 differentially expressed genes (239 genes ranked descending according to fold change) in each cell cluster as numbered on top.</p><p id="p-44"><strong>E.</strong> Heatmap of the active scores of each myeloid cell subtype as numbered on top, of which expression was regulated by transcription factors (TFs), as estimated using SCENIC analysis. Shown are the top 10 TFs having the highest difference in expression regulation estimates between each cluster and all other cells, tested with a Wilcoxon rank-sum test.</p><p id="p-45"><strong>F.</strong> Pseudo-time trajectory analysis of all myeloid cells with high variable genes. Each dot represents a single cell and is colored according to their clustering in <strong>A</strong>. The inlet t-SNE plot at each plot shows each cell with a pseudo-time score from dark blue to yellow, indicating early and terminal states, respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-46">TFs such as <em>SPI1</em> are involved in the development of monocytes to macrophages[<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>]. However, how TFs regulate the development in normal human body is still unclear. TF activity analysis revealed similar activation of certain TFs among the four classical monocytes, which is consistent with their close similarity in gene signatures (<strong><a id="xref-fig-4-7" class="xref-fig" href="#F4">Fig. 4D, E</a> and Additional file 24: Table S6</strong>). Higher activation of <em>SPI1</em> were determined in cell clusters of classical monocytes, non-classical monocytes and LIPA_macrophage, suggesting its involvement in the development of these cells. We also identified several TFs with higher activity in the four classical monocyte clusters, including <em>NFIL3</em>, <em>RXRA</em>, <em>ELF2</em>, <em>ETV6</em>, <em>ELK3</em>, <em>NFE2</em> and <em>EP300</em>. Their roles in maintaining the classical state of monocytes have yet to be investigated. Non-classical monocytes and macrophages shared several active TFs, such as <em>TCF7L2</em>, <em>STAT1</em>, <em>KLF11</em>, <em>ELF1</em>, <em>NR1H3</em>, <em>SPIC</em> and <em>HDAC2</em>, which is consistent with their close pattern in the hierarchical clustering. We observed multiple not-well-characterized TFs activated in the intermediate, non-classical monocytes, macrophages and DCs other than classical monocytes, including <em>JUN</em>, <em>KLF2</em>, <em>PRDM1</em>, <em>EGFR1 and ATF3</em>. We also observed cluster-specific TFs for myeloid cell clusters <strong>(<a id="xref-fig-4-8" class="xref-fig" href="#F4">Fig. 4E</a> and Additional file 24: Table S6</strong>). For instance, activation of <em>ZNF217</em> and <em>NR1H2</em> was higher in C14 macrophages. These results indicated that TFs combinations help shape the different states of myeloid cells in a steady state.</p><p id="p-47">Considering the previous understanding that tissue-resident macrophages are originated from circulating monocytes[<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>], we examined the connection between macrophages and monocytes. First, we observed the coexistence of macrophages and monocytes in the same organs (<strong><a id="xref-fig-4-9" class="xref-fig" href="#F4">Fig. 4A, B</a></strong>). Second, trajectory analysis revealed that the classical monocytes had initial state at the root, sprouting into branches with more developed states of monocytes and then macrophages either directly or via the two intermediate monocytes (<strong><a id="xref-fig-4-10" class="xref-fig" href="#F4">Fig. 4F</a> and Additional file 10: Figure S9B, C</strong>). Notably, high expression of proliferation markers including <em>MKI67</em>, <em>PCNA</em> and <em>TYMS</em> were exclusively detected in the C12 macrophages (mainly from the rectum) and Langerhans cells (C16, skin; <strong>Additional file 10: Figure S9D</strong>). GSEA analysis also revealed that most of the macrophage clusters had higher enrichment score of <em>G2M checkpoint and E2F targets</em> pathways (<strong>Additional file 10: Figure S9E</strong>). Additional trajectory analysis of macrophages in the rectum showed that C12 positioned in the middle position in the development of other clusters, suggesting that it was the origin of these clusters (<strong>Additional file 10: Figure S9F</strong>). Taken together, these observations suggest that circulating monocytes give rise to the macrophages in the organs and local microenvironment also contributes to the expansion and proliferation of different macrophage populations.</p><p id="p-48">It has been demonstrated that myeloid cells could act as professional APCs, with the strongest antigen-presenting ability for DCs[<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>]. We observed various antigen-presenting abilities as reflected by PAS for different myeloid clusters (<strong>Additional file 10: Figure S9G</strong>). Langerhans cells had stronger antigen-presenting ability than other macrophages and monocytes (<em>P</em> &lt; 2.2 × 10<sup>−16</sup>; <strong>Additional file 10: Figure S9G</strong>). In average, the classical monocytes had the lowest PAS among myeloid cell clusters (<em>P</em> &lt; 2.2 × 10<sup>−16</sup>). Surprisingly, DCs had similar PAS with macrophages (<em>P</em> = 0.16), which might be due to the limited number of DCs included in the analysis.</p></div><div id="sec-10" class="subsection"><h3>The similarity and heterogeneity of epithelial cells intra- and inter-tissues</h3><p id="p-49">We obtained a total of 18,090 epithelial cells from nine tested organ tissues (<strong><a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5A</a></strong>). Differential gene expression analysis revealed a clear distinct pattern of gene expression among the tissue samples (<strong>Additional file 11: Figure S10A and Additional file 25: Table S7</strong>). There were 189 genes with tissue-specific expression (FC ≥ 5, pct.1 ≥ 0.2; <strong>Additional file 25: Table S7 and Additional file 11: Figure S10A</strong>), indicating the heterogeneity of epithelial cells among organ tissues. GO analysis also revealed various biological function of epithelial cells from different tissues, among which <em>epithelial cell differentiation, humoral immune response</em> and <em>regulation of myeloid leukocyte activation</em> were common pathways enriched in the majority of epithelial cells, suggesting that the cells share common functions in epithelial cell development and regulating immune response (<strong>Additional file 11: Figure S10B</strong>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The heterogeneity of epithelial cells inter- and intra-organ tissues. A. t-SNE plots showed 18,090 epithelial cells, colored according to their origins of tissues (top panel) or cell clusters (bottom panel). B. Dot plotting visualization of the normalized expression of marker genes of each epithelial cluster. The size of the dot represented the percentage of cells with a cell type, and the color represented the average expression level. C. Dendrogram of 33 clusters based on their normalized mean expression values (correlation distance metric, average linkage). Only genes with ln(fold-change) above 0.25, p.adjust &lt; 0.05 and pct.1 &#x2265; 0.2 in each cluster were included in the analysis. D, E. Gene Ontology enrichment analysis results of each epithelial cell cluster in digestive organs (D) and non-digestive organs (E). Cell clustered as numbered below were colored according to their &#x2212;log10P values. Only the top 20 significant terms (p-value &lt; 0.05) were shown. F. Heatmap of the active scores of epithelial cell subtypes as numbered on top, of which expression was regulated by transcription factors (TFs), as estimated using SCENIC analysis. Shown are the top 10 TFs having the highest difference in expression regulation estimates between each cluster and all other cells, tested with a Wilcoxon rank-sum test." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-124545678" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;The heterogeneity of epithelial cells inter- and intra-organ tissues.&lt;/span&gt; &lt;strong&gt;A.&lt;/strong&gt; t-SNE plots showed 18,090 epithelial cells, colored according to their origins of tissues (top panel) or cell clusters (bottom panel). &lt;strong&gt;B.&lt;/strong&gt; Dot plotting visualization of the normalized expression of marker genes of each epithelial cluster. The size of the dot represented the percentage of cells with a cell type, and the color represented the average expression level. &lt;strong&gt;C.&lt;/strong&gt; Dendrogram of 33 clusters based on their normalized mean expression values (correlation distance metric, average linkage). Only genes with ln(fold-change) above 0.25, p.adjust &lt; 0.05 and pct.1 &#x2265; 0.2 in each cluster were included in the analysis. &lt;strong&gt;D, E.&lt;/strong&gt; Gene Ontology enrichment analysis results of each epithelial cell cluster in digestive organs (D) and non-digestive organs (E). Cell clustered as numbered below were colored according to their &#x2212;log&lt;sub&gt;10&lt;/sub&gt;&lt;em&gt;P&lt;/em&gt; values. Only the top 20 significant terms (p-value &lt; 0.05) were shown. &lt;strong&gt;F.&lt;/strong&gt; Heatmap of the active scores of epithelial cell subtypes as numbered on top, of which expression was regulated by transcription factors (TFs), as estimated using SCENIC analysis. Shown are the top 10 TFs having the highest difference in expression regulation estimates between each cluster and all other cells, tested with a Wilcoxon rank-sum test.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975/F5.medium.gif" width="302" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975/F5.medium.gif" width="302" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <span class="caption-title">The heterogeneity of epithelial cells inter- and intra-organ tissues.</span><p id="p-50" class="first-child"><strong>A.</strong> t-SNE plots showed 18,090 epithelial cells, colored according to their origins of tissues (top panel) or cell clusters (bottom panel).</p><p id="p-51"><strong>B.</strong> Dot plotting visualization of the normalized expression of marker genes of each epithelial cluster. The size of the dot represented the percentage of cells with a cell type, and the color represented the average expression level.</p><p id="p-52"><strong>C.</strong> Dendrogram of 33 clusters based on their normalized mean expression values (correlation distance metric, average linkage). Only genes with ln(fold-change) above 0.25, p.adjust &lt; 0.05 and pct.1 ≥ 0.2 in each cluster were included in the analysis.</p><p id="p-53"><strong>D, E.</strong> Gene Ontology enrichment analysis results of each epithelial cell cluster in digestive organs (D) and non-digestive organs (E). Cell clustered as numbered below were colored according to their −log<sub>10</sub><em>P</em> values. Only the top 20 significant terms (p-value &lt; 0.05) were shown.</p><p id="p-54"><strong>F.</strong> Heatmap of the active scores of epithelial cell subtypes as numbered on top, of which expression was regulated by transcription factors (TFs), as estimated using SCENIC analysis. Shown are the top 10 TFs having the highest difference in expression regulation estimates between each cluster and all other cells, tested with a Wilcoxon rank-sum test.</p><div class="sb-div caption-clear"></div></div></div><p id="p-55">The epithelial cells were further grouped into 33 clusters (<strong><a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5A</a></strong>). Grouping of close clusters was seen in all organ tissues, expect for C31 (liver and common bile duct) and C29 (the stomach, trachea and small intestine), suggesting the presence of heterogeneity in epithelial cells at both intra- and inter-organ contexts. The 33 clusters showed different expression profiles with 375 signature genes (FC ≥ 5, pct.1 ≥ 0.2 and pct.2 ≤ 0.2), most of which were expressed exclusively in one cluster (<strong><a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig. 5B</a>, Additional file 11: Figure S10C, D, and Additional file 25: Table S7</strong>. Hierarchical clustering analysis revealed closer grouping of cell clusters within tissues than cross tissues (<strong><a id="xref-fig-5-4" class="xref-fig" href="#F5">Fig. 5C</a></strong>). Cells from digestive organs including the small intestine, stomach, rectum and common bile duct were clustered closely, as were cells from non-digestive tissues including the skin, trachea and bladder (<strong><a id="xref-fig-5-5" class="xref-fig" href="#F5">Fig. 5C</a></strong>). Although the esophagus is a digestive organ, the epithelial cells were clustered much closer to the skin cells than cells from digestive organs.</p><p id="p-56">To explore the potential functions of cell from each cluster, we preformed GO analysis for two groups of cells from the 13 digestion-related clusters and the 20 non-digestion-related clusters, separately. For the cells of digestive system, biological functions related to metabolic process, energy synthesis and digestion pathways were commonly observed (<strong><a id="xref-fig-5-6" class="xref-fig" href="#F5">Fig. 5D</a></strong>). For the non-digestion-related cells, <em>cell cycle regulation</em> and <em>epidermis development</em> were the strongly enriched pathways (<strong><a id="xref-fig-5-7" class="xref-fig" href="#F5">Fig. 5E</a></strong>). As we observed the enrichment of immune-related pathways in most of the cell clusters (<strong><a id="xref-fig-5-8" class="xref-fig" href="#F5">Fig. 5D, E</a></strong>), we examined their antigen-presenting abilities, which revealed weak antigen-presenting ability for epithelial cells from the skin and esophagus (<strong>Additional file 11: Figure S10E</strong>).</p><p id="p-57">Next, we investigated the contribution of TFs in regulating the heterogenous transcriptional profiles of epithelial cells in and between organs. SCENIC analysis revealed that the regulation of TFs was similar among cell clusters of a same tissue but was very different for among clusters between tissues (<strong><a id="xref-fig-5-9" class="xref-fig" href="#F5">Fig. 5F</a></strong>). Interestingly, the digestion-associated epithelial clusters exhibited similar activation of TFs, as did the cell clusters belong to non-digestive tissues including the trachea, skin and esophagus (<strong><a id="xref-fig-5-10" class="xref-fig" href="#F5">Fig. 5F</a></strong>). In addition, we also observed some cluster-specific TFs in skin (<em>CEBPA</em>, <em>KLF4</em>, <em>EGR1</em>, <em>GATA3</em> and <em>NFIC)</em>, small intestine epithelial cells (<em>HNF4G</em>, <em>NR1H3</em>, <em>NR1I2</em>, <em>VDR</em> and <em>GATA4)</em>, tuff cells (<em>MTA3</em>, <em>SOX13</em>, <em>POU2F1</em>, <em>ETV7</em> and <em>POU2F3</em>), absorptive cells (<em>HES4</em>), and stem cells (<em>ASCL2</em>).</p></div><div id="sec-11" class="subsection"><h3>The similarity and heterogeneity of stromal cells</h3><p id="p-58">Endothelial cells (ECs) line up in a monolayer and form the interior surface of blood and lymphatic vessels as well as heart chambers. We identified a total of 7,137 ECs, including 6,863 blood endothelial cells (BECs, marked with <em>VWF</em>) and 274 lymphatic endothelial cells (LECs, marked with <em>LYVE1</em>). BECs and LECs could be further grouped into 13 and two clusters, respectively, which had unique gene signatures (<strong>Additional file 12: Figure S11A-C and Additional file 26: Table S8</strong>). Hierarchical cluster analysis revealed that the cell clusters from a same tissue were grouped closer than those between two different tissues (<strong>Additional file 12: Figure S11D</strong>), such as SELE_BECs and APOD_BECs from the skin, as well as MADCAM1_BEC and CD320_BEC from the common bile duct. Interestingly, a group of LECs (FCN3_LECs) were strictly identified in the liver and expressed only two of the four genes for LECs (<em>PECAM1</em> and <em>LYVE1</em>) and other liver-specific markers (<em>CD4</em>, <em>CD14</em>, <em>FCN2/3</em>, <em>OIT3</em> and <em>CLEC4G</em>), but not the other two genes for LECs (<em>PDPN</em> and <em>PROX1</em>[<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>]; <strong>Additional file 12: Figure S11B, E</strong>). The other group of LECs (CCL21_LEC) were from tissues other than the liver, with exclusively high expression of <em>CCL21</em>, the protein of which binds the chemokine receptor 7 (<em>CCR7</em>) and thereby promotes adhesion and migration of various immune cells[<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>]. This suggests that these LECs have higher potential to attract immune cells than the LECs in the liver. GO analysis results revealed that BECs and LECs from most of the clusters had common endothelial functions including <em>vasculature development</em> and <em>response to wounding</em>, as well as functions regulating immune response (<strong>Additional file 12: Figure S11F</strong>). Moreover, the cells from each cluster were shown to have specific biological functions, including <em>endocrine and hormone regulation</em> for liver BECs (TIMP1_BEC), <em>adaptive immune response</em> for heart BECs (ACKR1_BEC) and <em>regulation of inflammatory response</em> for liver LECs (FCN3_LEC). We further examined the antigen-presenting ability of cells from each cluster, which revealed higher ability for BECs from the skin (SELE_BEC and APOD_BEC, <em>P</em> &lt; 2.2×10<sup>−16</sup> and <em>P</em> = 0.003354, respectively) than that from other organs, and higher ability for the LECs in the liver (FCN3_LEC; <em>P</em> = 2.61×10<sup>−13</sup>) than the LECs from other tissues (<strong>Additional file 12: Figure S11G</strong>).</p><p id="p-59">We also identified a total of 17,835 fibroblasts and smooth muscle cells from nine tissues. These cells were further grouped into 14 fibroblast clusters (11,767 cells, <em>MMP2</em>), four smooth muscle cell clusters (3,201 cells, <em>ACTA2</em>), and another five clusters assigned as FibSmo (2,867 cells; marked with <em>MMP2</em> and <em>ACTA2</em>; <strong>Additional file 13: Figure S12A, B</strong>). We observed organ-specific distribution for fibroblasts of different clusters, but mixture of multiple organs for smooth muscle cell clusters (<strong>Additional file 13: Figure S12A, B</strong>), which is consistent with the previous findings in a mouse model[<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>]. DEG analysis revealed that cells from each cluster had unique signature (<strong>Additional file 13: Figure S12C and Additional file 26: Table S8</strong>). Hierarchical cluster analysis showed that cells from clusters were grouped as organ-specific manner, namely close distance for those within a same tissue (<strong>Additional file 13: Figure S12D</strong>). GO analysis revealed that fibroblasts, smooth muscle cells and FibSmo cells shared classical functions, including <em>blood vessel development</em>, <em>negative regulation of cell proliferation</em>, <em>response to wounding</em>, <em>regulation of cellular component movement</em> and <em>cell-substrate adhesion</em>, together with additional functions for cells from each cluster (<strong>Additional file 13: Figure S12E</strong>).</p></div><div id="sec-12" class="subsection"><h3>ACE2 expression across different tissues</h3><p id="p-60">ACE2 is considered as a potential receptor for the most recent outbreaking coronavirus (HCoV-19)[<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>, <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>]. We therefore examined the expression of <em>ACE2</em> in our dataset containing 43 cell clusters and observed extremely high expression of <em>ACE2</em> in only two clusters, namely small intestine epithelial cells (C15) and common bile duct epithelial cells (C26; <strong><a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1C</a>, Additional file 4: Figure S3B and Additional file 14: Figure S13A, B</strong>). The majority of ACE2-expressing cells were epithelial cells, which were detected in 11 of 15 organs, including common bile duct, rectum, skin and muscle, and other seven tissues recently reported (esophagus, trachea, small intestine, heart, liver, stomach and bladder[<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>]; <strong>Additional file 14: Figure S13B bottom panel</strong>). High expression of <em>ACE2</em> was observed in the heart (~3.01%), small intestine (~24.7%) and common bile duct (~19.09%), while low expression was observed in most of other tissues (&lt; 1% <em>ACE2</em> positive cells; <strong>Additional file 14: Figure S13B bottom panel</strong>). Specifically, we observed high expression of <em>ACE2</em> in the fibroblasts of the heart and the smooth muscle cells of the heart and muscle (<strong>Additional file 14: Figure S13A, B</strong>). These observations are consistent with the enquiry results from the Human Protein Atlas database (<a href="https://www.proteinatlas.org">https://www.proteinatlas.org</a>) and recent reports[<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>–<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>]. Moreover, in common bile duct tissue, GO enrichment analysis revealed enrichment of cell adhesion and immune-related pathways in ACE2-positive cells, including <em>neutrophil activation involved in immune response</em> (<em>ANXA2</em>, <em>ASAH1</em>, <em>B2M</em>, <em>FTL</em>, <em>LYZ</em>, <em>MGST1</em>, <em>CEACAM6</em>, <em>DNAJC3</em>, <em>TCN1</em>, and <em>ATP6AP2</em>), <em>cell-cell adhesion via plasma-membrane adhesion molecules</em> (<em>DSG2</em>, <em>ITGB1</em>, <em>EPCAM</em>, <em>CEACAM6</em>, <em>PTPRF</em>, and <em>CD164</em>; <strong>Additional file 26: Table S8</strong>).</p></div><div id="sec-13" class="subsection"><h3>Complex and broad intercellular communication networks within tissues</h3><p id="p-61">Since we observed heterogeneity for cells in each organ tissue, we explored their potential intercellular communication network. First, we examined molecular interactions based on ligand-receptor pairs in each tissue using CellPhoneDB. We observed a total of 19,742 significant interactions based on 468 ligand-receptor pairs among cell types and tissues, which varied from 252 in the lymph node to 2,725 in the trachea (<strong><a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig. 6A</a> and Additional file 27: Table S9</strong>). Among them, <em>MIF_CD74</em>, <em>HBEGF_CD44</em>, <em>CD55_ADGRE5</em>, <em>MIF_TNFRSF14</em> and <em>APP_CD74</em> were the top five frequent interacting pairs detected across different cell types (<strong><a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig. 6B</a></strong>), suggesting their important roles in mediating crosstalk between different cell types. Next, we focused on the interactions between any pairs of the major cell types including CD4<sup>+</sup> T, CD8<sup>+</sup> T, NK, B, plasma, myeloid, epithelial, endothelial, smooth muscle cells, and fibroblasts (<strong><a id="xref-fig-6-3" class="xref-fig" href="#F6">Fig. 6C</a></strong>). We observed that myeloid cell was the most active cell type interacting with the other types of cells (6,695 inter-cell interactions), among which the most active intensive interactions were between myeloid cells and epithelial cells (8.1% of the total inter-cell interactions) in the skin and trachea (52.6%). Interestingly, the interactions were common between myeloid cells and all kinds of epithelial cells, with the most frequent ligand-receptor pair <em>CD74_MIF</em> (<strong>Additional file 15: Figure S14</strong>), dysregulations of which have been reported in several tumours[<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>, <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>], suggesting their potential roles in tumorigenesis. CD8<sup>+</sup> Tcells were among the most active cell types interacting with other types of cells (total inter-cell 4511 interactions), among which the most active interactions were between CD8<sup>+</sup> T and myeloid cells (2.2% of the total inter-cell interactions). The interactions were found mainly in the liver, rectum, and spleen (52.1%). The frequent interaction pairs between CD8<sup>+</sup> T cell and myeloid cells were <em>RPS19_C5AR1</em>, <em>MIF_CD74</em>, <em>CD55_ADGRE5</em>, <em>ITGAL_ICAM1</em>, <em>aLb2 complex_ICAM1</em>, <em>CD99_PILRA</em> and <em>ANXA1_FPR1</em>, most of which play important roles in immune regulation[<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>–<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>] (<strong>Additional file 16: Figure S15</strong>). CD8<sup>+</sup> T cells also had broad interactions with non-immune cells (<strong>Additional file 17: Figure S16</strong>). For instance, the most frequent interacting cytokine and receptor pair <em>CXCL12_CXCR4</em>[<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>] was observed between fibroblasts and CD8<sup>+</sup> T cells, suggesting the attractive potential of fibroblasts for T cells migration into tissues (<strong>Additional file 17: Figure S16</strong>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Intercellular communication networks within tissues. A. Counts of significant ligand-receptor interactions in each organ as indicated in the x axis. B. Frequency of each ligand-receptor interaction in among all cell types. Shown are interactions with a frequency larger than 50. C. Counts of interactions between a major cell type as subtitled to other major cell types as indicated in the colored circles, as well as that within the major cell type. CD4, CD4+ T cells; CD8, CD8+ T cells; B, B cells; Plasma, plasma cells; Myeloid, myeloid cells; NK, NK cells; Epi, epithelial cells; Fib, fibroblasts; Smo, smooth muscle cells; FibSmo, FibSmo cells; and Endo, endothelial cells). Numbers in red indicate the counts of ligand-receptor pairs for each intercellular link." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-124545678" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Intercellular communication networks within tissues.&lt;/span&gt; &lt;strong&gt;A.&lt;/strong&gt; Counts of significant ligand-receptor interactions in each organ as indicated in the x axis. &lt;strong&gt;B.&lt;/strong&gt; Frequency of each ligand-receptor interaction in among all cell types. Shown are interactions with a frequency larger than 50. &lt;strong&gt;C.&lt;/strong&gt; Counts of interactions between a major cell type as subtitled to other major cell types as indicated in the colored circles, as well as that within the major cell type. CD4, CD4&lt;sup&gt;+&lt;/sup&gt; T cells; CD8, CD8&lt;sup&gt;+&lt;/sup&gt; T cells; B, B cells; Plasma, plasma cells; Myeloid, myeloid cells; NK, NK cells; Epi, epithelial cells; Fib, fibroblasts; Smo, smooth muscle cells; FibSmo, FibSmo cells; and Endo, endothelial cells). Numbers in red indicate the counts of ligand-receptor pairs for each intercellular link.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975/F6.medium.gif" width="440" height="393"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975/F6.medium.gif" width="440" height="393"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.996975/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6.</span> <span class="caption-title">Intercellular communication networks within tissues.</span><p id="p-62" class="first-child"><strong>A.</strong> Counts of significant ligand-receptor interactions in each organ as indicated in the x axis.</p><p id="p-63"><strong>B.</strong> Frequency of each ligand-receptor interaction in among all cell types. Shown are interactions with a frequency larger than 50.</p><p id="p-64"><strong>C.</strong> Counts of interactions between a major cell type as subtitled to other major cell types as indicated in the colored circles, as well as that within the major cell type. CD4, CD4<sup>+</sup> T cells; CD8, CD8<sup>+</sup> T cells; B, B cells; Plasma, plasma cells; Myeloid, myeloid cells; NK, NK cells; Epi, epithelial cells; Fib, fibroblasts; Smo, smooth muscle cells; FibSmo, FibSmo cells; and Endo, endothelial cells). Numbers in red indicate the counts of ligand-receptor pairs for each intercellular link.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-14"><h2 class="">DISCUSSION</h2><p id="p-65">Here we generated for the first time to our knowledge an adult human cell atlas (AHCA), by profiling the single cell transcriptome for 88,622 cells from 15 organs. The AHCA included 234 subtypes of cells, each of which was distinguished by multiple marker genes and transcriptional profiles and collectively contributed to the heterogeneity of major human organs. The AHCA empowered us to explore the developmental trajectories of major cell types and identify regulators and interacting networks that might play important roles in maintaining the homeostasis of human body. We have provided the AHCA publicly available as a resource to uncover key events during the development of human disease in the context of heterogeneity of cells and organs.</p><p id="p-66">It has been demonstrated that T<sub>N</sub> cells are generated at the thymus and populate lymphoid tissues sites where they differentiate to T<sub>EFF</sub> cells upon antigen stimulus, and subsequently develop into long-lived memory T cells[<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>]. However, how T cells develop into different state throughout human organs and the links across T cells of different states as well as the underlying regulatory networks, are largely unknown, especially in the context of one individual body. In our study, trajectory analysis revealed a clear development route from T<sub>N</sub> cells into T<sub>EFF</sub> cells and then CD4<sup>+</sup> and CD8<sup>+</sup> T<sub>RM</sub> cells in non-lymphoid organs with terminal development states, which is consistent with the previous findings[<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">57</a>]. We also noted that T<sub>RM</sub> cells exhibit development states at organ-specific patterns and consistently these cells were regulated by different types of transcription factors, suggesting that tissue microenvironment might regulate gene expression by affecting transcription factor’s activity, which in turn shape the specific T cell phenotypes. TCR analysis tracking cell of same linage revealed widespread links among subpopulations of T<sub>RM</sub> cells, which however, are considered as non-recirculating[<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>]. Moreover, intensive sharing of TCRs were observed among T<sub>RM</sub> cells, T<sub>EM</sub> cell and T<sub>EFF</sub> cells (<strong>Additional file 8: Figure S7H, I</strong>). Together with their development states, these results suggest that T<sub>EM</sub> cells and T<sub>EFF</sub> cells might enter the tissues and developed into T<sub>RM</sub> cells and IEL T cells.</p><p id="p-67">Infiltrating macrophages come from classical monocytes in pathological settings, such as cancer[<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>], while various origins of adult macrophages among tissues in the steady state have been reported[<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a>]. As such, how macrophages are renewed in maintenance of their hemostasis in tissues has been debated over whether the tissue-resident macrophages undergo local proliferation or recruitment of monocytes from peripheral blood[<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a>, <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a>]. In our study, multiple observations suggest that macrophages in organs are derived from either circulating monocytes or in-site expansion and proliferation of macrophage populations coping with local microenvironment. Consistently, entry of monocytes to steady-state nonlymphoid organs and self-maintenance of tissue macrophages have been reported in mouse models [<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a>, <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a>]. Moreover, we observed a more isolated clustering of epithelial cells and stromal cells compared to that of the immune cells, which have circulating capability. Because epithelial cells and stromal cells are a fundamental component forming protective barriers and supporting matrix for many organs, disruptions of their homeostasis have been implicated in various diseases[<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a>–<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">68</a>]. Accumulating studies have demonstrated remarkably functional heterogeneity of epithelial cells among tissues[<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">69</a>–<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">71</a>]. Consistently, our gene ontology enrichment analyses revealed very diverse functions of epithelial cells among different tissues, as well as stromal cells. Taken together, higher degree of heterogeneity might reflect higher degree of terminal differentiation states and distinct specific functions of these cells among different organs.</p><p id="p-68">The AHCA not only brings more detailed understanding of cell development and heterogeneity, but also reveals novel cell types and genes as well as regulatory factors that might be important for cell development. We identified a small proportion of novel cells in the skin tissue, in addition to the majority of immune cells, myeloid cells, epithelial cells and stromal cells. Transcription factors are known as the ‘master regulators’ for gene expression[<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">72</a>, <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">73</a>] and we identified numerous additional novel transcription factors in regulating the development of different cell state of major cell types, such as <em>CREM</em> in CD4<sup>+</sup> T<sub>RM</sub>, <em>MAFF</em> in CD8<sup>+</sup> T<sub>RM</sub>, and <em>ELF1</em> in both CD4<sup>+</sup> and CD8<sup>+</sup> T<sub>RM</sub>, <em>CTCF</em> and <em>TGIF1</em> in B cells, <em>KLF13</em> in plasma cells, as well as <em>BHLHE40, MYC, YY1, FOS, SOX2, BCL11A</em> and <em>KLF5</em> in non-digestive tissues containing the trachea, skin and esophagus. Such TFs networks not only extend our understanding for how they regulate gene expression and shape the different phenotypes, but also provide potential gene combinations in reprogramming application. The AHCA also provides useful data to explore the cellular networks at a single-cell resolution. We note that a large number of interactions between immune cells and other cells in all tissues, reflecting essential and broad communications between immune cells and other cell types in human body. In addition, we also observed that epithelial cells had the most frequent intra-cell interactions among all cell types, suggesting that epithelial cells could interact with each other intensively to regulate their biology functions in each organ (<strong>Additional file 18: Figure S17</strong>).</p><p id="p-69">Concerning the recent outbreak of COVID-19 caused by the new human coronavirus HCoV-19[<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">74</a>], we attempted to explore the usage of the AHCA in understanding the infection of this virus and the disease. The clinical course of HCoV-19 infected patients remains to be fully characterized, the disease pathogenesis is still largely unknown, and no pharmacological therapies of proven efficacy yet exist. The common symptoms include fever, fatigue, myalgia, dry cough, anorexia and shortness of breath[<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">75</a>]. The complications include acute respiratory distress syndrome, arrhythmia, acute cardiac injury, shock and acute kidney injury. Some severe patients died of multiple organs failure. Our dataset revealed <em>ACE2</em> expression in multiple cell types and organs, which overlapped with the suffered organs (lung, heart, trachea, stomach, small intestine, artery and muscle) of HCoV-19 infection[<a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">75</a>]. Moreover, we observed GO enrichment pathway of <em>neutrophil activation involved in immune response</em> in ACE2-positive cells, which might contribute to neutrophilia in considerable proportion of patients (38%)[<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">76</a>] and much severe neutrophilia in non-survivors[<a id="xref-ref-75-3" class="xref-bibr" href="#ref-75">75</a>]. Given that <em>ACE2</em> is likely a receptor for the coronavirus[<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">43</a>, <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">44</a>], these data suggest that ACE2-positive cells are the targets of the virus, by blocking the <em>ACE2</em> functions or triggering cell death. The skin and common bile duct also contain ACE2-positie cells, and thereby we note that these organs might also be the targets of the virus. Collectively, these findings provide hints for understanding the pathogenesis of HCoV-19 infection.</p><p id="p-70">We acknowledged that current AHCA has several limitations. First, although we obtained a sufficient number of sequencing reads for each sample, the number of genes detected in each cell was limited. This might underestimate the roles of some low expressed genes, such as long non-coding RNAs. Second, we obtained around 5,000 cells in average for each organ, which might limit our ability to identify rare cell types and thus underestimate the heterogeneity of intercell interactions in organs. Third, we included only 15 organ tissues from a single donor in our study. Further studies on gene expression profiling at both transcriptional and protein levels, as well as functional characterization with more organs from a larger number of donors, would provide much broader and detailed global view of human cell atlas and cell biology.</p></div><div class="section" id="sec-15"><h2 class="">Conclusions</h2><p id="p-71">We generated an adult human cell atlas, by profiling the single cell transcriptome for more than 88,000 cells of 15 organs from one research-consent donor. The AHCA uncovers the heterogeneity of cells in major human organs, containing more than 234 subtypes of cells. Comprehensive analyses of the AHCA enabled us to delineate the developmental trajectories of major cell types and to identify novel cell types, regulators, and key molecular events that might play important roles in maintaining the homeostasis of human body or otherwise developing into human diseases.</p></div><div class="section" id="sec-16"><h2 class="">MATERIALS AND METHODS</h2><div id="sec-17" class="subsection"><h3>Organ tissue collection</h3><p id="p-72">An adult male donor who died of traumatic brain injury was recruited with written consent from his family. The management of organs was complied with the organ procurement guidelines and the study was approved by the Ethics Committee of the First Affiliated Hospital of Sun Yat-sen University. Besides the organs for transplantation purpose, we collected tissues from 15 organs in sequence, including blood, bone marrow, liver, common bile duct, lymph node (hilar and mesenteric), spleen, heart (apical), urinary bladder, trachea, esophagus, stomach, small intestine, rectum, skin and muscle (thigh). All hollow viscera tissues were dissected according to the whole layer structure, and all parenchymal viscera tissues were obtained from the organ lower pole. All the tissue collection procedures were accomplished within 20 minutes to maximize cell viability. To avoid cross-contamination, we used different sets of sterilized surgical instruments. The blood and bone marrow samples were loaded into 10 ml anticoagulation tubes containing EDTA (BD Biosciences, Cat. no. BD-366643) and other tissues were placed in physiological saline (4°C) to wash away the blood and secretions, and then immediately in a D10 resuspension buffer, containing culture medium (DMEM medium; Gibco™, Cat. no. 11965092) with 10% fetal bovine serum (FBS; Gibco™, Cat. no. 10099141). All tissue samples were kept on ice and delivered to the laboratory within 40 mins for further processing.</p></div><div id="sec-18" class="subsection"><h3>Tissue dissociation and cell purification</h3><p id="p-73">All tissues were dissociated within 1.5 hours and viable cells were collected at the end using fluorescence-activated cell sorting (FACS; BD FACS Aria™ III). For solid tissues exclusive of liver, each fresh tissue was cut into 1 mm pieces and incubated with a proper digestive solution of enzyme cocktail (<strong>Additional file 19: Table S1</strong>), followed by neutralization with D10 and then passed through a 40 μm cell strainer (BD, Cat. no. 352340). The cell suspension was centrifuged at 300x g for 5 min at 4 °C, and the pellet was resuspended with 0.8% NH<sub>4</sub>Cl (Sigma-Aldrich, Cat. no. 254134-5G) red blood cells lysis buffer (RBCL) on ice for 10 min, followed by an additional wash with D10. The liver tissue was cut into pieces of 3-4 mm and incubated with 1 mM EGTA (Sigma-Aldrich, Cat. no. E0396-10G) in 1 × DBPS (Gibco™, Cat. no. 14190250) for 10 min at 37 °C with rotation at 50 rpm. After a washing with 1 × DPBS to remove EGTA, tissue was then incubated in pre-warmed digestion buffer (Supplementary information, Table S1) with rotation at 100 rpm at 37 °C for 30 min. The liver cell suspension was carefully passed through a 70 mm nylon cell strainer (BD, Cat. no. 352350), which was further centrifuged at 50× g for 3 min at 4°C to pellet hepatocytes. The supernatant was centrifuged at 300× g for 5 min at 4°C to pellet non-parenchymal cells. The pellet was resuspended and treated with RBCL. The blood and bone marrow samples were pelleted by centrifugation at 300x g for 5 min at 4 °C and was resuspended with RBCL on ice for 10 min, followed by an additional wash with D10. All cells from each tissue were resuspended with D10 to a concentration of 50-500 million cells per milliliter and stained with Calcein AM (Component A: AM) and Ethidium homodimer-1 (Component B: EH) in LIVE/DEAD Viability/Cytotoxicity Kit (Invitrogen, Cat. no. L3224) for 20 min on ice. Only the AM<sup>+</sup>EH<sup>−</sup> cells were collected by FACS for each tissue.</p></div><div id="sec-19" class="subsection"><h3>cDNA library preparation</h3><p id="p-74">The concentration of single cell suspension was determined using Cellometer Auto 2000 (Cellometer) and adjusted to 1,000 cells/μl. Appropriate number of cells were loaded into CHROMIUM instrument (10X Genomics, CA, USA) according to the standard protocol of the Chromium single cell V(D)J kit in order to capture 5,000 cells to 10,000 cells/chip position. In brief, mRNA transcripts from each sample were ligated with barcoded indexes at 5’-end and reverse transcribed into cDNA, using GemCode technology (10X Genomics, USA). cDNA libraries including the enriched fragments spanning the full-length V(D)J segments of T cell receptors (TCR) or B cell receptor (BCR), and 5’-end fragments for gene expression were separately constructed, which were subsequently subjected for high-throughput sequencing.</p></div><div id="sec-20" class="subsection"><h3>Single cell RNA-seq data processing</h3><p id="p-75">5’-end cDNA, TCR and BCR libraries were mixed and subjected for sequencing on Illumina HiSeq XTen instruments with pared-end 150 bp. Raw data (BCL files) from HiSeq platform was converted to fastq files using Illumina-implemented software bcl2fastq (version v2.19.0.316). cDNA reads were aligned to the human reference genome (hg38) and digital gene expression matrix was built using STAR algorithm in CellRanger (“count” option; version 3.0.1; 10X Genomics)[<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">77</a>]. TCR and BCR reads were aligned to human reference VDJ dataset (<a href="http://cf.10Xgenomics.com/supp/cell-vdj/refdata-cellranger-vdj-GRCh38-alts-ensembl-2.0.0.tar.gz">http://cf.10Xgenomics.com/supp/cell-vdj/refdata-cellranger-vdj-GRCh38-alts-ensembl-2.0.0.tar.gz</a>) using CellRanger (“vdj” option; version 3.1.0; 10X Genomics). Parameters were set as default except for “force-cells” as 13,000. Raw digital gene expression matrix in the “filtered_feature_bc_matrix” file folder generated by CellRanger was used for further analysis.</p></div><div id="sec-21" class="subsection"><h3>Cell clustering and differential gene expression analysis</h3><p id="p-76">Quality control filtering, variable gene selection, dimensionality reduction and clustering for cells were performed using R software (version 3.6.1; <a href="https://www.r-project.org">https://www.r-project.org</a>) and Seurat package[<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>] (version 3.0.1; <a href="https://satijalab.org/seurat">https://satijalab.org/seurat</a>) with default settings unless otherwise stated. For each tissue, we removed cells with low quality cells (UMI &lt; 1,000, gene number &lt; 500, and mitochondrial genome fragments ≥ 0.25) and genes with rare frequencies (0.1% of all cells). From the remaining cells, gene expression data for each sample was subtracted and scaled using “ScaleData” function (negative binomial model). Principal Component Analysis (PCA) and t-SNE implemented in the “RunPCA” and “RunTSNE” functions, respectively, were used to identify the deviations among cells. Genes with high variations were identified using “FindVariableGenes” and included for PCA (mean.cutoff ≥ 0.1, dispersion.cutoff ≥ 0.5). We used different number of Principal Components (PCs) and value of perplexity determined by elbow plot for each tissue and cell type (<strong>Additional file 28: Table S10</strong>). Cell clusters were identified using “FindClusters” function and shown using t-SNE with the same number of PCs. Differential expression markers or genes were determined using Wilcoxon test implemented in “FindAllMarkers” function, which was considered significant with an average natural logarithm (fold-change) of at least 0.25 and a Bonferroni-adjusted p-value lower than 0.05. Subsequently, the candidate markers were reviewed and were used to annotate cell clusters. For analyses of the merged data from all tissues, we used 30 PCs and resolution parameter set to 1 for cell clustering. For further analyses, we removed cell clusters that had multiple well-defined marker genes of different cell types, and with an extremely high number of UMIs (<strong>Additional file 21: Table S3</strong>).</p></div><div id="sec-22" class="subsection"><h3>Identification and removal of highly transcribed genes with contamination potentials</h3><p id="p-77">Because we observed that some cell-specific genes were broadly expressed among all types of cell in a tissue, for example, <em>APOC3</em> in enterocytes cells in small intestine with an average more than 200 UMIs in a cell, we suspected that if a fraction of a certain type of cells were broken during the sample processing, cell-specific genes with high transcriptions would be released and thus contaminated all cell droplets. Especially, such genes would screw the differential expression analysis of cell type among all tissues. Therefore, we identified these genes and removed them for comparation analyses within major cell types. We assumed that non-epithelial cells from a same linage have similar gene profiles at a certain degree and such that a few genes would have modest expression in only one organ. Here given an example for T cells, we grouped together the cells previously labeled with NK/T, T, and immune cells (<strong>Additional file 21: Table S3</strong>) in each tissue. Next, we determined the top 1% of genes with high transcription in each tissue sample based on the total number of UMIs and marked these as potentially contaminating genes. We further randomly sampled 300 cells for each cell type in a tissue and as such generated artificial data for all tissues, with which differential expression gene analysis was performed using “FindAllMarker” function. Any gene with normal logarithm of FC above 0.25 and with an expression in less than 5% cells of all other tissue samples was considered contaminating gene. After removing the above contaminating genes, we performed cluster analysis with the first 20 PCs and resolution of 1.5. After further removal of cell clusters that had multiple well-defined marker genes of different cell types, we repeated cluster analysis using a lower resolution setting (<strong>Additional file 28: Table S10</strong>) to identify CD4<sup>+</sup>, CD8<sup>+</sup> and NK cell (<strong>Additional file 22: Table S4</strong>) clusters.</p><p id="p-78">For B cells, plasma cells, endothelial cells, macrophages, monocytes and fibroblasts, we applied a similar strategy to remove contaminating genes and cell clusters with multiple cell-specific markers.</p></div><div id="sec-23" class="subsection"><h3>Trajectories analysis</h3><p id="p-79">We performed trajectories analysis using Monocle3 alpha[<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>] for all tissue-derived T cells, macrophages/monocytes, according to the general pipeline (<a href="http://cole-trapnell-lab.github.io/Monocle-release/Monocle3/">http://cole-trapnell-lab.github.io/Monocle-release/Monocle3/</a>). For T cells, after identification of T cells clusters for CD4<sup>+</sup> and CD8<sup>+</sup> T, raw gene expression counts of cells were imported to the software. Only genes matching the thresholds (both mean expression and dispersion ratio are greater than 0.1 and 0.2, respectively.) were used for cell ordering and training the pseudo-time trajectory. For trajectory analysis of macrophages/monocytes, we used different thresholds for genes selection (mean expression ≥ 0.3, dispersion ration ≥ 0.1).</p></div><div id="sec-24" class="subsection"><h3>TCR and BCR analysis</h3><p id="p-80">We assessed the enrichment of TCR and BCR in various organs using R package Startrac (version 0.1.0)[<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>], which included only the cells with the certain clonotypes assigned by Cellranger and with paired chains (α and β for T cells, heavy and light chains for B cells). In brief, cells sharing identical TCR or BCR clones between tissues were measured using migration-index score, and the degree of cell linking between different clusters of T cells or B cells was determined by transition-index score. TCR or BCR diversity (Shannon-index score) was calculated using a “1 - expansion-index score”. For detailed pipeline, please referred to the website (<a href="https://github.com/Japrin/STARTRAC/blob/master/vignettes/startrac.html">https://github.com/Japrin/STARTRAC/blob/master/vignettes/startrac.html</a>).</p></div><div id="sec-25" class="subsection"><h3>Presenting-antigen and proliferated scores</h3><p id="p-81">Presenting-antigen ability score (PAS) and proliferated score of each cell were calculated using “AddModuleScore” function implemented in the Seurat package, with gene sets in REACTOME database “MHC CLASS II ANTIGEN PRESENTATION” pathway, and proliferation-association genes (including <em>ZWINT, E2F1, FEN1, FOXM1, H2AFZ, HMGB2, MCM2, MCM3, MCM4, MCM5, MCM6, MKI67, MYBL2, PCNA, PLK1, CCND1, AURKA, BUB1, TOP2A, TYMS, DEK, CCNB1</em> and <em>CCNE1</em>)[<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">78</a>], respectively.</p></div><div id="sec-26" class="subsection"><h3>Gene Ontology (GO) pathway enrichment analysis</h3><p id="p-82">All GO enrichment analysis was performed using the online tools metascape[<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">79</a>] with “multiple gene list mode” (<a href="http://metascape.org/gp/index.html">http://metascape.org/gp/index.html</a>). For epithelial cells, we selected genes with FC ≥ 2 and p.adjust &lt; 0.05 for each tissue, while FC ≥ 1.5, p.adjust &lt; 0.05, and ptc.1 &gt; 0.2 for each cluster. For non-epithelial cells, we selected genes with lnFC ≥ 0.25, p.adjust &lt; 0.05 and ptc.1 &gt; 0.2 in each cluster. Only the genes ranked top 150 according the FC were used for comparisons in each tissue and subpopulation. The background was given as all the genes expressed in corresponding cell types. In addition, only the gene sets in the “GO Biological Processes” were considered.</p></div><div id="sec-27" class="subsection"><h3>Cellular interaction analysis</h3><p id="p-83">To investigate the cellular interaction, we identified the inferred paired molecules using CellphoneDB software (version 2.0)[<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>] with default parameters. To evaluate the interaction between major cell types, we first merged clusters within major cell types including T, B, NK, myeloid, endothelial cells, fibroblasts, smooth muscle cells and FibSmo cells within each tissue. Next, the merged cell types were as follows, CD4<sup>+</sup> T cells (T<sub>N</sub>, T<sub>CM</sub>, T<sub>reg</sub>, T<sub>EM</sub>, T<sub>EFF</sub>, T<sub>RM</sub>), CD8<sup>+</sup> T cells (T<sub>N</sub>, T<sub>CM</sub>, T<sub>EM</sub>, T<sub>EFF</sub>, T<sub>RM</sub>, IEL, γδ, MAIT). B cells (naïve and memory B cells), NK cells, plasma cells, myeloid (monocytes: Mon, macrophages: Mac and DCs), Endothelial cells (BECs and IECs), fibroblasts (Fib), smooth muscle cells (Smo), and FibSmo cells (FibSmo). We did not merge subpopulations of epithelial cells because of their high degree of heterogeneity in each tissue. Considering the test efficiency and computational burden, we focused on cell types with more than 30 cells and only randomly selected 250 cells of each cell type for analysis in each tissue. The significant ligand-receptor pairs were filtered with a <em>P</em> value less than 0.05. All the analyses above were performed as tissue independent. Visualization of interaction network was done using Cytoscape (version 3.7.0).</p></div><div id="sec-28" class="subsection"><h3>Single-cell regulatory network inference and clustering (SCENIC) analysis</h3><p id="p-84">We conducted SCENIC analysis on cells passed the filtering for each major cell types, using R package SCENIC (version 1.1.2.2) as previously described[<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">80</a>]. Regions for TFs searching were restricted to 10k distance centered the transcriptional start site (TSS) or 500bp upstream of the TSSs. Transcription factor binding motifs (TFBS) over-represented on a gene list and networks inferring were done using R package RcisTarget (version 1.4.0) and GENIE3 (version 1.6.0), respectively, with 20-thousand motifs database. We randomly selected no more than 250 cells for each cell cluster. The input matrix was the normalized expression matrix from Seurat. The cluster-specific TFs of one cluster were defined as the top 10 or 15 highly enriched TFs according to decreasing fold change compared to all other cell clusters using a Wilcoxon rank-sum test.</p></div><div id="sec-29" class="subsection"><h3>Analysis of differential pathway</h3><p id="p-85">Gene set variation analysis (GSVA) or gene set enrichment analysis (GSEA) was performed to identify significantly enriched or depleted groups of genes in each transcriptional dataset, using R package GSVA (version 1.32.0) or GSEA software (version 4.0.3) (<a href="https://www.gsea-msigdb.org/gsea/index.jsp">https://www.gsea-msigdb.org/gsea/index.jsp</a>) on the 50 hallmark pathways with default parameters, respectively.</p></div><div id="sec-30" class="subsection"><h3>Data sharing and code availability</h3><p id="p-86">The key raw data have been deposited in the Research Data Deposit (RDD; <a href="http://www.researchdata.org.cn">http://www.researchdata.org.cn</a>). The R scripts are available at <a href="https://github.com/bei-lab">https://github.com/bei-lab</a> or upon request.</p></div></div><div class="section" id="sec-32"><h2 class="">Conflict of Interests</h2><p id="p-87">The authors declare no conflict of interest.</p></div><div class="section" id="sec-33"><h2 class="">AUTHORSHIP CONTRIBUTIONS</h2><p id="p-88">J.-X.B., X.S.H. and Z.Y.G. conceived and supervised this study. S.H., L.-H.W., Y.L., Y.-Q.L., J.H.X., H.T.C., W.P., G.-W.L., P.P.W., B.L., X.X.J. and D.W. performed sample preparation, library construction, and data generation. S.H. performed bioinformatics analyses. S.H., J.-X.B. and Z.Y.G. wrote the manuscript. All authors read and approved the final manuscript.</p></div><div id="sec-34" class="subsection"><p id="p-90"><strong>Figure S1. Quality information of single-cell RNA sequencing, related to <a id="xref-fig-1-6" class="xref-fig" href="#F1">figure 1</a>.</strong></p><p id="p-91">Box plots showed the numbers of UMI and genes detected in each cell of each organ.</p><p id="p-92"><strong>Figure S2</strong>. <strong>Distribution of cells in each organ tissue, related to <a id="xref-fig-1-7" class="xref-fig" href="#F1">figure 1</a>.</strong></p><p id="p-93">Bar plots on the left panels showed the numbers of cells of different cell types, and the t-SNE plots on the right panels showed their relevant clustering. Cell types were determined by differential gene expression of known markers between clusters. The colors for each cell type are consistent between the bar plot and t-SNE plot for a same organ. Each dot in the t-SNE plot represented a single cell.</p><p id="p-94"><strong>Figure S3. t-SNE plot of the cell clusters in different organ samples, related to <a id="xref-fig-1-8" class="xref-fig" href="#F1">figure 1</a>.</strong></p><p id="p-95"><strong>A.</strong> t-SNE plots showed cells in skin colored by normalized expression of <em>COCH</em>, <em>SCGB2A2</em>, <em>SCGB1B2P</em>, <em>PIP</em>, <em>MUCL1</em>, and <em>DCD</em> genes.</p><p id="p-96"><strong>B.</strong> t-SNE plot showed cell clusters in 15 organs (88,622 cells). Each dot in the t-SNE plot represented a single cell, and each color-coded region indicated one cluster.</p><p id="p-97"><strong>Figure S4. Correlation analysis of cell types determined by whole dataset (all cell of 15 organs) and manual annotation in each organ, related to <a id="xref-fig-1-9" class="xref-fig" href="#F1">figure 1</a>.</strong></p><p id="p-98">x axis indicated clusters from main <strong><a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1C</a></strong>, and the y axis indicated clusters annotated manually, which were determined using cells from all tested 15 organs or from individual organ by genes with high expression, respectively. The result indicates similar cell types from distinct organs were clustered into a major cell type. In particular T cells, NK cell, B cells, plasma cells, endothelial cells, fibroblasts, smooth muscle cells, macrophages, monocytes and Schwan cells from different organs were grouped into limited clusters (regions outlined in dashed boxes).</p><p id="p-99"><strong>Figure S5. Expression of maker genes in major cell types from all 15 organ samples, related to <a id="xref-fig-1-11" class="xref-fig" href="#F1">figure 1</a>.</strong></p><p id="p-100"><strong>A.</strong> Dot plots showed the most highly expressed marker genes (x axis) of major cell types (y axis) in main <strong><a id="xref-fig-1-12" class="xref-fig" href="#F1">Figure 1C</a></strong>. The depth of the color from white to blue and the size of the dot represented the average expression from low to high and the percent of cells expressing the gene, respectively.</p><p id="p-101"><strong>B.</strong> t-SNE plots showed the normalized expression of representative markers for T/NK cells (<em>CD3E</em>), B cells (<em>MS4A1</em>), macrophages/monocytes (<em>CD14</em>), endothelial cells (<em>PECAM1</em>), fibroblasts (<em>MMP2</em>) and smooth muscle cells (<em>ACTA2</em>). Each dot represented a single cell. Color keys from grey to blue indicated relative expression levels from low to high.</p><p id="p-102"><strong>Figure S6. The heterogeneity and development trajectory analysis of T cells</strong>, <strong>related to <a id="xref-fig-2-15" class="xref-fig" href="#F2">figure 2</a>.</strong></p><p id="p-103"><strong>A.</strong> t-SNE plot of 7,122 CD4<sup>+</sup> T (top panel) and 13,372 CD8<sup>+</sup> T cells (bottom panel) from 15 organ tissues. Each point represented a cell, colored according to their origins of tissues as indicated on the right legend.</p><p id="p-104"><strong>B.</strong> Bar charts showed the proportions of cell types in each tissue for CD4<sup>+</sup> T cells (left panel) and CD8<sup>+</sup> T cells (right panel).</p><p id="p-105"><strong>C, D.</strong> Heatmap showed the expression of the signature genes for each of the CD4<sup>+</sup> (C) and CD8<sup>+</sup> (D) T cell clusters as indicated by numbers at the bottom.</p><p id="p-106"><strong>E,F.</strong> Pseudo-time trajectory analysis of each of the CD4<sup>+</sup> (E) and CD8<sup>+</sup> T cells (F) with high variable genes inferred by Monocle3 alpha. Each dot represented a single cell and colored according to their origin of tissue (top panels) or cluster (bottom panels).</p><p id="p-107"><strong>Figure S7. The heterogeneity and clonalities of T cell clones in human body, related to <a id="xref-fig-2-16" class="xref-fig" href="#F2">figure 2</a>.</strong></p><p id="p-108"><strong>A, B.</strong> Frequencies of the V and J segments of the TCR α (top panels) and β chains (bottom panels) for CD4<sup>+</sup> T cells (A) and CD8<sup>+</sup> T cells (B).</p><p id="p-109"><strong>C.</strong> Comparison of TCR clone sizes of CD4<sup>+</sup> and CD8<sup>+</sup> T cells in each tissue as indicated by different colors. Two-sided paired Student’s t-test was done.</p><p id="p-110"><strong>D.</strong> Bar plots showed the numbers of clonal cells in each organ for CD4<sup>+</sup> (top panel) and CD8<sup>+</sup> (bottom panel) T cells. The clonotypes are categorized as unique (n = 1), duplicated clonal (n = 2) and clonal (n ≥ 3) based on their numbers of cells sharing a TCR.</p><p id="p-111"><strong>E.</strong> Comparison of diversity between CD4<sup>+</sup> and CD8<sup>+</sup> T cells in each organ using Shannon entropy. A two-sided paired Student’s t-test was done.</p><p id="p-112"><strong>F.</strong> Bar plots showed the numbers of clonal cells in each cell cluster for CD4<sup>+</sup> (top panel) and CD8<sup>+</sup> (bottom panel) T cells. The definitions for clonotypes are as same as in <strong>D</strong>.</p><p id="p-113"><strong>G.</strong> Box plot showed the expansion- and transition-index scores of each CD4<sup>+</sup> or CD8<sup>+</sup> T cell cluster. The comparison was tested using a Wilcoxon rank-sum test.</p><p id="p-114"><strong>H,I.</strong> Heatmaps showed the pairwise transition-index scores for either two of the CD4<sup>+</sup> T cells (H) and CD8<sup>+</sup> T cells (I) clusters.</p><p id="p-115"><strong>J,K.</strong> Diversities of CD4<sup>+</sup> (J) and CD8<sup>+</sup> (K) T cells in each organ with color illustration in <strong>E</strong>. Shannon-index score was measured using TCR β chain or paired TCR α-β chains. Comparison was tested with a two-sided paired Student’s t-test.</p><p id="p-116"><strong>L.</strong> Diversities of subtypes of CD4<sup>+</sup> (top panel) and CD8<sup>+</sup> (bottom panel) T cell. Shannon-index score was measured using TCR β chain or paired TCR α-β chains.</p><p id="p-117"><strong>Figure S8. The heterogeneity of B cells and plasma cells, related to <a id="xref-fig-3-9" class="xref-fig" href="#F3">figure 3</a>.</strong></p><p id="p-118"><strong>A.</strong> Expression profiles of each B and plasma cell clusters as indicated by the numbers on top and illustrated at the right inlet box. 1,013 genes with FC ≥ 1.5 and p.adjust &lt; 0.05 were shown and IGK/L/H genes were removed.</p><p id="p-119"><strong>B.</strong> Scatter plot showed the reverse correlation between the expressions of <em>CD27</em> and <em>TCL1A</em> in all B and plasma cells. Cells with no expression of both the genes were excluded. Person’s R = −0.83, <em>P</em> = 0.</p><p id="p-120"><strong>C.</strong> Violin plot showed the presenting-antigen ability scores of each B and plasma cell cluster as indicated at the x axis.</p><p id="p-121"><strong>D.</strong> GSEA analysis results showed the significant enriched pathways of <em>oxidative phosphorylation</em> and <em>fatty acid metabolism</em> in naïve B in non-lymph node anatomical position. <strong>E-G.</strong> Frequencies of V and J gene segments of IGH (E), IGK (F) and IGL (G) chains.</p><p id="p-122"><strong>H,I.</strong> Bar plots showed the numbers of clonal cells in each organ (H) and cell cluster (I). The clonotypes are categorized as unique (n = 1), duplicated clonal (n = 2) and clonal (n ≥ 3) based on their numbers of cells sharing a BCR.</p><p id="p-123"><strong>Figure S9. Heterogeneity and trajectory analysis of macrophages, related to <a id="xref-fig-4-11" class="xref-fig" href="#F4">figure 4</a>.</strong></p><p id="p-124"><strong>A.</strong> t-SNE plots showed the normalized expression of marker genes of monocytes <em>(S100A8/9/12 and VCAN)</em>, macrophages (pan-marker: <em>C1QC, C1QA, C1QB, VSIG4;</em> Langerhans: <em>CD207)</em>, DC (<em>CD1C</em>) and subpopulation-specific genes <em>(CD14</em> and <em>CD16).</em> Each dot represented a single cell and the depth of color from grey to blue represented the expression level from low to high.</p><p id="p-125"><strong>B,C.</strong> Pseudo-time trajectory analysis of myeloid cells in each tissue (B) or cluster (C). Each dot represented a single cell.</p><p id="p-126"><strong>D.</strong> Violin plots showed the normalized expression of proliferated marker genes <em>(MKI67, TYMS</em> and <em>PCAN)</em> in each myeloid cell cluster as indicated at the x axis.</p><p id="p-127"><strong>E.</strong> GSVA analysis showed the pathways with significant difference of activation between monocytes and macrophages clusters. Activation was determined using GSVA. Comparison was tested with an R package limma.</p><p id="p-128"><strong>F.</strong> Violin plot showed the presenting-antigen ability scores of each myeloid cell cluster. comparisons were done using a Wilcoxon rank-sum test.</p><p id="p-129"><strong>G.</strong> Pseudo-time trajectory analysis of myeloid cells in rectum. Each point represented a single cell and was colored by cluster corresponding to <strong>C</strong>. High variable genes included in the analyses were inferred using Monocle3 alpha.</p><p id="p-130"><strong>Figure S10. Expression and antigen presenting ability of epithelial cells, related to <a id="xref-fig-5-11" class="xref-fig" href="#F5">figure 5</a>.</strong></p><p id="p-131"><strong>A.</strong> Scaled heatmap showed the expression of 189 tissue-specific genes in epithelial cells from different tissues as indicated at the bottom. The tissue-specific genes were defined as FC ≥ 5, pct.1 ≥ 0.2 and p.adj &lt; 0.05. The expression levels for high and low were indicated in red and blue, respectively.</p><p id="p-132"><strong>B.</strong> Top 20 pathways enriched significantly according to the GO analysis (<em>p</em> &lt; 0.05) in epithelial cell from each organ as indicated at the bottom. The depth of color represented −log10<em>P</em> calculated by online tool “metascape”.</p><p id="p-133"><strong>C.</strong> Scaled heatmap showed the expression of 375 signature genes in epithelial cell clusters as indicated by numbers on top. The signature genes were determined by FC ≥ 5, pct.1 ≥ 0.2, pct.2 ≤ 0.2 and p.adj &lt; 0.05.</p><p id="p-134"><strong>D.</strong> Heatmap showed the numbers of cluster-specific genes shared between clusters.</p><p id="p-135"><strong>E.</strong> Violin plot showed the presenting-antigen ability score of each epithelial cell cluster as indicated at the x axis.</p><p id="p-136"><strong>Figure S11. The heterogenous expression pattern of endothelial cells.</strong></p><p id="p-137"><strong>A.</strong> t-SNE showed 7,137 endothelial cells, colored by tissue (top panel) or cluster (15 clusters, bottom panel).</p><p id="p-138"><strong>B.</strong> Violin plots showed the log normalized expression of marker genes for pan-endothelial cells <em>(PECAM1)</em>, blood endothelial cells <em>(VWF)</em>, and lymphatic endothelial cells <em>(LYEV1</em> and <em>PDPN)</em> in each endothelial cell cluster as indicated at the bottom.</p><p id="p-139"><strong>C.</strong> Scaled heatmap showed the expression of the top 10 cluster-specific genes in endothelial cell clusters as indicated at the bottom. The cluster-specific genes were defined as pct.1 ≥ 0.2 and p.adj &lt; 0.05. The expression levels for high and low were indicated in red and blue, respectively.</p><p id="p-140"><strong>D.</strong> Dendrogram of cell clusters based on their normalized mean expression values of 15 clusters (correlation distance metric and average linkage). Only genes with lnFC above 0.25, p.adjust &lt; 0.05 and pct.1 ≥ 0.2 in each cluster are used to calculate the distance. Bar plots at the right illustrate the proportions of cells in corresponding organs, colored by tissue as in <strong>A</strong> (top panel).</p><p id="p-141"><strong>E.</strong> Scatter plot showed the differential gene expression of LECs in the liver (blue) and other organs (red) compared to all other endothelial cells. Selected signature genes for all LECs were indicated in grey box.</p><p id="p-142"><strong>F.</strong> GO enrichment analysis in each endothelial cell cluster. The point for each cluster in a pathway was colored by −log10<em>P</em> calculated using online tool “metascape”.</p><p id="p-143"><strong>G.</strong> Violin plot showed the presenting-antigen ability scores of each endothelial cell cluster, comparison was tested with a Wilcoxon rank-sum test. P-value was calculated as in the comparison of one cluster to all other cells.</p><p id="p-144"><strong>Figure S12. The similarity and heterogeneity of fibroblasts and smooth muscle cells.</strong></p><p id="p-145"><strong>A.</strong> t-SNE showed a total of 17,835 stromal cells, colored according to their origin of tissue (left panel) or cell cluster (15 clusters, right panel).</p><p id="p-146"><strong>B.</strong> Violin plots showed the log normalized expression of marker genes for fibroblasts <em>(MMP2)</em>, and smooth muscle cells (<em>ACTA2</em>) in each cell cluster.</p><p id="p-147"><strong>C.</strong> Scaled heatmap showed the expression of the top 10 cluster-specific genes in each stromal cell (fibroblasts, smooth muscle cell and FibSmo cell) cluster as indicated on top. The cluster-specific genes were defined as pct.1 ≥ 0.2 and p.adj &lt; 0.05. The expression levels for high and low were indicated in red and blue, respectively.</p><p id="p-148"><strong>D.</strong> Dendrogram of clusters based on the normalized mean expression values of 23 clusters (correlation distance metric, average linkage). Only genes with lnFC above 0.25, p.adjust &lt; 0.05 and pct.1 ≥ 0.2 in each cluster are used to calculate the distance. Bar plots at the right illustrate the proportions of cells in corresponding organs, colored by tissue as in <strong>A</strong> (left panel).</p><p id="p-149"><strong>E.</strong> Selected pathways in the GO enrichment analysis in each stromal cell cluster as indicated at the bottom. The point for each cluster in a pathway was colored by −log10<em>P</em> calculated using online tool “metascape”.</p><p id="p-150"><strong>Figure S13. Distribution of <em>ACE2</em> positive cells in human body.</strong></p><p id="p-151"><strong>A.</strong> t-SNE plot showed the normalized expression of <em>ACE2</em> in 15 organs. The expression levels for high and low were indicated in red and blue, respectively. Arrows indicated higher proportion of <em>ACE2</em> in common bile duct, small intestine, heart and muscle tissues.</p><p id="p-152"><strong>B.</strong> Violin plot (top panel) showing the normalized expression of <em>ACE2</em> in 43 cell clusters identified in 15 organs. Table (bottom panel) listed the proportions and cell types of <em>ACE2</em> positive cells in each organ.</p><p id="p-153"><strong>Figure S14. Overview of ligand-receptor interactions between epithelial cells and myeloid cells, related to <a id="xref-fig-6-4" class="xref-fig" href="#F6">figure 6</a>.</strong></p><p id="p-154">The significance of <em>P</em>-values was indicated as circle size (permutation test). The naming system was as follow, taking an example of “IL10_receptor IL10” in “Mac.Mucosal_Squamous_epithelial_cells_esophagus_10.Esophagus”: The interaction pair A_B, C.D.E: the means of the average expression levels of IL10 receptor in Mac cluster and <em>IL10</em> in cluster Mucosal_Squamous_epithelial_cells_esophagus_10 in tissue Esophagus.</p><p id="p-155"><strong>Figure S15. Overview of ligand-receptor interactions between CD8<sup>+</sup> T and myeloid cells, related to <a id="xref-fig-6-5" class="xref-fig" href="#F6">figure 6</a>.</strong></p><p id="p-156">Notes as in <strong>Figure S14</strong>.</p><p id="p-157"><strong>Figure S16. Overview of ligand-receptor interactions between CD8<sup>+</sup> T and fibroblasts, related to <a id="xref-fig-6-6" class="xref-fig" href="#F6">figure 6</a>.</strong></p><p id="p-158">Notes as in <strong>Figure S14</strong>.</p><p id="p-159"><strong>Figure S17. Overview of ligand-receptor interactions within epithelia cells.</strong></p><p id="p-160">Notes as in <strong>Figure S14</strong>.</p></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGEMENTS</h2><p id="p-89">We acknowledge Chunlin Luo, Xixi Chen, Xiangyu Xiong, Shuqiang Liu, Xiaochen Xu, Yaqing Zhou, Yanchun Feng, Jingjing He and Liping Xu for performing the tissues dissociation. This work was supported by grants as follows: the National Natural Science Foundation of China (81970564, 81471583 and 81570587), Guangdong Innovative and Entrepreneurial Research Team Program (2016ZT06S638), the Guangdong Provincial Key Laboratory Construction Projection on Organ Donation and Transplant Immunology (2013A061401007 and 2017B030314018), Guangdong Provincial Natural Science Funds for Major Basic Science Culture Project (2015A030308010), Guangdong Provincial Natural Science Funds for Distinguished Young Scholars (2015A030306025), Special Support Program for Training High Level Talents in Guangdong Province (2015TQ01R168), Pearl River Nova Program of Guangzhou (201506010014), and Science and Technology Program of Guangzhou (201704020150), National Program for Support of Top-Notch Young Professionals (J.-X.B.), Chang Jiang Scholars Program (J.-X.B.), Special Support Program of Guangdong (J.-X.B.) and Sun Yat-sen University Young Teacher Key Cultivate Project (17ykzd29).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-other" id="fn-1"><p id="p-1">Email addresses of authors: Shuai He, <span class="em-link"><span class="em-addr">heshuai{at}sysucc.org.cn</span></span>; Lin-he Wang, <span class="em-link"><span class="em-addr">wanglh6{at}mail2.sysu.edu.cn</span></span>; Yang Liu, <span class="em-link"><span class="em-addr">liuyang3{at}sysucc.org.cn</span></span>; Yi-qi Li, <span class="em-link"><span class="em-addr">liyq2{at}sysucc.org.cn</span></span>; Haitian Chen, <span class="em-link"><span class="em-addr">chenht3{at}mail2.sysu.edu.cn</span></span>; Jinghong Xu, <span class="em-link"><span class="em-addr">xujh38{at}mail2.sysu.edu.cn</span></span>; Wan Peng, <span class="em-link"><span class="em-addr">pengwan{at}sysucc.org.cn</span></span>; Guo-Wang Lin, <span class="em-link"><span class="em-addr">lingw{at}sysucc.org.cn</span></span>; Pan-Pan Wei, <span class="em-link"><span class="em-addr">weipp{at}sysucc.org.cn</span></span>; Bo Li, <span class="em-link"><span class="em-addr">libo47{at}mail.sysu.edu.cn</span></span>; Xiaojun Xia, <span class="em-link"><span class="em-addr">xiaxj{at}sysucc.org.cn</span></span>; Dan Wang, <span class="em-link"><span class="em-addr">wangdan{at}sysucc.org.cn</span></span>; Jin-Xin Bei, <span class="em-link"><span class="em-addr">beijx{at}sysucc.org.cn</span></span>; Xiaoshun He, <span class="em-link"><span class="em-addr">gdtrc{at}163.com</span></span>; Zhiyong Guo, <span class="em-link"><span class="em-addr">rockyucsf1981{at}126.com</span></span></p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">Reference</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.1" data-doi="10.1038/nbt.2038"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dalerba</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kalisky</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sahoo</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rajendran</span>  <span class="cit-name-given-names">PS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rothenberg</span>  <span class="cit-name-given-names">ME</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leyrat</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sim</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Okamoto</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">DM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">Single-cell dissection of transcriptional heterogeneity in human colon tumors</span>. <abbr class="cit-jnl-abbrev">Nat Biotechnol</abbr> <span class="cit-pub-date">2011</span>, <span class="cit-vol">29</span>:<span class="cit-fpage">1120</span>–<span class="cit-lpage">1127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DDalerba%26rft.auinit1%253DP.%26rft.volume%253D29%26rft.issue%253D12%26rft.spage%253D1120%26rft.epage%253D1127%26rft.atitle%253DSingle-cell%2Bdissection%2Bof%2Btranscriptional%2Bheterogeneity%2Bin%2Bhuman%2Bcolon%2Btumors.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2038%26rft_id%253Dinfo%253Apmid%252F22081019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22081019&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">2.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.18.996975v1.2" data-doi="10.1016/j.cell.2015.04.044"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mazutis</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Akartuna</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tallapragada</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Veres</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peshkin</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weitz</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kirschner</span>  <span class="cit-name-given-names">MW</span></span>: <span class="cit-article-title">Droplet barcoding for single-cell transcriptomics applied to embryonic stem cells</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-pub-date">2015</span>, <span class="cit-vol">161</span>:<span class="cit-fpage">1187</span>–<span class="cit-lpage">1201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D161%26rft.spage%253D1187%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.04.044%26rft_id%253Dinfo%253Apmid%252F26000487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.04.044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26000487&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.3" data-doi="10.1038/nmeth.2694"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">AR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neff</span>  <span class="cit-name-given-names">NF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kalisky</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dalerba</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Treutlein</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rothenberg</span>  <span class="cit-name-given-names">ME</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mburu</span>  <span class="cit-name-given-names">FM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mantalas</span>  <span class="cit-name-given-names">GL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sim</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clarke</span>  <span class="cit-name-given-names">MF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Quake</span>  <span class="cit-name-given-names">SR</span></span>: <span class="cit-article-title">Quantitative assessment of single-cell RNA-sequencing methods</span>. <abbr class="cit-jnl-abbrev">Nat Methods</abbr> <span class="cit-pub-date">2014</span>, <span class="cit-vol">11</span>:<span class="cit-fpage">41</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D11%26rft.spage%253D41%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2694%26rft_id%253Dinfo%253Apmid%252F24141493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2694&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24141493&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000329178200023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.4" data-doi="10.1038/nmeth.4150"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Qiu</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Packer</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">YA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C</span></span>: <span class="cit-article-title">Single-cell mRNA quantification and differential analysis with Census</span>. <abbr class="cit-jnl-abbrev">Nat Methods</abbr> <span class="cit-pub-date">2017</span>, <span class="cit-vol">14</span>:<span class="cit-fpage">309</span>–<span class="cit-lpage">315</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D14%26rft.spage%253D309%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.4150%26rft_id%253Dinfo%253Apmid%252F28114287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.4150&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28114287&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">5.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.18.996975v1.5" data-doi="10.1186/s12915-018-0570-z"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fabre</span>  <span class="cit-name-given-names">PJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leleu</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mascrez</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lo Giudice</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cobb</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duboule</span>  <span class="cit-name-given-names">D</span></span>: <span class="cit-article-title">Heterogeneous combinatorial expression of Hoxd genes in single cells during limb development</span>. <abbr class="cit-jnl-abbrev">BMC Biol</abbr> <span class="cit-pub-date">2018</span>, <span class="cit-vol">16</span>:<span class="cit-fpage">101</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBiol%26rft.volume%253D16%26rft.spage%253D101%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12915-018-0570-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12915-018-0570-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.6" data-doi="10.1038/nbt.4096"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoffman</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smibert</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Papalexi</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Satija</span>  <span class="cit-name-given-names">R</span></span>: <span class="cit-article-title">Integrating single-cell transcríptomic data across different conditions, technologies, and species</span>. <abbr class="cit-jnl-abbrev">Nat Biotechnol</abbr> <span class="cit-pub-date">2018</span>, <span class="cit-vol">36</span>:<span class="cit-fpage">411</span>–<span class="cit-lpage">420</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BBiotechnol%26rft.volume%253D36%26rft.spage%253D411%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.4096%26rft_id%253Dinfo%253Apmid%252F29608179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.4096&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29608179&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">7.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.18.996975v1.7" data-doi="10.1038/nbt.4091"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Haghverdi</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lun</span>  <span class="cit-name-given-names">ATL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marioni</span>  <span class="cit-name-given-names">JC</span></span>: <span class="cit-article-title">Batch effects in single-cell RNA-sequencing data are corrected by matching mutual nearest neighbors</span>. <abbr class="cit-jnl-abbrev">Nat Biotechnol</abbr> <span class="cit-pub-date">2018</span>, <span class="cit-vol">36</span>:<span class="cit-fpage">421</span>–<span class="cit-lpage">427</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BBiotechnol%26rft.volume%253D36%26rft.spage%253D421%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.4091%26rft_id%253Dinfo%253Apmid%252F29608177%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.4091&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29608177&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.8" data-doi="10.1038/s41586-018-0590-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tabula Muris</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth cit-collab">Overall c, Logistical c, Organ c, processing, Library p, sequencing, Computational data a, Cell type a, Writing g</span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">Single-cell transcriptomics of 20 mouse organs creates a Tabula Muris</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-pub-date">2018</span>, <span class="cit-vol">562</span>:<span class="cit-fpage">367</span>–<span class="cit-lpage">372</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D562%26rft.spage%253D367%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-018-0590-4%26rft_id%253Dinfo%253Apmid%252F30283141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-018-0590-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30283141&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.9" data-doi="10.1016/j.cell.2018.05.012"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fei</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saadatpour</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">Mapping the Mouse Cell Atlas by Microwell-Seq</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-pub-date">2018</span>, <span class="cit-vol">173</span>:<span class="cit-fpage">1307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D173%26rft.spage%253D1307%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2018.05.012%26rft_id%253Dinfo%253Apmid%252F29775597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2018.05.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29775597&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">10.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.18.996975v1.10" data-doi="10.1126/science.aar4362"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">DE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weinreb</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">ZM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Megason</span>  <span class="cit-name-given-names">SG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">AM</span></span>: <span class="cit-article-title">Single-cell mapping of gene expression landscapes and lineage in the zebrafish embryo</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-pub-date">2018</span>, <span class="cit-vol">360</span>:<span class="cit-fpage">981</span>–<span class="cit-lpage">987</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aar4362%26rft_id%253Dinfo%253Apmid%252F29700229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNjAvNjM5Mi85ODEiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yMC8yMDIwLjAzLjE4Ljk5Njk3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weinreb</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">DE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Megason</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peshkin</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kirschner</span>  <span class="cit-name-given-names">MW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">AM</span></span>: <span class="cit-article-title">The dynamics of gene expression in vertebrate embryogenesis at single-cell resolution</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-pub-date">2018</span>,<span class="cit-fpage">360</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.12" data-doi="10.1126/science.aam8999"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">AB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roco</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muscat</span>  <span class="cit-name-given-names">RA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuchina</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sample</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graybuck</span>  <span class="cit-name-given-names">LT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peeler</span>  <span class="cit-name-given-names">DJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mukherjee</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">Single-cell profiling of the developing mouse brain and spinal cord with split-pool barcoding</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-pub-date">2018</span>, <span class="cit-vol">360</span>:<span class="cit-fpage">176</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aam8999%26rft_id%253Dinfo%253Apmid%252F29545511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNjAvNjM4NS8xNzYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yMC8yMDIwLjAzLjE4Ljk5Njk3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.13" data-doi="10.1126/science.aat1699"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">TJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vieira Braga</span>  <span class="cit-name-given-names">FA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tran</span>  <span class="cit-name-given-names">MGB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">BJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ferdinand</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Collord</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Botting</span>  <span class="cit-name-given-names">RA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Popescu</span>  <span class="cit-name-given-names">DM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Loudon</span>  <span class="cit-name-given-names">KW</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">Single-cell transcriptomes from human kidneys reveal the cellular identity of renal tumors</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-pub-date">2018</span>, <span class="cit-vol">361</span>:<span class="cit-fpage">594</span>–<span class="cit-lpage">599</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DYoung%26rft.auinit1%253DM.%2BD.%26rft.volume%253D361%26rft.issue%253D6402%26rft.spage%253D594%26rft.epage%253D599%26rft.atitle%253DSingle-cell%2Btranscriptomes%2Bfrom%2Bhuman%2Bkidneys%2Breveal%2Bthe%2Bcellular%2Bidentity%2Bof%2Brenal%2Btumors%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aat1699%26rft_id%253Dinfo%253Apmid%252F30093597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNjEvNjQwMi81OTQiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yMC8yMDIwLjAzLjE4Ljk5Njk3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schiller</span>  <span class="cit-name-given-names">HB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Montoro</span>  <span class="cit-name-given-names">DT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">LM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rawlins</span>  <span class="cit-name-given-names">EL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">KB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Strunz</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vieira Braga</span>  <span class="cit-name-given-names">FA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Timens</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koppelman</span>  <span class="cit-name-given-names">GH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Budinger</span>  <span class="cit-name-given-names">GRS</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">The Human Lung Cell Atlas: A High-Resolution Reference Map of the Human Lung in Health and Disease</span>. <abbr class="cit-jnl-abbrev">Am J Respir Cell MoiBiol</abbr> <span class="cit-pub-date">2019</span>, <span class="cit-vol">61</span>:<span class="cit-fpage">31</span>–<span class="cit-lpage">41</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm%2BJ%2BRespir%2BCell%2BMoiBiol%26rft.volume%253D61%26rft.spage%253D31%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.15" data-doi="10.1016/j.celrep.2018.09.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">JB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sedgewick</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Finnegan</span>  <span class="cit-name-given-names">Al</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harirchian</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kwon</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fassett</span>  <span class="cit-name-given-names">MS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Golovato</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ghadially</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">Transcriptional Programming of Normal and Inflamed Human Epidermis at Single-Cell Resolution</span>. <abbr class="cit-jnl-abbrev">Cell Rep</abbr> <span class="cit-pub-date">2018</span>, <span class="cit-vol">25</span>:<span class="cit-fpage">871</span>–<span class="cit-lpage">883</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep%26rft.volume%253D25%26rft.spage%253D871%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2018.09.006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2018.09.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.16" data-doi="10.1038/s41586-018-0394-6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Plasschaert</span>  <span class="cit-name-given-names">LW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zilionis</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Choo-Wing</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Savova</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knehr</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roma</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jaffe</span>  <span class="cit-name-given-names">AB</span></span>: <span class="cit-article-title">A single-cell atlas of the airway epithelium reveals the CFTR-rich pulmonary ionocyte</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-pub-date">2018</span>, <span class="cit-vol">560</span>:<span class="cit-fpage">377</span>–<span class="cit-lpage">381</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D560%26rft.spage%253D377%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-018-0394-6%26rft_id%253Dinfo%253Apmid%252F30069046%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-018-0394-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30069046&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.17" data-doi="10.1038/s41556-018-0121-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Giladi</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Herzog</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lubling</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weiner</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yofe</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jaitin</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cabezas-Wallscheid</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dress</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ginhoux</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">Single-cell characterization of haematopoietic progenitors and their trajectories in homeostasis and perturbed haematopoiesis</span>. <abbr class="cit-jnl-abbrev">Nat Cell Biol</abbr> <span class="cit-pub-date">2018</span>, <span class="cit-vol">20</span>:<span class="cit-fpage">836</span>–<span class="cit-lpage">846</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCell%2BBiol%26rft.volume%253D20%26rft.spage%253D836%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41556-018-0121-4%26rft_id%253Dinfo%253Apmid%252F29915358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41556-018-0121-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29915358&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.18" data-doi="10.1038/s41586-018-0698-6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vento-Tormo</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Efremova</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Botting</span>  <span class="cit-name-given-names">RA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Turco</span>  <span class="cit-name-given-names">MY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vento-Tormo</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">KB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">JE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephenson</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Polanski</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goncalves</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">Single-cell reconstruction of the early matemal-fetal interface in humans</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-pub-date">2018</span>, <span class="cit-vol">563</span>:<span class="cit-fpage">347</span>–<span class="cit-lpage">353</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D563%26rft.spage%253D347%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-018-0698-6%26rft_id%253Dinfo%253Apmid%252F30429548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-018-0698-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30429548&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.19" data-doi="10.1038/s41586-019-0969-x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spielmann</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qiu</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ibrahim</span>  <span class="cit-name-given-names">DM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mundlos</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Christiansen</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Steemers</span>  <span class="cit-name-given-names">FJ</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">The single-cell transcriptional landscape of mammalian organogenesis</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-pub-date">2019</span>, <span class="cit-vol">566</span>:<span class="cit-fpage">496</span>–<span class="cit-lpage">502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D566%26rft.spage%253D496%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-019-0969-x%26rft_id%253Dinfo%253Apmid%252F30787437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-019-0969-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30787437&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.20" data-doi="10.1093/nar/gky900"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lan</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Quan</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">CellMarker: a manually curated resource of cell markers in human and mouse</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-pub-date">2019</span>, <span class="cit-vol">47</span>:<span class="cit-fpage">D721</span>–<span class="cit-lpage">d728</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D47%26rft.spage%253DD721%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky900%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gky900&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.21" data-doi="10.1172/JCI200421140"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mizumoto</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Takashima</span>  <span class="cit-name-given-names">A</span></span>: <span class="cit-article-title">CD1a and langerin: acting as more than Langerhans cell markers</span>. <abbr class="cit-jnl-abbrev">J Clin Invest</abbr> <span class="cit-pub-date">2004</span>, <span class="cit-vol">113</span>:<span class="cit-fpage">658</span>–<span class="cit-lpage">660</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BInvestigation%26rft.stitle%253DJCI%26rft.aulast%253DMizumoto%26rft.auinit1%253DN.%26rft.volume%253D113%26rft.issue%253D5%26rft.spage%253D658%26rft.epage%253D660%26rft.atitle%253DCD1a%2Band%2Blangerin%253A%2Bacting%2Bas%2Bmore%2Bthan%2BLangerhans%2Bcell%2Bmarkers%26rft_id%253Dinfo%253Adoi%252F10.1172%252FJCI200421140%26rft_id%253Dinfo%253Apmid%252F14991060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1172/JCI200421140&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14991060&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.22" data-doi="10.1677/ERC-06-0001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lacroix</span>  <span class="cit-name-given-names">M</span></span>: <span class="cit-article-title">Significance, detection and markers of disseminated breast cancer cells</span>. <abbr class="cit-jnl-abbrev">Endocr Relat Cancer</abbr> <span class="cit-pub-date">2006</span>, <span class="cit-vol">13</span>:<span class="cit-fpage">1033</span>–<span class="cit-lpage">1067</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEndocrine-Related%2BCancer%26rft.stitle%253DEndocr%2BRelat%2BCancer%26rft.aulast%253DLacroix%26rft.auinit1%253DM.%26rft.volume%253D13%26rft.issue%253D4%26rft.spage%253D1033%26rft.epage%253D1067%26rft.atitle%253DSignificance%252C%2Bdetection%2Band%2Bmarkers%2Bof%2Bdisseminated%2Bbreast%2Bcancer%2Bcells%26rft_id%253Dinfo%253Adoi%252F10.1677%252FERC-06-0001%26rft_id%253Dinfo%253Apmid%252F17158753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZXJjIjtzOjU6InJlc2lkIjtzOjk6IjEzLzQvMTAzMyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzIwLzIwMjAuMDMuMTguOTk2OTc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.23" data-doi="10.1016/j.immuni.2016.07.007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">MT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ong</span>  <span class="cit-name-given-names">DE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">FS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Teng</span>  <span class="cit-name-given-names">KW</span></span>, <span class="cit-auth"><span class="cit-name-surname">McGovern</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Narayanan</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">WQ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cerny</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">HK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anicete</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">A High-Dimensional Atlas of Human T Cell Diversity Reveals Tissue-Specific Trafficking and Cytokine Signatures</span>. <abbr class="cit-jnl-abbrev">Immunity</abbr> <span class="cit-pub-date">2016</span>, <span class="cit-vol">45</span>:<span class="cit-fpage">442</span>–<span class="cit-lpage">456</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunity%26rft.volume%253D45%26rft.spage%253D442%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.immuni.2016.07.007%26rft_id%253Dinfo%253Apmid%252F27521270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.immuni.2016.07.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27521270&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">JY</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">Lineage tracking reveals dynamic relationships of T cells in colorectal cancer</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-pub-date">2018</span>, <span class="cit-vol">564</span>:<span class="cit-fpage">268</span>–<span class="cit-lpage">272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D564%26rft.spage%253D268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rothenberg</span>  <span class="cit-name-given-names">EV</span></span>: <span class="cit-article-title">The chromatin landscape and transcription factors in T cell programming</span>. <abbr class="cit-jnl-abbrev">Trends Immunol</abbr> <span class="cit-pub-date">2014</span>, <span class="cit-vol">35</span>:<span class="cit-fpage">195</span>–<span class="cit-lpage">204</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BImmunol%26rft.volume%253D35%26rft.spage%253D195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.26" data-doi="10.1073/pnas.1009463108"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vaeth</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gogishvili</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bopp</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berberich-Siebelt</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gattenloehner</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Avots</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sparwasser</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grebe</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmitt</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">Regulatory T cells facilitate the nuclear accumulation of inducible cAMP early repressor (ICER) and suppress nuclear factor of activated T cell c1 (NFATc1)</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr> <span class="cit-pub-date">2011</span>, <span class="cit-vol">108</span>:<span class="cit-fpage">2480</span>–<span class="cit-lpage">2485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1009463108%26rft_id%253Dinfo%253Apmid%252F21262800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA4LzYvMjQ4MCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzIwLzIwMjAuMDMuMTguOTk2OTc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.27" data-doi="10.4049/jimmunol.176.3.1439"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Willinger</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Freeman</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Herbert</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hasegawa</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">McMichael</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Callan</span>  <span class="cit-name-given-names">MF</span></span>: <span class="cit-article-title">Human naive CD8 T cells down-regulate expression of the WNT pathway transcription factors lymphoid enhancer binding factor 1 and transcription factor 7 (T cell factor-1) following antigen encounter in vitro and in vivo</span>. <abbr class="cit-jnl-abbrev">J Immunol</abbr> <span class="cit-pub-date">2006</span>, <span class="cit-vol">176</span>:<span class="cit-fpage">1439</span>–<span class="cit-lpage">1446</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BImmunology%26rft.stitle%253DJ.%2BImmunol.%26rft.aulast%253DWillinger%26rft.auinit1%253DT.%26rft.volume%253D176%26rft.issue%253D3%26rft.spage%253D1439%26rft.epage%253D1446%26rft.atitle%253DHuman%2BNaive%2BCD8%2BT%2BCells%2BDown-Regulate%2BExpression%2Bof%2Bthe%2BWNT%2BPathway%2BTranscription%2BFactors%2BLymphoid%2BEnhancer%2BBinding%2BFactor%2B1%2Band%2BTranscription%2BFactor%2B7%2B%2528T%2BCell%2BFactor-1%2529%2Bfollowing%2BAntigen%2BEncounter%2BIn%2BVitro%2Band%2BIn%2BVivo%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.176.3.1439%26rft_id%253Dinfo%253Apmid%252F16424171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTA6IjE3Ni8zLzE0MzkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yMC8yMDIwLjAzLjE4Ljk5Njk3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.28" data-doi="10.1038/ni1565"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sebzda</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">JS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kahn</span>  <span class="cit-name-given-names">ML</span></span>: <span class="cit-article-title">Transcription factor KLF2 regulates the migration of naive T cells by restricting chemokine receptor expression patterns</span>. <abbr class="cit-jnl-abbrev">Nat Immunol</abbr> <span class="cit-pub-date">2008</span>, <span class="cit-vol">9</span>:<span class="cit-fpage">292</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bimmunology%26rft.stitle%253DNat%2BImmunol%26rft.aulast%253DSebzda%26rft.auinit1%253DE.%26rft.volume%253D9%26rft.issue%253D3%26rft.spage%253D292%26rft.epage%253D300%26rft.atitle%253DTranscription%2Bfactor%2BKLF2%2Bregulates%2Bthe%2Bmigration%2Bof%2Bnaive%2BT%2Bcells%2Bby%2Brestricting%2Bchemokine%2Breceptor%2Bexpression%2Bpatterns.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fni1565%26rft_id%253Dinfo%253Apmid%252F18246069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ni1565&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18246069&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253406700015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.29" data-doi="10.1038/ni.3683"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Karwacz</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miraldi</span>  <span class="cit-name-given-names">ER</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pokrovskii</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Madi</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yosef</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wortman</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Watters</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carriero</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Awasthi</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">Critical role of IRF1 and BATF in forming chromatin landscape during type 1 regulatory cell differentiation</span>. <abbr class="cit-jnl-abbrev">Nat Immunol</abbr> <span class="cit-pub-date">2017</span>, <span class="cit-vol">18</span>:<span class="cit-fpage">412</span>–<span class="cit-lpage">421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BImmunol%26rft.volume%253D18%26rft.spage%253D412%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fni.3683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ni.3683&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Diao</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Getzler</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rogal</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frederick</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Milner</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crotty</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goldrath</span>  <span class="cit-name-given-names">AW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pipkin</span>  <span class="cit-name-given-names">ME</span></span>: <span class="cit-article-title">The Transcription Factor Runx3 Establishes Chromatin Accessibility of cis-Regulatory Landscapes that Drive Memory Cytotoxic T Lymphocyte Formation</span>. <abbr class="cit-jnl-abbrev">Immunity</abbr> <span class="cit-pub-date">2018</span>, <span class="cit-vol">48</span>:<span class="cit-fpage">659</span>–<span class="cit-lpage">674.e656</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunity%26rft.volume%253D48%26rft.spage%253D659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.31" data-doi="10.1016/S0167-5699(00)01605-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Agematsu</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hokibara</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nagumo</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Komiyama</span>  <span class="cit-name-given-names">A</span></span>: <span class="cit-article-title">CD27: a memory B-cell marker</span>. <abbr class="cit-jnl-abbrev">Immunol Today</abbr> <span class="cit-pub-date">2000</span>, <span class="cit-vol">21</span>:<span class="cit-fpage">204</span>–<span class="cit-lpage">206</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunology%2Btoday%26rft.stitle%253DImmunol%2BToday%26rft.aulast%253DAgematsu%26rft.auinit1%253DK.%26rft.volume%253D21%26rft.issue%253D5%26rft.spage%253D204%26rft.epage%253D206%26rft.atitle%253DCD27%253A%2Ba%2Bmemory%2BB-cell%2Bmarker.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0167-5699%252800%252901605-4%26rft_id%253Dinfo%253Apmid%252F10782048%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0167-5699(00)01605-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10782048&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086894700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.32" data-doi="10.1182/blood-2009-03-208256"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Herling</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">KA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weit</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lilienthal</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hallek</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keating</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">D</span></span>: <span class="cit-article-title">High TCL1 levels are a marker of B-cell receptor pathway responsiveness and adverse outcome in chronic lymphocytic leukemia</span>. <abbr class="cit-jnl-abbrev">Blood</abbr> <span class="cit-pub-date">2009</span>, <span class="cit-vol">114</span>:<span class="cit-fpage">4675</span>–<span class="cit-lpage">4686</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBlood%26rft_id%253Dinfo%253Adoi%252F10.1182%252Fblood-2009-03-208256%26rft_id%253Dinfo%253Apmid%252F19770358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6ImJsb29kam91cm5hbCI7czo1OiJyZXNpZCI7czoxMToiMTE0LzIxLzQ2NzUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yMC8yMDIwLjAzLjE4Ljk5Njk3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.33" data-doi="10.4049/jimmunol.1103371"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yoon</span>  <span class="cit-name-given-names">HS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scharer</span>  <span class="cit-name-given-names">CD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Majumder</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">CW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zinzow-Kramer</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Skountzou</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koutsonanos</span>  <span class="cit-name-given-names">DG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ahmed</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boss</span>  <span class="cit-name-given-names">JM</span></span>: <span class="cit-article-title">ZBTB32 is an early repressor of the CIITA and MHC class II gene expression during B cell differentiation to plasma cells</span>. <abbr class="cit-jnl-abbrev">Journal of immunology (Baltimore, Md: 1950)</abbr> <span class="cit-pub-date">2012</span>, <span class="cit-vol">189</span>:<span class="cit-fpage">2393</span>–<span class="cit-lpage">2403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bimmunology%2B%2528Baltimore%252C%2BMd%253A%2B1950%2529%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.1103371%26rft_id%253Dinfo%253Apmid%252F22851713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTA6IjE4OS81LzIzOTMiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yMC8yMDIwLjAzLjE4Ljk5Njk3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.34" data-doi="10.1016/j.coi.2017.03.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shinnakasu</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kurosaki</span>  <span class="cit-name-given-names">T</span></span>: <span class="cit-article-title">Regulation of memory B and plasma cell differentiation</span>. <abbr class="cit-jnl-abbrev">Curr Opin Immunol</abbr> <span class="cit-pub-date">2017</span>, <span class="cit-vol">45</span>:<span class="cit-fpage">126</span>–<span class="cit-lpage">131</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr%2BOpin%2BImmunol%26rft.volume%253D45%26rft.spage%253D126%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.coi.2017.03.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.coi.2017.03.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.35" data-doi="10.4049/jimmunol.1300712"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fernandez</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ortiz</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martinez</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moreno de Alboran</span>  <span class="cit-name-given-names">I</span></span>: <span class="cit-article-title">The proto-oncogene c-myc regulates antibody secretion and Ig class switch recombination</span>. <abbr class="cit-jnl-abbrev">J Immunol</abbr> <span class="cit-pub-date">2013</span>, <span class="cit-vol">190</span>:<span class="cit-fpage">6135</span>–<span class="cit-lpage">6144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BImmunol%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.1300712%26rft_id%253Dinfo%253Apmid%252F23690468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTE6IjE5MC8xMi82MTM1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjAvMjAyMC4wMy4xOC45OTY5NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Roy</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ohta</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">YS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Metzig</span>  <span class="cit-name-given-names">MO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nutt</span>  <span class="cit-name-given-names">SL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">A</span></span>: <span class="cit-article-title">A Regulatory Circuit Controlling the Dynamics of NFkappaB cRel Transitions B Cells from Proliferation to Plasma Cell Differentiation</span>. <abbr class="cit-jnl-abbrev">Immunity</abbr> <span class="cit-pub-date">2019</span>, <span class="cit-vol">50</span>:<span class="cit-fpage">616</span>–<span class="cit-lpage">628 e616</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunity%26rft.volume%253D50%26rft.spage%253D616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Valledor</span>  <span class="cit-name-given-names">AF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Borràs</span>  <span class="cit-name-given-names">FE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cullell-Young</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Celada</span>  <span class="cit-name-given-names">A</span></span>: <span class="cit-article-title">Transcription factors that regulate monocyte/macrophage differentiation</span>. <abbr class="cit-jnl-abbrev">J Leukoc Biol</abbr> <span class="cit-pub-date">1998</span>, <span class="cit-vol">63</span>:<span class="cit-fpage">405</span>–<span class="cit-lpage">417</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BLeukocyte%2BBiology%26rft.stitle%253DJ.%2BLeukoc.%2BBiol.%26rft.aulast%253DValledor%26rft.auinit1%253DA.%26rft.volume%253D63%26rft.issue%253D4%26rft.spage%253D405%26rft.epage%253D417%26rft.atitle%253DTranscription%2Bfactors%2Bthat%2Bregulate%2Bmonocyte%252Fmacrophage%2Bdifferentiation%26rft_id%253Dinfo%253Apmid%252F9544570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9544570&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000072889800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.38" data-doi="10.1126/science.1240831"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kueh</span>  <span class="cit-name-given-names">HY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Champhekar</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Champhekhar</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nutt</span>  <span class="cit-name-given-names">SL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elowitz</span>  <span class="cit-name-given-names">MB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rothenberg</span>  <span class="cit-name-given-names">EV</span></span>: <span class="cit-article-title">Positive feedback between PU.1 and the cell cycle controls myeloid differentiation</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-pub-date">2013</span>, <span class="cit-vol">341</span>:<span class="cit-fpage">670</span>–<span class="cit-lpage">673</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1240831%26rft_id%253Dinfo%253Apmid%252F23868921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDEvNjE0Ni82NzAiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yMC8yMDIwLjAzLjE4Ljk5Njk3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.39" data-doi="10.1038/ni.2705"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Davies</span>  <span class="cit-name-given-names">LC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">JE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">PR</span></span>: <span class="cit-article-title">Tissue-resident macrophages</span>. <abbr class="cit-jnl-abbrev">Nature Immunology</abbr> <span class="cit-pub-date">2013</span>, <span class="cit-vol">14</span>:<span class="cit-fpage">986</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BImmunology%26rft.volume%253D14%26rft.spage%253D986%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fni.2705%26rft_id%253Dinfo%253Apmid%252F24048120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ni.2705&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24048120&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Roche</span>  <span class="cit-name-given-names">PA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Furuta</span>  <span class="cit-name-given-names">K</span></span>: <span class="cit-article-title">The ins and outs of MHC class ll-mediated antigen processing and presentation</span>. <abbr class="cit-jnl-abbrev">Nat Rev lmmunol</abbr> <span class="cit-pub-date">2015</span>, <span class="cit-vol">15</span>:<span class="cit-fpage">203</span>–<span class="cit-lpage">216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BRev%2Blmmunol%26rft.volume%253D15%26rft.spage%253D203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.41" data-doi="10.1182/blood-2006-10-053280"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Amatschek</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kriehuber</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reininger</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meraner</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wolpl</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schweifer</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haslinger</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stingl</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maurer</span>  <span class="cit-name-given-names">D</span></span>: <span class="cit-article-title">Blood and lymphatic endothelial cell-specific differentiation programs are stringently controlled by the tissue environment</span>. <abbr class="cit-jnl-abbrev">Blood</abbr> <span class="cit-pub-date">2007</span>, <span class="cit-vol">109</span>:<span class="cit-fpage">4777</span>–<span class="cit-lpage">4785</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBlood%26rft_id%253Dinfo%253Adoi%252F10.1182%252Fblood-2006-10-053280%26rft_id%253Dinfo%253Apmid%252F17289814%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6ImJsb29kam91cm5hbCI7czo1OiJyZXNpZCI7czoxMToiMTA5LzExLzQ3NzciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yMC8yMDIwLjAzLjE4Ljk5Njk3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.42" data-doi="10.1038/nri2588"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">SN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Germain</span>  <span class="cit-name-given-names">RN</span></span>: <span class="cit-article-title">Stromal cell contributions to the homeostasis and functionality of the immune system</span>. <abbr class="cit-jnl-abbrev">Nat Rev Immunol</abbr> <span class="cit-pub-date">2009</span>, <span class="cit-vol">9</span>:<span class="cit-fpage">618</span>–<span class="cit-lpage">629</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BImmunology%26rft.stitle%253DNat%2BRev%2BImmunol%26rft.aulast%253DMueller%26rft.auinit1%253DS.%2BN.%26rft.volume%253D9%26rft.issue%253D9%26rft.spage%253D618%26rft.epage%253D629%26rft.atitle%253DStromal%2Bcell%2Bcontributions%2Bto%2Bthe%2Bhomeostasis%2Band%2Bfunctionality%2Bof%2Bthe%2Bimmune%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnri2588%26rft_id%253Dinfo%253Apmid%252F19644499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nri2588&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19644499&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269263400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">P</span></span>: <span class="cit-article-title">Evolution of the novel coronavirus from the ongoing Wuhan outbreak and modeling of its spike protein for risk of human transmission</span>. <abbr class="cit-jnl-abbrev">Sci China Life Sci</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xin Zou</span>  <span class="cit-name-given-names">KC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiawei</span>  <span class="cit-name-given-names">Zou</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peiyi</span>  <span class="cit-name-given-names">Han</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jie</span>  <span class="cit-name-given-names">Hao</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zeguang</span>  <span class="cit-name-given-names">Han</span></span>: <span class="cit-article-title">The single-cell RNA-seq data analysis on the receptor ACE2 expression reveals the potential risk of different human organs vulnerable to Wuhan 2019-nCoV infection</span>. <abbr class="cit-jnl-abbrev">Front Med</abbr>:O-.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.46" data-doi="10.1002/path.1570"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hamming</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Timens</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bulthuis</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lely</span>  <span class="cit-name-given-names">AT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Navis</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Goor</span>  <span class="cit-name-given-names">H</span></span>: <span class="cit-article-title">Tissue distribution of ACE2 protein, the functional receptor for SARS coronavirus. A first step in understanding SARS pathogenesis</span>. <abbr class="cit-jnl-abbrev">J Pathol</abbr> <span class="cit-pub-date">2004</span>, <span class="cit-vol">203</span>:<span class="cit-fpage">631</span>–<span class="cit-lpage">637</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bpathology%26rft.stitle%253DJ%2BPathol%26rft.aulast%253DHamming%26rft.auinit1%253DI.%26rft.volume%253D203%26rft.issue%253D2%26rft.spage%253D631%26rft.epage%253D637%26rft.atitle%253DTissue%2Bdistribution%2Bof%2BACE2%2Bprotein%252C%2Bthe%2Bfunctional%2Breceptor%2Bfor%2BSARS%2Bcoronavirus.%2BA%2Bfirst%2Bstep%2Bin%2Bunderstanding%2BSARS%2Bpathogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpath.1570%26rft_id%253Dinfo%253Apmid%252F15141377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/path.1570&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15141377&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221633700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">47.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.18.996975v1.47" data-doi="10.1038/nature00786"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Crackower</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sarao</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oudit</span>  <span class="cit-name-given-names">GY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yagil</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kozieradzki</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scanga</span>  <span class="cit-name-given-names">SE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oliveira-dos-Santos</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">da Costa</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pei</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">Angiotensin-converting enzyme 2 is an essential regulator of heart function</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-pub-date">2002</span>, <span class="cit-vol">417</span>:<span class="cit-fpage">822</span>–<span class="cit-lpage">828</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DCrackower%26rft.auinit1%253DM.%2BA.%26rft.volume%253D417%26rft.issue%253D6891%26rft.spage%253D822%26rft.epage%253D828%26rft.atitle%253DAngiotensin-converting%2Benzyme%2B2%2Bis%2Ban%2Bessential%2Bregulator%2Bof%2Bheart%2Bfunction.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature00786%26rft_id%253Dinfo%253Apmid%252F12075344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature00786&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12075344&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176285600039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">48.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.18.996975v1.48" data-doi="10.1096/fj.08-107300"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kowalczuk</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Broer</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tietze</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vanslambrouck</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rasko</span>  <span class="cit-name-given-names">JE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Broer</span>  <span class="cit-name-given-names">S</span></span>: <span class="cit-article-title">A protein complex in the brush-border membrane explains a Hartnup disorder allele</span>. <abbr class="cit-jnl-abbrev">FASEB J</abbr> <span class="cit-pub-date">2008</span>, <span class="cit-vol">22</span>:<span class="cit-fpage">2880</span>–<span class="cit-lpage">2887</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFASEB%2BJ%26rft_id%253Dinfo%253Adoi%252F10.1096%252Ffj.08-107300%26rft_id%253Dinfo%253Apmid%252F18424768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1096/fj.08-107300&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18424768&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258089300028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.49" data-doi="10.1016/S0014-5793(02)03640-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Harmer</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Borman</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">KL</span></span>: <span class="cit-article-title">Quantitative mRNA expression profiling of ACE 2, a novel homologue of angiotensin converting enzyme</span>. <abbr class="cit-jnl-abbrev">FEBS Lett</abbr> <span class="cit-pub-date">2002</span>, <span class="cit-vol">532</span>:<span class="cit-fpage">107</span>–<span class="cit-lpage">110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DHarmer%26rft.auinit1%253DD.%26rft.volume%253D532%26rft.issue%253D1-2%26rft.spage%253D107%26rft.epage%253D110%26rft.atitle%253DQuantitative%2BmRNA%2Bexpression%2Bprofiling%2Bof%2BACE%2B2%252C%2Ba%2Bnovel%2Bhomologue%2Bof%2Bangiotensin%2Bconverting%2Benzyme.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0014-5793%252802%252903640-2%26rft_id%253Dinfo%253Apmid%252F12459472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0014-5793(02)03640-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12459472&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179799200021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.50" data-doi="10.1002/cncr.24245"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bach</span>  <span class="cit-name-given-names">JP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deuster</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Balzer-Geldsetzer</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dodel</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bacher</span>  <span class="cit-name-given-names">M</span></span>: <span class="cit-article-title">The role of macrophage inhibitory factor in tumorigenesis and central nervous system tumors</span>. <abbr class="cit-jnl-abbrev">Cancer</abbr> <span class="cit-pub-date">2009</span>, <span class="cit-vol">115</span>:<span class="cit-fpage">2031</span>–<span class="cit-lpage">2040</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%26rft.stitle%253DCancer%26rft.aulast%253DBach%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D115%26rft.issue%253D10%26rft.spage%253D2031%26rft.epage%253D2040%26rft.atitle%253DThe%2Brole%2Bof%2Bmacrophage%2Binhibitory%2Bfactor%2Bin%2Btumorigenesis%2Band%2Bcentral%2Bnervous%2Bsystem%2Btumors.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fcncr.24245%26rft_id%253Dinfo%253Apmid%252F19326434%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/cncr.24245&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19326434&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.51" data-doi="10.4049/jimmunol.181.9.6584"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">YH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">CY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">WC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">SW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">MK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">JY</span></span>: <span class="cit-article-title">Up-regulation of vascular endothelial growth factor-D expression in clear cell renal cell carcinoma by CD74: a critical role in cancer cell tumorigenesis</span>. <abbr class="cit-jnl-abbrev">J Immunol</abbr> <span class="cit-pub-date">2008</span>, <span class="cit-vol">181</span>:<span class="cit-fpage">6584</span>–<span class="cit-lpage">6594</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BImmunology%26rft.stitle%253DJ.%2BImmunol.%26rft.aulast%253DLiu%26rft.auinit1%253DY.-H.%26rft.volume%253D181%26rft.issue%253D9%26rft.spage%253D6584%26rft.epage%253D6594%26rft.atitle%253DUp-Regulation%2Bof%2BVascular%2BEndothelial%2BGrowth%2BFactor-D%2BExpression%2Bin%2BClear%2BCell%2BRenal%2BCell%2BCarcinoma%2Bby%2BCD74%253A%2BA%2BCritical%2BRole%2Bin%2BCancer%2BCell%2BTumorigenesis%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.181.9.6584%26rft_id%253Dinfo%253Apmid%252F18941249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTA6IjE4MS85LzY1ODQiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yMC8yMDIwLjAzLjE4Ljk5Njk3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.52" data-doi="10.4049/jimmunol.1401750"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Calame</span>  <span class="cit-name-given-names">DG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mueller-Ortiz</span>  <span class="cit-name-given-names">SL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morales</span>  <span class="cit-name-given-names">JE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wetsel</span>  <span class="cit-name-given-names">RA</span></span>: <span class="cit-article-title">The C5a anaphylatoxin receptor (C5aR1) protects against Listeria monocytogenes infection by inhibiting type 1 IFN expression</span>. <abbr class="cit-jnl-abbrev">J lmmunol</abbr> <span class="cit-pub-date">2014</span>, <span class="cit-vol">193</span>:<span class="cit-fpage">5099</span>–<span class="cit-lpage">5107</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2Blmmunol%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.1401750%26rft_id%253Dinfo%253Apmid%252F25297874%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTE6IjE5My8xMC81MDk5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjAvMjAyMC4wMy4xOC45OTY5NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">53.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.18.996975v1.53" data-doi="10.1158/1078-0432.CCR-07-1167"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mattes</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cardillo</span>  <span class="cit-name-given-names">TM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">HJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">CH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Govindan</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goldenberg</span>  <span class="cit-name-given-names">DM</span></span>: <span class="cit-article-title">CD74: a new candidate target for the immunotherapy of B-cell neoplasms</span>. <abbr class="cit-jnl-abbrev">Clin Cancer Res</abbr> <span class="cit-pub-date">2007</span>, <span class="cit-vol">13</span>:<span class="cit-fpage">5556s</span>–<span class="cit-lpage">5563s</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2BCancer%2BResearch%26rft.stitle%253DClin.%2BCancer%2BRes.%26rft.aulast%253DStein%26rft.auinit1%253DR.%26rft.volume%253D13%26rft.issue%253D18%26rft.spage%253D5556s%26rft.epage%253D5563s%26rft.atitle%253DCD74%253A%2BA%2BNew%2BCandidate%2BTarget%2Bfor%2Bthe%2BImmunotherapy%2Bof%2BB-Cell%2BNeoplasms%26rft_id%253Dinfo%253Adoi%252F10.1158%252F1078-0432.CCR-07-1167%26rft_id%253Dinfo%253Apmid%252F17875789%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6ImNsaW5jYW5yZXMiO3M6NToicmVzaWQiO3M6MTE6IjEzLzE4LzU1NTZzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjAvMjAyMC4wMy4xOC45OTY5NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">54.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.18.996975v1.54" data-doi="10.4049/jimmunol.1100646"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">EO</span></span>: <span class="cit-article-title">ICAM-1: getting a grip on leukocyte adhesion</span>. <abbr class="cit-jnl-abbrev">J Immunol</abbr> <span class="cit-pub-date">2011</span>, <span class="cit-vol">186</span>:<span class="cit-fpage">5021</span>–<span class="cit-lpage">5023</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BImmunology%26rft.stitle%253DJ.%2BImmunol.%26rft.aulast%253DLong%26rft.auinit1%253DE.%2BO.%26rft.volume%253D186%26rft.issue%253D9%26rft.spage%253D5021%26rft.epage%253D5023%26rft.atitle%253DICAM-1%253A%2BGetting%2Ba%2BGrip%2Bon%2BLeukocyte%2BAdhesion%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.1100646%26rft_id%253Dinfo%253Apmid%252F21505213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTA6IjE4Ni85LzUwMjEiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yMC8yMDIwLjAzLjE4Ljk5Njk3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.55"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Osei-Owusu</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Charlton</span>  <span class="cit-name-given-names">TM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">HK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Missiakas</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schneewind</span>  <span class="cit-name-given-names">O</span></span>: <span class="cit-article-title">FPR1 is the plague receptor on host immune cells</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-pub-date">2019</span>, <span class="cit-vol">574</span>:<span class="cit-fpage">57</span>–<span class="cit-lpage">62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D574%26rft.spage%253D57%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.56" data-doi="10.1073/pnas.0804286105"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Contento</span>  <span class="cit-name-given-names">RL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Molon</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boularan</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pozzan</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Manes</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marullo</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Viola</span>  <span class="cit-name-given-names">A</span></span>: <span class="cit-article-title">CXCR4-CCR5: a couple modulating T cell functions</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-pub-date">2008</span>, <span class="cit-vol">105</span>:<span class="cit-fpage">10101</span>–<span class="cit-lpage">10106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0804286105%26rft_id%253Dinfo%253Apmid%252F18632580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzI5LzEwMTAxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjAvMjAyMC4wMy4xOC45OTY5NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.57" data-doi="10.1016/j.cell.2014.10.026"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thome</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yudanin</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ohmura</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kubota</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grinshpun</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sathaliyawala</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kato</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lerner</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Farber</span>  <span class="cit-name-given-names">DL</span></span>: <span class="cit-article-title">Spatial map of human T cell compartmentalization and maintenance over decades of life</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-pub-date">2014</span>, <span class="cit-vol">159</span>:<span class="cit-fpage">814</span>–<span class="cit-lpage">828</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D159%26rft.spage%253D814%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2014.10.026%26rft_id%253Dinfo%253Apmid%252F25417158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2014.10.026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25417158&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.58" data-doi="10.1016/j.immuni.2014.12.007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schenkel</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Masopust</span>  <span class="cit-name-given-names">D</span></span>: <span class="cit-article-title">Tissue-resident memory T cells</span>. <abbr class="cit-jnl-abbrev">Immunity</abbr> <span class="cit-pub-date">2014</span>, <span class="cit-vol">41</span>:<span class="cit-fpage">886</span>–<span class="cit-lpage">897</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunity%26rft.volume%253D41%26rft.spage%253D886%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.immuni.2014.12.007%26rft_id%253Dinfo%253Apmid%252F25526304%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.immuni.2014.12.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25526304&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.59" data-doi="10.1038/nature10138"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">BZ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kitamura</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Campion</span>  <span class="cit-name-given-names">LR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kaiser</span>  <span class="cit-name-given-names">EA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">LA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pollard</span>  <span class="cit-name-given-names">JW</span></span>: <span class="cit-article-title">CCL2 recruits inflammatory monocytes to facilitate breast-tumour metastasis</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-pub-date">2011</span>, <span class="cit-vol">475</span>:<span class="cit-fpage">222</span>–<span class="cit-lpage">225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DQian%26rft.auinit1%253DB.%2BZ.%26rft.volume%253D475%26rft.issue%253D7355%26rft.spage%253D222%26rft.epage%253D225%26rft.atitle%253DCCL2%2Brecruits%2Binflammatory%2Bmonocytes%2Bto%2Bfacilitate%2Bbreast-tumour%2Bmetastasis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10138%26rft_id%253Dinfo%253Apmid%252F21654748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10138&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21654748&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292690500049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.60" data-doi="10.1016/j.immuni.2012.12.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yona</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">KW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mildner</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Varol</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Breker</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Strauss-Ayali</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Viukov</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guilliams</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Misharin</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">Fate mapping reveals origins and dynamics of monocytes and tissue macrophages under homeostasis</span>. <abbr class="cit-jnl-abbrev">Immunity</abbr> <span class="cit-pub-date">2013</span>, <span class="cit-vol">38</span>:<span class="cit-fpage">79</span>–<span class="cit-lpage">91</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunity%26rft.volume%253D38%26rft.spage%253D79%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.immuni.2012.12.001%26rft_id%253Dinfo%253Apmid%252F23273845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.immuni.2012.12.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23273845&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000314195500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.61"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hume</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ross</span>  <span class="cit-name-given-names">IL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Himes</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sasmono</span>  <span class="cit-name-given-names">RT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wells</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ravasi</span>  <span class="cit-name-given-names">T</span></span>: <span class="cit-article-title">The mononuclear phagocyte system revisited</span>. <abbr class="cit-jnl-abbrev">J Leukoc Biol</abbr> <span class="cit-pub-date">2002</span>, <span class="cit-vol">72</span>:<span class="cit-fpage">621</span>–<span class="cit-lpage">627</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BLeukocyte%2BBiology%26rft.stitle%253DJ.%2BLeukoc.%2BBiol.%26rft.aulast%253DHume%26rft.auinit1%253DD.%2BA.%26rft.volume%253D72%26rft.issue%253D4%26rft.spage%253D621%26rft.epage%253D627%26rft.atitle%253DThe%2Bmononuclear%2Bphagocyte%2Bsystem%2Brevisited%26rft_id%253Dinfo%253Apmid%252F12377929%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12377929&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178512100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.62"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hume</span>  <span class="cit-name-given-names">DA</span></span>: <span class="cit-article-title">The mononuclear phagocyte system</span>. <abbr class="cit-jnl-abbrev">Curr Opin lmmunol</abbr> <span class="cit-pub-date">2006</span>, <span class="cit-vol">18</span>:<span class="cit-fpage">49</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr%2BOpin%2Blmmunol%26rft.volume%253D18%26rft.spage%253D49%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.63" data-doi="10.1016/j.immuni.2013.08.007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jakubzick</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">EL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gibbings</span>  <span class="cit-name-given-names">SL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sojka</span>  <span class="cit-name-given-names">DK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schlitzer</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">TE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ivanov</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duan</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bala</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Condon</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">Minimal differentiation of classical monocytes as they survey steady-state tissues and transport antigen to lymph nodes</span>. <abbr class="cit-jnl-abbrev">Immunity</abbr> <span class="cit-pub-date">2013</span>, <span class="cit-vol">39</span>:<span class="cit-fpage">599</span>–<span class="cit-lpage">610</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunity%26rft.volume%253D39%26rft.spage%253D599%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.immuni.2013.08.007%26rft_id%253Dinfo%253Apmid%252F24012416%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.immuni.2013.08.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24012416&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000330949600021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.64" data-doi="10.1016/j.immuni.2013.04.004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hashimoto</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chow</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Noizat</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Teo</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beasley</span>  <span class="cit-name-given-names">MB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leboeuf</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Becker</span>  <span class="cit-name-given-names">CD</span></span>, <span class="cit-auth"><span class="cit-name-surname">See</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lucas</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">Tissue-resident macrophages self-maintain locally throughout adult life with minimal contribution from circulating monocytes</span>. <abbr class="cit-jnl-abbrev">Immunity</abbr> <span class="cit-pub-date">2013</span>, <span class="cit-vol">38</span>:<span class="cit-fpage">792</span>–<span class="cit-lpage">804</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunity%26rft.volume%253D38%26rft.spage%253D792%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.immuni.2013.04.004%26rft_id%253Dinfo%253Apmid%252F23601688%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.immuni.2013.04.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23601688&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000330942100020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">65.</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.65" data-doi="10.1038/nrm2523"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bryant</span>  <span class="cit-name-given-names">DM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mostov</span>  <span class="cit-name-given-names">KE</span></span>: <span class="cit-article-title">From cells to organs: building polarized tissue</span>. <abbr class="cit-jnl-abbrev">Nat Rev Mol Cell Biol</abbr> <span class="cit-pub-date">2008</span>, <span class="cit-vol">9</span>:<span class="cit-fpage">887</span>–<span class="cit-lpage">901</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DBryant%26rft.auinit1%253DD.%2BM.%26rft.volume%253D9%26rft.issue%253D11%26rft.spage%253D887%26rft.epage%253D901%26rft.atitle%253DFrom%2Bcells%2Bto%2Borgans%253A%2Bbuilding%2Bpolarized%2Btissue.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm2523%26rft_id%253Dinfo%253Apmid%252F18946477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm2523&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946477&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260312700014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">66.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.18.996975v1.66" data-doi="10.1016/j.cub.2014.06.068"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Macara</span>  <span class="cit-name-given-names">IG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guyer</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Richardson</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huo</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ahmed</span>  <span class="cit-name-given-names">SM</span></span>: <span class="cit-article-title">Epithelial homeostasis</span>. <abbr class="cit-jnl-abbrev">Curr Biol</abbr> <span class="cit-pub-date">2014</span>, <span class="cit-vol">24</span>:<span class="cit-fpage">R815</span>–<span class="cit-lpage">825</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr%2BBiol%26rft.volume%253D24%26rft.spage%253DR815%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2014.06.068%26rft_id%253Dinfo%253Apmid%252F25202877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2014.06.068&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25202877&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">67.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.18.996975v1.67" data-doi="10.1172/JCI93555"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Watt</span>  <span class="cit-name-given-names">FM</span></span>: <span class="cit-article-title">Fibroblast heterogeneity: implications for human disease</span>. <abbr class="cit-jnl-abbrev">J clin Invest</abbr> <span class="cit-pub-date">2018</span>, <span class="cit-vol">128</span>:<span class="cit-fpage">26</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2Bclin%2BInvest%26rft.volume%253D128%26rft.spage%253D26%26rft_id%253Dinfo%253Adoi%252F10.1172%252FJCI93555%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1172/JCI93555&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">68.</span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference 68. in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.68"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eckers</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haendeler</span>  <span class="cit-name-given-names">J</span></span>: <span class="cit-article-title">Endothelial cells in health and disease</span>. <abbr class="cit-jnl-abbrev">Antioxid Redox Signal</abbr> <span class="cit-pub-date">2015</span>, <span class="cit-vol">22</span>:<span class="cit-fpage">1209</span>–<span class="cit-lpage">1211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntioxid%2BRedox%2BSignal%26rft.volume%253D22%26rft.spage%253D1209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">69.</span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference 69. in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.69" data-doi="10.1038/nature24489"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Haber</span>  <span class="cit-name-given-names">AL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Biton</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rogel</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Herbst</span>  <span class="cit-name-given-names">RH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shekhar</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smillie</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burgin</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Delorey</span>  <span class="cit-name-given-names">TM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Howitt</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Katz</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">A single-cell survey of the small intestinal epithelium</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-pub-date">2017</span>, <span class="cit-vol">551</span>:<span class="cit-fpage">333</span>–<span class="cit-lpage">339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D551%26rft.spage%253D333%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature24489%26rft_id%253Dinfo%253Apmid%252F29144463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature24489&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29144463&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">70.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.18.996975v1.70"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">QH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pervolarakis</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blake</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">RT</span></span>, <span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Phung</span>  <span class="cit-name-given-names">AT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Willey</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jabart</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">Profiling human breast epithelial cells using single cell RNA sequencing identifies cell diversity</span>. <abbr class="cit-jnl-abbrev">Nature Communications</abbr> <span class="cit-pub-date">2018</span>, <span class="cit-vol">9</span>:<span class="cit-fpage">2028</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D9%26rft.spage%253D2028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">71.</span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference 71. in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.71" data-doi="10.1186/s13059-018-1416-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mao</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">F</span></span>: <span class="cit-article-title">Single-cell RNA-seq analysis unveils a prevalent epithelial/mesenchymal hybrid state during mouse organogenesis</span>. <abbr class="cit-jnl-abbrev">Genome Biol</abbr> <span class="cit-pub-date">2018</span>, <span class="cit-vol">19</span>:<span class="cit-fpage">31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D19%26rft.spage%253D31%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-018-1416-2%26rft_id%253Dinfo%253Apmid%252F29540203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-018-1416-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29540203&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">72.</span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference 72. in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.72" data-doi="10.1016/j.cell.2013.02.014"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">TI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">RA</span></span>: <span class="cit-article-title">Transcriptional regulation and its misregulation in disease</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-pub-date">2013</span>, <span class="cit-vol">152</span>:<span class="cit-fpage">1237</span>–<span class="cit-lpage">1251</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D152%26rft.spage%253D1237%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.02.014%26rft_id%253Dinfo%253Apmid%252F23498934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.02.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23498934&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316192500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">73.</span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference 73. in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.73" data-doi="10.1016/j.it.2014.03.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Khan</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dinner</span>  <span class="cit-name-given-names">AR</span></span>: <span class="cit-article-title">Gene regulatory networks in the immune system</span>. <abbr class="cit-jnl-abbrev">Trends Immunol</abbr> <span class="cit-pub-date">2014</span>, <span class="cit-vol">35</span>:<span class="cit-fpage">211</span>–<span class="cit-lpage">218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BImmunol%26rft.volume%253D35%26rft.spage%253D211%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.it.2014.03.006%26rft_id%253Dinfo%253Apmid%252F24768519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.it.2014.03.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24768519&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">74.</span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference 74. in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.74"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">GF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">D</span></span>: <span class="cit-article-title">A distinct name is needed for the new coronavirus</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">75.</span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference 75. in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.75"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiang</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiong</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">Clinical Characteristics of 138 Hospitalized Patients With 2019 Novel Coronavirus-lnfected Pneumonia in Wuhan, China</span>. <abbr class="cit-jnl-abbrev">JAMA</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">76.</span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference 76. in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.76" data-doi="10.1016/S0140-6736(20)30211-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gong</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qiu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">Epidemiological and clinical characteristics of 99 cases of 2019 novel coronavirus pneumonia in Wuhan, China: a descriptive study</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-pub-date">2020</span>, <span class="cit-vol">395</span>:<span class="cit-fpage">507</span>–<span class="cit-lpage">513</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D395%26rft.spage%253D507%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%252930211-7%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)30211-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=http://www.n&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">77.</span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference 77. in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.77" data-doi="10.1093/bioinformatics/bts635"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dobin</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schlesinger</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drenkow</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zaleski</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jha</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Batut</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chaisson</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gingeras</span>  <span class="cit-name-given-names">TR</span></span>: <span class="cit-article-title">STAR: ultrafast universal RNA-seq aligner</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-pub-date">2013</span>, <span class="cit-vol">29</span>:<span class="cit-fpage">15</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts635%26rft_id%253Dinfo%253Apmid%252F23104886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts635&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23104886&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312654600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">78.</span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference 78. in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.78" data-doi="10.1038/nrc1802"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Whitfield</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">George</span>  <span class="cit-name-given-names">LK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">GD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perou</span>  <span class="cit-name-given-names">CM</span></span>: <span class="cit-article-title">Common markers of proliferation</span>. <abbr class="cit-jnl-abbrev">Nat Rev Cancer</abbr> <span class="cit-pub-date">2006</span>, <span class="cit-vol">6</span>:<span class="cit-fpage">99</span>–<span class="cit-lpage">106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BCancer%26rft.stitle%253DNat%2BRev%2BCancer%26rft.aulast%253DWhitfield%26rft.auinit1%253DM.%2BL.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253D99%26rft.epage%253D106%26rft.atitle%253DCommon%2Bmarkers%2Bof%2Bproliferation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrc1802%26rft_id%253Dinfo%253Apmid%252F16491069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrc1802&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16491069&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234844000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">79.</span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference 79. in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.79"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pache</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Khodabakhshi</span>  <span class="cit-name-given-names">AH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tanaseichuk</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Benner</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chanda</span>  <span class="cit-name-given-names">SK</span></span>: <span class="cit-article-title">Metascape provides a biologist-oriented resource for the analysis of systems-level datasets</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-pub-date">2019</span>, <span class="cit-vol">10</span>:<span class="cit-fpage">1523</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D10%26rft.spage%253D1523%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">80.</span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference 80. in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.996975v1.80" data-doi="10.1038/nmeth.4463"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aibar</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gonzalez-Blas</span>  <span class="cit-name-given-names">CB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moerman</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huynh-Thu</span>  <span class="cit-name-given-names">VA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Imrichova</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hulselmans</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambow</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marine</span>  <span class="cit-name-given-names">JC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Geurts</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aerts</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">SCENIC: single-cell regulatory network inference and clustering</span>. <abbr class="cit-jnl-abbrev">Nat Methods</abbr> <span class="cit-pub-date">2017</span>, <span class="cit-vol">14</span>:<span class="cit-fpage">1083</span>–<span class="cit-lpage">1086</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D14%26rft.spage%253D1083%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.4463%26rft_id%253Dinfo%253Apmid%252F28991892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.4463&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28991892&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.996975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.18.996975v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.18.996975v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.03.19.998930v1" title="Transposable elements resistant to epigenetic resetting in the human germline are epigenetic hotspots for development and disease" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.18.996603v1" title="Reverse transcriptase enzyme and priming strategy affect quantification and diversity of environmental transcripts" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 20, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.18.996975v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.03.18.996975v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.18.996975v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.18.996975v1" class="active" data-icon-position="" data-hide-link-title="0">Single-cell transcriptome profiling an adult human cell atlas of 15 major organs</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1203872" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-fhRxxW4IcYUAb3fhKVtw_Ye8IPfnP54lh9gOqT5ABg8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1203872" id="node1203872--2" data-pisa="biorxiv;2020.03.18.996975v1" data-pisa-master="biorxiv;2020.03.18.996975" data-seqnum="1203872" data-apath="/biorxiv/early/2020/03/20/2020.03.18.996975.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Single-cell transcriptome profiling an adult human cell atlas of 15 major organs</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Shuai</span> <span class="nlm-surname">He</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Lin-he</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Yang</span> <span class="nlm-surname">Liu</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Yi-qi</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Haitian</span> <span class="nlm-surname">Chen</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Jinghong</span> <span class="nlm-surname">Xu</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Wan</span> <span class="nlm-surname">Peng</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Guo-Wang</span> <span class="nlm-surname">Lin</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Pan-Pan</span> <span class="nlm-surname">Wei</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Bo</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Xiaojun</span> <span class="nlm-surname">Xia</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Dan</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Jin-Xin</span> <span class="nlm-surname">Bei</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Xiaoshun</span> <span class="nlm-surname">He</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Zhiyong</span> <span class="nlm-surname">Guo</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.18.996975; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.18.996975 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.18.996975v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.18.996975v1&amp;title=Single-cell%20transcriptome%20profiling%20an%20adult%20human%20cell%20atlas%20of%2015%20major%20organs" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.18.996975v1&amp;title=Single-cell%20transcriptome%20profiling%20an%20adult%20human%20cell%20atlas%20of%2015%20major%20organs" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.18.996975v1&amp;text=Single-cell%20transcriptome%20profiling%20an%20adult%20human%20cell%20atlas%20of%2015%20major%20organs" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.18.996975v1&amp;title=Single-cell%20transcriptome%20profiling%20an%20adult%20human%20cell%20atlas%20of%2015%20major%20organs" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.18.996975v1&amp;t=Single-cell%20transcriptome%20profiling%20an%20adult%20human%20cell%20atlas%20of%2015%20major%20organs" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.18.996975v1&amp;title=Single-cell%20transcriptome%20profiling%20an%20adult%20human%20cell%20atlas%20of%2015%20major%20organs" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.18.996975v1&amp;title=Single-cell%20transcriptome%20profiling%20an%20adult%20human%20cell%20atlas%20of%2015%20major%20organs" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1203872" id="node1203872--3" data-pisa="biorxiv;2020.03.18.996975v1" data-pisa-master="biorxiv;2020.03.18.996975" data-seqnum="1203872" data-apath="/biorxiv/early/2020/03/20/2020.03.18.996975.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Single-cell transcriptome profiling an adult human cell atlas of 15 major organs</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Shuai</span> <span class="nlm-surname">He</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Lin-he</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Yang</span> <span class="nlm-surname">Liu</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Yi-qi</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Haitian</span> <span class="nlm-surname">Chen</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Jinghong</span> <span class="nlm-surname">Xu</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Wan</span> <span class="nlm-surname">Peng</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Guo-Wang</span> <span class="nlm-surname">Lin</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Pan-Pan</span> <span class="nlm-surname">Wei</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Bo</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Xiaojun</span> <span class="nlm-surname">Xia</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Dan</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Jin-Xin</span> <span class="nlm-surname">Bei</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Xiaoshun</span> <span class="nlm-surname">He</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Zhiyong</span> <span class="nlm-surname">Guo</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.18.996975; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.18.996975 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Single-cell%20transcriptome%20profiling%20an%20adult%20human%20cell%20atlas%20of%2015%20major%20organs&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=He%2C+Shuai&amp;rft.au=Wang%2C+Lin-he&amp;rft.au=Liu%2C+Yang&amp;rft.au=Li%2C+Yi-qi&amp;rft.au=Chen%2C+Haitian&amp;rft.au=Xu%2C+Jinghong&amp;rft.au=Peng%2C+Wan&amp;rft.au=Lin%2C+Guo-Wang&amp;rft.au=Wei%2C+Pan-Pan&amp;rft.au=Li%2C+Bo&amp;rft.au=Xia%2C+Xiaojun&amp;rft.au=Wang%2C+Dan&amp;rft.au=Bei%2C+Jin-Xin&amp;rft.au=He%2C+Xiaoshun&amp;rft.au=Guo%2C+Zhiyong"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1203872/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1203872/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1203872/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1203872/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1203872/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1203872/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1203872/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1203872/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1203872/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1203872/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1203872/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1203872/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.18.996975v1&amp;count=horizontal&amp;via=&amp;text=Single-cell%20transcriptome%20profiling%20an%20adult%20human%20cell%20atlas%20of%2015%20major%20organs&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.18.996975v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.18.996975v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.18.996975v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/cell-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Cell Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
