<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">PLoS Negl Trop Dis</span><span class="iso-abbrev" title="iso-abbrev">PLoS Negl Trop Dis</span><span class="publisher-id" title="publisher-id">plos</span><span class="pmc" title="pmc">plosntds</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">PLoS Neglected Tropical Diseases</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1935-2727</span><span class="issn-epub" tagx="issn" title="issn-epub">1935-2735</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, CA USA</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7053713">7053713</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pntd.0008049">10.1371/journal.pntd.0008049</a></span><span class="publisher-id" title="publisher-id">PNTD-D-19-01376</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Tropical Diseases</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Neglected Tropical Diseases</span>: <span class="subject" title="subject">Dengue Fever</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Infectious Diseases</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Diseases</span>: <span class="subject" title="subject">Dengue Fever</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Systematics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Phylogenetics</span>: <span class="subject" title="subject">Phylogenetic Analysis</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Taxonomy</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Systematics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Phylogenetics</span>: <span class="subject" title="subject">Phylogenetic Analysis</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computer and Information Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Data Management</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Taxonomy</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Systematics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Phylogenetics</span>: <span class="subject" title="subject">Phylogenetic Analysis</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and analysis methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Database and informatics methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Bioinformatics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Sequence analysis</span>: <span class="subject" title="subject">DNA sequence analysis</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Database and Informatics Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Bioinformatics</span>: <span class="subject" title="subject">Sequence Analysis</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">RNA viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Flaviviruses</span>: <span class="subject" title="subject">Dengue Virus</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medical Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Flaviviruses</span>: <span class="subject" title="subject">Dengue Virus</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathology and Laboratory Medicine</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Flaviviruses</span>: <span class="subject" title="subject">Dengue Virus</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Flaviviruses</span>: <span class="subject" title="subject">Dengue Virus</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span>: <span class="subject" title="subject">Molecular Genetics</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Biology</span>: <span class="subject" title="subject">Molecular Genetics</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Database and Informatics Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biological Databases</span>: <span class="subject" title="subject">Sequence Databases</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Database and Informatics Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Bioinformatics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Sequence Analysis</span>: <span class="subject" title="subject">Sequence Databases</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biochemistry</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Nucleic acids</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">RNA</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Messenger RNA</span>: <span class="subject" title="subject">Untranslated Regions</span></div> </div> </div> </div> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Genomic epidemiological characteristics of dengue fever in Guangdong province, China from 2013 to 2017</div> <div class="alt-title" title="alt-title">
Genomic epidemiological characteristics of dengue fever in Guangdong province, China</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">Bangyao</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Writing – original draft</span><a href="#aff001"><sup>1</sup></a><a href="#aff002"><sup>2</sup></a><a href="#aff003"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Xin</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Resources</span><a href="#aff004"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Huan</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Resources</span><a href="#aff004"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Haizhou</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Software</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff001"><sup>1</sup></a><a href="#aff002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">Lina</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Resources</span><a href="#aff005"><sup>5</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">Qiqi</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Resources</span><a href="#aff004"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">Mifang</span></span><span class="role" tagx="role" title="role">Funding acquisition</span><span class="role" tagx="role" title="role">Project administration</span><a href="#aff005"><sup>5</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">De</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Project administration</span><span class="role" tagx="role" title="role">Supervision</span><a href="#aff004"><sup>4</sup></a><a href="#cor001">*</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-3693-2726</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Di</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Funding acquisition</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Project administration</span><span class="role" tagx="role" title="role">Supervision</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff001"><sup>1</sup></a><a href="#aff002"><sup>2</sup></a><a href="#aff003"><sup>3</sup></a><a href="#cor001">*</a></span></div><span class="citation_author_institution" id="aff001">[1], <span class="addr-line" title="addr-line">CAS Key Laboratory of Special Pathogens and Biosafety, Wuhan Institute of Virology, Chinese Academy of Sciences, Wuhan, China</span></span><span class="citation_author_institution" id="aff002">[2], <span class="addr-line" title="addr-line">Computational Virology Group, Center for Bacteria and Viruses Resources and Bioinformation, Wuhan Institute of Virology, Chinese Academy of Sciences, Wuhan, China</span></span><span class="citation_author_institution" id="aff003">[3], <span class="addr-line" title="addr-line">University of Chinese Academy of Sciences, Beijing, China</span></span><span class="citation_author_institution" id="aff004">[4], <span class="addr-line" title="addr-line">Institute of Microbiology, Guangdong provincial Center for Disease Control and Prevention, Guangzhou, China</span></span><span class="citation_author_institution" id="aff005">[5], <span class="addr-line" title="addr-line">National Institute for Viral Disease Control and Prevention, Chinese Center for Disease Control and Prevention, Beijing, China</span></span><div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gubler</span><span class="given-names" tagx="given-names" title="given-names">Duane J.</span></span><span class="role" tagx="role" title="role">Editor</span><a href="#edit1" /></span></div><span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">Duke-NUS GMS, SINGAPORE</span></span><div class="author-notes" title="author-notes"> <div class="fn-type-COI-statement" title="COI-statement"> <p>The authors have declared that no competing interests exist.</p> </div> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">wu_de68@cdcp.org.cn</span> (DW); <span class="email" tagx="email" title="email">liud@wh.iov.cn</span> (DL)</div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-3-3</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020-3</span></span><span class="volume" tagx="volume" title="volume">14</span><span class="issue" tagx="issue" title="issue">3</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e0008049</span><span class="history" title="history"><span class="received" title="received">received: 2019-8-29</span><span class="accepted" title="accepted">accepted: 2020-1-11</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Sun et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</span></span></div><a class="self-uri" href="pntd.0008049.pdf" title="self-uri">pntd.0008049.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Dengue fever, a mosquito-borne viral disease in humans, has been endemic in many Southeast Asian countries. Since its first outbreak in 1978 in Foshan, Guangdong province, China, dengue has been continually epidemic in recent years in Guangdong, which raised the concern whether dengue infection is endemic in Guangdong. In this study, we performed phylogenetic, recombinant, and nucleotide variation analyses of 114 complete genome sequences of dengue virus serotypes 1–4 (DENV1-4) collected from 2013 to 2017 in 18 of 21 cities of Guangdong. Phylogenetic analyses revealed that DENV sequences did not form a single cluster, indicating that dengue fever was not endemic in Guangdong, although DENV1-4 co-circulated in Guangdong. Twenty intra-serotype recombinant isolates involving DENV1-4 were detected, but no inter-serotype recombinant events were identified in this study. Additionally, the most recombinant events were detected simultaneously in the gene <i>NS3</i> of DENV1-4. Nucleotide variation analyses showed that no significant intra-serotype differences were observed, whereas more significant inter-subtype differences were discovered in non-structural genes than in structural genes. Our investigation will facilitate the understanding of the current prevalent status of dengue fever in Guangdong and contribute to designing more effective preventive and control strategies for dengue infection.</p> </div> <div class="abstract" title="abstract"> <div class="title" tagx="title" title="title">
Author summary</div> <p>In 1978, dengue fever was first reported in Guangdong province, China, and this has been continuously prevalent in Guangdong in recent years. This is responsible for the heavy burden on the control of dengue, and raises the concern about whether dengue outbreaks have become endemic in Guangdong. Previous studies based on single <i>E</i> gene or few full-length genome sequences were inconclusive. In this study, we sequenced 114 DENV complete genomes of DENV1-4 obtained from 2013 to 2017 in Guangdong and further analyzed the epidemiological and molecular characteristics. Phylogenetic analyses revealed that dengue fever was not endemic in Guangdong, which was indirectly supported by results of our recombination analyses. Nucleotide variation analyses indicated that purification selection shaped dengue virus population. Our investigation will facilitate the development of more effective epidemiological surveillance strategies for dengue infection.</p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">China Mega-Project for Infectious Disease</span></div><span class="award-id" tagx="award-id" title="award-id">2017ZX10104001</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">Mifang</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">the China Ministry of Science and Technology Key Research, Development Program</span></div><span class="award-id" tagx="award-id" title="award-id">No. 2016YFC1200800</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-3693-2726</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Di</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">the National Natural Science Foundation of China</span></div><span class="award-id" tagx="award-id" title="award-id">No. 91631110</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-3693-2726</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Di</span></span></span></div> <div class="funding-statement" title="funding-statement">
This work was supported by China Mega-Project for Infectious Disease (2017ZX10104001). DL is supported by National Program for Support of Top-notch Young Professionals. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Data Availability" value="The DENV genome sequences generated in this study were deposited in the NCBI GenBank database under the accession no. MN018285-MN018398." /> </div> </div> <div class="title" tagx="title" title="title">
Data Availability</div> <p>The DENV genome sequences generated in this study were deposited in the NCBI GenBank database under the accession no. MN018285-MN018398.</p> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Dengue fever was listed as a major threat to global health in 2019 by the World Health Organization, aiming to reduce half of the deaths resulting from dengue infection by 2020 (WHO, <a href="https://www.who.int/emergencies/ten-threats-to-global-health-in-2019">https://www.who.int/emergencies/ten-threats-to-global-health-in-2019</a>). Dengue infection is a mosquito-borne disease widely distributed in tropical and subtropical countries with its clinical symptoms ranging from mild dengue fever to severe dengue hemorrhagic fever (DHF) or dengue shock syndrome (DSS)[<a href="#pntd.0008049.ref001">1</a>]. Approximately two-fifths of the world population are under threat of dengue infection with nearly 390 million infections annually, and it is mainly distributed in Southeast Asia, Africa, Eastern Mediterranean, Americas, and Western Pacific [<a href="#pntd.0008049.ref002">2</a>].</p> <p>Dengue fever is caused by dengue virus (DENV), belonging to the genus <i>Flavivirus</i> of family <i>Flaviviridae</i>. It is primarily transmitted by <i>Aedes aegypti</i> or <i>Aedes albopictus</i> mosquitoes [<a href="#pntd.0008049.ref003">3</a>] and is serologically classified into four serotypes (DENV1-4) [<a href="#pntd.0008049.ref004">4</a>, <a href="#pntd.0008049.ref005">5</a>]. The genome of DENV is a single-strand positive-sense RNA of approximately 11 kb, consisting of a single open reading frame that encodes three structural proteins (C, M, and E) and seven nonstructural (NS) proteins (NS1, NS2A, NS2B, NS3, NS4A, NS4B, and NS5), which are flanked by 5’ and 3’ untranslated regions (UTRs) [<a href="#pntd.0008049.ref006">6</a>].</p> <p>In China, dengue disease was first recorded in Foshan, Guangdong Province in 1978. Since then, sporadic dengue fever epidemics has occurred in Hainan, Guangxi, Fujian, Zhejiang, and Yunnan provinces in mainland China [<a href="#pntd.0008049.ref007">7</a>–<a href="#pntd.0008049.ref009">9</a>]. Particularly, dengue infection is reported almost annually in Guangdong in recent years, contributing to more than half of dengue fever cases in China [<a href="#pntd.0008049.ref010">10</a>]. In 2014, an unprecedented dengue fever outbreak occurred in Guangdong Province, leading to more than 40,000 infective cases, although great efforts were made to prevent and control dengue infection[<a href="#pntd.0008049.ref011">11</a>, <a href="#pntd.0008049.ref012">12</a>]. In subsequent years, hundreds of dengue cases at least per year have been observed in Guangdong (<a href="#pntd.0008049.t001">Table 1</a>, data from Guangdong provincial Center for Disease Control and Prevention). Continuous dengue epidemics have again raised the concern about whether dengue outbreaks have become endemic in Guangdong.</p> <div class="table-wrap_UNKNOWN" id="pntd.0008049.t001" orientation="portrait" position="float"><a href="https://dx.doi.org/10.1371/journal.pntd.0008049.t001">10.1371/journal.pntd.0008049.t001</a><span class="label" tagx="label" title="label">Table 1</span> <div class="title" tagx="title" title="title">
Dengue cases from 2013 to 2017 in Guangdong Province.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" id="pntd.0008049.t001g" /> <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div>  <thead> <tr> <th align="center" rowspan="1" colspan="1">Year</th> <th align="center" rowspan="1" colspan="1">2013</th> <th align="center" rowspan="1" colspan="1">2014</th> <th align="center" rowspan="1" colspan="1">2015</th> <th align="center" rowspan="1" colspan="1">2016</th> <th align="center" rowspan="1" colspan="1">2017</th> </tr> </thead> <tbody> <tr> <td align="center" rowspan="1" colspan="1">Dengue cases</td> <td align="center" rowspan="1" colspan="1">2894</td> <td align="center" rowspan="1" colspan="1">45189</td> <td align="center" rowspan="1" colspan="1">1683</td> <td align="center" rowspan="1" colspan="1">544</td> <td align="center" rowspan="1" colspan="1">1662</td> </tr> </tbody> </table> </div> </div> <p>In the current study, we sequenced 114 full-length genomes of DENV1-4 from 2013 to 2017, involving 18 of 21 cities in Guangdong Province and we further performed phylogenetic, recombinant, and nucleotide variation analyses. Our analyses revealed that dengue fever was not endemic in Guangdong but it is still necessary to strengthen the survey of dengue disease. By this investigation, we aimed to provide further support for the control of dengue fever in Guangdong Province.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Methods</div> <div class="ethicsstatement" title="sec"> <div class="title" tagx="title" title="title">
Ethics statement</div> <p>This study was performed in accordance with the Declaration of Helsinki and was approved by the ethics committee of Wuhan Institute of Virology. All the patients provided written informed consents.</p> </div> <div class="samplescollectionandvirusisolation" title="sec"> <div class="title" tagx="title" title="title">
Samples collection and virus isolation</div> <p>The patient's history of travelling was obtained through inquiry. Patients with DENV who did not leave Guangdong within two weeks prior to the onset of the disease were classified as local cases; otherwise classified as imported cases. After collecting serum from patients, type-specific monoclonal anti-DENV bodies were used to identify the serotype by indirect immunofluorescence. DENV positive samples were further used for viral isolation by inoculation into C6/36 cell lines. When cytopathic effect was clearly observed, cells were centrifuged to remove any debris and supernatant were collected and stored at ˗80°C.</p> </div> <div class="rnaextraction,rt-pcr,andsangersequencing" title="sec"> <div class="title" tagx="title" title="title">
RNA extraction, RT-PCR, and Sanger sequencing</div> <p>Viral RNA was extracted from 200 μL of cell supernatant using Viral DNA/RNA Extraction kit (QIAGEN, Germany) according to the manufacturer’s instructions. One-step RT-PCR kit (TaKaRa, Japan) was used to amplify DENV sequences according to the manufacturer’s protocols. PCR primer sequences of DENV are provided in <a href="#pntd.0008049.s005">S1 Table</a>. Sanger sequencing (ABI 3730) was performed to obtain DENV sequences after PCR products purification (Qiagen, Germany). Sequences assemblies were completed using BioEdit (version 7.1.3.0).</p> </div> <div class="datasetcollection" title="sec"> <div class="title" tagx="title" title="title">
Dataset collection</div> <p>We sequenced 114 whole genomes in this study (DENV-1 = 52, DENV-2 = 30, DENV-3 = 23, and DENV-4 = 9). Each analyses dataset of four DENV serotypes was composed of complete genome sequences in this study and those possessing the location and year of isolation available in GenBank database (up to December 31, 2018). Each complete sequence of DENV1-4 was firstly performed through BLASTN analyses and the top 50 hit complete sequences were selected. Next, top hit sequences were combined into a dataset, along with published complete genome sequences isolated in Asia from 2012 to 2017. Sequences with duplicated accession numbers, poor quality, and positive recombinant signal (see "Evidence of recombination�?) were excluded from dataset. Finally, four separated datasets (527 DENV-1, 314 DENV-2, 154 DENV-3, and 110 DENV-4 genome sequences) were collected for subsequent analyses. All published DENV complete genome accessions involved in this study are supplied in <a href="#pntd.0008049.s006">S2 Table</a>.</p> </div> <div class="phylogeneticanalyses" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic analyses</div> <p>Sequences were aligned using ClustalO [<a href="#pntd.0008049.ref013">13</a>] with subsequent manual correction by BioEdit (version 7.1.3.0). For phylogenetic analyses, UTR sequences were removed. Maximum likelihood (ML) phylogenetic trees were constructed using online web server (CIPRES) [<a href="#pntd.0008049.ref014">14</a>], as implemented in RAxML [<a href="#pntd.0008049.ref015">15</a>] program (version 8.2.10), under the General Time Reversible (GTR) model of nucleotide substitution with 1,000 bootstrap replicates. The final ML trees were visualized using Figtree (version 1.4.3).</p> </div> <div class="evidenceofrecombination" title="sec"> <div class="title" tagx="title" title="title">
Evidence of recombination</div> <p>Recombination detection was performed using Recombination Detection Program 4 (RDP4) [<a href="#pntd.0008049.ref016">16</a>] with UTR sequences eliminated. Seven methods were selected, including RDP [<a href="#pntd.0008049.ref016">16</a>], GENECONV [<a href="#pntd.0008049.ref017">17</a>], BootScan [<a href="#pntd.0008049.ref018">18</a>], MaxChi [<a href="#pntd.0008049.ref019">19</a>], Chimaera [<a href="#pntd.0008049.ref020">20</a>], SiScan [<a href="#pntd.0008049.ref021">21</a>], and 3Seq [<a href="#pntd.0008049.ref022">22</a>]. The recombinant event was determined when recombinant signal was detected by at least two methods and <i>p</i> value &amp;lt; 0.05 was considered positive recombinant signal. Detailed detection information of the recombinant isolates is shown in <a href="#pntd.0008049.s007">S3 Table</a>.</p> </div> <div class="singlenucleotidepolymorphism(snp)analyses" title="sec"> <div class="title" tagx="title" title="title">
Single Nucleotide Polymorphism (SNP) analyses</div> <p>The consensus sequences of dataset were obtained via cons command in EMBOSS package [<a href="#pntd.0008049.ref023">23</a>], and SNPs were directly called by aligning the complete genome sequences in this study to the consensus sequence using homemade PERL script (available at <a href="https://github.com/zer0liu/bioutils/tree/master/snp">https://github.com/zer0liu/bioutils/tree/master/snp</a>).</p> </div> <div class="accessionnumbers" title="sec"> <div class="title" tagx="title" title="title">
Accession numbers</div> <p>All dengue complete genomes generated in this study were submitted to NCBI GenBank database with accessions no. MN018285-MN018398. Detailed information is supplied in <a href="#pntd.0008049.s008">S4 Table</a>.</p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="sequencecharacteristicsinthisstudy" title="sec"> <div class="title" tagx="title" title="title">
Sequence characteristics in this study</div> <p>One hundred and fourteen DENV complete genomes were sequenced in this study (DENV-1 = 52, DENV-2 = 30, DENV-3 = 23, DENV-4 = 9). In contrast, there were 79 published DENV genome sequences (DENV-1 = 55, DENV-2 = 17, DENV-3 = 6, DENV-4 = 1) isolated from Guangdong province, China from 2012 to 2017 in GenBank (<a href="#pntd.0008049.g001">Fig 1A</a>). Except DENV-1, the sequence numbers of DENV2-4 in this study were more than those of published sequences (<a href="#pntd.0008049.g001">Fig 1A</a>). In temporal distribution, there were 4, 17, 43, 31, and 19 DENV genome sequences from 2013 to 2017, respectively (<a href="#pntd.0008049.g001">Fig 1B</a>). In 2015 and 2016, sequences from imported cases were more than sequences from local cases (23:20 in 2015, 22:9 in 2016) and in the other three years, it was the opposite (<a href="#pntd.0008049.g001">Fig 1B</a>). Sequences from local cases were a little higher than sequences from imported cases (59:55), with ratio of 33:19 in DENV-1, 13:17 in DENV-2, 11:12 in DENV-3, 2:7 in DENV-4 (<a href="#pntd.0008049.g001">Fig 1B</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pntd.0008049.g001"><span class="label" tagx="label" title="label">Fig 1</span></a></div>   <div class="title" tagx="title" title="title">
Statistical information of DENV genome sequences.</div> <p>(A) No. of DENV sequences both in the present study and those isolated from Guangdong province, China in GenBank from 2012 to 2017. (B, C) temporal distribution (B) and geographical distribution (C) of sequences in this study. "Local�? means that sequences were isolated from human cases in Guangdong and "Imported�? means that sequences were isolated from imported cases from other countries. The map image is copyright-free (available in standard map service system in China, <a href="http://bzdt.ch.mnr.gov.cn/">http://bzdt.ch.mnr.gov.cn/</a>).</p>   </div> <p>In geographical distribution, 75 of 114 DENV genome sequences were derived from five cities, Guangzhou, Shenzhen, Foshan, Dongguan, and Zhongshan (all five cities were located in the Pearl River Delta, Guangdong Province, which was characterized by developed economy, dense population, and frequent personnel flow). Moreover, sequences from Shenzhen and Foshan simultaneously included DENV1-4, and there were three DENV serotype sequences distributed in Guangzhou, Dongguan, Zhongshan, Shantou, Jiangmen, and Yangjiang. Only one DENV serotype was derived from Zhanjiang, Zhuhai, Zhaoqing, Shanwei, Huizhou, and Shaoguan. No sequences were distributed in Heyuan, Jieyang, or Meizhou (<a href="#pntd.0008049.g001">Fig 1C</a>).</p> </div> <div class="phylogeneticrelationship" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic relationship</div> <p>To illuminate the phylogenetic relationship among DENV genome sequences in this study, we conducted maximum likelihood (ML) trees of DENV1-4. Our results showed that the sequences in this study presented a scattered distribution and they were not grouped into a single cluster in DENV1-4 ML trees (Figs <a href="#pntd.0008049.g002">2</a> and <a href="#pntd.0008049.g003">3</a>). In addition, most imported cases of dengue infection were from Southeast and South Asian countries and most DENV isolates were closet to those from Southeast and South Asian countries (Figs <a href="#pntd.0008049.g002">2</a> and <a href="#pntd.0008049.g003">3</a>). For DENV-1, the sequences from the same year showed a discrete distribution along ML tree and the sequences from different years were not grouped into a single cluster. Furthermore, the most imported cases of DENV-1 infection were from Malaysia, Indonesia, Thailand, Viet Nam, the Philippines, and Sri Lanka with only one case from Brazil. Most DENV-1 isolates in our study were closely related to isolates from Singapore, Malaysia, Indonesia, and Sri Lanka (<a href="#pntd.0008049.g002">Fig 2A</a>). A similar distribution characteristic was observed in DENV-2 ML tree compared to DENV-1. The most imported DENV-2 cases were concentrated in Malaysia, Thailand, Maldives, the Philippines, Viet Nam, and Myanmar. Only a few imported DENV-2 cases were from Taiwan and New Guinea. Similarly, most DENV-2 isolates in our study were closely related to those from Malaysia, Singapore, and Indonesia (<a href="#pntd.0008049.g002">Fig 2B</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pntd.0008049.g002"><span class="label" tagx="label" title="label">Fig 2</span></a></div>   <p><b>Maximum-likelihood phylogenetic tree of DENV-1 (A) and DENV-2 (B) genomic coding regions.</b> UTR sequences and recombinant regions were excluded for phylogenetic analyses. Two published sequences (GenBank accession JQ920481, KX812530) were selected as their out-groups and the out-groups were removed for better presentation. "Locations�? means the country where of sequences were obtained, and "Imported�? means the original country of imported cases. Both are labelled by colored rectangles. The collection time are presented in red rectangles (sequences in this study) and in gray rectangles (sequences in GenBank). The recombinant isolates in this study are indicated by black stars. Bootstrap values are labelled at major nodes. Scale bar means nucleotide substitutions per site.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pntd.0008049.g003"><span class="label" tagx="label" title="label">Fig 3</span></a></div>   <p><b>Maximum-likelihood phylogenetic tree of DENV-3 (A) and DENV-4 (B) genomic coding regions.</b> Two published sequences (GenBank accession AB189121, AY037116) were selected as their out-groups and other description information are the same as in <a href="#pntd.0008049.g002">Fig 2</a>.</p>   </div> <p>DENV3-4 sequences in this study were also characterized by a discrete distribution along ML trees (<a href="#pntd.0008049.g003">Fig 3</a>). Meanwhile, most imported cases of DENV3-4 were from Malaysia, Indonesia, Thailand, India, the Philippines, Viet Nam, and Cambodia. Most isolates of DENV3-4 in the current study were closely related to those from Indonesia, Singapore, India, the Philippines, Thailand, and Cambodia (<a href="#pntd.0008049.g003">Fig 3</a>).</p> <p>Given that the potential recombinant events influence phylogenetic relationship, we further conducted ML trees of DENV1-4 without the recombinant regions excluded. The results showed that DENV1-4 sequences in the present study were not grouped into a single obvious cluster in their ML trees (<a href="#pntd.0008049.s002">S1</a> and <a href="#pntd.0008049.s003">S2</a> Figs) and most DENV isolates were closely related to those from Southeast and South Asian countries (<a href="#pntd.0008049.s002">S1</a> and <a href="#pntd.0008049.s003">S2</a> Figs). However, many isolates, especially the recombinant isolates (marked by black stars) changed their topological positions in the ML trees (<a href="#pntd.0008049.g002">Fig 2</a> and <a href="#pntd.0008049.s002">S1 Fig</a>, <a href="#pntd.0008049.g003">Fig 3</a> and <a href="#pntd.0008049.s003">S2 Fig</a>), which demonstrated that recombinant events took place in DENV genomes.</p> </div> <div class="evidenceofviralrecombination" title="sec"> <div class="title" tagx="title" title="title">
Evidence of viral recombination</div> <p>Recombination analyses of DENV genome CDS regions was performed using Recombination Detection Program 4 (RDP4). Twenty intra-serotype recombinant isolates were identified in this study with three in DENV-1, eight in DENV-2, five in DENV-3, and four in DENV-4 (<a href="#pntd.0008049.g004">Fig 4A</a>), and no inter-serotype recombinant events were detected. Among them, fourteen isolates were isolated from imported cases and six isolates were from local cases (<a href="#pntd.0008049.g004">Fig 4A</a>). In temporal distribution, nine, eight, and three recombinant isolates were obtained in 2015, 2016, and 2017, respectively, but no recombinant isolates were obtained in 2013 or 2014 (<a href="#pntd.0008049.g004">Fig 4B</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pntd.0008049.g004"><span class="label" tagx="label" title="label">Fig 4</span></a></div>   <div class="title" tagx="title" title="title">
Recombinant information of DENV genomic coding regions.</div> <p>UTR sequences were excluded for recombinant analyses. (A) The recombinant region distribution along the viral genome. Because of the little difference in the CDS regions of DENV1-4, only one genomic organization of DENV is represented in the upper panel. In the lower panel, the recombinant isolates in this study are labelled on the left (with local isolates in black and imported isolates in red) and the major parental sequences of recombinant isolates are marked on the right, with minor parental sequences in the middle. D1, D2, D3, and D4 represent DENV-1, DENV-2, DENV-3, and DENV-4, respectively. "Unknown�? means that only one parent sequence is detectable. (B) No. of recombinant isolates from Guangdong province, China (Local) or other countries (Imported) in indicated years. (C) Number (primary axis) and frequency (secondary axis) of recombinant event per DENV gene. Every involved gene will be counted once if the recombinant regions are across more than one gene, and the frequency per gene is calculated by its own recombination number divided by the total recombinant events.</p>   </div> <p>The number and frequency of recombinant event in each DENV gene were further examined and the results showed that recombinant event occurred in every gene of DENV genome except <i>2K</i> (<a href="#pntd.0008049.g004">Fig 4A and 4C</a>). The recombinant signal of <i>NS4A</i> was detected only in DENV-4, and other genes (but <i>2K</i>) were detected in at least two dengue serotypes. Furthermore, <i>NS3</i> not only had the most recombination events, but also the only one, in which recombination signal was detected simultaneously in DENV1-4. For the four DENV serotypes in this study, recombination signal was detected in two genes of DENV-1, seven genes of DENV-2, five genes of DENV-3, and nine genes of DENV-4 (<a href="#pntd.0008049.g004">Fig 4C</a>). Meanwhile, DENV-1 had the highest sequences but the lowest recombination frequency (3/52) while DENV-4 had the highest recombination frequency (4/9).</p> </div> <div class="nucleotidevariationofdenguegenome" title="sec"> <div class="title" tagx="title" title="title">
Nucleotide variation of dengue genome</div> <p>We implemented single nucleotide polymorphism (SNP) analyses of dengue genome via homemade PERL script. Total synonymous (S) and non-synonymous (N) mutations in coding sequence (CDS) and SNP in UTR within each dengue subtype were calculated. The results showed that synonymous mutations were much higher than non-synonymous mutations or SNP in UTR in each serotype (<a href="#pntd.0008049.s004">S3A Fig</a>). We further made a statistical analyses of SNP distribution along DENV genome within each subtype. The results showed that <i>NS5</i> had the maximum SNP while <i>2K</i> had the minimum (<a href="#pntd.0008049.s004">S3B Fig</a>). Considering that both the sequence number in DENV1-4 and the sequence length of each gene varied, SNP was then normalized. The normalized SNP represented an even distribution along DENV genome and it was further compared with that of CDS to examine the variation of each gene within subtypes by a 2 × 2 contingency table. Our results showed that no significant difference was found in intra-serotype DENV genes (<a href="#pntd.0008049.g005">Fig 5A</a>, <i>p</i> &amp;gt; 0.01), which indicated that each gene within the same serotype evolved at a similar rate.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pntd.0008049.g005"><span class="label" tagx="label" title="label">Fig 5</span></a></div>   <div class="title" tagx="title" title="title">
The normalized SNP distributions along DENV genome.</div> <p>(A) Intra-subtype comparison of normalized SNP distributions along DENV genomic regions. The normalized SNP are obtained by total SNPs per gene or UTR divided by the sequence number and gene length (kb). The normalized SNP of each gene is compared with that of CDS (one-tailed Fisher’s exact test for a 2 × 2 contingency table, and <i>p</i>-value &amp;lt; 0.01 is considered statistically significant). (B, C) Inter-subtype comparison of normalized SNP of UTR (B), non-synonymous mutations (C, red shade), and synonymous mutations (C, blue shade). Statistical significance was assessed using all pairwise Kruskal-Wallis one-way ANOVA test with <i>p</i> &amp;lt; 0.01. The values marked without the same superscript differ significantly.</p>   </div> <p>Next, we performed inter-subtype comparison of each gene. For viral UTRs, 5'-UTR of DENV-2 and DENV-4 had more mutations than of DENV-1 and DENV-3. 3'-UTR of DENV-4 possessed the highest SNPs among the four subtypes (<a href="#pntd.0008049.g005">Fig 5B</a>). In addition, rapider mutation of DENV3-4 than of DENV-1 occurred in both synonymous and non-synonymous SNP in CDS regions (<a href="#pntd.0008049.g005">Fig 5C</a>, CDS panel, <i>p</i> &amp;lt; 0.01). Genetically, consistent inter-subtype differences occurred in three genes (<i>NS2A</i>, <i>2K</i>, and <i>NS5</i>) between synonymous and non-synonymous mutations (<a href="#pntd.0008049.g005">Fig 5C</a>). For non-synonymous SNP, no significant inter-subtype difference was found in six genes (<i>C</i>, <i>M</i>, <i>NS2B</i>, <i>NS3</i>, <i>2K</i>, and <i>NS4B</i>, <i>p</i> &amp;gt; 0.01), and among the remaining five genes, <i>E</i> and <i>NS2A</i> of DENV-3 and <i>NS1</i>, <i>NS4A</i>, and <i>NS5</i> of DENV-4 exhibited the fastest variation rate, respectively. For synonymous SNP, significant inter-subtype differences occurred in all genes except <i>2K</i> (<a href="#pntd.0008049.g005">Fig 5C</a>, <i>p</i> &amp;lt; 0.01). In general, more significant inter-subtype differences appeared in non-structural genes than in structural genes. Furthermore, normalized non-synonymous SNP were significantly different between DENV3-4 and DENV1-2, indicating that DENV3-4 evolved more rapidly than DENV1-2, especially DENV-4, which appeared to have the fastest mutation rate (<a href="#pntd.0008049.g005">Fig 5C</a>).</p> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Discussion</div> <p>In some Southeast Asia countries, Dengue fever outbreaks had been endemic [<a href="#pntd.0008049.ref024">24</a>, <a href="#pntd.0008049.ref025">25</a>]. However, whether dengue disease is endemic in Guangdong province, China is controversial. Previous reports suggested that endemic prevalence introduced by imported isolates and endogenous epidemic outbreak co-triggered dengue outbreak in Guangdong in recent years [<a href="#pntd.0008049.ref026">26</a>–<a href="#pntd.0008049.ref028">28</a>], while Wang and his colleagues believed that DENV was an imported disease in Guangdong [<a href="#pntd.0008049.ref029">29</a>]. Compared to previous studies with conclusions made based on single <i>E</i> gene or few full-length genome sequences, the present study has implementation of phylogenetic, recombinant, and nucleotide variation analyses of 114 whole genome sequences combined with those published in GenBank of DENV1-4. This, we believe, could provide information that is much more reliable about molecular epidemiological characteristics of DENV in Guangdong in recent years.</p> <p>Genomic sequencing has played a crucial role in the prevention and control of viral infectious disease [<a href="#pntd.0008049.ref030">30</a>, <a href="#pntd.0008049.ref031">31</a>]. In this study, we sequenced 114 complete genomes of DENV1-4 from 2013 to 2017 and it provided a huge support for the research of dengue fever in Guangdong, China. In the meantime, DENV1-4 coexisted in Guangdong province, especially in the Pearl River Delta area, and this was in agreement with earlier reports [<a href="#pntd.0008049.ref026">26</a>]. Therefore, dengue epidemic situation in Guangdong is serious and further epidemiological surveillance on dengue disease is required.</p> <p>To illuminate viral phylogenetic relationships, ML trees of DENV1-4 were conducted. As shown in <a href="#pntd.0008049.g002">Fig 2A</a> and <a href="#pntd.0008049.g002">Fig 2B</a>, sequences from Singapore (DENV-1 from 2012 to 2016 and DENV-2 from 2007 to 2013) were grouped into a single obvious cluster. It revealed that successive dengue outbreaks in Singapore were caused by endemic circulating strains. A similar characteristic was presented in sequences from Thailand (<a href="#pntd.0008049.g003">Fig 3B</a>, DENV-4 from 1991 to 2010). All the above-mentioned characteristics showed that dengue fever had been endemic in Singapore and Thailand [<a href="#pntd.0008049.ref032">32</a>, <a href="#pntd.0008049.ref033">33</a>]. However, DENV1-4 sequences from the same or different years in this study represented a scattered distribution along ML trees. Although four closely related DENV-3 sequences were successively isolated from 2014 to 2017, three of them were from imported cases outside Guangdong Province (<a href="#pntd.0008049.g003">Fig 3A</a>). Furthermore, majority of isolates were closely related to those from Southeast and South Asian regions where most imported cases were obtained. This indicated that dengue epidemics in Guangdong were mainly induced by randomly imported cases from increasing tourism and economic exchanges between Southeast-South Asian countries and Guangdong province, China. Hence, we believe that dengue is still an imported disease in Guangdong.</p> <p>Recombination is considered an important driver of viral evolution and adaption [<a href="#pntd.0008049.ref034">34</a>]. Numerous DENV recombinant events have been identified in previous studies [<a href="#pntd.0008049.ref026">26</a>, <a href="#pntd.0008049.ref027">27</a>, <a href="#pntd.0008049.ref029">29</a>, <a href="#pntd.0008049.ref034">34</a>–<a href="#pntd.0008049.ref037">37</a>]. In this article, twenty recombinant isolates involving DENV1-4 were identified including each gene except <i>2K</i>, and this was further conformed by the inconsistency of ML trees constructed with or without recombinant regions. Frequent recombination generated multiple genotypes, giving rise to the current complicatedly epidemic situation in Guangdong (Figs <a href="#pntd.0008049.g002">2</a> and <a href="#pntd.0008049.g003">3</a>). Among these recombinant isolates, fourteen isolates were from imported cases while only six isolates were from Guangdong. This means that the majority of recombinant events took place outside Guangdong, which indirectly indicated that dengue disease is not endemic in Guangdong Province. Among DENV genes, <i>NS3</i> possessed the most recombinant events covering simultaneously DENV1-4. This may be in connection with its function that plays a crucial role in viral replication [<a href="#pntd.0008049.ref038">38</a>]. The precise recombinant mechanism needs further investigation via co-infection model of different dengue strains <i>in vitro</i>.</p> <p>Nucleotide mutation is another driver of viral evolution [<a href="#pntd.0008049.ref039">39</a>] and SNP analyses reflects the selection pressure within the viral population [<a href="#pntd.0008049.ref040">40</a>]. Our analyses showed that much more synonymous mutations were detected than non-synonymous mutations in DENV1-4, suggesting that most non-synonymous mutations of DENV1-4 were deleterious, and purification selection was shaping dengue virus population to prevent them from evolution <i>in situ</i>. Additionally, there were no obvious intra-serotype differences in DENV genes, but more significant inter-subtype differences in non-structural genes than in structural genes were detected. This was possibly because of the biological functional difference between the structural and non-structural proteins. For flavivirus, the structural proteins constitute the virion and they mediate cellular adsorption, penetration, and fusion while the non-structural proteins combined with their UTRs participate in viral replication, translation, and impairment of host antiviral responses [<a href="#pntd.0008049.ref041">41</a>–<a href="#pntd.0008049.ref044">44</a>]. The specific molecular mechanism still needs further investigation.</p> <p>Recently, the first vaccine for the prevention of dengue disease has been approved by the U.S. Food and Drug Administration, but there are some limitations in its global inoculation (<a href="https://www.fda.gov/news-events/press-announcements/first-fda-approved-vaccine-prevention-dengue-disease-endemic-regions">https://www.fda.gov/news-events/press-announcements/first-fda-approved-vaccine-prevention-dengue-disease-endemic-regions</a>). Moreover, the humid and rainy climate in Guangdong is suitable for the multiplication of mosquitoes and DENV transmission. Therefore, measures that are more effective for dengue prevention and control are urgently needed in the future. Our investigation has provided further knowledge on the prevalence of DENV in Guangdong and it is in favor with the formulation of prevention and control strategies for dengue infection.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting information</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Checklist</span> <div class="title" tagx="title" title="title">
STROBE checklist.</div> <p>(PDF)</p>  <div class="media" title="media"><a href="pntd.0008049.s001.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Fig</span> <p><b>Maximum-likelihood phylogenetic tree of DENV-1 (A) and DENV-2 (B) genome coding regions</b> (The recombinant regions of DENV1-2 sequences are not excluded and other information are described in <a href="#pntd.0008049.g002">Fig 2</a>).</p> <p>(TIF)</p>  <div class="media" title="media"><a href="pntd.0008049.s002.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S2 Fig</span> <p><b>Maximum-likelihood phylogenetic tree of DENV-3 (A) and DENV-4 (B) genome coding regions</b> (The recombinant regions of DENV3-4 sequences are not excluded and other information are described in <a href="#pntd.0008049.g003">Fig 3</a>).</p> <p>(TIF)</p>  <div class="media" title="media"><a href="pntd.0008049.s003.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S3 Fig</span> <div class="title" tagx="title" title="title">
The total SNPs among DENV genome.</div> <p>(A) Total SNPs within UTR (green), non-synonymous (red) and synonymous (blue) mutations of DENV1-4. (B) Total SNPs among each gene or UTR of DENV1-4.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="pntd.0008049.s004.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Table</span> <div class="title" tagx="title" title="title">
The PCR primer sequences used in this study.</div> <p>(PDF)</p>  <div class="media" title="media"><a href="pntd.0008049.s005.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S2 Table</span> <div class="title" tagx="title" title="title">
GenBank accessions of published DENV sequences in dataset.</div> <p>(PDF)</p>  <div class="media" title="media"><a href="pntd.0008049.s006.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S3 Table</span> <div class="title" tagx="title" title="title">
Recombination test information.</div> <p>(PDF)</p>  <div class="media" title="media"><a href="pntd.0008049.s007.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S4 Table</span> <div class="title" tagx="title" title="title">
GenBank accessions of DENV sequences in this study.</div> <p>(PDF)</p>  <div class="media" title="media"><a href="pntd.0008049.s008.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="pntd.0008049.ref001" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guarner</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hale</span><span class="given-names" tagx="given-names" title="given-names">GL</span></span>. <span class="mixed-article-title" title="mixed-article-title">Four human diseases with significant public health impact caused by mosquito-borne flaviviruses: West Nile, Zika, dengue and yellow fever</span>. <span class="source" tagx="source" title="source">Semin Diagn Pathol</span>. <span class="year" tagx="year" title="year">2019</span>; <span class="pub-id"><a href="https://dx.doi.org/10.1053/j.semdp.2019.04.009">10.1053/j.semdp.2019.04.009</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31006554">31006554</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref002" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhatt</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gething</span><span class="given-names" tagx="given-names" title="given-names">PW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brady</span><span class="given-names" tagx="given-names" title="given-names">OJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Messina</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farlow</span><span class="given-names" tagx="given-names" title="given-names">AW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moyes</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">The global distribution and burden of dengue</span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">496</span>(<span class="issue" tagx="issue" title="issue">7446</span>):<span class="fpage" tagx="fpage" title="fpage">504</span>–<span class="lpage" tagx="lpage" title="lpage">7</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nature12060">10.1038/nature12060</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23563266">23563266</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref003" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lambrechts</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scott</span><span class="given-names" tagx="given-names" title="given-names">TW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gubler</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span>. <span class="mixed-article-title" title="mixed-article-title">Consequences of the expanding global distribution of Aedes albopictus for dengue virus transmission</span>. <span class="source" tagx="source" title="source">PLoS Negl Trop Dis</span>. <span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">4</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">e646</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pntd.0000646">10.1371/journal.pntd.0000646</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20520794">20520794</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref004" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mustafa</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rasotgi</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jain</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>. <span class="mixed-article-title" title="mixed-article-title">Discovery of fifth serotype of dengue virus (DENV-5): A new public health dilemma in dengue control</span>. <span class="source" tagx="source" title="source">Medical journal, Armed Forces India</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">71</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">67</span>–<span class="lpage" tagx="lpage" title="lpage">70</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.mjafi.2014.09.011">10.1016/j.mjafi.2014.09.011</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25609867">25609867</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref005" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shepard</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Undurraga</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halasa</span><span class="given-names" tagx="given-names" title="given-names">YA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanaway</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span>. <span class="mixed-article-title" title="mixed-article-title">The global economic burden of dengue: a systematic analysis</span>. <span class="source" tagx="source" title="source">Lancet Infect Dis</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">16</span>(<span class="issue" tagx="issue" title="issue">8</span>):<span class="fpage" tagx="fpage" title="fpage">935</span>–<span class="lpage" tagx="lpage" title="lpage">41</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S1473-3099(16)00146-8">10.1016/S1473-3099(16)00146-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27091092">27091092</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref006" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuhn</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Michael</span><span class="given-names" tagx="given-names" title="given-names">GR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pletnev</span><span class="given-names" tagx="given-names" title="given-names">SV</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Corver</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lenches</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Structure of Dengue Virus: Implications for Flavivirus Organization, Maturation, and Fusion</span>. <span class="source" tagx="source" title="source">Cell</span>. <span class="year" tagx="year" title="year">2002</span>;<span class="volume" tagx="volume" title="volume">108</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">717</span>–<span class="lpage" tagx="lpage" title="lpage">725</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/s0092-8674(02)00660-8">10.1016/s0092-8674(02)00660-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11893341">11893341</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref007" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">QY</span></span>. <span class="mixed-article-title" title="mixed-article-title">Dengue fever in China</span>. <span class="source" tagx="source" title="source">Lancet</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">385</span>(<span class="issue" tagx="issue" title="issue">9978</span>):<span class="fpage" tagx="fpage" title="fpage">1621</span>–<span class="lpage" tagx="lpage" title="lpage">2</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S0140-6736(15)60793-0">10.1016/S0140-6736(15)60793-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25943817">25943817</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref008" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gutiérrez</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lionel</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">María</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Douglas</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">William</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guillermina</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Evaluation of the Diagnostic Utility of the Traditional and Revised WHO Dengue Case Definitions</span>. <span class="source" tagx="source" title="source">PLoS Negl Trop Dis</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">7</span>(<span class="issue" tagx="issue" title="issue">8</span>):<span class="fpage" tagx="fpage" title="fpage">e2385</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pntd.0002385">10.1371/journal.pntd.0002385</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23991237">23991237</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref009" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sang</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="etal" title="etal"><i>et al.</i></span>, 2005–2015. <span class="source" tagx="source" title="source">Int J Infect Dis</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">57(C)</span>:<span class="fpage" tagx="fpage" title="fpage">86</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ijid.2017.02.007">10.1016/j.ijid.2017.02.007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28214563">28214563</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref010" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pan</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kong</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Molecular epidemiology demonstrates that imported and local strains circulated during the 2014 dengue outbreak in Guangzhou, China</span>. <span class="source" tagx="source" title="source">Virol Sin</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">32</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">63</span>–<span class="lpage" tagx="lpage" title="lpage">72</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s12250-016-3872-8">10.1007/s12250-016-3872-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28120220">28120220</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref011" /><span class="label" tagx="label" title="label">11</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">HC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">XL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">BH</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">A survey of the 2014 dengue fever epidemic in Guangzhou, China</span>. <span class="source" tagx="source" title="source">Emerg Microbes Infect</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">4</span>(<span class="issue" tagx="issue" title="issue">9</span>):<span class="fpage" tagx="fpage" title="fpage">e57</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/emi.2015.57">10.1038/emi.2015.57</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26954995">26954995</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref012" /><span class="label" tagx="label" title="label">12</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gu</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span>. <span class="mixed-article-title" title="mixed-article-title">China in action: national strategies to combat against emerging infectious diseases</span>. <span class="source" tagx="source" title="source">Sci China Life Sci</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">60</span>(<span class="issue" tagx="issue" title="issue">12</span>):<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">3</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s11427-017-9141-3">10.1007/s11427-017-9141-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28058635">28058635</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref013" /><span class="label" tagx="label" title="label">13</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thompson</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibson</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Higgins</span><span class="given-names" tagx="given-names" title="given-names">DG</span></span>. <span class="mixed-article-title" title="mixed-article-title">Multiple sequence alignment using ClustalW and ClustalX</span>. <span class="source" tagx="source" title="source">Curr Protoc Bioinf</span>. <span class="year" tagx="year" title="year">2002</span>; (<span class="issue" tagx="issue" title="issue">Unit 2</span>)<span class="fpage" tagx="fpage" title="fpage">2.3.1</span>–<span class="lpage" tagx="lpage" title="lpage">2.3.22</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/0471250953.bi0203s00">10.1002/0471250953.bi0203s00</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18792934">18792934</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref014" /><span class="label" tagx="label" title="label">14</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Miller MA, Pfeiffer W, Schwartz T. Creating the CIPRES Science Gateway for inference of large phylogenetic trees in Proceedings of the Gateway Computing Environments Workshop (GCE), 2010; New Orleans, LA 1–8. <span class="pub-id"><a href="https://dx.doi.org/10.1109/GCE.2010.5676129">10.1109/GCE.2010.5676129</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref015" /><span class="label" tagx="label" title="label">15</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stamatakis</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="mixed-article-title" title="mixed-article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies</span>. <span class="source" tagx="source" title="source">Bioinf</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">30</span>(<span class="issue" tagx="issue" title="issue">9</span>):<span class="fpage" tagx="fpage" title="fpage">1312</span>–<span class="lpage" tagx="lpage" title="lpage">3</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btu033">10.1093/bioinformatics/btu033</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24451623">24451623</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref016" /><span class="label" tagx="label" title="label">16</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">DP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murrell</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Golden</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khoosal</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muhire</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>. <span class="mixed-article-title" title="mixed-article-title">RDP4: Detection and analysis of recombination patterns in virus genomes</span>. <span class="source" tagx="source" title="source">Virus Evol</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">1</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">vev003</span>–<span class="lpage" tagx="lpage" title="lpage">vev003</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/ve/vev003">10.1093/ve/vev003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27774277">27774277</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref017" /><span class="label" tagx="label" title="label">17</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Padidam</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sawyer</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fauquet</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span>. <span class="mixed-article-title" title="mixed-article-title">Possible emergence of new geminiviruses by frequent recombination</span>. <span class="source" tagx="source" title="source">Virology</span>. <span class="year" tagx="year" title="year">1999</span>;<span class="volume" tagx="volume" title="volume">265</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">218</span>–<span class="lpage" tagx="lpage" title="lpage">225</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1006/viro.1999.0056">10.1006/viro.1999.0056</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10600594">10600594</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref018" /><span class="label" tagx="label" title="label">18</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salminen</span><span class="given-names" tagx="given-names" title="given-names">MO</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carr</span><span class="given-names" tagx="given-names" title="given-names">JK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burke</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCutchan</span><span class="given-names" tagx="given-names" title="given-names">FE</span></span>. <span class="mixed-article-title" title="mixed-article-title">Identification of breakpoints in intergenotypic recombinants of HIV type 1 by bootscanning</span>. <span class="source" tagx="source" title="source">AIDS Res Hum Retroviruses</span>. <span class="year" tagx="year" title="year">1995</span>;<span class="volume" tagx="volume" title="volume">11</span>(<span class="issue" tagx="issue" title="issue">11</span>):<span class="fpage" tagx="fpage" title="fpage">1423</span>–<span class="lpage" tagx="lpage" title="lpage">5</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1089/aid.1995.11.1423">10.1089/aid.1995.11.1423</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8573403">8573403</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref019" /><span class="label" tagx="label" title="label">19</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>. <span class="mixed-article-title" title="mixed-article-title">Analyzing the mosaic structure of genes</span>. <span class="source" tagx="source" title="source">J Mol Evol</span>. <span class="year" tagx="year" title="year">1992</span>;<span class="volume" tagx="volume" title="volume">34</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">126</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/bf00182389">10.1007/bf00182389</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1556748">1556748</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref020" /><span class="label" tagx="label" title="label">20</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Posada</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crandall K</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="mixed-article-title" title="mixed-article-title">Evaluation of methods for detecting recombination from DNA sequences: computer simulations</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci</span>. <span class="year" tagx="year" title="year">2001</span>;<span class="volume" tagx="volume" title="volume">98</span>(<span class="issue" tagx="issue" title="issue">24</span>):<span class="fpage" tagx="fpage" title="fpage">13757</span>–<span class="lpage" tagx="lpage" title="lpage">13762</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.241370698">10.1073/pnas.241370698</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11717435">11717435</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref021" /><span class="label" tagx="label" title="label">21</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibbs</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Armstrong</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibbs</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>. <span class="mixed-article-title" title="mixed-article-title">Sister-scanning: a Monte Carlo procedure for assessing signals in recombinant sequences</span>. <span class="source" tagx="source" title="source">Bioinf</span>. <span class="year" tagx="year" title="year">2000</span>;<span class="volume" tagx="volume" title="volume">16</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="fpage" tagx="fpage" title="fpage">573</span>–<span class="lpage" tagx="lpage" title="lpage">582</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/16.7.573">10.1093/bioinformatics/16.7.573</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11038328">11038328</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref022" /><span class="label" tagx="label" title="label">22</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boni</span><span class="given-names" tagx="given-names" title="given-names">MF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Posada</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feldman</span><span class="given-names" tagx="given-names" title="given-names">MW</span></span>, <span class="mixed-article-title" title="mixed-article-title">An exact nonparametric method for inferring mosaic structure in sequence triplets</span>. <span class="source" tagx="source" title="source">Genet</span>. <span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">176</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">1035</span><span class="pub-id"><a href="https://dx.doi.org/10.1534/genetics.106.068874">10.1534/genetics.106.068874</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17409078">17409078</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref023" /><span class="label" tagx="label" title="label">23</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rice</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Longden</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bleasby</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="mixed-article-title" title="mixed-article-title">EMBOSS: the European Molecular Biology Open Software Suite</span>. <span class="source" tagx="source" title="source">Trends Genet</span>. <span class="year" tagx="year" title="year">2000</span>;<span class="volume" tagx="volume" title="volume">16</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">276</span>–<span class="lpage" tagx="lpage" title="lpage">277</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/s0168-9525(00)02024-2">10.1016/s0168-9525(00)02024-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10827456">10827456</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref024" /><span class="label" tagx="label" title="label">24</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ong</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rajarethinam</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yap</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ho</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ng</span><span class="given-names" tagx="given-names" title="given-names">LC</span></span>. <span class="mixed-article-title" title="mixed-article-title">A novel entomological index, Aedes aegypti Breeding Percentage, reveals the geographical spread of the dengue vector in Singapore and serves as a spatial risk indicator for dengue</span>. <span class="source" tagx="source" title="source">Parasites &amp;amp; vectors</span>. <span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">12</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">17</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s13071-018-3281-y">10.1186/s13071-018-3281-y</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30621762">30621762</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref025" /><span class="label" tagx="label" title="label">25</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moore</span><span class="given-names" tagx="given-names" title="given-names">PR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van den Hurk</span><span class="given-names" tagx="given-names" title="given-names">AF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mackenzie</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pyke</span><span class="given-names" tagx="given-names" title="given-names">AT</span></span>. <span class="mixed-article-title" title="mixed-article-title">Dengue viruses in Papua New Guinea: evidence of endemicity and phylogenetic variation, including the evolution of new genetic lineages</span>. <span class="source" tagx="source" title="source">Emerg Microbes Infect</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">6</span>(<span class="issue" tagx="issue" title="issue">12</span>):<span class="fpage" tagx="fpage" title="fpage">e114</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/emi.2017.103">10.1038/emi.2017.103</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29259329">29259329</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref026" /><span class="label" tagx="label" title="label">26</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bai</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">LC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Evolutionary and phylodynamic analyses of Dengue virus serotype I in Guangdong Province, China, between 1985 and 2015</span>. <span class="source" tagx="source" title="source">Virus Res</span>. <span class="year" tagx="year" title="year">2018</span>:S0168170218300091-. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virusres.2018.07.005">10.1016/j.virusres.2018.07.005</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29990510">29990510</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref027" /><span class="label" tagx="label" title="label">27</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bai</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tu</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fang</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Molecular epidemiology of dengue viruses in southern China from 1978 to 2006</span>. <span class="source" tagx="source" title="source">Virol J</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">8</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">322</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1743-422X-8-322">10.1186/1743-422X-8-322</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21703015">21703015</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref028" /><span class="label" tagx="label" title="label">28</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">HQ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ke</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maeda</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maeda</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takashima</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Molecular characterization of the E gene of dengue virus type 1 isolated in Guangdong province, China, in 2006</span>. <span class="source" tagx="source" title="source">Epidemiol Infect</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">137</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">73</span>–<span class="lpage" tagx="lpage" title="lpage">8</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1017/S0950268808000617">10.1017/S0950268808000617</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18387217">18387217</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref029" /><span class="label" tagx="label" title="label">29</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">HL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yao</span><span class="given-names" tagx="given-names" title="given-names">ZW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qin</span><span class="given-names" tagx="given-names" title="given-names">ZR</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Molecular characterization and phylogenetic analysis of dengue virus type 1 in Guangdong in 2014</span>. <span class="source" tagx="source" title="source">Springerplus</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">5</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">1942</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s40064-016-3604-4">10.1186/s40064-016-3604-4</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27933232">27933232</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref030" /><span class="label" tagx="label" title="label">30</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">WF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">GF</span></span>. <span class="mixed-article-title" title="mixed-article-title">Pathogen genomic surveillance elucidates the origins, transmission and evolution of emerging viral agents in China</span>. <span class="source" tagx="source" title="source">Sci China Life Sci</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">60</span>(<span class="issue" tagx="issue" title="issue">12</span>):<span class="fpage" tagx="fpage" title="fpage">1317</span>–<span class="lpage" tagx="lpage" title="lpage">30</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s11427-017-9211-0">10.1007/s11427-017-9211-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29270793">29270793</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref031" /><span class="label" tagx="label" title="label">31</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>. <span class="mixed-article-title" title="mixed-article-title">The triphibious warfare against viruses</span>. <span class="source" tagx="source" title="source">Sci China Life Sci</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">60</span>(<span class="issue" tagx="issue" title="issue">12</span>):<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">4</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s11427-017-9252-y">10.1007/s11427-017-9252-y</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28058635">28058635</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref032" /><span class="label" tagx="label" title="label">32</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kittichai</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sriprapun</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Konklong</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thonsangin</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chompoosri</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tawatsin</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Double Dengue Serotypes in Asymptomatic Populations Living in An Area of Thailand Endemic for Dengue Hemorrhagic Fever</span>. <span class="source" tagx="source" title="source">Thai J Vet Med</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">45</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">205</span>–<span class="lpage" tagx="lpage" title="lpage">12</span>.</span></li> <li tag="ref"><a name="pntd.0008049.ref033" /><span class="label" tagx="label" title="label">33</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">KS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lo</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">SSY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chua</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">LK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">HL</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Dengue virus surveillance in Singapore reveals high viral diversity through multiple introductions and in situ evolution</span>. <span class="source" tagx="source" title="source">Infect Genet Evol</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">12</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">77</span>–<span class="lpage" tagx="lpage" title="lpage">85</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.meegid.2011.10.012">10.1016/j.meegid.2011.10.012</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22036707">22036707</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref034" /><span class="label" tagx="label" title="label">34</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Worobey</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span>. <span class="mixed-article-title" title="mixed-article-title">Widespread intra-serotype recombination in natural populations of dengue virus</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci</span>. <span class="year" tagx="year" title="year">1999</span>;<span class="volume" tagx="volume" title="volume">96</span>(<span class="issue" tagx="issue" title="issue">13</span>):<span class="fpage" tagx="fpage" title="fpage">7352</span>–<span class="lpage" tagx="lpage" title="lpage">7</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.96.13.7352">10.1073/pnas.96.13.7352</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10377418">10377418</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref035" /><span class="label" tagx="label" title="label">35</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">SP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deng</span><span class="given-names" tagx="given-names" title="given-names">YQ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qin</span><span class="given-names" tagx="given-names" title="given-names">CF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Identification of a recombinant dengue virus type 1 with 3 recombination regions in natural populations in Guangdong province, China</span>. <span class="source" tagx="source" title="source">Arch Virol</span>. <span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">153</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">1175</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s00705-008-0090-1">10.1007/s00705-008-0090-1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18446424">18446424</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref036" /><span class="label" tagx="label" title="label">36</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mo</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zeng</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Molecular characterization of an imported dengue virus serotype 4 isolate from Thailand</span>. <span class="source" tagx="source" title="source">Arch Virol</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">163</span>(<span class="issue" tagx="issue" title="issue">10</span>):<span class="fpage" tagx="fpage" title="fpage">2903</span>–<span class="lpage" tagx="lpage" title="lpage">6</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s00705-018-3906-7">10.1007/s00705-018-3906-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29948381">29948381</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref037" /><span class="label" tagx="label" title="label">37</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Behura</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Severson</span><span class="given-names" tagx="given-names" title="given-names">DW</span></span>. <span class="mixed-article-title" title="mixed-article-title">Nucleotide substitutions in dengue virus serotypes from Asian and American countries: insights into intracodon recombination and purifying selection</span>. <span class="source" tagx="source" title="source">BMC Microbiol</span>. <span class="year" tagx="year" title="year">2013</span>; (<span class="issue" tagx="issue" title="issue">13</span>):<span class="fpage" tagx="fpage" title="fpage">37</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2180-13-37">10.1186/1471-2180-13-37</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23410119">23410119</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref038" /><span class="label" tagx="label" title="label">38</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cahour</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Falgout</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lai</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span>. <span class="mixed-article-title" title="mixed-article-title">Cleavage of the dengue virus polyprotein at the NS3/NS4A and NS4B/NS5 junctions is mediated by viral protease NS2B-NS3, whereas NS4A/NS4B may be processed by a cellular protease</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">1992</span>;<span class="volume" tagx="volume" title="volume">66</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">1535</span>–<span class="lpage" tagx="lpage" title="lpage">42</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/0166-0934(92)90058-L">10.1016/0166-0934(92)90058-L</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1531368">1531368</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref039" /><span class="label" tagx="label" title="label">39</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andino</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Domingo</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>. <span class="mixed-article-title" title="mixed-article-title">Viral quasispecies</span>. <span class="source" tagx="source" title="source">Virol</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">479-480</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">46</span>–<span class="lpage" tagx="lpage" title="lpage">51</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virol.2015.03.022">10.1016/j.virol.2015.03.022</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25824477">25824477</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref040" /><span class="label" tagx="label" title="label">40</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ni</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qian</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiao</span><span class="given-names" tagx="given-names" title="given-names">HX</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">WF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Intra-host dynamics of Ebola virus during 2014</span>. <span class="source" tagx="source" title="source">Nat Microbiol</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">1</span>(<span class="issue" tagx="issue" title="issue">11</span>):<span class="fpage" tagx="fpage" title="fpage">16151</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nmicrobiol.2016.151">10.1038/nmicrobiol.2016.151</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27595345">27595345</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref041" /><span class="label" tagx="label" title="label">41</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lazear</span><span class="given-names" tagx="given-names" title="given-names">HM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diamond</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>. <span class="mixed-article-title" title="mixed-article-title">Zika Virus: New Clinical Syndromes and Its Emergence in the Western Hemisphere</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">90</span>(<span class="issue" tagx="issue" title="issue">10</span>):<span class="fpage" tagx="fpage" title="fpage">4864</span>–<span class="lpage" tagx="lpage" title="lpage">75</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00252-16">10.1128/JVI.00252-16</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26962217">26962217</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref042" /><span class="label" tagx="label" title="label">42</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suthar</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diamond</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gale</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>. <span class="mixed-article-title" title="mixed-article-title">West Nile virus infection and immunity</span>. <span class="source" tagx="source" title="source">Nat Rev Microbiol</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">11</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">115</span>–<span class="lpage" tagx="lpage" title="lpage">128</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nrmicro2950">10.1038/nrmicro2950</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23321534">23321534</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref043" /><span class="label" tagx="label" title="label">43</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mukhopadhyay</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuhn</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rossmann</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span>. <span class="mixed-article-title" title="mixed-article-title">A structural perspective of the flavivirus life cycle</span>. <span class="source" tagx="source" title="source">Nat Rev Microbiol</span>. <span class="year" tagx="year" title="year">2005</span>;<span class="volume" tagx="volume" title="volume">3</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">13</span>–<span class="lpage" tagx="lpage" title="lpage">22</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nrmicro1067">10.1038/nrmicro1067</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15608696">15608696</a></span></span></li> <li tag="ref"><a name="pntd.0008049.ref044" /><span class="label" tagx="label" title="label">44</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diamond</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pierson</span><span class="given-names" tagx="given-names" title="given-names">TC</span></span>. <span class="mixed-article-title" title="mixed-article-title">Molecular Insight into Dengue Virus Pathogenesis and Its Implications for Disease Control</span>. <span class="source" tagx="source" title="source">Cell</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">162</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">488</span>–<span class="lpage" tagx="lpage" title="lpage">492</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cell.2015.07.005">10.1016/j.cell.2015.07.005</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26232221">26232221</a></span></span></li> </ul> </div> </div>  </body></html>