<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Viruses</span><span class="iso-abbrev" title="iso-abbrev">Viruses</span><span class="publisher-id" title="publisher-id">viruses</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Viruses</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1999-4915</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">MDPI</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5923494">5923494</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3390/v10040200">10.3390/v10040200</a></span><span class="publisher-id" title="publisher-id">viruses-10-00200</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Accounting for Space — Quantification of Cell-To-Cell Transmission Kinetics Using Virus Dynamics Models</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumberger</span><span class="given-names" tagx="given-names" title="given-names">Peter</span></span><a href="#af1-viruses-10-00200">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Durso-Cain</span><span class="given-names" tagx="given-names" title="given-names">Karina</span></span><a href="#af2-viruses-10-00200">2</a><a href="#af3-viruses-10-00200">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uprichard</span><span class="given-names" tagx="given-names" title="given-names">Susan L.</span></span><a href="#af2-viruses-10-00200">2</a><a href="#af3-viruses-10-00200">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dahari</span><span class="given-names" tagx="given-names" title="given-names">Harel</span></span><a href="#af3-viruses-10-00200">3</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-1198-6632</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graw</span><span class="given-names" tagx="given-names" title="given-names">Frederik</span></span><a href="#af1-viruses-10-00200">1</a><a href="#c1-viruses-10-00200">*</a></span></div><span class="citation_author_institution" id="af1-viruses-10-00200">[1], <span class="email" tagx="email" title="email">peter.kumberger@bioquant.uni-heidelberg.de</span></span><span class="citation_author_institution" id="af2-viruses-10-00200">[2], <span class="email" tagx="email" title="email">kdurso@luc.edu</span><span class="email" tagx="email" title="email">suprichard@luc.edu</span></span><span class="citation_author_institution" id="af3-viruses-10-00200">[3], <span class="email" tagx="email" title="email">hdahari@luc.edu</span></span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">*</span>Correspondence: <span class="email" tagx="email" title="email">frederik.graw@bioquant.uni-heidelberg.de</span>; Tel.: +49-6221-54-51309</div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-4-4</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2018-4</span></span><span class="volume" tagx="volume" title="volume">10</span><span class="issue" tagx="issue" title="issue">4</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">200</span><span class="history" title="history"><span class="received" title="received">received: 2018-2-28</span><span class="accepted" title="accepted">accepted: 2018-4-12</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2018</span></span><span class="license" title="license"><span class="license-p" title="license-p">Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>).</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Mathematical models based on ordinary differential equations (ODE) that describe the population dynamics of viruses and infected cells have been an essential tool to characterize and quantify viral infection dynamics. Although an important aspect of viral infection is the dynamics of viral spread, which includes transmission by cell-free virions and direct cell-to-cell transmission, models used so far ignored cell-to-cell transmission completely, or accounted for this process by simple mass-action kinetics between infected and uninfected cells. In this study, we show that the simple mass-action approach falls short when describing viral spread in a spatially-defined environment. Using simulated data, we present a model extension that allows correct quantification of cell-to-cell transmission dynamics within a monolayer of cells. By considering the decreasing proportion of cells that can contribute to cell-to-cell spread with progressing infection, our extension accounts for the transmission dynamics on a single cell level while still remaining applicable to standard population-based experimental measurements. While the ability to infer the proportion of cells infected by either of the transmission modes depends on the viral diffusion rate, the improved estimates obtained using our novel approach emphasize the need to correctly account for spatial aspects when analyzing viral spread.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
cell-to-cell transmission</div> <div class="kwd" title="kwd">
virus dynamics</div> <div class="kwd" title="kwd">
mathematical modeling</div> <div class="kwd" title="kwd">
spatial effects</div> </div> </div> </div> <div class="body" title="body"> <div class="1.introduction" title="sec"> <div class="title" tagx="title" title="title">
1. Introduction</div> <p>The way a pathogen spreads within a host has important implications for the successful establishment and maintenance of an infection, as well as the identification of therapeutic targets. Being cellular parasites, viruses have evolved different mechanisms to invade their corresponding target cells in order to replicate and spread infection. In general, these mechanisms can be categorized by two main modes of viral transmission, i.e., (i) cell-free (CF) spread by diffusing viral particles and (ii) direct cell-to-cell (CC) transmission between infected and uninfected cells [<a href="#B1-viruses-10-00200">1</a>]. While cell-free diffusion of viral particles allows the infection of distant cells, direct cell-to-cell transmission leads to local spread, avoids complex entry processes and is thought to have the potential to shield the virus from neutralizing antibodies and other antiviral clearance mechanisms [<a href="#B2-viruses-10-00200">2</a>,<a href="#B3-viruses-10-00200">3</a>]. Therefore, the latter transmission mode is usually thought to be more efficient in spreading infection [<a href="#B4-viruses-10-00200">4</a>,<a href="#B5-viruses-10-00200">5</a>]. In addition, CC is assumed to play an important role in the establishment of persistent infections and therapy failure [<a href="#B2-viruses-10-00200">2</a>,<a href="#B6-viruses-10-00200">6</a>,<a href="#B7-viruses-10-00200">7</a>]. The importance and contribution of each of the transmission modes to viral spread still has to be determined in detail. However, to address this question, a reliable quantification of the individual transmission kinetics is needed.</p> <p>Mathematical models have been an essential tool to analyze experimental data to assess viral infection dynamics. They have allowed the quantification of viral kinetics and identification of potential antiviral therapies (reviewed in [<a href="#B8-viruses-10-00200">8</a>,<a href="#B9-viruses-10-00200">9</a>,<a href="#B10-viruses-10-00200">10</a>]). These virus dynamics models are usually based on systems of ordinary differential equations (ODE) describing the change in concentrations of target and infected cells, as well as the viral load over time. The earliest models, including the so-called standard model of virus dynamics [<a href="#B8-viruses-10-00200">8</a>], concentrated on cell-free transmission describing the spread of infection by a contact process between virions and target cells [<a href="#B11-viruses-10-00200">11</a>,<a href="#B12-viruses-10-00200">12</a>]. Mathematically, this transmission was formulated by simple mass-action kinetics. While these models provided first estimates of viral infection kinetics and cellular turnover dynamics, they ignored the process of cell-to-cell transmission. With increasing awareness of the importance of cell-to-cell transmission for infection dynamics, revised models appeared that additionally accounted for this transmission mode by extending the standard model of virus dynamics [<a href="#B13-viruses-10-00200">13</a>,<a href="#B14-viruses-10-00200">14</a>,<a href="#B15-viruses-10-00200">15</a>,<a href="#B16-viruses-10-00200">16</a>,<a href="#B17-viruses-10-00200">17</a>]. These new models considered different specific aspects of cell-associated transmission, including the possibility to transfer multiple viral strains simultaneously [<a href="#B16-viruses-10-00200">16</a>,<a href="#B17-viruses-10-00200">17</a>], as well as considering a time-delay for viral production [<a href="#B13-viruses-10-00200">13</a>]. However, all these models described cell-to-cell transmission by allowing target cells to get infected proportional to the concentration of infected cells and assumed that target and infected cells are well mixed. This assumption is especially violated for viral spread among stationary cells, e.g., within solid tissue environments. As the infection progresses, infected cells successively become surrounded by other infected cells, hindering cells in the center of infected foci from contributing to cell-to-cell transmission. Such foci of infected cells have been observed, for example, in HCV-infected liver tissue [<a href="#B18-viruses-10-00200">18</a>,<a href="#B19-viruses-10-00200">19</a>,<a href="#B20-viruses-10-00200">20</a>]. Thus, previous models relying on the mass-action assumption overestimate the fraction of infected cells contributing to cell-to-cell transmission and lead to an inappropriate quantification of the infection dynamics. There have been several approaches to improve these quantifications for various pathogens by using probabilistic [<a href="#B21-viruses-10-00200">21</a>] and spatially explicit modeling frameworks, such as agent-based models [<a href="#B22-viruses-10-00200">22</a>,<a href="#B23-viruses-10-00200">23</a>]. However, especially the latter approach is difficult to be directly fit to experimental data. Therefore, finding an extension for the standard model of virus dynamics that appropriately accounts for cell-to-cell transmission would help to improve current analyses of viral spread.</p> <p>To this end, we developed an agent-based model (ABM) to simulate viral replication and spread in a monolayer of cells. The simulated data allowed us to reveal the deficiencies of standard virus dynamics models when analyzing viral spread by cell-to-cell transmission. Based on these analyses, we developed an extension for the standard model of virus dynamics to allow for an appropriate quantification of the infection kinetics due to cell-to-cell transmission. Our novel approach adjusts for the decreasing proportion of infected cells contributing to cell-to-cell spread with progressing infection, which is especially important for viral transmission among stationary cells. In contrast to current models, our proposed model provides a more accurate quantification of the underlying infection dynamics. As such, our extension provides a means to account for viral transmission dynamics on a single cell level within the context of a model that remains applicable to common population-based experimental measurements that are used to quantify rates characterizing viral infection and spread. Additionally, we tested the ability of our novel method in comparison to previous approaches to infer the contribution of each of the transmission modes to infection spread and found that reliable approximations of these contributions depend on the diffusion rates of cell-free virus, with our novel approach leading to more appropriate estimates. Thus, our analysis provides an improved understanding of how to reliably quantify cell-to-cell transmission dynamics for various viruses spreading within solid tissue-environments.</p> </div> <div class="2.materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
2. Materials and Methods</div> <div class="2.1.thestandardmodelofviraldynamics" title="sec"> <div class="title" tagx="title" title="title">
2.1. The Standard Model of Viral Dynamics</div> <p>The standard model of virus dynamics describes the coupled dynamics of the concentration of target cells, <i>T</i>, infected cells, <i>I</i>, and the extracellular viral load, <i>V</i> [<a href="#B8-viruses-10-00200">8</a>]. In general, target cells are assumed to get infected at rate <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
f</div> </div> </div></div></span> proportional to the viral concentration <i>V</i> and have an average lifetime of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
/</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mi_UNKNOWN">
T</div> </div> </div> </div></div></span>. Infected cells produce new virions with a viral production rate <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
�?</div> </div></div></span> and are lost with rate <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mi_UNKNOWN">
I</div> </div> </div></div></span>, while free virions are cleared with rate <i>c</i>. Previous model extensions accounted for cell-to-cell transmission by including an additional infection term, also allowing target cells to get infected at a rate <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
c</div> </div> </div></div></span> proportional to the concentration of infected cells [<a href="#B13-viruses-10-00200">13</a>,<a href="#B14-viruses-10-00200">14</a>,<a href="#B15-viruses-10-00200">15</a>]. Hereby, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
c</div> </div> </div></div></span> describes the rate of cell-to-cell transmission. In summary, the basic model accounting for both transmission modes is then described by the following system of ordinary differential equations: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(1)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
T</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
t</div> </div> </div> </div> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mi_UNKNOWN">
V</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mi_UNKNOWN">
I</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mi_UNKNOWN">
T</div> </div> <div class="mi_UNKNOWN">
T</div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
I</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
t</div> </div> </div> </div> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mi_UNKNOWN">
V</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mi_UNKNOWN">
I</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mi_UNKNOWN">
I</div> </div> <div class="mi_UNKNOWN">
I</div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
V</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
t</div> </div> </div> </div> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
�?</div> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
V</div> <div class="mo_UNKNOWN">
.</div> </div> </div> </div> </div> </div></div></div> <p /> <p>As we will later introduce different extensions and sub-models of this standard model, <a href="#viruses-10-00200-t001">Table 1</a> provides an overview of all the different models used in the current study.</p> </div> <div class="2.2.simulatingviralspreadina2dagent-basedmodel" title="sec"> <div class="title" tagx="title" title="title">
2.2. Simulating Viral Spread in a 2D Agent-Based Model</div> <p>We developed and simulated spread of a positive-strand RNA virus within a monolayer of cells in vitro using an agent-based modeling approach. Cells were distributed on a two-dimensional lattice with each node denoting a single cell. We assume that each cell has a hexagonal shape with <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
6</div> </div> </div></div></span> direct neighbors and the total hexagonal shaped grid comprising 24,031 cells in total (90 cells per side).</p> <p>A sketch of the different processes considered in the agent-based model is depicted in <a href="#viruses-10-00200-f001">Figure 1</a>A. Cells are stationary and can be either infected or uninfected. Upon infection of a cell, intracellular viral replication is modeled by an ordinary differential equation describing the accumulation of positive-strand RNA, <i>R</i>. We assume density dependent replication with a maximal replication rate <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
α</div> </div></div></span> and a carrying capacity of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mi_UNKNOWN">
cap</div> </div> </div></div></span> for each cell. Positive-strand RNA is degraded with rate <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
γ</div> </div></div></span> and exported from the cell with an export rate <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
�?</div> </div></div></span> contributing to the extracellular viral concentration, <i>V</i>. However, only a fraction of the intracellular viral RNA, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
inf</div> </div> </div></div></span>, is assumed to be infectious. Thus, the intracellular replication of positive-strand RNA in an infected cell at site <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> and its export is described by the following system of ordinary differential equations: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(2)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
t</div> </div> </div> </div> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
α</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mfenced_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="mfrac_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mi_UNKNOWN">
cap</div> </div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
γ</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
�?</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> </div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
t</div> </div> </div> </div> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
inf</div> </div> <div class="mi_UNKNOWN">
�?</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
c</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
.</div> </div> </div> </div> </div> </div></div></div> <p /> <p>Extracellular virus is capable of diffusing through the lattice with diffusion modeled as seen in [<a href="#B24-viruses-10-00200">24</a>] assuming that the viral concentration at grid site <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> changes from time step <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
n</div> </div> </div></div></span> to <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div></div></span> by: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(3)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mn_UNKNOWN">
0</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mn_UNKNOWN">
0</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
n</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> <div class="mi_UNKNOWN">
k</div> </div> <div class="munder_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
∈</div> <div class="mo_UNKNOWN">
Ω</div> </div> </div> <div class="mfenced_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
n</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mn_UNKNOWN">
0</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
n</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
,</div> </div> </div></div></div> with <i>k</i> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Ω</div> </div></div></span> denoting the number and set of neighboring grid sites, respectively, and <i>m</i> the fraction of viral particles that are assumed to diffuse. An uninfected cell can get infected by cell-free transmission at each time-step with probability <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
f</div> </div> </div></div></span> dependent on the extracellular viral concentration at the corresponding grid site, <i>V</i>, and a scaling factor <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
f</div> </div> </div></div></span>, i.e., <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mi_UNKNOWN">
V</div> </div> </div></div></span>. Cell-to-cell transmission is considered by the direct transmission of intracellular viral RNA from an infected cell to its uninfected neighbor and occurs with probability <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
c</div> </div> </div></div></span> dependent on the intracellular concentration of infectious viral RNA within the infected cell, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
inf</div> </div> <div class="mi_UNKNOWN">
R</div> </div> </div></div></span>, and a scaling factor <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
c</div> </div> </div></div></span>. Upon successful infection, the concentration of intracellular viral RNA within the infecting cell is reduced due to transfer of viral material into the target cell and the novel infected cell starts viral replication with one single viral RNA. Initial viral load for cells infected through cell-free infection is also one viral RNA. Intracellular viral replication is parameterized by fitting Equation (<a href="#FD2-viruses-10-00200">2</a>) to experimental data on the spread of HCV within in vitro cultures that measured the intracellular RNA per cell and the extracellular viral load over time [<a href="#B25-viruses-10-00200">25</a>]. The estimated parameter values considered within the simulation are shown in <a href="#app1-viruses-10-00200">Table S1</a> and the resulting time courses of the concentrations for intracellular RNA, <i>R</i>, and extracellular infectious virus, <i>V</i>, for one infected cell are shown in <a href="#viruses-10-00200-f001">Figure 1</a>B.<p /> <p>Resembling previous experimental protocols [<a href="#B21-viruses-10-00200">21</a>], our ABM-simulated infection is initialized by choosing a number of cells at random that are infected with probability <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
init</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div></div></span> during the first 17 h of a simulation. Hereby, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
init</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div></div></span> is defined by: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(4)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
init</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
:</div> <div class="mo_UNKNOWN">
=</div> <div class="mover_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
t</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
17</div> <div class="mi_UNKNOWN">
λ</div> </div> </div> </div> </div> <div class="mspace_UNKNOWN" /> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
,</div> </div> </div></div></div> with <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
^</div> </div> </div></div></span> denoting the expected total number of infected cells during initialization, and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> </div></div></span> the rate at which the inoculum used for infection looses its infectivity. At 17 h post infection, the total extracellular virus concentration is reset to zero, representing the change of media.<p /> <p>The simulated cell culture system was run for 10 days and the number of infected cells, as well as the viral concentration at indicated time points was noted. The appropriateness of different population-based modeling approaches to infer the underlying parameters characterizing both transmission modes was determined by fitting these models to the simulated ABM-data. The probabilities for cell-free, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
f</div> </div> </div></div></span>, and cell-to-cell transmission, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
c</div> </div> </div></div></span>, were varied between simulations to consider different contributions of each of the two transmission modes to infection spread. An example of a simulation of the agent-based model about 3 days after the initiation of infection assuming a scaling factor <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
28</div> <div class="mspace_UNKNOWN" /> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
min</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mspace_UNKNOWN" /> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
virus</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0.82</div> <div class="mspace_UNKNOWN" /> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
min</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mspace_UNKNOWN" /> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
RNA</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div></div></span> is shown in <a href="#viruses-10-00200-f001">Figure 1</a>C. The model was implemented in the <tt>C++</tt> programming language.</p> </div> <div class="2.3.parameterestimation" title="sec"> <div class="title" tagx="title" title="title">
2.3. Parameter Estimation</div> <p>The different mathematical models describing the spread of infection, e.g., Equation (<a href="#FD1-viruses-10-00200">1</a>), were fitted to the simulated data using the optim-function in the <i>R</i>-language of statistical computing [<a href="#B26-viruses-10-00200">26</a>]. Parameters characterizing the infection dynamics were determined maximizing the log-likelihood function given by: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(5)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
θ</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
=</div> </div> </div> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mfrac_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
m</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mfenced_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mo_UNKNOWN">
¯</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
θ</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mi_UNKNOWN">
t</div> </div> </div> </div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mi_UNKNOWN">
ln</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mspace_UNKNOWN" /> <div class="mspace_UNKNOWN" /> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
with</div> </div> </div> </div> </div> </div> </div></div></div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(6)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
=</div> </div> </div> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> </div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mi_UNKNOWN">
n</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
me</div> </div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mrow_UNKNOWN"> <div class="mspace_UNKNOWN" /> <div class="mspace_UNKNOWN" /> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
and</div> <div class="mspace_UNKNOWN" /> <div class="mspace_UNKNOWN" /> <div class="mspace_UNKNOWN" /> </div> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mo_UNKNOWN">
¯</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
:</div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mi_UNKNOWN">
n</div> </div> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mi_UNKNOWN">
n</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
j</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> </div> </div> </div></div></div> <p /> <p>Hereby, <i>n</i> determines the number of different simulations, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
j</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div></div></span> the measured concentration of infected cells or the viral concentration at sampling time point <i>t</i> for simulation <i>j</i>, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mi_UNKNOWN">
t</div> </div> </div></div></span> the empirical variation across all simulations, and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
θ</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> the corresponding model prediction with <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
…</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span>. Furthermore, we assume a constant absolute measurement error, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mi_UNKNOWN">
me</div> </div> </div></div></span>, that was additionally estimated. When the extracellular viral concentration and the number of infected cells were considered in the fit, a relative measurement error was assumed. Parameter identifiability was assessed using a profile likelihood approach [<a href="#B27-viruses-10-00200">27</a>], which was also used to calculate the 95%-confidence intervals.</p> <p>Model performance was compared using the corrected Akaike information criterion (AICc) [<a href="#B28-viruses-10-00200">28</a>] calculated by: </p><div class="disp-formula" title="disp-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
AICc</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
2</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
2</div> <div class="mi_UNKNOWN">
L</div> <div class="mo_UNKNOWN">
+</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mo_UNKNOWN">
,</div> </div> </div></div></div> where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
L</div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
max</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
θ</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> defines the maximal value of the log likelihood function, <i>k</i> the number of model parameters and <i>n</i> the number of data points the model is fitted to. Differences between models were evaluated by the <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
AICc</div> </div> </div></div></span> with the difference always calculated compared to the best performing model with the lowest AICc-value within the corresponding situation.<p /> </div> </div> <div class="3.results" title="sec"> <div class="title" tagx="title" title="title">
3. Results</div> <div class="3.1.standardmodelsofvirusdynamicsareinsufficienttodescribecell-to-celltransmissiondynamicsamongstationarycells" title="sec"> <div class="title" tagx="title" title="title">
3.1. Standard Models of Virus Dynamics Are Insufficient to Describe Cell-To-Cell Transmission Dynamics among Stationary Cells</div> <p>The standard model of virus dynamics has been extensively used to analyze time courses of infection. It describes the dynamics of the concentration of target cells, <i>T</i>, infected cells, <i>I</i>, and the extracellular viral load, <i>V</i>, accounting for the infection processes and viral turnover [<a href="#B8-viruses-10-00200">8</a>] (see <a href="#viruses-10-00200-f002">Figure 2</a>A and Equation (<a href="#FD1-viruses-10-00200">1</a>) in the Materials and Methods for a detailed description of the model). In this model, the two infection processes of CF- and CC-transmission are generally described by mass-action kinetics dependent on the viral concentration, <i>V</i>, and the concentration of infected cells, <i>I</i>. Examining the latter term addressing cell-to-cell transmission in more detail, it becomes clear that all infected cells are assumed to always contribute to CC-transmission. However, this assumption is most likely violated for viral spread within spatially defined environments, especially for viral spread among stationary cells, as, e.g., in solid tissues.</p> <p>To test the ability of the basic model defined in Equation (<a href="#FD1-viruses-10-00200">1</a>) to describe cell-to-cell transmission dynamics among stationary cells, we used an agent-based model (ABM) simulating viral spread in a 2D monolayer of cells. The ABM assumes cells to be distributed in a hexagonal grid, i.e., with each cell having six different neighbors, and accounts for intracellular viral replication and intercellular viral spread (see the Materials and Methods for a detailed explanation). For simplicity, proliferation and death of infected and uninfected cells were neglected. To concentrate on the dynamics of cell-to-cell transmission, we first studied infection spread by cell-to-cell transmission starting with a single infected and assuming unlimited target cell numbers. <a href="#viruses-10-00200-f002">Figure 2</a>B shows the increase in the total number of infected cells averaged over 100 individual ABM simulations that were evaluated every 24 h for 10 days. Fitting a model that assumes standard mass-action kinetics to describe cell-to-cell transmission (CC-model, Equation (<a href="#FD1-viruses-10-00200">1</a>) with <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mi_UNKNOWN">
I</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mi_UNKNOWN">
T</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span>) to these data, we find that it is incapable of providing a reasonable representation of the infection dynamics (<a href="#viruses-10-00200-f002">Figure 2</a>B, with reduced Chi-squared-statistics <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> <div class="mn_UNKNOWN">
8</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1096</div> </div> </div></div></span> indicating poor fit considering the standard deviation of the data). Such a model fails to explain the observed decreasing growth rate of a focus of infected cells over time. Thus, simple mass-action kinetics as it has been used previously [<a href="#B11-viruses-10-00200">11</a>,<a href="#B13-viruses-10-00200">13</a>,<a href="#B14-viruses-10-00200">14</a>] seems to be inappropriate when describing viral spread by cell-to-cell transmission among stationary cells.</p> </div> <div class="3.2.improveddescriptionofcell-to-celltransmissiondynamicsaccountingforspatialeffects" title="sec"> <div class="title" tagx="title" title="title">
3.2. Improved Description of Cell-To-Cell Transmission Dynamics Accounting for Spatial Effects</div> <p>Our ABM-simulated data indicate a non-constant growth rate for foci of infected cells by cell-to-cell transmission among stationary cells. To better understand the dynamics of focus growth by cell-to-cell transmission over time, we consider a virus only capable of spreading by cell-to-cell transmission. Newly-infected cells can only appear next to already infected cells leading to large foci of infected cells. The larger a focus gets, the larger the proportion of cells that are completely surrounded by other infected cells and, hence, can no longer contribute to cell-to-cell transmission (<a href="#viruses-10-00200-f003">Figure 3</a>A). Thus, the previously assumed proportionality between successful cell-to-cell transmissions and the concentration of infected cells (Equation (<a href="#FD1-viruses-10-00200">1</a>)) might not hold true and needs to be adapted.</p> <p>To this end, we derived a correction term <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div></div></span> that adjusts the infection term <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mi_UNKNOWN">
I</div> <div class="mi_UNKNOWN">
T</div> </div> </div></div></span> for the decreasing proportion of cells contributing to cell-to-cell transmission during the progression of infection. Following a single focus of infected cells in a 2D layer of stationary cells and assuming radial focus growth, i.e., all cells in a certain radius around the focus-founding cell are infected before the next expansion phase (<a href="#viruses-10-00200-f003">Figure 3</a>A), the proportion of cells able to contribute to cell-to-cell transmission, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
c</div> </div> </div></div></span>, can be calculated by the ratio between the number of cells in the perimeter of the focus and the total focus size, </p><div class="disp-formula" title="disp-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
#</div> <div class="mi_UNKNOWN">
cells</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
in</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
perimeter</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
of</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
focus</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
#</div> <div class="mi_UNKNOWN">
cells</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
in</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
focus</div> </div> </div> <div class="mo_UNKNOWN">
.</div> </div> </div></div></div> <p /> <p>The number of cells in the perimeter depends on the assumed number of neighbors per cell, <i>k</i>, and the radius of infection, i.e., the expansion phase, <i>n</i> (<a href="#viruses-10-00200-f003">Figure 3</a>A). Initially, having only one infected cell, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> </div></div></span> as the perimeter equals the whole focus size. After the first expansion phase, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> </div></div></span>, the number of cells in the perimeter is equivalent to the total number of neighbors of a cell, i.e., <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
k</div> </div> </div></div></span>. After the second and third expansion phase, the number of cells in the perimeter doubles and triples, respectively, while the total focus size grows slightly faster (<a href="#viruses-10-00200-f003">Figure 3</a>A). In general, the proportion of cells in a focus contributing to cell-to-cell transmission (CC contributors) after completion of expansion phase <i>n</i> and dependent on the assumed number of neighbors, <i>k</i>, is determined by: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(7)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mi_UNKNOWN">
n</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
k</div> <div class="msubsup_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mi_UNKNOWN">
n</div> </div> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mi_UNKNOWN">
k</div> <div class="mi_UNKNOWN">
n</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
k</div> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> </div> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
for</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
≥</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
.</div> </div> </div> </div> </div> </div></div></div> <p /> <p>Similarly, the total number of infected cells in a focus is calculated by: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(8)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
.</div> </div> </div></div></div> <p /> <p>By solving Equation (<a href="#FD8-viruses-10-00200">8</a>) for <i>n</i> and substituting in Equation (<a href="#FD7-viruses-10-00200">7</a>), we obtain: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(9)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msqrt_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
8</div> <div class="mi_UNKNOWN">
k</div> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
8</div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
k</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mi_UNKNOWN">
I</div> </div> </div> </div> </div></div></div> as a continuous function of the assumed number of neighbors per cell and the number of infected cells in the focus (see <a href="#viruses-10-00200-f003">Figure 3</a>B, dashed red line).<p /> <p>Obviously, Equation (<a href="#FD9-viruses-10-00200">9</a>) only holds true if <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
≥</div> <div class="mi_UNKNOWN">
k</div> </div> </div></div></span>. Otherwise, all infected cells in the focus can still contribute to cell-to-cell transmission as not all neighbors of the founder cell are infected (see <a href="#viruses-10-00200-f003">Figure 3</a>A). To allow for a better approximation of the proportion of contributors for small foci sizes, we define <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> </div></div></span> for <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
≤</div> <div class="mi_UNKNOWN">
k</div> </div> </div></div></span>. In order to obtain a smooth function for <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
c</div> </div> </div></div></span> dependent on the number of infected cells, we define a connection term <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mn_UNKNOWN">
2</div> </div> </div></div></span> for <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
z</div> </div> </div></div></span>, with <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
z</div> <div class="mo_UNKNOWN">
∈</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mo_UNKNOWN">
+</div> </div> </div> </div></div></span>, to connect the proportion of contributors for small focus sizes, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mn_UNKNOWN">
1</div> </div> </div></div></span>, with the general function for larger focus sizes obtained in Equation (<a href="#FD7-viruses-10-00200">7</a>) (see <a href="#viruses-10-00200-f003">Figure 3</a>B and <a href="#app1-viruses-10-00200">Supplementary Material Text S1</a> for a detailed derivation of the connection term <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mn_UNKNOWN">
2</div> </div> </div></div></span>). Hereby, the parameter <i>z</i> defines the number of infected cells over which the transition between full contributors and ringlike focus growth is used. Thus, the final adjustment term <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div></div></span> for a single focus comprising <i>I</i> infected cells is then given by: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(10)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfenced_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
for</div> </div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
≤</div> <div class="mi_UNKNOWN">
k</div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
a</div> <div class="msup_UNKNOWN"> <div class="mfenced_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
k</div> </div> <div class="mn_UNKNOWN">
3</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
b</div> <div class="msup_UNKNOWN"> <div class="mfenced_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
k</div> </div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
for</div> </div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
z</div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msqrt_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
8</div> <div class="mi_UNKNOWN">
k</div> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
8</div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
k</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mi_UNKNOWN">
I</div> </div> </div> </div> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
for</div> </div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
≥</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
z</div> <div class="mo_UNKNOWN">
,</div> </div> </div> </div> </div> </div> </div> </div> </div> </div> </div></div></div> where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
a</div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
a</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
z</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
z</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> are functions of the parameters <i>z</i> and <i>k</i> and define coefficients of the polynomial for <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mn_UNKNOWN">
2</div> </div> </div></div></span> ensuring the smoothness of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
c</div> </div> </div></div></span> (see <a href="#app1-viruses-10-00200">Text S1</a>). <a href="#viruses-10-00200-f003">Figure 3</a>C shows the development of the proportion of CC contributors for different numbers of neighbors per cell dependent on the focus size, <i>I</i>. Note that the adjustment term described in Equation (<a href="#FD10-viruses-10-00200">10</a>) is valid for a continuous variable <i>I</i>, e.g., <i>I</i> representing the concentration of infected cells. To approximate the proportion of cells contributing to cell-to-cell transmission in a culture comprising several foci, <i>I</i> can be replaced by <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
/</div> <div class="mi_UNKNOWN">
ϕ</div> </div> </div></div></span>, where <i>I</i> defines the total number of infected cells and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ϕ</div> </div></div></span> the number of different foci at the beginning.<p /> <p>With the definition of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
c</div> </div> </div></div></span>, an adjusted model for cell-to-cell transmission dynamics that extends Equation (<a href="#FD1-viruses-10-00200">1</a>) is then defined by: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(11)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
T</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
t</div> </div> </div> </div> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mi_UNKNOWN">
V</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mi_UNKNOWN">
I</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mi_UNKNOWN">
T</div> </div> <div class="mi_UNKNOWN">
T</div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
I</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
t</div> </div> </div> </div> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mi_UNKNOWN">
V</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mi_UNKNOWN">
I</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mi_UNKNOWN">
I</div> </div> <div class="mi_UNKNOWN">
I</div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
V</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
t</div> </div> </div> </div> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
�?</div> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
V</div> </div> </div> <div class="mtd_UNKNOWN"> <div class="mo_UNKNOWN">
.</div> </div> </div> </div> </div></div></div> <p /> <p>As we always neglect cellular turnover in our simulations to focus on the effect of the spatial conditions on the transmission processes, cell death is neglected in the following, meaning <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mi_UNKNOWN">
T</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mi_UNKNOWN">
I</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span> throughout the manuscript.</p> </div> <div class="3.3.adjustedmodelprovidesimproveddescriptionofcell-to-celltransmissiondynamicsforindividualfocusgrowth" title="sec"> <div class="title" tagx="title" title="title">
3.3. Adjusted Model Provides Improved Description of Cell-To-Cell Transmission Dynamics for Individual Focus Growth</div> <p>To test the ability of our adjusted model to describe individual focus growth by cell-to-cell transmission, we fit it to the same simulated data. The adjusted model (aCC-model, Equation (<a href="#FD11-viruses-10-00200">11</a>) with <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mi_UNKNOWN">
T</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mi_UNKNOWN">
I</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span>, see also <a href="#viruses-10-00200-t001">Table 1</a>) provides a visibly better description of the growth dynamics than the standard CC-model (<a href="#viruses-10-00200-f004">Figure 4</a>A). Model fits are significantly improved (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
AICc</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
823.1</div> </div> </div></div></span> compared to <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
AICc</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
29.2</div> </div> </div></div></span> for the CC- and aCC-model, respectively) and with the adjusted cell-to-cell transmission rate, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
c</div> </div> </div> </div></div></span>, we are able to explain the decreasing growth rate of a focus over time.</p> <p>Estimates for the transmission rate characterizing cell-to-cell spread using the aCC-model are roughly 1.5-times larger than for the CC-model (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1.49</div> <div class="mo_UNKNOWN">
×</div> <div class="msup_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
6</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
cell</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
h</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mspace_UNKNOWN" /> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mn_UNKNOWN">
1.47</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1.50</div> <div class="mo_UNKNOWN">
]</div> </div> </div> </div></div></span> aCC-model vs. <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1.04</div> <div class="mo_UNKNOWN">
×</div> <div class="msup_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
6</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
cell</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
h</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div></div></span><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
[</div> <div class="mn_UNKNOWN">
1.00</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1.07</div> <div class="mo_UNKNOWN">
]</div> </div> </div></div></span> CC-model, <a href="#viruses-10-00200-t002">Table 2</a>). Profile likelihood analysis indicates that all parameters are identifiable based on the given data (<a href="#viruses-10-00200-f004">Figure 4</a>A, insert and <a href="#app1-viruses-10-00200">Figure S1</a>).</p> <p>To more closely resemble the simulated scenario, we also considered a model that additionally incorporated a time-delay for the concentration of infected cells mediating cell-to-cell transmission, i.e., accounting for the time that cells need to become infectious. To this end, the term <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mi_UNKNOWN">
I</div> <div class="mi_UNKNOWN">
T</div> </div> </div></div></span> describing cell-to-cell transmission in the aCC-model is replaced by <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mi_UNKNOWN">
I</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
τ</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mi_UNKNOWN">
T</div> </div> </div></div></span>, with <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
τ</div> </div></div></span> defining the considered time-delay. This revised aCC-d<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN" /> <div class="mi_UNKNOWN">
I</div> </div> </div></div></span>-model further improves model predictions (<a href="#viruses-10-00200-f004">Figure 4</a>B and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
AICc</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
29.2</div> </div> </div></div></span> compared to the aCC-model, <a href="#viruses-10-00200-t002">Table 2</a>). However, including a time-delay on the standard-model, i.e., CC-d<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN" /> <div class="mi_UNKNOWN">
I</div> </div> </div></div></span>, does not improve model predictions substantially (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
AICc</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
810.9</div> </div> </div></div></span> compared to aCC-d<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN" /> <div class="mi_UNKNOWN">
I</div> </div> </div></div></span>), indicating that the additional consideration of an eclipse phase does not explain all the dynamics observed in the data.</p> <p>To validate our theoretically derived adjustment term <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
c</div> </div> </div></div></span>, we compared the predicted proportion of cells contributing to cell-to-cell transmission (CC contributors) to the proportions inferred from the individual ABM simulations. We found that the aCC-model predicts a slightly higher proportion of CC contributors dependent on the focus size than observed in the simulations (<a href="#viruses-10-00200-f004">Figure 4</a>C, green line). However, the aCC-model with an additional time-delay in the concentration of infected cells contributing to CC (aCC-d<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN" /> <div class="mi_UNKNOWN">
I</div> </div> </div></div></span>-model) leads to nice agreement in the predicted and observed proportions (<a href="#viruses-10-00200-f004">Figure 4</a>C, red-dotted line).</p> <p>In summary, our adjusted model is a more appropriate approach than previous models with simple mass-action kinetics when describing individual focus growth among stationary cells. In addition, it provides a good approximation of the proportion of cells capable of contributing to cell-to-cell transmission.</p> </div> <div class="3.4.determiningcell-to-celltransmissiondynamicsacrossmultiplefoci" title="sec"> <div class="title" tagx="title" title="title">
3.4. Determining Cell-To-Cell Transmission Dynamics across Multiple Foci</div> <p>Data from in vitro experiments usually comprise the simultaneous growth of multiple foci with measurements of viral load and infected cells obtained on a population level. Here, heterogeneity in the time at which individual foci are initiated, as well as processes such as merging of foci (<a href="#viruses-10-00200-f005">Figure 5</a>A) might violate the assumptions used for the derivation of the adjustment term <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
c</div> </div> </div></div></span>.</p> <p>To evaluate the appropriateness of our adjusted model to correctly infer cell-to-cell transmission dynamics given the simultaneous growth of multiple foci, we used our agent-based model to simulate viral spread assays that allow the initiation of multiple foci. According to previous experimental conditions [<a href="#B21-viruses-10-00200">21</a>,<a href="#B29-viruses-10-00200">29</a>,<a href="#B30-viruses-10-00200">30</a>], we consider a monolayer of cells in which infection is randomly initiated during a time period of 17 h. After this initial infection period, the extracellular virus concentration is set to zero and cell-free infection is turned off, resembling the removal of medium and the addition of antibodies targeting the viral envelope in the experiment [<a href="#B21-viruses-10-00200">21</a>,<a href="#B29-viruses-10-00200">29</a>,<a href="#B30-viruses-10-00200">30</a>]. Infection is then assumed to progress solely by cell-to-cell transmission. As before, the total number of infected cells in the simulations is sampled every 24 h (see the Materials and Methods for a detailed description of the ABM). To compare the CC- and aCC-model in their ability to describe cell-to-cell transmission dynamics, we separate the two processes of initiation of infection and subsequent focus growth. Therefore, only the data after the end of the initial infection period are considered for the analysis, i.e., taking the number of infected cells 17 h post infection as initial condition.</p> <p>Again, we observed that our aCC-model performs better than the CC-model that relies on the well-mixed assumption between target and infected cells (<a href="#viruses-10-00200-f005">Figure 5</a>B and <a href="#viruses-10-00200-t002">Table 2</a>, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
AIC</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
92.88</div> </div> </div></div></span>). In this more complex scenario, a model not accounting for a decreasing growth rate with progression of infection still fails to describe the decelerated infection dynamics. As already observed for a single focus, using the adjusted model (aCC) leads to approximately a 1.5-fold increase in the estimates for the cell-to-cell transmission rate (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1.13</div> <div class="mo_UNKNOWN">
×</div> <div class="msup_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
6</div> </div> </div> <div class="mspace_UNKNOWN" /> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
cell</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mspace_UNKNOWN" /> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
h</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div></div></span><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mn_UNKNOWN">
1.07</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1.18</div> <div class="mo_UNKNOWN">
]</div> </div> </div></div></span> CC-model, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1.72</div> <div class="mo_UNKNOWN">
×</div> <div class="msup_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
6</div> </div> </div> <div class="mspace_UNKNOWN" /> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
cell</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mspace_UNKNOWN" /> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
h</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div></div></span><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mn_UNKNOWN">
1.70</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1.74</div> <div class="mo_UNKNOWN">
]</div> </div> </div></div></span> aCC-model, <a href="#viruses-10-00200-t002">Table 2</a>). In addition, we found that estimates of the cell-to-cell transmission rate for the aCC-model are robust even if the assumed neighbor-distribution, <i>k</i>, deviates to some extent from the correct distribution (see <a href="#app1-viruses-10-00200">Supplementary Material Table S2</a>). Extending our adjusted aCC-model to additionally account for the time a cell needs to become infectious (aCC-<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
I</div> </div> </div></div></span>) does not improve model fits as we estimate a delay of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
τ</div> <div class="mo_UNKNOWN">
≈</div> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span>. This is expected as, in contrast to the scenario starting with a single infected cell, there are already some infectious cells present after the end of the initiation period. Therefore, an additional time-delay can be neglected and we will concentrate on the aCC-model for the underlying scenario.</p> <p>To confirm the suitability of the adjustment term <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
c</div> </div> </div></div></span> in such a complex scenario, we compared the proportion of CC contributors among all foci of infected cells inferred from the ABM simulations to the proportion predicted by our model (<a href="#viruses-10-00200-f005">Figure 5</a>C). Here, the adjustment seems to provide a worse prediction for the proportion of CC contributors than compared to single focus growth (<a href="#viruses-10-00200-f004">Figure 4</a>C). Especially for later time points with multiple infected cells, we seem to overestimate the proportion of cells contributing to infection spread. However, the discrepancy between the proportion of CC contributors in the ABM simulations and the predicted proportion by our aCC-model is due to an uneven comparison of both terms. For the ABM, the average increase in infected cells per time step, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
ABM</div> </div> <div class="mi_UNKNOWN">
I</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
I</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
I</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div></div></span>, can be described by: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(12)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
ABM</div> </div> <div class="mi_UNKNOWN">
I</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mo_UNKNOWN">
˜</div> </div> <div class="mi_UNKNOWN">
c</div> </div> <div class="msub_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> <div class="mo_UNKNOWN">
¯</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
c</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
˜</div> </div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mi_UNKNOWN">
I</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
with</div> <div class="mspace_UNKNOWN" /> <div class="mspace_UNKNOWN" /> <div class="mspace_UNKNOWN" /> <div class="msub_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> <div class="mo_UNKNOWN">
¯</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
c</div> </div> </div> </div> </div> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
:</div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
end</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
I</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
start</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mo_UNKNOWN">
˜</div> </div> <div class="mi_UNKNOWN">
c</div> </div> <div class="msubsup_UNKNOWN"> <div class="mo_UNKNOWN">
∫</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
start</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
end</div> </div> </div> <div class="mi_UNKNOWN">
I</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
s</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msub_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
˜</div> </div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
s</div> </div> </div> <div class="mo_UNKNOWN">
.</div> </div> </div> </div> </div> </div></div></div> <p /> <p>Here, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
start</div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
end</div> </div> </div></div></span> represent the start and end time of the simulation run, respectively, and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> <div class="mo_UNKNOWN">
¯</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
c</div> </div> </div> </div></div></span> the approximated average concentration of viral particles in the cells contributing to cell-to-cell transmission. The term <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mo_UNKNOWN">
˜</div> </div> <div class="mi_UNKNOWN">
c</div> </div> </div></div></span> defines the infectivity per intracellular viral particle, and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div></div></span> the proportion of infected cells contributing to cell-to-cell transmission by accounting for the availability of uninfected target cells in their neighborhood.</p> <p>On the other hand, the increase in infected cells in our aCC-model (Equation (<a href="#FD11-viruses-10-00200">11</a>) with <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mi_UNKNOWN">
I</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mi_UNKNOWN">
T</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span>) is given by: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(13)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
ODE</div> </div> <div class="mi_UNKNOWN">
I</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
I</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
I</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
.</div> </div> </div></div></div> <p /> <p>Both equations, Equations (<a href="#FD12-viruses-10-00200">12</a>) and (<a href="#FD13-viruses-10-00200">13</a>), describe the same increase, i.e., <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
ABM</div> </div> <div class="mi_UNKNOWN">
I</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
ODE</div> </div> <div class="mi_UNKNOWN">
I</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div></div></span> (<a href="#viruses-10-00200-f005">Figure 5</a>D) with <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mo_UNKNOWN">
˜</div> </div> <div class="mi_UNKNOWN">
c</div> </div> <div class="msub_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> <div class="mo_UNKNOWN">
¯</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
c</div> </div> </div> </div> </div></div></span>. Hereby, the proportion of contributors in Equation (<a href="#FD13-viruses-10-00200">13</a>), <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
I</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div></div></span>, needs to be corrected for the proportion of target cells that are still available (<a href="#viruses-10-00200-f005">Figure 5</a>C, dashed green line). This is already implicitly considered in the corresponding term <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
˜</div> </div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div></div></span> for the ABM. Thus, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
˜</div> </div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div></div></span> corresponds to <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
I</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
/</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mn_UNKNOWN">
0</div> </div> </div> </div></div></span>, where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
/</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mn_UNKNOWN">
0</div> </div> </div> </div></div></span> describes the proportion of available target cells at time <i>t</i> with <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span> defining the total number of target cells at the start of the infection dynamics. In summary, although heterogeneous growth of multiple foci and merging of foci violate the assumptions used to derive the adjustment term <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
c</div> </div> </div></div></span>, the term still provides a reliable approximation of the proportion of cells contributing to focus growth (<a href="#viruses-10-00200-f005">Figure 5</a>C) if corrected for the available proportion of target cells. Therefore, our extended aCC-model provides a better description of the observed infection dynamics than standard models when analyzing viral spread by cell-to-cell transmission among stationary cells.</p> </div> <div class="3.5.usingpopulationdynamicsmodelstodisentanglecombinedtransmissiondynamics" title="sec"> <div class="title" tagx="title" title="title">
3.5. Using Population Dynamics Models to Disentangle Combined Transmission Dynamics</div> <p>So far, we have only considered the spread of infection by cell-to-cell transmission and showed that the adjusted CC model (aCC) provides a good description of the underlying dynamics. However, during infection, both transmission modes most likely occur simultaneously. This leads to the question to which extent population dynamics models, as described by Equations (<a href="#FD1-viruses-10-00200">1</a>) and (<a href="#FD11-viruses-10-00200">11</a>), are generally able to disentangle the contribution of each of the transmission modes to viral spread. To this end, we simulated infection dynamics allowing for simultaneous occurrence of cell-free and cell-to-cell transmission in our agent-based model. Hereby, infected cells produce virions that are exported to the extracellular space and diffuse through the simulated space before infecting other cells (see the Materials and Methods). Sampling the number of infected cells and the extracellular viral load at various time points, we then determined the ability of the different ODE models (Equations (<a href="#FD1-viruses-10-00200">1</a>) and (<a href="#FD11-viruses-10-00200">11</a>)) to predict the proportion of cells infected by either cell-free (CF proportion) or cell-to-cell transmission as observed in the ABM-simulations.</p> <p>In the first step, we considered a scenario with virions diffusing slowly through the extracellular environment, i.e., considering a low viral diffusion rate. For simplicity, we again neglected cell proliferation and cell death within the ABM-simulations. As before, we observe that our adjusted model considering both transmission modes (aCCF-model, Equation (<a href="#FD11-viruses-10-00200">11</a>) with <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mi_UNKNOWN">
I</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mi_UNKNOWN">
T</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span>) provides a substantially better description of the observed dynamics than the standard model (CCF-model, Equation (<a href="#FD1-viruses-10-00200">1</a>) with <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mi_UNKNOWN">
I</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mi_UNKNOWN">
T</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span>) (<a href="#viruses-10-00200-f006">Figure 6</a>A, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
AICc</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
60</div> </div> </div></div></span>, <a href="#viruses-10-00200-t003">Table 3</a>). Based on the estimates for <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
c</div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
f</div> </div> </div></div></span>, the CCF-model also predicts that all cells are infected by only one of the two transmission modes. In contrast, the aCCF-model indicates an increasing proportion of cells infected by cell-free transmission, which is comparable to the observations in the simulated data (<a href="#viruses-10-00200-f006">Figure 6</a>B). However, at later time points, the CF proportion is generally underestimated which indicates an underestimation of the corresponding transmission rate, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
f</div> </div> </div></div></span>. Thus, while the standard model is incapable of inferring the contribution of each of the two transmission modes to infection spread, our adjusted model provides appropriate approximations, although it underestimates the CF proportion at later time points.</p> <p>Identifying the contribution of each of the transmission modes to viral spread could potentially be facilitated if one of the transmission rates is already known, or if it can be quantified based on separate experiments. Assuming that cell-to-cell transmission can be blocked, we simulated infection spread in our ABM only allowing for cell-free transmission. Fitting the standard model of virus dynamics (CF-model, Equation (<a href="#FD1-viruses-10-00200">1</a>), <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span>) to these data, we observe that a correct quantification of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
f</div> </div> </div></div></span> is impaired under conditions in which the extracellular virus diffuses slowly (<a href="#viruses-10-00200-f006">Figure 6</a>C). Specifically, in cases of low viral diffusion rates infection tends to spread locally, causing a model assuming mass-action kinetics for infection spread to provide a poor description of the dynamics. This could already be seen for local spread by cell-to-cell transmission (e.g., <a href="#viruses-10-00200-f004">Figure 4</a>A). However, given fast viral diffusion rates (∼1.7 <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
μ</div> </div></div></span>m<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN" /> <div class="mn_UNKNOWN">
2</div> </div> </div></div></span>s<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN" /> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div></div></span> resembling those observed for HIV-sized virions diffusing in water [<a href="#B31-viruses-10-00200">31</a>], <a href="#app1-viruses-10-00200">Table S1</a>), which are roughly 165-times faster than assumed before, a model describing cell-free transmission by mass-action kinetics would provide a reasonable description of the observed dynamics as the system approaches a well-mixed situation (<a href="#viruses-10-00200-f006">Figure 6</a>C). Thus, a separate quantification of the rate of cell-free transmission is only appropriate in case of fast viral diffusion rates allowing far-range viral spread.</p> <p>We therefore tested if the simultaneous analysis of simulated data from systems considering either only cell-free or both transmission modes in case of fast viral diffusion allows us to disentangle the contribution of each of the two transmission modes to infection spread. We found that both models, CCF and aCCF, provide reasonable descriptions of the dynamics for infected cells and extracellular virus (<a href="#viruses-10-00200-f006">Figure 6</a>D,E). Furthermore, both models lead to similar estimates for the cell-free and cell-to-cell transmission rates, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
f</div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
c</div> </div> </div></div></span>, respectively (<a href="#viruses-10-00200-f006">Figure 6</a>F and <a href="#viruses-10-00200-t003">Table 3</a>). In this scenario, there is no indication for the necessity of an adjustment term that corrects for a decreasing frequency of CC contributors (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
AICc</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
9.5</div> </div> </div></div></span>, <a href="#viruses-10-00200-t003">Table 3</a>). Due to a fast viral diffusion, many new foci are initiated during the time course of infection, which in turn leads to small average focus sizes. Despite their ability to describe the infection kinetics (<a href="#viruses-10-00200-f006">Figure 6</a>F), both models slightly overestimate the proportion of cells infected by cell-free virus, although capturing the general dynamics (<a href="#viruses-10-00200-f006">Figure 6</a>D,E).</p> <p>In summary, population dynamics models are able to provide a rough approximation for the proportion of cells infected by either of the transmission modes if viral diffusion rates allow for far-range viral spread, and in case one of the transmission kinetics can be quantified separately. In the case of high viral diffusion, there is generally no evidence for using the adjusted aCCF-model. However, if CF infection tends to spread locally, i.e., if viral diffusion rates are low, using the adjusted model aCCF-model is recommended as it provides far better approximations than the previously used CCF-model.</p> </div> </div> <div class="4.discussion" title="sec"> <div class="title" tagx="title" title="title">
4. Discussion</div> <p>Direct cell-to-cell transmission of viral material between infected and uninfected cells has been found to play an important role for various viruses, including hepatitis C virus, influenza and HIV-1 (reviewed in [<a href="#B1-viruses-10-00200">1</a>]). As cell-to-cell transmission is thought to have the potential to shield the virus from host neutralizing antibodies and other viral clearance mechanisms [<a href="#B2-viruses-10-00200">2</a>,<a href="#B3-viruses-10-00200">3</a>,<a href="#B21-viruses-10-00200">21</a>,<a href="#B32-viruses-10-00200">32</a>], it has been estimated that this route of transmission is much more efficient than cell-free viral spread [<a href="#B4-viruses-10-00200">4</a>,<a href="#B5-viruses-10-00200">5</a>]. However, to reliably infer the contribution of this transmission mode for the spread of infection, appropriate mathematical models that analyze the infection dynamics are essential.</p> <p>In the current study we show that standard population-based approaches fall short to explain viral spread within spatially defined environments given cell-associated viral spread. In particular for stationary cells, ODE models relying on mass-action kinetics to describe spread of infection by cell-to-cell (CC) transmission do not explain the observed dynamics (<a href="#viruses-10-00200-f002">Figure 2</a>B). This can be explained by the fact that such models assume that all infected cells are capable of contributing to cell-to-cell spread while the actual proportion is decreasing with increasing focus sizes. Spatial effects limit the proportion of contributors as soon as infected cells are completely surrounded by other infected cells. Models not accounting for this loss underestimate the actual transmission kinetics by overestimating the population of cells driving the spread of infection. Extending previous virus dynamics models by adding a newly developed adjustment term that accounts for this decreasing proportion of CC contributors over time, we were able to correctly describe the underlying dynamics. This novel approach provides more appropriate estimates for the kinetics of cell-to-cell transmission among stationary cells, with estimates for the cell-to-cell transmission rates being roughly 1.5-times larger than estimates relying on the uncorrected models. In this scenario, estimates based on mathematical models that rely on mass-action kinetics only provide a lower bound for the transmission kinetics as they tend to overestimate the cell population contributing to infection. To what extent this also applies for cell-to-cell transmission among motile target cells still has to be investigated. However, it has already been shown that the assumption of a well-mixed situation has to be taken with care if cell-cell interactions take time, and if multiple cells interact simultaneously [<a href="#B33-viruses-10-00200">33</a>,<a href="#B34-viruses-10-00200">34</a>,<a href="#B35-viruses-10-00200">35</a>]. With the observed importance and assumed efficiency of a direct viral transmission pathway between infected and uninfected cells for several viruses, it is more important than ever to correctly account for this mode of transmission when analyzing viral spread.</p> <p>As appropriate experimental methods to distinguish between cells infected by either route of transmission are lacking, mathematical models currently represent a feasible alternative strategy to estimate the contribution of each of the transmission modes to viral spread. However, disentangling the contribution of cell-free and cell-to-cell transmission to viral spread by population dynamics models in case of their simultaneous occurrence is generally challenging (<a href="#viruses-10-00200-f006">Figure 6</a>). Our analysis based on simulated data indicates that our extended ODE model provides a better approximation for the proportion of cells infected by each transmission mode compared to previous approaches, when cell-free virus tends to spread locally (<a href="#viruses-10-00200-f006">Figure 6</a>B). However, an underestimation of the proportion of cells infected by cell-free transmission at later time points is still observed, and is to be expected. Specifically, when viral diffusion rates are low growth patterns are more likely to resemble those relying on cell-to-cell transmission dynamics causing the contribution of cell-free transmission to be underestimated as it cannot be reliably separated from growth of foci by cell-to-cell transmission. Analysis of data from experiments where either cell-free [<a href="#B21-viruses-10-00200">21</a>] or cell-to-cell transmission [<a href="#B14-viruses-10-00200">14</a>] is inhibited can be used to quantify one of the transmission kinetics separately. This additional information clearly improves the quantification of the contribution of each of the transmission modes to viral spread. Hereby, the kinetics of cell-free transmission can only be reliably estimated by population-dynamics models if viral diffusion rates allow for far-range viral spread leading to non-local growth patterns. However, even with such prior-knowledge biased estimates can still be obtained as the mean-field approximations by ODE models can only insufficiently account for the effects both transmission modes have on each other (<a href="#viruses-10-00200-f006">Figure 6</a>F).</p> <p>For simplicity, our analysis explicitly concentrated on the viral transmission dynamics and neglected infected cell death and the turnover of uninfected target cells in the simulated data and analysis. As such, our approach could be especially relevant for non-cytopathic viruses, such as hepatitis B and C virus, where cell death can be ignored until effective immune responses are induced [<a href="#B36-viruses-10-00200">36</a>,<a href="#B37-viruses-10-00200">37</a>]. To which extent target cell turnover, as well as recovery of tissue could affect the dynamics and, thus, the appropriateness of our extension term, remains to be investigated. However, several theoretical studies including target cell dynamics already revealed the limitations of modeling approaches using simple mass-action kinetics in case of spatial effects [<a href="#B24-viruses-10-00200">24</a>,<a href="#B38-viruses-10-00200">38</a>]. In ecological and epidemiological studies, it has also been shown that the consideration of space for contact-dependent infection spread has important implications on the epidemiological and evolutionary dynamics [<a href="#B39-viruses-10-00200">39</a>,<a href="#B40-viruses-10-00200">40</a>]. Spatial separation of the susceptible population can lead to delayed and more reduced infection kinetics, as well as a higher likelihood of epidemics to go extinct [<a href="#B39-viruses-10-00200">39</a>]. A number of different modeling frameworks have been developed that study infection spread within populations and that have also been used to determine the effect of space in the context of viral spread within tissues. These modeling frameworks include meta-population models [<a href="#B24-viruses-10-00200">24</a>], pairwise-approximation or spatial moment equations [<a href="#B41-viruses-10-00200">41</a>], models based on partial differential equations [<a href="#B31-viruses-10-00200">31</a>], and even sophisticated agent-based models and cellular automata [<a href="#B22-viruses-10-00200">22</a>,<a href="#B23-viruses-10-00200">23</a>]. However, these studies mainly concentrated on the qualitative behavior of the systems due to the general difficulty of fitting them to experimental data. Nevertheless, they already revealed differences in the infection dynamics in comparison to non-spatial models and limitations for their assumptions [<a href="#B24-viruses-10-00200">24</a>,<a href="#B38-viruses-10-00200">38</a>,<a href="#B42-viruses-10-00200">42</a>]. Thus, especially in the context of experimental systems of increasing complexity that mimic natural tissue structures [<a href="#B43-viruses-10-00200">43</a>,<a href="#B44-viruses-10-00200">44</a>,<a href="#B45-viruses-10-00200">45</a>], reliably accounting for space will be important when analyzing viral spread.</p> <p>Our novel approach represents a first step towards an improved quantification of cell-to-cell transmission dynamics within complex environments using ODE-based virus dynamics models. It is especially relevant for viruses spreading within solid tissues, such as hepatitis C virus within the liver [<a href="#B18-viruses-10-00200">18</a>,<a href="#B20-viruses-10-00200">20</a>] or influenza spreading in lung epithelial tissue [<a href="#B23-viruses-10-00200">23</a>]. Most importantly, the extended model formalism allows for consideration of viral spread on the cellular level, while remaining applicable to the standard population-based experimental data typically available for analysis (e.g., [<a href="#B14-viruses-10-00200">14</a>]). While individual cell-based approaches, such as agent-based models as presented here and elsewhere [<a href="#B22-viruses-10-00200">22</a>,<a href="#B46-viruses-10-00200">46</a>], allow a more detailed description and analysis of the transmission kinetics, the complex data required to utilize such models is less often available, and the computational complexity of the models themselves can be limiting. In combination with the advances in computational methods that allow parameter estimation for such sophisticated models [<a href="#B47-viruses-10-00200">47</a>], these approaches could enhance the possibility to infer the kinetics and contribution of each of the transmission modes to viral spread for different pathogens and, thus, could provide necessary information for the development of potent antiviral strategies.</p> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>Peter Kumberger and Frederik Graw were funded by the Center for Modeling and Simulation in the Biosciences (BIOMS). The research was supported in part by the U.S. National Institute of Health (NIH) Grant R01-AI078881 (Susan L. Uprichard and Harel Dahari). We acknowledge financial support by Deutsche Forschungsgemeinschaft within the funding program Open Access Publishing, by the Baden-Württemberg Ministry of Science, Research and the Arts and by Ruprecht-Karls-Universität Heidelberg. The funders had no role in the design of the study; in the collection, analysis or interpretation of data; in the writing of the manuscript; nor in the decision to publish the results.</p> </div> <div class="app-group" title="app-group"> <div class="app" title="app"> <div class="title" tagx="title" title="title">
Supplementary Materials</div> <p>The following are available online at <span class="uri" title="uri"><a href="http://www.mdpi.com/1999-4915/10/4/200/s1">http://www.mdpi.com/1999-4915/10/4/200/s1</a></span>: Figure S1: Profile likelihood analysis for parameter estimates considering single focus growth, Table S1: Parameters used for simulating viral spread in the agent-based model, Table S2: Estimates for the aCC-model given multiple foci using different fixed values for the assumed number of neighbors per cell, <i>k</i>, Text S1: Mathematical derivation of a continuous adjustment term for cells contributing to cell-to-cell transmission.</p> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="viruses-10-00200-s001.zip">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>F.G. conceived of and designed the study. P.K. and F.G. performed the analysis and developed the methods. P.K., S.L.U., H.D. and F.G. analyzed the data. K.D. and S.L.U. contributed reagents/materials/analysis tools. P.K., S.L.U., H.D. and F.G. wrote the paper.</p> <div class="title" tagx="title" title="title">
Conflicts of Interest</div> <p>The authors declare no conflict of interest.</p> <div class="glossary" title="glossary"> <div class="title" tagx="title" title="title">
Abbreviations</div> <p>The following abbreviations are used in this manuscript:</p><span class="array" tagx="array" title="array">   CC Cell-to-cell   CF Cell-free  </span></div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="B1-viruses-10-00200" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sattentau</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Avoiding the void: Cell-to-cell spread of human viruses</span><span class="source" tagx="source" title="source">Nat. Rev. Microbiol.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">815</span><span class="lpage" tagx="lpage" title="lpage">826</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrmicro1972">10.1038/nrmicro1972</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18923409">18923409</a></span></span></li> <li tag="ref"><a name="B2-viruses-10-00200" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sigal</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">J.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balazs</span><span class="given-names" tagx="given-names" title="given-names">A.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dekel</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mayo</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milo</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baltimore</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cell-to-cell spread of HIV permits ongoing replication despite antiretroviral therapy</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">477</span><span class="fpage" tagx="fpage" title="fpage">95</span><span class="lpage" tagx="lpage" title="lpage">98</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature10347">10.1038/nature10347</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21849975">21849975</a></span></span></li> <li tag="ref"><a name="B3-viruses-10-00200" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brimacombe</span><span class="given-names" tagx="given-names" title="given-names">C.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grove</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meredith</span><span class="given-names" tagx="given-names" title="given-names">L.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Syder</span><span class="given-names" tagx="given-names" title="given-names">A.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Flores</span><span class="given-names" tagx="given-names" title="given-names">M.V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Timpe</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krieger</span><span class="given-names" tagx="given-names" title="given-names">S.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baumert</span><span class="given-names" tagx="given-names" title="given-names">T.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tellinghuisen</span><span class="given-names" tagx="given-names" title="given-names">T.L.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Neutralizing antibody-resistant hepatitis C virus cell-to-cell transmission</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">85</span><span class="fpage" tagx="fpage" title="fpage">596</span><span class="lpage" tagx="lpage" title="lpage">605</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.01592-10">10.1128/JVI.01592-10</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20962076">20962076</a></span></span></li> <li tag="ref"><a name="B4-viruses-10-00200" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hubner</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spinelli</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">B.K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Predominant mode of human immunodeficiency virus transfer between T cells is mediated by sustained Env-dependent neutralization-resistant virological synapses</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">81</span><span class="fpage" tagx="fpage" title="fpage">12582</span><span class="lpage" tagx="lpage" title="lpage">12595</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00381-07">10.1128/JVI.00381-07</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17728240">17728240</a></span></span></li> <li tag="ref"><a name="B5-viruses-10-00200" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sato</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Orenstein</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dimitrov</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cell-to-cell spread of HIV-1 occurs within minutes and may not involve the participation of virus particles</span><span class="source" tagx="source" title="source">Virology</span><span class="year" tagx="year" title="year">1992</span><span class="volume" tagx="volume" title="volume">186</span><span class="fpage" tagx="fpage" title="fpage">712</span><span class="lpage" tagx="lpage" title="lpage">724</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0042-6822(92)90038-Q">10.1016/0042-6822(92)90038-Q</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1370739">1370739</a></span></span></li> <li tag="ref"><a name="B6-viruses-10-00200" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abela</span><span class="given-names" tagx="given-names" title="given-names">I.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berlinger</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schanz</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reynell</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gunthard</span><span class="given-names" tagx="given-names" title="given-names">H.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rusert</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trkola</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cell-cell transmission enables HIV-1 to evade inhibition by potent CD4bs directed antibodies</span><span class="source" tagx="source" title="source">PLoS Pathog.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1002634</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1002634">10.1371/journal.ppat.1002634</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22496655">22496655</a></span></span></li> <li tag="ref"><a name="B7-viruses-10-00200" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barretto</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sainz</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hussain</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uprichard</span><span class="given-names" tagx="given-names" title="given-names">S.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Determining the involvement and therapeutic implications of host cellular factors in hepatitis C virus cell-to-cell spread</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">88</span><span class="fpage" tagx="fpage" title="fpage">5050</span><span class="lpage" tagx="lpage" title="lpage">5061</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.03241-13">10.1128/JVI.03241-13</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24554660">24554660</a></span></span></li> <li tag="ref"><a name="B8-viruses-10-00200" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perelson</span><span class="given-names" tagx="given-names" title="given-names">A.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Modelling viral and immune system dynamics</span><span class="source" tagx="source" title="source">Nat. Rev. Immunol.</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">28</span><span class="lpage" tagx="lpage" title="lpage">36</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nri700">10.1038/nri700</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11905835">11905835</a></span></span></li> <li tag="ref"><a name="B9-viruses-10-00200" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Canini</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perelson</span><span class="given-names" tagx="given-names" title="given-names">A.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Viral kinetic modeling: State of the art</span><span class="source" tagx="source" title="source">J. Pharmacokinet. Pharmacodyn.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">41</span><span class="fpage" tagx="fpage" title="fpage">431</span><span class="lpage" tagx="lpage" title="lpage">443</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10928-014-9363-3">10.1007/s10928-014-9363-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24961742">24961742</a></span></span></li> <li tag="ref"><a name="B10-viruses-10-00200" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graw</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perelson</span><span class="given-names" tagx="given-names" title="given-names">A.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Modeling Viral Spread</span><span class="source" tagx="source" title="source">Annu. Rev. Virol.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">555</span><span class="lpage" tagx="lpage" title="lpage">572</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev-virology-110615-042249">10.1146/annurev-virology-110615-042249</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27618637">27618637</a></span></span></li> <li tag="ref"><a name="B11-viruses-10-00200" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perelson</span><span class="given-names" tagx="given-names" title="given-names">A.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neumann</span><span class="given-names" tagx="given-names" title="given-names">A.U.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Markowitz</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leonard</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ho</span><span class="given-names" tagx="given-names" title="given-names">D.D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">HIV-1 dynamics in vivo: Virion clearance rate, infected cell life-span, and viral generation time</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">1996</span><span class="volume" tagx="volume" title="volume">271</span><span class="fpage" tagx="fpage" title="fpage">1582</span><span class="lpage" tagx="lpage" title="lpage">1586</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.271.5255.1582">10.1126/science.271.5255.1582</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8599114">8599114</a></span></span></li> <li tag="ref"><a name="B12-viruses-10-00200" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neumann</span><span class="given-names" tagx="given-names" title="given-names">A.U.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lam</span><span class="given-names" tagx="given-names" title="given-names">N.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dahari</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gretch</span><span class="given-names" tagx="given-names" title="given-names">D.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wiley</span><span class="given-names" tagx="given-names" title="given-names">T.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Layden</span><span class="given-names" tagx="given-names" title="given-names">T.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perelson</span><span class="given-names" tagx="given-names" title="given-names">A.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Hepatitis C viral dynamics in vivo and the antiviral efficacy of interferon-alpha therapy</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">282</span><span class="fpage" tagx="fpage" title="fpage">103</span><span class="lpage" tagx="lpage" title="lpage">107</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.282.5386.103">10.1126/science.282.5386.103</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9756471">9756471</a></span></span></li> <li tag="ref"><a name="B13-viruses-10-00200" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Culshaw</span><span class="given-names" tagx="given-names" title="given-names">R.V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruan</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webb</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A mathematical model of cell-to-cell spread of HIV-1 that includes a time delay</span><span class="source" tagx="source" title="source">J. Math. Biol.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">46</span><span class="fpage" tagx="fpage" title="fpage">425</span><span class="lpage" tagx="lpage" title="lpage">444</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00285-002-0191-5">10.1007/s00285-002-0191-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12750834">12750834</a></span></span></li> <li tag="ref"><a name="B14-viruses-10-00200" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iwami</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takeuchi</span><span class="given-names" tagx="given-names" title="given-names">J.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakaoka</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mammano</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clavel</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Inaba</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kobayashi</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Misawa</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aihara</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koyanagi</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Cell-to-cell infection by HIV contributes over half of virus infection</span><span class="source" tagx="source" title="source">eLife</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">e08150</span><span class="pub-id"><a href="https://dx.doi.org/10.7554/eLife.08150">10.7554/eLife.08150</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26441404">26441404</a></span></span></li> <li tag="ref"><a name="B15-viruses-10-00200" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Groppelli</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pellegrino</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Williams</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Borrow</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chain</span><span class="given-names" tagx="given-names" title="given-names">B.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jolly</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Hybrid spreading mechanisms and T cell activation shape the dynamics of HIV-1 infection</span><span class="source" tagx="source" title="source">PLoS Comput. Biol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">11</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1004179</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pcbi.1004179">10.1371/journal.pcbi.1004179</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25837979">25837979</a></span></span></li> <li tag="ref"><a name="B16-viruses-10-00200" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Komarova</span><span class="given-names" tagx="given-names" title="given-names">N.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levy</span><span class="given-names" tagx="given-names" title="given-names">D.N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wodarz</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Synaptic transmission and the susceptibility of HIV infection to anti-viral drugs</span><span class="source" tagx="source" title="source">Sci. Rep.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">2103</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/srep02103">10.1038/srep02103</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23811684">23811684</a></span></span></li> <li tag="ref"><a name="B17-viruses-10-00200" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Komarova</span><span class="given-names" tagx="given-names" title="given-names">N.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wodarz</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Virus dynamics in the presence of synaptic transmission</span><span class="source" tagx="source" title="source">Math. Biosci.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">242</span><span class="fpage" tagx="fpage" title="fpage">161</span><span class="lpage" tagx="lpage" title="lpage">171</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.mbs.2013.01.003">10.1016/j.mbs.2013.01.003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23357287">23357287</a></span></span></li> <li tag="ref"><a name="B18-viruses-10-00200" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kandathil</span><span class="given-names" tagx="given-names" title="given-names">A.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graw</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quinn</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hwang</span><span class="given-names" tagx="given-names" title="given-names">H.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Torbenson</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perelson</span><span class="given-names" tagx="given-names" title="given-names">A.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ray</span><span class="given-names" tagx="given-names" title="given-names">S.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thomas</span><span class="given-names" tagx="given-names" title="given-names">D.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ribeiro</span><span class="given-names" tagx="given-names" title="given-names">R.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balagopal</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Use of laser capture microdissection to map hepatitis C virus-positive hepatocytes in human liver</span><span class="source" tagx="source" title="source">Gastroenterology</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">145</span><span class="fpage" tagx="fpage" title="fpage">1404</span><span class="lpage" tagx="lpage" title="lpage">1413</span><span class="pub-id"><a href="https://dx.doi.org/10.1053/j.gastro.2013.08.034">10.1053/j.gastro.2013.08.034</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23973767">23973767</a></span></span></li> <li tag="ref"><a name="B19-viruses-10-00200" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graw</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balagopal</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kandathil</span><span class="given-names" tagx="given-names" title="given-names">A.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ray</span><span class="given-names" tagx="given-names" title="given-names">S.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thomas</span><span class="given-names" tagx="given-names" title="given-names">D.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ribeiro</span><span class="given-names" tagx="given-names" title="given-names">R.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perelson</span><span class="given-names" tagx="given-names" title="given-names">A.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Inferring viral dynamics in chronically HCV infected patients from the spatial distribution of infected hepatocytes</span><span class="source" tagx="source" title="source">PLoS Comput. Biol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1003934</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pcbi.1003934">10.1371/journal.pcbi.1003934</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25393308">25393308</a></span></span></li> <li tag="ref"><a name="B20-viruses-10-00200" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wieland</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Makowska</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Campana</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Calabrese</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dill</span><span class="given-names" tagx="given-names" title="given-names">M.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chung</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chisari</span><span class="given-names" tagx="given-names" title="given-names">F.V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heim</span><span class="given-names" tagx="given-names" title="given-names">M.H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Simultaneous detection of hepatitis C virus and interferon stimulated gene expression in infected human liver</span><span class="source" tagx="source" title="source">Hepatology</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">59</span><span class="fpage" tagx="fpage" title="fpage">2121</span><span class="lpage" tagx="lpage" title="lpage">2130</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/hep.26770">10.1002/hep.26770</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24122862">24122862</a></span></span></li> <li tag="ref"><a name="B21-viruses-10-00200" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graw</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">D.N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perelson</span><span class="given-names" tagx="given-names" title="given-names">A.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uprichard</span><span class="given-names" tagx="given-names" title="given-names">S.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dahari</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Quantification of Hepatitis C Virus Cell-to-Cell Spread Using a Stochastic Modeling Approach</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">89</span><span class="fpage" tagx="fpage" title="fpage">6551</span><span class="lpage" tagx="lpage" title="lpage">6561</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00016-15">10.1128/JVI.00016-15</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25833046">25833046</a></span></span></li> <li tag="ref"><a name="B22-viruses-10-00200" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bauer</span><span class="given-names" tagx="given-names" title="given-names">A.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beauchemin</span><span class="given-names" tagx="given-names" title="given-names">C.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perelson</span><span class="given-names" tagx="given-names" title="given-names">A.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Agent-based modeling of host-pathogen systems: The successes and challenges</span><span class="source" tagx="source" title="source">Inform. Sci.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">179</span><span class="fpage" tagx="fpage" title="fpage">1379</span><span class="lpage" tagx="lpage" title="lpage">1389</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ins.2008.11.012">10.1016/j.ins.2008.11.012</a></span></span></li> <li tag="ref"><a name="B23-viruses-10-00200" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beauchemin</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Samuel</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tuszynski</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A simple cellular automaton model for influenza A viral infections</span><span class="source" tagx="source" title="source">J. Theor. Biol.</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">232</span><span class="fpage" tagx="fpage" title="fpage">223</span><span class="lpage" tagx="lpage" title="lpage">234</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jtbi.2004.08.001">10.1016/j.jtbi.2004.08.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15530492">15530492</a></span></span></li> <li tag="ref"><a name="B24-viruses-10-00200" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Funk</span><span class="given-names" tagx="given-names" title="given-names">G.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jansen</span><span class="given-names" tagx="given-names" title="given-names">V.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonhoeffer</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Killingback</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Spatial models of virus-immune dynamics</span><span class="source" tagx="source" title="source">J. Theor. Biol.</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">233</span><span class="fpage" tagx="fpage" title="fpage">221</span><span class="lpage" tagx="lpage" title="lpage">236</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jtbi.2004.10.004">10.1016/j.jtbi.2004.10.004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15619362">15619362</a></span></span></li> <li tag="ref"><a name="B25-viruses-10-00200" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aunins</span><span class="given-names" tagx="given-names" title="given-names">T.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marsh</span><span class="given-names" tagx="given-names" title="given-names">K.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Subramanya</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uprichard</span><span class="given-names" tagx="given-names" title="given-names">S.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perelson</span><span class="given-names" tagx="given-names" title="given-names">A.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chatterjee</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Intracellular hepatitis C modeling predicts infection dynamics and viral protein mechanisms</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2018</span><span class="comment">in press</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.02098-17">10.1128/JVI.02098-17</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29563295">29563295</a></span></span></li> <li tag="ref"><a name="B26-viruses-10-00200" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="collab'">collab: R Core Team</span></span><span class="source" tagx="source" title="source">R: A Language and Environment for Statistical Computing</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">R Foundation for Statistical Computing</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Vienna, Austria</span><span class="year" tagx="year" title="year">2016</span></span></li> <li tag="ref"><a name="B27-viruses-10-00200" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raue</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kreutz</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maiwald</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bachmann</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schilling</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klingmüller</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Timmer</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structural and practical identifiability analysis of partially observed dynamical models by exploiting the profile likelihood</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">1923</span><span class="lpage" tagx="lpage" title="lpage">1929</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btp358">10.1093/bioinformatics/btp358</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505944">19505944</a></span></span></li> <li tag="ref"><a name="B28-viruses-10-00200" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burnham</span><span class="given-names" tagx="given-names" title="given-names">K.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anderson</span><span class="given-names" tagx="given-names" title="given-names">D.R.</span></span></span><span class="source" tagx="source" title="source">Model Selection and Multinomal Inference: A Practical Information-Theoretic Approach</span><span class="edition" tagx="edition" title="edition">2nd ed.</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York, NY, USA</span><span class="year" tagx="year" title="year">2002</span></span></li> <li tag="ref"><a name="B29-viruses-10-00200" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sabahi</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marsh</span><span class="given-names" tagx="given-names" title="given-names">K.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dahari</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Corcoran</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lamora</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garry</span><span class="given-names" tagx="given-names" title="given-names">R.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uprichard</span><span class="given-names" tagx="given-names" title="given-names">S.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The rate of hepatitis C virus infection initiation in vitro is directly related to particle density</span><span class="source" tagx="source" title="source">Virology</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">407</span><span class="fpage" tagx="fpage" title="fpage">110</span><span class="lpage" tagx="lpage" title="lpage">119</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virol.2010.07.026">10.1016/j.virol.2010.07.026</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20800257">20800257</a></span></span></li> <li tag="ref"><a name="B30-viruses-10-00200" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barretto</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uprichard</span><span class="given-names" tagx="given-names" title="given-names">S.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Hepatitis C virus Cell-to-cell Spread Assay</span><span class="source" tagx="source" title="source">Bio-Protocol</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">4</span><span class="pub-id"><a href="https://dx.doi.org/10.21769/BioProtoc.1365">10.21769/BioProtoc.1365</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29104884">29104884</a></span></span></li> <li tag="ref"><a name="B31-viruses-10-00200" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Strain</span><span class="given-names" tagx="given-names" title="given-names">M.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richman</span><span class="given-names" tagx="given-names" title="given-names">D.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">J.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levine</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Spatiotemporal dynamics of HIV propagation</span><span class="source" tagx="source" title="source">J. Theor. Biol.</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">218</span><span class="fpage" tagx="fpage" title="fpage">85</span><span class="lpage" tagx="lpage" title="lpage">96</span><span class="pub-id"><a href="https://dx.doi.org/10.1006/jtbi.2002.3055">10.1006/jtbi.2002.3055</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12297072">12297072</a></span></span></li> <li tag="ref"><a name="B32-viruses-10-00200" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reh</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Magnus</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schanz</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weber</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uhr</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rusert</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trkola</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Capacity of Broadly Neutralizing Antibodies to Inhibit HIV-1 Cell-Cell Transmission Is Strain- and Epitope-Dependent</span><span class="source" tagx="source" title="source">PLoS Pathog.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">11</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1004966</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1004966">10.1371/journal.ppat.1004966</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26158270">26158270</a></span></span></li> <li tag="ref"><a name="B33-viruses-10-00200" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graw</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Regoes</span><span class="given-names" tagx="given-names" title="given-names">R.R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Investigating CTL Mediated Killing with a 3D Cellular Automaton</span><span class="source" tagx="source" title="source">PLOS Comput. Biol.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">12</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pcbi.1000466">10.1371/journal.pcbi.1000466</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19696876">19696876</a></span></span></li> <li tag="ref"><a name="B34-viruses-10-00200" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gadhamsetty</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marée</span><span class="given-names" tagx="given-names" title="given-names">A.F.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beltman</span><span class="given-names" tagx="given-names" title="given-names">J.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Boer</span><span class="given-names" tagx="given-names" title="given-names">R.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A General Functional Response of Cytotoxic T Lymphocyte-Mediated Killing of Target Cells</span><span class="source" tagx="source" title="source">Biophys. J.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">106</span><span class="fpage" tagx="fpage" title="fpage">1780</span><span class="lpage" tagx="lpage" title="lpage">1791</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bpj.2014.01.048">10.1016/j.bpj.2014.01.048</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24739177">24739177</a></span></span></li> <li tag="ref"><a name="B35-viruses-10-00200" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pilyugin</span><span class="given-names" tagx="given-names" title="given-names">S.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antia</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Modeling immune responses with handling time</span><span class="source" tagx="source" title="source">Bull. Math. Biol.</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">62</span><span class="fpage" tagx="fpage" title="fpage">869</span><span class="lpage" tagx="lpage" title="lpage">890</span><span class="pub-id"><a href="https://dx.doi.org/10.1006/bulm.2000.0181">10.1006/bulm.2000.0181</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11016088">11016088</a></span></span></li> <li tag="ref"><a name="B36-viruses-10-00200" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dahari</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Major</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mihalik</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rice</span><span class="given-names" tagx="given-names" title="given-names">C.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perelson</span><span class="given-names" tagx="given-names" title="given-names">A.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feinstone</span><span class="given-names" tagx="given-names" title="given-names">S.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neumann</span><span class="given-names" tagx="given-names" title="given-names">A.U.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mathematical modeling of primary hepatitis C infection: Noncytolytic clearance and early blockage of virion production</span><span class="source" tagx="source" title="source">Gastroenterology</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">128</span><span class="fpage" tagx="fpage" title="fpage">1056</span><span class="lpage" tagx="lpage" title="lpage">1066</span><span class="pub-id"><a href="https://dx.doi.org/10.1053/j.gastro.2005.01.049">10.1053/j.gastro.2005.01.049</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15825086">15825086</a></span></span></li> <li tag="ref"><a name="B37-viruses-10-00200" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Serti</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Block</span><span class="given-names" tagx="given-names" title="given-names">P.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chung</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chayama</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rehermann</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">T.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Hepatitis B virus evades innate immunity of hepatocytes but activates cytokine production by macrophages</span><span class="source" tagx="source" title="source">Hepatology</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">66</span><span class="fpage" tagx="fpage" title="fpage">1779</span><span class="lpage" tagx="lpage" title="lpage">1793</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/hep.29348">10.1002/hep.29348</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28665004">28665004</a></span></span></li> <li tag="ref"><a name="B38-viruses-10-00200" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beauchemin</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Probing the effects of the well-mixed assumption on viral infection dynamics</span><span class="source" tagx="source" title="source">J. Theor. Biol.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">242</span><span class="fpage" tagx="fpage" title="fpage">464</span><span class="lpage" tagx="lpage" title="lpage">477</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jtbi.2006.03.014">10.1016/j.jtbi.2006.03.014</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16650441">16650441</a></span></span></li> <li tag="ref"><a name="B39-viruses-10-00200" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keeling</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rohani</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="source" tagx="source" title="source">Modeling Infectious Diseases in Humans and Animals</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Princeton University Press</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Princeton, NJ, USA</span><span class="year" tagx="year" title="year">2008</span></span></li> <li tag="ref"><a name="B40-viruses-10-00200" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lion</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baalen</span><span class="given-names" tagx="given-names" title="given-names">M.V.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Self-structuring in spatial evolutionary ecology</span><span class="source" tagx="source" title="source">Ecol. Lett.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">277</span><span class="lpage" tagx="lpage" title="lpage">295</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1461-0248.2007.01132.x">10.1111/j.1461-0248.2007.01132.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18070102">18070102</a></span></span></li> <li tag="ref"><a name="B41-viruses-10-00200" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lion</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Moment equations in spatial evolutionary ecology</span><span class="source" tagx="source" title="source">J. Theor. Biol.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">405</span><span class="fpage" tagx="fpage" title="fpage">46</span><span class="lpage" tagx="lpage" title="lpage">57</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jtbi.2015.10.014">10.1016/j.jtbi.2015.10.014</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26555844">26555844</a></span></span></li> <li tag="ref"><a name="B42-viruses-10-00200" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holder</span><span class="given-names" tagx="given-names" title="given-names">B.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liao</span><span class="given-names" tagx="given-names" title="given-names">L.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simon</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boivin</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beauchemin</span><span class="given-names" tagx="given-names" title="given-names">C.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Design considerations in building in silico equivalents of common experimental influenza virus assays</span><span class="source" tagx="source" title="source">Autoimmunity</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">44</span><span class="fpage" tagx="fpage" title="fpage">282</span><span class="lpage" tagx="lpage" title="lpage">293</span><span class="pub-id"><a href="https://dx.doi.org/10.3109/08916934.2011.523267">10.3109/08916934.2011.523267</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21244331">21244331</a></span></span></li> <li tag="ref"><a name="B43-viruses-10-00200" /><span class="label" tagx="label" title="label">43.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fackler</span><span class="given-names" tagx="given-names" title="given-names">O.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murooka</span><span class="given-names" tagx="given-names" title="given-names">T.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Imle</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mempel</span><span class="given-names" tagx="given-names" title="given-names">T.R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Adding new dimensions: Towards an integrative understanding of HIV-1 spread</span><span class="source" tagx="source" title="source">Nat. Rev. Microbiol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">563</span><span class="lpage" tagx="lpage" title="lpage">574</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrmicro3309">10.1038/nrmicro3309</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25029025">25029025</a></span></span></li> <li tag="ref"><a name="B44-viruses-10-00200" /><span class="label" tagx="label" title="label">44.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glushakova</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baibakov</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Margolis</span><span class="given-names" tagx="given-names" title="given-names">L.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zimmerberg</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Infection of human tonsil histocultures: A model for HIV pathogenesis</span><span class="source" tagx="source" title="source">Nat. Med.</span><span class="year" tagx="year" title="year">1995</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">1320</span><span class="lpage" tagx="lpage" title="lpage">1322</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nm1295-1320">10.1038/nm1295-1320</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7489416">7489416</a></span></span></li> <li tag="ref"><a name="B45-viruses-10-00200" /><span class="label" tagx="label" title="label">45.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baker</span><span class="given-names" tagx="given-names" title="given-names">B.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">C.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Deconstructing the third dimension: How 3D culture microenvironments alter cellular cues</span><span class="source" tagx="source" title="source">J. Cell. Sci.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">125</span><span class="fpage" tagx="fpage" title="fpage">3015</span><span class="lpage" tagx="lpage" title="lpage">3024</span><span class="pub-id"><a href="https://dx.doi.org/10.1242/jcs.079509">10.1242/jcs.079509</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22797912">22797912</a></span></span></li> <li tag="ref"><a name="B46-viruses-10-00200" /><span class="label" tagx="label" title="label">46.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beltman</span><span class="given-names" tagx="given-names" title="given-names">J.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maree</span><span class="given-names" tagx="given-names" title="given-names">A.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lynch</span><span class="given-names" tagx="given-names" title="given-names">J.N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Boer</span><span class="given-names" tagx="given-names" title="given-names">R.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Lymph node topology dictates T cell migration behavior</span><span class="source" tagx="source" title="source">J. Exp. Med.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">204</span><span class="fpage" tagx="fpage" title="fpage">771</span><span class="lpage" tagx="lpage" title="lpage">780</span><span class="pub-id"><a href="https://dx.doi.org/10.1084/jem.20061278">10.1084/jem.20061278</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17389236">17389236</a></span></span></li> <li tag="ref"><a name="B47-viruses-10-00200" /><span class="label" tagx="label" title="label">47.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jagiella</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rickert</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Theis</span><span class="given-names" tagx="given-names" title="given-names">F.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hasenauer</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Parallelization and High-Performance Computing Enables Automated Statistical Inference of Multi-scale Models</span><span class="source" tagx="source" title="source">Cell Syst.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">194</span><span class="lpage" tagx="lpage" title="lpage">206</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cels.2016.12.002">10.1016/j.cels.2016.12.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28089542">28089542</a></span></span></li> </ul> </div> </div> <div class="floats-group" title="floats-group"> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>(<b>A</b>) Schematic of the agent-based model simulating intracellular viral replication and spread by cell-free (CF) and cell-to-cell (CC) transmission among stationary cells. Cells are assumed to be hexagonally shaped and can be either uninfected (gray) or infected (red). The parameters <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
c</div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
f</div> </div> </div></div></span> define the probability of infection by CC- and CF-transmission, respectively, dependent on the intra- and extra-cellular viral load at the corresponding grid sites; (<b>B</b>) Simulated time courses of intracellular viral load (black line) and produced extracellular virus (gray line) for one infected cell; (<b>C</b>) Realization of simulation outcomes after around three days post infection assuming simultaneous occurrence of CF- and CC-transmission (left) or only CC-transmission (right). Cells infected by CF or CC-transmission are indicated in blue and orange, respectively.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>(<b>A</b>) Sketch of a viral dynamics model in a well-mixed system as given in Equation (<a href="#FD1-viruses-10-00200">1</a>). In case of cell-to-cell (CC) transmission target cells (<i>T</i>, gray) become infected (<i>I</i>, red) with rate <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
c</div> </div> </div></div></span> dependent on the concentration of target and infected cells. Infection by cell-free virus occurs with rate <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
f</div> </div> </div></div></span> dependent on the viral load, <i>V</i>, and the target cell concentration. Viral particles are produced by infected cells with rate <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
�?</div> </div></div></span> and cleared at rate <i>c</i>. For simplicity, the processes of cell death and proliferation are neglected here; (<b>B</b>) Simulated dynamics and model predictions for viral spread only considering CC among stationary cells starting with a single infected cell. Black dots indicate the mean and vertical lines represent <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
95</div> <div class="mo_UNKNOWN">
%</div> </div> </div></div></span>-confidence interval for the total number of infected cells averaged over 100 individual ABM simulations sampled every 24 h. The red line indicates the best fit of the basic model as described in Equation (<a href="#FD1-viruses-10-00200">1</a>) with <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mi_UNKNOWN">
I</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mi_UNKNOWN">
T</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span> to the data, with the red shaded area indicating the 95%-confidence interval for model predictions using the estimated standard deviation.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Proportion of cells contributing to cell-to-cell (CC) transmission. (<b>A</b>) Sketch of cells contributing to focus growth for different numbers of neighbors per cell and different expansion phases assuming radial focus growth. Whereas cells in the perimeter of the focus (orange) can contribute to CC-transmission, cells completely enclosed by other infected cells (red) cannot; (<b>B</b>) Proportion of cells contributing to CC-transmission dependent on the focus size assuming that each cell has six neighbors (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
6</div> </div> </div></div></span>). The theoretically-determined relationship (Equation (<a href="#FD9-viruses-10-00200">9</a>), red dashed line) underestimates the true proportion for small focus sizes. The black line indicates the complete adjustment term as described in Equation (<a href="#FD10-viruses-10-00200">10</a>)) with <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
z</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
10</div> </div> </div></div></span>; (<b>C</b>) Proportion of CC contributors with respect to focus size assuming either <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
4</div> </div> </div></div></span> (red), 6 (green) or 8 (blue) neighbors (neigh) for each cell. Parameter <i>z</i> is set to 10.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>Simulated data and corresponding model predictions describing growth of an individual focus of infected cells. (<b>A</b>) Individual focus growth by cell-to-cell (CC) transmission among stationary cells starting with a single infected cell. Black dots indicate the mean and 95%-confidence interval for the total number of infected cells averaged over 100 individual ABM simulations that were sampled every 24 h. Model predictions using standard mass-action kinetics (CC-model, Equation (<a href="#FD1-viruses-10-00200">1</a>) with <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span>, magenta) or our adjusted model (aCC-model, Equation (<a href="#FD11-viruses-10-00200">11</a>) with <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span>, green) to describe cell-to-cell transmission dynamics. Insertion: Profile likelihoods of the estimated cell-to-cell transmission rate, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
c</div> </div> </div></div></span>, using either the CC- (magenta) or aCC- (green) model; (<b>B</b>) An additional model extension considering a time-delay for the time infected cells need to become infectious (aCC-<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
I</div> </div> </div></div></span>, red) additionally improves model predictions; (<b>C</b>) Proportion of CC contributors as predicted by the aCC-model (green line), and the extended aCC-<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
I</div> </div> </div></div></span>-model (red dashed line) in comparison to the actual proportion observed in the ABM simulations (gray lines). The corresponding parameter estimates are given in <a href="#viruses-10-00200-t002">Table 2</a>.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>(<b>A</b>) Sketch of the same number of infected cells distributed on two foci that are either separate or build one merged focus. Cells contributing to cell-to-cell (CC) transmission (orange) and non-contributors (red) are indicated. Boundaries of separate foci are marked in blue; (<b>B</b>) Simulated data and model predictions for a scenario allowing for multiple foci growth. Black dots indicate the mean and 95%-confidence interval for the total number of infected cells averaged over 100 individual ABM simulations that were sampled every 24 h. Colored lines indicate the best fit using either the CC- (magenta) or adjusted aCC-model (green). Corresponding parameter estimates are shown in <a href="#viruses-10-00200-t002">Table 2</a>; (<b>C</b>) Proportion of CC contributors predicted by the adjustment term (green solid line) as defined in Equation (<a href="#FD10-viruses-10-00200">10</a>) in comparison to the proportions inferred from each individual simulation (gray lines). The dashed green line represents the corrected adjustment term multiplied with the proportion of target cells relative to the number of target cells at the beginning of the simulation, i.e., <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
I</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
/</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mn_UNKNOWN">
0</div> </div> </div> </div></div></span>; (<b>D</b>) Approximate increase of infected cells calculated per minute in the ABM (black dots, mean and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
95</div> <div class="mo_UNKNOWN">
%</div> </div> </div></div></span>-confidence intervals based on 100 simulations) and the ODE model (red line) as defined by Equations (<a href="#FD12-viruses-10-00200">12</a>) and (<a href="#FD13-viruses-10-00200">13</a>), respectively.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6</span></a></div>   <p>Disentangling viral transmission modes: (<b>A</b>,<b>B</b>) Simulated data and best model fits for a scenario assuming simultaneous occurrence of both transmission modes given slow diffusion of cell-free virus. Black dots indicate the mean and the 95%-confidence interval of the data obtained from 100 individual simulations runs of the ABM sampled every 24 h. (<b>A</b>) Lines show the best fits of a model allowing for both transmission modes without (CCF-model, magenta, Equation (<a href="#FD1-viruses-10-00200">1</a>)) or with adjusted cell-to-cell transmission dynamics (aCCF-model, green, Equation (<a href="#FD11-viruses-10-00200">11</a>)) fitted to the number of infected cells; (<b>B</b>) The observed (black dots) and predicted (solid lines) proportion of cells infected by cell-free (CF) transmission dependent on the best fit shown in panel (<b>A</b>); (<b>C</b>) Simulations of a scenario only allowing for CF-transmission with slow viral diffusion (gray) and fast viral diffusion (black). Lines indicate the best fit of the standard model (Equation (<a href="#FD1-viruses-10-00200">1</a>) with <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span>) to the data; (<b>D</b>–<b>F</b>) Similar scenario as in (<b>A</b>) with simultaneous occurrence of both transmission modes this time assuming high viral diffusion rates (black, CC/CF trans) and additionally considering a scenario with only CF-transmission (gray, CF trans). Both scenarios are fitted simultaneously. The number of infected cells (<b>D</b>), as well as the dynamics of the extracellular viral load (<b>E</b>) are shown. (<b>F</b>) The observed (black) and predicted proportions of cells infected through CF-transmission in the scenario allowing for both transmission modes. Corresponding parameter estimates for each of the different models are shown in <a href="#viruses-10-00200-t003">Table 3</a>.</p>   </div> <div class="table-wrap_UNKNOWN" id="viruses-10-00200-t001" orientation="portrait" position="float">
viruses-10-00200-t001_Table 1<span class="label" tagx="label" title="label">Table 1</span> <p>Overview of all different models considered that were applied to the ABM-simulated data. As we always neglect cellular turnover in our simulations to focus on the effect of the spatial conditions on the transmission processes, cell death is always neglected within the model equations, meaning <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mi_UNKNOWN">
T</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mi_UNKNOWN">
I</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span>.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="center" valign="middle" style="border-bottom:solid thin;border-top:solid thin" rowspan="1" colspan="1">Name</th> <th align="left" valign="middle" style="border-bottom:solid thin;border-top:solid thin" rowspan="1" colspan="1">Description</th> <th align="center" valign="middle" style="border-bottom:solid thin;border-top:solid thin" rowspan="1" colspan="1">Equation #</th> <th align="center" valign="middle" style="border-bottom:solid thin;border-top:solid thin" rowspan="1" colspan="1">Assumptions</th> </tr> </thead> <tbody> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">CF</td> <td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">cell-free (CF) transmission model</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">(<a href="#FD1-viruses-10-00200">1</a>)</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span></td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">CC</td> <td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">cell-to-cell (CC) transmission model</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">(<a href="#FD1-viruses-10-00200">1</a>)</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span></td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">CCF</td> <td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">CF and CC model</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">(<a href="#FD1-viruses-10-00200">1</a>)</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">aCC</td> <td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">adjusted CC model</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">(<a href="#FD11-viruses-10-00200">11</a>)</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span></td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">aCC-d<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN" /> <div class="mi_UNKNOWN">
I</div> </div> </div></div></span></td> <td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">adjusted CC model with delay on infected cells</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">(<a href="#FD11-viruses-10-00200">11</a>)</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span>; <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
τ</div> <div class="mi_UNKNOWN">
I</div> </div> </div></div></span> included</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">aCCF</td> <td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">CF and adjusted CC model</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">(<a href="#FD11-viruses-10-00200">11</a>)</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="viruses-10-00200-t002" orientation="portrait" position="float">
viruses-10-00200-t002_Table 2<span class="label" tagx="label" title="label">Table 2</span> <p>Estimates of the cell-to-cell (CC) transmission dynamics in the simulated data. The CC-, aCC- and aCC-d<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN" /> <div class="mi_UNKNOWN">
I</div> </div> </div></div></span> model (see <a href="#viruses-10-00200-t001">Table 1</a> for equations) are fitted to the concentration of infected cells obtained for the growth of single and multiple foci. The parameter estimate of the best fit is given with the <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
95</div> <div class="mo_UNKNOWN">
%</div> </div> </div></div></span>-confidence interval shown in brackets. Model performance was assessed using the corrected Akaike information criterion (AICc). The difference in the corrected AIC values, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
AICc</div> </div> </div></div></span>, is calculated relative to the <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
AICc</div> </div></div></span> of the best fitting model (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
AICc</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span>) for single and multiple focus growth, respectively. For the aCC-model, the neighbor distribution was defined by <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
6</div> </div> </div></div></span>. The parameter <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> </div></div></span> scales the number of infected cells within the adjustment term and defines a quantity representing irregular focus growth, with <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> </div></div></span> indicating perfect circular focus growth. For multiple foci, the number of infected cells is divided by <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ϕ</div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
ψ</div> <div class="mi_UNKNOWN">
θ</div> </div> </div></div></span>, where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ψ</div> </div></div></span> defines the number of initially infected cells (see also <a href="#app1-viruses-10-00200">Supplementary Material Text S1</a>). Viral turnover rates, i.e., <i>c</i> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
�?</div> </div></div></span>, are not taken into account when analyzing only CC-transmission dynamics.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="center" valign="middle" style="border-bottom:solid thin;border-top:solid thin" rowspan="1" colspan="1">Model:</th> <th align="center" valign="middle" style="border-bottom:solid thin;border-top:solid thin" rowspan="1" colspan="1">CC</th> <th align="center" valign="middle" style="border-bottom:solid thin;border-top:solid thin" rowspan="1" colspan="1">aCC</th> <th align="center" valign="middle" style="border-bottom:solid thin;border-top:solid thin" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
aCC</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
I</div> </div> </div> </div> </div></div></span></th> </tr> </thead> <tbody> <tr> <td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td colspan="3" align="center" valign="middle" style="border-bottom:solid thin" rowspan="1"><b>Single focus</b></td> </tr> <tr> <td align="left" valign="middle" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
c</div> </div> </div></div></span><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mo_UNKNOWN">
×</div> <div class="msup_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
6</div> </div> </div> <div class="mspace_UNKNOWN" /> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
h</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
cell</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span></td> <td align="center" valign="middle" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1.04</div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
[</div> <div class="mn_UNKNOWN">
1.00</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1.07</div> <div class="mo_UNKNOWN">
]</div> </div> </div></div></span></td> <td align="center" valign="middle" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1.49</div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
[</div> <div class="mn_UNKNOWN">
1.47</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1.50</div> <div class="mo_UNKNOWN">
]</div> </div> </div></div></span></td> <td align="center" valign="middle" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2.04</div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
[</div> <div class="mn_UNKNOWN">
1.82</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
2.34</div> <div class="mo_UNKNOWN">
]</div> </div> </div></div></span></td> </tr> <tr> <td align="left" valign="middle" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> </div></div></span></td> <td align="center" valign="middle" rowspan="1" colspan="1">-</td> <td align="center" valign="middle" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2.7</div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
[</div> <div class="mn_UNKNOWN">
2.58</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
2.82</div> <div class="mo_UNKNOWN">
]</div> </div> </div></div></span></td> <td align="center" valign="middle" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1.62</div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
[</div> <div class="mn_UNKNOWN">
1.31</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1.94</div> <div class="mo_UNKNOWN">
]</div> </div> </div></div></span></td> </tr> <tr> <td align="left" valign="middle" rowspan="1" colspan="1"><i>z</i> (cells)</td> <td align="center" valign="middle" rowspan="1" colspan="1">-</td> <td align="center" valign="middle" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
13.6</div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
[</div> <div class="mn_UNKNOWN">
8.82</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
17.9</div> <div class="mo_UNKNOWN">
]</div> </div> </div></div></span></td> <td align="center" valign="middle" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
20.2</div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
[</div> <div class="mn_UNKNOWN">
12.5</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
28.6</div> <div class="mo_UNKNOWN">
]</div> </div> </div></div></span></td> </tr> <tr> <td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
τ</div> </div></div></span> (hours)</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
5.71</div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
[</div> <div class="mn_UNKNOWN">
3.73</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
7.89</div> <div class="mo_UNKNOWN">
]</div> </div> </div></div></span></td> </tr> <tr> <td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
AICc</div> </div> </div></div></span></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">823.1</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">29.2</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0</td> </tr> <tr> <td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td colspan="3" align="center" valign="middle" style="border-bottom:solid thin" rowspan="1"><b>Multiple foci</b></td> </tr> <tr> <td align="left" valign="middle" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
c</div> </div> </div></div></span><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mo_UNKNOWN">
×</div> <div class="msup_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
6</div> </div> </div> <div class="mspace_UNKNOWN" /> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
h</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
cell</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span></td> <td align="center" valign="middle" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1.13</div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
[</div> <div class="mn_UNKNOWN">
1.07</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1.18</div> <div class="mo_UNKNOWN">
]</div> </div> </div></div></span></td> <td align="center" valign="middle" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1.72</div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
[</div> <div class="mn_UNKNOWN">
1.70</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1.74</div> <div class="mo_UNKNOWN">
]</div> </div> </div></div></span></td> <td align="center" valign="middle" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1.72</div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
[</div> <div class="mn_UNKNOWN">
1.70</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1.79</div> <div class="mo_UNKNOWN">
]</div> </div> </div></div></span></td> </tr> <tr> <td align="left" valign="middle" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> </div></div></span></td> <td align="center" valign="middle" rowspan="1" colspan="1">-</td> <td align="center" valign="middle" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1.78</div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
[</div> <div class="mn_UNKNOWN">
1.70</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1.86</div> <div class="mo_UNKNOWN">
]</div> </div> </div></div></span></td> <td align="center" valign="middle" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1.78</div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
[</div> <div class="mn_UNKNOWN">
1.66</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1.86</div> <div class="mo_UNKNOWN">
]</div> </div> </div></div></span></td> </tr> <tr> <td align="left" valign="middle" rowspan="1" colspan="1"><i>z</i> (cells)</td> <td align="center" valign="middle" rowspan="1" colspan="1">-</td> <td align="center" valign="middle" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2.19</div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
(</div> <div class="mn_UNKNOWN">
0</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
6.61</div> <div class="mo_UNKNOWN">
]</div> </div> </div></div></span></td> <td align="center" valign="middle" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2.09</div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
(</div> <div class="mn_UNKNOWN">
0</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
6.31</div> <div class="mo_UNKNOWN">
]</div> </div> </div></div></span></td> </tr> <tr> <td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
τ</div> </div></div></span> (hours)</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1.65</div> <div class="mo_UNKNOWN">
×</div> <div class="msup_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
5</div> </div> </div> <div class="mspace_UNKNOWN" /> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mn_UNKNOWN">
0</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
0.664</div> <div class="mo_UNKNOWN">
]</div> </div> </div> </div></div></span></td> </tr> <tr> <td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
AICc</div> </div> </div></div></span></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">92.88</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">7.33</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="viruses-10-00200-t003" orientation="portrait" position="float">
viruses-10-00200-t003_Table 3<span class="label" tagx="label" title="label">Table 3</span> <p>Estimated transmission rates for model predictions fitted to simulated data allowing simultaneous transmission dynamics. The transmission rates describing cell-free, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
f</div> </div> </div></div></span>, and cell-to-cell transmission, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
c</div> </div> </div></div></span>, are denoted in <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
×</div> <div class="msup_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
9</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
h</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
cell</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
×</div> <div class="msup_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
8</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
h</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div></div></span>, respectively. For the simultaneous fit of both transmission modes to the number of infected cells without separate quantification of cell-free transmission (<a href="#viruses-10-00200-f006">Figure 6</a>A) no reasonable values for <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
f</div> </div> </div></div></span> can be obtained, i.e., <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
f</div> </div> </div></div></span> is not identifiable (ni). Confidence intervals are calculated from 2 × <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> <div class="mn_UNKNOWN">
3</div> </div> </div></div></span> (CC/CF) and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> <div class="mn_UNKNOWN">
3</div> </div> </div></div></span> (CC/CF + CF) individual fits using random starting conditions and considering all parameter combinations within a range of 3.84 of the best fit likelihood. The difference in the corrected AIC values, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
AICc</div> </div> </div></div></span>, is calculated relative to the <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
AICc</div> </div></div></span> of the best fitting model for each scenario, i.e., <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
AICc</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span>.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="center" valign="middle" style="border-top:solid thin" rowspan="1" colspan="1">Scenario:</th> <th colspan="2" align="center" valign="middle" style="border-top:solid thin" rowspan="1">Both Transmission Modes</th> <th colspan="2" align="center" valign="middle" style="border-top:solid thin" rowspan="1">Both Transmission Modes + Only CF</th> </tr> <tr> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <th colspan="2" align="center" valign="middle" style="border-bottom:solid thin" rowspan="1">(CC/CF)</th> <th colspan="2" align="center" valign="middle" style="border-bottom:solid thin" rowspan="1">(CC/CF + CF)</th> </tr> </thead> <tbody> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Model:</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">CCF</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">aCCF</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">CCF</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">aCCF</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
c</div> </div> </div></div></span></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2.37 (0, 2.41]</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2.74 [2.67, 3.00]</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1.52 [1.50, 1.54]</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1.52 [1.50, 1.63]</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
f</div> </div> </div></div></span></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ni</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ni</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">4.30 [3.65, 5.20]</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">4.30 [3.67, 4.80]</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
AICc</div> </div> </div></div></span></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">60</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">9.5</td> </tr> </tbody> </table> </div> </div>  </body></html>