<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/04/06/2020.03.13.990242.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.13.990242v4" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Accurate Identification of SARS-CoV-2 from Viral Genome Sequences using Deep Learning" />
<meta name="DC.Identifier" content="10.1101/2020.03.13.990242" />
<meta name="DC.Date" content="2020-04-06" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="One of the reasons for the fast spread of SARS-CoV-2 is the lack of accuracy in detection tools in the clinical field. Molecular techniques, such as quantitative real-time RT-PCR and nucleic acid sequencing methods, are widely used to identify pathogens. For this particular virus, however, they have an overall unsatisfying detection rate, due to its relatively recent emergence and still not completely understood features. In addition, SARS-CoV-2 is remarkably similar to other Coronaviruses, and it can present with other respiratory infections, making identification even harder. To tackle this issue, we propose an assisted detection test, combining molecular testing with deep learning. The proposed approach employs a state-of-the-art deep convolutional neural network, able to automatically create features starting from the genome sequence of the virus. Experiments on data from the Novel Coronavirus Resource (2019nCoVR) show that the proposed approach is able to correctly classify SARS-CoV-2, distinguishing it from other coronavirus strains, such as MERS-CoV, HCoV-NL63, HCoV-OC43, HCoV-229E, HCoV-HKU1, and SARS-CoV regardless of missing information and errors in sequencing (noise). From a dataset of 553 complete genome non-repeated sequences that vary from 1,260 to 31,029 bps in length, the proposed approach classifies the different coronaviruses with an average accuracy of 98.75% in a 10-fold cross-validation, identifying SARS-CoV-2 with an AUC of 98%, specificity of 0.9939 and sensitivity of 1.00 in a binary classification. Then, using the same basis, we classify SARS-CoV-2 from 384 complete viral genome sequences with human host, that contain the gene ORF1ab from the NCBI with a 10-fold accuracy of 98.17%, a specificity of 0.9797 and sensitivity of 1.00. Furthermore, an in-depth analysis of the results allow us to identify base pairs sequences that are unique to SARS-CoV-2 and do not appear in other virus strains, that could then be used as a base for designing new primers and examined by experts to extract further insights. These preliminary results seem encouraging enough to identify deep learning as a promising research venue to develop assisted detection tests for SARS-CoV-2. At this end the interaction between viromics and deep learning , will hopefully help to solve global infection problems. In addition, we offer our code and processed data to be used for diagnostic purposes by medical doctors, virologists and scientists involved in solving the SARS-CoV-2 pandemic. As more data become available we will update our system." />
<meta name="DC.Contributor" content="Alejandro Lopez-Rincon" />
<meta name="DC.Contributor" content="Alberto Tonda" />
<meta name="DC.Contributor" content="Lucero Mendoza-Maldonado" />
<meta name="DC.Contributor" content="Eric Claassen" />
<meta name="DC.Contributor" content="Johan Garssen" />
<meta name="DC.Contributor" content="Aletta D. Kraneveld" />
<meta name="article:published_time" content="2020-04-06" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Accurate Identification of SARS-CoV-2 from Viral Genome Sequences using Deep Learning" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;One of the reasons for the fast spread of SARS-CoV-2 is the lack of accuracy in detection tools in the clinical field. Molecular techniques, such as quantitative real-time RT-PCR and nucleic acid sequencing methods, are widely used to identify pathogens. For this particular virus, however, they have an overall unsatisfying detection rate, due to its relatively recent emergence and still not completely understood features. In addition, SARS-CoV-2 is remarkably similar to other Coronaviruses, and it can present with other respiratory infections, making identification even harder. To tackle this issue, we propose an assisted detection test, combining molecular testing with deep learning. The proposed approach employs a state-of-the-art deep convolutional neural network, able to automatically create features starting from the genome sequence of the virus. Experiments on data from the Novel Coronavirus Resource (2019nCoVR) show that the proposed approach is able to correctly classify SARS-CoV-2, distinguishing it from other coronavirus strains, such as MERS-CoV, HCoV-NL63, HCoV-OC43, HCoV-229E, HCoV-HKU1, and SARS-CoV regardless of missing information and errors in sequencing (noise). From a dataset of 553 complete genome non-repeated sequences that vary from 1,260 to 31,029 bps in length, the proposed approach classifies the different coronaviruses with an average accuracy of 98.75% in a 10-fold cross-validation, identifying SARS-CoV-2 with an AUC of 98%, specificity of 0.9939 and sensitivity of 1.00 in a binary classification. Then, using the same basis, we classify SARS-CoV-2 from 384 complete viral genome sequences with human host, that contain the gene &lt;i&gt;ORF1ab&lt;/i&gt; from the NCBI with a 10-fold accuracy of 98.17% , a specificity of 0.9797 and sensitivity of 1.00. These preliminary results seem encouraging enough to identify deep learning as a promising research venue to develop assisted detection tests for SARS-CoV-2. At this end the interaction between viromics and &lt;i&gt;deep learning&lt;/i&gt;, will hopefully help to solve global infection problems. In addition, we offer our code and processed data to be used for diagnostic purposes by medical doctors, virologists and scientists involved in solving the SARS-CoV-2 pandemic. As more data become available we will update our system.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.13.990242v4" />
<meta name="citation_id" content="2020.03.13.990242v4" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.13.990242v4" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.13.990242v4.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.13.990242v4.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.13.990242" />
<meta name="citation_num_pages" content="41" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.13.990242" />
<meta name="citation_author" content="Alejandro Lopez-Rincon" />
<meta name="citation_author_institution" content="Division of Pharmacology, Utrecht Institute for Pharmaceutical Sciences, Faculty of Science, Utrecht University" />
<meta name="citation_author_email" content="alejandro.lopezrn@hotmail.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4491-5889" />
<meta name="citation_author" content="Alberto Tonda" />
<meta name="citation_author_institution" content="INRAe UMR 518 MIA-Paris" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5895-4809" />
<meta name="citation_author" content="Lucero Mendoza-Maldonado" />
<meta name="citation_author_institution" content="Centro Universitario de Ciencias de la Salud, Universidad de Guadalajara" />
<meta name="citation_author" content="Eric Claassen" />
<meta name="citation_author_institution" content="Athena Institute, Vrije Universiteit" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6768-9039" />
<meta name="citation_author" content="Johan Garssen" />
<meta name="citation_author_institution" content="Division of Pharmacology, Utrecht Institute for Pharmaceutical Sciences, Faculty of Science, Utrecht University" />
<meta name="citation_author_institution" content="Department Immunology, Danone Nutricia research" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8678-9182" />
<meta name="citation_author" content="Aletta D. Kraneveld" />
<meta name="citation_author_institution" content="Division of Pharmacology, Utrecht Institute for Pharmaceutical Sciences, Faculty of Science, Utrecht University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9819-383X" />
<meta name="citation_reference" content="P. C. Woo, Y. Huang, S. K. Lau, K.-Y. Yuen, Coronavirus genomics and bioinformatics analysis, viruses 2 (8) (2010) 1804–1820." />
<meta name="citation_reference" content="R. Lu, X. Zhao, J. Li, P. Niu, B. Yang, H. Wu, W. Wang, H. Song, B. Huang, N. Zhu, et al., Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding, The Lancet 395 (10224) (2020) 565–574." />
<meta name="citation_reference" content="V. M. Corman, O. Landt, M. Kaiser, R. Molenkamp, A. Meijer, D. K. Chu, T. Bleicker, S. Brünink, J. Schneider, M. L. Schmidt, et al., Detection of 2019 novel coronavirus (2019-ncov) by real-time rt-pcr, Eurosurveillance 25 (3) (2020)." />
<meta name="citation_reference" content="D. K. Chu, Y. Pan, S. Cheng, K. P. Hui, P. Krishnan, Y. Liu, D. Y. Ng, C. K. Wan, P. Yang, Q. Wang, et al., Molecular diagnosis of a novel coronavirus (2019-ncov) causing an outbreak of pneumonia, Clinical chemistry (2020)." />
<meta name="citation_reference" content="D. A. Marston, L. M. McElhinney, R. J. Ellis, D. L. Horton, E. L. Wise, S. L. Leech, D. David, X. de Lamballerie, A. R. Fooks, Next generation sequencing of viral rna genomes, BMC genomics 14 (1) (2013) 444." />
<meta name="citation_reference" content="Beijing Institute of Genomics, Chinese Academy of Science, China National Center for Bioinformation &amp; National Genomics Data Center, https://bigd.big.ac.cn/ncov/?lang=en, online; accessed 27 January 2020 (2013)." />
<meta name="citation_reference" content="W. H. Organization, WHO report Coronavirus disease 2019 (COVID-19), World Health Organization, Geneva:, 2020., licence: CC BY-NC-SA 3.0 IGO." />
<meta name="citation_reference" content="Y. Wang, H. Kang, X. Liu, Z. Tong, Combination of rt-qpcr testing and clinical features for diagnosis of covid-19 facilitates management of sars-cov-2 outbreak, Journal of Medical Virology (2020)." />
<meta name="citation_reference" content="H. C. Metsky, C. A. Freije, T.-S. F. Kosoko-Thoroddsen, P. C. Sabeti, C. Myhrvold, Crispr-based surveillance for covid-19 using genomically-comprehensive machine learning design, bioRxiv (2020)." />
<meta name="citation_reference" content="M. Wang, Q. Wu, W. Xu, B. Qiao, J. Wang, H. Zheng, S. Jiang, J. Mei, Z. Wu, Y. Deng, et al., Clinical diagnosis of 8274 samples with 2019-novel coronavirus in wuhan, medRxiv (2020)." />
<meta name="citation_reference" content="S. Wang, B. Kang, J. Ma, X. Zeng, M. Xiao, J. Guo, M. Cai, J. Yang, Y. Li, X. Meng, et al., A deep learning algorithm using ct images to screen for corona virus disease (covid-19), medRxiv (2020)." />
<meta name="citation_reference" content="J. Y. Kim, P. G. Choe, Y. Oh, K. J. Oh, J. Kim, S. J. Park, J. H. Park, H. K. Na, M.-d. Oh, The first case of 2019 novel coronavirus pneumonia imported into korea from wuhan, china: implication for infection prevention and control measures, Journal of Korean Medical Science 35 (5) (2020)." />
<meta name="citation_reference" content="W. R. Pearson, [5] rapid and sensitive sequence comparison with fastp and fasta (1990)." />
<meta name="citation_reference" content="S. F. Altschul, W. Gish, W. Miller, E. W. Myers, D. J. Lipman, Basic local alignment search tool, Journal of molecular biology 215 (3) (1990) 403–410." />
<meta name="citation_reference" content="L. Pinello, G. Lo Bosco, G.-C. Yuan, Applications of alignment-free methods in epigenomics, Briefings in Bioinformatics 15 (3) (2014) 419–430." />
<meta name="citation_reference" content="S. Vinga, J. Almeida, Alignment-free sequence comparison—a review, Bioinformatics 19 (4) (2003) 513–523." />
<meta name="citation_reference" content="D. Bzhalava, J. Ekström, F. Lysholm, E. Hultin, H. Faust, B. Persson, M. Lehtinen, E.-M. de Villiers, J. Dillner, Phylogenetically diverse tt virus viremia among pregnant women, Virology 432 (2) (2012) 427–434." />
<meta name="citation_reference" content="N. G. Nguyen, V. A. Tran, D. L. Ngo, D. Phan, F. R. Lumbanraja, M. R. Faisal, B. Abapihi, M. Kubo, K. Satou, et al., Dna sequence classification by convolutional neural network, Journal of Biomedical Science and Engineering 9 (05) (2016) 280." />
<meta name="citation_reference" content="R. Rizzo, A. Fiannaca, M. La Rosa, A. Urso, A deep learning approach to dna sequence classification, in: International Meeting on Computational Intelligence Methods for Bioinformatics and Biostatistics, Springer, 2015, pp. 129–140." />
<meta name="citation_reference" content="A. Tampuu, Z. Bzhalava, J. Dillner, R. Vicente, Viraminer: Deep learning on raw dna sequences for identifying viral genomes in human samples, PloS one 14 (9) (2019)." />
<meta name="citation_reference" content="S. T. Sherry, M.-H. Ward, M. Kholodov, J. Baker, L. Phan, E. M. Smigielski, K. Sirotkin, dbsnp: the ncbi database of genetic variation, Nucleic acids research 29 (1) (2001) 308–311." />
<meta name="citation_reference" content="C. d. S. Ribeiro, M. Y. van Roode, G. B. Haringhuizen, M. P. Koopmans, E. Claassen, L. H. van de Burgwal, How ownership rights over microorganisms affect infectious disease control and innovation: a root-cause analysis of barriers to data sharing as experienced by key stakeholders, PloS one 13 (5) (2018)." />
<meta name="citation_reference" content="J. H. Simon, E. Claassen, C. E. Correa, A. D. Osterhaus, Managing severe acute respiratory syndrome (sars) intellectual property rights: the possible role of patent pooling, Bulletin of the World Health Organization 83 (2005) 707–710." />
<meta name="citation_reference" content="C. d. S. Ribeiro, M. P. Koopmans, G. B. Haringhuizen, Threats to timely sharing of pathogen sequence data, Science 362 (6413) (2018) 404–406." />
<meta name="citation_reference" content="A. Lopez-Rincon, A. Tonda, M. Elati, O. Schwander, B. Piwowarski, P. Gal-linari, Evolutionary optimization of convolutional neural networks for cancer mirna biomarkers classification, Applied Soft Computing 65 (2018) 91–100." />
<meta name="citation_reference" content="A. Vabret, T. Mourez, S. Gouarin, J. Petitjean, F. Freymuth, An outbreak of coronavirus oc43 respiratory infection in normandy, france, Clinical infectious diseases 36 (8) (2003) 985–989." />
<meta name="citation_reference" content="L.-J. Cui, C. Zhang, T. Zhang, R.-J. Lu, Z.-D. Xie, L.-L. Zhang, C.-Y. Liu, W.-M. Zhou, L. Ruan, X.-J. Ma, et al., Human coronaviruses hcov-nl63 and hcov-hku1 in hospitalized children with acute respiratory infections in beijing, china, Advances in virology 2011 (2011)." />
<meta name="citation_reference" content="F. Zeng, C. Chan, M. Chan, J. Chen, K. Chow, C. Hon, K. Hui, J. Li, V. Li, C. Wang, et al., The complete genome sequence of severe acute respiratory syndrome coronavirus strain hku-39849 (hk-39), Experimental Biology and Medicine 228 (7) (2003) 866–873." />
<meta name="citation_reference" content="I. Mizrachi, Genbank: the nucleotide sequence database, The NCBI Handbook [Internet], updated 22 (2007)." />
<meta name="citation_reference" content="M. Abadi, A. Agarwal, P. Barham, E. Brevdo, Z. Chen, C. Citro, G. S. Corrado, A. Davis, J. Dean, M. Devin, S. Ghemawat, I. Goodfellow, A. Harp, G. Irving, M. Isard, Y. Jia, R. Jozefowicz, L. Kaiser, M. Kudlur, J. Lev-enberg, D. Mané, R. Monga, S. Moore, D. Murray, C. Olah, M. Schuster, J. Shlens, B. Steiner, I. Sutskever, K. Talwar, P. Tucker, V. Vanhoucke, V. Vasudevan, F. Viegas, O. Vinyals, P. Warden, M. Wattenberg, M. Wicke, Y. Yu, X. Zheng, TensorFlow: Large-scale machine learning on heterogeneous systems, software available from tensorflow.org (2015). URL https://www.tensorflow.org/" />
<meta name="citation_reference" content="F. Chollet, et al., Keras, https://keras.io (2015)." />
<meta name="citation_reference" content="A.-M. šimundić, Measures of diagnostic accuracy: basic definitions, Ejifcc 19 (4) (2009) 203." />
<meta name="citation_reference" content="J. N. Mandrekar, Receiver operating characteristic curve in diagnostic test assessment, Journal of Thoracic Oncology 5 (9) (2010) 1315–1316." />
<meta name="citation_reference" content="A. Lopez-Rincon, M. Martinez-Archundia, G. U. Martinez-Ruiz, A. Schoenhuth, A. Tonda, Automatic discovery of 100-mirna signature for cancer classification using ensemble feature selection, BMC bioinformatics 20 (1) (2019) 480." />
<meta name="citation_reference" content="Y. Shu, J. McCauley, Gisaid: Global initiative on sharing all influenza data-from vision to reality, Eurosurveillance 22 (13) (2017)." />
<meta name="citation_reference" content="C.-S. Shi, N. R. Nabar, N.-N. Huang, J. H. Kehrl, Sars-coronavirus open reading frame-8b triggers intracellular stress pathways and activates nlrp3 inflammasomes, Cell death discovery 5 (1) (2019) 1–12." />
<meta name="citation_reference" content="N. Kanzawa, K. Nishigaki, T. Hayashi, Y. Ishii, S. Furukawa, A. Niiro, F. Yasui, M. Kohara, K. Morita, K. Matsushima, et al., Augmentation of chemokine production by severe acute respiratory syndrome coronavirus 3a/x1 and 7a/x4 proteins through nf-ĸb activation, FEBS letters 580 (30) (2006) 6807–6812." />
<meta name="citation_reference" content="K. Padhan, C. Tanwar, A. Hussain, P. Y. Hui, M. Y. Lee, C. Y. Cheung, J. S. M. Peiris, S. Jameel, Severe acute respiratory syndrome coronavirus orf3a protein interacts with caveolin, Journal of General Virology 88 (11) (2007) 3067–3077." />
<meta name="citation_reference" content="A. Untergasser, H. Nijveen, X. Rao, T. Bisseling, R. Geurts, J. A. Leunissen, Primer3plus, an enhanced web interface to primer3, Nucleic acids research 35 (suppl_2) (2007) W71–W74." />
<meta name="citation_reference" content="R. Kalendar, D. Lee, A. H. Schulman, et al., Fastpcr software for pcr primer and probe design and repeat search, Genes, Genomes and Genomics 3 (1) (2009) 1–14." />
<meta name="citation_reference" content="T. Phan, Genetic diversity and evolution of sars-cov-2, Infection, Genetics and Evolution 81 (2020) 104260." />
<meta name="citation_reference" content="R. L. Graham, J. S. Sparks, L. D. Eckerle, A. C. Sims, M. R. Denison, Sars coronavirus replicase proteins in pathogenesis, Virus research 133 (1) (2008) 88–100." />
<meta name="citation_reference" content="Y.-Z. Zhang, E. C. Holmes, A genomic perspective on the origin and emergence of sars-cov-2, Cell (2020)." />
<meta name="twitter:title" content="Accurate Identification of SARS-CoV-2 from Viral Genome Sequences using Deep Learning" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="One of the reasons for the fast spread of SARS-CoV-2 is the lack of accuracy in detection tools in the clinical field. Molecular techniques, such as quantitative real-time RT-PCR and nucleic acid sequencing methods, are widely used to identify pathogens. For this particular virus, however, they have an overall unsatisfying detection rate, due to its relatively recent emergence and still not completely understood features. In addition, SARS-CoV-2 is remarkably similar to other Coronaviruses, and it can present with other respiratory infections, making identification even harder. To tackle this issue, we propose an assisted detection test, combining molecular testing with deep learning. The proposed approach employs a state-of-the-art deep convolutional neural network, able to automatically create features starting from the genome sequence of the virus. Experiments on data from the Novel Coronavirus Resource (2019nCoVR) show that the proposed approach is able to correctly classify SARS-CoV-2, distinguishing it from other coronavirus strains, such as MERS-CoV, HCoV-NL63, HCoV-OC43, HCoV-229E, HCoV-HKU1, and SARS-CoV regardless of missing information and errors in sequencing (noise). From a dataset of 553 complete genome non-repeated sequences that vary from 1,260 to 31,029 bps in length, the proposed approach classifies the different coronaviruses with an average accuracy of 98.75% in a 10-fold cross-validation, identifying SARS-CoV-2 with an AUC of 98%, specificity of 0.9939 and sensitivity of 1.00 in a binary classification. Then, using the same basis, we classify SARS-CoV-2 from 384 complete viral genome sequences with human host, that contain the gene ORF1ab from the NCBI with a 10-fold accuracy of 98.17%, a specificity of 0.9797 and sensitivity of 1.00. Furthermore, an in-depth analysis of the results allow us to identify base pairs sequences that are unique to SARS-CoV-2 and do not appear in other virus strains, that could then be used as a base for designing new primers and examined by experts to extract further insights. These preliminary results seem encouraging enough to identify deep learning as a promising research venue to develop assisted detection tests for SARS-CoV-2. At this end the interaction between viromics and deep learning , will hopefully help to solve global infection problems. In addition, we offer our code and processed data to be used for diagnostic purposes by medical doctors, virologists and scientists involved in solving the SARS-CoV-2 pandemic. As more data become available we will update our system." />
<meta name="og-title" property="og:title" content="Accurate Identification of SARS-CoV-2 from Viral Genome Sequences using Deep Learning" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.13.990242v4" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="One of the reasons for the fast spread of SARS-CoV-2 is the lack of accuracy in detection tools in the clinical field. Molecular techniques, such as quantitative real-time RT-PCR and nucleic acid sequencing methods, are widely used to identify pathogens. For this particular virus, however, they have an overall unsatisfying detection rate, due to its relatively recent emergence and still not completely understood features. In addition, SARS-CoV-2 is remarkably similar to other Coronaviruses, and it can present with other respiratory infections, making identification even harder. To tackle this issue, we propose an assisted detection test, combining molecular testing with deep learning. The proposed approach employs a state-of-the-art deep convolutional neural network, able to automatically create features starting from the genome sequence of the virus. Experiments on data from the Novel Coronavirus Resource (2019nCoVR) show that the proposed approach is able to correctly classify SARS-CoV-2, distinguishing it from other coronavirus strains, such as MERS-CoV, HCoV-NL63, HCoV-OC43, HCoV-229E, HCoV-HKU1, and SARS-CoV regardless of missing information and errors in sequencing (noise). From a dataset of 553 complete genome non-repeated sequences that vary from 1,260 to 31,029 bps in length, the proposed approach classifies the different coronaviruses with an average accuracy of 98.75% in a 10-fold cross-validation, identifying SARS-CoV-2 with an AUC of 98%, specificity of 0.9939 and sensitivity of 1.00 in a binary classification. Then, using the same basis, we classify SARS-CoV-2 from 384 complete viral genome sequences with human host, that contain the gene ORF1ab from the NCBI with a 10-fold accuracy of 98.17%, a specificity of 0.9797 and sensitivity of 1.00. Furthermore, an in-depth analysis of the results allow us to identify base pairs sequences that are unique to SARS-CoV-2 and do not appear in other virus strains, that could then be used as a base for designing new primers and examined by experts to extract further insights. These preliminary results seem encouraging enough to identify deep learning as a promising research venue to develop assisted detection tests for SARS-CoV-2. At this end the interaction between viromics and deep learning , will hopefully help to solve global infection problems. In addition, we offer our code and processed data to be used for diagnostic purposes by medical doctors, virologists and scientists involved in solving the SARS-CoV-2 pandemic. As more data become available we will update our system." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-04-06" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.13.990242v4.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.13.990242v4.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.13.990242v4.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.13.990242v4" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1231123" />
    <title>Accurate Identification of SARS-CoV-2 from Viral Genome Sequences using Deep Learning | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"sy8k5Y013SpwKxtQFpVD5rfvJ86E9dg_amBwqBdkvIg","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1231123","apath":"\/biorxiv\/early\/2020\/04\/06\/2020.03.13.990242.atom","pisa":"biorxiv;2020.03.13.990242v4","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.13.990242v4"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.13.990242v4","title":"Accurate Identification of SARS-CoV-2 from Viral Genome Sequences using Deep Learning","identifier":"node\/1231123"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022AEjX3zc5FkWXwVlkE9QWqLfY5Tyluii63tXrky8D\\\\\\\/T\\\\\\\/kiTveY74A82AWn\\\\\\\/F0gzwDKS\\\\\\\/Pe1TiqcpDcPPdRBA9vNfQpz18cP\\\\\\\/X3sNqgbpyX46N7gFy7e5znq1jdruufKCyNZ8A0GzeC5BrNroK3rvblnHOROSR7mRsgNgrcwlJJulNWBUjgzLKrVCnG5yZBYExkQy0s1tp5ae+wD4r4L4UtX7ysuLByF1fjfA3AdO4O5RFUHme8wkI3O09dfKk4mtdJJyv7zHSTCj5mR0csxALz2SBVO5azmVFRCGJIeC65EcC4dQa12wtz7SJj1czEitC3YJyyTT+EHVJDUu9qv51CohIjxmcYxqnbE2GW+ycCugqMZonCvDPQVJtxN2JW7SoL05M21IJ9tL6PUWt0PUVOB6kPA7twYwEdzXNbRrXyuB61seeQLMjmOlru4YNq+Nui98DvYP5QU2K2P+AUxdmZ3XRhQu0RvkDMadhpxZSfamdqQoJn7QUVPzoWFkli0cW7G33M6LZgUfnmafh7KP+A4LYdYcBW2zTFab9SGpgw1A1MjzpAd81Wl4v88sAUHXp+FgziL6HdhfL5EQsXmT1WPMQBLomDxJxYx\\\\\\\/YHbr8IYuzo7O2T\\\\\\\/wXFCzFZgRIvDC4wT\\\\\\\/sISoOyiEzKaMb\\\\\\\/aU+6OFiQMdSTtI+gAH25MHvW73SS7LjXtPZjyTsKLTcADmdbXAXVhWZwpgN16qxN0lgZDU4kWem022C7+ZcATuZvBcNv+FFNEse39z9vxJJ1TKIoGWQOSpvUeVDpMwqwPxG+TjunW7bmRo0uDngYNolhh906FW4YLqcg46F42UzBhOUzHYUxgVg2t4UMioIEMMmFe\\\\\\\/p7c7vdXDQO1hwCQf73OzXUB0f9X50gNYfHgJvpmJ9idfRtt62qzX29BOJxXVOps7dLdzn0gg+H+\\\\\\\/EDwD6OsPpV9EYWbtMzL\\\\\\\/+GCrkRnUpZBLYf2d7QCdPyFsiNjGDFpLLUkqpUJxsxzavtBgVk8vi8syhQsJEOhNmP\\\\\\\/1lwCO\\\\\\\/bnY4uobUlBBDlrquCySi2qaBXjPbw4tFrNIL9bsJ55o+DxVRTLsM8pH7fOoJtKBYkyCLo+8+jsHxIsmRSrlDHfJlfn3OMNEkhvV1Bo4wE1hBlvATX0YhA2nFWT5KWlQkLZr7HKlql8Ic0sV2NAcCv8ElK3HW4KN0f4ON\\\\\\\/zVAZDg5FLOOx3c\\\\\\\/9DSNN7ceH\\\\\\\/ziXpH4RcCqzD8b6BPh5\\\\\\\/WaewaLgIH81OqnRXu8vaVTwbgkry6usFkOvFDvng3dOIFfmibxNZdAmcx12xG6ci6+6fvKZEPBK+dBC9twO5ICXSfB+0W+f20QFDeTRNGEyQ9i\\\u0022,\\\u0022iv\\\u0022:\\\u0022a8rMJ42NKzvfTTiyUsaTcg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022ea6a3bc0689aab8f444bc761dff68cc558e23490813bf4873ed47c6c270ea04a\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.13.990242v4.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1231123 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.13.990242v4.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vFhtJpFDzJM4tgemGmpWY9CgsXR8PrBk651vB1rJ6YE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1231123" id="node1231123" data-pisa="biorxiv;2020.03.13.990242v4" data-pisa-master="biorxiv;2020.03.13.990242" data-apath="/biorxiv/early/2020/04/06/2020.03.13.990242.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Accurate Identification of SARS-CoV-2 from Viral Genome Sequences using Deep Learning</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4491-5889" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Alejandro</span> <span class="nlm-surname">Lopez-Rincon</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5895-4809" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Alberto</span> <span class="nlm-surname">Tonda</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Lucero</span> <span class="nlm-surname">Mendoza-Maldonado</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-6768-9039" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Eric</span> <span class="nlm-surname">Claassen</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-8678-9182" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Johan</span> <span class="nlm-surname">Garssen</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-9819-383X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Aletta D.</span> <span class="nlm-surname">Kraneveld</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.13.990242 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1231123" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Alejandro Lopez-Rincon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Division of Pharmacology, Utrecht Institute for Pharmaceutical Sciences, Faculty of Science, Utrecht University</span>, Universiteitsweg 99, 358f CG Utrecht, <span class='nlm-country'>the Netherlands</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alejandro%2BLopez-Rincon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lopez-Rincon%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlejandro%2BLopez-Rincon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4491-5889" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alejandro Lopez-Rincon</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alejandro.lopezrn@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">alejandro.lopezrn@hotmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alberto Tonda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>INRAe UMR 518 MIA-Paris</span>, c/o 113 rue Nationale, 75103, Paris, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alberto%2BTonda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tonda%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlberto%2BTonda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5895-4809" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alberto Tonda</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Lucero Mendoza-Maldonado </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Centro Universitario de Ciencias de la Salud, Universidad de Guadalajara</span>, Sierra Mojada No. 950, Col. Independencia C.P. ff.3f.8 Guadalajara, Jalisco, <span class='nlm-country'>Mexico</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lucero%2BMendoza-Maldonado%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mendoza-Maldonado%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALucero%2BMendoza-Maldonado%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Eric Claassen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Athena Institute, Vrije Universiteit</span>, De Boelelaan 1085, 1081 HV Amsterdam, <span class='nlm-country'>the Netherlands</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eric%2BClaassen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Claassen%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEric%2BClaassen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6768-9039" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Eric Claassen</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Johan Garssen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Division of Pharmacology, Utrecht Institute for Pharmaceutical Sciences, Faculty of Science, Utrecht University</span>, Universiteitsweg 99, 358f CG Utrecht, <span class='nlm-country'>the Netherlands</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Department Immunology, Danone Nutricia research</span>, Uppsalalaan 12, 358f CT Utrecht, <span class='nlm-country'>the Netherlands</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Johan%2BGarssen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garssen%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohan%2BGarssen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8678-9182" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Johan Garssen</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Aletta D. Kraneveld </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Division of Pharmacology, Utrecht Institute for Pharmaceutical Sciences, Faculty of Science, Utrecht University</span>, Universiteitsweg 99, 358f CG Utrecht, <span class='nlm-country'>the Netherlands</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aletta%2BD.%2BKraneveld%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kraneveld%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAletta%2BD.%2BKraneveld%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9819-383X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aletta D. Kraneveld</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.13.990242v4" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1231123" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1231123/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.13.990242v4.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1231123" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1231123/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.13.990242v4.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1231123" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1231123/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.13.990242v4.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1231123" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1231123/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.13.990242v4.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:1231123" data-trigger="external-links" data-url-enabled="1">Data/Code</a><a href="/panels_ajax_tab/article_tab_data_code/node:1231123/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.13.990242v4.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1231123" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1231123/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">One of the reasons for the fast spread of SARS-CoV-2 is the lack of accuracy in detection tools in the clinical field. Molecular techniques, such as quantitative real-time RT-PCR and nucleic acid sequencing methods, are widely used to identify pathogens. For this particular virus, however, they have an overall unsatisfying detection rate, due to its relatively recent emergence and still not completely understood features. In addition, SARS-CoV-2 is remarkably similar to other Coronaviruses, and it can present with other respiratory infections, making identification even harder. To tackle this issue, we propose an assisted detection test, combining molecular testing with deep learning. The proposed approach employs a state-of-the-art deep convolutional neural network, able to automatically create features starting from the genome sequence of the virus. Experiments on data from the Novel Coronavirus Resource (2019nCoVR) show that the proposed approach is able to correctly classify SARS-CoV-2, distinguishing it from other coronavirus strains, such as MERS-CoV, HCoV-NL63, HCoV-OC43, HCoV-229E, HCoV-HKU1, and SARS-CoV regardless of missing information and errors in sequencing (noise). From a dataset of 553 complete genome non-repeated sequences that vary from 1,260 to 31,029 bps in length, the proposed approach classifies the different coronaviruses with an average accuracy of 98.75% in a 10-fold cross-validation, identifying SARS-CoV-2 with an AUC of 98%, specificity of 0.9939 and sensitivity of 1.00 in a binary classification. Then, using the same basis, we classify SARS-CoV-2 from 384 complete viral genome sequences with human host, that contain the gene <em>ORF1ab</em> from the NCBI with a 10-fold accuracy of 98.17%, a specificity of 0.9797 and sensitivity of 1.00. Furthermore, an in-depth analysis of the results allow us to identify base pairs sequences that are unique to SARS-CoV-2 and do not appear in other virus strains, that could then be used as a base for designing new primers and examined by experts to extract further insights. These preliminary results seem encouraging enough to identify deep learning as a promising research venue to develop assisted detection tests for SARS-CoV-2. At this end the interaction between viromics and <em>deep learning</em>, will hopefully help to solve global infection problems. In addition, we offer our code and processed data to be used for diagnostic purposes by medical doctors, virologists and scientists involved in solving the SARS-CoV-2 pandemic. As more data become available we will update our system.</p></div><div class="section" id="sec-1"><h2 class="">1. Introduction</h2><p id="p-5">The Coronaviridae family presents a positive sense, single-strand RNA genome. This viruses have been identified in avian and mammal hosts, including humans. Coronaviruses have genomes from 26.4 kilo base-pairs (kbps) to 31.7 kbps, with G + C contents varying from 32% to 43%, and human-infecting coronaviruses include SARS-CoV, MERS-CoV, HCoV-OC43, HCoV-229E, HCoV-NL63 and HCoV-HKU1 [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>]. In December 2019, SARS-CoV-2, a novel, human-infecting Coronavirus was identified in Wuhan, China, using Next Generation Sequencing [<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>].</p><p id="p-6">As a typical RNA virus, new mutations appears every replication cycle of Coronavirus, and its average evolutionary rate is roughly 10-4 nucleotide substitutions per site each year [<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>]. In the specific case of SARS-CoV-2, RT-qPCR testing using primers in ORF1ab and N genes have been used to identified the infection in humans. However, this method presents a high false negative rate (FNR), with a detection rate of 30-50% [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>]. This low detection rate can be explained by the variation of viral RNA sequences within virus species, and the viral load in different anatomic sites [<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>]. Population mutation frequency of site 8,872 located in ORF1ab gene and site 28,144 located in ORF8 gene gradually increased from 0 to 29% as the epidemic progressed [<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>].</p><p id="p-7">As of March 6<sup><em>th</em></sup> of 2020, the new SARS-CoV-2 has 98,192 confirmed cases across 88 countries, with 17,481 cases outside of China [<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>]. In addition, SARS-CoV-2 has an estimated mortality rate of 3-4%, and it is spreading faster than SARS-CoV and MERS-CoV [<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>]. SARS-CoV-2 assays can yield false positives if they are not targeted specifically to SARS-CoV-2, as the virus is closely related to other Coronavirus organisms. In addition, SARS-CoV-2 may present with other respiratory infections, which make it even more difficult to identify [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>]. Thus, it is fundamental to improve existing diagnostic tools to contain the spread. For example, diagnostic tools combining computed tomography (CT) scans with deep learning have been proposed, achieving an improved detection accuracy of 82.9% [<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>]. Another solution for identifying SARS-CoV-2 is additional sequencing of the viral complementary DNA (cDNA). We can use sequencing data with cDNA, resulting from the PCR of the original viral RNA; e,g, Real-Time PCR amplicons (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>) to identify the SARS-CoV-2 [<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>].</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="PCR Amplicons sequencing procedure." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-188909711" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;PCR Amplicons sequencing procedure.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F1.medium.gif" width="440" height="213"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F1.medium.gif" width="440" height="213"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <p id="p-8" class="first-child">PCR Amplicons sequencing procedure.</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">Classification using viral sequencing techniques is mainly based on alignment methods such as FASTA [<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>] and BLAST [<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>]. These methods rely on the assumption that DNA sequences share common features, and their order prevails among different sequences [<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>]. However, these methods suffer from the necessity of needing base sequences for the detection [<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>]. Nevertheless, it is necessary to develop innovative improved diagnostic tools that target the genome to improve the identification of pathogenic variants, as sometimes several tests, are needed to have an accurate diagnosis. As an alternative deep learning methods have been suggested for classification of DNA sequences, as these methods do not need pre-selected features to identify or classify DNA sequences. Deep Learning has been efficiently used for classification of DNA sequences, using one-hot label encoding and Convolution Neural Networks (CNN) [<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>], albeit the examples in literature are featuring DNA sequences of length up to 500 bps, only.</p><p id="p-10">In particular, for the case of viruses, Next Generation Sequencing (NGS) genomic samples might not be identified by BLAST, as there are no reference sequences valid for all genomes, as viruses have high mutation frequency [<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>]. Alternative solutions based on deep learning have been proposed to classify viruses, by dividing sequences into pieces of fixed lengths, from 300 bps [<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a>] to 3,000 bps [<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>]. However, this approach has the negative effect of potentially ignoring part of the information contained in the input sequence, that is disregarded if it cannot completely fill a piece of fixed size.</p><p id="p-11">Given the impact of the world-wide outbreak, international efforts have been made to simplify the access to viral genomic data and metadata through international repositories, such as; the 2019 Novel Coronavirus Resource (2019nCoVR) repository [<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>] and the National Center for Biotechnology Information (NCBI) [<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>], expecting that the easiness to acquire information would make it possible to develop medical countermeasures to control the disease worldwide, as it happened in similar cases earlier [<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>]. Thus, taking advantage of the available information of international resources without any political and/or economic borders, we propose an innovative system based on viral gene sequencing.</p><p id="p-12">Differently from previous works in literature, that use of deep learning with fixed length features and one-hot label encoding, in this work we propose the use of a different encoding to input the full sequence as a whole. In addition, we use as base input 31,029 as an input vector, which is the maximum length of available DNA sequences for Coronavirus. Finally, we propose a novel architecture for the deep network, inspired by successful applications in cancer detection starting from miRNA [<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>].</p></div><div class="section" id="sec-2"><h2 class="">2. Methods</h2><div id="sec-3" class="subsection"><h3>2.1. Data</h3><div id="sec-4" class="subsection"><h4>2.1.1. Classification of Coronaviruses</h4><p id="p-13">SARS-CoV-2 identification can give wrong results, as the virus is difficult to distinguish from other Coronaviruses, due to their genetic similarity. In addition, people with SARS-CoV-2 may present other infections besides the virus [<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>, <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>]. Therefore, it is important to be able to properly classify SARS-CoV-2 from other Coronaviruses.</p><p id="p-14">From the repository 2019 Novel Coronavirus Resource (2019nCoVR) [<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">6</a>], we downloaded all the available sequences with the query <em>Nucleotide Complete-ness=“complete” AND host=“homo sapiens”</em>, for a total of 588 samples. Next, we removed all repeated sequences, resulting in 553 unique sequences of variable length (1,260-31,029 bps). The data was organized and labeled as summarized by <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. We grouped HCoV-229E and HCoV-OC43 in the same class, as they are mostly known as Coronaviruses responsible for the common cold [<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>]; the two available samples of HCoV-4408 were also added to the same class, as it is a Betacoronavirus 1, as HCoV-OC43. In a similar fashion, we grouped HCoV-NL63 and HCoV-HKU1, as they are both associated with acute respiratory infections (ARI) [<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>]. Finally, we grouped SARS-CoV/SARS-CoV-P2/SARS-CoV HKU-39849 [<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>]/SARS-CoV GDH-BJH01 organisms together, as they are all strains of SARS.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1238326/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1238326/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1238326" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1:</span> <p id="p-15" class="first-child">Organism, assigned label, and number of samples in the unique sequences obtained from the repository [<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">6</a>]. We use the NCBI organism naming convention [<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>].</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">To encode the cDNA data into an input tensor for the CNN, we assigned numeric values to the different bases; C=0.25, T=0.50, G=0.75, A=1.0 (see <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>). All missing entries were assigned the value 0.0. This procedure is different from previous methods, that relied upon one-hot encoding [<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>, <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">20</a>], and has the advantages of making the input more human-readable and do not multiply the amount of memory required to store the information. We divide the available samples in two parts, 90% for training and validation (80% training, 10% validation), and 10% for testing, in a 10-fold cross-validation scheme. k-fold cross-validation is a procedure by which available data is divided into <em>k</em> parts, called <em>folds</em>. At each iteration <em>i</em>, the i-th fold is used as a test set, while all the other folds are used as training. At the end of the k-th iteration, the average performance of the model in test over all folds provides a good estimate of the generality of the results. In this particular case, we use stratified folds, that preserve the same proportion of classes in every fold. The procedure is summarized by <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Coding for the input sequences." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-188909711" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Coding for the input sequences.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F2.medium.gif" width="440" height="85"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F2.medium.gif" width="440" height="85"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <p id="p-17" class="first-child">Coding for the input sequences.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Scheme of a k-fold cross-validation. Available data is divided into k parts. At each iteration i, the i-th fold is used for testing, while all the others are used as a training set." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-188909711" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Scheme of a &lt;em&gt;k&lt;/em&gt;-fold cross-validation. Available data is divided into &lt;em&gt;k&lt;/em&gt; parts. At each iteration &lt;em&gt;i&lt;/em&gt;, the i-th fold is used for testing, while all the others are used as a training set.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F3.medium.gif" width="430" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F3.medium.gif" width="430" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <p id="p-18" class="first-child">Scheme of a <em>k</em>-fold cross-validation. Available data is divided into <em>k</em> parts. At each iteration <em>i</em>, the i-th fold is used for testing, while all the others are used as a training set.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h4>2.1.2. Separating SARS-CoV-2 from other viruses containing gene ORF1ab</h4><p id="p-19">Two thirds of the Coronaviruses’ genome contain the ORF1ab gene [<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>]. Therefore, it is important that we are able to differentiate SARS-CoV-2 from similar viruses, like Astroviruses. From the NCBI repository [<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a>], we downloaded the genome sequences corresponding to the following search: <em>gene=“ORF1ab” AND host=“homo sapiens” AND “complete genome”</em>. This resulted in 402 sequences, distributed as described in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. For this data, we assigned SARS-CoV-2 label 0, and grouped the rest of the organisms together in label 1. Next, we removed all the repeated sequences, obtaining a total of 384 unique sequences, with 45 samples belonging to SARS-CoV-2. The genomic data was translated to digits using the encoding previously described in <a id="xref-sec-4-1" class="xref-sec" href="#sec-4">Subsection 2.1.1</a>.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1238341/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1238341/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1238341" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2:</span> <p id="p-20" class="first-child">Organism, assigned label, and number of samples in the unique sequences obtained from the repository NCBI [<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">30</a>].</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-6" class="subsection"><h3>2.2. Convolutional Neural Network</h3><p id="p-21">The deep learning model used for the experiments is a CNN with a convolutional layer with max pooling, a fully connected layer, and a final soft-max layer, as described in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4</a>. The input is a vector of 31,029 elements, which is the maximum size of the genome sequences in the dataset. The convolutional layer is characterized by three hyperparameters, as shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5</a>: <em>w</em> =12, <em>wd</em> = 21, <em>h</em> = 148. The fully connected layer has 196 ReLu units and it is set with a dropout probability of <em>pd</em> = 0.5 during training, to improve generality; moreover, a <em>l</em>2 regularization is applied to the cross-categorical entropy loss function, considering all weights in the convolutional layer, with <em>β</em> = 10<sup>-3</sup>. The optimizer used for the weights is Adaptive Moment Estimation (Adam) [<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>], with learning rate <em>lr</em> = 10<sup>-5</sup>, run for 500 epochs. The hyper-parameters used in the experiments were selected after a set of preliminary trials. All the necessary code was developed in Python 3, using the <kbd>tensorflow</kbd> [<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>] and <kbd>keras</kbd> [<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>] libraries for deep learning, and has been made available on an open GitHub repository<sup><a id="xref-fn-3-1" class="xref-fn" href="#fn-3">1</a></sup>.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Architecture of the deep convolutional neural network used in the experiments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-188909711" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Architecture of the deep convolutional neural network used in the experiments.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F4.medium.gif" width="440" height="156"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F4.medium.gif" width="440" height="156"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4:</span> <p id="p-22" class="first-child">Architecture of the deep convolutional neural network used in the experiments.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Structure of a convolutional layer in the network. For each of the w = 12 filters, the convolution window of size wd = 21 is slid over the data, one step at a time. The results of the convolution are then passed through a max pooling layer of size h = 148, that helps making the representation approximately invariant to small translations of the input." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-188909711" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Structure of a convolutional layer in the network. For each of the &lt;em&gt;w&lt;/em&gt; = 12 filters, the convolution window of size &lt;em&gt;wd&lt;/em&gt; = 21 is slid over the data, one step at a time. The results of the convolution are then passed through a max pooling layer of size &lt;em&gt;h&lt;/em&gt; = 148, that helps making the representation approximately invariant to small translations of the input.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F5.medium.gif" width="440" height="338"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F5.medium.gif" width="440" height="338"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5:</span> <p id="p-23" class="first-child">Structure of a convolutional layer in the network. For each of the <em>w</em> = 12 filters, the convolution window of size <em>wd</em> = 21 is slid over the data, one step at a time. The results of the convolution are then passed through a max pooling layer of size <em>h</em> = 148, that helps making the representation approximately invariant to small translations of the input.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-7"><h2 class="">3. Results</h2><div id="sec-8" class="subsection"><h3>3.1. Classification of SARS-CoV-2 among Coronaviruses</h3><p id="p-24">In the first test, we separated the SARS-CoV-2 from other sequences available at the repository 2019 Novel Coronavirus Resource (2019nCoVR) [<a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">6</a>]. We obtained a 10-fold mean test accuracy of <em>μ</em> = 0.9875 with <em>σ</em> = 0.0160. The resulting confusion matrix (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig. 6</a>) shows that only 3 out of the 66 SARS-CoV-2 sequences were mistakenly assigned to another class. The binarized curve of the test (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Fig. 7</a>) has an area under the curve (AUC) of 0.98, with a specificity of 0.9939 and sensitivity of 1.00. This is considered an outstanding performance, according to the guidelines provided by [<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>].</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Confusion matrix resulting from the test of a 10-fold cross-validation, comprising 553 samples belonging to 5 different classes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-188909711" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Confusion matrix resulting from the test of a 10-fold cross-validation, comprising 553 samples belonging to 5 different classes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F6.medium.gif" width="414" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F6.medium.gif" width="414" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6:</span> <p id="p-25" class="first-child">Confusion matrix resulting from the test of a 10-fold cross-validation, comprising 553 samples belonging to 5 different classes.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Binarized ROC curve of the 553 sequences, where we consider samples belonging to SARS-CoV-2 as class 0, and all the rest as class 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-188909711" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Binarized ROC curve of the 553 sequences, where we consider samples belonging to SARS-CoV-2 as class 0, and all the rest as class 1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F7.medium.gif" width="440" height="344"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F7.medium.gif" width="440" height="344"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7:</span> <p id="p-26" class="first-child">Binarized ROC curve of the 553 sequences, where we consider samples belonging to SARS-CoV-2 as class 0, and all the rest as class 1.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">As viruses are characterized by high mutation frequencies, to assess the robustness of our approach, we performed further experiments where we added noise to the dataset, simulating possible future mutations. 5% noise was added by randomly selecting 1,551 positions from each sequence, from the 31,029 available, and modifying each selected base to another, or to a missing value, randomly. A new 10-fold cross-validation classification run on the noisy dataset yields an average accuracy <em>μ</em> = 0.9674 with a <em>σ</em> = 0.0158. <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figs. 8</a> and <a id="xref-fig-9-1" class="xref-fig" href="#F9">9</a> show the resulting confusion matrix and ROC curve, respectively. This gives a AUC of 0.97, with a specificity of 0.9939 and sensitivity of 0.90.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Confusion matrix resulting from the test of a 10-fold cross-validation, comprising 553 samples belonging to 5 different classes, with a 5% noise in the dataset." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-188909711" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Confusion matrix resulting from the test of a 10-fold cross-validation, comprising 553 samples belonging to 5 different classes, with a 5% noise in the dataset.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F8.medium.gif" width="413" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F8.medium.gif" width="413" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8:</span> <p id="p-28" class="first-child">Confusion matrix resulting from the test of a 10-fold cross-validation, comprising 553 samples belonging to 5 different classes, with a 5% noise in the dataset.</p><div class="sb-div caption-clear"></div></div></div><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Binarized ROC curve of the 553 sequences, where we consider samples belonging to SARS-CoV-2 as class 0, and all the rest as class 1, with 5% added noise." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-188909711" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Binarized ROC curve of the 553 sequences, where we consider samples belonging to SARS-CoV-2 as class 0, and all the rest as class 1, with 5% added noise.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F9.medium.gif" width="440" height="344"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F9.medium.gif" width="440" height="344"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9:</span> <p id="p-29" class="first-child">Binarized ROC curve of the 553 sequences, where we consider samples belonging to SARS-CoV-2 as class 0, and all the rest as class 1, with 5% added noise.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>3.2. Separating SARS-CoV-2 from other viruses containing gene ORF1ab</h3><p id="p-30">In a next batch of experiments, we aim to distinguish SARS-CoV-2 from other genome sequences from NCBI [<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">30</a>], with the following search parameters: <em>gene=“ORF1ab” AND host=“homo sapiens” AND “complete genome”</em>. We get a 10-fold average accuracy of μ = 0.9817 with a σ = 0.0167. The resulting confusion matrix (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig. 6</a>) shows that 7 out of the 45 SARS-CoV-2 sequences, were classified in another class. The ROC curve of the test (<a id="xref-fig-11-1" class="xref-fig" href="#F11">Fig. 11</a>) has an area under the curve (AUC) of 0.92, with a specificity of 0.9797 and sensitivity of 1.00.</p><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Confusion Matrix of the proposed approach on the 384 NCBI sequences, binarizing the problem with only two classes. Label 0 corresponds to SARS-CoV-2, label 1 to all the other virus strains." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-188909711" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Confusion Matrix of the proposed approach on the 384 NCBI sequences, binarizing the problem with only two classes. Label 0 corresponds to SARS-CoV-2, label 1 to all the other virus strains.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 10:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F10.medium.gif" width="413" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 10:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F10.medium.gif" width="413" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 10:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 10:</span> <p id="p-31" class="first-child">Confusion Matrix of the proposed approach on the 384 NCBI sequences, binarizing the problem with only two classes. Label 0 corresponds to SARS-CoV-2, label 1 to all the other virus strains.</p><div class="sb-div caption-clear"></div></div></div><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="ROC curve of the proposed approach classifying the 384 NCBI sequences, where we consider SARS-CoV-2, as class 0 and the rest as 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-188909711" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;ROC curve of the proposed approach classifying the 384 NCBI sequences, where we consider SARS-CoV-2, as class 0 and the rest as 1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 11:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F11.medium.gif" width="440" height="346"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 11:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F11.medium.gif" width="440" height="346"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 11:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 11:</span> <p id="p-32" class="first-child">ROC curve of the proposed approach classifying the 384 NCBI sequences, where we consider SARS-CoV-2, as class 0 and the rest as 1.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-10"><h2 class="">4. Feature Detection</h2><p id="p-33">The convolutional layers of CNNs <em>de-facto</em> learn new features to characterize the problem, directly from the data. In this specific case, the new features are specific sequences of base pairs that can more easily separate different virus strains (<a id="xref-fig-12-1" class="xref-fig" href="#F12">Fig. 12</a>). By analyzing the result of each filter in a convolutional layer, and how its output interacts with the corresponding max pooling layer, it is possible to detect human-readable sequences of base pairs that might provide domain experts with important information. It is important to notice that these sequences are not bound to specific locations of the genome; thanks to its structure, the CNN is able to detect them and recognize their importance even if their position is displaced in different samples.</p><div id="F12" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overall procedure to find the specific SARS-CoV-2 sequences." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-188909711" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Overall procedure to find the specific SARS-CoV-2 sequences.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 12:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F12.medium.gif" width="440" height="240"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 12:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F12.medium.gif" width="440" height="240"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 12:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 12:</span> <p id="p-34" class="first-child">Overall procedure to find the specific SARS-CoV-2 sequences.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">For this purpose, we use the trained CNN described in <a id="xref-sec-6-1" class="xref-sec" href="#sec-6">Subsection 2.2</a>, that obtained an accuracy of 98.75% in a 10-fold cross-validation. In a first step, we plot the inputs and outputs of the convolutional layer, to visually inspect for patterns. As an example, in <a id="xref-fig-13-1" class="xref-fig" href="#F13">Fig. 13</a> we report the visualization of the first 2,500 bps of each of the 553 samples considered in the first experiment. Each filter slides a 21-bps window over the input, and for each step produces a single value. The output of a filter is thus a sequence of values in (0,1): <a id="xref-fig-14-1" class="xref-fig" href="#F14">Fig. 14</a>,<a id="xref-fig-15-1" class="xref-fig" href="#F15">15</a> shows the outputs of four out of the twelve filters in the CNN, for the first 2,500 bps of all 553 samples, mapped to a monochromatic image where the closest a value is to 1, the whiter the corresponding pixel is.</p><div id="F13" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F13.large.jpg?width=800&amp;height=600&amp;carousel=1" title="cDNA visualization for the first 2,500 bps from the input dataset, for each of the 553 samples. Each sample is represented by a horizontal line of pixels. Colored pixels represent bases: G=green, C=blue, A=red, T=orange, missing=black." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-188909711" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;cDNA visualization for the first 2,500 bps from the input dataset, for each of the 553 samples. Each sample is represented by a horizontal line of pixels. Colored pixels represent bases: G=green, C=blue, A=red, T=orange, missing=black.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 13:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F13.medium.gif" width="438" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 13:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F13.medium.gif" width="438" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 13:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 13:</span> <p id="p-36" class="first-child">cDNA visualization for the first 2,500 bps from the input dataset, for each of the 553 samples. Each sample is represented by a horizontal line of pixels. Colored pixels represent bases: G=green, C=blue, A=red, T=orange, missing=black.</p><div class="sb-div caption-clear"></div></div></div><div id="F14" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F14.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The output of convolutional filter 0, for the input given in Fig. 13. The output of the filters is a series of continuous values in (0, 1), here represented in grayscale, with higher values closer to white." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-188909711" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The output of convolutional filter 0, for the input given in Fig. 13. The output of the filters is a series of continuous values in (0, 1), here represented in grayscale, with higher values closer to white.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 14:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F14.medium.gif" width="439" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 14:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F14.medium.gif" width="439" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 14:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 14:</span> <p id="p-37" class="first-child">The output of convolutional filter 0, for the input given in <a id="xref-fig-13-2" class="xref-fig" href="#F13">Fig. 13</a>. The output of the filters is a series of continuous values in (0, 1), here represented in grayscale, with higher values closer to white.</p><div class="sb-div caption-clear"></div></div></div><div id="F15" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F15.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The output of convolutional filter 1, for the input given in Fig. 13. The output of the filters is a series of continuous values in (0, 1), here represented in grayscale, with higher values closer to white" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-188909711" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The output of convolutional filter 1, for the input given in Fig. 13. The output of the filters is a series of continuous values in (0, 1), here represented in grayscale, with higher values closer to white&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 15:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F15.medium.gif" width="439" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 15:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F15.medium.gif" width="439" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F15.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 15:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F15.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 15:</span> <p id="p-38" class="first-child">The output of convolutional filter 1, for the input given in <a id="xref-fig-13-3" class="xref-fig" href="#F13">Fig. 13</a>. The output of the filters is a series of continuous values in (0, 1), here represented in grayscale, with higher values closer to white</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">The output of the max pooling layer for each filter is then further inspected for patterns. An example of the output of the max pooling layer for the first two filters is displayed in <a id="xref-fig-16-1" class="xref-fig" href="#F16">Fig. 16</a>: it is noticeable how the different classes can be already visually distinguished. At this step, we identify filter 1 as the most promising, as it seems to focus on the a few relevant points in the genome, and it is thus most likely able to identify meaningful sequences.</p><div id="F16" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F16.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Visualization of the output of the max pooling for the first two filters of the CNN, with the data from the convolutional filters (Fig. 14,15) in input. Different patterns for samples from different classes are recognizable from a simple visual inspection." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-188909711" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Visualization of the output of the max pooling for the first two filters of the CNN, with the data from the convolutional filters (Fig. 14,15) in input. Different patterns for samples from different classes are recognizable from a simple visual inspection.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 16:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F16.medium.gif" width="343" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 16:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F16.medium.gif" width="343" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F16.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 16:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F16.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 16:</span> <p id="p-40" class="first-child">Visualization of the output of the max pooling for the first two filters of the CNN, with the data from the convolutional filters (<a id="xref-fig-14-2" class="xref-fig" href="#F14">Fig. 14</a>,<a id="xref-fig-15-2" class="xref-fig" href="#F15">15</a>) in input. Different patterns for samples from different classes are recognizable from a simple visual inspection.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">Given this data, it is now possible to identify the 21-bps sequences (created by the first convolutional filter) that obtained the highest output values in the max pooling layer of filter 1, in a section of 148 positions. This process results in 210 (31,029 divided by 148) <em>max pooling features</em>, each one identifying the 21-bps sequence that obtained the highest value from the convolutional filter, in a specific 148-position interval of the original genome: the first max pooling feature will cover positions 1-148, the second will cover position 149-296, and so on. We graph the whole set of max pooling features for the complete data, <a id="xref-fig-17-1" class="xref-fig" href="#F17">Fig. 17</a>.</p><div id="F17" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F17.large.jpg?width=800&amp;height=600&amp;carousel=1" title="cDNA visualization for the selected 210 21-bps-long sequences selected from the input dataset. Each sample is represented by a horizontal line of pixels. Colored pixels represent bases: G=green, C=blue, A=red, T=orange, missing=black. We divide the whole information, for visualization purposes; from visual inspection we can see the similarity of the patterns between the classes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-188909711" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;cDNA visualization for the selected 210 21-bps-long sequences selected from the input dataset. Each sample is represented by a horizontal line of pixels. Colored pixels represent bases: G=green, C=blue, A=red, T=orange, missing=black. We divide the whole information, for visualization purposes; from visual inspection we can see the similarity of the patterns between the classes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 17:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F17.medium.gif" width="440" height="273"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 17:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F17.medium.gif" width="440" height="273"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F17.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 17:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F17.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 17:</span> <p id="p-42" class="first-child">cDNA visualization for the selected 210 21-bps-long sequences selected from the input dataset. Each sample is represented by a horizontal line of pixels. Colored pixels represent bases: G=green, C=blue, A=red, T=orange, missing=black. We divide the whole information, for visualization purposes; from visual inspection we can see the similarity of the patterns between the classes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">Analyzing the different sequence values appearing in the max pooling feature space, a total of 3,827 unique 21-bps cDNA sequences, that can potentially be very informative for identifying different virus strains. For example, sequence “AGGTAACAAACCAACCAACTT” is only found inside the class of SARS-CoV-2, in 59 out of 66 available samples. Sequence “CACGAGTAACTCGTC-TATCTT” is present only in SARS-CoV-2, in 63 out of the 66 samples.</p><p id="p-44">The combination of the convolutional and max pooling layer allows the CNN to identify sequences even if they are slightly displaced in the genome (by up to 148 positions). As some samples might present sequences that are displaced even more, in the next experiments we decided to just consider the relative frequency of the 21-pbs sequences identified at the previous step, creating a <em>sequence feature space</em>, to verify whether the appearance of specific sequences could be enough to differentiate between virus strains.</p><div id="sec-11" class="subsection"><h3>4.1. Example 1</h3><p id="p-45">We downloaded the dataset from the NGDC repository [<a id="xref-ref-6-6" class="xref-bibr" href="#ref-6">6</a>] on March 15<sup>15</sup> 2020. We removed repeated sequences and applied the whole procedure to translate the data into the sequence feature space. This leave us with a frequency table of 3,827 features with 583 samples (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Next, we ran a state-of-the-art feature selection algorithm [<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>], to reduce the sequences needed to identify different virus strain to the bare minimum. Remarkably, we are then able to classify exactly all samples using only 53 of the original 3,827 sequences, obtaining a 100% accuracy in a 10-fold cross-validation with a simpler and more traditional classifier, such as Logistic Regression.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1238347/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1238347/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1238347" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3:</span> <p id="p-46" class="first-child">Organism, assigned label, and number of samples in the unique sequences obtained from the repository [<a id="xref-ref-6-7" class="xref-bibr" href="#ref-6">6</a>]. We use the NCBI organism naming convention [<a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">30</a>].</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>4.2. Example 2</h3><p id="p-47">We downloaded data from NCBI [<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a>] on March 15<sup>th</sup> 2020, with the following query=<em>gene=“ORF1ab” AND host=“homo sapiens” AND ‘‘complete genome”</em>. The query resulted in 407 non-repeated sequences (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>), with 68 sequences belonging to SARS-CoV-2. Then, we applied the whole procedure to translate the data into the sequence feature space, and we run the same state-of-the-art feature selection algorithm [<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">36</a>]. This give us a list of 10 different sequences: just checking for their presence is enough to differentiate between SARS-CoV-2 and other viruses in the dataset with a 100% accuracy. Each of the sequences only appears in SARS-CoV-2.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1238333/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1238333/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1238333" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4:</span> <p id="p-48" class="first-child">Organism, assigned label, and number of samples in the unique sequences obtained from the repository NCBI [<a id="xref-ref-30-6" class="xref-bibr" href="#ref-30">30</a>].</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>4.3. Example 3</h3><p id="p-49">We downloaded data from NCBI [<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">22</a>] on March 17<sup>th</sup> 2020, with the following query= “virus <em>AND host=“homo sapiens” AND ‘complete genome”</em>, restricting the size from 1,000 to 35,000. This gives us a total of 20,603 results, where only 32 samples are SARS-CoV-2 samples and 20,571 are from other taxa, including; Hepatitis B, Dengue, Human immunodeficiency, Human orthopneumovirus, Enterovirus A, Hepacivirus C, Chikungunya virus, Zaire ebolavirus, Human respirovirus 3, Orthohepevirus A, Norovirus GII, Hepatitis delta virus, Mumps rubulavirus, Enterovirus D, Zika virus, Measles morbillivirus, Enterovirus C, Human T-cell leukemia virus type I, Yellow fever virus, Adeno-associated virus, rhinovirus (A, B and C), for more than 900 viruses. Then, we we applied the whole procedure to translate the data into the sequence feature space and run the feature reduction algorithm [<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">36</a>]. This results in 2 sequences of 21 bps: just by checking for their presence, we are able to separate SARS-CoV-2 from the rest of the samples with a 100% accuracy. The sequences are: <strong>AATAGAA-GAATTATTCTATTC</strong> and <strong>CGATAACAACTTCTGTGGCCC</strong>.</p></div><div id="sec-14" class="subsection"><h3>4.4. Example 4</h3><p id="p-50">From the GISAID repository [<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>], we downloaded the last 323 sequences available for SARS-CoV-2, from different countries. Then, we calculate the frequency table of the 21-bps sequences from examples 2 and 3, to see which sequences remain and could be use for detection. The results are in <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a> in percentage of appearance in the GISAID sequences.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1238345/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1238345/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1238345" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5:</span> <p id="p-51" class="first-child">Sequences that only exist in SARS-CoV-2, that help differentiate between the virus and other taxa as displayed in <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>.</p><div class="sb-div caption-clear"></div></div></div><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1238346/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1238346/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1238346" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6:</span> <p id="p-52" class="first-child">Frequency table in percentages for the sequences in examples 2 and 3 in the 323 sequences from GISAID [<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">37</a>].</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>4.5. Biological features and molecular techniques</h3><p id="p-53">After deep learning analysis, we identify that the sequence TAGCACTCTC-CAAGGGTGTTC is exclusive for SARS-CoV-2 and shows a frequency of 100% in viral genomes available from different countries in the GISAID [<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">37</a>] and NCBI [<a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">22</a>]. Using NC045512.2 as reference SARS-CoV-2 sequence, we identify this unique sequence is located from 25604 to 25624 nucleotides in ORF3a gene. In SARS-CoV, this gene encodes a protein of 274 aa, that is related with necrotic cell death [<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>], chemokine production, inflammatory response [<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>] and may play an important role in virus life cycle [<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>]. With this information, we design a specific primer set for detection of SARS-CoV-2 using Primer3plus [<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>]. We use TAGCACTCTCCAAGGGTGTTC as forward primer and GCAAAGC-CAAAGCCTCATTA as reverse primer. Then, we run an <em>In silico PCR</em> test using FastPCR 6.7 [<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>] with default parameters, this yields the results from <a id="xref-fig-18-1" class="xref-fig" href="#F18">Fig. 18</a>.</p><div id="F18" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F18.large.jpg?width=800&amp;height=600&amp;carousel=1" title="In silico PCR Test results using TAGCACTCTCCAAGGGTGTTC and GCAAAGCCAAAGCCTCATTA sequences as primers in NC045512.2 as reference SARS-CoV-2 sequence using FastPCR 6.7 program [42]." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-188909711" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;In silico PCR Test results using TAGCACTCTCCAAGGGTGTTC and GCAAAGCCAAAGCCTCATTA sequences as primers in NC045512.2 as reference SARS-CoV-2 sequence using FastPCR 6.7 program [42].&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 18:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F18.medium.gif" width="440" height="338"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 18:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F18.medium.gif" width="440" height="338"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F18.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 18:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F18.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 18:</span> <p id="p-54" class="first-child">In silico PCR Test results using TAGCACTCTCCAAGGGTGTTC and GCAAAGCCAAAGCCTCATTA sequences as primers in NC045512.2 as reference SARS-CoV-2 sequence using FastPCR 6.7 program [<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">42</a>].</p><div class="sb-div caption-clear"></div></div></div><p id="p-55">These primers (Forward 5’ TAGCACTCTCCAAGGGTGTTC 3’ and Reverse 5’ GCAAAGCCAAAGCCTCATTA 3’) could identify and differentiate SARS-CoV-2 from other coronavirus species through the PCR method. Furthermore, we propose to create a multiplex PCR using the 21 nt. unique sequences for SARS-CoV-2 identified through deep learning to develop accurate molecular diagnostic techniques. However, it is necessary to test it in laboratory and carry out the validation with patients samples.</p><div id="sec-16" class="subsection"><h4>Severity Identification</h4><div id="sec-17" class="subsection"><h5>Experiment 5: Severity Detection</h5><p id="p-56">The data collected from the GISAID repository [<a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">37</a>], also reports metadata, including the status of patients. While most of the metadata is missing, we selected 169 patients for which the status is reported; in the dataset, 52 are annotated as <em>asymptomatic</em> and 117 as <em>hospitalized</em>. We reached to the submitters of some of the sequences, and they reported that <em>hospitalized</em> meant that the patients presented evident symptoms of SARS-CoV-2, and could thus be considered <em>symptomatic</em>. We then applied the previously described methodology to discover specific sequences to separate asymptomatic from symptomatic (hospitalized) patients, and reduced the number of 21-bps sequences to the necessary minimum, using a feature reduction algorithm [<a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">36</a>]. The algorithm ultimately returns an optimal set of 32 sequences, of 21 bps each: Simply checking for their presence inside a patient sample makes it possible to separate asymptomatic from symptomatic patients with 94% accuracy.</p><p id="p-57">For each of the discovered sequences we then calculate the frequency of appearance in both asymptomatic and symptomatic patients. The results are reported in <a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a>. It is important to notice that the sequences discovered for this experiment do not overlap with those identified in previous validations, as the objective of this last test is considerably different: While in previous experiments the aim was to separate SARS-CoV-2 samples from other virus strains, here the goal is to separate SARS-CoV-2 patients that require hospitalization from those who do not.</p><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1238335/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1238335/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1238335" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7:</span> <p id="p-58" class="first-child">Appearance frequency table for the 169 sequences. We only show the sequences with the biggest percentage differences between symptomatic and asymptomatic patients.</p><div class="sb-div caption-clear"></div></div></div><p id="p-59">In contrast to Experiments 1-4 where the 21-length bps sequences were scattered, in Experiment 5, seven of the sequences are clustered together in the symptomatic cases <a id="xref-fig-19-1" class="xref-fig" href="#F19">Fig. 19</a>, and four of the sequences in asymptomatic cases <a id="xref-fig-20-1" class="xref-fig" href="#F20">Fig. 20</a>. These sequences are located in the same region of ORF1ab gene, but contain a mutation that discriminates between symptomatic and asympotomatic patients (c.11083G &gt; <em>T</em>). This transversion results in the substitution of leucine to phenylalanine (p.L3606F). A previous report identified the same mutation in the ORF1ab gene [<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>]. ORF1ab proteins play an important role in pathogenesis and viral replication. These might be through the interaction of structural and non-structural proteins, besides the regulatory sequences in viral RNA. Furthermore, it has been described that mutations in ORF1ab are positively selected during trans-species transmission of SARS-CoV and SARS-like coronaviruses. Thus, we suggest that the missense mutation in ORF1ab gene (c.11083G &gt; <em>T</em>) could alters the viral load during the infection between symptomatic and asym-potomatic patients [<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>].</p><div id="F19" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F19.large.jpg?width=800&amp;height=600&amp;carousel=1" title="7 of the first 16 sequences clustered together in a 36-bps sequence, which primarily appears in symptomatic cases." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-188909711" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;7 of the first 16 sequences clustered together in a 36-bps sequence, which primarily appears in symptomatic cases.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 19:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F19.medium.gif" width="440" height="209"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 19:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F19.medium.gif" width="440" height="209"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F19.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 19:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F19.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 19:</span> <p id="p-60" class="first-child">7 of the first 16 sequences clustered together in a 36-bps sequence, which primarily appears in symptomatic cases.</p><div class="sb-div caption-clear"></div></div></div><div id="F20" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F20.large.jpg?width=800&amp;height=600&amp;carousel=1" title="4 of the first 16 sequences clustered together in a 28-bps sequence, which primarily appears in asymptomatic cases." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-188909711" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;4 of the first 16 sequences clustered together in a 28-bps sequence, which primarily appears in asymptomatic cases.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 20:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F20.medium.gif" width="440" height="200"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 20:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F20.medium.gif" width="440" height="200"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F20.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 20:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F20.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 20:</span> <p id="p-61" class="first-child">4 of the first 16 sequences clustered together in a 28-bps sequence, which primarily appears in asymptomatic cases.</p><div class="sb-div caption-clear"></div></div></div><p id="p-62"><a id="xref-fig-21-1" class="xref-fig" href="#F21">Fig. 21</a>. In summary, we get the <a id="xref-table-wrap-8-1" class="xref-table" href="#T8">Table 8</a>. With one extra symptomatic case the presents a missing value, giving the following sequence; <strong>TTTGTTCTTTTTTT-TNTATGAAAATGCCTTTTTACC</strong>.</p><div id="T8" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1238329/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1238329/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1238329" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 8:</span> <p id="p-63" class="first-child">Frequency of appearance of the sequences in asymptomatic and symptomatic cases.</p><div class="sb-div caption-clear"></div></div></div><p id="p-64">Finally, using NC045512.2 as the reference SARS-CoV-2 sequence, and <strong>TTTTTTTT[G/T]TATGAAAATGCCTTTTTAC</strong> as target sequence, we generate a primer set using <em>Primer3plus</em> [<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">41</a>]. This outputs <strong>5’TTCCAAAGT-GCAGTGAAAAGAA3’</strong> as forward primer and <strong>5’TTGCAAAAGCAGA-CATAGCAA3’</strong> as reverse primer with a total length of 175 bps. Then, we run an <em>in-silico PCR</em> test using FastPCR 6.7 [<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">42</a>] with default parameters, this yields the results reported in <a id="xref-fig-22-1" class="xref-fig" href="#F22">Fig. 22</a>. Nevertheless, the results of the PCR Amplicons will need to be sequenced, to differentiate between the two possible sequences.</p><div id="F21" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F21.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Single-nucleotide polymorphism (SNP) between two discovered sequences, that separates asymptomatic from symptomatic patients with a 85% accuracy." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-188909711" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Single-nucleotide polymorphism (SNP) between two discovered sequences, that separates asymptomatic from symptomatic patients with a 85% accuracy.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 21:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F21.medium.gif" width="340" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 21:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F21.medium.gif" width="340" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F21.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 21:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F21.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 21:</span> <p id="p-65" class="first-child">Single-nucleotide polymorphism (SNP) between two discovered sequences, that separates asymptomatic from symptomatic patients with a 85% accuracy.</p><div class="sb-div caption-clear"></div></div></div><div id="F22" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F22.large.jpg?width=800&amp;height=600&amp;carousel=1" title="In-silico PCR test results from the FastPCR 6.7 software [42] using sequences TTCCAAAGTGCAGTGAAAAGAA and TTGCAAAAGCAGACATAGCAA as primers, in NC045512.2 used as a reference SARS-CoV-2 sequence." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-188909711" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;In-silico PCR test results from the FastPCR 6.7 software [42] using sequences TTCCAAAGTGCAGTGAAAAGAA and TTGCAAAAGCAGACATAGCAA as primers, in NC045512.2 used as a reference SARS-CoV-2 sequence.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 22:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F22.medium.gif" width="440" height="368"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 22:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F22.medium.gif" width="440" height="368"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F22.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 22:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.13.990242/F22.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 22:</span> <p id="p-66" class="first-child">In-silico PCR test results from the FastPCR 6.7 software [<a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">42</a>] using sequences TTCCAAAGTGCAGTGAAAAGAA and TTGCAAAAGCAGACATAGCAA as primers, in NC045512.2 used as a reference SARS-CoV-2 sequence.</p><div class="sb-div caption-clear"></div></div></div></div></div></div></div><div class="section" id="sec-18"><h2 class="">5. Conclusion</h2><p id="p-67">Being able to reliably identify SARS-CoV-2 and distinguish it from other similar pathogens is important to contain its spread. The time of processing samples and the availability of reliable diagnostic tests is a challenge during an outbreak. Developing innovative diagnostic tools that target the genome to improve the identification of pathogens, can help reduce health costs and time to identify the infection, instead of using unsuitable treatments or testing. Moreover, it is necessary to perform an accurate classification to identify the different species of Coronavirus, the genetic variants that could appear in the future, and the co-infections with other pathogens.</p><p id="p-68">Given the high transmissibility of the SARS-CoV-2, the proper diagnosis of the disease is urgent, to stop the virus from spreading further. Considering the false negatives given by the standard nucleic acid detection, better implementations such as using deep learning are necessary in order to to properly detect the virus. While the accuracy of current nucleic acid testing is around 30-50%, and CT scans with deep learning go up at 83%, we believe that the use of the sequences detected by a CNN-based system has the potential to improve the accuracy of the diagnosis above 95%.</p><p id="p-69">Our results, show that by targeting only 12 21-bps specific sequences, we are able to distinguish SARS-CoV-2, from any other virus (&gt; 99%). In addition, with 85% accuracy is possible to predict if an infected person will need to be hospitalized or will be asymptomatic. These findings could help to identify patients with SARS-CoV-2 that are susceptible to develop severe acute respiratory infection and make a better clinical management. Nevertheless, our conclusions hold only for the data currently at our disposal. Further testing is necessary to confirm these promising results so it is essential to create multidisciplinary groups that work to stop the outbreak. Finally, as an interesting remark, by comparing the discovered sequences against other hosts, we noticed that from the 12 sequences exclusive to SARS-CoV-2, 1 of them appears in all of the 9 sequences from <em>Manis Javanina.</em> In contrast, 5 of the sequences of SARS-CoV-2 appear in the only sample available from <em>Rhinolophus Affinis.</em> The addition of these sequences sum up to 6 of the 12 sequences that we used to characterized the SARS-CoV-2. This is consistent with the findings of Zhang et al. [<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>], and could point to the zootonic origin of the virus.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-update fn-group-summary-of-updates" id="fn-1"><p id="p-3">Added a section on Feature Design</p></li><li class="fn-dataset fn-group-external-links" id="fn-2"><p id="p-4">
<a href="http://github.org/albertotonda/deep-learning-coronaviru">http://github.org/albertotonda/deep-learning-coronaviru</a>
</p></li><li class="fn" id="fn-3"><p id="p-70"><a class="rev-xref" href="#xref-fn-3-1">↵</a><span class="fn-label">1</span> <a href="https://github.com/albertotonda/deep-learning-coronavirus-genome">https://github.com/albertotonda/deep-learning-coronavirus-genome</a></p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">[1].</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference [1] in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.1" data-doi="10.3390/v2081803"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">P. C.</span> <span class="cit-name-surname">Woo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Huang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. K.</span> <span class="cit-name-surname">Lau</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.-Y.</span> <span class="cit-name-surname">Yuen</span></span>, <span class="cit-article-title">Coronavirus genomics and bioinformatics analysis</span>, <abbr class="cit-jnl-abbrev">viruses</abbr> <span class="cit-vol">2</span> (<span class="cit-issue">8</span>) (<span class="cit-pub-date">2010</span>) <span class="cit-fpage">1804</span>–<span class="cit-lpage">1820</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DViruses%26rft.aulast%253DWoo%26rft.auinit1%253DP.%2BC.%26rft.volume%253D2%26rft.issue%253D8%26rft.spage%253D1804%26rft.epage%253D1820%26rft.atitle%253DCoronavirus%2Bgenomics%2Band%2Bbioinformatics%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv2081803%26rft_id%253Dinfo%253Apmid%252F21994708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v2081803&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21994708&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.13.990242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[2].</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference [2] in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.2"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Lu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Niu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Yang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Wu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Song</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Huang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Zhu</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</span>, <abbr class="cit-jnl-abbrev">The Lancet</abbr> <span class="cit-vol">395</span> (<span class="cit-issue">10224</span>) (<span class="cit-pub-date">2020</span>) <span class="cit-fpage">565</span>–<span class="cit-lpage">574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BLancet%26rft.volume%253D395%26rft.spage%253D565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[3].</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference [3] in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.3"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">V. M.</span> <span class="cit-name-surname">Corman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Landt</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kaiser</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Molenkamp</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Meijer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. K.</span> <span class="cit-name-surname">Chu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Bleicker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Brünink</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Schneider</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Schmidt</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Detection of 2019 novel coronavirus (2019-ncov) by real-time rt-pcr</span>, <abbr class="cit-jnl-abbrev">Eurosurveillance</abbr> <span class="cit-vol">25</span> (<span class="cit-issue">3</span>) (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[4].</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference [4] in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.4"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D. K.</span> <span class="cit-name-surname">Chu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Pan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Cheng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. P.</span> <span class="cit-name-surname">Hui</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Krishnan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. Y.</span> <span class="cit-name-surname">Ng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. K.</span> <span class="cit-name-surname">Wan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Yang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Molecular diagnosis of a novel coronavirus (2019-ncov) causing an outbreak of pneumonia</span>, <abbr class="cit-jnl-abbrev">Clinical chemistry</abbr> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[5].</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference [5] in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.5" data-doi="10.1186/1471-2164-14-444"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Marston</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. M.</span> <span class="cit-name-surname">McElhinney</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Ellis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. L.</span> <span class="cit-name-surname">Horton</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. L.</span> <span class="cit-name-surname">Wise</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Leech</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">David</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">de Lamballerie</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Fooks</span></span>, <span class="cit-article-title">Next generation sequencing of viral rna genomes</span>, <abbr class="cit-jnl-abbrev">BMC genomics</abbr> <span class="cit-vol">14</span> (<span class="cit-issue">1</span>) (<span class="cit-pub-date">2013</span>) <span class="cit-fpage">444</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%26rft.volume%253D14%26rft.spage%253D444%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-444%26rft_id%253Dinfo%253Apmid%252F23822119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-444&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23822119&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.13.990242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[6].</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference [6] in text" id="ref-6">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.13.990242v4.6"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Beijing Institute of Genomics</span>, <span class="cit-article-title">Chinese Academy of Science</span>, <span class="cit-source">China National Center for Bioinformation &amp; National Genomics Data Center</span>, <a href="https://bigd.big.ac.cn/ncov/?lang=en">https://bigd.big.ac.cn/ncov/?lang=en</a>, online; accessed <span class="cit-date-in-citation">27 January 2020</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[7].</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference [7] in text" id="ref-7">↵</a><div class="cit ref-cit ref-book" id="cit-2020.03.13.990242v4.7"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">W. H.</span> <span class="cit-name-surname">Organization</span></span>, <span class="cit-source">WHO report Coronavirus disease 2019 (COVID-19)</span>, <span class="cit-publ-name">World Health Organization</span>, <span class="cit-publ-loc">Geneva</span>:, <span class="cit-pub-date">2020</span>., licence: CC BY-NC-SA 3.0 IGO.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[8].</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference [8] in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.8"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Kang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Tong</span></span>, <span class="cit-article-title">Combination of rt-qpcr testing and clinical features for diagnosis of covid-19 facilitates management of sars-cov-2 outbreak</span>, <abbr class="cit-jnl-abbrev">Journal of Medical Virology</abbr> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[9].</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference [9] in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.9"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H. C.</span> <span class="cit-name-surname">Metsky</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Freije</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.-S. F.</span> <span class="cit-name-surname">Kosoko-Thoroddsen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. C.</span> <span class="cit-name-surname">Sabeti</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Myhrvold</span></span>, <span class="cit-article-title">Crispr-based surveillance for covid-19 using genomically-comprehensive machine learning design</span>, <abbr class="cit-jnl-abbrev">bioRxiv</abbr> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[10].</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference [10] in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.10"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Wu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Xu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Qiao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Zheng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Jiang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Mei</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Wu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Deng</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Clinical diagnosis of 8274 samples with 2019-novel coronavirus in wuhan</span>, <abbr class="cit-jnl-abbrev">medRxiv</abbr> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[11].</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference [11] in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.11"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Kang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ma</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zeng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Xiao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Guo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Cai</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Yang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Meng</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">A deep learning algorithm using ct images to screen for corona virus disease (covid-19)</span>, <abbr class="cit-jnl-abbrev">medRxiv</abbr> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[12].</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference [12] in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.12"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. Y.</span> <span class="cit-name-surname">Kim</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. G.</span> <span class="cit-name-surname">Choe</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Oh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. J.</span> <span class="cit-name-surname">Oh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Kim</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Park</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Park</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H. K.</span> <span class="cit-name-surname">Na</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.-d.</span> <span class="cit-name-surname">Oh</span></span>, <span class="cit-article-title">The first case of 2019 novel coronavirus pneumonia imported into korea from wuhan, china: implication for infection prevention and control measures</span>, <abbr class="cit-jnl-abbrev">Journal of Korean Medical Science</abbr> <span class="cit-vol">35</span> (<span class="cit-issue">5</span>) (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[13].</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference [13] in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.13"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">W. R.</span> <span class="cit-name-surname">Pearson</span></span>, <abbr class="cit-jnl-abbrev">[5] rapid and sensitive sequence comparison with fastp and fasta</abbr> (<span class="cit-pub-date">1990</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[14].</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference [14] in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.14" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. F.</span> <span class="cit-name-surname">Altschul</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Gish</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Miller</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. W.</span> <span class="cit-name-surname">Myers</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Lipman</span></span>, <span class="cit-article-title">Basic local alignment search tool</span>, <abbr class="cit-jnl-abbrev">Journal of molecular biology</abbr> <span class="cit-vol">215</span> (<span class="cit-issue">3</span>) (<span class="cit-pub-date">1990</span>) <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.13.990242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[15].</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference [15] in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.15" data-doi="10.1093/bib/bbt078"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Pinello</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Lo Bosco</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.-C.</span> <span class="cit-name-surname">Yuan</span></span>, <span class="cit-article-title">Applications of alignment-free methods in epigenomics</span>, <abbr class="cit-jnl-abbrev">Briefings in Bioinformatics</abbr> <span class="cit-vol">15</span> (<span class="cit-issue">3</span>) (<span class="cit-pub-date">2014</span>) <span class="cit-fpage">419</span>–<span class="cit-lpage">430</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBriefings%2Bin%2BBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbt078%26rft_id%253Dinfo%253Apmid%252F24197932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/bbt078&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24197932&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.13.990242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[16].</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference [16] in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.16" data-doi="10.1093/bioinformatics/btg005"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Vinga</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Almeida</span></span>, <span class="cit-article-title">Alignment-free sequence comparison—a review</span>, <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span> (<span class="cit-issue">4</span>) (<span class="cit-pub-date">2003</span>) <span class="cit-fpage">513</span>–<span class="cit-lpage">523</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DVinga%26rft.auinit1%253DS.%26rft.volume%253D19%26rft.issue%253D4%26rft.spage%253D513%26rft.epage%253D523%26rft.atitle%253DAlignment-free%2Bsequence%2Bcomparison--a%2Breview%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg005%26rft_id%253Dinfo%253Apmid%252F12611807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12611807&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.13.990242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181410700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[17].</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference [17] in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.17"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Bzhalava</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ekström</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Lysholm</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Hultin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Faust</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Persson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lehtinen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.-M.</span> <span class="cit-name-surname">de Villiers</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Dillner</span></span>, <span class="cit-article-title">Phylogenetically diverse tt virus viremia among pregnant women</span>, <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">432</span> (<span class="cit-issue">2</span>) (<span class="cit-pub-date">2012</span>) <span class="cit-fpage">427</span>–<span class="cit-lpage">434</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D432%26rft.spage%253D427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[18].</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference [18] in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.18"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">N. G.</span> <span class="cit-name-surname">Nguyen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V. A.</span> <span class="cit-name-surname">Tran</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. L.</span> <span class="cit-name-surname">Ngo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Phan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F. R.</span> <span class="cit-name-surname">Lumbanraja</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Faisal</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Abapihi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kubo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Satou</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Dna sequence classification by convolutional neural network</span>, <abbr class="cit-jnl-abbrev">Journal of Biomedical Science and Engineering</abbr> <span class="cit-vol">9</span> (<span class="cit-issue">05</span>) (<span class="cit-pub-date">2016</span>) <span class="cit-fpage">280</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiomedical%2BScience%2Band%2BEngineering%26rft.volume%253D9%26rft.spage%253D280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[19].</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference [19] in text" id="ref-19">↵</a><div class="cit ref-cit ref-book" id="cit-2020.03.13.990242v4.19"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Rizzo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Fiannaca</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">La Rosa</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Urso</span></span>, <span class="cit-chapter-title">A deep learning approach to dna sequence classification</span>, in: <span class="cit-source">International Meeting on Computational Intelligence Methods for Bioinformatics and Biostatistics</span>, <span class="cit-publ-name">Springer</span>, <span class="cit-pub-date">2015</span>, pp. <span class="cit-fpage">129</span>–<span class="cit-lpage">140</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[20].</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference [20] in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.20"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Tampuu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Bzhalava</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Dillner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Vicente</span></span>, <span class="cit-article-title">Viraminer: Deep learning on raw dna sequences for identifying viral genomes in human samples</span>, <abbr class="cit-jnl-abbrev">PloS one</abbr> <span class="cit-vol">14</span> (<span class="cit-issue">9</span>) (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[21].</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference [21] in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.21"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ren</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Song</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Deng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Ahlgren</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Fuhrman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Xie</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Sun</span></span>, <span class="cit-article-title">Identifying viruses from metagenomic data by deep learning</span>, <abbr class="cit-jnl-abbrev">arXiv</abbr> preprint<span class="cit-pub-id-sep cit-pub-id-arxiv-sep"> </span><span class="cit-pub-id-scheme">arXiv:</span><span class="cit-pub-id cit-pub-id-arxiv">1806.07810</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[22].</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference [22] in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.22" data-doi="10.1093/nar/29.1.308"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. T.</span> <span class="cit-name-surname">Sherry</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.-H.</span> <span class="cit-name-surname">Ward</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kholodov</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Baker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Phan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. M.</span> <span class="cit-name-surname">Smigielski</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Sirotkin</span></span>, <span class="cit-article-title">dbsnp: the ncbi database of genetic variation</span>, <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr> <span class="cit-vol">29</span> (<span class="cit-issue">1</span>) (<span class="cit-pub-date">2001</span>) <span class="cit-fpage">308</span>–<span class="cit-lpage">311</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSherry%26rft.auinit1%253DS.%2BT.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D308%26rft.epage%253D311%26rft.atitle%253DdbSNP%253A%2Bthe%2BNCBI%2Bdatabase%2Bof%2Bgenetic%2Bvariation%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F29.1.308%26rft_id%253Dinfo%253Apmid%252F11125122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/29.1.308&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11125122&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.13.990242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166360300086&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[23].</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference [23] in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.23"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C. d. S.</span> <span class="cit-name-surname">Ribeiro</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. Y.</span> <span class="cit-name-surname">van Roode</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. B.</span> <span class="cit-name-surname">Haringhuizen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. P.</span> <span class="cit-name-surname">Koopmans</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Claassen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. H.</span> <span class="cit-name-surname">van de Burgwal</span></span>, <span class="cit-article-title">How ownership rights over microorganisms affect infectious disease control and innovation: a root-cause analysis of barriers to data sharing as experienced by key stakeholders</span>, <abbr class="cit-jnl-abbrev">PloS one</abbr> <span class="cit-vol">13</span> (<span class="cit-issue">5</span>) (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[24].</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference [24] in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.24"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Simon</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Claassen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. E.</span> <span class="cit-name-surname">Correa</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Osterhaus</span></span>, <span class="cit-article-title">Managing severe acute respiratory syndrome (sars) intellectual property rights: the possible role of patent pooling</span>, <abbr class="cit-jnl-abbrev">Bulletin of the World Health Organization</abbr> <span class="cit-vol">83</span> (<span class="cit-pub-date">2005</span>) <span class="cit-fpage">707</span>–<span class="cit-lpage">710</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBulletin%2Bof%2Bthe%2BWorld%2BHealth%2BOrganization%26rft.stitle%253DBull%2BWorld%2BHealth%2BOrgan%26rft.aulast%253DSimon%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D83%26rft.issue%253D9%26rft.spage%253D707%26rft.epage%253D710%26rft.atitle%253DManaging%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome%2B%2528SARS%2529%2Bintellectual%2Bproperty%2Brights%253A%2Bthe%2Bpossible%2Brole%2Bof%2Bpatent%2Bpooling.%26rft_id%253Dinfo%253Apmid%252F16211163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=16211163&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.13.990242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[25].</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference [25] in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.25" data-doi="10.1126/science.aau5229"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C. d. S.</span> <span class="cit-name-surname">Ribeiro</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. P.</span> <span class="cit-name-surname">Koopmans</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. B.</span> <span class="cit-name-surname">Haringhuizen</span></span>, <span class="cit-article-title">Threats to timely sharing of pathogen sequence data</span>, <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">362</span> (<span class="cit-issue">6413</span>) (<span class="cit-pub-date">2018</span>) <span class="cit-fpage">404</span>–<span class="cit-lpage">406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253Ddos%2BS.%2BRibeiro%26rft.auinit1%253DC.%26rft.volume%253D362%26rft.issue%253D6413%26rft.spage%253D404%26rft.epage%253D406%26rft.atitle%253DThreats%2Bto%2Btimely%2Bsharing%2Bof%2Bpathogen%2Bsequence%2Bdata%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aau5229%26rft_id%253Dinfo%253Apmid%252F30361362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNjIvNjQxMy80MDQiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wNi8yMDIwLjAzLjEzLjk5MDI0Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[26].</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference [26] in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.26"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Lopez-Rincon</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Tonda</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Elati</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Schwander</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Piwowarski</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Gal-linari</span></span>, <span class="cit-article-title">Evolutionary optimization of convolutional neural networks for cancer mirna biomarkers classification</span>, <abbr class="cit-jnl-abbrev">Applied Soft Computing</abbr> <span class="cit-vol">65</span> (<span class="cit-pub-date">2018</span>) <span class="cit-fpage">91</span>–<span class="cit-lpage">100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2BSoft%2BComputing%26rft.volume%253D65%26rft.spage%253D91%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[27].</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference [27] in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.27" data-doi="10.1086/374222"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Vabret</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Mourez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Gouarin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Petitjean</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Freymuth</span></span>, <span class="cit-article-title">An outbreak of coronavirus oc43 respiratory infection in normandy, france</span>, <abbr class="cit-jnl-abbrev">Clinical infectious diseases</abbr> <span class="cit-vol">36</span> (<span class="cit-issue">8</span>) (<span class="cit-pub-date">2003</span>) <span class="cit-fpage">985</span>–<span class="cit-lpage">989</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2Binfectious%2Bdiseases%26rft_id%253Dinfo%253Adoi%252F10.1086%252F374222%26rft_id%253Dinfo%253Apmid%252F12684910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/374222&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12684910&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.13.990242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182124700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[28].</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference [28] in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.28"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L.-J.</span> <span class="cit-name-surname">Cui</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.-J.</span> <span class="cit-name-surname">Lu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.-D.</span> <span class="cit-name-surname">Xie</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.-L.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.-Y.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.-M.</span> <span class="cit-name-surname">Zhou</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Ruan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.-J.</span> <span class="cit-name-surname">Ma</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Human coronaviruses hcov-nl63 and hcov-hku1 in hospitalized children with acute respiratory infections in beijing, china</span>, <abbr class="cit-jnl-abbrev">Advances in virology</abbr> <span class="cit-vol">2011</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[29].</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference [29] in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.29"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Zeng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Chan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Chan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Chow</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Hon</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Hui</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">The complete genome sequence of severe acute respiratory syndrome coronavirus strain hku-39849 (hk-39)</span>, <abbr class="cit-jnl-abbrev">Experimental Biology and Medicine</abbr> <span class="cit-vol">228</span> (<span class="cit-issue">7</span>) (<span class="cit-pub-date">2003</span>) <span class="cit-fpage">866</span>–<span class="cit-lpage">873</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExperimental%2BBiology%2Band%2BMedicine%26rft.stitle%253DExp%2BBiol%2BMed%2B%2528Maywood%2529%26rft.aulast%253DZeng%26rft.auinit1%253DF.%2BY.%26rft.volume%253D228%26rft.issue%253D7%26rft.spage%253D866%26rft.epage%253D873%26rft.atitle%253DThe%2BComplete%2BGenome%2BSequence%2Bof%2BSevere%2BAcute%2BRespiratory%2BSyndrome%2BCoronavirus%2BStrain%2BHKU-39849%2B%2528HK-39%2529%26rft_id%253Dinfo%253Apmid%252F12876307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12876307&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.13.990242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[30].</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference [30] in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.30"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Mizrachi</span></span>, <span class="cit-article-title">Genbank: the nucleotide sequence database</span>, <abbr class="cit-jnl-abbrev">The NCBI Handbook [Internet], updated</abbr> <span class="cit-vol">22</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[31].</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference [31] in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.31"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D. P.</span> <span class="cit-name-surname">Kingma</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ba</span></span>, <span class="cit-article-title">Adam: A method for stochastic optimization</span>, <abbr class="cit-jnl-abbrev">arXiv</abbr> preprint<span class="cit-pub-id-sep cit-pub-id-arxiv-sep"> </span><span class="cit-pub-id-scheme">arXiv:</span><span class="cit-pub-id cit-pub-id-arxiv">1412.6980</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[32].</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference [32] in text" id="ref-32">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.13.990242v4.32"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Abadi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Agarwal</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Barham</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Brevdo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Citro</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. S.</span> <span class="cit-name-surname">Corrado</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Davis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Dean</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Devin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Ghemawat</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Goodfellow</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Harp</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Irving</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Isard</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Jia</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Jozefowicz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Kaiser</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kudlur</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Lev-enberg</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Mané</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Monga</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Moore</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Murray</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Olah</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Schuster</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Shlens</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Steiner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Sutskever</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Talwar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Tucker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Vanhoucke</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Vasudevan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Viegas</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Vinyals</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Warden</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Wattenberg</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Wicke</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Yu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zheng</span></span>, <span class="cit-source">TensorFlow: Large-scale machine learning on heterogeneous systems, software available from tensorflow.org</span> (<span class="cit-pub-date">2015</span>). URL <a href="https://www.tensorflow.org/">https://www.tensorflow.org/</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[33].</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference [33] in text" id="ref-33">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.13.990242v4.33"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Chollet</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-source">Keras</span>, <a href="https://keras.io">https://keras.io</a> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[34].</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference [34] in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.34"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.-M.</span> <span class="cit-name-surname">šimundić</span></span>, <span class="cit-article-title">Measures of diagnostic accuracy: basic definitions</span>, <abbr class="cit-jnl-abbrev">Ejifcc</abbr> <span class="cit-vol">19</span> (<span class="cit-issue">4</span>) (<span class="cit-pub-date">2009</span>) <span class="cit-fpage">203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEjifcc%26rft.volume%253D19%26rft.spage%253D203%26rft_id%253Dinfo%253Apmid%252F27683318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=27683318&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.13.990242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[35].</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference [35] in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.35" data-doi="10.1097/JTO.0b013e3181ec173d"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. N.</span> <span class="cit-name-surname">Mandrekar</span></span>, <span class="cit-article-title">Receiver operating characteristic curve in diagnostic test assessment</span>, <abbr class="cit-jnl-abbrev">Journal of Thoracic Oncology</abbr> <span class="cit-vol">5</span> (<span class="cit-issue">9</span>) (<span class="cit-pub-date">2010</span>) <span class="cit-fpage">1315</span>–<span class="cit-lpage">1316</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BThoracic%2BOncology%26rft.volume%253D5%26rft.spage%253D1315%26rft_id%253Dinfo%253Adoi%252F10.1097%252FJTO.0b013e3181ec173d%26rft_id%253Dinfo%253Apmid%252F20736804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1097/JTO.0b013e3181ec173d&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20736804&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.13.990242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281602900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[36].</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference [36] in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.36"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Lopez-Rincon</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Martinez-Archundia</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. U.</span> <span class="cit-name-surname">Martinez-Ruiz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Schoenhuth</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Tonda</span></span>, <span class="cit-article-title">Automatic discovery of 100-mirna signature for cancer classification using ensemble feature selection</span>, <abbr class="cit-jnl-abbrev">BMC bioinformatics</abbr> <span class="cit-vol">20</span> (<span class="cit-issue">1</span>) (<span class="cit-pub-date">2019</span>) <span class="cit-fpage">480</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%26rft.volume%253D20%26rft.spage%253D480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[37].</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference [37] in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.37"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Shu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">McCauley</span></span>, <span class="cit-article-title">Gisaid: Global initiative on sharing all influenza data-from vision to reality</span>, <abbr class="cit-jnl-abbrev">Eurosurveillance</abbr> <span class="cit-vol">22</span> (<span class="cit-issue">13</span>) (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[38].</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference [38] in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.38"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.-S.</span> <span class="cit-name-surname">Shi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N. R.</span> <span class="cit-name-surname">Nabar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.-N.</span> <span class="cit-name-surname">Huang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Kehrl</span></span>, <span class="cit-article-title">Sars-coronavirus open reading frame-8b triggers intracellular stress pathways and activates nlrp3 inflammasomes</span>, <abbr class="cit-jnl-abbrev">Cell death discovery</abbr> <span class="cit-vol">5</span> (<span class="cit-issue">1</span>) (<span class="cit-pub-date">2019</span>) <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Bdeath%2Bdiscovery%26rft.volume%253D5%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[39].</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference [39] in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.39" data-doi="10.1016/j.febslet.2006.11.046"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Kanzawa</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Nishigaki</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Hayashi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Ishii</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Furukawa</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Niiro</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Yasui</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kohara</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Morita</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Matsushima</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Augmentation of chemokine production by severe acute respiratory syndrome coronavirus 3a/x1 and 7a/x4 proteins through nf-ĸb activation</span>, <abbr class="cit-jnl-abbrev">FEBS letters</abbr> <span class="cit-vol">580</span> (<span class="cit-issue">30</span>) (<span class="cit-pub-date">2006</span>) <span class="cit-fpage">6807</span>–<span class="cit-lpage">6812</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DKanzawa%26rft.auinit1%253DN.%26rft.volume%253D580%26rft.issue%253D30%26rft.spage%253D6807%26rft.epage%253D6812%26rft.atitle%253DAugmentation%2Bof%2Bchemokine%2Bproduction%2Bby%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome%2Bcoronavirus%2B3a%252FX1%2Band%2B7a%252FX4%2Bproteins%2Bthrough%2BNF-kappaB%2Bactivation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.febslet.2006.11.046%26rft_id%253Dinfo%253Apmid%252F17141229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.febslet.2006.11.046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17141229&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.13.990242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[40].</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference [40] in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.40" data-doi="10.1099/vir.0.82856-0"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Padhan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Tanwar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Hussain</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. Y.</span> <span class="cit-name-surname">Hui</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. Y.</span> <span class="cit-name-surname">Lee</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. Y.</span> <span class="cit-name-surname">Cheung</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. S. M.</span> <span class="cit-name-surname">Peiris</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Jameel</span></span>, <span class="cit-article-title">Severe acute respiratory syndrome coronavirus orf3a protein interacts with caveolin</span>, <abbr class="cit-jnl-abbrev">Journal of General Virology</abbr> <span class="cit-vol">88</span> (<span class="cit-issue">11</span>) (<span class="cit-pub-date">2007</span>) <span class="cit-fpage">3067</span>–<span class="cit-lpage">3077</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DPadhan%26rft.auinit1%253DK.%26rft.volume%253D88%26rft.issue%253D11%26rft.spage%253D3067%26rft.epage%253D3077%26rft.atitle%253DSevere%2Bacute%2Brespiratory%2Bsyndrome%2Bcoronavirus%2BOrf3a%2Bprotein%2Binteracts%2Bwith%2Bcaveolin%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.82856-0%26rft_id%253Dinfo%253Apmid%252F17947532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.82856-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17947532&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.13.990242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[41].</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference [41] in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.41" data-doi="10.1093/nar/gkm306"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Untergasser</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Nijveen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Rao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Bisseling</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Geurts</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Leunissen</span></span>, <span class="cit-article-title">Primer3plus, an enhanced web interface to primer3</span>, <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr> <span class="cit-vol">35</span> (suppl_<span class="cit-issue">2</span>) (<span class="cit-pub-date">2007</span>) <span class="cit-fpage">W71</span>–<span class="cit-lpage">W74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm306%26rft_id%253Dinfo%253Apmid%252F17485472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkm306&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17485472&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.13.990242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255311500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[42].</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference [42] in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.42"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Kalendar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Lee</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. H.</span> <span class="cit-name-surname">Schulman</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Fastpcr software for pcr primer and probe design and repeat search</span>, <abbr class="cit-jnl-abbrev">Genes, Genomes and Genomics</abbr> <span class="cit-vol">3</span> (<span class="cit-issue">1</span>) (<span class="cit-pub-date">2009</span>) <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%252C%2BGenomes%2Band%2BGenomics%26rft.volume%253D3%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[43].</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference [43] in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.43"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Phan</span></span>, <span class="cit-article-title">Genetic diversity and evolution of sars-cov-2</span>, <abbr class="cit-jnl-abbrev">Infection, Genetics and Evolution</abbr> <span class="cit-vol">81</span> (<span class="cit-pub-date">2020</span>) <span class="cit-fpage">104260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%252C%2BGenetics%2Band%2BEvolution%26rft.volume%253D81%26rft.spage%253D104260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[44].</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference [44] in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.44" data-doi="10.1016/j.virusres.2007.02.017"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Graham</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Sparks</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. D.</span> <span class="cit-name-surname">Eckerle</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Sims</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Denison</span></span>, <span class="cit-article-title">Sars coronavirus replicase proteins in pathogenesis</span>, <abbr class="cit-jnl-abbrev">Virus research</abbr> <span class="cit-vol">133</span> (<span class="cit-issue">1</span>) (<span class="cit-pub-date">2008</span>) <span class="cit-fpage">88</span>–<span class="cit-lpage">100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DGraham%26rft.auinit1%253DR.%2BL.%26rft.volume%253D133%26rft.issue%253D1%26rft.spage%253D88%26rft.epage%253D100%26rft.atitle%253DSARS%2Bcoronavirus%2Breplicase%2Bproteins%2Bin%2Bpathogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2007.02.017%26rft_id%253Dinfo%253Apmid%252F17397959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2007.02.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17397959&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.13.990242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[45].</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference [45] in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990242v4.45"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Y.-Z.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. C.</span> <span class="cit-name-surname">Holmes</span></span>, <span class="cit-article-title">A genomic perspective on the origin and emergence of sars-cov-2</span>, <abbr class="cit-jnl-abbrev">Cell</abbr> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.13.990242v4.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.13.990242v4" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.04.05.026146v1" title="CovProfile: profiling the viral genome and gene expressions of SARS-COV2" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/765230v2" title="Co-option of the lineage-specific LAVA retrotransposon in the gibbon genome" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;April 06, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.13.990242v4.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/early/2020/04/06/2020.03.13.990242.external-links" target="_self" class="link-icon"><i class="icon-file"></i> <span class="title">Data/Code</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.13.990242v4.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.13.990242v4" class="active" data-icon-position="" data-hide-link-title="0">Accurate Identification of SARS-CoV-2 from Viral Genome Sequences using Deep Learning</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1231123" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-e1J74yH-KBPEKk19n1Wj6U_MVKreZc0C-uewHIMRwVQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1231123" id="node1231123--2" data-pisa="biorxiv;2020.03.13.990242v4" data-pisa-master="biorxiv;2020.03.13.990242" data-seqnum="1231123" data-apath="/biorxiv/early/2020/04/06/2020.03.13.990242.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Accurate Identification of SARS-CoV-2 from Viral Genome Sequences using Deep Learning</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Alejandro</span> <span class="nlm-surname">Lopez-Rincon</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Alberto</span> <span class="nlm-surname">Tonda</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Lucero</span> <span class="nlm-surname">Mendoza-Maldonado</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Eric</span> <span class="nlm-surname">Claassen</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Johan</span> <span class="nlm-surname">Garssen</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Aletta D.</span> <span class="nlm-surname">Kraneveld</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.13.990242; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.13.990242 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.13.990242v4" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.990242v4&amp;title=Accurate%20Identification%20of%20SARS-CoV-2%20from%20Viral%20Genome%20Sequences%20using%20Deep%20Learning" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.990242v4&amp;title=Accurate%20Identification%20of%20SARS-CoV-2%20from%20Viral%20Genome%20Sequences%20using%20Deep%20Learning" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.990242v4&amp;text=Accurate%20Identification%20of%20SARS-CoV-2%20from%20Viral%20Genome%20Sequences%20using%20Deep%20Learning" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.990242v4&amp;title=Accurate%20Identification%20of%20SARS-CoV-2%20from%20Viral%20Genome%20Sequences%20using%20Deep%20Learning" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.990242v4&amp;t=Accurate%20Identification%20of%20SARS-CoV-2%20from%20Viral%20Genome%20Sequences%20using%20Deep%20Learning" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.990242v4&amp;title=Accurate%20Identification%20of%20SARS-CoV-2%20from%20Viral%20Genome%20Sequences%20using%20Deep%20Learning" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.990242v4&amp;title=Accurate%20Identification%20of%20SARS-CoV-2%20from%20Viral%20Genome%20Sequences%20using%20Deep%20Learning" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1231123" id="node1231123--3" data-pisa="biorxiv;2020.03.13.990242v4" data-pisa-master="biorxiv;2020.03.13.990242" data-seqnum="1231123" data-apath="/biorxiv/early/2020/04/06/2020.03.13.990242.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Accurate Identification of SARS-CoV-2 from Viral Genome Sequences using Deep Learning</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Alejandro</span> <span class="nlm-surname">Lopez-Rincon</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Alberto</span> <span class="nlm-surname">Tonda</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Lucero</span> <span class="nlm-surname">Mendoza-Maldonado</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Eric</span> <span class="nlm-surname">Claassen</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Johan</span> <span class="nlm-surname">Garssen</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Aletta D.</span> <span class="nlm-surname">Kraneveld</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.13.990242; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.13.990242 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Accurate%20Identification%20of%20SARS-CoV-2%20from%20Viral%20Genome%20Sequences%20using%20Deep%20Learning&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Lopez-Rincon%2C+Alejandro&amp;rft.au=Tonda%2C+Alberto&amp;rft.au=Mendoza-Maldonado%2C+Lucero&amp;rft.au=Claassen%2C+Eric&amp;rft.au=Garssen%2C+Johan&amp;rft.au=Kraneveld%2C+Aletta+D."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1231123/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1231123/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1231123/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1231123/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1231123/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1231123/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1231123/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1231123/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1231123/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1231123/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1231123/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1231123/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.990242v4&amp;count=horizontal&amp;via=&amp;text=Accurate%20Identification%20of%20SARS-CoV-2%20from%20Viral%20Genome%20Sequences%20using%20Deep%20Learning&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.990242v4" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.990242v4&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.13.990242v4" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">Bioinformatics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
