<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Virusdisease</span><span class="iso-abbrev" title="iso-abbrev">Virusdisease</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">VirusDisease</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">2347-3584</span><span class="issn-epub" tagx="issn" title="issn-epub">2347-3517</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer India</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New Delhi</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6531531">6531531</a></span><span class="publisher-id" title="publisher-id">513</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1007/s13337-019-00513-w">10.1007/s13337-019-00513-w</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Original Article</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Isolation and recombinant analysis of variants of porcine epidemic diarrhea virus strains from Beijing, China</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Peng</span></span><a href="#Aff1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">Jinyan</span></span><a href="#Aff1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Xinze</span></span><a href="#Aff1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">Jiaojiao</span></span><a href="#Aff1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gu</span><span class="given-names" tagx="given-names" title="given-names">Xuejia</span></span><a href="#Aff1" /></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-5775-3175</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruan</span><span class="given-names" tagx="given-names" title="given-names">Wenke</span></span><span class="address" title="address"><span class="phone" tagx="phone" title="phone">86-(10)-8079-5593</span><span class="email" tagx="email" title="email">wenkeruan@hotmail.com</span></span><a href="#Aff1" /></span><span class="citation_author_institution" id="Aff1">[], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0004 1798 6793</span><span class="institution-id" title="institution-id">grid.411626.6</span><span class="institution" title="institution">Beijing Key Laboratory of Traditional Chinese Veterinary Medicine, College of Animal Science and Technology, </span><span class="institution" title="institution">Beijing University of Agriculture, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-2-2</span></span><span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2019-6</span></span><span class="volume" tagx="volume" title="volume">30</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">294</span><span class="lpage" tagx="lpage" title="lpage">301</span><span class="history" title="history"><span class="received" title="received">received: 2018-11-14</span><span class="accepted" title="accepted">accepted: 2019-2-1</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p id="Par1">Porcine epidemic diarrhea virus (PEDV) is a highly infectious virus infecting pigs with high morbidity, especially for newborn piglets. Several PEDV strains were isolated from the intestinal tracts of diarrheic piglets from the Beijing area, China. Sequencing of the whole-genome of the PEDV isolates (GenBank numbers MG546687-MG546690) yielded sequences of 28033–28038 nt. The phylogenetic tree revealed that these strains from the Beijing area belonged to group II, while the vaccine strain, CV777, belonged to group I. We also determined the genetic correlation between these strains and CV777 strain. However, it showed that these strains in the Beijing area had unique mutations. The sequence identity of PEDV strains showed that these strains are most similar to these strains LZW, CH/JX-1/2013, USAIllinois972013, USAKansas1252014, CH/GDZQ/2014, SHQPYM2013, AJ1102, CHZMDZY11, KoreaK14JB01, and CHYJ130330, respectively. The possible recombination events indicate that PEDV in this studies were possibly recombinant strain formed by parent strains USAIllinois972013, KoreaK14JB01, CHYJ130330, and CHZMDZY11. These PEDV strains has been genetic recombination and mutations. The variant strains characterized in this study help to the evolutionary analysis of PEDV.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
PEDV</div> <div class="kwd" title="kwd">
Sequence analysis</div> <div class="kwd" title="kwd">
Isolation</div> <div class="kwd" title="kwd">
Phylogenetic tree</div> <div class="kwd" title="kwd">
Whole-genome</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100001809</span><span class="institution" title="institution">National Natural Science Foundation of China</span></div> </div><span class="award-id" tagx="award-id" title="award-id">31572499</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruan</span><span class="given-names" tagx="given-names" title="given-names">Wenke</span></span></span></div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© Indian Virological Society 2019" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="Par2">Porcine epidemic diarrhea (PED) is a highly infectious disease of swine. It causes severe watery diarrhea, vomiting, dehydration and death in suckling and nursing piglets that are typically 1–2 weeks of age [<a href="#CR1">1</a>, <a href="#CR2">2</a>]. Porcine epidemic diarrhea virus (PEDV) is a member of the Coronaviridae family [<a href="#CR3">3</a>, <a href="#CR4">4</a>]. The virus strain was first isolated in Belgium in 1973 and designated coronavirus CV777 [<a href="#CR1">1</a>]. There were only a few reports in the following decades [<a href="#CR5">5</a>–<a href="#CR7">7</a>]. However, a new PEDV strain has emerged in Asia since 2010 [<a href="#CR7">7</a>–<a href="#CR10">10</a>]. In China, the positive rate of PEDV in some provinces was over 50% in 2010–2011 [<a href="#CR7">7</a>]. In USA, classical PEDV first identified in the state of Ohio in 2013, and a variant PEDV was reported in 2014 [<a href="#CR11">11</a>]. In Japan, more than 490,000 pigs died of PED from 2013 to 2015 [<a href="#CR12">12</a>]. In Korea, PED outbreaks reoccurred in 2013 [<a href="#CR13">13</a>, <a href="#CR14">14</a>]. In Europe, isolates were reported from the Ukraine, Belgium, Holland, France, Germany, Portugal and Italy from 2013 to 2015 [<a href="#CR15">15</a>]. These isolates were closely related to those strains reported from North America [<a href="#CR11">11</a>, <a href="#CR15">15</a>].</p> <p id="Par3">Diarrhea disease broke out in some large-scale pig farms in the Beijing area of China from 2015 to 2016. The newborn piglets had high mortality rates, resulting in heavy loss to pig farms. The CV777 vaccines used failed. PEDV strains were isolated and identified from the small intestine of diarrhea samples obtained at these pig farms. Whole genome sequencing and recombinant analysis was performed, with the aim of examining the genomes for viral mutations and recombination. The molecular epidemiological information of PEDV can help develop efficient diagnostic reagent and vaccine.</p> </div> <div class="materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="virusisolationandidentification" title="sec"> <div class="title" tagx="title" title="title">
Virus isolation and identification</div> <p id="Par4">Intestine samples of dead piglets with diarrhea were collected from several piglets infected with PEDV from 2015 to 2016. These infected piglets were chosen from 4 different large-scale swine farms located in Beijing and Hebei provinces in China. PEDV detection were positive via RT-PCR. For total RNA extraction, samples were frozen with liquid nitrogen. All procedures were approved by the Animal Care and Use Committee of Beijing University of Agriculture (Beijing, China).</p> <p id="Par5">The fresh intestine samples were collected under sterile conditions, made then into slurry, filter sterilization. Vero E6 cells were washed with PBS twice. After adsorption for 60 min at 37 °C, PEDV growth medium was added [<a href="#CR16">16</a>]. The Vero E6 cell cultures were observed for 4 days for cytopathic effects (CPE).</p> </div> </div> <div class="completenucleotidesequenceofpedv" title="sec"> <div class="title" tagx="title" title="title">
Complete nucleotide sequence of PEDV</div> <p id="Par6">Total RNA was obtained from intestine samples using TRIzol (Invitrogen, USA), per the manufacturer’s instructions (Promega, USA), and was used for cDNA synthesis. PCR was performed to amplify the target gene using specific primers (reference sequence number: NC003436.1), as shown in Table <a href="#Tab1">1</a>. PCR reaction contained primers, cDNA, dNTP mix and Hifi polymerase (Transgen, China) in 1× Hifi buffer. PCR products were sequenced from both ends by Sangon Biotech co. ltd. The coding sequences for PEDV were deposited in the GenBank database under accession numbers MG546687-MG546690. <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 1</span> </div></p><p>Primers of the PEDV genes</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left">Genes</th> <th align="left">Forward primer (5′–3′)</th> <th align="left">Reverse primer (5′–3′)</th> <th align="left">Position</th> </tr> </thead> <tbody> <tr> <td align="left">PEDVORF1P1</td> <td align="left">ATGGCTAGCAACCATGTCACATTGG</td> <td align="left">TGCATGCTTACCCTTACGTGGACC</td> <td align="left">293–4285</td> </tr> <tr> <td align="left">PEDV ORF1P2</td> <td align="left">GCAAAGGCCATTGATGTTTATACCA</td> <td align="left">TCGAAGCACAAACCAGATGTACCAA</td> <td align="left">4286–8033</td> </tr> <tr> <td align="left">PEDV ORF1P3</td> <td align="left">TCTTTATTAACTTCGACCAGTGGCA</td> <td align="left">GAACTTACGTGGTGGTTCTAGCTCA</td> <td align="left">8034–11986</td> </tr> <tr> <td align="left">PEDV ORF1P4</td> <td align="left">ATTTCAGATAAACCGGACCTGCGTG</td> <td align="left">GCACAAAAACCATGCCAGGAACAAG</td> <td align="left">11987–15954</td> </tr> <tr> <td align="left">PEDV ORF1P5</td> <td align="left">ATCGGTGAGTTTGTGTTTGAGAAGG</td> <td align="left">GTCAAAGCAGAACCACTTAGCATCC</td> <td align="left">15955–18214</td> </tr> <tr> <td align="left">PEDV ORF1P6</td> <td align="left">ATGTGGTTCGCAAGCGTATAGTTCA</td> <td align="left">TCATTTGTTTACGTTGACCAAATGATTAGA</td> <td align="left">18215–20633</td> </tr> <tr> <td align="left">PEDV ORF3</td> <td align="left">ATGTTTCTTGGACTTTTTCAATAC</td> <td align="left">TCATTCACTAATTGTAGCATACTCG</td> <td align="left">24794–25448</td> </tr> <tr> <td align="left">PEDVE</td> <td align="left">ATGCTACAATTAGTGAATGATAATGGG</td> <td align="left">TTATACGTCAATAACAGTACTGGGG</td> <td align="left">25449–25686</td> </tr> <tr> <td align="left">PEDVM</td> <td align="left">ATGTCTAACGGTTCTATTCCC</td> <td align="left">TTAGACTAAATGAAGCACTTTCTC</td> <td align="left">25687–26396</td> </tr> <tr> <td align="left">PEDVN</td> <td align="left">ATGGCTTCTGTCAGCTTTCAGGATCG</td> <td align="left">TTAATTTCCTGTGTCGAAGATCTCGTTGA</td> <td align="left">26397–27704</td> </tr> <tr> <td align="left">PEDV5UTR</td> <td align="left">ACTTAAAAAGATTTTCTATCTACGGATA</td> <td align="left">AGCCGGCAGTTACTGGTTT</td> <td align="left">1–292</td> </tr> <tr> <td align="left">PEDV3UTR</td> <td align="left">ACAATGTTAGACCGGCTTATCCT</td> <td align="left">GTGTATCCATATCAACACCGTCAG</td> <td align="left">27705–28038</td> </tr> <tr> <td align="left">PEDVS</td> <td align="left">ATGAAGTCTTTAACCTACTTCTGGTTGTT</td> <td align="left">TCACTGCACGTGGACCTTTTC</td> <td align="left">20634–24793</td> </tr> </tbody> </table>  <p /> <div class="sequenceanalysis" title="sec"> <div class="title" tagx="title" title="title">
Sequence analysis</div> <p id="Par7">Nucleotide and amino acid sequences of PEDV were aligned by the ClustalW method. Sequence alignments of the whole genome and construction of trees were also performed by the neighbor-joining method in MEGA7 software. Information of the other strains that were chosen in this study, including years and locations of isolation and GenBank accession numbers proposed, are shown in Table <a href="#Tab2">2</a>. <div class="table-wrap_UNKNOWN" id="Tab2"><span class="label" tagx="label" title="label">Table 2</span> </div></p><p>PEDV strains used in this study</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left">Strains</th> <th align="left">Location</th> <th align="left">Date</th> <th align="left">Accession no.</th> </tr> </thead> <tbody> <tr> <td align="left">CH/BJ9/2015</td> <td align="left">Beijing, China</td> <td align="left">2015</td> <td align="left">MG546687</td> </tr> <tr> <td align="left">CH/BJ2/2016</td> <td align="left">Beijing, China</td> <td align="left">2016</td> <td align="left">MG546688</td> </tr> <tr> <td align="left">CH/BJ11/2016</td> <td align="left">Beijing, China</td> <td align="left">2016</td> <td align="left">MG546690</td> </tr> <tr> <td align="left">CH/HB9/2016</td> <td align="left">Hebei, China</td> <td align="left">2016</td> <td align="left">MG546689</td> </tr> <tr> <td align="left">CV777</td> <td align="left">Belgium</td> <td align="left">1978</td> <td align="left">AF353511</td> </tr> <tr> <td align="left">CH/GDZQ/2014</td> <td align="left">Gangdong, China</td> <td align="left">2014</td> <td align="left">KM242131</td> </tr> <tr> <td align="left">CHZMDZY11</td> <td align="left">Henan, China</td> <td align="left">2011</td> <td align="left">KC196276</td> </tr> <tr> <td align="left">CH/JX-1/2013</td> <td align="left">Jiangxi, China</td> <td align="left">2013</td> <td align="left">KF760557</td> </tr> <tr> <td align="left">SHQPYM2013</td> <td align="left">Shanghai, China</td> <td align="left">2013</td> <td align="left">KJ196348</td> </tr> <tr> <td align="left">CHYJ130330</td> <td align="left">Gangdong, China</td> <td align="left">2013</td> <td align="left">KJ020932</td> </tr> <tr> <td align="left">CH/YNKM-8/2013</td> <td align="left">Yunnan, China</td> <td align="left">2013</td> <td align="left">KF761675</td> </tr> <tr> <td align="left">CH/FJZZ-9/2012</td> <td align="left">Fujian, China</td> <td align="left">2012</td> <td align="left">KC140102</td> </tr> <tr> <td align="left">CH/GDGZ/2012</td> <td align="left">Gangdong, China</td> <td align="left">2012</td> <td align="left">KF384500</td> </tr> <tr> <td align="left">AJ1102</td> <td align="left">Hubei, China</td> <td align="left">2011</td> <td align="left">JX188454</td> </tr> <tr> <td align="left">LC</td> <td align="left">Gangdong, China</td> <td align="left">2012</td> <td align="left">JX489155</td> </tr> <tr> <td align="left">KC189944</td> <td align="left">Hubei, China</td> <td align="left">2013</td> <td align="left">KC189944</td> </tr> <tr> <td align="left">SD-M</td> <td align="left">Shandong, China</td> <td align="left">2012</td> <td align="left">JX560761</td> </tr> <tr> <td align="left">ZJCZ4</td> <td align="left">Zhejiang, China</td> <td align="left">2012</td> <td align="left">JX524137</td> </tr> <tr> <td align="left">LZW</td> <td align="left">Beijing, China</td> <td align="left">2014</td> <td align="left">KJ777678</td> </tr> <tr> <td align="left">KoreaK14JB01</td> <td align="left">Korea</td> <td align="left">2014</td> <td align="left">KJ623926</td> </tr> <tr> <td align="left">USAIllinois972013</td> <td align="left">Illinois, USA</td> <td align="left">2013</td> <td align="left">KJ645689</td> </tr> <tr> <td align="left">USAKansas1252014</td> <td align="left">Kansas, USA</td> <td align="left">2014</td> <td align="left">KJ645701</td> </tr> <tr> <td align="left">SeCoV/Italy/213306/2009</td> <td align="left">Italy</td> <td align="left">2009</td> <td align="left">KR061459</td> </tr> </tbody> </table>  <p /> </div> <div class="recombinationanalysis" title="sec"> <div class="title" tagx="title" title="title">
Recombination analysis</div> <p id="Par8">Comparison and analysis of the whole PEDV genome sequences and the reference PEDV strains were conducted using MegAlign programs (DNAStar, Madison, WI).</p> <p id="Par9">The putative recombinant sequence of these strains in this study and its parental strains were conducted using recombination detection program (RDP) version 4.97 with seven recombination detection methods (RDP, GENECONV, Chimaera, MaxChi, BootScan, SiScan, and 3Seq). In order to improve the reliability of detection, the recombination breakpoints through at least 5 of these methods was taken as confirmatory for any putative recombination event (<i>p</i> value is 0.01). The phylogenetic tree of recombination and non-recombination were constructed to verify the reliability of these results.</p> </div> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="virusisolationandidentification" title="sec"> <div class="title" tagx="title" title="title">
Virus isolation and identification</div> <p id="Par10">Vero E6 cells were used for viral multiplication. Cytopathic effects (CPE) appeared 48 h after inoculating virus, and more than 70% CPE with cell clustering, detachment, and syncytium formation were observed at 72 h. These results indicated that the isolated PEDV could cause pathological changes in Vero E6 cells (Fig. <a href="#Fig1">1</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <p>Pathogenicity experiment of PEDV-infected Vero cells in different periods. Cytopathic effects appeared 48 h after the initial infection, and more than 50% cytopathic effects were observed at 72 h. No cytopathic effects occurred in the control wells</p>   </div> <p /> </div> <div class="sequenceanalysisofthewholegenome" title="sec"> <div class="title" tagx="title" title="title">
Sequence analysis of the whole genome</div> <p id="Par11">The organization of the genome of PEDV was characteristic of the gene order 5′-ORF1a/1b-S-ORF3-E-M-N-3′ [<a href="#CR2">2</a>, <a href="#CR4">4</a>, <a href="#CR17">17</a>, <a href="#CR18">18</a>]. Sequencing of the complete gene of the four isolates (GenBank numbers: MG546687-MG546690) yielded sequences of 28033–28038 bp, which compared with 18 typical strains available in GenBank (Table <a href="#Tab2">2</a>). The phylogenetic tree showed that these 22 PEDV strains divided into two groups (Fig. <a href="#Fig2">2</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <p>Phylogenetic analysis of the PEDV nucleotide sequences of 22 PEDV isolates, including the reference strains. The trees were constructed by the neighbor-joining method in MEGA7 software. Bootstrap values were indicated for each node from 1000 replicates. The names of the strains, years and places of isolation, as well as GenBank accession numbers proposed, are shown in Table <a href="#Tab2">2</a>. The strains in this study are indicated by underline</p>   </div> <p /> </div> <div class="recombinationanalysisofpedv" title="sec"> <div class="title" tagx="title" title="title">
Recombination analysis of PEDV</div> <p id="Par12">The sequence identity between PEDV strains in this studies and typical strains in terms of their according genes or regions were calculated. The highest percentage of nucleotide identities of 5′-UTR, ORF1a/1b, S, M, N, E, ORF3, and 3′-UTR of PEDV compared with other PEDV strains were show in Table <a href="#Tab3">3</a>. For ORF1a/1b gene, these strains are most similar to these strains, LZW, SHQPYM2013, AJ1102, CH/YNKM-8/2013, CH/JX-1/2013, USAKansas1252014, and CHZMDZY11, respectively. For S gene, these strains are most similar to these strains LZW, KoreaK14JB01, USAIllinois972013, USAKansas1252014, and CH/GDZQ/2014, respectively. The percentage of nucleotide identities of S of PEDV strains in this study compared with recombined strains SeCoV is 90.4–90.7%. For M gene, these strains are most similar to these strains USAIllinois972013 and USAKansas1252014. For N gene, these strains are most similar to these strains LZW and CHYJ130330. For E gene, these strains are most similar to these strains LZW, CH/JX-1/2013, and CH/YNKM-8/2013. For ORF3 gene, these strains are most similar to these strains LZW, CH/JX-1/2013, USAIllinois972013, USAKansas1252014, and CH/GDZQ/2014, respectively (Table <a href="#Tab3">3</a>). <div class="table-wrap_UNKNOWN" id="Tab3"><span class="label" tagx="label" title="label">Table 3</span> </div></p><p>Percentage of sequence identities of the complete genome and different regions of PEDV strains compared with those of other PEDV strains</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left">Strain</th> <th align="left">5′-UTR</th> <th align="left">ORF1a/1b</th> <th align="left">S</th> <th align="left">M</th> <th align="left">N</th> <th align="left">E</th> <th align="left">ORF3</th> <th align="left">3′-UTR</th> </tr> </thead> <tbody> <tr> <td align="left" colspan="9">CH/BJ9/2015</td> </tr> <tr> <td align="left"> Strain</td> <td align="left">CH/JX-1/2013</td> <td align="left">LZW</td> <td align="left">LZW</td> <td align="left">USAIllinois972013, USAKansas1252014</td> <td align="left">LZW</td> <td align="left">CH/JX-1/2013, CH/YNKM-8/2013, LZW</td> <td align="left">LZW</td> <td align="left">CH/JX-1/2013, CHYJ130330</td> </tr> <tr> <td align="left"> Percent<sup>a</sup></td> <td align="left">99.3</td> <td align="left">99.7</td> <td align="left">99.2</td> <td align="left">99.7</td> <td align="left">99.4</td> <td align="left">99.1</td> <td align="left">100</td> <td align="left">99.4</td> </tr> <tr> <td align="left" colspan="9">CH/BJ2/2016</td> </tr> <tr> <td align="left"> Strain</td> <td align="left">AJ1102, CH/YNKM-8/2013, LC</td> <td align="left">SHQPYM2013</td> <td align="left">KoreaK14JB01, USAIllinois972013, USAKansas1252014</td> <td align="left">USAIllinois972013, USAKansas1252014</td> <td align="left">CHYJ130330</td> <td align="left">CH/JX-1/2013, CH/YNKM-8/2013, LZW</td> <td align="left">CH/JX-1/2013, USAIllinois972013, USAKansas1252014</td> <td align="left">CH/JX-1/2013, CHYJ130330</td> </tr> <tr> <td align="left"> Percent<sup>a</sup></td> <td align="left">98.6</td> <td align="left">99.2</td> <td align="left">98.9</td> <td align="left">99.7</td> <td align="left">98.9</td> <td align="left">99.6</td> <td align="left">99.7</td> <td align="left">99.4</td> </tr> <tr> <td align="left" colspan="9">CH/BJ11/2016</td> </tr> <tr> <td align="left"> Strain</td> <td align="left">AJ1102, CH/YNKM-8/2013, LC</td> <td align="left">AJ1102, CH/YNKM-8/2013, CH/JX-1/2013, LZW, SHQPYM2013, USAKansas1252014</td> <td align="left">KoreaK14JB01, USAIllinois972013, USAKansas1252014</td> <td align="left">USAIllinois972013, USAKansas1252014</td> <td align="left">LZW</td> <td align="left">CH/JX-1/2013, CH/YNKM-8/2013, LZW</td> <td align="left">CH/JX-1/2013, USAIllinois972013, USAKansas1252014</td> <td align="left">CH/JX-1/2013, CHYJ130330</td> </tr> <tr> <td align="left"> Percent<sup>a</sup></td> <td align="left">98.6</td> <td align="left">99.1</td> <td align="left">98.8</td> <td align="left">99.7</td> <td align="left">99.2</td> <td align="left">99.6</td> <td align="left">99.7</td> <td align="left">99.4</td> </tr> <tr> <td align="left" colspan="9">CH/HB9/2016</td> </tr> <tr> <td align="left"> Strain</td> <td align="left">ZJCZ</td> <td align="left">CHZMDZY11</td> <td align="left">CH/GDZQ/2014</td> <td align="left">USAIllinois972013, USAKansas1252014</td> <td align="left">LZW</td> <td align="left">CH/JX-1/2013, CH/YNKM-8/2013, LZW</td> <td align="left">CH/GDZQ/2014</td> <td align="left">CH/JX-1/2013, CHYJ130330</td> </tr> <tr> <td align="left"> Percent<sup>a</sup></td> <td align="left">99.7</td> <td align="left">99.2</td> <td align="left">98.9</td> <td align="left">99.7</td> <td align="left">99.2</td> <td align="left">99.1</td> <td align="left">99.6</td> <td align="left">99.4</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p><sup>a</sup>The highest nucleotide precent identities of different regions</p> </div>  <p /> <p id="Par13">Frequent mutation and recombination of genes may occur in epidemics of PEDV. In order to detect these possible recombination events, 19 reference strains were used as parent strains for RDP, GENECONV, Chimaera, MaxChi, BootScan, SiScan, and 3Seq program analysis. The results showed that PEDV in this study were possibly recombinant strain formed by parent strains USAIllinois972013, KoreaK14JB01, CHYJ130330 and CHZMDZY11 (Fig. <a href="#Fig3">3</a>). USAIllinois972013 is major parent of PEDV strain CH/BJ9/2015, the recombination region locates on 4669-8176nt; KoreaK14JB01 is minor parent of PEDV strains CH/BJ2/2016 and CH/BJ11/2016, the recombination region locates on 21744-27721nt; CHYJ130330 is major parent of PEDV strain CH/HB9/2016 and CHZMDZY11 is minor parent of CH/HB9/2016, their recombination region locates on 2218-15942nt. However, there may be a deviation in the prediction of software. It requires further confirms these recombination events. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3</span></a></div>   <p>Results of the recombination analysis of PEDV strains. The putative recombinant sequence of PEDV strains and its parent strains were conducted using recombination detection program version 4.97 (RDP) with seven recombination detection methods (RDP, GENECONV, Chimaera, MaxChi, BootScan, SiScan, and 3Seq, <i>p</i> value is 0.01)</p>   </div> <p /> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="Par14">Since 2011, a large number of piglets died as a result of PEDV outbreaks, causing much economic loss in China [<a href="#CR2">2</a>–<a href="#CR4">4</a>, <a href="#CR7">7</a>–<a href="#CR10">10</a>, <a href="#CR18">18</a>–<a href="#CR21">21</a>]. In this study, the deceased piglets that provided intestine samples had been immunized with the CV777 vaccine previously. This showed that the classic vaccine strain exhibited lower protective efficacy against the variant PEDV [<a href="#CR22">22</a>, <a href="#CR23">23</a>]. The phylogenetic trees showed that, these PEDV isolates from the Beijing area belonged to group II, while the traditional vaccine strain, CV777, belonged to group I based on the comparison of several important genes in PEDV, such as ORF1a/1b, S, M, N, E, and ORF3. However, N, as a conserved gene, was inconsistent with the above results, which also suggested that the N sequence or protein is suitable for universal detection. The ORF1a/1b gene were two long ORFs which encoding the non-structural replicase polyproteins of PEDV. The strain from 2015 in the Beijing area shared the greatest homology in ORF1a/1b with LZW, which was also isolated in Beijing; it showed the continuity of heredity.</p> <p id="Par15">The S sequence of isolates in 2015 in the Beijing area showed a closer relationship with a previous isolate, LZW. Other isolates in 2016 in the Beijing area showed a closer relationship with previous isolates in Guangdong. This revealed that S genes accumulated variations faster based on inherent genetic characteristics or recombinant genes from other isolates [<a href="#CR10">10</a>, <a href="#CR20">20</a>, <a href="#CR24">24</a>].</p> <p id="Par16">The 1–1180 bp section in the S1 region of PEDV genes is prone to mutate [<a href="#CR24">24</a>]. Sequence analysis revealed that isolated PEDV strains and these variant PEDV strains were reported had insertion in 167–178 sites (GTGAAAACCAGG) and 421–423 sites (TGA), also deletion in 218 (A), 471–474 (ATAT) and 481–482 (TG) comparing with CV777. These insertion, deletion, or introducing stop codon (TGA) of sequence resulted in frameshift mutation and abnormally transcripts of S1 gene. It is a reason for failed vaccination with CV777 vaccine. However, different from other recent isolates, these strains in the Beijing area had shared sites with CV777 in the S1 region, such as A5G (this specific nucleotide change led to the amino acid change K–R), C70T, and T1043C (which led to the amino acid change F–S). This determined the genetic correlation between these strains in the Beijing area and the CV777 strain. It was remarkable that E sequences of isolates in the Beijing area had distinct differences from other isolates. There were unique variations in the E gene in these strains, such as T27G. This showed that these isolates from the Beijing area had unique mutations. This might provide a gene reference for other mutations of PEDV in the future.</p> <p id="Par17">Viral gene recombination and mutation results in evolution [<a href="#CR25">25</a>]. Gene recombination and mutation often occurs in coronavirus, such as PEDV, SARSCoV, and infectious bronchitis virus (IBV) [<a href="#CR26">26</a>–<a href="#CR29">29</a>]. There is evidence that the S gene of epidemic PEDV strains and genome of transmissible gastroenteritis virus (TEGV) strains have recombined when both were present in the same host to form a new recombinant porcine coronavirus, SeCoV [<a href="#CR30">30</a>, <a href="#CR31">31</a>]. This S sequence of SeCoV strain share above 90% homology with S sequence of PEDV strains in this study. However, analysis of the recombination of PEDV strains showed that, no recombination event was occurred between SeCoV strain and PEDV strains in this study. These PEDV strains isolated in Beijing area shows recombination events with PEDV isolated in USA, PEDV isolated in Korea, and PEDV isolated in China in sequence. It suggests the accelerated spread of PEDV virus.</p> <p id="Par18">Viral recombination and mutation causes immune failure. The new and effective vaccine is critical to control the occurrence of PEDV. The study of the effective vaccine must be based on the characteristic analysis of variant viruses [<a href="#CR32">32</a>]. The new data on the evolutionary analysis of PEDV is provided in this study. These data will be helpful for research on PEDV vaccine and control.</p> <p id="Par19">Several PEDV strains were isolated from the intestinal tracts of diarrheic piglets from the Beijing area, China. These PEDV strains has been genetic recombination and mutations. The variant strains characterized in this study help to the evolutionary analysis of PEDV.</p> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisher's Note</b></p> <p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>The authors would like to thank the pig farm owners their collaboration.</p> </div> <div class="title" tagx="title" title="title">
Funding</div> <p>This work was supported by the National Natural Science Foundation of China (No. 31572499). The funders did not play any role in the design, conclusions or interpretation of the study.</p> <div class="title" tagx="title" title="title">
Conflict of interest</div> <p id="Par20">The authors declare that they have no competing interests.</p> <div class="title" tagx="title" title="title">
Ethics approval and consent to participate</div> <p id="Par23">Experimental animals used in the study were cared for in accordance with the internationally accepted standards for the care and handling of laboratory animals. All procedures were approved by the Animal Care and Use Committee of Beijing University of Agriculture (Beijing, China). The Animal Ethics Committee approval number was BUAACUC2016-1. All samples were collected after informed written consent was obtained from the owners.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pensaert</span><span class="given-names" tagx="given-names" title="given-names">MB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bouck</span><span class="given-names" tagx="given-names" title="given-names">PD</span></span></span><span class="mixed-article-title" title="mixed-article-title">A new coronavirus-like particle associated with diarrhea in swine</span><span class="source" tagx="source" title="source">Adv Virol</span><span class="year" tagx="year" title="year">1978</span><span class="volume" tagx="volume" title="volume">58</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">243</span><span class="lpage" tagx="lpage" title="lpage">247</span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">ZL</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular characterization and phylogenetic analysis of porcine epidemic diarrhea virus (PEDV) field strains in south China</span><span class="source" tagx="source" title="source">Virus Genes</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">45</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">181</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11262-012-0735-8">10.1007/s11262-012-0735-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22528639">22528639</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Sequence and phylogenetic analysis of nucleocapsid genes of porcine epidemic diarrhea virus (PEDV) strains in China</span><span class="source" tagx="source" title="source">Adv Virol</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">158</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">1267</span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">DQ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Whole-genome analysis of porcine epidemic diarrhea virus (PEDV) from eastern China</span><span class="source" tagx="source" title="source">Adv Virol</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">159</span><span class="issue" tagx="issue" title="issue">10</span><span class="fpage" tagx="fpage" title="fpage">2777</span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takahashi</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Okada</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ohshima</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="mixed-article-title" title="mixed-article-title">An outbreak of swine diarrhea of a new-type associated with coronavirus-like particles in Japan</span><span class="source" tagx="source" title="source">Nihon Juigaku Zasshi</span><span class="year" tagx="year" title="year">1983</span><span class="volume" tagx="volume" title="volume">45</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">829</span><span class="lpage" tagx="lpage" title="lpage">832</span><span class="pub-id"><a href="https://dx.doi.org/10.1292/jvms1939.45.829">10.1292/jvms1939.45.829</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6323804">6323804</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="mixed-article-title" title="mixed-article-title">Porcine epidemic diarrhoea virus: a comprehensive review of molecular epidemiology, diagnosis, and vaccines</span><span class="source" tagx="source" title="source">Virus Genes</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">44</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">167</span><span class="lpage" tagx="lpage" title="lpage">175</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11262-012-0713-1">10.1007/s11262-012-0713-1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22270324">22270324</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiao</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Porcine epidemic diarrhea in China</span><span class="source" tagx="source" title="source">Virus Res</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">226</span><span class="fpage" tagx="fpage" title="fpage">7</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virusres.2016.05.026">10.1016/j.virusres.2016.05.026</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27261169">27261169</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pan</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Isolation and characterization of a variant porcine epidemic diarrhea virus in China</span><span class="source" tagx="source" title="source">Virology Journal</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">9</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">195</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1743-422X-9-195">10.1186/1743-422X-9-195</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22967434">22967434</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Genetic variability and phylogeny of current chinese porcine epidemic diarrhea virus strains based on spike, ORF3, and membrane genes</span><span class="source" tagx="source" title="source">Sci World J</span><span class="year" tagx="year" title="year">2014</span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hao</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Bioinformatics insight into the spike glycoprotein gene of field porcine epidemic diarrhea strains during 2011–2013 in Guangdong, China</span><span class="source" tagx="source" title="source">Virus Genes</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">49</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">58</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11262-014-1055-y">10.1007/s11262-014-1055-y</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24771495">24771495</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Byrum</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">New variant of porcine epidemic diarrhea virus, United States, 2014</span><span class="source" tagx="source" title="source">Emerg Infect Dis</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">20</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">917</span><span class="lpage" tagx="lpage" title="lpage">919</span><span class="pub-id"><a href="https://dx.doi.org/10.3201/eid2005.140195">10.3201/eid2005.140195</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24750580">24750580</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Horie</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Isolation and molecular characterization of porcine epidemic diarrhea viruses collected in Japan in 2014</span><span class="source" tagx="source" title="source">Adv Virol</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">161</span><span class="issue" tagx="issue" title="issue">8</span><span class="fpage" tagx="fpage" title="fpage">2189</span><span class="lpage" tagx="lpage" title="lpage">2195</span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chung</span><span class="given-names" tagx="given-names" title="given-names">HC</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">New emergence pattern with variant porcine epidemic diarrhea viruses, South Korea, 2012–2015</span><span class="source" tagx="source" title="source">Virus Res</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">226</span><span class="fpage" tagx="fpage" title="fpage">14</span><span class="lpage" tagx="lpage" title="lpage">19</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virusres.2016.06.013">10.1016/j.virusres.2016.06.013</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27345861">27345861</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="mixed-article-title" title="mixed-article-title">Outbreak-related porcine epidemic diarrhea virus strains similar to US strains, South Korea, 2013</span><span class="source" tagx="source" title="source">Emerg Infect Dis</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">20</span><span class="issue" tagx="issue" title="issue">7</span><span class="fpage" tagx="fpage" title="fpage">1223</span><span class="pub-id"><a href="https://dx.doi.org/10.3201/eid2007.140294">10.3201/eid2007.140294</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24960370">24960370</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pensaert</span><span class="given-names" tagx="given-names" title="given-names">MB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martelli</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="mixed-article-title" title="mixed-article-title">Porcine epidemic diarrhea: a retrospect from Europe and matters of debate</span><span class="source" tagx="source" title="source">Virus Res</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">226</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">6</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virusres.2016.05.030">10.1016/j.virusres.2016.05.030</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27317168">27317168</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oka</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Cell culture isolation and sequence analysis of genetically diverse US porcine epidemic diarrhea virus strains including a novel strain with a large deletion in the spike gene</span><span class="source" tagx="source" title="source">Vet Microbiol</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">173</span><span class="issue" tagx="issue" title="issue">3–4</span><span class="fpage" tagx="fpage" title="fpage">258</span><span class="lpage" tagx="lpage" title="lpage">269</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.vetmic.2014.08.012">10.1016/j.vetmic.2014.08.012</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25217400">25217400</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sj</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular characterization and phylogenetic analysis of porcine epidemic diarrhea virus (PEDV) field isolates in Korea</span><span class="source" tagx="source" title="source">Adv Virol</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">156</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">577</span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Isolation, molecular characterization and an artificial infection model for a variant porcine epidemic diarrhea virus strain from Jiangsu Province, China</span><span class="source" tagx="source" title="source">Adv Virol</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">8</span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sung</span><span class="given-names" tagx="given-names" title="given-names">MH</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Evolutionary characterization of the emerging porcine epidemic diarrhea virus worldwide and 2014 epidemic in Taiwan</span><span class="source" tagx="source" title="source">Infect Genet Evol</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">36</span><span class="fpage" tagx="fpage" title="fpage">108</span><span class="lpage" tagx="lpage" title="lpage">115</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.meegid.2015.09.011">10.1016/j.meegid.2015.09.011</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26375730">26375730</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Detection and phylogenetic analysis of porcine epidemic diarrhea virus in central China based on the ORF3 gene and the S1 gene</span><span class="source" tagx="source" title="source">Virol J</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">13</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">192</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12985-016-0646-8">10.1186/s12985-016-0646-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27887624">27887624</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fan</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Characterization of Chinese porcine epidemic diarrhea virus with novel insertions and deletions in genome</span><span class="source" tagx="source" title="source">Sci Rep</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">44209</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/srep44209">10.1038/srep44209</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28276526">28276526</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moon</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kang</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="mixed-article-title" title="mixed-article-title">Porcine epidemic diarrhea: a review of current epidemiology and available vaccines</span><span class="source" tagx="source" title="source">Clin Exp Vaccine Res</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">4</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">166</span><span class="lpage" tagx="lpage" title="lpage">176</span><span class="pub-id"><a href="https://dx.doi.org/10.7774/cevr.2015.4.2.166">10.7774/cevr.2015.4.2.166</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26273575">26273575</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Annamalai</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Cross protective immune responses in nursing piglets infected with a US spike-insertion deletion porcine epidemic diarrhea virus strain and challenged with an original US PEDV strain</span><span class="source" tagx="source" title="source">Vet Res</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">48</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">61</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s13567-017-0469-7">10.1186/s13567-017-0469-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28985754">28985754</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niraj</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">S1 domain of the porcine epidemic diarrhea virus spike protein as a vaccine antigen</span><span class="source" tagx="source" title="source">Virol J</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">13</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">10</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12985-015-0456-4">10.1186/s12985-015-0456-4</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26728778">26728778</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Epidemiology, genetic recombination, and pathogenesis of coronaviruses</span><span class="source" tagx="source" title="source">Trends Microbiol</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">24</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">490</span><span class="lpage" tagx="lpage" title="lpage">502</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tim.2016.03.003">10.1016/j.tim.2016.03.003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27012512">27012512</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thor</span><span class="given-names" tagx="given-names" title="given-names">SW</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Recombination in avian gamma-coronavirus infectious bronchitis virus</span><span class="source" tagx="source" title="source">Viruses</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">3</span><span class="issue" tagx="issue" title="issue">9</span><span class="fpage" tagx="fpage" title="fpage">1777</span><span class="lpage" tagx="lpage" title="lpage">1799</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/v3091777">10.3390/v3091777</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21994806">21994806</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Evolution of infectious bronchitis virus in China over the past two decades</span><span class="source" tagx="source" title="source">J Gen Virol</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">97</span><span class="issue" tagx="issue" title="issue">7</span><span class="fpage" tagx="fpage" title="fpage">1566</span><span class="lpage" tagx="lpage" title="lpage">1574</span><span class="pub-id"><a href="https://dx.doi.org/10.1099/jgv.0.000464">10.1099/jgv.0.000464</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27008625">27008625</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Evolution and variation of the SARS-CoV genome</span><span class="source" tagx="source" title="source">Genom Proteom Bioinform</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">1</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">216</span><span class="lpage" tagx="lpage" title="lpage">225</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S1672-0229(03)01027-1">10.1016/S1672-0229(03)01027-1</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Viral evolution and the emergence of SARS coronavirus</span><span class="source" tagx="source" title="source">Philos Trans R Soc Lond B Biol Sci</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">359</span><span class="issue" tagx="issue" title="issue">1447</span><span class="fpage" tagx="fpage" title="fpage">1059</span><span class="lpage" tagx="lpage" title="lpage">1065</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rstb.2004.1478">10.1098/rstb.2004.1478</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15306390">15306390</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boniotti</span><span class="given-names" tagx="given-names" title="given-names">MB</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Porcine epidemic diarrhea virus and discovery of a recombinant swine enteric coronavirus, Italy</span><span class="source" tagx="source" title="source">Emerg Infect Dis</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">22</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">83</span><span class="lpage" tagx="lpage" title="lpage">87</span><span class="pub-id"><a href="https://dx.doi.org/10.3201/eid2201.150544">10.3201/eid2201.150544</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26689738">26689738</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belsham</span><span class="given-names" tagx="given-names" title="given-names">GJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Characterization of a novel chimeric swine enteric coronavirus from diseased pigs in central eastern Europe in 2016</span><span class="source" tagx="source" title="source">Transbound Emerg Dis</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">63</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">595</span><span class="lpage" tagx="lpage" title="lpage">601</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/tbed.12579">10.1111/tbed.12579</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27619563">27619563</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graham</span><span class="given-names" tagx="given-names" title="given-names">RL</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Evaluation of a recombination-resistant coronavirus as a broadly applicable, rapidly implementable vaccine platform</span><span class="source" tagx="source" title="source">Commun Biol</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">179</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s42003-018-0175-7">10.1038/s42003-018-0175-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30393776">30393776</a></span></span></li> </ul> </div> </div>  </body></html>