<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/04/03/2020.03.13.990598.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.13.990598v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="When Darkness Becomes a Ray of Light in the Dark Times: Understanding the COVID-19 via the Comparative Analysis of the Dark Proteomes of SARS-CoV-2, Human SARS and Bat SARS-Like Coronaviruses" />
<meta name="DC.Identifier" content="10.1101/2020.03.13.990598" />
<meta name="DC.Date" content="2020-04-03" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Recently emerged coronavirus designated as SARS-CoV-2 (also known as 2019 novel coronavirus (2019-nCoV) or Wuhan coronavirus) is a causative agent of coronavirus disease 2019 (COVID-19), which is rapidly spreading throughout the world now. More than 9,00,000 cases of SARS-CoV-2 infection and more than 47,000 COVID-19-associated mortalities have been reported worldwide till the writing of this article, and these numbers are increasing every passing hour. World Health Organization (WHO) has declared the SARS-CoV-2 spread as a global public health emergency and admitted that the COVID-19 is a pandemic now. The multiple sequence alignment data correlated with the already published reports on the SARS-CoV-2 evolution and indicated that this virus is closely related to the bat Severe Acute Respiratory Syndrome-like coronavirus (bat SARS-like CoV) and the well-studied Human SARS coronavirus (SARS CoV). The disordered regions in viral proteins are associated with the viral infectivity and pathogenicity. Therefore, in this study, we have exploited a set of complementary computational approaches to examine the dark proteomes of SARS-CoV-2, bat SARS-like, and human SARS CoVs by analysing the prevalence of intrinsic disorder in their proteins. According to our findings, SARS-CoV-2 proteome contains very significant levels of structural order. In fact, except for Nucleocapsid, Nsp8, and ORF6, the vast majority of SARS-CoV-2 proteins are mostly ordered proteins containing less intrinsically disordered protein regions (IDPRs). However, IDPRs found in SARS-CoV-2 proteins are functionally important. For example, cleavage sites in its replicase 1ab polyprotein are found to be highly disordered, and almost all SARS-CoV-2 proteins were shown to contain molecular recognition features (MoRFs), which are intrinsic disorder-based protein-protein interaction sites that are commonly utilized by proteins for interaction with specific partners. The results of our extensive investigation of the dark side of the SARS-CoV-2 proteome will have important implications for the structural and non-structural biology of SARS or SARS-like coronaviruses.

Significance The infection caused by a novel coronavirus (SARS-CoV-2) that causes severe respiratory disease with pneumonia-like symptoms in humans is responsible for the current COVID-19 pandemic. No in-depth information on structures and functions of SARS-CoV-2 proteins is currently available in the public domain, and no effective anti-viral drugs and/or vaccines are designed for the treatment of this infection. Our study provides the first comparative analysis of the order- and disorder-based features of the SARS-CoV-2 proteome relative to human SARS and bat CoV that may be useful for structure-based drug discovery.

*   ACE2
    :   Angiotensin-converting enzyme 2
    CDF
    :   Cumulative distribution function
    CH
    :   Charge hydropathy
    COVID-19
    :   Coronavirus disease 2019
    CTD
    :   Cterminal domain
    DMVs
    :   Double-membrane vesicles
    ICTV
    :   International committee on taxonomy of viruses
    IDP
    :   Intrinsically disordered proteins
    IDPRs
    :   Intrinsically disordered protein regions
    IFN
    :   Interferon
    MoRFs
    :   Molecular recognition features
    MSA
    :   Multiple sequence alignment
    Nsps
    :   Non-structural proteins
    NTD
    :   N-terminal domain
    PONDR
    :   Predictor of natural disordered regions
    PPID
    :   Predicted percentage of intrinsic disorder
    Pprint
    :   Prediction of Protein RNA-Interaction
    RBD
    :   Receptor binding domain
    SARS
    :   Severe acute respiratory syndrome
    TRS
    :   Transcriptional regulatory sequences
    VLPs
    :   Virus-like particles
    WHO
    :   World health organization" />
<meta name="DC.Contributor" content="Rajanish Giri" />
<meta name="DC.Contributor" content="Taniya Bhardwaj" />
<meta name="DC.Contributor" content="Meenakshi Shegane" />
<meta name="DC.Contributor" content="Bhuvaneshwari R. Gehi" />
<meta name="DC.Contributor" content="Prateek Kumar" />
<meta name="DC.Contributor" content="Kundlik Gadhave" />
<meta name="DC.Contributor" content="Christopher J. Oldfield" />
<meta name="DC.Contributor" content="Vladimir N. Uversky" />
<meta name="article:published_time" content="2020-04-03" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="When Darkness Becomes a Ray of Light in the Dark Times: Understanding the COVID-19 via the Comparative Analysis of the Dark Proteomes of SARS-CoV-2, Human SARS and Bat SARS-Like Coronaviruses" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Recently emerged coronavirus designated as SARS-CoV-2 (also known as 2019 novel coronavirus (2019-nCoV) or Wuhan coronavirus) is a causative agent of coronavirus disease 2019 (COVID-19), which is rapidly spreading throughout the world now. More than 9,00,000 cases of SARS-CoV-2 infection and more than 47,000 COVID-19-associated mortalities have been reported worldwide till the writing of this article, and these numbers are increasing every passing hour. World Health Organization (WHO) has declared the SARS-CoV-2 spread as a global public health emergency and admitted that the COVID-19 is a pandemic now. The multiple sequence alignment data correlated with the already published reports on the SARS-CoV-2 evolution and indicated that this virus is closely related to the bat Severe Acute Respiratory Syndrome-like coronavirus (bat SARS-like CoV) and the well-studied Human SARS coronavirus (SARS CoV). The disordered regions in viral proteins are associated with the viral infectivity and pathogenicity. Therefore, in this study, we have exploited a set of complementary computational approaches to examine the dark proteomes of SARS-CoV-2, bat SARS-like, and human SARS CoVs by analysing the prevalence of intrinsic disorder in their proteins. According to our findings, SARS-CoV-2 proteome contains very significant levels of structural order. In fact, except for Nucleocapsid, Nsp8, and ORF6, the vast majority of SARS-CoV-2 proteins are mostly ordered proteins containing less intrinsically disordered protein regions (IDPRs). However, IDPRs found in SARS-CoV-2 proteins are functionally important. For example, cleavage sites in its replicase 1ab polyprotein are found to be highly disordered, and almost all SARS-CoV-2 proteins were shown to contain molecular recognition features (MoRFs), which are intrinsic disorder-based protein-protein interaction sites that are commonly utilized by proteins for interaction with specific partners. The results of our extensive investigation of the dark side of the SARS- CoV-2 proteome will have important implications for the structural and non-structural biology of SARS or SARS-like coronaviruses.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.13.990598v2" />
<meta name="citation_id" content="2020.03.13.990598v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.13.990598v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.13.990598v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.13.990598v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.13.990598" />
<meta name="citation_num_pages" content="63" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.13.990598" />
<meta name="citation_author" content="Rajanish Giri" />
<meta name="citation_author_institution" content="Indian Institute of Technology Mandi, School of Basic Sciences, VPO Kamand" />
<meta name="citation_author_email" content="rajanishgiri@iitmandi.ac.in" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2046-836X" />
<meta name="citation_author" content="Taniya Bhardwaj" />
<meta name="citation_author_institution" content="Indian Institute of Technology Mandi, School of Basic Sciences, VPO Kamand" />
<meta name="citation_author" content="Meenakshi Shegane" />
<meta name="citation_author_institution" content="Indian Institute of Technology Mandi, School of Basic Sciences, VPO Kamand" />
<meta name="citation_author" content="Bhuvaneshwari R. Gehi" />
<meta name="citation_author_institution" content="Indian Institute of Technology Mandi, School of Basic Sciences, VPO Kamand" />
<meta name="citation_author" content="Prateek Kumar" />
<meta name="citation_author_institution" content="Indian Institute of Technology Mandi, School of Basic Sciences, VPO Kamand" />
<meta name="citation_author" content="Kundlik Gadhave" />
<meta name="citation_author_institution" content="Indian Institute of Technology Mandi, School of Basic Sciences, VPO Kamand" />
<meta name="citation_author" content="Christopher J. Oldfield" />
<meta name="citation_author_institution" content="Department of Computer Science, Virginia Commonwealth University" />
<meta name="citation_author" content="Vladimir N. Uversky" />
<meta name="citation_author_institution" content="Department of Molecular Medicine and Byrd Alzheimer’s Research Institute, Morsani College of Medicine, University of South Florida" />
<meta name="citation_author_institution" content="Laboratory of New Methods in Biology, Institute for Biological Instrumentation of the Russian Academy of Sciences, Federal Research Center “Pushchino Scientific Center for Biological Research of the Russian Academy of Sciences”" />
<meta name="citation_reference" content="Yang X, Yu Y, Xu J, Shu H, Xia J, Liu H, et al. Clinical course and outcomes of critically ill patients with SARS-CoV-2 pneumonia in Wuhan, China: a single-centered, retrospective, observational study. Lancet Respir Med. 2020 [cited 29 Feb 2020]. doi:10.1016/S2213-2600(20)30079-5" />
<meta name="citation_reference" content="Coronavirus disease 2019. [cited 29 Feb 2020]. Available: https://www.who.int/emergencies/diseases/novel-coronavirus-2019" />
<meta name="citation_reference" content="Gorbalenya AE, Enjuanes L, Ziebuhr J, Snijder EJ. Nidovirales: evolving the largest RNA virus genome. Virus Res. 2006;117: 17–37. doi:10.1016/j.virusres.2006.01.017" />
<meta name="citation_reference" content="Corman VM, Lienau J, Witzenrath M. [Coronaviruses as the cause of respiratory infections]. Internist (Berl). 2019;60: 1136–1145. doi:10.1007/s00108-019-00671-5" />
<meta name="citation_reference" content="Woo PCY, Lau SKP, Lam CSF, Lau CCY, Tsang AKL, Lau JHN, et al. Discovery of Seven Novel Mammalian and Avian Coronaviruses in the Genus Deltacoronavirus Supports Bat Coronaviruses as the Gene Source of Alphacoronavirus and Betacoronavirus and Avian Coronaviruses as the Gene Source of Gammacoronavirus and Deltacoronavi. J Virol. 2012;86: 3995–4008. doi:10.1128/jvi.06540-11" />
<meta name="citation_reference" content="Cotten M, Lam TT, Watson SJ, Palser AL, Petrova V, Grant P, et al. Full-genome deep sequencing and phylogenetic analysis of novel human betacoronavirus. Emerg Infect Dis. 2013;19: 736–742. doi:10.3201/eid1905.130057" />
<meta name="citation_reference" content="Masters PS. The molecular biology of coronaviruses. Adv Virus Res. 2006;66: 193–292. doi:10.1016/S0065-3527(06)66005-3" />
<meta name="citation_reference" content="Hussain S, Pan J, Chen Y, Yang Y, Xu J, Peng Y, et al. Identification of Novel Subgenomic RNAs and Noncanonical Transcription Initiation Signals of Severe Acute Respiratory Syndrome Coronavirus. J Virol. 2005;79: 5288–5295. doi:10.1128/jvi.79.9.5288-5295.2005" />
<meta name="citation_reference" content="Snijder EJ, van der Meer Y, Zevenhoven-Dobbe J, Onderwater JJM, van der Meulen J, Koerten HK, et al. Ultrastructure and Origin of Membrane Vesicles Associated with the Severe Acute Respiratory Syndrome Coronavirus Replication Complex. J Virol. 2006;80: 5927–5940. doi:10.1128/jvi.02501-05" />
<meta name="citation_reference" content="Sawicki SG, Sawicki DL, Siddell SG. A Contemporary View of Coronavirus Transcription. J Virol. 2007;81: 20–29. doi:10.1128/jvi.01358-06" />
<meta name="citation_reference" content="Van Der Lee R, Buljan M, Lang B, Weatheritt RJ, Daughdrill GW, Dunker AK, et al. Classification of intrinsically disordered regions and proteins. Chemical Reviews. American Chemical Society; 2014. pp. 6589–6631. doi:10.1021/cr400525m" />
<meta name="citation_reference" content="Oldfield CJ, Dunker AK. Intrinsically Disordered Proteins and Intrinsically Disordered Protein Regions. Annu Rev Biochem. 2014;83: 553–584. doi:10.1146/annurev-biochem-072711-164947" />
<meta name="citation_reference" content="Wright PE, Dyson HJ. Intrinsically unstructured proteins: Re-assessing the protein structure-function paradigm. J Mol Biol. 1999;293: 321–331. doi:10.1006/jmbi.1999.3110" />
<meta name="citation_reference" content="Dunker AK, Cortese MS, Romero P, Iakoucheva LM, Uversky VN. Flexible nets. The roles of intrinsic disorder in protein interaction networks. FEBS J. 2005;272: 5129–48. doi:10.1111/j.1742-4658.2005.04948.x" />
<meta name="citation_reference" content="Dunker AK, Brown CJ, Obradovic Z. Identification and functions of usefully disordered proteins. Advances in Protein Chemistry. 2002. pp. 25–49. doi:10.1016/S0065-3233(02)62004-2" />
<meta name="citation_reference" content="Dunker AK, Silman I, Uversky VN, Sussman JL. Function and structure of inherently disordered proteins. Current Opinion in Structural Biology. 2008. pp. 756–764. doi:10.1016/j.sbi.2008.10.002" />
<meta name="citation_reference" content="Liu J, Perumal NB, Oldfield CJ, Su EW, Uversky VN, Dunker AK. Intrinsic disorder in transcription factors. Biochemistry. 2006;45: 6873–88. doi:10.1021/bi0602718" />
<meta name="citation_reference" content="Uversky VN, Oldfield CJ, Dunker AK. Showing your ID: intrinsic disorder as an ID for recognition, regulation and cell signaling. J Mol Recognit. 2005;18: 343–84. doi:10.1002/jmr.747" />
<meta name="citation_reference" content="Yan J, Kurgan L. DRNApred, fast sequence-based method that accurately predicts and discriminates DNA- and RNA-binding residues. Nucleic Acids Res. 2017;45: e84. doi:10.1093/nar/gkx059" />
<meta name="citation_reference" content="Peng Z KL. High-throughput prediction of RNA, DNA and protein binding regions mediated by intrinsic disorder. Nucleic Acids Res. 2015;43: e121." />
<meta name="citation_reference" content="Wu F, Zhao S, Yu B, Chen Y-M, Wang W, Song Z-G, et al. A new coronavirus associated with human respiratory disease in China. Nature. 2020. doi:10.1038/s41586-020-2008-3" />
<meta name="citation_reference" content="Giri R, Kumar D, Sharma N, Uversky VN. Intrinsically Disordered Side of the Zika Virus Proteome. Frontiers in cellular and infection microbiology 2016, 6, 144. teome. Front Cell Infect Microbiol. 2016;6: 144. doi:10.3389/fcimb.2016.00144" />
<meta name="citation_reference" content="Xue B, W. Williams R, J. Oldfield C, Kian-Meng Goh G, Keith Dunker A, N. Uversky V. Viral Disorder or Disordered Viruses: Do Viral Proteins Possess Unique Features? Protein Pept Lett. 2010;17: 932–951. doi:10.2174/092986610791498984" />
<meta name="citation_reference" content="Singh A, Kumar A, Yadav R, Uversky VN, Giri R. Deciphering the dark proteome of Chikungunya virus. Sci Rep. 2018;8: 5822. doi:10.1038/s41598-018-23969-0" />
<meta name="citation_reference" content="Ward JJ, Sodhi JS, McGuffin LJ, Buxton BF, Jones DT. Prediction and Functional Analysis of Native Disorder in Proteins from the Three Kingdoms of Life. J Mol Biol. 2004;337: 635–645. doi:10.1016/j.jmb.2004.02.002" />
<meta name="citation_reference" content="Clark K, Karsch-Mizrachi I, Lipman DJ, Ostell J, Sayers EW. GenBank. Nucleic Acids Res. 2016;44: D67–D72. doi:10.1093/nar/gkv1276" />
<meta name="citation_reference" content="Sievers F, Wilm A, Dineen D, Gibson TJ, Karplus K, Li W, et al. Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega. Mol Syst Biol. 2011;7. doi:10.1038/msb.2011.75" />
<meta name="citation_reference" content="Robert X, Gouet P. Deciphering key features in protein structures with the new ENDscript server. Nucleic Acids Res. 2014;42: W320–4. doi:10.1093/nar/gku316" />
<meta name="citation_reference" content="Peng K, Radivojac P, Vucetic S, Dunker AK, Obradovic Z. Length-dependent prediction of protein intrinsic disorder. BMC Bioinformatics. 2006;7: 208. doi:10.1186/1471-2105-7-208" />
<meta name="citation_reference" content="Peng K, Vucetic S, Radivojac P, Brown CJ, Dunker AK, Obradovic Z. Optimizing long intrinsic disorder predictors with protein evolutionary information. J Bioinform Comput Biol. 2005;3: 35–60. doi:10.1142/S0219720005000886" />
<meta name="citation_reference" content="Xue B, Dunbrack RL, Williams RW, Dunker AK, Uversky VN. PONDR-FIT: a meta-predictor of intrinsically disordered amino acids. Biochim Biophys Acta. 2010;1804: 996–1010. doi:10.1016/j.bbapap.2010.01.011" />
<meta name="citation_reference" content="Romero P, Obradovic Z, Li X, Garner EC, Brown CJ, Dunker AK. Sequence complexity of disordered protein. Proteins Struct Funct Genet. 2001;42: 38–48. doi:10.1002/1097-0134(20010101)42:1&lt;38::AID-PROT50&gt;3.0.CO;2-3" />
<meta name="citation_reference" content="Mészáros B, Erdos G, Dosztányi Z. IUPred2A: context-dependent prediction of protein disorder as a function of redox state and protein binding. Nucleic Acids Res. 2018;46: W329–W337. doi:10.1093/nar/gky384" />
<meta name="citation_reference" content="Gadhave K, Gehi BR, Kumar P, Xue B, Uversky VN, Giri R. The dark side of Alzheimer’s disease: unstructured biology of proteins from the amyloid cascade signaling pathway. Cell Mol Life Sci. 2020 [cited 29 Feb 2020]. doi:10.1007/s00018-019-03414-9" />
<meta name="citation_reference" content="Garg N, Kumar P, Gadhave K, Giri R. The dark proteome of cancer: Intrinsic disorderedness and functionality of HIF-1α along with its interacting proteins. Prog Mol Biol Transl Sci. 2019;166: 371–403. doi:10.1016/bs.pmbts.2019.05.006" />
<meta name="citation_reference" content="Uversky VN, Gillespie JR, Fink AL. Why are “natively unfolded” proteins unstructured under physiologic conditions? Proteins Struct Funct Genet. 2000;41: 415–427. doi:10.1002/1097-0134(20001115)41:3&lt;415::AID-PROT130&gt;3.0.CO;2-7" />
<meta name="citation_reference" content="Huang F, Oldfield C, Meng J, Hsu WL, Xue B, Uversky VN, et al. Subclassifying disordered proteins by the CH-CDF plot method. Pacific Symposium on Biocomputing. 2012. pp. 128–139. doi:10.1142/9789814366496_0013" />
<meta name="citation_reference" content="Malhis N, Wong ETC, Nassar R, Gsponer J. Computational Identification of MoRFs in Protein Sequences Using Hierarchical Application of Bayes Rule. PLoS One. 2015;10: e0141603. doi:10.1371/journal.pone.0141603" />
<meta name="citation_reference" content="Mészáros B, Simon I, Dosztányi Z. Prediction of protein binding regions in disordered proteins. PLoS Comput Biol. 2009;5: e1000376. doi:10.1371/journal.pcbi.1000376" />
<meta name="citation_reference" content="Dosztányi Z, Mészáros B, Simon I. ANCHOR: web server for predicting protein binding regions in disordered proteins. Bioinformatics. 2009;25: 2745–6. doi:10.1093/bioinformatics/btp518" />
<meta name="citation_reference" content="Disfani FM, Hsu W-L, Mizianty MJ, Oldfield CJ, Xue B, Dunker AK, et al. MoRFpred, a computational tool for sequence-based prediction and characterization of short disorder-to-order transitioning binding regions in proteins. Bioinformatics. 2012;28: i75–83. doi:10.1093/bioinformatics/bts209" />
<meta name="citation_reference" content="Jones DT, Cozzetto D. DISOPRED3: Precise disordered region predictions with annotated protein-binding activity. Bioinformatics. 2015;31: 857–863. doi:10.1093/bioinformatics/btu744" />
<meta name="citation_reference" content="Peng Z, Wang C, Uversky VN, Kurgan L. Prediction of Disordered RNA, DNA, and Protein Binding Regions Using DisoRDPbind. Methods Mol Biol. 2017;1484: 187–203. doi:10.1007/978-1-4939-6406-2_14" />
<meta name="citation_reference" content="Kumar M, Gromiha MM, Raghava GPS. Prediction of RNA binding sites in a protein using SVM and PSSM profile. Proteins. 2008;71: 189–94. doi:10.1002/prot.21677" />
<meta name="citation_reference" content="Wu A, Peng Y, Huang B, Ding X, Wang X, Niu P, et al. Genome Composition and Divergence of the Novel Coronavirus (2019-nCoV) Originating in China. Cell Host Microbe. 2020 [cited 7 Mar 2020]. doi:10.1016/j.chom.2020.02.001" />
<meta name="citation_reference" content="Rajagopalan K, Mooney SM, Parekh N, Getzenberg RH, Kulkarni P. A majority of the cancer/testis antigens are intrinsically disordered proteins. J Cell Biochem. 2011;112: 3256–67. doi:10.1002/jcb.23252" />
<meta name="citation_reference" content="Mishra PM, Uversky VN, Giri R. Molecular Recognition Features in Zika Virus Proteome. J Mol Biol. 2018;430: 2372–2388. doi:10.1016/j.jmb.2017.10.018" />
<meta name="citation_reference" content="Gypas F, Tsaousis GN, Hamodrakas SJ. mpMoRFsDB: a database of molecular recognition features in membrane proteins. Bioinformatics. 2013;29: 2517–8. doi:10.1093/bioinformatics/btt427" />
<meta name="citation_reference" content="Oldfield CJ, Peng Z, Kurgan L. Disordered RNA-Binding Region Prediction with DisoRDPbind. Methods Mol Biol. 2020;2106: 225–239. doi:10.1007/978-1-0716-0231-7_14" />
<meta name="citation_reference" content="Cavanagh D, Davis PJ. Coronavirus IBV: removal of spike glycopolypeptide S1 by urea abolishes infectivity and haemagglutination but not attachment to cells. J Gen Virol. 1986;67 (Pt 7): 1443–8. doi:10.1099/0022-1317-67-7-1443" />
<meta name="citation_reference" content="Graham RL, Baric RS. Recombination, Reservoirs, and the Modular Spike: Mechanisms of Coronavirus Cross-Species Transmission. J Virol. 2010;84: 3134–3146. doi:10.1128/jvi.01394-09" />
<meta name="citation_reference" content="Belouzard S, Millet JK, Licitra BN, Whittaker GR. Mechanisms of coronavirus cell entry mediated by the viral spike protein. Viruses. 2012. pp. 1011–1033. doi:10.3390/v4061011" />
<meta name="citation_reference" content="de Haan CAM, te Lintelo E, Li Z, Raaben M, Wurdinger T, Bosch BJ, et al. Cooperative Involvement of the S1 and S2 Subunits of the Murine Coronavirus Spike Protein in Receptor Binding and Extended Host Range. J Virol. 2006;80: 10909–10918. doi:10.1128/jvi.00950-06" />
<meta name="citation_reference" content="McBride CE, Li J, Machamer CE. The Cytoplasmic Tail of the Severe Acute Respiratory Syndrome Coronavirus Spike Protein Contains a Novel Endoplasmic Reticulum Retrieval Signal That Binds COPI and Promotes Interaction with Membrane Protein. J Virol. 2007;81: 2418–2428. doi:10.1128/jvi.02146-06" />
<meta name="citation_reference" content="Li F, Li W, Farzan M, Harrison SC. Structure of SARS coronavirus spike receptor-binding domain complexed with receptor. Science. 2005;309: 1864–8. doi:10.1126/science.1116480" />
<meta name="citation_reference" content="Broer R, Boson B, Spaan W, Cosset F-L, Corver J. Important Role for the Transmembrane Domain of Severe Acute Respiratory Syndrome Coronavirus Spike Protein during Entry. J Virol. 2006;80: 1302–1310. doi:10.1128/jvi.80.3.1302-1310.2006" />
<meta name="citation_reference" content="Song W, Gui M, Wang X, Xiang Y. Cryo-EM structure of the SARS coronavirus spike glycoprotein in complex with its host cell receptor ACE2. PLoS Pathog. 2018;14: e1007236. doi:10.1371/journal.ppat.1007236" />
<meta name="citation_reference" content="Wrapp D, Wang N, Corbett KS, Goldsmith JA, Hsieh C-L, Abiona O, et al. Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation. Science. 2020 [cited 29 Feb 2020]. doi:10.1126/science.abb2507" />
<meta name="citation_reference" content="Ruch TR, Machamer CE. The coronavirus E protein: assembly and beyond. Viruses. 2012;4: 363–82. doi:10.3390/v4030363" />
<meta name="citation_reference" content="Ujike M, Taguchi F. Incorporation of spike and membrane glycoproteins into coronavirus virions. Viruses. MDPI AG; 2015. pp. 1700–1725. doi:10.3390/v7041700" />
<meta name="citation_reference" content="DeDiego ML, Alvarez E, Almazan F, Rejas MT, Lamirande E, Roberts A, et al. A Severe Acute Respiratory Syndrome Coronavirus That Lacks the E Gene Is Attenuated In Vitro and In Vivo. J Virol. 2007;81: 1701–1713. doi:10.1128/jvi.01467-06" />
<meta name="citation_reference" content="Torres J, Wang J, Parthasarathy K, Liu DX. The transmembrane oligomers of coronavirus protein E. Biophys J. 2005;88: 1283–90. doi:10.1529/biophysj.104.051730" />
<meta name="citation_reference" content="Li Y, Surya W, Claudine S, Torres J. Structure of a conserved Golgi complex-targeting signal in coronavirus envelope proteins. J Biol Chem. 2014;289: 12535–49. doi:10.1074/jbc.M114.560094" />
<meta name="citation_reference" content="Wahyu Surya MS and JT. Structural and Functional Aspects of Viroporins in Human Respiratory Viruses: Respiratory Syncytial Virus and Coronaviruses. 2013." />
<meta name="citation_reference" content="Teoh KT, Siu YL, Chan WL, Schlüter MA, Liu CJ, Peiris JSM, et al. The SARS coronavirus E protein interacts with PALS1 and alters tight junction formation and epithelial morphogenesis. Mol Biol Cell. 2010;21: 3838–3852. doi:10.1091/mbc.E10-04-0338" />
<meta name="citation_reference" content="Tseng Y-T, Chang C-H, Wang S-M, Huang K-J, Wang C-T. Identifying SARS-CoV membrane protein amino acid residues linked to virus-like particle assembly. PLoS One. 2013;8: e64013. doi:10.1371/journal.pone.0064013" />
<meta name="citation_reference" content="Tseng Y-T, Wang S-M, Huang K-J, Lee AI-R, Chiang C-C, Wang C-T. Self-assembly of severe acute respiratory syndrome coronavirus membrane protein. J Biol Chem. 2010;285: 12862–72. doi:10.1074/jbc.M109.030270" />
<meta name="citation_reference" content="Corse E, Machamer CE. The cytoplasmic tails of infectious bronchitis virus E and M proteins mediate their interaction. Virology. 2003;312: 25–34. doi:10.1016/S0042-6822(03)00175-2" />
<meta name="citation_reference" content="Narayanan K, Chen C-J, Maeda J, Makino S. Nucleocapsid-independent specific viral RNA packaging via viral envelope protein and viral RNA signal. J Virol. 2003;77: 2922–7. doi:10.1128/jvi.77.5.2922-2927.2003" />
<meta name="citation_reference" content="Nal B, Chan C, Kien F, Siu L, Tse J, Chu K, et al. Differential maturation and subcellular localization of severe acute respiratory syndrome coronavirus surface proteins S, M and E. J Gen Virol. 2005;86: 1423–34. doi:10.1099/vir.0.80671-0" />
<meta name="citation_reference" content="Voss D, Pfefferle S, Drosten C, Stevermann L, Traggiai E, Lanzavecchia A, et al. Studies on membrane topology, N-glycosylation and functionality of SARS-CoV membrane protein. Virol J. 2009;6: 79. doi:10.1186/1743-422X-6-79" />
<meta name="citation_reference" content="Neuman BW, Kiss G, Kunding AH, Bhella D, Baksh MF, Connelly S, et al. A structural analysis of M protein in coronavirus assembly and morphology. J Struct Biol. 2011;174: 11–22. doi:10.1016/j.jsb.2010.11.021" />
<meta name="citation_reference" content="Liu J, Sun Y, Qi J, Chu F, Wu H, Gao F, et al. The membrane protein of severe acute respiratory syndrome coronavirus acts as a dominant immunogen revealed by a clustering region of novel functionally and structurally defined cytotoxic T-lymphocyte epitopes. J Infect Dis. 2010;202: 1171–80. doi:10.1086/656315" />
<meta name="citation_reference" content="Goh GK-M, Dunker AK, Uversky V. Prediction of Intrinsic Disorder in MERS-CoV/HCoV-EMC Supports a High Oral-Fecal Transmission. PLoS Curr. 2013;5. doi:10.1371/currents.outbreaks.22254b58675cdebc256dbe3c5aa6498b" />
<meta name="citation_reference" content="Perrier A, Bonnin A, Desmarets L, Danneels A, Goffard A, Rouillé Y, et al. The C-terminal domain of the MERS coronavirus M protein contains a trans-Golgi network localization signal. J Biol Chem. 2019;294: 14406–14421. doi:10.1074/jbc.RA119.008964" />
<meta name="citation_reference" content="McBride R, van Zyl M, Fielding BC. The coronavirus nucleocapsid is a multifunctional protein. Viruses. MDPI AG; 2014. pp. 2991–3018. doi:10.3390/v6082991" />
<meta name="citation_reference" content="Saikatendu KS, Joseph JS, Subramanian V, Neuman BW, Buchmeier MJ, Stevens RC, et al. Ribonucleocapsid formation of severe acute respiratory syndrome coronavirus through molecular action of the N-terminal domain of N protein. J Virol. 2007;81: 3913–21. doi:10.1128/JVI.02236-06" />
<meta name="citation_reference" content="Siu YL, Teoh KT, Lo J, Chan CM, Kien F, Escriou N, et al. The M, E, and N Structural Proteins of the Severe Acute Respiratory Syndrome Coronavirus Are Required for Efficient Assembly, Trafficking, and Release of Virus-Like Particles. J Virol. 2008;82: 11318–11330. doi:10.1128/jvi.01052-08" />
<meta name="citation_reference" content="Chang C-K, Hsu Y-L, Chang Y-H, Chao F-A, Wu M-C, Huang Y-S, et al. Multiple Nucleic Acid Binding Sites and Intrinsic Disorder of Severe Acute Respiratory Syndrome Coronavirus Nucleocapsid Protein: Implications for Ribonucleocapsid Protein Packaging. J Virol. 2009;83: 2255–2264. doi:10.1128/jvi.02001-08" />
<meta name="citation_reference" content="Huang Q, Yu L, Petros AM, Gunasekera A, Liu Z, Xu N, et al. Structure of the N-terminal RNA-binding domain of the SARS CoV nucleocapsid protein. Biochemistry. 2004;43: 6059–63. doi:10.1021/bi036155b" />
<meta name="citation_reference" content="Yu I-M, Oldham ML, Zhang J, Chen J. Crystal structure of the severe acute respiratory syndrome (SARS) coronavirus nucleocapsid protein dimerization domain reveals evolutionary linkage between corona- and arteriviridae. J Biol Chem. 2006;281: 17134–9. doi:10.1074/jbc.M602107200" />
<meta name="citation_reference" content="He R, Leeson A, Ballantine M, Andonov A, Baker L, Dobie F, et al. Characterization of protein-protein interactions between the nucleocapsid protein and membrane protein of the SARS coronavirus. Virus Res. 2004;105: 121–5. doi:10.1016/j.virusres.2004.05.002" />
<meta name="citation_reference" content="Luo H, Chen Q, Chen J, Chen K, Shen X, Jiang H. The nucleocapsid protein of SARS coronavirus has a high binding affinity to the human cellular heterogeneous nuclear ribonucleoprotein A1. FEBS Lett. 2005;579: 2623–8. doi:10.1016/j.febslet.2005.03.080" />
<meta name="citation_reference" content="He R, Dobie F, Ballantine M, Leeson A, Li Y, Bastien N, et al. Analysis of multimerization of the SARS coronavirus nucleocapsid protein. Biochem Biophys Res Commun. 2004;316: 476–83. doi:10.1016/j.bbrc.2004.02.074" />
<meta name="citation_reference" content="Nelson GW, Stohlman SA. Localization of the RNA-binding domain of mouse hepatitis virus nucleocapsid protein. J Gen Virol. 1993;74 (Pt 9): 1975–9. doi:10.1099/0022-1317-74-9-1975" />
<meta name="citation_reference" content="Gunasekaran K, Tsai C-J, Nussinov R. Analysis of ordered and disordered protein complexes reveals structural features discriminating between stable and unstable monomers. J Mol Biol. 2004;341: 1327–41. doi:10.1016/j.jmb.2004.07.002" />
<meta name="citation_reference" content="Oldfield CJ, Meng J, Yang JY, Yang MQ, Uversky VN, Dunker AK. Flexible nets: disorder and induced fit in the associations of p53 and 14-3-3 with their partners. BMC Genomics. 2008;9 Suppl 1: S1. doi:10.1186/1471-2164-9-S1-S1" />
<meta name="citation_reference" content="Wu Z, Hu G, Yang J, Peng Z, Uversky VN, Kurgan L. In various protein complexes, disordered protomers have large per-residue surface areas and area of protein-, DNA- and RNA-binding interfaces. FEBS Lett. 2015;589: 2561–9. doi:10.1016/j.febslet.2015.08.014" />
<meta name="citation_reference" content="Narayanan K, Huang C, Makino S. SARS coronavirus accessory proteins. Virus Res. 2008;133: 113–21. doi:10.1016/j.virusres.2007.10.009" />
<meta name="citation_reference" content="Yu C-J, Chen Y-C, Hsiao C-H, Kuo T-C, Chang SC, Lu C-Y, et al. Identification of a novel protein 3a from severe acute respiratory syndrome coronavirus. FEBS Lett. 2004;565: 111–6. doi:10.1016/j.febslet.2004.03.086" />
<meta name="citation_reference" content="Yuan X, Li J, Shan Y, Yang Z, Zhao Z, Chen B, et al. Subcellular localization and membrane association of SARS-CoV 3a protein. Virus Res. 2005;109: 191–202. doi:10.1016/j.virusres.2005.01.001" />
<meta name="citation_reference" content="Tan Y-J, Teng E, Shen S, Tan THP, Goh P-Y, Fielding BC, et al. A Novel Severe Acute Respiratory Syndrome Coronavirus Protein, U274, Is Transported to the Cell Surface and Undergoes Endocytosis. J Virol. 2004;78: 6723–6734. doi:10.1128/jvi.78.13.6723-6734.2004" />
<meta name="citation_reference" content="Lu W, Zheng B-J, Xu K, Schwarz W, Du L, Wong CKL, et al. Severe acute respiratory syndrome-associated coronavirus 3a protein forms an ion channel and modulates virus release. Proc Natl Acad Sci U S A. 2006;103: 12540–5. doi:10.1073/pnas.0605402103" />
<meta name="citation_reference" content="Tan Y-J. The Severe Acute Respiratory Syndrome (SARS)-coronavirus 3a protein may function as a modulator of the trafficking properties of the spike protein. Virol J. 2005;2: 5. doi:10.1186/1743-422X-2-5" />
<meta name="citation_reference" content="McBride R, Fielding BC. The role of severe acute respiratory syndrome (SARS)-coronavirus accessory proteins in virus pathogenesis. Viruses. 2012;4: 2902–23. doi:10.3390/v4112902" />
<meta name="citation_reference" content="Yuan X, Shan Y, Yao Z, Li J, Zhao Z, Chen J, et al. Mitochondrial location of severe acute respiratory syndrome coronavirus 3b protein. Mol Cells. 2006;21: 186–91. Available: http://www.ncbi.nlm.nih.gov/pubmed/16682811" />
<meta name="citation_reference" content="Yuan X, Yao Z, Shan Y, Chen B, Yang Z, Wu J, et al. Nucleolar localization of non-structural protein 3b, a protein specifically encoded by the severe acute respiratory syndrome coronavirus. Virus Res. 2005;114: 70–9. doi:10.1016/j.virusres.2005.06.001" />
<meta name="citation_reference" content="Varshney B, Lal SK. SARS-CoV accessory protein 3b induces AP-1 transcriptional activity through activation of JNK and ERK pathways. Biochemistry. 2011;50: 5419–25. doi:10.1021/bi200303r" />
<meta name="citation_reference" content="Yuan X, Shan Y, Zhao Z, Chen J, Cong Y. G0/G1 arrest and apoptosis induced by SARS-CoV 3b protein in transfected cells. Virol J. 2005;2: 66. doi:10.1186/1743-422X-2-66" />
<meta name="citation_reference" content="Khan S, Fielding BC, Tan THP, Chou C-F, Shen S, Lim SG, et al. Over-expression of severe acute respiratory syndrome coronavirus 3b protein induces both apoptosis and necrosis in Vero E6 cells. Virus Res. 2006;122: 20–7. doi:10.1016/j.virusres.2006.06.005" />
<meta name="citation_reference" content="Law PTW, Wong C-H, Au TCC, Chuck C-P, Kong S-K, Chan PKS, et al. The 3a protein of severe acute respiratory syndrome-associated coronavirus induces apoptosis in Vero E6 cells. J Gen Virol. 2005;86: 1921–30. doi:10.1099/vir.0.80813-0" />
<meta name="citation_reference" content="Kopecky-Bromberg SA, Martinez-Sobrido L, Frieman M, Baric RA, Palese P. Severe Acute Respiratory Syndrome Coronavirus Open Reading Frame (ORF) 3b, ORF 6, and Nucleocapsid Proteins Function as Interferon Antagonists. J Virol. 2007;81: 548–557. doi:10.1128/jvi.01782-06" />
<meta name="citation_reference" content="Frieman M, Yount B, Heise M, Kopecky-Bromberg SA, Palese P, Baric RS. Severe Acute Respiratory Syndrome Coronavirus ORF6 Antagonizes STAT1 Function by Sequestering Nuclear Import Factors on the Rough Endoplasmic Reticulum/Golgi Membrane. J Virol. 2007;81: 9812–9824. doi:10.1128/jvi.01012-07" />
<meta name="citation_reference" content="Netland J, Ferraro D, Pewe L, Olivares H, Gallagher T, Perlman S. Enhancement of Murine Coronavirus Replication by Severe Acute Respiratory Syndrome Coronavirus Protein 6 Requires the N-Terminal Hydrophobic Region but Not C-Terminal Sorting Motifs. J Virol. 2007;81: 11520–11525. doi:10.1128/jvi.01308-07" />
<meta name="citation_reference" content="Gunalan V, Mirazimi A, Tan Y-J. A putative diacidic motif in the SARS-CoV ORF6 protein influences its subcellular localization and suppression of expression of co-transfected expression constructs. BMC Res Notes. 2011;4: 446. doi:10.1186/1756-0500-4-446" />
<meta name="citation_reference" content="Zhou H, Ferraro D, Zhao J, Hussain S, Shao J, Trujillo J, et al. The N-Terminal Region of Severe Acute Respiratory Syndrome Coronavirus Protein 6 Induces Membrane Rearrangement and Enhances Virus Replication. J Virol. 2010;84: 3542–3551. doi:10.1128/jvi.02570-09" />
<meta name="citation_reference" content="Fielding BC, Tan Y-J, Shuo S, Tan THP, Ooi E-E, Lim SG, et al. Characterization of a Unique Group-Specific Protein (U122) of the Severe Acute Respiratory Syndrome Coronavirus. J Virol. 2004;78: 7311–7318. doi:10.1128/jvi.78.14.7311-7318.2004" />
<meta name="citation_reference" content="Huang C, Ito N, Tseng C-TK, Makino S. Severe Acute Respiratory Syndrome Coronavirus 7a Accessory Protein Is a Viral Structural Protein. J Virol. 2006;80: 7287–7294. doi:10.1128/jvi.00414-06" />
<meta name="citation_reference" content="Kanzawa N, Nishigaki K, Hayashi T, Ishii Y, Furukawa S, Niiro A, et al. Augmentation of chemokine production by severe acute respiratory syndrome coronavirus 3a/X1 and 7a/X4 proteins through NF-kappaB activation. FEBS Lett. 2006;580: 6807–12. doi:10.1016/j.febslet.2006.11.046" />
<meta name="citation_reference" content="Law HKW, Cheung CY, Ng HY, Sia SF, Chan YO, Luk W, et al. Chemokine up-regulation in SARS-coronavirus-infected, monocyte-derived human dendritic cells. Blood. 2005;106: 2366–74. doi:10.1182/blood-2004-10-4166" />
<meta name="citation_reference" content="Tan Y-X, Tan THP, Lee MJ-R, Tham P-Y, Gunalan V, Druce J, et al. Induction of Apoptosis by the Severe Acute Respiratory Syndrome Coronavirus 7a Protein Is Dependent on Its Interaction with the Bcl-XL Protein. J Virol. 2007;81: 6346–6355. doi:10.1128/jvi.00090-07" />
<meta name="citation_reference" content="Schaecher SR, Mackenzie JM, Pekosz A. The ORF7b Protein of Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) Is Expressed in Virus-Infected Cells and Incorporated into SARS-CoV Particles. J Virol. 2007;81: 718–731. doi:10.1128/jvi.01691-06" />
<meta name="citation_reference" content="Kopecky-Bromberg SA, Martinez-Sobrido L, Palese P. 7a Protein of Severe Acute Respiratory Syndrome Coronavirus Inhibits Cellular Protein Synthesis and Activates p38 Mitogen-Activated Protein Kinase. J Virol. 2006;80: 785–793. doi:10.1128/jvi.80.2.785-793.2006" />
<meta name="citation_reference" content="Nelson CA, Pekosz A, Lee CA, Diamond MS, Fremont DH. Structure and intracellular targeting of the SARS-coronavirus Orf7a accessory protein. Structure. 2005;13: 75–85. doi:10.1016/j.str.2004.10.010" />
<meta name="citation_reference" content="Hänel K, Stangler T, Stoldt M, Willbold D. Solution structure of the X4 protein coded by the SARS related coronavirus reveals an immunoglobulin like fold and suggests a binding activity to integrin I domains. J Biomed Sci. 2006;13: 281–293. doi:10.1007/s11373-005-9043-9" />
<meta name="citation_reference" content="Oostra M, de Haan CAM, Rottier PJM. The 29-Nucleotide Deletion Present in Human but Not in Animal Severe Acute Respiratory Syndrome Coronaviruses Disrupts the Functional Expression of Open Reading Frame 8. J Virol. 2007;81: 13876–13888. doi:10.1128/jvi.01631-07" />
<meta name="citation_reference" content="Chinese SARS Molecular Epidemiology Consortium. Molecular evolution of the SARS coronavirus during the course of the SARS epidemic in China. Science. 2004;303: 1666–9. doi:10.1126/science.1092002" />
<meta name="citation_reference" content="Keng C-T, Choi Y-W, Welkers MRA, Chan DZL, Shen S, Gee Lim S, et al. The human severe acute respiratory syndrome coronavirus (SARS-CoV) 8b protein is distinct from its counterpart in animal SARS-CoV and down-regulates the expression of the envelope protein in infected cells. Virology. 2006;354: 132–42. doi:10.1016/j.virol.2006.06.026" />
<meta name="citation_reference" content="Xu K, Zheng B-J, Zeng R, Lu W, Lin Y-P, Xue L, et al. Severe acute respiratory syndrome coronavirus accessory protein 9b is a virion-associated protein. Virology. 2009;388: 279–85. doi:10.1016/j.virol.2009.03.032" />
<meta name="citation_reference" content="Sharma K, Åkerström S, Sharma AK, Chow VTK, Teow S, Abrenica B, et al. SARS-CoV 9b protein diffuses into nucleus, undergoes active Crm1 mediated nucleocytoplasmic export and triggers apoptosis when retained in the nucleus. PLoS One. 2011;6: e19436. doi:10.1371/journal.pone.0019436" />
<meta name="citation_reference" content="Meier C, Aricescu AR, Assenberg R, Aplin RT, Gilbert RJC, Grimes JM, et al. The crystal structure of ORF-9b, a lipid binding protein from the SARS coronavirus. Structure. 2006;14: 1157–65. doi:10.1016/j.str.2006.05.012" />
<meta name="citation_reference" content="Thiel V, Ivanov KA, Putics Á, Hertzig T, Schelle B, Bayer S, et al. Mechanisms and enzymes involved in SARS coronavirus genome expression. J Gen Virol. 2003;84: 2305–2315. doi:10.1099/vir.0.19424-0" />
<meta name="citation_reference" content="Fan K, Wei P, Feng Q, Chen S, Huang C, Ma L, et al. Biosynthesis, purification, and substrate specificity of severe acute respiratory syndrome coronavirus 3C-like proteinase. J Biol Chem. 2004;279: 1637–42. doi:10.1074/jbc.M310875200" />
<meta name="citation_reference" content="Lokugamage KG, Narayanan K, Huang C, Makino S. Severe Acute Respiratory Syndrome Coronavirus Protein nsp1 Is a Novel Eukaryotic Translation Inhibitor That Represses Multiple Steps of Translation Initiation. J Virol. 2012;86: 13598–13608. doi:10.1128/jvi.01958-12" />
<meta name="citation_reference" content="Almeida MS, Johnson MA, Herrmann T, Geralt M, Wuthrich K. Novel-Barrel Fold in the Nuclear Magnetic Resonance Structure of the Replicase Nonstructural Protein 1 from the Severe Acute Respiratory Syndrome Coronavirus. J Virol. 2007;81: 3151–3161. doi:10.1128/jvi.01939-06" />
<meta name="citation_reference" content="Jauregui AR, Savalia D, Lowry VK, Farrell CM, Wathelet MG. Identification of residues of SARS-CoV nsp1 that differentially affect inhibition of gene expression and antiviral signaling. PLoS One. 2013;8: e62416. doi:10.1371/journal.pone.0062416" />
<meta name="citation_reference" content="Narayanan K, Ramirez SI, Lokugamage KG, Makino S. Coronavirus nonstructural protein 1: Common and distinct functions in the regulation of host and viral gene expression. Virus Res. 2015;202: 89–100. doi:10.1016/j.virusres.2014.11.019" />
<meta name="citation_reference" content="Cornillez-Ty CT, Liao L, Yates JR, Kuhn P, Buchmeier MJ. Severe Acute Respiratory Syndrome Coronavirus Nonstructural Protein 2 Interacts with a Host Protein Complex Involved in Mitochondrial Biogenesis and Intracellular Signaling. J Virol. 2009;83: 10314–10318. doi:10.1128/jvi.00842-09" />
<meta name="citation_reference" content="Graham RL, Sims AC, Brockway SM, Baric RS, Denison MR. The nsp2 Replicase Proteins of Murine Hepatitis Virus and Severe Acute Respiratory Syndrome Coronavirus Are Dispensable for Viral Replication. J Virol. 2005;79: 13399–13411. doi:10.1128/jvi.79.21.13399-13411.2005" />
<meta name="citation_reference" content="Frieman M, Ratia K, Johnston RE, Mesecar AD, Baric RS. Severe Acute Respiratory Syndrome Coronavirus Papain-Like Protease Ubiquitin-Like Domain and Catalytic Domain Regulate Antagonism of IRF3 and NF-B Signaling. J Virol. 2009;83: 6689–6705. doi:10.1128/jvi.02220-08" />
<meta name="citation_reference" content="Ratia K, Saikatendu KS, Santarsiero BD, Barretto N, Baker SC, Stevens RC, et al. Severe acute respiratory syndrome coronavirus papain-like protease: structure of a viral deubiquitinating enzyme. Proc Natl Acad Sci U S A. 2006;103: 5717–22. doi:10.1073/pnas.0510851103" />
<meta name="citation_reference" content="Serrano P, Johnson MA, Almeida MS, Horst R, Herrmann T, Joseph JS, et al. Nuclear Magnetic Resonance Structure of the N-Terminal Domain of Nonstructural Protein 3 from the Severe Acute Respiratory Syndrome Coronavirus. J Virol. 2007;81: 12049–12060. doi:10.1128/jvi.00969-07" />
<meta name="citation_reference" content="Alvarez E, DeDiego ML, Nieto-Torres JL, Jiménez-Guardeño JM, Marcos-Villar L, Enjuanes L. The envelope protein of severe acute respiratory syndrome coronavirus interacts with the non-structural protein 3 and is ubiquitinated. Virology. 2010;402: 281–91. doi:10.1016/j.virol.2010.03.015" />
<meta name="citation_reference" content="Angelini MM, Akhlaghpour M, Neuman BW, Buchmeier MJ. Severe acute respiratory syndrome coronavirus nonstructural proteins 3, 4, and 6 induce double-membrane vesicles. MBio. 2013;4. doi:10.1128/mBio.00524-13" />
<meta name="citation_reference" content="Hagemeijer MC, Ulasli M, Vonk AM, Reggiori F, Rottier PJM, de Haan CAM. Mobility and Interactions of Coronavirus Nonstructural Protein 4. J Virol. 2011;85: 4572–4577. doi:10.1128/jvi.00042-11" />
<meta name="citation_reference" content="Sakai Y, Kawachi K, Terada Y, Omori H, Matsuura Y, Kamitani W. Two-amino acids change in the nsp4 of SARS coronavirus abolishes viral replication. Virology. 2017;510: 165–174. doi:10.1016/j.virol.2017.07.019" />
<meta name="citation_reference" content="Oostra M, te Lintelo EG, Deijs M, Verheije MH, Rottier PJM, de Haan CAM. Localization and Membrane Topology of Coronavirus Nonstructural Protein 4: Involvement of the Early Secretory Pathway in Replication. J Virol. 2007;81: 12323–12336. doi:10.1128/jvi.01506-07" />
<meta name="citation_reference" content="Tomar S, Johnston ML, St John SE, Osswald HL, Nyalapatla PR, Paul LN, et al. Ligand-induced Dimerization of Middle East Respiratory Syndrome (MERS) Coronavirus nsp5 Protease (3CLpro): IMPLICATIONS FOR nsp5 REGULATION AND THE DEVELOPMENT OF ANTIVIRALS. J Biol Chem. 2015;290: 19403–22. doi:10.1074/jbc.M115.651463" />
<meta name="citation_reference" content="Sparks JS, Donaldson EF, Lu X, Baric RS, Denison MR. A Novel Mutation in Murine Hepatitis Virus nsp5, the Viral 3C-Like Proteinase, Causes Temperature-Sensitive Defects in Viral Growth and Protein Processing. J Virol. 2008;82: 5999–6008. doi:10.1128/jvi.00203-08" />
<meta name="citation_reference" content="Anand K, Palm GJ, Mesters JR, Siddell SG, Ziebuhr J, Hilgenfeld R. Structure of coronavirus main proteinase reveals combination of a chymotrypsin fold with an extra alpha-helical domain. EMBO J. 2002;21: 3213–24. doi:10.1093/emboj/cdf327" />
<meta name="citation_reference" content="Cottam EM, Whelband MC, Wileman T. Coronavirus NSP6 restricts autophagosome expansion. Autophagy. 2014;10: 1426–41. doi:10.4161/auto.29309" />
<meta name="citation_reference" content="te Velthuis AJW, van den Worm SHE, Snijder EJ. The SARS-coronavirus nsp7+nsp8 complex is a unique multimeric RNA polymerase capable of both de novo initiation and primer extension. Nucleic Acids Res. 2012;40: 1737–47. doi:10.1093/nar/gkr893" />
<meta name="citation_reference" content="Zhai Y, Sun F, Li X, Pang H, Xu X, Bartlam M, et al. Insights into SARS-CoV transcription and replication from the structure of the nsp7-nsp8 hexadecamer. Nat Struct Mol Biol. 2005;12: 980–6. doi:10.1038/nsmb999" />
<meta name="citation_reference" content="Kirchdoerfer RN, Ward AB. Structure of the SARS-CoV nsp12 polymerase bound to nsp7 and nsp8 co-factors. Nat Commun. 2019;10: 2342. doi:10.1038/s41467-019-10280-3" />
<meta name="citation_reference" content="Egloff M-P, Ferron F, Campanacci V, Longhi S, Rancurel C, Dutartre H, et al. The severe acute respiratory syndrome-coronavirus replicative protein nsp9 is a single-stranded RNA-binding subunit unique in the RNA virus world. Proc Natl Acad Sci U S A. 2004;101: 3792–6. doi:10.1073/pnas.0307877101" />
<meta name="citation_reference" content="Ponnusamy R, Moll R, Weimar T, Mesters JR, Hilgenfeld R. Variable oligomerization modes in coronavirus non-structural protein 9. J Mol Biol. 2008;383: 1081–96. doi:10.1016/j.jmb.2008.07.071" />
<meta name="citation_reference" content="Miknis ZJ, Donaldson EF, Umland TC, Rimmer RA, Baric RS, Schultz LW. Severe Acute Respiratory Syndrome Coronavirus nsp9 Dimerization Is Essential for Efficient Viral Growth. J Virol. 2009;83: 3007–3018. doi:10.1128/jvi.01505-08" />
<meta name="citation_reference" content="Bouvet M, Imbert I, Subissi L, Gluais L, Canard B, Decroly E. RNA 3’-end mismatch excision by the severe acute respiratory syndrome coronavirus nonstructural protein nsp10/nsp14 exoribonuclease complex. Proc Natl Acad Sci U S A. 2012;109: 9372–7. doi:10.1073/pnas.1201130109" />
<meta name="citation_reference" content="Bouvet M, Debarnot C, Imbert I, Selisko B, Snijder EJ, Canard B, et al. In vitro reconstitution of SARS-coronavirus mRNA cap methylation. PLoS Pathog. 2010;6: e1000863. doi:10.1371/journal.ppat.1000863" />
<meta name="citation_reference" content="Ma Y, Wu L, Shaw N, Gao Y, Wang J, Sun Y, et al. Structural basis and functional analysis of the SARS coronavirus nsp14-nsp10 complex. Proc Natl Acad Sci U S A. 2015;112: 9436–41. doi:10.1073/pnas.1508686112" />
<meta name="citation_reference" content="Ahn D-G, Choi J-K, Taylor DR, Oh J-W. Biochemical characterization of a recombinant SARS coronavirus nsp12 RNA-dependent RNA polymerase capable of copying viral RNA templates. Arch Virol. 2012;157: 2095–104. doi:10.1007/s00705-012-1404-x" />
<meta name="citation_reference" content="Adedeji AO, Marchand B, Te Velthuis AJW, Snijder EJ, Weiss S, Eoff RL, et al. Mechanism of nucleic acid unwinding by SARS-CoV helicase. PLoS One. 2012;7: e36521. doi:10.1371/journal.pone.0036521" />
<meta name="citation_reference" content="Jia Z, Yan L, Ren Z, Wu L, Wang J, Guo J, et al. Delicate structural coordination of the Severe Acute Respiratory Syndrome coronavirus Nsp13 upon ATP hydrolysis. Nucleic Acids Res. 2019;47: 6538–6550. doi:10.1093/nar/gkz409" />
<meta name="citation_reference" content="Minskaia E, Hertzig T, Gorbalenya AE, Campanacci V, Cambillau C, Canard B, et al. Discovery of an RNA virus 3’-&gt;5’ exoribonuclease that is critically involved in coronavirus RNA synthesis. Proc Natl Acad Sci U S A. 2006;103: 5108–13. doi:10.1073/pnas.0508200103" />
<meta name="citation_reference" content="Ivanov KA, Hertzig T, Rozanov M, Bayer S, Thiel V, Gorbalenya AE, et al. Major genetic marker of nidoviruses encodes a replicative endoribonuclease. Proc Natl Acad Sci U S A. 2004;101: 12694–12699. doi:10.1073/pnas.0403127101" />
<meta name="citation_reference" content="Ricagno S, Egloff M-P, Ulferts R, Coutard B, Nurizzo D, Campanacci V, et al. Crystal structure and mechanistic determinants of SARS coronavirus nonstructural protein 15 define an endoribonuclease family. Proc Natl Acad Sci U S A. 2006;103: 11892–7. doi:10.1073/pnas.0601708103" />
<meta name="citation_reference" content="Decroly E, Imbert I, Coutard B, Bouvet M, Selisko B, Alvarez K, et al. Coronavirus Nonstructural Protein 16 Is a Cap-0 Binding Enzyme Possessing (Nucleoside-2’O)-Methyltransferase Activity. J Virol. 2008;82: 8071–8084. doi:10.1128/jvi.00407-08" />
<meta name="citation_reference" content="Chen Y, Su C, Ke M, Jin X, Xu L, Zhang Z, et al. Biochemical and structural insights into the mechanisms of SARS coronavirus RNA ribose 2’-O-methylation by nsp16/nsp10 protein complex. PLoS Pathog. 2011;7: e1002294. doi:10.1371/journal.ppat.1002294" />
<meta name="twitter:title" content="When Darkness Becomes a Ray of Light in the Dark Times: Understanding the COVID-19 via the Comparative Analysis of the Dark Proteomes of SARS-CoV-2, Human SARS and Bat SARS-Like Coronaviruses" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Recently emerged coronavirus designated as SARS-CoV-2 (also known as 2019 novel coronavirus (2019-nCoV) or Wuhan coronavirus) is a causative agent of coronavirus disease 2019 (COVID-19), which is rapidly spreading throughout the world now. More than 9,00,000 cases of SARS-CoV-2 infection and more than 47,000 COVID-19-associated mortalities have been reported worldwide till the writing of this article, and these numbers are increasing every passing hour. World Health Organization (WHO) has declared the SARS-CoV-2 spread as a global public health emergency and admitted that the COVID-19 is a pandemic now. The multiple sequence alignment data correlated with the already published reports on the SARS-CoV-2 evolution and indicated that this virus is closely related to the bat Severe Acute Respiratory Syndrome-like coronavirus (bat SARS-like CoV) and the well-studied Human SARS coronavirus (SARS CoV). The disordered regions in viral proteins are associated with the viral infectivity and pathogenicity. Therefore, in this study, we have exploited a set of complementary computational approaches to examine the dark proteomes of SARS-CoV-2, bat SARS-like, and human SARS CoVs by analysing the prevalence of intrinsic disorder in their proteins. According to our findings, SARS-CoV-2 proteome contains very significant levels of structural order. In fact, except for Nucleocapsid, Nsp8, and ORF6, the vast majority of SARS-CoV-2 proteins are mostly ordered proteins containing less intrinsically disordered protein regions (IDPRs). However, IDPRs found in SARS-CoV-2 proteins are functionally important. For example, cleavage sites in its replicase 1ab polyprotein are found to be highly disordered, and almost all SARS-CoV-2 proteins were shown to contain molecular recognition features (MoRFs), which are intrinsic disorder-based protein-protein interaction sites that are commonly utilized by proteins for interaction with specific partners. The results of our extensive investigation of the dark side of the SARS-CoV-2 proteome will have important implications for the structural and non-structural biology of SARS or SARS-like coronaviruses.

Significance The infection caused by a novel coronavirus (SARS-CoV-2) that causes severe respiratory disease with pneumonia-like symptoms in humans is responsible for the current COVID-19 pandemic. No in-depth information on structures and functions of SARS-CoV-2 proteins is currently available in the public domain, and no effective anti-viral drugs and/or vaccines are designed for the treatment of this infection. Our study provides the first comparative analysis of the order- and disorder-based features of the SARS-CoV-2 proteome relative to human SARS and bat CoV that may be useful for structure-based drug discovery.

*   ACE2
    :   Angiotensin-converting enzyme 2
    CDF
    :   Cumulative distribution function
    CH
    :   Charge hydropathy
    COVID-19
    :   Coronavirus disease 2019
    CTD
    :   Cterminal domain
    DMVs
    :   Double-membrane vesicles
    ICTV
    :   International committee on taxonomy of viruses
    IDP
    :   Intrinsically disordered proteins
    IDPRs
    :   Intrinsically disordered protein regions
    IFN
    :   Interferon
    MoRFs
    :   Molecular recognition features
    MSA
    :   Multiple sequence alignment
    Nsps
    :   Non-structural proteins
    NTD
    :   N-terminal domain
    PONDR
    :   Predictor of natural disordered regions
    PPID
    :   Predicted percentage of intrinsic disorder
    Pprint
    :   Prediction of Protein RNA-Interaction
    RBD
    :   Receptor binding domain
    SARS
    :   Severe acute respiratory syndrome
    TRS
    :   Transcriptional regulatory sequences
    VLPs
    :   Virus-like particles
    WHO
    :   World health organization" />
<meta name="og-title" property="og:title" content="When Darkness Becomes a Ray of Light in the Dark Times: Understanding the COVID-19 via the Comparative Analysis of the Dark Proteomes of SARS-CoV-2, Human SARS and Bat SARS-Like Coronaviruses" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.13.990598v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Recently emerged coronavirus designated as SARS-CoV-2 (also known as 2019 novel coronavirus (2019-nCoV) or Wuhan coronavirus) is a causative agent of coronavirus disease 2019 (COVID-19), which is rapidly spreading throughout the world now. More than 9,00,000 cases of SARS-CoV-2 infection and more than 47,000 COVID-19-associated mortalities have been reported worldwide till the writing of this article, and these numbers are increasing every passing hour. World Health Organization (WHO) has declared the SARS-CoV-2 spread as a global public health emergency and admitted that the COVID-19 is a pandemic now. The multiple sequence alignment data correlated with the already published reports on the SARS-CoV-2 evolution and indicated that this virus is closely related to the bat Severe Acute Respiratory Syndrome-like coronavirus (bat SARS-like CoV) and the well-studied Human SARS coronavirus (SARS CoV). The disordered regions in viral proteins are associated with the viral infectivity and pathogenicity. Therefore, in this study, we have exploited a set of complementary computational approaches to examine the dark proteomes of SARS-CoV-2, bat SARS-like, and human SARS CoVs by analysing the prevalence of intrinsic disorder in their proteins. According to our findings, SARS-CoV-2 proteome contains very significant levels of structural order. In fact, except for Nucleocapsid, Nsp8, and ORF6, the vast majority of SARS-CoV-2 proteins are mostly ordered proteins containing less intrinsically disordered protein regions (IDPRs). However, IDPRs found in SARS-CoV-2 proteins are functionally important. For example, cleavage sites in its replicase 1ab polyprotein are found to be highly disordered, and almost all SARS-CoV-2 proteins were shown to contain molecular recognition features (MoRFs), which are intrinsic disorder-based protein-protein interaction sites that are commonly utilized by proteins for interaction with specific partners. The results of our extensive investigation of the dark side of the SARS-CoV-2 proteome will have important implications for the structural and non-structural biology of SARS or SARS-like coronaviruses.

Significance The infection caused by a novel coronavirus (SARS-CoV-2) that causes severe respiratory disease with pneumonia-like symptoms in humans is responsible for the current COVID-19 pandemic. No in-depth information on structures and functions of SARS-CoV-2 proteins is currently available in the public domain, and no effective anti-viral drugs and/or vaccines are designed for the treatment of this infection. Our study provides the first comparative analysis of the order- and disorder-based features of the SARS-CoV-2 proteome relative to human SARS and bat CoV that may be useful for structure-based drug discovery.

*   ACE2
    :   Angiotensin-converting enzyme 2
    CDF
    :   Cumulative distribution function
    CH
    :   Charge hydropathy
    COVID-19
    :   Coronavirus disease 2019
    CTD
    :   Cterminal domain
    DMVs
    :   Double-membrane vesicles
    ICTV
    :   International committee on taxonomy of viruses
    IDP
    :   Intrinsically disordered proteins
    IDPRs
    :   Intrinsically disordered protein regions
    IFN
    :   Interferon
    MoRFs
    :   Molecular recognition features
    MSA
    :   Multiple sequence alignment
    Nsps
    :   Non-structural proteins
    NTD
    :   N-terminal domain
    PONDR
    :   Predictor of natural disordered regions
    PPID
    :   Predicted percentage of intrinsic disorder
    Pprint
    :   Prediction of Protein RNA-Interaction
    RBD
    :   Receptor binding domain
    SARS
    :   Severe acute respiratory syndrome
    TRS
    :   Transcriptional regulatory sequences
    VLPs
    :   Virus-like particles
    WHO
    :   World health organization" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-04-03" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.13.990598v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.13.990598v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.13.990598v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.13.990598v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1225706" />
    <title>When Darkness Becomes a Ray of Light in the Dark Times: Understanding the COVID-19 via the Comparative Analysis of the Dark Proteomes of SARS-CoV-2, Human SARS and Bat SARS-Like Coronaviruses | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"hteFe6FmVAMfcCHL7HiXWkQ3viA5pkU_cPp-feMx6yA","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1225706","apath":"\/biorxiv\/early\/2020\/04\/03\/2020.03.13.990598.atom","pisa":"biorxiv;2020.03.13.990598v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.13.990598v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.13.990598v2","title":"When Darkness Becomes a Ray of Light in the Dark Times: Understanding the COVID-19 via the Comparative Analysis of the Dark Proteomes of SARS-CoV-2, Human SARS and Bat SARS-Like Coronaviruses","identifier":"node\/1225706"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00225IwLAus68wUQ5XqRlRarwT1kb9W6utG+GORK7lMN3QUFqzUtpNMqZQt8O4Bu20m5GtUZ7V4RUiPMSm4kVl0mMoV9vmHtNjGfZAdYsEK43RnwvY2+7wP3jU8LGmtH0MRJMdWgr+ROxnrzXda0KZuc1dHTYMdiuUHhmiYLdXl79oGsfQwTunTriQdU1yKVf3XgEOtTluK8pGaIA2SvkUEYSBWfPSMI9bO1poA4TPDX6M1M+cODJzvR+zFtk0lwlqsYR6uRbP684smcZormEy8GVrmYUTU\\\\\\\/pcC6xlcACjjD+YVL1Id3g\\\\\\\/\\\\\\\/l5mHlberEfTcoVaGJTmBhaDGjVyy5n6IO7dHBDrXOKQEnZ\\\\\\\/bvSOwV1B43Kaydl83J95xatM4GF3M7YBSAozIr+mzQGJvlDYjYq4vyoeFyPw0MgptfZj\\\\\\\/PAXqsIK\\\\\\\/EUGYyF6DXmjaGRE\\\\\\\/YqDpmRiVl9aPYMqZOjPuP2ePf0dl+w0lsRqx8r5GwE9goN8C1Swl481PmYv4KsFQvzFdboEYGBCHcid7iv5HEErIrcxqdU0Fy+bikelw16oT3xWQCmS5d3XTp88c5U1mBnAiMeMFE1bV05sWsdigrYjig25sYvIwYnTlcfRxiLzw8AdTwgYCogI0hn3EicMAYps5UkTCvdxCi+8cSYNgVMjKIeg+OLlMvrObH8krIpNSKg7bBy7stUaGN9sPVb\\\\\\\/cHkyLnN8qt3dH7eUpqAGMA3A4eULtsa+\\\\\\\/tf54+6tjilBUfoY7lOQjQXQLWf5YeFSrqlejWuO1oEE\\\\\\\/2Y9PXjdd+krmkQ1Q\\\\\\\/Db9fJCohbfxGtl8WXuGRM16JKPT9ijHeNRtNamCWIr5LacWab1sqEfS\\\\\\\/IjlyOnsFibNUId7tFzS4as91HcCfMjlOwEoYGKW7Pe4z1DN2A6X++F0b6qCk4OoUNPsCq9qMqnV\\\\\\\/K37u7CX3LNSAJwzDyp6ycrS5XOoHritKcR38b6ifF3N2aKqTfC\\\\\\\/eQwCYkV\\\\\\\/A4HU9CTENexXCC7trT+6G+QVCmpnYGISt9WayVW1XwumiPd+EMlQyWCT\\\\\\\/3FV2YuDRKQDyA6BTejnuuF4coL+9YpqS7DneR9\\\\\\\/+j1M9QI\\\\\\\/rpNvLmsxUY7SBc1QUQ44DJrK2bjNkOPjmWF7JrG6TjS89aEr6wTlEf7gH\\\\\\\/9VQkSTa7DY+vlIHaSu6z8E4hg4jYVOxEXd3eH8VTYKp52F2612jb3k\\\\\\\/lkamt3uwFPxg7E55bo5UmMpxzIupdxunjMJHYaLc0c1PJq95QTRVKWjTxmb7mlsWR4YZ+\\\\\\\/OsKgkDDe4kXQNWQ5+L1bV35\\\\\\\/Pl9ubIzCqh0iUWxBsy+IEJaUEsN9feN5qr0IjTtjSsBexh\\\u0022,\\\u0022iv\\\u0022:\\\u0022TEK51H8ctUGtBzCfE+nLpA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002237f70742dbd3d09c8960f481aa471a2f23e323b921206690a56732eddf1fcb2f\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.13.990598v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1225706 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.13.990598v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-aGWPuOcXmGe-M1vdT2f533Vh5pjp749iJzYYoRufmXs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1225706" id="node1225706" data-pisa="biorxiv;2020.03.13.990598v2" data-pisa-master="biorxiv;2020.03.13.990598" data-apath="/biorxiv/early/2020/04/03/2020.03.13.990598.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">When Darkness Becomes a Ray of Light in the Dark Times: Understanding the COVID-19 via the Comparative Analysis of the Dark Proteomes of SARS-CoV-2, Human SARS and Bat SARS-Like Coronaviruses</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2046-836X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Rajanish</span> <span class="nlm-surname">Giri</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Taniya</span> <span class="nlm-surname">Bhardwaj</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Meenakshi</span> <span class="nlm-surname">Shegane</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Bhuvaneshwari R.</span> <span class="nlm-surname">Gehi</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Prateek</span> <span class="nlm-surname">Kumar</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Kundlik</span> <span class="nlm-surname">Gadhave</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Christopher J.</span> <span class="nlm-surname">Oldfield</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Vladimir N.</span> <span class="nlm-surname">Uversky</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.13.990598 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1225706" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Rajanish Giri </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Indian Institute of Technology Mandi, School of Basic Sciences, VPO Kamand</span>, Himachal Pradesh 175005, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rajanish%2BGiri%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giri%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARajanish%2BGiri%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2046-836X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rajanish Giri</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rajanishgiri@iitmandi.ac.in" class="" data-icon-position="" data-hide-link-title="0">rajanishgiri@iitmandi.ac.in</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Taniya Bhardwaj </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Indian Institute of Technology Mandi, School of Basic Sciences, VPO Kamand</span>, Himachal Pradesh 175005, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Taniya%2BBhardwaj%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bhardwaj%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATaniya%2BBhardwaj%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Meenakshi Shegane </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Indian Institute of Technology Mandi, School of Basic Sciences, VPO Kamand</span>, Himachal Pradesh 175005, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Meenakshi%2BShegane%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shegane%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMeenakshi%2BShegane%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Bhuvaneshwari R. Gehi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Indian Institute of Technology Mandi, School of Basic Sciences, VPO Kamand</span>, Himachal Pradesh 175005, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bhuvaneshwari%2BR.%2BGehi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gehi%20BR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABhuvaneshwari%2BR.%2BGehi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Prateek Kumar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Indian Institute of Technology Mandi, School of Basic Sciences, VPO Kamand</span>, Himachal Pradesh 175005, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Prateek%2BKumar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kumar%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APrateek%2BKumar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Kundlik Gadhave </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Indian Institute of Technology Mandi, School of Basic Sciences, VPO Kamand</span>, Himachal Pradesh 175005, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kundlik%2BGadhave%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gadhave%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKundlik%2BGadhave%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Christopher J. Oldfield </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Computer Science, Virginia Commonwealth University</span>, Richmond, VA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BJ.%2BOldfield%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oldfield%20CJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristopher%2BJ.%2BOldfield%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Vladimir N. Uversky </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Molecular Medicine and Byrd Alzheimer’s Research Institute, Morsani College of Medicine, University of South Florida</span>, Tampa, Florida, <span class='nlm-country'>United States of America</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Laboratory of New Methods in Biology, Institute for Biological Instrumentation of the Russian Academy of Sciences, Federal Research Center “Pushchino Scientific Center for Biological Research of the Russian Academy of Sciences”</span>, Pushchino, Moscow region, 142290 <span class='nlm-country'>Russia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vladimir%2BN.%2BUversky%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Uversky%20VN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVladimir%2BN.%2BUversky%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.13.990598v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1225706" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1225706/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.13.990598v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1225706" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1225706/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.13.990598v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1225706" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1225706/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.13.990598v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1225706" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1225706/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.13.990598v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1225706" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1225706/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.13.990598v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1225706" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1225706/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Recently emerged coronavirus designated as SARS-CoV-2 (also known as 2019 novel coronavirus (2019-nCoV) or Wuhan coronavirus) is a causative agent of coronavirus disease 2019 (COVID-19), which is rapidly spreading throughout the world now. More than 9,00,000 cases of SARS-CoV-2 infection and more than 47,000 COVID-19-associated mortalities have been reported worldwide till the writing of this article, and these numbers are increasing every passing hour. World Health Organization (WHO) has declared the SARS-CoV-2 spread as a global public health emergency and admitted that the COVID-19 is a pandemic now. The multiple sequence alignment data correlated with the already published reports on the SARS-CoV-2 evolution and indicated that this virus is closely related to the bat Severe Acute Respiratory Syndrome-like coronavirus (bat SARS-like CoV) and the well-studied Human SARS coronavirus (SARS CoV). The disordered regions in viral proteins are associated with the viral infectivity and pathogenicity. Therefore, in this study, we have exploited a set of complementary computational approaches to examine the dark proteomes of SARS-CoV-2, bat SARS-like, and human SARS CoVs by analysing the prevalence of intrinsic disorder in their proteins. According to our findings, SARS-CoV-2 proteome contains very significant levels of structural order. In fact, except for Nucleocapsid, Nsp8, and ORF6, the vast majority of SARS-CoV-2 proteins are mostly ordered proteins containing less intrinsically disordered protein regions (IDPRs). However, IDPRs found in SARS-CoV-2 proteins are functionally important. For example, cleavage sites in its replicase 1ab polyprotein are found to be highly disordered, and almost all SARS-CoV-2 proteins were shown to contain molecular recognition features (MoRFs), which are intrinsic disorder-based protein-protein interaction sites that are commonly utilized by proteins for interaction with specific partners. The results of our extensive investigation of the dark side of the SARS-CoV-2 proteome will have important implications for the structural and non-structural biology of SARS or SARS-like coronaviruses.</p><div id="sec-1" class="subsection"><p id="p-3"><strong>Significance</strong> The infection caused by a novel coronavirus (SARS-CoV-2) that causes severe respiratory disease with pneumonia-like symptoms in humans is responsible for the current COVID-19 pandemic. No in-depth information on structures and functions of SARS-CoV-2 proteins is currently available in the public domain, and no effective anti-viral drugs and/or vaccines are designed for the treatment of this infection. Our study provides the first comparative analysis of the order- and disorder-based features of the SARS-CoV-2 proteome relative to human SARS and bat CoV that may be useful for structure-based drug discovery.</p></div></div><div class="section" id="sec-2"><h2 class="">Introduction</h2><p id="p-5">Emerging Coronavirus disease 2019 (COVID-19) is a recent pandemic, which is recently declared as a public health emergency by the World Health Organization (WHO). Since its first appearance in visitors of the Wuhan’s seafood and meat market, China, reported in December 2019, COVID-19 has now large scale socio-economic impact [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>]. According to the WHO, till 2<sup>nd</sup> April 2020, the infection spread over at least 170 countries and territories, where there were more than 0.93 million confirmed cases, with more than 47,000 patients died due to COVID-19. One should also keep in mind that these data on the COVID-19 spread and related casualties are rapidly becoming outdated, almost with the speed of typing of these sentences. Symptoms of COVID-19, which is a respiratory illness, include fever, cough, sore throat, shortness of breath, as well as mild gastrointestinal (GI) symptoms in some patients, and, in more serious cases, the infection can cause severe pneumonia and even death [<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>].</p><p id="p-6">According to the International Committee on Taxonomy of Viruses (ICTV), SARS-CoV-2 comes under the coronavirinae sub-family of <em>coronaviridae</em> family of order nidovirales. Viruses of the nidovirales order are enveloped, non-segmented positive-sense, single-stranded RNA viruses [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>]. The major variations among <em>Nidovirus</em> family occur in the number, type, and sizes of the structural proteins [<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>]. The family <em>coronaviridae</em> comprises of vertebrate infecting viruses that transmit horizontally mainly through oral/fecal route and cause gastrointestinal and respiratory problems to the host [<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>]. Sub-family <em>coronavirinae</em> consists of four genera, namely: alpha, beta, gamma, and delta coronaviruses based on the phylogenetic clustering of viruses [5,6]. <em>Coronavirinae</em> having the largest genomes among the RNA viruses, incorporate their ~30kb genomes inside the enveloped capsid. These viruses have variations among genome due to significant differences in the structure and morphology of their nucleocapsids [<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>].</p><p id="p-7">SARS-Coronavirus genomic RNA includes a 5’ cap, leader sequence, UTR, a replicase gene, genes for structural and accessory proteins, 3’ UTR, and a poly-A tail (<strong><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a></strong>). Two-third of genome codes for replicase gene (~20kb) containing genes for all non-structural viral proteins, while the remaining part of the genome (~10kb) contains genes for accessory proteins interspersed between the genes responsible for coding structural proteins [<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>]. For the expression of structural and accessory proteins, additional transcriptional regulatory sequences (TRS) are present within the viral genome. 5’ and 3’ UTRs contain stem-loop regions required for viral RNA synthesis [<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>]. The ~20kb (replicase gene) ssRNA is translated first into two long polyproteins: replicase polyprotein 1a and 1ab inside host cells. The newly formed polyproteins, after cleavage by two viral proteases, result in 16 non-structural proteins that perform a wide range of functions for viruses inside the host cell. They also induce ER-derived double-membrane vesicles (DMVs) for viral replication and transcription. Structural proteins shape the outer cover of the virion, while accessory proteins are mostly involved in host immune evasion [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>,<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>].</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome architecture of SARS-CoV-2. SARS-CoV-2 contains a positive single-stranded mRNA as genetic material. The 5&#x2019; capped mRNA has a leader sequence (LS), poly-A tail at 3&#x2019; end, and 5&#x2019; and 3&#x2019; UTR. It contains the following genes: ORF1a, ORF1b, Spike (S), ORF3a, ORF3b, Envelope (E), Membrane (M), ORF6, ORF7a, ORF7b, ORF8, ORF9b, ORF14, Nucleocapsid (N), and ORF10." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Genome architecture of SARS-CoV-2.&lt;/span&gt; SARS-CoV-2 contains a positive single-stranded mRNA as genetic material. The 5&#x2019; capped mRNA has a leader sequence (LS), poly-A tail at 3&#x2019; end, and 5&#x2019; and 3&#x2019; UTR. It contains the following genes: ORF1a, ORF1b, Spike (S), ORF3a, ORF3b, Envelope (E), Membrane (M), ORF6, ORF7a, ORF7b, ORF8, ORF9b, ORF14, Nucleocapsid (N), and ORF10.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F1.medium.gif" width="440" height="105"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F1.medium.gif" width="440" height="105"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">Genome architecture of SARS-CoV-2.</span><p id="p-8" class="first-child">SARS-CoV-2 contains a positive single-stranded mRNA as genetic material. The 5’ capped mRNA has a leader sequence (LS), poly-A tail at 3’ end, and 5’ and 3’ UTR. It contains the following genes: ORF1a, ORF1b, Spike (S), ORF3a, ORF3b, Envelope (E), Membrane (M), ORF6, ORF7a, ORF7b, ORF8, ORF9b, ORF14, Nucleocapsid (N), and ORF10.</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">In this study, we analyzed the dark side of the SARS-CoV-2 proteome (i.e., a part of a proteome that includes proteins or protein regions, which are not amenable to experimental structure determination by existing means and inaccessible to homology modelling; i.e., intrinsically disordered proteins (IDPs) and intrinsically disordered protein regions (IDPRs)), in order to better understand an interplay between the ordered and disordered components of the proteome. In classical structure-function-paradigm, it is believed that a unique, stable, and well-defined 3-dimensional structure is a prerequisite for a protein to accomplish its unique biological function. Although this notion dominated scientific minds for over the hundred years, eventually an idea of the presence of functional intrinsic disorder in proteins came to the attention of the structural biologists. According to this “heretic” viewpoint, a noticeable amount of biologically active proteins (of protein regions) fail to fold into the well-defined structures and instead remain disordered, existing as highly dynamic ensembles of rapidly interconverting conformations under the physiological conditions. These proteins and protein regions are known now as intrinsically disordered proteins (IDPs) and intrinsically disordered protein regions (IDPRs), respectively. The propensity of being functional intrinsically disordered proteins (similar to the propensity of forming unique biologically active structures of ordered proteins) is determined by the amino acid sequences [<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>–<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>]. IDPs exhibit their biological functions in numerous biological processes commonly associated with cellular signalling, gene regulation, and control by interacting with their physiological partners [<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>–<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>]. These functions of IDPs and IDPs are regulated by their protein-protein, protein-RNA, protein-DNA interactions [<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>,<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>]. Molecular recognition features (MoRFs) are the regions in IDPs implicated in regulation of IDPs function by protein-protein interactions and serve as the primary stage in molecular recognition.</p><p id="p-10">Zhang and colleagues have reported the genomic sequence of SARS-CoV-2 with GenBank accession number <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=NC_045512&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom">NC_045512</a> having 29,903 nucleotides. The virus was isolated from the bronchoalveolar lavage fluid of a patient, went through a circle or renaming, from novel Wuhan seafood market pneumonia virus to deadly Wuhan coronavirus, to the 2019 novel coronavirus (2019-nCoV) or the Wuhan-2019 novel coronavirus (Wuhan-2019-nCoV, and was eventually named SARS-CoV-2 by the WHO [<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>].</p><p id="p-11">It is known that the IDPs/IDPRs are present in all three kingdoms of life, and viral proteins often contain unstructured regions that have been strongly correlated with their virulence [<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>–<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>]. In this report, we investigated the disordered side of the SARS-CoV-2 proteome using a complementary set of computational approaches to check the prevalence of IDPRs in various SARS-CoV-2 proteins and to shed some light on their disorder-related functions. We also have comprehensively analyzed IDPRs among the closely related viruses, such as human SARS CoV and bat SARS-like CoV. Furthermore, we have also identified protein functions related to protein-protein interactions, RNA binding, and DNA binding from all three viruses. Since these three viruses are closely related, our study provides important means for a better understanding of the sequence and structural peculiarities of their evolution. We believe that this study will help the structural and non-structural biologists to design and perform experiments for a more in-depth understanding of this virus and its pathogenicity. This also will have long-term implications for developing new drugs or vaccines against this currently unpreventable infection.</p></div><div class="section" id="sec-3"><h2 class="">Materials and Methods</h2><div id="sec-4" class="subsection"><h3>Sequence retrieval and multiple sequence alignment</h3><p id="p-12">The protein sequences of Bat CoV (SARS-like) and Human SARS CoV were retrieved from UniProt (UniProt IDs for individual proteins are listed in <strong><a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a></strong>). The translated sequences of SARS-CoV-2 proteins [GenBank database [<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>] (Accession ID: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=NC_045512.2&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom">NC_045512.2</a>) ] were obtained from GenBank. We used these sequences for performing multiple sequence alignment (MSA) and predicting the IDPRs. We have used Clustal Omega [<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>] for protein sequence alignment and Esprit 3.0 [<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>] for constructing the aligned images.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1232231/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/1232231/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1:</span> <p id="p-13" class="first-child">Mean PPID scores of structural and accessory proteins from SARS-CoV-2, Human SARS, and Bat CoVs.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Per-residue predictions of intrinsic disorder predisposition</h3><p id="p-14">For the prediction of the intrinsic disorder predisposition of CoV proteomes, we have used multiple predictors, such as members of the PONDR<sup>®</sup> (Predictor of Natural Disordered Regions) family including PONDR<sup>®</sup>VLS2 [<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>], PONDR<sup>®</sup>VL3 [<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>], PONDR<sup>®</sup>FIT [<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>], and PONDR<sup>®</sup> VLXT [<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>], as well as the IUPred platform for predicting long (≥30 residues) and short IDPRs (,30 residues) [<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>]. These computational tools predict residues/regions, which do not have the tendency to form an ordered structure. Residues with disorder scores exceeding the threshold value of 0.5 are considered as intrinsically disordered residues, whereas residues with the predicted disorder scores between 0.2 and 0.5 are considered flexible. Complete predicted percent of intrinsic disorder (PPID) in a query protein was calculated for every protein of all the three viruses from outputs of six predictors. The detailed methodology has been given in our previous reports [<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>,<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>].</p></div><div id="sec-6" class="subsection"><h3>Combined CH-CDF analysis to predict disorder predisposition of proteins</h3><p id="p-15">Charge hydropathy plot [<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>] and PONDR<sup>®</sup> VLXT-based cumulative distribution function are two binary predictors of disorder (i.e., tool evaluating entire protein as mostly ordered or mostly disordered), which are available on the PONDR web server (<a href="https://www.pondr.com/">https://www.pondr.com/</a>). Combining the result from these binary predictors helps one to classify the proteins into different groups, depending on the flavours of their global disorder [<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>].</p></div><div id="sec-7" class="subsection"><h3>Molecular recognition features (MoRFs) determination in CoV proteomes</h3><p id="p-16">The authentic online bioinformatics predictors that use a different set of algorithms for the prediction of MoRFs were used: These include MoRFchibi_web [<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>], ANCHOR [<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>,<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>], MoRFPred [<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>], and DISOPRED3 [<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>]. The protein residues with ANCHOR, MoRFPred, and DISOPRED3 score above the threshold value of 0.5 and MoRFchibi_web score above the threshold value of 0.725 are considered MoRF regions.</p></div><div id="sec-8" class="subsection"><h3>Identification of DNA and RNA binding regions in CoV proteomes</h3><p id="p-17">Often IDPs and IDPRs facilitate interactions with RNAs and DNAs and regulates many cellular functions [<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>]. Thus, for predicting the DNA binding residues in CoV proteins, we have used two online servers: DRNAPred [<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a>] and DisoRDPbind [<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">43</a>]. For RNA binding residues, we used Pprint (Prediction of Protein RNA-Interaction) [<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>] and DisoRDPbind [<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">43</a>].</p></div></div><div class="section" id="sec-9"><h2 class="">Results and Discussion</h2><div id="sec-10" class="subsection"><h3>Comprehensive computational analysis of the intrinsic disorder in proteins from SARS-CoV-2, human SARS and bat CoV (SARS-like)</h3><p id="p-18">The mean values of the predicted percentage of intrinsic disorder scores (mean PPIDs), that were obtained by averaging the predicted disorder scores from six disorder predictors (<strong>Supplementary Table 1–3</strong>) for each protein of SARS-CoV-2 as well as Human SARS, and Bat CoV are represented in <strong><a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a></strong>.</p><p id="p-19"><strong><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figures 2A, 2B</a></strong>, and <strong><a id="xref-fig-2-2" class="xref-fig" href="#F2">2C</a></strong> are 2D-disorder plots generated for SARS-CoV-2, Human SARS and Bat CoV proteins, respectively, and represent the PPID<sub>PONDR-FIT</sub> <em>vs</em>. PPID<sub>Mean</sub> plots. Based on their predicted levels of intrinsic disorder, proteins can be classified as highly ordered (PPID &lt; 10%), moderately disordered (10% ≤ PPID &lt; 30%) and highly disordered (PPID ≥ 30%) [<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>]. From the data in <strong><a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>, <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figures 2A, 2B</a></strong>, and <strong><a id="xref-fig-2-4" class="xref-fig" href="#F2">2C</a></strong>, as well as the PPID based classification, we conclude that the Nucleocapsid protein from all three strains of coronavirus possesses the highest percentage of the disorder and is classified as highly disordered protein. ORF3b protein in Bat Cov, ORF6 protein in SARS-CoV-2, Human SARS, and Bat CoV, and ORF9b protein in Human SARS and SARS CoV belong to the class of moderately disordered proteins. While the structured proteins, namely, Spike glycoprotein (S), an Envelope protein (E) and Membrane protein (M) as well as accessory proteins ORF3a, ORF7a, ORF8 (ORF8a and ORF8b in case of Human SARS) of all three strains of coronavirus are ordered proteins. ORF14 and ORF10 proteins also belong to the class of ordered proteins.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of overall disorder status of proteins of SARS-CoV&#x2013;2, Human SARS, and Bat CoV (SARS-like): 2D plot representing PPIDPONDR-FIT vs. PPIDMean in (A) SARS-CoV-2 (B) Human SARS and (C) Bat CoV. In CH-CDF plot of the proteins of (D) SARS-CoV-2 (E) Human SARS and (F) Bat CoV, the Y coordinate of each protein spot signifies distance of corresponding protein from the boundary in CH plot and the X coordinate value corresponds to the average distance of the CDF curve for the respective protein from the CDF boundary." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of overall disorder status of proteins of SARS-CoV&#x2013;2, Human SARS, and Bat CoV (SARS-like):&lt;/span&gt; 2D plot representing PPID&lt;sub&gt;PONDR-FIT&lt;/sub&gt; &lt;em&gt;vs&lt;/em&gt;. PPID&lt;sub&gt;Mean&lt;/sub&gt; in &lt;strong&gt;(A)&lt;/strong&gt; SARS-CoV-2 &lt;strong&gt;(B)&lt;/strong&gt; Human SARS and &lt;strong&gt;(C)&lt;/strong&gt; Bat CoV. In CH-CDF plot of the proteins of &lt;strong&gt;(D)&lt;/strong&gt; SARS-CoV-2 &lt;strong&gt;(E)&lt;/strong&gt; Human SARS and &lt;strong&gt;(F)&lt;/strong&gt; Bat CoV, the Y coordinate of each protein spot signifies distance of corresponding protein from the boundary in CH plot and the X coordinate value corresponds to the average distance of the CDF curve for the respective protein from the CDF boundary.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F2.medium.gif" width="440" height="215"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F2.medium.gif" width="440" height="215"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Analysis of overall disorder status of proteins of SARS-CoV–2, Human SARS, and Bat CoV (SARS-like):</span><p id="p-20" class="first-child">2D plot representing PPID<sub>PONDR-FIT</sub> <em>vs</em>. PPID<sub>Mean</sub> in <strong>(A)</strong> SARS-CoV-2 <strong>(B)</strong> Human SARS and <strong>(C)</strong> Bat CoV. In CH-CDF plot of the proteins of <strong>(D)</strong> SARS-CoV-2 <strong>(E)</strong> Human SARS and <strong>(F)</strong> Bat CoV, the Y coordinate of each protein spot signifies distance of corresponding protein from the boundary in CH plot and the X coordinate value corresponds to the average distance of the CDF curve for the respective protein from the CDF boundary.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">In order to further investigate the nature of the disorder in proteins of SARS-CoV-2, Human SARS, and Bat CoV, we utilized the combined CH-CDF tool that uses the outputs of two binary classifiers of disorder, Charge hydropathy (CH) plot and Cumulative distribution function (CDF) plot. This helped in retrieving more detailed characterization of the global disorder predisposition of the query proteins and their classification according to the disorder “favours”. The CH plot is a linear classifier that differentiates between proteins that are predisposed to possess extended disordered conformations that include random coils and pre-molten globules from proteins that have compact conformations (ordered proteins and molten globule-like proteins). The other binary predictor, CDF is a nonlinear classifier that uses the PONDR<sup>®</sup>VLXT scores to discriminate ordered globular proteins from all disordered conformations, which include native molten globules, pre-molten globules, and random coils. The CH-CDF plot can be divided into four quadrants: Q1 (bottom right quadrant) is an area of CH-CDF phase space that is expected to include ordered proteins; Q2 (bottom left quadrant) includes proteins predicted to be disordered by CDF and compact by CH (i.e., native molten globules and hybrid proteins containing high levels of both ordered and disordered regions); Q3 (top left quadrant) contains proteins that are predicted to be disordered by both CH and CDF analysis (i.e., highly disordered proteins with the extended disorder); and Q4 (top right quadrant) possesses proteins disordered according to CH but ordered according to CDF analysis [<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a>]. <strong><a id="xref-fig-2-5" class="xref-fig" href="#F2">Figures 2D, 2E</a></strong> and <strong><a id="xref-fig-2-6" class="xref-fig" href="#F2">2F</a></strong> represent the CH-CDF analysis of proteins of SARS-CoV-2, Human SARS, and Bat CoV and show that all the proteins are located within the two quadrants Q1 and Q2. The CH-CDF analysis leads to the conclusion that all proteins of SARS-CoV-2, Human SARS, and Bat CoV are ordered except Nucleocapsid protein, which is predicted to be disordered by CDF but ordered by CH and hence lies in Q2.</p><p id="p-22">Molecular recognition features (MoRFs) are short interaction-prone disordered regions found within IDPs/IDPRs that commence a disorder-to-order transition upon binding to their partners [<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>,<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>]. These regions are important for protein-protein interactions and may initiate an early step in molecular recognition [<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">48</a>]. In this study, we have analyzed and compared MoRFs (protein-binding regions) in SARS-CoV-2 with Human SARS and Bat CoV. The results of this analysis are summarized in <strong><a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a></strong>, which clearly shows that most of the SARS-CoV-2 proteins contain at least one MoRF, indicating that disorder does play an important role in the functionality of these viral proteins. All of the SARS-CoV-2 proteins have been predicted to contain MoRFs except ORF7b and Nsp13 proteins for which none of the predictors have located any MoRF. MoRFs in Human SARS and Bat CoV proteomes are listed in <strong>Supplementary Tables 7 and 8</strong>. Similar to SARS-CoV-2 proteome, Bat CoV proteins ORF7b, and Nsp13 were not predicted to have any MoRF by any of the servers used. In Human SARS proteome the proteins ORF7b, Nsp13, Nsp2, and Nsp15 proteins does not show the presence of any MoRF. Interestingly, structural N protein from SARS-CoV-2, Human SARS, and Bat CoV shows high number of short and long MoRF regions, signifying its central role in virus pathogenesis.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1232214/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/1232214/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2:</span> <p id="p-23" class="first-child">Predicted MoRFs regions in SARS-CoV-2 proteins</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Nucleotide-binding propensity in proteins of coronaviruses</h3><p id="p-24">In addition to protein-protein interactions/protein-binding functions, IDPs and IDRs also mediates functions by facilitating their interactions with nucleotides such as DNA and RNA [<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a>,<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>]. Therefore, we have used a combination of two different online servers for locating protein residues that are showing the propensity to bind with DNA as well as RNA. Nucleotide-binding residues in proteins of three studied coronaviruses are listed in <strong>Supplementary Table 9-11</strong>. These results mostly contain the residues in viral proteins that suitably bind viral RNA and host DNA. Interestingly, all the viral proteins of SARS-CoV-2, Human SARS, and Bat CoV have shown the propensity to bind to nucleic acid (DNA/RNA). In particular, structural (S, M, and N proteins) and non-structural (Nsp 2, 3, 4, 5, 6, 12, 13, 14, 15, and 16) proteins of all three viruses displays large number RNA binding residues. However, ORF3a, ORF3b, ORF6, ORF7a, ORF7b, ORF8, ORF9b, ORF10, and ORF14 shows less RNA binding and more DNA binding residues.</p></div><div id="sec-12" class="subsection"><h3>Intrinsic disorder analysis of structural proteins of Coronaviruses</h3><p id="p-25">Coronaviruses encode four structural proteins, namely, Spike (S), Envelope (E) glycoprotein, Membrane (M), and Nucleocapsid (N) proteins, which are translated from the last ~10kb nucleotides and form the outer cover of the CoVs, encapsulating their single-stranded genomic RNA.</p><div id="sec-13" class="subsection"><h4>Spike (S) glycoprotein</h4><p id="p-26">S protein is a large multifunctional protein forming the exterior of the CoV particles [<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>,<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>]. It forms surface homotrimers and contains two distinct ectodomain regions known as S1 and S2. In some CoVs, the S protein is actually cleaved into these subunits, which are joined non-covalently, whereas an additional proteolytic cleavage within the N-terminal part of the S2 subunit that takes place upon virus endocytosis generates spike proteins S2’. Subunit S1 initiates viral infection by binding to the host cell receptors, S2 acts as a class I viral fusion protein that mediates fusion of the virion and cellular membranes and thereby promotes the viral entry into the host cells, whereas S2’ serves as a viral fusion peptide [<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>,<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>]. Spike binds to the virion M protein through its C-terminal transmembrane region [<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>]. Belonging to a class I viral fusion protein, S protein binds to specific surface receptor angiotensin-converting enzyme 2 (ACE2) on host cell plasma membrane through its N-terminal receptor-binding domain (RBD) and mediates viral entry into host cells [<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>].</p><p id="p-27">The S protein consists of an N-terminal signal peptide, a long extracellular domain, a single-pass transmembrane domain, and a short intracellular domain [<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>]. A 3.60 Å resolution structure (PDB ID: 6ACC) of S protein from Human SARS complexed with its host binding partner ACE2 has been obtained by cryo-electron microscopy (cryo-EM). In this PDB structure, few residues (1-17, 240-243, 661-673, 812-831 and 1120-1203) are missing [<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>], suggesting their flexible nature. Also, the structure of the S protein (3.5 Å) from SARS-CoV-2 has been recently deduced by Wrapp <em>et. al.</em> using electron microscopy (PDB ID: 6VSB) [<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>] (<strong><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a></strong>). In this structure, residues 1-26, 67-78, 96-98, 143-155, 177-186, 247-260, 329-334, 444-448, 455-490, 501-502, 621-639, 673-686, 812-814, 829-851, 1147-1288 were missing, suggesting that the corresponding regions of a protein are characterized by high conformational flexibility. The biophysical analysis reported in previous study has also revealed that the S protein from SARS-CoV-2 has a higher binding affinity to ACE2 than S protein from Human SARS [<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">58</a>].</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Structure and intrinsic disorder propensity of Spike glycoprotein from CoVs. (A) A 3.50 &#xC5; resolution structure of spike glycoprotein from SARS-CoV-2 obtained by cryo-EM (PDB ID: 6VSB). This homotrimeric structure includes three chains, A (violet-red), B (dark khaki), and C (turquoise). Evaluation of intrinsic disorder predisposition of S protein from SARS-CoV-2 (B), Human SARS (C), and Bat CoV (D). Graphs in (B), (C) and (D) show the disorder profiles generated by PONDR&#xAE; VSL2 (black line), PONDR&#xAE; VL3 (red line), PONDR&#xAE; VLXT (blue line), PONDR&#xAE; FIT (green line), IUPred long (purple line) and IUPred short (golden line). The mean disorder propensity which has been calculated by averaging the disorder scores from all six predictors is represented by a short-dot line (sky-blue line) in the graph. The light sky-blue shadow region signifies the mean error distribution. The residues missing in the PDB structure or the residues for which PDB structure is unavailable are represented by the grey-coloured area in the corresponding graphs. (E) Aligned disorder profiles generated for spike glycoprotein from SARS-CoV-2 (black line), Human SARS (red line), and Bat CoV (green line) based on the outputs of the PONDR&#xAE; VSL2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Structure and intrinsic disorder propensity of Spike glycoprotein from CoVs.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; A 3.50 &#xC5; resolution structure of spike glycoprotein from SARS-CoV-2 obtained by cryo-EM (PDB ID: 6VSB). This homotrimeric structure includes three chains, A (violet-red), B (dark khaki), and C (turquoise). Evaluation of intrinsic disorder predisposition of S protein from SARS-CoV-2 &lt;strong&gt;(B)&lt;/strong&gt;, Human SARS &lt;strong&gt;(C)&lt;/strong&gt;, and Bat CoV &lt;strong&gt;(D)&lt;/strong&gt;. Graphs in &lt;strong&gt;(B), (C)&lt;/strong&gt; and &lt;strong&gt;(D)&lt;/strong&gt; show the disorder profiles generated by PONDR&lt;sup&gt;&#xAE;&lt;/sup&gt; VSL2 (black line), PONDR&lt;sup&gt;&#xAE;&lt;/sup&gt; VL3 (red line), PONDR&lt;sup&gt;&#xAE;&lt;/sup&gt; VLXT (blue line), PONDR&lt;sup&gt;&#xAE;&lt;/sup&gt; FIT (green line), IUPred long (purple line) and IUPred short (golden line). The mean disorder propensity which has been calculated by averaging the disorder scores from all six predictors is represented by a short-dot line (sky-blue line) in the graph. The light sky-blue shadow region signifies the mean error distribution. The residues missing in the PDB structure or the residues for which PDB structure is unavailable are represented by the grey-coloured area in the corresponding graphs. &lt;strong&gt;(E)&lt;/strong&gt; Aligned disorder profiles generated for spike glycoprotein from SARS-CoV-2 (black line), Human SARS (red line), and Bat CoV (green line) based on the outputs of the PONDR&lt;sup&gt;&#xAE;&lt;/sup&gt; VSL2.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F3.medium.gif" width="440" height="268"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F3.medium.gif" width="440" height="268"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Structure and intrinsic disorder propensity of Spike glycoprotein from CoVs.</span><p id="p-28" class="first-child"><strong>(A)</strong> A 3.50 Å resolution structure of spike glycoprotein from SARS-CoV-2 obtained by cryo-EM (PDB ID: 6VSB). This homotrimeric structure includes three chains, A (violet-red), B (dark khaki), and C (turquoise). Evaluation of intrinsic disorder predisposition of S protein from SARS-CoV-2 <strong>(B)</strong>, Human SARS <strong>(C)</strong>, and Bat CoV <strong>(D)</strong>. Graphs in <strong>(B), (C)</strong> and <strong>(D)</strong> show the disorder profiles generated by PONDR<sup>®</sup> VSL2 (black line), PONDR<sup>®</sup> VL3 (red line), PONDR<sup>®</sup> VLXT (blue line), PONDR<sup>®</sup> FIT (green line), IUPred long (purple line) and IUPred short (golden line). The mean disorder propensity which has been calculated by averaging the disorder scores from all six predictors is represented by a short-dot line (sky-blue line) in the graph. The light sky-blue shadow region signifies the mean error distribution. The residues missing in the PDB structure or the residues for which PDB structure is unavailable are represented by the grey-coloured area in the corresponding graphs. <strong>(E)</strong> Aligned disorder profiles generated for spike glycoprotein from SARS-CoV-2 (black line), Human SARS (red line), and Bat CoV (green line) based on the outputs of the PONDR<sup>®</sup> VSL2.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">MSA analysis among all three coronaviruses demonstrates that S protein of SARS-CoV-2 has a 77.71% sequence identity with Bat CoV and 77.14% identity with Human SARS <strong>(Supplementary Figure S1A)</strong>. All three S proteins are found to have a conserved C-terminal region. However, the N-terminal regions of S proteins display noticeable differences. Given that there is significant sequence variation RBD located at the N-terminal region of S protein, this might be the reason behind variation in its virulence and its receptor-mediated binding and entry into the host cell.</p><p id="p-30">According to our intrinsic disorder propensity analysis, S protein from all three CoVs analysed in this study are highly structured, as their predicted disorder propensity lies below 10% (<strong><a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a></strong>). In fact, the mean PPID scores of SARS-CoV-2, Human SARS CoV, and Bat CoV are calculated to be 1.41%, 1.12%, and 1.85%, respectively. <strong><a id="xref-fig-3-2" class="xref-fig" href="#F3">Figures 3B, 3C</a></strong>, and <strong><a id="xref-fig-3-3" class="xref-fig" href="#F3">3D</a></strong> represent the intrinsic disorder profiles of S proteins from SARS-CoV-2, Human SARS and Bat CoV obtained from six disorder predictors. Finally, <strong><a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3E</a></strong> shows aligned disorder profiles of S proteins from these CoVs and illustrates remarkable similarity in their disorder propensity, especially in the C-terminal region.</p><p id="p-31">It is of interest to map known functional regions of S proteins to their corresponding disorder profiles. The maturation of S protein requires specific posttranslational modification (PTM), proteolytic cleavage that happens at two stages. First, host cell furin or another cellular protease nicks the S precursor to generate S1 and S2 proteins, whereas the second cleavage that takes place after the viral attachment to host cell receptors leads to the release of a fusion peptide generating the S2’ subunit. In Human SARS CoV, the first and second cleavage site is located at residues R<sub>667</sub> and R<sub>797</sub>, respectively, whereas in Bat CoV, the corresponding cleavage sites are residues R<sub>654</sub> and R<sub>784</sub>. As it follows from <strong><a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a></strong>, these cleavage sites are located within the IDPRs. In Human SARS CoV S protein, fusion peptide (residues 770-788) is located within a flexible region, is characterized by the mean disorder score of 0.232±0.053. Similarly, in Bat CoV S protein, fusion peptide (residues 757-775) has a mean disorder score of 0.320±0.046. S protein contains two heptad repeat regions that form coiled-coil structure during viral and target cell membrane fusion, assuming a trimer-of-hairpins structure needed for the functional positioning of the fusion peptide. In Human SARS CoV S protein, heptad repeat regions are formed by residues 902-952 and 1145-1184, which have mean disorder scores of 0.458±0.067 and 0.353±0.062, respectively. The analogous situation is observed for the S protein from Bat CoV, where these heptad repeat regions are positioned at residues 889-939 (0.44±0.11) and 1132-1171 (0.353±0.062). Another functional region found in S proteins is the receptor-binding domain (residues 306-527 and 310-514 in Human SARS CoV and Bat CoV, respectively) containing a receptor-binding motif responsible for interaction with human ACE2. In human S protein of Human SARS CoV this motif (residues 424-494) is not only characterized by structural flexibility, possessing a mean disorder score of 0.30±0.16, but also contains a disordered region (residues 461-466). Since S protein is known as spike glycoprotein, it contains numerous glycosylation sites. Due to rather close similarity of disorder profiles of S proteins analysed here, we can assume that all the aforementioned indications of the functional importance of disorder and flexible regions in S proteins from SARS CoV and Bat CoV are also applicable to SARS-CoV-2 S protein. Finally, <strong><a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a></strong> shows that S protein from SARS-CoV-2 contain one MoRF region at its C-terminal (residues 1265-1272) by MoRFchibi_web, two MoRF regions ((residues 2-6) &amp; (residues 819-823)) by MoRFPred, and one MoRF region at N-terminal (residues 1-10) by DISOPRED3. These results indicating that intrinsic disorder is important for its interaction with binding partners. Interestingly, the N-terminal region of S protein (residues 1-10) from all three viruses are observed to be a disorder-based protein binding region by two predictors (MoRFPred and DISOPRED3). N-terminal MoRF displays its role in viral interaction with host receptor and C-terminal MoRF displays its role in M protein interaction and viral assembly. Moreover, MoRF region mainly lies in the N- and C-terminal regions suggesting a possible role during cleavage as well. In addition to protein-binding regions, S protein also shows many nucleotide-binding residues. <strong>Tables 9, 10, and 11</strong> shows that numerous RNA binding residues predicted by PPRint in all three viruses and a single RNA binding residue were predicted by DisoRDPbind in human SARS. Further, DRNApred and DisoRDPbind predicted the presence of many DNA binding residues in S protein of all three viruses. These results signify the role of S protein functions related to molecular recognition (protein-protein interaction, RNA binding, and DNA binding) such as interaction with host cell membrane and further viral infection. Therefore, identified IDPs/IDPRs and residues/regions from S protein crucial for molecular recognition can be targeted for disorder-based drug discovery.</p></div><div id="sec-14" class="subsection"><h4>Envelope (E) small membrane protein</h4><p id="p-32">Envelope (E) protein is a small, multifunctional inner membrane protein that plays an important role in the assembly and morphogenesis of virions in the cell [<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>–<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a>]. E protein consists of two ectodomains associated with N- and C-terminal regions, and a transmembrane domain. It homo-oligomerize to form pentameric membrane destabilizing transmembrane (TM) hairpins to form a pore necessary for its ion channel activity [<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a>]. <strong><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a></strong> shows the NMR-structure (PDB ID: 2MM4) of Human SARS envelope glycoprotein of 8-65 residues [<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a>].</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of structural features and intrinsic disorder predisposition of Envelope glycoprotein. (A) The solution NMR generated structure of envelope glycoprotein of Human SARS (PDB ID: 2MM4). The structure is shown for the region consists of residues 8-65. (B) MSA profile of envelope glycoproteins of SARS-CoV-2, Human SARS CoV and Bat CoV. Red shaded sequences represent identical regions of these proteins. (C-E) graph represents the intrinsic disorder profiles of E protein from SARS-CoV-2 (C), Human SARS CoV (D), and Bat CoV (E). The colour schemes are the same as in the corresponding plots of Figure 3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of structural features and intrinsic disorder predisposition of Envelope glycoprotein.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; The solution NMR generated structure of envelope glycoprotein of Human SARS (PDB ID: 2MM4). The structure is shown for the region consists of residues 8-65. &lt;strong&gt;(B)&lt;/strong&gt; MSA profile of envelope glycoproteins of SARS-CoV-2, Human SARS CoV and Bat CoV. Red shaded sequences represent identical regions of these proteins. (&lt;strong&gt;C-E)&lt;/strong&gt; graph represents the intrinsic disorder profiles of E protein from SARS-CoV-2 &lt;strong&gt;(C)&lt;/strong&gt;, Human SARS CoV &lt;strong&gt;(D)&lt;/strong&gt;, and Bat CoV &lt;strong&gt;(E)&lt;/strong&gt;. The colour schemes are the same as in the corresponding plots of &lt;strong&gt;Figure 3&lt;/strong&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F4.medium.gif" width="440" height="249"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F4.medium.gif" width="440" height="249"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">Analysis of structural features and intrinsic disorder predisposition of Envelope glycoprotein.</span><p id="p-33" class="first-child"><strong>(A)</strong> The solution NMR generated structure of envelope glycoprotein of Human SARS (PDB ID: 2MM4). The structure is shown for the region consists of residues 8-65. <strong>(B)</strong> MSA profile of envelope glycoproteins of SARS-CoV-2, Human SARS CoV and Bat CoV. Red shaded sequences represent identical regions of these proteins. (<strong>C-E)</strong> graph represents the intrinsic disorder profiles of E protein from SARS-CoV-2 <strong>(C)</strong>, Human SARS CoV <strong>(D)</strong>, and Bat CoV <strong>(E)</strong>. The colour schemes are the same as in the corresponding plots of <strong><a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a></strong>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">MSA results illustrate (<strong><a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a></strong>) that this protein is highly conserved, with only three amino acid substitutions in E protein of SARS-CoV-2 conferring its 96% sequence similarity with Human SARS and Bat CoV. Bat CoV shares 100% sequence identity with Human SARS. Mean PPID calculated for SARS-CoV-2, Human SARS, and Bat CoV E proteins are 5.33%, 6.58%, and 6.58% respectively (<strong><a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a></strong>). The E protein is found to have a reasonably well-predicted structure. Our predictions suggest that the residues of N- and C-terminals are displaying a higher tendency for the disorder. The last 18 hydrophilic residues (residues 59-76) have been reported to adopt a random-coil conformation with and without the addition of lipid membranes [<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a>]. Literature suggests that the last four amino acids of the C-terminal region of E protein containing a PZD-binding motif are involved in protein-protein interactions with a tight junction protein PALS1. Our results support literature as we identified long N-terminal region of approximately 30 residues long as disorder-based protein binding region in all three viruses (see <strong><a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>, Supplementary Table 7</strong> and <strong>8</strong>). PALS1 is involved in maintaining the polarity of epithelial cells in mammals [<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a>]. Respective graphs in <strong><a id="xref-fig-4-3" class="xref-fig" href="#F4">Figures 4C, 4D</a></strong>, and <strong><a id="xref-fig-4-4" class="xref-fig" href="#F4">4E</a></strong> show the predicted intrinsic disorder profiles for E proteins of SARS-CoV-2, Human SARS, and Bat CoV. We speculate that the disordered region content may be facilitating the interactions with other proteins as well. In agreement with this hypothesis, <strong><a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a></strong> shows that in E protein from SARS-CoV-2, the C-terminal domain serves as protein-binding region. We found that the residues from 45-75 is a long MoRF in E proteins of all three viruses as predicted by MoRFchibi_web (<strong><a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>, Supplementary Table 7</strong> and <strong>8</strong>). As aforementioned, these randomly-coiled binding-residues at C-terminus may gain structure while assisting the protein-protein interaction mediated by E protein. One more MoRF region (residues 26-30) in the transmembrane domain was observed by DISOPRED3 in the E protein of all three viruses. Since these residues are the part of ion channel, we speculate that these residues do specific interactions and may be guiding the specifi functions of ion channel activity.</p><p id="p-35">Few RNA binding residues by PPRint and DisoRDPbind and several DNA binding residues by DRNApred are predicted for E protein in all three viruses.</p></div><div id="sec-15" class="subsection"><h4>Membrane (M) glycoprotein</h4><p id="p-36">Membrane (M) glycoprotein plays an important role in virion assembly by interacting with the nucleocapsid (N) and E proteins [<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">66</a>–<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">68</a>]. Protein M interacts specifically with a short viral packaging signal containing coronavirus RNA in the absence of N protein, thereby highlighting an important nucleocapsid-independent viral RNA packaging mechanism inside the host cells [<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">69</a>]. It gains high-mannose N-glycans in ER, which are subsequently modified into complex N-glycans in the Golgi complex. Glycosylation of M protein is observed to be not essential for virion fusion in cell culture [<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">70</a>,<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">71</a>].</p><p id="p-37">Cryo-EM and Tomography data indicate that M forms two distinct conformations, a compact M protein having high flexibility and low spike density, and an elongated M protein having a rigid structure and narrow range of membrane curvature [<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">72</a>]. Some regions of M glycoproteins might serve as important dominant immunogens. Although no structural information is available for the full-length M protein as of yet, a short peptide of the membrane glycoprotein (residues 88-96) from Human SARS CoV was co-crystallized with a complex between A-2 alpha chain of the HLA class I histocompatibility antigen and β2-microglobulin (PDB ID: 3I6G) [<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">73</a>]. <strong><a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a></strong> shows that within this complex, the co-crystallized M protein region exists in an extended conformation.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of intrinsic disorder propensity of Membrane glycoprotein. (A) A 2.20 &#xC5; resolution crystal structure of a complex between the A-2 alpha chain of HLA class I histocompatibility antigen, &#x3B2;2-microglobulin and the immunogen peptide of the membrane glycoprotein (9 amino acids) from Human SARS CoV (PDB ID: 3I6G). This complex crystallized is a dimer. Therefore, each member is present here in two copies. Chains corresponding to the fragment of the membrane glycoprotein (residues 88-96) are shown by red and pink colours, A-2 alpha chain and &#x3B2;2-microglobulin in this dimer of heterotrimers structure are shown as blue/ice-blue and tan/yellow surfaces. (B) MSA profile of Membrane glycoproteins of SARS-CoV-2, Human SARS and Bat CoV (SARS-like). Red shaded sequences represent identical regions of these proteins. (C-E) graphs represent the intrinsic disorder profiles of M protein from SARS-CoV-2, (D) Human SARS CoV, and (E) Bat CoV. The colour schemes are the same as in the corresponding plots of Figure 3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of intrinsic disorder propensity of Membrane glycoprotein.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; A 2.20 &#xC5; resolution crystal structure of a complex between the A-2 alpha chain of HLA class I histocompatibility antigen, &#x3B2;2-microglobulin and the immunogen peptide of the membrane glycoprotein (9 amino acids) from Human SARS CoV (PDB ID: 3I6G). This complex crystallized is a dimer. Therefore, each member is present here in two copies. Chains corresponding to the fragment of the membrane glycoprotein (residues 88-96) are shown by red and pink colours, A-2 alpha chain and &#x3B2;2-microglobulin in this dimer of heterotrimers structure are shown as blue/ice-blue and tan/yellow surfaces. &lt;strong&gt;(B)&lt;/strong&gt; MSA profile of Membrane glycoproteins of SARS-CoV-2, Human SARS and Bat CoV (SARS-like). Red shaded sequences represent identical regions of these proteins. &lt;strong&gt;(C-E)&lt;/strong&gt; graphs represent the intrinsic disorder profiles of M protein from SARS-CoV-2, &lt;strong&gt;(D)&lt;/strong&gt; Human SARS CoV, and &lt;strong&gt;(E)&lt;/strong&gt; Bat CoV. The colour schemes are the same as in the corresponding plots of &lt;strong&gt;Figure 3&lt;/strong&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F5.medium.gif" width="440" height="244"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F5.medium.gif" width="440" height="244"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <span class="caption-title">Analysis of intrinsic disorder propensity of Membrane glycoprotein.</span><p id="p-38" class="first-child"><strong>(A)</strong> A 2.20 Å resolution crystal structure of a complex between the A-2 alpha chain of HLA class I histocompatibility antigen, β2-microglobulin and the immunogen peptide of the membrane glycoprotein (9 amino acids) from Human SARS CoV (PDB ID: 3I6G). This complex crystallized is a dimer. Therefore, each member is present here in two copies. Chains corresponding to the fragment of the membrane glycoprotein (residues 88-96) are shown by red and pink colours, A-2 alpha chain and β2-microglobulin in this dimer of heterotrimers structure are shown as blue/ice-blue and tan/yellow surfaces. <strong>(B)</strong> MSA profile of Membrane glycoproteins of SARS-CoV-2, Human SARS and Bat CoV (SARS-like). Red shaded sequences represent identical regions of these proteins. <strong>(C-E)</strong> graphs represent the intrinsic disorder profiles of M protein from SARS-CoV-2, <strong>(D)</strong> Human SARS CoV, and <strong>(E)</strong> Bat CoV. The colour schemes are the same as in the corresponding plots of <strong><a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3</a></strong>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">M protein of SARS-CoV-2 has a sequence similarity of 90.1% with Bat CoV and 89.6% with Human SARS M proteins (<strong><a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a></strong>). Our analysis revealed that the intrinsic disorder levels in M proteins of SARS-COV-2, Human SARS CoV, and Bat CoV are relatively low since these proteins show the PPID values of 2.70%, 1.36%, and 1.36% respectively. This is in line with the previous publication by Goh <em>et al.</em> on Human SARS HKU4 where they found the mean PPID of 4% using additional predictors such as TopIDP and FoldIndex along with the predictors used in our study [<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">74</a>]. <strong><a id="xref-fig-5-3" class="xref-fig" href="#F5">Figures 5C, 5D</a></strong>, and <strong><a id="xref-fig-5-4" class="xref-fig" href="#F5">5E</a></strong> represent per-residue disorder profiles generated for M proteins of SARS-CoV-2, Human SARS CoV, and Bat CoV and show that with the exception to their N- and C-terminal regions, these proteins are mostly ordered. The last 20 residues of MERS-CoV M protein are important for intracellular trafficking and contains a determinant that localizes it into the Golgi network [<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">75</a>]. Our results in <strong><a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a></strong> illustrates that the disordered C-tail of the M protein is predicted to have disorder based protein-binding region and therefore can serve as a binding site for its specific partner required for its localization inside the host cell. A long MoRF region (residues 186-220) at the C-terminal of M protein in all three viruses were observed by MoRFchibi_web. Two MoRF regions (one at N-terminus (residues 1-16) and one at C-terminus (residues 205-221)) was observed by DISOPRED3 in human SARS and Bat CoV. However, single MoRF (residues 117-132) observed in SARS-CoV-2 by DISOPRED3. MoRFPred also predicts a short MoRF at C-terminus of SARS-CoV-2 (residues 214-222), Human SARS (residues 214-221), and Bat CoV (residues 211-221) (<strong><a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a>, Supplementary Tables 7 and 8</strong>). Furthermore, the M protein from all three viruses displays strong tendency to bind with RNA (as predicted by PPRint and DisoRDPbind) and DNA (as predicted by DRNApred and DisoRDPbind) (see <strong>Supplementary Tables 9, 10, and 11</strong>). Our understanding on M protein of CoVs (IDPs and MoRF at C-terminus and molecular recognition) elucidates its crucial role in interaction with the N and E proteins for viral assembly.</p></div><div id="sec-16" class="subsection"><h4>Nucleocapsid (N) protein</h4><p id="p-40">Nucleocapsid (N) protein is one of the major viral proteins playing several significant roles in transcription, and virion assembly of coronaviruses [<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">76</a>]. It binds to viral genomic RNA forming a ribonucleoprotein core required for the RNA encapsidation during viral particle assembly [<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">77</a>]. SARS-CoV virus-like particles (VLPs) formation has been reported to depend upon either M and E proteins or M and N proteins. For the effective production and release of VLPs, co-expression of E or N proteins with M protein is necessary [<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">78</a>].</p><p id="p-41">N protein of Human SARS consists of two structural domains, the N-terminal RNA-binding domain (NTD: 45-181 residues) and the C-terminal dimerization domain (CTD: 248-365 residues) with a disordered patch in between these domains. N protein has been demonstrated to bind viral RNA using both NTD and CTD [<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">79</a>]. <strong><a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A1</a></strong> displays the NMR solution structure of the NTD of Human SARS CoV nucleocapsid protein (45-181 residues) (PDB ID: 1SSK) [<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">80</a>]. <strong><a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6A2</a></strong> shows an X-ray crystal structure of the CTD of Human SARS CoV nucleocapsid protein (270-366 residues) (PDB ID: 2GIB) [<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">81</a>]. A model of the domain organization of the N-protein from SARS-CoV-2 is shown in <strong><a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B</a></strong>.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of structural properties and intrinsic disorder propensity of the coronaviral Nucleocapsid protein. (A1) NMR solution structure of the NTD (residues 45-181) of nucleocapsid from Human SARS CoV (PDB ID: 1SSK). (A2) X-ray crystal structure with the 1.75&#xC5; resolution of the CTD (residues 270-366) of nucleocapsid protein from Human SARS (PDB ID: 2GIB). The structure is a homodimer of chains A (violet-red) and B (dark khaki). Residues 270-289 and 362-366 have shown disorder propensity in the available structures (forest green colour). (B) Representation of the predicted disordered regions of SARS-CoV-2 on N protein. (C-E) graphs represent the intrinsic disorder profiles of N protein from (C) SARS-CoV-2, (D) Human SARS CoV, and (E) Bat CoV. The colour schemes are the same as in corresponding plots of Figure 3. (F) Aligned disorder profiles generated for N protein from SARS-CoV-2 (black line), Human SARS CoV (red line), and Bat CoV (green line) based on the outputs of the PONDR&#xAE; VSL2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of structural properties and intrinsic disorder propensity of the coronaviral Nucleocapsid protein.&lt;/span&gt; &lt;strong&gt;(A1)&lt;/strong&gt; NMR solution structure of the NTD (residues 45-181) of nucleocapsid from Human SARS CoV (PDB ID: 1SSK). &lt;strong&gt;(A2)&lt;/strong&gt; X-ray crystal structure with the 1.75&#xC5; resolution of the CTD (residues 270-366) of nucleocapsid protein from Human SARS (PDB ID: 2GIB). The structure is a homodimer of chains A (violet-red) and B (dark khaki). Residues 270-289 and 362-366 have shown disorder propensity in the available structures (forest green colour). &lt;strong&gt;(B)&lt;/strong&gt; Representation of the predicted disordered regions of SARS-CoV-2 on N protein. &lt;strong&gt;(C-E)&lt;/strong&gt; graphs represent the intrinsic disorder profiles of N protein from &lt;strong&gt;(C)&lt;/strong&gt; SARS-CoV-2, &lt;strong&gt;(D)&lt;/strong&gt; Human SARS CoV, and &lt;strong&gt;(E)&lt;/strong&gt; Bat CoV. The colour schemes are the same as in corresponding plots of &lt;strong&gt;Figure 3&lt;/strong&gt;. &lt;strong&gt;(F)&lt;/strong&gt; Aligned disorder profiles generated for N protein from SARS-CoV-2 (black line), Human SARS CoV (red line), and Bat CoV (green line) based on the outputs of the PONDR&lt;sup&gt;&#xAE;&lt;/sup&gt; VSL2.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F6.medium.gif" width="440" height="392"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F6.medium.gif" width="440" height="392"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6.</span> <span class="caption-title">Analysis of structural properties and intrinsic disorder propensity of the coronaviral Nucleocapsid protein.</span><p id="p-42" class="first-child"><strong>(A1)</strong> NMR solution structure of the NTD (residues 45-181) of nucleocapsid from Human SARS CoV (PDB ID: 1SSK). <strong>(A2)</strong> X-ray crystal structure with the 1.75Å resolution of the CTD (residues 270-366) of nucleocapsid protein from Human SARS (PDB ID: 2GIB). The structure is a homodimer of chains A (violet-red) and B (dark khaki). Residues 270-289 and 362-366 have shown disorder propensity in the available structures (forest green colour). <strong>(B)</strong> Representation of the predicted disordered regions of SARS-CoV-2 on N protein. <strong>(C-E)</strong> graphs represent the intrinsic disorder profiles of N protein from <strong>(C)</strong> SARS-CoV-2, <strong>(D)</strong> Human SARS CoV, and <strong>(E)</strong> Bat CoV. The colour schemes are the same as in corresponding plots of <strong><a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3</a></strong>. <strong>(F)</strong> Aligned disorder profiles generated for N protein from SARS-CoV-2 (black line), Human SARS CoV (red line), and Bat CoV (green line) based on the outputs of the PONDR<sup>®</sup> VSL2.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">The 419 amino acid-long N protein of SARS-CoV-2 shows a percentage identity of 88.76% with N protein of Bat CoV N protein and 89.74% with Human SARS N protein (<strong>Supplementary Figure S1B)</strong>. Our analysis revealed that the N proteins of coronaviruses contain the highest levels of intrinsic disorder (see <strong><a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6</a></strong> and <strong><a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a></strong>). In fact, N proteins from SARS-COV-2 Human SARS CoV, and Bat CoV are characterized by the mean PPID of 64.91%, 71.09%, and 65.80%, respectively. In accordance with the previously evaluated intrinsic disorder predisposition [<a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">74</a>], N protein is highly disordered in all three SARS viruses analysed in this study (<strong><a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a></strong>). Graphs in <strong><a id="xref-fig-6-5" class="xref-fig" href="#F6">Figures 6C, 6D</a></strong>, and <strong><a id="xref-fig-6-6" class="xref-fig" href="#F6">6E</a></strong> depict the disorder profiles of SARS-CoV-2, Human SARS CoV, and Bat CoV nucleocapsid proteins and show that their N- and C-terminal regions are completely disordered, and all three proteins also contain the central unstructured segment. As expected, the intrinsic disorder predisposition of the N protein of SARS-CoV-2 is remarkably similar to that for the N protein of Human SARS CoV as reported in a previous study [<a id="xref-ref-74-3" class="xref-bibr" href="#ref-74">74</a>]. This is further supported by <strong><a id="xref-fig-6-7" class="xref-fig" href="#F6">Figure 6F</a></strong>, where PONDR<sup>®</sup> VSL2-generated disorder profiles of these three proteins are overlapped to show almost complete coincidence of their major disorder-related features. It is clear that in N-proteins, the N- and C-termini and a log central segment are completely disordered. <strong><a id="xref-fig-6-8" class="xref-fig" href="#F6">Figure 6C</a></strong> shows that in the N protein from SARS-CoV-2, residues 1-57, 64-102, 145-162, 166-289, and 362-422 are found to be disordered. Many of these residues are lying within the NTD and CTD regions, and which, due to their structural plasticity, were not crystallized in Human SARS CoV N protein. SARS-CoV-2 has a disordered segment from 168-289 residues while Human SARS has predicted to have an unstructured segment from 145-289 residues. Overall, all three N proteins are found to be highly disordered.</p><p id="p-44">The N protein from Human SARS CoV has one phosphorylation site (residue S<sub>177</sub>) and several regions with compositional biases, such as Ser-rich (residues 181-213), Poly-Leu, Poly-Gln, and Ploy-Lys (residues 220-225, 240-245, and 370-376), all predicted to be disordered. Similarly, in N protein from Bat CoV, S<sub>176</sub> is phosphorylated, and this protein has Ser-rich, Poly-Leu, and Ploy-Lys regions (residues 176-206, 219-224, and 369-375, respectively), all of which are disordered. It has been reported to interact using the central disordered region with M protein, hnRNP A1, and self N-N interaction [<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">82</a>–<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">84</a>]. The middle flexible region is also responsible for its RNA-binding activity [<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">85</a>]. Deletion of 184-196 residues, 169-308 residues, 161-210 residues of N abolishes its multimerization, RNA-binding capacity, and hnRNP A1 interactions respectively. <strong>Supplementary Table 7</strong> and <strong>8</strong>, and <strong><a id="xref-table-wrap-2-8" class="xref-table" href="#T2">Table 2</a></strong> shows that N protein is heavily decorated with numerous MoRFs, suggesting that this protein is a promiscuous binder. Long disorder-based protein bonding regions at N- and C-terminus of N protein of all three viruses were observed by all four predictors (MoRFchibi_web, ANCHOR, MoRFPred, and DISOPRED3). Indeed, this is the single protein where we found many MoRFs as compared with the other structural, non-structural and accessory proteins of CoVs. The MoRFs present in these regions may mediate the above-mentioned interactions of N proteins. <strong><a id="xref-fig-6-9" class="xref-fig" href="#F6">Figure 6A2</a></strong> represents another important disorder-related functional feature of the N protein. In fact, the CTD homodimer shown there is characterized by highly intertwined morphology, which is typically a result of binding-induced folding [<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">86</a>–<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">88</a>], indicating that a very significant part of CTD gains structure during dimerization. We identified numerous RNA binding residues in all three viruses using PPRint server. This finding supports the function of N protein as it interacts with genomic RNA for a ribonucleoprotein core formation which is crucial step for RNA encapsidation during viral particle assembly. In addition, DRNApred and DisoRDPbind predicts multiple DNA binding residues for N protein in SARS-CoV-2, Human SARS, and Bat CoV. The long flexible (IDPRs) regions at N and C-terminus of SARS-CoV-2 have long protein-binding as well as nucleotide-binding regions that may have important role in its interaction with viral RNA. These flexible regions can be targeted to inhibit interaction of N protein with viral genomic RNA.</p></div></div><div id="sec-17" class="subsection"><h3>Intrinsic disorder analysis of accessory proteins of Coronaviruses</h3><p id="p-45">Literature suggests that some viral proteins are translated from the genes interspersed in between the genes of structural proteins. These proteins are known as accessory proteins, and many of them are proposed to be involved in viral pathogenesis [<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">89</a>].</p><div id="sec-18" class="subsection"><h4>Proteins ORF3a and ORF3b</h4><p id="p-46">ORF3a is a multifunctional protein with the molecular weight of ~31 kDa that has been found to localize in different organelles inside the host cells. Also referred to as <em>U274</em>, <em>X1</em>, and <em>ORF3</em>, the gene for this protein is present between the <em>S</em> and <em>E</em> genes of the SARS-CoV genome [<a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">90</a>–<a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">92</a>]. The homo-tetrameric complex of ORF3a has been demonstrated to form a potassium-ion channel on the host cell plasma membrane [<a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">93</a>]. It performs a major function during virion assembly by co-localizing with E, M, and S viral proteins [<a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">94</a>,<a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">95</a>]. ORF3b protein can be found in the cytoplasm, nucleolus, and outer membrane of mitochondria of the host cells [<a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">96</a>,<a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">97</a>]. In Huh 7 cells, its over-expression has been linked with the activation of AP-1 via ERK and JNK pathways [<a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">98</a>]. Transfection of ORF3b-EGFP leads to cell growth arrest at the G0/G1 phase of Vero, 293, and COS-7 cells [<a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">99</a>]. ORF3a induces apoptosis via caspase 8/9 directed mitochondrial-mediated pathways, while ORF3b is reported to affect only the caspase 3-related pathways [<a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">100</a>,<a id="xref-ref-101-1" class="xref-bibr" href="#ref-101">101</a>].</p><p id="p-47">On performing MSA, results of which are shown in <strong><a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7D</a></strong>, we found that ORF3a protein from SARS-COV-2 is slightly evolutionary closer to the ORF3a of Bat CoV (73.36%) than to the ORF3a of Human SARS CoV (72.99%). Graphs in <strong><a id="xref-fig-7-2" class="xref-fig" href="#F7">Figures 7A, 7B</a></strong>, and <strong><a id="xref-fig-7-3" class="xref-fig" href="#F7">7C</a></strong> depict the propensity for disorder in ORF3a proteins of novel SARS-CoV-2, Human SARS CoV, and Bat CoV (SARS-like), respectively. Mean PPIDs in these ORF3a proteins are 9.1% (SARS-CoV-2), 8.8% (Human SARS), and 6.2% (Bat CoV (SARS-like)). ORF3a of SARS CoV-2 shows protein-binding regions at its N-terminus (by MoRFchibi_web (residues 1-6), MoRFPred (residues 7-12), and DISOPRED3 (residues 1-19)) and at C-terminus (by MoRFchibi_web (residues 261-268) and MoRFPred (residues 259-263)) (<strong><a id="xref-table-wrap-2-9" class="xref-table" href="#T2">Table 2</a></strong>). Similarly, ORF3a of Human SARS and Bat CoV also shows MoRFs at N- and C-terminus with the help of MoRFchibi_web and MoRFPred (<strong>Supplementary Tables 7 and 8</strong>). These protein-binding regions in ORF3a may have role in its co-localization with E, M, and S viral proteins. Apart from MoRFs, it also displays several nucleotide-binding residues in all three viruses (see <strong>Supplementary Tables 9, 10, and 11</strong>). In fact, this represents maximum number of RNA and DNA binding residues as compared with all other accessory proteins. These results indicate that the IDPs/IDPRs of this protein could be utilized in molecular recognition (protein-protein, protein-RNA, and protein-DNA interaction).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of intrinsic disorder propensity of ORF3a protein. (A-C) graphs represent the intrinsic disorder profiles of ORF3a protein from (A) SARS-CoV-2, (B) Human SARS CoV, and (C) Bat CoV. The colour schemes are the same as in the corresponding plots of Figure 3. (D) MSA profile of ORF3a proteins from SARS-CoV-2, Human SARS CoV, and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of intrinsic disorder propensity of ORF3a protein.&lt;/span&gt; &lt;strong&gt;(A-C)&lt;/strong&gt; graphs represent the intrinsic disorder profiles of ORF3a protein from &lt;strong&gt;(A)&lt;/strong&gt; SARS-CoV-2, &lt;strong&gt;(B)&lt;/strong&gt; Human SARS CoV, and &lt;strong&gt;(C)&lt;/strong&gt; Bat CoV. The colour schemes are the same as in the corresponding plots of &lt;strong&gt;Figure 3. (D)&lt;/strong&gt; MSA profile of ORF3a proteins from SARS-CoV-2, Human SARS CoV, and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F7.medium.gif" width="440" height="314"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F7.medium.gif" width="440" height="314"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7.</span> <span class="caption-title">Analysis of intrinsic disorder propensity of ORF3a protein.</span><p id="p-48" class="first-child"><strong>(A-C)</strong> graphs represent the intrinsic disorder profiles of ORF3a protein from <strong>(A)</strong> SARS-CoV-2, <strong>(B)</strong> Human SARS CoV, and <strong>(C)</strong> Bat CoV. The colour schemes are the same as in the corresponding plots of <strong><a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3</a>. (D)</strong> MSA profile of ORF3a proteins from SARS-CoV-2, Human SARS CoV, and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues.</p><div class="sb-div caption-clear"></div></div></div><p id="p-49">According to the intrinsic disorder predisposition analysis of ORF3b proteins, their mean PPID values in SARS-CoV-2, Human SARS CoV, and Bat CoV are 0%, 7.1%, and 23.1% respectively, as represented in <strong><a id="xref-fig-8-1" class="xref-fig" href="#F8">Figures 8A, 8B</a></strong>, and <strong><a id="xref-fig-8-2" class="xref-fig" href="#F8">8C</a></strong>. MSA results (<strong><a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8D</a></strong>) demonstrate that ORF3b of SARS-CoV-2 is not closer to ORF3b protein of Human SARS and ORF3b protein of Bat-CoV, having a sequence similarity of only 54.6% and 59.1%, respectively. As we can see in <strong><a id="xref-table-wrap-2-10" class="xref-table" href="#T2">Table 2</a></strong>, there is not a single MoRF found in ORF3b of SARS-CoV-2. However, for Human SARS we identified three MoRFs (residues 32-37, 41-70, and 125-153) and for Bat CoV one MoRF at N-terminus (residues 1-38) by MoRFchibi_web server.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of intrinsic disorder propensity of ORF 3b protein of Human SARS CoV. (A) The graph represents the results of intrinsic disorder analysis in SARS-CoV-2, (B) Human SARS, and (C) Bat CoV. The colour schemes are the same as in the corresponding plots of Figure 3. (D) MSA profile of ORF3b proteins from SARS-CoV-2, Human SARS CoV, and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of intrinsic disorder propensity of ORF 3b protein of Human SARS CoV.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; The graph represents the results of intrinsic disorder analysis in SARS-CoV-2, &lt;strong&gt;(B)&lt;/strong&gt; Human SARS, and &lt;strong&gt;(C)&lt;/strong&gt; Bat CoV. The colour schemes are the same as in the corresponding plots of &lt;strong&gt;Figure 3. (D)&lt;/strong&gt; MSA profile of ORF3b proteins from SARS-CoV-2, Human SARS CoV, and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F8.medium.gif" width="440" height="236"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F8.medium.gif" width="440" height="236"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8.</span> <span class="caption-title">Analysis of intrinsic disorder propensity of ORF 3b protein of Human SARS CoV.</span><p id="p-50" class="first-child"><strong>(A)</strong> The graph represents the results of intrinsic disorder analysis in SARS-CoV-2, <strong>(B)</strong> Human SARS, and <strong>(C)</strong> Bat CoV. The colour schemes are the same as in the corresponding plots of <strong><a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3</a>. (D)</strong> MSA profile of ORF3b proteins from SARS-CoV-2, Human SARS CoV, and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h4>Protein ORF6</h4><p id="p-51">ORF6 is a short coronavirus protein with just 63 residues. Also known as P6, this membrane-associated protein serves as an interferon (IFN) antagonist [<a id="xref-ref-102-1" class="xref-bibr" href="#ref-102">102</a>]. It downregulates the IFN pathway by blocking a nuclear import protein, karyopherin α2. Using its C-terminal residues, ORF6 disrupts karyopherin import complex in the cytosol and, therefore, hampers the movement of transcription factors like STAT1 into the nucleus [<a id="xref-ref-102-2" class="xref-bibr" href="#ref-102">102</a>,<a id="xref-ref-103-1" class="xref-bibr" href="#ref-103">103</a>]. It contains a YSEL motif near its C-terminal region, which functions in protein internalization from the plasma membrane into the endosomal vesicles [<a id="xref-ref-104-1" class="xref-bibr" href="#ref-104">104</a>]. Another study has also demonstrated the presence of ORF6 in endosomal/lysosomal compartments [<a id="xref-ref-104-2" class="xref-bibr" href="#ref-104">104</a>,<a id="xref-ref-105-1" class="xref-bibr" href="#ref-105">105</a>].</p><p id="p-52">MSA results demonstrate that (<strong><a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9D</a></strong>), SARS-CoV-2 ORF6 is closer to ORF6 protein of Human SARS CoV, having a sequence similarity of 68.85% than to the ORF6 of Bat CoV (SARS-like) (67.21%). Novel SARS-CoV-2 ORF6 is predicted to be the second most disordered structural protein, with PPID of 22.95%, and with especially disordered C-terminal region.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of intrinsic disorder propensity of ORF6 protein. Graphs represent the intrinsic disorder analysis of ORF6 proteins from SARS-CoV-2 (A), Human SARS (B), and Bat CoV (C). The colour schemes are the same as in the corresponding plots of Figure 3. (D) MSA profile of ORF6 of SARS-CoV-2, Human SARS CoV, and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of intrinsic disorder propensity of ORF6 protein.&lt;/span&gt; Graphs represent the intrinsic disorder analysis of ORF6 proteins from SARS-CoV-2 &lt;strong&gt;(A)&lt;/strong&gt;, Human SARS &lt;strong&gt;(B)&lt;/strong&gt;, and Bat CoV &lt;strong&gt;(C)&lt;/strong&gt;. The colour schemes are the same as in the corresponding plots of &lt;strong&gt;Figure 3. (D)&lt;/strong&gt; MSA profile of ORF6 of SARS-CoV-2, Human SARS CoV, and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F9.medium.gif" width="440" height="237"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F9.medium.gif" width="440" height="237"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9.</span> <span class="caption-title">Analysis of intrinsic disorder propensity of ORF6 protein.</span><p id="p-53" class="first-child">Graphs represent the intrinsic disorder analysis of ORF6 proteins from SARS-CoV-2 <strong>(A)</strong>, Human SARS <strong>(B)</strong>, and Bat CoV <strong>(C)</strong>. The colour schemes are the same as in the corresponding plots of <strong><a id="xref-fig-3-11" class="xref-fig" href="#F3">Figure 3</a>. (D)</strong> MSA profile of ORF6 of SARS-CoV-2, Human SARS CoV, and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues.</p><div class="sb-div caption-clear"></div></div></div><p id="p-54">Our analysis of the intrinsic disorder predisposition using six predictors revealed the mean PPID in ORF6 proteins of SARS-CoV-2, Human SARS, and Bat CoV to be 22.95%, 20.63%, and 20.63%, respectively (<strong><a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a></strong>). Graphs in <strong><a id="xref-fig-9-2" class="xref-fig" href="#F9">Figures 9A, 9B</a></strong> and <strong><a id="xref-fig-9-3" class="xref-fig" href="#F9">9C</a></strong> illustrate that ORF6 proteins from all three studied coronaviruses are expected to be moderately disordered proteins with the high disorder content in their C-terminal regions. These disordered regions are important for the biological activities of ORF6. As aforementioned, this hydrophilic region contains lysosomal targeting motif (YSEL) and diacidic motif (DDEE) responsible for binding and recognition during translocation [<a id="xref-ref-104-3" class="xref-bibr" href="#ref-104">104</a>]. However, the N-terminal region does not contain a noticeable disorder. The 1-38 residues of the N-terminal region of Human SARS CoV ORF6 was shown to be α-helical and embedded in the membrane, although ORF6 is not a transmembrane protein [<a id="xref-ref-106-1" class="xref-bibr" href="#ref-106">106</a>]. A long MoRF region ((residues 26-61 in SARS-CoV-2), (residues 31-63 in Human SARS), and (residues 30-60 in Bat Cov)) is also present at C-terminus of ORF6 proteins which are tabulated in <strong><a id="xref-table-wrap-2-11" class="xref-table" href="#T2">Table 2</a></strong>, and <strong>Supplementary Tables 7</strong> and <strong>8</strong>. No predictor other than MoRFchibi_web has located MoRFs in this protein. <strong>Supplementary Table 9, 10, and 11</strong> shows nucleotide-binding residues in ORF6 of all three viruses. It represents very few RNA binding residues by PPRint and few DNA binding residues by DRNApred.</p></div><div id="sec-20" class="subsection"><h4>ORF7a and ORF7b proteins</h4><p id="p-55">Alternatively called U122, ORF7a is a type I transmembrane protein [<a id="xref-ref-107-1" class="xref-bibr" href="#ref-107">107</a>,<a id="xref-ref-108-1" class="xref-bibr" href="#ref-108">108</a>]. It has been proven to localize in ER, Golgi, and peri-nuclear space. The presence of a KRKTE motif near the C-terminal region is needed for importing this protein from the ER to the Golgi apparatus [<a id="xref-ref-107-2" class="xref-bibr" href="#ref-107">107</a>,<a id="xref-ref-108-2" class="xref-bibr" href="#ref-108">108</a>]. ORF7a contributes to viral pathogenesis by activating the release of pro-inflammatory cytokines and chemokines, such as IL-8 and RANTES [<a id="xref-ref-109-1" class="xref-bibr" href="#ref-109">109</a>,<a id="xref-ref-110-1" class="xref-bibr" href="#ref-110">110</a>]. In another study, overexpression of BCL-XL in 293T cells blocked the ORF7a mediated apoptosis [<a id="xref-ref-111-1" class="xref-bibr" href="#ref-111">111</a>]. On the other hand, ORF7b is an integral membrane protein that has been shown to localize in the Golgi complex [<a id="xref-ref-112-1" class="xref-bibr" href="#ref-112">112</a>,<a id="xref-ref-113-1" class="xref-bibr" href="#ref-113">113</a>]. The same reports also confirm the role of ORF7b as an accessory as well as a structural protein in SARS-CoV virion [<a id="xref-ref-112-2" class="xref-bibr" href="#ref-112">112</a>,<a id="xref-ref-113-2" class="xref-bibr" href="#ref-113">113</a>].</p><p id="p-56"><strong><a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure 10D</a></strong> represents the 1.8 Å X-ray crystal structure of the 14-96 fragment of the ORF7a from Human SARS CoV (PDB ID: 1XAK) and demonstrates the compact seven-stranded <em>β</em> topology of this protein, which is similar to that of the Ig-superfamily members [<a id="xref-ref-114-1" class="xref-bibr" href="#ref-114">114</a>]. Importantly, in this crystal structure, residues 82-96 constituted the region with missing electron density, indicating high structural flexibility of this segment. In line with this hypothesis, the NMR solution structure of the 16-99 fragment of the ORF7a from Human SARS CoV (PDB ID: 1YO4) showed that residues 81-99 are highly disordered [<a id="xref-ref-115-1" class="xref-bibr" href="#ref-115">115</a>]. At the domain level, the structure of the ORF7a protein includes a signal peptide, a luminal domain, a transmembrane domain, and a short cytoplasmic tail at the C-terminus [<a id="xref-ref-95-2" class="xref-bibr" href="#ref-95">95</a>,<a id="xref-ref-114-2" class="xref-bibr" href="#ref-114">114</a>].</p><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of intrinsic disorder propensity of ORF7a protein. The graphs represent the results of the intrinsic disorder analysis of ORF7a protein from SARS-CoV-2 (A), Human SARS (B), and Bat CoV (C). The colour schemes are the same as in the corresponding plots of Figure 3. (D) A 1.8 &#xC5; resolution X-ray structure of ORF7a from Human SARS CoV (PDB ID: 1XAK). It consists of only A chain of 14-96 residues (violet colour). (E) MSA profile of ORF7a from SARS-CoV-2, Human SARS CoV, and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of intrinsic disorder propensity of ORF7a protein.&lt;/span&gt; The graphs represent the results of the intrinsic disorder analysis of ORF7a protein from SARS-CoV-2 &lt;strong&gt;(A)&lt;/strong&gt;, Human SARS &lt;strong&gt;(B)&lt;/strong&gt;, and Bat CoV &lt;strong&gt;(C)&lt;/strong&gt;. The colour schemes are the same as in the corresponding plots of &lt;strong&gt;Figure 3. (D)&lt;/strong&gt; A 1.8 &#xC5; resolution X-ray structure of ORF7a from Human SARS CoV (PDB ID: 1XAK). It consists of only A chain of 14-96 residues (violet colour). &lt;strong&gt;(E)&lt;/strong&gt; MSA profile of ORF7a from SARS-CoV-2, Human SARS CoV, and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 10." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F10.medium.gif" width="440" height="234"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 10." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F10.medium.gif" width="440" height="234"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 10." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 10.</span> <span class="caption-title">Analysis of intrinsic disorder propensity of ORF7a protein.</span><p id="p-57" class="first-child">The graphs represent the results of the intrinsic disorder analysis of ORF7a protein from SARS-CoV-2 <strong>(A)</strong>, Human SARS <strong>(B)</strong>, and Bat CoV <strong>(C)</strong>. The colour schemes are the same as in the corresponding plots of <strong><a id="xref-fig-3-12" class="xref-fig" href="#F3">Figure 3</a>. (D)</strong> A 1.8 Å resolution X-ray structure of ORF7a from Human SARS CoV (PDB ID: 1XAK). It consists of only A chain of 14-96 residues (violet colour). <strong>(E)</strong> MSA profile of ORF7a from SARS-CoV-2, Human SARS CoV, and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues.</p><div class="sb-div caption-clear"></div></div></div><p id="p-58">We found that 121-residue-long ORF7a protein of SARS-CoV-2 shares 89.26% and 85.95% sequence identity with ORF7a proteins of Bat CoV and Human SARS CoV, respectively (<strong><a id="xref-fig-10-2" class="xref-fig" href="#F10">Figure 10E</a></strong>). On the other hand, the ORF7b of SARS-CoV-2 is found to be closer to ORF7b of Human SARS than to ORF7b of Bat CoV, showing sequence identities of 81.40% and 79.07%, respectively (see <strong><a id="xref-fig-11-1" class="xref-fig" href="#F11">Figure 11D</a></strong>).</p><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of intrinsic disorder propensity of ORF7b protein. The graphs represent the results of the intrinsic disorder analysis of the ORF7b protein from SARS-CoV-2 (A), Human SARS CoV (B), and Bat CoV (C). The colour schemes are the same as in the corresponding plots of Figure 3. (D) MSA profile of ORF7b from SARS CoV-2, Human SARS CoV, and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of intrinsic disorder propensity of ORF7b protein.&lt;/span&gt; The graphs represent the results of the intrinsic disorder analysis of the ORF7b protein from SARS-CoV-2 &lt;strong&gt;(A)&lt;/strong&gt;, Human SARS CoV &lt;strong&gt;(B)&lt;/strong&gt;, and Bat CoV &lt;strong&gt;(C)&lt;/strong&gt;. The colour schemes are the same as in the corresponding plots of &lt;strong&gt;Figure 3. (D)&lt;/strong&gt; MSA profile of ORF7b from SARS CoV-2, Human SARS CoV, and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 11." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F11.medium.gif" width="440" height="186"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 11." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F11.medium.gif" width="440" height="186"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 11." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 11.</span> <span class="caption-title">Analysis of intrinsic disorder propensity of ORF7b protein.</span><p id="p-59" class="first-child">The graphs represent the results of the intrinsic disorder analysis of the ORF7b protein from SARS-CoV-2 <strong>(A)</strong>, Human SARS CoV <strong>(B)</strong>, and Bat CoV <strong>(C)</strong>. The colour schemes are the same as in the corresponding plots of <strong><a id="xref-fig-3-13" class="xref-fig" href="#F3">Figure 3</a>. (D)</strong> MSA profile of ORF7b from SARS CoV-2, Human SARS CoV, and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues.</p><div class="sb-div caption-clear"></div></div></div><p id="p-60">As can be observed from <strong><a id="xref-table-wrap-1-9" class="xref-table" href="#T1">Table 1</a></strong>, our disorder predisposition analyses resulted in the overall PPID for ORF7a proteins of 1.65% for SARS-CoV-2, 0.82% for Bat CoV and 0.82% for Human SARS CoV. Mean PPIDs estimated for ORF7b proteins are 9.30% for SARS-CoV-2, 4.55% for Bat CoV and 4.55% Human SARS CoV. <strong><a id="xref-fig-10-3" class="xref-fig" href="#F10">Figures 10A, 10B</a></strong>, and <strong><a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10C</a></strong> represent the residues predisposed for disorder in ORF7a proteins of SARS-CoV-2, Human SARS CoV, and Bat CoV, respectively. <strong><a id="xref-table-wrap-2-12" class="xref-table" href="#T2">Table 2</a></strong> shows that ORF7a protein is expected to have several MoRFs indicating the potential involvement of this protein in disorder-dependent protein-protein interactions. At the N-terminus, we observed one MoRF region (residues 1-10) with the help of DISOPRED3 in all three viruses. In addition to protein binding regions, ORF7a also contains several RNA and DNA binding residues. Analysis also represents, ORF7b proteins from all three viruses have low disorder content, likewise, they are not predicted to contain any MoRF by any of the predictors used in this study (<strong><a id="xref-table-wrap-2-13" class="xref-table" href="#T2">Table 2</a>, Supplementary Table 7</strong> and <strong>8</strong>). Although the ORF7b does not contain protein-binding regions, it was found to contain nucleotide (RNS and DNA) binding regions in the protein <strong><a id="xref-fig-11-2" class="xref-fig" href="#F11">Figures 11A, 11B</a></strong>, and <strong><a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11C</a></strong> depict the residues predisposed for disorder in ORF7b proteins of SARS-CoV-2, Human SARS CoV, and Bat CoV, respectively. According to our analysis, both proteins in all three studied coronaviruses have a mostly ordered structure.</p></div><div id="sec-21" class="subsection"><h4>Proteins ORF8a and ORF8b</h4><p id="p-61">In animals and isolates from early human infections, the <em>ORF8</em> gene codes for a single ORF8 protein. However, in late infections, more specifically, at middle and late stages, a 29 nucleotide deletion in the <em>ORF8</em> gene led to the formation of two distinct proteins, ORF8a and ORF8b containing 39 and 84 residues respectively [<a id="xref-ref-116-1" class="xref-bibr" href="#ref-116">116</a>,<a id="xref-ref-117-1" class="xref-bibr" href="#ref-117">117</a>].</p><p id="p-62">Both proteins have conformations different from that of the longer ORF8 protein. It has been reported that overexpression of ORF8b resulted in the downregulation of E protein while the proteins ORF8a and ORF8/ORF8ab have no effect on the expression of protein E. Also, ORF8/ORF8ab was found to interact very strongly with proteins S, ORF3a, and ORF7a. ORF8a interacts with S and E proteins, whereas ORF8b protein interacts with E, M, ORF3a and ORF7a proteins [<a id="xref-ref-118-1" class="xref-bibr" href="#ref-118">118</a>]. The disorder-based protein binding regions of this protein identified in this study may have important role in interaction with other proteins.</p><p id="p-63">ORF8 protein found in early SARS-CoV-2 isolates having 121 residues and according to our analysis, it shares a 90.05% sequence identity with ORF8 protein of Bat CoV (<strong><a id="xref-fig-12-1" class="xref-fig" href="#F12">Figure 12C</a></strong>). Furthermore, <strong><a id="xref-fig-12-2" class="xref-fig" href="#F12">Figures 12A</a></strong> and <strong><a id="xref-fig-12-3" class="xref-fig" href="#F12">12B</a></strong> show that there is no intrinsic disorder in both ORF8 proteins from SARS-CoV-2 and Bat CoV. Therefore, these two proteins predicted to be completely structured having a mean PPID of 0.00%. In ORF8a and ORF8b proteins of the Human SARS, the predicted disorder is estimated to be 2.56% and 2.38%, respectively <strong>(<a id="xref-table-wrap-1-10" class="xref-table" href="#T1">Table 1</a>)</strong>. Graphs in <strong><a id="xref-fig-13-1" class="xref-fig" href="#F13">Figures 13A</a></strong> and <strong><a id="xref-fig-13-2" class="xref-fig" href="#F13">13B</a></strong> illustrate the presence of some disorder near the N- and C-terminals of ORF8a and ORF8b proteins. <strong><a id="xref-table-wrap-2-14" class="xref-table" href="#T2">Table 2</a></strong> shows the identified MoRF regions in ORF8 of SARS-CoV-2. It shows three MoRF regions (residues 1-5, 26-52, and 69-91) by MoRFchibi_web and one MoRF region (residues 1-10) by DISOPRED3. In Human SARS, the N-terminus of both ORF8a (residues 1-39) and ORF8b (residues 1-83) was found to be MoRF by MoRFchibi_web server (<strong>Supplementary Table 7</strong>). Further, four protein-binding regions (residues 26-53, 70-91, 98-104, and 113-130) were identified by MoRFchibi_web server in Bat CoV (<strong>Supplementary Table 8</strong>). Apart from protein-binding, ORF8 of SARS-CoV-2, ORF8a and ORF8b of Human SARS, and ORF8 of Bat CoV also comprise several nucleotide-binding residues (see <strong>Supplementary Table 9, 10</strong>, and <strong>11</strong>).</p><div id="F12" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of intrinsic disorder propensity of ORF8 protein. The graphs represent the results of the intrinsic disorder analysis of the ORF8 protein from SARS-CoV-2 (A) and Bat CoV (B). The colour schemes are the same as in the corresponding plots of Figure 3. (C) MSA profile of ORF8 protein from SARS-CoV-2 and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of intrinsic disorder propensity of ORF8 protein.&lt;/span&gt; The graphs represent the results of the intrinsic disorder analysis of the ORF8 protein from SARS-CoV-2 &lt;strong&gt;(A)&lt;/strong&gt; and Bat CoV &lt;strong&gt;(B)&lt;/strong&gt;. The colour schemes are the same as in the corresponding plots of &lt;strong&gt;Figure 3. (C)&lt;/strong&gt; MSA profile of ORF8 protein from SARS-CoV-2 and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 12." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F12.medium.gif" width="440" height="324"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 12." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F12.medium.gif" width="440" height="324"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 12." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 12.</span> <span class="caption-title">Analysis of intrinsic disorder propensity of ORF8 protein.</span><p id="p-64" class="first-child">The graphs represent the results of the intrinsic disorder analysis of the ORF8 protein from SARS-CoV-2 <strong>(A)</strong> and Bat CoV <strong>(B)</strong>. The colour schemes are the same as in the corresponding plots of <strong><a id="xref-fig-3-14" class="xref-fig" href="#F3">Figure 3</a>. (C)</strong> MSA profile of ORF8 protein from SARS-CoV-2 and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues.</p><div class="sb-div caption-clear"></div></div></div><div id="F13" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F13.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of intrinsic disorder propensity of ORF8a and ORF8b proteins from Human SARS CoV. (A) The graph represents the disorder profile generated for ORF8a protein from Human SARS CoV. (B) The graph shows the disorder profile generated for ORF8b protein from Human SARS CoV. The colour schemes are the same as in the corresponding plots of Figure 3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of intrinsic disorder propensity of ORF8a and ORF8b proteins from Human SARS CoV.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; The graph represents the disorder profile generated for ORF8a protein from Human SARS CoV. &lt;strong&gt;(B)&lt;/strong&gt; The graph shows the disorder profile generated for ORF8b protein from Human SARS CoV. The colour schemes are the same as in the corresponding plots of &lt;strong&gt;Figure 3&lt;/strong&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 13." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F13.medium.gif" width="440" height="171"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 13." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F13.medium.gif" width="440" height="171"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 13." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 13.</span> <span class="caption-title">Analysis of intrinsic disorder propensity of ORF8a and ORF8b proteins from Human SARS CoV.</span><p id="p-65" class="first-child"><strong>(A)</strong> The graph represents the disorder profile generated for ORF8a protein from Human SARS CoV. <strong>(B)</strong> The graph shows the disorder profile generated for ORF8b protein from Human SARS CoV. The colour schemes are the same as in the corresponding plots of <strong><a id="xref-fig-3-15" class="xref-fig" href="#F3">Figure 3</a></strong>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-22" class="subsection"><h4>ORF9b protein</h4><p id="p-66">This protein is expressed from an alternative ORF within the <em>N</em> gene through a leaky ribosome binding process [<a id="xref-ref-119-1" class="xref-bibr" href="#ref-119">119</a>]. Inside the host cells, ORF9b enters the nucleus, which is a cell cycle-independent process and represents a passive entry. This protein was shown to interact with a nuclear export protein receptor Exportin 1 (Crm1), using which it translocate out of the nucleus [<a id="xref-ref-120-1" class="xref-bibr" href="#ref-120">120</a>]. Our MoRFs analysis shows the presence of disorder-based protein binding regions in ORF9b protein which may have role in its interaction with Crm1 and further translocation outside the nucleus. A 2.8 Å resolution crystal structure of ORF9b protein from Human SARS CoV (PDB ID: 2CME) shows the presence of a dimeric tent-like <em>β</em>-structure along with the central hydrophobic amino acids (<strong><a id="xref-fig-14-1" class="xref-fig" href="#F14">Figure 14D</a>)</strong>. The published structure has the highly polarized distribution of charges, with positively charged residues on the one side of the tent and negatively charged on the other [<a id="xref-ref-121-1" class="xref-bibr" href="#ref-121">121</a>].</p><div id="F14" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F14.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of intrinsic disorder propensity of ORF9b protein. The graphs represent results of the intrinsic disorder analysis of the ORF9b of SARS-CoV-2 (A) and Human SARS CoV (B), as well as Bat CoV of ORF9 protein (C). The colour schemes are the same as in the corresponding plots of Figure 3. (D) A 2.8 &#xC5; resolution X-ray crystal structure of ORF9b protein from Human SARS CoV (PDB ID: 2CME). This asymmetric unit includes four ORF9b homodimers and consists of A, B, E, and G chains of 9-98 residues (purple colour) and C, D, F, and H chains of 10-98 residues (purple colour). (E) MSA of ORF9b of Human SARS CoV and ORF9 of Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of intrinsic disorder propensity of ORF9b protein.&lt;/span&gt; The graphs represent results of the intrinsic disorder analysis of the ORF9b of SARS-CoV-2 &lt;strong&gt;(A)&lt;/strong&gt; and Human SARS CoV &lt;strong&gt;(B)&lt;/strong&gt;, as well as Bat CoV of ORF9 protein &lt;strong&gt;(C)&lt;/strong&gt;. The colour schemes are the same as in the corresponding plots of &lt;strong&gt;Figure 3. (D)&lt;/strong&gt; A 2.8 &#xC5; resolution X-ray crystal structure of ORF9b protein from Human SARS CoV (PDB ID: 2CME). This asymmetric unit includes four ORF9b homodimers and consists of A, B, E, and G chains of 9-98 residues (purple colour) and C, D, F, and H chains of 10-98 residues (purple colour). &lt;strong&gt;(E)&lt;/strong&gt; MSA of ORF9b of Human SARS CoV and ORF9 of Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 14." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F14.medium.gif" width="440" height="231"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 14." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F14.medium.gif" width="440" height="231"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 14." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 14.</span> <span class="caption-title">Analysis of intrinsic disorder propensity of ORF9b protein.</span><p id="p-67" class="first-child">The graphs represent results of the intrinsic disorder analysis of the ORF9b of SARS-CoV-2 <strong>(A)</strong> and Human SARS CoV <strong>(B)</strong>, as well as Bat CoV of ORF9 protein <strong>(C)</strong>. The colour schemes are the same as in the corresponding plots of <strong><a id="xref-fig-3-16" class="xref-fig" href="#F3">Figure 3</a>. (D)</strong> A 2.8 Å resolution X-ray crystal structure of ORF9b protein from Human SARS CoV (PDB ID: 2CME). This asymmetric unit includes four ORF9b homodimers and consists of A, B, E, and G chains of 9-98 residues (purple colour) and C, D, F, and H chains of 10-98 residues (purple colour). <strong>(E)</strong> MSA of ORF9b of Human SARS CoV and ORF9 of Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues.</p><div class="sb-div caption-clear"></div></div></div><p id="p-68">Based on the sequence availability of accession ID <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=NC_045512.2&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom">NC_045512.2</a>, the translated protein sequence of ORF9b is not reported for the SARS-CoV-2 as of yet. However, based on the report by Wu and colleagues [<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>], the sequences of the SARS-CoV-2 are already annotated. Therefore, we took the corresponding amino acid sequences from that study and conducted the intrinsic disorder analysis. According to the MSA, results shown in <strong><a id="xref-fig-14-2" class="xref-fig" href="#F14">Figure 14E</a></strong>, ORF9b protein from SARS-CoV-2 shares 73.2% identity with Human SARS and 74.23% identity with Bat CoV.</p><p id="p-69">Our IDP analysis (<strong><a id="xref-table-wrap-1-11" class="xref-table" href="#T1">Table 1</a></strong>) shows that ORF9b from Human SARS is a moderately unstructured protein with a mean PPID estimated to 26.53%. As depicted in <strong><a id="xref-fig-14-3" class="xref-fig" href="#F14">Figure 14A, 14B</a></strong>, and <strong><a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14C</a></strong>, disorder mainly lies near the N-terminal end 1-10 residues and 28-40 residues near the central region with a well-ordered inner core of Human SARS ORF9b protein. The x-ray crystal structure of ORF9b has a missing electron density of the first 8 residues and 26-37 residues near the central region. This indicates that the corresponding regions are disordered, which are difficult to crystallize due to their highly dynamic structural organization. SARS-CoV-2 ORF9b protein with a mean PPID of 10.31% has an N-terminal (1-10 residues) predicted disordered segment. ORF9b of Bat CoV is shown to have an intrinsic disorder content of 9.28%, comparatively lower than that of the Human SARS ORF9b protein. MoRFs lies in the N-terminal region of ORF9b proteins (<strong><a id="xref-table-wrap-2-15" class="xref-table" href="#T2">Table 2</a>, Supplementary Table 7</strong> and <strong>8</strong>). In the absence of other viral proteins, its first 41 residues have been demonstrated to induce membranous structures similar to DMVs [<a id="xref-ref-106-2" class="xref-bibr" href="#ref-106">106</a>]. The available crystal structure also has the missing electron density in the N-terminal region suggests that these flexible amino acids are likely to interact with host lipids. The first 3-29 residues of SARS-CoV2 are identified as disorder-based protein binding region that may have role in its interaction with host lipids and formation of DMVs. <strong>Supplementary Tables 9, 10</strong>, and <strong>11</strong> represents nucleotide-binding residues for ORF9b of SARS-CoV-2, Human SARS, and Bat CoV.</p></div><div id="sec-23" class="subsection"><h4>ORF10 protein</h4><p id="p-70">The newly emerged SARS-CoV-2 has an ORF10 protein of 38 amino acids. ORF10 of SARS-CoV-2 has a 100% sequence similarity with ORF10 of Bat CoV strain Bat-SL-CoVZC45 [<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>]. However, we did not conduct the disorder analysis for ORF10 from the Bat-SL-CoVZC45 strain, since all our studies reported here are related to a different strain of Bat CoV (reviewed strain HKU3-1). Therefore, we report here only the results of disorder analysis for the ORF10 protein from SARS-CoV-2, according to which this protein has a mean PPID of 0.00% (see also <strong><a id="xref-fig-15-1" class="xref-fig" href="#F15">Figure 15</a></strong> for disorder profile of ORF10). This protein contains a MoRF from 3-7 residues at its N-terminus as predicted by MoRFchibi_web. Further, we found its tendency to nucleotides and found the presence of few RNA binding sites, however, it does not contain DNA binding residues.</p><div id="F15" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F15.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of intrinsic disorder propensity of ORF10 protein of SARS-CoV-2. The colour schemes are the same as in the corresponding plots of Figure 3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of intrinsic disorder propensity of ORF10 protein of SARS-CoV-2.&lt;/span&gt; The colour schemes are the same as in the corresponding plots of &lt;strong&gt;Figure 3&lt;/strong&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 15." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F15.medium.gif" width="440" height="357"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 15." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F15.medium.gif" width="440" height="357"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F15.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 15." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F15.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 15.</span> <span class="caption-title">Analysis of intrinsic disorder propensity of ORF10 protein of SARS-CoV-2.</span><p id="p-71" class="first-child">The colour schemes are the same as in the corresponding plots of <strong><a id="xref-fig-3-17" class="xref-fig" href="#F3">Figure 3</a></strong>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-24" class="subsection"><h4>Protein ORF14</h4><p id="p-72">This is a 70-amino-acid-long uncharacterized protein of unknown function, which is present in Human SARS and Bat CoV. In SARS-CoV-2, ORF14 is a 73-amino-acid-long protein. According to the MSA, ORF14 of SARS-CoV-2 has 77.1% identity with Human-SARS and 72.9% identity with Bat CoV as represented in <strong><a id="xref-fig-16-1" class="xref-fig" href="#F16">Figure 16D</a></strong>. We have performed the intrinsic disorder analysis to see the peculiarities of the distribution of disorder predisposition in this protein. <strong><a id="xref-fig-16-2" class="xref-fig" href="#F16">Figures 16A, 16B</a></strong>, and <strong><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16C</a></strong> show the resulting disorder profiles of ORF14 of SARS-CoV-2, Human SARS CoV, and Bat CoV. Although these proteins have calculated mean PPID values of 0.00%, 2.86%, and 0.00% respectively, <strong><a id="xref-fig-16-3" class="xref-fig" href="#F16">Figure 16</a></strong> shows that they have flexible N- and C-terminal regions. This protein can use intrinsic disorder or structural flexibility for protein-protein interactions since it possesses MoRFs. It mainly contains MoRFs at N- and C-terminal regions as tabulated in (<strong><a id="xref-table-wrap-2-16" class="xref-table" href="#T2">Table 2</a>, Supplementary Table 7</strong> and <strong>8</strong>). It was also found to contain several RNA and DNA binding residues (<strong>Supplementary Table 9, 10</strong>, and <strong>11)</strong>. These results indicating its vital role in protein function related to molecular recognition such as protein-protein, protein-RNA, and Protein-DNA interaction.</p><div id="F16" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F16.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of intrinsic disorder propensity of ORF 14 protein. The graphs represent the results of the intrinsic disorder analysis of the ORF14 protein from SARS-CoV-2 (A), Human SARS CoV (B), and Bat CoV (C). The colour schemes are the same as in the corresponding plots of Figure 3. (D) MSA profile of ORF14 from SARS CoV-2, Human SARS CoV, and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of intrinsic disorder propensity of ORF 14 protein.&lt;/span&gt; The graphs represent the results of the intrinsic disorder analysis of the ORF14 protein from SARS-CoV-2 &lt;strong&gt;(A)&lt;/strong&gt;, Human SARS CoV &lt;strong&gt;(B)&lt;/strong&gt;, and Bat CoV &lt;strong&gt;(C)&lt;/strong&gt;. The colour schemes are the same as in the corresponding plots of &lt;strong&gt;Figure 3. (D)&lt;/strong&gt; MSA profile of ORF14 from SARS CoV-2, Human SARS CoV, and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 16." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F16.medium.gif" width="440" height="225"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 16." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F16.medium.gif" width="440" height="225"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F16.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 16." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F16.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 16.</span> <span class="caption-title">Analysis of intrinsic disorder propensity of ORF 14 protein.</span><p id="p-73" class="first-child">The graphs represent the results of the intrinsic disorder analysis of the ORF14 protein from SARS-CoV-2 <strong>(A)</strong>, Human SARS CoV <strong>(B)</strong>, and Bat CoV <strong>(C)</strong>. The colour schemes are the same as in the corresponding plots of <strong><a id="xref-fig-3-18" class="xref-fig" href="#F3">Figure 3</a>. (D)</strong> MSA profile of ORF14 from SARS CoV-2, Human SARS CoV, and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues.</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-25" class="subsection"><h3>Intrinsic disorder analysis of non-structural proteins of Coronaviruses</h3><p id="p-74">In coronaviruses, due to ribosomal leakage during translation, two-third of the RNA genome is processed into two polyproteins: (i) Replicase polyprotein 1a and (ii) Replicase polyprotein 1ab. Both contain non-structural proteins (Nsp1-10) in addition to different proteins required for viral replication and pathogenesis. Replicase polyprotein 1a contains an additional Nsp11 protein of 13 amino acids, the function of which is not investigated yet. The longer replicase polyprotein 1ab of 7073 amino acids accommodates five other non-structural proteins (Nsp12-16) [<a id="xref-ref-122-1" class="xref-bibr" href="#ref-122">122</a>]. These proteins assist in ER membrane-induced vesicle formation, which acts as sites for replication and transcription. In addition to this, non-structural proteins work as proteases, helicases, and mRNA capping and methylation enzymes, crucial for virus survival and replication inside host cells [<a id="xref-ref-122-2" class="xref-bibr" href="#ref-122">122</a>,<a id="xref-ref-123-1" class="xref-bibr" href="#ref-123">123</a>].</p></div><div id="sec-26" class="subsection"><h3>Global analysis of intrinsic disorder in the replicase polyprotein 1ab</h3><p id="p-75"><strong><a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a></strong> represents the PPID mean scores of 15 non-structural proteins (Nsps) derived from the Replicase polyprotein 1ab in SARS-CoV-2, Human SARS CoV, and Bat CoV. These values were obtained by combining the results from six disorder predictors (see <strong>Supplementary Table S4-S6</strong>). <strong><a id="xref-fig-17-1" class="xref-fig" href="#F17">Figures 17A, 17B</a></strong>, and <strong><a id="xref-fig-17-2" class="xref-fig" href="#F17">17C</a></strong> represent the 2D-disorder plots of the Nsps coded by ORF1ab in SARS-CoV-2, Human SARS Cov, and Bat CoV, respectively. Based on the mean PPID scores in <strong><a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>, <a id="xref-fig-17-3" class="xref-fig" href="#F17">Figures 17A, 17B, 17C</a></strong>, and taking into PPID based classification [<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">46</a>], we conclude that none of the Nsps in SARS-CoV-2, Human SARS CoV, and Bat CoV are highly disordered. The highest disorder was observed for Nsp8 proteins in all three Coronaviruses. Both Nsp1 and Nsp8 are moderately disordered proteins (10% ≤ PPID ≤ 30%). We also observed that Nsp2, Nsp3, Nsp5, Nsp6, Nsp7, Nsp9, Nsp10, Nsp15, and Nsp16 have less than 10% disordered residues and hence, belong to the category of mostly ordered proteins. Other non-structural proteins, namely, Nsp4, Nsp12, Nsp13, and Nsp14 have negligible levels of disorder (PPID &lt; 1%), which tells us that these are highly structured proteins.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1232223/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/1232223/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3.</span> <p id="p-76" class="first-child">Evaluation of mean predicted percentage disorder in non-structural proteins of novel SARS-CoV-2, Human SARS, and Bat CoV.</p><div class="sb-div caption-clear"></div></div></div><div id="F17" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F17.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of the overall intrinsic disorder status of non-structural proteins (Nsps) coded by ORF1ab in SARS-CoV-2, Human SARS CoV, and Bat CoV: 2D plot representing PPIDPONDR-FIT vs PPIDMean in (A) SARS-CoV-2 (B) Human SARS and (C) Bat CoV (SARS-like). In CH-CDF plot of the proteins of (D) SARS-CoV-2 (E) Human SARS and (F) Bat CoV, the Y coordinate of each protein spot signifies distance of corresponding protein from the boundary in CH plot and the X coordinate value corresponds to the average distance of the CDF curve for the respective protein from the CDF boundary." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of the overall intrinsic disorder status of non-structural proteins (Nsps) coded by &lt;em&gt;ORF1ab&lt;/em&gt; in SARS-CoV-2, Human SARS CoV, and Bat CoV:&lt;/span&gt; 2D plot representing PPID&lt;sub&gt;PONDR-FIT&lt;/sub&gt; vs PPID&lt;sub&gt;Mean&lt;/sub&gt; in &lt;strong&gt;(A)&lt;/strong&gt; SARS-CoV-2 &lt;strong&gt;(B)&lt;/strong&gt; Human SARS and &lt;strong&gt;(C)&lt;/strong&gt; Bat CoV (SARS-like). In CH-CDF plot of the proteins of &lt;strong&gt;(D)&lt;/strong&gt; SARS-CoV-2 &lt;strong&gt;(E)&lt;/strong&gt; Human SARS and &lt;strong&gt;(F)&lt;/strong&gt; Bat CoV, the Y coordinate of each protein spot signifies distance of corresponding protein from the boundary in CH plot and the X coordinate value corresponds to the average distance of the CDF curve for the respective protein from the CDF boundary.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 17." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F17.medium.gif" width="440" height="246"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 17." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F17.medium.gif" width="440" height="246"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F17.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 17." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F17.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 17.</span> <span class="caption-title">Analysis of the overall intrinsic disorder status of non-structural proteins (Nsps) coded by <em>ORF1ab</em> in SARS-CoV-2, Human SARS CoV, and Bat CoV:</span><p id="p-77" class="first-child">2D plot representing PPID<sub>PONDR-FIT</sub> vs PPID<sub>Mean</sub> in <strong>(A)</strong> SARS-CoV-2 <strong>(B)</strong> Human SARS and <strong>(C)</strong> Bat CoV (SARS-like). In CH-CDF plot of the proteins of <strong>(D)</strong> SARS-CoV-2 <strong>(E)</strong> Human SARS and <strong>(F)</strong> Bat CoV, the Y coordinate of each protein spot signifies distance of corresponding protein from the boundary in CH plot and the X coordinate value corresponds to the average distance of the CDF curve for the respective protein from the CDF boundary.</p><div class="sb-div caption-clear"></div></div></div><p id="p-78">The CH-CDF analysis of the Nsps from SARS-CoV-2, Human SARS and Bat CoV have been represented in <strong><a id="xref-fig-17-4" class="xref-fig" href="#F17">Figures 17D, 17E</a></strong>, and <strong><a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17F</a></strong> respectively. It was observed that all the Nsps of the three coronaviruses are located within the quadrant Q1 of the CH-CDF phase space, indicating that all the Nsps are predicted to be mostly ordered.</p><div id="sec-27" class="subsection"><h4>Replicase polyprotein 1ab</h4><p id="p-79">The longer replicase polyprotein 1ab is a 7,073 amino acid-long polypeptide, which contains 15 non-structural proteins listed in <strong><a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a></strong>. Nsp1, Nsp2, and Nsp3 are cleaved using a viral papain-like proteinase (Nsp3/PL-Pro), while the rest of Nsps are cleaved by another viral 3C-like proteinase, Nsp5/3CL-Pro. We mapped the cleavage sites of the replicase 1ab polyprotein from Human SARS CoV to the disorder profile of this polyprotein. <strong><a id="xref-fig-18-1" class="xref-fig" href="#F18">Figure 18</a></strong> represents the results of this analysis by showing zoomed-in regions surrounding all the cleavage sites with few residues spanning at both terminals. Interestingly, we observed that all the cleavage sites are largely disordered, suggesting that intrinsic disorder may have a crucial role in the maturation of individual non-structural proteins. As the Nsps of Human SARS CoV are evolutionary close to the Nsps of SARS-CoV-2, we hypothesize that the cleavage sites in the SARS-CoV-2 replicase 1ab polyprotein are also intrinsically disordered or flexible. To shed more light on other implications of IDPRs, the structural and functional properties of Nsps and their predicted IDPRs are thoroughly described below.</p><div id="F18" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F18.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Intrinsic disorder at the cleavage sites of the replicase 1ab polyprotein of Human SARS CoV. Plots A to N denotes the cleavage sites (magenta coloured bar for PL-Pro protease and grey coloured bar for 3CL-Pro protease) in relation to disordered regions present between the individual proteins (Nsp1-16) of replicase 1ab polyprotein of Human SARS. All proteins are represented by different coloured horizontal bars." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Intrinsic disorder at the cleavage sites of the replicase 1ab polyprotein of Human SARS CoV.&lt;/span&gt; Plots &lt;strong&gt;A&lt;/strong&gt; to &lt;strong&gt;N&lt;/strong&gt; denotes the cleavage sites (magenta coloured bar for PL-Pro protease and grey coloured bar for 3CL-Pro protease) in relation to disordered regions present between the individual proteins (Nsp1-16) of replicase 1ab polyprotein of Human SARS. All proteins are represented by different coloured horizontal bars.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 18." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F18.medium.gif" width="440" height="249"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 18." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F18.medium.gif" width="440" height="249"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F18.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 18." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F18.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 18.</span> <span class="caption-title">Intrinsic disorder at the cleavage sites of the replicase 1ab polyprotein of Human SARS CoV.</span><p id="p-80" class="first-child">Plots <strong>A</strong> to <strong>N</strong> denotes the cleavage sites (magenta coloured bar for PL-Pro protease and grey coloured bar for 3CL-Pro protease) in relation to disordered regions present between the individual proteins (Nsp1-16) of replicase 1ab polyprotein of Human SARS. All proteins are represented by different coloured horizontal bars.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-28" class="subsection"><h4>Non-structural protein 1 (Nsp1)</h4><p id="p-81">This protein acts as a host translation inhibitor as it binds to the 40S subunit of the ribosome and blocks the translation of cap-dependent mRNAs as well as mRNAs that uses the internal ribosome entry site (IRES) [<a id="xref-ref-124-1" class="xref-bibr" href="#ref-124">124</a>]. <strong><a id="xref-fig-19-1" class="xref-fig" href="#F19">Figure 19D</a></strong> shows the NMR solution structure (PDB ID: 2GDT) of Human SARS nsp1 protein (13-128 residues), whereas residues 117-180 were not included in this structural analysis [<a id="xref-ref-125-1" class="xref-bibr" href="#ref-125">125</a>].</p><div id="F19" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F19.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of intrinsic disorder propensity and structural characterization of Nsp1. Intrinsic disorder profiles generated for Nsp1 from SARS-CoV-2 (A), Human SARS CoV (B), and Bat CoV (C). The colour schemes are the same as in the corresponding plots of Figure 3. (D) NMR solution structure of a 13-128 fragment of Human SARS Nsp1 (PDB ID: 2GDT). (E) MSA profile of Nsp1 from SARS CoV-2, Human SARS CoV, and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of intrinsic disorder propensity and structural characterization of Nsp1.&lt;/span&gt; Intrinsic disorder profiles generated for Nsp1 from SARS-CoV-2 &lt;strong&gt;(A)&lt;/strong&gt;, Human SARS CoV &lt;strong&gt;(B)&lt;/strong&gt;, and Bat CoV &lt;strong&gt;(C)&lt;/strong&gt;. The colour schemes are the same as in the corresponding plots of &lt;strong&gt;Figure 3. (D)&lt;/strong&gt; NMR solution structure of a 13-128 fragment of Human SARS Nsp1 (PDB ID: 2GDT). &lt;strong&gt;(E)&lt;/strong&gt; MSA profile of Nsp1 from SARS CoV-2, Human SARS CoV, and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 19." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F19.medium.gif" width="440" height="255"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 19." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F19.medium.gif" width="440" height="255"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F19.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 19." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F19.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 19.</span> <span class="caption-title">Analysis of intrinsic disorder propensity and structural characterization of Nsp1.</span><p id="p-82" class="first-child">Intrinsic disorder profiles generated for Nsp1 from SARS-CoV-2 <strong>(A)</strong>, Human SARS CoV <strong>(B)</strong>, and Bat CoV <strong>(C)</strong>. The colour schemes are the same as in the corresponding plots of <strong><a id="xref-fig-3-19" class="xref-fig" href="#F3">Figure 3</a>. (D)</strong> NMR solution structure of a 13-128 fragment of Human SARS Nsp1 (PDB ID: 2GDT). <strong>(E)</strong> MSA profile of Nsp1 from SARS CoV-2, Human SARS CoV, and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues.</p><div class="sb-div caption-clear"></div></div></div><p id="p-83">SARS-CoV-2 Nsp1 shares 84.44% and 83.80% sequence identity with Nsp1s of Human SARS CoV and Bat CoV, respectively. Its N-terminal region is found to be more conserved than the rest of the protein sequence (<strong><a id="xref-fig-19-2" class="xref-fig" href="#F19">Figure 19E</a>)</strong>. Mean PPIDs of Nsp1s from SARS-CoV-2, Human SARS CoV, and Bat CoV are 12.78%, 14.44%, and 12.85%, respectively. <strong><a id="xref-fig-19-3" class="xref-fig" href="#F19">Figure 19A, 19B</a></strong>, and <strong><a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">19C</a></strong> represent the graphs of predicted per-residue intrinsic disorder propensity of these Nsp1s. According to the analysis, the following regions are predicted to be disordered: SARS-CoV-2 (residues 1-7 and 165-180), Human SARS CoV (residues 1-5 and 165-180), and Bat CoV (residues 1-5 and 165-179). NMR solution structure of Nsp1 from Human SARS revealed the presence of two unstructured segments near the N-terminal (1-12 residues) and C-terminal (129-179 residues) regions [<a id="xref-ref-125-2" class="xref-bibr" href="#ref-125">125</a>]. The disordered region (128–180 residues) at C-terminus is important for Nsp1 expression [<a id="xref-ref-126-1" class="xref-bibr" href="#ref-126">126</a>]. Based on sequence homology with Human SARS CoV Nsp1, the predicted disordered C-terminal region of SARS-CoV-2 Nsp1 may play a critical role in its expression. Alanine mutants at K164 and H165 in the C-terminal region of Nsp1 protein is reported to abolish its binding with the 40S subunit of the host ribosome [<a id="xref-ref-127-1" class="xref-bibr" href="#ref-127">127</a>]. In conjunction with this data, several MoRFs are present in the unstructured segments of Nsp1 proteins. These regions are tabulated in <strong><a id="xref-table-wrap-2-17" class="xref-table" href="#T2">Table 2</a></strong>, and <strong>Supplementary Tables 7</strong> and <strong>8</strong>.</p></div><div id="sec-29" class="subsection"><h4>Non-structural protein 2 (Nsp2)</h4><p id="p-84">This protein functions by disrupting the host survival pathway via interaction with the host proteins Prohibitin-1 and Prohibitin-2 [<a id="xref-ref-128-1" class="xref-bibr" href="#ref-128">128</a>]. Reverse genetic deletion in the coding sequence of Nsp2 of the SARS virus attenuated little viral growth and replication and allowed the recovery of mutant virulent viruses. This indicates the dispensable nature of the Nsp2 protein for SARS viruses [<a id="xref-ref-129-1" class="xref-bibr" href="#ref-129">129</a>].</p><p id="p-85">The sequence identity of the Nsp2 protein from SARS-CoV-2 with Nsp2s of Human SARS CoV and Bat CoV amounts to 68.34% and 68.97%, respectively (<strong>Supplementary Figure S2A</strong>). We have estimated the mean PPIDs of Nsp2s of SARS-CoV-2, Human SARS CoV, and Bat CoV to be 5.17%, 2.04%, and 2.03% respectively (see <strong><a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a></strong>). The per-residues predisposition for the intrinsic disorder of Nsp2s from SARS-CoV-2, Human SARS CoV, and Bat CoV are depicted in <strong><a id="xref-fig-20-1" class="xref-fig" href="#F20">Figures 20A, 20B</a></strong>, and <strong><a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">20C</a></strong>. According to this analysis, the following regions in Nsp2 proteins are predicted to be disordered, residues 570-595 (SARS-CoV-2), residues 110-115 (Human SARS), and residues 112-116 (Bat CoV). As listed in <strong><a id="xref-table-wrap-2-18" class="xref-table" href="#T2">Table 2</a></strong>, and <strong>Supplementary Tables 7</strong> and <strong>8</strong>, Human SARS CoV does not contain MoRF while SARS-CoV-2 and Bat CoV have an N-terminally located MoRF region predicted by MoRFchibi_web.</p><div id="F20" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F20.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of the intrinsic disorder propensity of Nsp2. Intrinsic disorder profiles generated for Nsp2 from SARS-CoV-2 (A), Human SARS CoV (B), and Bat CoV (C). The colour schemes are the same as in the corresponding plots of Figure 3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of the intrinsic disorder propensity of Nsp2.&lt;/span&gt; Intrinsic disorder profiles generated for Nsp2 from SARS-CoV-2 &lt;strong&gt;(A)&lt;/strong&gt;, Human SARS CoV &lt;strong&gt;(B)&lt;/strong&gt;, and Bat CoV &lt;strong&gt;(C)&lt;/strong&gt;. The colour schemes are the same as in the corresponding plots of &lt;strong&gt;Figure 3&lt;/strong&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 20." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F20.medium.gif" width="440" height="118"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 20." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F20.medium.gif" width="440" height="118"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F20.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 20." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F20.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 20.</span> <span class="caption-title">Analysis of the intrinsic disorder propensity of Nsp2.</span><p id="p-86" class="first-child">Intrinsic disorder profiles generated for Nsp2 from SARS-CoV-2 <strong>(A)</strong>, Human SARS CoV <strong>(B)</strong>, and Bat CoV <strong>(C)</strong>. The colour schemes are the same as in the corresponding plots of <strong><a id="xref-fig-3-20" class="xref-fig" href="#F3">Figure 3</a></strong>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-30" class="subsection"><h4>Non-structural protein 3 (Nsp3)</h4><p id="p-87">Nsp3 is an almost 2,000-residue-long viral papain-like protease (PLP) that affects the phosphorylation and activation of IRF3 and therefore antagonizes the IFN pathway [<a id="xref-ref-130-1" class="xref-bibr" href="#ref-130">130</a>]. It was also demonstrated that Nsp3 works by stabilizing NF-κ<em>β</em> inhibitor further blocking the NF-κ<em>β</em> pathway [<a id="xref-ref-130-2" class="xref-bibr" href="#ref-130">130</a>]. <strong><a id="xref-fig-21-1" class="xref-fig" href="#F21">Figure 21D</a></strong> represents the 1.85 Å resolution X-ray crystal structure of the catalytic core of Nsp3 protein from Human SARS CoV (PDB ID: 2FE8), which was obtained by Andrew and colleagues [<a id="xref-ref-131-1" class="xref-bibr" href="#ref-131">131</a>]. This structure consists of the residues 723-1036 of Nsp3. The structure revealed folds similar to a deubiquitinating enzyme in-vitro deubiquitinating activity of which was found to be efficiently high [<a id="xref-ref-131-2" class="xref-bibr" href="#ref-131">131</a>].</p><div id="F21" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F21.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of intrinsic disorder propensity of Nsp3. Intrinsic disorder profiles generated for Nsp3 from SARS-CoV-2 (A), Human SARS CoV (B), and Bat CoV (C). The colour schemes are the same as in the corresponding plots of Figure 3. (D) A 1.85 &#xC5; resolution X-ray crystal structure of the region containing residues 723-1036 of Nsp3 of Human SARS CoV (PDB ID: 2FE8). (E) Aligned disorder profiles generated for Nsp3 from SARS-CoV-2 (black line), Human SARS CoV (red line), and Bat CoV (green line) based on the outputs of the PONDR&#xAE; VSL2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of intrinsic disorder propensity of Nsp3.&lt;/span&gt; Intrinsic disorder profiles generated for Nsp3 from SARS-CoV-2 &lt;strong&gt;(A)&lt;/strong&gt;, Human SARS CoV &lt;strong&gt;(B)&lt;/strong&gt;, and Bat CoV &lt;strong&gt;(C)&lt;/strong&gt;. The colour schemes are the same as in the corresponding plots of &lt;strong&gt;Figure 3. (D)&lt;/strong&gt; A 1.85 &#xC5; resolution X-ray crystal structure of the region containing residues 723-1036 of Nsp3 of Human SARS CoV (PDB ID: 2FE8). &lt;strong&gt;(E)&lt;/strong&gt; Aligned disorder profiles generated for Nsp3 from SARS-CoV-2 (black line), Human SARS CoV (red line), and Bat CoV (green line) based on the outputs of the PONDR&lt;sup&gt;&#xAE;&lt;/sup&gt; VSL2.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 21." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F21.medium.gif" width="440" height="250"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 21." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F21.medium.gif" width="440" height="250"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F21.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 21." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F21.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 21.</span> <span class="caption-title">Analysis of intrinsic disorder propensity of Nsp3.</span><p id="p-88" class="first-child">Intrinsic disorder profiles generated for Nsp3 from SARS-CoV-2 <strong>(A)</strong>, Human SARS CoV <strong>(B)</strong>, and Bat CoV <strong>(C)</strong>. The colour schemes are the same as in the corresponding plots of <strong><a id="xref-fig-3-21" class="xref-fig" href="#F3">Figure 3</a>. (D)</strong> A 1.85 Å resolution X-ray crystal structure of the region containing residues 723-1036 of Nsp3 of Human SARS CoV (PDB ID: 2FE8). <strong>(E)</strong> Aligned disorder profiles generated for Nsp3 from SARS-CoV-2 (black line), Human SARS CoV (red line), and Bat CoV (green line) based on the outputs of the PONDR<sup>®</sup> VSL2.</p><div class="sb-div caption-clear"></div></div></div><p id="p-89">Nsp3 protein of SARS-CoV-2 contains several substituted residues throughout the protein. It is equally close with both Nsp3 proteins of Human SARS and Bat CoV sharing respective 76.69% and 76.31% identity <strong>(Supplementary Figure S2B)</strong>. According to our results, the mean PPIDs of Nsp3 proteins of SARS-CoV-2, Human SARS, and Bat CoV are 7.40%, 7.91%, and 7.78% respectively (<strong><a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a></strong>). Graphs in <strong><a id="xref-fig-21-2" class="xref-fig" href="#F21">Figures 21A, 21B</a></strong>, and <strong><a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">21C</a></strong> portray the tendency of Nsp3 proteins of SARS-CoV-2, Human SARS, and Bat CoV for the intrinsic disorder. Nsp3 proteins of all three studied SARS viruses were found to be highly structured and characterized by rather similar disorder profiles. This is further supported by <strong><a id="xref-fig-21-3" class="xref-fig" href="#F21">Figure 21E</a></strong>, where PONDR<sup>®</sup> VSL2-generated disorder profiles of these three proteins are overlapped to show almost complete coincidence of their major disorder-related features. According to the mean disorder analysis (see <strong><a id="xref-fig-21-4" class="xref-fig" href="#F21">Figures 21A, 21B</a></strong>, and <strong><a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">21C</a></strong>), Nsp3 proteins are predicted to have the following IDPRs, SARS-CoV-2 (1-5, 105-199, 1221-1238), Human SARS (102-189, 355-384, 1195-1223) and Bat CoV (107-182, 352-376, 1191-1217). The First 112 residues in Nsp3 represent a ubiquitin-like globular fold while 113-183 residues form the flexible acidic domain rich in glutamic acid. It is thought to bind and ubiquitinate viral E protein using the N-terminal acidic domain [<a id="xref-ref-132-1" class="xref-bibr" href="#ref-132">132</a>,<a id="xref-ref-133-1" class="xref-bibr" href="#ref-133">133</a>]. This unstructured segment has many MoRFs predicted by ANCHOR and MoRFPred servers which may facilitate the protein-protein interaction (<strong><a id="xref-table-wrap-2-19" class="xref-table" href="#T2">Table 2</a></strong>). Interestingly, Nsp3 of all three viruses was found with highest number of RNA-binding residues (<strong>Supplementary Tables 9, 10</strong>, and <strong>11</strong>).</p></div><div id="sec-31" class="subsection"><h4>Non-structural protein 4 (Nsp4)</h4><p id="p-90">Nsp4 has been reported to induce the formation of the double-membrane vesicles (DMVs) with the co-expression of full-length Nsp3 and Nsp6 proteins for optimal replication inside host cells [<a id="xref-ref-134-1" class="xref-bibr" href="#ref-134">134</a>–<a id="xref-ref-136-1" class="xref-bibr" href="#ref-136">136</a>]. It localizes itself in ER-membrane, when expressed alone but is demonstrated to be present in replication units in infected cells. It was observed that Nsp4 protein contains a tetraspanning transmembrane region having its N- and C-terminals in the cytosol [<a id="xref-ref-137-1" class="xref-bibr" href="#ref-137">137</a>]. No crystal or NMR solution structure is reported for this protein as of yet.</p><p id="p-91">Nsp4 protein of SARS-CoV-2 has multiple substitutions near the N-terminal region and has a quite conserved C-terminus <strong>(Supplementary Figure S2C)</strong>. It is found to be closer to Nsp4 of Bat CoV (81.40% identity) than to Human SARS Nsp4 (80%). Mean PPIDs of Nsp4s from SARS-CoV-2, Human SARS, and Bat CoV are estimated to be 0.80%, 0.60%, and 0.60% respectively. The low level of intrinsic disorder is further illustrated by <strong><a id="xref-fig-22-1" class="xref-fig" href="#F22">Figures 22A, 22B</a></strong>, and <strong><a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22C</a></strong>. With PPIDs around zero, Nsp4 were classified as highly structured proteins, which, however, contain some flexible regions. Likewise, <strong><a id="xref-table-wrap-2-20" class="xref-table" href="#T2">Table 2</a></strong> shows the presence of only N- and C-terminal MoRFs which possibly assist in cleavage of Nsp4 protein from long polyproteins 1a and 1ab.</p><div id="F22" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F22.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of intrinsic disorder propensity of Nsp4. Intrinsic disorder profiles generated for Nsp4 from SARS-CoV-2 (A), Human SARS CoV (B), and Bat CoV (C). The colour schemes are the same as in the corresponding plots of Figure 3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of intrinsic disorder propensity of Nsp4.&lt;/span&gt; Intrinsic disorder profiles generated for Nsp4 from SARS-CoV-2 &lt;strong&gt;(A)&lt;/strong&gt;, Human SARS CoV &lt;strong&gt;(B)&lt;/strong&gt;, and Bat CoV &lt;strong&gt;(C)&lt;/strong&gt;. The colour schemes are the same as in the corresponding plots of &lt;strong&gt;Figure 3&lt;/strong&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 22." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F22.medium.gif" width="440" height="118"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 22." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F22.medium.gif" width="440" height="118"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F22.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 22." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F22.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 22.</span> <span class="caption-title">Analysis of intrinsic disorder propensity of Nsp4.</span><p id="p-92" class="first-child">Intrinsic disorder profiles generated for Nsp4 from SARS-CoV-2 <strong>(A)</strong>, Human SARS CoV <strong>(B)</strong>, and Bat CoV <strong>(C)</strong>. The colour schemes are the same as in the corresponding plots of <strong><a id="xref-fig-3-22" class="xref-fig" href="#F3">Figure 3</a></strong>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-32" class="subsection"><h4>Non-structural protein 5 (Nsp5)</h4><p id="p-93">Also referred to as 3CL-pro, Nsp5 works as a protease that cleaves the replicase polyproteins (1a and 1ab) at 11 major sites [<a id="xref-ref-138-1" class="xref-bibr" href="#ref-138">138</a>,<a id="xref-ref-139-1" class="xref-bibr" href="#ref-139">139</a>]. X-ray crystal structure with 1.5 Å resolution (PDB ID: 5C5O) obtained for Human SARS CoV Nsp5 is shown in <strong><a id="xref-fig-23-1" class="xref-fig" href="#F23">Figure 23D</a></strong>. Here, 3CL-protease is bound to a phenyl-beta-alanyl (S, R)-N-declin type inhibitor. Another crystal structure resolved to 1.96 Å revealed a chymotrypsin-like fold and a conserved substrate-binding site connected to a novel α-helical fold [<a id="xref-ref-140-1" class="xref-bibr" href="#ref-140">140</a>]. Recently, the X-ray crystal structure (resolution 2.16 Å) was solved for the SARS-CoV-2 Nsp5 in complex with an inhibitor N3 (PDB ID: 6LU7) (<strong><a id="xref-fig-23-2" class="xref-fig" href="#F23">Figure 23E</a>)</strong>.</p><div id="F23" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F23.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of intrinsic disorder propensity of Nsp5. Intrinsic disorder profiles generated for Nsp5 from SARS-CoV-2 (A), Human SARS CoV (B), and Bat CoV (C). The colour schemes are the same as in the corresponding plots of Figure 3. (D) A 1.50 &#xC5; X-ray crystal structure of Nsp5 from Human SARS CoV (PDB ID: 5C5O). (E) A 2.16 &#xC5; X-ray crystal structure of SARS-CoV-2 Nsp5 (PDB ID: 6LU7)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of intrinsic disorder propensity of Nsp5.&lt;/span&gt; Intrinsic disorder profiles generated for Nsp5 from SARS-CoV-2 &lt;strong&gt;(A)&lt;/strong&gt;, Human SARS CoV &lt;strong&gt;(B)&lt;/strong&gt;, and Bat CoV &lt;strong&gt;(C)&lt;/strong&gt;. The colour schemes are the same as in the corresponding plots of &lt;strong&gt;Figure 3. (D)&lt;/strong&gt; A 1.50 &#xC5; X-ray crystal structure of Nsp5 from Human SARS CoV (PDB ID: 5C5O). &lt;strong&gt;(E)&lt;/strong&gt; A 2.16 &#xC5; X-ray crystal structure of SARS-CoV-2 Nsp5 (PDB ID: 6LU7).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 23." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F23.medium.gif" width="440" height="209"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 23." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F23.medium.gif" width="440" height="209"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F23.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 23." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F23.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 23.</span> <span class="caption-title">Analysis of intrinsic disorder propensity of Nsp5.</span><p id="p-94" class="first-child">Intrinsic disorder profiles generated for Nsp5 from SARS-CoV-2 <strong>(A)</strong>, Human SARS CoV <strong>(B)</strong>, and Bat CoV <strong>(C)</strong>. The colour schemes are the same as in the corresponding plots of <strong><a id="xref-fig-3-23" class="xref-fig" href="#F3">Figure 3</a>. (D)</strong> A 1.50 Å X-ray crystal structure of Nsp5 from Human SARS CoV (PDB ID: 5C5O). <strong>(E)</strong> A 2.16 Å X-ray crystal structure of SARS-CoV-2 Nsp5 (PDB ID: 6LU7).</p><div class="sb-div caption-clear"></div></div></div><p id="p-95">Nsp5 protein is found to be highly conserved in all three studied CoV viruses. SARS-CoV-2 Nsp5 shares a 96.08% sequence identity with Human SARS Nsp5 and 95.42% with Nsp5 of Bat CoV <strong>(Supplementary Figure S2D)</strong>. Therefore, it not surprising that our analysis demonstrated the identical mean PPID values of 1.96% for Nsp5s from SARS-CoV-2, Human SARS, and Bat CoV (<strong><a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a></strong>). The predicted per-residue intrinsic disorder propensity of SARS-CoV-2, Human SARS, and Bat CoV Nsp5s are presented in <strong><a id="xref-fig-23-3" class="xref-fig" href="#F23">Figures 23A, 23B</a></strong>, and <strong><a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23C</a></strong>, respectively. As the graphs depict, Nsp5s have several flexible regions and N-terminally IDPR of six residues. Due to the low flexibility of this protein, a single MoRF predicted by MoRFchibi_web is present in the N-terminal region (residues 3-8) in Nsp5s of all three viruses (<strong><a id="xref-table-wrap-2-21" class="xref-table" href="#T2">Table 2</a>, Supplementary Tables 7</strong> and <strong>8</strong>). Further, the identified nucleotide-binding residues in Nsp5 of all three viruses are tabulated in <strong>Supplementary Tables 9, 10</strong>, and <strong>11</strong>.</p></div><div id="sec-33" class="subsection"><h4>Non-structural protein 6 (Nsp6)</h4><p id="p-96">Nsp6 protein is involved in blocking ER-induced autophagosome/autolysosome vesicle formation that functions in restricting viral production inside host cells. It induces autophagy by activating the omegasome pathway, which is normally utilized by cells in response to starvation. SARS Nsp6 leads to the generation of small autophagosome vesicles thereby limiting their expansion [<a id="xref-ref-141-1" class="xref-bibr" href="#ref-141">141</a>].</p><p id="p-97">Nsp6 of SARS-CoV-2 is equally close to Nsp6s from both Human SARS and Bat CoV, having a sequence identity of 87.24% (<strong><a id="xref-fig-24-1" class="xref-fig" href="#F24">Figure 24D</a></strong>). According to our analysis, mean PPIDs for Nsp6s are calculated to be 1.03%, 1.03%, and 1.03% for SARS-CoV-2, Human SARS CoV, and Bat CoV, respectively. <strong><a id="xref-fig-24-2" class="xref-fig" href="#F24">Figures 24A, 24B</a></strong>, and <strong><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24C</a></strong> show the corresponding graphs of intrinsic disorder tendency of Nsp6s from SARS-CoV-2, Human SARS CoV, and Bat CoV and demonstrate that these proteins are highly ordered and show low flexibility. As it is a membrane protein, Nsp6 proteins are predicted to have only a single MoRF near the N-terminal region (residues 1-19 in SARS-CoV-2, residues 1-22 in Human SARS, and residues 1-21 in Bat CoV) by the DISOPRED3 server (<strong><a id="xref-table-wrap-2-22" class="xref-table" href="#T2">Table 2</a>, Supplementary Tables 7</strong> and <strong>8</strong>). The role of these protein-binding regions for the induction of autophagy is need to be elucidated.</p><div id="F24" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F24.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of intrinsic disorder propensity of Nsp6. Intrinsic disorder profiles generated for Nsp6 from SARS-CoV-2 (A), Human SARS CoV (B), and Bat CoV (C). The colour schemes are the same as in the corresponding plots of Figure 3. (D) MSA profile of Nsp6 of SARS-CoV-2, Human SARS, and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of intrinsic disorder propensity of Nsp6.&lt;/span&gt; Intrinsic disorder profiles generated for Nsp6 from SARS-CoV-2 &lt;strong&gt;(A)&lt;/strong&gt;, Human SARS CoV &lt;strong&gt;(B)&lt;/strong&gt;, and Bat CoV &lt;strong&gt;(C)&lt;/strong&gt;. The colour schemes are the same as in the corresponding plots of &lt;strong&gt;Figure 3. (D)&lt;/strong&gt; MSA profile of Nsp6 of SARS-CoV-2, Human SARS, and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 24." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F24.medium.gif" width="440" height="324"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 24." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F24.medium.gif" width="440" height="324"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F24.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 24." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F24.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 24.</span> <span class="caption-title">Analysis of intrinsic disorder propensity of Nsp6.</span><p id="p-98" class="first-child">Intrinsic disorder profiles generated for Nsp6 from SARS-CoV-2 <strong>(A)</strong>, Human SARS CoV <strong>(B)</strong>, and Bat CoV <strong>(C)</strong>. The colour schemes are the same as in the corresponding plots of <strong><a id="xref-fig-3-24" class="xref-fig" href="#F3">Figure 3</a>. (D)</strong> MSA profile of Nsp6 of SARS-CoV-2, Human SARS, and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-34" class="subsection"><h4>Non-structural proteins 7 and 8 (Nsp7 and 8)</h4><p id="p-99">The ~10 kDa Nsp7 helps in primase-independent <em>de novo</em> initiation of viral RNA replication by forming a hexadecameric ring-like structure with Nsp8 protein [<a id="xref-ref-142-1" class="xref-bibr" href="#ref-142">142</a>,<a id="xref-ref-143-1" class="xref-bibr" href="#ref-143">143</a>]. Both non-structural proteins 7 and 8 contribute 8 molecules to the ring-structured multimeric viral RNA polymerase. Site-directed mutagenesis in Nsp8 revealed a D/ExD/E motif essential for the <em>in vitro</em> catalysis [<a id="xref-ref-142-2" class="xref-bibr" href="#ref-142">142</a>]. <strong><a id="xref-fig-25-1" class="xref-fig" href="#F25">Figure 25D</a></strong> depicts the 3.1 Å resolution electron microscopy-based structure (PDB ID: 6NUR) of the RDRP-nsp8-nsp7 complex bound to the Nsp12. The structure identified conserved neutral Nsp7 and Nsp8 binding sites overlapping with finger and thumb domains on Nsp12 of the virus [<a id="xref-ref-144-1" class="xref-bibr" href="#ref-144">144</a>].</p><div id="F25" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F25.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of intrinsic disorder propensity of Nsp7. Intrinsic disorder profiles generated for Nsp7 from SARS-CoV-2 (A), Human SARS CoV (B), and Bat CoV (C). The colour schemes are the same as in the corresponding plots of Figure 3. (D) A 3.10 &#xC5; resolution cryo-EM structure of Nsp12-Nsp8-Nsp7 complex (PDB ID: 6NUR). Chain C includes 2-71 residues of Nsp7 (gold colour), chains B and D (dark khaki) represent 77-191 residues of Nsp8 and chain A signifies residues 117-896 and 907-920 of Nsp12 (RNA-directed RNA polymerase) (orange colour) from Human SARS CoV. (E) MSA profile of NSP7 of SARS-CoV-2, Human SARS and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of intrinsic disorder propensity of Nsp7.&lt;/span&gt; Intrinsic disorder profiles generated for Nsp7 from SARS-CoV-2 &lt;strong&gt;(A)&lt;/strong&gt;, Human SARS CoV &lt;strong&gt;(B)&lt;/strong&gt;, and Bat CoV &lt;strong&gt;(C)&lt;/strong&gt;. The colour schemes are the same as in the corresponding plots of &lt;strong&gt;Figure 3. (D)&lt;/strong&gt; A 3.10 &#xC5; resolution cryo-EM structure of Nsp12-Nsp8-Nsp7 complex (PDB ID: 6NUR). Chain C includes 2-71 residues of Nsp7 (gold colour), chains B and D (dark khaki) represent 77-191 residues of Nsp8 and chain A signifies residues 117-896 and 907-920 of Nsp12 (RNA-directed RNA polymerase) (orange colour) from Human SARS CoV. &lt;strong&gt;(E)&lt;/strong&gt; MSA profile of NSP7 of SARS-CoV-2, Human SARS and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 25." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F25.medium.gif" width="440" height="258"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 25." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F25.medium.gif" width="440" height="258"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F25.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 25." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F25.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 25.</span> <span class="caption-title">Analysis of intrinsic disorder propensity of Nsp7.</span><p id="p-100" class="first-child">Intrinsic disorder profiles generated for Nsp7 from SARS-CoV-2 <strong>(A)</strong>, Human SARS CoV <strong>(B)</strong>, and Bat CoV <strong>(C)</strong>. The colour schemes are the same as in the corresponding plots of <strong><a id="xref-fig-3-25" class="xref-fig" href="#F3">Figure 3</a>. (D)</strong> A 3.10 Å resolution cryo-EM structure of Nsp12-Nsp8-Nsp7 complex (PDB ID: 6NUR). Chain C includes 2-71 residues of Nsp7 (gold colour), chains B and D (dark khaki) represent 77-191 residues of Nsp8 and chain A signifies residues 117-896 and 907-920 of Nsp12 (RNA-directed RNA polymerase) (orange colour) from Human SARS CoV. <strong>(E)</strong> MSA profile of NSP7 of SARS-CoV-2, Human SARS and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues.</p><div class="sb-div caption-clear"></div></div></div><p id="p-101">We found that Nsp7 of SARS-CoV-2 share 100% sequence identity with Nsp7 of Bat CoV and 98.80% with Nsp7 from Human SARS (<strong><a id="xref-fig-25-2" class="xref-fig" href="#F25">Figure 25E</a></strong>), while SARS-CoV-2 Nsp8 is closer to Nsp8 of Human SARS (97.47%) than to Nsp8 of Bat CoV (96.46%) (<strong><a id="xref-fig-26-1" class="xref-fig" href="#F26">Figure 26D</a></strong>).</p><div id="F26" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F26.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of intrinsic disorder propensity of Nsp8. Intrinsic disorder profiles generated for Nsp8 from SARS-CoV-2 (A), Human SARS CoV (B), and Bat CoV (C). The colour schemes are the same as in the corresponding plots of Figure 3. (D) MSA profile of Nsp8 of SARS-CoV-2, Human SARS and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of intrinsic disorder propensity of Nsp8.&lt;/span&gt; Intrinsic disorder profiles generated for Nsp8 from SARS-CoV-2 &lt;strong&gt;(A)&lt;/strong&gt;, Human SARS CoV &lt;strong&gt;(B)&lt;/strong&gt;, and Bat CoV &lt;strong&gt;(C)&lt;/strong&gt;. The colour schemes are the same as in the corresponding plots of &lt;strong&gt;Figure 3. (D)&lt;/strong&gt; MSA profile of Nsp8 of SARS-CoV-2, Human SARS and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 26." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F26.medium.gif" width="440" height="274"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 26." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F26.medium.gif" width="440" height="274"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F26.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 26." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F26.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 26.</span> <span class="caption-title">Analysis of intrinsic disorder propensity of Nsp8.</span><p id="p-102" class="first-child">Intrinsic disorder profiles generated for Nsp8 from SARS-CoV-2 <strong>(A)</strong>, Human SARS CoV <strong>(B)</strong>, and Bat CoV <strong>(C)</strong>. The colour schemes are the same as in the corresponding plots of <strong><a id="xref-fig-3-26" class="xref-fig" href="#F3">Figure 3</a>. (D)</strong> MSA profile of Nsp8 of SARS-CoV-2, Human SARS and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues.</p><div class="sb-div caption-clear"></div></div></div><p id="p-103">Due to the high levels of sequence identity, mean PPIDs of all Nsp7s were found to be identical and equal to 9.64%. Both SARS-CoV-2 and Human SARS Nsp8 proteins were calculated to have a mean PPID of 23.74% and, for Nsp8 of Bat CoV mean disorder is predicted to be 22.22%. <strong><a id="xref-fig-25-3" class="xref-fig" href="#F25">Figures 25A, 25B</a></strong>, and <strong><a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25C</a></strong> display the intrinsic disorder profiles for Nsp7s, whereas <strong><a id="xref-fig-26-2" class="xref-fig" href="#F26">Figures 26A, 26B</a></strong>, and <strong><a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26C</a></strong> represent the predicted intrinsic disorder propensity of Nsp8s. As our analysis suggests, Nsp7s might have a well-predicted structure, while Nsp8s are moderately disordered. Nsp8s are predicted to have a long IDPR (residues 44-84) in both SARS-CoV-2 and Human SARS, and a bit shorter IDPR in Bat CoV (residues 48-84). Furthermore, SARS-CoV Nsp7 using its N-terminus residues (V11, C13, V17, and V21) forms a hydrophobic core with Nsp8 residues (M92, M95, L96, M99, and L103). Additionally, H-bonding takes place between Nsp7 Q24 and Nsp8 T89 residues [<a id="xref-ref-143-2" class="xref-bibr" href="#ref-143">143</a>]. These amino acids are the part of MoRFs predicted in Nsp7 and Nsp8 proteins. The results are tabulated in both <strong><a id="xref-table-wrap-2-23" class="xref-table" href="#T2">Table 2</a></strong>, and <strong>Supplementary Tables 7</strong> and <strong>8</strong>. Three protein binding regions in Nsp7 of SARS-CoV-2 (residues 1-30, 39-58, and 65-83), Human SARS (residues 1-30, 44-58, and 64-83), and Bat CoV (residues 1-30, 39-58, and 65-83) was identified by MoRFchibi_web server. Nsp7 was found with the presence of very few nucleotide-binding regions and Nsp8 contains several DNA as well as RNA binding residues (see <strong>Supplementary Tables 9, 10</strong>, and <strong>11</strong>).</p></div><div id="sec-35" class="subsection"><h4>Non-structural protein 9 (Nsp9)</h4><p id="p-104">Nsp9 protein is a single-stranded RNA-binding protein [<a id="xref-ref-145-1" class="xref-bibr" href="#ref-145">145</a>]. It might protect RNA from nucleases by binding and stabilizing viral nucleic acids during replication or transcription [<a id="xref-ref-145-2" class="xref-bibr" href="#ref-145">145</a>]. Our results on nucleotide-binding tendency of Nsp9 shows the presence of several RNA binding and few DNA binding residues in Nsp9 of SARS-CoV-2, Human SARS, and Bat CoV (<strong>Supplementary Tables 9, 10</strong>, and <strong>11</strong>). Presumed to evolve from a protease, Nsp9 forms a dimer using its GXXXG motif [<a id="xref-ref-146-1" class="xref-bibr" href="#ref-146">146</a>,<a id="xref-ref-147-1" class="xref-bibr" href="#ref-147">147</a>]. <strong><a id="xref-fig-27-1" class="xref-fig" href="#F27">Figure 27D</a></strong> shows a 2.7 Å crystal structure of the homodimer of Human SARS Nsp9 (PDB ID: 1QZ8) that identified a unique and previously unreported for other proteins, oligosaccharide/oligonucleotide fold-like fold [<a id="xref-ref-145-3" class="xref-bibr" href="#ref-145">145</a>]. Here, each monomer contains a cone-shaped β-barrel and a C-terminal α-helix arranged into a compact domain [<a id="xref-ref-145-4" class="xref-bibr" href="#ref-145">145</a>].</p><div id="F27" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F27.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of intrinsic disorder propensity of Nsp9. Intrinsic disorder profiles generated for Nsp9 from SARS-CoV-2 (A), Human SARS CoV (B), and Bat CoV (C). The colour schemes are the same as in the corresponding plots of Figure 3. (D) A 2.70 &#xC5; X-ray crystal structure of 3-113 residues of Nsp9 from Human SARS CoV (PDB ID: 1QZ8). (E) MSA profile of Nsp9s from SARS-CoV-2, Human SARS, and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of intrinsic disorder propensity of Nsp9.&lt;/span&gt; Intrinsic disorder profiles generated for Nsp9 from SARS-CoV-2 &lt;strong&gt;(A)&lt;/strong&gt;, Human SARS CoV &lt;strong&gt;(B)&lt;/strong&gt;, and Bat CoV &lt;strong&gt;(C)&lt;/strong&gt;. The colour schemes are the same as in the corresponding plots of &lt;strong&gt;Figure 3. (D)&lt;/strong&gt; A 2.70 &#xC5; X-ray crystal structure of 3-113 residues of Nsp9 from Human SARS CoV (PDB ID: 1QZ8). &lt;strong&gt;(E)&lt;/strong&gt; MSA profile of Nsp9s from SARS-CoV-2, Human SARS, and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 27." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F27.medium.gif" width="440" height="228"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 27." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F27.medium.gif" width="440" height="228"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F27.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 27." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F27.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 27.</span> <span class="caption-title">Analysis of intrinsic disorder propensity of Nsp9.</span><p id="p-105" class="first-child">Intrinsic disorder profiles generated for Nsp9 from SARS-CoV-2 <strong>(A)</strong>, Human SARS CoV <strong>(B)</strong>, and Bat CoV <strong>(C)</strong>. The colour schemes are the same as in the corresponding plots of <strong><a id="xref-fig-3-27" class="xref-fig" href="#F3">Figure 3</a>. (D)</strong> A 2.70 Å X-ray crystal structure of 3-113 residues of Nsp9 from Human SARS CoV (PDB ID: 1QZ8). <strong>(E)</strong> MSA profile of Nsp9s from SARS-CoV-2, Human SARS, and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues.</p><div class="sb-div caption-clear"></div></div></div><p id="p-106">Nsp9 of SARS-CoV-2 is equally similar to Nsp9s from both Human SARS and Bat CoV, having a percentage identity of 97.35%. The difference in three amino acids at 34, 35 and 48 positions accounts for these similarity scores (<strong><a id="xref-fig-27-2" class="xref-fig" href="#F27">Figure 27E</a></strong>). As calculated, the mean PPIDs of Nsp9s of SARS-CoV-2, Human SARS CoV, and Bat CoV are 7.08%, 7.96%, and 7.08% respectively. <strong><a id="xref-fig-27-3" class="xref-fig" href="#F27">Figures 27A, 27B</a></strong>, and <strong><a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27C</a></strong> depict the predicted intrinsic disorder propensity in the Nsp9 protein from SARS-CoV-2, Human SARS, and Bat CoV. According to our analysis, all three Nsp9s are rather structured but contain flexible regions. Nsp9 contains conserved residues (R10, K52, Y53, R55, R74, F75, K86, Y87, F90, K92, R99, and R111) of positively charged side chains suitable for binding with the negatively charged phosphate backbone of RNA and aromatic side-chain amino acids providing stacking interactions [<a id="xref-ref-145-5" class="xref-bibr" href="#ref-145">145</a>].</p><p id="p-107">These residues are a part of multiple disorder-based binding sites predicted by MoRFchibi_web server (<strong><a id="xref-table-wrap-2-24" class="xref-table" href="#T2">Table 2</a>, Supplementary Table 7</strong> and <strong>8</strong>).</p></div><div id="sec-36" class="subsection"><h4>Non-structural protein 10 (Nsp10)</h4><p id="p-108">Nsp10 performs several functions for SARS-CoV. It forms a complex with Nsp14 for dsRNA hydrolysis in 3′ to 5′ direction and activates its exonuclease activity [<a id="xref-ref-148-1" class="xref-bibr" href="#ref-148">148</a>]. It also stimulates the methyltransferase (MTase) activity of Nsp14 required during RNA-cap formation after replication [<a id="xref-ref-149-1" class="xref-bibr" href="#ref-149">149</a>]. <strong><a id="xref-fig-28-1" class="xref-fig" href="#F28">Figure 28D</a></strong> represents the X-ray crystal structure of the Nsp10/Nsp14 complex (PDB ID: 5C8T) [<a id="xref-ref-150-1" class="xref-bibr" href="#ref-150">150</a>]. In agreement with the results of previous biochemical experimental studies, the structure identified important interactions with the ExoN (exonuclease domain) of Nsp14 without affecting its N7-MTase activity [<a id="xref-ref-148-2" class="xref-bibr" href="#ref-148">148</a>,<a id="xref-ref-149-2" class="xref-bibr" href="#ref-149">149</a>].</p><div id="F28" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F28.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of intrinsic disorder propensity of Nsp10. Intrinsic disorder profiles generated for Nsp10 from SARS-CoV-2 (A), Human SARS CoV (B), and Bat CoV (C). The colour schemes are the same as in the corresponding plots of Figure 3. (D) A 3.20 &#xC5; X-ray crystal structure of the Nsp10/Nsp14 complex (PDB ID: 5C8T). In this structure, A and C chain (cornflower blue colour) signify show 1-131 residues of Nsp10 and B and D chains correspond to residues 1-453 and 465-525 of Nsp14 (dim grey colour). (E) MSA profile of Nsp10s from SARS-CoV-2, Human SARS and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of intrinsic disorder propensity of Nsp10.&lt;/span&gt; Intrinsic disorder profiles generated for Nsp10 from SARS-CoV-2 &lt;strong&gt;(A)&lt;/strong&gt;, Human SARS CoV &lt;strong&gt;(B)&lt;/strong&gt;, and Bat CoV &lt;strong&gt;(C)&lt;/strong&gt;. The colour schemes are the same as in the corresponding plots of &lt;strong&gt;Figure 3. (D)&lt;/strong&gt; A 3.20 &#xC5; X-ray crystal structure of the Nsp10/Nsp14 complex (PDB ID: 5C8T). In this structure, A and C chain (cornflower blue colour) signify show 1-131 residues of Nsp10 and B and D chains correspond to residues 1-453 and 465-525 of Nsp14 (dim grey colour). &lt;strong&gt;(E)&lt;/strong&gt; MSA profile of Nsp10s from SARS-CoV-2, Human SARS and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 28." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F28.medium.gif" width="440" height="240"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 28." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F28.medium.gif" width="440" height="240"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F28.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 28." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F28.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 28.</span> <span class="caption-title">Analysis of intrinsic disorder propensity of Nsp10.</span><p id="p-109" class="first-child">Intrinsic disorder profiles generated for Nsp10 from SARS-CoV-2 <strong>(A)</strong>, Human SARS CoV <strong>(B)</strong>, and Bat CoV <strong>(C)</strong>. The colour schemes are the same as in the corresponding plots of <strong><a id="xref-fig-3-28" class="xref-fig" href="#F3">Figure 3</a>. (D)</strong> A 3.20 Å X-ray crystal structure of the Nsp10/Nsp14 complex (PDB ID: 5C8T). In this structure, A and C chain (cornflower blue colour) signify show 1-131 residues of Nsp10 and B and D chains correspond to residues 1-453 and 465-525 of Nsp14 (dim grey colour). <strong>(E)</strong> MSA profile of Nsp10s from SARS-CoV-2, Human SARS and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues.</p><div class="sb-div caption-clear"></div></div></div><p id="p-110">SARS-CoV-2 Nsp10 protein is quite conserved having a 97.12% sequence identity with Nsp10 of Human SARS and 97.84% with Nsp10 of Bat CoV (<strong><a id="xref-fig-28-2" class="xref-fig" href="#F28">Figure 28E</a></strong>). Mean PPIDs of all three studied Nsp10 proteins are found to be 5.04%. <strong><a id="xref-fig-28-3" class="xref-fig" href="#F28">Figures 28A, 28B</a></strong>, and <strong><a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">28C</a></strong> represent disorder profiles of Nsp10s and signify the lack of long IDPRs but presence flexible regions in these proteins. Furthermore, <strong><a id="xref-table-wrap-2-25" class="xref-table" href="#T2">Table 2</a></strong>, and <strong>Supplementary Table 7</strong> and <strong>8</strong> shows that all three virus Nps10 has multiple MoRFs. For SARS-CoV-2, three MoRFs (residues 25-32, 91-99, and 133-138) was identified by MoRFchibi_web server and one MoRF (residues 11-18) was predicted by MoRFPred server. Interestingly, the SARS-CoV Nsp10 residues F16, F19, and V21 form van der Waals interactions with many of the Nsp14 amino acids [<a id="xref-ref-150-2" class="xref-bibr" href="#ref-150">150</a>] and one residue (F16) is located in MoRF region which we have identified. Furthermore, many nucleotide-binding residues which are found in all three viruses (<strong>Supplementary Table 9, 10</strong>, and <strong>11</strong>) and above-mentioned residues are not found to interact with DNA/RNA.</p></div><div id="sec-37" class="subsection"><h4>Non-structural protein 12 (Nsp12)</h4><p id="p-111">In coronaviruses, Nsp12 is an RNA-dependent RNA Polymerase (RDRP). It carries out both primer-independent and primer-dependent synthesis of viral RNA with Mn<sup>2+</sup> as its metallic co-factor and viral Nsp7 and 8 as protein co-factors [<a id="xref-ref-151-1" class="xref-bibr" href="#ref-151">151</a>]. As aforementioned, a 3.1Å resolution structure of Human SARS Nsp12 in association with Nsp7 and Nsp8 proteins (PDB ID: 6NUR) has been reported using electron microscopy (<strong><a id="xref-fig-25-4" class="xref-fig" href="#F25">Figure 25D</a>)</strong>. Nsp12 has a polymerase domain similar to “right hand”, finger domain (398–581, 628–687 residues), palm domain (582–627, 688–815 residues) and a thumb domain (816–919) [<a id="xref-ref-144-2" class="xref-bibr" href="#ref-144">144</a>].</p><p id="p-112">SARS-CoV-2 Nsp12 protein has a highly conserved C-terminal region <strong>(Supplementary Figure S2E)</strong>. It is found to share a 96.35% sequence identity with Human SARS Nsp12 and 95.60% with Bat CoV Nsp12. Mean PPID values for all three Nsp12s are estimated to be 0.43% (<strong><a id="xref-table-wrap-3-7" class="xref-table" href="#T3">Table 3</a></strong>). <strong><a id="xref-fig-29-1" class="xref-fig" href="#F29">Figures 29A, 29B</a></strong>, and <strong><a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29C</a></strong> show that although these proteins are mostly ordered, they have multiple flexible regions. As RDRP protein is observed to be mostly structured, significant MoRFs in disordered regions are not found (<strong><a id="xref-table-wrap-2-26" class="xref-table" href="#T2">Table 2</a>, Supplementary Table 7</strong> and <strong>8</strong>).</p><div id="F29" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F29.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of intrinsic disorder propensity of Nsp12. Intrinsic disorder profiles generated for Nsp12 from SARS-CoV-2 (A), Human SARS CoV (B), and Bat CoV (C). The colour schemes are the same as in the corresponding plots of Figure 3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of intrinsic disorder propensity of Nsp12.&lt;/span&gt; Intrinsic disorder profiles generated for Nsp12 from SARS-CoV-2 &lt;strong&gt;(A)&lt;/strong&gt;, Human SARS CoV &lt;strong&gt;(B)&lt;/strong&gt;, and Bat CoV &lt;strong&gt;(C)&lt;/strong&gt;. The colour schemes are the same as in the corresponding plots of &lt;strong&gt;Figure 3&lt;/strong&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 29." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F29.medium.gif" width="440" height="121"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 29." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F29.medium.gif" width="440" height="121"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F29.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 29." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F29.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 29.</span> <span class="caption-title">Analysis of intrinsic disorder propensity of Nsp12.</span><p id="p-113" class="first-child">Intrinsic disorder profiles generated for Nsp12 from SARS-CoV-2 <strong>(A)</strong>, Human SARS CoV <strong>(B)</strong>, and Bat CoV <strong>(C)</strong>. The colour schemes are the same as in the corresponding plots of <strong><a id="xref-fig-3-29" class="xref-fig" href="#F3">Figure 3</a></strong>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-38" class="subsection"><h4>Non-structural protein 13 (Nsp13)</h4><p id="p-114">Nsp13 functions as a viral helicase and unwinds dsDNA/dsRNA in 5’ to 3’ direction [<a id="xref-ref-152-1" class="xref-bibr" href="#ref-152">152</a>]. Recombinant viral helicase expressed in <em>E.coli</em> Rosetta 2 strain was reported to unwind ~280 bp per second [<a id="xref-ref-152-2" class="xref-bibr" href="#ref-152">152</a>]. <strong><a id="xref-fig-30-1" class="xref-fig" href="#F30">Figure 30D</a></strong> represents a 2.8 Å X-ray crystal structure of Human SARS Nsp13 (PDB ID: 6JYT) [<a id="xref-ref-153-1" class="xref-bibr" href="#ref-153">153</a>]. This helicase contains a <em>β</em>19-<em>β</em>20 loop on 1A domain, which is primarily responsible for its unwinding activity. Furthermore, the study revealed an important interaction of Nsp12 with Nsp13 that further enhances its helicase activity [<a id="xref-ref-153-2" class="xref-bibr" href="#ref-153">153</a>].</p><div id="F30" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F30.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of intrinsic disorder propensity of Nsp13. Intrinsic disorder profiles generated for Nsp13 from SARS-CoV-2 (A), Human SARS CoV (B), and Bat CoV (C). The colour schemes are the same as in the corresponding plots of Figure 3. (D) A 2.80 &#xC5; X-ray crystal structure of 1-596 residues of Human SARS Nsp13 (PDB ID: 6JYT)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of intrinsic disorder propensity of Nsp13.&lt;/span&gt; Intrinsic disorder profiles generated for Nsp13 from SARS-CoV-2 &lt;strong&gt;(A)&lt;/strong&gt;, Human SARS CoV &lt;strong&gt;(B)&lt;/strong&gt;, and Bat CoV &lt;strong&gt;(C)&lt;/strong&gt;. The colour schemes are the same as in the corresponding plots of &lt;strong&gt;Figure 3. (D)&lt;/strong&gt; A 2.80 &#xC5; X-ray crystal structure of 1-596 residues of Human SARS Nsp13 (PDB ID: 6JYT).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 30." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F30.medium.gif" width="440" height="327"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 30." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F30.medium.gif" width="440" height="327"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F30.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 30." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F30.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 30.</span> <span class="caption-title">Analysis of intrinsic disorder propensity of Nsp13.</span><p id="p-115" class="first-child">Intrinsic disorder profiles generated for Nsp13 from SARS-CoV-2 <strong>(A)</strong>, Human SARS CoV <strong>(B)</strong>, and Bat CoV <strong>(C)</strong>. The colour schemes are the same as in the corresponding plots of <strong><a id="xref-fig-3-30" class="xref-fig" href="#F3">Figure 3</a>. (D)</strong> A 2.80 Å X-ray crystal structure of 1-596 residues of Human SARS Nsp13 (PDB ID: 6JYT).</p><div class="sb-div caption-clear"></div></div></div><p id="p-116">The 601-amino-acid-long Nsp13 of SARS-CoV-2 is almost completely conserved, as it shares 99.83% with Nsp13 of Humans SARS and 98.84% with Nsp13 of Bat CoV <strong>(Supplementary Figure S2F)</strong>. In accordance with our results, the mean PPIDs of all three Nsp13 proteins are estimated to be 0.67%. <strong><a id="xref-fig-30-2" class="xref-fig" href="#F30">Figures 30A, 30B</a></strong>, and <strong><a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30C</a></strong> show that Nsp13s contain multiple flexible regions but do not possess significant disorder. As expected, being a low disorder protein Nsp13 does not contain any MoRF region and not a single binding-region is located by any server used in all three viruses (<strong><a id="xref-table-wrap-2-27" class="xref-table" href="#T2">Table 2</a>, Supplementary Table 7</strong> and <strong>8</strong>). It has many nucleotide-binding residues (RNA and DNA) which are tabulated in <strong>Supplementary Tables 9, 10</strong>, and <strong>11</strong>.</p></div><div id="sec-39" class="subsection"><h4>Non-structural protein 14</h4><p id="p-117">Nsp14 is a multifunctional viral protein that acts as an Exoribonuclease (ExoN) and methyltransferase (N7-MTase) in SARS coronaviruses. It’s 3’ to 5’ exonuclease activity lies in the conserved DEDD residues related to exonuclease superfamily [<a id="xref-ref-154-1" class="xref-bibr" href="#ref-154">154</a>]. Its guanine-N7 methyltransferase activity depends upon the S-adenosyl-L-methionine (AdoMet) as a cofactor [<a id="xref-ref-149-3" class="xref-bibr" href="#ref-149">149</a>]. As aforementioned, Nsp14 requires Nsp10 for activating its ExoN and N7-MTase activity inside the host cells. <strong><a id="xref-fig-28-4" class="xref-fig" href="#F28">Figure 28D</a></strong> depicts the 3.2 Å crystal structure of human SARS nsp10/nsp14 complex (PDB ID: 5C8T), where amino acids 1-287 form the ExoN domain and 288-527 residues form the N7-MTase domain of nsp14. A loop (residues 288-301) is essential for its N7-MTase activity [<a id="xref-ref-150-3" class="xref-bibr" href="#ref-150">150</a>].</p><p id="p-118">SARS-CoV-2 Nsp14 protein shares a 95.07% identity with Human SARS Nsp14 and 94.69% with Bat CoV Nsp14 (<strong>Supplementary Figure S2G</strong>). Mean PPID values for Nsp14s from SARS-CoV-2 and Human SARS is calculated to be 0.38%, while the Nsp14 from Bat CoV has a mean PPID 0.57%. Predicted per-residue intrinsic disorder propensity of Nsp14s from SARS-CoV-2, Human SARS, and Bat CoV is represented in <strong><a id="xref-fig-31-1" class="xref-fig" href="#F31">Figures 31A, 31B</a></strong>, and <strong><a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31C</a></strong>, respectively. As can be observed from these plots and corresponding PPID values, all Nsp14s are found to be highly structured. Likewise, <strong><a id="xref-table-wrap-2-28" class="xref-table" href="#T2">Table 2</a></strong> shows Nsp14 contains two protein binding regions (residues 8-13 and 441-445) predicted by the MoRFPred server in all three viruses. As shown in <strong>Supplementary Tables 9, 10</strong>, and <strong>11</strong>, the ORF14 represents multiple nucleotide-binding residues.</p><div id="F31" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F31.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of intrinsic disorder propensity of Nsp14. Intrinsic disorder profiles generated for Nsp14 from SARS-CoV-2 (A), Human SARS CoV (B), and Bat CoV (C). The colour schemes are the same as in the corresponding plots of Figure 3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of intrinsic disorder propensity of Nsp14.&lt;/span&gt; Intrinsic disorder profiles generated for Nsp14 from SARS-CoV-2 &lt;strong&gt;(A)&lt;/strong&gt;, Human SARS CoV &lt;strong&gt;(B)&lt;/strong&gt;, and Bat CoV &lt;strong&gt;(C)&lt;/strong&gt;. The colour schemes are the same as in the corresponding plots of &lt;strong&gt;Figure 3&lt;/strong&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 31." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F31.medium.gif" width="440" height="120"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 31." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F31.medium.gif" width="440" height="120"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F31.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 31." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F31.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 31.</span> <span class="caption-title">Analysis of intrinsic disorder propensity of Nsp14.</span><p id="p-119" class="first-child">Intrinsic disorder profiles generated for Nsp14 from SARS-CoV-2 <strong>(A)</strong>, Human SARS CoV <strong>(B)</strong>, and Bat CoV <strong>(C)</strong>. The colour schemes are the same as in the corresponding plots of <strong><a id="xref-fig-3-31" class="xref-fig" href="#F3">Figure 3</a></strong>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-40" class="subsection"><h4>Non-structural protein 15 (Nsp15)</h4><p id="p-120">Nsp15 is a uridylate-specific RNA Endonuclease (NendoU) that creates a 2′–3′ cyclic phosphates after cleavage. Its endonuclease activity depends upon Mn<sup>2+</sup> ions as co-factors. Conserved in Nidovirus, it acts as an important genetic marker due to its absence in other RNA viruses [<a id="xref-ref-155-1" class="xref-bibr" href="#ref-155">155</a>]. <strong><a id="xref-fig-32-1" class="xref-fig" href="#F32">Figure 32D</a></strong> represents a 2.6 Å crystal structure of Uridylate-specific Nsp15 (PDB ID: 2H85) that was deduced by Bruno and colleagues using X-ray diffraction [<a id="xref-ref-156-1" class="xref-bibr" href="#ref-156">156</a>]. The monomeric Nsp15 has three domains: N-terminal domain (1-62 residues) formed by a three anti-parallel <em>β</em>-strands and two α-helices packed together; a middle domain (63–191 residues) that contains an α-helix connected via a 39-amino-acid-long coil to an ordered region containing two α-helices and five <em>β</em>-strands; and a C-terminal domain (192–345 residues) consisting of two anti-parallel three <em>β</em>-strand sheets on each side of a central α-helical core [<a id="xref-ref-156-2" class="xref-bibr" href="#ref-156">156</a>].</p><div id="F32" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F32.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of intrinsic disorder propensity of Nsp15. Intrinsic disorder profiles generated for Nsp15 from SARS-CoV-2 (A), Human SARS CoV (B), and Bat CoV (C). The colour schemes are the same as in the corresponding plots of Figure 3. (D) A 2.60 &#xC5; X-ray crystal structure of Nsp15 from Human SARS CoV (PDB ID: 2H85) (rosy brown colour). Residues 151-157 predicted to be disordered are represented in forest-green colour." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of intrinsic disorder propensity of Nsp15.&lt;/span&gt; Intrinsic disorder profiles generated for Nsp15 from SARS-CoV-2 &lt;strong&gt;(A)&lt;/strong&gt;, Human SARS CoV &lt;strong&gt;(B)&lt;/strong&gt;, and Bat CoV &lt;strong&gt;(C)&lt;/strong&gt;. The colour schemes are the same as in the corresponding plots of &lt;strong&gt;Figure 3. (D)&lt;/strong&gt; A 2.60 &#xC5; X-ray crystal structure of Nsp15 from Human SARS CoV (PDB ID: 2H85) (rosy brown colour). Residues 151-157 predicted to be disordered are represented in forest-green colour.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 31." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F32.medium.gif" width="440" height="355"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 31." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F32.medium.gif" width="440" height="355"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F32.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 31." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F32.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 31.</span> <span class="caption-title">Analysis of intrinsic disorder propensity of Nsp15.</span><p id="p-121" class="first-child">Intrinsic disorder profiles generated for Nsp15 from SARS-CoV-2 <strong>(A)</strong>, Human SARS CoV <strong>(B)</strong>, and Bat CoV <strong>(C)</strong>. The colour schemes are the same as in the corresponding plots of <strong><a id="xref-fig-3-32" class="xref-fig" href="#F3">Figure 3</a>. (D)</strong> A 2.60 Å X-ray crystal structure of Nsp15 from Human SARS CoV (PDB ID: 2H85) (rosy brown colour). Residues 151-157 predicted to be disordered are represented in forest-green colour.</p><div class="sb-div caption-clear"></div></div></div><p id="p-122">The Nsp15 is found to be quite conserved across human SARS and bat CoVs. SARS-CoV-2 Nsp15 shares an 88.73% sequence identity with Nsp15 of Human SARS and 88.15% with Nsp15 of Bat CoV (<strong>Supplementary Figure S2H</strong>). Calculated mean PPIDs of Nsp15s from SARS-CoV-2, Human SARS, and Bat CoV are 1.73%, 2.60%, and 2.60%, respectively. Similar to many other non-structural proteins of coronaviruses, Nsp15s from SARS-CoV-2, Human SARS, and Bat CoV are predicted to possess multiple flexible regions but contain virtually no IDPRs (see <strong><a id="xref-fig-32-2" class="xref-fig" href="#F32">Figures 32A, 32B</a></strong>, and <strong><a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">32C</a></strong>). Similarly, no significant disorder-binding regions are predicted in Nsp15 proteins (<strong><a id="xref-table-wrap-2-29" class="xref-table" href="#T2">Table 2</a></strong>). SARS-CoV-2 contain one MoRF (residues 9-13) predicted by MoRFPred server. Human SARS do not have a single MoRF while Bat CoV possesses two very short binding regions (<strong>Supplementary Table 7</strong> and <strong>8</strong>). <strong>Supplementary Table 9, 10</strong>, and <strong>11</strong> depicts the presence of many RNA binding residues and few DNA binding residues in Nsp15 of all three viruses.</p></div><div id="sec-41" class="subsection"><h4>Non-structural protein 16 (Nsp16)</h4><p id="p-123">Nsp16 protein is another MTase domain-containing protein. As methylation of coronavirus mRNAs occurs in steps, three proteins Nsp10, Nsp14, and Nsp16 acts one after another. The first event requires the initiation trigger from Nsp10 protein, after which Nsp14 methylates capped mRNAs forming cap-0 (7Me) GpppA-RNAs. Nsp16 protein, along with its co-activator protein Nsp10, acts on cap-0 (7Me) GpppA-RNAs to give rise to final cap-1 (7Me)GpppA(2’OMe)-RNAs [<a id="xref-ref-149-4" class="xref-bibr" href="#ref-149">149</a>,<a id="xref-ref-157-1" class="xref-bibr" href="#ref-157">157</a>]. A 2 Å X-ray crystal structure of the Human SARS nsp10-nsp16 complex is depicted in <strong><a id="xref-fig-33-1" class="xref-fig" href="#F33">Figure 33D</a></strong>(PDB ID: 3R24) [<a id="xref-ref-158-1" class="xref-bibr" href="#ref-158">158</a>]. The structure consists of a characteristic fold present in class I MTase family comprising of α-helices and loops surrounding a seven-stranded β-sheet [<a id="xref-ref-158-2" class="xref-bibr" href="#ref-158">158</a>].</p><div id="F33" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F33.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of intrinsic disorder propensity of Nsp16. (A) Intrinsic disorder profiles generated for Nsp15 from SARS-CoV-2 (A), Human SARS CoV (B), and Bat CoV (C). The colour schemes are the same as in the corresponding plots of Figure 3. (D) A 2.60 &#xC5; X-ray crystal structure of the Human SARS Nsp10-Nsp16 complex (PDB ID: 3R24). Chain A shown in turquoise colour corresponds to the residues 3-294 of Nsp16" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of intrinsic disorder propensity of Nsp16.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; Intrinsic disorder profiles generated for Nsp15 from SARS-CoV-2 &lt;strong&gt;(A)&lt;/strong&gt;, Human SARS CoV &lt;strong&gt;(B)&lt;/strong&gt;, and Bat CoV &lt;strong&gt;(C)&lt;/strong&gt;. The colour schemes are the same as in the corresponding plots of &lt;strong&gt;Figure 3. (D)&lt;/strong&gt; A 2.60 &#xC5; X-ray crystal structure of the Human SARS Nsp10-Nsp16 complex (PDB ID: 3R24). Chain A shown in turquoise colour corresponds to the residues 3-294 of Nsp16&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 33." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F33.medium.gif" width="440" height="357"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 33." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F33.medium.gif" width="440" height="357"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F33.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 33." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F33.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 33.</span> <span class="caption-title">Analysis of intrinsic disorder propensity of Nsp16.</span><p id="p-124" class="first-child"><strong>(A)</strong> Intrinsic disorder profiles generated for Nsp15 from SARS-CoV-2 <strong>(A)</strong>, Human SARS CoV <strong>(B)</strong>, and Bat CoV <strong>(C)</strong>. The colour schemes are the same as in the corresponding plots of <strong><a id="xref-fig-3-33" class="xref-fig" href="#F3">Figure 3</a>. (D)</strong> A 2.60 Å X-ray crystal structure of the Human SARS Nsp10-Nsp16 complex (PDB ID: 3R24). Chain A shown in turquoise colour corresponds to the residues 3-294 of Nsp16</p><div class="sb-div caption-clear"></div></div></div><p id="p-125">Nsp16 protein of SARS-CoV-2 is found to be equally similar to Nsp16s from Human SARS and Bat CoV (93.29%) (<strong>Supplementary Figure S2I</strong>). Mean PPIDs for Nsp16s from SARS-CoV-2, Human SARS, and Bat CoV are 5.37%, 3.02%, and 3.02%, respectively. In line with these PPIDs values, <strong><a id="xref-fig-33-2" class="xref-fig" href="#F33">Figures 33A</a>, <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">33B</a></strong>, and <strong><a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">33C</a></strong> show that Nsp16s are mostly ordered proteins containing several flexible regions. Correspondingly, no significant MoRFs are present in this protein (<strong><a id="xref-table-wrap-2-30" class="xref-table" href="#T2">Table 2</a>, Supplementary Table 7</strong> and <strong>8</strong>). A single MoRF (residues 151-156) were found with the help of MoRFPred in all three viruses. Further, several RNA-binding and few DNA-binding residues are also identified (<strong>Supplementary Table 9, 10, and 11</strong>).</p></div><div id="sec-42" class="subsection"><h4>Replicase polyprotein 1a</h4><p id="p-126">Since replicase polyprotein 1a contains non-structural proteins 1-10 identical to those found in replicase polyprotein 1ab, we did not perform their disorder analysis separately. However, replicase polyprotein 1a has one additional non-structural protein designated as Nsp11.</p></div><div id="sec-43" class="subsection"><h4>Non-structural protein 11 (Nsp11)</h4><p id="p-127">Nsp11 is a small uncharacterized protein cleaved from the replicase polyprotein 1a. This small protein with unknown function requires experimental insights to further characterize this protein. The intrinsic disorder predicting software used in this study requires amino acid sequences, which are at least 30-residue long. Therefore, because of their short sequences (just 13 residues) Nsp11s from all three studied coronaviruses were not checked for the intrinsic disorder, disorder-based protein binding regions, and nucleotide-binding residues. Based on the MSA outputs, Nsp11 from SARS-CoV-2 was found to have a sequence identity of 84.62% with Nsp11s from Human SARS and Bat CoV (<strong><a id="xref-fig-34-1" class="xref-fig" href="#F34">Figure 34</a></strong>).</p><div id="F34" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F34.large.jpg?width=800&amp;height=600&amp;carousel=1" title="MSA profile of Nsp11 of SARS-CoV-2, Human SARS and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818817006" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;MSA profile of Nsp11 of SARS-CoV-2, Human SARS and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 34." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F34.medium.gif" width="440" height="120"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 34." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F34.medium.gif" width="440" height="120"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F34.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 34." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.13.990598/F34.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 34.</span> <p id="p-128" class="first-child">MSA profile of Nsp11 of SARS-CoV-2, Human SARS and Bat CoV. Red shaded sequences represent identical regions of these proteins, whereas red characters show similar residues.</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-44" class="subsection"><h3>Concluding remarks</h3><p id="p-129">The emergence of new viruses and associated deaths around the globe represent one of the major concerns of modern times. Despite its pandemic nature, there is very little information available in the public domain regarding the structures and functions of SARS-CoV-2 proteins. Based on its similarity with Human SARS CoV and Bat CoV, the published reports have suggested the functions of SARS-CoV-2 proteins. In this study, we utilized information available on SARS-CoV-2 genome and translated proteome from GenBank, and carried out a comprehensive computational analysis of the prevalence of the intrinsic disorder in SARS-CoV-2 proteins. Additionally, a comparison was also made with proteins from close relatives of SARS-CoV-2 from the same group of beta coronaviruses, Human SARS CoV and Bat CoV. Our analysis revealed that in these three CoVs, the N proteins are highly disordered, possessing the PPID values of more than 60%. These viruses also have several moderately disordered proteins, such as Nsp8, Orf6, and Orf9b. Although other proteins have shown lower disorder content, almost all of them contain at least some IDPRs, and all CoV proteins analysed in this study definitely have multiple flexible regions. Importantly, our study provides novel information on presence of intrinsic disorder at the cleavage sites of the replicase 1ab polyprotein of CoVs. This observation confirms the crucial role of IDPRs in maturation of individual proteins. We also established that many of these proteins contain disorder-based binding motifs. Since IDPs/IDPRs might undergo structural transition upon association with their physiological partners, our study generates important grounds for better understanding of the functionality of these proteins, their interactions with other viral proteins, as well as interaction with host proteins in different physiological conditions. This will also guide structural biologists to carry out a structure-based analysis of SARS-CoV-2 proteome to explore the path for the development of new drugs and vaccines.</p></div><div id="sec-45" class="subsection"><h3>Future perspective</h3><p id="p-130">The periodical outbreaks of pathogens worldwide always remind the lack of suitable drugs or vaccines for proper cure or treatment. In 2003, nearly 750 deaths were reported due to the SARS outbreak in more than 24 countries. But this time, the outbreak of Wuhan’s novel coronavirus (SARS-CoV-2) has quickly surpassed this number, indicating more causalities soon. The lack of accurate information and ignorance of primary symptoms are major reasons, which cause many infection cases. Although efficient transmission from human to human is confirmed, the actual reasons for fast SARS-CoV-2 spread are still unknown, but some assumptions were made by researchers and Chinese authorities. The fast spread of SARS-CoV-2, COVID-19 pandemic, and associated introduction of quarantine also have made major impacts on economy and education worldwide due to several restrictions, such as limited transportation, restrained or frozen traveling, halted attendance of mass events, the introduction of distant teaching and learning, etc. Due to advancements in sequencing techniques, the full genome sequence of SARS-CoV-2 was made available in a few days of the first infection report from Wuhan, China. However, massive subsequent research needs to be done to identify the actual cause of SARS-CoV-2 infectivity and to design suitable treatment in the coming future. Certain possibilities can be explored with the available information. The mutational pressure study on this virus will be very interesting to see if this virus transforms from Bat SARS to Human SARS to SARS-CoV-2. More in-depth experimental studies using molecular and cell biology techniques to establish structure-function relationships are required for a better understanding of the functioning of SARS-CoV-2 proteins. Additionally, based on the sequence homology and information on protein-protein interactions, the associated viral and host proteins should be explored, for finding means suitable for limiting replication, maturation, and ultimately pathogenesis of this virus. Although structural biology techniques (so-called rational drug design) can be used in drug development utilizing high throughput screening of compounds virtually or experimentally, the applicability of these techniques is limited by the presence of intrinsic disorder in target proteins. Therefore, the thorough disorder analysis of three coronaviruses conducted in this study will help structural biologists to rationally design experiments keeping this information in mind.</p></div></div><div class="section" id="sec-47"><h2 class="">Authors Contribution</h2><p id="p-153">RG: Conception and design, interpretation of data, writing, and review of the manuscript, and study supervision. VNU: Conception and design, acquisition and interpretation of data, writing, and review of the manuscript. MS, TB, PK, BRG, KG: acquisition and interpretation of data, writing of the manuscript.</p></div><div class="section COI-statement" id="sec-48"><h2 class="">Conflict of Interest</h2><p id="p-155">All authors declare that there is no financial competing interest.</p></div><div class="section" id="sec-49"><h2 class="">Supplementary Figure legends</h2><p id="p-156"><strong>Supplementary Table 1.</strong> Evaluation of intrinsic disorder in structural and accessory proteins of SARS-CoV-2.</p><p id="p-157"><strong>Supplementary Table 2.</strong> Evaluation of intrinsic disorder in structural and accessory proteins of Human SARS.</p><p id="p-158"><strong>Supplementary Table 3.</strong> Evaluation of intrinsic disorder in structural and accessory proteins of Bat CoV.</p><p id="p-159"><strong>Supplementary Table 4.</strong> Evaluation of intrinsic disorder in non-structural proteins SARS-CoV-2.</p><p id="p-160"><strong>Supplementary Table 5.</strong> Evaluation of intrinsic disorder in non-structural proteins of Human SARS.</p><p id="p-161"><strong>Supplementary Table 6.</strong> Evaluation of intrinsic disorder in non-structural proteins of Bat CoV.</p><p id="p-162"><strong>Supplementary Table 7:</strong> Predicted MoRF residues in Human SARS proteins.</p><p id="p-163"><strong>Supplementary Table 8:</strong> Predicted MoRF residues in Bat CoV proteins.</p><p id="p-164"><strong>Supplementary Table 9:</strong> Predicted nucleotide-binding residues in SARS-CoV-2 proteins.</p><p id="p-165"><strong>Supplementary Table 10:</strong> Predicted nucleotide-binding residues in Human SARS proteins.</p><p id="p-166"><strong>Supplementary Table 11:</strong> Predicted nucleotide-binding residues in BAT CoV proteins.</p><p id="p-167"><strong>Supplementary Figures S1.</strong> Multiple sequence alignment of structural proteins of all three studied coronaviruses are generated using Clustal Omega. The aligned images are created using Esprit 3.0.</p><p id="p-168"><strong>Figure S1A.</strong> MSA of SARS-CoV-2, Human SARS, and Bat CoV spike glycoproteins.</p><p id="p-169"><strong>Figure S1B.</strong> MSA of SARS-CoV-2, Human SARS, and Bat CoV Nucleoproteins.</p><p id="p-170"><strong>Supplementary Figure S2.</strong> Multiple sequence alignment of non-structural proteins of all three studied coronaviruses are generated using Clustal Omega. The aligned images are created using Esprit 3.0.</p><p id="p-171"><strong>Figure S2A.</strong> MSA of SARS-CoV-2, Human SARS, and Bat CoV Nsp2 proteins.</p><p id="p-172"><strong>Figure S2B.</strong> MSA of SARS-CoV-2, Human SARS, and Bat CoV Nsp3 proteins.</p><p id="p-173"><strong>Figure S2C.</strong> MSA of SARS-CoV-2, Human SARS, and Bat CoV Nsp4 proteins.</p><p id="p-174"><strong>Figure S2D.</strong> MSA of SARS-CoV-2, Human SARS, and Bat CoV Nsp5 proteins.</p><p id="p-175"><strong>Figure S2E.</strong> MSA of SARS-CoV-2, Human SARS, and Bat CoV Nsp12 proteins.</p><p id="p-176"><strong>Figure S2F.</strong> MSA of SARS-CoV-2, Human SARS, and Bat CoV Nsp13 proteins.</p><p id="p-177"><strong>Figure S2G.</strong> MSA of SARS-CoV-2, Human SARS, and Bat CoV Nsp14 proteins.</p><p id="p-178"><strong>Figure S2H.</strong> MSA of SARS-CoV-2, Human SARS, and Bat CoV Nsp15 proteins.</p><p id="p-179"><strong>Figure S2I.</strong> MSA of SARS-CoV-2, Human SARS, and Bat CoV Nsp16 proteins.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-154">All the authors would like to thank IIT Mandi for providing facilities. MS and BRG were supported by MHRD for funding. KG was supported by the Department of Biotechnology (DBT), India (BT/PR16871/NER/95/329/2015). PK was supported by IIT Mandi-IIT Ropar-PGI Chandigarh, BioX consortium grant (IITM/INT/RG/18). TB is grateful to the Department of Science and Technology for her INSPIRE fellowship for Funding.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-update fn-group-summary-of-updates" id="fn-1"><p id="p-4">We have added the molecular recognition features data and results.</p></li></ul></div><div class="section glossary"><h2 class="">List of Abbreviations</h2><dl id="def-list-1"><dt id="def-item-1">ACE2<span class="def-item-dt-sep"></span></dt><dd>Angiotensin-converting enzyme 2<span class="def-item-dd-sep"></span></dd><dt id="def-item-2">CDF<span class="def-item-dt-sep"></span></dt><dd>Cumulative distribution function<span class="def-item-dd-sep"></span></dd><dt id="def-item-3">CH<span class="def-item-dt-sep"></span></dt><dd>Charge hydropathy<span class="def-item-dd-sep"></span></dd><dt id="def-item-4">COVID-19<span class="def-item-dt-sep"></span></dt><dd>Coronavirus disease 2019<span class="def-item-dd-sep"></span></dd><dt id="def-item-5">CTD<span class="def-item-dt-sep"></span></dt><dd>Cterminal domain<span class="def-item-dd-sep"></span></dd><dt id="def-item-6">DMVs<span class="def-item-dt-sep"></span></dt><dd>Double-membrane vesicles<span class="def-item-dd-sep"></span></dd><dt id="def-item-7">ICTV<span class="def-item-dt-sep"></span></dt><dd>International committee on taxonomy of viruses<span class="def-item-dd-sep"></span></dd><dt id="def-item-8">IDP<span class="def-item-dt-sep"></span></dt><dd>Intrinsically disordered proteins<span class="def-item-dd-sep"></span></dd><dt id="def-item-9">IDPRs<span class="def-item-dt-sep"></span></dt><dd>Intrinsically disordered protein regions<span class="def-item-dd-sep"></span></dd><dt id="def-item-10">IFN<span class="def-item-dt-sep"></span></dt><dd>Interferon<span class="def-item-dd-sep"></span></dd><dt id="def-item-11">MoRFs<span class="def-item-dt-sep"></span></dt><dd>Molecular recognition features<span class="def-item-dd-sep"></span></dd><dt id="def-item-12">MSA<span class="def-item-dt-sep"></span></dt><dd>Multiple sequence alignment<span class="def-item-dd-sep"></span></dd><dt id="def-item-13">Nsps<span class="def-item-dt-sep"></span></dt><dd>Non-structural proteins<span class="def-item-dd-sep"></span></dd><dt id="def-item-14">NTD<span class="def-item-dt-sep"></span></dt><dd>N-terminal domain<span class="def-item-dd-sep"></span></dd><dt id="def-item-15">PONDR<span class="def-item-dt-sep"></span></dt><dd>Predictor of natural disordered regions<span class="def-item-dd-sep"></span></dd><dt id="def-item-16">PPID<span class="def-item-dt-sep"></span></dt><dd>Predicted percentage of intrinsic disorder<span class="def-item-dd-sep"></span></dd><dt id="def-item-17">Pprint<span class="def-item-dt-sep"></span></dt><dd>Prediction of Protein RNA-Interaction<span class="def-item-dd-sep"></span></dd><dt id="def-item-18">RBD<span class="def-item-dt-sep"></span></dt><dd>Receptor binding domain<span class="def-item-dd-sep"></span></dd><dt id="def-item-19">SARS<span class="def-item-dt-sep"></span></dt><dd>Severe acute respiratory syndrome<span class="def-item-dd-sep"></span></dd><dt id="def-item-20">TRS<span class="def-item-dt-sep"></span></dt><dd>Transcriptional regulatory sequences<span class="def-item-dd-sep"></span></dd><dt id="def-item-21">VLPs<span class="def-item-dt-sep"></span></dt><dd>Virus-like particles<span class="def-item-dd-sep"></span></dd><dt id="def-item-22">WHO<span class="def-item-dt-sep"></span></dt><dd>World health organization</dd></dl></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.1" data-doi="10.1016/S2213-2600(20)30079-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shu</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Clinical course and outcomes of critically ill patients with SARS-CoV-2 pneumonia in Wuhan, China: a single-centered, retrospective, observational study</span>. <abbr class="cit-jnl-abbrev">Lancet Respir Med.</abbr> <span class="cit-pub-date">2020</span> [cited 29 Feb 2020].<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/S2213-2600(20)30079-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%2BRespir%2BMed.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS2213-2600%252820%252930079-5%26rft_id%253Dinfo%253Apmid%252F32105632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S2213-2600(20)30079-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32105632&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.13.990598v2.2"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Coronavirus disease 2019</span>. [cited 29 Feb 2020]. Available: <a href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019">https://www.who.int/emergencies/diseases/novel-coronavirus-2019</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.3" data-doi="10.1016/j.virusres.2006.01.017"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Enjuanes</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ.</span></span> <span class="cit-article-title">Nidovirales: evolving the largest RNA virus genome</span>. <abbr class="cit-jnl-abbrev">Virus Res.</abbr> <span class="cit-pub-date">2006</span>;<span class="cit-vol">117</span>: <span class="cit-fpage">17</span>–<span class="cit-lpage">37</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virusres.2006.01.017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DGorbalenya%26rft.auinit1%253DA.%2BE.%26rft.volume%253D117%26rft.issue%253D1%26rft.spage%253D17%26rft.epage%253D37%26rft.atitle%253DNidovirales%253A%2Bevolving%2Bthe%2Blargest%2BRNA%2Bvirus%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2006.01.017%26rft_id%253Dinfo%253Apmid%252F16503362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2006.01.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16503362&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236746000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.4" data-doi="10.1007/s00108-019-00671-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Corman</span>  <span class="cit-name-given-names">VM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lienau</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Witzenrath</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">[Coronaviruses as the cause of respiratory infections]</span>. <abbr class="cit-jnl-abbrev">Internist (Berl).</abbr> <span class="cit-pub-date">2019</span>;<span class="cit-vol">60</span>: <span class="cit-fpage">1136</span>–<span class="cit-lpage">1145</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/s00108-019-00671-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternist%2B%2528Berl%2529.%26rft.volume%253D60%26rft.spage%253D1136%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00108-019-00671-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00108-019-00671-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">5.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.13.990598v2.5" data-doi="10.1128/JVI.06540-11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woo</span>  <span class="cit-name-given-names">PCY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">SKP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">CSF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">CCY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tsang</span>  <span class="cit-name-given-names">AKL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">JHN</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Discovery of Seven Novel Mammalian and Avian Coronaviruses in the Genus Deltacoronavirus Supports Bat Coronaviruses as the Gene Source of Alphacoronavirus and Betacoronavirus and Avian Coronaviruses as the Gene Source of Gammacoronavirus and Deltacoronavi</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2012</span>;<span class="cit-vol">86</span>: <span class="cit-fpage">3995</span>–<span class="cit-lpage">4008</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.06540-11</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.06540-11%26rft_id%253Dinfo%253Apmid%252F22278237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg2LzcvMzk5NSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAzLzIwMjAuMDMuMTMuOTkwNTk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">6.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.13.990598v2.6" data-doi="10.3201/eid1905.130057"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cotten</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">TT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Watson</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Palser</span>  <span class="cit-name-given-names">AL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Petrova</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Full-genome deep sequencing and phylogenetic analysis of novel human betacoronavirus</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis.</abbr> <span class="cit-pub-date">2013</span>;<span class="cit-vol">19</span>: <span class="cit-fpage">736</span>–<span class="cit-lpage">742</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.3201/eid1905.130057</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg%2BInfect%2BDis.%26rft.volume%253D19%26rft.spage%253D736%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid1905.130057%26rft_id%253Dinfo%253Apmid%252F23693015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid1905.130057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23693015&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.7" data-doi="10.1016/S0065-3527(06)66005-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Masters</span>  <span class="cit-name-given-names">PS.</span></span> <span class="cit-article-title">The molecular biology of coronaviruses</span>. <abbr class="cit-jnl-abbrev">Adv Virus Res.</abbr> <span class="cit-pub-date">2006</span>;<span class="cit-vol">66</span>: <span class="cit-fpage">193</span>–<span class="cit-lpage">292</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/S0065-3527(06)66005-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bvirus%2Bresearch%26rft.stitle%253DAdv%2BVirus%2BRes%26rft.aulast%253DMasters%26rft.auinit1%253DP.%2BS.%26rft.volume%253D66%26rft.spage%253D193%26rft.epage%253D292%26rft.atitle%253DThe%2Bmolecular%2Bbiology%2Bof%2Bcoronaviruses.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-3527%252806%252966005-3%26rft_id%253Dinfo%253Apmid%252F16877062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0065-3527(06)66005-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16877062&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240137400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.8" data-doi="10.1128/JVI.79.9.5288-5295.2005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hussain</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Identification of Novel Subgenomic RNAs and Noncanonical Transcription Initiation Signals of Severe Acute Respiratory Syndrome Coronavirus</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2005</span>;<span class="cit-vol">79</span>: <span class="cit-fpage">5288</span>–<span class="cit-lpage">5295</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.79.9.5288-5295.2005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DHussain%26rft.auinit1%253DS.%26rft.volume%253D79%26rft.issue%253D9%26rft.spage%253D5288%26rft.epage%253D5295%26rft.atitle%253DIdentification%2Bof%2BNovel%2BSubgenomic%2BRNAs%2Band%2BNoncanonical%2BTranscription%2BInitiation%2BSignals%2Bof%2BSevere%2BAcute%2BRespiratory%2BSyndrome%2BCoronavirus%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.79.9.5288-5295.2005%26rft_id%253Dinfo%253Apmid%252F15827143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijc5LzkvNTI4OCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAzLzIwMjAuMDMuMTMuOTkwNTk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.9" data-doi="10.1128/JVI.02501-05"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">van der Meer</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zevenhoven-Dobbe</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Onderwater</span>  <span class="cit-name-given-names">JJM</span></span>, <span class="cit-auth"><span class="cit-name-surname">van der Meulen</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koerten</span>  <span class="cit-name-given-names">HK</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Ultrastructure and Origin of Membrane Vesicles Associated with the Severe Acute Respiratory Syndrome Coronavirus Replication Complex</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2006</span>;<span class="cit-vol">80</span>: <span class="cit-fpage">5927</span>–<span class="cit-lpage">5940</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.02501-05</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DSnijder%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D80%26rft.issue%253D12%26rft.spage%253D5927%26rft.epage%253D5940%26rft.atitle%253DUltrastructure%2Band%2BOrigin%2Bof%2BMembrane%2BVesicles%2BAssociated%2Bwith%2Bthe%2BSevere%2BAcute%2BRespiratory%2BSyndrome%2BCoronavirus%2BReplication%2BComplex%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02501-05%26rft_id%253Dinfo%253Apmid%252F16731931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4MC8xMi81OTI3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDMvMjAyMC4wMy4xMy45OTA1OTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.10" data-doi="10.1128/JVI.01358-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sawicki</span>  <span class="cit-name-given-names">SG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sawicki</span>  <span class="cit-name-given-names">DL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Siddell</span>  <span class="cit-name-given-names">SG.</span></span> <span class="cit-article-title">A Contemporary View of Coronavirus Transcription</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2007</span>;<span class="cit-vol">81</span>: <span class="cit-fpage">20</span>–<span class="cit-lpage">29</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.01358-06</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01358-06%26rft_id%253Dinfo%253Apmid%252F16928755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjc6IjgxLzEvMjAiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wMy8yMDIwLjAzLjEzLjk5MDU5OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.11" data-doi="10.1021/cr400525m"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Van Der Lee</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buljan</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lang</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weatheritt</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Daughdrill</span>  <span class="cit-name-given-names">GW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dunker</span>  <span class="cit-name-given-names">AK</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Classification of intrinsically disordered regions and proteins</span>. <abbr class="cit-jnl-abbrev">Chemical Reviews. American Chemical Society</abbr>; <span class="cit-pub-date">2014</span>. pp. <span class="cit-fpage">6589</span>–<span class="cit-lpage">6631</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1021/cr400525m</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChemical%2BReviews.%2BAmerican%2BChemical%2BSociety%26rft.volume%253D114%26rft.spage%253D6589%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fcr400525m%26rft_id%253Dinfo%253Apmid%252F24773235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/cr400525m&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24773235&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000338980600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">12.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.13.990598v2.12" data-doi="10.1146/annurev-biochem-072711-164947"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Oldfield</span>  <span class="cit-name-given-names">CJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dunker</span>  <span class="cit-name-given-names">AK.</span></span> <span class="cit-article-title">Intrinsically Disordered Proteins and Intrinsically Disordered Protein Regions</span>. <abbr class="cit-jnl-abbrev">Annu Rev Biochem.</abbr> <span class="cit-pub-date">2014</span>;<span class="cit-vol">83</span>: <span class="cit-fpage">553</span>–<span class="cit-lpage">584</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1146/annurev-biochem-072711-164947</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BBiochem.%26rft.volume%253D83%26rft.spage%253D553%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-biochem-072711-164947%26rft_id%253Dinfo%253Apmid%252F24606139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-biochem-072711-164947&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24606139&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.13" data-doi="10.1006/jmbi.1999.3110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">PE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dyson</span>  <span class="cit-name-given-names">HJ.</span></span> <span class="cit-article-title">Intrinsically unstructured proteins: Re-assessing the protein structure-function paradigm</span>. <abbr class="cit-jnl-abbrev">J Mol Biol.</abbr> <span class="cit-pub-date">1999</span>;<span class="cit-vol">293</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">331</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1006/jmbi.1999.3110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DWright%26rft.auinit1%253DP.%2BE.%26rft.volume%253D293%26rft.issue%253D2%26rft.spage%253D321%26rft.epage%253D331%26rft.atitle%253DIntrinsically%2Bunstructured%2Bproteins%253A%2Bre-assessing%2Bthe%2Bprotein%2Bstructure-function%2Bparadigm.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1999.3110%26rft_id%253Dinfo%253Apmid%252F10550212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1999.3110&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10550212&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083284300016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.14" data-doi="10.1111/j.1742-4658.2005.04948.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dunker</span>  <span class="cit-name-given-names">AK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cortese</span>  <span class="cit-name-given-names">MS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Romero</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Iakoucheva</span>  <span class="cit-name-given-names">LM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Uversky</span>  <span class="cit-name-given-names">VN.</span></span> <span class="cit-article-title">Flexible nets. The roles of intrinsic disorder in protein interaction networks</span>. <abbr class="cit-jnl-abbrev">FEBS J.</abbr> <span class="cit-pub-date">2005</span>;<span class="cit-vol">272</span>: <span class="cit-fpage">5129</span>–<span class="cit-lpage">48</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1742-4658.2005.04948.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BJournal%26rft.stitle%253DFEBS%2BJ%26rft.aulast%253DDunker%26rft.auinit1%253DA.%2BK.%26rft.volume%253D272%26rft.issue%253D20%26rft.spage%253D5129%26rft.epage%253D5148%26rft.atitle%253DFlexible%2Bnets.%2BThe%2Broles%2Bof%2Bintrinsic%2Bdisorder%2Bin%2Bprotein%2Binteraction%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1742-4658.2005.04948.x%26rft_id%253Dinfo%253Apmid%252F16218947%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1742-4658.2005.04948.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16218947&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">15.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.13.990598v2.15" data-doi="10.1016/S0065-3233(02)62004-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dunker</span>  <span class="cit-name-given-names">AK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">CJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Obradovic</span>  <span class="cit-name-given-names">Z.</span></span> <span class="cit-article-title">Identification and functions of usefully disordered proteins</span>. <abbr class="cit-jnl-abbrev">Advances in Protein Chemistry</abbr>. <span class="cit-pub-date">2002</span>. pp. <span class="cit-fpage">25</span>–<span class="cit-lpage">49</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/S0065-3233(02)62004-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bprotein%2Bchemistry%26rft.stitle%253DAdv%2BProtein%2BChem%26rft.aulast%253DDunker%26rft.auinit1%253DA.%2BK.%26rft.volume%253D62%26rft.spage%253D25%26rft.epage%253D49%26rft.atitle%253DIdentification%2Band%2Bfunctions%2Bof%2Busefully%2Bdisordered%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-3233%252802%252962004-2%26rft_id%253Dinfo%253Apmid%252F12418100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0065-3233(02)62004-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12418100&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179263800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.16" data-doi="10.1016/j.sbi.2008.10.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dunker</span>  <span class="cit-name-given-names">AK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Silman</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Uversky</span>  <span class="cit-name-given-names">VN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sussman</span>  <span class="cit-name-given-names">JL.</span></span> <span class="cit-article-title">Function and structure of inherently disordered proteins</span>. <abbr class="cit-jnl-abbrev">Current Opinion in Structural Biology</abbr>. <span class="cit-pub-date">2008</span>. pp. <span class="cit-fpage">756</span>–<span class="cit-lpage">764</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.sbi.2008.10.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bstructural%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BStruct%2BBiol%26rft.aulast%253DDunker%26rft.auinit1%253DA.%2BK.%26rft.volume%253D18%26rft.issue%253D6%26rft.spage%253D756%26rft.epage%253D764%26rft.atitle%253DFunction%2Band%2Bstructure%2Bof%2Binherently%2Bdisordered%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.sbi.2008.10.002%26rft_id%253Dinfo%253Apmid%252F18952168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.sbi.2008.10.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18952168&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262064300017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.17" data-doi="10.1021/bi0602718"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perumal</span>  <span class="cit-name-given-names">NB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oldfield</span>  <span class="cit-name-given-names">CJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">EW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Uversky</span>  <span class="cit-name-given-names">VN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dunker</span>  <span class="cit-name-given-names">AK.</span></span> <span class="cit-article-title">Intrinsic disorder in transcription factors</span>. <abbr class="cit-jnl-abbrev">Biochemistry</abbr>. <span class="cit-pub-date">2006</span>;<span class="cit-vol">45</span>: <span class="cit-fpage">6873</span>–<span class="cit-lpage">88</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1021/bi0602718</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%2B%2528Easton%2529%26rft.stitle%253DBiochemistry%2B%2528Easton%2529%26rft.aulast%253DLiu%26rft.auinit1%253DJ.%26rft.volume%253D45%26rft.issue%253D22%26rft.spage%253D6873%26rft.epage%253D6888%26rft.atitle%253DIntrinsic%2Bdisorder%2Bin%2Btranscription%2Bfactors.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi0602718%26rft_id%253Dinfo%253Apmid%252F16734424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi0602718&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16734424&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237920900012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.18" data-doi="10.1002/jmr.747"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Uversky</span>  <span class="cit-name-given-names">VN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oldfield</span>  <span class="cit-name-given-names">CJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dunker</span>  <span class="cit-name-given-names">AK.</span></span> <span class="cit-article-title">Showing your ID: intrinsic disorder as an ID for recognition, regulation and cell signaling</span>. <abbr class="cit-jnl-abbrev">J Mol Recognit.</abbr> <span class="cit-pub-date">2005</span>;<span class="cit-vol">18</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">84</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1002/jmr.747</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Brecognition%2B%253A%2B%2BJMR%26rft.stitle%253DJ%2BMol%2BRecognit%26rft.aulast%253DUversky%26rft.auinit1%253DV.%2BN.%26rft.volume%253D18%26rft.issue%253D5%26rft.spage%253D343%26rft.epage%253D384%26rft.atitle%253DShowing%2Byour%2BID%253A%2Bintrinsic%2Bdisorder%2Bas%2Ban%2BID%2Bfor%2Brecognition%252C%2Bregulation%2Band%2Bcell%2Bsignaling.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjmr.747%26rft_id%253Dinfo%253Apmid%252F16094605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jmr.747&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16094605&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231887200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.19" data-doi="10.1093/nar/gkx059"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kurgan</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">DRNApred, fast sequence-based method that accurately predicts and discriminates DNA- and RNA-binding residues</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-pub-date">2017</span>;<span class="cit-vol">45</span>: <span class="cit-fpage">e84</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gkx059</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx059%26rft_id%253Dinfo%253Apmid%252F28132027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx059&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28132027&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.20" data-doi="10.1093/nar/gkv585"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">Z KL.</span></span> <span class="cit-article-title">High-throughput prediction of RNA, DNA and protein binding regions mediated by intrinsic disorder</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-pub-date">2015</span>;<span class="cit-vol">43</span>: <span class="cit-fpage">e121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv585%26rft_id%253Dinfo%253Apmid%252F26109352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv585&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26109352&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.21" data-doi="10.1038/s41586-020-2008-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y-M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Z-G</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">A new coronavirus associated with human respiratory disease in China</span>. <abbr class="cit-jnl-abbrev">Nature.</abbr> <span class="cit-pub-date">2020</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41586-020-2008-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2008-3%26rft_id%253Dinfo%253Apmid%252F32015508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-020-2008-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32015508&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.22" data-doi="10.3389/fcimb.2016.00144"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Giri</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Uversky</span>  <span class="cit-name-given-names">VN.</span></span> <span class="cit-article-title">Intrinsically Disordered Side of the Zika Virus Proteome. Frontiers in cellular and infection microbiology 2016, 6, 144. teome</span>. <abbr class="cit-jnl-abbrev">Front Cell Infect Microbiol.</abbr> <span class="cit-pub-date">2016</span>;<span class="cit-vol">6</span>: <span class="cit-fpage">144</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.3389/fcimb.2016.00144</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront%2BCell%2BInfect%2BMicrobiol.%26rft.volume%253D6%26rft.spage%253D144%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffcimb.2016.00144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fcimb.2016.00144&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.23" data-doi="10.2174/092986610791498984"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xue</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Williams R</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Oldfield C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kian-Meng Goh</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keith Dunker</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Uversky V</span></span>. <span class="cit-article-title">Viral Disorder or Disordered Viruses: Do Viral Proteins Possess Unique Features</span>? <abbr class="cit-jnl-abbrev">Protein Pept Lett.</abbr> <span class="cit-pub-date">2010</span>;<span class="cit-vol">17</span>: <span class="cit-fpage">932</span>–<span class="cit-lpage">951</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.2174/092986610791498984</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtein%2Band%2Bpeptide%2Bletters%26rft.stitle%253DProtein%2BPept%2BLett%26rft.aulast%253DXue%26rft.auinit1%253DB.%26rft.volume%253D17%26rft.issue%253D8%26rft.spage%253D932%26rft.epage%253D951%26rft.atitle%253DViral%2Bdisorder%2Bor%2Bdisordered%2Bviruses%253A%2Bdo%2Bviral%2Bproteins%2Bpossess%2Bunique%2Bfeatures%253F%26rft_id%253Dinfo%253Adoi%252F10.2174%252F092986610791498984%26rft_id%253Dinfo%253Apmid%252F20450483%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2174/092986610791498984&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20450483&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.24" data-doi="10.1038/s41598-018-23969-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yadav</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Uversky</span>  <span class="cit-name-given-names">VN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Giri</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Deciphering the dark proteome of Chikungunya virus</span>. <abbr class="cit-jnl-abbrev">Sci Rep.</abbr> <span class="cit-pub-date">2018</span>;<span class="cit-vol">8</span>: <span class="cit-fpage">5822</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41598-018-23969-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep.%26rft.volume%253D8%26rft.spage%253D5822%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-018-23969-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-018-23969-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.25" data-doi="10.1016/j.jmb.2004.02.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sodhi</span>  <span class="cit-name-given-names">JS</span></span>, <span class="cit-auth"><span class="cit-name-surname">McGuffin</span>  <span class="cit-name-given-names">LJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buxton</span>  <span class="cit-name-given-names">BF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">DT.</span></span> <span class="cit-article-title">Prediction and Functional Analysis of Native Disorder in Proteins from the Three Kingdoms of Life</span>. <abbr class="cit-jnl-abbrev">J Mol Biol.</abbr> <span class="cit-pub-date">2004</span>;<span class="cit-vol">337</span>: <span class="cit-fpage">635</span>–<span class="cit-lpage">645</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.jmb.2004.02.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DWard%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D337%26rft.issue%253D3%26rft.spage%253D635%26rft.epage%253D645%26rft.atitle%253DPrediction%2Band%2Bfunctional%2Banalysis%2Bof%2Bnative%2Bdisorder%2Bin%2Bproteins%2Bfrom%2Bthe%2Bthree%2Bkingdoms%2Bof%2Blife.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2004.02.002%26rft_id%253Dinfo%253Apmid%252F15019783%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2004.02.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15019783&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220337500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.26" data-doi="10.1093/nar/gkv1276"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Karsch-Mizrachi</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">DJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ostell</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sayers</span>  <span class="cit-name-given-names">EW.</span></span> <span class="cit-article-title">GenBank</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-pub-date">2016</span>;<span class="cit-vol">44</span>: <span class="cit-fpage">D67</span>–<span class="cit-lpage">D72</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gkv1276</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1276%26rft_id%253Dinfo%253Apmid%252F26590407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1276&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26590407&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.27" data-doi="10.1038/msb.2011.75"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sievers</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilm</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dineen</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">TJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Karplus</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega</span>. <abbr class="cit-jnl-abbrev">Mol Syst Biol.</abbr> <span class="cit-pub-date">2011</span>;<span class="cit-vol">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/msb.2011.75</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BSyst%2BBiol.%26rft.volume%253D7%26rft.spage%253D539%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2011.75%26rft_id%253Dinfo%253Apmid%252F21988835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/msb.2011.75&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21988835&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296652600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.28" data-doi="10.1093/nar/gku316"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Robert</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gouet</span>  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">Deciphering key features in protein structures with the new ENDscript server</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-pub-date">2014</span>;<span class="cit-vol">42</span>: <span class="cit-fpage">W320</span>–<span class="cit-lpage">4</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gku316</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku316%26rft_id%253Dinfo%253Apmid%252F24753421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku316&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24753421&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000339715000053&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.29" data-doi="10.1186/1471-2105-7-208"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Radivojac</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vucetic</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dunker</span>  <span class="cit-name-given-names">AK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Obradovic</span>  <span class="cit-name-given-names">Z.</span></span> <span class="cit-article-title">Length-dependent prediction of protein intrinsic disorder</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr>. <span class="cit-pub-date">2006</span>;<span class="cit-vol">7</span>: <span class="cit-fpage">208</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/1471-2105-7-208</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DPeng%26rft.auinit1%253DK.%26rft.volume%253D7%26rft.spage%253D208%26rft.epage%253D208%26rft.atitle%253DLength-dependent%2Bprediction%2Bof%2Bprotein%2Bintrinsic%2Bdisorder.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-7-208%26rft_id%253Dinfo%253Apmid%252F16618368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-7-208&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16618368&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.30" data-doi="10.1142/S0219720005000886"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vucetic</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Radivojac</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">CJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dunker</span>  <span class="cit-name-given-names">AK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Obradovic</span>  <span class="cit-name-given-names">Z.</span></span> <span class="cit-article-title">Optimizing long intrinsic disorder predictors with protein evolutionary information</span>. <abbr class="cit-jnl-abbrev">J Bioinform Comput Biol.</abbr> <span class="cit-pub-date">2005</span>;<span class="cit-vol">3</span>: <span class="cit-fpage">35</span>–<span class="cit-lpage">60</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1142/S0219720005000886</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bbioinformatics%2Band%2Bcomputational%2Bbiology%26rft.stitle%253DJ%2BBioinform%2BComput%2BBiol%26rft.aulast%253DPeng%26rft.auinit1%253DK.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D35%26rft.epage%253D60%26rft.atitle%253DOptimizing%2Blong%2Bintrinsic%2Bdisorder%2Bpredictors%2Bwith%2Bprotein%2Bevolutionary%2Binformation.%26rft_id%253Dinfo%253Adoi%252F10.1142%252FS0219720005000886%26rft_id%253Dinfo%253Apmid%252F15751111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1142/S0219720005000886&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15751111&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.31" data-doi="10.1016/j.bbapap.2010.01.011"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xue</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dunbrack</span>  <span class="cit-name-given-names">RL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">RW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dunker</span>  <span class="cit-name-given-names">AK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Uversky</span>  <span class="cit-name-given-names">VN.</span></span> <span class="cit-article-title">PONDR-FIT: a meta-predictor of intrinsically disordered amino acids</span>. <abbr class="cit-jnl-abbrev">Biochim Biophys Acta.</abbr> <span class="cit-pub-date">2010</span>;<span class="cit-vol">1804</span>: <span class="cit-fpage">996</span>–<span class="cit-lpage">1010</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.bbapap.2010.01.011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253DXue%26rft.auinit1%253DB.%26rft.volume%253D1804%26rft.issue%253D4%26rft.spage%253D996%26rft.epage%253D1010%26rft.atitle%253DPONDR-FIT%253A%2Ba%2Bmeta-predictor%2Bof%2Bintrinsically%2Bdisordered%2Bamino%2Bacids.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbapap.2010.01.011%26rft_id%253Dinfo%253Apmid%252F20100603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbapap.2010.01.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20100603&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275516500040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.32" data-doi="10.1002/1097-0134(20010101)42:1&lt;38::AID-PROT50&gt;3.0.CO;2-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Romero</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Obradovic</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garner</span>  <span class="cit-name-given-names">EC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">CJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dunker</span>  <span class="cit-name-given-names">AK.</span></span> <span class="cit-article-title">Sequence complexity of disordered protein</span>. <abbr class="cit-jnl-abbrev">Proteins Struct Funct Genet.</abbr> <span class="cit-pub-date">2001</span>;<span class="cit-vol">42</span>: <span class="cit-fpage">38</span>–<span class="cit-lpage">48</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1002/1097-0134(20010101)42:1&lt;38::AID-PROT50&gt;3.0.CO;2-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteins%26rft.stitle%253DProteins%26rft.aulast%253DRomero%26rft.auinit1%253DP.%26rft.volume%253D42%26rft.issue%253D1%26rft.spage%253D38%26rft.epage%253D48%26rft.atitle%253DSequence%2Bcomplexity%2Bof%2Bdisordered%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F1097-0134%252820010101%252942%253A1%253C38%253A%253AAID-PROT50%253E3.0.CO%253B2-3%26rft_id%253Dinfo%253Apmid%252F11093259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/1097-0134(20010101)42:1&lt;38::AID-PROT50&gt;3.0.CO;2-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11093259&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166077200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.33" data-doi="10.1093/nar/gky384"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mészáros</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Erdos</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dosztányi</span>  <span class="cit-name-given-names">Z.</span></span> <span class="cit-article-title">IUPred2A: context-dependent prediction of protein disorder as a function of redox state and protein binding</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-pub-date">2018</span>;<span class="cit-vol">46</span>: <span class="cit-fpage">W329</span>–<span class="cit-lpage">W337</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gky384</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D46%26rft.spage%253DW329%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky384%26rft_id%253Dinfo%253Apmid%252F29860432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gky384&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29860432&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.34" data-doi="10.1007/s00018-019-03414-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gadhave</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gehi</span>  <span class="cit-name-given-names">BR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xue</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Uversky</span>  <span class="cit-name-given-names">VN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Giri</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">The dark side of Alzheimer’s disease: unstructured biology of proteins from the amyloid cascade signaling pathway</span>. <abbr class="cit-jnl-abbrev">Cell Mol Life Sci.</abbr> <span class="cit-pub-date">2020</span> [cited 29 Feb 2020].<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/s00018-019-03414-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BMol%2BLife%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00018-019-03414-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00018-019-03414-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.35" data-doi="10.1016/bs.pmbts.2019.05.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Garg</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gadhave</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Giri</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">The dark proteome of cancer: Intrinsic disorderedness and functionality of HIF-1α along with its interacting proteins</span>. <abbr class="cit-jnl-abbrev">Prog Mol Biol Transl Sci.</abbr> <span class="cit-pub-date">2019</span>;<span class="cit-vol">166</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">403</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/bs.pmbts.2019.05.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProg%2BMol%2BBiol%2BTransl%2BSci.%26rft.volume%253D166%26rft.spage%253D371%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fbs.pmbts.2019.05.006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/bs.pmbts.2019.05.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.36" data-doi="10.1002/1097-0134(20001115)41:3&lt;415::AID-PROT130&gt;3.0.CO;2-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Uversky</span>  <span class="cit-name-given-names">VN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gillespie</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">AL.</span></span> <span class="cit-article-title">Why are “natively unfolded” proteins unstructured under physiologic conditions</span>? <abbr class="cit-jnl-abbrev">Proteins Struct Funct Genet.</abbr> <span class="cit-pub-date">2000</span>;<span class="cit-vol">41</span>: <span class="cit-fpage">415</span>–<span class="cit-lpage">427</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1002/1097-0134(20001115)41:3&lt;415::AID-PROT130&gt;3.0.CO;2-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteins%26rft.stitle%253DProteins%26rft.aulast%253DUversky%26rft.auinit1%253DV.%2BN.%26rft.volume%253D41%26rft.issue%253D3%26rft.spage%253D415%26rft.epage%253D427%26rft.atitle%253DWhy%2Bare%2B%2526quot%253Bnatively%2Bunfolded%2526quot%253B%2Bproteins%2Bunstructured%2Bunder%2Bphysiologic%2Bconditions%253F%26rft_id%253Dinfo%253Adoi%252F10.1002%252F1097-0134%252820001115%252941%253A3%253C415%253A%253AAID-PROT130%253E3.0.CO%253B2-7%26rft_id%253Dinfo%253Apmid%252F11025552%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/1097-0134(20001115)41:3&lt;415::AID-PROT130&gt;3.0.CO;2-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11025552&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000090071100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.37" data-doi="10.1142/9789814366496_0013"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oldfield</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meng</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hsu</span>  <span class="cit-name-given-names">WL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xue</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Uversky</span>  <span class="cit-name-given-names">VN</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Subclassifying disordered proteins by the CH-CDF plot method</span>. <abbr class="cit-jnl-abbrev">Pacific Symposium on Biocomputing</abbr>. <span class="cit-pub-date">2012</span>. pp. <span class="cit-fpage">128</span>–<span class="cit-lpage">139</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1142/9789814366496_0013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPacific%2BSymposium%2Bon%2BBiocomputing%26rft_id%253Dinfo%253Adoi%252F10.1142%252F9789814366496_0013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1142/9789814366496_0013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.38" data-doi="10.1371/journal.pone.0141603"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Malhis</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">ETC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nassar</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gsponer</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Computational Identification of MoRFs in Protein Sequences Using Hierarchical Application of Bayes Rule</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">10</span>: <span class="cit-fpage">e0141603</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0141603</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0141603%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0141603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0141603&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.39" data-doi="10.1371/journal.pcbi.1000376"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mészáros</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dosztányi</span>  <span class="cit-name-given-names">Z.</span></span> <span class="cit-article-title">Prediction of protein binding regions in disordered proteins</span>. <abbr class="cit-jnl-abbrev">PLoS Comput Biol.</abbr> <span class="cit-pub-date">2009</span>;<span class="cit-vol">5</span>: <span class="cit-fpage">e1000376</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pcbi.1000376</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DMeszaros%26rft.auinit1%253DB.%26rft.volume%253D5%26rft.issue%253D5%26rft.spage%253De1000376%26rft.epage%253De1000376%26rft.atitle%253DPrediction%2Bof%2Bprotein%2Bbinding%2Bregions%2Bin%2Bdisordered%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1000376%26rft_id%253Dinfo%253Apmid%252F19412530%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1000376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19412530&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.40" data-doi="10.1093/bioinformatics/btp518"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dosztányi</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mészáros</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">I.</span></span> <span class="cit-article-title">ANCHOR: web server for predicting protein binding regions in disordered proteins</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2009</span>;<span class="cit-vol">25</span>: <span class="cit-fpage">2745</span>–<span class="cit-lpage">6</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btp518</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp518%26rft_id%253Dinfo%253Apmid%252F19717576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp518&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19717576&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270685200022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.41" data-doi="10.1093/bioinformatics/bts209"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Disfani</span>  <span class="cit-name-given-names">FM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hsu</span>  <span class="cit-name-given-names">W-L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mizianty</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oldfield</span>  <span class="cit-name-given-names">CJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xue</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dunker</span>  <span class="cit-name-given-names">AK</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">MoRFpred, a computational tool for sequence-based prediction and characterization of short disorder-to-order transitioning binding regions in proteins</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">28</span>: <span class="cit-fpage">i75</span>–<span class="cit-lpage">83</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/bts209</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts209%26rft_id%253Dinfo%253Apmid%252F22689782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts209&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22689782&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305419800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.42" data-doi="10.1093/bioinformatics/btu744"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">DT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cozzetto</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">DISOPRED3: Precise disordered region predictions with annotated protein-binding activity</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">31</span>: <span class="cit-fpage">857</span>–<span class="cit-lpage">863</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btu744</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu744%26rft_id%253Dinfo%253Apmid%252F25391399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu744&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25391399&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.43" data-doi="10.1007/978-1-4939-6406-2_14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Uversky</span>  <span class="cit-name-given-names">VN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kurgan</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">Prediction of Disordered RNA, DNA, and Protein Binding Regions Using DisoRDPbind</span>. <abbr class="cit-jnl-abbrev">Methods Mol Biol.</abbr> <span class="cit-pub-date">2017</span>;<span class="cit-vol">1484</span>: <span class="cit-fpage">187</span>–<span class="cit-lpage">203</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/978-1-4939-6406-2_14</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol%2BBiol.%26rft.volume%253D1484%26rft.spage%253D187%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4939-6406-2_14%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4939-6406-2_14&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.44" data-doi="10.1002/prot.21677"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gromiha</span>  <span class="cit-name-given-names">MM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Raghava</span>  <span class="cit-name-given-names">GPS.</span></span> <span class="cit-article-title">Prediction of RNA binding sites in a protein using SVM and PSSM profile</span>. <abbr class="cit-jnl-abbrev">Proteins</abbr>. <span class="cit-pub-date">2008</span>;<span class="cit-vol">71</span>: <span class="cit-fpage">189</span>–<span class="cit-lpage">94</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1002/prot.21677</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteins%26rft.stitle%253DProteins%26rft.aulast%253DKumar%26rft.auinit1%253DM.%26rft.volume%253D71%26rft.issue%253D1%26rft.spage%253D189%26rft.epage%253D194%26rft.atitle%253DPrediction%2Bof%2BRNA%2Bbinding%2Bsites%2Bin%2Ba%2Bprotein%2Busing%2BSVM%2Band%2BPSSM%2Bprofile.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fprot.21677%26rft_id%253Dinfo%253Apmid%252F17932917%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/prot.21677&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17932917&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254035500018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.45" data-doi="10.1016/j.chom.2020.02.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Genome Composition and Divergence of the Novel Coronavirus (2019-nCoV) Originating in China</span>. <abbr class="cit-jnl-abbrev">Cell Host Microbe.</abbr> <span class="cit-pub-date">2020</span> [cited 7 Mar 2020].<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.chom.2020.02.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2BMicrobe.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2020.02.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2020.02.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.46" data-doi="10.1002/jcb.23252"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rajagopalan</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mooney</span>  <span class="cit-name-given-names">SM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Parekh</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Getzenberg</span>  <span class="cit-name-given-names">RH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kulkarni</span>  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">A majority of the cancer/testis antigens are intrinsically disordered proteins</span>. <abbr class="cit-jnl-abbrev">J Cell Biochem.</abbr> <span class="cit-pub-date">2011</span>;<span class="cit-vol">112</span>: <span class="cit-fpage">3256</span>–<span class="cit-lpage">67</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1002/jcb.23252</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcellular%2Bbiochemistry%26rft.stitle%253DJ%2BCell%2BBiochem%26rft.aulast%253DRajagopalan%26rft.auinit1%253DK.%26rft.volume%253D112%26rft.issue%253D11%26rft.spage%253D3256%26rft.epage%253D3267%26rft.atitle%253DA%2Bmajority%2Bof%2Bthe%2Bcancer%252Ftestis%2Bantigens%2Bare%2Bintrinsically%2Bdisordered%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcb.23252%26rft_id%253Dinfo%253Apmid%252F21748782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jcb.23252&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21748782&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.47" data-doi="10.1016/j.jmb.2017.10.018"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mishra</span>  <span class="cit-name-given-names">PM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Uversky</span>  <span class="cit-name-given-names">VN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Giri</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Molecular Recognition Features in Zika Virus Proteome</span>. <abbr class="cit-jnl-abbrev">J Mol Biol.</abbr> <span class="cit-pub-date">2018</span>;<span class="cit-vol">430</span>: <span class="cit-fpage">2372</span>–<span class="cit-lpage">2388</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.jmb.2017.10.018</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BMol%2BBiol.%26rft.volume%253D430%26rft.spage%253D2372%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2017.10.018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2017.10.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.48" data-doi="10.1093/bioinformatics/btt427"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gypas</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tsaousis</span>  <span class="cit-name-given-names">GN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hamodrakas</span>  <span class="cit-name-given-names">SJ.</span></span> <span class="cit-article-title">mpMoRFsDB: a database of molecular recognition features in membrane proteins</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2013</span>;<span class="cit-vol">29</span>: <span class="cit-fpage">2517</span>–<span class="cit-lpage">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btt427</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt427%26rft_id%253Dinfo%253Apmid%252F23894139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt427&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23894139&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.49" data-doi="10.1007/978-1-0716-0231-7_14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Oldfield</span>  <span class="cit-name-given-names">CJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kurgan</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">Disordered RNA-Binding Region Prediction with DisoRDPbind</span>. <abbr class="cit-jnl-abbrev">Methods Mol Biol.</abbr> <span class="cit-pub-date">2020</span>;<span class="cit-vol">2106</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">239</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/978-1-0716-0231-7_14</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol%2BBiol.%26rft.volume%253D2106%26rft.spage%253D225%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-0716-0231-7_14%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-0716-0231-7_14&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.50" data-doi="10.1099/0022-1317-67-7-1443"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cavanagh</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">PJ.</span></span> <span class="cit-article-title">Coronavirus IBV: removal of spike glycopolypeptide S1 by urea abolishes infectivity and haemagglutination but not attachment to cells</span>. <abbr class="cit-jnl-abbrev">J Gen Virol.</abbr> <span class="cit-pub-date">1986</span>;<span class="cit-vol">67</span> (<span class="cit-issue">Pt 7</span>): <span class="cit-fpage">1443</span>–<span class="cit-lpage">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1099/0022-1317-67-7-1443</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DCavanagh%26rft.auinit1%253DD.%26rft.volume%253D67%26rft.issue%253D7%26rft.spage%253D1443%26rft.epage%253D1448%26rft.atitle%253DCoronavirus%2BIBV%253A%2BRemoval%2Bof%2BSpike%2BGlycopolypeptide%2BS1%2Bby%2BUrea%2BAbolishes%2BInfectivity%2Band%2BHaemagglutination%2Bbut%2BNot%2BAttachment%2Bto%2BCells%26rft_id%253Dinfo%253Adoi%252F10.1099%252F0022-1317-67-7-1443%26rft_id%253Dinfo%253Apmid%252F3014054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/0022-1317-67-7-1443&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3014054&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986D294800021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.51" data-doi="10.1128/JVI.01394-09"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">RL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>  <span class="cit-name-given-names">RS.</span></span> <span class="cit-article-title">Recombination, Reservoirs, and the Modular Spike: Mechanisms of Coronavirus Cross-Species Transmission</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2010</span>;<span class="cit-vol">84</span>: <span class="cit-fpage">3134</span>–<span class="cit-lpage">3146</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.01394-09</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01394-09%26rft_id%253Dinfo%253Apmid%252F19906932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg0LzcvMzEzNCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAzLzIwMjAuMDMuMTMuOTkwNTk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.52" data-doi="10.3390/v4061011"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Belouzard</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Millet</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Licitra</span>  <span class="cit-name-given-names">BN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whittaker</span>  <span class="cit-name-given-names">GR.</span></span> <span class="cit-article-title">Mechanisms of coronavirus cell entry mediated by the viral spike protein</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr>. <span class="cit-pub-date">2012</span>. pp. <span class="cit-fpage">1011</span>–<span class="cit-lpage">1033</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.3390/v4061011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DViruses%26rft.aulast%253DBelouzard%26rft.auinit1%253DS.%26rft.volume%253D4%26rft.issue%253D6%26rft.spage%253D1011%26rft.epage%253D1033%26rft.atitle%253DMechanisms%2Bof%2Bcoronavirus%2Bcell%2Bentry%2Bmediated%2Bby%2Bthe%2Bviral%2Bspike%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv4061011%26rft_id%253Dinfo%253Apmid%252F22816037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v4061011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22816037&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305801900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.53" data-doi="10.1128/JVI.00950-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">de Haan</span>  <span class="cit-name-given-names">CAM</span></span>, <span class="cit-auth"><span class="cit-name-surname">te Lintelo</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Raaben</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wurdinger</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bosch</span>  <span class="cit-name-given-names">BJ</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Cooperative Involvement of the S1 and S2 Subunits of the Murine Coronavirus Spike Protein in Receptor Binding and Extended Host Range</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2006</span>;<span class="cit-vol">80</span>: <span class="cit-fpage">10909</span>–<span class="cit-lpage">10918</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.00950-06</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00950-06%26rft_id%253Dinfo%253Apmid%252F16956938%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4MC8yMi8xMDkwOSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAzLzIwMjAuMDMuMTMuOTkwNTk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.54" data-doi="10.1128/JVI.02146-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McBride</span>  <span class="cit-name-given-names">CE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Machamer</span>  <span class="cit-name-given-names">CE.</span></span> <span class="cit-article-title">The Cytoplasmic Tail of the Severe Acute Respiratory Syndrome Coronavirus Spike Protein Contains a Novel Endoplasmic Reticulum Retrieval Signal That Binds COPI and Promotes Interaction with Membrane Protein</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2007</span>;<span class="cit-vol">81</span>: <span class="cit-fpage">2418</span>–<span class="cit-lpage">2428</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.02146-06</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02146-06%26rft_id%253Dinfo%253Apmid%252F17166901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgxLzUvMjQxOCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAzLzIwMjAuMDMuMTMuOTkwNTk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.55" data-doi="10.1126/science.1116480"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Farzan</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harrison</span>  <span class="cit-name-given-names">SC.</span></span> <span class="cit-article-title">Structure of SARS coronavirus spike receptor-binding domain complexed with receptor</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-pub-date">2005</span>;<span class="cit-vol">309</span>: <span class="cit-fpage">1864</span>–<span class="cit-lpage">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1116480</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLi%26rft.auinit1%253DF.%26rft.volume%253D309%26rft.issue%253D5742%26rft.spage%253D1864%26rft.epage%253D1868%26rft.atitle%253DStructure%2Bof%2BSARS%2BCoronavirus%2BSpike%2BReceptor-Binding%2BDomain%2BComplexed%2Bwith%2BReceptor%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1116480%26rft_id%253Dinfo%253Apmid%252F16166518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDkvNTc0Mi8xODY0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDMvMjAyMC4wMy4xMy45OTA1OTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.56" data-doi="10.1128/JVI.80.3.1302-1310.2006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Broer</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boson</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spaan</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cosset</span>  <span class="cit-name-given-names">F-L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Corver</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Important Role for the Transmembrane Domain of Severe Acute Respiratory Syndrome Coronavirus Spike Protein during Entry</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2006</span>;<span class="cit-vol">80</span>: <span class="cit-fpage">1302</span>–<span class="cit-lpage">1310</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.80.3.1302-1310.2006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DBroer%26rft.auinit1%253DR.%26rft.volume%253D80%26rft.issue%253D3%26rft.spage%253D1302%26rft.epage%253D1310%26rft.atitle%253DImportant%2BRole%2Bfor%2Bthe%2BTransmembrane%2BDomain%2Bof%2BSevere%2BAcute%2BRespiratory%2BSyndrome%2BCoronavirus%2BSpike%2BProtein%2Bduring%2BEntry%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.80.3.1302-1310.2006%26rft_id%253Dinfo%253Apmid%252F16415007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgwLzMvMTMwMiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAzLzIwMjAuMDMuMTMuOTkwNTk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.57" data-doi="10.1371/journal.ppat.1007236"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gui</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiang</span>  <span class="cit-name-given-names">Y.</span></span> <span class="cit-article-title">Cryo-EM structure of the SARS coronavirus spike glycoprotein in complex with its host cell receptor ACE2</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-pub-date">2018</span>;<span class="cit-vol">14</span>: <span class="cit-fpage">e1007236</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.ppat.1007236</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D14%26rft.spage%253De1007236%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1007236%26rft_id%253Dinfo%253Apmid%252F30102747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1007236&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30102747&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.58" data-doi="10.1126/science.abb2507"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wrapp</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Corbett</span>  <span class="cit-name-given-names">KS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goldsmith</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hsieh</span>  <span class="cit-name-given-names">C-L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abiona</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-pub-date">2020</span> [cited 29 Feb 2020].<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.abb2507</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.abb2507%26rft_id%253Dinfo%253Apmid%252F32075877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNjcvNjQ4My8xMjYwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDMvMjAyMC4wMy4xMy45OTA1OTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.59" data-doi="10.3390/v4030363"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ruch</span>  <span class="cit-name-given-names">TR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Machamer</span>  <span class="cit-name-given-names">CE.</span></span> <span class="cit-article-title">The coronavirus E protein: assembly and beyond</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">4</span>: <span class="cit-fpage">363</span>–<span class="cit-lpage">82</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.3390/v4030363</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DViruses%26rft.aulast%253DRuch%26rft.auinit1%253DT.%2BR.%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253D363%26rft.epage%253D382%26rft.atitle%253DThe%2Bcoronavirus%2BE%2Bprotein%253A%2Bassembly%2Band%2Bbeyond.%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv4030363%26rft_id%253Dinfo%253Apmid%252F22590676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v4030363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22590676&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">60.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.13.990598v2.60" data-doi="10.3390/v7041700"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ujike</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taguchi</span>  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">Incorporation of spike and membrane glycoproteins into coronavirus virions</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr>. <span class="cit-auth cit-collab">MDPI AG</span>; <span class="cit-pub-date">2015</span>. pp. <span class="cit-fpage">1700</span>–<span class="cit-lpage">1725</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.3390/v7041700</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D7%26rft.spage%253D1700%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv7041700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v7041700&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.61" data-doi="10.1128/JVI.01467-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">DeDiego</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alvarez</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Almazan</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rejas</span>  <span class="cit-name-given-names">MT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lamirande</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">A Severe Acute Respiratory Syndrome Coronavirus That Lacks the E Gene Is Attenuated In Vitro and In Vivo</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2007</span>;<span class="cit-vol">81</span>: <span class="cit-fpage">1701</span>–<span class="cit-lpage">1713</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.01467-06</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01467-06%26rft_id%253Dinfo%253Apmid%252F17108030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgxLzQvMTcwMSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAzLzIwMjAuMDMuMTMuOTkwNTk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.62" data-doi="10.1529/biophysj.104.051730"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Torres</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Parthasarathy</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">DX.</span></span> <span class="cit-article-title">The transmembrane oligomers of coronavirus protein E</span>. <abbr class="cit-jnl-abbrev">Biophys J.</abbr> <span class="cit-pub-date">2005</span>;<span class="cit-vol">88</span>: <span class="cit-fpage">1283</span>–<span class="cit-lpage">90</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1529/biophysj.104.051730</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiophysical%2BJournal%26rft.stitle%253DBiophys.%2BJ%26rft.aulast%253DTorres%26rft.auinit1%253DJ.%26rft.volume%253D88%26rft.issue%253D2%26rft.spage%253D1283%26rft.epage%253D1290%26rft.atitle%253DThe%2BTransmembrane%2BOligomers%2Bof%2BCoronavirus%2BProtein%2BE%26rft_id%253Dinfo%253Adoi%252F10.1529%252Fbiophysj.104.051730%26rft_id%253Dinfo%253Apmid%252F15713601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1529/biophysj.104.051730&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15713601&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.63" data-doi="10.1074/jbc.M114.560094"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Surya</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Claudine</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Torres</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Structure of a conserved Golgi complex-targeting signal in coronavirus envelope proteins</span>. <abbr class="cit-jnl-abbrev">J Biol Chem.</abbr> <span class="cit-pub-date">2014</span>;<span class="cit-vol">289</span>: <span class="cit-fpage">12535</span>–<span class="cit-lpage">49</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1074/jbc.M114.560094</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M114.560094%26rft_id%253Dinfo%253Apmid%252F24668816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODkvMTgvMTI1MzUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wMy8yMDIwLjAzLjEzLjk5MDU5OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.64"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wahyu Surya</span>  <span class="cit-name-given-names">MS</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">JT.</span> <span class="cit-name-surname">Structural</span></span> and <abbr class="cit-jnl-abbrev">Functional Aspects of Viroporins in Human Respiratory Viruses: Respiratory Syncytial Virus and Coronaviruses</abbr>. <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">65.</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.65" data-doi="10.1091/mbc.E10-04-0338"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Teoh</span>  <span class="cit-name-given-names">KT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Siu</span>  <span class="cit-name-given-names">YL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">WL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schlüter</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">CJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peiris</span>  <span class="cit-name-given-names">JSM</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">The SARS coronavirus E protein interacts with PALS1 and alters tight junction formation and epithelial morphogenesis</span>. <abbr class="cit-jnl-abbrev">Mol Biol Cell.</abbr> <span class="cit-pub-date">2010</span>;<span class="cit-vol">21</span>: <span class="cit-fpage">3838</span>–<span class="cit-lpage">3852</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1091/mbc.E10-04-0338</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BCell.%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E10-04-0338%26rft_id%253Dinfo%253Apmid%252F20861307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyMS8yMi8zODM4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDMvMjAyMC4wMy4xMy45OTA1OTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">66.</span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66. in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.66" data-doi="10.1371/journal.pone.0064013"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tseng</span>  <span class="cit-name-given-names">Y-T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">C-H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S-M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">K-J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C-T.</span></span> <span class="cit-article-title">Identifying SARS-CoV membrane protein amino acid residues linked to virus-like particle assembly</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>. <span class="cit-pub-date">2013</span>;<span class="cit-vol">8</span>: <span class="cit-fpage">e64013</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0064013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De64013%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0064013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0064013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">67.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.13.990598v2.67" data-doi="10.1074/jbc.M109.030270"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tseng</span>  <span class="cit-name-given-names">Y-T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S-M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">K-J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">AI-R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chiang</span>  <span class="cit-name-given-names">C-C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C-T.</span></span> <span class="cit-article-title">Self-assembly of severe acute respiratory syndrome coronavirus membrane protein</span>. <abbr class="cit-jnl-abbrev">J Biol Chem.</abbr> <span class="cit-pub-date">2010</span>;<span class="cit-vol">285</span>: <span class="cit-fpage">12862</span>–<span class="cit-lpage">72</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1074/jbc.M109.030270</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M109.030270%26rft_id%253Dinfo%253Apmid%252F20154085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODUvMTcvMTI4NjIiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wMy8yMDIwLjAzLjEzLjk5MDU5OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">68.</span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference 68. in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.68" data-doi="10.1016/S0042-6822(03)00175-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Corse</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Machamer</span>  <span class="cit-name-given-names">CE.</span></span> <span class="cit-article-title">The cytoplasmic tails of infectious bronchitis virus E and M proteins mediate their interaction</span>. <abbr class="cit-jnl-abbrev">Virology</abbr>. <span class="cit-pub-date">2003</span>;<span class="cit-vol">312</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">34</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/S0042-6822(03)00175-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DCorse%26rft.auinit1%253DE.%26rft.volume%253D312%26rft.issue%253D1%26rft.spage%253D25%26rft.epage%253D34%26rft.atitle%253DThe%2Bcytoplasmic%2Btails%2Bof%2Binfectious%2Bbronchitis%2Bvirus%2BE%2Band%2BM%2Bproteins%2Bmediate%2Btheir%2Binteraction.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0042-6822%252803%252900175-2%26rft_id%253Dinfo%253Apmid%252F12890618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0042-6822(03)00175-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12890618&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">69.</span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference 69. in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.69" data-doi="10.1128/JVI.77.5.2922-2927.2003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Narayanan</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C-J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maeda</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Makino</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Nucleocapsid-independent specific viral RNA packaging via viral envelope protein and viral RNA signal</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2003</span>;<span class="cit-vol">77</span>: <span class="cit-fpage">2922</span>–<span class="cit-lpage">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.77.5.2922-2927.2003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DNarayanan%26rft.auinit1%253DK.%26rft.volume%253D77%26rft.issue%253D5%26rft.spage%253D2922%26rft.epage%253D2927%26rft.atitle%253DNucleocapsid-Independent%2BSpecific%2BViral%2BRNA%2BPackaging%2Bvia%2BViral%2BEnvelope%2BProtein%2Band%2BViral%2BRNA%2BSignal%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.77.5.2922-2927.2003%26rft_id%253Dinfo%253Apmid%252F12584316%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijc3LzUvMjkyMiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAzLzIwMjAuMDMuMTMuOTkwNTk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">70.</span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference 70. in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.70" data-doi="10.1099/vir.0.80671-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nal</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kien</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Siu</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tse</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Differential maturation and subcellular localization of severe acute respiratory syndrome coronavirus surface proteins S, M and E</span>. <abbr class="cit-jnl-abbrev">J Gen Virol.</abbr> <span class="cit-pub-date">2005</span>;<span class="cit-vol">86</span>: <span class="cit-fpage">1423</span>–<span class="cit-lpage">34</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1099/vir.0.80671-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DNal%26rft.auinit1%253DB.%26rft.volume%253D86%26rft.issue%253D5%26rft.spage%253D1423%26rft.epage%253D1434%26rft.atitle%253DDifferential%2Bmaturation%2Band%2Bsubcellular%2Blocalization%2Bof%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome%2Bcoronavirus%2Bsurface%2Bproteins%2BS%252C%2BM%2Band%2BE%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.80671-0%26rft_id%253Dinfo%253Apmid%252F15831954%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.80671-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15831954&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">71.</span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference 71. in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.71" data-doi="10.1186/1743-422X-6-79"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Voss</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pfefferle</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drosten</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stevermann</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Traggiai</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lanzavecchia</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Studies on membrane topology, N-glycosylation and functionality of SARS-CoV membrane protein</span>. <abbr class="cit-jnl-abbrev">Virol J.</abbr> <span class="cit-pub-date">2009</span>;<span class="cit-vol">6</span>: <span class="cit-fpage">79</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/1743-422X-6-79</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DVirol%2BJ%26rft.aulast%253DVosz%26rft.auinit1%253DD.%26rft.volume%253D6%26rft.spage%253D79%26rft.epage%253D79%26rft.atitle%253DStudies%2Bon%2Bmembrane%2Btopology%252C%2BN-glycosylation%2Band%2Bfunctionality%2Bof%2BSARS-CoV%2Bmembrane%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1743-422X-6-79%26rft_id%253Dinfo%253Apmid%252F19534833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1743-422X-6-79&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19534833&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">72.</span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference 72. in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.72" data-doi="10.1016/j.jsb.2010.11.021"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Neuman</span>  <span class="cit-name-given-names">BW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kiss</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kunding</span>  <span class="cit-name-given-names">AH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bhella</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baksh</span>  <span class="cit-name-given-names">MF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Connelly</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">A structural analysis of M protein in coronavirus assembly and morphology</span>. <abbr class="cit-jnl-abbrev">J Struct Biol.</abbr> <span class="cit-pub-date">2011</span>;<span class="cit-vol">174</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">22</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.jsb.2010.11.021</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bstructural%2Bbiology%26rft.stitle%253DJ%2BStruct%2BBiol%26rft.aulast%253DNeuman%26rft.auinit1%253DB.%2BW.%26rft.volume%253D174%26rft.issue%253D1%26rft.spage%253D11%26rft.epage%253D22%26rft.atitle%253DA%2Bstructural%2Banalysis%2Bof%2BM%2Bprotein%2Bin%2Bcoronavirus%2Bassembly%2Band%2Bmorphology.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jsb.2010.11.021%26rft_id%253Dinfo%253Apmid%252F21130884%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jsb.2010.11.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21130884&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">73.</span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference 73. in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.73" data-doi="10.1086/656315"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">The membrane protein of severe acute respiratory syndrome coronavirus acts as a dominant immunogen revealed by a clustering region of novel functionally and structurally defined cytotoxic T-lymphocyte epitopes</span>. <abbr class="cit-jnl-abbrev">J Infect Dis.</abbr> <span class="cit-pub-date">2010</span>;<span class="cit-vol">202</span>: <span class="cit-fpage">1171</span>–<span class="cit-lpage">80</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1086/656315</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BInfect%2BDis.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F656315%26rft_id%253Dinfo%253Apmid%252F20831383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/656315&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20831383&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">74.</span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference 74. in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.74" data-doi="10.1371/currents.outbreaks.22254b58675cdebc256dbe3c5aa6498b"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goh</span>  <span class="cit-name-given-names">GK-M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dunker</span>  <span class="cit-name-given-names">AK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Uversky</span>  <span class="cit-name-given-names">V.</span></span> <span class="cit-article-title">Prediction of Intrinsic Disorder in MERS-CoV/HCoV-EMC Supports a High Oral-Fecal Transmission</span>. <abbr class="cit-jnl-abbrev">PLoS Curr</abbr>. <span class="cit-pub-date">2013</span>;<span class="cit-vol">5</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/currents.outbreaks.22254b58675cdebc256dbe3c5aa6498b</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BCurr%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fcurrents.outbreaks.22254b58675cdebc256dbe3c5aa6498b%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/currents.outbreaks.22254b58675cdebc256dbe3c5aa6498b&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">75.</span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference 75. in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.75" data-doi="10.1074/jbc.RA119.008964"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Perrier</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bonnin</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Desmarets</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Danneels</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goffard</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rouillé</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">The C-terminal domain of the MERS coronavirus M protein contains a trans-Golgi network localization signal</span>. <abbr class="cit-jnl-abbrev">J Biol Chem.</abbr> <span class="cit-pub-date">2019</span>;<span class="cit-vol">294</span>: <span class="cit-fpage">14406</span>–<span class="cit-lpage">14421</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1074/jbc.RA119.008964</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.RA119.008964%26rft_id%253Dinfo%253Apmid%252F31399512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyOTQvMzkvMTQ0MDYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wMy8yMDIwLjAzLjEzLjk5MDU5OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">76.</span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference 76. in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.76" data-doi="10.3390/v6082991"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McBride</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Zyl</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fielding</span>  <span class="cit-name-given-names">BC.</span></span> <span class="cit-article-title">The coronavirus nucleocapsid is a multifunctional protein</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr>. <span class="cit-auth cit-collab">MDPI AG</span>; <span class="cit-pub-date">2014</span>. pp. <span class="cit-fpage">2991</span>–<span class="cit-lpage">3018</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.3390/v6082991</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D6%26rft.spage%253D2991%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv6082991%26rft_id%253Dinfo%253Apmid%252F25105276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v6082991&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25105276&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">77.</span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference 77. in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.77" data-doi="10.1128/JVI.02236-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Saikatendu</span>  <span class="cit-name-given-names">KS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joseph</span>  <span class="cit-name-given-names">JS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Subramanian</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neuman</span>  <span class="cit-name-given-names">BW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buchmeier</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">RC</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Ribonucleocapsid formation of severe acute respiratory syndrome coronavirus through molecular action of the N-terminal domain of N protein</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2007</span>;<span class="cit-vol">81</span>: <span class="cit-fpage">3913</span>–<span class="cit-lpage">21</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.02236-06</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02236-06%26rft_id%253Dinfo%253Apmid%252F17229691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgxLzgvMzkxMyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAzLzIwMjAuMDMuMTMuOTkwNTk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">78.</span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference 78. in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.78" data-doi="10.1128/JVI.01052-08"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Siu</span>  <span class="cit-name-given-names">YL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Teoh</span>  <span class="cit-name-given-names">KT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lo</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kien</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Escriou</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">The M, E, and N Structural Proteins of the Severe Acute Respiratory Syndrome Coronavirus Are Required for Efficient Assembly, Trafficking, and Release of Virus-Like Particles</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2008</span>;<span class="cit-vol">82</span>: <span class="cit-fpage">11318</span>–<span class="cit-lpage">11330</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.01052-08</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01052-08%26rft_id%253Dinfo%253Apmid%252F18753196%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4Mi8yMi8xMTMxOCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAzLzIwMjAuMDMuMTMuOTkwNTk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">79.</span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference 79. in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.79" data-doi="10.1128/JVI.02001-08"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">C-K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hsu</span>  <span class="cit-name-given-names">Y-L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">Y-H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">F-A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">M-C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y-S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Multiple Nucleic Acid Binding Sites and Intrinsic Disorder of Severe Acute Respiratory Syndrome Coronavirus Nucleocapsid Protein: Implications for Ribonucleocapsid Protein Packaging</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2009</span>;<span class="cit-vol">83</span>: <span class="cit-fpage">2255</span>–<span class="cit-lpage">2264</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.02001-08</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02001-08%26rft_id%253Dinfo%253Apmid%252F19052082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgzLzUvMjI1NSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAzLzIwMjAuMDMuMTMuOTkwNTk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">80.</span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference 80. in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.80" data-doi="10.1021/bi036155b"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Petros</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gunasekera</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Structure of the N-terminal RNA-binding domain of the SARS CoV nucleocapsid protein</span>. <abbr class="cit-jnl-abbrev">Biochemistry</abbr>. <span class="cit-pub-date">2004</span>;<span class="cit-vol">43</span>: <span class="cit-fpage">6059</span>–<span class="cit-lpage">63</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1021/bi036155b</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%2B%2528Easton%2529%26rft.stitle%253DBiochemistry%2B%2528Easton%2529%26rft.aulast%253DHuang%26rft.auinit1%253DQ.%26rft.volume%253D43%26rft.issue%253D20%26rft.spage%253D6059%26rft.epage%253D6063%26rft.atitle%253DStructure%2Bof%2Bthe%2BN-terminal%2BRNA-binding%2Bdomain%2Bof%2Bthe%2BSARS%2BCoV%2Bnucleocapsid%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi036155b%26rft_id%253Dinfo%253Apmid%252F15147189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi036155b&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15147189&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">81.</span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference 81. in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.81" data-doi="10.1074/jbc.M602107200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">I-M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oldham</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Crystal structure of the severe acute respiratory syndrome (SARS) coronavirus nucleocapsid protein dimerization domain reveals evolutionary linkage between corona- and arteriviridae</span>. <abbr class="cit-jnl-abbrev">J Biol Chem.</abbr> <span class="cit-pub-date">2006</span>;<span class="cit-vol">281</span>: <span class="cit-fpage">17134</span>–<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1074/jbc.M602107200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M602107200%26rft_id%253Dinfo%253Apmid%252F16627473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODEvMjUvMTcxMzQiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wMy8yMDIwLjAzLjEzLjk5MDU5OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">82.</span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference 82. in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.82" data-doi="10.1016/j.virusres.2004.05.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leeson</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ballantine</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andonov</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dobie</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Characterization of protein-protein interactions between the nucleocapsid protein and membrane protein of the SARS coronavirus</span>. <abbr class="cit-jnl-abbrev">Virus Res.</abbr> <span class="cit-pub-date">2004</span>;<span class="cit-vol">105</span>: <span class="cit-fpage">121</span>–<span class="cit-lpage">5</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virusres.2004.05.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DHe%26rft.auinit1%253DR.%26rft.volume%253D105%26rft.issue%253D2%26rft.spage%253D121%26rft.epage%253D125%26rft.atitle%253DCharacterization%2Bof%2Bprotein-protein%2Binteractions%2Bbetween%2Bthe%2Bnucleocapsid%2Bprotein%2Band%2Bmembrane%2Bprotein%2Bof%2Bthe%2BSARS%2Bcoronavirus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2004.05.002%26rft_id%253Dinfo%253Apmid%252F15351485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2004.05.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15351485&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">83.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.13.990598v2.83" data-doi="10.1016/j.febslet.2005.03.080"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">H.</span></span> <span class="cit-article-title">The nucleocapsid protein of SARS coronavirus has a high binding affinity to the human cellular heterogeneous nuclear ribonucleoprotein A1</span>. <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-pub-date">2005</span>;<span class="cit-vol">579</span>: <span class="cit-fpage">2623</span>–<span class="cit-lpage">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.febslet.2005.03.080</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DLuo%26rft.auinit1%253DH.%26rft.volume%253D579%26rft.issue%253D12%26rft.spage%253D2623%26rft.epage%253D2628%26rft.atitle%253DThe%2Bnucleocapsid%2Bprotein%2Bof%2BSARS%2Bcoronavirus%2Bhas%2Ba%2Bhigh%2Bbinding%2Baffinity%2Bto%2Bthe%2Bhuman%2Bcellular%2Bheterogeneous%2Bnuclear%2Bribonucleoprotein%2BA1.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.febslet.2005.03.080%26rft_id%253Dinfo%253Apmid%252F15862300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.febslet.2005.03.080&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15862300&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229051600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">84.</span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference 84. in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.84" data-doi="10.1016/j.bbrc.2004.02.074"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dobie</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ballantine</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leeson</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bastien</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Analysis of multimerization of the SARS coronavirus nucleocapsid protein</span>. <abbr class="cit-jnl-abbrev">Biochem Biophys Res Commun.</abbr> <span class="cit-pub-date">2004</span>;<span class="cit-vol">316</span>: <span class="cit-fpage">476</span>–<span class="cit-lpage">83</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.bbrc.2004.02.074</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2Band%2Bbiophysical%2Bresearch%2Bcommunications%26rft.stitle%253DBiochem%2BBiophys%2BRes%2BCommun%26rft.aulast%253DHe%26rft.auinit1%253DR.%26rft.volume%253D316%26rft.issue%253D2%26rft.spage%253D476%26rft.epage%253D483%26rft.atitle%253DAnalysis%2Bof%2Bmultimerization%2Bof%2Bthe%2BSARS%2Bcoronavirus%2Bnucleocapsid%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbrc.2004.02.074%26rft_id%253Dinfo%253Apmid%252F15020242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbrc.2004.02.074&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15020242&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220333200027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">85.</span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference 85. in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.85" data-doi="10.1099/0022-1317-74-9-1975"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">GW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stohlman</span>  <span class="cit-name-given-names">SA.</span></span> <span class="cit-article-title">Localization of the RNA-binding domain of mouse hepatitis virus nucleocapsid protein</span>. <abbr class="cit-jnl-abbrev">J Gen Virol.</abbr> <span class="cit-pub-date">1993</span>;<span class="cit-vol">74</span> (<span class="cit-issue">Pt 9</span>): <span class="cit-fpage">1975</span>–<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1099/0022-1317-74-9-1975</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DNelson%26rft.auinit1%253DG.%2BW.%26rft.volume%253D74%26rft.issue%253D9%26rft.spage%253D1975%26rft.epage%253D1979%26rft.atitle%253DLocalization%2Bof%2Bthe%2BRNA-binding%2Bdomain%2Bof%2Bmouse%2Bhepatitis%2Bvirus%2Bnucleocapsid%2Bprotein%26rft_id%253Dinfo%253Adoi%252F10.1099%252F0022-1317-74-9-1975%26rft_id%253Dinfo%253Apmid%252F8397288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/0022-1317-74-9-1975&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8397288&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">86.</span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference 86. in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.86" data-doi="10.1016/j.jmb.2004.07.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gunasekaran</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">C-J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nussinov</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Analysis of ordered and disordered protein complexes reveals structural features discriminating between stable and unstable monomers</span>. <abbr class="cit-jnl-abbrev">J Mol Biol.</abbr> <span class="cit-pub-date">2004</span>;<span class="cit-vol">341</span>: <span class="cit-fpage">1327</span>–<span class="cit-lpage">41</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.jmb.2004.07.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DGunasekaran%26rft.auinit1%253DK.%26rft.volume%253D341%26rft.issue%253D5%26rft.spage%253D1327%26rft.epage%253D1341%26rft.atitle%253DAnalysis%2Bof%2Bordered%2Band%2Bdisordered%2Bprotein%2Bcomplexes%2Breveals%2Bstructural%2Bfeatures%2Bdiscriminating%2Bbetween%2Bstable%2Band%2Bunstable%2Bmonomers.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2004.07.002%26rft_id%253Dinfo%253Apmid%252F15321724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2004.07.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15321724&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223480000015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">87.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.13.990598v2.87" data-doi="10.1186/1471-2164-9-S2-S1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Oldfield</span>  <span class="cit-name-given-names">CJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meng</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">JY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">MQ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Uversky</span>  <span class="cit-name-given-names">VN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dunker</span>  <span class="cit-name-given-names">AK.</span></span> <span class="cit-article-title">Flexible nets: disorder and induced fit in the associations of p53 and 14-3-3 with their partners</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr>. <span class="cit-pub-date">2008</span>;<span class="cit-vol">9</span> Suppl <span class="cit-issue">1</span>: <span class="cit-fpage">S1</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/1471-2164-9-S1-S1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D9%26rft.spage%253DS1%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-S2-S1%26rft_id%253Dinfo%253Apmid%252F18831774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-S2-S1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18831774&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">88.</span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference 88. in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.88" data-doi="10.1016/j.febslet.2015.08.014"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Uversky</span>  <span class="cit-name-given-names">VN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kurgan</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">In various protein complexes, disordered protomers have large per-residue surface areas and area of protein-, DNA- and RNA-binding interfaces</span>. <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-pub-date">2015</span>;<span class="cit-vol">589</span>: <span class="cit-fpage">2561</span>–<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.febslet.2015.08.014</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLett.%26rft.volume%253D589%26rft.spage%253D2561%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.febslet.2015.08.014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.febslet.2015.08.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">89.</span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference 89. in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.89" data-doi="10.1016/j.virusres.2007.10.009"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Narayanan</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Makino</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">SARS coronavirus accessory proteins</span>. <abbr class="cit-jnl-abbrev">Virus Res.</abbr> <span class="cit-pub-date">2008</span>;<span class="cit-vol">133</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">21</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virusres.2007.10.009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DNarayanan%26rft.auinit1%253DK.%26rft.volume%253D133%26rft.issue%253D1%26rft.spage%253D113%26rft.epage%253D121%26rft.atitle%253DSARS%2Bcoronavirus%2Baccessory%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2007.10.009%26rft_id%253Dinfo%253Apmid%252F18045721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2007.10.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18045721&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">90.</span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference 90. in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.90" data-doi="10.1016/j.febslet.2004.03.086"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">C-J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y-C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hsiao</span>  <span class="cit-name-given-names">C-H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuo</span>  <span class="cit-name-given-names">T-C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">SC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">C-Y</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Identification of a novel protein 3a from severe acute respiratory syndrome coronavirus</span>. <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-pub-date">2004</span>;<span class="cit-vol">565</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">6</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.febslet.2004.03.086</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DYu%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D565%26rft.issue%253D1-3%26rft.spage%253D111%26rft.epage%253D116%26rft.atitle%253DIdentification%2Bof%2Ba%2Bnovel%2Bprotein%2B3a%2Bfrom%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome%2Bcoronavirus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.febslet.2004.03.086%26rft_id%253Dinfo%253Apmid%252F15135062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.febslet.2004.03.086&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15135062&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221397400021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">91.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.13.990598v2.91" data-doi="10.1016/j.virusres.2005.01.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shan</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Subcellular localization and membrane association of SARS-CoV 3a protein</span>. <abbr class="cit-jnl-abbrev">Virus Res.</abbr> <span class="cit-pub-date">2005</span>;<span class="cit-vol">109</span>: <span class="cit-fpage">191</span>–<span class="cit-lpage">202</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virusres.2005.01.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DYuan%26rft.auinit1%253DX.%26rft.volume%253D109%26rft.issue%253D2%26rft.spage%253D191%26rft.epage%253D202%26rft.atitle%253DSubcellular%2Blocalization%2Band%2Bmembrane%2Bassociation%2Bof%2BSARS-CoV%2B3a%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2005.01.001%26rft_id%253Dinfo%253Apmid%252F15763150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2005.01.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15763150&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228095800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">92.</span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference 92. in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.92" data-doi="10.1128/JVI.78.13.6723-6734.2004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">Y-J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Teng</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">THP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goh</span>  <span class="cit-name-given-names">P-Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fielding</span>  <span class="cit-name-given-names">BC</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">A Novel Severe Acute Respiratory Syndrome Coronavirus Protein, U274, Is Transported to the Cell Surface and Undergoes Endocytosis</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2004</span>;<span class="cit-vol">78</span>: <span class="cit-fpage">6723</span>–<span class="cit-lpage">6734</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.78.13.6723-6734.2004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DTan%26rft.auinit1%253DY.-J.%26rft.volume%253D78%26rft.issue%253D13%26rft.spage%253D6723%26rft.epage%253D6734%26rft.atitle%253DA%2BNovel%2BSevere%2BAcute%2BRespiratory%2BSyndrome%2BCoronavirus%2BProtein%252C%2BU274%252C%2BIs%2BTransported%2Bto%2Bthe%2BCell%2BSurface%2Band%2BUndergoes%2BEndocytosis%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.78.13.6723-6734.2004%26rft_id%253Dinfo%253Apmid%252F15194747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI3OC8xMy82NzIzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDMvMjAyMC4wMy4xMy45OTA1OTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">93.</span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference 93. in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.93" data-doi="10.1073/pnas.0605402103"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">B-J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schwarz</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">CKL</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Severe acute respiratory syndrome-associated coronavirus 3a protein forms an ion channel and modulates virus release</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>. <span class="cit-pub-date">2006</span>;<span class="cit-vol">103</span>: <span class="cit-fpage">12540</span>–<span class="cit-lpage">5</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.0605402103</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0605402103%26rft_id%253Dinfo%253Apmid%252F16894145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzMzLzEyNTQwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDMvMjAyMC4wMy4xMy45OTA1OTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">94.</span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference 94. in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.94" data-doi="10.1186/1743-422X-2-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">Y-J.</span></span> <span class="cit-article-title">The Severe Acute Respiratory Syndrome (SARS)-coronavirus 3a protein may function as a modulator of the trafficking properties of the spike protein</span>. <abbr class="cit-jnl-abbrev">Virol J.</abbr> <span class="cit-pub-date">2005</span>;<span class="cit-vol">2</span>: <span class="cit-fpage">5</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/1743-422X-2-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DVirol%2BJ%26rft.aulast%253DTan%26rft.auinit1%253DY.%2BJ.%26rft.volume%253D2%26rft.spage%253D5%26rft.epage%253D5%26rft.atitle%253DThe%2BSevere%2BAcute%2BRespiratory%2BSyndrome%2B%2528SARS%2529-coronavirus%2B3a%2Bprotein%2Bmay%2Bfunction%2Bas%2Ba%2Bmodulator%2Bof%2Bthe%2Btrafficking%2Bproperties%2Bof%2Bthe%2Bspike%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1743-422X-2-5%26rft_id%253Dinfo%253Apmid%252F15703085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1743-422X-2-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15703085&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">95.</span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference 95. in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.95" data-doi="10.3390/v4112902"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McBride</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fielding</span>  <span class="cit-name-given-names">BC.</span></span> <span class="cit-article-title">The role of severe acute respiratory syndrome (SARS)-coronavirus accessory proteins in virus pathogenesis</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">4</span>: <span class="cit-fpage">2902</span>–<span class="cit-lpage">23</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.3390/v4112902</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DViruses%26rft.aulast%253DMcBride%26rft.auinit1%253DR.%26rft.volume%253D4%26rft.issue%253D11%26rft.spage%253D2902%26rft.epage%253D2923%26rft.atitle%253DThe%2BRole%2Bof%2BSevere%2BAcute%2BRespiratory%2BSyndrome%2B%2B%2528SARS%2529-Coronavirus%2BAccessory%2BProteins%2Bin%2BVirus%2BPathogenesis.%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv4112902%26rft_id%253Dinfo%253Apmid%252F23202509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v4112902&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23202509&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">96.</span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference 96. in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.96"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shan</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Mitochondrial location of severe acute respiratory syndrome coronavirus 3b protein</span>. <abbr class="cit-jnl-abbrev">Mol Cells.</abbr> <span class="cit-pub-date">2006</span>;<span class="cit-vol">21</span>: <span class="cit-fpage">186</span>–<span class="cit-lpage">91</span>. Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/16682811">http://www.ncbi.nlm.nih.gov/pubmed/16682811</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecules%2Band%2Bcells%26rft.stitle%253DMol%2BCells%26rft.aulast%253DYuan%26rft.auinit1%253DX.%26rft.volume%253D21%26rft.issue%253D2%26rft.spage%253D186%26rft.epage%253D191%26rft.atitle%253DMitochondrial%2Blocation%2Bof%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome%2Bcoronavirus%2B3b%2Bprotein.%26rft_id%253Dinfo%253Apmid%252F16682811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=16682811&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">97.</span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference 97. in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.97" data-doi="10.1016/j.virusres.2005.06.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shan</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Nucleolar localization of non-structural protein 3b, a protein specifically encoded by the severe acute respiratory syndrome coronavirus</span>. <abbr class="cit-jnl-abbrev">Virus Res.</abbr> <span class="cit-pub-date">2005</span>;<span class="cit-vol">114</span>: <span class="cit-fpage">70</span>–<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virusres.2005.06.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DYuan%26rft.auinit1%253DX.%26rft.volume%253D114%26rft.issue%253D1-2%26rft.spage%253D70%26rft.epage%253D79%26rft.atitle%253DNucleolar%2Blocalization%2Bof%2Bnon-structural%2Bprotein%2B3b%252C%2Ba%2Bprotein%2Bspecifically%2Bencoded%2Bby%2Bthe%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome%2Bcoronavirus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2005.06.001%26rft_id%253Dinfo%253Apmid%252F16046244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2005.06.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16046244&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">98.</span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference 98. in text" id="ref-98">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.98" data-doi="10.1021/bi200303r"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Varshney</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lal</span>  <span class="cit-name-given-names">SK.</span></span> <span class="cit-article-title">SARS-CoV accessory protein 3b induces AP-1 transcriptional activity through activation of JNK and ERK pathways</span>. <abbr class="cit-jnl-abbrev">Biochemistry</abbr>. <span class="cit-pub-date">2011</span>;<span class="cit-vol">50</span>: <span class="cit-fpage">5419</span>–<span class="cit-lpage">25</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1021/bi200303r</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBiochemistry%26rft.aulast%253DVarshney%26rft.auinit1%253DB.%26rft.volume%253D50%26rft.issue%253D24%26rft.spage%253D5419%26rft.epage%253D5425%26rft.atitle%253DSARS-CoV%2Baccessory%2Bprotein%2B3b%2Binduces%2BAP-1%2Btranscriptional%2Bactivity%2Bthrough%2Bactivation%2Bof%2BJNK%2Band%2BERK%2Bpathways.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi200303r%26rft_id%253Dinfo%253Apmid%252F21561061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi200303r&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21561061&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">99.</span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference 99. in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.99" data-doi="10.1186/1743-422X-2-66"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shan</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cong</span>  <span class="cit-name-given-names">Y.</span></span> <span class="cit-article-title">G0/G1 arrest and apoptosis induced by SARS-CoV 3b protein in transfected cells</span>. <abbr class="cit-jnl-abbrev">Virol J.</abbr> <span class="cit-pub-date">2005</span>;<span class="cit-vol">2</span>: <span class="cit-fpage">66</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/1743-422X-2-66</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DVirol%2BJ%26rft.aulast%253DYuan%26rft.auinit1%253DX.%26rft.volume%253D2%26rft.spage%253D66%26rft.epage%253D66%26rft.atitle%253DG0%252FG1%2Barrest%2Band%2Bapoptosis%2Binduced%2Bby%2BSARS-CoV%2B3b%2Bprotein%2Bin%2Btransfected%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1743-422X-2-66%26rft_id%253Dinfo%253Apmid%252F16107218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1743-422X-2-66&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16107218&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">100.</span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference 100. in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.100" data-doi="10.1016/j.virusres.2006.06.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Khan</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fielding</span>  <span class="cit-name-given-names">BC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">THP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chou</span>  <span class="cit-name-given-names">C-F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">SG</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Over-expression of severe acute respiratory syndrome coronavirus 3b protein induces both apoptosis and necrosis in Vero E6 cells</span>. <abbr class="cit-jnl-abbrev">Virus Res.</abbr> <span class="cit-pub-date">2006</span>;<span class="cit-vol">122</span>: <span class="cit-fpage">20</span>–<span class="cit-lpage">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virusres.2006.06.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DKhan%26rft.auinit1%253DS.%26rft.volume%253D122%26rft.issue%253D1-2%26rft.spage%253D20%26rft.epage%253D27%26rft.atitle%253DOver-expression%2Bof%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome%2Bcoronavirus%2B3b%2Bprotein%2Binduces%2Bboth%2Bapoptosis%2Band%2Bnecrosis%2Bin%2BVero%2BE6%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2006.06.005%26rft_id%253Dinfo%253Apmid%252F16965829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2006.06.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16965829&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">101.</span><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference 101. in text" id="ref-101">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.101" data-doi="10.1099/vir.0.80813-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Law</span>  <span class="cit-name-given-names">PTW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">C-H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Au</span>  <span class="cit-name-given-names">TCC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chuck</span>  <span class="cit-name-given-names">C-P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">S-K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">PKS</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">The 3a protein of severe acute respiratory syndrome-associated coronavirus induces apoptosis in Vero E6 cells</span>. <abbr class="cit-jnl-abbrev">J Gen Virol.</abbr> <span class="cit-pub-date">2005</span>;<span class="cit-vol">86</span>: <span class="cit-fpage">1921</span>–<span class="cit-lpage">30</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1099/vir.0.80813-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DLaw%26rft.auinit1%253DP.%2BT.%2BW.%26rft.volume%253D86%26rft.issue%253D7%26rft.spage%253D1921%26rft.epage%253D1930%26rft.atitle%253DThe%2B3a%2Bprotein%2Bof%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome-associated%2Bcoronavirus%2Binduces%2Bapoptosis%2Bin%2BVero%2BE6%2Bcells%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.80813-0%26rft_id%253Dinfo%253Apmid%252F15958670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.80813-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15958670&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230220300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">102.</span><a class="rev-xref-ref" href="#xref-ref-102-1" title="View reference 102. in text" id="ref-102">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.102" data-doi="10.1128/JVI.01782-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kopecky-Bromberg</span>  <span class="cit-name-given-names">SA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martinez-Sobrido</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frieman</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>  <span class="cit-name-given-names">RA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Palese</span>  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">Severe Acute Respiratory Syndrome Coronavirus Open Reading Frame (ORF) 3b, ORF 6, and Nucleocapsid Proteins Function as Interferon Antagonists</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2007</span>;<span class="cit-vol">81</span>: <span class="cit-fpage">548</span>–<span class="cit-lpage">557</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.01782-06</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01782-06%26rft_id%253Dinfo%253Apmid%252F17108024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjgxLzIvNTQ4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDMvMjAyMC4wMy4xMy45OTA1OTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">103.</span><a class="rev-xref-ref" href="#xref-ref-103-1" title="View reference 103. in text" id="ref-103">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.103" data-doi="10.1128/JVI.01012-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Frieman</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yount</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heise</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kopecky-Bromberg</span>  <span class="cit-name-given-names">SA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Palese</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>  <span class="cit-name-given-names">RS.</span></span> <span class="cit-article-title">Severe Acute Respiratory Syndrome Coronavirus ORF6 Antagonizes STAT1 Function by Sequestering Nuclear Import Factors on the Rough Endoplasmic Reticulum/Golgi Membrane</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2007</span>;<span class="cit-vol">81</span>: <span class="cit-fpage">9812</span>–<span class="cit-lpage">9824</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.01012-07</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01012-07%26rft_id%253Dinfo%253Apmid%252F17596301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4MS8xOC85ODEyIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDMvMjAyMC4wMy4xMy45OTA1OTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">104.</span><a class="rev-xref-ref" href="#xref-ref-104-1" title="View reference 104. in text" id="ref-104">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.104" data-doi="10.1128/JVI.01308-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Netland</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ferraro</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pewe</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Olivares</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gallagher</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perlman</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Enhancement of Murine Coronavirus Replication by Severe Acute Respiratory Syndrome Coronavirus Protein 6 Requires the N-Terminal Hydrophobic Region but Not C-Terminal Sorting Motifs</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2007</span>;<span class="cit-vol">81</span>: <span class="cit-fpage">11520</span>–<span class="cit-lpage">11525</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.01308-07</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01308-07%26rft_id%253Dinfo%253Apmid%252F17670827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4MS8yMC8xMTUyMCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAzLzIwMjAuMDMuMTMuOTkwNTk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">105.</span><a class="rev-xref-ref" href="#xref-ref-105-1" title="View reference 105. in text" id="ref-105">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.105" data-doi="10.1186/1756-0500-4-446"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gunalan</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mirazimi</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">Y-J.</span></span> <span class="cit-article-title">A putative diacidic motif in the SARS-CoV ORF6 protein influences its subcellular localization and suppression of expression of co-transfected expression constructs</span>. <abbr class="cit-jnl-abbrev">BMC Res Notes.</abbr> <span class="cit-pub-date">2011</span>;<span class="cit-vol">4</span>: <span class="cit-fpage">446</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/1756-0500-4-446</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BRes%2BNotes%26rft.aulast%253DGunalan%26rft.auinit1%253DV.%26rft.volume%253D4%26rft.spage%253D446%26rft.epage%253D446%26rft.atitle%253DA%2Bputative%2Bdiacidic%2Bmotif%2Bin%2Bthe%2BSARS-CoV%2BORF6%2Bprotein%2Binfluences%2Bits%2Bsubcellular%2Blocalization%2Band%2Bsuppression%2Bof%2Bexpression%2Bof%2Bco-transfected%2Bexpression%2Bconstructs.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1756-0500-4-446%26rft_id%253Dinfo%253Apmid%252F22026976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1756-0500-4-446&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22026976&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">106.</span><a class="rev-xref-ref" href="#xref-ref-106-1" title="View reference 106. in text" id="ref-106">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.106" data-doi="10.1128/JVI.02570-09"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ferraro</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hussain</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shao</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Trujillo</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">The N-Terminal Region of Severe Acute Respiratory Syndrome Coronavirus Protein 6 Induces Membrane Rearrangement and Enhances Virus Replication</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2010</span>;<span class="cit-vol">84</span>: <span class="cit-fpage">3542</span>–<span class="cit-lpage">3551</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.02570-09</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02570-09%26rft_id%253Dinfo%253Apmid%252F20106914%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg0LzcvMzU0MiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAzLzIwMjAuMDMuMTMuOTkwNTk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">107.</span><a class="rev-xref-ref" href="#xref-ref-107-1" title="View reference 107. in text" id="ref-107">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.107" data-doi="10.1128/JVI.78.14.7311-7318.2004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fielding</span>  <span class="cit-name-given-names">BC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">Y-J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shuo</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">THP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ooi</span>  <span class="cit-name-given-names">E-E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">SG</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Characterization of a Unique Group-Specific Protein (U122) of the Severe Acute Respiratory Syndrome Coronavirus</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2004</span>;<span class="cit-vol">78</span>: <span class="cit-fpage">7311</span>–<span class="cit-lpage">7318</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.78.14.7311-7318.2004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DFielding%26rft.auinit1%253DB.%2BC.%26rft.volume%253D78%26rft.issue%253D14%26rft.spage%253D7311%26rft.epage%253D7318%26rft.atitle%253DCharacterization%2Bof%2Ba%2BUnique%2BGroup-Specific%2BProtein%2B%2528U122%2529%2Bof%2Bthe%2BSevere%2BAcute%2BRespiratory%2BSyndrome%2BCoronavirus%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.78.14.7311-7318.2004%26rft_id%253Dinfo%253Apmid%252F15220404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI3OC8xNC83MzExIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDMvMjAyMC4wMy4xMy45OTA1OTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">108.</span><a class="rev-xref-ref" href="#xref-ref-108-1" title="View reference 108. in text" id="ref-108">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.108" data-doi="10.1128/JVI.00414-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ito</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tseng</span>  <span class="cit-name-given-names">C-TK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Makino</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Severe Acute Respiratory Syndrome Coronavirus 7a Accessory Protein Is a Viral Structural Protein</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2006</span>;<span class="cit-vol">80</span>: <span class="cit-fpage">7287</span>–<span class="cit-lpage">7294</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.00414-06</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DHuang%26rft.auinit1%253DC.%26rft.volume%253D80%26rft.issue%253D15%26rft.spage%253D7287%26rft.epage%253D7294%26rft.atitle%253DSevere%2BAcute%2BRespiratory%2BSyndrome%2BCoronavirus%2B7a%2BAccessory%2BProtein%2BIs%2Ba%2BViral%2BStructural%2BProtein%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00414-06%26rft_id%253Dinfo%253Apmid%252F16840309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4MC8xNS83Mjg3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDMvMjAyMC4wMy4xMy45OTA1OTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">109.</span><a class="rev-xref-ref" href="#xref-ref-109-1" title="View reference 109. in text" id="ref-109">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.109" data-doi="10.1016/j.febslet.2006.11.046"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kanzawa</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nishigaki</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ishii</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Furukawa</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Niiro</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Augmentation of chemokine production by severe acute respiratory syndrome coronavirus 3a/X1 and 7a/X4 proteins through NF-kappaB activation</span>. <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-pub-date">2006</span>;<span class="cit-vol">580</span>: <span class="cit-fpage">6807</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.febslet.2006.11.046</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DKanzawa%26rft.auinit1%253DN.%26rft.volume%253D580%26rft.issue%253D30%26rft.spage%253D6807%26rft.epage%253D6812%26rft.atitle%253DAugmentation%2Bof%2Bchemokine%2Bproduction%2Bby%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome%2Bcoronavirus%2B3a%252FX1%2Band%2B7a%252FX4%2Bproteins%2Bthrough%2BNF-kappaB%2Bactivation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.febslet.2006.11.046%26rft_id%253Dinfo%253Apmid%252F17141229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.febslet.2006.11.046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17141229&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">110.</span><a class="rev-xref-ref" href="#xref-ref-110-1" title="View reference 110. in text" id="ref-110">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.110" data-doi="10.1182/blood-2004-10-4166"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Law</span>  <span class="cit-name-given-names">HKW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">CY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">HY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sia</span>  <span class="cit-name-given-names">SF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">YO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luk</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Chemokine up-regulation in SARS-coronavirus-infected, monocyte-derived human dendritic cells</span>. <abbr class="cit-jnl-abbrev">Blood.</abbr> <span class="cit-pub-date">2005</span>;<span class="cit-vol">106</span>: <span class="cit-fpage">2366</span>–<span class="cit-lpage">74</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1182/blood-2004-10-4166</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBlood.%26rft_id%253Dinfo%253Adoi%252F10.1182%252Fblood-2004-10-4166%26rft_id%253Dinfo%253Apmid%252F15860669%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6ImJsb29kam91cm5hbCI7czo1OiJyZXNpZCI7czoxMDoiMTA2LzcvMjM2NiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAzLzIwMjAuMDMuMTMuOTkwNTk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">111.</span><a class="rev-xref-ref" href="#xref-ref-111-1" title="View reference 111. in text" id="ref-111">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.111" data-doi="10.1128/JVI.00090-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">Y-X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">THP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">MJ-R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tham</span>  <span class="cit-name-given-names">P-Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gunalan</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Druce</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Induction of Apoptosis by the Severe Acute Respiratory Syndrome Coronavirus 7a Protein Is Dependent on Its Interaction with the Bcl-XL Protein</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2007</span>;<span class="cit-vol">81</span>: <span class="cit-fpage">6346</span>–<span class="cit-lpage">6355</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.00090-07</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00090-07%26rft_id%253Dinfo%253Apmid%252F17428862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4MS8xMi82MzQ2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDMvMjAyMC4wMy4xMy45OTA1OTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">112.</span><a class="rev-xref-ref" href="#xref-ref-112-1" title="View reference 112. in text" id="ref-112">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.112" data-doi="10.1128/JVI.01691-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schaecher</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mackenzie</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pekosz</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">The ORF7b Protein of Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) Is Expressed in Virus-Infected Cells and Incorporated into SARS-CoV Particles</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2007</span>;<span class="cit-vol">81</span>: <span class="cit-fpage">718</span>–<span class="cit-lpage">731</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.01691-06</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01691-06%26rft_id%253Dinfo%253Apmid%252F17079322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjgxLzIvNzE4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDMvMjAyMC4wMy4xMy45OTA1OTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">113.</span><a class="rev-xref-ref" href="#xref-ref-113-1" title="View reference 113. in text" id="ref-113">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.113" data-doi="10.1128/JVI.80.2.785-793.2006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kopecky-Bromberg</span>  <span class="cit-name-given-names">SA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martinez-Sobrido</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Palese</span>  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">7a Protein of Severe Acute Respiratory Syndrome Coronavirus Inhibits Cellular Protein Synthesis and Activates p38 Mitogen-Activated Protein Kinase</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2006</span>;<span class="cit-vol">80</span>: <span class="cit-fpage">785</span>–<span class="cit-lpage">793</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.80.2.785-793.2006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DKopecky-Bromberg%26rft.auinit1%253DS.%2BA.%26rft.volume%253D80%26rft.issue%253D2%26rft.spage%253D785%26rft.epage%253D793%26rft.atitle%253D7a%2BProtein%2Bof%2BSevere%2BAcute%2BRespiratory%2BSyndrome%2BCoronavirus%2BInhibits%2BCellular%2BProtein%2BSynthesis%2Band%2BActivates%2Bp38%2BMitogen-Activated%2BProtein%2BKinase%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.80.2.785-793.2006%26rft_id%253Dinfo%253Apmid%252F16378980%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjgwLzIvNzg1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDMvMjAyMC4wMy4xMy45OTA1OTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">114.</span><a class="rev-xref-ref" href="#xref-ref-114-1" title="View reference 114. in text" id="ref-114">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.114" data-doi="10.1016/j.str.2004.10.010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pekosz</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Diamond</span>  <span class="cit-name-given-names">MS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fremont</span>  <span class="cit-name-given-names">DH.</span></span> <span class="cit-article-title">Structure and intracellular targeting of the SARS-coronavirus Orf7a accessory protein</span>. <abbr class="cit-jnl-abbrev">Structure.</abbr> <span class="cit-pub-date">2005</span>;<span class="cit-vol">13</span>: <span class="cit-fpage">75</span>–<span class="cit-lpage">85</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.str.2004.10.010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStructure%2B%2528London%2529%26rft.stitle%253DStructure%2B%2528London%2529%26rft.aulast%253DNelson%26rft.auinit1%253DC.%2BA.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D75%26rft.epage%253D85%26rft.atitle%253DStructure%2Band%2Bintracellular%2Btargeting%2Bof%2Bthe%2BSARS-coronavirus%2BOrf7a%2Baccessory%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.str.2004.10.010%26rft_id%253Dinfo%253Apmid%252F15642263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.str.2004.10.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15642263&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">115.</span><a class="rev-xref-ref" href="#xref-ref-115-1" title="View reference 115. in text" id="ref-115">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.115" data-doi="10.1007/s11373-005-9043-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hänel</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stangler</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stoldt</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Willbold</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">Solution structure of the X4 protein coded by the SARS related coronavirus reveals an immunoglobulin like fold and suggests a binding activity to integrin I domains</span>. <abbr class="cit-jnl-abbrev">J Biomed Sci.</abbr> <span class="cit-pub-date">2006</span>;<span class="cit-vol">13</span>: <span class="cit-fpage">281</span>–<span class="cit-lpage">293</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/s11373-005-9043-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bbiomedical%2Bscience%26rft.stitle%253DJ%2BBiomed%2BSci%26rft.aulast%253DHanel%26rft.auinit1%253DK.%26rft.volume%253D13%26rft.issue%253D3%26rft.spage%253D281%26rft.epage%253D293%26rft.atitle%253DSolution%2Bstructure%2Bof%2Bthe%2BX4%2Bprotein%2Bcoded%2Bby%2Bthe%2BSARS%2Brelated%2Bcoronavirus%2Breveals%2Ban%2Bimmunoglobulin%2Blike%2Bfold%2Band%2Bsuggests%2Ba%2Bbinding%2Bactivity%2Bto%2Bintegrin%2BI%2Bdomains.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11373-005-9043-9%26rft_id%253Dinfo%253Apmid%252F16328780%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11373-005-9043-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16328780&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">116.</span><a class="rev-xref-ref" href="#xref-ref-116-1" title="View reference 116. in text" id="ref-116">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.116" data-doi="10.1128/JVI.01631-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Oostra</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Haan</span>  <span class="cit-name-given-names">CAM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rottier</span>  <span class="cit-name-given-names">PJM.</span></span> <span class="cit-article-title">The 29-Nucleotide Deletion Present in Human but Not in Animal Severe Acute Respiratory Syndrome Coronaviruses Disrupts the Functional Expression of Open Reading Frame 8</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2007</span>;<span class="cit-vol">81</span>: <span class="cit-fpage">13876</span>–<span class="cit-lpage">13888</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.01631-07</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01631-07%26rft_id%253Dinfo%253Apmid%252F17928347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4MS8yNC8xMzg3NiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAzLzIwMjAuMDMuMTMuOTkwNTk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">117.</span><a class="rev-xref-ref" href="#xref-ref-117-1" title="View reference 117. in text" id="ref-117">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.117" data-doi="10.1126/science.1092002"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Chinese SARS Molecular Epidemiology Consortium</span>. <span class="cit-article-title">Molecular evolution of the SARS coronavirus during the course of the SARS epidemic in China</span>. <abbr class="cit-jnl-abbrev">Science.</abbr> <span class="cit-pub-date">2004</span>;<span class="cit-vol">303</span>: <span class="cit-fpage">1666</span>–<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1092002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience.%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1092002%26rft_id%253Dinfo%253Apmid%252F14752165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDMvNTY2NC8xNjY2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDMvMjAyMC4wMy4xMy45OTA1OTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">118.</span><a class="rev-xref-ref" href="#xref-ref-118-1" title="View reference 118. in text" id="ref-118">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.118" data-doi="10.1016/j.virol.2006.06.026"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Keng</span>  <span class="cit-name-given-names">C-T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">Y-W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Welkers</span>  <span class="cit-name-given-names">MRA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">DZL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gee Lim</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">The human severe acute respiratory syndrome coronavirus (SARS-CoV) 8b protein is distinct from its counterpart in animal SARS-CoV and down-regulates the expression of the envelope protein in infected cells</span>. <abbr class="cit-jnl-abbrev">Virology.</abbr> <span class="cit-pub-date">2006</span>;<span class="cit-vol">354</span>: <span class="cit-fpage">132</span>–<span class="cit-lpage">42</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virol.2006.06.026</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DKeng%26rft.auinit1%253DC.%2BT.%26rft.volume%253D354%26rft.issue%253D1%26rft.spage%253D132%26rft.epage%253D142%26rft.atitle%253DThe%2Bhuman%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome%2Bcoronavirus%2B%2528SARS-CoV%2529%2B8b%2Bprotein%2Bis%2Bdistinct%2Bfrom%2Bits%2Bcounterpart%2Bin%2Banimal%2BSARS-CoV%2Band%2Bdown-regulates%2Bthe%2Bexpression%2Bof%2Bthe%2Benvelope%2Bprotein%2Bin%2Binfected%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2006.06.026%26rft_id%253Dinfo%253Apmid%252F16876844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2006.06.026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16876844&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">119.</span><a class="rev-xref-ref" href="#xref-ref-119-1" title="View reference 119. in text" id="ref-119">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.119" data-doi="10.1016/j.virol.2009.03.032"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">B-J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Y-P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xue</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Severe acute respiratory syndrome coronavirus accessory protein 9b is a virion-associated protein</span>. <abbr class="cit-jnl-abbrev">Virology.</abbr> <span class="cit-pub-date">2009</span>;<span class="cit-vol">388</span>: <span class="cit-fpage">279</span>–<span class="cit-lpage">85</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virol.2009.03.032</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DVirology%26rft.aulast%253DXu%26rft.auinit1%253DK.%26rft.volume%253D388%26rft.issue%253D2%26rft.spage%253D279%26rft.epage%253D285%26rft.atitle%253DSevere%2Bacute%2Brespiratory%2Bsyndrome%2Bcoronavirus%2Baccessory%2Bprotein%2B9b%2Bis%2Ba%2Bvirion-associated%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2009.03.032%26rft_id%253Dinfo%253Apmid%252F19394665%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2009.03.032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19394665&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">120.</span><a class="rev-xref-ref" href="#xref-ref-120-1" title="View reference 120. in text" id="ref-120">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.120" data-doi="10.1371/journal.pone.0019436"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Åkerström</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">AK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chow</span>  <span class="cit-name-given-names">VTK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Teow</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abrenica</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">SARS-CoV 9b protein diffuses into nucleus, undergoes active Crm1 mediated nucleocytoplasmic export and triggers apoptosis when retained in the nucleus</span>. <abbr class="cit-jnl-abbrev">PLoS One.</abbr> <span class="cit-pub-date">2011</span>;<span class="cit-vol">6</span>: <span class="cit-fpage">e19436</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0019436</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSharma%26rft.auinit1%253DK.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19436%26rft.epage%253De19436%26rft.atitle%253DSARS-CoV%2B9b%2Bprotein%2Bdiffuses%2Binto%2Bnucleus%252C%2Bundergoes%2Bactive%2BCrm1%2Bmediated%2Bnucleocytoplasmic%2Bexport%2Band%2Btriggers%2Bapoptosis%2Bwhen%2Bretained%2Bin%2Bthe%2Bnucleus.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019436%26rft_id%253Dinfo%253Apmid%252F21637748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019436&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21637748&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">121.</span><a class="rev-xref-ref" href="#xref-ref-121-1" title="View reference 121. in text" id="ref-121">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.121" data-doi="10.1016/j.str.2006.05.012"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Meier</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aricescu</span>  <span class="cit-name-given-names">AR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Assenberg</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aplin</span>  <span class="cit-name-given-names">RT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">RJC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grimes</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">The crystal structure of ORF-9b, a lipid binding protein from the SARS coronavirus</span>. <abbr class="cit-jnl-abbrev">Structure.</abbr> <span class="cit-pub-date">2006</span>;<span class="cit-vol">14</span>: <span class="cit-fpage">1157</span>–<span class="cit-lpage">65</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.str.2006.05.012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStructure%2B%2528London%2529%26rft.stitle%253DStructure%2B%2528London%2529%26rft.aulast%253DMeier%26rft.auinit1%253DC.%26rft.volume%253D14%26rft.issue%253D7%26rft.spage%253D1157%26rft.epage%253D1165%26rft.atitle%253DThe%2Bcrystal%2Bstructure%2Bof%2BORF-9b%252C%2Ba%2Blipid%2Bbinding%2Bprotein%2Bfrom%2Bthe%2BSARS%2Bcoronavirus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.str.2006.05.012%26rft_id%253Dinfo%253Apmid%252F16843897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.str.2006.05.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16843897&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">122.</span><a class="rev-xref-ref" href="#xref-ref-122-1" title="View reference 122. in text" id="ref-122">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.122" data-doi="10.1099/vir.0.19424-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thiel</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ivanov</span>  <span class="cit-name-given-names">KA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Putics</span>  <span class="cit-name-given-names">Á</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hertzig</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schelle</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bayer</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Mechanisms and enzymes involved in SARS coronavirus genome expression</span>. <abbr class="cit-jnl-abbrev">J Gen Virol.</abbr> <span class="cit-pub-date">2003</span>;<span class="cit-vol">84</span>: <span class="cit-fpage">2305</span>–<span class="cit-lpage">2315</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1099/vir.0.19424-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DThiel%26rft.auinit1%253DV.%26rft.volume%253D84%26rft.issue%253D9%26rft.spage%253D2305%26rft.epage%253D2315%26rft.atitle%253DMechanisms%2Band%2Benzymes%2Binvolved%2Bin%2BSARS%2Bcoronavirus%2Bgenome%2Bexpression%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.19424-0%26rft_id%253Dinfo%253Apmid%252F12917450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.19424-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12917450&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184993100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">123.</span><a class="rev-xref-ref" href="#xref-ref-123-1" title="View reference 123. in text" id="ref-123">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.123" data-doi="10.1074/jbc.M310875200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Biosynthesis, purification, and substrate specificity of severe acute respiratory syndrome coronavirus 3C-like proteinase</span>. <abbr class="cit-jnl-abbrev">J Biol Chem.</abbr> <span class="cit-pub-date">2004</span>;<span class="cit-vol">279</span>: <span class="cit-fpage">1637</span>–<span class="cit-lpage">42</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1074/jbc.M310875200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M310875200%26rft_id%253Dinfo%253Apmid%252F14561748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyNzkvMy8xNjM3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDMvMjAyMC4wMy4xMy45OTA1OTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">124.</span><a class="rev-xref-ref" href="#xref-ref-124-1" title="View reference 124. in text" id="ref-124">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.124" data-doi="10.1128/JVI.01958-12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lokugamage</span>  <span class="cit-name-given-names">KG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Narayanan</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Makino</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Severe Acute Respiratory Syndrome Coronavirus Protein nsp1 Is a Novel Eukaryotic Translation Inhibitor That Represses Multiple Steps of Translation Initiation</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2012</span>;<span class="cit-vol">86</span>: <span class="cit-fpage">13598</span>–<span class="cit-lpage">13608</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.01958-12</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01958-12%26rft_id%253Dinfo%253Apmid%252F23035226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4Ni8yNC8xMzU5OCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAzLzIwMjAuMDMuMTMuOTkwNTk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">125.</span><a class="rev-xref-ref" href="#xref-ref-125-1" title="View reference 125. in text" id="ref-125">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.125" data-doi="10.1128/JVI.01939-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Almeida</span>  <span class="cit-name-given-names">MS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Herrmann</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Geralt</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wuthrich</span>  <span class="cit-name-given-names">K.</span></span> <span class="cit-article-title">Novel-Barrel Fold in the Nuclear Magnetic Resonance Structure of the Replicase Nonstructural Protein 1 from the Severe Acute Respiratory Syndrome Coronavirus</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2007</span>;<span class="cit-vol">81</span>: <span class="cit-fpage">3151</span>–<span class="cit-lpage">3161</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.01939-06</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01939-06%26rft_id%253Dinfo%253Apmid%252F17202208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgxLzcvMzE1MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAzLzIwMjAuMDMuMTMuOTkwNTk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">126.</span><a class="rev-xref-ref" href="#xref-ref-126-1" title="View reference 126. in text" id="ref-126">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.126" data-doi="10.1371/journal.pone.0062416"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jauregui</span>  <span class="cit-name-given-names">AR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Savalia</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lowry</span>  <span class="cit-name-given-names">VK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Farrell</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wathelet</span>  <span class="cit-name-given-names">MG.</span></span> <span class="cit-article-title">Identification of residues of SARS-CoV nsp1 that differentially affect inhibition of gene expression and antiviral signaling</span>. <abbr class="cit-jnl-abbrev">PLoS One.</abbr> <span class="cit-pub-date">2013</span>;<span class="cit-vol">8</span>: <span class="cit-fpage">e62416</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0062416</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne.%26rft.volume%253D8%26rft.spage%253De62416%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0062416%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0062416&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">127.</span><a class="rev-xref-ref" href="#xref-ref-127-1" title="View reference 127. in text" id="ref-127">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.127" data-doi="10.1016/j.virusres.2014.11.019"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Narayanan</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ramirez</span>  <span class="cit-name-given-names">SI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lokugamage</span>  <span class="cit-name-given-names">KG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Makino</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Coronavirus nonstructural protein 1: Common and distinct functions in the regulation of host and viral gene expression</span>. <abbr class="cit-jnl-abbrev">Virus Res.</abbr> <span class="cit-pub-date">2015</span>;<span class="cit-vol">202</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">100</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virusres.2014.11.019</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2014.11.019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2014.11.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">128.</span><a class="rev-xref-ref" href="#xref-ref-128-1" title="View reference 128. in text" id="ref-128">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.128" data-doi="10.1128/JVI.00842-09"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cornillez-Ty</span>  <span class="cit-name-given-names">CT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yates</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buchmeier</span>  <span class="cit-name-given-names">MJ.</span></span> <span class="cit-article-title">Severe Acute Respiratory Syndrome Coronavirus Nonstructural Protein 2 Interacts with a Host Protein Complex Involved in Mitochondrial Biogenesis and Intracellular Signaling</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2009</span>;<span class="cit-vol">83</span>: <span class="cit-fpage">10314</span>–<span class="cit-lpage">10318</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.00842-09</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00842-09%26rft_id%253Dinfo%253Apmid%252F19640993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4My8xOS8xMDMxNCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAzLzIwMjAuMDMuMTMuOTkwNTk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">129.</span><a class="rev-xref-ref" href="#xref-ref-129-1" title="View reference 129. in text" id="ref-129">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.129" data-doi="10.1128/JVI.79.21.13399-13411.2005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">RL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sims</span>  <span class="cit-name-given-names">AC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brockway</span>  <span class="cit-name-given-names">SM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>  <span class="cit-name-given-names">RS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Denison</span>  <span class="cit-name-given-names">MR.</span></span> <span class="cit-article-title">The nsp2 Replicase Proteins of Murine Hepatitis Virus and Severe Acute Respiratory Syndrome Coronavirus Are Dispensable for Viral Replication</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2005</span>;<span class="cit-vol">79</span>: <span class="cit-fpage">13399</span>–<span class="cit-lpage">13411</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.79.21.13399-13411.2005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DGraham%26rft.auinit1%253DR.%2BL.%26rft.volume%253D79%26rft.issue%253D21%26rft.spage%253D13399%26rft.epage%253D13411%26rft.atitle%253DThe%2Bnsp2%2BReplicase%2BProteins%2Bof%2BMurine%2BHepatitis%2BVirus%2Band%2BSevere%2BAcute%2BRespiratory%2BSyndrome%2BCoronavirus%2BAre%2BDispensable%2Bfor%2BViral%2BReplication%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.79.21.13399-13411.2005%26rft_id%253Dinfo%253Apmid%252F16227261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI3OS8yMS8xMzM5OSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAzLzIwMjAuMDMuMTMuOTkwNTk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">130.</span><a class="rev-xref-ref" href="#xref-ref-130-1" title="View reference 130. in text" id="ref-130">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.130" data-doi="10.1128/JVI.02220-08"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Frieman</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ratia</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">RE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mesecar</span>  <span class="cit-name-given-names">AD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>  <span class="cit-name-given-names">RS.</span></span> <span class="cit-article-title">Severe Acute Respiratory Syndrome Coronavirus Papain-Like Protease Ubiquitin-Like Domain and Catalytic Domain Regulate Antagonism of IRF3 and NF-B Signaling</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2009</span>;<span class="cit-vol">83</span>: <span class="cit-fpage">6689</span>–<span class="cit-lpage">6705</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.02220-08</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02220-08%26rft_id%253Dinfo%253Apmid%252F19369340%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4My8xMy82Njg5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDMvMjAyMC4wMy4xMy45OTA1OTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">131.</span><a class="rev-xref-ref" href="#xref-ref-131-1" title="View reference 131. in text" id="ref-131">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.131" data-doi="10.1073/pnas.0510851103"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ratia</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saikatendu</span>  <span class="cit-name-given-names">KS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Santarsiero</span>  <span class="cit-name-given-names">BD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barretto</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">SC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">RC</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Severe acute respiratory syndrome coronavirus papain-like protease: structure of a viral deubiquitinating enzyme</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>. <span class="cit-pub-date">2006</span>;<span class="cit-vol">103</span>: <span class="cit-fpage">5717</span>–<span class="cit-lpage">22</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.0510851103</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0510851103%26rft_id%253Dinfo%253Apmid%252F16581910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAzLzE1LzU3MTciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wMy8yMDIwLjAzLjEzLjk5MDU5OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">132.</span><a class="rev-xref-ref" href="#xref-ref-132-1" title="View reference 132. in text" id="ref-132">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.132" data-doi="10.1128/JVI.00969-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Serrano</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Almeida</span>  <span class="cit-name-given-names">MS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horst</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Herrmann</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joseph</span>  <span class="cit-name-given-names">JS</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Nuclear Magnetic Resonance Structure of the N-Terminal Domain of Nonstructural Protein 3 from the Severe Acute Respiratory Syndrome Coronavirus</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2007</span>;<span class="cit-vol">81</span>: <span class="cit-fpage">12049</span>–<span class="cit-lpage">12060</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.00969-07</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00969-07%26rft_id%253Dinfo%253Apmid%252F17728234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4MS8yMS8xMjA0OSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAzLzIwMjAuMDMuMTMuOTkwNTk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">133.</span><a class="rev-xref-ref" href="#xref-ref-133-1" title="View reference 133. in text" id="ref-133">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.133" data-doi="10.1016/j.virol.2010.03.015"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Alvarez</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">DeDiego</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nieto-Torres</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiménez-Guardeño</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marcos-Villar</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Enjuanes</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">The envelope protein of severe acute respiratory syndrome coronavirus interacts with the non-structural protein 3 and is ubiquitinated</span>. <abbr class="cit-jnl-abbrev">Virology.</abbr> <span class="cit-pub-date">2010</span>;<span class="cit-vol">402</span>: <span class="cit-fpage">281</span>–<span class="cit-lpage">91</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virol.2010.03.015</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DVirology%26rft.aulast%253DAlvarez%26rft.auinit1%253DE.%26rft.volume%253D402%26rft.issue%253D2%26rft.spage%253D281%26rft.epage%253D291%26rft.atitle%253DThe%2Benvelope%2Bprotein%2Bof%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome%2Bcoronavirus%2Binteracts%2Bwith%2Bthe%2Bnon-structural%2Bprotein%2B3%2Band%2Bis%2Bubiquitinated.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2010.03.015%26rft_id%253Dinfo%253Apmid%252F20409569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2010.03.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20409569&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">134.</span><a class="rev-xref-ref" href="#xref-ref-134-1" title="View reference 134. in text" id="ref-134">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.134" data-doi="10.1128/mBio.00524-13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Angelini</span>  <span class="cit-name-given-names">MM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Akhlaghpour</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neuman</span>  <span class="cit-name-given-names">BW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buchmeier</span>  <span class="cit-name-given-names">MJ.</span></span> <span class="cit-article-title">Severe acute respiratory syndrome coronavirus nonstructural proteins 3, 4, and 6 induce double-membrane vesicles</span>. <abbr class="cit-jnl-abbrev">MBio.</abbr> <span class="cit-pub-date">2013</span>;<span class="cit-fpage">4</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/mBio.00524-13</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.00524-13%26rft_id%253Dinfo%253Apmid%252F23943763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibWJpbyI7czo1OiJyZXNpZCI7czoxMzoiNC80L2UwMDUyNC0xMyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAzLzIwMjAuMDMuMTMuOTkwNTk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">135.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.13.990598v2.135" data-doi="10.1128/JVI.00042-11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hagemeijer</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ulasli</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vonk</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reggiori</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rottier</span>  <span class="cit-name-given-names">PJM</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Haan</span>  <span class="cit-name-given-names">CAM.</span></span> <span class="cit-article-title">Mobility and Interactions of Coronavirus Nonstructural Protein 4</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2011</span>;<span class="cit-vol">85</span>: <span class="cit-fpage">4572</span>–<span class="cit-lpage">4577</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.00042-11</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00042-11%26rft_id%253Dinfo%253Apmid%252F21345958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg1LzkvNDU3MiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAzLzIwMjAuMDMuMTMuOTkwNTk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">136.</span><a class="rev-xref-ref" href="#xref-ref-136-1" title="View reference 136. in text" id="ref-136">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.136" data-doi="10.1016/j.virol.2017.07.019"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sakai</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kawachi</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Terada</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Omori</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matsuura</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kamitani</span>  <span class="cit-name-given-names">W.</span></span> <span class="cit-article-title">Two-amino acids change in the nsp4 of SARS coronavirus abolishes viral replication</span>. <abbr class="cit-jnl-abbrev">Virology.</abbr> <span class="cit-pub-date">2017</span>;<span class="cit-vol">510</span>: <span class="cit-fpage">165</span>–<span class="cit-lpage">174</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virol.2017.07.019</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology.%26rft.volume%253D510%26rft.spage%253D165%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2017.07.019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2017.07.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">137.</span><a class="rev-xref-ref" href="#xref-ref-137-1" title="View reference 137. in text" id="ref-137">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.137" data-doi="10.1128/JVI.01506-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Oostra</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">te Lintelo</span>  <span class="cit-name-given-names">EG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deijs</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Verheije</span>  <span class="cit-name-given-names">MH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rottier</span>  <span class="cit-name-given-names">PJM</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Haan</span>  <span class="cit-name-given-names">CAM.</span></span> <span class="cit-article-title">Localization and Membrane Topology of Coronavirus Nonstructural Protein 4: Involvement of the Early Secretory Pathway in Replication</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2007</span>;<span class="cit-vol">81</span>: <span class="cit-fpage">12323</span>–<span class="cit-lpage">12336</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.01506-07</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01506-07%26rft_id%253Dinfo%253Apmid%252F17855519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4MS8yMi8xMjMyMyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAzLzIwMjAuMDMuMTMuOTkwNTk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">138.</span><a class="rev-xref-ref" href="#xref-ref-138-1" title="View reference 138. in text" id="ref-138">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.138" data-doi="10.1074/jbc.M115.651463"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tomar</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">St John</span>  <span class="cit-name-given-names">SE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Osswald</span>  <span class="cit-name-given-names">HL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nyalapatla</span>  <span class="cit-name-given-names">PR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">LN</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Ligand-induced Dimerization of Middle East Respiratory Syndrome (MERS) Coronavirus nsp5 Protease (3CLpro): IMPLICATIONS FOR nsp5 REGULATION AND THE DEVELOPMENT OF ANTIVIRALS</span>. <abbr class="cit-jnl-abbrev">J Biol Chem.</abbr> <span class="cit-pub-date">2015</span>;<span class="cit-vol">290</span>: <span class="cit-fpage">19403</span>–<span class="cit-lpage">22</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1074/jbc.M115.651463</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M115.651463%26rft_id%253Dinfo%253Apmid%252F26055715%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyOTAvMzIvMTk0MDMiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wMy8yMDIwLjAzLjEzLjk5MDU5OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">139.</span><a class="rev-xref-ref" href="#xref-ref-139-1" title="View reference 139. in text" id="ref-139">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.139" data-doi="10.1128/JVI.00203-08"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sparks</span>  <span class="cit-name-given-names">JS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Donaldson</span>  <span class="cit-name-given-names">EF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>  <span class="cit-name-given-names">RS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Denison</span>  <span class="cit-name-given-names">MR.</span></span> <span class="cit-article-title">A Novel Mutation in Murine Hepatitis Virus nsp5, the Viral 3C-Like Proteinase, Causes Temperature-Sensitive Defects in Viral Growth and Protein Processing</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2008</span>;<span class="cit-vol">82</span>: <span class="cit-fpage">5999</span>–<span class="cit-lpage">6008</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.00203-08</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00203-08%26rft_id%253Dinfo%253Apmid%252F18385240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Mi8xMi81OTk5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDMvMjAyMC4wMy4xMy45OTA1OTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">140.</span><a class="rev-xref-ref" href="#xref-ref-140-1" title="View reference 140. in text" id="ref-140">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.140" data-doi="10.1093/emboj/cdf327"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anand</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Palm</span>  <span class="cit-name-given-names">GJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mesters</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Siddell</span>  <span class="cit-name-given-names">SG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hilgenfeld</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Structure of coronavirus main proteinase reveals combination of a chymotrypsin fold with an extra alpha-helical domain</span>. <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-pub-date">2002</span>;<span class="cit-vol">21</span>: <span class="cit-fpage">3213</span>–<span class="cit-lpage">24</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/emboj/cdf327</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DAnand%26rft.auinit1%253DK.%26rft.volume%253D21%26rft.issue%253D13%26rft.spage%253D3213%26rft.epage%253D3224%26rft.atitle%253DStructure%2Bof%2Bcoronavirus%2Bmain%2Bproteinase%2Breveals%2Bcombination%2Bof%2Ba%2Bchymotrypsin%2Bfold%2Bwith%2Ban%2Bextra%2B%257Balpha%257D-helical%2Bdomain%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252Fcdf327%26rft_id%253Dinfo%253Apmid%252F12093723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjEvMTMvMzIxMyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAzLzIwMjAuMDMuMTMuOTkwNTk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">141.</span><a class="rev-xref-ref" href="#xref-ref-141-1" title="View reference 141. in text" id="ref-141">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.141" data-doi="10.4161/auto.29309"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cottam</span>  <span class="cit-name-given-names">EM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whelband</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wileman</span>  <span class="cit-name-given-names">T.</span></span> <span class="cit-article-title">Coronavirus NSP6 restricts autophagosome expansion</span>. <abbr class="cit-jnl-abbrev">Autophagy.</abbr> <span class="cit-pub-date">2014</span>;<span class="cit-vol">10</span>: <span class="cit-fpage">1426</span>–<span class="cit-lpage">41</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.4161/auto.29309</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAutophagy.%26rft.volume%253D10%26rft.spage%253D1426%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fauto.29309%26rft_id%253Dinfo%253Apmid%252F24991833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/auto.29309&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24991833&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">142.</span><a class="rev-xref-ref" href="#xref-ref-142-1" title="View reference 142. in text" id="ref-142">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.142" data-doi="10.1093/nar/gkr893"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">te Velthuis</span>  <span class="cit-name-given-names">AJW</span></span>, <span class="cit-auth"><span class="cit-name-surname">van den Worm</span>  <span class="cit-name-given-names">SHE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ.</span></span> <span class="cit-article-title">The SARS-coronavirus nsp7+nsp8 complex is a unique multimeric RNA polymerase capable of both de novo initiation and primer extension</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-pub-date">2012</span>;<span class="cit-vol">40</span>: <span class="cit-fpage">1737</span>–<span class="cit-lpage">47</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gkr893</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr893%26rft_id%253Dinfo%253Apmid%252F22039154%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr893&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22039154&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000301069400034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">143.</span><a class="rev-xref-ref" href="#xref-ref-143-1" title="View reference 143. in text" id="ref-143">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.143" data-doi="10.1038/ng1622"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhai</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pang</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bartlam</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Insights into SARS-CoV transcription and replication from the structure of the nsp7-nsp8 hexadecamer</span>. <abbr class="cit-jnl-abbrev">Nat Struct Mol Biol.</abbr> <span class="cit-pub-date">2005</span>;<span class="cit-vol">12</span>: <span class="cit-fpage">980</span>–<span class="cit-lpage">6</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nsmb999</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DSharon%26rft.auinit1%253DI.%26rft.volume%253D12%26rft.issue%253D11%26rft.spage%253D980%26rft.epage%253D986%26rft.atitle%253DInsights%2Binto%2BSARS-CoV%2Btranscription%2Band%2Breplication%2Bfrom%2Bthe%2Bstructure%2Bof%2Bthe%2Bnsp7-nsp8%2Bhexadecamer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1622%26rft_id%253Dinfo%253Apmid%252F16228002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1622&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16228002&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233115100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">144.</span><a class="rev-xref-ref" href="#xref-ref-144-1" title="View reference 144. in text" id="ref-144">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.144" data-doi="10.1038/s41467-019-10280-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kirchdoerfer</span>  <span class="cit-name-given-names">RN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">AB.</span></span> <span class="cit-article-title">Structure of the SARS-CoV nsp12 polymerase bound to nsp7 and nsp8 co-factors</span>. <abbr class="cit-jnl-abbrev">Nat Commun.</abbr> <span class="cit-pub-date">2019</span>;<span class="cit-vol">10</span>: <span class="cit-fpage">2342</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41467-019-10280-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun.%26rft.volume%253D10%26rft.spage%253D2342%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-019-10280-3%26rft_id%253Dinfo%253Apmid%252F31138817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-019-10280-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=31138817&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">145.</span><a class="rev-xref-ref" href="#xref-ref-145-1" title="View reference 145. in text" id="ref-145">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.145" data-doi="10.1073/pnas.0307877101"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Egloff</span>  <span class="cit-name-given-names">M-P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ferron</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Campanacci</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Longhi</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rancurel</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dutartre</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">The severe acute respiratory syndrome-coronavirus replicative protein nsp9 is a single-stranded RNA-binding subunit unique in the RNA virus world</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A.</abbr> <span class="cit-pub-date">2004</span>;<span class="cit-vol">101</span>: <span class="cit-fpage">3792</span>–<span class="cit-lpage">6</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.0307877101</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0307877101%26rft_id%253Dinfo%253Apmid%252F15007178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzExLzM3OTIiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wMy8yMDIwLjAzLjEzLjk5MDU5OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">146.</span><a class="rev-xref-ref" href="#xref-ref-146-1" title="View reference 146. in text" id="ref-146">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.146" data-doi="10.1016/j.jmb.2008.07.071"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ponnusamy</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moll</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weimar</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mesters</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hilgenfeld</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Variable oligomerization modes in coronavirus non-structural protein 9</span>. <abbr class="cit-jnl-abbrev">J Mol Biol.</abbr> <span class="cit-pub-date">2008</span>;<span class="cit-vol">383</span>: <span class="cit-fpage">1081</span>–<span class="cit-lpage">96</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.jmb.2008.07.071</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DPonnusamy%26rft.auinit1%253DR.%26rft.volume%253D383%26rft.issue%253D5%26rft.spage%253D1081%26rft.epage%253D1096%26rft.atitle%253DVariable%2Boligomerization%2Bmodes%2Bin%2Bcoronavirus%2Bnon-structural%2Bprotein%2B9.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2008.07.071%26rft_id%253Dinfo%253Apmid%252F18694760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2008.07.071&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18694760&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">147.</span><a class="rev-xref-ref" href="#xref-ref-147-1" title="View reference 147. in text" id="ref-147">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.147" data-doi="10.1128/JVI.01505-08"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Miknis</span>  <span class="cit-name-given-names">ZJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Donaldson</span>  <span class="cit-name-given-names">EF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Umland</span>  <span class="cit-name-given-names">TC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rimmer</span>  <span class="cit-name-given-names">RA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>  <span class="cit-name-given-names">RS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schultz</span>  <span class="cit-name-given-names">LW.</span></span> <span class="cit-article-title">Severe Acute Respiratory Syndrome Coronavirus nsp9 Dimerization Is Essential for Efficient Viral Growth</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2009</span>;<span class="cit-vol">83</span>: <span class="cit-fpage">3007</span>–<span class="cit-lpage">3018</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.01505-08</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01505-08%26rft_id%253Dinfo%253Apmid%252F19153232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgzLzcvMzAwNyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAzLzIwMjAuMDMuMTMuOTkwNTk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">148.</span><a class="rev-xref-ref" href="#xref-ref-148-1" title="View reference 148. in text" id="ref-148">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.148" data-doi="10.1073/pnas.1201130109"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bouvet</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Imbert</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Subissi</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gluais</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Canard</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Decroly</span>  <span class="cit-name-given-names">E.</span></span> <span class="cit-article-title">RNA 3’-end mismatch excision by the severe acute respiratory syndrome coronavirus nonstructural protein nsp10/nsp14 exoribonuclease complex</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A.</abbr> <span class="cit-pub-date">2012</span>;<span class="cit-vol">109</span>: <span class="cit-fpage">9372</span>–<span class="cit-lpage">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1201130109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1201130109%26rft_id%253Dinfo%253Apmid%252F22635272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA5LzI0LzkzNzIiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wMy8yMDIwLjAzLjEzLjk5MDU5OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">149.</span><a class="rev-xref-ref" href="#xref-ref-149-1" title="View reference 149. in text" id="ref-149">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.149" data-doi="10.1371/journal.ppat.1000863"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bouvet</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Debarnot</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Imbert</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Selisko</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Canard</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">In vitro reconstitution of SARS-coronavirus mRNA cap methylation</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-pub-date">2010</span>;<span class="cit-vol">6</span>: <span class="cit-fpage">e1000863</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.ppat.1000863</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DBouvet%26rft.auinit1%253DM.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253De1000863%26rft.epage%253De1000863%26rft.atitle%253DIn%2Bvitro%2Breconstitution%2Bof%2BSARS-coronavirus%2BmRNA%2Bcap%2Bmethylation.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1000863%26rft_id%253Dinfo%253Apmid%252F20421945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1000863&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20421945&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">150.</span><a class="rev-xref-ref" href="#xref-ref-150-1" title="View reference 150. in text" id="ref-150">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.150" data-doi="10.1073/pnas.1508686112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Structural basis and functional analysis of the SARS coronavirus nsp14-nsp10 complex</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A.</abbr> <span class="cit-pub-date">2015</span>;<span class="cit-vol">112</span>: <span class="cit-fpage">9436</span>–<span class="cit-lpage">41</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1508686112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1508686112%26rft_id%253Dinfo%253Apmid%252F26159422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzMwLzk0MzYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wMy8yMDIwLjAzLjEzLjk5MDU5OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">151.</span><a class="rev-xref-ref" href="#xref-ref-151-1" title="View reference 151. in text" id="ref-151">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.151" data-doi="10.1007/s00705-012-1404-x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ahn</span>  <span class="cit-name-given-names">D-G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">J-K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">DR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oh</span>  <span class="cit-name-given-names">J-W.</span></span> <span class="cit-article-title">Biochemical characterization of a recombinant SARS coronavirus nsp12 RNA-dependent RNA polymerase capable of copying viral RNA templates</span>. <abbr class="cit-jnl-abbrev">Arch Virol.</abbr> <span class="cit-pub-date">2012</span>;<span class="cit-vol">157</span>: <span class="cit-fpage">2095</span>–<span class="cit-lpage">104</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/s00705-012-1404-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArchives%2Bof%2Bvirology%26rft.stitle%253DArch%2BVirol%26rft.aulast%253DAhn%26rft.auinit1%253DD.%2BG.%26rft.volume%253D157%26rft.issue%253D11%26rft.spage%253D2095%26rft.epage%253D2104%26rft.atitle%253DBiochemical%2Bcharacterization%2Bof%2Ba%2Brecombinant%2BSARS%2Bcoronavirus%2Bnsp12%2BRNA-dependent%2BRNA%2Bpolymerase%2Bcapable%2Bof%2Bcopying%2Bviral%2BRNA%2Btemplates.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00705-012-1404-x%26rft_id%253Dinfo%253Apmid%252F22791111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00705-012-1404-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22791111&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">152.</span><a class="rev-xref-ref" href="#xref-ref-152-1" title="View reference 152. in text" id="ref-152">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.152" data-doi="10.1371/journal.pone.0036521"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Adedeji</span>  <span class="cit-name-given-names">AO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marchand</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Te Velthuis</span>  <span class="cit-name-given-names">AJW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weiss</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eoff</span>  <span class="cit-name-given-names">RL</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Mechanism of nucleic acid unwinding by SARS-CoV helicase</span>. <abbr class="cit-jnl-abbrev">PLoS One.</abbr> <span class="cit-pub-date">2012</span>;<span class="cit-vol">7</span>: <span class="cit-fpage">e36521</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0036521</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DAdedeji%26rft.auinit1%253DA.%2BO.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De36521%26rft.epage%253De36521%26rft.atitle%253DMechanism%2Bof%2Bnucleic%2Bacid%2Bunwinding%2Bby%2BSARS-CoV%2Bhelicase.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0036521%26rft_id%253Dinfo%253Apmid%252F22615777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0036521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22615777&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">153.</span><a class="rev-xref-ref" href="#xref-ref-153-1" title="View reference 153. in text" id="ref-153">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.153" data-doi="10.1093/nar/gkz409"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Delicate structural coordination of the Severe Acute Respiratory Syndrome coronavirus Nsp13 upon ATP hydrolysis</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-pub-date">2019</span>;<span class="cit-vol">47</span>: <span class="cit-fpage">6538</span>–<span class="cit-lpage">6550</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gkz409</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D47%26rft.spage%253D6538%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkz409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkz409&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">154.</span><a class="rev-xref-ref" href="#xref-ref-154-1" title="View reference 154. in text" id="ref-154">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.154" data-doi="10.1073/pnas.0508200103"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Minskaia</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hertzig</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Campanacci</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cambillau</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Canard</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Discovery of an RNA virus 3’-&gt;5’ exoribonuclease that is critically involved in coronavirus RNA synthesis</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A.</abbr> <span class="cit-pub-date">2006</span>;<span class="cit-vol">103</span>: <span class="cit-fpage">5108</span>–<span class="cit-lpage">13</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.0508200103</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0508200103%26rft_id%253Dinfo%253Apmid%252F16549795%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAzLzEzLzUxMDgiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wMy8yMDIwLjAzLjEzLjk5MDU5OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">155.</span><a class="rev-xref-ref" href="#xref-ref-155-1" title="View reference 155. in text" id="ref-155">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.155" data-doi="10.1073/pnas.0403127101"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ivanov</span>  <span class="cit-name-given-names">KA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hertzig</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rozanov</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bayer</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thiel</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Major genetic marker of nidoviruses encodes a replicative endoribonuclease</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A.</abbr> <span class="cit-pub-date">2004</span>;<span class="cit-vol">101</span>: <span class="cit-fpage">12694</span>–<span class="cit-lpage">12699</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.0403127101</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0403127101%26rft_id%253Dinfo%253Apmid%252F15304651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzM0LzEyNjk0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDMvMjAyMC4wMy4xMy45OTA1OTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">156.</span><a class="rev-xref-ref" href="#xref-ref-156-1" title="View reference 156. in text" id="ref-156">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.156" data-doi="10.1073/pnas.0601708103"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ricagno</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Egloff</span>  <span class="cit-name-given-names">M-P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ulferts</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coutard</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nurizzo</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Campanacci</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Crystal structure and mechanistic determinants of SARS coronavirus nonstructural protein 15 define an endoribonuclease family</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A.</abbr> <span class="cit-pub-date">2006</span>;<span class="cit-vol">103</span>: <span class="cit-fpage">11892</span>–<span class="cit-lpage">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.0601708103</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0601708103%26rft_id%253Dinfo%253Apmid%252F16882730%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzMyLzExODkyIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDMvMjAyMC4wMy4xMy45OTA1OTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">157.</span><a class="rev-xref-ref" href="#xref-ref-157-1" title="View reference 157. in text" id="ref-157">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.157" data-doi="10.1128/JVI.00407-08"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Decroly</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Imbert</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coutard</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bouvet</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Selisko</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alvarez</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Coronavirus Nonstructural Protein 16 Is a Cap-0 Binding Enzyme Possessing (Nucleoside-2’O)-Methyltransferase Activity</span>. <abbr class="cit-jnl-abbrev">J Virol.</abbr> <span class="cit-pub-date">2008</span>;<span class="cit-vol">82</span>: <span class="cit-fpage">8071</span>–<span class="cit-lpage">8084</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.00407-08</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00407-08%26rft_id%253Dinfo%253Apmid%252F18417574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Mi8xNi84MDcxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDMvMjAyMC4wMy4xMy45OTA1OTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">158.</span><a class="rev-xref-ref" href="#xref-ref-158-1" title="View reference 158. in text" id="ref-158">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990598v2.158" data-doi="10.1371/journal.ppat.1002294"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ke</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Biochemical and structural insights into the mechanisms of SARS coronavirus RNA ribose 2’-O-methylation by nsp16/nsp10 protein complex</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-pub-date">2011</span>;<span class="cit-vol">7</span>: <span class="cit-fpage">e1002294</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.ppat.1002294</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DChen%26rft.auinit1%253DY.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De1002294%26rft.epage%253De1002294%26rft.atitle%253DBiochemical%2Band%2Bstructural%2Binsights%2Binto%2Bthe%2Bmechanisms%2Bof%2BSARS%2Bcoronavirus%2BRNA%2Bribose%2B2%2527-O-methylation%2Bby%2Bnsp16%252Fnsp10%2Bprotein%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002294%26rft_id%253Dinfo%253Apmid%252F22022266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002294&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22022266&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.13.990598.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.13.990598v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.13.990598v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.03.05.976167v2" title="Direct RNA sequencing and early evolution of SARS-CoV-2" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.04.03.023416v1" title="Pex11β knockdown decreases peroxisome abundance and reverses the inhibitory effect of palmitate on pancreatic beta-cell function" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;April 03, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.13.990598v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.03.13.990598v2.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.13.990598v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.13.990598v2" class="active" data-icon-position="" data-hide-link-title="0">When Darkness Becomes a Ray of Light in the Dark Times: Understanding the COVID-19 via the Comparative Analysis of the Dark Proteomes of SARS-CoV-2, Human SARS and Bat SARS-Like Coronaviruses</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1225706" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-JmaXoePPcr8GUPDv0LxAbCdMuulAnm10eUtmOvd2538" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1225706" id="node1225706--2" data-pisa="biorxiv;2020.03.13.990598v2" data-pisa-master="biorxiv;2020.03.13.990598" data-seqnum="1225706" data-apath="/biorxiv/early/2020/04/03/2020.03.13.990598.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">When Darkness Becomes a Ray of Light in the Dark Times: Understanding the COVID-19 via the Comparative Analysis of the Dark Proteomes of SARS-CoV-2, Human SARS and Bat SARS-Like Coronaviruses</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Rajanish</span> <span class="nlm-surname">Giri</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Taniya</span> <span class="nlm-surname">Bhardwaj</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Meenakshi</span> <span class="nlm-surname">Shegane</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Bhuvaneshwari R.</span> <span class="nlm-surname">Gehi</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Prateek</span> <span class="nlm-surname">Kumar</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Kundlik</span> <span class="nlm-surname">Gadhave</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Christopher J.</span> <span class="nlm-surname">Oldfield</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Vladimir N.</span> <span class="nlm-surname">Uversky</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.13.990598; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.13.990598 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.13.990598v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.990598v2&amp;title=When%20Darkness%20Becomes%20a%20Ray%20of%20Light%20in%20the%20Dark%20Times%3A%20Understanding%20the%20COVID-19%20via%20the%20Comparative%20Analysis%20of%20the%20Dark%20Proteomes%20of%20SARS-CoV-2%2C%20Human%20SARS%20and%20Bat%20SARS-Like%20Coronaviruses" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.990598v2&amp;title=When%20Darkness%20Becomes%20a%20Ray%20of%20Light%20in%20the%20Dark%20Times%3A%20Understanding%20the%20COVID-19%20via%20the%20Comparative%20Analysis%20of%20the%20Dark%20Proteomes%20of%20SARS-CoV-2%2C%20Human%20SARS%20and%20Bat%20SARS-Like%20Coronaviruses" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.990598v2&amp;text=When%20Darkness%20Becomes%20a%20Ray%20of%20Light%20in%20the%20Dark%20Times%3A%20Understanding%20the%20COVID-19%20via%20the%20Comparative%20Analysis%20of%20the%20Dark%20Proteomes%20of%20SARS-CoV-2%2C%20Human%20SARS%20and%20Bat%20SARS-Like%20Coronaviruses" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.990598v2&amp;title=When%20Darkness%20Becomes%20a%20Ray%20of%20Light%20in%20the%20Dark%20Times%3A%20Understanding%20the%20COVID-19%20via%20the%20Comparative%20Analysis%20of%20the%20Dark%20Proteomes%20of%20SARS-CoV-2%2C%20Human%20SARS%20and%20Bat%20SARS-Like%20Coronaviruses" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.990598v2&amp;t=When%20Darkness%20Becomes%20a%20Ray%20of%20Light%20in%20the%20Dark%20Times%3A%20Understanding%20the%20COVID-19%20via%20the%20Comparative%20Analysis%20of%20the%20Dark%20Proteomes%20of%20SARS-CoV-2%2C%20Human%20SARS%20and%20Bat%20SARS-Like%20Coronaviruses" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.990598v2&amp;title=When%20Darkness%20Becomes%20a%20Ray%20of%20Light%20in%20the%20Dark%20Times%3A%20Understanding%20the%20COVID-19%20via%20the%20Comparative%20Analysis%20of%20the%20Dark%20Proteomes%20of%20SARS-CoV-2%2C%20Human%20SARS%20and%20Bat%20SARS-Like%20Coronaviruses" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.990598v2&amp;title=When%20Darkness%20Becomes%20a%20Ray%20of%20Light%20in%20the%20Dark%20Times%3A%20Understanding%20the%20COVID-19%20via%20the%20Comparative%20Analysis%20of%20the%20Dark%20Proteomes%20of%20SARS-CoV-2%2C%20Human%20SARS%20and%20Bat%20SARS-Like%20Coronaviruses" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1225706" id="node1225706--3" data-pisa="biorxiv;2020.03.13.990598v2" data-pisa-master="biorxiv;2020.03.13.990598" data-seqnum="1225706" data-apath="/biorxiv/early/2020/04/03/2020.03.13.990598.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">When Darkness Becomes a Ray of Light in the Dark Times: Understanding the COVID-19 via the Comparative Analysis of the Dark Proteomes of SARS-CoV-2, Human SARS and Bat SARS-Like Coronaviruses</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Rajanish</span> <span class="nlm-surname">Giri</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Taniya</span> <span class="nlm-surname">Bhardwaj</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Meenakshi</span> <span class="nlm-surname">Shegane</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Bhuvaneshwari R.</span> <span class="nlm-surname">Gehi</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Prateek</span> <span class="nlm-surname">Kumar</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Kundlik</span> <span class="nlm-surname">Gadhave</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Christopher J.</span> <span class="nlm-surname">Oldfield</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Vladimir N.</span> <span class="nlm-surname">Uversky</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.13.990598; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.13.990598 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=When%20Darkness%20Becomes%20a%20Ray%20of%20Light%20in%20the%20Dark%20Times%3A%20Understanding%20the%20COVID-19%20via%20the%20Comparative%20Analysis%20of%20the%20Dark%20Proteomes%20of%20SARS-CoV-2%2C%20Human%20SARS%20and%20Bat%20SARS-Like%20Coronaviruses&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Giri%2C+Rajanish&amp;rft.au=Bhardwaj%2C+Taniya&amp;rft.au=Shegane%2C+Meenakshi&amp;rft.au=Gehi%2C+Bhuvaneshwari+R.&amp;rft.au=Kumar%2C+Prateek&amp;rft.au=Gadhave%2C+Kundlik&amp;rft.au=Oldfield%2C+Christopher+J.&amp;rft.au=Uversky%2C+Vladimir+N."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1225706/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1225706/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1225706/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1225706/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1225706/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1225706/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1225706/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1225706/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1225706/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1225706/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1225706/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1225706/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.990598v2&amp;count=horizontal&amp;via=&amp;text=When%20Darkness%20Becomes%20a%20Ray%20of%20Light%20in%20the%20Dark%20Times%3A%20Understanding%20the%20COVID-19%20via%20the%20Comparative%20Analysis%20of%20the%20Dark%20Proteomes%20of%20SARS-CoV-2%2C%20Human%20SARS%20and%20Bat%20SARS-Like%20Coronaviruses&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.990598v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.990598v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.13.990598v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">Bioinformatics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
