<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/04/06/2020.03.20.001008.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.03.20.001008v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="DIRECT RT-qPCR DETECTION OF SARS-CoV-2 RNA FROM PATIENT NASOPHARYNGEAL SWABS WITHOUT AN RNA EXTRACTION STEP"/>
  <meta name="DC.Identifier" content="10.1101/2020.03.20.001008"/>
  <meta name="DC.Date" content="2020-04-06"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="The ongoing COVID-19 pandemic has caused an unprecedented need for rapid diagnostic testing. The Centers for Disease Control and Prevention (CDC) and the World Health Organization (WHO) recommend a standard assay that includes an RNA extraction step from a nasopharyngeal (NP) swab followed by reverse transcription-quantitative polymerase chain reaction (RT-qPCR) to detect the purified SARS-CoV-2 RNA. The current global shortage of RNA extraction kits has caused a severe bottleneck to COVID-19 testing. We hypothesized that SARS-CoV-2 RNA could be detected from NP samples via a direct RT-qPCR assay that omits the RNA extraction step altogether, and tested this hypothesis on a series of blinded clinical samples. The direct RT-qPCR approach correctly identified 92% of NP samples (n = 155) demonstrated to be positive for SARS-CoV-2 RNA by traditional clinical diagnostic RT-qPCR that included an RNA extraction. Thus, direct RT-qPCR could be a front-line approach to identify the substantial majority of COVID-19 patients, reserving a repeat test with RNA extraction for those individuals with high suspicion of infection but an initial negative result. This strategy would drastically ease supply chokepoints of COVID-19 testing and should be applicable throughout the world."/>
  <meta name="DC.Contributor" content="Emily A. Bruce"/>
  <meta name="DC.Contributor" content="Meei-Li Huang"/>
  <meta name="DC.Contributor" content="Garrett A. Perchetti"/>
  <meta name="DC.Contributor" content="Scott Tighe"/>
  <meta name="DC.Contributor" content="Pheobe Laaguiby"/>
  <meta name="DC.Contributor" content="Jessica J. Hoffman"/>
  <meta name="DC.Contributor" content="Diana L. Gerrard"/>
  <meta name="DC.Contributor" content="Arun K. Nalla"/>
  <meta name="DC.Contributor" content="Yulun Wei"/>
  <meta name="DC.Contributor" content="Alexander L. Greninger"/>
  <meta name="DC.Contributor" content="Sean A. Diehl"/>
  <meta name="DC.Contributor" content="David J. Shirley"/>
  <meta name="DC.Contributor" content="Debra G. B. Leonard"/>
  <meta name="DC.Contributor" content="Christopher D. Huston"/>
  <meta name="DC.Contributor" content="Beth D. Kirkpatrick"/>
  <meta name="DC.Contributor" content="Julie A. Dragon"/>
  <meta name="DC.Contributor" content="Jessica W. Crothers"/>
  <meta name="DC.Contributor" content="Keith R. Jerome"/>
  <meta name="DC.Contributor" content="Jason W. Botten"/>
  <meta name="article:published_time" content="2020-04-06"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="DIRECT RT-qPCR DETECTION OF SARS-CoV-2 RNA FROM PATIENT NASOPHARYNGEAL SWABS WITHOUT AN RNA EXTRACTION STEP"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;The ongoing COVID-19 pandemic has caused an unprecedented need for rapid diagnostic testing. The Centers for Disease Control and Prevention (CDC) and the World Health Organization (WHO) recommend a standard assay that includes an RNA extraction step from a nasopharyngeal (NP) swab followed by reverse transcription-quantitative polymerase chain reaction (RT-qPCR) to detect the purified SARS-CoV-2 RNA. The current global shortage of RNA extraction kits has caused a severe bottleneck to COVID-19 testing. We hypothesized that SARS-CoV-2 RNA could be detected from NP samples via a direct RT-qPCR assay that omits the RNA extraction step altogether, and tested this hypothesis on a series of blinded clinical samples. The direct RT-qPCR approach correctly identified 92% of NP samples (n = 155) demonstrated to be positive for SARS-CoV-2 RNA by traditional clinical diagnostic RT-qPCR that included an RNA extraction. Thus, direct RT-qPCR could be a front-line approach to identify the substantial majority of COVID-19 patients, reserving a repeat test with RNA extraction for those individuals with high suspicion of infection but an initial negative result. This strategy would drastically ease supply chokepoints of COVID-19 testing and should be applicable throughout the world.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.03.20.001008v2"/>
  <meta name="citation_id" content="2020.03.20.001008v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.20.001008v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.20.001008v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.20.001008v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.20.001008.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.03.20.001008"/>
  <meta name="citation_num_pages" content="14"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.03.20.001008"/>
  <meta name="citation_author" content="Emily A. Bruce"/>
  <meta name="citation_author_institution" content="Department of Medicine, Division of Immunobiology, Robert Larner, M.D. College of Medicine, University of Vermont"/>
  <meta name="citation_author_email" content="Emily.bruce@med.uvm.edu"/>
  <meta name="citation_author_email" content="kjerome@fredhutch.org"/>
  <meta name="citation_author_email" content="Jason.botten@med.uvm.edu"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8391-370X"/>
  <meta name="citation_author" content="Meei-Li Huang"/>
  <meta name="citation_author_institution" content="Virology Division, Department of Laboratory Medicine, University of Washington"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2058-2776"/>
  <meta name="citation_author" content="Garrett A. Perchetti"/>
  <meta name="citation_author_institution" content="Virology Division, Department of Laboratory Medicine, University of Washington"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3786-0509"/>
  <meta name="citation_author" content="Scott Tighe"/>
  <meta name="citation_author_institution" content="Vermont Integrative Genomics Resource, Robert Larner, M.D. College of Medicine, University of Vermont"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3988-0741"/>
  <meta name="citation_author" content="Pheobe Laaguiby"/>
  <meta name="citation_author_institution" content="Vermont Integrative Genomics Resource, Robert Larner, M.D. College of Medicine, University of Vermont"/>
  <meta name="citation_author" content="Jessica J. Hoffman"/>
  <meta name="citation_author_institution" content="Vermont Integrative Genomics Resource, Robert Larner, M.D. College of Medicine, University of Vermont"/>
  <meta name="citation_author" content="Diana L. Gerrard"/>
  <meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, University of Vermont Medical Center"/>
  <meta name="citation_author" content="Arun K. Nalla"/>
  <meta name="citation_author_institution" content="Virology Division, Department of Laboratory Medicine, University of Washington"/>
  <meta name="citation_author" content="Yulun Wei"/>
  <meta name="citation_author_institution" content="Virology Division, Department of Laboratory Medicine, University of Washington"/>
  <meta name="citation_author" content="Alexander L. Greninger"/>
  <meta name="citation_author_institution" content="Virology Division, Department of Laboratory Medicine, University of Washington"/>
  <meta name="citation_author_institution" content="Vaccine and Infectious Disease Division, Fred Hutchinson Cancer Research Center"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7443-0527"/>
  <meta name="citation_author" content="Sean A. Diehl"/>
  <meta name="citation_author_institution" content="Department of Microbiology and Molecular Genetics, Robert Larner, M.D. College of Medicine, University of Vermont"/>
  <meta name="citation_author_institution" content="Vaccine Testing Center, Robert Larner, M.D. College of Medicine, University of Vermont"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9700-235X"/>
  <meta name="citation_author" content="David J. Shirley"/>
  <meta name="citation_author_institution" content="IXIS LLC, Data Science Division"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4529-0403"/>
  <meta name="citation_author" content="Debra G. B. Leonard"/>
  <meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, Robert Larner, M.D. College of Medicine, University of Vermont and the University of Vermont Health Network"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1431-4935"/>
  <meta name="citation_author" content="Christopher D. Huston"/>
  <meta name="citation_author_institution" content="Department of Microbiology and Molecular Genetics, Robert Larner, M.D. College of Medicine, University of Vermont"/>
  <meta name="citation_author_institution" content="Department of Medicine, Division of Infectious Disease, University of Vermont Medical Center"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5393-139X"/>
  <meta name="citation_author" content="Beth D. Kirkpatrick"/>
  <meta name="citation_author_institution" content="Department of Microbiology and Molecular Genetics, Robert Larner, M.D. College of Medicine, University of Vermont"/>
  <meta name="citation_author_institution" content="Vaccine Testing Center, Robert Larner, M.D. College of Medicine, University of Vermont"/>
  <meta name="citation_author_institution" content="Department of Medicine, Division of Infectious Disease, University of Vermont Medical Center"/>
  <meta name="citation_author" content="Julie A. Dragon"/>
  <meta name="citation_author_institution" content="Vermont Integrative Genomics Resource, Robert Larner, M.D. College of Medicine, University of Vermont"/>
  <meta name="citation_author_institution" content="Department of Microbiology and Molecular Genetics, Robert Larner, M.D. College of Medicine, University of Vermont"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8031-5381"/>
  <meta name="citation_author" content="Jessica W. Crothers"/>
  <meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, Robert Larner, M.D. College of Medicine, University of Vermont and the University of Vermont Health Network"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4594-5051"/>
  <meta name="citation_author" content="Keith R. Jerome"/>
  <meta name="citation_author_institution" content="Virology Division, Department of Laboratory Medicine, University of Washington"/>
  <meta name="citation_author_institution" content="Vaccine and Infectious Disease Division, Fred Hutchinson Cancer Research Center"/>
  <meta name="citation_author_email" content="Emily.bruce@med.uvm.edu"/>
  <meta name="citation_author_email" content="kjerome@fredhutch.org"/>
  <meta name="citation_author_email" content="Jason.botten@med.uvm.edu"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8212-3789"/>
  <meta name="citation_author" content="Jason W. Botten"/>
  <meta name="citation_author_institution" content="Department of Medicine, Division of Immunobiology, Robert Larner, M.D. College of Medicine, University of Vermont"/>
  <meta name="citation_author_institution" content="Department of Microbiology and Molecular Genetics, Robert Larner, M.D. College of Medicine, University of Vermont"/>
  <meta name="citation_author_institution" content="Vaccine Testing Center, Robert Larner, M.D. College of Medicine, University of Vermont"/>
  <meta name="citation_author_email" content="Emily.bruce@med.uvm.edu"/>
  <meta name="citation_author_email" content="kjerome@fredhutch.org"/>
  <meta name="citation_author_email" content="Jason.botten@med.uvm.edu"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6839-0306"/>
  <meta name="citation_reference" content="Centers for Disease Control and Prevention &amp;; Division of Viral Diseases. CDC 2019-Novel Coronavirus (2019-nCoV) Real-Time RT-PCR Diagnostic Panel. (https://www.fda.gov/media/134922/download)."/>
  <meta name="citation_reference" content="Chu, D.K.W., et al. Molecular Diagnosis of a Novel Coronavirus (2019-nCoV) Causing an Outbreak of Pneumonia. Clin Chem (2020)."/>
  <meta name="citation_reference" content="Corman, V.M., et al. Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR. Euro Surveill 25 (2020)."/>
  <meta name="citation_reference" content="Bruce, E.A., et al. RT-qPCR DETECTION OF SARS-CoV-2 RNA FROM PATIENT NASOPHARYNGEAL SWAB USING QIAGEN RNEASY KITS OR DIRECTLY VIA OMISSION OF AN RNA EXTRACTION STEP. (Preprint at https://www.biorxiv.org/content/10.1101/2020.03.20.001008v1, 2020)."/>
  <meta name="citation_reference" content="Nelson, A.C., et al. Analytical Validation of a COVID-19 qRT-PCR Detection Assay Using a 384-well Format and Three Extraction Methods. (Preprint at https://www.biorxiv.org/content/10.1101/2020.04.02.022186v1, 2020)."/>
  <meta name="citation_reference" content="Beltran-Pavez, C., et al. SARS-CoV-2 detection from nasopharyngeal swab samples without RNA extraction. (Preprint at https://www.biorxiv.org/content/10.1101/2020.03.28.013508v1, 2020)."/>
  <meta name="citation_reference" content="Fomsgaard, A.S. &amp;; Rosenstierne, M.W. An alternative workflow for molecular detection of SARS-CoV-2 - escape from the NA extraction kit-shortage. (Preprint at https://www.medrxiv.org/content/10.1101/2020.03.27.20044495v1, 2020)."/>
  <meta name="citation_reference" content="Casto, A.M., et al. Comparative Performance of SARS-CoV-2 Detection Assays using Seven Different Primer/Probe Sets and One Assay Kit. (Preprint at https://www.medrxiv.org/content/10.1101/2020.03.13.20035618v1 2020)."/>
  <meta name="twitter:title" content="DIRECT RT-qPCR DETECTION OF SARS-CoV-2 RNA FROM PATIENT NASOPHARYNGEAL SWABS WITHOUT AN RNA EXTRACTION STEP"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="The ongoing COVID-19 pandemic has caused an unprecedented need for rapid diagnostic testing. The Centers for Disease Control and Prevention (CDC) and the World Health Organization (WHO) recommend a standard assay that includes an RNA extraction step from a nasopharyngeal (NP) swab followed by reverse transcription-quantitative polymerase chain reaction (RT-qPCR) to detect the purified SARS-CoV-2 RNA. The current global shortage of RNA extraction kits has caused a severe bottleneck to COVID-19 testing. We hypothesized that SARS-CoV-2 RNA could be detected from NP samples via a direct RT-qPCR assay that omits the RNA extraction step altogether, and tested this hypothesis on a series of blinded clinical samples. The direct RT-qPCR approach correctly identified 92% of NP samples (n = 155) demonstrated to be positive for SARS-CoV-2 RNA by traditional clinical diagnostic RT-qPCR that included an RNA extraction. Thus, direct RT-qPCR could be a front-line approach to identify the substantial majority of COVID-19 patients, reserving a repeat test with RNA extraction for those individuals with high suspicion of infection but an initial negative result. This strategy would drastically ease supply chokepoints of COVID-19 testing and should be applicable throughout the world."/>
  <meta name="og-title" property="og:title" content="DIRECT RT-qPCR DETECTION OF SARS-CoV-2 RNA FROM PATIENT NASOPHARYNGEAL SWABS WITHOUT AN RNA EXTRACTION STEP"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.20.001008v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="The ongoing COVID-19 pandemic has caused an unprecedented need for rapid diagnostic testing. The Centers for Disease Control and Prevention (CDC) and the World Health Organization (WHO) recommend a standard assay that includes an RNA extraction step from a nasopharyngeal (NP) swab followed by reverse transcription-quantitative polymerase chain reaction (RT-qPCR) to detect the purified SARS-CoV-2 RNA. The current global shortage of RNA extraction kits has caused a severe bottleneck to COVID-19 testing. We hypothesized that SARS-CoV-2 RNA could be detected from NP samples via a direct RT-qPCR assay that omits the RNA extraction step altogether, and tested this hypothesis on a series of blinded clinical samples. The direct RT-qPCR approach correctly identified 92% of NP samples (n = 155) demonstrated to be positive for SARS-CoV-2 RNA by traditional clinical diagnostic RT-qPCR that included an RNA extraction. Thus, direct RT-qPCR could be a front-line approach to identify the substantial majority of COVID-19 patients, reserving a repeat test with RNA extraction for those individuals with high suspicion of infection but an initial negative result. This strategy would drastically ease supply chokepoints of COVID-19 testing and should be applicable throughout the world."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-04-06"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>DIRECT RT-qPCR DETECTION OF SARS-CoV-2 RNA FROM PATIENT NASOPHARYNGEAL SWABS WITHOUT AN RNA EXTRACTION STEP | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1231882 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">ABSTRACT</h2>
    <p id="p-4">The ongoing COVID-19 pandemic has caused an unprecedented need for rapid diagnostic testing. The Centers for Disease Control and Prevention (CDC) and the World Health Organization (WHO) recommend a standard assay that includes an RNA extraction step from a nasopharyngeal (NP) swab followed by reverse transcription-quantitative polymerase chain reaction (RT-qPCR) to detect the purified SARS-CoV-2 RNA. The current global shortage of RNA extraction kits has caused a severe bottleneck to COVID-19 testing. We hypothesized that SARS-CoV-2 RNA could be detected from NP samples via a direct RT-qPCR assay that omits the RNA extraction step altogether, and tested this hypothesis on a series of blinded clinical samples. The direct RT-qPCR approach correctly identified 92% of NP samples (n = 155) demonstrated to be positive for SARS-CoV-2 RNA by traditional clinical diagnostic RT-qPCR that included an RNA extraction. Thus, direct RT-qPCR could be a front-line approach to identify the substantial majority of COVID-19 patients, reserving a repeat test with RNA extraction for those individuals with high suspicion of infection but an initial negative result. This strategy would drastically ease supply chokepoints of COVID-19 testing and should be applicable throughout the world.</p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">MAIN</h2>
    <p id="p-6">The ongoing COVID-19 pandemic has put exceptional strain on public health laboratories, hospital laboratories, and commercial laboratories as they attempt to keep up with demands for SARS-CoV-2 testing. The current diagnostic testing methods recommended by the Centers for Disease Control and Prevention (CDC) in the United States and the World Health Organization (WHO) are traditional RT-qPCR assays that require two steps: first, an RNA extraction from patient nasopharyngeal (NP) swab material, followed by RT-qPCR amplification of the extracted RNA to detect viral RNA 
     <sup>
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>-
      <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>
     </sup>. The major bottleneck to widespread SARS-CoV-2 testing lies at the RNA extraction step. The simplest manual kit (the Qiagen Viral RNA Mini) is no longer available, and reagents and supplies for the larger automated instruments are extremely limited with uncertain supply chains. While substitution of other RNA extraction kits 
     <sup>
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>,
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>
     </sup> is possible, they too are in limited supply. The current bottleneck is not simply the availability of RNA extraction kits, but also the cost of the extraction assay, the labor and time required to perform it, and the fact that it is rate limiting compared to the downstream RT-qPCR analysis. To address this issue, we tested the unconventional approach of skipping the RNA extraction step altogether and directly loading patient swab material into the RT-qPCR mix. Herein, we report that this approach (which we refer to hereafter as “direct RT-qPCR”) correctly identified 92% of samples (n =155) previously shown to be positive for SARS-CoV-2 RNA by conventional RT-qPCR featuring an RNA extraction. Thus, our results suggest that this streamlined assay could greatly alleviate constraints to COVID-19 testing in many regions of the world.
    </p>
    <p id="p-7">We initially conducted a pilot experiment using NP swabs from two COVID-19 patients who had previously been verified for SARS-CoV-2 infection by the Vermont Department of Health Laboratory (VDHL) using the CDC’s recommended RT-qPCR test. Both patient samples, which were originally collected as NP swabs in 3 mL of M6 viral transport medium (termed diluent hereafter), were pooled (equi-volume). RNA was extracted from 140 µL of the pooled sample using the QIAamp Viral RNA Mini kit, and purified RNA representing 11.3 µL of the original swab diluent was detected as positive via standard RT-qPCR using the CDC 2019-nCoV N3 primer/probe set, with a CT of 18.7. In parallel, we added 7 µL of the pooled COVID-19 patient NP swab diluent directly to the RT-qPCR reaction, and found that SARS-CoV-2 RNA was successfully detected in the absence of an RNA extraction step. Compared to the same pooled NP swab diluent extracted with the QIAamp Viral RNA Mini kit (after adjusting for the quantity of swab diluent added in each case), adding the NP diluent directly into the RT-qPCR reaction resulted in an ∼ 4 CT drop in sensitivity (
     <strong>
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1a</a>
     </strong>). Preheating the NP diluent for five minutes at 70°C prior to RT-qPCR had no impact on viral RNA detection. These results provided proof-of-principle that successful detection of SARS-CoV-2 RNA from an NP swab sample by RT-qPCR could be done in the absence of an RNA extraction step.
    </p>
    <div id="F1" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.20.001008/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="SARS-CoV-2 RNA can be detected from COVID-19 patient nasopharyngeal swabs by RT-qPCR without an RNA extraction step (a) Nasopharyngeal (NP) swab diluents from two confirmed COVID-19 patients were pooled, and using the 2019-nCoV_N3 primer/probe set, the mixture was either i) subjected to RNA extraction using the Qiagen QIAamp Viral RNA Mini kit followed by subsequent testing by RT-qPCR (using the equivalent of 11.3 ul swab diluent) or ii) directly added to the RT-qPCR reaction, with or without a preheating step (five minutes at 70[[#xB0]]C, [[#x201C]]NP sample + heat[[#x201D]]). As a control, the indicated quantities of the CDC 2019-nCoV Positive Control SARS-CoV-2 synthetic RNA was spiked into M6 transport media, purified using the QIAamp Viral RNA Mini kit, and screened by RT-qPCR. NP swab samples from seven additional donors were screened by direct RT-qPCR for SARS-CoV-2 RNA using (b) the 2019-nCoV_N1 primer/probe set, (c) the 2019-nCoV_N2 primer/probe set, or (d) for human RNase P RNA using the RP primer probe set. NP swab samples from donors 1 [[#x2013]] 4 were previously shown to contain SARS-CoV-2 RNA by standard clinical RT-qPCR, while donors 5 [[#x2013]] 7 were negative. For each primer/probe set, 7 [[#xB5]]L (a) or 3 [[#xB5]]L (b, c, d) of NP swab diluent was tested in the RT-qPCR reaction per donor. For the N1 and N2 primer probe sets, the fully synthetic SARS-CoV-2 RNA Control 2 from Twist Bioscience was loaded at serial 10-fold dilutions (A, 3[[#xD7]]106 copies; B, 3[[#xD7]]105 copies; C, 3[[#xD7]]104 copies; D, 3[[#xD7]]103 copies; E, 3[[#xD7]]102 copies; F, 3[[#xD7]]101 copies) as indicated in panels b and c. No template control (NTC) wells were included for each primer/probe set and each was negative. For panels b and c, the correlation coefficients (R2) of the standard curves were 0.999 and 0.995, respectively. The dashed line at cycle 40 in each graph indicates the limit of detection." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-990139409" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;SARS-CoV-2 RNA can be detected from COVID-19 patient nasopharyngeal swabs by RT-qPCR without an RNA extraction step&lt;;/span&gt;; (&lt;;strong&gt;;a&lt;;/strong&gt;;) Nasopharyngeal (NP) swab diluents from two confirmed COVID-19 patients were pooled, and using the 2019-nCoV_N3 primer/probe set, the mixture was either i) subjected to RNA extraction using the Qiagen QIAamp Viral RNA Mini kit followed by subsequent testing by RT-qPCR (using the equivalent of 11.3 ul swab diluent) or ii) directly added to the RT-qPCR reaction, with or without a preheating step (five minutes at 70[[#xB0]]C, [[#x201C]]NP sample + heat[[#x201D]]). As a control, the indicated quantities of the CDC 2019-nCoV Positive Control SARS-CoV-2 synthetic RNA was spiked into M6 transport media, purified using the QIAamp Viral RNA Mini kit, and screened by RT-qPCR. NP swab samples from seven additional donors were screened by direct RT-qPCR for SARS-CoV-2 RNA using (&lt;;strong&gt;;b&lt;;/strong&gt;;) the 2019-nCoV_N1 primer/probe set, (&lt;;strong&gt;;c&lt;;/strong&gt;;) the 2019-nCoV_N2 primer/probe set, or (&lt;;strong&gt;;d&lt;;/strong&gt;;) for human RNase P RNA using the RP primer probe set. NP swab samples from donors 1 [[#x2013]] 4 were previously shown to contain SARS-CoV-2 RNA by standard clinical RT-qPCR, while donors 5 [[#x2013]] 7 were negative. For each primer/probe set, 7 [[#xB5]]L (&lt;;strong&gt;;a&lt;;/strong&gt;;) or 3 [[#xB5]]L (&lt;;strong&gt;;b, c, d&lt;;/strong&gt;;) of NP swab diluent was tested in the RT-qPCR reaction per donor. For the N1 and N2 primer probe sets, the fully synthetic SARS-CoV-2 RNA Control 2 from Twist Bioscience was loaded at serial 10-fold dilutions (A, 3[[#xD7]]10&lt;;sup&gt;;6&lt;;/sup&gt;; copies; B, 3[[#xD7]]10&lt;;sup&gt;;5&lt;;/sup&gt;; copies; C, 3[[#xD7]]10&lt;;sup&gt;;4&lt;;/sup&gt;; copies; D, 3[[#xD7]]10&lt;;sup&gt;;3&lt;;/sup&gt;; copies; E, 3[[#xD7]]10&lt;;sup&gt;;2&lt;;/sup&gt;; copies; F, 3[[#xD7]]10&lt;;sup&gt;;1&lt;;/sup&gt;; copies) as indicated in panels &lt;;strong&gt;;b&lt;;/strong&gt;; and &lt;;strong&gt;;c&lt;;/strong&gt;;. No template control (NTC) wells were included for each primer/probe set and each was negative. For panels &lt;;strong&gt;;b&lt;;/strong&gt;; and &lt;;strong&gt;;c&lt;;/strong&gt;;, the correlation coefficients (R&lt;;sup&gt;;2&lt;;/sup&gt;;) of the standard curves were 0.999 and 0.995, respectively. The dashed line at cycle 40 in each graph indicates the limit of detection.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig. 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.20.001008/F1.medium.gif" width="261" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig. 1" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.20.001008/F1.medium.gif" width="261" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.20.001008/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1" data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.20.001008/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig. 1</span>
      <span class="caption-title">SARS-CoV-2 RNA can be detected from COVID-19 patient nasopharyngeal swabs by RT-qPCR without an RNA extraction step</span>
      <p id="p-8" class="first-child">(
       <strong>a</strong>) Nasopharyngeal (NP) swab diluents from two confirmed COVID-19 patients were pooled, and using the 2019-nCoV_N3 primer/probe set, the mixture was either i) subjected to RNA extraction using the Qiagen QIAamp Viral RNA Mini kit followed by subsequent testing by RT-qPCR (using the equivalent of 11.3 ul swab diluent) or ii) directly added to the RT-qPCR reaction, with or without a preheating step (five minutes at 70°C, “NP sample + heat”). As a control, the indicated quantities of the CDC 2019-nCoV Positive Control SARS-CoV-2 synthetic RNA was spiked into M6 transport media, purified using the QIAamp Viral RNA Mini kit, and screened by RT-qPCR. NP swab samples from seven additional donors were screened by direct RT-qPCR for SARS-CoV-2 RNA using (
       <strong>b</strong>) the 2019-nCoV_N1 primer/probe set, (
       <strong>c</strong>) the 2019-nCoV_N2 primer/probe set, or (
       <strong>d</strong>) for human RNase P RNA using the RP primer probe set. NP swab samples from donors 1 – 4 were previously shown to contain SARS-CoV-2 RNA by standard clinical RT-qPCR, while donors 5 – 7 were negative. For each primer/probe set, 7 µL (
       <strong>a</strong>) or 3 µL (
       <strong>b, c, d</strong>) of NP swab diluent was tested in the RT-qPCR reaction per donor. For the N1 and N2 primer probe sets, the fully synthetic SARS-CoV-2 RNA Control 2 from Twist Bioscience was loaded at serial 10-fold dilutions (A, 3×10
       <sup>6</sup> copies; B, 3×10
       <sup>5</sup> copies; C, 3×10
       <sup>4</sup> copies; D, 3×10
       <sup>3</sup> copies; E, 3×10
       <sup>2</sup> copies; F, 3×10
       <sup>1</sup> copies) as indicated in panels 
       <strong>b</strong> and 
       <strong>c</strong>. No template control (NTC) wells were included for each primer/probe set and each was negative. For panels 
       <strong>b</strong> and 
       <strong>c</strong>, the correlation coefficients (R
       <sup>2</sup>) of the standard curves were 0.999 and 0.995, respectively. The dashed line at cycle 40 in each graph indicates the limit of detection.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-9">We next sought to validate the direct RT-qPCR approach on additional samples, determine the optimal volume of NP swab diluent for use in the direct RT-qPCR assay, and further address the potential impact of a prior heating step on assay sensitivity. NP samples from three COVID-19 patients who had previously been shown to be positive for SARS-CoV-2 RNA at high, intermediate, or low copy load by the Department of Laboratory Medicine at the University of Washington (UW) in Seattle were heated or not at 95°C for 10 minutes and then directly loaded into RT-qPCR reactions at 1, 3, or 5 µL volumes, or subjected to RNA extraction via the Roche MagNA Pure 96 platform prior to loading the equivalent of ∼20 µL of swab diluent into the RT-qPCR reaction. The main findings of this experiment, shown in 
     <strong>
      <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>
     </strong>, were that i) SARS-CoV-2 RNA could be detected in all three viral copy level samples at either input volume by direct RT-qPCR, provided they were heated first, and ii) addition of less NP diluent led to more sensitive detection of target RNA. Thus, heating appears important for subsequent detection of low viral copy samples, presumably by denaturing inhibitors of the RT and/or PCR enzymes present in the NP diluent. The best sensitivity for SARS-CoV-2 detection was achieved when 3 µL of swab diluent was used for direct RT-qPCR 
     <strong>(
      <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>)
     </strong>.
    </p>
    <div id="T1" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1238739/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup">
         <a href="/highwire/markup/1238739/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
        <li class="download-ppt last">
         <a href="/highwire/powerpoint/1238739" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Table 1.</span>
      <span class="caption-title">Detection of SARS-CoV-2 RNA from NP swab diluent by direct RT-qPCR and the impact of heat and loading volume on assay sensitivity.</span>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-10">The CDC RT-qPCR method includes an internal control primer/probe set for detection of human RNase P. To ensure that direct RT-qPCR was able to detect the presence of this gene in NP swab diluent, and to test whether our approach was generalizable to other primer/probe sets, we screened seven donors (n = 4 COVID-19 positive; n = 3 COVID-19 negative) using the 2019-nCoV_N1, 2019-nCoV_N2, and RNase P primer/probe sets. Both the 2019-nCoV_N1 and 2019-nCoV_N2 primer/probe sets specifically detected SARS-CoV-2 RNA in each sample from COVID-19 patients, but not in samples from the negative donors (
     <strong>
      <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1b,c</a>
     </strong>). In addition, RNase P RNA was successfully detected in each donor by direct RT-qPCR (
     <strong>
      <a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1d</a>
     </strong>). Collectively, these results confirmed i) the validity of our direct RT-qPCR approach with additional virus-specific primer/probe sets for detection of SARS-CoV-2 RNA, and ii) the presence of intact cellular RNA, as detected with an internal primer-probe set targeting a host gene, in all samples tested.
    </p>
    <p id="p-11">To get an accurate view of how omission of the RNA extraction would perform in a real-world clinical setting, we tested in a blinded fashion a panel of 150 NP swabs from COVID-19 patients representing high (CT less than 20), intermediate (CT of 20 - 30), or low (CT of more than 30) viral RNA loads by direct RT-qPCR. The SARS-CoV-2 RNA copy levels in these samples were previously determined at UW by standard clinical RT-qPCR that included RNA extraction. To address the potential impact of a prior heating step on detection sensitivity, NP swab samples were heated or not for 10 min at 95°C prior to use in the downstream RNA extraction and/or direct RT-qPCR. For each NP swab sample, a 3 µL volume of diluent was used for direct RT-qPCR. In parallel, a 30 µL aliquot of each sample was used for RNA extraction and an equivalent of 3 µL of the original swab diluent used for RT-qPCR, allowing for a one to one comparison with the direct RT-qPCR method. To control for inhibitors of the RT or PCR enzymes and/or RNase activity in the swab diluent, an aliquot of each swab diluent was spiked with 4 x 10
     <sup>4</sup> copies of EXO control RNA for subsequent detection with an EXO primer-probe set; no inhibition was noted under any of the tested conditions. 
     <strong>Supplementary Table 1</strong> shows the full results of this experiment while 
     <strong>
      <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>
     </strong> provides a summary. 
     <strong>
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>
     </strong> shows the CT values from the original clinical RT-qPCR done on the equivalent of 20 µL of NP swab diluent versus those obtained by direct or standard RT-qPCR when measuring 3 µL (or equivalent) of swab diluent. There were several key observations from this experiment. First, preheating NP swab samples prior to direct RT-qPCR enhanced assay sensitivity (e.g. 138 of 150 samples were detected following a heat step, compared with only 126 of 150 positive in the absence of preheating). Second, extraction of RNA prior to RT-qPCR did not enhance detection of SARS-CoV-2 positive samples when an equivalent amount of diluent was screened by direct RT-qPCR (138 of 150 samples were detected by each approach). This suggests one of the primary advantages of kit-based RNA extraction comes from concentrating the sample material so more RNA can be loaded into the RT-qPCR reaction. Third, 11 of the 12 samples not detected by direct RT-qPCR were from donors that had extremely low loads of viral RNA as originally determined in the clinical test (CTs ranging from 33 to 38). Fourth, preheating samples reduced sensitivity when using the Roche MagnaPure 96 RNA extraction system, but had little effect on the direct RT-qPCR approach. Finally, the results from the EXO spiking experiment indicate that the NP swab diluent does not inhibit the direct RT-qPCR assay. Collectively, these results demonstrate that the direct RT-qPCR assay described here is capable of reliably detecting the substantial majority of COVID-19 patients.
    </p>
    <div id="T2" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1238735/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup">
         <a href="/highwire/markup/1238735/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
        <li class="download-ppt last">
         <a href="/highwire/powerpoint/1238735" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Table 2.</span>
      <span class="caption-title">Detection sensitivity of direct RT-qPCR versus standard RT-qPCR on NP swabs containing a range of SARS-CoV-2 viral RNA loads.</span>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F2" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.20.001008/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Distribution of CT values from COVID-19 patient NP swabs following direct RT-qPCR versus standard RT-qPCR that includes RNA extraction. 150 NP swab samples representing high (CT values less than 20), intermediate (CT values of 20-30), or low (CT values of more than 30) SARS-CoV-2 RNA loads as determined by standard clinical RT-qPCR at UW in Seattle (aqua circles) were analyzed by the indicated method. All assays used the 2019-nCoV_N2 primer/probe set. Direct RT-qPCR was performed on 3 [[#xB5]]L of NP swab diluent after heating for 10 minutes at 95[[#xB0]]C (green circles). In parallel, RNA was extracted from 30 [[#xB5]]L of NP swab diluent that had been previously heated at 95[[#xB0]]C for 10 minutes, and RNA representing 3 [[#xB5]]L of the original diluent was used in RT-qPCR (purple circles) to allow a head to head comparison to direct RT-qPCR on the same quantity of NP swab diluent. The limit of detection (40 CT) is denoted with a dashed line. Samples with CT values above this cutoff were considered negative for SARS-CoV-2 RNA. The fitted curves are LOESS-smoothed (locally estimated scatterplot smoothing) CT values, with 95% confidence interval in gray, against the mean of CT values detected in the clinical RT-qPCR assay with primer sets N1 &amp;; N2. Samples are ordered by the latter mean. The full data set for this experiment and controls is provided in Supplementary Table 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-990139409" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Distribution of CT values from COVID-19 patient NP swabs following direct RT-qPCR versus standard RT-qPCR that includes RNA extraction.&lt;;/span&gt;; 150 NP swab samples representing high (CT values less than 20), intermediate (CT values of 20-30), or low (CT values of more than 30) SARS-CoV-2 RNA loads as determined by standard clinical RT-qPCR at UW in Seattle (aqua circles) were analyzed by the indicated method. All assays used the 2019-nCoV_N2 primer/probe set. Direct RT-qPCR was performed on 3 [[#xB5]]L of NP swab diluent after heating for 10 minutes at 95[[#xB0]]C (green circles). In parallel, RNA was extracted from 30 [[#xB5]]L of NP swab diluent that had been previously heated at 95[[#xB0]]C for 10 minutes, and RNA representing 3 [[#xB5]]L of the original diluent was used in RT-qPCR (purple circles) to allow a head to head comparison to direct RT-qPCR on the same quantity of NP swab diluent. The limit of detection (40 CT) is denoted with a dashed line. Samples with CT values above this cutoff were considered negative for SARS-CoV-2 RNA. The fitted curves are LOESS-smoothed (locally estimated scatterplot smoothing) CT values, with 95% confidence interval in gray, against the mean of CT values detected in the clinical RT-qPCR assay with primer sets N1 &amp;; N2. Samples are ordered by the latter mean. The full data set for this experiment and controls is provided in &lt;;strong&gt;;Supplementary Table 1&lt;;/strong&gt;;.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig. 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.20.001008/F2.medium.gif" width="440" height="260"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig. 2" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.20.001008/F2.medium.gif" width="440" height="260"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.20.001008/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2" data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.20.001008/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig. 2</span>
      <span class="caption-title">Distribution of CT values from COVID-19 patient NP swabs following direct RT-qPCR versus standard RT-qPCR that includes RNA extraction.</span>
      <p id="p-12" class="first-child">150 NP swab samples representing high (CT values less than 20), intermediate (CT values of 20-30), or low (CT values of more than 30) SARS-CoV-2 RNA loads as determined by standard clinical RT-qPCR at UW in Seattle (aqua circles) were analyzed by the indicated method. All assays used the 2019-nCoV_N2 primer/probe set. Direct RT-qPCR was performed on 3 µL of NP swab diluent after heating for 10 minutes at 95°C (green circles). In parallel, RNA was extracted from 30 µL of NP swab diluent that had been previously heated at 95°C for 10 minutes, and RNA representing 3 µL of the original diluent was used in RT-qPCR (purple circles) to allow a head to head comparison to direct RT-qPCR on the same quantity of NP swab diluent. The limit of detection (40 CT) is denoted with a dashed line. Samples with CT values above this cutoff were considered negative for SARS-CoV-2 RNA. The fitted curves are LOESS-smoothed (locally estimated scatterplot smoothing) CT values, with 95% confidence interval in gray, against the mean of CT values detected in the clinical RT-qPCR assay with primer sets N1 &amp;; N2. Samples are ordered by the latter mean. The full data set for this experiment and controls is provided in 
       <strong>Supplementary Table 1</strong>.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-13">COVID-19 testing demands are currently overwhelming the world’s clinical laboratories. A major chokepoint is the RNA extraction step, due to both the time and labor for this step and the critical shortage of extraction kits and required reagents. As a means to circumvent these limitations on clinical testing availability, we have shown that a direct RT-qPCR approach that omits an RNA extraction step can effectively identify SARS-CoV-2 RNA from NP swabs. Further, direct RT-qPCR is able to detect host RNA from NP swabs, which enables us to rule out false negatives due to inadequate swabbing of patients resulting in a paucity of intact RNA in the specimen, or due to the presence of inhibitors. When applied to a large collection of clinical NP specimens representative of the range of COVID-19 patients in the state of Washington, the direct RT-qPCR assay correctly identified 92% of the donors screened. The only donors that our simplified approach missed were those with the very lowest load of viral RNA (and who were on the very cusp of detection even when RNA extraction was used). Of the 1,872 cases with a positive diagnosis at UW by our team at the time of this writing, only ∼25% would fall into this low copy range, which indicates that this assay will detect the substantial majority of COVID-19 patients. Indeed, after we posted our initial results on bioRxiv 
     <sup>
      <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>
     </sup>, two additional groups posted preliminary data on bioRxiv and medRxiv replicating these findings, using standard clinical diagnostic methods in Chile and Denmark 
     <sup>
      <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>,
      <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>
     </sup>. The simplified approach we present here may be especially well suited for general screening of the public and identification of “silent carrier” individuals. Where resources permit, we envision that a two-step testing algorithm in which a repeat test including RNA extraction could be applied for symptomatic patients, health care workers, and others with a high index of suspicion but an initial negative result. Such a two-step strategy would reduce the need for RNA extraction for a substantial portion of future COVID-19 tests. Importantly, many testing sites in the developing world may not even have the capacity to perform RNA extraction, but could easily run RT-qPCR reactions. This new approach would open up a viable avenue for testing in these regions.
    </p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Methods</h2>
    <div id="sec-3" class="subsection">
     <h3>Samples</h3>
     <div id="sec-4" class="subsection">
      <h4>University of Vermont</h4>
      <p id="p-14">Clinical nasopharyngeal swabs were collected in 3 ml M6 transport medium. Patients were confirmed to be negative (3) or positive (6) for COVID-19 by the Vermont Department of Health using the CDC 2019-Novel Coronavirus (2019-nCoV) Real-Time RT-PCR Diagnostic Panel 
       <sup>
        <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>
       </sup>. Limited quantities of this material were available, so we initially pooled equal amounts of sample from two COVID-19 patients (
       <strong>
        <a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1a</a>
       </strong>). Subsequently, four additional COVID-19 confirmed patients and four COVID-19 negative donors were tested individually for the presence of SARS-CoV-2 RNA or RNase P by direct RT-qPCR of the sample material (
       <strong>
        <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figs. 1b-d</a>
       </strong>).
      </p>
     </div>
     <div id="sec-5" class="subsection">
      <h4>University of Washington</h4>
      <p id="p-15">Clinical nasopharyngeal swabs were collected in 3 mL of Universal transport medium (UTM). 150 samples from patients confirmed to be COVID-19 positive by the University of Washington Medical Laboratory with ranges of viral loads (CT values of SARS-CoV-2 with the N2 primer from the original clinical test run at UW range from 10.17 to 38.13) and sufficient volume were selected for this study (
       <strong>
        <a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2</a>, 
        <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Tables 1</a> and 
        <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">2</a>, and Supplementary Table 1
       </strong>).
      </p>
      <p id="p-16">All patient samples at each testing site were stored at 4 oC in between sample collection and transport to the laboratory, and again until clinical testing was carried out, maintaining a standardized sample acquisition and processing protocol.</p>
     </div>
    </div>
    <div id="sec-6" class="subsection">
     <h3>RNA Extraction</h3>
     <div id="sec-7" class="subsection">
      <h4>University of Vermont</h4>
      <p id="p-17">To compare the effect of nucleic acid extraction to direct RT-PCR, RNA from a pooled patient sample was extracted using the QIAamp Viral RNA Mini (Qiagen, Cat. No. 52904) kit according to the manufacturer’s instructions. 140 µL of pooled nasopharyngeal swab diluent was extracted and eluted in 60 µL; 5 µL (equivalent to 11 µL of original sample) was used for real-time RT-PCR (
       <strong>
        <a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig. 1a</a>
       </strong>).
      </p>
     </div>
     <div id="sec-8" class="subsection">
      <h4>University of Washington</h4>
      <p id="p-18">To compare the effect of nucleic acid extraction, RNA from 30 µL or 200 µL of patient sample was extracted using the Roche MagNA Pure 96 (Roche Lifesciences) platform, eluted into 50 µL of buffer; 5 µL of RNA (equivalent to 3 µL of original sample for a 30 µL extraction) was used for real-time RT-PCR. To monitor RNA recovery and RT-PCR efficiency, 40,000 copies of EXO internal control RNA were added into the lysis buffer and went through the extraction process with each sample (
       <strong>
        <a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2</a>, 
        <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Tables 1</a> and 
        <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">2</a>, and Supplementary Table 1
       </strong>).
      </p>
     </div>
    </div>
    <div id="sec-9" class="subsection">
     <h3>RT-qPCR Detection</h3>
     <div id="sec-10" class="subsection">
      <h4>University of Vermont</h4>
      <p id="p-19">In 
       <strong>
        <a id="xref-fig-1-7" class="xref-fig" href="#F1">Fig. 1a</a>
       </strong>, 7 µL of pooled nasopharyngeal swab diluent (heated to 70°C for 5 minutes, or not), or 5 µL extracted RNA, was used as input material for the New England Biolabs Luna Universal Probe One Step RT qPCR kit (Cat #E3006S, lot #10066679) according to the IDT recommendation for primers/probe (1.5 µL primer/probe per reaction) using primer set N3 from IDT’s 2019-nCoV CDC Emergency Use Authorization Kits (20 µL reaction). Subsequently, in 
       <strong>
        <a id="xref-fig-1-8" class="xref-fig" href="#F1">Figs. 1b-d</a>
       </strong>, 3 µL of nasopharyngeal swab diluent was used as input material for the ThermoFisher TaqPath™ 1-Step RT-qPCR Master Mix, CG (Cat #A15299) using primer sets N1, N2, and RP (human RNase P) from IDT’s CDC Emergency Use Authorization Kits (20 µL reaction). Known copies of positive control RNA (CDC 2019-nCoV Positive Control 
       <em>in</em> vitro transcribed RNA for 
       <strong>
        <a id="xref-fig-1-9" class="xref-fig" href="#F1">Fig. 1a</a>
       </strong>; fully synthetic SARS-CoV-2 RNA Control 2 [Cat# MN908947.3, Twist Bioscience] for 
       <strong>
        <a id="xref-fig-1-10" class="xref-fig" href="#F1">Fig. 1b,c</a>
       </strong>) were used to compare viral RNA quantities in patient samples. Each patient sample, as well as no template control (water as well as M6 transport media) was run in duplicate, the synthetic RNA standard was run in triplicate. The initial experiment was run on an ABI QuantStudio 6 Flex (
       <strong>
        <a id="xref-fig-1-11" class="xref-fig" href="#F1">Fig. 1a</a>
       </strong>), while subsequent experiments were run on an ABI 7500 Fast (
       <strong>
        <a id="xref-fig-1-12" class="xref-fig" href="#F1">Figs. 1b-d</a>
       </strong>). Reagent preparations were performed in a PCR-free clean room equipped with PCR workstations (AirClean Systems). Clinical sample manipulations (RNA extractions or direct addition of NP swab diluent to RT-qPCR plates) were done in a Class IIA Biosafety Cabinet using BSL-2 precautions. RT-qPCR plates were placed directly in biosafety waste at the conclusion of each run.
      </p>
     </div>
     <div id="sec-11" class="subsection">
      <h4>University of Washington</h4>
      <p id="p-20">
       <strong>
        <a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2</a>, 
        <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Tables 1</a> and 
        <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">2</a>, and Supplementary Table 1
       </strong>: For each patient sample, 250 µL was heat treated at 95oC for 10 minutes, or not. For direct RT-PCR, 3 µL of this sample, or 5 µL of extracted RNA, was added directly into the RT-PCR reaction mixture. Each real-time RT-PCR reaction contained 400 nM of CDC N2 forward and reverse primers and 100 nM of N2 probe. To monitor potential RT-PCR inhibition, each RT-PCR reaction was spiked with 40,000 copies of EXO internal control RNA and EXO primers (100 nM of EXO forward and 200 nM of EXO reverse) and probes (62.5 nM). Real-time RT-PCR assays were performed using AgPath-ID One Step RT-PCR kit (Life Technologies) and an ABI 7500 Real Time PCR system was used to perform the RT-PCR reactions 
       <sup>
        <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>
       </sup>. For samples that were subjected to RNA extraction, 40,000 copies of EXO RNA was spiked into the lysis buffer. Extraction efficiency for each sample was tracked based on the percentage of the spiked EXO RNA that was detected by RT-qPCR. The primer-probe sequences and RT-qPCR conditions for all methods used are below:
      </p>
      <div id="T3" class="table pos-float">
       <div class="table-inline table-callout-links">
        <div class="callout">
         <span>View this table:</span>
         <ul class="callout-links">
          <li class="view-inline first">
           <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1238738/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
          </li>
          <li class="view-popup last">
           <a href="/highwire/markup/1238738/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
          </li>
         </ul>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="sec-12" class="subsection">
     <h3>Study Design</h3>
     <p id="p-21">For both the University of Vermont and the University of Washington, sample selection (including information regarding CT values) and RNA extraction/RT-PCR of samples were performed by separate individuals; the person running the assays was blinded to the original clinical CT value of the samples.</p>
    </div>
   </div>
   <div class="section" id="sec-14">
    <h2 class="">Author Contributions</h2>
    <p id="p-23">EAB, MLH, ST, KRJ, and JWB conceived and designed experiments. EAB, MLH, GAP, YW, ST, PL, JH, DG performed experiments. JWC, ALG, and KRJ provided specimens. EAB, ST, DJS, MLH, KRJ, and JWB analyzed the data. EAB, CDH, KRJ, and JWB wrote the manuscript and all authors participated in the editorial process and approved the manuscript.</p>
   </div>
   <div class="section" id="sec-15">
    <h2 class="">Ethical Statement</h2>
    <p id="p-24">Patient samples were de-identified and were not considered Human Subjects Research due to the quality improvement and public health intent of the work.</p>
   </div>
   <div class="section" id="sec-16">
    <h2 class="">Competing Interests</h2>
    <p id="p-25">The authors have declared that no competing interests exist. While DJS was employed at IXIS LLC at the time of this study, his employment there did not create a competing interest. Further, IXIS LLC had no involvement in this study.</p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgements</h2>
    <p id="p-22">The authors gratefully acknowledge funding from the Office of the Vice President for Research at the University of Vermont (JWB), the M.J. Murdock Charitable Trust (KRJ), and NIH grants P30GM118228-04 (Immunology and Infectious Disease COBRE) (EB, JAD), P20GM125498-02 (Translational Global Infectious Diseases Research COBRE) (BDK, CDH, SAD, JAD), U01AI1141997 (JWB, SAD, and BDK), R41AI132047 (JWB), and U54 GM115516-01 (JAD). We also thank the staff at the Vermont Department of Health involved in COVID-19 diagnostic testing in particular Helen Reid, Cheryl Achilles, and Christine Matusevich as well as Dr. Deborah Donnell, Dr. Ken Hampel, Dr. Colm Atkins, Dr. Kishana Taylor, Dr. Robyn Kent, Daniel Vellone, and Denise Francis for technical comments or experimental help.</p>
   </div>
   <div class="section fn-group" id="fn-group-1">
    <h2>Footnotes</h2>
    <ul>
     <li class="fn-update fn-group-summary-of-updates" id="fn-3">
      <p id="p-5">The original version of this manuscript reported that addition of nasopharyngeal (NP) swab diluent from COVID-19 positive donors directly into a RT-qPCR reaction (ie. skipping the RNA extraction step) was a streamlined approach that could identify SARS-CoV-2 RNA in NP swab samples. Our initial finding was based on a single patient NP sample (which is shown in Figure 1a of this revision). The remainder of the data in this revision is new. Specifically, we have now optimized conditions for this direct RT-PCR test (new Table 1), and replicated our result using 154 additional clinical samples (new Figures 1b,c,d and 2, new Table 2, and new Supplementary Table 1). Furthermore, we have included COVID-19 negative samples, which enables control for the presence of RNA in the swab specimens to rule out missing a positive due to an absence of intact RNA in the specimen (Figure 1d). Importantly, new collaborators (The Jerome laboratory at UW Seattle) joined the UVM team to generate this new data and are included as coauthors. Note that this revision removes Figures 2 and 3 of the original version (dealing with the use of alternative RNA extraction kits).</p>
     </li>
    </ul>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.03.20.001008v2.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">Centers for Disease Control and Prevention &amp;; Division of Viral Diseases</span>. 
         <span class="cit-source">CDC 2019-Novel Coronavirus (2019-nCoV) Real-Time RT-PCR Diagnostic Panel</span>. (
         <a href="https://www.fda.gov/media/134922/download">https://www.fda.gov/media/134922/download</a>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.20.001008v2.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chu</span>, 
          <span class="cit-name-given-names">D.K.W.</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Molecular Diagnosis of a Novel Coronavirus (2019-nCoV) Causing an Outbreak of Pneumonia</span>. 
         <abbr class="cit-jnl-abbrev">Clin Chem</abbr> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.20.001008v2.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Corman</span>, 
          <span class="cit-name-given-names">V.M.</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR</span>. 
         <abbr class="cit-jnl-abbrev">Euro Surveill</abbr>
         <span class="cit-vol">25</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.03.20.001008v2.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bruce</span>, 
          <span class="cit-name-given-names">E.A.</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-source">RT-qPCR DETECTION OF SARS-CoV-2 RNA FROM PATIENT NASOPHARYNGEAL SWAB USING QIAGEN RNEASY KITS OR DIRECTLY VIA OMISSION OF AN RNA EXTRACTION STEP. (Preprint at</span>
         <a href="https://www.biorxiv.org/content/10.1101/2020.03.20.001008v1">https://www.biorxiv.org/content/10.1101/2020.03.20.001008v1</a>, 
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.20.001008v2.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Nelson</span>, 
          <span class="cit-name-given-names">A.C.</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <abbr class="cit-jnl-abbrev">Analytical Validation of a COVID-19 qRT-PCR Detection Assay Using a 384-well Format and Three Extraction Methods. (Preprint at</abbr>
         <a href="https://www.biorxiv.org/content/10.1101/2020.04.02.022186v1">https://www.biorxiv.org/content/10.1101/2020.04.02.022186v1</a>, 
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.03.20.001008v2.6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Beltran-Pavez</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-source">SARS-CoV-2 detection from nasopharyngeal swab samples without RNA extraction. (Preprint at</span>
         <a href="https://www.biorxiv.org/content/10.1101/2020.03.28.013508v1">https://www.biorxiv.org/content/10.1101/2020.03.28.013508v1</a>, 
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.03.20.001008v2.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Fomsgaard</span>, 
          <span class="cit-name-given-names">A.S.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Rosenstierne</span>, 
          <span class="cit-name-given-names">M.W.</span>
         </span>
         <span class="cit-source">An alternative workflow for molecular detection of SARS-CoV-2 - escape from the NA extraction kit-shortage. (Preprint at</span>
         <a href="https://www.medrxiv.org/content/10.1101/2020.03.27.20044495v1">https://www.medrxiv.org/content/10.1101/2020.03.27.20044495v1</a>, 
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.03.20.001008v2.8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Casto</span>, 
          <span class="cit-name-given-names">A.M.</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-source">Comparative Performance of SARS-CoV-2 Detection Assays using Seven Different Primer/Probe Sets and One Assay Kit. (Preprint at</span>
         <a href="https://www.medrxiv.org/content/10.1101/2020.03.13.20035618v1">https://www.medrxiv.org/content/10.1101/2020.03.13.20035618v1</a> 2020).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
