<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2018/10/15/443457.atom"/>
  <meta name="HW.pisa" content="biorxiv;443457v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Characterisation of the faecal virome of captive and wild Tasmanian devils using virus-like particles metagenomics and meta-transcriptomics"/>
  <meta name="DC.Identifier" content="10.1101/443457"/>
  <meta name="DC.Date" content="2018-10-15"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2018, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Background The Tasmanian devil is an endangered carnivorous marsupial threatened by devil facial tumour disease (DFTD). While research on DFTD has been extensive, little is known about the viruses present in devils, and whether any of these are of potential conservation relevance for this endangered species. Methods Using both metagenomics based on virus-like particle (VLP) enrichment and sequence-independent amplification (VLP metagenomics), and meta-transcriptomics based on bulk RNA sequencing, we characterised and compared the faecal viromes of captive and wild Tasmanian devils. Results A total of 54 devil faecal samples collected from captive (n = 2) and wild (n = 4) populations were processed for virome characterisation using both approaches. We detected many novel, highly divergent viruses, including vertebrate viruses, bacteriophage and other dietary associated plant and insect viruses. In total, 18 new vertebrate viruses, including novel sapelovirus, astroviruses, bocaviruses, papillomaviruses and gammaherpesvirus were identified, as well as known mammalian pathogens including rabbit haemorrhagic disease virus 2 (RHDV2). Captive devils showed significantly lower levels of viral diversity than wild devils. Comparison of the two methodological approaches revealed substantial differences in the number and types of viruses detected, with meta-transcriptomics mainly identifying RNA viruses, and VLP metagenomics largely identifying DNA viruses. Conclusion This study has greatly expanded our knowledge of eukaryotic viruses in the Tasmanian devil and provides important baseline information that will contribute to the conservation and captive management of this endangered species. In addition, our results showed that a combination of VLP metagenomics and meta-transcriptomics may be a more comprehensive approach to virome characterisation than either method alone. * SH : Stony Head MI : Maria Island BR : Buckbys Road wMW : wukalina/Mt William National Park VLP : Virus-like particles RHDV : Rabbit haemorrhagic disease virus DeSV : Devil sapelovirus DeAstV1 : DeRoV1 and -2 Devil rotavirus 1 and 2 DePBV1-6 : Devil picobirnavirus 1-6 DeBoV1 and -2 : Devil bocavirus 1 and 2 DePV1 and -2 : Devil papillomavirus 1 and 2 DePyV1 and -2 : Devil polyomavirus 1 and 2 DaHV-3 : Dasyurid herpesvirus 3 DeCV : Devil circovirus"/>
  <meta name="DC.Contributor" content="Rowena Chong"/>
  <meta name="DC.Contributor" content="Mang Shi"/>
  <meta name="DC.Contributor" content="Catherine E Grueber"/>
  <meta name="DC.Contributor" content="Edward C Holmes"/>
  <meta name="DC.Contributor" content="Carolyn Hogg"/>
  <meta name="DC.Contributor" content="Katherine Belov"/>
  <meta name="DC.Contributor" content="Vanessa R Barrs"/>
  <meta name="article:published_time" content="2018-10-15"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Characterisation of the faecal virome of captive and wild Tasmanian devils using virus-like particles metagenomics and meta-transcriptomics"/>
  <meta name="citation_abstract" lang="en" content="&lt;;h3&gt;;Abstract&lt;;/h3&gt;; &lt;;h3&gt;;Background&lt;;/h3&gt;; &lt;;p&gt;;The Tasmanian devil is an endangered carnivorous marsupial threatened by devil facial tumour disease (DFTD). While research on DFTD has been extensive, little is known about the viruses present in devils, and whether any of these are of potential conservation relevance for this endangered species.&lt;;/p&gt;;&lt;;h3&gt;;Methods&lt;;/h3&gt;; &lt;;p&gt;;Using both metagenomics based on virus-like particle (VLP) enrichment and sequence-independent amplification (VLP metagenomics), and meta-transcriptomics based on bulk RNA sequencing, we characterised and compared the faecal viromes of captive and wild Tasmanian devils.&lt;;/p&gt;;&lt;;h3&gt;;Results&lt;;/h3&gt;; &lt;;p&gt;;A total of 54 devil faecal samples collected from captive (n = 2) and wild (n = 4) populations were processed for virome characterisation using both approaches. We detected many novel, highly divergent viruses, including vertebrate viruses, bacteriophage and other dietary associated plant and insect viruses. In total, 18 new vertebrate viruses, including novel sapelovirus, astroviruses, bocaviruses, papillomaviruses and gammaherpesvirus were identified, as well as known mammalian pathogens including rabbit haemorrhagic disease virus 2 (RHDV2). Captive devils showed significantly lower levels of viral diversity than wild devils. Comparison of the two methodological approaches revealed substantial differences in the number and types of viruses detected, with meta-transcriptomics mainly identifying RNA viruses, and VLP metagenomics largely identifying DNA viruses.&lt;;/p&gt;;&lt;;h3&gt;;Conclusion&lt;;/h3&gt;; &lt;;p&gt;;This study has greatly expanded our knowledge of eukaryotic viruses in the Tasmanian devil and provides important baseline information that will contribute to the conservation and captive management of this endangered species. In addition, our results showed that a combination of VLP metagenomics and meta-transcriptomics may be a more comprehensive approach to virome characterisation than either method alone.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2018/01/01"/>
  <meta name="citation_mjid" content="biorxiv;443457v1"/>
  <meta name="citation_id" content="443457v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/443457v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/443457v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/443457v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457.full.pdf"/>
  <meta name="citation_doi" content="10.1101/443457"/>
  <meta name="citation_num_pages" content="42"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="443457"/>
  <meta name="citation_author" content="Rowena Chong"/>
  <meta name="citation_author_institution" content="School of Life and Environmental Sciences, University of Sydney"/>
  <meta name="citation_author" content="Mang Shi"/>
  <meta name="citation_author_institution" content="Marie Bashir Institute for Infectious Diseases and Biosecurity, Sydney Medical School, University of Sydney"/>
  <meta name="citation_author_institution" content="School of Life and Environmental Sciences and Sydney Medical School, Charles Perkins Centre, University of Sydney"/>
  <meta name="citation_author" content="Catherine E Grueber"/>
  <meta name="citation_author_institution" content="School of Life and Environmental Sciences, University of Sydney"/>
  <meta name="citation_author_institution" content="San Diego Zoo Global"/>
  <meta name="citation_author" content="Edward C Holmes"/>
  <meta name="citation_author_institution" content="Marie Bashir Institute for Infectious Diseases and Biosecurity, Sydney Medical School, University of Sydney"/>
  <meta name="citation_author_institution" content="School of Life and Environmental Sciences and Sydney Medical School, Charles Perkins Centre, University of Sydney"/>
  <meta name="citation_author" content="Carolyn Hogg"/>
  <meta name="citation_author_institution" content="School of Life and Environmental Sciences, University of Sydney"/>
  <meta name="citation_author" content="Katherine Belov"/>
  <meta name="citation_author_institution" content="School of Life and Environmental Sciences, University of Sydney"/>
  <meta name="citation_author" content="Vanessa R Barrs"/>
  <meta name="citation_author_institution" content="Marie Bashir Institute for Infectious Diseases and Biosecurity, Sydney Medical School, University of Sydney"/>
  <meta name="citation_author_institution" content="Sydney School of Veterinary Science, University of Sydney"/>
  <meta name="citation_author_email" content="vanessa.barrs@sydney.edu.au"/>
  <meta name="citation_reference" content="Pemberton, D., S. Gales, B. Bauer, R. Gales, B. Lazenby, and K. Medlock, The diet of the Tasmanian devil, Sarcophilus harrisii, as determined from analysis of scat and stomach contents. Papers and Proceedings of the Royal Society of Tasmania, 2008. 142(2): p. 13–22."/>
  <meta name="citation_reference" content="Owen, D. and D. Pemberton, Made for travelling rough’: devil ecology, in Tasmanian Devil: A Unique and Threatened Animal. 2005, Allen &amp;; Unwin."/>
  <meta name="citation_reference" content="Lazenby, B.T., M.W. Tobler, W.E. Brown, C.E. Hawkins, G.J. Hocking, F. Hume, S. Huxtable, P. Iles, M.E. Jones, and C. Lawrence, Density trends and demographic signals uncover the long-term impact of transmissible cancer in Tasmanian devils. Journal of Applied Ecology, 2018. 55(3): p. 1368–1379."/>
  <meta name="citation_reference" content="Pye, R.J., D. Pemberton, C. Tovar, J.M. Tubio, K.A. Dun, S. Fox, J. Darby, D. Hayes, G.W. Knowles, and A. Kreiss, A second transmissible cancer in Tasmanian devils. Proceedings of the National Academy of Sciences, 2016. 113(2): p. 374–379."/>
  <meta name="citation_reference" content="Hogg, C., A. Lee, C. Srb, and C. Hibbard, Metapopulation management of an endangered species with limited genetic diversity in the presence of disease: the Tasmanian devil Sarcophilus harrisii. International Zoo Yearbook, 2017. 51(1): p. 137–153."/>
  <meta name="citation_reference" content="Jones, M.E., P.J. Jarman, C.M. Lees, H. Hesterman, R.K. Hamede, N.J. Mooney, D. Mann, C.E. Pukk, J. Bergfeld, and H. McCallum, Conservation management of Tasmanian devils in the context of an emerging, extinction-threatening disease: devil facial tumor disease. EcoHealth, 2007. 4(3): p. 326–337."/>
  <meta name="citation_reference" content="Loh, R., J. Bergfeld, D. Hayes, A. O’hara, S. Pyecroft, S. Raidal, and R. Sharpe, The pathology of devil facial tumor disease (DFTD) in Tasmanian devils (Sarcophilus harrisii). Veterinary Pathology, 2006. 43(6): p. 890–895."/>
  <meta name="citation_reference" content="McCallum, H., Tasmanian devil facial tumour disease: lessons for conservation biology. Trends in Ecology &amp;; Evolution, 2008. 23(11): p. 631–637."/>
  <meta name="citation_reference" content="Kreiss, A., Y. Cheng, F. Kimble, B. Wells, S. Donovan, K. Belov, and G.M. Woods, Allorecognition in the Tasmanian devil (Sarcophilus harrisii), an endangered marsupial species with limited genetic diversity. PLOS. one, 2011. 6(7): p. e22402."/>
  <meta name="citation_reference" content="Pearse, A.-M. and K. Swift, Allograft theory: transmission of devil facial-tumour disease. Nature, 2006. 439(7076): p. 549–549."/>
  <meta name="citation_reference" content="Grueber, C.E., E. Peel, R. Gooley, and K. Belov, Genomic insights into a contagious cancer in Tasmanian devils. Trends in Genetics, 2015. 31(9): p. 528–535."/>
  <meta name="citation_reference" content="Stalder, K., P.K. Vaz, J.R. Gilkerson, R. Baker, P. Whiteley, N. Ficorilli, L. Tatarczuch, T. Portas, K. Skogvold, and G.A. Anderson, Prevalence and clinical significance of herpesvirus infection in populations of Australian marsupials. PLO. one, 2015. 10(7): p. e0133807."/>
  <meta name="citation_reference" content="Shi, M., X.-D. Lin, X. Chen, J.-H. Tian, L.-J. Chen, K. Li, W. Wang, J.-S. Eden, J.-J. Shen, and L. Liu, The evolutionary history of vertebrate RNA viruses. Nature, 2018: p. 1."/>
  <meta name="citation_reference" content="Weisbecker, V. and R. Beck, Marsupial and Monotreme Evolution And Biogeography. 2015. 1–32."/>
  <meta name="citation_reference" content="Djikeng, A., R. Halpin, R. Kuzmickas, J. DePasse, J. Feldblyum, N. Sengamalay, C. Afonso, X. Zhang, N.G. Anderson, and E. Ghedin, Viral genome sequencing by random priming methods. BMC. genomics, 2008. 9(1): p. 5."/>
  <meta name="citation_reference" content="Thurber, R.V., Methods in Viral Metagenomics. Handbook of Molecular Microbial Ecology II: Metagenomics in different habitats: Wiley-Blackwell, 2011."/>
  <meta name="citation_reference" content="Conceicao-Neto, N., M. Zeller, H. Lefrere, P. De. Bruyn, L. Beller, W. Deboutte, C.K. Yinda, R. Lavigne, P. Maes, M. Van Ranst, E. Heylen, and J. Matthijnssens, Modular approach to customise sample preparation procedures for viral metagenomics: a reproducible protocol for virome analysis. Scientific Report, 2015. 5: p. 16532."/>
  <meta name="citation_reference" content="Hall, R.J., J. Wang, A.K. Todd, A.B. Bissielo, S. Yen, H. Strydom, N.E. Moore, X. Ren, Q.S. Huang, and P.E. Carter, Evaluation of rapid and simple techniques for the enrichment of viruses prior to metagenomic virus discovery. Journal of virological methods, 2014. 195: p. 194–204."/>
  <meta name="citation_reference" content="Shi, M., X.-D. Lin, J.-H. Tian, L.-J. Chen, X. Chen, C.-X. Li, X.-C. Qin, J. Li, J.-P. Cao, and J.-S. Eden, Redefining the invertebrate RNA virosphere. Nature, 2016. 540(7634): p. 539."/>
  <meta name="citation_reference" content="Shi, M., P. Neville, J. Nicholson, J.-S. Eden, A. Imrie, and E.C. Holmes, High-Resolution Metatranscriptomics Reveals the Ecological Dynamics of Mosquito-Associated RNA Viruses in Western Australia. Journal of Virology, 2017. 91(17): p. e00680–17."/>
  <meta name="citation_reference" content="Hawkins, C.E., C. Baars, H. Hesterman, G.J. Hocking, M.E. Jones, B. Lazenby, D. Mann, N. Mooney, D. Pemberton, S. Pyecroft, M. Restani, and J. Wiersma, Emerging disease and population decline of an island endemic, the Tasmanian devil Sarcophilus harrisii. Biological Conservation, 2006. 131(2): p. 307–324."/>
  <meta name="citation_reference" content="Bolger, A.M., M. Lohse, and B. Usadel, Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics, 2014. 30(15): p. 2114–2120."/>
  <meta name="citation_reference" content="Grabherr, M.G., B.J. Haas, M. Yassour, J.Z. Levin, D.A. Thompson, I. Amit, X. Adiconis, L. Fan, R. Raychowdhury, and Q. Zeng, Full-length transcriptome assembly from RNA-Seq data without a reference genome. Nature biotechnology, 2011. 29(7): p. 644."/>
  <meta name="citation_reference" content="Rost, B., Twilight zone of protein sequence alignments. Protein Engineering, Design and Selection, 1999. 12(2): p. 85–94."/>
  <meta name="citation_reference" content="Shi, M., X.-D. Lin, X. Chen, J.-H. Tian, L.-J. Chen, K. Li, W. Wang, J.-S. Eden, J.-J. Shen, and L. Liu, The evolutionary history of vertebrate RNA viruses. Nature, 2018. 556(7700): p. 197."/>
  <meta name="citation_reference" content="Langmead, B. and S.L. Salzberg, Fast gapped-read alignment with Bowtie 2. Nature methods, 2012. 9(4): p. 357."/>
  <meta name="citation_reference" content="Li, B. and C.N. Dewey, RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome. BMC. bioinformatics, 2011. 12(1): p. 323."/>
  <meta name="citation_reference" content="Kearse, M., R. Moir, A. Wilson, S. Stones-Havas, M. Cheung, S. Sturrock, S. Buxton, A. Cooper, S. Markowitz, and C. Duran, Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data. Bioinformatics, 2012. 28(12): p. 1647–1649."/>
  <meta name="citation_reference" content="Katoh, K. and D.M. Standley, MAFFT multiple sequence alignment software version 7: improvements in performance and usability. Molecular biology and evolution, 2013. 30(4): p. 772–780."/>
  <meta name="citation_reference" content="Capella-Gutiérrez, S., J.M. Silla-Martínez, and T. Gabaldón, trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses. Bioinformatics, 2009. 25(15): p. 1972–1973."/>
  <meta name="citation_reference" content="Guindon, S. and O. Gascuel, A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood. Systematic biology, 2003. 52(5): p. 696–704."/>
  <meta name="citation_reference" content="Caporaso, J.G., J. Kuczynski, J. Stombaugh, K. Bittinger, F.D. Bushman, E.K. Costello, N. Fierer, A.G. Pena, J.K. Goodrich, J.I. Gordon, G.A. Huttley, S.T. Kelley, D. Knights, J.E. Koenig, R.E. Ley, C.A. Lozupone, D. McDonald, B.D. Muegge, M. Pirrung, J. Reeder, J.R. Sevinsky, P.J. Turnbaugh, W.A. Walters, J. Widmann, T. Yatsunenko, J. Zaneveld, and R. Knight, QIIME allows analysis of high-throughput community sequencing data. Nat Methods, 2010. 7(5): p. 335–6."/>
  <meta name="citation_reference" content="R Core Team, R: A language and environment for statistical computing. 2018: R Foundation for Statistical Computing, Vienna, Austria."/>
  <meta name="citation_reference" content="Abrantes, J., W. Van Der Loo, J. Le. Pendu, and P.J. Esteves, Rabbit haemorrhagic disease (RHD) and rabbit haemorrhagic disease virus (RHDV): a review. Veterinary research, 2012. 43(1): p. 12."/>
  <meta name="citation_reference" content="Hall, R.N., J.E. Mahar, A.J. Read, R. Mourant, M. Piper, N. Huang, and T. Strive, A strain-specific multiplex RT-PCR for Australian rabbit haemorrhagic disease viruses uncovers a new recombinant virus variant in rabbits and hares. Transboundary and emerging diseases, 2018. 65(2): p. e444–e456."/>
  <meta name="citation_reference" content="Martín, I., T. García, V. Fajardo, M. Rojas, N. Pegels, P. Hernández, and I. González, Polymerase chain reaction detection of rabbit DNA in food and animal feed. World Rabbit Science, 2010. 17(1): p. 27–36."/>
  <meta name="citation_reference" content="Burland, T.G., DNASTAR’s Lasergene sequence analysis software, in Bioinformatics methods and protocols. 2000, Springer. p. 71–91."/>
  <meta name="citation_reference" content="Snijder, E.J. and M.C. Horzinek, Toroviruses: replication, evolution and comparison with other members of the coronavirus-like superfamily. Journal of general virology, 1993. 74(11): p. 2305–2316."/>
  <meta name="citation_reference" content="King, A.M.Q., M.J. Adams, E.B. Carstens, and E.J. Lefkowitz, Virus Taxonomy: Classification and Nomenclature of Viruses: Ninth Report of the International Committee on Taxonomy of Viruses. 2012, Elsevier: San Diego."/>
  <meta name="citation_reference" content="Koonin, E.V., Y.I. Wolf, K. Nagasaki, and V.V. Dolja, The Big Bang of picorna-like virus evolution antedates the radiation of eukaryotic supergroups. Nature Reviews Microbiology, 2008. 6(12): p. 925."/>
  <meta name="citation_reference" content="Knowles, N., T. Hovi, T. Hyypiä, A. King, A.M. Lindberg, M. Pallansch, A. Palmenberg, P. Simmonds, T. Skern, and G. Stanway, Family-Picornaviridae. 2012."/>
  <meta name="citation_reference" content="Herrmann, J.E., D.N. Taylor, P. Echeverri, and N.R. Blacklow, Astroviruses as a cause of gastroenteritis in children. New England Journal of Medicine, 1991. 324(25): p. 1757–1760."/>
  <meta name="citation_reference" content="Finkbeiner, S.R., Y. Li, S. Ruone, C. Conrardy, N. Gregoricus, D. Toney, H.W. Virgin, L.J. Anderson, J. Vinjé, and D. Wang, Identification of a novel astrovirus (astrovirus VA1) associated with an outbreak of acute gastroenteritis. Journal of virology, 2009. 83(20): p. 10836–10839."/>
  <meta name="citation_reference" content="Méndez, E. and C.F. Arias, Astrovirus, in Clinical Virology, Third Edition. 2009, American Society of Microbiology. p. 1145–1153."/>
  <meta name="citation_reference" content="citation_pages=953-959;citation_year=2012"/>
  <meta name="citation_reference" content="Estes, M.K. and J. Cohen, Rotavirus gene structure and function. Microbiological reviews, 1989. 53(4): p. 410–449."/>
  <meta name="citation_reference" content="Ganesh, B., G. Masachessi, and Z. Mladenova, Animal picobirnavirus. Virusdisease, 2014. 25(2): p. 223–238."/>
  <meta name="citation_reference" content="Siegl, G., R.C. Bates, K.I. Berns, B.J. Carter, D.C. Kelly, E. Kurstak, and P. Tattersall, Characteristics and taxonomy of Parvoviridae. Intervirology, 1985. 23(2): p. 61–73."/>
  <meta name="citation_reference" content="Walboomers, J.M., M.V. Jacobs, M.M. Manos, F.X. Bosch, J.A. Kummer, K.V. Shah, P.J. Snijders, J. Peto, C.J. Meijer, and N. Muñoz, Human papillomavirus is a necessary cause of invasive cervical cancer worldwide. The Journal of pathology, 1999. 189(1): p. 12–19."/>
  <meta name="citation_reference" content="Chaturvedi, A.K., E.A. Engels, R.M. Pfeiffer, B.Y. Hernandez, W. Xiao, E. Kim, B. Jiang, M.T. Goodman, M. Sibug-Saber, and W. Cozen, Human papillomavirus and rising oropharyngeal cancer incidence in the United States. Journal of clinical oncology, 2011. 29(32): p. 4294."/>
  <meta name="citation_reference" content="Wilson, V.G., M. West, K. Woytek, and D. Rangasamy, Papillomavirus E1 proteins: form, function, and features. Virus Genes, 2002. 24(3): p. 275–90."/>
  <meta name="citation_reference" content="Johne, R., C.B. Buck, T. Allander, W.J. Atwood, R.L. Garcea, M.J. Imperiale, E.O. Major, T. Ramqvist, and L.C. Norkin, Taxonomical developments in the family Polyomaviridae. Archives of virology, 2011. 156(9): p. 1627–1634."/>
  <meta name="citation_reference" content="Buck, C.B., K. Van Doorslaer, A. Peretti, E.M. Geoghegan, M.J. Tisza, P. An, J.P. Katz, J.M. Pipas, A.A. McBride, and A.C. Camus, The ancient evolutionary history of polyomaviruses. PLoS pathogens, 2016. 12(4): p. e1005574."/>
  <meta name="citation_reference" content="Woolford, L., A. Rector, M. Van Ranst, A. Ducki, M.D. Bennett, P.K. Nicholls, K.S. Warren, R.A. Swan, G.E. Wilcox, and A.J. O’Hara, A novel virus detected in papillomas and carcinomas of the endangered western barred bandicoot (Perameles bougainville) exhibits genomic features of both the Papillomaviridae and Polyomaviridae. Journal of virology, 2007. 81(24): p. 13280–13290."/>
  <meta name="citation_reference" content="Bennett, M.D., L. Woolford, H. Stevens, M. Van Ranst, T. Oldfield, M. Slaven, A.J. O’Hara, K.S. Warren, and P.K. Nicholls, Genomic characterization of a novel virus found in papillomatous lesions from a southern brown bandicoot (Isoodon obesulus) in Western Australia. Virology, 2008. 376(1): p. 173–182."/>
  <meta name="citation_reference" content="Lukert, P., G. De. Boer, J. Dale, P.K. Keese, M. McNulty, J.W. Randles, and I. Tischer, Family Circoviridae. 1995, Springer-Verlag."/>
  <meta name="citation_reference" content="Shi, M., Y.-Z. Zhang, and E.C. Holmes, Meta-transcriptomics and The Evolutionary Biology of RNA Viruses. Virus research, 2017."/>
  <meta name="citation_reference" content="Kleiner, M., L.V. Hooper, and B.A. Duerkop, Evaluation of methods to purify virus-like particles for metagenomic sequencing of intestinal viromes. BMC. genomics, 2015. 16(1): p. 7."/>
  <meta name="citation_reference" content="Li, L., X. Deng, E.T. Mee, S. Collot-Teixeira, R. Anderson, S. Schepelmann, P.D. Minor, and E. Delwart, Comparing viral metagenomics methods using a highly multiplexed human viral pathogens reagent. Journal of virological methods, 2015. 213: p. 139–146."/>
  <meta name="citation_reference" content="Karlsson, O.E., S. Belák, and F. Granberg, The effect of preprocessing by sequence-independent, single-primer amplification (SISPA) on metagenomic detection of viruses. Biosecurity and bioterrorism: biodefense strategy, practice, and science, 2013. 11(S1): p. S227–S234."/>
  <meta name="citation_reference" content="Rosseel, T., O. Ozhelvaci, G. Freimanis, and S. Van Borm, Evaluation of convenient pretreatment protocols for RNA virus metagenomics in serum and tissue samples. Journal of Virological Methods, 2015. 222: p. 72–80."/>
  <meta name="citation_reference" content="Łusiak-Szelachowska, M., B. Weber-Dąbrowska, E. Jończyk-Matysiak, R. Wojciechowska, and A. Górski, Bacteriophages in the gastrointestinal tract and their implications. Gut Pathogens, 2017. 9: p. 44."/>
  <meta name="citation_reference" content="Carding, S.R., N. Davis, and L. Hoyles, Review article: the human intestinal virome in health and disease. Alimentary Pharmacology &amp;; Therapeutics, 2017. 46(9): p. 800–815."/>
  <meta name="citation_reference" content="Marzano, S.-Y.L., B.D. Nelson, O. Ajayi-Oyetunde, C.A. Bradley, T.J. Hughes, G.L. Hartman, D.M. Eastburn, and L.L. Domier, Identification of diverse mycoviruses through metatranscriptomics characterization of the viromes of five major fungal plant pathogens. Journal of virology, 2016. 90(15): p. 6846–6863."/>
  <meta name="citation_reference" content="Cheng, Y., S. Fox, D. Pemberton, C. Hogg, A.T. Papenfuss, and K. Belov, The Tasmanian devil microbiome—implications for conservation and management. Microbiome, 2015. 3(1): p. 1."/>
  <meta name="citation_reference" content="Wise, P., D. Lee, S. Peck, J. Clarke, S. Thalmann, J. Hockley, D. Schaap, and D. Pemberton, The conservation introduction of Tasmanian devils to Maria Island National Park: a response to devil facial tumor disease (DFTD). Global Re-introduction Perspectives: 2016. Case-studies from around the globe, 2016: p. 166."/>
  <meta name="citation_reference" content="Cheng, Y., K. Heasman, S. Peck, E. Peel, R.M. Gooley, A.T. Papenfuss, C.J. Hogg, and K. Belov, Significant decline in anticancer immune capacity during puberty in the Tasmanian devil. Scientific Reports, 2017. 7: p. 44716."/>
  <meta name="citation_reference" content="Kreiss, A., C. Tovar, D. Obendorf, K. Dun, and G. Woods, A murine xenograft model for a transmissible cancer in Tasmanian devils. Veterinary pathology, 2011. 48(2): p. 475–481."/>
  <meta name="citation_reference" content="Wiedinger, K., C. Bitsaktsis, and S. Chang, Reactivation of human polyomaviruses in immunocompromised states. Journal of neurovirology, 2014. 20(1): p. 1–8."/>
  <meta name="citation_reference" content="Li, L., X. Deng, P. Linsuwanon, D. Bangsberg, M.B. Bwana, P. Hunt, J.N. Martin, S.G. Deeks, and E. Delwart, AIDS Alters the Commensal Plasma Virome. Journal of Virology, 2013. 87(19): p. 10912."/>
  <meta name="citation_reference" content="Zhang, W., S. Yang, T. Shan, R. Hou, Z. Liu, W. Li, L. Guo, Y. Wang, P. Chen, X. Wang, F. Feng, H. Wang, C. Chen, Q. Shen, C. Zhou, X. Hua, L. Cui, X. Deng, Z. Zhang, D. Qi, and E. Delwart, Virome comparisons in wild-diseased and healthy captive giant pandas. Microbiome, 2017. 5(1): p. 90."/>
  <meta name="citation_reference" content="Cheng, Y., C. Sanderson, M. Jones, and K. Belov, Low MHC class II diversity in the Tasmanian devil (Sarcophilus harrisii). Immunogenetics, 2012. 64(7): p. 525–533."/>
  <meta name="citation_reference" content="Morris, K., J.J. Austin, and K. Belov, Low major histocompatibility complex diversity in the Tasmanian devil predates European settlement and may explain susceptibility to disease epidemics. Biology letters, 2013. 9(1): p. 20120900."/>
  <meta name="citation_reference" content="Miller, W., V.M. Hayes, A. Ratan, D.C. Petersen, N.E. Wittekindt, J. Miller, B. Walenz, J. Knight, J. Qi, and F. Zhao, Genetic diversity and population structure of the endangered marsupial Sarcophilus harrisii (Tasmanian devil). Proceedings of the National Academy of Sciences, 2011. 108(30): p. 12348–12353."/>
  <meta name="citation_reference" content="Willi, Y., J. Van Buskirk, and A.A. Hoffmann, Limits to the adaptive potential of small populations. Annu. Rev. Ecol. Evol. Syst., 2006. 37: p. 433–458."/>
  <meta name="citation_reference" content="Pearman, P.B. and T.W. Garner, Susceptibility of Italian agile frog populations to an emerging strain of Ranavirus parallels population genetic diversity. Ecology Letters, 2005. 8(4): p. 401–408."/>
  <meta name="citation_reference" content="O’Brien, S.J., M.E. Roelke, L. Marker, A. Newman, C. Winkler, D. Meltzer, L. Colly, J. Evermann, M. Bush, and D.E. Wildt, Genetic basis for species vulnerability in the cheetah. Science, 1985. 227(4693): p. 1428–1434."/>
  <meta name="citation_reference" content="Van Ranst, M., A. Fuse, P. Fiten, E. Beuken, H. Pfister, R.D. Burk, and G. Opdenakker, Human papillomavirus type 13 and pygmy chimpanzee papillomavirus type 1: comparison of the genome organizations. Virology, 1992. 190(2): p. 587–596."/>
  <meta name="citation_reference" content="Tachezy, R., G. Duson, A. Rector, A.B. Jenson, J.P. Sundberg, and M. Van Ranst, Cloning and genomic characterization of Felis domesticus papillomavirus type 1. Virology, 2002. 301(2): p. 313–321."/>
  <meta name="twitter:title" content="Characterisation of the faecal virome of captive and wild Tasmanian devils using virus-like particles metagenomics and meta-transcriptomics"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Background The Tasmanian devil is an endangered carnivorous marsupial threatened by devil facial tumour disease (DFTD). While research on DFTD has been extensive, little is known about the viruses present in devils, and whether any of these are of potential conservation relevance for this endangered species. Methods Using both metagenomics based on virus-like particle (VLP) enrichment and sequence-independent amplification (VLP metagenomics), and meta-transcriptomics based on bulk RNA sequencing, we characterised and compared the faecal viromes of captive and wild Tasmanian devils. Results A total of 54 devil faecal samples collected from captive (n = 2) and wild (n = 4) populations were processed for virome characterisation using both approaches. We detected many novel, highly divergent viruses, including vertebrate viruses, bacteriophage and other dietary associated plant and insect viruses. In total, 18 new vertebrate viruses, including novel sapelovirus, astroviruses, bocaviruses, papillomaviruses and gammaherpesvirus were identified, as well as known mammalian pathogens including rabbit haemorrhagic disease virus 2 (RHDV2). Captive devils showed significantly lower levels of viral diversity than wild devils. Comparison of the two methodological approaches revealed substantial differences in the number and types of viruses detected, with meta-transcriptomics mainly identifying RNA viruses, and VLP metagenomics largely identifying DNA viruses. Conclusion This study has greatly expanded our knowledge of eukaryotic viruses in the Tasmanian devil and provides important baseline information that will contribute to the conservation and captive management of this endangered species. In addition, our results showed that a combination of VLP metagenomics and meta-transcriptomics may be a more comprehensive approach to virome characterisation than either method alone. * SH : Stony Head MI : Maria Island BR : Buckbys Road wMW : wukalina/Mt William National Park VLP : Virus-like particles RHDV : Rabbit haemorrhagic disease virus DeSV : Devil sapelovirus DeAstV1 : DeRoV1 and -2 Devil rotavirus 1 and 2 DePBV1-6 : Devil picobirnavirus 1-6 DeBoV1 and -2 : Devil bocavirus 1 and 2 DePV1 and -2 : Devil papillomavirus 1 and 2 DePyV1 and -2 : Devil polyomavirus 1 and 2 DaHV-3 : Dasyurid herpesvirus 3 DeCV : Devil circovirus"/>
  <meta name="og-title" property="og:title" content="Characterisation of the faecal virome of captive and wild Tasmanian devils using virus-like particles metagenomics and meta-transcriptomics"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/443457v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Background The Tasmanian devil is an endangered carnivorous marsupial threatened by devil facial tumour disease (DFTD). While research on DFTD has been extensive, little is known about the viruses present in devils, and whether any of these are of potential conservation relevance for this endangered species. Methods Using both metagenomics based on virus-like particle (VLP) enrichment and sequence-independent amplification (VLP metagenomics), and meta-transcriptomics based on bulk RNA sequencing, we characterised and compared the faecal viromes of captive and wild Tasmanian devils. Results A total of 54 devil faecal samples collected from captive (n = 2) and wild (n = 4) populations were processed for virome characterisation using both approaches. We detected many novel, highly divergent viruses, including vertebrate viruses, bacteriophage and other dietary associated plant and insect viruses. In total, 18 new vertebrate viruses, including novel sapelovirus, astroviruses, bocaviruses, papillomaviruses and gammaherpesvirus were identified, as well as known mammalian pathogens including rabbit haemorrhagic disease virus 2 (RHDV2). Captive devils showed significantly lower levels of viral diversity than wild devils. Comparison of the two methodological approaches revealed substantial differences in the number and types of viruses detected, with meta-transcriptomics mainly identifying RNA viruses, and VLP metagenomics largely identifying DNA viruses. Conclusion This study has greatly expanded our knowledge of eukaryotic viruses in the Tasmanian devil and provides important baseline information that will contribute to the conservation and captive management of this endangered species. In addition, our results showed that a combination of VLP metagenomics and meta-transcriptomics may be a more comprehensive approach to virome characterisation than either method alone. * SH : Stony Head MI : Maria Island BR : Buckbys Road wMW : wukalina/Mt William National Park VLP : Virus-like particles RHDV : Rabbit haemorrhagic disease virus DeSV : Devil sapelovirus DeAstV1 : DeRoV1 and -2 Devil rotavirus 1 and 2 DePBV1-6 : Devil picobirnavirus 1-6 DeBoV1 and -2 : Devil bocavirus 1 and 2 DePV1 and -2 : Devil papillomavirus 1 and 2 DePyV1 and -2 : Devil polyomavirus 1 and 2 DaHV-3 : Dasyurid herpesvirus 3 DeCV : Devil circovirus"/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2018-10-15"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Characterisation of the faecal virome of captive and wild Tasmanian devils using virus-like particles metagenomics and meta-transcriptomics | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-133234 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <div id="sec-1" class="subsection">
     <p id="p-2">
      <strong>Background</strong> The Tasmanian devil is an endangered carnivorous marsupial threatened by devil facial tumour disease (DFTD). While research on DFTD has been extensive, little is known about the viruses present in devils, and whether any of these are of potential conservation relevance for this endangered species.
     </p>
    </div>
    <div id="sec-2" class="subsection">
     <p id="p-3">
      <strong>Methods</strong> Using both metagenomics based on virus-like particle (VLP) enrichment and sequence-independent amplification (VLP metagenomics), and meta-transcriptomics based on bulk RNA sequencing, we characterised and compared the faecal viromes of captive and wild Tasmanian devils.
     </p>
    </div>
    <div id="sec-3" class="subsection">
     <p id="p-4">
      <strong>Results</strong> A total of 54 devil faecal samples collected from captive (n = 2) and wild (n = 4) populations were processed for virome characterisation using both approaches. We detected many novel, highly divergent viruses, including vertebrate viruses, bacteriophage and other dietary associated plant and insect viruses. In total, 18 new vertebrate viruses, including novel sapelovirus, astroviruses, bocaviruses, papillomaviruses and gammaherpesvirus were identified, as well as known mammalian pathogens including rabbit haemorrhagic disease virus 2 (RHDV2). Captive devils showed significantly lower levels of viral diversity than wild devils. Comparison of the two methodological approaches revealed substantial differences in the number and types of viruses detected, with meta-transcriptomics mainly identifying RNA viruses, and VLP metagenomics largely identifying DNA viruses.
     </p>
    </div>
    <div id="sec-4" class="subsection">
     <p id="p-5">
      <strong>Conclusion</strong> This study has greatly expanded our knowledge of eukaryotic viruses in the Tasmanian devil and provides important baseline information that will contribute to the conservation and captive management of this endangered species. In addition, our results showed that a combination of VLP metagenomics and meta-transcriptomics may be a more comprehensive approach to virome characterisation than either method alone.
     </p>
    </div>
   </div>
   <div class="section" id="sec-5">
    <h2 class="">Background</h2>
    <p id="p-6">The Tasmanian devil (
     <em>Sarcophilus harrisii</em>) is the world’s largest extant carnivorous marsupial found only on the island state of Tasmania, Australia. Being predominantly a scavenger, the diet of devils largely comprises of carrion of mammals, such as wallabies, possums and kangaroos, although they are also known to consume and digest fish, insects, fruit and vegetation [
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>, 
     <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>].
    </p>
    <p id="p-7">Listed as endangered, the Tasmanian devil is facing the threat of extinction due to a contagious cancer, devil facial tumour disease (DFTD), that has caused drastic declines in wild devil populations by 77% since its discovery in 1996 [
     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>]. There are currently two forms of DFTD affecting devils; DFT1 and DFT2 [
     <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>]. In an attempt to save the species from extinction, an insurance population was established in 2006 to serve as an important source of animals for supplementing wild populations at risk of population crashes due to DFTD [
     <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>, 
     <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>]. While extensive research has focused on DFTD itself as well as devil genetic diversity and susceptibility to the disease over the past decade [
     <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>-
     <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>], our understanding of other disease threats to devils remains limited. Specifically, virological studies are scarce and limited to the identification of a single gammaherpesvirus (DaHV-2), for which captivity was identified as a significant risk factor [
     <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>]. A comprehensive characterisation of the viral communities inhabiting the Tasmanian devil is an essential step to improving our understanding of host-microbe relationships, and maximising health and conservation management of the species.
    </p>
    <p id="p-8">Comparative analysis of marsupial viruses with those from diverse vertebrate hosts, including eutherian mammals, birds and other vertebrates will also provide a deeper understanding of the phylogenetic history of the viruses infecting this evolutionary unique group of mammals [
     <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>, 
     <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>].
    </p>
    <p id="p-9">To date, the most widely used method for studying viral metagenomics relies on the enrichment of virus-like particles (VLP) and subsequent sequence-independent amplification prior to sequencing [
     <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>-
     <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>]. Removal of non-viral genomic host and bacterial nucleic acids and enrichment of VLP is often necessary for the detection of low-titre viruses [
     <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>, 
     <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>]. More recently, the use of RNA sequencing of total non-ribosomal RNA from environmental samples gave rise to viral meta-transcriptomics, which has been successfully applied to characterise the viromes of diverse invertebrate and vertebrate species [
     <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>, 
     <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>, 
     <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>]. To our knowledge, no studies have been conducted to compare these two approaches, although doing so would allow us to understand the detection capabilities and biases associated with these different nucleic acid extraction and sample treatment methods.
    </p>
    <p id="p-10">We characterised the faecal virome of wild and captive Tasmanian devil using both the metagenomics approach based on VLP enrichment and sequence-independent amplification (hereafter “VLP metagenomics”) and the meta-transcriptomics approach based on RNA sequencing of total non-ribosomal RNA (hereafter “meta-transcriptomics”). Our objectives were threefold: (1) to provide a comprehensive characterisation of the faecal virome of Tasmanian devils; (2) to compare the two virome characterisation approaches, highlighting their advantages and potential challenges; and (3) to compare the faecal viromes of wild and captive Tasmanian devils.</p>
   </div>
   <div class="section" id="sec-6">
    <h2 class="">Materials and Methods</h2>
    <div id="sec-7" class="subsection">
     <h3>Sample collection</h3>
     <p id="p-11">Faecal samples were collected from wild Tasmanian devils between September 2016 and June 2017 from four locations in Tasmania (
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>); 
      <strong>A</strong> Stony head (SH), 
      <strong>B</strong> Buckby’s Road (BR), 
      <strong>C</strong> Maria Island (MI), and 
      <strong>D</strong> wukalina/Mt William National Park (wMW), as well as from captive devils at two Australian mainland zoos in June and July 2017 (Zoo A and Zoo B). Wild devils were trapped overnight during routine monitoring trips by Save the Tasmanian Devil Program staff, using baited PVC-pipe traps [
      <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>]. Upon capture, each animal was subjected to a thorough health check, including body weight measurement, estimation of body condition score and observation for signs of disease. Fresh faecal samples were collected from either the traps, or from the hessian bag, during processing of the devils. For captive devils, faeces were collected from the enclosures shortly after defecation. All samples were stored in either liquid nitrogen or a portable -80
      <sup>°</sup>C freezer (Stirling Ultracold, Global Cooling Inc.) immediately after collection. After arriving at the laboratory, samples were separated into two aliquots to be used in the extraction of total RNA for meta-transcriptomics and the enrichment of VLP for VLP metagenomics.
     </p>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Map of Tasmania, Australia showing the four wild sampling sites." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1318907993" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Map of Tasmania, Australia showing the four wild sampling sites.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457/F1.medium.gif" width="403" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457/F1.medium.gif" width="403" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 1.</span>
       <p id="p-12" class="first-child">Map of Tasmania, Australia showing the four wild sampling sites.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-8" class="subsection">
     <h3>Meta-transcriptomics</h3>
     <div id="sec-9" class="subsection">
      <h4>Total RNA extraction, library preparation and sequencing</h4>
      <p id="p-13">Samples were disrupted and homogenized in 600 μl of lysis buffer with 1.44 mm ceramic beads using a Bead Ruptor Homogenizer (Omni International) at 5 m.s
       <sup>-1</sup> for 5 min. Total RNA was isolated using the Qiagen RNeasy Plus Mini Kit (Qiagen) following the manufacturer’s instructions. The extracted total RNAs were pooled based on their source locations at equal mass amount, with each pool containing 5-10 samples. The RNA pools were depleted of host and bacteria rRNA using a Ribo-Zero-Gold (Epidemiology) kit (Illumina) before constructing sequencing libraries using a TruSeq total RNA Library Preparation Kit (Illumina). Paired-end (75 bp) sequencing of each library was performed on a NextSeq500 HO platform (Illumina) at the Ramaciotti Centre for Genomics (Sydney, Australia).
      </p>
     </div>
     <div id="sec-10" class="subsection">
      <h4>VLP Metagenomics</h4>
      <div id="sec-11" class="subsection">
       <h5>VLP enrichment and nucleic acid extraction</h5>
       <p id="p-14">A second aliquot from each faecal sample was processed for the VLP metagenomics approach, as described previously with minor modifications [
        <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">17</a>].
       </p>
       <p id="p-15">VLPs were enriched as follows: Faecal suspensions consisting of 50 mg faeces in 700 μl of were homogenized for 1 min using the Bead Ruptor Homogenizer (Omni International) at 5 m.s
        <sup>- 1</sup> and centrifuged at 15000 × g for 3 min at 4 
        <sup>°</sup>C. The resulting supernatants were filtered through 0.45 μm membrane filters (Corning) for 1 min at 15000 × g. The filtrates were treated with 0.8 μl benzonase (Sigma Aldrich), 7 μl Turbo DNase (Ambion), 1 μl micrococcal nuclease (New England Biolab) and 14.88 μl 1 x Turbo DNase buffer (Ambion) at 37 °C for 2 h to digest free-floating nucleic acids. Viral DNA and RNA were then simultaneously extracted using the QIAamp Viral RNA Mini Kit (Qiagen) according to the manufacturer’s instructions, except without addition of carrier RNA to the lysis buffer [
        <a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">17</a>] .
       </p>
      </div>
      <div id="sec-12" class="subsection">
       <h5>Random amplification</h5>
       <p id="p-16">Extracted viral nucleic acids were pooled based on their source locations at equal mass amounts, using the same representative samples as the total RNA preparations for meta-transcriptomics described above. The pooled extractions were subjected to first and second strand synthesis and random PCR amplification for 22 cycles using the Complete Whole Transcriptome Amplification (WTA2) kit (Sigma-Aldrich). The denaturation temperature was increased from 70°C to 95°C to allow for the denaturation and amplification of both dsDNA and RNA [
        <a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">17</a>]. WTA2 PCR products were then purified using Agencourt AMPure XP beads (Beckman Coulter) prior to library preparation and sequencing.
       </p>
      </div>
      <div id="sec-13" class="subsection">
       <h5>Library preparation and Illumina Sequencing</h5>
       <p id="p-17">Sequencing libraries were constructed using the Nextera XT DNA Library Preparation Kit (Illumina) according to the manufacturer’s instructions with the following modifications as shown in [
        <a id="xref-ref-17-6" class="xref-bibr" href="#ref-17">17</a>] : (i) the tagmentation time was reduced to 4 min to increase fragment size, (ii) input DNA was increased to 1.2 ng/ μl
        <sup>-1</sup> and (iii) reagent quantities were halved to further increase the fragment size. Paired-end (100 bp) sequencing of each library was performed on a Hiseq2500 platform (Illumina) at the Ramaciotti Centre for Genomics (Sydney).
       </p>
      </div>
     </div>
     <div id="sec-14" class="subsection">
      <h4>Assembly and annotation</h4>
      <p id="p-18">Sequencing reads were de-multiplexed and quality trimmed with Trimmomatic [
       <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>] and assembled 
       <em>de novo</em> using Trinity [
       <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>]. Resulting contigs were compared against the non-redundant nucleotide (nt) and protein (nr) databases on GenBank using Blastn and Blastx, respectively, with an e-value cut-off at 1E-5. Blast search using Blastx was also conducted against a bespoke database containing all viral RNA-dependent RNA polymerase (RdRp) protein reference sequences downloaded from GenBank. Taxonomic information at the domain level (i.e. Eukarya, Bacteria, or Archaea), as well as for viruses, was first assigned based on the blastn results and then based on blastx results. Potential virus-related sequences were further categorized into families and orders based on their genetic similarity to their closest relatives and/or their phylogenetic position. Similarly, the assignment of viruses to the broader groups of their hosts (i.e. Eukarya, Bacteria, or Archaea) was based on their phylogenetic relationship to viruses with reliable host information obtained either using experimental or phylogenetic approaches. The genetic identity cut-off for host assignment was set to be 40% based on the most conserved proteins such as RdRp or DNA polymerases [
       <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>]. The threshold was set based on the intra-family diversity of most vertebrate-specific virus families/genera [
       <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>]. The assignment of vertebrate host was based on phylogenetic analyses, in which a potential devil associated virus is expected to either cluster within, or form a sister group to, an existing mammalian virus group.
      </p>
      <p id="p-19">To compare the abundance of each transcript/contig, we calculated the percentage of total reads in the library. The abundance of host transcripts/contigs was estimated by mapping reads against Tasmanian devil genome using Bowtie2 [
       <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>], whereas those of other organisms, namely, viruses, bacteria, archaea, and non-host eukaryotes, were estimated using the RSEM approach [
       <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>] implemented in Trinity.
      </p>
      <p id="p-20">For each virus, the genome sequence was further extended by merging related contigs. Gaps in the genome were filled either by mapping the reads against the existing scaffold or by RT-PCR and Sanger sequencing. Putative ORFs in viral genomes were predicted by the Geneious 8.1 software [
       <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>] or NCBI ORF finder, and annotated based on their similarity to previous published virus genomes.
      </p>
     </div>
     <div id="sec-15" class="subsection">
      <h4>Phylogenetic analysis</h4>
      <p id="p-21">Nucleotide sequences of complete or partial genomes and amino acid sequences from the conserved domain (e.g. the RNA-dependent RNA polymerase; RdRp) of the newly characterised viral sequences were aligned with those of reference viruses representative of the diversity of the corresponding virus family or species. The alignment was performed using the E-INS-I algorithm implemented in MAFFT (version 7) [
       <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>]. The quality of the alignments was subsequently assessed, and all ambiguously aligned regions were removed using TrimAl (version 1.2) [
       <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>]. Phylogenetic trees of aligned amino acid (all data sets with the exception of RHDV) or nucleotide (RHDV) sequences were then inferred using the maximum likelihood method implemented in PhyML version (3.0) [
       <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>], utilising the best-fit substitution model and the Subtree Pruning and Regrafting (SPR) branch-swapping algorithm.
      </p>
     </div>
     <div id="sec-16" class="subsection">
      <h4>Analyses of virome ecology</h4>
      <p id="p-22">Viral abundance tables (Additional file 2: Table S2 and Additional file 3: Table S3) were generated based on complete or near complete viral contigs and the percentage of reads mapped to them using Bowtie 2 [
       <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a>] in each library. QIIME (v 1.9) [
       <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>] was used to perform ecological and statistical analysis to compare viromes of different populations. Within-library virotype richness (alpha diversity measured using the 
       <em>Chao1</em> metric) and dissimilarity between libraries (beta diversity measured using the Euclidean metric) for both VLP metagenomics and meta-transcriptomics were calculated based on levels of viral abundance. The statistical significance of differences in alpha diversity was evaluated by the Monte Carlo method (999 permutations), with a null hypothesis that diversity is equal in all libraries with a significance threshold of α = 0.05. Levels of viral abundance were also used to produce heatmaps and dendrograms from hierarchical clustering. Principal coordinates analysis (PCoA) was performed on the Euclidean distance matrix as calculated in QIIME, and additional cluster analysis was conducted using K-Means clustering in R [
       <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>].
      </p>
     </div>
     <div id="sec-17" class="subsection">
      <h4>PCR confirmation and Sanger sequencing of rabbit haemorrhagic disease virus (RHDV)</h4>
      <p id="p-23">Contigs with high percentage of similarity (&gt;;97%) to rabbit haemorrhagic disease virus (RHDV) were detected in one of the meta-transcriptomics libraries. RHDV is used as a biocontrol for rabbits in Australia and causes fatal hepatitis in European rabbits (
       <em>Oryctolagus cuniculus</em>) and some hare species [
       <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>]. To confirm the detection of RHDV, RT-PCR was performed on individual faecal RNA extractions using the Qiagen OneStep Ahead RT-PCR kit (Qiagen) and primers from a previously validated Australian RHDV strain-specific PCR [
       <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>] as well as two additional primer sets manually designed based on the current meta-transcriptomics assembled contigs (Additional file 4: Table S4). PCR products were separated on 1.5% agarose gel (Bio-Rad Laboratories, Hercules, CA, USA) in 1x tris-acetate EDTA, and visualized using SYBR Safe DNA gel stain (Life Technologies, Carlsbad, CA, USA). Sanger sequencing of positive PCR products was performed at the Australian Genome Research Facility (Sydney, Australia). In addition, DNA was extracted from faecal samples using the ISOLATE Fecal DNA kit (Bioline, London, UK) and presence of rabbit DNA was tested using primers targeting a 110 bp region of the 
       <em>Oryctolagus cuniculus</em> 12S mitochondrial rRNA gene (Fwd: 5′□CAAAAGTAAGCTCAATTACCACCGTAc3′; Rev: n5′□ATAAGGGCTTTCGTATATTCGGAAc3′) [
       <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>]. Rabbit DNA extracted from rabbit liver obtained from collaborators at CSIRO using the Bioline Isolate II Genomic DNA kit (Bioline) was used as a positive control. Cleanup, primer trimming and sequence analysis of Sanger data was performed using DNASTAR Lasergene [
       <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>] and Geneious [
       <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">28</a>].
      </p>
     </div>
    </div>
   </div>
   <div class="section" id="sec-18">
    <h2 class="">Results</h2>
    <div id="sec-19" class="subsection">
     <h3>Overview of the devil virome</h3>
     <p id="p-24">Location Captive/Wild Meta-transcriptomics No. reads No. contigs VLP metagenomics No.reads No. contigs We characterised the faecal virome of six pools consisting of a total of 54 faecal samples collected from Tasmanian devils from four wild and two captive sites using both meta-transcriptomics and VLP metagenomics approaches. Meta-transcriptomic libraries resulted in 128 - 140 million reads per pool (793,038,436 reads in total), which were assembled 
      <em>de novo</em> into 196,919 – 358,327 contigs (
      <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Blast analyses of sequence reads from the meta-transcriptomic protocol revealed large proportions of reads from Bacteria (55.11 – 67.77%), and only 4.32 – 7.88% from Eukarya. Mapping reads to the Tasmanian devil genome revealed that 10.75 – 18.99 % of reads originated from the host. The percentage of reads related to Archaea was less than 0.02%, and for viruses between 0.68 – 1.16% (
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2a</a>).
     </p>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/616382/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/616382/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/616382" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 1</span>
       <p id="p-25" class="first-child">Library information of meta-transcriptomics and VLP metagenomics in the present study</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Overview of the Tasmanian devil faecal virome characterized by meta-transcriptomics and VLP metagenomics. Sequencing library/sampling site are represented at the bottom of the bar charts by SH for Stony Head, BR for Buckbys Road, wMW for wukalina/Mt William National Park, MI for Maria Island and Zoo A and B for the two captive populations. a) Proportions of all sequence reads in meta-transcriptomics (top) and VLP metagenomics (bottom) showing the proportions of reads belong to bacteria, eukaryote, archaea, viruses, host and unidentified. b) Proportions of RNA and DNA viruses detected by meta-transcriptomics (top) and VLP metagenomics (bottom). c) Virome composition and the proportions of viral groups in meta-transcriptomics (top) and VLP metagenomics (bottom). d) Estimated counts as calculated by RSEM (all six sequencing libraries from both VLP metagenomics and meta-transcriptomics combined and log transformed) to a selection of viruses, showing the differences in viruses detected by VLP metagenomics (blue) and meta-transcriptomics (orange)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1318907993" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Overview of the Tasmanian devil faecal virome characterized by meta-transcriptomics and VLP metagenomics. Sequencing library/sampling site are represented at the bottom of the bar charts by SH for Stony Head, BR for Buckbys Road, wMW for wukalina/Mt William National Park, MI for Maria Island and Zoo A and B for the two captive populations. &lt;;strong&gt;;a)&lt;;/strong&gt;; Proportions of all sequence reads in meta-transcriptomics (top) and VLP metagenomics (bottom) showing the proportions of reads belong to bacteria, eukaryote, archaea, viruses, host and unidentified. &lt;;strong&gt;;b)&lt;;/strong&gt;; Proportions of RNA and DNA viruses detected by meta-transcriptomics (top) and VLP metagenomics (bottom). &lt;;strong&gt;;c)&lt;;/strong&gt;; Virome composition and the proportions of viral groups in meta-transcriptomics (top) and VLP metagenomics (bottom). &lt;;strong&gt;;d)&lt;;/strong&gt;; Estimated counts as calculated by RSEM (all six sequencing libraries from both VLP metagenomics and meta-transcriptomics combined and log transformed) to a selection of viruses, showing the differences in viruses detected by VLP metagenomics (blue) and meta-transcriptomics (orange).&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457/F2.medium.gif" width="342" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457/F2.medium.gif" width="342" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 2.</span>
       <p id="p-26" class="first-child">Overview of the Tasmanian devil faecal virome characterized by meta-transcriptomics and VLP metagenomics. Sequencing library/sampling site are represented at the bottom of the bar charts by SH for Stony Head, BR for Buckbys Road, wMW for wukalina/Mt William National Park, MI for Maria Island and Zoo A and B for the two captive populations. 
        <strong>a)</strong> Proportions of all sequence reads in meta-transcriptomics (top) and VLP metagenomics (bottom) showing the proportions of reads belong to bacteria, eukaryote, archaea, viruses, host and unidentified. 
        <strong>b)</strong> Proportions of RNA and DNA viruses detected by meta-transcriptomics (top) and VLP metagenomics (bottom). 
        <strong>c)</strong> Virome composition and the proportions of viral groups in meta-transcriptomics (top) and VLP metagenomics (bottom). 
        <strong>d)</strong> Estimated counts as calculated by RSEM (all six sequencing libraries from both VLP metagenomics and meta-transcriptomics combined and log transformed) to a selection of viruses, showing the differences in viruses detected by VLP metagenomics (blue) and meta-transcriptomics (orange).
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-27">VLP metagenomics resulted in 26 – 49 million reads per pool (237,174,236 reads in total), which were assembled 
      <em>de novo</em> into 49,813 – 313,354 contigs (
      <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). In comparison to meta-transcriptomics, the VLP metagenomics protocol resulted in smaller proportions of non-viral nucleic acids (host and bacterial) and hence enriched viral nucleic acids (
      <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2a</a>). The proportion of viral reads using VLP metagenomics varied from 14.69 – 60.02%, while the proportion of reads mapped to non-viral components were 17 – 49.46% for Bacteria, 0.17–2.42% for Eukarya, 1.29 –18.67% for host and less than 0.01% for Archaea (
      <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2a</a>). For both approaches, a substantial proportion of reads had no significant similarity to other sequences in the databases on GenBank (10.22 – 29.20%).
     </p>
     <p id="p-28">Despite detecting a smaller proportion of virus-related sequences with meta-transcriptomics than with VLP metagenomics, viruses detected with the meta-transcriptomics approach fell into a wide range of viral groups, of which 49.87 – 97.51% had the closest hits to RNA viruses and 2.49 – 50.13% to DNA viruses. Conversely, for VLP metagenomics, over 95.54% of the virus-related sequences had closest hits to DNA viruses, and less than 5% were identified as RNA viruses (
      <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2b</a>). Meta-transcriptomics revealed high levels of viral diversity across all libraries, with the most abundant viral groups detected being the 
      <em>Caudovirales, Luteo-Sobemo, Narna-Levi, Partiti-Picobirna, Picorna-Calici</em> and 
      <em>Tombus-Noda</em> (
      <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2c</a>). Conversely, VLP metagenomics revealed relatively lower viral diversity across the same libraries, with 
      <em>Caudovirales</em> dominating significant proportions of the viral reads (69.89-99.49%). Other viral groups identified by VLP metagenomics, at much lower abundances, include the 
      <em>Microviridae, Circoviridae, Genomoviridae, Parvoviridae, Herpesviridae, Polyomaviridae and Papillomaviridae</em> (
      <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2c</a>). Overall, VLP metagenomics and meta-transcriptomics differed in the viruses detected as well as the expected counts (transcript abundance) as measured by the RSEM analysis (
      <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2d</a>).
     </p>
     <p id="p-29">The percentage of vertebrate viruses detected by meta-transcriptomics ranged between 0 –9.41% of the total viral reads, while large proportions of viral reads belonged to either non-vertebrate eukaryotic viruses (45.08 – 97.51%), which included plant viruses, insect viruses and mycoviruses, or bacteriophage (2.48 – 48.91%) from the families 
      <em>Siphoviridae, Podoviridae, Myoviridae and Microviridae</em>. In the VLP metagenomics data set, the percentage of reads associated with vertebrate viruses was also small, ranging between 0.04 – 0.84%, while the percentages of bacteriophage and other eukaryotic viruses ranged between 79.17 – 99.91% and 0.04 – 19.99%, respectively. Detailed information on all vertebrate viruses identified is presented in Additional file 1: Table S1.
     </p>
    </div>
   </div>
   <div class="section" id="sec-20">
    <h2 class="">Detection of viruses previously identified in other mammalian hosts (Rabbit haemorrhagic disease virus and Torovirus)</h2>
    <p id="p-30">Rabbit haemorrhagic disease virus (RHDV) is a calicivirus in the genus 
     <em>Lagovirus</em> [
     <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a>]. All lagoviruses have a single-stranded, positive-sense RNA genome of approximately 7.5 kb and share a similar genome structure comprising two open reading frames (ORFs). Using meta-transcriptomics, we detected genomes with high nucleotide and amino acid similarity (&gt;;98 %) to RHDV in one of the wild devil meta-transcriptomic libraries (BR), with genome coverage of 98.1%. Phylogenetic analysis based on the nucleotide sequences of the major capsid and non-structural protein genes revealed that the RHDV detected here clustered with RHDV strain GI.2 (also called RHDV2) (
     <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3a and b</a>), which was first detected in Australia in May 2015 and has since became the dominant circulating strain nationwide. RHDV-specific RT-PCR and sequencing confirmed the presence of RHDV2 in 4 of the 9 devil faecal samples from the BR meta-transcriptomics pool. In addition, no rabbit associated genes were detected during the initial sequence analysis. Additional PCR targeting a short fragment of rabbit mtDNA (&lt;;300bp) also did not detect any rabbit DNA in the original faecal samples from BR. Further RT-PCR and sequencing performed on the faecal RNA extractions from the remaining pools confirmed the presence of RHDV in 1 of 10 devils from wMW, 2 of 9 devils from Zoo A and 1 of 9 from Zoo B. One of the 4 additional RHDV positive samples, from Zoo A contained detectable levels of rabbit mtDNA as confirmed by PCR.
    </p>
    <p id="p-31">Torovirus is a genus of viruses in the viral family 
     <em>Coronaviridae</em> with a linear, positive-sense RNA genome of about 28 – 28.5kb [
     <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>]. We identified the complete viral genome (28,463 bp) of a novel torovirus most closely related to Bovine torovirus (Bredavirus) in one of the meta-transcriptomic libraries (wMW). BLAST search revealed that the torovirus detected here shared 96% nucleotide similarity and 97% amino acid similarity with Bovine torovirus, a respiratory and enteric pathogen of cattle that causes gastroenteritis and severe diarrhoea, particularly in young calves. We determined the full genome structure of the novel torovirus strain, which included ORF 1a and ORF 1b encoding the two polyproteins (pp1a and pp1ab), ORF 2 encoding for the spike protein (S), ORF 3 encoding the membrane protein (M), ORF 4 encoding the hemagglutinin-esterase protein (HE), and ORF 5 encoding the nucleocapsid protein (N) [
     <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>].
    </p>
    <div id="F3" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Maximum likelihood phylogenetic trees of viruses detected that were previously identified in other mammalian hosts. a) phylogenetic analysis of representative RHDV strains based on the 1414 nt nucleotide sequence of major capsid protein. b) phylogenetic analysis of representative RHDV strains based on the 5890 nt nucleotide sequence of the non-structural protein. c) phylogenetic analysis based on the 4762 aa spike protein of the novel torovirus strain detected here with other previously identified toroviruses. All trees are mid-point rooted and scaled to either the number of amino acid substitution or nucleotide substitutions per site based on the nature of alignment. The viral sequences detected in this study are shown in red in each tree." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1318907993" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Maximum likelihood phylogenetic trees of viruses detected that were previously identified in other mammalian hosts. &lt;;strong&gt;;a)&lt;;/strong&gt;; phylogenetic analysis of representative RHDV strains based on the 1414 nt nucleotide sequence of major capsid protein. &lt;;strong&gt;;b)&lt;;/strong&gt;; phylogenetic analysis of representative RHDV strains based on the 5890 nt nucleotide sequence of the non-structural protein. &lt;;strong&gt;;c)&lt;;/strong&gt;; phylogenetic analysis based on the 4762 aa spike protein of the novel torovirus strain detected here with other previously identified toroviruses. All trees are mid-point rooted and scaled to either the number of amino acid substitution or nucleotide substitutions per site based on the nature of alignment. The viral sequences detected in this study are shown in red in each tree.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457/F3.medium.gif" width="315" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457/F3.medium.gif" width="315" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 3.</span>
      <p id="p-32" class="first-child">Maximum likelihood phylogenetic trees of viruses detected that were previously identified in other mammalian hosts. 
       <strong>a)</strong> phylogenetic analysis of representative RHDV strains based on the 1414 nt nucleotide sequence of major capsid protein. 
       <strong>b)</strong> phylogenetic analysis of representative RHDV strains based on the 5890 nt nucleotide sequence of the non-structural protein. 
       <strong>c)</strong> phylogenetic analysis based on the 4762 aa spike protein of the novel torovirus strain detected here with other previously identified toroviruses. All trees are mid-point rooted and scaled to either the number of amino acid substitution or nucleotide substitutions per site based on the nature of alignment. The viral sequences detected in this study are shown in red in each tree.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-33">Based on the phylogenetic analysis of the spike protein amino acid (aa) sequence (4762 aa), clustering of the novel torovirus strain with other toroviruses isolated from cows in the USA, Japan and Europe indicated a bovine origin, although this will need to be confirmed with wider sampling (
     <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3c</a>).
    </p>
   </div>
   <div class="section" id="sec-21">
    <h2 class="">Detection and characterisation of novel marsupial-associated viruses (
     <em>Picornaviridae, Astroviridae, Reoviridae, Picobirnaviridae, Parvoviridae, Papillomaviridae, Herpesviridae, Polyomaviridae</em> and 
     <em>Circoviridae</em>)
    </h2>
    <p id="p-34">The order 
     <em>Picornavirales</em> includes a diverse group of vertebrate-infecting RNA viruses, including enteroviruses and rhinoviruses, that can cause a wide range of diseases, such as poliomyelitis, hand, foot and mouth disease, encephalitis, respiratory tract infections and the common cold [
     <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>]. Members from the order are small, non-enveloped, positive-sense, single-stranded RNA viruses about 7-8.8 kb in size [
     <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>]. There are currently five recognized families: 
     <em>Picornaviridae, Secoviridae, Iflaviridae, Marnaviridae</em> and 
     <em>Dicistroviridae</em> [
     <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">39</a>]. We identified a novel member of the 
     <em>Picornaviridae</em> in one of the meta-transcriptomic libraries (wMW) and obtained the complete genome of 8015 bp. According to the ICTV, members of a 
     <em>Picornavirus</em> genus should share at least 40% amino acid sequence identity in the polyprotein region [
     <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">39</a>]. The encoded 2396 aa polyprotein exhibited 45.5% amino acid similarity to Simian sapelovirus, placing it in the genus 
     <em>Sapelovirus</em>. We have provisionally named this newly identified virus Devil sapelovirus (DeSV). Phylogenetic analysis based on the amino acid sequence of the RdRp domains showed that DeSV formed a sister lineage to sapeloviruses identified from eutherian mammals (i.e. Porcine and Simian sapelovirus) (
     <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4a</a>).
    </p>
    <div id="F4" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Phylogenetic analyses and genomic structures of the RNA viruses identified in the faeces of Tasmanian devils. All phylogenetic analyses were performed based on the amino acid sequence of the RdRp. a) Devil sapelovirus, b) Devil astrovirus, c) Devil rotavirus 1 and 2 and d) Devil picobirnavirus 1-6. For Devil astrovirus where the whole genome sequence was obtained, the genomic structure was also included and are shown below the corresponding phylogenetic tree. Predicted ORFs of these genomes are labelled with information of the potential protein or protein domain they encode. All trees are mid-point rooted and scaled to the number amino acid substitutions per site. The newly discovered viruses are shown in red in each phylogenetic tree." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1318907993" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Phylogenetic analyses and genomic structures of the RNA viruses identified in the faeces of Tasmanian devils. All phylogenetic analyses were performed based on the amino acid sequence of the RdRp. &lt;;strong&gt;;a)&lt;;/strong&gt;; Devil sapelovirus, &lt;;strong&gt;;b)&lt;;/strong&gt;; Devil astrovirus, &lt;;strong&gt;;c)&lt;;/strong&gt;; Devil rotavirus 1 and 2 and &lt;;strong&gt;;d)&lt;;/strong&gt;; Devil picobirnavirus 1-6. For Devil astrovirus where the whole genome sequence was obtained, the genomic structure was also included and are shown below the corresponding phylogenetic tree. Predicted ORFs of these genomes are labelled with information of the potential protein or protein domain they encode. All trees are mid-point rooted and scaled to the number amino acid substitutions per site. The newly discovered viruses are shown in red in each phylogenetic tree.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457/F4.medium.gif" width="322" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457/F4.medium.gif" width="322" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 4.</span>
      <p id="p-35" class="first-child">Phylogenetic analyses and genomic structures of the RNA viruses identified in the faeces of Tasmanian devils. All phylogenetic analyses were performed based on the amino acid sequence of the RdRp. 
       <strong>a)</strong> Devil sapelovirus, 
       <strong>b)</strong> Devil astrovirus, 
       <strong>c)</strong> Devil rotavirus 1 and 2 and 
       <strong>d)</strong> Devil picobirnavirus 1-6. For Devil astrovirus where the whole genome sequence was obtained, the genomic structure was also included and are shown below the corresponding phylogenetic tree. Predicted ORFs of these genomes are labelled with information of the potential protein or protein domain they encode. All trees are mid-point rooted and scaled to the number amino acid substitutions per site. The newly discovered viruses are shown in red in each phylogenetic tree.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-36">Astroviruses are single-stranded positive-sense RNA viruses from the family 
     <em>Astrovirida</em>e, that can cause diarrhoea and gastroenteritis in infected hosts [
     <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>, 
     <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>]. The family has a genome size of 6.4 to 7.3kb and has been identified in a broad range of vertebrate hosts including humans, other mammals, reptiles, amphibians, fish and birds [
     <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>]. We detected astrovirus-related sequences in 5 of the 6 meta-transcriptomic libraries (SH, wMW, BR, Zoo A and Zoo B). We identified one complete and one near-complete genome sequence with 81.4% pairwise nucleotide identity, denoting 2 separate strains of a single astrovirus species, which we have tentatively named Devil astrovirus 1 (DeAstV1) [
     <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>]. The DeAstV1 identified here has a genome structure typical of other astroviruses, with 3 putative open reading frames (ORF1a, ORF1b and ORF2) each encoding for the protease, RdRp and capsid, respectively. In addition, we found a ribosomal frameshift motif (AAAAAAC) within the ORF1a/1b overlap region. Phylogenetic analysis based on the conserved RdRp domain showed that DeAstV1 formed a distinct cluster that is more closely related to astroviruses of mammalian hosts (Mamastroviruses) than those of avian hosts (Avastroviruses) (
     <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4b</a>).
    </p>
    <p id="p-37">Rotaviruses of the family 
     <em>Reoviridae</em> are non-enveloped, double-stranded RNA viruses that infect only vertebrates through faecal-oral transmission [
     <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>]. They are the most common cause of acute viral gastroenteritis and can be found in a wide range of vertebrate hosts, predominately terrestrial mammals and birds. We detected rotavirus sequences in 3 of 6 meta-transcriptomic libraries (SH, BR and wMW, all are wild sites). Among them, we were able to identify 2 distinct segments (3481 bp and 3479 bp in length) encoding rotavirus VP1 (i.e. RdRp) from the two meta-transcriptomics libraries (BR and wMW, respectively). The two RdRp sequences shared &gt;;90 % nucleotide identity with each other, indicative of two different strains belonging to the same species. In addition, a contig encoding a partial rotavirus VP1 of 282 aa sharing 51 % sequence similarity with rotavirus H was also detected in meta-transcriptomics library wMW. We provisionally named the two viruses Devil rotavirus 1 and Devil rotavirus 2 (DeRoV1 and -2). The VP1 of DeRoV1 shared the highest amino acid similarity of ~51% with Rotavirus G, while DeRoV2 shared the highest amino acid similarity of 44% with Rotavirus H. Phylogenetic analysis with other previously characterised rotavirus species based on VP1 suggested that DeRoV1 and 2 form a distinct cluster that is most closely related to the cluster that contains Rotavirus G chicken, Human rotavirus B, Rotavirus I and Adult diarrheal rotavirus strain, which are associated with avian and mammalian hosts (
     <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4c</a>).
    </p>
    <p id="p-38">Picobirnaviruses (PBV) are small, positive-stranded non-enveloped RNA viruses with abroad host ranges found in the faeces of many mammal, bird and reptile species [
     <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>]. We detected picobirnavirus sequences that encoded complete and partial viral RdRp (330 – 557 aa) in 4 of the 6 meta-transcriptomics libraries, all of which comprised of faecal samples collected from wild devils (BR, wMW, SH and MI). However, picobirnavirus sequences detected in library MI were too short to be phylogenetically informative and were discarded in the phylogenetic analysis. We provisionally named the novel picobirnaviruses detected here Devil picobirnavirus 1, 2, 3, 4, 5, and 6 (DePBV1 -6) with two separate strains occurring in DePBV1 and DePBV5. Phylogenetic analysis based on the RdRp domain of these viruses with other representative members from the family showed that the 6 novel Devil picobirnaviruses are highly diverse and widely distributed across the phylogenetic tree (
     <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4d</a>).
    </p>
    <p id="p-39">
     <em>Parvoviridae</em> are a family of small, non-enveloped, ssDNA viruses [
     <a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">39</a>]. Members of 
     <em>Parvoviridae</em> are associated with a number of diseases, including erythema infectiosum in humans, hepatic and enteric diseases across a broad host range [
     <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>]. We identified two new members of the vertebrate-associated sub-family 
     <em>Parvovirinae</em> in the faeces of Tasmanian devils from two of the VLP metagenomics libraries (MI and BR). We recovered partial and near-complete protein sequences sharing ~50% identity to California sea lion bocavirus and Porcine bocavirus, respectively. Two bocavirus-related sequences detected here shared &gt;;97% amino acid sequence similarity, denoting two separate strains of the same species, which we have provisionally named Devil bocavirus 1 (DeBoV1). A third bocavirus-related sequence was also identified in VLP metagenomics library BR, sharing 71.13% amino acid similarity with DeBoV1. We provisionally named this second virus Devil bocavirus 2 (DeBoV2). Phylogenetic analysis of DeBoV1 and 2 in the context of other representative viruses from the 
     <em>Parvoviridae</em> family further confirmed the clustering of DeBoVs within the diversity of mammalian bocaviruses, although the branching order involved DeBoVs and other members of bocavirus remain unresolved with current topology (
     <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5a</a>).
    </p>
    <div id="F5" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Phylogenetic analyses and genomic structures of the DNA viruses identified in the faeces of Tasmanian devils. a) Devil bocavirus 1 and 2 based on the amino acid sequences of the NS1 protein. b) Devil papillomavirus 1 and 2 based on the amino acids of the E1 protein. c) Devil polyomavirus 1 and 2 based on the amino acid sequences of the LTAg and VP1 proteins. For devil polyomavirus 1, the whole genome sequence was obtained and the genomic structure is shown next to the VP1 tree. Predicted ORFs of these genomes are labelled with information of the potential protein or protein domain they encode. d) Dasyurid herpesvirus 3 based on the amino acid sequence of the DNA polymerase. e) Devil circovirus 1 based on the amino acid sequence of the replicase protein. All trees are mid-point rooted and scaled to the number of amino acid substitutions per site. The newly discovered viruses are shown in red." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1318907993" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Phylogenetic analyses and genomic structures of the DNA viruses identified in the faeces of Tasmanian devils. &lt;;strong&gt;;a)&lt;;/strong&gt;; Devil bocavirus 1 and 2 based on the amino acid sequences of the NS1 protein. &lt;;strong&gt;;b)&lt;;/strong&gt;; Devil papillomavirus 1 and 2 based on the amino acids of the E1 protein. &lt;;strong&gt;;c)&lt;;/strong&gt;; Devil polyomavirus 1 and 2 based on the amino acid sequences of the LTAg and VP1 proteins. For devil polyomavirus 1, the whole genome sequence was obtained and the genomic structure is shown next to the VP1 tree. Predicted ORFs of these genomes are labelled with information of the potential protein or protein domain they encode. &lt;;strong&gt;;d)&lt;;/strong&gt;; Dasyurid herpesvirus 3 based on the amino acid sequence of the DNA polymerase. &lt;;strong&gt;;e)&lt;;/strong&gt;; Devil circovirus 1 based on the amino acid sequence of the replicase protein. All trees are mid-point rooted and scaled to the number of amino acid substitutions per site. The newly discovered viruses are shown in red.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457/F5.medium.gif" width="320" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457/F5.medium.gif" width="320" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 5.</span>
      <p id="p-40" class="first-child">Phylogenetic analyses and genomic structures of the DNA viruses identified in the faeces of Tasmanian devils. 
       <strong>a)</strong> Devil bocavirus 1 and 2 based on the amino acid sequences of the NS1 protein. 
       <strong>b)</strong> Devil papillomavirus 1 and 2 based on the amino acids of the E1 protein. 
       <strong>c)</strong> Devil polyomavirus 1 and 2 based on the amino acid sequences of the LTAg and VP1 proteins. For devil polyomavirus 1, the whole genome sequence was obtained and the genomic structure is shown next to the VP1 tree. Predicted ORFs of these genomes are labelled with information of the potential protein or protein domain they encode. 
       <strong>d)</strong> Dasyurid herpesvirus 3 based on the amino acid sequence of the DNA polymerase. 
       <strong>e)</strong> Devil circovirus 1 based on the amino acid sequence of the replicase protein. All trees are mid-point rooted and scaled to the number of amino acid substitutions per site. The newly discovered viruses are shown in red.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-41">Papillomaviruses from the family 
     <em>Papillomaviridae</em> are small, nonenveloped, icosahedral viruses with a circular double-stranded DNA genome of about 8 kbp in size [
     <a id="xref-ref-39-5" class="xref-bibr" href="#ref-39">39</a>], and are associated with the development of benign and malignant tumours [
     <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>, 
     <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>]. We identified fragmented genome of two novel species of papillomavirus from a single VLP metagenomics library (MI), among which we were able to retrieve two longer fragments of 1225 and 1335 bp in length, respectively, both of which encode partial E1 protein, an ATP-dependent DNA helicase required for viral replication [
     <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>]. Since the two fragments shared approximately 64% similarity to each other, it suggests the presence of two distinct papillomavirus species, which we have tentatively designated as Devil papillomavirus 1 and 2 (DePV1 and -2). Phylogenetic analysis based on the E1 protein showed that DePV1 and 2 form a distinct cluster with 
     <em>Bettongia penicillata</em> papillomavirus type 1 (BpPV1) isolated from the woylie (
     <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5b</a>). While the marsupial papillomaviruses viruses are clustered together in the phylogenetic tree, their relationship with viruses identified from eutherian mammals remain unresolved.
    </p>
    <p id="p-42">Polyomaviruses from the family 
     <em>Polyomaviridae</em> are small, nonenveloped, circular DNA viruses about 5 kb in size that have been identified in a wide variety of mammalian and avian hosts [
     <a id="xref-ref-39-6" class="xref-bibr" href="#ref-39">39</a>, 
     <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>]. We detected two novel polymaviruses from the faeces of Tasmanian devils in 3 VLP metagenomics libraries (MI, wMW and Zoo B), which we have tentatively designated Devil polyomavirus 1 and 2 (DePyv1 and -2), respectively. We recovered the complete circular genome of 4894 bp for DePyv1 (
     <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5c</a>) and partial gene sequence (2251 bp) of the large T antigen (LTAg) protein for DePyv2. Phylogenetic analyses revealed strikingly different evolutionary histories for the structural and non-structural parts of the genome (
     <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5c</a>), indicative of recombination [
     <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>]. In the LTAg phylogeny, DePyV1 and -2 formed a distinct lineage with another marsupial virus Bandicoot papillomatosis carcinomatosis virus type 2 (BPCV-2), which in turn is clustered with polyomaviruses of avian hosts (
     <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5c</a>). In contrast, in the VP1 phylogeny, the marsupial viruses showed no close relationship with the avian viruses. Interestingly, the Bandicoot papillomatosis carcinomatosis viruses (BPCV1 and -2) only showed close relationship to DePyV1 at the LTAg region but not the VP1 region. Indeed, both BPCV1 and -2 are hybrid viruses between the family 
     <em>Papillomaviridae</em> and 
     <em>Polyomaviridae</em> [
     <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>, 
     <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>]. While its non-structural genes (i.e. large T and small T antigen genes) are related to polyomavirus, its structural genes (i.e. L1 and L2 protein genes) are related to papillomavirus [
     <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">54</a>].
    </p>
    <p id="p-43">
     <em>Herpesviridae</em> are a large family of DNA viruses with large linear, dsDNA genome of about 120-240 kb, and that comprise 3 subfamilies; 
     <em>Alphaherpesvirinae</em>, 
     <em>Betaherpesvirinae</em> and 
     <em>Gammaherpesvirinae</em> [
     <a id="xref-ref-39-7" class="xref-bibr" href="#ref-39">39</a>]. In one of the captive VLP metagenomics libraries (Zoo A), we identified more than 70 contigs matching different regions of a herpesvirus genome, which totaled 62,821 bp in length and included partial gene sequences of the DNA polymerase (575 aa), major DNA binding protein (465 aa), helicase (396 aa), glycoprotein M (378 aa) and H (427 aa), and major capsid protein (358 aa), amongst others. We tentatively named the novel herpesvirus Dasyurid herpesvirus 3 (DaHV-3). Phylogenetic analysis based on these non-structural and structural proteins suggested that DaHV-3 clustered with other gammaherpesviruses from the 
     <em>Gammaherpesvirinae</em> subfamily (Additional file 5: Figure S1 and Additional file 6: Figure S2). Further analysis of the 
     <em>Gammaherpesvirinae</em> phylogenetic structure based on the DNA polymerase showed that DaHV-3 forms a distinct lineage that is most closely related to Bovine gammaherpesvirus 6 and Common bottlenose dolphin gammaherpesvirus 1 strain Sarasota (
     <a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5d</a>). The previously characterised Dasyurid herpesvirus 2 (DaHV-2) isolated from Tasmanian devils [
     <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>] could not be included in the phylogenetic analysis because there are no available sequences from the same genomic regions. A BLASTx search of the DNA polymerase showed that DaHV-3 exhibited the greatest amino acid similarity (93.3%) with the previously characterised Macropodid herpesvirus 3(MaHV-3) [
     <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">12</a>], whose DNA polymerase amino acid sequence was also too short (&lt;;50% of the other representative herpesviruses) to be included in the phylogenetic analysis.
    </p>
    <p id="p-44">Circoviruses from the family 
     <em>Circoviridae</em> have a circular ssDNA genome which range from 1.7 to 2.3 kb in size [
     <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>]. Here, we identified circovirus-related sequences and recovered the partial replicase gene sequences (899 bp) in one of the wild devil metagenomes (SH) and tentatively named it Devil circovirus (DeCV). Phylogenetic analysis based on the Rep proteins of the novel DeCV and representative strains of circoviruses and cycloviruses suggested that DeCV is clustered with circoviruses previously isolated from bats and pigs, sharing the highest sequence identity (62%) with bat circovirus (AIF76281.1) (
     <a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5e</a>).
    </p>
   </div>
   <div class="section" id="sec-22">
    <h2 class="">Other viruses: plant and insect viruses, and bacteriophage</h2>
    <p id="p-45">In both the VLP metagenomics and meta-transcriptomics analyses, large proportions of viral sequence reads could be attributed to viruses that infect plants, insects and bacteria. Indeed, bacteriophage sequences from the order 
     <em>Caudovirale</em>s were detected in all libraries and made up over 90% of all viral VLP metagenomic reads and up to 48.91% of the meta-transcriptomic reads. Sequences related to newly identified arthropod viruses, such as Wuhan fly virus and Wuhan mosquito virus were also detected. Most of the insect viruses detected belong to RNA virus groups the 
     <em>Bunyavirales</em>, the 
     <em>Mononegavirales</em>, and the 
     <em>Chuviridae</em>, as well as DNA virus subfamily 
     <em>Densovirinae</em> from the 
     <em>Parvoviridae</em> family. The detection of these arthropod virus related sequences indicates the possible ingestion of arthropods and/or the environmental contamination of faeces. Despite being a carnivorous species, sequences related to various plant and fungal viruses were also observed in all libraries, including various sobemoviruses, tombusviruses and mitoviruses. Herbivorous species such as wallabies often make up a large proportion of devil’s diet, and the presence of plant viruses in these prey items might be easily detected in the faeces of devil through deep sequencing, especially if the intestines of the prey were consumed [
     <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>]. Nonetheless, devils have also been observed to consume vegetation, and the abundance of highly diverse eukaryotic and vertebrate viruses that are thought to be associated with diverse host taxa are reflective of devils’ overall generalist diet [
     <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">1</a>].
    </p>
   </div>
   <div class="section" id="sec-23">
    <h2 class="">Virome ecology: comparison between devil populations</h2>
    <p id="p-46">Within-library viral diversity (alpha diversity) as characterised by our meta-transcriptomics approach was significantly different between captive and wild populations (
     <em>p</em> &lt;; 0.05). In general, captive populations had lower diversity in their faecal viromes compared to wild populations, as measured using the chao1 metric. However, when considering bacteriophage alone, we did not find any significant associations between devil populations (captive or wild) in either diversity or abundance.
    </p>
    <p id="p-47">The meta-transcriptomics analysis of the virome of Maria Island (MI) devils displayed a similar level of overall viral diversity to the two captive populations (Zoo A and Zoo B), and was lower than that found in the three other wild populations (wMW, BR and SH) (
     <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6a</a>). Conversely, alpha diversity as characterized from VLP metagenomics data did not differ significantly between libraries (
     <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6b</a>). Cluster analysis indicated that in the meta-transcriptomics data, wild and captive devils fell into two distinct clusters, while in VLP metagenomics data, BR formed its own cluster and the remaining populations formed a second cluster (
     <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6a</a> and b).
    </p>
    <div id="F6" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Heatmaps (left) showing the hierarchical clustering and percentage of reads from each library mapping to various viral groups, and principal coordinate analysis (PCoA) plots (right) showing the similarity relations among libraries based on Euclidean distances as seen in a) meta-transcriptomics and b) VLP metagenomics." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1318907993" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Heatmaps (left) showing the hierarchical clustering and percentage of reads from each library mapping to various viral groups, and principal coordinate analysis (PCoA) plots (right) showing the similarity relations among libraries based on Euclidean distances as seen in &lt;;strong&gt;;a)&lt;;/strong&gt;; meta-transcriptomics and &lt;;strong&gt;;b)&lt;;/strong&gt;; VLP metagenomics.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457/F6.medium.gif" width="439" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457/F6.medium.gif" width="439" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/15/443457/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 6.</span>
      <p id="p-48" class="first-child">Heatmaps (left) showing the hierarchical clustering and percentage of reads from each library mapping to various viral groups, and principal coordinate analysis (PCoA) plots (right) showing the similarity relations among libraries based on Euclidean distances as seen in 
       <strong>a)</strong> meta-transcriptomics and 
       <strong>b)</strong> VLP metagenomics.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
   </div>
   <div class="section" id="sec-24">
    <h2 class="">Discussion</h2>
    <p id="p-49">We provide the first comprehensive characterisation of the faecal virome of an endangered marsupial species, the Tasmanian devil. Using both VLP metagenomics and meta-transcriptomics, we identified a huge diversity of viruses in the faeces of devils, including vertebrate viruses, bacteriophage and other eukaryotic viruses. Novel viruses identified as potentially marsupial-associated – including, sapelovirus, rotaviruses, picobirnaviruses, astrovirus, bocaviruses, papillomaviruses, herpesvirus and polyomaviruses – were detected and have greatly expanded our current knowledge of viruses that are found within this unique group of mammals.</p>
    <p id="p-50">Our comparison of VLP metagenomics and meta-transcriptomics approaches revealed marked differences in terms of the viruses detected. In general, VLP metagenomics mainly detected DNA viruses, while meta-transcriptomics detected both DNA and RNA viruses, although the DNA viruses detected with meta-transcriptomics were limited to those with relatively high abundance (
     <a id="xref-fig-2-8" class="xref-fig" href="#F2">Fig 2d</a>). A high abundance level is often indicative of an active viral infection, during which DNA viruses are transcribed into RNA intermediates that can be readily detected by RNA sequencing [
     <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>]. Conversely, RNA viruses identified in meta-transcriptomic were rarely detected in VLP metagenomics, even if they were highly abundant based on the RSEM estimated counts.
    </p>
    <p id="p-51">The use of VLP enrichment and sequence-independent amplification in VLP metagenomics increased the number of viral reads in each library compared to the meta-transcriptomic approach. However, the viral compositions of all six VLP metagenomes were highly skewed towards DNA viruses, particularly DNA bacteriophage from the order 
     <em>Caudovirales</em>. This is consistent with a previous study comparing various enrichment methods, in which bacteriophage accounted for &gt;;80% of all reads in all of the enrichment methods tested, but &lt;;5% when no enrichment steps were incorporated [
     <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>]. Despite being able to substantially increase the total number of viral reads in the metagenomes, sequence-independent amplification is bias prone, resulting in fewer viruses being detected and lower genome coverage due to preferential amplification of certain sequences [
     <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>-
     <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a>]. However, the over-representation of bacteriophage found here could also be attributed to the fact that they make up the bulk of the virobiota in the gut, which is dominated by bacteria [
     <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a>, 
     <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a>]. Regardless of its known bias [
     <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">58</a>, 
     <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">59</a>], VLP metagenomics still holds merit for use in virome characterisation due to its ability to identify low abundance DNA viruses, which is especially relevant for dormant or non-active viruses.
    </p>
    <p id="p-52">Compared to VLP metagenomics, meta-transcriptomics is non-viral specific, requires less sample processing, and reveals the entire transcriptome within a sample [
     <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a>, 
     <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a>]. Omitting the need for VLP enrichment and additional sample processing, the likelihood of biased detection is plausibly reduced in meta-transcriptomics. In this study, the proportions of viral reads sequenced by meta-transcriptomics were less than 2% per library, but the numbers of viral groups detected were significantly higher than those detected in VLP metagenomics, which included both RNA and DNA viruses.
    </p>
    <p id="p-53">Importantly, then, our results show that the taxonomic compositions of viral communities as revealed by VLP metagenomics and meta-transcriptomics were not interchangeable and neither of the approaches was able to detect all viruses present. However, these two approaches were complementary to one-another, and an integrated approach using both VLP metagenomics and meta-transcriptomics will prove to be a powerful tool for obtaining a complete overview of both the taxonomic and functional profiles of viral communities in a sample.</p>
    <p id="p-54">Ecological analysis of virome composition and diversity revealed significant differences between captive and wild devil populations, especially when characterised using meta-transcriptomics. The two captive populations displayed lower levels of viral diversity compared to the four wild populations. This loss of diversity within the captive populations is consistent with trends previously observed in the gut bacteriome, where captive devils also exhibited lower bacterial diversity compared to wild devils [
     <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a>]. The microbiome is dynamic and sensitive to changes in the environment. Hence, the extreme lifestyle and diet changes that often occur in captivity are likely to affect the viral communities of devils. Interestingly, Maria Island devils had viromes that are more similar to the two captive populations. Maria Island is a 115 km
     <sup>2</sup> island off the east coast of Tasmania (
     <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig 1</a>) and is home to approximately 100 free ranging devils. There were no Tasmanian devils living on Maria Island until 27 devils were introduced to the island in 2012 and 2013 from several captive populations [
     <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">66</a>]. Non-native vertebrate species, such as cats, black rats and house mice, are present on the island, whilst other species such as rabbits and livestock are absent, although all these species are common in the wild on mainland Tasmania. Two factors may have potentially contributed to the lower viral diversity observed on Maria Island. First, due to its isolation from mainland Tasmania, animal movements or immigrations are limited to only marine or bird species. This restriction to animal movements (particularly vertebrate species) between the mainland and Maria Island may in turn limit the introduction of viruses to the island. Second, captive-born devils are likely to have a “captive-type” virome, that is, lower viral diversity compared to wild devils. Indeed, some of the devils included in this study are captive-born animals recently translocated to the island. As samples were pooled prior to sequencing, it was not possible to distinguish between viromes of captive-born and wild-born devils. To understand whether captive devils can reacquire a “wild-type” virome following translocations, future studies should focus on comparing captive born translocated devils with the incumbent devils in the wild.
    </p>
    <p id="p-55">Overall, we detected sequences related to 20 vertebrate viruses, including 18 novel marsupial related viruses and two known mammalian viruses (RHDV2 and torovirus). While some of the viruses identified in this study come from families that include important pathogens, their pathogenic potential in devils is unclear. It is also important to note that some of these viruses may in fact be dietary viruses or occur naturally as part of the normal gut flora and therefore do not necessarily lead to disease. In the case of RHDV, we showed that RHDV2 was present in both captive and wild devils. In some areas of Tasmania, rabbits are common and may be a dietary component of wild devils. They are also regularly fed to devils in captivity, along with other meat sources such as wallaby and chicken. Feeding records provided by the two zoos in this study confirmed the feeding of rabbits to some of the sampled devils (zoo A) around the time of faecal collection. While we did not detect rabbit genes by meta-transcriptomics, VLP metagenomics or PCR in any of the wild devil samples that tested positive for RHDV, we did detect rabbit mtDNA in the stools of one captive devil from zoo A that tested positive for RHDV by PCR. Thus, more specific diagnostic techniques such as targeted PCR of blood or internal tissues (i.e. liver), 
     <em>in situ</em> hybridization and serological assays are required to determine whether these viruses can actively replicate in devils and cause disease or are simply gut contaminants. Nevertheless, exposure to host-adapted viruses could pose significant health threats, especially for devils that are immunocompromised due to DFTD, old age or other concurrent diseases [
     <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">67</a>, 
     <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">68</a>]. Furthermore, even commensal or latent viral infections can be exacerbated or reactivated in immunocompromised hosts [
     <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">69</a>, 
     <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">70</a>]. For example, in the giant panda, while a number of viruses (e.g. papillomaviruses, picornaviruses and anelloviruses) were detected in both sick and healthy animals, the virus titres were much higher in the diseased individuals, indicating a compromised balance between host immune response and virus replication [
     <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">71</a>].
    </p>
    <p id="p-56">Tasmanian devils have low genetic diversity both across their genomes and at functionally important loci such as the major histocompatibility complex (MHC) [
     <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">72</a>-
     <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">74</a>]. This lack of diversity renders them particularly vulnerable to changes in the environment, including the emergence of new pathogens [
     <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">75</a>]. For instance, populations of Italian agile frogs with lower microsatellite diversity have been shown to be more susceptible to an emerging strain of Ranavirus (FV3-frog virus 3) and experience higher mortality rates than populations with higher diversity [
     <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">76</a>]. Similarly in cheetahs, a coronavirus-associated feline infectious peritonitis outbreak causing mass mortality in a captive breeding colony was linked to the species’ extreme genetic monomorphism, particularly at the MHC [
     <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">77</a>]. While none of the devils included in this study exhibited overt signs of disease at the time of sample collection, the viruses described here will provide a fundamental baseline of the normal Tasmanian devil faecal virome, which can be used as a reference for comparing healthy and diseased animals.
    </p>
    <p id="p-57">Phylogenetic analyses of the newly identified viruses, including divergent members of their respective viral families, provided us with insights into the evolutionary history of marsupial-associated viruses relative to viruses of eutherian mammals and other host taxa. Generally, a long-term relationship between viruses and hosts are expected for mammalian viruses [
     <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>]. Strong evidence for this lies in the observation that devil viruses are usually clustered with other marsupial viruses, as a marsupial-specific lineage that is distinct from the eutherian viruses, as observed in papillomavirus [
     <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">78</a>, 
     <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">79</a>], polyomavirus, and herpesvirus (
     <a id="xref-fig-5-8" class="xref-fig" href="#F5">Fig. 5</a>). Furthermore, in several cases the branching order of viruses broadly reflects that of their hosts such that a general co-divergence can be inferred. For instance, in the phylogenies of 
     <em>Picornaviridae</em> and 
     <em>Astroviridae</em>, the devil (marsupial) viruses formed sister clade to eutherian viruses, which in turn are sister to avian viruses, consistent with the evolutionary history of the host. Although such a relationship is not observed in every virus phylogeny, a deep divergence between eutherian and marsupial viruses is typical of our dataset. This observation indicates that the timescale of virus evolution is very likely to reflect that of the hosts [
     <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">25</a>].
    </p>
    <p id="p-58">The gut virome is increasingly recognized as an integral component of the gut microbiome, and studies of the devil virome will continue to shine light on the biology and health of this iconic endangered species. For example, bacteriophage, which we have shown to dominate the devil’s faecal virome, can contribute to host health by maintaining the diversity and structure of the gut bacteriome through direct interactions with the bacterial communities. While the functions of bacteriophage on devil health remain to be determined, future studies will be able to exploit the extensive microbiomic data that is now available to answer important questions about the host-microbe relationship between devils and their microbiome [
     <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">65</a>].
    </p>
   </div>
   <div class="section" id="sec-25">
    <h2 class="">Conclusion</h2>
    <p id="p-59">We provided the first comprehensive characterisation of the faecal virome of the Tasmanian devil, greatly expanding our knowledge of the diverse groups of vertebrate viruses observed in this endangered species. Similar to their bacterial microbiome, captive devils have significantly lower diversity in their faecal viromes compared to wild devils, likely reflective of their captive diets and lifestyle. Identification of vertebrate and marsupial-specific viruses in devils provides potential candidate viruses for future disease surveillance and routine screening as part of the broader conservation management of devils. However, future work will first need to focus on elucidating the pathogenic impact of these viruses on devil health. Finally, we showed that a combination of VLP metagenomics and meta-transcriptomics may be a more comprehensive virome characterisation approach that will encompass both DNA and RNA viruses.</p>
   </div>
   <div class="section" id="sec-26">
    <h2 class="">Funding</h2>
    <p id="p-75">This work was funded by an Australian Research Council (ARC; LP140100508) grant to KB/CEG/CH/VB.</p>
    <p id="p-76">ECH is funded by an ARC Australian Laureate Fellowship (FL170100022).</p>
   </div>
   <div class="section" id="sec-27">
    <h2 class="">Availability of data and material</h2>
    <p id="p-77">Raw sequence reads generated in this study are available in NCBI SRA database under the BioProject ID PRJNA495667. Viral genomes described in detail here are available in the figshare repository 
     <a href="https://doi.org/10.6084/m9.figshare.7185146.v3">https://doi.org/10.6084/m9.figshare.7185146.v3</a> and GenBank (submission currently under process).
    </p>
   </div>
   <div class="section" id="sec-28">
    <h2 class="">Authors’ contributions</h2>
    <p id="p-78">RC, VB, MS, CH, CEG and KB designed the experiments. CH coordinated sample collection from all sites, with RC, CEG, CH participating in collection from the zoos, BR, SH (respectively). RC carried out the extractions and PCRs. RC, MS analysed the data. RC wrote the manuscript and all authors critically reviewed and edited the manuscript.</p>
   </div>
   <div class="section" id="sec-29">
    <h2 class="">Ethics approval and consent to participate</h2>
    <p id="p-79">Wild Tasmanian devils were trapped by the Save the Tasmanian Devil Program staff under the Tasmanian Government Department of Primary Industries, Parks, Water and Environment Standard Operating Procedure “Trapping and handling wild Tasmanian devils”.</p>
   </div>
   <div class="section" id="sec-30">
    <h2 class="">Consent for publication</h2>
    <p id="p-80">Not applicable</p>
   </div>
   <div class="section COI-statement" id="sec-31">
    <h2 class="">Competing interests</h2>
    <p id="p-81">The authors declare no competing interests.</p>
   </div>
   <div class="section" id="sec-32">
    <h2 class="">Additional files</h2>
    <p id="p-82">
     <strong>Additional file 1: Table S1</strong> List of vertebrate viruses identified by meta-transcriptomics and VLP metagenomics, including information on the sequence library, identification, classification, virus and strain name, sample type, genome length and percentage reads, Blastx hits to known viruses. (.xls)
    </p>
    <p id="p-83">
     <strong>Additional file 2: Table S2.</strong> Virus abundance table showing the proportion of reads mapped to viral contigs in each meta-transcriptomic library. (.xls)
    </p>
    <p id="p-84">
     <strong>Additional file 3: Table S3.</strong> Virus abundance table showing the proportion of reads mapped to viral contigs in each VLP metagenomics library. (.xls)
    </p>
    <p id="p-85">
     <strong>Additional file 4: Table S4.</strong> List of primer sets used for PCR confirmation of RHDV2. (.xls)
    </p>
    <p id="p-86">
     <strong>Additional file 5: Figure S1.</strong> Maximum likelihood phylogenetic tree of Dasyurid herpesvirus 3 in the context of representatives of the 
     <em>Herpesviridae</em> family based on the amino acid sequences of glycoprotein M. The tree is mid-point rooted and scaled to the number of amino acid substitutions per site. The newly discovered Dasyurid herpesvirus 3 is shown in red. (.pdf)
    </p>
    <p id="p-87">
     <strong>Additional file 6: Figure S2.</strong> Maximum likelihood phylogenetic tree of Dasyurid herpesvirus 3 in the context of representatives of the 
     <em>Herpesviridae</em> family based on the amino acid sequences of major capsid. The tree is mid-point rooted and scaled to the number of amino acid substitutions per site. The newly discovered DaHV-3 is shown in red. (.pdf)
    </p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgements</h2>
    <p id="p-74">Thank you to the field teams of the Save the Tasmanian Devil Program and the University of Sydney who collected faecal samples. Thanks also to N. Mooney for his assistance at the BR site, the two zoos (and their keeping staff) who provided access to faecal samples from the captive devils, J. Mahar for providing rabbit liver tissue and A.V. Lee for the map in 
     <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>.
    </p>
   </div>
   <div class="section glossary">
    <h2 class="">Abbreviations</h2>
    <dl id="def-list-1">
     <dt id="def-item-1">SH
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>Stony Head
      <span class="def-item-dd-sep"/>
     </dd>
     <dt id="def-item-2">MI
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>Maria Island
      <span class="def-item-dd-sep"/>
     </dd>
     <dt id="def-item-3">BR
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>Buckbys Road
      <span class="def-item-dd-sep"/>
     </dd>
     <dt id="def-item-4">wMW
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>wukalina/Mt William National Park
      <span class="def-item-dd-sep"/>
     </dd>
     <dt id="def-item-5">VLP
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>Virus-like particles
      <span class="def-item-dd-sep"/>
     </dd>
     <dt id="def-item-6">RHDV
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>Rabbit haemorrhagic disease virus
      <span class="def-item-dd-sep"/>
     </dd>
     <dt id="def-item-7">DeSV
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>Devil sapelovirus
      <span class="def-item-dd-sep"/>
     </dd>
     <dt id="def-item-8">DeAstV1
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>DeRoV1 and -2 Devil rotavirus 1 and 2
      <span class="def-item-dd-sep"/>
     </dd>
     <dt id="def-item-9">DePBV1-6
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>Devil picobirnavirus 1-6
      <span class="def-item-dd-sep"/>
     </dd>
     <dt id="def-item-10">DeBoV1 and -2
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>Devil bocavirus 1 and 2
      <span class="def-item-dd-sep"/>
     </dd>
     <dt id="def-item-11">DePV1 and -2
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>Devil papillomavirus 1 and 2
      <span class="def-item-dd-sep"/>
     </dd>
     <dt id="def-item-12">DePyV1 and -2
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>Devil polyomavirus 1 and 2
      <span class="def-item-dd-sep"/>
     </dd>
     <dt id="def-item-13">DaHV-3
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>Dasyurid herpesvirus 3
      <span class="def-item-dd-sep"/>
     </dd>
     <dt id="def-item-14">DeCV
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>Devil circovirus</dd>
    </dl>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pemberton</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Gales</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">B.</span>
          <span class="cit-name-surname">Bauer</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Gales</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">B.</span>
          <span class="cit-name-surname">Lazenby</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.</span>
          <span class="cit-name-surname">Medlock</span>
         </span>, 
         <span class="cit-article-title">The diet of the Tasmanian devil, Sarcophilus harrisii, as determined from analysis of scat and stomach contents</span>. 
         <abbr class="cit-jnl-abbrev">Papers and Proceedings of the Royal Society of Tasmania</abbr>, 
         <span class="cit-pub-date">2008</span>. 
         <span class="cit-vol">142</span>(
         <span class="cit-issue">2</span>): p. 
         <span class="cit-fpage">13</span>–
         <span class="cit-lpage">22</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPapers%2Band%2BProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2Bof%2BTasmania%26rft.volume%253D142%26rft.spage%253D13%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Owen</span>, 
          <span class="cit-name-given-names">D.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Pemberton</span>
         </span>, 
         <span class="cit-article-title">Made for travelling rough’: devil ecology, in Tasmanian Devil: A Unique and Threatened Animal</span>. 
         <span class="cit-pub-date">2005</span>, 
         <abbr class="cit-jnl-abbrev">Allen &amp;; Unwin</abbr>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lazenby</span>, 
          <span class="cit-name-given-names">B.T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.W.</span>
          <span class="cit-name-surname">Tobler</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">W.E.</span>
          <span class="cit-name-surname">Brown</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.E.</span>
          <span class="cit-name-surname">Hawkins</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">G.J.</span>
          <span class="cit-name-surname">Hocking</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">F.</span>
          <span class="cit-name-surname">Hume</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Huxtable</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Iles</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.E.</span>
          <span class="cit-name-surname">Jones</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Lawrence</span>
         </span>, 
         <span class="cit-article-title">Density trends and demographic signals uncover the long-term impact of transmissible cancer in Tasmanian devils</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Applied Ecology</abbr>, 
         <span class="cit-pub-date">2018</span>. 
         <span class="cit-vol">55</span>(
         <span class="cit-issue">3</span>): p. 
         <span class="cit-fpage">1368</span>–
         <span class="cit-lpage">1379</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BApplied%2BEcology%26rft.volume%253D55%26rft.spage%253D1368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.4" data-doi="10.1073/pnas.1519691113">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pye</span>, 
          <span class="cit-name-given-names">R.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Pemberton</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Tovar</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.M.</span>
          <span class="cit-name-surname">Tubio</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.A.</span>
          <span class="cit-name-surname">Dun</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Fox</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Darby</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Hayes</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">G.W.</span>
          <span class="cit-name-surname">Knowles</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Kreiss</span>
         </span>, 
         <span class="cit-article-title">A second transmissible cancer in Tasmanian devils</span>. 
         <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr>, 
         <span class="cit-pub-date">2016</span>. 
         <span class="cit-vol">113</span>(
         <span class="cit-issue">2</span>): p. 
         <span class="cit-fpage">374</span>–
         <span class="cit-lpage">379</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1519691113%26rft_id%253Dinfo%253Apmid%252F26711993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMTMvMi8zNzQiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8xMC8xNS80NDM0NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hogg</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Lee</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Srb</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Hibbard</span>
         </span>, 
         <span class="cit-article-title">Metapopulation management of an endangered species with limited genetic diversity in the presence of disease: the Tasmanian devil Sarcophilus harrisii</span>. 
         <abbr class="cit-jnl-abbrev">International Zoo Yearbook</abbr>, 
         <span class="cit-pub-date">2017</span>. 
         <span class="cit-vol">51</span>(
         <span class="cit-issue">1</span>): p. 
         <span class="cit-fpage">137</span>–
         <span class="cit-lpage">153</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DInternational%2BZoo%2BYearbook%26rft.volume%253D51%26rft.spage%253D137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.6" data-doi="10.1007/s10393-007-0120-6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jones</span>, 
          <span class="cit-name-given-names">M.E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.J.</span>
          <span class="cit-name-surname">Jarman</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.M.</span>
          <span class="cit-name-surname">Lees</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">H.</span>
          <span class="cit-name-surname">Hesterman</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.K.</span>
          <span class="cit-name-surname">Hamede</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">N.J.</span>
          <span class="cit-name-surname">Mooney</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Mann</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.E.</span>
          <span class="cit-name-surname">Pukk</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Bergfeld</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">H.</span>
          <span class="cit-name-surname">McCallum</span>
         </span>, 
         <span class="cit-article-title">Conservation management of Tasmanian devils in the context of an emerging, extinction-threatening disease: devil facial tumor disease</span>. 
         <abbr class="cit-jnl-abbrev">EcoHealth</abbr>, 
         <span class="cit-pub-date">2007</span>. 
         <span class="cit-vol">4</span>(
         <span class="cit-issue">3</span>): p. 
         <span class="cit-fpage">326</span>–
         <span class="cit-lpage">337</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEcoHealth%26rft.volume%253D4%26rft.spage%253D326%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10393-007-0120-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1007/s10393-007-0120-6&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=000250173600010&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.7" data-doi="10.1354/vp.43-6-890">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Loh</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Bergfeld</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Hayes</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">O’hara</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Pyecroft</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Raidal</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Sharpe</span>
         </span>, 
         <span class="cit-article-title">The pathology of devil facial tumor disease (DFTD) in Tasmanian devils (Sarcophilus harrisii)</span>. 
         <abbr class="cit-jnl-abbrev">Veterinary Pathology</abbr>, 
         <span class="cit-pub-date">2006</span>. 
         <span class="cit-vol">43</span>(
         <span class="cit-issue">6</span>): p. 
         <span class="cit-fpage">890</span>–
         <span class="cit-lpage">895</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVeterinary%2BPathology%26rft.stitle%253DVeterinary%2BPathology%26rft.aulast%253DLoh%26rft.auinit1%253DR.%26rft.volume%253D43%26rft.issue%253D6%26rft.spage%253D890%26rft.epage%253D895%26rft.atitle%253DThe%2BPathology%2Bof%2BDevil%2BFacial%2BTumor%2BDisease%2B%2528DFTD%2529%2Bin%2BTasmanian%2BDevils%2B%2528Sarcophilus%2Bharrisii%2529%26rft_id%253Dinfo%253Adoi%252F10.1354%252Fvp.43-6-890%26rft_id%253Dinfo%253Apmid%252F17099145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1354/vp.43-6-890&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17099145&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000242225800003&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-443457v1.8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">McCallum</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-article-title">Tasmanian devil facial tumour disease: lessons for conservation biology</span>. 
         <abbr class="cit-jnl-abbrev">Trends in Ecology &amp;; Evolution</abbr>, 
         <span class="cit-pub-date">2008</span>. 
         <span class="cit-vol">23</span>(
         <span class="cit-issue">11</span>): p. 
         <span class="cit-fpage">631</span>–
         <span class="cit-lpage">637</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.volume%253D23%26rft.spage%253D631%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-443457v1.9" data-doi="10.1371/journal.pone.0022402">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kreiss</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.</span>
          <span class="cit-name-surname">Cheng</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">F.</span>
          <span class="cit-name-surname">Kimble</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">B.</span>
          <span class="cit-name-surname">Wells</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Donovan</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.</span>
          <span class="cit-name-surname">Belov</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">G.M.</span>
          <span class="cit-name-surname">Woods</span>
         </span>, 
         <span class="cit-article-title">Allorecognition in the Tasmanian devil (Sarcophilus harrisii), an endangered marsupial species with limited genetic diversity</span>. 
         <abbr class="cit-jnl-abbrev">PLOS. one</abbr>, 
         <span class="cit-pub-date">2011</span>. 
         <span class="cit-vol">6</span>(
         <span class="cit-issue">7</span>): p. 
         <span class="cit-fpage">e22402</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DKreiss%26rft.auinit1%253DA.%26rft.volume%253D6%26rft.issue%253D7%26rft.spage%253De22402%26rft.epage%253De22402%26rft.atitle%253DAllorecognition%2Bin%2Bthe%2BTasmanian%2Bdevil%2B%2528Sarcophilus%2Bharrisii%2529%252C%2Ban%2Bendangered%2Bmarsupial%2Bspecies%2Bwith%2Blimited%2Bgenetic%2Bdiversity.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0022402%26rft_id%253Dinfo%253Apmid%252F21811598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0022402&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21811598&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-443457v1.10" data-doi="10.1038/439549a">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pearse</span>, 
          <span class="cit-name-given-names">A.-M.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.</span>
          <span class="cit-name-surname">Swift</span>
         </span>, 
         <span class="cit-article-title">Allograft theory: transmission of devil facial-tumour disease</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>, 
         <span class="cit-pub-date">2006</span>. 
         <span class="cit-vol">439</span>(
         <span class="cit-issue">7076</span>): p. 
         <span class="cit-fpage">549</span>–
         <span class="cit-lpage">549</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPearse%26rft.auinit1%253DA.%2BM.%26rft.volume%253D439%26rft.issue%253D7076%26rft.spage%253D549%26rft.epage%253D549%26rft.atitle%253DAllograft%2Btheory%253A%2Btransmission%2Bof%2Bdevil%2Bfacial-tumour%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F439549a%26rft_id%253Dinfo%253Apmid%252F16452970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/439549a&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16452970&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000235033900031&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.11" data-doi="10.1016/j.tig.2015.05.001">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Grueber</span>, 
          <span class="cit-name-given-names">C.E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.</span>
          <span class="cit-name-surname">Peel</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Gooley</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.</span>
          <span class="cit-name-surname">Belov</span>
         </span>, 
         <span class="cit-article-title">Genomic insights into a contagious cancer in Tasmanian devils</span>. 
         <abbr class="cit-jnl-abbrev">Trends in Genetics</abbr>, 
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-vol">31</span>(
         <span class="cit-issue">9</span>): p. 
         <span class="cit-fpage">528</span>–
         <span class="cit-lpage">535</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BGenetics%26rft.volume%253D31%26rft.spage%253D528%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2015.05.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.tig.2015.05.001&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.12">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Stalder</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.K.</span>
          <span class="cit-name-surname">Vaz</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.R.</span>
          <span class="cit-name-surname">Gilkerson</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Baker</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Whiteley</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">N.</span>
          <span class="cit-name-surname">Ficorilli</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L.</span>
          <span class="cit-name-surname">Tatarczuch</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Portas</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.</span>
          <span class="cit-name-surname">Skogvold</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">G.A.</span>
          <span class="cit-name-surname">Anderson</span>
         </span>, 
         <span class="cit-article-title">Prevalence and clinical significance of herpesvirus infection in populations of Australian marsupials</span>. 
         <abbr class="cit-jnl-abbrev">PLO. one</abbr>, 
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-vol">10</span>(
         <span class="cit-issue">7</span>): p. 
         <span class="cit-fpage">e0133807</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLO.%2Bone%26rft.volume%253D10%26rft.spage%253De0133807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.13">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Shi</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">X.-D.</span>
          <span class="cit-name-surname">Lin</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">X.</span>
          <span class="cit-name-surname">Chen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.-H.</span>
          <span class="cit-name-surname">Tian</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L.-J.</span>
          <span class="cit-name-surname">Chen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.</span>
          <span class="cit-name-surname">Li</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">W.</span>
          <span class="cit-name-surname">Wang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.-S.</span>
          <span class="cit-name-surname">Eden</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.-J.</span>
          <span class="cit-name-surname">Shen</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">L.</span>
          <span class="cit-name-surname">Liu</span>
         </span>, 
         <span class="cit-article-title">The evolutionary history of vertebrate RNA viruses</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>, 
         <span class="cit-pub-date">2018</span>: p. 
         <span class="cit-fpage">1</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Weisbecker</span>, 
          <span class="cit-name-given-names">V.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Beck</span>
         </span>, 
         <span class="cit-article-title">Marsupial and Monotreme Evolution And Biogeography</span>. 
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">32</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.15" data-doi="10.1186/1471-2164-9-5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Djikeng</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Halpin</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Kuzmickas</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">DePasse</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Feldblyum</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">N.</span>
          <span class="cit-name-surname">Sengamalay</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Afonso</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">X.</span>
          <span class="cit-name-surname">Zhang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">N.G.</span>
          <span class="cit-name-surname">Anderson</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.</span>
          <span class="cit-name-surname">Ghedin</span>
         </span>, 
         <span class="cit-article-title">Viral genome sequencing by random priming methods</span>. 
         <abbr class="cit-jnl-abbrev">BMC. genomics</abbr>, 
         <span class="cit-pub-date">2008</span>. 
         <span class="cit-vol">9</span>(
         <span class="cit-issue">1</span>): p. 
         <span class="cit-fpage">5</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DDjikeng%26rft.auinit1%253DA.%26rft.volume%253D9%26rft.spage%253D5%26rft.epage%253D5%26rft.atitle%253DViral%2Bgenome%2Bsequencing%2Bby%2Brandom%2Bpriming%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-5%26rft_id%253Dinfo%253Apmid%252F18179705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-5&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18179705&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-443457v1.16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Thurber</span>, 
          <span class="cit-name-given-names">R.V.</span>
         </span>, 
         <span class="cit-article-title">Methods in Viral Metagenomics</span>. 
         <abbr class="cit-jnl-abbrev">Handbook of Molecular Microbial Ecology II: Metagenomics in different habitats: Wiley-Blackwell</abbr>, 
         <span class="cit-pub-date">2011</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">17.</span>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.17">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Conceicao-Neto</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Zeller</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">H.</span>
          <span class="cit-name-surname">Lefrere</span>
         </span>, P. 
         <span class="cit-auth">
          <span class="cit-name-given-names">De.</span>
          <span class="cit-name-surname">Bruyn</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L.</span>
          <span class="cit-name-surname">Beller</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">W.</span>
          <span class="cit-name-surname">Deboutte</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.K.</span>
          <span class="cit-name-surname">Yinda</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Lavigne</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Maes</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Van Ranst</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.</span>
          <span class="cit-name-surname">Heylen</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Matthijnssens</span>
         </span>, 
         <span class="cit-article-title">Modular approach to customise sample preparation procedures for viral metagenomics: a reproducible protocol for virome analysis</span>. 
         <abbr class="cit-jnl-abbrev">Scientific Report</abbr>, 
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-vol">5</span>: p. 
         <span class="cit-fpage">16532</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScientific%2BReport%26rft.volume%253D5%26rft.spage%253D16532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">18.</span>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.18" data-doi="10.1016/j.jviromet.2013.08.035">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hall</span>, 
          <span class="cit-name-given-names">R.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Wang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.K.</span>
          <span class="cit-name-surname">Todd</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.B.</span>
          <span class="cit-name-surname">Bissielo</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Yen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">H.</span>
          <span class="cit-name-surname">Strydom</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">N.E.</span>
          <span class="cit-name-surname">Moore</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">X.</span>
          <span class="cit-name-surname">Ren</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Q.S.</span>
          <span class="cit-name-surname">Huang</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.E.</span>
          <span class="cit-name-surname">Carter</span>
         </span>, 
         <span class="cit-article-title">Evaluation of rapid and simple techniques for the enrichment of viruses prior to metagenomic virus discovery</span>. 
         <abbr class="cit-jnl-abbrev">Journal of virological methods</abbr>, 
         <span class="cit-pub-date">2014</span>. 
         <span class="cit-vol">195</span>: p. 
         <span class="cit-fpage">194</span>–
         <span class="cit-lpage">204</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirological%2Bmethods%26rft.volume%253D195%26rft.spage%253D194%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jviromet.2013.08.035%26rft_id%253Dinfo%253Apmid%252F24036074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.jviromet.2013.08.035&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24036074&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">19.</span>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.19" data-doi="10.1038/nature20167">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Shi</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">X.-D.</span>
          <span class="cit-name-surname">Lin</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.-H.</span>
          <span class="cit-name-surname">Tian</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L.-J.</span>
          <span class="cit-name-surname">Chen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">X.</span>
          <span class="cit-name-surname">Chen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.-X.</span>
          <span class="cit-name-surname">Li</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">X.-C.</span>
          <span class="cit-name-surname">Qin</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Li</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.-P.</span>
          <span class="cit-name-surname">Cao</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.-S.</span>
          <span class="cit-name-surname">Eden</span>
         </span>, 
         <span class="cit-article-title">Redefining the invertebrate RNA virosphere</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>, 
         <span class="cit-pub-date">2016</span>. 
         <span class="cit-vol">540</span>(
         <span class="cit-issue">7634</span>): p. 
         <span class="cit-fpage">539</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D540%26rft.spage%253D539%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature20167%26rft_id%253Dinfo%253Apmid%252F27880757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature20167&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=27880757&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">20.</span>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.20">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Shi</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Neville</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Nicholson</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.-S.</span>
          <span class="cit-name-surname">Eden</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Imrie</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.C.</span>
          <span class="cit-name-surname">Holmes</span>
         </span>, 
         <span class="cit-article-title">High-Resolution Metatranscriptomics Reveals the Ecological Dynamics of Mosquito-Associated RNA Viruses in Western Australia</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>, 
         <span class="cit-pub-date">2017</span>. 
         <span class="cit-vol">91</span>(
         <span class="cit-issue">17</span>): p. 
         <span class="cit-fpage">e00680</span>–
         <span class="cit-lpage">17</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.volume%253D91%26rft.spage%253De00680%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">21.</span>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.21" data-doi="10.1016/j.biocon.2006.04.010">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hawkins</span>, 
          <span class="cit-name-given-names">C.E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Baars</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">H.</span>
          <span class="cit-name-surname">Hesterman</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">G.J.</span>
          <span class="cit-name-surname">Hocking</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.E.</span>
          <span class="cit-name-surname">Jones</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">B.</span>
          <span class="cit-name-surname">Lazenby</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Mann</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">N.</span>
          <span class="cit-name-surname">Mooney</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Pemberton</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Pyecroft</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Restani</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Wiersma</span>
         </span>, 
         <span class="cit-article-title">Emerging disease and population decline of an island endemic, the Tasmanian devil Sarcophilus harrisii</span>. 
         <abbr class="cit-jnl-abbrev">Biological Conservation</abbr>, 
         <span class="cit-pub-date">2006</span>. 
         <span class="cit-vol">131</span>(
         <span class="cit-issue">2</span>): p. 
         <span class="cit-fpage">307</span>–
         <span class="cit-lpage">324</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBiological%2BConservation%26rft.volume%253D131%26rft.spage%253D307%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.biocon.2006.04.010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.biocon.2006.04.010&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">22.</span>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.22" data-doi="10.1093/bioinformatics/btu170">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bolger</span>, 
          <span class="cit-name-given-names">A.M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Lohse</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">B.</span>
          <span class="cit-name-surname">Usadel</span>
         </span>, 
         <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, 
         <span class="cit-pub-date">2014</span>. 
         <span class="cit-vol">30</span>(
         <span class="cit-issue">15</span>): p. 
         <span class="cit-fpage">2114</span>–
         <span class="cit-lpage">2120</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24695404&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000340049100004&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">23.</span>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.23" data-doi="10.1038/nbt.1883">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Grabherr</span>, 
          <span class="cit-name-given-names">M.G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">B.J.</span>
          <span class="cit-name-surname">Haas</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Yassour</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.Z.</span>
          <span class="cit-name-surname">Levin</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.A.</span>
          <span class="cit-name-surname">Thompson</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">I.</span>
          <span class="cit-name-surname">Amit</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">X.</span>
          <span class="cit-name-surname">Adiconis</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L.</span>
          <span class="cit-name-surname">Fan</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Raychowdhury</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Q.</span>
          <span class="cit-name-surname">Zeng</span>
         </span>, 
         <span class="cit-article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome</span>. 
         <abbr class="cit-jnl-abbrev">Nature biotechnology</abbr>, 
         <span class="cit-pub-date">2011</span>. 
         <span class="cit-vol">29</span>(
         <span class="cit-issue">7</span>): p. 
         <span class="cit-fpage">644</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DGrabherr%26rft.auinit1%253DM.%2BG.%26rft.volume%253D29%26rft.issue%253D7%26rft.spage%253D644%26rft.epage%253D652%26rft.atitle%253DFull-length%2Btranscriptome%2Bassembly%2Bfrom%2BRNA-Seq%2Bdata%2Bwithout%2Ba%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1883%26rft_id%253Dinfo%253Apmid%252F21572440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nbt.1883&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21572440&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">24.</span>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.24" data-doi="10.1093/protein/12.2.85">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rost</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-article-title">Twilight zone of protein sequence alignments</span>. 
         <abbr class="cit-jnl-abbrev">Protein Engineering, Design and Selection</abbr>, 
         <span class="cit-pub-date">1999</span>. 
         <span class="cit-vol">12</span>(
         <span class="cit-issue">2</span>): p. 
         <span class="cit-fpage">85</span>–
         <span class="cit-lpage">94</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProtein%2BEngineering%2BDesign%2Band%2BSelection%26rft.stitle%253DProtein%2BEng%2BDes%2BSel%26rft.aulast%253DRost%26rft.auinit1%253DB.%26rft.volume%253D12%26rft.issue%253D2%26rft.spage%253D85%26rft.epage%253D94%26rft.atitle%253DTwilight%2Bzone%2Bof%2Bprotein%2Bsequence%2Balignments%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fprotein%252F12.2.85%26rft_id%253Dinfo%253Apmid%252F10195279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/protein/12.2.85&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=10195279&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000079066100001&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">25.</span>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.25" data-doi="10.1038/s41586-018-0012-7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Shi</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">X.-D.</span>
          <span class="cit-name-surname">Lin</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">X.</span>
          <span class="cit-name-surname">Chen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.-H.</span>
          <span class="cit-name-surname">Tian</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L.-J.</span>
          <span class="cit-name-surname">Chen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.</span>
          <span class="cit-name-surname">Li</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">W.</span>
          <span class="cit-name-surname">Wang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.-S.</span>
          <span class="cit-name-surname">Eden</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.-J.</span>
          <span class="cit-name-surname">Shen</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">L.</span>
          <span class="cit-name-surname">Liu</span>
         </span>, 
         <span class="cit-article-title">The evolutionary history of vertebrate RNA viruses</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>, 
         <span class="cit-pub-date">2018</span>. 
         <span class="cit-vol">556</span>(
         <span class="cit-issue">7700</span>): p. 
         <span class="cit-fpage">197</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D556%26rft.spage%253D197%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-018-0012-7%26rft_id%253Dinfo%253Apmid%252F29618816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41586-018-0012-7&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=29618816&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">26.</span>
      <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.26" data-doi="10.1038/nmeth.1923">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Langmead</span>, 
          <span class="cit-name-given-names">B.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.L.</span>
          <span class="cit-name-surname">Salzberg</span>
         </span>, 
         <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2</span>. 
         <abbr class="cit-jnl-abbrev">Nature methods</abbr>, 
         <span class="cit-pub-date">2012</span>. 
         <span class="cit-vol">9</span>(
         <span class="cit-issue">4</span>): p. 
         <span class="cit-fpage">357</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Bmethods%26rft.volume%253D9%26rft.spage%253D357%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22388286&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000302218500017&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">27.</span>
      <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.27" data-doi="10.1186/1471-2105-12-323">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">B.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.N.</span>
          <span class="cit-name-surname">Dewey</span>
         </span>, 
         <span class="cit-article-title">RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome</span>. 
         <abbr class="cit-jnl-abbrev">BMC. bioinformatics</abbr>, 
         <span class="cit-pub-date">2011</span>. 
         <span class="cit-vol">12</span>(
         <span class="cit-issue">1</span>): p. 
         <span class="cit-fpage">323</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DLi%26rft.auinit1%253DB.%26rft.volume%253D12%26rft.spage%253D323%26rft.epage%253D323%26rft.atitle%253DRSEM%253A%2Baccurate%2Btranscript%2Bquantification%2Bfrom%2BRNA-Seq%2Bdata%2Bwith%2Bor%2Bwithout%2Ba%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-323%26rft_id%253Dinfo%253Apmid%252F21816040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-323&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21816040&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">28.</span>
      <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.28" data-doi="10.1093/bioinformatics/bts199">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kearse</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Moir</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Wilson</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Stones-Havas</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Cheung</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Sturrock</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Buxton</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Cooper</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Markowitz</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Duran</span>
         </span>, 
         <span class="cit-article-title">Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, 
         <span class="cit-pub-date">2012</span>. 
         <span class="cit-vol">28</span>(
         <span class="cit-issue">12</span>): p. 
         <span class="cit-fpage">1647</span>–
         <span class="cit-lpage">1649</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts199%26rft_id%253Dinfo%253Apmid%252F22543367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts199&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22543367&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000305419800052&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">29.</span>
      <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.29" data-doi="10.1093/molbev/mst010">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Katoh</span>, 
          <span class="cit-name-given-names">K.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.M.</span>
          <span class="cit-name-surname">Standley</span>
         </span>, 
         <span class="cit-article-title">MAFFT multiple sequence alignment software version 7: improvements in performance and usability</span>. 
         <abbr class="cit-jnl-abbrev">Molecular biology and evolution</abbr>, 
         <span class="cit-pub-date">2013</span>. 
         <span class="cit-vol">30</span>(
         <span class="cit-issue">4</span>): p. 
         <span class="cit-fpage">772</span>–
         <span class="cit-lpage">780</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMolecular%2Bbiology%2Band%2Bevolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst010%26rft_id%253Dinfo%253Apmid%252F23329690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/molbev/mst010&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23329690&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000317002300004&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">30.</span>
      <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.30" data-doi="10.1093/bioinformatics/btp348">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Capella-Gutiérrez</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.M.</span>
          <span class="cit-name-surname">Silla-Martínez</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Gabaldón</span>
         </span>, 
         <span class="cit-article-title">trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, 
         <span class="cit-pub-date">2009</span>. 
         <span class="cit-vol">25</span>(
         <span class="cit-issue">15</span>): p. 
         <span class="cit-fpage">1972</span>–
         <span class="cit-lpage">1973</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp348%26rft_id%253Dinfo%253Apmid%252F19505945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp348&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19505945&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000268107100022&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">31.</span>
      <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.31" data-doi="10.1080/10635150390235520">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Guindon</span>, 
          <span class="cit-name-given-names">S.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">O.</span>
          <span class="cit-name-surname">Gascuel</span>
         </span>, 
         <span class="cit-article-title">A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood</span>. 
         <abbr class="cit-jnl-abbrev">Systematic biology</abbr>, 
         <span class="cit-pub-date">2003</span>. 
         <span class="cit-vol">52</span>(
         <span class="cit-issue">5</span>): p. 
         <span class="cit-fpage">696</span>–
         <span class="cit-lpage">704</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSystematic%2Bbiology%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150390235520%26rft_id%253Dinfo%253Apmid%252F14530136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1080/10635150390235520&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=14530136&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000185732500010&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">32.</span>
      <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.32" data-doi="10.1038/nmeth.f.303">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Caporaso</span>, 
          <span class="cit-name-given-names">J.G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Kuczynski</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Stombaugh</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.</span>
          <span class="cit-name-surname">Bittinger</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">F.D.</span>
          <span class="cit-name-surname">Bushman</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.K.</span>
          <span class="cit-name-surname">Costello</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">N.</span>
          <span class="cit-name-surname">Fierer</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.G.</span>
          <span class="cit-name-surname">Pena</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.K.</span>
          <span class="cit-name-surname">Goodrich</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.I.</span>
          <span class="cit-name-surname">Gordon</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">G.A.</span>
          <span class="cit-name-surname">Huttley</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.T.</span>
          <span class="cit-name-surname">Kelley</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Knights</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.E.</span>
          <span class="cit-name-surname">Koenig</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.E.</span>
          <span class="cit-name-surname">Ley</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.A.</span>
          <span class="cit-name-surname">Lozupone</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">McDonald</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">B.D.</span>
          <span class="cit-name-surname">Muegge</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Pirrung</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Reeder</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.R.</span>
          <span class="cit-name-surname">Sevinsky</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.J.</span>
          <span class="cit-name-surname">Turnbaugh</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">W.A.</span>
          <span class="cit-name-surname">Walters</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Widmann</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Yatsunenko</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Zaneveld</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Knight</span>
         </span>, 
         <span class="cit-article-title">QIIME allows analysis of high-throughput community sequencing data</span>. 
         <abbr class="cit-jnl-abbrev">Nat Methods</abbr>, 
         <span class="cit-pub-date">2010</span>. 
         <span class="cit-vol">7</span>(
         <span class="cit-issue">5</span>): p. 
         <span class="cit-fpage">335</span>–
         <span class="cit-lpage">6</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DCaporaso%26rft.auinit1%253DJ.%2BG.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253D335%26rft.epage%253D336%26rft.atitle%253DQIIME%2Ballows%2Banalysis%2Bof%2Bhigh-throughput%2Bcommunity%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.f.303%26rft_id%253Dinfo%253Apmid%252F20383131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nmeth.f.303&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20383131&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000277175100003&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">33.</span>
      <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.33">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">R Core Team</span>, 
         <span class="cit-article-title">R: A language and environment for statistical computing</span>. 
         <span class="cit-pub-date">2018</span>: 
         <abbr class="cit-jnl-abbrev">R Foundation for Statistical Computing, Vienna, Austria</abbr>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">34.</span>
      <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.34" data-doi="10.1186/1297-9716-43-12">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Abrantes</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">W.</span>
          <span class="cit-name-surname">Van Der Loo</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J. Le.</span>
          <span class="cit-name-surname">Pendu</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.J.</span>
          <span class="cit-name-surname">Esteves</span>
         </span>, 
         <span class="cit-article-title">Rabbit haemorrhagic disease (RHD) and rabbit haemorrhagic disease virus (RHDV): a review</span>. 
         <abbr class="cit-jnl-abbrev">Veterinary research</abbr>, 
         <span class="cit-pub-date">2012</span>. 
         <span class="cit-vol">43</span>(
         <span class="cit-issue">1</span>): p. 
         <span class="cit-fpage">12</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVeterinary%2Bresearch%26rft.stitle%253DVet%2BRes%26rft.aulast%253DAbrantes%26rft.auinit1%253DJ.%26rft.volume%253D43%26rft.issue%253D1%26rft.spage%253D12%26rft.epage%253D12%26rft.atitle%253DRabbit%2Bhaemorrhagic%2Bdisease%2B%2528RHD%2529%2Band%2Brabbit%2Bhaemorrhagic%2Bdisease%2Bvirus%2B%2528RHDV%2529%253A%2Ba%2Breview.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9716-43-12%26rft_id%253Dinfo%253Apmid%252F22325049%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/1297-9716-43-12&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22325049&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">35.</span>
      <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.35">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hall</span>, 
          <span class="cit-name-given-names">R.N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.E.</span>
          <span class="cit-name-surname">Mahar</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.J.</span>
          <span class="cit-name-surname">Read</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Mourant</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Piper</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">N.</span>
          <span class="cit-name-surname">Huang</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Strive</span>
         </span>, 
         <span class="cit-article-title">A strain-specific multiplex RT-PCR for Australian rabbit haemorrhagic disease viruses uncovers a new recombinant virus variant in rabbits and hares</span>. 
         <abbr class="cit-jnl-abbrev">Transboundary and emerging diseases</abbr>, 
         <span class="cit-pub-date">2018</span>. 
         <span class="cit-vol">65</span>(
         <span class="cit-issue">2</span>): p. 
         <span class="cit-fpage">e444</span>–
         <span class="cit-lpage">e456</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTransboundary%2Band%2Bemerging%2Bdiseases%26rft.volume%253D65%26rft.spage%253D444e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">36.</span>
      <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.36">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Martín</span>, 
          <span class="cit-name-given-names">I.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">García</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">V.</span>
          <span class="cit-name-surname">Fajardo</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Rojas</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">N.</span>
          <span class="cit-name-surname">Pegels</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Hernández</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">I.</span>
          <span class="cit-name-surname">González</span>
         </span>, 
         <span class="cit-article-title">Polymerase chain reaction detection of rabbit DNA in food and animal feed</span>. 
         <abbr class="cit-jnl-abbrev">World Rabbit Science</abbr>, 
         <span class="cit-pub-date">2010</span>. 
         <span class="cit-vol">17</span>(
         <span class="cit-issue">1</span>): p. 
         <span class="cit-fpage">27</span>–
         <span class="cit-lpage">36</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DWorld%2BRabbit%2BScience%26rft.volume%253D17%26rft.spage%253D27%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">37.</span>
      <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a>
      <div class="cit ref-cit ref-book" id="cit-443457v1.37">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Burland</span>, 
          <span class="cit-name-given-names">T.G.</span>
         </span>, 
         <span class="cit-chapter-title">DNASTAR’s Lasergene sequence analysis software, in Bioinformatics methods and protocols</span>. 
         <span class="cit-pub-date">2000</span>, 
         <span class="cit-publ-name">Springer. p</span>. 
         <span class="cit-fpage">71</span>–
         <span class="cit-lpage">91</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">38.</span>
      <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.38">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Snijder</span>, 
          <span class="cit-name-given-names">E.J.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.C.</span>
          <span class="cit-name-surname">Horzinek</span>
         </span>, 
         <span class="cit-article-title">Toroviruses: replication, evolution and comparison with other members of the coronavirus-like superfamily</span>. 
         <abbr class="cit-jnl-abbrev">Journal of general virology</abbr>, 
         <span class="cit-pub-date">1993</span>. 
         <span class="cit-vol">74</span>(
         <span class="cit-issue">11</span>): p. 
         <span class="cit-fpage">2305</span>–
         <span class="cit-lpage">2316</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DSnijder%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D74%26rft.issue%253D11%26rft.spage%253D2305%26rft.epage%253D2316%26rft.atitle%253DToroviruses%253A%2Breplication%252C%2Bevolution%2Band%2Bcomparison%2Bwith%2Bother%2Bmembers%2Bof%2Bthe%2Bcoronavirus-like%2Bsuperfamily%26rft_id%253Dinfo%253Apmid%252F8245847%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=8245847&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1993MF24400001&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">39.</span>
      <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.39">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">King</span>, 
          <span class="cit-name-given-names">A.M.Q.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.J.</span>
          <span class="cit-name-surname">Adams</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.B.</span>
          <span class="cit-name-surname">Carstens</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.J.</span>
          <span class="cit-name-surname">Lefkowitz</span>
         </span>, 
         <span class="cit-article-title">Virus Taxonomy: Classification and Nomenclature of Viruses: Ninth Report of the International Committee on Taxonomy of Viruses</span>. 
         <span class="cit-pub-date">2012</span>, 
         <abbr class="cit-jnl-abbrev">Elsevier: San Diego</abbr>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">40.</span>
      <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.40" data-doi="10.1038/nrmicro2030">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Koonin</span>, 
          <span class="cit-name-given-names">E.V.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.I.</span>
          <span class="cit-name-surname">Wolf</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.</span>
          <span class="cit-name-surname">Nagasaki</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">V.V.</span>
          <span class="cit-name-surname">Dolja</span>
         </span>, 
         <span class="cit-article-title">The Big Bang of picorna-like virus evolution antedates the radiation of eukaryotic supergroups</span>. 
         <abbr class="cit-jnl-abbrev">Nature Reviews Microbiology</abbr>, 
         <span class="cit-pub-date">2008</span>. 
         <span class="cit-vol">6</span>(
         <span class="cit-issue">12</span>): p. 
         <span class="cit-fpage">925</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DKoonin%26rft.auinit1%253DE.%2BV.%26rft.volume%253D6%26rft.issue%253D12%26rft.spage%253D925%26rft.epage%253D939%26rft.atitle%253DThe%2BBig%2BBang%2Bof%2Bpicorna-like%2Bvirus%2Bevolution%2Bantedates%2Bthe%2Bradiation%2Bof%2Beukaryotic%2Bsupergroups.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro2030%26rft_id%253Dinfo%253Apmid%252F18997823%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nrmicro2030&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18997823&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000260903500014&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">41.</span>
      <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.41">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Knowles</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Hovi</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Hyypiä</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">King</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.M.</span>
          <span class="cit-name-surname">Lindberg</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Pallansch</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Palmenberg</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Simmonds</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Skern</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">G.</span>
          <span class="cit-name-surname">Stanway</span>
         </span>, 
         <span class="cit-article-title">Family-Picornaviridae</span>. 
         <span class="cit-pub-date">2012</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">42.</span>
      <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.42" data-doi="10.1056/NEJM199106203242501">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Herrmann</span>, 
          <span class="cit-name-given-names">J.E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.N.</span>
          <span class="cit-name-surname">Taylor</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Echeverri</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">N.R.</span>
          <span class="cit-name-surname">Blacklow</span>
         </span>, 
         <span class="cit-article-title">Astroviruses as a cause of gastroenteritis in children</span>. 
         <abbr class="cit-jnl-abbrev">New England Journal of Medicine</abbr>, 
         <span class="cit-pub-date">1991</span>. 
         <span class="cit-vol">324</span>(
         <span class="cit-issue">25</span>): p. 
         <span class="cit-fpage">1757</span>–
         <span class="cit-lpage">1760</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.stitle%253DNEJM%26rft.aulast%253DHerrmann%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D324%26rft.issue%253D25%26rft.spage%253D1757%26rft.epage%253D1760%26rft.atitle%253DAstroviruses%2Bas%2Ba%2Bcause%2Bof%2Bgastroenteritis%2Bin%2Bchildren%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJM199106203242501%26rft_id%253Dinfo%253Apmid%252F1645459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1056/NEJM199106203242501&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=1645459&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1991FR68100001&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">43.</span>
      <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.43" data-doi="10.1128/JVI.00998-09">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Finkbeiner</span>, 
          <span class="cit-name-given-names">S.R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.</span>
          <span class="cit-name-surname">Li</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Ruone</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Conrardy</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">N.</span>
          <span class="cit-name-surname">Gregoricus</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Toney</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">H.W.</span>
          <span class="cit-name-surname">Virgin</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L.J.</span>
          <span class="cit-name-surname">Anderson</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Vinjé</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Wang</span>
         </span>, 
         <span class="cit-article-title">Identification of a novel astrovirus (astrovirus VA1) associated with an outbreak of acute gastroenteritis</span>. 
         <abbr class="cit-jnl-abbrev">Journal of virology</abbr>, 
         <span class="cit-pub-date">2009</span>. 
         <span class="cit-vol">83</span>(
         <span class="cit-issue">20</span>): p. 
         <span class="cit-fpage">10836</span>–
         <span class="cit-lpage">10839</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00998-09%26rft_id%253Dinfo%253Apmid%252F19706703%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4My8yMC8xMDgzNiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzEwLzE1LzQ0MzQ1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">44.</span>
      <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.44">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Méndez</span>, 
          <span class="cit-name-given-names">E.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.F.</span>
          <span class="cit-name-surname">Arias</span>
         </span>, 
         <span class="cit-article-title">Astrovirus, in Clinical Virology, Third Edition</span>. 
         <span class="cit-pub-date">2009</span>, 
         <abbr class="cit-jnl-abbrev">American Society of Microbiology</abbr>. p. 
         <span class="cit-fpage">1145</span>–
         <span class="cit-lpage">1153</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">45.</span>
      <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a>
      <div class="cit ref-cit ref-book" id="cit-443457v1.45">
       <div class="cit-metadata">
        <ol class="duplicate">
         <li>
          <span class="cit-ed">
           <span class="cit-name-given-names">A.M.Q.</span>
           <span class="cit-name-surname">King</span>
          </span>
         </li>
        </ol>
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bosch</span>
          <span class="cit-name-given-names">A.</span>
         </span>, G.S., 
         <span class="cit-auth">
          <span class="cit-name-surname">Krishna</span>
          <span class="cit-name-given-names">N.K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Méndez</span>
          <span class="cit-name-given-names">E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Monroe</span>
          <span class="cit-name-given-names">S.S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pantin-Jackwood</span>
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schultz-Cherry</span>
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-chapter-title">Family - Astroviridae, in Virus Taxonomy: Ninth Report of the International Committee on Taxonomy of Viruses</span>. 
         <span class="cit-ed">
          <span class="cit-name-given-names">A.M.Q.</span>
          <span class="cit-name-surname">King</span>
         </span>, 
         <span class="cit-etal">et al.</span>, Editors. 
         <span class="cit-pub-date">2012</span>, 
         <span class="cit-publ-name">Elsevier</span>: 
         <span class="cit-publ-loc">San Diego</span>. p. 
         <span class="cit-fpage">953</span>–
         <span class="cit-lpage">959</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">46.</span>
      <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.46">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Estes</span>, 
          <span class="cit-name-given-names">M.K.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Cohen</span>
         </span>, 
         <span class="cit-article-title">Rotavirus gene structure and function</span>. 
         <abbr class="cit-jnl-abbrev">Microbiological reviews</abbr>, 
         <span class="cit-pub-date">1989</span>. 
         <span class="cit-vol">53</span>(
         <span class="cit-issue">4</span>): p. 
         <span class="cit-fpage">410</span>–
         <span class="cit-lpage">449</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DEstes%26rft.auinit1%253DM%2BK%26rft.volume%253D53%26rft.issue%253D4%26rft.spage%253D410%26rft.epage%253D449%26rft.atitle%253DRotavirus%2Bgene%2Bstructure%2Band%2Bfunction.%26rft_id%253Dinfo%253Apmid%252F2556635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI1My80LzQxMCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzEwLzE1LzQ0MzQ1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">47.</span>
      <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.47" data-doi="10.1007/s13337-014-0207-y">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ganesh</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">G.</span>
          <span class="cit-name-surname">Masachessi</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Z.</span>
          <span class="cit-name-surname">Mladenova</span>
         </span>, 
         <span class="cit-article-title">Animal picobirnavirus</span>. 
         <abbr class="cit-jnl-abbrev">Virusdisease</abbr>, 
         <span class="cit-pub-date">2014</span>. 
         <span class="cit-vol">25</span>(
         <span class="cit-issue">2</span>): p. 
         <span class="cit-fpage">223</span>–
         <span class="cit-lpage">238</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVirusdisease%26rft.volume%253D25%26rft.spage%253D223%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs13337-014-0207-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1007/s13337-014-0207-y&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">48.</span>
      <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.48" data-doi="10.1159/000149587">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Siegl</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.C.</span>
          <span class="cit-name-surname">Bates</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.I.</span>
          <span class="cit-name-surname">Berns</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">B.J.</span>
          <span class="cit-name-surname">Carter</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.C.</span>
          <span class="cit-name-surname">Kelly</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.</span>
          <span class="cit-name-surname">Kurstak</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Tattersall</span>
         </span>, 
         <span class="cit-article-title">Characteristics and taxonomy of Parvoviridae</span>. 
         <abbr class="cit-jnl-abbrev">Intervirology</abbr>, 
         <span class="cit-pub-date">1985</span>. 
         <span class="cit-vol">23</span>(
         <span class="cit-issue">2</span>): p. 
         <span class="cit-fpage">61</span>–
         <span class="cit-lpage">73</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DIntervirology%26rft.stitle%253DIntervirology%26rft.aulast%253DSiegl%26rft.auinit1%253DG.%26rft.volume%253D23%26rft.issue%253D2%26rft.spage%253D61%26rft.epage%253D73%26rft.atitle%253DCharacteristics%2Band%2Btaxonomy%2Bof%2BParvoviridae.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000149587%26rft_id%253Dinfo%253Apmid%252F3980186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1159/000149587&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=3980186&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1985ABS4400001&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">49.</span>
      <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.49" data-doi="10.1002/(SICI)1096-9896(199909)189:1&lt;;12::AID-PATH431&gt;;3.0.CO;2-F">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Walboomers</span>, 
          <span class="cit-name-given-names">J.M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.V.</span>
          <span class="cit-name-surname">Jacobs</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.M.</span>
          <span class="cit-name-surname">Manos</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">F.X.</span>
          <span class="cit-name-surname">Bosch</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.A.</span>
          <span class="cit-name-surname">Kummer</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.V.</span>
          <span class="cit-name-surname">Shah</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.J.</span>
          <span class="cit-name-surname">Snijders</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Peto</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.J.</span>
          <span class="cit-name-surname">Meijer</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">N.</span>
          <span class="cit-name-surname">Muñoz</span>
         </span>, 
         <span class="cit-article-title">Human papillomavirus is a necessary cause of invasive cervical cancer worldwide</span>. 
         <abbr class="cit-jnl-abbrev">The Journal of pathology</abbr>, 
         <span class="cit-pub-date">1999</span>. 
         <span class="cit-vol">189</span>(
         <span class="cit-issue">1</span>): p. 
         <span class="cit-fpage">12</span>–
         <span class="cit-lpage">19</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bpathology%26rft.stitle%253DJ%2BPathol%26rft.aulast%253DWalboomers%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D189%26rft.issue%253D1%26rft.spage%253D12%26rft.epage%253D19%26rft.atitle%253DHuman%2Bpapillomavirus%2Bis%2Ba%2Bnecessary%2Bcause%2Bof%2Binvasive%2Bcervical%2Bcancer%2Bworldwide.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291096-9896%2528199909%2529189%253A1%253C12%253A%253AAID-PATH431%253E3.0.CO%253B2-F%26rft_id%253Dinfo%253Apmid%252F10451482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/(SICI)1096-9896(199909)189:1&lt;;12::AID-PATH431&gt;;3.0.CO;2-F&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=10451482&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000082360300003&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">50.</span>
      <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.50" data-doi="10.1200/JCO.2011.36.4596">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chaturvedi</span>, 
          <span class="cit-name-given-names">A.K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.A.</span>
          <span class="cit-name-surname">Engels</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.M.</span>
          <span class="cit-name-surname">Pfeiffer</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">B.Y.</span>
          <span class="cit-name-surname">Hernandez</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">W.</span>
          <span class="cit-name-surname">Xiao</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.</span>
          <span class="cit-name-surname">Kim</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">B.</span>
          <span class="cit-name-surname">Jiang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.T.</span>
          <span class="cit-name-surname">Goodman</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Sibug-Saber</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">W.</span>
          <span class="cit-name-surname">Cozen</span>
         </span>, 
         <span class="cit-article-title">Human papillomavirus and rising oropharyngeal cancer incidence in the United States</span>. 
         <abbr class="cit-jnl-abbrev">Journal of clinical oncology</abbr>, 
         <span class="cit-pub-date">2011</span>. 
         <span class="cit-vol">29</span>(
         <span class="cit-issue">32</span>): p. 
         <span class="cit-fpage">4294</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bclinical%2Boncology%26rft_id%253Dinfo%253Adoi%252F10.1200%252FJCO.2011.36.4596%26rft_id%253Dinfo%253Apmid%252F21969503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNvIjtzOjU6InJlc2lkIjtzOjEwOiIyOS8zMi80Mjk0IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMTAvMTUvNDQzNDU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">51.</span>
      <a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.51" data-doi="10.1023/A:1015336817836">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wilson</span>, 
          <span class="cit-name-given-names">V.G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">West</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.</span>
          <span class="cit-name-surname">Woytek</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Rangasamy</span>
         </span>, 
         <span class="cit-article-title">Papillomavirus E1 proteins: form, function, and features</span>. 
         <abbr class="cit-jnl-abbrev">Virus Genes</abbr>, 
         <span class="cit-pub-date">2002</span>. 
         <span class="cit-vol">24</span>(
         <span class="cit-issue">3</span>): p. 
         <span class="cit-fpage">275</span>–
         <span class="cit-lpage">90</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVirus%2Bgenes%26rft.stitle%253DVirus%2BGenes%26rft.aulast%253DWilson%26rft.auinit1%253DV.%2BG.%26rft.volume%253D24%26rft.issue%253D3%26rft.spage%253D275%26rft.epage%253D290%26rft.atitle%253DPapillomavirus%2BE1%2Bproteins%253A%2Bform%252C%2Bfunction%252C%2Band%2Bfeatures.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1015336817836%26rft_id%253Dinfo%253Apmid%252F12086149%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1023/A:1015336817836&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=12086149&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000175308500010&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">52.</span>
      <a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.52" data-doi="10.1007/s00705-011-1008-x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Johne</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.B.</span>
          <span class="cit-name-surname">Buck</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Allander</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">W.J.</span>
          <span class="cit-name-surname">Atwood</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.L.</span>
          <span class="cit-name-surname">Garcea</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.J.</span>
          <span class="cit-name-surname">Imperiale</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.O.</span>
          <span class="cit-name-surname">Major</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Ramqvist</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">L.C.</span>
          <span class="cit-name-surname">Norkin</span>
         </span>, 
         <span class="cit-article-title">Taxonomical developments in the family Polyomaviridae</span>. 
         <abbr class="cit-jnl-abbrev">Archives of virology</abbr>, 
         <span class="cit-pub-date">2011</span>. 
         <span class="cit-vol">156</span>(
         <span class="cit-issue">9</span>): p. 
         <span class="cit-fpage">1627</span>–
         <span class="cit-lpage">1634</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DArchives%2Bof%2Bvirology%26rft.stitle%253DArch%2BVirol%26rft.aulast%253DJohne%26rft.auinit1%253DR.%26rft.volume%253D156%26rft.issue%253D9%26rft.spage%253D1627%26rft.epage%253D1634%26rft.atitle%253DTaxonomical%2Bdevelopments%2Bin%2Bthe%2Bfamily%2BPolyomaviridae.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00705-011-1008-x%26rft_id%253Dinfo%253Apmid%252F21562881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1007/s00705-011-1008-x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21562881&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">53.</span>
      <a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.53" data-doi="10.1371/journal.ppat.1005574">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Buck</span>, 
          <span class="cit-name-given-names">C.B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.</span>
          <span class="cit-name-surname">Van Doorslaer</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Peretti</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.M.</span>
          <span class="cit-name-surname">Geoghegan</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.J.</span>
          <span class="cit-name-surname">Tisza</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">An</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.P.</span>
          <span class="cit-name-surname">Katz</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.M.</span>
          <span class="cit-name-surname">Pipas</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.A.</span>
          <span class="cit-name-surname">McBride</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.C.</span>
          <span class="cit-name-surname">Camus</span>
         </span>, 
         <span class="cit-article-title">The ancient evolutionary history of polyomaviruses</span>. 
         <abbr class="cit-jnl-abbrev">PLoS pathogens</abbr>, 
         <span class="cit-pub-date">2016</span>. 
         <span class="cit-vol">12</span>(
         <span class="cit-issue">4</span>): p. 
         <span class="cit-fpage">e1005574</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2Bpathogens%26rft.volume%253D12%26rft.spage%253De1005574%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1005574%26rft_id%253Dinfo%253Apmid%252F27093155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1005574&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=27093155&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">54.</span>
      <a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.54" data-doi="10.1128/JVI.01662-07">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Woolford</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Rector</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Van Ranst</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Ducki</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.D.</span>
          <span class="cit-name-surname">Bennett</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.K.</span>
          <span class="cit-name-surname">Nicholls</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.S.</span>
          <span class="cit-name-surname">Warren</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.A.</span>
          <span class="cit-name-surname">Swan</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">G.E.</span>
          <span class="cit-name-surname">Wilcox</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.J.</span>
          <span class="cit-name-surname">O’Hara</span>
         </span>, 
         <span class="cit-article-title">A novel virus detected in papillomas and carcinomas of the endangered western barred bandicoot (Perameles bougainville) exhibits genomic features of both the Papillomaviridae and Polyomaviridae</span>. 
         <abbr class="cit-jnl-abbrev">Journal of virology</abbr>, 
         <span class="cit-pub-date">2007</span>. 
         <span class="cit-vol">81</span>(
         <span class="cit-issue">24</span>): p. 
         <span class="cit-fpage">13280</span>–
         <span class="cit-lpage">13290</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01662-07%26rft_id%253Dinfo%253Apmid%252F17898069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4MS8yNC8xMzI4MCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzEwLzE1LzQ0MzQ1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">55.</span>
      <a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.55" data-doi="10.1016/j.virol.2008.03.014">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bennett</span>, 
          <span class="cit-name-given-names">M.D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L.</span>
          <span class="cit-name-surname">Woolford</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">H.</span>
          <span class="cit-name-surname">Stevens</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Van Ranst</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Oldfield</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Slaven</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.J.</span>
          <span class="cit-name-surname">O’Hara</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.S.</span>
          <span class="cit-name-surname">Warren</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.K.</span>
          <span class="cit-name-surname">Nicholls</span>
         </span>, 
         <span class="cit-article-title">Genomic characterization of a novel virus found in papillomatous lesions from a southern brown bandicoot (Isoodon obesulus) in Western Australia</span>. 
         <abbr class="cit-jnl-abbrev">Virology</abbr>, 
         <span class="cit-pub-date">2008</span>. 
         <span class="cit-vol">376</span>(
         <span class="cit-issue">1</span>): p. 
         <span class="cit-fpage">173</span>–
         <span class="cit-lpage">182</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DBennett%26rft.auinit1%253DM.%2BD.%26rft.volume%253D376%26rft.issue%253D1%26rft.spage%253D173%26rft.epage%253D182%26rft.atitle%253DGenomic%2Bcharacterization%2Bof%2Ba%2Bnovel%2Bvirus%2Bfound%2Bin%2Bpapillomatous%2Blesions%2Bfrom%2Ba%2Bsouthern%2Bbrown%2Bbandicoot%2B%2528Isoodon%2Bobesulus%2529%2Bin%2BWestern%2BAustralia.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2008.03.014%26rft_id%253Dinfo%253Apmid%252F18440042%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.virol.2008.03.014&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18440042&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">56.</span>
      <a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a>
      <div class="cit ref-cit ref-book" id="cit-443457v1.56">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lukert</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">G. De.</span>
          <span class="cit-name-surname">Boer</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Dale</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.K.</span>
          <span class="cit-name-surname">Keese</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">McNulty</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.W.</span>
          <span class="cit-name-surname">Randles</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">I.</span>
          <span class="cit-name-surname">Tischer</span>
         </span>, 
         <span class="cit-chapter-title">Family Circoviridae</span>. 
         <span class="cit-pub-date">1995</span>, 
         <span class="cit-publ-name">Springer-Verlag</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">57.</span>
      <a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.57">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Shi</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.-Z.</span>
          <span class="cit-name-surname">Zhang</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.C.</span>
          <span class="cit-name-surname">Holmes</span>
         </span>, 
         <span class="cit-article-title">Meta-transcriptomics and The Evolutionary Biology of RNA Viruses</span>. 
         <abbr class="cit-jnl-abbrev">Virus research</abbr>, 
         <span class="cit-pub-date">2017</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">58.</span>
      <a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.58" data-doi="10.1186/s12864-014-1207-4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kleiner</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L.V.</span>
          <span class="cit-name-surname">Hooper</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">B.A.</span>
          <span class="cit-name-surname">Duerkop</span>
         </span>, 
         <span class="cit-article-title">Evaluation of methods to purify virus-like particles for metagenomic sequencing of intestinal viromes</span>. 
         <abbr class="cit-jnl-abbrev">BMC. genomics</abbr>, 
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-vol">16</span>(
         <span class="cit-issue">1</span>): p. 
         <span class="cit-fpage">7</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC.%2Bgenomics%26rft.volume%253D16%26rft.spage%253D7%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-014-1207-4%26rft_id%253Dinfo%253Apmid%252F25608871%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/s12864-014-1207-4&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25608871&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">59.</span>
      <a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.59" data-doi="10.1016/j.jviromet.2014.12.002">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">X.</span>
          <span class="cit-name-surname">Deng</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.T.</span>
          <span class="cit-name-surname">Mee</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Collot-Teixeira</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Anderson</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Schepelmann</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.D.</span>
          <span class="cit-name-surname">Minor</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.</span>
          <span class="cit-name-surname">Delwart</span>
         </span>, 
         <span class="cit-article-title">Comparing viral metagenomics methods using a highly multiplexed human viral pathogens reagent</span>. 
         <abbr class="cit-jnl-abbrev">Journal of virological methods</abbr>, 
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-vol">213</span>: p. 
         <span class="cit-fpage">139</span>–
         <span class="cit-lpage">146</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirological%2Bmethods%26rft.volume%253D213%26rft.spage%253D139%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jviromet.2014.12.002%26rft_id%253Dinfo%253Apmid%252F25497414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.jviromet.2014.12.002&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25497414&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">60.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-443457v1.60">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Karlsson</span>, 
          <span class="cit-name-given-names">O.E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Belák</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">F.</span>
          <span class="cit-name-surname">Granberg</span>
         </span>, 
         <span class="cit-article-title">The effect of preprocessing by sequence-independent, single-primer amplification (SISPA) on metagenomic detection of viruses</span>. 
         <abbr class="cit-jnl-abbrev">Biosecurity and bioterrorism: biodefense strategy, practice, and science</abbr>, 
         <span class="cit-pub-date">2013</span>. 
         <span class="cit-vol">11</span>(
         <span class="cit-issue">S1</span>): p. 
         <span class="cit-fpage">S227</span>–
         <span class="cit-lpage">S234</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBiosecurity%2Band%2Bbioterrorism%253A%2Bbiodefense%2Bstrategy%252C%2Bpractice%252C%2Band%2Bscience%26rft.volume%253D11%26rft.spage%253D227S%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">61.</span>
      <a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.61" data-doi="10.1016/j.jviromet.2015.05.010">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rosseel</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">O.</span>
          <span class="cit-name-surname">Ozhelvaci</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">G.</span>
          <span class="cit-name-surname">Freimanis</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Van Borm</span>
         </span>, 
         <span class="cit-article-title">Evaluation of convenient pretreatment protocols for RNA virus metagenomics in serum and tissue samples</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Virological Methods</abbr>, 
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-vol">222</span>: p. 
         <span class="cit-fpage">72</span>–
         <span class="cit-lpage">80</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirological%2BMethods%26rft.volume%253D222%26rft.spage%253D72%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jviromet.2015.05.010%26rft_id%253Dinfo%253Apmid%252F26025457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.jviromet.2015.05.010&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26025457&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">62.</span>
      <a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.62">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Łusiak-Szelachowska</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">B.</span>
          <span class="cit-name-surname">Weber-Dąbrowska</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.</span>
          <span class="cit-name-surname">Jończyk-Matysiak</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Wojciechowska</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Górski</span>
         </span>, 
         <span class="cit-article-title">Bacteriophages in the gastrointestinal tract and their implications</span>. 
         <abbr class="cit-jnl-abbrev">Gut Pathogens</abbr>, 
         <span class="cit-pub-date">2017</span>. 
         <span class="cit-vol">9</span>: p. 
         <span class="cit-fpage">44</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGut%2BPathogens%26rft.volume%253D9%26rft.spage%253D44%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">63.</span>
      <a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.63" data-doi="10.1111/apt.14280">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Carding</span>, 
          <span class="cit-name-given-names">S.R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">N.</span>
          <span class="cit-name-surname">Davis</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">L.</span>
          <span class="cit-name-surname">Hoyles</span>
         </span>, 
         <span class="cit-article-title">Review article: the human intestinal virome in health and disease</span>. 
         <abbr class="cit-jnl-abbrev">Alimentary Pharmacology &amp;; Therapeutics</abbr>, 
         <span class="cit-pub-date">2017</span>. 
         <span class="cit-vol">46</span>(
         <span class="cit-issue">9</span>): p. 
         <span class="cit-fpage">800</span>–
         <span class="cit-lpage">815</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAlimentary%2BPharmacology%2B%2526%2BTherapeutics%26rft.volume%253D46%26rft.spage%253D800%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fapt.14280%26rft_id%253Dinfo%253Apmid%252F28869283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/apt.14280&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=28869283&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">64.</span>
      <a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.64" data-doi="10.1128/JVI.00357-16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Marzano</span>, 
          <span class="cit-name-given-names">S.-Y.L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">B.D.</span>
          <span class="cit-name-surname">Nelson</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">O.</span>
          <span class="cit-name-surname">Ajayi-Oyetunde</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.A.</span>
          <span class="cit-name-surname">Bradley</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.J.</span>
          <span class="cit-name-surname">Hughes</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">G.L.</span>
          <span class="cit-name-surname">Hartman</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.M.</span>
          <span class="cit-name-surname">Eastburn</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">L.L.</span>
          <span class="cit-name-surname">Domier</span>
         </span>, 
         <span class="cit-article-title">Identification of diverse mycoviruses through metatranscriptomics characterization of the viromes of five major fungal plant pathogens</span>. 
         <abbr class="cit-jnl-abbrev">Journal of virology</abbr>, 
         <span class="cit-pub-date">2016</span>. 
         <span class="cit-vol">90</span>(
         <span class="cit-issue">15</span>): p. 
         <span class="cit-fpage">6846</span>–
         <span class="cit-lpage">6863</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00357-16%26rft_id%253Dinfo%253Apmid%252F27194764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI5MC8xNS82ODQ2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMTAvMTUvNDQzNDU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">65.</span>
      <a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.65" data-doi="10.1186/s40168-014-0066-1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Cheng</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Fox</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Pemberton</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Hogg</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.T.</span>
          <span class="cit-name-surname">Papenfuss</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.</span>
          <span class="cit-name-surname">Belov</span>
         </span>, 
         <span class="cit-article-title">The Tasmanian devil microbiome—implications for conservation and management</span>. 
         <abbr class="cit-jnl-abbrev">Microbiome</abbr>, 
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-vol">3</span>(
         <span class="cit-issue">1</span>): p. 
         <span class="cit-fpage">1</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMicrobiome%26rft.volume%253D3%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs40168-014-0066-1%26rft_id%253Dinfo%253Apmid%252F25621171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/s40168-014-0066-1&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25621171&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">66.</span>
      <a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66. in text" id="ref-66">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.66">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wise</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Lee</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Peck</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Clarke</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Thalmann</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Hockley</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Schaap</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Pemberton</span>
         </span>, 
         <span class="cit-article-title">The conservation introduction of Tasmanian devils to Maria Island National Park: a response to devil facial tumor disease (DFTD)</span>. 
         <abbr class="cit-jnl-abbrev">Global Re-introduction Perspectives: 2016. Case-studies from around the globe</abbr>, 
         <span class="cit-pub-date">2016</span>: p. 
         <span class="cit-fpage">166</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">67.</span>
      <a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference 67. in text" id="ref-67">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.67">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Cheng</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.</span>
          <span class="cit-name-surname">Heasman</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Peck</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.</span>
          <span class="cit-name-surname">Peel</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.M.</span>
          <span class="cit-name-surname">Gooley</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.T.</span>
          <span class="cit-name-surname">Papenfuss</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.J.</span>
          <span class="cit-name-surname">Hogg</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.</span>
          <span class="cit-name-surname">Belov</span>
         </span>, 
         <span class="cit-article-title">Significant decline in anticancer immune capacity during puberty in the Tasmanian devil</span>. 
         <abbr class="cit-jnl-abbrev">Scientific Reports</abbr>, 
         <span class="cit-pub-date">2017</span>. 
         <span class="cit-vol">7</span>: p. 
         <span class="cit-fpage">44716</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScientific%2BReports%26rft.volume%253D7%26rft.spage%253D44716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">68.</span>
      <a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference 68. in text" id="ref-68">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.68" data-doi="10.1177/0300985810380398">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kreiss</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Tovar</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Obendorf</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.</span>
          <span class="cit-name-surname">Dun</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">G.</span>
          <span class="cit-name-surname">Woods</span>
         </span>, 
         <span class="cit-article-title">A murine xenograft model for a transmissible cancer in Tasmanian devils</span>. 
         <abbr class="cit-jnl-abbrev">Veterinary pathology</abbr>, 
         <span class="cit-pub-date">2011</span>. 
         <span class="cit-vol">48</span>(
         <span class="cit-issue">2</span>): p. 
         <span class="cit-fpage">475</span>–
         <span class="cit-lpage">481</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVeterinary%2Bpathology%26rft_id%253Dinfo%253Adoi%252F10.1177%252F0300985810380398%26rft_id%253Dinfo%253Apmid%252F20861503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1177/0300985810380398&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20861503&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">69.</span>
      <a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference 69. in text" id="ref-69">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.69" data-doi="10.1007/s13365-014-0234-x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wiedinger</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Bitsaktsis</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Chang</span>
         </span>, 
         <span class="cit-article-title">Reactivation of human polyomaviruses in immunocompromised states</span>. 
         <abbr class="cit-jnl-abbrev">Journal of neurovirology</abbr>, 
         <span class="cit-pub-date">2014</span>. 
         <span class="cit-vol">20</span>(
         <span class="cit-issue">1</span>): p. 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">8</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bneurovirology%26rft.volume%253D20%26rft.issue%253D1%26rft.spage%253D1%26rft.atitle%253DREACTIVATION%2BOF%2BHUMAN%2BPOLYOMAVIRUSES%2BIN%2BIMMUNOCOMPROMISED%2BSTATES%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs13365-014-0234-x%26rft_id%253Dinfo%253Apmid%252F24481784%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1007/s13365-014-0234-x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24481784&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">70.</span>
      <a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference 70. in text" id="ref-70">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.70" data-doi="10.1128/JVI.01839-13">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">X.</span>
          <span class="cit-name-surname">Deng</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Linsuwanon</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Bangsberg</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.B.</span>
          <span class="cit-name-surname">Bwana</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Hunt</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.N.</span>
          <span class="cit-name-surname">Martin</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.G.</span>
          <span class="cit-name-surname">Deeks</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.</span>
          <span class="cit-name-surname">Delwart</span>
         </span>, 
         <span class="cit-article-title">AIDS Alters the Commensal Plasma Virome</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>, 
         <span class="cit-pub-date">2013</span>. 
         <span class="cit-vol">87</span>(
         <span class="cit-issue">19</span>): p. 
         <span class="cit-fpage">10912</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01839-13%26rft_id%253Dinfo%253Apmid%252F23903845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4Ny8xOS8xMDkxMiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzEwLzE1LzQ0MzQ1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">71.</span>
      <a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference 71. in text" id="ref-71">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.71" data-doi="10.1186/s40168-017-0308-0">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Yang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Shan</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Hou</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Z.</span>
          <span class="cit-name-surname">Liu</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">W.</span>
          <span class="cit-name-surname">Li</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L.</span>
          <span class="cit-name-surname">Guo</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.</span>
          <span class="cit-name-surname">Wang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Chen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">X.</span>
          <span class="cit-name-surname">Wang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">F.</span>
          <span class="cit-name-surname">Feng</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">H.</span>
          <span class="cit-name-surname">Wang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Chen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Q.</span>
          <span class="cit-name-surname">Shen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Zhou</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">X.</span>
          <span class="cit-name-surname">Hua</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L.</span>
          <span class="cit-name-surname">Cui</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">X.</span>
          <span class="cit-name-surname">Deng</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Z.</span>
          <span class="cit-name-surname">Zhang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Qi</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.</span>
          <span class="cit-name-surname">Delwart</span>
         </span>, 
         <span class="cit-article-title">Virome comparisons in wild-diseased and healthy captive giant pandas</span>. 
         <abbr class="cit-jnl-abbrev">Microbiome</abbr>, 
         <span class="cit-pub-date">2017</span>. 
         <span class="cit-vol">5</span>(
         <span class="cit-issue">1</span>): p. 
         <span class="cit-fpage">90</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMicrobiome%26rft.volume%253D5%26rft.spage%253D90%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs40168-017-0308-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/s40168-017-0308-0&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">72.</span>
      <a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference 72. in text" id="ref-72">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.72" data-doi="10.1007/s00251-012-0614-4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Cheng</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Sanderson</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Jones</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.</span>
          <span class="cit-name-surname">Belov</span>
         </span>, 
         <span class="cit-article-title">Low MHC class II diversity in the Tasmanian devil (Sarcophilus harrisii)</span>. 
         <abbr class="cit-jnl-abbrev">Immunogenetics</abbr>, 
         <span class="cit-pub-date">2012</span>. 
         <span class="cit-vol">64</span>(
         <span class="cit-issue">7</span>): p. 
         <span class="cit-fpage">525</span>–
         <span class="cit-lpage">533</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DImmunogenetics%26rft.volume%253D64%26rft.spage%253D525%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00251-012-0614-4%26rft_id%253Dinfo%253Apmid%252F22460528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1007/s00251-012-0614-4&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22460528&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000305130300004&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">73.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-443457v1.73">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Morris</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.J.</span>
          <span class="cit-name-surname">Austin</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.</span>
          <span class="cit-name-surname">Belov</span>
         </span>, 
         <span class="cit-article-title">Low major histocompatibility complex diversity in the Tasmanian devil predates European settlement and may explain susceptibility to disease epidemics</span>. 
         <abbr class="cit-jnl-abbrev">Biology letters</abbr>, 
         <span class="cit-pub-date">2013</span>. 
         <span class="cit-vol">9</span>(
         <span class="cit-issue">1</span>): p. 
         <span class="cit-fpage">20120900</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBiology%2Bletters%26rft.volume%253D9%26rft.spage%253D20120900%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">74.</span>
      <a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference 74. in text" id="ref-74">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.74" data-doi="10.1073/pnas.1102838108">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Miller</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">V.M.</span>
          <span class="cit-name-surname">Hayes</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Ratan</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.C.</span>
          <span class="cit-name-surname">Petersen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">N.E.</span>
          <span class="cit-name-surname">Wittekindt</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Miller</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">B.</span>
          <span class="cit-name-surname">Walenz</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Knight</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Qi</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">F.</span>
          <span class="cit-name-surname">Zhao</span>
         </span>, 
         <span class="cit-article-title">Genetic diversity and population structure of the endangered marsupial Sarcophilus harrisii (Tasmanian devil)</span>. 
         <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr>, 
         <span class="cit-pub-date">2011</span>. 
         <span class="cit-vol">108</span>(
         <span class="cit-issue">30</span>): p. 
         <span class="cit-fpage">12348</span>–
         <span class="cit-lpage">12353</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1102838108%26rft_id%253Dinfo%253Apmid%252F21709235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzMwLzEyMzQ4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMTAvMTUvNDQzNDU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">75.</span>
      <a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference 75. in text" id="ref-75">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.75" data-doi="10.1146/annurev.ecolsys.37.091305.110145">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Willi</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Van Buskirk</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.A.</span>
          <span class="cit-name-surname">Hoffmann</span>
         </span>, 
         <span class="cit-article-title">Limits to the adaptive potential of small populations</span>. 
         <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Evol. Syst.</abbr>, 
         <span class="cit-pub-date">2006</span>. 
         <span class="cit-vol">37</span>: p. 
         <span class="cit-fpage">433</span>–
         <span class="cit-lpage">458</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BEcol.%2BEvol.%2BSyst.%26rft.volume%253D37%26rft.spage%253D433%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.37.091305.110145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.37.091305.110145&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">76.</span>
      <a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference 76. in text" id="ref-76">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.76" data-doi="10.1111/j.1461-0248.2005.00735.x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pearman</span>, 
          <span class="cit-name-given-names">P.B.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.W.</span>
          <span class="cit-name-surname">Garner</span>
         </span>, 
         <span class="cit-article-title">Susceptibility of Italian agile frog populations to an emerging strain of Ranavirus parallels population genetic diversity</span>. 
         <abbr class="cit-jnl-abbrev">Ecology Letters</abbr>, 
         <span class="cit-pub-date">2005</span>. 
         <span class="cit-vol">8</span>(
         <span class="cit-issue">4</span>): p. 
         <span class="cit-fpage">401</span>–
         <span class="cit-lpage">408</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEcology%2BLetters%26rft.volume%253D8%26rft.spage%253D401%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2005.00735.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2005.00735.x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=000228020200008&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">77.</span>
      <a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference 77. in text" id="ref-77">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.77" data-doi="10.1126/science.2983425">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">O’Brien</span>, 
          <span class="cit-name-given-names">S.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.E.</span>
          <span class="cit-name-surname">Roelke</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L.</span>
          <span class="cit-name-surname">Marker</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Newman</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Winkler</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Meltzer</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L.</span>
          <span class="cit-name-surname">Colly</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Evermann</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Bush</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.E.</span>
          <span class="cit-name-surname">Wildt</span>
         </span>, 
         <span class="cit-article-title">Genetic basis for species vulnerability in the cheetah</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>, 
         <span class="cit-pub-date">1985</span>. 
         <span class="cit-vol">227</span>(
         <span class="cit-issue">4693</span>): p. 
         <span class="cit-fpage">1428</span>–
         <span class="cit-lpage">1434</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DO%2527Brien%26rft.auinit1%253DS.%26rft.volume%253D227%26rft.issue%253D4693%26rft.spage%253D1428%26rft.epage%253D1434%26rft.atitle%253DGenetic%2Bbasis%2Bfor%2Bspecies%2Bvulnerability%2Bin%2Bthe%2Bcheetah%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.2983425%26rft_id%253Dinfo%253Apmid%252F2983425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyMjcvNDY5My8xNDI4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMTAvMTUvNDQzNDU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">78.</span>
      <a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference 78. in text" id="ref-78">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.78" data-doi="10.1016/0042-6822(92)90896-W">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Van Ranst</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Fuse</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Fiten</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.</span>
          <span class="cit-name-surname">Beuken</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">H.</span>
          <span class="cit-name-surname">Pfister</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.D.</span>
          <span class="cit-name-surname">Burk</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">G.</span>
          <span class="cit-name-surname">Opdenakker</span>
         </span>, 
         <span class="cit-article-title">Human papillomavirus type 13 and pygmy chimpanzee papillomavirus type 1: comparison of the genome organizations</span>. 
         <abbr class="cit-jnl-abbrev">Virology</abbr>, 
         <span class="cit-pub-date">1992</span>. 
         <span class="cit-vol">190</span>(
         <span class="cit-issue">2</span>): p. 
         <span class="cit-fpage">587</span>–
         <span class="cit-lpage">596</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DVan%2BRanst%26rft.auinit1%253DM.%26rft.volume%253D190%26rft.issue%253D2%26rft.spage%253D587%26rft.epage%253D596%26rft.atitle%253DHuman%2Bpapillomavirus%2Btype%2B13%2Band%2Bpygmy%2Bchimpanzee%2Bpapillomavirus%2Btype%2B1%253A%2Bcomparison%2Bof%2Bthe%2Bgenome%2Borganizations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0042-6822%252892%252990896-W%26rft_id%253Dinfo%253Apmid%252F1325697%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/0042-6822(92)90896-W&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=1325697&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1992JM76500003&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">79.</span>
      <a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference 79. in text" id="ref-79">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-443457v1.79" data-doi="10.1006/viro.2002.1566">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Tachezy</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">G.</span>
          <span class="cit-name-surname">Duson</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Rector</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.B.</span>
          <span class="cit-name-surname">Jenson</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.P.</span>
          <span class="cit-name-surname">Sundberg</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Van Ranst</span>
         </span>, 
         <span class="cit-article-title">Cloning and genomic characterization of Felis domesticus papillomavirus type 1</span>. 
         <abbr class="cit-jnl-abbrev">Virology</abbr>, 
         <span class="cit-pub-date">2002</span>. 
         <span class="cit-vol">301</span>(
         <span class="cit-issue">2</span>): p. 
         <span class="cit-fpage">313</span>–
         <span class="cit-lpage">321</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DTachezy%26rft.auinit1%253DR.%26rft.volume%253D301%26rft.issue%253D2%26rft.spage%253D313%26rft.epage%253D321%26rft.atitle%253DCloning%2Band%2Bgenomic%2Bcharacterization%2Bof%2BFelis%2Bdomesticus%2Bpapillomavirus%2Btype%2B1.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fviro.2002.1566%26rft_id%253Dinfo%253Apmid%252F12359433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1006/viro.2002.1566&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=12359433&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F15%2F443457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000178376400012&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
