<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/embed/inline-graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2017/12/20/173211.atom" />
<meta name="HW.pisa" content="biorxiv;173211v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="MERS-CoV spillover at the camel-human interface" />
<meta name="DC.Identifier" content="10.1101/173211" />
<meta name="DC.Date" content="2017-12-20" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2017, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Middle East respiratory syndrome coronavirus (MERS-CoV) is a zoonotic virus from camels causing significant mortality and morbidity in humans in the Arabian Peninsula. The epidemiology of the virus remains poorly understood, and while case-based and seroepidemiological studies have been employed extensively throughout the epidemic, viral sequence data have not been utilised to their full potential. Here we use existing MERS-CoV sequence data to explore its phylodynamics in two of its known major hosts, humans and camels. We employ structured coalescent models to show that long-term MERS-CoV evolution occurs exclusively in camels, whereas humans act as a transient, and ultimately terminal host. By analysing the distribution of human outbreak cluster sizes and zoonotic introduction times we show that human outbreaks in the Arabian peninsula are driven by seasonally varying zoonotic transfer of viruses from camels. Without heretofore unseen evolution of host tropism, MERS-CoV is unlikely to become endemic in humans." />
<meta name="DC.Contributor" content="Gytis Dudas" />
<meta name="DC.Contributor" content="Luiz Max Carvalho" />
<meta name="DC.Contributor" content="Andrew Rambaut" />
<meta name="DC.Contributor" content="Trevor Bedford" />
<meta name="DC.Contributor" content="Ali M. Somily" />
<meta name="DC.Contributor" content="Mazin Barry" />
<meta name="DC.Contributor" content="Sarah S. Al Subaie" />
<meta name="DC.Contributor" content="Abdulaziz A. BinSaeed" />
<meta name="DC.Contributor" content="Fahad A. Alzamil" />
<meta name="DC.Contributor" content="Waleed Zaher" />
<meta name="DC.Contributor" content="Theeb Al Qahtani" />
<meta name="DC.Contributor" content="Khaldoon Al Jerian" />
<meta name="DC.Contributor" content="Scott J.N. McNabb" />
<meta name="DC.Contributor" content="Imad A. Al-Jahdali" />
<meta name="DC.Contributor" content="Ahmed M. Alotaibi" />
<meta name="DC.Contributor" content="Nahid A. Batarfi" />
<meta name="DC.Contributor" content="Matthew Cotten" />
<meta name="DC.Contributor" content="Simon J. Watson" />
<meta name="DC.Contributor" content="Spela Binter" />
<meta name="DC.Contributor" content="Paul Kellam" />
<meta name="article:published_time" content="2017-12-20" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="MERS-CoV spillover at the camel-human interface" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Middle East respiratory syndrome coronavirus (MERS-CoV) is a zoonotic virus from camels causing significant mortality and morbidity in humans in the Arabian Peninsula. The epidemiology of the virus remains poorly understood, and while case-based and seroepidemiological studies have been employed extensively throughout the epidemic, viral sequence data have not been utilised to their full potential. Here we use existing MERS-CoV sequence data to explore its phylodynamics in two of its known major hosts, humans and camels. We employ structured coalescent models to show that long-term MERS-CoV evolution occurs exclusively in camels, whereas humans act as a transient, and ultimately terminal host. By analysing the distribution of human outbreak cluster sizes and zoonotic introduction times we show that human outbreaks in the Arabian peninsula are driven by seasonally varying zoonotic transfer of viruses from camels. Without heretofore unseen evolution of host tropism, MERS-CoV is unlikely to become endemic in humans.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2017/01/01" />
<meta name="citation_mjid" content="biorxiv;173211v2" />
<meta name="citation_id" content="173211v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/173211v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/173211v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/173211v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211.full.pdf" />
<meta name="citation_doi" content="10.1101/173211" />
<meta name="citation_num_pages" content="45" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="173211" />
<meta name="citation_author" content="Gytis Dudas" />
<meta name="citation_author_institution" content="College of Medicine, King Saud University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0227-4158" />
<meta name="citation_author" content="Luiz Max Carvalho" />
<meta name="citation_author_institution" content="Institute of Evolutionary Biology, University of Edinburgh" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5736-5578" />
<meta name="citation_author" content="Andrew Rambaut" />
<meta name="citation_author_institution" content="Institute of Evolutionary Biology, University of Edinburgh" />
<meta name="citation_author_institution" content="Fogarty International Center, National Institutes of Health" />
<meta name="citation_author" content="Trevor Bedford" />
<meta name="citation_author_institution" content="Vaccine and Infectious Disease Division, Fred Hutchinson Cancer Research Center" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4039-5794" />
<meta name="citation_author" content="Ali M. Somily" />
<meta name="citation_author_institution" content="College of Medicine, King Saud University" />
<meta name="citation_author" content="Mazin Barry" />
<meta name="citation_author_institution" content="College of Medicine, King Saud University" />
<meta name="citation_author" content="Sarah S. Al Subaie" />
<meta name="citation_author_institution" content="College of Medicine, King Saud University" />
<meta name="citation_author" content="Abdulaziz A. BinSaeed" />
<meta name="citation_author_institution" content="College of Medicine, King Saud University" />
<meta name="citation_author" content="Fahad A. Alzamil" />
<meta name="citation_author_institution" content="College of Medicine, King Saud University" />
<meta name="citation_author" content="Waleed Zaher" />
<meta name="citation_author_institution" content="College of Medicine, King Saud University" />
<meta name="citation_author" content="Theeb Al Qahtani" />
<meta name="citation_author_institution" content="College of Medicine, King Saud University" />
<meta name="citation_author" content="Khaldoon Al Jerian" />
<meta name="citation_author_institution" content="College of Medicine, King Saud University" />
<meta name="citation_author" content="Scott J.N. McNabb" />
<meta name="citation_author_institution" content="Rollins School of Public Health, Emory University" />
<meta name="citation_author" content="Imad A. Al-Jahdali" />
<meta name="citation_author_institution" content="Deputy Minister. Ex. General Director King Fahad General Hospital, Jeddah and Occupational and environmental medicine, Um AlQura University" />
<meta name="citation_author" content="Ahmed M. Alotaibi" />
<meta name="citation_author_institution" content="Department of Intensive Care, Prince Mohammed bin Abdulaziz Hospital" />
<meta name="citation_author" content="Nahid A. Batarfi" />
<meta name="citation_author_institution" content="Epidemiology section, Command and Control Center (CCC) Ministry of Health" />
<meta name="citation_author" content="Matthew Cotten" />
<meta name="citation_author_institution" content="Wellcome Trust Sanger Institute" />
<meta name="citation_author" content="Simon J. Watson" />
<meta name="citation_author_institution" content="Wellcome Trust Sanger Institute" />
<meta name="citation_author" content="Spela Binter" />
<meta name="citation_author_institution" content="Wellcome Trust Sanger Institute" />
<meta name="citation_author" content="Paul Kellam" />
<meta name="citation_author_institution" content="Wellcome Trust Sanger Institute" />
<meta name="citation_reference" content="Abdallah H, Faye B. 2013. Typology of camel farming system in Saudi Arabia. Emirates Journal of Food and Agriculture. 25:250." />
<meta name="citation_reference" content="Adney DR, van Doremalen N, Brown VR, Bushmaker T, Scott D, de Wit E, Bowen RA, Munster VJ. 2014. Replication and Shedding of MERS-CoV in Upper Respiratory Tract of Inoculated Dromedary Camels. Emerging Infectious Diseases. 20:1999–2005." />
<meta name="citation_reference" content="Ali MA, Shehata MM, Gomaa MR, et al. (18 co-authors). 2017. Systematic, active surveillance for Middle East respiratory syndrome coronavirus in camels in Egypt. Emerg Microbes Infect. 6:e1." />
<meta name="citation_reference" content="Almutairi SE, Boujenane I, Musaad A, Awad-Acharari F. 2010. Non-genetic factors influencing reproductive traits and calving weight in Saudi camels. Trop Anim Health Prod. 42:1087–1092." />
<meta name="citation_reference" content="Antia R, Regoes RR, Koella JC, Bergstrom CT. 2003. The role of evolution in the emergence of infectious diseases. Nature. 426:658–661." />
<meta name="citation_reference" content="Assiri A, Al-Tawfiq JA, Al-Rabeeah AA, et al. (13 co-authors). 2013a. Epidemiological, demographic, and clinical characteristics of 47 cases of Middle East respiratory syndrome coronavirus disease from Saudi Arabia: a descriptive study. Lancet Infect Dis. 13:752–761." />
<meta name="citation_reference" content="Assiri A, McGeer A, Perl TM, et al. (18 co-authors). 2013b. Hospital outbreak of Middle East respiratory syndrome coronavirus. N Engl J Med. 369:407–416." />
<meta name="citation_reference" content="Bedford T, Cobey S, Pascual M. 2011. Strength and tempo of selection revealed in viral gene genealogies. BMC Evol Biol. 11:220." />
<meta name="citation_reference" content="Blumberg S, Lloyd-Smith JO. 2013. Inference of R0 and transmission heterogeneity from the size distribution of stuttering chains. PLoS Comput Biol. 9:e1002993." />
<meta name="citation_reference" content="Boheemen Sv, Graaf Md, Lauber C, et al. (11 co-authors). 2012. Genomic characterization of a newly discovered coronavirus associated with acute respiratory distress syndrome in humans. mBio. 3:e00473–12." />
<meta name="citation_reference" content="Boni MF, Posada D, Feldman MW. 2007. An exact nonparametric method for inferring mosaic structure in sequence triplets. Genetics. 176:1035–1047." />
<meta name="citation_reference" content="Bouckaert R, Heled J, Kühnert D, Vaughan T, Wu CH, Xie D, Suchard MA, Rambaut A, Drummond AJ. 2014. BEAST 2: A software platform for Bayesian evolutionary analysis. PLoS Comput Biol. 10:e1003537." />
<meta name="citation_reference" content="Bruen TC, Philippe H, Bryant D. 2006. A simple and robust statistical test for detecting the presence of recombination. Genetics. 172:2665–2681." />
<meta name="citation_reference" content="Carpenter B, Gelman A, Hoffman M, Lee D, Goodrich B, Betancourt M, Brubaker MA, Guo J, Li P, Riddell A. 2016. Stan: A probabilistic programming language. J Stat Softw. 20:1–37." />
<meta name="citation_reference" content="Cauchemez S, Nouvellet P, Cori A, et al. (25 co-authors). 2016. Unraveling the drivers of MERS-CoV transmission. Proc Natl Acad Sci USA. 113:9081–9086." />
<meta name="citation_reference" content="Chen X, Chughtai AA, Dyda A, MacIntyre CR. 2017. Comparative epidemiology of Middle East respiratory syndrome coronavirus (MERS-CoV) in Saudi Arabia and South Korea. Emerg Microbes Infect. 6:e51." />
<meta name="citation_reference" content="Chu DK, Poon LL, Gomaa MM, et al. (13 co-authors). 2014. MERS coronaviruses in dromedary camels, Egypt. Emerg Infect Dis. 20:1049–1053." />
<meta name="citation_reference" content="Corman VM, Jores J, Meyer B, et al. (13 co-authors). 2014. Antibodies against MERS coronavirus in dromedary camels, Kenya, 1992-2013. Emerg Infect Dis. 20:1319–1322." />
<meta name="citation_reference" content="Cotten M, Watson SJ, Kellam P, et al. (22 co-authors). 2013. Transmission and evolution of the Middle East respiratory syndrome coronavirus in Saudi Arabia: a descriptive genomic study. Lancet. 382:1993–2002." />
<meta name="citation_reference" content="Drummond AJ, Ho SYW, Phillips MJ, Rambaut A. 2006. Relaxed phylogenetics and dating with confidence. PLoS Biol. 4:e88." />
<meta name="citation_reference" content="Drummond AJ, Suchard MA, Xie D, Rambaut A. 2012. Bayesian phylogenetics with BEAUti and the BEAST 1.7. Mol Biol Evol. 29:1969–1973." />
<meta name="citation_reference" content="Dudas G, Bedford T, Lycett S, Rambaut A. 2015. Reassortment between influenza B lineages and the emergence of a coadapted PB1-PB2-HA gene complex. Mol Biol Evol. 32:162–172." />
<meta name="citation_reference" content="Dudas G, Carvalho LM, Bedford T, et al. (96 co-authors). 2017. Virus genomes reveal factors that spread and sustained the Ebola epidemic. Nature. 544:309–315." />
<meta name="citation_reference" content="Dudas G, Rambaut A. 2016. MERS-CoV recombination: implications about the reservoir and potential for adaptation. Virus Evol. 2:vev023." />
<meta name="citation_reference" content="Fagbo SF, Skakni L, Chu DKW, Garbati MA, Joseph M, Hakawi AM. 2015. Molecular epidemiology of hospital outbreak of Middle East respiratory syndrome, Riyadh, Saudi Arabia, 2014. Emerg Infect Dis. 21:1981." />
<meta name="citation_reference" content="Faria NR, Suchard MA, Rambaut A, Streicker DG, Lemey P. 2013. Simultaneously reconstructing viral cross-species transmission history and identifying the underlying constraints. Phil Trans R Soc B. 368:20120196." />
<meta name="citation_reference" content="Frost SD, Volz EM. 2010. Viral phylodynamics and the search for an ‘effective number of infections’. Philos Trans Royal Soc B Trans R Soc B. 365:1879–1890." />
<meta name="citation_reference" content="Gill MS, Lemey P, Faria NR, Rambaut A, Shapiro B, Suchard MA. 2013. Improving bayesian population dynamics inference: A coalescent-based model for multiple loci. Mol Biol Evol. 30:713." />
<meta name="citation_reference" content="Gire SK, Goba A, Andersen KG, et al. (58 co-authors). 2014. Genomic surveillance elucidates Ebola virus origin and transmission during the 2014 outbreak. Science. 345:1369–1372." />
<meta name="citation_reference" content="Grubaugh ND, Ladner JT, Kraemer MU, et al. (67 co-authors). 2017. Genomic epidemiology reveals multiple introductions of Zika virus into the United States. Nature. 546:401–405." />
<meta name="citation_reference" content="Guindon S, Gascuel O, Rannala B. 2003. A Simple, Fast, and Accurate Algorithm to Estimate Large Phylogenies by Maximum Likelihood. Systematic Biology. 52:696–704." />
<meta name="citation_reference" content="Hasegawa M, Kishino H, Yano Ta. 1985. Dating of the human-ape splitting by a molecular clock of mitochondrial DNA. J Mol Evol. 22:160–174." />
<meta name="citation_reference" content="Heled J, Bouckaert RR. 2013. Looking for trees in the forest: summary tree from posterior samples. BMC Evolutionary Biology. 13:221." />
<meta name="citation_reference" content="Herrewegh AAPM, Smeenk I, Horzinek MC, Rottier PJM, Groot RJd. 1998. Feline coronavirus type II strains 79-1683 and 79-1146 originate from a double recombination between feline coronavirus type I and canine coronavirus. J Virol. 72:4508–4514." />
<meta name="citation_reference" content="Hon CC, Lam TY, Shi ZL, Drummond AJ, Yip CW, Zeng F, Lam PY, Leung FCC. 2008. Evidence of the recombinant origin of a bat severe acute respiratory syndrome (SARS)-like coronavirus and its implications on the direct ancestor of SARS coronavirus. J Virol. 82:1819–1826." />
<meta name="citation_reference" content="Katoh K, Standley DM. 2013. MAFFT Multiple Sequence Alignment Software Version 7: Improvements in Performance and Usability. Molecular Biology and Evolution. 30:772–780." />
<meta name="citation_reference" content="Keck JG, Matsushima GK, Makino S, Fleming JO, Vannier DM, Stohlman SA, Lai MM. 1988. In vivo RNA-RNA recombination of coronavirus in mouse brain. J Virol. 62:1810–1813." />
<meta name="citation_reference" content="Kottier SA, Cavanagh D, Britton P. 1995. Experimental evidence of recombination in coronavirus infectious bronchitis virus. Virology. 213:569–580." />
<meta name="citation_reference" content="Kühnert D, Stadler T, Vaughan TG, Drummond AJ. 2016. Phylodynamics with Migration: A Computational Framework to Quantify Population Structure from Genomic Data. Molecular Biology and Evolution. 33:2102–2116." />
<meta name="citation_reference" content="Lai MM, Baric RS, Makino S, Keck JG, Egbert J, Leibowitz JL, Stohlman SA. 1985. Recombination between nonsegmented RNA genomes of murine coronaviruses. J Virol. 56:449–456." />
<meta name="citation_reference" content="Lemey P, Rambaut A, Drummond AJ, Suchard MA. 2009. Bayesian Phylogeography Finds Its Roots. PLOS Computational Biology. 5:e1000520." />
<meta name="citation_reference" content="Lipsitch M, Barclay W, Raman R, et al. (11 co-authors). 2016. Viral factors in influenza pandemic risk assessment. eLife. 5:e18491." />
<meta name="citation_reference" content="Liu D, Shi W, Shi Y, et al. (11 co-authors). 2013. Origin and diversity of novel avian influenza A H7N9 viruses causing human infection: phylogenetic, structural, and coalescent analyses. Lancet. 381:1926–1932." />
<meta name="citation_reference" content="Lloyd-Smith JO, Schreiber SJ, Kopp PE, Getz WM. 2005. Superspreading and the effect of individual variation on disease emergence. Nature. 438:355–359." />
<meta name="citation_reference" content="Lycett S, Bodewes R, Pohlmann A, et al. (11 co-authors). 2016. Role for migratory wild birds in the global spread of avian influenza H5N8. Science. 354:213–217." />
<meta name="citation_reference" content="Maio ND, Wu CH, O’Reilly KM, Wilson D. 2015. New Routes to Phylogeography: A Bayesian Structured Coalescent Approximation. PLOS Genetics. 11:e1005421." />
<meta name="citation_reference" content="Makino S, Keck JG, Stohlman SA, Lai MM. 1986. High-frequency RNA recombination of murine coronaviruses. J Virol. 57:729–737." />
<meta name="citation_reference" content="Martinez VP, Bellomo C, San Juan J, Pinna D, Forlenza R, Elder M, Padula PJ. 2005. Person-to-person transmission of Andes virus. Emerg Infect Dis. 11:1848–1853." />
<meta name="citation_reference" content="Martinez-Bakker M, Bakker KM, King AA, Rohani P. 2014. Human birth seasonality: latitudinal gradient and interplay with childhood disease dynamics. In: Proc R Soc B. volume 281, p. 20132438." />
<meta name="citation_reference" content="Memish ZA, Cotten M, Meyer B, et al. (17 co-authors). 2014. Human infection with MERS coronavirus after exposure to infected camels, Saudi Arabia, 2013. Emerg Infect Dis. 20:1012." />
<meta name="citation_reference" content="Mueller NF, Rasmussen DA, Stadler T. 2017. MASCOT: Parameter and state inference under the marginal structured coalescent approximation. bioRxiv. p. 188516." />
<meta name="citation_reference" content="Müller MA, Corman VM, Jores J, et al. (12 co-authors). 2014. MERS coronavirus neutralizing antibodies in camels, Eastern Africa, 1983-1997. Emerg Infect Dis. 20." />
<meta name="citation_reference" content="Müller MA, Meyer B, Corman VM, et al. (19 co-authors). 2015. Presence of Middle East respiratory syndrome coronavirus antibodies in Saudi Arabia: a nationwide, crosssectional, serological study. Lancet Infect Dis. 15:559–564." />
<meta name="citation_reference" content="Notohara M. 1990. The coalescent and the genealogical process in geographically structured population. J Math Biol. 29:59–75." />
<meta name="citation_reference" content="Park M, Loverdo C, Schreiber SJ, Lloyd-Smith JO. 2013. Multiple scales of selection influence the evolutionary emergence of novel pathogens. Philos Trans Royal Soc B. 368:20120333." />
<meta name="citation_reference" content="Park SS, Wernery U, Corman VM, et al. (19 co-authors). 2015. Acute Middle East respiratory syndrome coronavirus infection in livestock dromedaries, Dubai, 2014. Emerg Infect Dis. 21:1019." />
<meta name="citation_reference" content="Pond, KLS, Posada D, Gravenor MB, Woelk CH, Frost SDW. 2006. GARD: a genetic algorithm for recombination detection. Bioinformatics. 22:3096–3098." />
<meta name="citation_reference" content="Quick J, Ashton P, Calus S, et al. (18 co-authors). 2015. Rapid draft sequencing and real-time nanopore sequencing in a hospital outbreak of Salmonella. Genome Biology. 16:114." />
<meta name="citation_reference" content="Rasmussen DA, Volz EM, Koelle K. 2014. Phylodynamic Inference for Structured Epidemiological Models. PLOS Computational Biology. 10:e1003570." />
<meta name="citation_reference" content="Reed KD, Melski JW, Graham MB, et al. (19 co-authors). 2004. The detection of monkeypox in humans in the Western Hemisphere. N Engl J Med. 350:342–350." />
<meta name="citation_reference" content="Reusken CB, Haagmans BL, Müller MA, et al. (24 co-authors). 2013. Middle East respiratory syndrome coronavirus neutralising serum antibodies in dromedary camels: a comparative serological study. Lancet Infect Dis. 13:859–866." />
<meta name="citation_reference" content="Reusken CB, Messadi L, Feyisa A, et al. (17 co-authors). 2014. Geographic distribution of MERS coronavirus among dromedary camels, Africa. Emerg Infect Dis. 20:1370–1374." />
<meta name="citation_reference" content="Reusken CBEM, Farag EABA, Haagmans BL, et al. (21 co-authors). 2015. Occupational exposure to dromedaries and risk for MERS-CoV infection, Qatar, 2013–-2014. Emerg Infect Dis. 21:1422." />
<meta name="citation_reference" content="Smith GJD, Bahl J, Vijaykrishna D, Zhang J, Poon LLM, Chen H, Webster RG, Peiris JSM, Guan Y. 2009. Dating the emergence of pandemic influenza viruses. Proc Natl Acad Sci USA. 106:11709–11712." />
<meta name="citation_reference" content="The WHO MERS-CoV Research Group. 2013. State of knowledge and data gaps of Middle East respiratory syndrome coronavirus (MERS-CoV) in humans. PLoS Curr. Edition 1." />
<meta name="citation_reference" content="Turner TL, Hahn MW, Nuzhdin SV. 2005. Genomic islands of speciation in Anopheles gambiae. PLoS Biol. 3:e285." />
<meta name="citation_reference" content="Vaughan TG, Kühnert D, Popinga A, Welch D, Drummond AJ. 2014. Efficient Bayesian inference under the structured coalescent. Bioinformatics. 30:2272–2279." />
<meta name="citation_reference" content="Volz EM. 2011. Complex Population Dynamics and the Coalescent under Neutrality. Genetics. p. genetics.111.134627." />
<meta name="citation_reference" content="Volz EM, Koelle K, Bedford T. 2013. Viral phylodynamics. PLoS Comput Biol. 9:e1002947." />
<meta name="citation_reference" content="Wernery U. 2001. Camelid immunoglobulins and their importance for the new-born – a review. J Vet Med B. 48:561–568." />
<meta name="citation_reference" content="World Health Organization. 2016. Disease outbreak news – 2016 december 19. Available at http://www.who.int/csr/don/19-december-2016-2-mers-saudi-arabia/en/." />
<meta name="citation_reference" content="World Health Organization. 2017. WHO MERS-CoV global summary and assessment of risk. Available at http://www.who.int/emergencies/mers-cov/risk-assessment-july-2017.pdf." />
<meta name="citation_reference" content="Yang Z. 1994. Maximum likelihood phylogenetic estimation from DNA sequences with variable rates over sites: Approximate methods. J Mol Evol. 39:306–314." />
<meta name="citation_reference" content="Zaki AM, van Boheemen S, Bestebroer TM, Osterhaus AD, Fouchier RA. 2012. Isolation of a novel coronavirus from a man with pneumonia in Saudi Arabia. N Engl J Med. 367:1814–1820." />
<meta name="citation_reference" content="Zhang Z, Shen L, Gu X. 2016. Evolutionary dynamics of MERS-CoV: potential recombination, positive selection and transmission. Sci Rep. 6:25049." />
<meta name="twitter:title" content="MERS-CoV spillover at the camel-human interface" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Middle East respiratory syndrome coronavirus (MERS-CoV) is a zoonotic virus from camels causing significant mortality and morbidity in humans in the Arabian Peninsula. The epidemiology of the virus remains poorly understood, and while case-based and seroepidemiological studies have been employed extensively throughout the epidemic, viral sequence data have not been utilised to their full potential. Here we use existing MERS-CoV sequence data to explore its phylodynamics in two of its known major hosts, humans and camels. We employ structured coalescent models to show that long-term MERS-CoV evolution occurs exclusively in camels, whereas humans act as a transient, and ultimately terminal host. By analysing the distribution of human outbreak cluster sizes and zoonotic introduction times we show that human outbreaks in the Arabian peninsula are driven by seasonally varying zoonotic transfer of viruses from camels. Without heretofore unseen evolution of host tropism, MERS-CoV is unlikely to become endemic in humans." />
<meta name="og-title" property="og:title" content="MERS-CoV spillover at the camel-human interface" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/173211v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Middle East respiratory syndrome coronavirus (MERS-CoV) is a zoonotic virus from camels causing significant mortality and morbidity in humans in the Arabian Peninsula. The epidemiology of the virus remains poorly understood, and while case-based and seroepidemiological studies have been employed extensively throughout the epidemic, viral sequence data have not been utilised to their full potential. Here we use existing MERS-CoV sequence data to explore its phylodynamics in two of its known major hosts, humans and camels. We employ structured coalescent models to show that long-term MERS-CoV evolution occurs exclusively in camels, whereas humans act as a transient, and ultimately terminal host. By analysing the distribution of human outbreak cluster sizes and zoonotic introduction times we show that human outbreaks in the Arabian peninsula are driven by seasonally varying zoonotic transfer of viruses from camels. Without heretofore unseen evolution of host tropism, MERS-CoV is unlikely to become endemic in humans." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2017-12-20" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/173211v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/173211v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/173211v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/173211v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/72610" />
    <title>MERS-CoV spillover at the camel-human interface | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"oytrlydIGYmlSrQrkMvDTS612CQvAYRMIPOTuCnHN4c","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"72610","apath":"\/biorxiv\/early\/2017\/12\/20\/173211.atom","pisa":"biorxiv;173211v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;173211v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/173211v2","title":"MERS-CoV spillover at the camel-human interface","identifier":"node\/72610"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022UECit\\\\\\\/t5kmGw2FA5fr+Xze52pSOOqjg4coSNdu3YYr9Qh5gmyBIjWQjKOgo2irp+g3gDzzwqZ3S9ex+hWScoKZ6uuBvmcTkIUYKcX4mEBvASrEOUB\\\\\\\/NMV8Bh2pGJM67q8KyhTbiPZO\\\\\\\/4Z8g+BY+FDXmKE\\\\\\\/ci9Y2BDxMr3W99BtnaCNOuFLv8K0h3mePiycRqA5YIOgHEeI4fvM\\\\\\\/YMpuWwX+zdt4NdmbXiOHWRYf5PIlUoDLl0n2JI0eN7Dc2QIEXwtfG05SXsCQ1tQZ8UjxTZTK3SfGx062OinbVGS2h+h114wqF83x+dvuML36pBAL\\\\\\\/eACwPfOR0sYM0UGDIoLnfLROtCLQ0FFVYLgBnziqaP\\\\\\\/xe276NVmGAbjZvfbuCLgrpD58UaI3RpzErE50mTmSQV9fk\\\\\\\/XwkryRKmq3vbdkFkdM+QR9MjmKEpkJjkTkG58AMpNKQYvJ6ZuA2xzonlsTfQAFsw26lqVSjqsDvjuKP\\\\\\\/v4i3QfwJ3EjXxd2Hapv7bfIZHIzD9YZsckrafIGJayrjwAe2zn5gBHxoFXiWfUZUUwU55Flg7a9E1PeN5ubFw55myFMsFJmwKFRMwwgwAHip+lQkTwLubl03B8Gv0VGlOKZ21bQyeHWwYkndr+yz6WtFVbbfqd1PLcmzzjBHS+hnO\\\\\\\/YDnVXwqHAY5NpOYnVDDjr4vQD0QLPe3\\\\\\\/qSVl\\\\\\\/XfmfTcu8rF1dvTTVoe2jem+csUtx0zZA5ojBSmLg5fJqF7t1FGcmHHWdctUvOuzdsWhW5MiI2yFsBScV3zmeISAkTZIa1imAU6SlhTRcr8cnm2PiPr4Ndk9Ym34QTbxTU09mz1qPtoRvjk83iNMTujevcoUhCNS4q\\\\\\\/vbNm7n\\\\\\\/8P+XBoei5C7BKWSXZH4Ke4Umpa8dkDD2ve4wXZN+JGhFODUyeqTwq9gzfGclsAyqbQIJmbUI1sVR3RfZMgcTQ33G3pAXPqj46yNI2upKnP7BuLrrusAZ2NkxD+aPw1DAqAXN\\\\\\\/rHVrE6uG6i7OV9p1Loaty5VTUDhLcFPwekdtvOYfUpTUAKWI2i6Vc4Cv7x6360tEA6ghkszdFAPC2DbnGsgnHEQ39SPEfqNfDGg7MKhEKlpThZyBTjzg27a0bvtyJOo\\\\\\\/q4ib+5zAa3GY3poi7elJSgbBPQbGZUyLUymQ4ce4VF9W4Y0Z0j6Ev6VqOdp2IWXyTQiseWDvSL6+mHU9Ot8Xy1wlPKOFmUgs1Lk9MAoeIGALpa82cDF0SlvoGJQCrcETyxRw6NsnaIOMA6B3ZFoF6LRbwE1pqaBLP+EsWfHmm8PJ1Q1vbkIWF1CE5RMY2C5c=\\\u0022,\\\u0022iv\\\u0022:\\\u0022a42pDo6UXjj+RtquQI5wPw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00227d83857870bee74e61bc360215438f5c473c9d93d5005b2a4b014333d20571b4\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/173211v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-72610 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/173211v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vaEhvn6r7JmHyHkJgo_HNEnkYBnSsAKTlYQj_3XasNo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="72610" id="node72610" data-pisa="biorxiv;173211v2" data-pisa-master="biorxiv;173211" data-apath="/biorxiv/early/2017/12/20/173211.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">MERS-CoV spillover at the camel-human interface</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0227-4158" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Gytis</span> <span class="nlm-surname">Dudas</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5736-5578" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Luiz Max</span> <span class="nlm-surname">Carvalho</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Andrew</span> <span class="nlm-surname">Rambaut</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-4039-5794" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Trevor</span> <span class="nlm-surname">Bedford</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Ali M.</span> <span class="nlm-surname">Somily</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Mazin</span> <span class="nlm-surname">Barry</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Sarah S.</span> <span class="nlm-surname">Al Subaie</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Abdulaziz A.</span> <span class="nlm-surname">BinSaeed</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Fahad A.</span> <span class="nlm-surname">Alzamil</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Waleed</span> <span class="nlm-surname">Zaher</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Theeb Al</span> <span class="nlm-surname">Qahtani</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Khaldoon Al</span> <span class="nlm-surname">Jerian</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Scott J.N.</span> <span class="nlm-surname">McNabb</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Imad A.</span> <span class="nlm-surname">Al-Jahdali</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Ahmed M.</span> <span class="nlm-surname">Alotaibi</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Nahid A.</span> <span class="nlm-surname">Batarfi</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">Matthew</span> <span class="nlm-surname">Cotten</span></span>, <span class="highwire-citation-author" data-delta="17"><span class="nlm-given-names">Simon J.</span> <span class="nlm-surname">Watson</span></span>, <span class="highwire-citation-author" data-delta="18"><span class="nlm-given-names">Spela</span> <span class="nlm-surname">Binter</span></span>, <span class="highwire-citation-author" data-delta="19"><span class="nlm-given-names">Paul</span> <span class="nlm-surname">Kellam</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/173211 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node72610" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gytis Dudas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>College of Medicine, King Saud University</span>, Riyadh, <span class='nlm-country'>Kingdom of Saudi Arabia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gytis%2BDudas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dudas%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGytis%2BDudas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0227-4158" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gytis Dudas</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Luiz Max Carvalho </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Institute of Evolutionary Biology, University of Edinburgh</span>, Edinburgh, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luiz%2BMax%2BCarvalho%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carvalho%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALuiz%2BMax%2BCarvalho%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5736-5578" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Luiz Max Carvalho</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Andrew Rambaut </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Institute of Evolutionary Biology, University of Edinburgh</span>, Edinburgh, <span class='nlm-country'>UK</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Fogarty International Center, National Institutes of Health</span>, Bethesda, MD, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BRambaut%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rambaut%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BRambaut%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Trevor Bedford </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Vaccine and Infectious Disease Division, Fred Hutchinson Cancer Research Center</span>, Seattle, WA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Trevor%2BBedford%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bedford%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATrevor%2BBedford%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4039-5794" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Trevor Bedford</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ali M. Somily </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>College of Medicine, King Saud University</span>, Riyadh, <span class='nlm-country'>Kingdom of Saudi Arabia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ali%2BM.%2BSomily%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Somily%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAli%2BM.%2BSomily%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Mazin Barry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>College of Medicine, King Saud University</span>, Riyadh, <span class='nlm-country'>Kingdom of Saudi Arabia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mazin%2BBarry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barry%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMazin%2BBarry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Sarah S. Al Subaie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>College of Medicine, King Saud University</span>, Riyadh, <span class='nlm-country'>Kingdom of Saudi Arabia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sarah%2BS.%2BAl%2BSubaie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Al%20Subaie%20SS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASarah%2BS.%2BAl%2BSubaie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Abdulaziz A. BinSaeed </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>College of Medicine, King Saud University</span>, Riyadh, <span class='nlm-country'>Kingdom of Saudi Arabia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abdulaziz%2BA.%2BBinSaeed%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=BinSaeed%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbdulaziz%2BA.%2BBinSaeed%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Fahad A. Alzamil </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>College of Medicine, King Saud University</span>, Riyadh, <span class='nlm-country'>Kingdom of Saudi Arabia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fahad%2BA.%2BAlzamil%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Alzamil%20FA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFahad%2BA.%2BAlzamil%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Waleed Zaher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>College of Medicine, King Saud University</span>, Riyadh, <span class='nlm-country'>Kingdom of Saudi Arabia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Waleed%2BZaher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zaher%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWaleed%2BZaher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Theeb Al Qahtani </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>College of Medicine, King Saud University</span>, Riyadh, <span class='nlm-country'>Kingdom of Saudi Arabia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Theeb%2BAl%2BQahtani%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Qahtani%20TA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATheeb%2BAl%2BQahtani%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Khaldoon Al Jerian </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>College of Medicine, King Saud University</span>, Riyadh, <span class='nlm-country'>Kingdom of Saudi Arabia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Khaldoon%2BAl%2BJerian%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jerian%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKhaldoon%2BAl%2BJerian%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Scott J.N. McNabb </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Rollins School of Public Health, Emory University</span>, Atlanta, GA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Scott%2BJ.N.%2BMcNabb%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McNabb%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AScott%2BJ.N.%2BMcNabb%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Imad A. Al-Jahdali </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Deputy Minister. Ex. General Director King Fahad General Hospital, Jeddah and Occupational and environmental medicine, Um AlQura University</span>, <span class='nlm-country'>Kingdom of Saudi Arabia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Imad%2BA.%2BAl-Jahdali%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Al-Jahdali%20IA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AImad%2BA.%2BAl-Jahdali%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Ahmed M. Alotaibi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Department of Intensive Care, Prince Mohammed bin Abdulaziz Hospital</span>, Riyadh, <span class='nlm-country'>Kingdom of Saudi Arabia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ahmed%2BM.%2BAlotaibi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Alotaibi%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAhmed%2BM.%2BAlotaibi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Nahid A. Batarfi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Epidemiology section, Command and Control Center (CCC) Ministry of Health</span>, Jeddah</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nahid%2BA.%2BBatarfi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Batarfi%20NA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANahid%2BA.%2BBatarfi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-16"><div class="author-tooltip-name">Matthew Cotten </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Wellcome Trust Sanger Institute</span>, Hinxton, <span class='nlm-country'>United Kingdom</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BCotten%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cotten%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BCotten%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-17"><div class="author-tooltip-name">Simon J. Watson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Wellcome Trust Sanger Institute</span>, Hinxton, <span class='nlm-country'>United Kingdom</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simon%2BJ.%2BWatson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Watson%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASimon%2BJ.%2BWatson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-18"><div class="author-tooltip-name">Spela Binter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Wellcome Trust Sanger Institute</span>, Hinxton, <span class='nlm-country'>United Kingdom</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Spela%2BBinter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Binter%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASpela%2BBinter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-19"><div class="author-tooltip-name">Paul Kellam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Wellcome Trust Sanger Institute</span>, Hinxton, <span class='nlm-country'>United Kingdom</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BKellam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kellam%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaul%2BKellam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/173211v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:72610" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:72610/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/173211v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:72610" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:72610/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/173211v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:72610" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:72610/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/173211v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:72610" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:72610/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/173211v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:72610" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:72610/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Middle East respiratory syndrome coronavirus (MERS-CoV) is a zoonotic virus from camels causing significant mortality and morbidity in humans in the Arabian Peninsula. The epidemiology of the virus remains poorly understood, and while case-based and seroepidemiological studies have been employed extensively throughout the epidemic, viral sequence data have not been utilised to their full potential. Here we use existing MERS-CoV sequence data to explore its phylodynamics in two of its known major hosts, humans and camels. We employ structured coalescent models to show that long-term MERS-CoV evolution occurs exclusively in camels, whereas humans act as a transient, and ultimately terminal host. By analysing the distribution of human outbreak cluster sizes and zoonotic introduction times we show that human outbreaks in the Arabian peninsula are driven by seasonally varying zoonotic transfer of viruses from camels. Without heretofore unseen evolution of host tropism, MERS-CoV is unlikely to become endemic in humans.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">Middle East respiratory syndrome coronavirus (MERS-CoV), endemic in camels in the Arabian Peninsula, is the causative agent of zoonotic infections and limited outbreaks in humans. The virus, first discovered in 2012 (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Zaki et al., 2012</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Boheemen et al., 2012</a>), has caused more than 2000 infections and over 700 deaths, according to the World Health Organization (WHO) (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">World Health Organization, 2017</a>). Its epidemiology remains obscure, largely because infections are observed among the most severely affected individuals, such as older males with comorbidities (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Assiri et al., 2013a</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">The WHO MERS-CoV Research Group, 2013</a>). While contact with camels is often reported, other patients do not recall contact with any livestock, suggesting an unobserved community contribution to the outbreak (<a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">The WHO MERS-CoV Research Group, 2013</a>). Previous studies on MERS-CoV epidemiology have used serology to identify factors associated with MERS-CoV exposure in potential risk groups (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Reusken et al., 2015</a>, <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">2013</a>). Such data have shown high seroprevalence in camels (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Müller et al., 2014</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Corman et al., 2014</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Chu et al., 2014</a>; <a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Reusken et al., 2013</a>, <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">2014</a>) and evidence of contact with MERS-CoV in workers with occupational exposure to camels (<a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Reusken et al., 2015</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Müller et al., 2015</a>). Separately, epidemiological modelling approaches have been used to look at incidence reports through time, space and across hosts (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Cauchemez et al., 2016</a>).</p><p id="p-4">Although such epidemiological approaches yield important clues about exposure patterns and potential for larger outbreaks, much inevitably remains opaque to such approaches due to difficulties in linking cases into transmission clusters in the absence of detailed information. Where sequence data are relatively cheap to produce, genomic epidemiological approaches can fill this critical gap in outbreak scenarios (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Liu et al., 2013</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Gire et al., 2014</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Grubaugh et al., 2017</a>). These data often yield a highly detailed picture of an epidemic when complete genome sequencing is performed consistently and appropriate metadata collected (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Dudas et al., 2017</a>). Sequence data can help discriminate between multiple and single source scenarios (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Gire et al., 2014</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Quick et al., 2015</a>), which are fundamental to quantifying risk (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Grubaugh et al., 2017</a>). Sequencing MERS-CoV has been performed as part of initial attempts to link human infections with the camel reservoir (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Memish et al., 2014</a>), nosocomial outbreak investigations (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Assiri et al., 2013b</a>) and routine surveillance (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Park et al., 2015</a>). A large portion of MERS-CoV sequences come from outbreaks within hospitals, where sequence data have been used to determine whether infections were isolated introductions or were part of a larger hospital-associated outbreak (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Fagbo et al., 2015</a>). Similar studies on MERS-CoV have taken place at broader geographic scales, such as cities (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Cotten et al., 2013</a>).</p><p id="p-5">It is widely accepted that recorded human MERS-CoV infections are a result of at least several introductions of the virus into humans (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Cotten et al., 2013</a>) and that contact with camels is a major risk factor for developing MERS, per WHO guidelines (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">World Health Organization, 2016</a>). Previous studies attempting to quantify the actual number of spillover infections have either relied on case-based epidemiological approaches (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Cauchemez et al., 2016</a>) or employed methods agnostic to signals of population structure within sequence data (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Zhang et al., 2016</a>). Here we use a dataset of 274 MERS-CoV genomes to investigate transmission patterns of the virus between humans and camels.</p><p id="p-6">Here, we use an explicit model of metapopulation structure and migration between discrete subpopulations, referred to here as demes (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Vaughan et al., 2014</a>), derived from the structured coalescent (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Notohara, 1990</a>). Unlike approaches that model host species as a discrete phylogenetic trait of the virus using continuous-time Markov processes (or simpler, parsimony based, approaches) (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Faria et al., 2013</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Lycett et al., 2016</a>), population structure models explicitly incorporate contrasts in deme effective population sizes and migration between demes. By estimating independent coalescence rates for MERS-CoV in humans and camels, as well as migration patterns between the two demes, we show that long-term viral evolution of MERS-CoV occurs exclusively in camels. Our results suggest that spillover events into humans are seasonal and might be associated with the calving season in camels. However, we find that MERS-CoV, once introduced into humans, follows transient transmission chains that soon abate. Using Monte Carlo simulations we show that <em>R</em><sub>0</sub> for MERS-CoV circulating in humans is much lower than the epidemic threshold of 1.0 and that correspondingly the virus has been introduced into humans hundreds of times.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3>MERS-CoV is predominantly a camel virus</h3><p id="p-7">The structured coalescent approach we employ (see Methods) identifies camels as a reservoir host where most of MERS-CoV evolution takes place (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>), while human MERS outbreaks are transient and terminal with respect to long-term evolution of the virus (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure S1</a>). Across 174 MERS-CoV genomes collected from humans, we estimate a median of 56 separate camel-to-human cross-species transmissions (95% highest posterior density (HPD): 48–63). While we estimate a median of 3 (95% HPD: 0–12) human- to-camel migrations, the 95% HPD interval includes zero and we find that no such migrations are found in the maximum clade credibility tree (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Correspondingly, we observe substantially higher camel-to-human migration rate estimates than human-to- camel migration rate estimates (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure S2</a>). This inference derives from the tree structure wherein human viruses appear as clusters of highly related sequences nested within the diversity seen in camel viruses, which themselves show significantly higher diversity and less clustering. This manifests as different rates of coalescence with camel viruses showing a scaled effective population size <em>N<sub>e</sub>τ</em> of 3.49 years (95% HPD: 2.71–4.38) and human viruses showing a scaled effective population of 0.24 years (95% HPD: 0.14–0.34).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Evolutionary history of MERS-CoV partitioned between camels and humans This is the same tree as shown in Figure 1, but with contiguous stretches of MERS-CoV evolutionary history split by inferred host: camels (top in orange) and humans (bottom in blue). This visualisation highlights the ephemeral nature of MERS-CoV outbreaks in humans, compared to continuous circulation of the virus in camels." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-108077617" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Evolutionary history of MERS-CoV partitioned between camels and humans&lt;/span&gt; This is the same tree as shown in Figure 1, but with contiguous stretches of MERS-CoV evolutionary history split by inferred host: camels (top in orange) and humans (bottom in blue). This visualisation highlights the ephemeral nature of MERS-CoV outbreaks in humans, compared to continuous circulation of the virus in camels.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F7.medium.gif" width="298" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S1." src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F7.medium.gif" width="298" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure S1.</span> <span class="caption-title">Evolutionary history of MERS-CoV partitioned between camels and humans</span><p id="p-54" class="first-child">This is the same tree as shown in <a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1</a>, but with contiguous stretches of MERS-CoV evolutionary history split by inferred host: camels (top in orange) and humans (bottom in blue). This visualisation highlights the ephemeral nature of MERS-CoV outbreaks in humans, compared to continuous circulation of the virus in camels.</p><div class="sb-div caption-clear"></div></div></div><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Posterior backwards migration rate estimates for two choices of prior Negligible flow of MERS-CoV lineages from humans into camels is recovered regardless of prior choice (note that rates are backwards in time). Plots show the 95% highest posterior density for the estimated migration rate from the human deme into the camel deme looking backwards in time (orange) and vice versa (blue). Dotted lines indicate exponential priors specified for migration rates, with mean 1.0 (bottom) or 10.0 (top)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-108077617" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Posterior backwards migration rate estimates for two choices of prior&lt;/span&gt; Negligible flow of MERS-CoV lineages from humans into camels is recovered regardless of prior choice (note that rates are backwards in time). Plots show the 95% highest posterior density for the estimated migration rate from the human deme into the camel deme looking backwards in time (orange) and &lt;em&gt;vice versa&lt;/em&gt; (blue). Dotted lines indicate exponential priors specified for migration rates, with mean 1.0 (bottom) or 10.0 (top).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F8.medium.gif" width="417" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S2." src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F8.medium.gif" width="417" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S2.</span> <span class="caption-title">Posterior backwards migration rate estimates for two choices of prior</span><p id="p-55" class="first-child">Negligible flow of MERS-CoV lineages from humans into camels is recovered regardless of prior choice (note that rates are backwards in time). Plots show the 95% highest posterior density for the estimated migration rate from the human deme into the camel deme looking backwards in time (orange) and <em>vice versa</em> (blue). Dotted lines indicate exponential priors specified for migration rates, with mean 1.0 (bottom) or 10.0 (top).</p><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Typed maximum clade credibility tree of MERS-CoV genomes from 174 human viruses and 100 camel viruses Maximum clade credibility (MCC) tree showing inferred ancestral hosts for MERS-CoV recovered with the structured coalescent. The vast majority of MERS-CoV evolution is inferred to occur in camels (orange) with human outbreaks (blue) representing evolutionary dead-ends for the virus. Confidence in host assignment is depicted as a colour gradient, with increased uncertainty in host assignment (posterior probabilities close to 0.5) shown as grey. While large clusters of human cases are apparent in the tree, significant contributions to human outbreaks are made by singleton sequences, likely representing recent cross-species transmissions that were caught early." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-108077617" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Typed maximum clade credibility tree of MERS-CoV genomes from 174 human viruses and 100 camel viruses&lt;/span&gt; Maximum clade credibility (MCC) tree showing inferred ancestral hosts for MERS-CoV recovered with the structured coalescent. The vast majority of MERS-CoV evolution is inferred to occur in camels (orange) with human outbreaks (blue) representing evolutionary dead-ends for the virus. Confidence in host assignment is depicted as a colour gradient, with increased uncertainty in host assignment (posterior probabilities close to 0.5) shown as grey. While large clusters of human cases are apparent in the tree, significant contributions to human outbreaks are made by singleton sequences, likely representing recent cross-species transmissions that were caught early.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F1.medium.gif" width="439" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F1.medium.gif" width="439" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 1.</span> <span class="caption-title">Typed maximum clade credibility tree of MERS-CoV genomes from 174 human viruses and 100 camel viruses</span><p id="p-8" class="first-child">Maximum clade credibility (MCC) tree showing inferred ancestral hosts for MERS-CoV recovered with the structured coalescent. The vast majority of MERS-CoV evolution is inferred to occur in camels (orange) with human outbreaks (blue) representing evolutionary dead-ends for the virus. Confidence in host assignment is depicted as a colour gradient, with increased uncertainty in host assignment (posterior probabilities close to 0.5) shown as grey. While large clusters of human cases are apparent in the tree, significant contributions to human outbreaks are made by singleton sequences, likely representing recent cross-species transmissions that were caught early.</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">We believe that the small number of inferred human-to-camel migrations are induced by the migration rate prior, while some are derived from phylogenetic proximity of human sequences to the apparent “backbone” of the phylogenetic tree. This is most apparent in lineages in early-mid 2013 that lead up to sequences comprising the MERS-CoV clade dominant in 2015, where owing to poor sampling of MERS-CoV genetic diversity from camels the model cannot completely dismiss humans as a potential alternative host. The first sequences of MERS-CoV from camels do not appear in our data until November 2013. Our finding of negligible human-to-camel transmission is robust to choice of prior (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure S2</a>).</p><p id="p-10">The repeated and asymmetric introductions of short-lived clusters of MERS-CoV sequences from camels into humans leads us to conclude that MERS-CoV epidemiology in humans is dominated by zoonotic transmission (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-fig-7-2" class="xref-fig" href="#F7">S1</a>). We observe dense terminal clusters of MERS-CoV circulating in humans that are of no subsequent relevance to the evolution of the virus. These clusters of presumed human-to-human transmission are then embedded within extensive diversity of MERS-CoV lineages inferred to be circulating in camels, a classic pattern of source-sink dynamics. Our findings suggest that instances of human infection with MERS-CoV are more common than currently thought, with exceedingly short transmission chains mostly limited to primary cases that might be mild and ultimately not detected by surveillance or sequencing. Structured coalescent analyses recover the camel-centered picture of MERS-CoV evolution despite sequence data heavily skewed towards non-uniformly sampled human cases and are robust to choice of prior. Comparing these results with a currently standard discrete trait analysis (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Lemey et al., 2009</a>) approach for ancestral state reconstruction shows dramatic differences in host reconstruction at internal nodes (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure S3</a>). Discrete trait analysis reconstruction identifies both camels and humans as important hosts for MERS-CoV persistence, but with humans as the ultimate source of camel infections. A similar approach has been attempted previously (<a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Zhang et al., 2016</a>), but this interpretation of MERS-CoV evolution disagrees with lack of continuing human transmission chains outside of Arabian peninsula, low seroprevalence in humans and very high seroprevalence in camels across Saudi Arabia. We suspect that this particular discrete trait analysis reconstruction is false due to biased data, <em>i.e.</em> having nearly twice as many MERS-CoV sequences from humans (<em>n</em> = 174) than from camels (<em>n</em> = 100) and the inability of the model to account for and quantify vastly different rates of coalescence in the phylogenetic vicinity of both types of sequences. We can replicate these results by either applying the same model to current data (<a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure S3</a>) or by enforcing equal coalescence rates within each deme in the structured coalescent model (<a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure S4</a>).</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Maximum clade credibility (MCC) tree with ancestral state reconstruction according to a discrete trait model MCC tree is presented the same as Figure 1 and Figure S4, with colours indicating the most probable state reconstruction at internal nodes. Unlike the structured coalescent summary shown in Figure 1 where camels are reconstructed as the main host where MERS-CoV persists, the discrete trait approach identifies both camels and humans as major hosts with humans being the source of MERS-CoV infection in camels." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-108077617" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Maximum clade credibility (MCC) tree with ancestral state reconstruction according to a discrete trait model&lt;/span&gt; MCC tree is presented the same as Figure 1 and Figure S4, with colours indicating the most probable state reconstruction at internal nodes. Unlike the structured coalescent summary shown in Figure 1 where camels are reconstructed as the main host where MERS-CoV persists, the discrete trait approach identifies both camels and humans as major hosts with humans being the source of MERS-CoV infection in camels.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F9.medium.gif" width="438" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S3." src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F9.medium.gif" width="438" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S3.</span> <span class="caption-title">Maximum clade credibility (MCC) tree with ancestral state reconstruction according to a discrete trait model</span><p id="p-56" class="first-child">MCC tree is presented the same as <a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-fig-10-3" class="xref-fig" href="#F10">Figure S4</a>, with colours indicating the most probable state reconstruction at internal nodes. Unlike the structured coalescent summary shown in <a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1</a> where camels are reconstructed as the main host where MERS-CoV persists, the discrete trait approach identifies both camels and humans as major hosts with humans being the source of MERS-CoV infection in camels.</p><div class="sb-div caption-clear"></div></div></div><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Maximum clade credibility (MCC) tree of structured coalescent model with enforced equal coalescence rates MCC tree is presented the same as Figures 1 and S3, with colours indicating the most probable state reconstruction at internal nodes. Similar to Figure S3 enforcing equal coalescence rates between demes in a structured coalescent model identifies humans as a major MERS-CoV host and the source of viruses in camels." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-108077617" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Maximum clade credibility (MCC) tree of structured coalescent model with enforced equal coalescence rates&lt;/span&gt; MCC tree is presented the same as Figures 1 and S3, with colours indicating the most probable state reconstruction at internal nodes. Similar to Figure S3 enforcing equal coalescence rates between demes in a structured coalescent model identifies humans as a major MERS-CoV host and the source of viruses in camels.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F10.medium.gif" width="438" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S4." src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F10.medium.gif" width="438" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S4.</span> <span class="caption-title">Maximum clade credibility (MCC) tree of structured coalescent model with enforced equal coalescence rates</span><p id="p-57" class="first-child">MCC tree is presented the same as <a id="xref-fig-1-11" class="xref-fig" href="#F1">Figures 1</a> and <a id="xref-fig-9-4" class="xref-fig" href="#F9">S3</a>, with colours indicating the most probable state reconstruction at internal nodes. Similar to <a id="xref-fig-9-5" class="xref-fig" href="#F9">Figure S3</a> enforcing equal coalescence rates between demes in a structured coalescent model identifies humans as a major MERS-CoV host and the source of viruses in camels.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>MERS-CoV shows seasonal introductions</h3><p id="p-11">We use the posterior distribution of MERS-CoV introduction events from camels to humans (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>) to model seasonal variation in zoonotic transfer of viruses. We identify four months (April, May, June, July) when the odds of MERS-CoV introductions are increased (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>) and four when the odds are decreased (August, September, November, December). Camel calving is reported to occur from October to February (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Almutairi et al., 2010</a>), with rapidly declining maternal antibody levels in calves within the first weeks after birth (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Wernery, 2001</a>). It is possible that MERS-CoV sweeps through each new camel generation once critical mass of susceptibles is reached (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Martinez-Bakker et al., 2014</a>), leading to a sharp rise in prevalence of the virus in camels and resulting in increased force of infection into the human population. Strong influx of susceptibles and subsequent sweeping outbreaks in camels may explain evidence of widespread exposure to MERS-CoV in camels from seroepidemiology (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Müller et al., 2014</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Corman et al., 2014</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Chu et al., 2014</a>; <a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Reusken et al., 2013</a>, <a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">2014</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Seasonality of MERS-CoV introduction events A) Posterior density estimates partitioned by month showing the 95% highest posterior density interval for relative odds ratios of MERS-CoV introductions into humans. Posterior means are indicated with circles. Evidence for increased or decreased risk (95% HPD excludes 1.0) for introductions are indicated by black or white circles, respectively. Hatched area spanning October to February indicates the camel calving season. B) Sequence cluster sizes and inferred dates of introduction events. Each introduction event is shown as a vertical line positioned based on the median introduction time, as recovered by structured coalescent analyses and coloured by time of year with height indicating number of descendant sequences recovered from human cases. 95% highest posterior density intervals for introductions of MERS-CoV into humans are indicated with coloured lines, coloured by median estimated introduction time. The black dotted line indicates the joint probability density for introductions. We find little correlation between date and size of introduction (Spearman p = 0.06, p = 0.68)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-108077617" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Seasonality of MERS-CoV introduction events&lt;/span&gt; A) Posterior density estimates partitioned by month showing the 95% highest posterior density interval for relative odds ratios of MERS-CoV introductions into humans. Posterior means are indicated with circles. Evidence for increased or decreased risk (95% HPD excludes 1.0) for introductions are indicated by black or white circles, respectively. Hatched area spanning October to February indicates the camel calving season. B) Sequence cluster sizes and inferred dates of introduction events. Each introduction event is shown as a vertical line positioned based on the median introduction time, as recovered by structured coalescent analyses and coloured by time of year with height indicating number of descendant sequences recovered from human cases. 95% highest posterior density intervals for introductions of MERS-CoV into humans are indicated with coloured lines, coloured by median estimated introduction time. The black dotted line indicates the joint probability density for introductions. We find little correlation between date and size of introduction (Spearman &lt;em&gt;p&lt;/em&gt; = 0.06, &lt;em&gt;p&lt;/em&gt; = 0.68).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F2.medium.gif" width="440" height="315"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F2.medium.gif" width="440" height="315"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Seasonality of MERS-CoV introduction events</span><p id="p-12" class="first-child">A) Posterior density estimates partitioned by month showing the 95% highest posterior density interval for relative odds ratios of MERS-CoV introductions into humans. Posterior means are indicated with circles. Evidence for increased or decreased risk (95% HPD excludes 1.0) for introductions are indicated by black or white circles, respectively. Hatched area spanning October to February indicates the camel calving season. B) Sequence cluster sizes and inferred dates of introduction events. Each introduction event is shown as a vertical line positioned based on the median introduction time, as recovered by structured coalescent analyses and coloured by time of year with height indicating number of descendant sequences recovered from human cases. 95% highest posterior density intervals for introductions of MERS-CoV into humans are indicated with coloured lines, coloured by median estimated introduction time. The black dotted line indicates the joint probability density for introductions. We find little correlation between date and size of introduction (Spearman <em>p</em> = 0.06, <em>p</em> = 0.68).</p><div class="sb-div caption-clear"></div></div></div><p id="p-13">Although we find evidence of seasonality in zoonotic spillover timing, no such relationship exists for sizes of human sequence clusters (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). This is entirely expected, since little seasonality in human behaviour that could facilitate MERS-CoV transmission is expected following an introduction. Similarly, we do not observe any trend in human sequence cluster sizes over time (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>, Spearman <em>ρ</em> = 0.06, <em>p</em> = 0.68), suggesting that MERS-CoV outbreaks in humans are neither growing nor shrinking in size. This is not surprising either, since MERS-CoV is a camel virus that has to date, experienced little-to-no selective pressure to improve transmissibility between humans.</p></div><div id="sec-5" class="subsection"><h3>MERS-CoV is poorly suited for human transmission</h3><p id="p-14">Structured coalescent approaches clearly show humans to be a terminal host for MERS- CoV, implying poor transmissibility. However, we wanted to translate this observation into an estimate of the basic reproductive number <em>R</em><sub>0</sub> to provide an intuitive epidemic behaviour metric that does not require expertise in reading phylogenies. The parameter R<sub>0</sub> determines expected number of secondary cases in a single infections as well as the distribution of total cases that result from a single introduction event into the human population (<a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">Equation 1</a>, Methods). We estimate <em>R</em><sub>0</sub> along with other relevant parameters via Monte Carlo simulation in two steps. First, we simulate case counts across multiple outbreaks totaling 2000 cases using <a id="xref-disp-formula-2-2" class="xref-disp-formula" href="#disp-formula-2">Equation 1</a> and then we subsample from each case cluster to simulate sequencing of a fraction of cases. Sequencing simulations are performed via a multivariate hypergeometric distribution, where the probability of sequencing from a particular cluster depends on available sequencing capacity (number of trials), numbers of cases in the cluster (number of successes) and number of cases outside the cluster (number of failures). In addition, each hypergeometric distribution used to simulate sequencing is concentrated via a bias parameter, that enriches for large sequence clusters at the expense of smaller ones (for its effects see <a id="xref-fig-11-1" class="xref-fig" href="#F11">Figure S5</a>). This is a particularly pressing issue, since <em>a priori</em> we expect large hospital outbreaks of MERS to be overrepresented in sequence data, whereas sequences from primary cases will be sampled exceedingly rarely. We record the number, mean, standard deviation and skewness (third standardised moment) of sequence cluster sizes in each simulation (left-hand panel in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>) and extract the subset of Monte Carlo simulations in which these summary statistics fall within the 95% highest posterior density observed in the empirical MERS-CoV data from structured coalescent analyses. We record <em>R<sub>0</sub></em> values, as well as the number of case clusters (equivalent to number of zoonotic introductions), for these empirically matched simulations. A schematic of this Monte Carlo procedure is shown in <a id="xref-fig-12-1" class="xref-fig" href="#F12">Figure S6</a>. Since the total number of cases is fixed at 2000, higher R<sub>0</sub> results in fewer larger transmission clusters, while lower <em>R<sub>0</sub></em> results in many smaller transmission clusters.</p><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Monte Carlo simulations of human transmission clusters From top to bottom each row corresponds to departures from completely random sequencing efforts with respect to case cluster size (bias parameter=1.0) to sequencing increasingly biased towards capturing large case clusters (bias=2.0, bias=3.0). Leftmost scatter plots show the distribution of individual Monte Carlo simulation sequence cluster size statistics (mean and skewness) coloured by the R0 value used for the simulation. The dotted rectangle identifies the 95% highest posterior density bounds for sequence cluster size mean and skewness observed for empirical MERS-CoV data. The distribution of Ro values matching empirical data are shown in the middle, on the same y-axis across all levels of the bias parameter. Under unbiased sequencing (bias=1.0) only 0.45% of simulations fit our phylogenetic observations, while 1.79% and 1.67% of simulations fit for bias levels of 2.0 and 3.0, respectively. Correspondingly, we estimate 11.6% support for a model with bias level 1.0, 45.7% support for a model with bias level 2.0, and 42.7% support for a model with bias level 3.0. Bins falling inside the 95% percentiles are coloured by R0, as in the leftmost scatter plot. While the 95% percentiles for R0 values are close to 1.0 (0.71-0.98) for the unbiased sequencing simulation (i.e. uniform sequencing efforts, in which every case is equally likely to be sequenced), we also note that increasing levels of bias are considerably more to likely to generate MERS-CoV-like sequence clusters. The distribution of total number of introductions associated with simulations matching MERS-CoV sequence clusters is shown in the plots on the right, on the same y-axis across all levels of bias. Darker shade of grey indicates bins falling within the 95% percentiles. The median number of cross-species introductions observed in simulations matching empirical data without bias are 346 (95% percentiles 262&#x2013;439). These numbers jump up to 568 (95% percentiles 430&#x2013;727) for bias = 2.0 and 656 (95% percentiles 488&#x2013;853) for bias = 3.0 simulations. Model averaging would suggest plausible numbers of introductions between 311 and 811." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-108077617" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Monte Carlo simulations of human transmission clusters&lt;/span&gt; From top to bottom each row corresponds to departures from completely random sequencing efforts with respect to case cluster size (bias parameter=1.0) to sequencing increasingly biased towards capturing large case clusters (bias=2.0, bias=3.0). Leftmost scatter plots show the distribution of individual Monte Carlo simulation sequence cluster size statistics (mean and skewness) coloured by the &lt;em&gt;R&lt;/em&gt;&lt;sub&gt;0&lt;/sub&gt; value used for the simulation. The dotted rectangle identifies the 95% highest posterior density bounds for sequence cluster size mean and skewness observed for empirical MERS-CoV data. The distribution of Ro values matching empirical data are shown in the middle, on the same y-axis across all levels of the bias parameter. Under unbiased sequencing (bias=1.0) only 0.45% of simulations fit our phylogenetic observations, while 1.79% and 1.67% of simulations fit for bias levels of 2.0 and 3.0, respectively. Correspondingly, we estimate 11.6% support for a model with bias level 1.0, 45.7% support for a model with bias level 2.0, and 42.7% support for a model with bias level 3.0. Bins falling inside the 95% percentiles are coloured by R&lt;sub&gt;0&lt;/sub&gt;, as in the leftmost scatter plot. While the 95% percentiles for &lt;em&gt;R&lt;/em&gt;&lt;sub&gt;0&lt;/sub&gt; values are close to 1.0 (0.71-0.98) for the unbiased sequencing simulation (&lt;em&gt;i.e.&lt;/em&gt; uniform sequencing efforts, in which every case is equally likely to be sequenced), we also note that increasing levels of bias are considerably more to likely to generate MERS-CoV-like sequence clusters. The distribution of total number of introductions associated with simulations matching MERS-CoV sequence clusters is shown in the plots on the right, on the same y-axis across all levels of bias. Darker shade of grey indicates bins falling within the 95% percentiles. The median number of cross-species introductions observed in simulations matching empirical data without bias are 346 (95% percentiles 262&#x2013;439). These numbers jump up to 568 (95% percentiles 430&#x2013;727) for bias = 2.0 and 656 (95% percentiles 488&#x2013;853) for bias = 3.0 simulations. Model averaging would suggest plausible numbers of introductions between 311 and 811.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F11.medium.gif" width="440" height="408"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S5." src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F11.medium.gif" width="440" height="408"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S5.</span> <span class="caption-title">Monte Carlo simulations of human transmission clusters</span><p id="p-58" class="first-child">From top to bottom each row corresponds to departures from completely random sequencing efforts with respect to case cluster size (bias parameter=1.0) to sequencing increasingly biased towards capturing large case clusters (bias=2.0, bias=3.0). Leftmost scatter plots show the distribution of individual Monte Carlo simulation sequence cluster size statistics (mean and skewness) coloured by the <em>R</em><sub>0</sub> value used for the simulation. The dotted rectangle identifies the 95% highest posterior density bounds for sequence cluster size mean and skewness observed for empirical MERS-CoV data. The distribution of Ro values matching empirical data are shown in the middle, on the same y-axis across all levels of the bias parameter. Under unbiased sequencing (bias=1.0) only 0.45% of simulations fit our phylogenetic observations, while 1.79% and 1.67% of simulations fit for bias levels of 2.0 and 3.0, respectively. Correspondingly, we estimate 11.6% support for a model with bias level 1.0, 45.7% support for a model with bias level 2.0, and 42.7% support for a model with bias level 3.0. Bins falling inside the 95% percentiles are coloured by R<sub>0</sub>, as in the leftmost scatter plot. While the 95% percentiles for <em>R</em><sub>0</sub> values are close to 1.0 (0.71-0.98) for the unbiased sequencing simulation (<em>i.e.</em> uniform sequencing efforts, in which every case is equally likely to be sequenced), we also note that increasing levels of bias are considerably more to likely to generate MERS-CoV-like sequence clusters. The distribution of total number of introductions associated with simulations matching MERS-CoV sequence clusters is shown in the plots on the right, on the same y-axis across all levels of bias. Darker shade of grey indicates bins falling within the 95% percentiles. The median number of cross-species introductions observed in simulations matching empirical data without bias are 346 (95% percentiles 262–439). These numbers jump up to 568 (95% percentiles 430–727) for bias = 2.0 and 656 (95% percentiles 488–853) for bias = 3.0 simulations. Model averaging would suggest plausible numbers of introductions between 311 and 811.</p><div class="sb-div caption-clear"></div></div></div><div id="F12" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Monte Carlo simulation schematic Case clusters are simulated according to Equation 1 until an outbreak size of 2000 cases is reached. We sample 174 cases from each simulation to represent sequencing of human MERS cases. &#x2018;Sequencing&#x2019; is carried out by using multivariate hypergeometric sampling, representing sampling cases without replacement to be sequenced. Sequencing simulations take place at three levels of bias: 1.0, where every case is equally likely to be sequenced, and 2.0 and 3.0, where cases from larger clusters are increasingly more likely to be sequenced. The distribution of simulated sequence clusters is summarised by its mean, median and standard deviation. A simulation is considered to match if the mean, median and standard deviation of its sequence cluster sizes falls within the 95% highest posterior density interval of observed MERS-CoV sequence clusters. R0 values that ultimately generate data matching empirical observations, as well as associated numbers of &#x2018;introductions&#x2019; are retained as estimates. These estimates are summarised in Figure 3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-108077617" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Monte Carlo simulation schematic&lt;/span&gt; Case clusters are simulated according to Equation 1 until an outbreak size of 2000 cases is reached. We sample 174 cases from each simulation to represent sequencing of human MERS cases. &#x2018;Sequencing&#x2019; is carried out by using multivariate hypergeometric sampling, representing sampling cases without replacement to be sequenced. Sequencing simulations take place at three levels of bias: 1.0, where every case is equally likely to be sequenced, and 2.0 and 3.0, where cases from larger clusters are increasingly more likely to be sequenced. The distribution of simulated sequence clusters is summarised by its mean, median and standard deviation. A simulation is considered to match if the mean, median and standard deviation of its sequence cluster sizes falls within the 95% highest posterior density interval of observed MERS-CoV sequence clusters. &lt;em&gt;R&lt;/em&gt;&lt;sub&gt;0&lt;/sub&gt; values that ultimately generate data matching empirical observations, as well as associated numbers of &#x2018;introductions&#x2019; are retained as estimates. These estimates are summarised in Figure 3.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F12.medium.gif" width="440" height="385"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S6." src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F12.medium.gif" width="440" height="385"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S6.</span> <span class="caption-title">Monte Carlo simulation schematic</span><p id="p-59" class="first-child">Case clusters are simulated according to <a id="xref-disp-formula-2-4" class="xref-disp-formula" href="#disp-formula-2">Equation 1</a> until an outbreak size of 2000 cases is reached. We sample 174 cases from each simulation to represent sequencing of human MERS cases. ‘Sequencing’ is carried out by using multivariate hypergeometric sampling, representing sampling cases without replacement to be sequenced. Sequencing simulations take place at three levels of bias: 1.0, where every case is equally likely to be sequenced, and 2.0 and 3.0, where cases from larger clusters are increasingly more likely to be sequenced. The distribution of simulated sequence clusters is summarised by its mean, median and standard deviation. A simulation is considered to match if the mean, median and standard deviation of its sequence cluster sizes falls within the 95% highest posterior density interval of observed MERS-CoV sequence clusters. <em>R</em><sub>0</sub> values that ultimately generate data matching empirical observations, as well as associated numbers of ‘introductions’ are retained as estimates. These estimates are summarised in <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3</a>.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Monte Carlo simulations of human transmission clusters Leftmost scatter plot shows the distribution of individual Monte Carlo simulation sequence cluster size statistics (mean and skewness) coloured by the R0 value used for the simulation. The dotted rectangle identifies the 95% highest posterior density bounds for sequence cluster size mean and skewness observed for empirical MERS-CoV data. The distribution of R0 values that fall within 95% HPDs for sequence cluster size mean, standard deviation, skewness and number of introductions, is shown in the middle, on the same y-axis. Bins falling inside the 95% percentiles are coloured by R0, as in the leftmost scatter plot. The distribution of total number of introductions associated with simulations matching MERS-CoV sequence clusters is shown on the right. Darker shade of grey indicates bins falling within the 95% percentiles. Monte Carlo simulations indicate R0 for MERS-CoV in humans is likely to be below 1.0, with numbers of zoonotic transmissions numbering in the hundreds." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-108077617" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Monte Carlo simulations of human transmission clusters&lt;/span&gt; Leftmost scatter plot shows the distribution of individual Monte Carlo simulation sequence cluster size statistics (mean and skewness) coloured by the &lt;em&gt;R&lt;/em&gt;&lt;sub&gt;0&lt;/sub&gt; value used for the simulation. The dotted rectangle identifies the 95% highest posterior density bounds for sequence cluster size mean and skewness observed for empirical MERS-CoV data. The distribution of &lt;em&gt;R&lt;/em&gt;&lt;sub&gt;0&lt;/sub&gt; values that fall within 95% HPDs for sequence cluster size mean, standard deviation, skewness and number of introductions, is shown in the middle, on the same &lt;em&gt;y-axis&lt;/em&gt;. Bins falling inside the 95% percentiles are coloured by &lt;em&gt;R&lt;/em&gt;&lt;sub&gt;0&lt;/sub&gt;, as in the leftmost scatter plot. The distribution of total number of introductions associated with simulations matching MERS-CoV sequence clusters is shown on the right. Darker shade of grey indicates bins falling within the 95% percentiles. Monte Carlo simulations indicate &lt;em&gt;R&lt;/em&gt;&lt;sub&gt;0&lt;/sub&gt; for MERS-CoV in humans is likely to be below 1.0, with numbers of zoonotic transmissions numbering in the hundreds.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F3.medium.gif" width="440" height="138"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F3.medium.gif" width="440" height="138"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Monte Carlo simulations of human transmission clusters</span><p id="p-15" class="first-child">Leftmost scatter plot shows the distribution of individual Monte Carlo simulation sequence cluster size statistics (mean and skewness) coloured by the <em>R</em><sub>0</sub> value used for the simulation. The dotted rectangle identifies the 95% highest posterior density bounds for sequence cluster size mean and skewness observed for empirical MERS-CoV data. The distribution of <em>R</em><sub>0</sub> values that fall within 95% HPDs for sequence cluster size mean, standard deviation, skewness and number of introductions, is shown in the middle, on the same <em>y-axis</em>. Bins falling inside the 95% percentiles are coloured by <em>R</em><sub>0</sub>, as in the leftmost scatter plot. The distribution of total number of introductions associated with simulations matching MERS-CoV sequence clusters is shown on the right. Darker shade of grey indicates bins falling within the 95% percentiles. Monte Carlo simulations indicate <em>R</em><sub>0</sub> for MERS-CoV in humans is likely to be below 1.0, with numbers of zoonotic transmissions numbering in the hundreds.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">We find that observed phylogenetic patterns of sequence clustering strongly support <em>R</em>0 below 1.0 (middle panel in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). Mean <em>R</em><sub>0</sub> value observed in matching simulations is 0.72 (95% percentiles 0.57-0.90), suggesting the inability of the virus to sustain transmission in humans. Lower values for Ro in turn suggest relatively large numbers of zoonotic transfers of viruses into humans (right-hand panel in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). Median number of cross-species introductions observed in matching simulations is 592 (95% percentiles 311-811). Our results suggest a large number of unobserved MERS primary cases. Given this, we also performed simulations where the total number of cases is doubled to 4000 to explore the impact of dramatic underestimation of MERS cases. In these analyses <em>R</em><sub>0</sub> values tend to decrease even further as numbers of introductions go up, although very few simulations match currently observed MERS-CoV sequence data (<a id="xref-fig-13-1" class="xref-fig" href="#F13">Figure S7</a>).</p><div id="F13" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F13.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results of Monte Carlo simulations with vast underestimation of cases The plot is identical to Figure S5, but instead of 2000 cases, simulations were run with 4000 cases. With more unobserved cases the Ro values matching observed MERS-CoV sequence clusters can only be smaller, with a corresponding increase in numbers of zoonotic transmissions. However, the numbers of simulations that match MERS-CoV data go down as well." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-108077617" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Results of Monte Carlo simulations with vast underestimation of cases&lt;/span&gt; The plot is identical to Figure S5, but instead of 2000 cases, simulations were run with 4000 cases. With more unobserved cases the Ro values matching observed MERS-CoV sequence clusters can only be smaller, with a corresponding increase in numbers of zoonotic transmissions. However, the numbers of simulations that match MERS-CoV data go down as well.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F13.medium.gif" width="440" height="411"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S7." src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F13.medium.gif" width="440" height="411"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S7." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S7.</span> <span class="caption-title">Results of Monte Carlo simulations with vast underestimation of cases</span><p id="p-60" class="first-child">The plot is identical to <a id="xref-fig-11-3" class="xref-fig" href="#F11">Figure S5</a>, but instead of 2000 cases, simulations were run with 4000 cases. With more unobserved cases the Ro values matching observed MERS-CoV sequence clusters can only be smaller, with a corresponding increase in numbers of zoonotic transmissions. However, the numbers of simulations that match MERS-CoV data go down as well.</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">Overall, our analyses indicate that MERS-CoV is poorly suited for human-to-human transmission, with an estimated <em>R</em><sub>0</sub> &lt; 0.90 and sequence sampling likely to be biased towards large hospital outbreaks (<a id="xref-fig-11-2" class="xref-fig" href="#F11">Figure S5</a>). All matching simulations exhibit highly skewed distributions of case cluster sizes with long tails (<a id="xref-fig-14-1" class="xref-fig" href="#F14">Figure S8</a>), which is qualitatively similar to the results of (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Cauchemez et al., 2016</a>). We find that simulated sequence cluster sizes resemble observed sequence cluster sizes in the posterior distribution, with a slight reduction in mid-sized clusters in simulated data (<a id="xref-fig-15-1" class="xref-fig" href="#F15">Figure S9</a>). Given these findings, and the fact that large outbreaks of MERS occurred in hospitals, the combination of frequent spillover of MERS-CoV into humans and occasional outbreak amplification via poor hygiene practices (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Assiri et al., 2013b</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Chen et al., 2017</a>) appear sufficient to explain observed epidemiological patterns of MERS-CoV.</p><div id="F14" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F14.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Boxplots of matching simulated case and sequence cluster distributions Boxplots indicate frequency of case (blue, top) and sequence (red, bottom) cluster sizes across simulations at different bias levels, marginalised across R0 values. Outliers are shown with transparency, medians are indicated with thick black lines. Case clusters exhibit a strong skew with large numbers of singleton introductions and a substantial tail at higher levels of bias." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-108077617" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Boxplots of matching simulated case and sequence cluster distributions&lt;/span&gt; Boxplots indicate frequency of case (blue, top) and sequence (red, bottom) cluster sizes across simulations at different bias levels, marginalised across &lt;em&gt;R&lt;/em&gt;&lt;sub&gt;0&lt;/sub&gt; values. Outliers are shown with transparency, medians are indicated with thick black lines. Case clusters exhibit a strong skew with large numbers of singleton introductions and a substantial tail at higher levels of bias.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F14.medium.gif" width="440" height="359"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S8." src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F14.medium.gif" width="440" height="359"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S8." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S8.</span> <span class="caption-title">Boxplots of matching simulated case and sequence cluster distributions</span><p id="p-61" class="first-child">Boxplots indicate frequency of case (blue, top) and sequence (red, bottom) cluster sizes across simulations at different bias levels, marginalised across <em>R</em><sub>0</sub> values. Outliers are shown with transparency, medians are indicated with thick black lines. Case clusters exhibit a strong skew with large numbers of singleton introductions and a substantial tail at higher levels of bias.</p><div class="sb-div caption-clear"></div></div></div><div id="F15" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F15.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Quantile-quantile (Q-Q) plot of empirical and simulated sequence cluster sizes Density of sequence cluster size percentiles (1st-99th, calculated across a grid of 50 values) calculated for random states from the posterior distribution (x-axis) and matching simulations (y-axis). Most values fall on the one-to-one line, with a heavier tail in mid-sized sequence clusters in empirical data, manifesting as a greater density of points below the one-to-one line in the middle." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-108077617" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Quantile-quantile (Q-Q) plot of empirical and simulated sequence cluster sizes&lt;/span&gt; Density of sequence cluster size percentiles (1st-99th, calculated across a grid of 50 values) calculated for random states from the posterior distribution (x-axis) and matching simulations (y-axis). Most values fall on the one-to-one line, with a heavier tail in mid-sized sequence clusters in empirical data, manifesting as a greater density of points below the one-to-one line in the middle.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S9." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F15.medium.gif" width="440" height="365"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S9." src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F15.medium.gif" width="440" height="365"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F15.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S9." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F15.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S9.</span> <span class="caption-title">Quantile-quantile (Q-Q) plot of empirical and simulated sequence cluster sizes</span><p id="p-62" class="first-child">Density of sequence cluster size percentiles (1st-99th, calculated across a grid of 50 values) calculated for random states from the posterior distribution (x-axis) and matching simulations (y-axis). Most values fall on the one-to-one line, with a heavier tail in mid-sized sequence clusters in empirical data, manifesting as a greater density of points below the one-to-one line in the middle.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Recombination shapes MERS-CoV diversity</h3><p id="p-18">Recombination has been shown to occur in all genera of coronaviruses, including MERS- CoV (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Lai et al., 1985</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Makino et al., 1986</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Keck et al., 1988</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Kottier et al., 1995</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Herrewegh et al., 1998</a>). In order to quantify the degree to recombination has shaped MERS-CoV genetic diversity we used two recombination detection approaches across partitions of taxa corresponding to inferred MERS-CoV clades. Both methods rely on sampling parental and recombinant alleles within the alignment, although each quantifies different signals of recombination. One hallmark of recombination is the ability to carry alleles derived via mutation from one lineage to another, which appear as repeated mutations taking place in the recipient lineage, somewhere else in the tree. The PHI (pairwise homoplasy index) test quantifies the appearance of these excessive repeat mutations (homoplasies) within an alignment (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Bruen et al., 2006</a>). Another hallmark of recombination is clustering of alleles along the genome, due to how template switching, the primary mechanism of recombination in RNA viruses, occurs. 3Seq relies on the clustering of nucleotide similarities along the genome between sequence triplets – two potential parent-donors and one candidate offspring-recipient sequences (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Boni et al., 2007</a>).</p><p id="p-19">Both tests can give spurious results in cases of extreme rate heterogeneity and sampling over time (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Dudas and Rambaut, 2016</a>), but both tests have not been reported to fail simultaneously. PHI and 3Seq methods consistently identify most of the apparent ‘backbone’ of the MERS-CoV phylogeny as encompassing sequences with evidence of recombination (<a id="xref-fig-16-1" class="xref-fig" href="#F16">Figure S10</a>). Neither method can identify where in the tree recombination occurred, but each full asterisk in <a id="xref-fig-16-2" class="xref-fig" href="#F16">Figure S10</a> should be interpreted as the minimum partition of data that still captures both donor and recipient alleles involved in a recombination event. This suggests a non-negligible contribution of recombination in shaping existing MERS-CoV diversity. As done previously (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Dudas and Rambaut, 2016</a>), we show large numbers of homoplasies in MERS-CoV data (<a id="xref-fig-17-1" class="xref-fig" href="#F17">Figure S11</a>) with some evidence of genomic clustering of such alleles. These results are consistent with high incidence of MERS-CoV in camels (<a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Müller et al., 2014</a>; <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Corman et al., 2014</a>; <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Chu et al., 2014</a>; <a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">Reusken et al., 2014</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Ali et al., 2017</a>), allowing for co-infection with distinct genotypes and thus recombination to occur.</p><div id="F16" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F16.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Tests of recombination across MERS-CoV clades Maximum clade credibility tree of MERS-CoV genomes annotated with results of two recombination detection tests (PHI and 3Seq) applied to descendent sequences of each clade. Both tests identify large portions of existing sequence data as containing signals of recombination. Note that markings do not indicate where recombinations have occurred on the tree, merely the minimum distance in sequence/time space between recombining lineages." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-108077617" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Tests of recombination across MERS-CoV clades&lt;/span&gt; Maximum clade credibility tree of MERS-CoV genomes annotated with results of two recombination detection tests (PHI and 3Seq) applied to descendent sequences of each clade. Both tests identify large portions of existing sequence data as containing signals of recombination. Note that markings do not indicate where recombinations have occurred on the tree, merely the minimum distance in sequence/time space between recombining lineages.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S10." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F16.medium.gif" width="438" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S10." src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F16.medium.gif" width="438" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F16.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S10." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F16.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S10.</span> <span class="caption-title">Tests of recombination across MERS-CoV clades</span><p id="p-63" class="first-child">Maximum clade credibility tree of MERS-CoV genomes annotated with results of two recombination detection tests (PHI and 3Seq) applied to descendent sequences of each clade. Both tests identify large portions of existing sequence data as containing signals of recombination. Note that markings do not indicate where recombinations have occurred on the tree, merely the minimum distance in sequence/time space between recombining lineages.</p><div class="sb-div caption-clear"></div></div></div><div id="F17" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F17.large.jpg?width=800&amp;height=600&amp;carousel=1" title="MERS-CoV genomes exhibit high numbers of non-clonal loci Ancestral state reconstruction (right) identifies a large number of sites in which mutations have occurred more than once in the tree (homoplasies, orange) or are reversions (red) from a state arising in an ancestor. Mutations that apparently only occur once in the tree (synapomorphies) are shown in grey. The maximum likelihood phylogeny on the left is coloured by whether sequences were sampled in humans (blue) or camels (orange)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-108077617" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;MERS-CoV genomes exhibit high numbers of non-clonal loci&lt;/span&gt; Ancestral state reconstruction (right) identifies a large number of sites in which mutations have occurred more than once in the tree (homoplasies, orange) or are reversions (red) from a state arising in an ancestor. Mutations that apparently only occur once in the tree (synapomorphies) are shown in grey. The maximum likelihood phylogeny on the left is coloured by whether sequences were sampled in humans (blue) or camels (orange).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S11." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F17.medium.gif" width="440" height="233"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S11." src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F17.medium.gif" width="440" height="233"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F17.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S11." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F17.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S11.</span> <span class="caption-title">MERS-CoV genomes exhibit high numbers of non-clonal loci</span><p id="p-64" class="first-child">Ancestral state reconstruction (right) identifies a large number of sites in which mutations have occurred more than once in the tree (homoplasies, orange) or are reversions (red) from a state arising in an ancestor. Mutations that apparently only occur once in the tree (synapomorphies) are shown in grey. The maximum likelihood phylogeny on the left is coloured by whether sequences were sampled in humans (blue) or camels (orange).</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">Owing to these findings, we performed a sensitivity analysis in which we partitioned the MERS-CoV genome into two fragments and identified human outbreak clusters within each fragment. We find strong similarity in the grouping of human cases into outbreak clusters between fragments (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). Between the two trees in <a id="xref-fig-4-2" class="xref-fig" href="#F4">figure 4B</a> four (out of 54) ‘human’ clades are expanded where either singleton introductions or two-taxon clades in fragment 2 join other clades in fragment 1. For the reverse comparison there are five ‘human’ clades (out of 53) in fragment 2 that are expanded. All such clades have low divergence (<a id="xref-fig-4-3" class="xref-fig" href="#F4">figure 4B</a>) and thus incongruences in human clades are more likely to be caused by differences in deme assignment rather than actual recombination. And while we observe evidence of distinct phylogenetic trees from different parts of the MERS-CoV genome (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B</a>), human clades are minimally affected and large portions of the posterior probability density in both parts of the genome are concentrated in shared clades (<a id="xref-fig-18-1" class="xref-fig" href="#F18">Figure S12</a>). Additionally, we observe the same source-sink dynamics between camel and human populations in trees constructed from separate genomic fragments as were observed in the original full genome tree (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figures 1</a>, <a id="xref-fig-4-5" class="xref-fig" href="#F4">4B</a>).</p><div id="F18" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F18.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Human clade sharing between genomic fragments 1 and 2. Central scatter plot shows the posterior probability of human clades shared between genomic fragments 1 and 2, in their respective trees. Left and bottom scatter plots track the posterior probability of human clades only observed in fragment 2 (left) or fragment 1 (bottom). The cumulative probability of human clades present in either tree are tracked by plots on the right (fragment 2) and top (fragment 1). Most of the probability mass is concentrated within human clades that are present in trees of both genomic fragment 1 and 2 (0.9701 and 0.9474 of all human clades across posteriors, respectively)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-108077617" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Human clade sharing between genomic fragments 1 and 2.&lt;/span&gt; Central scatter plot shows the posterior probability of human clades shared between genomic fragments 1 and 2, in their respective trees. Left and bottom scatter plots track the posterior probability of human clades only observed in fragment 2 (left) or fragment 1 (bottom). The cumulative probability of human clades present in either tree are tracked by plots on the right (fragment 2) and top (fragment 1). Most of the probability mass is concentrated within human clades that are present in trees of both genomic fragment 1 and 2 (0.9701 and 0.9474 of all human clades across posteriors, respectively).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S12." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F18.medium.gif" width="440" height="422"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S12." src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F18.medium.gif" width="440" height="422"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F18.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S12." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F18.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S12.</span> <span class="caption-title">Human clade sharing between genomic fragments 1 and 2.</span><p id="p-65" class="first-child">Central scatter plot shows the posterior probability of human clades shared between genomic fragments 1 and 2, in their respective trees. Left and bottom scatter plots track the posterior probability of human clades only observed in fragment 2 (left) or fragment 1 (bottom). The cumulative probability of human clades present in either tree are tracked by plots on the right (fragment 2) and top (fragment 1). Most of the probability mass is concentrated within human clades that are present in trees of both genomic fragment 1 and 2 (0.9701 and 0.9474 of all human clades across posteriors, respectively).</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Recombinant features of MERS-CoV phylogenies A) Marginal posterior probabilities of taxa collected from humans belonging to the same clade in phylogenies derived from different parts of the genome. Taxa are ordered according to phylogeny of fragment 2 (genome positions 21001 to 29364) reduced to just the human tips and is displayed on the left. Human clusters are largely well-supported as monophyletic and consistent across trees of both genomic fragments. B) Tanglegram connecting the same taxa between a phylogeny derived from fragment 1 (left, genome positions 1 to 21000) and fragment 2 (right, genome positions 21001 to 29364), reduced to just the human tips and branches with posterior probability &lt; 0.1 collapsed. Human clusters exhibit limited diversity and corresponding low levels of incongruence within an introduction cluster." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-108077617" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Recombinant features of MERS-CoV phylogenies&lt;/span&gt; A) Marginal posterior probabilities of taxa collected from humans belonging to the same clade in phylogenies derived from different parts of the genome. Taxa are ordered according to phylogeny of fragment 2 (genome positions 21001 to 29364) reduced to just the human tips and is displayed on the left. Human clusters are largely well-supported as monophyletic and consistent across trees of both genomic fragments. B) Tanglegram connecting the same taxa between a phylogeny derived from fragment 1 (left, genome positions 1 to 21000) and fragment 2 (right, genome positions 21001 to 29364), reduced to just the human tips and branches with posterior probability &lt; 0.1 collapsed. Human clusters exhibit limited diversity and corresponding low levels of incongruence within an introduction cluster.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F4.medium.gif" width="440" height="182"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F4.medium.gif" width="440" height="182"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">Recombinant features of MERS-CoV phylogenies</span><p id="p-21" class="first-child">A) Marginal posterior probabilities of taxa collected from humans belonging to the same clade in phylogenies derived from different parts of the genome. Taxa are ordered according to phylogeny of fragment 2 (genome positions 21001 to 29364) reduced to just the human tips and is displayed on the left. Human clusters are largely well-supported as monophyletic and consistent across trees of both genomic fragments. B) Tanglegram connecting the same taxa between a phylogeny derived from fragment 1 (left, genome positions 1 to 21000) and fragment 2 (right, genome positions 21001 to 29364), reduced to just the human tips and branches with posterior probability &lt; 0.1 collapsed. Human clusters exhibit limited diversity and corresponding low levels of incongruence within an introduction cluster.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">Observed departures from strictly clonal evolution suggest that while recombination is an issue for inferring MERS-CoV phylogenies, its effect on the human side of MERS outbreaks is minimal, as expected if humans represent a transient host with little opportunity for co-infection. MERS-CoV evolution on the reservoir side is complicated by recombination, though is nonetheless still largely amenable to phylogenetic methods. Amongst other parameters of interest, recombination is expected to interfere with molecular clocks, where transferred genomic regions can give the impression of branches undergoing rapid evolution, or branches where recombination results in reversions appearing to evolve slow. In addition to its potential to influence tree topology, recombination in molecular sequence data is an erratic force with unpredictable effects. We suspect that the effects of recombination in MERS-CoV data are reigned in by a relatively small effective population size of the virus in Saudi Arabia (see next section) where haplotypes are fixed or nearly fixed, thus preventing an accumulation of genetic diversity that would then be reshuffled via recombination. Nevertheless, we choose not to report on any particular estimates for times of common ancestors (tMRCAs), even though these are expected to be somewhat robust for dating human clusters, and we do not report on the evolutionary rate of the virus, even though it appears to fall firmly within the expected range for RNA viruses: regression of nucleotide differences to Jordan-N3/2012 genome against sequence collection dates yields a rate of 4.59 × 10<sup>-4</sup> subs/site/year, Bayesian structured coalescent estimate from primary analysis 9.57 × 10<sup>-4</sup> (95% HPDs: 8.28 — 10.9 × 10<sup>-4</sup>) subs/site/year.</p></div><div id="sec-7" class="subsection"><h3>MERS-CoV shows population turnover in camels</h3><p id="p-23">Here we attempt to investigate MERS-CoV demographic patterns in the camel reservoir. We supplement camel sequence data with a single earliest sequence from each human cluster, treating viral diversity present in humans as a sentinel sample of MERS-CoV diversity circulating in camels. This removes conflicting demographic signals sampled during human outbreaks, where densely sampled closely related sequences from humans could be misconstrued as evidence of demographic crash in the viral population.</p><p id="p-24">Despite lack of convergence, neither of the two demographic reconstructions show evidence of fluctuations in the relative genetic diversity (<em>N<sub>e</sub>τ</em>) of MERS-CoV over time (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). We recover a similar demographic trajectory when estimating <em>N<sub>e</sub>τ</em> over time with a skygrid tree prior across the genome split into ten fragments with independent phylogenetic trees to account for confounding effects of recombination (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figures 5</a>, <a id="xref-fig-19-1" class="xref-fig" href="#F19">S13</a>). However, we do note that coalescence rate estimates are high relative to the sampling time period, with a mean estimate of <em>N<sub>e</sub>τ</em> at 3.49 years (95% HPD: 2.71-4.38), and consequently MERS-CoV phylogeny resembles a ladder, as often seen in human influenza A virus phylogenies (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bedford et al., 2011</a>).</p><div id="F19" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F19.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Skygrid comparison between whole and fragmented genomes Inferred median Ne&#x3C4; recovered using a skygrid tree prior on whole genome (bottom) and ten genomic fragments with independent trees (left), coloured by time. Dotted line indicates the one-to-one line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-108077617" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Skygrid comparison between whole and fragmented genomes&lt;/span&gt; Inferred median &lt;em&gt;N&lt;sub&gt;e&lt;/sub&gt;&#x3C4;&lt;/em&gt; recovered using a skygrid tree prior on whole genome (bottom) and ten genomic fragments with independent trees (left), coloured by time. Dotted line indicates the one-to-one line.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S13." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F19.medium.gif" width="440" height="355"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S13." src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F19.medium.gif" width="440" height="355"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F19.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S13." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F19.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S13.</span> <span class="caption-title">Skygrid comparison between whole and fragmented genomes</span><p id="p-66" class="first-child">Inferred median <em>N<sub>e</sub>τ</em> recovered using a skygrid tree prior on whole genome (bottom) and ten genomic fragments with independent trees (left), coloured by time. Dotted line indicates the one-to-one line.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Demographic history of MERS-CoV in Arabian peninsula camels Demographic history of MERS-CoV in camels, as inferred via a skygrid coalescent tree prior (Gill et al., 2013). Three skygrid reconstructions are shown, red and orange for each of the stationary distributions reached by MCMC with the whole genome and a black one where the genome was split into ten partitions. Shaded interval indicates the 95% highest posterior density interval for the product of generation time and effective population size, Ne&#x3C4;. Midline tracks the inferred median of Ne&#x3C4;." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-108077617" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Demographic history of MERS-CoV in Arabian peninsula camels&lt;/span&gt; Demographic history of MERS-CoV in camels, as inferred via a skygrid coalescent tree prior (Gill et al., 2013). Three skygrid reconstructions are shown, red and orange for each of the stationary distributions reached by MCMC with the whole genome and a black one where the genome was split into ten partitions. Shaded interval indicates the 95% highest posterior density interval for the product of generation time and effective population size, &lt;em&gt;N&lt;sub&gt;e&lt;/sub&gt;&#x3C4;&lt;/em&gt;. Midline tracks the inferred median of &lt;em&gt;N&lt;sub&gt;e&lt;/sub&gt;&#x3C4;&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F5.medium.gif" width="440" height="273"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F5.medium.gif" width="440" height="273"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <span class="caption-title">Demographic history of MERS-CoV in Arabian peninsula camels</span><p id="p-25" class="first-child">Demographic history of MERS-CoV in camels, as inferred via a skygrid coalescent tree prior (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Gill et al., 2013</a>). Three skygrid reconstructions are shown, red and orange for each of the stationary distributions reached by MCMC with the whole genome and a black one where the genome was split into ten partitions. Shaded interval indicates the 95% highest posterior density interval for the product of generation time and effective population size, <em>N<sub>e</sub>τ</em>. Midline tracks the inferred median of <em>N<sub>e</sub>τ</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">This empirically estimated effectived population can be compared to the expected effective population size in a simple epidemiological model. At endemic equilibrium, we expect scaled effective population size <em>N<sub>e</sub>τ</em> to follow I / 2<em>β</em>, where <em>β</em> is the equilibrium rate of transmission and <em>I</em> is the equilibrium number of infecteds (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Frost and Volz, 2010</a>). We assume that <em>β</em> is constant and is equal to the rate of recovery. Given a 20 day duration of infection in camels (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Adney et al., 2014</a>), we arrive at <em>β</em> = 365/20 = 18.25 infections per year. Given extremely high seroprevalence estimates within camels in Saudi Arabia (Muller et al., 2014; <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Corman et al., 2014</a>; <a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Chu et al., 2014</a>; <a id="xref-ref-61-4" class="xref-bibr" href="#ref-61">Reusken et al., 2013</a>, <a id="xref-ref-62-4" class="xref-bibr" href="#ref-62">2014</a>), we expect camels to usually be infected within their first year of life. Therefore we can estimate the rough number of camel infections per year as the number of calves produced each year. We find there are 830 000 camels in Saudi Arabia (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abdallah and Faye, 2013</a>) and that female camels in Saudi Arabia have an average fecundity of 45% (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Abdallah and Faye, 2013</a>). Thus, we expect 830 000 × 0.50 × 0.45 = 186 750 new calves produced yearly and correspondingly 186 750 new infections every year, which spread over 20 day intervals gives an average prevalence of <em>I</em> = 10 233 infections. This results in an expected scaled effective population size <em>N<sub>e</sub>τ</em> = 280.4 years.</p><p id="p-27">Comparing expected <em>N<sub>e</sub>τ</em> to empirical <em>N<sub>e</sub>τ</em> we arrive at a ratio of 80.3 (64.0–103.5). This is less than the equivalent ratio for human measles virus (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Bedford et al., 2011</a>) and is in line with the expectation from neutral evolutionary dynamics plus some degree of transmission heterogeneity (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Volz et al., 2013</a>) and seasonal troughs in prevalence. Thus, we believe that the ladder-like appearance of the MERS-CoV tree can likely be explained by entirely demographic factors.</p></div></div><div class="section" id="sec-8"><h2 class="">Discussion</h2><div id="sec-9" class="subsection"><h3>MERS-CoV epidemiology</h3><p id="p-28">In this study we aimed to understand the drivers of MERS coronavirus transmission in humans and what role the camel reservoir plays in perpetuating the epidemic in the Arabian peninsula by using sequence data collected from both hosts (174 from humans and 100 from camels). We showed that currently existing models of population structure (<a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Vaughan et al., 2014</a>) can identify distinct demographic modes in MERS-CoV genomic data, where viruses continuously circulating in camels repeatedly jump into humans and cause small outbreaks doomed to extinction (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figures 1</a>, <a id="xref-fig-7-3" class="xref-fig" href="#F7">S1</a>). This inference succeeds under different choices of priors for unknown demographic parameters (<a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure S2</a>) and in the presence of strong biases in sequence sampling schemes (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). When rapid coalescence in the human deme is not allowed (<a id="xref-fig-10-2" class="xref-fig" href="#F10">Figure S4</a>) structured coalescent inference loses power and ancestral state reconstruction is nearly identical to that of discrete trait analysis (<a id="xref-fig-9-3" class="xref-fig" href="#F9">Figure S3</a>). When allowed different deme-specific population sizes, the structured coalescent model succeeds because a large proportion of human sequences fall into tightly connected clusters, which informs a low estimate for the population size of the human deme. This in turn informs the inferred state of long ancestral branches in the phylogeny, <em>i.e.</em> because these long branches are not immediately coalescing, they are most likely in camels.</p><p id="p-29">From sequence data we identify at least 50 zoonotic introductions of MERS-CoV into humans from the reservoir (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>), from which we extrapolate that hundreds more such introductions must have taken place (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>). Although we recover migration rates from our model (<a id="xref-fig-8-4" class="xref-fig" href="#F8">Figure S2</a>), these only pertain to sequences and in no way reflect the epidemiologically relevant <em>per capita</em> rates of zoonotic spillover events. We also looked at potential seasonality in MERS-CoV spillover into humans. Our analyses indicated a period of three months where the odds of a sequenced spillover event are increased, with timing consistent with an enzootic amongst camel calves (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). As a result of our identification of large and asymmetric flow of viral lineages into humans we also find that the basic reproduction number for MERS-CoV in humans is well below the epidemic threshold (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>). Having said that, there are highly customisable coalescent methods available that extend the methods used here to accommodate time varying migration rates and population sizes, integrate alternative sources of information and fit to stochastic nonlinear models (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Rasmussen et al., 2014</a>), which would be more appropriate for MERS-CoV. Some distinct aspects of MERS-CoV epidemiology could not be captured in our methodology, such as hospital outbreaks where <em>R</em><sub>0</sub> is expected to be consistently closer to 1.0 compared to community transmission of MERS-CoV. Outside of coalescent-based models there are population structure models that explicitly relate epidemiological parameters to the branching process observed in sequence data (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Kühnert et al., 2016</a>), but often rely on specifying numerous informative priors and can suffer from MCMC convergence issues.</p><p id="p-30">Strong population structure in viruses often arises through limited gene flow, either due to geography (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Dudas et al., 2017</a>), ecology (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Smith et al., 2009</a>) or evolutionary forces (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Turner et al., 2005</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Dudas et al., 2015</a>). On a smaller scale population structure can unveil important details about transmission patterns, such as identifying reservoirs and understanding spillover trends and risk, much as we have done here. When properly understood naturally arising barriers to gene flow can be exploited for more efficient disease control and prevention, as well as risk management.</p></div><div id="sec-10" class="subsection"><h3>Transmissibility differences between zoonoses and pandemics</h3><p id="p-31">Severe acute respiratory syndrome (SARS) coronavirus, a Betacoronavirus like MERS-CoV, caused a serious epidemic in humans in 2003, with over 8000 cases and nearly 800 deaths. Since MERS-CoV was also able to cause significant pathogenicity in the human host it was inevitable that parallels would be drawn between MERS-CoV and SARS-CoV at the time of MERS discovery in 2012. Although we describe the epidemiology of MERS-CoV from sequence data, indications that MERS-CoV has poor capacity to spread human-to-human existed prior to any sequence data. SARS-CoV swept through the world in a short period of time, but MERS cases trickled slowly and were restricted to the Arabian Peninsula or resulted in self-limiting outbreaks outside of the region, a pattern strongly indicative of repeat zoonotic spillover. Infectious disease surveillance and control measures remain limited, so much like the SARS epidemic in 2003 or the H1N1 pandemic in 2009, zoonotic pathogens with <em>R</em><sub>0</sub> &gt; 1.0 are probably going to be discovered after spreading beyond the original location of spillover. Even though our results show that MERS-CoV does not appear to present an imminent global threat, we would like to highlight that its epidemiology is nonetheless concerning.</p><p id="p-32">Pathogens <em>Bacillus anthracis</em>, Andes hantavirus (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Martinez et al., 2005</a>), monkeypox (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Reed et al., 2004</a>) and influenza A are able to jump species barriers but only influenza A viruses have historically resulted in pandemics (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lipsitch et al., 2016</a>). MERS-CoV may join the list of pathogens able to jump species barriers but not spread efficiently in the new host. Since its emergence in 2012, MERS-CoV has caused self-limiting outbreaks in humans with no evidence of worsening outbreaks over time. However, sustained evolution of the virus in the reservoir and continued flow of viral lineages into humans provides the substrate for a more transmissible variant of MERS-CoV to possibly emerge. Previous modeling studies (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Antia et al., 2003</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Park et al., 2013</a>) suggest a positive relationship between initial R0 in the human host and probability of evolutionary emergence of a novel strain which passes the supercritical threshold of <em>R</em><sub>0</sub> &gt; 1.0. This leaves MERS-CoV in a precarious position; on one hand its current <em>R</em><sub>0</sub> of ~0.7 is certainly less than 1, but its proximity to the supercritical threshold raises alarm. With very little known about the fitness landscape or adaptive potential of MERS-CoV, it is incredibly challenging to predict the likelihood of the emergence more transmissible variants. In light of these difficulties, we encourage continued genomic surveillance of MERS-CoV in the camel reservoir and from sporadic human cases to rapidly identify a supercritical variant, if one does emerge.</p></div></div><div class="section" id="sec-11"><h2 class="">Methods</h2><div id="sec-12" class="subsection"><h3>Sequence data</h3><p id="p-33">All MERS-CoV sequences were downloaded from GenBank and accession numbers are given in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table S1</a>. Sequences without accessions were kindly shared by Ali M. Somily, Mazin Barry, Sarah S. Al Subaie, Abdulaziz A. BinSaeed, Fahad A. Alzamil, Waleed Zaher, Theeb Al Qahtani, Khaldoon Al Jerian, Scott J.N. McNabb, Imad A. Al-Jahdali, Ahmed M. Alotaibi, Nahid A. Batarfi, Matthew Cotten, Simon J. Watson, Spela Binter, and Paul Kellam prior to publication. Fragments of some strains submitted to GenBank as separate accessions were assembled into a single sequence. Only sequences covering at least 50% of MERS-CoV genome were kept, to facilitate later analyses where the alignment is split into two parts in order to account for effects of recombination (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Dudas and Rambaut, 2016</a>). Sequences were annotated with available collection dates and hosts, designated as camel or human, aligned with MAFFT (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Katoh and Standley, 2013</a>), and edited manually. Protein coding sequences were extracted and concatenated, reducing alignment length from 30130 down to 29364, which allowed for codon-partitioned substitution models to be used. The final dataset consisted of 174 genomes from human infections and 100 genomes from camel infections (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table S1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/425998/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/425998/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S1.</span> <p id="p-53" class="first-child">Strain names, accessions (where available), identified host and reported collection dates for MERS-CoV genomes used in this study.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Phylogenetic analyses</h3><div id="sec-14" class="subsection"><h4>Primary analysis, structured coalescent</h4><p id="p-34">For our primary analysis, the MultiTypeTree module (<a id="xref-ref-67-3" class="xref-bibr" href="#ref-67">Vaughan et al., 2014</a>) of BEAST v2.4.3 (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Bouckaert et al., 2014</a>) was used to specify a structured coalescent model with two demes – humans and camels. At time of writing structured population models are available in BEAST v2 (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Bouckaert et al., 2014</a>) but not in BEAST v1 (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Drummond et al., 2012</a>). We use the more computationally intensive MultiTypeTree module (<a id="xref-ref-67-4" class="xref-bibr" href="#ref-67">Vaughan et al., 2014</a>) over approximate methods also available in BEAST v2, such as BASTA (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Maio et al., 2015</a>), MASCOT (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Mueller et al., 2017</a>), and PhyDyn (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Volz, 2011</a>). Structured coalescent model implemented in the MultiTypeTree module (<a id="xref-ref-67-5" class="xref-bibr" href="#ref-67">Vaughan et al., 2014</a>) estimates four parameters: rate of coalescence in human viruses, rate of coalescence in camel viruses, rate of migration from the human deme to the camel deme and rate of migration from the camel deme to the human deme. Analyses were run on codon position partitioned data with two separate HKY+Γ<sub>4</sub> (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Hasegawa et al., 1985</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Yang, 1994</a>) nucleotide substitution models specified for codon positions 1+2 and 3. A relaxed molecular clock with branch rates drawn from a lognormal distribution (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Drummond et al., 2006</a>) was used to infer the evolutionary rate from date calibrated tips. Default priors were used for all parameters except for migration rates between demes for which an exponential prior with mean 1.0 was used. All analyses were run for 200 million steps across ten independent Markov chains (MCMC runs) and states were sampled every 20 000 steps. Due to the complexity of multitype tree parameter space 50% of states from every analysis were discarded as burn-in, convergence assessed in Tracer v1.6 and states combined using LogCombiner distributed with BEAST v2.4.3 (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Bouckaert et al., 2014</a>). Three chains out of ten did not converge and were discarded altogether. This left 70 000 states on which to base posterior inference. Posterior sets of typed (where migrating branches from structured coalescent are collapsed, and migration information is left as a switch in state between parent and descendant nodes) trees were summarised using TreeAnnotator v2.4.3 with the common ancestor heights option (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Heled and Bouckaert, 2013</a>). A maximum likelihood phylogeny showing just the genetic relationships between MERS-CoV genomes from camels and humans was recovered using PhyML (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Guindon et al., 2003</a>) under a HKY+Γ<sub>4</sub> (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Hasegawa et al., 1985</a>; <a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Yang, 1994</a>) nucleotide substitution model and is shown in <a id="xref-fig-20-1" class="xref-fig" href="#F20">Figure S14</a>.</p><div id="F20" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F20.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Maximum likelihood (ML) tree of MERS-CoV genomes coloured by origin of sequence Maximum likelihood tree shows genetic divergence between MERS-CoV genomes collected from camels (orange tips) and humans (blue tips)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-108077617" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Maximum likelihood (ML) tree of MERS-CoV genomes coloured by origin of sequence&lt;/span&gt; Maximum likelihood tree shows genetic divergence between MERS-CoV genomes collected from camels (orange tips) and humans (blue tips).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S14." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F20.medium.gif" width="432" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S14." src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F20.medium.gif" width="432" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F20.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S14." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F20.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S14.</span> <span class="caption-title">Maximum likelihood (ML) tree of MERS-CoV genomes coloured by origin of sequence</span><p id="p-67" class="first-child">Maximum likelihood tree shows genetic divergence between MERS-CoV genomes collected from camels (orange tips) and humans (blue tips).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h4>Control, structured coalescent with different prior</h4><p id="p-35">As a secondary analysis to test robustness to choice of prior, we set up an analysis where we increased the mean of the exponential distribution prior for migration rate to 10.0. All other parameters were identical to the primary analysis and as before 10 independent MCMC chains were run. In this case, two chains did not converge. This left 80 000 states on which to base posterior inference. Posterior sets of typed trees were summarised using TreeAnnotator v2.4.3 with the common ancestor heights option (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Heled and Bouckaert, 2013</a>).</p></div><div id="sec-16" class="subsection"><h4>Control, structured coalescent with equal deme sizes</h4><p id="p-36">To better understand where statistical power of the structured coalescent model lies we set up a tertiary analysis where a model was set up identically to the first structured coalescent analysis, but deme population sizes were enforced to have the same size. This analysis allowed us to differentiate whether statistical power in our analysis is coming from effective population size contrasts between demes or the backwards-in-time migration rate estimation. Five replicate chains were set up, two of which failed to converge after 200 million states. Combining the three converging runs left us with 15 000 trees sampled from the posterior distribution, which were summarised in TreeAnnotator v2.4.3 with the common ancestor heights option (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Heled and Bouckaert, 2013</a>).</p></div><div id="sec-17" class="subsection"><h4>Control, structured coalescent with more than one tree per genome</h4><p id="p-37">Due to concerns that recombination might affect our conclusions (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Dudas and Rambaut, 2016</a>), as an additional secondary analysis, we also considered a scenario where alignments were split into two fragments (fragment 1 comprised of positions 1-21000, fragment 2 of positions 21000-29364), with independent clocks, trees and migration rates, but shared substitution models and deme population sizes. Fragment positions were chosen based on consistent identification of the region around nucleotide 21000 as a probable breakpoint by GARD (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Pond et al., 2006</a>) by previous studies into SARS and MERS coronaviruses (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Hon et al., 2008</a>; <a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Dudas and Rambaut, 2016</a>). All analyses were set to run for 200 million states, subsampling every 20 000 states. Chains not converging after 200 million states were discarded. 20% of the states were discarded as burn-in, convergence assessed with Tracer 1.6 and combined with LogCombiner. Three chains out of ten did not converge. This left 70 000 states on which to base posterior inference. Posterior sets of typed trees were summarised using TreeAnnotator v2.4.3 with the common ancestor heights option (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Heled and Bouckaert, 2013</a>).</p></div><div id="sec-18" class="subsection"><h4>Control, discrete trait analysis</h4><p id="p-38">A currently widely used approach to infer ancestral states in phylogenies relies on treating traits of interest (such as geography, host, <em>etc.)</em> as features evolving along a phylogeny as continuous time Markov chains with an arbitrary number of states (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Lemey et al., 2009</a>). Unlike structured coalescent methods, such discrete trait approaches are independent from the tree <em>(i.e.</em> demographic) prior and thus unable to influence coalescence rates under different trait states. Such models have been used in the past to infer the number of MERS-CoV host jumps (<a id="xref-ref-75-3" class="xref-bibr" href="#ref-75">Zhang et al., 2016</a>) with results contradicting other sources of information. In order to test how a discrete trait approach compares to the structured coalescent we used our 274 MERS-CoV genome data set in BEAST v2.4.3 (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Bouckaert et al., 2014</a>) and specified identical codon-partitioned nucleotide substitution and molecular clock models to our structured coalescent analysis. To give the most comparable results we used a constant population size coalescent model, as this is the demographic function for each deme in the structured coalescent model. Five replicate MCMC analyses were run for 200 million states, three of which converged onto the same posterior distribution. The converging chains were combined after removing 20 million states as burn-in to give a total of 27 000 trees drawn from the posterior distribution. These trees were then summarised using TreeAnnotator v2.4.5 with the common ancestor heights option (<a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Heled and Bouckaert, 2013</a>).</p></div><div id="sec-19" class="subsection"><h4>Introduction seasonality</h4><p id="p-39">We extracted the times of camel-to-human introductions from the posterior distribution of multitype trees. This distribution of introduction times was then discretised as follows: for sample <em>k</em> = 1, 2,…, <em>L</em> from the posterior, <em>Z<sub>ijk</sub></em> was 1 if there as an introduction in month <em>i</em> and year <em>j</em> and 0 otherwise. We model the sum of introductions at month <em>i</em> and year <em>j</em> across the posterior sample <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/12/20/173211/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/12/20/173211/embed/inline-graphic-1.gif"/></noscript></span></span> with the hierarchical model:
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/12/20/173211/embed/graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/12/20/173211/embed/graphic-6.gif"/></noscript></span>

</span>
where <em>α<sub>j</sub></em> represents the contribution of year to expected introduction count and <em>β<sub>i</sub></em> represents the contribution of month to expected introduction count. Here, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/12/20/173211/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/12/20/173211/embed/inline-graphic-2.gif"/></noscript></span></span>. We sampled posterior values from this model via the Markov chain Monte Carlo methods implemented in Stan (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Carpenter et al., 2016</a>). Odds ratios of introductions were computed for each month <em>i</em> as OR<sub>i</sub> = exp(<em>β<sub>i</sub></em>).</p></div><div id="sec-20" class="subsection"><h4>Epidemiological analyses</h4><p id="p-40">Here, we employ a Monte Carlo simulation approach to identify parameters consistent with observed patterns of sequence clustering (<a id="xref-fig-12-2" class="xref-fig" href="#F12">Figure S6</a>). Our structured coalescent analyses indicate that every MERS outbreak is a contained cross-species spillover of the virus from camels into humans. The distribution of the number of these cross-species transmissions and their sizes thus contain information about the underlying transmission process. At heart, we expect fewer larger clusters if fundamental reproductive number <em>R</em><sub>0</sub> is large and more smaller clusters if <em>R</em><sub>0</sub> is small. A similar approach was used in <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Grubaugh et al. (2017)</a> to estimate <em>R</em><sub>0</sub> for Zika introductions into Florida.</p><p id="p-41">Branching process theory provides an analytical distribution for the number of eventual cases <em>j</em> in a transmission chain resulting from a single introduction event with <em>R</em><sub>0</sub> and dispersion parameter <em>ω</em> (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Blumberg and Lloyd-Smith, 2013</a>). This distribution follows
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/12/20/173211/embed/graphic-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/12/20/173211/embed/graphic-7.gif"/></noscript></span>

</span></p><p id="p-42">Here, <em>R</em><sub>0</sub> represents the expected number of secondary cases following a single infection and <em>ω</em> represents the dispersion parameter assuming secondary cases follow a negative binomial distribution (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Lloyd-Smith et al., 2005</a>), so that smaller values represent larger degrees of heterogeneity in the transmission process.</p><p id="p-43">As of 10 May 2017, the World Health Organization has been notified of 1952 cases of MERS-CoV (<a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">World Health Organization, 2017</a>). We thus simulated final transmission chain sizes using <a id="xref-disp-formula-2-3" class="xref-disp-formula" href="#disp-formula-2">Equation 1</a> until we reached an epidemic comprised of <em>N</em> = 2000 cases. 10 000 simulations were run for 121 uniformly spaced values of <em>R</em><sub>0</sub> across the range [0.5-1.1] with dispersion parameter <em>ω</em> fixed to 0.1 following expectations from previous studies of coronavirus behavior (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Lloyd-Smith et al., 2005</a>). Each simulation results in a vector of outbreak sizes <strong>c</strong>, where <em>c<sub>i</sub></em> is the size of the <em>i</em>th transmission cluster and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/12/20/173211/embed/inline-graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/12/20/173211/embed/inline-graphic-3.gif"/></noscript></span></span> and <em>K</em> is the number of clusters generated.</p><p id="p-44">Following the underlying transmission process generating case clusters <strong>c</strong> we simulate a secondary process of sampling some fraction of cases and sequencing them to generate data analogous to what we empirically observe. We sample from the case cluster size vector <strong>c</strong> without replacement according to a multivariate hypergeometric distribution (Algorithm 1). The resulting sequence cluster size vector s contains <em>K</em> entries, some of which are zero <em>(i.e.</em> case clusters not sequenced), but <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/12/20/173211/embed/inline-graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/12/20/173211/embed/inline-graphic-4.gif"/></noscript></span></span> which reflects the number of human MERS-CoV sequences used in this study. Note that this “sequencing capacity” parameter does not vary over time, even though MERS-CoV sequencing efforts have varied in intensity, starting out slow due to lack of awareness, methods and materials and increasing in response to hospital outbreaks later. As the default sampling scheme operates under equiprobable sequencing, we also simulated biased sequencing by using concentrated hypergeometric distributions where the probability mass function is squared (bias = 2) or cubed (bias = 3) and then normalized. Here, bias enriches the hypergeometric distribution so that sequences are sampled with weights proportional to <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/12/20/173211/embed/inline-graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/12/20/173211/embed/inline-graphic-5.gif"/></noscript></span></span>. Thus, bias makes larger clusters more likely to be ‘sequenced’.</p><p id="p-45">After simulations were completed, we identified simulations in which the recovered distribution of sequence cluster sizes s fell within the 95% highest posterior density intervals for four summary statistics of empirical MERS-CoV sequence cluster sizes recovered via structured coalescent analysis: number of sequence clusters, mean, standard deviation and skewness (third central moment). These values were 48-61 for number of sequence clusters, 2.87-3.65 for mean sequence cluster size, 4.84-6.02 for standard deviation of sequence cluster sizes, and 415.40-621.06 for skewness of sequence cluster sizes.</p><p id="p-46">We performed a smaller set of simulations with 2500 replicates and twice the number of cases, <em>i.e.</em> <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/12/20/173211/embed/inline-graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/12/20/173211/embed/inline-graphic-6.gif"/></noscript></span></span>, to explore a dramatically underreported epidemic. Additionally, we performed additional smaller set of simulations on a rougher grid of <em>R</em><sub>0</sub> values (23 values, 0.50-1.05), with 5 values of dispersion parameter <em>ω</em> (0.002, 0.04, 0.1, 0.5, 1.0) and 3 levels of bias (1, 2, 3) to justify our choice of dispersion parameter <em>ω</em> that was fixed to 0.1 in the main analyses (<a id="xref-fig-21-1" class="xref-fig" href="#F21">Figure S15</a>).</p><div id="F21" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F21.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Numbers of epidemiological simulations conforming to empirical observations Numbers indicate the total number of epidemiological simulations under each combination of bias and dispersion parameter &#x3C9; that result in MERS-CoV-like sequence cluster sizes. More simulations match observations with bias&gt; 1 and &#x3C9; &#x2248; 0.1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-108077617" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Numbers of epidemiological simulations conforming to empirical observations&lt;/span&gt; Numbers indicate the total number of epidemiological simulations under each combination of bias and dispersion parameter &#x3C9; that result in MERS-CoV-like sequence cluster sizes. More simulations match observations with bias&gt; 1 and &#x3C9; &#x2248; 0.1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S15." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F21.medium.gif" width="440" height="285"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S15." src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F21.medium.gif" width="440" height="285"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F21.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S15." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F21.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S15.</span> <span class="caption-title">Numbers of epidemiological simulations conforming to empirical observations</span><p id="p-68" class="first-child">Numbers indicate the total number of epidemiological simulations under each combination of bias and dispersion parameter ω that result in MERS-CoV-like sequence cluster sizes. More simulations match observations with bias&gt; 1 and ω ≈ 0.1.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-anchor type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F6.medium.gif" width="440" height="208"/><noscript><img class="highwire-fragment fragment-image" alt="Figure6" src="https://www.biorxiv.org/content/biorxiv/early/2017/12/20/173211/F6.medium.gif" width="440" height="208"/></noscript></span></div></div></div></div><div id="statement-1" class="statement"><span class="statement-label">Algorithm 1:</span><h3>Multivariate hypergeometric sampling scheme.</h3><p id="p-47">Pseudocode describes the multivariate hypergeometric sampling scheme that simulates sequencing. Probability of sequencing a given number of cases from a case cluster depends on cluster size and sequences left (<em>i.e.</em> “sequencing capacity”). The bias parameter determines how probability mass function of the hypergeometric distribution is concentrated.</p></div></div><div id="sec-21" class="subsection"><h4>Demographic inference of MERS-CoV in the reservoir</h4><p id="p-48">In order to infer the demographic history of MERS-CoV in camels we used the results of structured coalescent analyses to identify introductions of the virus into humans. The oldest sequence from each cluster introduced into humans was kept for further analysis. This procedure removes lineages coalescing rapidly in humans, which would otherwise introduce a strong signal of low effective population size. These subsampled MERS-CoV sequences from humans were combined with existing sequence data from camels to give us a dataset with minimal demographic signal coming from epidemiological processes in humans. Sequences belonging to the outgroup clade where most of MERS-CoV sequences from Egypt fall were removed out of concern that MERS epidemics in Saudi Arabia and Egypt are distinct epidemics with relatively poor sampling in the latter. Were more sequences of MERS-CoV available from other parts of Africa we speculate they would fall outside of the diversity that has been sampled in Saudi Arabia and cluster with early MERS-CoV sequences from Jordan and sequences from Egyptian camels. However, currently there are no indications of what MERS-CoV diversity looks like in camels east of Saudi Arabia. A flexible skygrid tree prior (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Gill et al., 2013</a>) was used to recover estimates of relative genetic diversity (<em>N<sub>e</sub>τ</em>) at 50 evenly spaced grid points across six years, ending at the most recent tip in the tree (2015 August) in BEAST v1.8.4 (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Drummond et al., 2012</a>), under a relaxed molecular clock with rates drawn from a lognormal distribution (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Drummond et al., 2006</a>) and codon position partitioned (positions 1 + 2 and 3) HKY+Γ<sub>4</sub> (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Hasegawa et al., 1985</a>; <a id="xref-ref-73-3" class="xref-bibr" href="#ref-73">Yang, 1994</a>) nucleotide substitution models. At time of writing advanced flexible coalescent tree priors from the skyline family, such as skygrid (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Gill et al., 2013</a>) are available in BEAST v1 (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Drummond et al., 2012</a>) but not in BEAST v2 (<a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Bouckaert et al., 2014</a>). We set up five independent MCMC chains to run for 500 million states, sampling every 50 000 states. This analysis suffered from poor convergence, where two chains converged onto one stationary distribution, two to another and the last chain onto a third stationary distribution, with high effective sample sizes. Demographic trajectories recovered by the two main stationary distributions are very similar and differences between the two appear to be caused by convergence onto subtly different tree topologies. This non-convergence effect may have been masked previously by the use of all available MERS-CoV sequences from humans which may have lead MCMC towards one of the multiple stationary distributions.</p><p id="p-49">To ensure that recombination was not interfering with the skygrid reconstruction we also split our MERS-CoV alignment into ten parts 2937 nucleotides long. These were then used as separate partitions with independent trees and clock rates in BEAST v1.8.4 (<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Drummond et al., 2012</a>). Nucleotide substitution and relaxed clock models were set up identically to the whole genome skygrid analysis described above (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Drummond et al., 2006</a>; <a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Hasegawa et al., 1985</a>; <a id="xref-ref-73-4" class="xref-bibr" href="#ref-73">Yang, 1994</a>). Skygrid coalescent tree prior (<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Gill et al., 2013</a>) was used jointly across all ten partitions for demographic inference. Five MCMC chains were set up, each running for 200 million states, sampling every 20 000 states.</p></div></div><div id="sec-22" class="subsection"><h3>Data availability</h3><p id="p-50">Sequence data and all analytical code is publicly available at <a href="https://github.com/blab/structured-mers">https://github.com/blab/structured-mers</a>.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-51">We would like to thank Allison Black for useful discussion and advice. GD is supported by the Mahan postdoctoral fellowship from the Fred Hutchinson Cancer Research Center. TB is a Pew Biomedical Scholar and is supported by NIH R35 GM119774-01. AR was supported in part by the European Union Seventh Framework Programme for research, technological development and demonstration under Grant Agreement no. 278433-PREDEMICS and no. 725422-RESERVOIRDOCS, and the Wellcome Trust through project 206298/Z/17/Z.</p><p id="p-52">We gratefully acknowledge the contribution of the following scientists for sharing MERS- CoV sequence data before publication:</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Abdallah</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Faye</span>  <span class="cit-name-given-names">B.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Typology of camel farming system in Saudi Arabia</span>. <abbr class="cit-jnl-abbrev">Emirates Journal of Food and Agriculture</abbr>. <span class="cit-vol">25</span>:<span class="cit-fpage">250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmirates%2BJournal%2Bof%2BFood%2Band%2BAgriculture%26rft.volume%253D25%26rft.spage%253D250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.2" data-doi="10.3201/eid2012.141280"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Adney</span>  <span class="cit-name-given-names">DR</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Doremalen</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">VR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bushmaker</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Wit</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bowen</span>  <span class="cit-name-given-names">RA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Munster</span>  <span class="cit-name-given-names">VJ.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Replication and Shedding of MERS-CoV in Upper Respiratory Tract of Inoculated Dromedary Camels</span>. <abbr class="cit-jnl-abbrev">Emerging Infectious Diseases</abbr>. <span class="cit-vol">20</span>:<span class="cit-fpage">1999</span>–<span class="cit-lpage">2005</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2BInfectious%2BDiseases%26rft.volume%253D20%26rft.spage%253D1999%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid2012.141280%26rft_id%253Dinfo%253Apmid%252F25418529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid2012.141280&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25418529&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ali</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shehata</span>  <span class="cit-name-given-names">MM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gomaa</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-etal">et al.</span> (18 co-authors). <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Systematic, active surveillance for Middle East respiratory syndrome coronavirus in camels in Egypt</span>. <abbr class="cit-jnl-abbrev">Emerg Microbes Infect</abbr>. <span class="cit-vol">6</span>:<span class="cit-fpage">e1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg%2BMicrobes%2BInfect%26rft.volume%253D6%26rft.spage%253D1e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Almutairi</span>  <span class="cit-name-given-names">SE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boujenane</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Musaad</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Awad-Acharari</span>  <span class="cit-name-given-names">F.</span></span> <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Non-genetic factors influencing reproductive traits and calving weight in Saudi camels</span>. <abbr class="cit-jnl-abbrev">Trop Anim Health Prod</abbr>. <span class="cit-vol">42</span>:<span class="cit-fpage">1087</span>–<span class="cit-lpage">1092</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTropical%2Banimal%2Bhealth%2Band%2Bproduction%26rft.stitle%253DTrop%2BAnim%2BHealth%2BProd%26rft.aulast%253DAlmutairi%26rft.auinit1%253DS.%2BE.%26rft.volume%253D42%26rft.issue%253D6%26rft.spage%253D1087%26rft.epage%253D1092%26rft.atitle%253DNon-genetic%2Bfactors%2Binfluencing%2Breproductive%2Btraits%2Band%2Bcalving%2Bweight%2Bin%2BSaudi%2Bcamels.%26rft_id%253Dinfo%253Apmid%252F20195755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20195755&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.5" data-doi="10.1038/nature02104"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Antia</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Regoes</span>  <span class="cit-name-given-names">RR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koella</span>  <span class="cit-name-given-names">JC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bergstrom</span>  <span class="cit-name-given-names">CT.</span></span> <span class="cit-pub-date">2003</span>. <span class="cit-article-title">The role of evolution in the emergence of infectious diseases</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-vol">426</span>:<span class="cit-fpage">658</span>–<span class="cit-lpage">661</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAntia%26rft.auinit1%253DR.%26rft.volume%253D426%26rft.issue%253D6967%26rft.spage%253D658%26rft.epage%253D661%26rft.atitle%253DThe%2Brole%2Bof%2Bevolution%2Bin%2Bthe%2Bemergence%2Bof%2Binfectious%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02104%26rft_id%253Dinfo%253Apmid%252F14668863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14668863&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187132800040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.6" data-doi="10.1016/S1473-3099(13)70204-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Assiri</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Al-Tawfiq</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Al-Rabeeah</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-etal">et al.</span> (13 co-authors). <span class="cit-pub-date">2013a</span>. <span class="cit-article-title">Epidemiological, demographic, and clinical characteristics of 47 cases of Middle East respiratory syndrome coronavirus disease from Saudi Arabia: a descriptive study</span>. <abbr class="cit-jnl-abbrev">Lancet Infect Dis</abbr>. <span class="cit-vol">13</span>:<span class="cit-fpage">752</span>–<span class="cit-lpage">761</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%2BInfect%2BDis%26rft.volume%253D13%26rft.spage%253D752%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1473-3099%252813%252970204-4%26rft_id%253Dinfo%253Apmid%252F23891402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1473-3099(13)70204-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23891402&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000323465900020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.7" data-doi="10.1056/NEJMoa1306742"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Assiri</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">McGeer</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perl</span>  <span class="cit-name-given-names">TM</span></span>, <span class="cit-etal">et al.</span> (18 co-authors). <span class="cit-pub-date">2013b</span>. <span class="cit-article-title">Hospital outbreak of Middle East respiratory syndrome coronavirus</span>. <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>. <span class="cit-vol">369</span>:<span class="cit-fpage">407</span>–<span class="cit-lpage">416</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft.volume%253D369%26rft.spage%253D407%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1306742%26rft_id%253Dinfo%253Apmid%252F23782161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa1306742&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23782161&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000322547400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.8" data-doi="10.1186/1471-2148-11-220"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bedford</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cobey</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pascual</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Strength and tempo of selection revealed in viral gene genealogies</span>. <abbr class="cit-jnl-abbrev">BMC Evol Biol</abbr>. <span class="cit-vol">11</span>:<span class="cit-fpage">220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DBedford%26rft.auinit1%253DT.%26rft.volume%253D11%26rft.spage%253D220%26rft.epage%253D220%26rft.atitle%253DStrength%2Band%2Btempo%2Bof%2Bselection%2Brevealed%2Bin%2Bviral%2Bgene%2Bgenealogies.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-11-220%26rft_id%253Dinfo%253Apmid%252F21787390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-11-220&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21787390&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.9" data-doi="10.1371/journal.pcbi.1002993"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Blumberg</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lloyd-Smith</span>  <span class="cit-name-given-names">JO.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Inference of <em>R</em><sub>0</sub> and transmission heterogeneity from the size distribution of stuttering chains</span>. <abbr class="cit-jnl-abbrev">PLoS Comput Biol</abbr>. <span class="cit-vol">9</span>:<span class="cit-fpage">e1002993</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BComput%2BBiol%26rft.volume%253D9%26rft.spage%253De1002993%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002993%26rft_id%253Dinfo%253Apmid%252F23658504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002993&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23658504&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.10" data-doi="10.1128/mBio.00473-12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Boheemen</span>  <span class="cit-name-given-names">Sv</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graaf</span>  <span class="cit-name-given-names">Md</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lauber</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-etal">et al.</span> (11 co-authors). <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Genomic characterization of a newly discovered coronavirus associated with acute respiratory distress syndrome in humans</span>. <abbr class="cit-jnl-abbrev">mBio</abbr>. <span class="cit-vol">3</span>:<span class="cit-fpage">e00473</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DmBio%26rft.volume%253D3%26rft.spage%253De00473%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.00473-12%26rft_id%253Dinfo%253Apmid%252F23170002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/mBio.00473-12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23170002&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.11" data-doi="10.1534/genetics.106.068874"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Boni</span>  <span class="cit-name-given-names">MF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feldman</span>  <span class="cit-name-given-names">MW.</span></span> <span class="cit-pub-date">2007</span>. <span class="cit-article-title">An exact nonparametric method for inferring mosaic structure in sequence triplets</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>. <span class="cit-vol">176</span>:<span class="cit-fpage">1035</span>–<span class="cit-lpage">1047</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D176%26rft.spage%253D1035%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.068874%26rft_id%253Dinfo%253Apmid%252F17409078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.106.068874&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17409078&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247870700027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.12" data-doi="10.1371/journal.pcbi.1003537"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bouckaert</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heled</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kühnert</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vaughan</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">CH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suchard</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">AJ.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">BEAST 2: A software platform for Bayesian evolutionary analysis</span>. <abbr class="cit-jnl-abbrev">PLoS Comput Biol</abbr>. <span class="cit-vol">10</span>:<span class="cit-fpage">e1003537</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BComput%2BBiol%26rft.volume%253D10%26rft.spage%253De1003537%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1003537%26rft_id%253Dinfo%253Apmid%252F24722319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1003537&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24722319&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.13" data-doi="10.1534/genetics.105.048975"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bruen</span>  <span class="cit-name-given-names">TC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Philippe</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bryant</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">A simple and robust statistical test for detecting the presence of recombination</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>. <span class="cit-vol">172</span>:<span class="cit-fpage">2665</span>–<span class="cit-lpage">2681</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D172%26rft.spage%253D2665%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.048975%26rft_id%253Dinfo%253Apmid%252F16489234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.105.048975&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16489234&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237225800052&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carpenter</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gelman</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoffman</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goodrich</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Betancourt</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brubaker</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Riddell</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Stan: A probabilistic programming language</span>. <abbr class="cit-jnl-abbrev">J Stat Softw</abbr>. <span class="cit-vol">20</span>:<span class="cit-fpage">1</span>–<span class="cit-lpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BStat%2BSoftw%26rft.volume%253D20%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.15" data-doi="10.1073/pnas.1519235113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cauchemez</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nouvellet</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cori</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> (25 co-authors). <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Unraveling the drivers of MERS-CoV transmission</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr>. <span class="cit-vol">113</span>:<span class="cit-fpage">9081</span>–<span class="cit-lpage">9086</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1519235113%26rft_id%253Dinfo%253Apmid%252F27457935%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEzLzMyLzkwODEiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNy8xMi8yMC8xNzMyMTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chughtai</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dyda</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">MacIntyre</span>  <span class="cit-name-given-names">CR.</span></span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Comparative epidemiology of Middle East respiratory syndrome coronavirus (MERS-CoV) in Saudi Arabia and South Korea</span>. <abbr class="cit-jnl-abbrev">Emerg Microbes Infect</abbr>. <span class="cit-vol">6</span>:<span class="cit-fpage">e51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg%2BMicrobes%2BInfect%26rft.volume%253D6%26rft.spage%253De51%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.17" data-doi="10.3201/eid2006.140299"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">DK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poon</span>  <span class="cit-name-given-names">LL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gomaa</span>  <span class="cit-name-given-names">MM</span></span>, <span class="cit-etal">et al.</span> (13 co-authors). <span class="cit-pub-date">2014</span>. <span class="cit-article-title">MERS coronaviruses in dromedary camels, Egypt</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr>. <span class="cit-vol">20</span>:<span class="cit-fpage">1049</span>–<span class="cit-lpage">1053</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg%2BInfect%2BDis%26rft.volume%253D20%26rft.spage%253D1049%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid2006.140299%26rft_id%253Dinfo%253Apmid%252F24856660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid2006.140299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24856660&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.18" data-doi="10.3201/eid2008.140596"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Corman</span>  <span class="cit-name-given-names">VM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jores</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-etal">et al.</span> (13 co-authors). <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Antibodies against MERS coronavirus in dromedary camels, Kenya, 1992-2013</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr>. <span class="cit-vol">20</span>:<span class="cit-fpage">1319</span>–<span class="cit-lpage">1322</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg%2BInfect%2BDis%26rft.volume%253D20%26rft.spage%253D1319%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid2008.140596%26rft_id%253Dinfo%253Apmid%252F25075637%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid2008.140596&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25075637&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.19" data-doi="10.1016/S0140-6736(1013)61887-61885"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cotten</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Watson</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kellam</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-etal">et al.</span> (22 co-authors). <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Transmission and evolution of the Middle East respiratory syndrome coronavirus in Saudi Arabia: a descriptive genomic study</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr>. <span class="cit-vol">382</span>:<span class="cit-fpage">1993</span>–<span class="cit-lpage">2002</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D382%26rft.spage%253D1993%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%25281013%252961887-61885%26rft_id%253Dinfo%253Apmid%252F24055451%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(1013)61887-61885&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24055451&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000328223700026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.20" data-doi="10.1371/journal.pbio.0040088"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">SYW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Relaxed phylogenetics and dating with confidence</span>. <abbr class="cit-jnl-abbrev">PLoS Biol</abbr>. <span class="cit-vol">4</span>:<span class="cit-fpage">e88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DDrummond%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D4%26rft.issue%253D5%26rft.spage%253De88%26rft.epage%253De88%26rft.atitle%253DRelaxed%2Bphylogenetics%2Band%2Bdating%2Bwith%2Bconfidence.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0040088%26rft_id%253Dinfo%253Apmid%252F16683862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0040088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16683862&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.21" data-doi="10.1093/molbev/mss075"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suchard</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Bayesian phylogenetics with BEAUti and the BEAST 1.7</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr>. <span class="cit-vol">29</span>:<span class="cit-fpage">1969</span>–<span class="cit-lpage">1973</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss075%26rft_id%253Dinfo%253Apmid%252F22367748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mss075&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22367748&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000307171300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.22" data-doi="10.1093/molbev/msu287"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dudas</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bedford</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lycett</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Reassortment between influenza B lineages and the emergence of a coadapted PB1-PB2-HA gene complex</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr>. <span class="cit-vol">32</span>:<span class="cit-fpage">162</span>–<span class="cit-lpage">172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu287%26rft_id%253Dinfo%253Apmid%252F25323575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu287&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25323575&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.23" data-doi="10.1038/nature22040"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dudas</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carvalho</span>  <span class="cit-name-given-names">LM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bedford</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-etal">et al.</span> (96 co-authors). <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Virus genomes reveal factors that spread and sustained the Ebola epidemic</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-vol">544</span>:<span class="cit-fpage">309</span>–<span class="cit-lpage">315</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D544%26rft.spage%253D309%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature22040%26rft_id%253Dinfo%253Apmid%252F28405027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature22040&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28405027&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.24" data-doi="10.1093/ve/vev023"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dudas</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">MERS-CoV recombination: implications about the reservoir and potential for adaptation</span>. <abbr class="cit-jnl-abbrev">Virus Evol</abbr>. <span class="cit-vol">2</span>:<span class="cit-fpage">vev023</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fve%252Fvev023%26rft_id%253Dinfo%253Apmid%252F27774293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/ve/vev023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27774293&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.25" data-doi="10.3201/eid2111.150944"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fagbo</span>  <span class="cit-name-given-names">SF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Skakni</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">DKW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garbati</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joseph</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hakawi</span>  <span class="cit-name-given-names">AM.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Molecular epidemiology of hospital outbreak of Middle East respiratory syndrome, Riyadh, Saudi Arabia, 2014</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr>. <span class="cit-vol">21</span>:<span class="cit-fpage">1981</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg%2BInfect%2BDis%26rft.volume%253D21%26rft.spage%253D1981%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid2111.150944%26rft_id%253Dinfo%253Apmid%252F26484549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid2111.150944&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26484549&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.26" data-doi="10.1098/rstb.2012.0196"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Faria</span>  <span class="cit-name-given-names">NR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suchard</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Streicker</span>  <span class="cit-name-given-names">DG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lemey</span>  <span class="cit-name-given-names">P.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Simultaneously reconstructing viral cross-species transmission history and identifying the underlying constraints</span>. <abbr class="cit-jnl-abbrev">Phil Trans R Soc B</abbr>. <span class="cit-vol">368</span>:<span class="cit-fpage">20120196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2012.0196%26rft_id%253Dinfo%253Apmid%252F23382420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2012.0196&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23382420&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Frost</span>  <span class="cit-name-given-names">SD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Volz</span>  <span class="cit-name-given-names">EM.</span></span> <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Viral phylodynamics and the search for an ‘effective number of infections’</span>. <abbr class="cit-jnl-abbrev">Philos Trans Royal Soc B Trans R Soc B</abbr>. <span class="cit-vol">365</span>:<span class="cit-fpage">1879</span>–<span class="cit-lpage">1890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos%2BTrans%2BRoyal%2BSoc%2BB%2BTrans%2BR%2BSoc%2BB%26rft.volume%253D365%26rft.spage%253D1879%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.28" data-doi="10.1093/molbev/mss265"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gill</span>  <span class="cit-name-given-names">MS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lemey</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Faria</span>  <span class="cit-name-given-names">NR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suchard</span>  <span class="cit-name-given-names">MA.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Improving bayesian population dynamics inference: A coalescent-based model for multiple loci</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr>. <span class="cit-vol">30</span>:<span class="cit-fpage">713</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss265%26rft_id%253Dinfo%253Apmid%252F23180580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mss265&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23180580&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000314891800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.29" data-doi="10.1126/science.1259657"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gire</span>  <span class="cit-name-given-names">SK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goba</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">KG</span></span>, <span class="cit-etal">et al.</span> (58 co-authors). <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Genomic surveillance elucidates Ebola virus origin and transmission during the 2014 outbreak</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-vol">345</span>:<span class="cit-fpage">1369</span>–<span class="cit-lpage">1372</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1259657%26rft_id%253Dinfo%253Apmid%252F25214632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDUvNjIwMi8xMzY5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMTIvMjAvMTczMjExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.30" data-doi="10.1038/nature22400"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grubaugh</span>  <span class="cit-name-given-names">ND</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ladner</span>  <span class="cit-name-given-names">JT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kraemer</span>  <span class="cit-name-given-names">MU</span></span>, <span class="cit-etal">et al.</span> (67 co-authors). <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Genomic epidemiology reveals multiple introductions of Zika virus into the United States</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-vol">546</span>:<span class="cit-fpage">401</span>–<span class="cit-lpage">405</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D546%26rft.spage%253D401%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature22400%26rft_id%253Dinfo%253Apmid%252F28538723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature22400&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28538723&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.31" data-doi="10.1080/10635150390235520"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guindon</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gascuel</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rannala</span>  <span class="cit-name-given-names">B.</span></span> <span class="cit-pub-date">2003</span>. <span class="cit-article-title">A Simple, Fast, and Accurate Algorithm to Estimate Large Phylogenies by Maximum Likelihood</span>. <abbr class="cit-jnl-abbrev">Systematic Biology</abbr>. <span class="cit-vol">52</span>:<span class="cit-fpage">696</span>–<span class="cit-lpage">704</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150390235520%26rft_id%253Dinfo%253Apmid%252F14530136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10635150390235520&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14530136&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185732500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.32" data-doi="10.1007/BF02101694"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hasegawa</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kishino</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yano</span>  <span class="cit-name-given-names">Ta.</span></span> <span class="cit-pub-date">1985</span>. <span class="cit-article-title">Dating of the human-ape splitting by a molecular clock of mitochondrial DNA</span>. <abbr class="cit-jnl-abbrev">J Mol Evol</abbr>. <span class="cit-vol">22</span>:<span class="cit-fpage">160</span>–<span class="cit-lpage">174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DHasegawa%26rft.auinit1%253DM.%26rft.volume%253D22%26rft.issue%253D2%26rft.spage%253D160%26rft.epage%253D174%26rft.atitle%253DDating%2Bof%2Bthe%2Bhuman-ape%2Bsplitting%2Bby%2Ba%2Bmolecular%2Bclock%2Bof%2Bmitochondrial%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02101694%26rft_id%253Dinfo%253Apmid%252F3934395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02101694&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3934395&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1985AWB1600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Heled</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bouckaert</span>  <span class="cit-name-given-names">RR.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Looking for trees in the forest: summary tree from posterior samples</span>. <abbr class="cit-jnl-abbrev">BMC Evolutionary Biology</abbr>. <span class="cit-vol">13</span>:<span class="cit-fpage">221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvolutionary%2BBiology%26rft.volume%253D13%26rft.spage%253D221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Herrewegh</span>  <span class="cit-name-given-names">AAPM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smeenk</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horzinek</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rottier</span>  <span class="cit-name-given-names">PJM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Groot</span>  <span class="cit-name-given-names">RJd.</span></span> <span class="cit-pub-date">1998</span>. <span class="cit-article-title">Feline coronavirus type II strains 79-1683 and 79-1146 originate from a double recombination between feline coronavirus type I and canine coronavirus</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-vol">72</span>:<span class="cit-fpage">4508</span>–<span class="cit-lpage">4514</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DHerrewegh%26rft.auinit1%253DA.%2BA.%25A0P.%25A0M.%26rft.volume%253D72%26rft.issue%253D5%26rft.spage%253D4508%26rft.epage%253D4514%26rft.atitle%253DFeline%2BCoronavirus%2BType%2BII%2BStrains%2B79-1683%25A0and%2B79-1146%25A0Originate%2Bfrom%2Ba%2BDouble%2BRecombination%2Bbetween%2BFeline%2BCoronavirus%2BType%2BI%2Band%2BCanine%2BCoronavirus%26rft_id%253Dinfo%253Apmid%252F9557750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjcyLzUvNDUwOCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzEyLzIwLzE3MzIxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.35" data-doi="10.1128/JVI.01926-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hon</span>  <span class="cit-name-given-names">CC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">TY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">ZL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yip</span>  <span class="cit-name-given-names">CW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">PY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leung</span>  <span class="cit-name-given-names">FCC.</span></span> <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Evidence of the recombinant origin of a bat severe acute respiratory syndrome (SARS)-like coronavirus and its implications on the direct ancestor of SARS coronavirus</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-vol">82</span>:<span class="cit-fpage">1819</span>–<span class="cit-lpage">1826</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01926-07%26rft_id%253Dinfo%253Apmid%252F18057240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgyLzQvMTgxOSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzEyLzIwLzE3MzIxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.36" data-doi="10.1093/molbev/mst010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Standley</span>  <span class="cit-name-given-names">DM.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">MAFFT Multiple Sequence Alignment Software Version 7: Improvements in Performance and Usability</span>. <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>. <span class="cit-vol">30</span>:<span class="cit-fpage">772</span>–<span class="cit-lpage">780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst010%26rft_id%253Dinfo%253Apmid%252F23329690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23329690&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000317002300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Keck</span>  <span class="cit-name-given-names">JG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matsushima</span>  <span class="cit-name-given-names">GK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Makino</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fleming</span>  <span class="cit-name-given-names">JO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vannier</span>  <span class="cit-name-given-names">DM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stohlman</span>  <span class="cit-name-given-names">SA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">MM.</span></span> <span class="cit-pub-date">1988</span>. <span class="cit-article-title">In vivo RNA-RNA recombination of coronavirus in mouse brain</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-vol">62</span>:<span class="cit-fpage">1810</span>–<span class="cit-lpage">1813</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DKeck%26rft.auinit1%253DJ%2BG%26rft.volume%253D62%26rft.issue%253D5%26rft.spage%253D1810%26rft.epage%253D1813%26rft.atitle%253DIn%2Bvivo%2BRNA-RNA%2Brecombination%2Bof%2Bcoronavirus%2Bin%2Bmouse%2Bbrain.%26rft_id%253Dinfo%253Apmid%252F2833625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjYyLzUvMTgxMCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzEyLzIwLzE3MzIxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.38" data-doi="10.1006/viro.1995.0029"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kottier</span>  <span class="cit-name-given-names">SA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cavanagh</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Britton</span>  <span class="cit-name-given-names">P.</span></span> <span class="cit-pub-date">1995</span>. <span class="cit-article-title">Experimental evidence of recombination in coronavirus infectious bronchitis virus</span>. <abbr class="cit-jnl-abbrev">Virology</abbr>. <span class="cit-vol">213</span>:<span class="cit-fpage">569</span>–<span class="cit-lpage">580</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DKottier%26rft.auinit1%253DS.%2BA.%26rft.volume%253D213%26rft.issue%253D2%26rft.spage%253D569%26rft.epage%253D580%26rft.atitle%253DExperimental%2Bevidence%2Bof%2Brecombination%2Bin%2Bcoronavirus%2Binfectious%2Bbronchitis%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fviro.1995.0029%26rft_id%253Dinfo%253Apmid%252F7491781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/viro.1995.0029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7491781&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995TE73200029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.39" data-doi="10.1093/molbev/msw064"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kühnert</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stadler</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vaughan</span>  <span class="cit-name-given-names">TG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">AJ.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Phylodynamics with Migration: A Computational Framework to Quantify Population Structure from Genomic Data</span>. <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>. <span class="cit-vol">33</span>:<span class="cit-fpage">2102</span>–<span class="cit-lpage">2116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw064%26rft_id%253Dinfo%253Apmid%252F27189573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27189573&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">MM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>  <span class="cit-name-given-names">RS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Makino</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keck</span>  <span class="cit-name-given-names">JG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Egbert</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leibowitz</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stohlman</span>  <span class="cit-name-given-names">SA.</span></span> <span class="cit-pub-date">1985</span>. <span class="cit-article-title">Recombination between nonsegmented RNA genomes of murine coronaviruses</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-vol">56</span>:<span class="cit-fpage">449</span>–<span class="cit-lpage">456</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DLai%26rft.auinit1%253DM%2BM%26rft.volume%253D56%26rft.issue%253D2%26rft.spage%253D449%26rft.epage%253D456%26rft.atitle%253DRecombination%2Bbetween%2Bnonsegmented%2BRNA%2Bgenomes%2Bof%2Bmurine%2Bcoronaviruses.%26rft_id%253Dinfo%253Apmid%252F2997467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjU2LzIvNDQ5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMTIvMjAvMTczMjExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lemey</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suchard</span>  <span class="cit-name-given-names">MA.</span></span> <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Bayesian Phylogeography Finds Its Roots</span>. <abbr class="cit-jnl-abbrev">PLOS Computational Biology</abbr>. <span class="cit-vol">5</span>:<span class="cit-fpage">e1000520</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComputational%2BBiology%26rft.volume%253D5%26rft.spage%253De1000520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.42" data-doi="10.7554/eLife.18491"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lipsitch</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barclay</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Raman</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-etal">et al.</span> (11 co-authors). <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Viral factors in influenza pandemic risk assessment</span>. <abbr class="cit-jnl-abbrev">eLife</abbr>. <span class="cit-vol">5</span>:<span class="cit-fpage">e18491</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D5%26rft.spage%253De18491%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.18491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.18491&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.43" data-doi="10.1016/S0140-6736(13)60938-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-etal">et al.</span> (11 co-authors). <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Origin and diversity of novel avian influenza A H7N9 viruses causing human infection: phylogenetic, structural, and coalescent analyses</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr>. <span class="cit-vol">381</span>:<span class="cit-fpage">1926</span>–<span class="cit-lpage">1932</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D381%26rft.spage%253D1926%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252813%252960938-1%26rft_id%253Dinfo%253Apmid%252F23643111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(13)60938-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23643111&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000320319500033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.44" data-doi="10.1038/nature04153"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lloyd-Smith</span>  <span class="cit-name-given-names">JO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schreiber</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kopp</span>  <span class="cit-name-given-names">PE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Getz</span>  <span class="cit-name-given-names">WM.</span></span> <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Superspreading and the effect of individual variation on disease emergence</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-vol">438</span>:<span class="cit-fpage">355</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLloyd-Smith%26rft.auinit1%253DJ.%2BO.%26rft.volume%253D438%26rft.issue%253D7066%26rft.spage%253D355%26rft.epage%253D359%26rft.atitle%253DSuperspreading%2Band%2Bthe%2Beffect%2Bof%2Bindividual%2Bvariation%2Bon%2Bdisease%2Bemergence.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04153%26rft_id%253Dinfo%253Apmid%252F16292310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04153&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16292310&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233300200048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.45" data-doi="10.1126/science.aaf8852"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lycett</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bodewes</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pohlmann</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> (11 co-authors). <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Role for migratory wild birds in the global spread of avian influenza H5N8</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-vol">354</span>:<span class="cit-fpage">213</span>–<span class="cit-lpage">217</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DThe%2BGlobal%2BConsortium%2Bfor%2BH5N8%2Band%2BRelated%2BInfluen%26rft.auinit1%253D%2B%26rft.volume%253D354%26rft.issue%253D6309%26rft.spage%253D213%26rft.epage%253D217%26rft.atitle%253DRole%2Bfor%2Bmigratory%2Bwild%2Bbirds%2Bin%2Bthe%2Bglobal%2Bspread%2Bof%2Bavian%2Binfluenza%2BH5N8%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaf8852%26rft_id%253Dinfo%253Apmid%252F27738169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNTQvNjMwOS8yMTMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNy8xMi8yMC8xNzMyMTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maio</span>  <span class="cit-name-given-names">ND</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">CH</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Reilly</span>  <span class="cit-name-given-names">KM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">New Routes to Phylogeography: A Bayesian Structured Coalescent Approximation</span>. <abbr class="cit-jnl-abbrev">PLOS Genetics</abbr>. <span class="cit-vol">11</span>:<span class="cit-fpage">e1005421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BGenetics%26rft.volume%253D11%26rft.spage%253De1005421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Makino</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keck</span>  <span class="cit-name-given-names">JG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stohlman</span>  <span class="cit-name-given-names">SA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">MM.</span></span> <span class="cit-pub-date">1986</span>. <span class="cit-article-title">High-frequency RNA recombination of murine coronaviruses</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-vol">57</span>:<span class="cit-fpage">729</span>–<span class="cit-lpage">737</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DMakino%26rft.auinit1%253DS%26rft.volume%253D57%26rft.issue%253D3%26rft.spage%253D729%26rft.epage%253D737%26rft.atitle%253DHigh-frequency%2BRNA%2Brecombination%2Bof%2Bmurine%2Bcoronaviruses.%26rft_id%253Dinfo%253Apmid%252F3005623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjU3LzMvNzI5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMTIvMjAvMTczMjExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.48" data-doi="10.3201/eid1112.050501"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martinez</span>  <span class="cit-name-given-names">VP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bellomo</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">San Juan</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pinna</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Forlenza</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elder</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Padula</span>  <span class="cit-name-given-names">PJ.</span></span> <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Person-to-person transmission of Andes virus</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr>. <span class="cit-vol">11</span>:<span class="cit-fpage">1848</span>–<span class="cit-lpage">1853</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2Binfectious%2Bdiseases%26rft.stitle%253DEmerg%2BInfect%2BDis%26rft.aulast%253DMartinez%26rft.auinit1%253DV.%2BP.%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253D1848%26rft.epage%253D1853%26rft.atitle%253DPerson-to-person%2Btransmission%2Bof%2BAndes%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid1112.050501%26rft_id%253Dinfo%253Apmid%252F16485469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid1112.050501&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16485469&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233768300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.49" data-doi="10.1098/rspb.2013.2438"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martinez-Bakker</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bakker</span>  <span class="cit-name-given-names">KM</span></span>, <span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rohani</span>  <span class="cit-name-given-names">P.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Human birth seasonality: latitudinal gradient and interplay with childhood disease dynamics</span>. In: <abbr class="cit-jnl-abbrev">Proc R Soc B</abbr>. volume <span class="cit-vol">281</span>, p. <span class="cit-fpage">20132438</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BR%2BSoc%2BB%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2013.2438%26rft_id%253Dinfo%253Apmid%252F24695423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2013.2438&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24695423&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.50" data-doi="10.3201/eid2006.140402"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Memish</span>  <span class="cit-name-given-names">ZA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cotten</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-etal">et al.</span> (17 co-authors). <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Human infection with MERS coronavirus after exposure to infected camels, Saudi Arabia, 2013</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr>. <span class="cit-vol">20</span>:<span class="cit-fpage">1012</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg%2BInfect%2BDis%26rft.volume%253D20%26rft.spage%253D1012%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid2006.140402%26rft_id%253Dinfo%253Apmid%252F24857749%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid2006.140402&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24857749&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.51"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">NF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rasmussen</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stadler</span>  <span class="cit-name-given-names">T.</span></span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">MASCOT: Parameter and state inference under the marginal structured coalescent approximation</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>. p. <span class="cit-fpage">188516</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.52"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Corman</span>  <span class="cit-name-given-names">VM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jores</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> (12 co-authors). <span class="cit-pub-date">2014</span>. <span class="cit-article-title">MERS coronavirus neutralizing antibodies in camels, Eastern Africa, 1983-1997</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr>. <span class="cit-vol">20</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.53" data-doi="10.1016/S1473-3099(15)70090-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Corman</span>  <span class="cit-name-given-names">VM</span></span>, <span class="cit-etal">et al.</span> (19 co-authors). <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Presence of Middle East respiratory syndrome coronavirus antibodies in Saudi Arabia: a nationwide, crosssectional, serological study</span>. <abbr class="cit-jnl-abbrev">Lancet Infect Dis</abbr>. <span class="cit-vol">15</span>:<span class="cit-fpage">559</span>–<span class="cit-lpage">564</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%2BInfect%2BDis%26rft.volume%253D15%26rft.spage%253D559%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1473-3099%252815%252970090-3%26rft_id%253Dinfo%253Apmid%252F25863564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1473-3099(15)70090-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25863564&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.54" data-doi="10.1007/BF00173909"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Notohara</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">1990</span>. <span class="cit-article-title">The coalescent and the genealogical process in geographically structured population</span>. <abbr class="cit-jnl-abbrev">J Math Biol</abbr>. <span class="cit-vol">29</span>:<span class="cit-fpage">59</span>–<span class="cit-lpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmathematical%2Bbiology%26rft.stitle%253DJ%2BMath%2BBiol%26rft.aulast%253DNotohara%26rft.auinit1%253DM.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D59%26rft.epage%253D75%26rft.atitle%253DThe%2Bcoalescent%2Band%2Bthe%2Bgenealogical%2Bprocess%2Bin%2Bgeographically%2Bstructured%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00173909%26rft_id%253Dinfo%253Apmid%252F2277236%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00173909&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2277236&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990EE60800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.55" data-doi="10.1098/rstb.2012.0333"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Loverdo</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schreiber</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lloyd-Smith</span>  <span class="cit-name-given-names">JO.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Multiple scales of selection influence the evolutionary emergence of novel pathogens</span>. <abbr class="cit-jnl-abbrev">Philos Trans Royal Soc B</abbr>. <span class="cit-vol">368</span>:<span class="cit-fpage">20120333</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos%2BTrans%2BRoyal%2BSoc%2BB%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2012.0333%26rft_id%253Dinfo%253Apmid%252F23382433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2012.0333&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23382433&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.56" data-doi="10.3201/eid2106.150038"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">SS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wernery</span>  <span class="cit-name-given-names">U</span></span>, <span class="cit-auth"><span class="cit-name-surname">Corman</span>  <span class="cit-name-given-names">VM</span></span>, <span class="cit-etal">et al.</span> (19 co-authors). <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Acute Middle East respiratory syndrome coronavirus infection in livestock dromedaries, Dubai, 2014</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr>. <span class="cit-vol">21</span>:<span class="cit-fpage">1019</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg%2BInfect%2BDis%26rft.volume%253D21%26rft.spage%253D1019%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid2106.150038%26rft_id%253Dinfo%253Apmid%252F25989145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid2106.150038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25989145&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.57" data-doi="10.1093/bioinformatics/btl474"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pond</span>,  <span class="cit-name-given-names">KLS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gravenor</span>  <span class="cit-name-given-names">MB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Woelk</span>  <span class="cit-name-given-names">CH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frost</span>  <span class="cit-name-given-names">SDW</span></span>. <span class="cit-pub-date">2006</span>. <span class="cit-article-title">GARD: a genetic algorithm for recombination detection</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-vol">22</span>:<span class="cit-fpage">3096</span>–<span class="cit-lpage">3098</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl474%26rft_id%253Dinfo%253Apmid%252F17110367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btl474&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17110367&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242715200019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.58" data-doi="10.1186/s13059-015-0677-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Quick</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ashton</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> (18 co-authors). <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Rapid draft sequencing and real-time nanopore sequencing in a hospital outbreak of Salmonella</span>. <abbr class="cit-jnl-abbrev">Genome Biology</abbr>. <span class="cit-vol">16</span>:<span class="cit-fpage">114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%26rft.volume%253D16%26rft.spage%253D114%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0677-2%26rft_id%253Dinfo%253Apmid%252F26025440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0677-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26025440&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.59"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rasmussen</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Volz</span>  <span class="cit-name-given-names">EM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koelle</span>  <span class="cit-name-given-names">K.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Phylodynamic Inference for Structured Epidemiological Models</span>. <abbr class="cit-jnl-abbrev">PLOS Computational Biology</abbr>. <span class="cit-vol">10</span>:<span class="cit-fpage">e1003570</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComputational%2BBiology%26rft.volume%253D10%26rft.spage%253De1003570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.60" data-doi="10.1056/NEJMoa032299"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reed</span>  <span class="cit-name-given-names">KD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Melski</span>  <span class="cit-name-given-names">JW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">MB</span></span>, <span class="cit-etal">et al.</span> (19 co-authors). <span class="cit-pub-date">2004</span>. <span class="cit-article-title">The detection of monkeypox in humans in the Western Hemisphere</span>. <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>. <span class="cit-vol">350</span>:<span class="cit-fpage">342</span>–<span class="cit-lpage">350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.stitle%253DNEJM%26rft.aulast%253DReed%26rft.auinit1%253DK.%2BD.%26rft.volume%253D350%26rft.issue%253D4%26rft.spage%253D342%26rft.epage%253D350%26rft.atitle%253DThe%2BDetection%2Bof%2BMonkeypox%2Bin%2BHumans%2Bin%2Bthe%2BWestern%2BHemisphere%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa032299%26rft_id%253Dinfo%253Apmid%252F14736926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa032299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14736926&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188254800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.61" data-doi="10.1016/S1473-3099(13)70164-6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reusken</span>  <span class="cit-name-given-names">CB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haagmans</span>  <span class="cit-name-given-names">BL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-etal">et al.</span> (24 co-authors). <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Middle East respiratory syndrome coronavirus neutralising serum antibodies in dromedary camels: a comparative serological study</span>. <abbr class="cit-jnl-abbrev">Lancet Infect Dis</abbr>. <span class="cit-vol">13</span>:<span class="cit-fpage">859</span>–<span class="cit-lpage">866</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%2BInfect%2BDis%26rft.volume%253D13%26rft.spage%253D859%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1473-3099%252813%252970164-6%26rft_id%253Dinfo%253Apmid%252F23933067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1473-3099(13)70164-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23933067&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000324899700020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.62" data-doi="10.3201/eid2008.140590"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reusken</span>  <span class="cit-name-given-names">CB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Messadi</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feyisa</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> (17 co-authors). <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Geographic distribution of MERS coronavirus among dromedary camels, Africa</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr>. <span class="cit-vol">20</span>:<span class="cit-fpage">1370</span>–<span class="cit-lpage">1374</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg%2BInfect%2BDis%26rft.volume%253D20%26rft.spage%253D1370%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid2008.140590%26rft_id%253Dinfo%253Apmid%252F25062254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid2008.140590&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25062254&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.63" data-doi="10.3201/eid2108.150481"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reusken</span>  <span class="cit-name-given-names">CBEM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Farag</span>  <span class="cit-name-given-names">EABA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haagmans</span>  <span class="cit-name-given-names">BL</span></span>, <span class="cit-etal">et al.</span> (21 co-authors). <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Occupational exposure to dromedaries and risk for MERS-CoV infection, Qatar, 2013–-2014</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr>. <span class="cit-vol">21</span>:<span class="cit-fpage">1422</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg%2BInfect%2BDis%26rft.volume%253D21%26rft.spage%253D1422%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid2108.150481%26rft_id%253Dinfo%253Apmid%252F26196891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid2108.150481&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26196891&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.64" data-doi="10.1073/pnas.0904991106"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">GJD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bahl</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vijaykrishna</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poon</span>  <span class="cit-name-given-names">LLM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Webster</span>  <span class="cit-name-given-names">RG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peiris</span>  <span class="cit-name-given-names">JSM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">Y.</span></span> <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Dating the emergence of pandemic influenza viruses</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr>. <span class="cit-vol">106</span>:<span class="cit-fpage">11709</span>–<span class="cit-lpage">11712</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0904991106%26rft_id%253Dinfo%253Apmid%252F19597152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzI4LzExNzA5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMTIvMjAvMTczMjExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.65"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">The WHO MERS-CoV Research Group</span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">State of knowledge and data gaps of Middle East respiratory syndrome coronavirus (MERS-CoV) in humans</span>. <abbr class="cit-jnl-abbrev">PLoS Curr</abbr>. Edition <span class="cit-edition">1</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.66" data-doi="10.1371/journal.pbio.0030285"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">TL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">MW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nuzhdin</span>  <span class="cit-name-given-names">SV.</span></span> <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Genomic islands of speciation in Anopheles gambiae</span>. <abbr class="cit-jnl-abbrev">PLoS Biol</abbr>. <span class="cit-vol">3</span>:<span class="cit-fpage">e285</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DTurner%26rft.auinit1%253DT.%2BL.%26rft.volume%253D3%26rft.issue%253D9%26rft.spage%253De285%26rft.epage%253De285%26rft.atitle%253DGenomic%2Bislands%2Bof%2Bspeciation%2Bin%2BAnopheles%2Bgambiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0030285%26rft_id%253Dinfo%253Apmid%252F16076241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0030285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16076241&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.67" data-doi="10.1093/bioinformatics/btu201"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vaughan</span>  <span class="cit-name-given-names">TG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kühnert</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Popinga</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Welch</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">AJ.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Efficient Bayesian inference under the structured coalescent</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-vol">30</span>:<span class="cit-fpage">2272</span>–<span class="cit-lpage">2279</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu201%26rft_id%253Dinfo%253Apmid%252F24753484%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu201&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24753484&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.68"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Volz</span>  <span class="cit-name-given-names">EM.</span></span> <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Complex Population Dynamics and the Coalescent under Neutrality</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>. p. genetics.111.134627.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.69" data-doi="10.1371/journal.pcbi.1002947"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Volz</span>  <span class="cit-name-given-names">EM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koelle</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bedford</span>  <span class="cit-name-given-names">T.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Viral phylodynamics</span>. <abbr class="cit-jnl-abbrev">PLoS Comput Biol</abbr>. <span class="cit-vol">9</span>:<span class="cit-fpage">e1002947</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BComput%2BBiol%26rft.volume%253D9%26rft.spage%253De1002947%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002947%26rft_id%253Dinfo%253Apmid%252F23555203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002947&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23555203&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.70"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wernery</span>  <span class="cit-name-given-names">U.</span></span> <span class="cit-pub-date">2001</span>. <span class="cit-article-title">Camelid immunoglobulins and their importance for the new-born – a review</span>. <abbr class="cit-jnl-abbrev">J Vet Med B</abbr>. <span class="cit-vol">48</span>:<span class="cit-fpage">561</span>–<span class="cit-lpage">568</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVet%2BMed%2BB%26rft.volume%253D48%26rft.spage%253D561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-web" id="cit-173211v2.71"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">World Health Organization</span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Disease outbreak news – 2016 december 19</span>. Available at <a href="http://www.who.int/csr/don/19-december-2016-2-mers-saudi-arabia/en/">http://www.who.int/csr/don/19-december-2016-2-mers-saudi-arabia/en/</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-web" id="cit-173211v2.72"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">World Health Organization</span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">WHO MERS-CoV global summary and assessment of risk</span>. Available at <a href="http://www.who.int/emergencies/mers-cov/risk-assessment-july-2017.pdf">http://www.who.int/emergencies/mers-cov/risk-assessment-july-2017.pdf</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.73" data-doi="10.1007/BF00160154"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span> <span class="cit-pub-date">1994</span>. <span class="cit-article-title">Maximum likelihood phylogenetic estimation from DNA sequences with variable rates over sites: Approximate methods</span>. <abbr class="cit-jnl-abbrev">J Mol Evol</abbr>. <span class="cit-vol">39</span>:<span class="cit-fpage">306</span>–<span class="cit-lpage">314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DYang%26rft.auinit1%253DZ.%26rft.volume%253D39%26rft.issue%253D3%26rft.spage%253D306%26rft.epage%253D314%26rft.atitle%253DMaximum%2Blikelihood%2Bphylogenetic%2Bestimation%2Bfrom%2BDNA%2Bsequences%2Bwith%2Bvariable%2Brates%2Bover%2Bsites%253A%2Bapproximate%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00160154%26rft_id%253Dinfo%253Apmid%252F7932792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00160154&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7932792&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PB53900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.74" data-doi="10.1056/NEJMoa1211721"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zaki</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Boheemen</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bestebroer</span>  <span class="cit-name-given-names">TM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Osterhaus</span>  <span class="cit-name-given-names">AD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fouchier</span>  <span class="cit-name-given-names">RA.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Isolation of a novel coronavirus from a man with pneumonia in Saudi Arabia</span>. <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>. <span class="cit-vol">367</span>:<span class="cit-fpage">1814</span>–<span class="cit-lpage">1820</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.stitle%253DNEJM%26rft.aulast%253DZaki%26rft.auinit1%253DA.%2BM.%26rft.volume%253D367%26rft.issue%253D19%26rft.spage%253D1814%26rft.epage%253D1820%26rft.atitle%253DIsolation%2Bof%2Ba%2Bnovel%2Bcoronavirus%2Bfrom%2Ba%2Bman%2Bwith%2Bpneumonia%2Bin%2BSaudi%2BArabia.%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1211721%26rft_id%253Dinfo%253Apmid%252F23075143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa1211721&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23075143&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F12%2F20%2F173211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310773200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-173211v2.75" data-doi="10.1038/srep25049"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">X.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Evolutionary dynamics of MERS-CoV: potential recombination, positive selection and transmission</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr>. <span class="cit-vol">6</span>:<span class="cit-fpage">25049</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D6%26rft.spage%253D25049%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep25049%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep25049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/173211v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F173211v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/185751v3" title="Temperate phages as self-replicating weapons in bacterial competition" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/236687v3" title="The compound topology of a continent-wide interaction network explained by an integrative hypothesis of specialization" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;December 20, 2017.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/173211v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/173211v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/173211v2" class="active" data-icon-position="" data-hide-link-title="0">MERS-CoV spillover at the camel-human interface</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/72610" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-kNEocWjTZaFM_coz3R0ycHEjmDjaIWYPe2keYfRRXM8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="72610" id="node72610--2" data-pisa="biorxiv;173211v2" data-pisa-master="biorxiv;173211" data-seqnum="72610" data-apath="/biorxiv/early/2017/12/20/173211.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">MERS-CoV spillover at the camel-human interface</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Gytis</span> <span class="nlm-surname">Dudas</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Luiz Max</span> <span class="nlm-surname">Carvalho</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Andrew</span> <span class="nlm-surname">Rambaut</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Trevor</span> <span class="nlm-surname">Bedford</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Ali M.</span> <span class="nlm-surname">Somily</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Mazin</span> <span class="nlm-surname">Barry</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Sarah S.</span> <span class="nlm-surname">Al Subaie</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Abdulaziz A.</span> <span class="nlm-surname">BinSaeed</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Fahad A.</span> <span class="nlm-surname">Alzamil</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Waleed</span> <span class="nlm-surname">Zaher</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Theeb Al</span> <span class="nlm-surname">Qahtani</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Khaldoon Al</span> <span class="nlm-surname">Jerian</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Scott J.N.</span> <span class="nlm-surname">McNabb</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Imad A.</span> <span class="nlm-surname">Al-Jahdali</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Ahmed M.</span> <span class="nlm-surname">Alotaibi</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Nahid A.</span> <span class="nlm-surname">Batarfi</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">Matthew</span> <span class="nlm-surname">Cotten</span></span>, <span class="highwire-citation-author" data-delta="17"><span class="nlm-given-names">Simon J.</span> <span class="nlm-surname">Watson</span></span>, <span class="highwire-citation-author" data-delta="18"><span class="nlm-given-names">Spela</span> <span class="nlm-surname">Binter</span></span>, <span class="highwire-citation-author" data-delta="19"><span class="nlm-given-names">Paul</span> <span class="nlm-surname">Kellam</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">173211; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/173211 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/173211v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/173211v2&amp;title=MERS-CoV%20spillover%20at%20the%20camel-human%20interface" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/173211v2&amp;title=MERS-CoV%20spillover%20at%20the%20camel-human%20interface" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/173211v2&amp;text=MERS-CoV%20spillover%20at%20the%20camel-human%20interface" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/173211v2&amp;title=MERS-CoV%20spillover%20at%20the%20camel-human%20interface" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/173211v2&amp;t=MERS-CoV%20spillover%20at%20the%20camel-human%20interface" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/173211v2&amp;title=MERS-CoV%20spillover%20at%20the%20camel-human%20interface" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/173211v2&amp;title=MERS-CoV%20spillover%20at%20the%20camel-human%20interface" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="72610" id="node72610--3" data-pisa="biorxiv;173211v2" data-pisa-master="biorxiv;173211" data-seqnum="72610" data-apath="/biorxiv/early/2017/12/20/173211.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">MERS-CoV spillover at the camel-human interface</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Gytis</span> <span class="nlm-surname">Dudas</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Luiz Max</span> <span class="nlm-surname">Carvalho</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Andrew</span> <span class="nlm-surname">Rambaut</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Trevor</span> <span class="nlm-surname">Bedford</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Ali M.</span> <span class="nlm-surname">Somily</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Mazin</span> <span class="nlm-surname">Barry</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Sarah S.</span> <span class="nlm-surname">Al Subaie</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Abdulaziz A.</span> <span class="nlm-surname">BinSaeed</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Fahad A.</span> <span class="nlm-surname">Alzamil</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Waleed</span> <span class="nlm-surname">Zaher</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Theeb Al</span> <span class="nlm-surname">Qahtani</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Khaldoon Al</span> <span class="nlm-surname">Jerian</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Scott J.N.</span> <span class="nlm-surname">McNabb</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Imad A.</span> <span class="nlm-surname">Al-Jahdali</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Ahmed M.</span> <span class="nlm-surname">Alotaibi</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Nahid A.</span> <span class="nlm-surname">Batarfi</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">Matthew</span> <span class="nlm-surname">Cotten</span></span>, <span class="highwire-citation-author" data-delta="17"><span class="nlm-given-names">Simon J.</span> <span class="nlm-surname">Watson</span></span>, <span class="highwire-citation-author" data-delta="18"><span class="nlm-given-names">Spela</span> <span class="nlm-surname">Binter</span></span>, <span class="highwire-citation-author" data-delta="19"><span class="nlm-given-names">Paul</span> <span class="nlm-surname">Kellam</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">173211; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/173211 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=MERS-CoV%20spillover%20at%20the%20camel-human%20interface&amp;rft.volume&amp;rft.issue&amp;rft.date=2017-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Dudas%2C+Gytis&amp;rft.au=Carvalho%2C+Luiz+Max&amp;rft.au=Rambaut%2C+Andrew&amp;rft.au=Bedford%2C+Trevor&amp;rft.au=Somily%2C+Ali+M.&amp;rft.au=Barry%2C+Mazin&amp;rft.au=Al+Subaie%2C+Sarah+S.&amp;rft.au=BinSaeed%2C+Abdulaziz+A.&amp;rft.au=Alzamil%2C+Fahad+A.&amp;rft.au=Zaher%2C+Waleed&amp;rft.au=Qahtani%2C+Theeb+Al&amp;rft.au=Jerian%2C+Khaldoon+Al&amp;rft.au=McNabb%2C+Scott+J.N.&amp;rft.au=Al-Jahdali%2C+Imad+A.&amp;rft.au=Alotaibi%2C+Ahmed+M.&amp;rft.au=Batarfi%2C+Nahid+A.&amp;rft.au=Cotten%2C+Matthew&amp;rft.au=Watson%2C+Simon+J.&amp;rft.au=Binter%2C+Spela&amp;rft.au=Kellam%2C+Paul"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/72610/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/72610/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/72610/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/72610/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/72610/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/72610/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/72610/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/72610/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/72610/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/72610/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/72610/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/72610/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/173211v2&amp;count=horizontal&amp;via=&amp;text=MERS-CoV%20spillover%20at%20the%20camel-human%20interface&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/173211v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/173211v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/173211v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/epidemiology" class="highlight" data-icon-position="" data-hide-link-title="0">Epidemiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__jnN2NeYkYJiOsZIjmjzMyRUq-BPexbFCB6tt10YTdRs__C0WUnZ9bv2nOjtXYdN5MmFM-XqUq6hlvZvg6KPPOnNo__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
