<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/03/02/2020.02.29.971101.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.02.29.971101v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Mutations, Recombination and Insertion in the Evolution of 2019-nCoV"/>
  <meta name="DC.Identifier" content="10.1101/2020.02.29.971101"/>
  <meta name="DC.Date" content="2020-03-02"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Background The 2019 novel coronavirus (2019-nCoV or SARS-CoV-2) has spread more rapidly than any other betacoronavirus including SARS-CoV and MERS-CoV. However, the mechanisms responsible for infection and molecular evolution of this virus remained unclear. Methods We collected and analyzed 120 genomic sequences of 2019-nCoV including 11 novel genomes from patients in China. Through comprehensive analysis of the available genome sequences of 2019-nCoV strains, we have tracked multiple inheritable SNPs and determined the evolution of 2019-nCoV relative to other coronaviruses. Results Systematic analysis of 120 genomic sequences of 2019-nCoV revealed co-circulation of two genetic subgroups with distinct SNPs markers, which can be used to trace the 2019-nCoV spreading pathways to different regions and countries. Although 2019-nCoV, human and bat SARS-CoV share high homologous in overall genome structures, they evolved into two distinct groups with different receptor entry specificities through potential recombination in the receptor binding regions. In addition, 2019-nCoV has a unique four amino acid insertion between S1 and S2 domains of the spike protein, which created a potential furin or TMPRSS2 cleavage site. Conclusions Our studies provided comprehensive insights into the evolution and spread of the 2019-nCoV. Our results provided evidence suggesting that 2019-nCoV may increase its infectivity through the receptor binding domain recombination and a cleavage site insertion. One Sentence Summary Novel 2019-nCoV sequences revealed the evolution and specificity of betacoronavirus with possible mechanisms of enhanced infectivity."/>
  <meta name="DC.Contributor" content="Aiping Wu"/>
  <meta name="DC.Contributor" content="Peihua Niu"/>
  <meta name="DC.Contributor" content="Lulan Wang"/>
  <meta name="DC.Contributor" content="Hangyu Zhou"/>
  <meta name="DC.Contributor" content="Xiang Zhao"/>
  <meta name="DC.Contributor" content="Wenling Wang"/>
  <meta name="DC.Contributor" content="Jingfeng Wang"/>
  <meta name="DC.Contributor" content="Chengyang Ji"/>
  <meta name="DC.Contributor" content="Xiao Ding"/>
  <meta name="DC.Contributor" content="Xianyue Wang"/>
  <meta name="DC.Contributor" content="Roujian Lu"/>
  <meta name="DC.Contributor" content="Sarah Gold"/>
  <meta name="DC.Contributor" content="Saba Aliyari"/>
  <meta name="DC.Contributor" content="Shilei Zhang"/>
  <meta name="DC.Contributor" content="Ellee Vikram"/>
  <meta name="DC.Contributor" content="Angela Zou"/>
  <meta name="DC.Contributor" content="Emily Lenh"/>
  <meta name="DC.Contributor" content="Janet Chen"/>
  <meta name="DC.Contributor" content="Fei Ye"/>
  <meta name="DC.Contributor" content="Na Han"/>
  <meta name="DC.Contributor" content="Yousong Peng"/>
  <meta name="DC.Contributor" content="Haitao Guo"/>
  <meta name="DC.Contributor" content="Guizhen Wu"/>
  <meta name="DC.Contributor" content="Taijiao Jiang"/>
  <meta name="DC.Contributor" content="Wenjie Tan"/>
  <meta name="DC.Contributor" content="Genhong Cheng"/>
  <meta name="article:published_time" content="2020-03-02"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Mutations, Recombination and Insertion in the Evolution of 2019-nCoV"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;Background: The 2019 novel coronavirus (2019-nCoV or SARS-CoV-2) has spread more rapidly than any other betacoronavirus including SARS-CoV and MERS-CoV. However, the mechanisms responsible for infection and molecular evolution of this virus remained unclear. Methods: We collected and analyzed 120 genomic sequences of 2019-nCoV including 11 novel genomes from patients in China. Through comprehensive analysis of the available genome sequences of 2019-nCoV strains, we have tracked multiple inheritable SNPs and determined the evolution of 2019-nCoV relative to other coronaviruses. Results: Systematic analysis of 120 genomic sequences of 2019-nCoV revealed co-circulation of two genetic subgroups with distinct SNPs markers, which can be used to trace the 2019-nCoV spreading pathways to different regions and countries. Although 2019-nCoV, human and bat SARS-CoV share high homologous in overall genome structures, they evolved into two distinct groups with different receptor entry specificities through potential recombination in the receptor binding regions. In addition, 2019-nCoV has a unique four amino acid insertion between S1 and S2 domains of the spike protein, which created a potential furin or TMPRSS2 cleavage site. Conclusions: Our studies provided comprehensive insights into the evolution and spread of the 2019-nCoV. Our results provided evidence suggesting that 2019-nCoV may increase its infectivity through the receptor binding domain recombination and a cleavage site insertion.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.02.29.971101v1"/>
  <meta name="citation_id" content="2020.02.29.971101v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.29.971101v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.29.971101v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.29.971101v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.02.29.971101"/>
  <meta name="citation_num_pages" content="38"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.02.29.971101"/>
  <meta name="citation_author" content="Aiping Wu"/>
  <meta name="citation_author_institution" content="Center for Systems Medicine, Institute of Basic Medical Sciences, Chinese Academy of Medical Sciences &amp;; Peking Union Medical College"/>
  <meta name="citation_author_institution" content="Suzhou Institute of Systems Medicine"/>
  <meta name="citation_author" content="Peihua Niu"/>
  <meta name="citation_author_institution" content="NHC Key Laboratory of Biosafety, National Institute for Viral Disease Control and Prevention"/>
  <meta name="citation_author" content="Lulan Wang"/>
  <meta name="citation_author_institution" content="Department of Microbiology, Immunology and Molecular Genetics, University of California"/>
  <meta name="citation_author" content="Hangyu Zhou"/>
  <meta name="citation_author_institution" content="Center for Systems Medicine, Institute of Basic Medical Sciences, Chinese Academy of Medical Sciences &amp;; Peking Union Medical College"/>
  <meta name="citation_author_institution" content="Suzhou Institute of Systems Medicine"/>
  <meta name="citation_author" content="Xiang Zhao"/>
  <meta name="citation_author_institution" content="NHC Key Laboratory of Biosafety, National Institute for Viral Disease Control and Prevention"/>
  <meta name="citation_author" content="Wenling Wang"/>
  <meta name="citation_author_institution" content="NHC Key Laboratory of Biosafety, National Institute for Viral Disease Control and Prevention"/>
  <meta name="citation_author" content="Jingfeng Wang"/>
  <meta name="citation_author_institution" content="Center for Systems Medicine, Institute of Basic Medical Sciences, Chinese Academy of Medical Sciences &amp;; Peking Union Medical College"/>
  <meta name="citation_author_institution" content="Suzhou Institute of Systems Medicine"/>
  <meta name="citation_author_institution" content="Department of Microbiology, Immunology and Molecular Genetics, University of California"/>
  <meta name="citation_author" content="Chengyang Ji"/>
  <meta name="citation_author_institution" content="Center for Systems Medicine, Institute of Basic Medical Sciences, Chinese Academy of Medical Sciences &amp;; Peking Union Medical College"/>
  <meta name="citation_author_institution" content="Suzhou Institute of Systems Medicine"/>
  <meta name="citation_author" content="Xiao Ding"/>
  <meta name="citation_author_institution" content="Center for Systems Medicine, Institute of Basic Medical Sciences, Chinese Academy of Medical Sciences &amp;; Peking Union Medical College"/>
  <meta name="citation_author_institution" content="Suzhou Institute of Systems Medicine"/>
  <meta name="citation_author" content="Xianyue Wang"/>
  <meta name="citation_author_institution" content="Center for Systems Medicine, Institute of Basic Medical Sciences, Chinese Academy of Medical Sciences &amp;; Peking Union Medical College"/>
  <meta name="citation_author_institution" content="Suzhou Institute of Systems Medicine"/>
  <meta name="citation_author" content="Roujian Lu"/>
  <meta name="citation_author_institution" content="NHC Key Laboratory of Biosafety, National Institute for Viral Disease Control and Prevention"/>
  <meta name="citation_author" content="Sarah Gold"/>
  <meta name="citation_author_institution" content="Department of Microbiology, Immunology and Molecular Genetics, University of California"/>
  <meta name="citation_author" content="Saba Aliyari"/>
  <meta name="citation_author_institution" content="Department of Microbiology, Immunology and Molecular Genetics, University of California"/>
  <meta name="citation_author" content="Shilei Zhang"/>
  <meta name="citation_author_institution" content="Department of Microbiology, Immunology and Molecular Genetics, University of California"/>
  <meta name="citation_author" content="Ellee Vikram"/>
  <meta name="citation_author_institution" content="Department of Microbiology, Immunology and Molecular Genetics, University of California"/>
  <meta name="citation_author" content="Angela Zou"/>
  <meta name="citation_author_institution" content="Department of Microbiology, Immunology and Molecular Genetics, University of California"/>
  <meta name="citation_author" content="Emily Lenh"/>
  <meta name="citation_author_institution" content="Department of Microbiology, Immunology and Molecular Genetics, University of California"/>
  <meta name="citation_author" content="Janet Chen"/>
  <meta name="citation_author_institution" content="Department of Microbiology, Immunology and Molecular Genetics, University of California"/>
  <meta name="citation_author" content="Fei Ye"/>
  <meta name="citation_author_institution" content="NHC Key Laboratory of Biosafety, National Institute for Viral Disease Control and Prevention"/>
  <meta name="citation_author" content="Na Han"/>
  <meta name="citation_author_institution" content="Center for Systems Medicine, Institute of Basic Medical Sciences, Chinese Academy of Medical Sciences &amp;; Peking Union Medical College"/>
  <meta name="citation_author_institution" content="Suzhou Institute of Systems Medicine"/>
  <meta name="citation_author" content="Yousong Peng"/>
  <meta name="citation_author_institution" content="College of Biology, Hunan Provincial Key Laboratory of Medical Virology, Hunan University"/>
  <meta name="citation_author" content="Haitao Guo"/>
  <meta name="citation_author_institution" content="Department of Microbiology and Molecular Genetics, University of Pittsburgh"/>
  <meta name="citation_author" content="Guizhen Wu"/>
  <meta name="citation_author_institution" content="NHC Key Laboratory of Biosafety, National Institute for Viral Disease Control and Prevention"/>
  <meta name="citation_author_email" content="gcheng@mednet.ucla.edu"/>
  <meta name="citation_author_email" content="tanwj@ivdc.chinacdc.cn"/>
  <meta name="citation_author_email" content="taijiao@ibms.pumc.edu.cn"/>
  <meta name="citation_author_email" content="wugz@ivdc.chinacdc.cn"/>
  <meta name="citation_author" content="Taijiao Jiang"/>
  <meta name="citation_author_institution" content="Center for Systems Medicine, Institute of Basic Medical Sciences, Chinese Academy of Medical Sciences &amp;; Peking Union Medical College"/>
  <meta name="citation_author_institution" content="Suzhou Institute of Systems Medicine"/>
  <meta name="citation_author_email" content="gcheng@mednet.ucla.edu"/>
  <meta name="citation_author_email" content="tanwj@ivdc.chinacdc.cn"/>
  <meta name="citation_author_email" content="taijiao@ibms.pumc.edu.cn"/>
  <meta name="citation_author_email" content="wugz@ivdc.chinacdc.cn"/>
  <meta name="citation_author" content="Wenjie Tan"/>
  <meta name="citation_author_institution" content="NHC Key Laboratory of Biosafety, National Institute for Viral Disease Control and Prevention"/>
  <meta name="citation_author_email" content="gcheng@mednet.ucla.edu"/>
  <meta name="citation_author_email" content="tanwj@ivdc.chinacdc.cn"/>
  <meta name="citation_author_email" content="taijiao@ibms.pumc.edu.cn"/>
  <meta name="citation_author_email" content="wugz@ivdc.chinacdc.cn"/>
  <meta name="citation_author" content="Genhong Cheng"/>
  <meta name="citation_author_institution" content="Department of Microbiology, Immunology and Molecular Genetics, University of California"/>
  <meta name="citation_author_email" content="gcheng@mednet.ucla.edu"/>
  <meta name="citation_author_email" content="tanwj@ivdc.chinacdc.cn"/>
  <meta name="citation_author_email" content="taijiao@ibms.pumc.edu.cn"/>
  <meta name="citation_author_email" content="wugz@ivdc.chinacdc.cn"/>
  <meta name="citation_reference" content="WHO. Clinical management of severe acute respiratory infection when Novel coronavirus (nCoV) infection is suspected: interim guidance. 2020."/>
  <meta name="citation_reference" content="WHO. Confirmed 2019-nCoV Cases Globally Global Map. 2020."/>
  <meta name="citation_reference" content="Munster VJ, Koopmans M, van Doremalen N, van Riel D, de Wit E. A Novel Coronavirus Emerging in China - Key Questions for Impact Assessment. The New England journal of medicine 2020."/>
  <meta name="citation_reference" content="Chan JF, Yuan S, Kok KH, et al. A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster. Lancet (London, England) 2020."/>
  <meta name="citation_reference" content="Zhu N, Zhang D, Wang W, et al. A Novel Coronavirus from Patients with Pneumonia in China, 2019. The New England journal of medicine 2020."/>
  <meta name="citation_reference" content="Liu DX, Fung TS, Chong KK, Shukla A, Hilgenfeld R. Accessory proteins of SARS-CoV and other coronaviruses. Antiviral research 2014;109:97–109."/>
  <meta name="citation_reference" content="Huang C, Wang Y, Li X, et al. Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. Lancet (London, England) 2020."/>
  <meta name="citation_reference" content="Shukla A, Hilgenfeld R. Acquisition of new protein domains by coronaviruses: analysis of overlapping genes coding for proteins N and 9b in SARS coronavirus. Virus genes 2015;50:29–38."/>
  <meta name="citation_reference" content="Li F. Receptor recognition and cross-species infections of SARS coronavirus. Antiviral research 2013;100:246–54."/>
  <meta name="citation_reference" content="Li F. Structure, Function, and Evolution of Coronavirus Spike Proteins. Annual review of virology 2016;3:237–61."/>
  <meta name="citation_reference" content="Belouzard S, Chu VC, Whittaker GR. Activation of the SARS coronavirus spike protein via sequential proteolytic cleavage at two distinct sites. Proceedings of the National Academy of Sciences of the United States of America 2009;106:5871–6."/>
  <meta name="citation_reference" content="Zheng Y, Shang J, Yang Y, et al. Lysosomal Proteases Are a Determinant of Coronavirus Tropism. Journal of virology 2018;92."/>
  <meta name="citation_reference" content="Lu G, Wang Q, Gao GF. Bat-to-human: spike features determining ‘host jump’ of coronaviruses SARS-CoV, MERS-CoV, and beyond. Trends in microbiology 2015;23:468–78."/>
  <meta name="citation_reference" content="White JM, Delos SE, Brecher M, Schornberg K. Structures and mechanisms of viral membrane fusion proteins: multiple variations on a common theme. Critical reviews in biochemistry and molecular biology 2008;43:189–219."/>
  <meta name="citation_reference" content="Peng Zhou X-LY, Xian-Guang Wang, Ben Hu, Lei Zhang, Wei Zhang, Hao-Rui Si, Yan Zhu, Bei Li, Chao-Lin Huang, Hui-Dong Chen, Jing Chen, Yun Luo, Hua Guo, Ren-Di Jiang, Mei-Qin Liu YC, Xu-Rui Shen, Xi Wang, Xiao-Shuang Zheng, Kai Zhao, Quan-Jiao Chen FD, Lin-Lin Liu, Bing Yan, Fa-Xian Zhan, Yan-Yi Wang, Geng-Fu Xiao &amp;;, Shi Z-L. A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature 2020."/>
  <meta name="citation_reference" content="Wan Y, Shang J, Graham R, Baric RS, Li F. Receptor recognition by novel coronavirus from Wuhan: An analysis based on decade-long structural studies of SARS. Journal of virology 2020."/>
  <meta name="citation_reference" content="Liu P, Chen W, Chen JP. Viral Metagenomics Revealed Sendai Virus and Coronavirus Infection of Malayan Pangolins (Manis javanica). Viruses 2019;11."/>
  <meta name="citation_reference" content="de Haan CA, Rottier PJ. Molecular interactions in the assembly of coronaviruses. Advances in virus research 2005;64:165–230."/>
  <meta name="citation_reference" content="White JM, Whittaker GR. Fusion of Enveloped Viruses in Endosomes. Traffic (Copenhagen, Denmark) 2016;17:593–614."/>
  <meta name="citation_reference" content="Daniel Wrapp NW, Kizzmekia S. Corbett, Jory A. Goldsmith, Ching-Lin Hsieh, Olubukola Abiona, Barney S. Graham, View ORCID ProfileJason S. McLellan. Cryo-EM Structure of the 2019-nCoV Spike in the Prefusion Conformation. 2020."/>
  <meta name="citation_reference" content="Lau SK, Feng Y, Chen H, et al. Severe Acute Respiratory Syndrome (SARS) Coronavirus ORF8 Protein Is Acquired from SARS-Related Coronavirus from Greater Horseshoe Bats through Recombination. Journal of virology 2015;89:10532–47."/>
  <meta name="citation_reference" content="Ge XY, Li JL, Yang XL, et al. Isolation and characterization of a bat SARS-like coronavirus that uses the ACE2 receptor. Nature 2013;503:535–8."/>
  <meta name="citation_reference" content="Lu R, Zhao X, Li J, et al. Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding. Lancet (London, England) 2020."/>
  <meta name="citation_reference" content="Cohen J. Mining coronavirus genomes for clues to the outbreak’s origins. Science (New York, NY) 2020."/>
  <meta name="citation_reference" content="Li W, Shi Z, Yu M, et al. Bats are natural reservoirs of SARS-like coronaviruses. Science (New York, NY) 2005;310:676–9."/>
  <meta name="citation_reference" content="Hu B, Zeng LP, Yang XL, et al. Discovery of a rich gene pool of bat SARS-related coronaviruses provides new insights into the origin of SARS coronavirus. PLoS pathogens 2017;13:e1006698."/>
  <meta name="citation_reference" content="Drosten C, Kellam P, Memish ZA. Evidence for camel-to-human transmission of MERS coronavirus. The New England journal of medicine 2014;371:1359–60."/>
  <meta name="citation_reference" content="Hulswit RJ, de Haan CA, Bosch BJ. Coronavirus Spike Protein and Tropism Changes. Advances in virus research 2016;96:29–57."/>
  <meta name="citation_reference" content="Wrapp D, Wang N, Corbett KS, et al. Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation. Science (New York, NY) 2020."/>
  <meta name="citation_reference" content="Bertram S, Dijkman R, Habjan M, et al. TMPRSS2 activates the human coronavirus 229E for cathepsin-independent host cell entry and is expressed in viral target cells in the respiratory epithelium. Journal of virology 2013;87:6150–60."/>
  <meta name="citation_reference" content="Watanabe R, Matsuyama S, Shirato K, et al. Entry from the cell surface of severe acute respiratory syndrome coronavirus with cleaved S protein as revealed by pseudotype virus bearing cleaved S protein. Journal of virology 2008;82:11985–91."/>
  <meta name="citation_reference" content="Kumar S, Stecher G, Tamura K. MEGA7: Molecular Evolutionary Genetics Analysis Version 7.0 for Bigger Datasets. Molecular biology and evolution 2016;33:1870–4."/>
  <meta name="citation_reference" content="Waterhouse A, Bertoni M, Bienert S, et al. SWISS-MODEL: homology modelling of protein structures and complexes. Nucleic acids research 2018;46:W296–w303."/>
  <meta name="citation_reference" content="Song W, Gui M, Wang X, Xiang Y. Cryo-EM structure of the SARS coronavirus spike glycoprotein in complex with its host cell receptor ACE2. PLoS pathogens 2018;14:e1007236."/>
  <meta name="citation_reference" content="Kumar S, Stecher G, Tamura K. MEGA7: Molecular Evolutionary Genetics Analysis Version 7.0 for Bigger Datasets. Molecular biology and evolution 2016;33:1870–4."/>
  <meta name="citation_reference" content="Waterhouse A, Bertoni M, Bienert S, et al. SWISS-MODEL: homology modelling of protein structures and complexes. Nucleic acids research 2018;46:W296–w303."/>
  <meta name="citation_reference" content="Song W, Gui M, Wang X, Xiang Y. Cryo-EM structure of the SARS coronavirus spike glycoprotein in complex with its host cell receptor ACE2. PLoS pathogens 2018;14:e1007236."/>
  <meta name="twitter:title" content="Mutations, Recombination and Insertion in the Evolution of 2019-nCoV"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Background The 2019 novel coronavirus (2019-nCoV or SARS-CoV-2) has spread more rapidly than any other betacoronavirus including SARS-CoV and MERS-CoV. However, the mechanisms responsible for infection and molecular evolution of this virus remained unclear. Methods We collected and analyzed 120 genomic sequences of 2019-nCoV including 11 novel genomes from patients in China. Through comprehensive analysis of the available genome sequences of 2019-nCoV strains, we have tracked multiple inheritable SNPs and determined the evolution of 2019-nCoV relative to other coronaviruses. Results Systematic analysis of 120 genomic sequences of 2019-nCoV revealed co-circulation of two genetic subgroups with distinct SNPs markers, which can be used to trace the 2019-nCoV spreading pathways to different regions and countries. Although 2019-nCoV, human and bat SARS-CoV share high homologous in overall genome structures, they evolved into two distinct groups with different receptor entry specificities through potential recombination in the receptor binding regions. In addition, 2019-nCoV has a unique four amino acid insertion between S1 and S2 domains of the spike protein, which created a potential furin or TMPRSS2 cleavage site. Conclusions Our studies provided comprehensive insights into the evolution and spread of the 2019-nCoV. Our results provided evidence suggesting that 2019-nCoV may increase its infectivity through the receptor binding domain recombination and a cleavage site insertion. One Sentence Summary Novel 2019-nCoV sequences revealed the evolution and specificity of betacoronavirus with possible mechanisms of enhanced infectivity."/>
  <meta name="og-title" property="og:title" content="Mutations, Recombination and Insertion in the Evolution of 2019-nCoV"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.29.971101v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Background The 2019 novel coronavirus (2019-nCoV or SARS-CoV-2) has spread more rapidly than any other betacoronavirus including SARS-CoV and MERS-CoV. However, the mechanisms responsible for infection and molecular evolution of this virus remained unclear. Methods We collected and analyzed 120 genomic sequences of 2019-nCoV including 11 novel genomes from patients in China. Through comprehensive analysis of the available genome sequences of 2019-nCoV strains, we have tracked multiple inheritable SNPs and determined the evolution of 2019-nCoV relative to other coronaviruses. Results Systematic analysis of 120 genomic sequences of 2019-nCoV revealed co-circulation of two genetic subgroups with distinct SNPs markers, which can be used to trace the 2019-nCoV spreading pathways to different regions and countries. Although 2019-nCoV, human and bat SARS-CoV share high homologous in overall genome structures, they evolved into two distinct groups with different receptor entry specificities through potential recombination in the receptor binding regions. In addition, 2019-nCoV has a unique four amino acid insertion between S1 and S2 domains of the spike protein, which created a potential furin or TMPRSS2 cleavage site. Conclusions Our studies provided comprehensive insights into the evolution and spread of the 2019-nCoV. Our results provided evidence suggesting that 2019-nCoV may increase its infectivity through the receptor binding domain recombination and a cleavage site insertion. One Sentence Summary Novel 2019-nCoV sequences revealed the evolution and specificity of betacoronavirus with possible mechanisms of enhanced infectivity."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-03-02"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Mutations, Recombination and Insertion in the Evolution of 2019-nCoV | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1177175 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <div id="sec-1" class="subsection">
     <p id="p-3">
      <strong>Background</strong> The 2019 novel coronavirus (2019-nCoV or SARS-CoV-2) has spread more rapidly than any other betacoronavirus including SARS-CoV and MERS-CoV. However, the mechanisms responsible for infection and molecular evolution of this virus remained unclear.
     </p>
    </div>
    <div id="sec-2" class="subsection">
     <p id="p-4">
      <strong>Methods</strong> We collected and analyzed 120 genomic sequences of 2019-nCoV including 11 novel genomes from patients in China. Through comprehensive analysis of the available genome sequences of 2019-nCoV strains, we have tracked multiple inheritable SNPs and determined the evolution of 2019-nCoV relative to other coronaviruses.
     </p>
    </div>
    <div id="sec-3" class="subsection">
     <p id="p-5">
      <strong>Results</strong> Systematic analysis of 120 genomic sequences of 2019-nCoV revealed co-circulation of two genetic subgroups with distinct SNPs markers, which can be used to trace the 2019-nCoV spreading pathways to different regions and countries. Although 2019-nCoV, human and bat SARS-CoV share high homologous in overall genome structures, they evolved into two distinct groups with different receptor entry specificities through potential recombination in the receptor binding regions. In addition, 2019-nCoV has a unique four amino acid insertion between S1 and S2 domains of the spike protein, which created a potential furin or TMPRSS2 cleavage site.
     </p>
    </div>
    <div id="sec-4" class="subsection">
     <p id="p-6">
      <strong>Conclusions</strong> Our studies provided comprehensive insights into the evolution and spread of the 2019-nCoV. Our results provided evidence suggesting that 2019-nCoV may increase its infectivity through the receptor binding domain recombination and a cleavage site insertion.
     </p>
    </div>
    <div id="sec-5" class="subsection">
     <p id="p-7">
      <strong>One Sentence Summary</strong> Novel 2019-nCoV sequences revealed the evolution and specificity of betacoronavirus with possible mechanisms of enhanced infectivity.
     </p>
    </div>
   </div>
   <div class="section" id="sec-6">
    <h2 class="">Introduction</h2>
    <p id="p-8">A new coronavirus, named the Novel Coronavirus 2019 (2019-nCoV or SARS-CoV-2), has emerged and been transmitted to the human population
     <sup>
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>
     </sup>. The outbreak originating from Wuhan, China began in December of 2019 and currently has, as of Feb 23
     <sup>th</sup>, 2020, over 77,000 confirmed cases globally and over 2400 fatalities
     <sup>
      <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>
     </sup>. The current infection has spread outwards from China to 29 other countries or regions including South Korea, Japan, Thailand, Singapore, Vietnam, Taiwan, Nepal, and the United States
     <sup>
      <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>
     </sup>. Series of pneumonia cases from Wuhan linked to this virus, with symptoms such as fever, dry cough, and dyspnea
     <sup>
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>,
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>
     </sup>. Most patients had no significant upper respiratory tract symptoms, suggesting that target cells are located lower in the airway. Consequently, the virus was isolated and sequenced from epithelial cells of the lower respiratory tract
     <sup>
      <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>
     </sup>.
    </p>
    <p id="p-9">Coronaviruses are enveloped, positive sense, single-stranded RNA viruses
     <sup>
      <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>
     </sup> that undergo rapid mutation and recombination
     <sup>
      <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>,
      <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>
     </sup>, of its receptor binding domain (RBD) to adapt to a large pool of species
     <sup>
      <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>–
      <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>
     </sup>. Analysis of the 2019-nCoV shows that it is a novel betacoronavirus belonging to the lineage B (subgenus: 
     <em>sarbecovirus</em>) which also includes Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV)
     <sup>
      <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>
     </sup>. The current 2019-nCoV genome is most phylogenetically similar to Bat/SARS/RaTG13/Yunnan strain, which was first isolated in 2013 in Yunnan, China
     <sup>
      <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>
     </sup>. Recent studies hinted that pangolin-CoV may be a possible intermediate hosts candidate for the 2019-nCoV
     <sup>
      <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>,
      <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>
     </sup>. Recently, this novel CoV has been confirmed to use the same cell entry receptor, ACE2, as SARS-CoV
     <sup>
      <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>
     </sup>. Coronaviruses (CoV) are capable of transmitting among different host species via shifting tropism and variable receptor targeting
     <sup>
      <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>–
      <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>
     </sup>. These characteristics are mediated by changes in the receptor binding domain (RBD) of the spike surface glycoprotein
     <sup>
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>,
      <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>
     </sup>. Similar to other well-known enveloped viruses, CoV initiates infection through fusion of this spike protein with the host cell membrane
     <sup>
      <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>
     </sup>. Their spike protein is comprised of S1 and S2 subunits, which are responsible for host receptor recognition and membrane fusion, respectively
     <sup>
      <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>,
      <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>,
      <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>,
      <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>
     </sup>. 2019-nCoV has recently been confirmed to use the human ACE2 as receptor
     <sup>
      <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>
     </sup>.
    </p>
    <p id="p-10">Although the fatality rate of 2019-nCoV is lower and the symptoms are milder than those of SARS, the transmissibility of 2019-nCoV appears to be higher
     <sup>
      <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>
     </sup>. The molecular mechanisms responsible for such rapid transmission and spread of the 2019-nCoV still remain elusive. Here, we have analyzed 120 2019-nCoV genome sequences including 11 novel strains isolated from patients to determine the viral evolution and divergence. Our studies suggest that 2019-nCoV may increase its infectivity through the receptor binding domain recombination and a furin or TMPRSS2 cleavage site insertion.
    </p>
   </div>
   <div class="section" id="sec-7">
    <h2 class="">Results</h2>
    <p id="p-11">The first cases of human-to-human 2019-nCoV infection occurred mid-December 2019, To date, the total confirmed cases and confirmed death surpassed SARS by a magnitude within a shorter period of time
     <sup>
      <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>
     </sup>. In an attempt to associate the pathologic property of 2019-nCoV with specific virulence factor, we compared the epidemiological information with sequencing data obtained from the WHO
     <sup>
      <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>
     </sup>. We compared the rate of infection for 2019-nCoV to that of the most recent betacoronavirus outbreaks, SARS in November 2002 and MERS in September 2012 (
     <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig 1A</a>.). 2019-nCoV appears to be transmitted much more quickly than SARS and MERS. To date, there are more confirmed 2019-nCoV cases than that of the whole 2002 SARS outbreak. Coronavirus sequences have been published frequently and consistently over the last 18 years, which included SARS strains isolated from different countries during 2002 SARS outbreak, sporadic CoV strains mainly reported in China, MERS strains isolated from middle east countries such as Saudi Arabia and United Arab Emirates (
     <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig 1B</a>). To further understand the evolution of the betacoronaviruses and track the mutations accumulated with 2019-nCoV, we have collected and sequenced 11 full-length 2019-nCoV genome sequences from new patients identified in multiple Chinese cities including Wuhan (
     <a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig 1C</a>). The phylogenetic analysis indicated that 11 new strains of 2019-nCoV clustered together with other 2019-nCoV strains and were more homologous to the bat CoV RaTG13 strain than human SARS, MERS and other CoVs (
     <a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig S1A</a>). At the amino acid levels, they only had a few random substitutions at positions with consensus sequences identical to the corresponding amino acid sequences in human and bat SARS (
     <a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig S1B</a>).
    </p>
    <div id="F5" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="(A) Phylogenetic analysis of 46 full-length translated genomes of 2019-nCoV and 27 beta coronaviruses available in the public domain were aligned and estimated using MEGA7. 11 newly reported nCoV strains were highlighted using stars. (B) Amino acid substitutions within the same set of genomes were identified, and compared to consensus sequence of Human, and Bat-SARS." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-561062230" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;strong&gt;;(A)&lt;;/strong&gt;; Phylogenetic analysis of 46 full-length translated genomes of 2019-nCoV and 27 beta coronaviruses available in the public domain were aligned and estimated using MEGA7. 11 newly reported nCoV strains were highlighted using stars. &lt;;strong&gt;;(B)&lt;;/strong&gt;; Amino acid substitutions within the same set of genomes were identified, and compared to consensus sequence of Human, and Bat-SARS.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Supplemental Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F5.medium.gif" width="440" height="307"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Supplemental Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F5.medium.gif" width="440" height="307"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplemental Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Supplemental Figure 1.</span>
      <p id="p-33" class="first-child">
       <strong>(A)</strong> Phylogenetic analysis of 46 full-length translated genomes of 2019-nCoV and 27 beta coronaviruses available in the public domain were aligned and estimated using MEGA7. 11 newly reported nCoV strains were highlighted using stars. 
       <strong>(B)</strong> Amino acid substitutions within the same set of genomes were identified, and compared to consensus sequence of Human, and Bat-SARS.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F1" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="(A) Data of total confirmed cases during SARS (orange line), MERS (yellow line) and 2019-nCoV (grey line) epidemic. (B) Number of sequences published and available in public domains in Asia and Middle East from Dec 2002 to Feb 2020. All panels are current as of Feb 24th, 2020. (C) Information about samples taken from patients infected with 2019-nCoV. nCoV is the 2019 novel coronavirus. SARS-CoV is severe acute respiratory syndrome coronavirus. MERS-CoV is Middle East respiratory syndrome coronavirus." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-561062230" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;strong&gt;;(A)&lt;;/strong&gt;; Data of total confirmed cases during SARS (orange line), MERS (yellow line) and 2019-nCoV (grey line) epidemic. &lt;;strong&gt;;(B)&lt;;/strong&gt;; Number of sequences published and available in public domains in Asia and Middle East from Dec 2002 to Feb 2020. All panels are current as of Feb 24&lt;;sup&gt;;th&lt;;/sup&gt;;, 2020. &lt;;strong&gt;;(C)&lt;;/strong&gt;; Information about samples taken from patients infected with 2019-nCoV. nCoV is the 2019 novel coronavirus. SARS-CoV is severe acute respiratory syndrome coronavirus. MERS-CoV is Middle East respiratory syndrome coronavirus.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F1.medium.gif" width="382" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F1.medium.gif" width="382" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 1.</span>
      <p id="p-12" class="first-child">
       <strong>(A)</strong> Data of total confirmed cases during SARS (orange line), MERS (yellow line) and 2019-nCoV (grey line) epidemic. 
       <strong>(B)</strong> Number of sequences published and available in public domains in Asia and Middle East from Dec 2002 to Feb 2020. All panels are current as of Feb 24
       <sup>th</sup>, 2020. 
       <strong>(C)</strong> Information about samples taken from patients infected with 2019-nCoV. nCoV is the 2019 novel coronavirus. SARS-CoV is severe acute respiratory syndrome coronavirus. MERS-CoV is Middle East respiratory syndrome coronavirus.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-13">To identify novel inherited mutations, we used SARS-CoV-2 strain (EPI_ISL_402125) as root to construct the phylogenetic tree for all 120 available complete genomes of the novel coronavirus from GISAID (updated February 18
     <sup>th</sup>, 2020). Considering potential sequencing errors at both ends, the genome sequence variations among different strains of 2019-nCoV isolated from patients located in different cities were low considering only several mutations in about 30kb genome per isolate. Based on the nucleotide positions 8517 and 27641, the 2019-nCoV strains can be divided into two major groups (
     <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figs 2A</a> and 
     <a id="xref-fig-6-1" class="xref-fig" href="#F6">S2</a>). All the group 1 strains have thymine at 8517 and cytosine at 27641, which are same as corresponding nucleotides in SARS, whereas the group 2 stains have cytosine at 8517 and thymine at 27641 (
     <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figs 2A</a> and 
     <a id="xref-fig-7-1" class="xref-fig" href="#F7">S3</a>). Epidemiological data G1 and G2 strains revealed that the collection date and location of the earliest G1 strain (EPI_ISL_406801) was January 5
     <sup>th</sup>, 2020 in Wuhan, whereas the earliest G2 strain was isolated in December 24
     <sup>th</sup>, 2019 in Wuhan (
     <a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig 2A</a>). The existence of both genetic groups in the same city indicated co-circulation, but evolved convergently at the early outbreak. Within each group, we also observed additional shared mutations added to multiple strains. Based on these potentially inheritable mutations and the identifying times and locations, we generated a mutation tree map to track individual shared mutations and show the relationships among different isolates (
     <a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig 2A</a> and 
     <a id="xref-fig-6-2" class="xref-fig" href="#F6">S2</a>). For example, five strains identified in Guangdong from Jan. 10-15 all share the same mutation in nucleotide position 28578 on the background of group 1 might be transmitted by the same person. The similar strain may transmit to three patients identified in Japan on Jan. 29-31 with additional mutation at nucleotide position 2397 and to a patient identified in USA on Jan. 22 with additional mutation at nucleotide position 10818. The G10818T is very interesting as it is shared by several independent strains in both group 1 and group 2, which will lead to L3606F amino acid substitution within the orf1ab polyprotein (
     <a id="xref-fig-7-2" class="xref-fig" href="#F7">Fig S3</a>). It is not clear if the common mutation both in group 1 and group 2 strains at 10818 site has any growth advantage but pangolin and bat CoV have L3606V substitution at the same position (
     <a id="xref-fig-7-3" class="xref-fig" href="#F7">Fig S3</a>). When the group 1 and group 2 strains were placed onto geographic map (
     <a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig 2B</a>), it seems that both groups have been transmitted to most countries and regions with reported 2019-nCoV cases with few exceptions, suggesting these two groups are rapidly transmissible.
    </p>
    <div id="F6" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Co-circulation of two SARS-CoV-2 genotypes from Wuhan to other regions. The genomes of groups G1 and G2 are grouped individually by their collection locations. Five sampled locations are highlighted in different colours. SNPs are shown in small circles." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-561062230" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Co-circulation of two SARS-CoV-2 genotypes from Wuhan to other regions. The genomes of groups G1 and G2 are grouped individually by their collection locations. Five sampled locations are highlighted in different colours. SNPs are shown in small circles.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Supplemental Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F6.medium.gif" width="359" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Supplemental Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F6.medium.gif" width="359" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplemental Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Supplemental Figure 2.</span>
      <p id="p-34" class="first-child">Co-circulation of two SARS-CoV-2 genotypes from Wuhan to other regions. The genomes of groups G1 and G2 are grouped individually by their collection locations. Five sampled locations are highlighted in different colours. SNPs are shown in small circles.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F7" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F7.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Major SNPs and amino acid substitutions among SARS-CoV-2 and similar coronaviruses isolated from bat and pangolin. (A) Two SNP markers in SARS-CoV-2 and similar coronaviruses isolated from bat and pangolin. (B) Comparison of two other minor SNPs. (C) Amino acid substitutions caused by non-synonymous SNPs (substitutions that appeared only once among all the sample genomes were removed). Strain EPI_ISL_402125 is used as reference. The numbering positions are listed at the bottom.Encoded proteins are highlighted as coloured bars at the top. Amino acids are shown as single letter or dot when it is the same as the reference strain." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-561062230" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Major SNPs and amino acid substitutions among SARS-CoV-2 and similar coronaviruses isolated from bat and pangolin. &lt;;strong&gt;;(A)&lt;;/strong&gt;; Two SNP markers in SARS-CoV-2 and similar coronaviruses isolated from bat and pangolin. &lt;;strong&gt;;(B)&lt;;/strong&gt;; Comparison of two other minor SNPs. &lt;;strong&gt;;(C)&lt;;/strong&gt;; Amino acid substitutions caused by non-synonymous SNPs (substitutions that appeared only once among all the sample genomes were removed). Strain EPI_ISL_402125 is used as reference. The numbering positions are listed at the bottom.Encoded proteins are highlighted as coloured bars at the top. Amino acids are shown as single letter or dot when it is the same as the reference strain.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Supplemental figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F7.medium.gif" width="440" height="392"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Supplemental figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F7.medium.gif" width="440" height="392"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplemental figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Supplemental figure 3.</span>
      <p id="p-35" class="first-child">Major SNPs and amino acid substitutions among SARS-CoV-2 and similar coronaviruses isolated from bat and pangolin. 
       <strong>(A)</strong> Two SNP markers in SARS-CoV-2 and similar coronaviruses isolated from bat and pangolin. 
       <strong>(B)</strong> Comparison of two other minor SNPs. 
       <strong>(C)</strong> Amino acid substitutions caused by non-synonymous SNPs (substitutions that appeared only once among all the sample genomes were removed). Strain EPI_ISL_402125 is used as reference. The numbering positions are listed at the bottom.Encoded proteins are highlighted as coloured bars at the top. Amino acids are shown as single letter or dot when it is the same as the reference strain.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F2" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="(A) Sequence alignment of 120 full-length genomes of 2019-nCoV including 11 newly reported genomes (highlighted by stars), ~30,000 base pairs in length, nucleotide substitutions to an early sequenced strain EPI_ISL_402125 as the root of phylogenetic tree. Two sub-groups were coloured in blue and red. The trailing dots on the right represent the SNPs in the viral genome. The first group (G1) possesses 8517T and 27641C. The second group (G2) possesses 8517C and 27641T. All genomes were clustered using ML method. Inherited SNPs that were share between multiple strains were highlighted. The divergent evolution of G1 and G2 was identified by tracking individual shared mutations. The horizontal axis represents the difference between reference sequence and the node strain. (B) Geographical of the spread of different genetic groups of 2019-nCoV. The red and blue lines represent predicted transmission pathways with G1 and G2 strains, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-561062230" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;strong&gt;;(A)&lt;;/strong&gt;; Sequence alignment of 120 full-length genomes of 2019-nCoV including 11 newly reported genomes (highlighted by stars), ~30,000 base pairs in length, nucleotide substitutions to an early sequenced strain EPI_ISL_402125 as the root of phylogenetic tree. Two sub-groups were coloured in blue and red. The trailing dots on the right represent the SNPs in the viral genome. The first group (G1) possesses 8517T and 27641C. The second group (G2) possesses 8517C and 27641T. All genomes were clustered using ML method. Inherited SNPs that were share between multiple strains were highlighted. The divergent evolution of G1 and G2 was identified by tracking individual shared mutations. The horizontal axis represents the difference between reference sequence and the node strain. &lt;;strong&gt;;(B)&lt;;/strong&gt;; Geographical of the spread of different genetic groups of 2019-nCoV. The red and blue lines represent predicted transmission pathways with G1 and G2 strains, respectively.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F2.medium.gif" width="406" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F2.medium.gif" width="406" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 2.</span>
      <p id="p-14" class="first-child">
       <strong>(A)</strong> Sequence alignment of 120 full-length genomes of 2019-nCoV including 11 newly reported genomes (highlighted by stars), ~30,000 base pairs in length, nucleotide substitutions to an early sequenced strain EPI_ISL_402125 as the root of phylogenetic tree. Two sub-groups were coloured in blue and red. The trailing dots on the right represent the SNPs in the viral genome. The first group (G1) possesses 8517T and 27641C. The second group (G2) possesses 8517C and 27641T. All genomes were clustered using ML method. Inherited SNPs that were share between multiple strains were highlighted. The divergent evolution of G1 and G2 was identified by tracking individual shared mutations. The horizontal axis represents the difference between reference sequence and the node strain. 
       <strong>(B)</strong> Geographical of the spread of different genetic groups of 2019-nCoV. The red and blue lines represent predicted transmission pathways with G1 and G2 strains, respectively.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-15">The most closely related strain, betacoronavirus RaTG13, was isolated from 
     <em>Rhinolophus affinis</em>
     <sup>
      <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>
     </sup> (
     <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig 3A</a>). We performed additional phylogenetic analyses using nucleotide sequences for specific viral proteins such asorf1a, spike, matrix and nucleocapsid (
     <a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3B</a>), and found the same close relationship of the RaTG13 strain and other bat SARS-like CoV strains. We further estimated that the divergences of most proteins between 2019-nCoV and RaTG13 happened between 2005 and 2012 whereas those between human SARS and bat SARS-like CoV happened between 1990-2002 (
     <a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig 3B</a>). When the full-length spike protein sequences were compared, 2019-nCoV shares 39% sequence homology to human and bat SARS as compared to MERS or other CoV at 29%. Interestingly, we found that 2019-nCoV and pangolin-CoV share near identical amino acid sequence in the RBD (aa 315-550 region) of the spike protein, but not for RaTG13 (
     <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figs 3A</a> and 
     <a id="xref-fig-3-5" class="xref-fig" href="#F3">3D</a>). To confirm this finding, we have compared the pangolin-CoV reported by Liu 
     <em>et al</em> with previously isolated but unpublished pangolin-CoV sequences (
     <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure S4</a>). Based on the alignment and phylogenetic analysis, we found that the consensus sequence of 2019-nCoV has the highest identity to BetaCoV/pangolin/Guangdong/P2S/2019 (EPI_ISL_410544), whereas additional mutations and indels were found in the pangolin-CoV strains isolated in the Guangxi province. Both BetaCoV/pangolin/Guangdong/P2S/2019 and Pangolin-CoV were isolated from Guangdong province of China in 2019. Next we examined the phylogenetic relationship of the consensus spike protein sequence of 2019-nCoV against 25 representative CoV strains including Hu-CoV, SARS and MERS and five new pangolin-CoV strains using a ML method (
     <a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure S4</a>). The results demonstrate that the Spike protein of 2019-nCoV is likely derived from Pangolin-CoV but not RaTG13, all of which could be in the same lineage as BAT_SARS-CoV/bat-SL-CoVZC542. The fact that while 2019-nCoV is most homologous to RaTG13 in overall genome structure the RBD of the spike protein is most homologous to Pangolin-CoV, suggests the possible recombination between RaTG13 like and Pangolin-CoV like strains happens in the evolution of 2019-nCoV. We have manually examined all amino acid mutations in the genome. Due to the presence of the unsequenced regions in the Pangolin-CoV sequence, we did not include any positions flanking these regions. We found that when comparing Pangolin-CoV and 2019-nCoV, in addition to the RBD, regions in nsp14 and 15 also shares consecutive sequences (
     <a id="xref-fig-3-6" class="xref-fig" href="#F3">Fig 3D</a>).
    </p>
    <div id="F8" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F8.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Phylogenetic analysis of the genome sequences from represented novel pangolin strains using the Maximum Likelihood method based on the JTT matrix-based model on MEGA v7.0." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-561062230" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Phylogenetic analysis of the genome sequences from represented novel pangolin strains using the Maximum Likelihood method based on the JTT matrix-based model on MEGA v7.0.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Supplemental figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F8.medium.gif" width="440" height="325"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Supplemental figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F8.medium.gif" width="440" height="325"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplemental figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Supplemental figure 4.</span>
      <p id="p-36" class="first-child">Phylogenetic analysis of the genome sequences from represented novel pangolin strains using the Maximum Likelihood method based on the JTT matrix-based model on MEGA v7.0.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F3" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="(A) Amino acid homology graph of Pangolin-CoV (blue line) and RaTG13/2013 (green line) against 2019-nCoV. Conserved receptor binding domain (RBD) highlighted in yellow. (B) Schematic phylogenetic trees based on the molecular clock analysis for the 2019-nCoV with the orf1a, spike, matrix and nucleocapsid genes. The phylogenetic clades that contain the 2019-nCoV, the SARS-like bat CoVs and the SARS viruses in human and the other hosts are colored by red, violet and light blue lines, respectively. The SARS-like bat CoV Yunnan-RaTG13 strain was highlight by the green line. The Divergence time in the molecular clock analysis between 2019-nCoV and RaTG13 were labelled. (C) Phylogenetic analysis of the aligned spike protein sequences from represented betacoronavirus strains using the ML method based on the JTT matrix-based model on MEGA v7.0. Alignment was performed using MUSCLE. 2019-nCoV, RaTG13 and Pangolin-CoV were highlighted with arrow. (D) Amino acid substitutions of Pangolin-CoV, 2019-nCoV and RaTG13. ORFa/b and spike proteins encoded by Pangolin-CoV and Bat/Yunnan/RaTG13 were aligned against 2019-nCoVs using MUSCLE. Sites flanking an unidentified amino acid were excluded." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-561062230" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;strong&gt;;(A)&lt;;/strong&gt;; Amino acid homology graph of Pangolin-CoV (blue line) and RaTG13/2013 (green line) against 2019-nCoV. Conserved receptor binding domain (RBD) highlighted in yellow. &lt;;strong&gt;;(B)&lt;;/strong&gt;; Schematic phylogenetic trees based on the molecular clock analysis for the 2019-nCoV with the orf1a, spike, matrix and nucleocapsid genes. The phylogenetic clades that contain the 2019-nCoV, the SARS-like bat CoVs and the SARS viruses in human and the other hosts are colored by red, violet and light blue lines, respectively. The SARS-like bat CoV Yunnan-RaTG13 strain was highlight by the green line. The Divergence time in the molecular clock analysis between 2019-nCoV and RaTG13 were labelled. &lt;;strong&gt;;(C)&lt;;/strong&gt;; Phylogenetic analysis of the aligned spike protein sequences from represented betacoronavirus strains using the ML method based on the JTT matrix-based model on MEGA v7.0. Alignment was performed using MUSCLE. 2019-nCoV, RaTG13 and Pangolin-CoV were highlighted with arrow. &lt;;strong&gt;;(D)&lt;;/strong&gt;; Amino acid substitutions of Pangolin-CoV, 2019-nCoV and RaTG13. ORFa/b and spike proteins encoded by Pangolin-CoV and Bat/Yunnan/RaTG13 were aligned against 2019-nCoVs using MUSCLE. Sites flanking an unidentified amino acid were excluded.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F3.medium.gif" width="440" height="303"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F3.medium.gif" width="440" height="303"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 3.</span>
      <p id="p-16" class="first-child">
       <strong>(A)</strong> Amino acid homology graph of Pangolin-CoV (blue line) and RaTG13/2013 (green line) against 2019-nCoV. Conserved receptor binding domain (RBD) highlighted in yellow. 
       <strong>(B)</strong> Schematic phylogenetic trees based on the molecular clock analysis for the 2019-nCoV with the orf1a, spike, matrix and nucleocapsid genes. The phylogenetic clades that contain the 2019-nCoV, the SARS-like bat CoVs and the SARS viruses in human and the other hosts are colored by red, violet and light blue lines, respectively. The SARS-like bat CoV Yunnan-RaTG13 strain was highlight by the green line. The Divergence time in the molecular clock analysis between 2019-nCoV and RaTG13 were labelled. 
       <strong>(C)</strong> Phylogenetic analysis of the aligned spike protein sequences from represented betacoronavirus strains using the ML method based on the JTT matrix-based model on MEGA v7.0. Alignment was performed using MUSCLE. 2019-nCoV, RaTG13 and Pangolin-CoV were highlighted with arrow. 
       <strong>(D)</strong> Amino acid substitutions of Pangolin-CoV, 2019-nCoV and RaTG13. ORFa/b and spike proteins encoded by Pangolin-CoV and Bat/Yunnan/RaTG13 were aligned against 2019-nCoVs using MUSCLE. Sites flanking an unidentified amino acid were excluded.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-17">To further evaluate the 2019-nCoV relationship with other SARS CoVs, we analyzed the RBM of the 2019-nCoV and different human/bat SARS viruses and observed that they can be clearly divided into two distinct clades (
     <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig 4A</a>). The clade I viruses include 2019-nCoV, pangolin CoVs, 12 of bat SARS (bat SARS CoV I) such as RaTG13, and human SARS (
     <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table S1</a>). The clade II viruses include 49 of bat SARS viruses (bat SARS CoV II) such as ZXC21 and ZC45, which share about 90% overall nucleotide and amino acid identity as the 2019-nCoV (
     <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table S2</a>). The major difference between these two clades is that the RBM of the clade II viruses has regions with 5 and 13-14 amino acids shorter than that of the clade I viruses (
     <a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4B</a>). Previous structural analysis have demonstrated that the 13-14 amino acid region of the SARS RBM forms a distinct loop structure, which is stabilized with a disulfide bond between two cysteine residues
     <sup>
      <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">9</a>,
      <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">15</a>,
      <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">16</a>,
      <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>
     </sup>. Although the amino acid sequences of 2019-nCoV within this loop region are very different from those of human SARS, the two cysteine residues are conserved (
     <a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4B</a>). Interestingly, all the CoV viruses known to use ACE2 as entry receptor are within the clade I, including the 2019-nCoV, which can also infect cells through ACE2 receptor-based on recent studies
     <sup>
      <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>
     </sup> (
     <a id="xref-fig-9-1" class="xref-fig" href="#F9">Fig. S5</a>). On the other hand, there is no report on using ACE2 as entry receptor for the clade II viruses, despite their overall genome sequence homology with 2019-nCoV. Therefore, our studies not only emphasize the critical role of the RBM in determining the specificity of entry receptors but also raise the question on how homologous strains of betacoronavirus switch tropism through mutations such as indel or recombination in the RBM.
    </p>
    <div id="F9" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F9.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="(A) Phylogenetic analyses of amino acid sequences of 2019-nCoV, human/civet SARS CoV and Bat SARS-like CoV from different bat species. The evolutionary history was inferred by using the Maximum Likelihood method based on the JTT matrix-based model conducting in MEGA7 software. The scale bar represents the number of substitutions per site. (B) Multiple alignment of the amino acid sequences of the receptor-binding motifs of the spike proteins of 2019-nCoV and SARS CoV and the corresponding sequences of bat SARS-like CoVs in different Rhinolophus species. Highly conserved residues are highlighted in yellow. Amino acid deletions in some bat SARS-like CoVs are labeled with blue. The five critical residues for receptor binding in 2019-nCoV at positions 455, 486, 493, 494 and 501(corresponding to 442,472,479,487,491 human/civet SARS CoV respectively) are highlighted in pink." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-561062230" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;strong&gt;;(A)&lt;;/strong&gt;; Phylogenetic analyses of amino acid sequences of 2019-nCoV, human/civet SARS CoV and Bat SARS-like CoV from different bat species. The evolutionary history was inferred by using the Maximum Likelihood method based on the JTT matrix-based model conducting in MEGA7 software. The scale bar represents the number of substitutions per site. &lt;;strong&gt;;(B)&lt;;/strong&gt;; Multiple alignment of the amino acid sequences of the receptor-binding motifs of the spike proteins of 2019-nCoV and SARS CoV and the corresponding sequences of bat SARS-like CoVs in different &lt;;em&gt;;Rhinolophus&lt;;/em&gt;; species. Highly conserved residues are highlighted in yellow. Amino acid deletions in some bat SARS-like CoVs are labeled with blue. The five critical residues for receptor binding in 2019-nCoV at positions 455, 486, 493, 494 and 501(corresponding to 442,472,479,487,491 human/civet SARS CoV respectively) are highlighted in pink.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Supplemental Fig 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F9.medium.gif" width="393" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Supplemental Fig 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F9.medium.gif" width="393" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplemental Fig 5." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Supplemental Fig 5.</span>
      <p id="p-37" class="first-child">
       <strong>(A)</strong> Phylogenetic analyses of amino acid sequences of 2019-nCoV, human/civet SARS CoV and Bat SARS-like CoV from different bat species. The evolutionary history was inferred by using the Maximum Likelihood method based on the JTT matrix-based model conducting in MEGA7 software. The scale bar represents the number of substitutions per site. 
       <strong>(B)</strong> Multiple alignment of the amino acid sequences of the receptor-binding motifs of the spike proteins of 2019-nCoV and SARS CoV and the corresponding sequences of bat SARS-like CoVs in different 
       <em>Rhinolophus</em> species. Highly conserved residues are highlighted in yellow. Amino acid deletions in some bat SARS-like CoVs are labeled with blue. The five critical residues for receptor binding in 2019-nCoV at positions 455, 486, 493, 494 and 501(corresponding to 442,472,479,487,491 human/civet SARS CoV respectively) are highlighted in pink.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="T1" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1179707/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup">
         <a href="/highwire/markup/1179707/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
        <li class="download-ppt last">
         <a href="/highwire/powerpoint/1179707" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Supplemental Table 1:</span>
      <p id="p-39" class="first-child">Sequences of Bat SARS-like CoV Clade I collected from NCBI</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="T2" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1179706/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup last">
         <a href="/highwire/markup/1179706/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Supplemental Table 2:</span>
      <p id="p-40" class="first-child">Sequences of Bat SARS-like CoV clade II collected from NCBI</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F4" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="(A) Molecular Phylogenetic analysis of receptor binding motifs of 2019-nCoV, human/civet SARS virus and bat SARS-like virus. The evolutionary history was inferred by using the Maximum Likelihood method based on the JTT matrix-based model conducting in MEGA7 software. The scale bar represents the number of substitutions per site. (B) Specific amino acid variations in the RBM of the spike proteins of the 2019-nCoV and Bat SARS-CoV sub-lineages. (C) Coronavirus Spike proteins and their potential furin or TMPRSS2 cleavage sites between S1 and S2 domain. Insertion mutation at amino acid position 681 of 2019-nCoV spike protein and its sequence comparison with human and bat SARS-like CoVs. The multiple alignment was conducted by using the Clustal in MEGA7 software with default parameters. Furin consensus target sites were labeled with red; residues immediately downstream of the cleavage site are italicized and highlighted in green." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-561062230" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;strong&gt;;(A)&lt;;/strong&gt;; Molecular Phylogenetic analysis of receptor binding motifs of 2019-nCoV, human/civet SARS virus and bat SARS-like virus. The evolutionary history was inferred by using the Maximum Likelihood method based on the JTT matrix-based model conducting in MEGA7 software. The scale bar represents the number of substitutions per site. &lt;;strong&gt;;(B)&lt;;/strong&gt;; Specific amino acid variations in the RBM of the spike proteins of the 2019-nCoV and Bat SARS-CoV sub-lineages. &lt;;strong&gt;;(C)&lt;;/strong&gt;; Coronavirus Spike proteins and their potential furin or TMPRSS2 cleavage sites between S1 and S2 domain. Insertion mutation at amino acid position 681 of 2019-nCoV spike protein and its sequence comparison with human and bat SARS-like CoVs. The multiple alignment was conducted by using the Clustal in MEGA7 software with default parameters. Furin consensus target sites were labeled with red; residues immediately downstream of the cleavage site are italicized and highlighted in green.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F4.medium.gif" width="440" height="325"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F4.medium.gif" width="440" height="325"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 4.</span>
      <p id="p-18" class="first-child">
       <strong>(A)</strong> Molecular Phylogenetic analysis of receptor binding motifs of 2019-nCoV, human/civet SARS virus and bat SARS-like virus. The evolutionary history was inferred by using the Maximum Likelihood method based on the JTT matrix-based model conducting in MEGA7 software. The scale bar represents the number of substitutions per site. 
       <strong>(B)</strong> Specific amino acid variations in the RBM of the spike proteins of the 2019-nCoV and Bat SARS-CoV sub-lineages. 
       <strong>(C)</strong> Coronavirus Spike proteins and their potential furin or TMPRSS2 cleavage sites between S1 and S2 domain. Insertion mutation at amino acid position 681 of 2019-nCoV spike protein and its sequence comparison with human and bat SARS-like CoVs. The multiple alignment was conducted by using the Clustal in MEGA7 software with default parameters. Furin consensus target sites were labeled with red; residues immediately downstream of the cleavage site are italicized and highlighted in green.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-19">Our study also revealed that the 2019-nCoV has a unique four amino acid insertion (681-PRRA-684) within the spike protein, or within nucleotide position 23,619-23,632 (
     <a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig 4C</a>). Interestingly, such an insertion (PRRA) within the spike protein of 2019-nCoV creates a potential cleavage site RRAR for the mammalian furin protein, which the consensus sequence is RXXR. The potential furin or TMPRSS2 cleavage site is inserted at the boundary between the S1 and S2 domains of the spike protein, and the first proline residue of the PRRA insertion may introduce a beta turn into the polypeptide chain. To understand the uniqueness of this insertion, we performed sequence comparison using represented SARS-CoV strains from human, civet and bat. Our study showed that this insertion is unique to the 2019-nCoV (
     <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4C</a> and 
     <a id="xref-fig-10-1" class="xref-fig" href="#F10">S6</a>). When compared to other CoV family members, we found that similar insertion has been identified and are located in the structural boundary between the S1 and S2 domains of the spike protein (
     <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4C</a>).
    </p>
    <div id="F10" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F10.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Unique insertion of a potential furin or TMPRSS2 cleavage site between S1 and S2 domain of the 2019-nCoV spike protein. (A) Insertion mutation at amino acid position 681 of 2019-nCoV spike protein and its sequence comparison with human and bat SARS-like CoVs. The multiple alignment was conducted by using the Clustal in MEGA7 software with default parameters." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-561062230" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Unique insertion of a potential furin or TMPRSS2 cleavage site between S1 and S2 domain of the 2019-nCoV spike protein. (A) Insertion mutation at amino acid position 681 of 2019-nCoV spike protein and its sequence comparison with human and bat SARS-like CoVs. The multiple alignment was conducted by using the Clustal in MEGA7 software with default parameters.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Supplemental Fig 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F10.medium.gif" width="440" height="390"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Supplemental Fig 6." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F10.medium.gif" width="440" height="390"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplemental Fig 6." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.971101/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Supplemental Fig 6.</span>
      <p id="p-38" class="first-child">Unique insertion of a potential furin or TMPRSS2 cleavage site between S1 and S2 domain of the 2019-nCoV spike protein. (A) Insertion mutation at amino acid position 681 of 2019-nCoV spike protein and its sequence comparison with human and bat SARS-like CoVs. The multiple alignment was conducted by using the Clustal in MEGA7 software with default parameters.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
   </div>
   <div class="section" id="sec-8">
    <h2 class="">Discussion</h2>
    <p id="p-20">The 2019-nCoV is still spreading rapidly from Wuhan to different cities in China and other countries, at a magnitude faster than SARS and MERS
     <sup>
      <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">4</a>
     </sup>. We have systematically analyzed and tracked the genome mutations among 120 different strains of 2019-nCoV. Although most substitutions are 
     <em>de novo</em> mutations, we have identified multiple SNPs shared by different strains of 2019-nCoV. Due to the low mutation rate and large genome capital, we hypothesis that these shared mutations are inherited SNPs from large group of populations, likely transmitted from people to people. Based on this assumption, we have generated the SNP tree to track mutation and spread of 2019-nCoV. Interestingly, the 2019-nCoV stains we have analyzed fall into two distinct groups with different nucleotide polymorphisms at positions 8782 and 28144. Although it remains to be determined if these two distinct groups of 2019-nCoV were evolved before or after transmission from animal to human, both groups were first detected in wuhan and then spread to different regions in China and multiple countries. By combining additional inheritable mutations in subsequent generations of 2019-nCoV with the times and locations of the patient sample collections, we can trace possible viral transmission pathways.
    </p>
    <p id="p-21">2019-nCoV as a member of the betacoronaviruses, shares similar genome structures as bat SARS, human SARS, MERS with nucleotide identity over 88%, 79%, about and 50%,respectively
     <sup>
      <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>
     </sup>. Its closest relative is the Beta CoV RaTG13, isolated from bat in Yunnan province, China, in 2013, which shares more than 96% identical nucleotides throughout the genome of over 30 kb
     <sup>
      <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">15</a>,
      <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>
     </sup>. Our evolution clock analysis estimated that 2019-nCoV diverged from RaTG13 and human SARS-CoV at about 12 and 30 years ago, respectively. Beside point mutations, there is also a potential evidence of recombination as a mechanism for the evolution of 2019-nCoV. We found that 2019-nCoV shares high identity with RaTG13 throughout the genome except for the RBD of spike protein, which is closer to Pangolin-CoV isolated in Guangdong
     <sup>
      <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>
     </sup>. Based on these findings we hypothesized that recombination in the RBD of the spike protein may happen between RaTG13 and Pangolin-CoV like strains during the evolution of 2019-nCoV.
    </p>
    <p id="p-22">Bat is believed to be the original host for the 2019-nCoV but its intermediate host before transmitting to human is not known whereas civets and camels are widely considered as the intermediate hosts for human SARS and MERS, respectively
     <sup>
      <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a>,
      <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>–
      <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>
     </sup>. Human SARS viruses are known to use the ACE2 while MERS use DPP4 as their receptors to infect host cells
     <sup>
      <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">16</a>,
      <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>
     </sup>. Recent studies have indicated ACE2 as the entry receptor for 2019-nCoV
     <sup>
      <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>
     </sup> although other host cell factors such as TMPRSS2 are likely involved
     <sup>
      <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>
     </sup>. When we used the receptor binding motifs to conduct phylogenic analysis, we could separate 2019-nCoV, human and bat SARS into two distinct clades. Interestingly, all the viruses known to use ACE2 as the entry receptor are within the clade I whereas all the bat SARS viruses which do not use ACE2 entry receptors belong to clade II. Therefore, we predict that the clade I CoV viruses including the 2019-nCoV can while clade II CoV cannot infect host cells through ACE2. Based on the available genome sequences, there are many more clade II bat CoV (over 49 stains) than clade I bat CoV (about 12 strains). Some of the clade II bat CoV such as ZXC21 and ZC45 are more homologous to the 2019-nCoV than the clade I bat CoV and human SARS. It would be interesting to find out if homologous betacoronavirus can switch tropism through recombination in the RBM.
    </p>
    <p id="p-23">We have also identified a unique four amino acid (PRRA) insertion between S1 and S2 domains of the spike protein, which may function as a furin or TMPRSS2 cleavage site. It has been shown that CoV may undergo a protease cleavage to trigger the viruscell membrane fusion
     <sup>
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>,
      <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">14</a>
     </sup>. This flexibility in priming and triggering the fusion machinery greatly modulates the viral pathogenicity and tropism of different coronaviruses
     <sup>
      <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">18</a>,
      <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a>
     </sup>. However, such protease cleavage has not been detected in SARS-CoV
     <sup>
      <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">19</a>
     </sup>. Introducing a cleavage site into SARS-CoV resulted in spike protein cleavage and potentiated the membrane fusion activity
     <sup>
      <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>
     </sup>. In addition, introducing a cleaved spike protein into a SARS-CoV pseudotype virus allowed it to directly enter host cells
     <sup>
      <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>
     </sup>. Based on previous sequencing and structural analysis, the 2019-nCoV spike protein were predicted to interact with the ACE2 receptor to trigger the fusion with the host cell membrane and initiate infection
     <sup>
      <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a>
     </sup>. Therefore, mutations or indels altering the S1-S2 subunits should significantly impact viral infection. We hypothesize that the PRRA insertion may render the spike protein to cleavage process, which triggers the viral fusion event.
    </p>
    <p id="p-24">Although the exact mechanisms responsible for such a high infection rates remains to be further investigated, our data on both recombination in the RBD and the unique furin or TMPRSS2 cleavage site insertion between the S1 and S2 domains of the spike protein may explain why the transmission of this newly emerged virus is significantly increased compared to the related beta-coronaviruses, including SARS and MERS. Further tracking the genome mutations with additional strains of 2019-nCoV isolated from patients in different locations at different time points will provide insights to understand the molecular evolution of this rapid spreading viruses. More importantly, comparison of protein sequence and structural differences between 2019-nCoV and other beta coronaviruses will provide insights to rapidly develop novel strategies to treat or prevent diseases associated with the novel emerging infectious viruses.</p>
   </div>
   <div class="section" id="sec-9">
    <h2 class="">Supplementary Materials for</h2>
    <div class="section" id="sec-10">
     <h3>Experimental Procedures</h3>
     <div class="section" id="sec-11">
      <h4>Patients and samples</h4>
      <p id="p-26">The whole-genome sequences of 2019-nCoV from 11 samples were generated by a combination of Sanger, Illumina, and Oxford nanopore sequencing. First, viral RNAs were extracted directly from clinical samples with the QIAamp Viral RNA Mini Kit, and then used to synthesize cDNA with the SuperScript III Reverse Transcriptase (ThermoFisher, Waltham, MA, USA) and N6 random primers, followed by second-strand synthesis with DNA Polymerase I, Large (Klenow) Fragment (ThermoFisher). Viral cDNA libraries were prepared with use of the Nextera XT Library Prep Kit (Illumina, San Diego, CA, USA), then purified with Agencourt AMPure XP beads (Beckman Coulter, Brea, CA, USA), followed by quantification with an Invitrogen Qubit 2.0 Fluorometer. The resulting DNA libraries were sequenced on either the MiSeq or iSeq platforms (Illumina) using a 300-cycle reagent kit. About 1 ·2–5 GB of data were obtained for each sample.</p>
     </div>
     <div class="section" id="sec-12">
      <h4>Genome and amino acid comparisons</h4>
      <p id="p-27">The start and stop codons of each predicted gene were manually checked to ensure the completeness. We tried to infer the possible evolution pathway and track the cotransmission of 2019-nCoV. To avoid random mutations that may mislead the inference, we removed the substitution that appeared only once among all the sample genomes. Nucleotide alignment was performed using MUSCLE (version 2.2.25+), and base-pair comparison was performed using Basebybase (v1.0). Nucleotide substitutions were determine using a consensus sequence of 2019-nCoV. The comparison of amino acids within the 2019-nCoV were carried out. Additionally, for each ORF, the amino acid sequences of 2019-nCoV, Human SARS-CoV and Bat SARS-CoV were compared using the MUSCLE.</p>
     </div>
     <div class="section" id="sec-13">
      <h4>Phylogenetic analysis</h4>
      <p id="p-28">Given the whole protein sequences corresponding for the 2019-nCoV, 2019-nCoVs were obtained from ViPR – Coronavirus and GISAID database on 18
       <sup>th</sup> Feb, 2020. We performed a representative search with the related sequences of the betacoronaviridae virus using the BLAST (version 2.2.25+). Based on the blast results, we excluded the strains without the sample date, location, host and species information. Then the screened sequences were aligned by the MUSCLE. The sequences were removed fot that with no less than 28000bp or more than 100 unsolved nucleotides as ‘N’. At last, 109 public genomes of 2019-nCoV were kept. The phylogenetic tree for the whole genome was constructed using the Molecular Evolutionary Genetics Analysis (MEGA) (version 7.0)
       <sup>
        <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">32</a>
       </sup> by the maximum likelihood method under the General Time Reversible (GTR) nucleotide substitution model, while the phylogenetic trees for the individual genes were constructed by the FigTree software v1.4.3.
      </p>
      <p id="p-29">For the spike protein, given the whole genomes of 2019 nCoVs (118 strains) and other SARS-like viruses isolated from Bat(1 strains) and Pangolin (6 strains), the genes was predicted by GeneMarkS (version 3.36). The predicted genes were then against the proteome of SARS-CoV by BLAST (version 2.2.25+). After that, Spike surface glycoprotein (S) were picked, and RBD domain in Spike was also discriminated. For the protein sequence of Spike genes, we got, multiple sequence alignment was made by using the FFT-NS-2 algorithm in MAFFT v7.407 program, and three regions (50~246aa; RBD domain; 600~796aa) were picked up. Finally, three phylogenetic trees were constructed using the Molecular Evolutionary Genetics Analysis (MEGA) software (version 10.0.5) by the maximum likelihood method with the bootstrap tests (100 replicates), and Fasttree software (version 1.43) were used to visualize these trees.</p>
     </div>
     <div class="section" id="sec-14">
      <h4>SNP recognition and comparison</h4>
      <p id="p-30">The SNPs of each sequence were defined as the sites variant from the reference sequence. The ancestral sequence of the phylogenetic tree was used as the reference sequence, which was estimated by python package TimeTree using Jukes-Cantor model and default settings. Site with the unsolved nucleotide N and gap was ignored. The site number of the SNP was decided by sequentially combined the CDS sequences in the whole genome. The phylogenetic tree and the matching SNP for the tips was plotted by ggtree. The geo-distribution and genotype of strains with SNP sites in Wuhan and other areas were plotted by ggplot2. The SNP sites of 2019-nCoV were compared with SARS-like virus in one bat and eight pangolin genome sequences. The pangolin sequences EPI_ISL_410543 and EPI_ISL_410544 were removed for too many unsolved nucleotides. The adjacent nucleotide of the two SNP sites 8517 and 27641 were extracted from the alignment. The translation-reading frame was inferred by using the GenBank annotations for the EPI_ISL_402125 strain.</p>
     </div>
     <div class="section" id="sec-15">
      <h4>Tracking individual shared mutations</h4>
      <p id="p-31">The divergent evolution of G1 and G2 was tracked with the help of shared mutations as following steps. First, we removed the substitution that appeared only once among all the genomes to avoid random mutations or sequencing errors that might mislead the inference. Second, genomes in G1 and G2 were grouped by the number of varied nucleotides compared with the reference sequence of G1 (EPI_ISL_402125) or G2 (EPI_ISL_406801). In total, we divided all the genomes in G1 and G2 separately into three groups with 1, 2 and 3 varied nucleotides. Third, the strains in groups with 1 varied nucleotides were plotted in the tree with branch length of 1, then, strains with 2 varied nucleotides were plotted in the tree with branch length of 2, it should be noted that the strains with substitution site contained the site in group 1 was plotted behind the branch with responding site. Similar ways were used to group 3.</p>
     </div>
     <div class="section" id="sec-16">
      <h4>Structure prediction and analysis</h4>
      <p id="p-32">Based on the computer-guided homology modeling method, the structural models were constructed by SWISS-MODEL online server
       <sup>
        <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">33</a>
       </sup>. The model of nCoV 2019 Spike protein used the Cryo-EM structure of SARS coronavirus S-protein (PDB ID: 6ACD)
       <sup>
        <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">34</a>
       </sup> as the template.
      </p>
     </div>
    </div>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgment</h2>
    <p id="p-25">This work was supported by NIH R01AI069120 and R01AI140718 grants, the National Key Plan for Scientific Research and Development of China (2016YFD0500301, 2016YFC1200200), CAMS Initiative for Innovative Medicine (CAMS-I2M, 2016-I2M-1-005), the National Natural Science Foundation of China (U1603126), the Central PublicInterest Scientific Institution Basal Research Fund (2016ZX310195, 2017PT31026 and 2018PT31016), China postdoctoral science foundation grant (2019M660548). We thank Dr. Fang Li for helpful discussion.</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">WHO</span>. 
         <span class="cit-article-title">Clinical management of severe acute respiratory infection when Novel coronavirus (nCoV) infection is suspected: interim guidance</span>. 
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">WHO</span>. 
         <span class="cit-article-title">Confirmed 2019-nCoV Cases Globally Global Map</span>. 
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Munster</span>
          <span class="cit-name-given-names">VJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Koopmans</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">van Doremalen</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">van Riel</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">de Wit</span>
          <span class="cit-name-given-names">E</span>
         </span>. 
         <span class="cit-article-title">A Novel Coronavirus Emerging in China - Key Questions for Impact Assessment</span>. 
         <abbr class="cit-jnl-abbrev">The New England journal of medicine</abbr>
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chan</span>
          <span class="cit-name-given-names">JF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yuan</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kok</span>
          <span class="cit-name-given-names">KH</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster</span>. 
         <abbr class="cit-jnl-abbrev">Lancet (London, England)</abbr>
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A Novel Coronavirus from Patients with Pneumonia in China, 2019</span>. 
         <abbr class="cit-jnl-abbrev">The New England journal of medicine</abbr>
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.6" data-doi="10.1016/j.antiviral.2014.06.013">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>
          <span class="cit-name-given-names">DX</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fung</span>
          <span class="cit-name-given-names">TS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chong</span>
          <span class="cit-name-given-names">KK</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shukla</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hilgenfeld</span>
          <span class="cit-name-given-names">R</span>
         </span>. 
         <span class="cit-article-title">Accessory proteins of SARS-CoV and other coronaviruses</span>. 
         <abbr class="cit-jnl-abbrev">Antiviral research</abbr>
         <span class="cit-pub-date">2014</span>;
         <span class="cit-vol">109</span>:
         <span class="cit-fpage">97</span>–
         <span class="cit-lpage">109</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAntiviral%2Bresearch%26rft.volume%253D109%26rft.spage%253D97%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.antiviral.2014.06.013%26rft_id%253Dinfo%253Apmid%252F24995382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.antiviral.2014.06.013&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24995382&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.29.971101.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</span>. 
         <abbr class="cit-jnl-abbrev">Lancet (London, England)</abbr>
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.8" data-doi="10.1007/s11262-014-1139-8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Shukla</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hilgenfeld</span>
          <span class="cit-name-given-names">R</span>
         </span>. 
         <span class="cit-article-title">Acquisition of new protein domains by coronaviruses: analysis of overlapping genes coding for proteins N and 9b in SARS coronavirus</span>. 
         <abbr class="cit-jnl-abbrev">Virus genes</abbr>
         <span class="cit-pub-date">2015</span>;
         <span class="cit-vol">50</span>:
         <span class="cit-fpage">29</span>–
         <span class="cit-lpage">38</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVirus%2Bgenes%26rft.volume%253D50%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11262-014-1139-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1007/s11262-014-1139-8&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.9" data-doi="10.1016/j.antiviral.2013.08.014">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">F</span>
         </span>. 
         <span class="cit-article-title">Receptor recognition and cross-species infections of SARS coronavirus</span>. 
         <abbr class="cit-jnl-abbrev">Antiviral research</abbr>
         <span class="cit-pub-date">2013</span>;
         <span class="cit-vol">100</span>:
         <span class="cit-fpage">246</span>–
         <span class="cit-lpage">54</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAntiviral%2Bresearch%26rft.volume%253D100%26rft.spage%253D246%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.antiviral.2013.08.014%26rft_id%253Dinfo%253Apmid%252F23994189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.antiviral.2013.08.014&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23994189&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.29.971101.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000328179300027&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.10">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">F</span>
         </span>. 
         <span class="cit-article-title">Structure, Function, and Evolution of Coronavirus Spike Proteins</span>. 
         <abbr class="cit-jnl-abbrev">Annual review of virology</abbr>
         <span class="cit-pub-date">2016</span>;
         <span class="cit-vol">3</span>:
         <span class="cit-fpage">237</span>–
         <span class="cit-lpage">61</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bvirology%26rft.volume%253D3%26rft.spage%253D237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.11" data-doi="10.1073/pnas.0809524106">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Belouzard</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chu</span>
          <span class="cit-name-given-names">VC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Whittaker</span>
          <span class="cit-name-given-names">GR</span>
         </span>. 
         <span class="cit-article-title">Activation of the SARS coronavirus spike protein via sequential proteolytic cleavage at two distinct sites</span>. 
         <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr>
         <span class="cit-pub-date">2009</span>;
         <span class="cit-vol">106</span>:
         <span class="cit-fpage">5871</span>–
         <span class="cit-lpage">6</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0809524106%26rft_id%253Dinfo%253Apmid%252F19321428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA2LzE0LzU4NzEiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8wMi8yMDIwLjAyLjI5Ljk3MTEwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.12">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zheng</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shang</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Lysosomal Proteases Are a Determinant of Coronavirus Tropism</span>. 
         <abbr class="cit-jnl-abbrev">Journal of virology</abbr>
         <span class="cit-pub-date">2018</span>;
         <span class="cit-vol">92</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.13" data-doi="10.1016/j.tim.2015.06.003">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">Q</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gao</span>
          <span class="cit-name-given-names">GF</span>
         </span>. 
         <span class="cit-article-title">Bat-to-human: spike features determining ‘host jump’ of coronaviruses SARS-CoV, MERS-CoV, and beyond</span>. 
         <abbr class="cit-jnl-abbrev">Trends in microbiology</abbr>
         <span class="cit-pub-date">2015</span>;
         <span class="cit-vol">23</span>:
         <span class="cit-fpage">468</span>–
         <span class="cit-lpage">78</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bmicrobiology%26rft.volume%253D23%26rft.spage%253D468%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2015.06.003%26rft_id%253Dinfo%253Apmid%252F26206723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.tim.2015.06.003&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26206723&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.29.971101.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.14" data-doi="10.1080/10409230802058320">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">White</span>
          <span class="cit-name-given-names">JM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Delos</span>
          <span class="cit-name-given-names">SE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Brecher</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schornberg</span>
          <span class="cit-name-given-names">K</span>
         </span>. 
         <span class="cit-article-title">Structures and mechanisms of viral membrane fusion proteins: multiple variations on a common theme</span>. 
         <abbr class="cit-jnl-abbrev">Critical reviews in biochemistry and molecular biology</abbr>
         <span class="cit-pub-date">2008</span>;
         <span class="cit-vol">43</span>:
         <span class="cit-fpage">189</span>–
         <span class="cit-lpage">219</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCritical%2BReviews%2Bin%2BBiochemistry%2Band%2BMolecular%2BBiology%26rft.stitle%253DCRIT%2BREV%2BBIOCHEM%2BMOL%2BBIOL%26rft.aulast%253DWhite%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D43%26rft.issue%253D3%26rft.spage%253D189%26rft.epage%253D219%26rft.atitle%253DStructures%2Band%2Bmechanisms%2Bof%2Bviral%2Bmembrane%2Bfusion%2Bproteins%253A%2Bmultiple%2Bvariations%2Bon%2Ba%2Bcommon%2Btheme.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10409230802058320%26rft_id%253Dinfo%253Apmid%252F18568847%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1080/10409230802058320&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18568847&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.29.971101.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000256840600002&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Peng Zhou</span>
          <span class="cit-name-given-names">X-LY</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Xian-Guang</span>
          <span class="cit-name-surname">Wang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Ben</span>
          <span class="cit-name-surname">Hu</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Lei</span>
          <span class="cit-name-surname">Zhang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Wei</span>
          <span class="cit-name-surname">Zhang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Hao-Rui</span>
          <span class="cit-name-surname">Si</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Yan</span>
          <span class="cit-name-surname">Zhu</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Bei</span>
          <span class="cit-name-surname">Li</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Chao-Lin</span>
          <span class="cit-name-surname">Huang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Hui-Dong</span>
          <span class="cit-name-surname">Chen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Jing</span>
          <span class="cit-name-surname">Chen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Yun</span>
          <span class="cit-name-surname">Luo</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Hua</span>
          <span class="cit-name-surname">Guo</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Ren-Di</span>
          <span class="cit-name-surname">Jiang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mei-Qin Liu</span>
          <span class="cit-name-given-names">YC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Xu-Rui</span>
          <span class="cit-name-surname">Shen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Xi</span>
          <span class="cit-name-surname">Wang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Xiao-Shuang</span>
          <span class="cit-name-surname">Zheng</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Kai</span>
          <span class="cit-name-surname">Zhao</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Quan-Jiao Chen</span>
          <span class="cit-name-given-names">FD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Lin-Lin</span>
          <span class="cit-name-surname">Liu</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Bing</span>
          <span class="cit-name-surname">Yan</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Fa-Xian</span>
          <span class="cit-name-surname">Zhan</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Yan-Yi</span>
          <span class="cit-name-surname">Wang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Geng-Fu</span>
          <span class="cit-name-surname">Xiao</span>
         </span> &amp;;, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shi</span>
          <span class="cit-name-given-names">Z-L</span>
         </span>. 
         <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wan</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shang</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Graham</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Baric</span>
          <span class="cit-name-given-names">RS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">F</span>
         </span>. 
         <span class="cit-article-title">Receptor recognition by novel coronavirus from Wuhan: An analysis based on decade-long structural studies of SARS</span>. 
         <abbr class="cit-jnl-abbrev">Journal of virology</abbr>
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">17.</span>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.17">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">JP</span>
         </span>. 
         <span class="cit-article-title">Viral Metagenomics Revealed Sendai Virus and Coronavirus Infection of Malayan Pangolins (Manis javanica)</span>. 
         <abbr class="cit-jnl-abbrev">Viruses</abbr>
         <span class="cit-pub-date">2019</span>;
         <span class="cit-vol">11</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">18.</span>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.18" data-doi="10.1016/S0065-3527(05)64006-7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">de Haan</span>
          <span class="cit-name-given-names">CA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rottier</span>
          <span class="cit-name-given-names">PJ</span>
         </span>. 
         <span class="cit-article-title">Molecular interactions in the assembly of coronaviruses</span>. 
         <abbr class="cit-jnl-abbrev">Advances in virus research</abbr>
         <span class="cit-pub-date">2005</span>;
         <span class="cit-vol">64</span>:
         <span class="cit-fpage">165</span>–
         <span class="cit-lpage">230</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bvirus%2Bresearch%26rft.stitle%253DAdv%2BVirus%2BRes%26rft.aulast%253Dde%2BHaan%26rft.auinit1%253DC.%2BA.%26rft.volume%253D64%26rft.spage%253D165%26rft.epage%253D230%26rft.atitle%253DMolecular%2Binteractions%2Bin%2Bthe%2Bassembly%2Bof%2Bcoronaviruses.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-3527%252805%252964006-7%26rft_id%253Dinfo%253Apmid%252F16139595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0065-3527(05)64006-7&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16139595&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.29.971101.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000231014200006&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">19.</span>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.19" data-doi="10.1111/tra.12389">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">White</span>
          <span class="cit-name-given-names">JM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Whittaker</span>
          <span class="cit-name-given-names">GR</span>
         </span>. 
         <span class="cit-article-title">Fusion of Enveloped Viruses in Endosomes</span>. 
         <abbr class="cit-jnl-abbrev">Traffic (Copenhagen, Denmark)</abbr>
         <span class="cit-pub-date">2016</span>;
         <span class="cit-vol">17</span>:
         <span class="cit-fpage">593</span>–
         <span class="cit-lpage">614</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTraffic%2B%2528Copenhagen%252C%2BDenmark%2529%26rft.volume%253D17%26rft.spage%253D593%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftra.12389%26rft_id%253Dinfo%253Apmid%252F26935856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/tra.12389&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26935856&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.29.971101.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">20.</span>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.20">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Daniel Wrapp</span>
          <span class="cit-name-given-names">NW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Kizzmekia S.</span>
          <span class="cit-name-surname">Corbett</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Jory A.</span>
          <span class="cit-name-surname">Goldsmith</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Ching-Lin</span>
          <span class="cit-name-surname">Hsieh</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Olubukola</span>
          <span class="cit-name-surname">Abiona</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Barney S.</span>
          <span class="cit-name-surname">Graham</span>
         </span>, View ORCID 
         <span class="cit-auth">
          <span class="cit-name-given-names">ProfileJason S.</span>
          <span class="cit-name-surname">McLellan</span>
         </span>. 
         <span class="cit-article-title">Cryo-EM Structure of the 2019-nCoV Spike in the Prefusion Conformation</span>. 
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">21.</span>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.21" data-doi="10.1128/JVI.01048-15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lau</span>
          <span class="cit-name-given-names">SK</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Feng</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Severe Acute Respiratory Syndrome (SARS) Coronavirus ORF8 Protein Is Acquired from SARS-Related Coronavirus from Greater Horseshoe Bats through Recombination</span>. 
         <abbr class="cit-jnl-abbrev">Journal of virology</abbr>
         <span class="cit-pub-date">2015</span>;
         <span class="cit-vol">89</span>:
         <span class="cit-fpage">10532</span>–
         <span class="cit-lpage">47</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01048-15%26rft_id%253Dinfo%253Apmid%252F26269185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4OS8yMC8xMDUzMiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzAyLzIwMjAuMDIuMjkuOTcxMTAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">22.</span>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.22" data-doi="10.1038/nature12711">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ge</span>
          <span class="cit-name-given-names">XY</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">JL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">XL</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Isolation and characterization of a bat SARS-like coronavirus that uses the ACE2 receptor</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-pub-date">2013</span>;
         <span class="cit-vol">503</span>:
         <span class="cit-fpage">535</span>–
         <span class="cit-lpage">8</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D503%26rft.spage%253D535%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12711%26rft_id%253Dinfo%253Apmid%252F24172901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature12711&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24172901&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.29.971101.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000327464200045&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">23.</span>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.23">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</span>. 
         <abbr class="cit-jnl-abbrev">Lancet (London, England)</abbr>
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">24.</span>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.24">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Cohen</span>
          <span class="cit-name-given-names">J</span>
         </span>. 
         <span class="cit-article-title">Mining coronavirus genomes for clues to the outbreak’s origins</span>. 
         <abbr class="cit-jnl-abbrev">Science (New York, NY)</abbr>
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">25.</span>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.25">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shi</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yu</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Bats are natural reservoirs of SARS-like coronaviruses</span>. 
         <abbr class="cit-jnl-abbrev">Science (New York, NY)</abbr>
         <span class="cit-pub-date">2005</span>;
         <span class="cit-vol">310</span>:
         <span class="cit-fpage">676</span>–
         <span class="cit-lpage">9</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%2B%2528New%2BYork%252C%2BNY%2529%26rft.volume%253D310%26rft.spage%253D676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">26.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.29.971101v1.26" data-doi="10.1371/journal.ppat.1006698">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zeng</span>
          <span class="cit-name-given-names">LP</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">XL</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Discovery of a rich gene pool of bat SARS-related coronaviruses provides new insights into the origin of SARS coronavirus</span>. 
         <abbr class="cit-jnl-abbrev">PLoS pathogens</abbr>
         <span class="cit-pub-date">2017</span>;
         <span class="cit-vol">13</span>:
         <span class="cit-fpage">e1006698</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2Bpathogens%26rft.volume%253D13%26rft.spage%253De1006698%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1006698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1006698&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">27.</span>
      <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.27" data-doi="10.1056/NEJMc1409847">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Drosten</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kellam</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Memish</span>
          <span class="cit-name-given-names">ZA</span>
         </span>. 
         <span class="cit-article-title">Evidence for camel-to-human transmission of MERS coronavirus</span>. 
         <abbr class="cit-jnl-abbrev">The New England journal of medicine</abbr>
         <span class="cit-pub-date">2014</span>;
         <span class="cit-vol">371</span>:
         <span class="cit-fpage">1359</span>–
         <span class="cit-lpage">60</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BNew%2BEngland%2Bjournal%2Bof%2Bmedicine%26rft.volume%253D371%26rft.spage%253D1359%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMc1409847%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1056/NEJMc1409847&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">28.</span>
      <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.28" data-doi="10.1016/bs.aivir.2016.08.004">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hulswit</span>
          <span class="cit-name-given-names">RJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">de Haan</span>
          <span class="cit-name-given-names">CA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bosch</span>
          <span class="cit-name-given-names">BJ</span>
         </span>. 
         <span class="cit-article-title">Coronavirus Spike Protein and Tropism Changes</span>. 
         <abbr class="cit-jnl-abbrev">Advances in virus research</abbr>
         <span class="cit-pub-date">2016</span>;
         <span class="cit-vol">96</span>:
         <span class="cit-fpage">29</span>–
         <span class="cit-lpage">57</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bvirus%2Bresearch%26rft.volume%253D96%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fbs.aivir.2016.08.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/bs.aivir.2016.08.004&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">29.</span>
      <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.29">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wrapp</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Corbett</span>
          <span class="cit-name-given-names">KS</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation</span>. 
         <abbr class="cit-jnl-abbrev">Science (New York, NY)</abbr>
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">30.</span>
      <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.30" data-doi="10.1128/JVI.03372-12">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bertram</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dijkman</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Habjan</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">TMPRSS2 activates the human coronavirus 229E for cathepsin-independent host cell entry and is expressed in viral target cells in the respiratory epithelium</span>. 
         <abbr class="cit-jnl-abbrev">Journal of virology</abbr>
         <span class="cit-pub-date">2013</span>;
         <span class="cit-vol">87</span>:
         <span class="cit-fpage">6150</span>–
         <span class="cit-lpage">60</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.03372-12%26rft_id%253Dinfo%253Apmid%252F23536651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Ny8xMS82MTUwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMDIvMjAyMC4wMi4yOS45NzExMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">31.</span>
      <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.31" data-doi="10.1128/JVI.01412-08">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Watanabe</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Matsuyama</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shirato</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Entry from the cell surface of severe acute respiratory syndrome coronavirus with cleaved S protein as revealed by pseudotype virus bearing cleaved S protein</span>. 
         <abbr class="cit-jnl-abbrev">Journal of virology</abbr>
         <span class="cit-pub-date">2008</span>;
         <span class="cit-vol">82</span>:
         <span class="cit-fpage">11985</span>–
         <span class="cit-lpage">91</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01412-08%26rft_id%253Dinfo%253Apmid%252F18786990%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4Mi8yMy8xMTk4NSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzAyLzIwMjAuMDIuMjkuOTcxMTAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">32.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.29.971101v1.32" data-doi="10.1093/molbev/msw054">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kumar</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Stecher</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tamura</span>
          <span class="cit-name-given-names">K</span>
         </span>. 
         <span class="cit-article-title">MEGA7: Molecular Evolutionary Genetics Analysis Version 7.0 for Bigger Datasets</span>. 
         <abbr class="cit-jnl-abbrev">Molecular biology and evolution</abbr>
         <span class="cit-pub-date">2016</span>;
         <span class="cit-vol">33</span>:
         <span class="cit-fpage">1870</span>–
         <span class="cit-lpage">4</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMolecular%2Bbiology%2Band%2Bevolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw054%26rft_id%253Dinfo%253Apmid%252F27004904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/molbev/msw054&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=27004904&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.29.971101.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">33.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.29.971101v1.33" data-doi="10.1093/nar/gky427">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Waterhouse</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bertoni</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bienert</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">SWISS-MODEL: homology modelling of protein structures and complexes</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr>
         <span class="cit-pub-date">2018</span>;
         <span class="cit-vol">46</span>:
         <span class="cit-fpage">W296</span>–
         <span class="cit-lpage">w303</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft.volume%253D46%26rft.spage%253DW296%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky427%26rft_id%253Dinfo%253Apmid%252F29788355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gky427&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=29788355&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.29.971101.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">34.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.29.971101v1.34" data-doi="10.1371/journal.ppat.1007236">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Song</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gui</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xiang</span>
          <span class="cit-name-given-names">Y</span>
         </span>. 
         <span class="cit-article-title">Cryo-EM structure of the SARS coronavirus spike glycoprotein in complex with its host cell receptor ACE2</span>. 
         <abbr class="cit-jnl-abbrev">PLoS pathogens</abbr>
         <span class="cit-pub-date">2018</span>;
         <span class="cit-vol">14</span>:
         <span class="cit-fpage">e1007236</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2Bpathogens%26rft.volume%253D14%26rft.spage%253De1007236%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1007236%26rft_id%253Dinfo%253Apmid%252F30102747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1007236&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=30102747&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.29.971101.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <div class="section ref-list" id="ref-list-2">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">32.</span>
      <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 32. in text" id="ref-35">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.35" data-doi="10.1093/molbev/msw054">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kumar</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Stecher</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tamura</span>
          <span class="cit-name-given-names">K</span>
         </span>. 
         <span class="cit-article-title">MEGA7: Molecular Evolutionary Genetics Analysis Version 7.0 for Bigger Datasets</span>. 
         <abbr class="cit-jnl-abbrev">Molecular biology and evolution</abbr>
         <span class="cit-pub-date">2016</span>;
         <span class="cit-vol">33</span>:
         <span class="cit-fpage">1870</span>–
         <span class="cit-lpage">4</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMolecular%2Bbiology%2Band%2Bevolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw054%26rft_id%253Dinfo%253Apmid%252F27004904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/molbev/msw054&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=27004904&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.29.971101.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">33.</span>
      <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 33. in text" id="ref-36">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.36" data-doi="10.1093/nar/gky427">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Waterhouse</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bertoni</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bienert</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">SWISS-MODEL: homology modelling of protein structures and complexes</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr>
         <span class="cit-pub-date">2018</span>;
         <span class="cit-vol">46</span>:
         <span class="cit-fpage">W296</span>–
         <span class="cit-lpage">w303</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft.volume%253D46%26rft.spage%253DW296%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky427%26rft_id%253Dinfo%253Apmid%252F29788355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gky427&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=29788355&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.29.971101.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">34.</span>
      <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 34. in text" id="ref-37">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.29.971101v1.37" data-doi="10.1371/journal.ppat.1007236">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Song</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gui</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xiang</span>
          <span class="cit-name-given-names">Y</span>
         </span>. 
         <span class="cit-article-title">Cryo-EM structure of the SARS coronavirus spike glycoprotein in complex with its host cell receptor ACE2</span>. 
         <abbr class="cit-jnl-abbrev">PLoS pathogens</abbr>
         <span class="cit-pub-date">2018</span>;
         <span class="cit-vol">14</span>:
         <span class="cit-fpage">e1007236</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2Bpathogens%26rft.volume%253D14%26rft.spage%253De1007236%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1007236%26rft_id%253Dinfo%253Apmid%252F30102747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1007236&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=30102747&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.29.971101.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
