<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2019/04/01/410282/embed/graphic-2.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2019/04/01/410282.atom"/>
  <meta name="HW.pisa" content="biorxiv;410282v3"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="A Novel Method for the Capture-based Purification of Whole Viral Native RNA Genomes"/>
  <meta name="DC.Identifier" content="10.1101/410282"/>
  <meta name="DC.Date" content="2019-04-01"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Current technologies for targeted characterization and manipulation of viral RNA primarily involve amplification or ultracentrifugation with isopycnic gradients of viral particles to decrease host RNA background. The former strategy is non-compatible for characterizing properties innate to RNA strands such as secondary structure, RNA-RNA interactions, and also for nanopore direct RNA sequencing involving the sequencing of native RNA strands. The latter strategy, ultracentrifugation, causes loss in genomic information due to its inability to retrieve unassembled viral RNA. To address this, we developed a novel application of current nucleic acid hybridization technologies for direct characterization of RNA. In particular, we modified a current enrichment protocol to capture whole viral native RNA genomes for downstream RNA assays to circumvent the abovementioned problems. This technique involves hybridization of biotinylated baits at 500 nucleotides (nt) intervals, stringent washes and release of free native RNA strands using DNase I treatment, with a turnaround time of about 6 h 15 min. RT-qPCR was used as the primary proof of concept that capture-based purification indeed removes host background. Subsequently, capture-based purification was applied to direct RNA sequencing as proof of concept that capture-based purification can be coupled with downstream RNA assays. We report that this protocol was able to successfully purify viral RNA by 561-791 fold. We also report that application of this protocol to direct RNA sequencing yielded a reduction in human host RNA background by 1580 fold, a 99.91% recovery of viral genome with at least 15x coverage, and a mean coverage across the genome of 120x. This report is, to the best of our knowledge, the first description of a capture-based purification method for assays that involve direct manipulation or characterisation of native RNA. This report also describes a successful application of capture-based purification as a direct RNA sequencing strategy that addresses certain limitations of current strategies in sequencing RNA viral genomes."/>
  <meta name="DC.Contributor" content="Cedric Chih Shen Tan"/>
  <meta name="DC.Contributor" content="Sebastian Maurer-Stroh"/>
  <meta name="DC.Contributor" content="Yue Wan"/>
  <meta name="DC.Contributor" content="October Michael Sessions"/>
  <meta name="DC.Contributor" content="Paola Florez de Sessions"/>
  <meta name="article:published_time" content="2019-04-01"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="A Novel Method for the Capture-based Purification of Whole Viral Native RNA Genomes"/>
  <meta name="citation_abstract" lang="en" content="&lt;;h3&gt;;ABSTRACT&lt;;/h3&gt;; &lt;;p&gt;;Current technologies for targeted characterization and manipulation of viral RNA primarily involve amplification or ultracentrifugation with isopycnic gradients of viral particles to decrease host RNA background. The former strategy is non-compatible for characterizing properties innate to RNA strands such as secondary structure, RNA-RNA interactions, and also for nanopore direct RNA sequencing involving the sequencing of native RNA strands. The latter strategy, ultracentrifugation, causes loss in genomic information due to its inability to retrieve unassembled viral RNA. To address this, we developed a novel application of current nucleic acid hybridization technologies for direct characterization of RNA. In particular, we modified a current enrichment protocol to capture whole viral native RNA genomes for downstream RNA assays to circumvent the abovementioned problems. This technique involves hybridization of biotinylated baits at 500 nucleotides (nt) intervals, stringent washes and release of free native RNA strands using DNase I treatment, with a turnaround time of about 6 h 15 min. RT-qPCR was used as the primary proof of concept that capture-based purification indeed removes host background. Subsequently, capture-based purification was applied to direct RNA sequencing as proof of concept that capture-based purification can be coupled with downstream RNA assays. We report that this protocol was able to successfully purify viral RNA by 561-791 fold. We also report that application of this protocol to direct RNA sequencing yielded a reduction in human host RNA background by 1580 fold, a 99.91% recovery of viral genome with at least 15x coverage, and a mean coverage across the genome of 120x. This report is, to the best of our knowledge, the first description of a capture-based purification method for assays that involve direct manipulation or characterisation of native RNA. This report also describes a successful application of capture-based purification as a direct RNA sequencing strategy that addresses certain limitations of current strategies in sequencing RNA viral genomes.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2019/01/01"/>
  <meta name="citation_mjid" content="biorxiv;410282v3"/>
  <meta name="citation_id" content="410282v3"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/410282v3"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/410282v3.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/410282v3.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/04/01/410282.full.pdf"/>
  <meta name="citation_doi" content="10.1101/410282"/>
  <meta name="citation_num_pages" content="23"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="410282"/>
  <meta name="citation_author" content="Cedric Chih Shen Tan"/>
  <meta name="citation_author_institution" content="Genome Institute of Singapore (GIS), Agency for Science, Technology and Research (A*STAR)"/>
  <meta name="citation_author_institution" content="A*STAR Graduate Academy"/>
  <meta name="citation_author_institution" content="University College London"/>
  <meta name="citation_author_email" content="cedriccstan@gmail.com"/>
  <meta name="citation_author" content="Sebastian Maurer-Stroh"/>
  <meta name="citation_author_institution" content="Bioinfomatics Institute (BII)"/>
  <meta name="citation_author" content="Yue Wan"/>
  <meta name="citation_author_institution" content="Genome Institute of Singapore (GIS), Agency for Science, Technology and Research (A*STAR)"/>
  <meta name="citation_author" content="October Michael Sessions"/>
  <meta name="citation_author_institution" content="National University of Singapore, Saw Swee Hock School of Public Health"/>
  <meta name="citation_author" content="Paola Florez de Sessions"/>
  <meta name="citation_author_institution" content="Genome Institute of Singapore (GIS), Agency for Science, Technology and Research (A*STAR)"/>
  <meta name="citation_reference" content="Acinas SG, Sarma-Rupavtarm R, Klepac-Ceraj V, Polz MF (2005) PCR-induced sequence artifacts and bias: Insights from comparison of two 16s rRNA clone libraries constructed from the same sample. Applied and Environmental Microbiology, doi: 10.1128/AEM.71.12.8966-8969.2005"/>
  <meta name="citation_reference" content="Buclez PO, Dias Florencio G, Relizani K, Beley C, Garcia L, Benchaouir R (2016) Rapid, scalable, and low-cost purification of recombinant adeno-associated virus produced by baculovirus expression vector system. Molecular Therapy - Methods and Clinical Development, doi: 10.1038/mtm.2016.35"/>
  <meta name="citation_reference" content="Cantara WA, Olson ED, Musier-Forsyth K (2014) Progress and outlook in structural biology of large viral RNAs. Virus Research, doi: 10.1016/j.virusres.2014.06.007"/>
  <meta name="citation_reference" content="Cowan D, Meyer Q, Stafford W, Muyanga S, Cameron R, Wittwer P (2005) Metagenomic gene discovery: Past, present and future. Trends in Biotechnology, doi: 10.1016/j.tibtech.2005.04.001"/>
  <meta name="citation_reference" content="Dethoff EA, Boerneke MA, Gokhale NS, Muhire BM, Martin DP, Sacco MT, McFadden MJ, Weinstein JB, Messer WB, Horner SM (2018) Pervasive tertiary structure in the dengue virus RNA genome. Proceedings of the National Academy of Sciences 115(45)National Acad Sciences, : 11513–11518"/>
  <meta name="citation_reference" content="Deviatkin AA, Lukashev AN, Markelov MM, Gmyl L V., Shipulin GA (2017) Enrichment of Viral Nucleic Acids by Solution Hybrid Selection with Genus Specific Oligonucleotides. Scientific Reports 7(1)Springer US, : 1–7, doi: 10.1038/s41598-017-10342-w"/>
  <meta name="citation_reference" content="Garalde DR, Snell EA, Jachimowicz D, Sipos B, Lloyd JH, Bruce M, Pantic N, Admassu T, James P, Warland A, Jordan M, Ciccone J, Serra S, Keenan J, Martin S, McNeill L, Wallace EJ, Jayasinghe L, Wright C, Blasco J, Young S, Brocklebank D, Juul S, Clarke J, Heron AJ, Turner DJ (2018) Highly parallel direct RN A sequencing on an array of nanopores. Nature Methods 15(3): 201–206, doi: 10.1038/nmeth.4577"/>
  <meta name="citation_reference" content="García-Alcalde F, Okonechnikov K, Carbonell J, Cruz LM, Götz S, Tarazona S, Dopazo J, Meyer TF, Conesa A (2012) Qualimap: Evaluating next-generation sequencing alignment data. Bioinformatics, doi: 10.1093/bioinformatics/bts503"/>
  <meta name="citation_reference" content="Hall RJ, Wang J, Todd AK, Bissielo AB, Yen S, Strydom H, Moore NE, Ren X, Huang QS, Carter PE, Peacey M (2014) Evaluation of rapid and simple techniques for the enrichment of viruses prior to metagenomic virus discovery. Journal of Virological Methods, doi: 10.1016/j.jviromet.2013.08.035"/>
  <meta name="citation_reference" content="Ho ZJM, Hapuarachchi HC, Barkham T, Chow A, Ng LC, Lee JMV, Leo YS, Prem K, Lim YHG, de Sessions PF, Rabaa MA, Chong CS, Tan CH, Rajarethinam J, Tan J, Anderson DE, Ong X, Cook AR, Chong CY, Hsu LY, Yap G, Lai YL, Chawla T, Pan L, Sim S, Chen ICM, Thoon KC, Yung CF, Li JH, Ng HLD, Nandar K, Ooi PL, Lin RTP, Aw P, Uehara A, Pratim De P, Soon W, Hibberd ML, Ng HH, Maurer-Stroh S, Sessions OM (2017) Outbreak of Zika virus infection in Singapore: an epidemiological, entomological, virological, and clinical analysis. The Lancet Infectious Diseases, doi: 10.1016/S1473-3099(17)30249-9"/>
  <meta name="citation_reference" content="Jensen RH, Mollerup S, Mourier T, Hansen TA, Fridholm H, Nielsen LP, Willerslev E, Hansen AJ, Vinner L (2015) Target-dependent enrichment of virions determines the reduction of high-throughput sequencing in virus discovery. PLoS ONE 10(4): 1–18, doi: 10.1371/journal.pone.0122636"/>
  <meta name="citation_reference" content="Keller MW, Rambo-Martin BL, Wilson MM, Ridenour CA, Shepard SS, Stark TJ, Neuhaus EB, Dugan VG, Wentworth DE, Barnes JR (2018) Direct RNA Sequencing of the Coding Complete Influenza A Virus Genome. Scientific Reports, doi: 10.1038/s41598-018-32615-8"/>
  <meta name="citation_reference" content="Liu CC, Guo MS, Lin FHY, Hsiao KN, Chang KHW, Chou AH, Wang YC, Chen YC, Yang CS, Chong PCS (2011) Purification and characterization of enterovirus 71 viral particles produced from vero cells grown in a serum-free microcarrier bioreactor system. PLoS ONE, doi: 10.1371/journal.pone.0020005"/>
  <meta name="citation_reference" content="Low JGH, Ooi EE, Tolfvenstam T, Leo YS, Hibberd ML, Ng LC, Lai YL, Yap GS, Li CS, Vasudevan SG (2006) Early Dengue infection and outcome study (EDEN)-study design and preliminary findings. ANNALS-ACADEMY OF MEDICINE SINGAPORE 35(11)Academy of Medicine, Singapore; 1999, : 783"/>
  <meta name="citation_reference" content="Mamanova L, Coffey AJ, Scott CE, Kozarewa I, Turner EH, Kumar A, Howard E, Shendure J, Turner DJ (2010) Target-enrichment strategies for next-generation sequencing. Nature Methods 7(2)Nature Publishing Group, : 111–118, doi: 10.1038/nmeth.1419"/>
  <meta name="citation_reference" content="Manokaran G, Finol E, Wang C, Gunaratne J, Bahl J, Ong EZ, Tan HC, Sessions OM, Ward AM, Gubler DJ, Harris E, Garcia-Blanco MA, Ooi EE (2015) Dengue subgenomic RNA binds TRIM25 to inhibit interferon expression for epidemiological fitness. Science, doi: 10.1007/BF03020102"/>
  <meta name="citation_reference" content="Marston DA, McElhinney LM, Ellis RJ, Horton DL, Wise EL, Leech SL, David D, de Lamballerie X, Fooks AR (2013) Next generation sequencing of viral RNA genomes. BMC Genomics, doi: 10.1186/1471-2164-14-444"/>
  <meta name="citation_reference" content="Moon SL, Dodd BJT, Brackney DE, Wilusz CJ, Ebel GD, Wilusz J (2015) Flavivirus sfRNA suppresses antiviral RNA interference in cultured cells and mosquitoes and directly interacts with the RNAi machinery. Virology, doi: 10.1016/j.virol.2015.08.009"/>
  <meta name="citation_reference" content="Ozsolak F, Milos PM (2011) RNA sequencing: Advances, challenges and opportunities. Nature Reviews Genetics, doi: 10.1038/nrg2934"/>
  <meta name="citation_reference" content="Ozsolak F, Platt AR, Jones DR, Reifenberger JG, Sass LE, McInerney P, Thompson JF, Bowers J, Jarosz M, Milos PM (2009) Direct RNA sequencing. Nature, doi: 10.1038/nature08390"/>
  <meta name="citation_reference" content="Peattie DA (1979) Direct chemical method for sequencing RNA. Proceedings of the National Academy of Sciences, doi: 10.1073/pnas.76.4.1760"/>
  <meta name="citation_reference" content="Pompon J, Manuel M, Ng GK, Wong B, Shan C, Manokaran G, Soto-Acosta R, Bradrick SS, Ooi EE, Missé D (2017) Dengue subgenomic flaviviral RNA disrupts immunity in mosquito salivary glands to increase virus transmission. PLoS pathogens 13(7)Public Library of Science, : e1006535"/>
  <meta name="citation_reference" content="Romero-López C, Berzal-Herranz A (2009) A long-range RNA-RNA interaction between the 51] and 31] ends of the HCV genome. RNA, doi: 10.1261/rna.1680809"/>
  <meta name="citation_reference" content="Schmittgen TD, Livak KJ (2008) Analyzing real-time PCR data by the comparative CTmethod. Nature Protocols, doi: 10.1038/nprot.2008.73"/>
  <meta name="citation_reference" content="Taylor S, Wakem M, Dijkman G, Alsarraj M, Nguyen M (2010) A practical approach to RT-qPCR- Publishing data that conform to the MIQE guidelines. Methods 50(4)Elsevier Inc., : S1–S5, doi: 10.1016/j.ymeth.2010.01.005"/>
  <meta name="citation_reference" content="Thomson E, Ip CLC, Badhan A, Christiansen MT, Adamson W, Ansari MA, Bibby D, Breuer J, Brown A, Bowden R, Bryant J, Bonsall D, Silva Filipe A Da, Hinds C, Hudson E, Klenerman P, Lythgow K, Mbisa JL, McLauchlan J, Myers R, Piazza P, Roy S, Trebes A, Sreenu VB, Witteveldt J, Barnes E, Simmonds P (2016) Comparison of next-generation sequencing technologies for comprehensive assessment of full-length hepatitis C viral genomes. Journal of Clinical Microbiology, doi: 10.1128/JCM.00330-16"/>
  <meta name="citation_reference" content="Witteveldt J, Blundell R, Maarleveld JJ, Mcfadden N, Evans DJ, Simmonds P (2014) The influence of viral RNA secondary structure on interactions with innate host cell defences. Nucleic Acids Research, doi: 10.1093/nar/gkt1291"/>
  <meta name="citation_reference" content="Wongsurawat T, Jenjaroenpun P, Taylor MK, Lee J, Tolardo AL, Parvathareddy J, Kandel S, Wadley TD, Kaewnapan B, Athipanyasilp N (2019) Rapid Sequencing of Multiple RNA Viruses in their Native Form. Frontiers in microbiology 10Frontiers Media SA"/>
  <meta name="twitter:title" content="A Novel Method for the Capture-based Purification of Whole Viral Native RNA Genomes"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Current technologies for targeted characterization and manipulation of viral RNA primarily involve amplification or ultracentrifugation with isopycnic gradients of viral particles to decrease host RNA background. The former strategy is non-compatible for characterizing properties innate to RNA strands such as secondary structure, RNA-RNA interactions, and also for nanopore direct RNA sequencing involving the sequencing of native RNA strands. The latter strategy, ultracentrifugation, causes loss in genomic information due to its inability to retrieve unassembled viral RNA. To address this, we developed a novel application of current nucleic acid hybridization technologies for direct characterization of RNA. In particular, we modified a current enrichment protocol to capture whole viral native RNA genomes for downstream RNA assays to circumvent the abovementioned problems. This technique involves hybridization of biotinylated baits at 500 nucleotides (nt) intervals, stringent washes and release of free native RNA strands using DNase I treatment, with a turnaround time of about 6 h 15 min. RT-qPCR was used as the primary proof of concept that capture-based purification indeed removes host background. Subsequently, capture-based purification was applied to direct RNA sequencing as proof of concept that capture-based purification can be coupled with downstream RNA assays. We report that this protocol was able to successfully purify viral RNA by 561-791 fold. We also report that application of this protocol to direct RNA sequencing yielded a reduction in human host RNA background by 1580 fold, a 99.91% recovery of viral genome with at least 15x coverage, and a mean coverage across the genome of 120x. This report is, to the best of our knowledge, the first description of a capture-based purification method for assays that involve direct manipulation or characterisation of native RNA. This report also describes a successful application of capture-based purification as a direct RNA sequencing strategy that addresses certain limitations of current strategies in sequencing RNA viral genomes."/>
  <meta name="og-title" property="og:title" content="A Novel Method for the Capture-based Purification of Whole Viral Native RNA Genomes"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/410282v3"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Current technologies for targeted characterization and manipulation of viral RNA primarily involve amplification or ultracentrifugation with isopycnic gradients of viral particles to decrease host RNA background. The former strategy is non-compatible for characterizing properties innate to RNA strands such as secondary structure, RNA-RNA interactions, and also for nanopore direct RNA sequencing involving the sequencing of native RNA strands. The latter strategy, ultracentrifugation, causes loss in genomic information due to its inability to retrieve unassembled viral RNA. To address this, we developed a novel application of current nucleic acid hybridization technologies for direct characterization of RNA. In particular, we modified a current enrichment protocol to capture whole viral native RNA genomes for downstream RNA assays to circumvent the abovementioned problems. This technique involves hybridization of biotinylated baits at 500 nucleotides (nt) intervals, stringent washes and release of free native RNA strands using DNase I treatment, with a turnaround time of about 6 h 15 min. RT-qPCR was used as the primary proof of concept that capture-based purification indeed removes host background. Subsequently, capture-based purification was applied to direct RNA sequencing as proof of concept that capture-based purification can be coupled with downstream RNA assays. We report that this protocol was able to successfully purify viral RNA by 561-791 fold. We also report that application of this protocol to direct RNA sequencing yielded a reduction in human host RNA background by 1580 fold, a 99.91% recovery of viral genome with at least 15x coverage, and a mean coverage across the genome of 120x. This report is, to the best of our knowledge, the first description of a capture-based purification method for assays that involve direct manipulation or characterisation of native RNA. This report also describes a successful application of capture-based purification as a direct RNA sequencing strategy that addresses certain limitations of current strategies in sequencing RNA viral genomes."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2019-04-01"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>A Novel Method for the Capture-based Purification of Whole Viral Native RNA Genomes | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-443071 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">ABSTRACT</h2>
    <p id="p-3">Current technologies for targeted characterization and manipulation of viral RNA primarily involve amplification or ultracentrifugation with isopycnic gradients of viral particles to decrease host RNA background. The former strategy is non-compatible for characterizing properties innate to RNA strands such as secondary structure, RNA-RNA interactions, and also for nanopore direct RNA sequencing involving the sequencing of native RNA strands. The latter strategy, ultracentrifugation, causes loss in genomic information due to its inability to retrieve unassembled viral RNA. To address this, we developed a novel application of current nucleic acid hybridization technologies for direct characterization of RNA. In particular, we modified a current enrichment protocol to capture whole viral native RNA genomes for downstream RNA assays to circumvent the abovementioned problems. This technique involves hybridization of biotinylated baits at 500 nucleotides (nt) intervals, stringent washes and release of free native RNA strands using DNase I treatment, with a turnaround time of about 6 h 15 min. RT-qPCR was used as the primary proof of concept that capture-based purification indeed removes host background. Subsequently, capture-based purification was applied to direct RNA sequencing as proof of concept that capture-based purification can be coupled with downstream RNA assays. We report that this protocol was able to successfully purify viral RNA by 561-791 fold. We also report that application of this protocol to direct RNA sequencing yielded a reduction in human host RNA background by 1580 fold, a 99.91% recovery of viral genome with at least 15x coverage, and a mean coverage across the genome of 120x. This report is, to the best of our knowledge, the first description of a capture-based purification method for assays that involve direct manipulation or characterisation of native RNA. This report also describes a successful application of capture-based purification as a direct RNA sequencing strategy that addresses certain limitations of current strategies in sequencing RNA viral genomes.</p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">INTRODUCTION</h2>
    <p id="p-4">Viruses with RNA genomes are the cause of many infectious diseases with serious consequences to human health and mortality such as flaviviruses, HIV-1, SARS-coronavirus, HTLV-1 and influenza virus (
     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Cantara et al. 2014</a>). It has been shown that RNA-RNA interactions (
     <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Romero-López and Berzal-Herranz 2009</a>) and intramolecular structure (
     <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Witteveldt et al. 2014</a>) of viral RNA genomes play an important role in viral replication. However, extraction of viral RNA directly from culture often yields viral RNA with high host RNA background (
     <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Marston et al. 2013</a>).
    </p>
    <p id="p-5">To deal with this issue, two main strategies are employed. Firstly, extracted viral RNA can be reverse transcribed to complementary DNA (cDNA) and amplified via polymerase chain reaction (PCR) to suit the sensitivities of the downstream assays (
     <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Ozsolak and Milos 2011</a>). Following conversion to cDNA and amplification, 
     <em>in vitro</em> transcription is then performed to retrieve genetic information in its RNA form, since both RNA-RNA interactions and RNA intramolecular structure can only be observed whilst in this form (
     <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Peattie 1979</a>). Indeed, both Romero-López et al. (2018) and Witteveldt et al. (2013) amplified viral RNA obtained from cell culture via PCR amplification of viral cDNA clones followed by 
     <em>in vitro</em> transcription. It has been shown, however, that such indirect methods for probing inherent properties of RNA increases experimental throughput at the expense of introducing PCR-induced artifacts and biases (
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Acinas et al. 2005</a>), potentially clouding our understanding of these properties (
     <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Ozsolak et al. 2009</a>). Secondly, isopycnic ultracentrifugation can be used to retrieve high purity samples of viral particles containing viral RNA, which can be subsequently extracted for downstream RNA assays. One example is the use of a sucrose density gradient to purify DENV2 RNA for RNA tertiary structure analysis (
     <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Dethoff et al. 2018</a>). While the use of isopycnic centrifugation allows for a high purity of viral RNA, only RNA from encapsulated viruses are obtained (
     <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Liu et al. 2011</a>), making this technique unfeasible for the study of unassembled viral genomes, which have also been shown to be crucial to our understanding of viral replication machinery (
     <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Manokaran et al. 2015</a>; 
     <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Moon et al. 2015</a>).
    </p>
    <p id="p-6">With the advent of 3
     <sup>rd</sup> generation nanopore sequencing, direct RNA sequencing of native RNA strands is heralded as a PCR and cDNA-bias free sequencing technology (
     <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Garalde et al. 2018</a>). However, the direct RNA sequencing of RNA viral genomes is similarly affected by high amounts of host RNA background in viral cultures. The only method to date which addresses this issue is the use of custom sequencing adapters which has been demonstrated to yield high sequencing coverage for RNA viruses. 
     <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Keller et al. (2018)</a> utilized a custom sequencing adapter complimentary to the 3’ end of 
     <em>influenza A</em> viral genome to selectively sequence only the viral RNA. However, this strategy is unviable for samples with diverse viral strains or genotypes where the 3’ sequence is unknown or non-conserved. An appropriate example is the wide diversity in 
     <em>hepatitis C</em> virus genotypes, which has caused issues in PCR primer design for target enrichment strategies due to sequence ambiguity (
     <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Thomson et al. 2016</a>). Similarly, 
     <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cowan et al. (2005)</a> noted that PCR-based enrichment techniques where a priori knowledge of target sequences is required for PCR primer design render the enrichment strategies ineffective in the characterization of novel viruses. It follows that the same sequence ambiguity in viral genomes would pose a problem for using customized sequencing adapters during direct RNA sequencing.
    </p>
    <p id="p-7">From the abovementioned problems, we identified a need for an alternative technique that isolates viral RNA molecules from host RNA background without PCR amplification or cDNA synthesis. In the case of direct RNA sequencing, such a technique should address the limitations presented by current strategies. In this study, we describe a capture-based purification method that leverages on current nucleic acid hybridization and enrichment techniques. This novel method is able to isolate native viral RNA molecules from a high host RNA background without the use of PCR amplification or cDNA synthesis whilst retaining genetic information. Our capture-based purification method is distinguished from conventional hybridization-based or PCR-based enrichment techniques explored in the review paper by 
     <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Mamanova et al. (2010)</a> by virtue of not having any PCR amplification or cDNA conversion steps. Such a purification method, when merged with current technologies for probing and characterizing inherent RNA properties, could potentially catalyze the study of viral RNAs and elucidate previously inaccessible characteristics of viral RNA genomes.
    </p>
    <p id="p-8">Reverse-transcription quantitative polymerase chain reaction (RT-qPCR) was used as the primary proof of concept that capture-based purification indeed removes host background. Subsequently, capture-based purification was applied to direct RNA sequencing as proof of concept that capture-based purification can be coupled with downstream RNA assays. The application of capture-based purification to direct RNA sequencing as an alternative to current strategies will also be discussed.</p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">MATERIALS AND METHODS</h2>
    <div id="sec-3" class="subsection">
     <h3>Viral culture</h3>
     <p id="p-9">Viral inoculum and Huh-7 hepatocellular carcinoma cells (Japanese Tissue Repository) were cultured in DMEM media (10% Fetal Bovine Serum, 1% Penicillin Streptomycin) up to 70-90% confluency in Corning® 150 cm
      <sup>2</sup> cell culture flasks (Sigma Aldrich). Prior to infection, spent media was discarded and Huh-7 cell monolayer was washed with 10 mL 1x PBS twice. Viral inoculum was provided by 
      <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Low et al. (2006)</a>. A multiplicity of infection of 1 was used, 2 mL 
      <em>dengue virus strain D1/SG/05K2402DK1/2005-1-I</em> (DENV1) was diluted with 2 mL pure DMEM media and mixed via pipetting up and down 5 times. Four milliliters of diluted virus was pipetted into each flask and incubated at 37°C for 1 h. Flasks were rocked every 15 min. Subsequently, inoculum was removed via pipette and 30 mL of pure DMEM media was pipetted to each flask was incubated at 37°C for 30 h.
     </p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>Trizol extraction of total RNA</h3>
     <p id="p-10">Extractions were separately performed on viral supernatant and cell lysates. Media of infected flasks (viral supernatant) was transferred to a 50 mL Falcon tube. Subsequently the cell monolayer of each flask was treated with 8 mL of Trizol LS Reagent (Invitrogen) and 2.25 mL of 1x PBS (Thermo Fisher Scientific) followed by incubation at 37°C for 10 min. One milliliter aliquots of this cell lysate mixture were then pipetted to 1.5 mL Eppendorf tubes. 250 μL aliquots of viral supernatant were also pipetted into individual tubes and treated with 750 μL of Trizol LS Regeant. 200 μL of chloroform was then added to individual aliquots of viral supernatant and cell lysate before incubation at room temperature for 10 min. Tubes were shaken vigorously by hand for 30 s, and subsequently inverted by hand every 3 min. Each tube was centrifuged at 12 000 g, 4°C for 15 min. Upper aqueous phase in each tube was carefully removed and pipetted to a new 1.5 mL Eppendorf tube (approximately 500 μL). Two microliters of Glycoblue Coprecipitant (Invitrogen) and 500 μL of isopropanol (Sigma Aldrich) was added to each new tube and stored in −80°C overnight (or longer). Prior to the capture step, tubes were centrifuged at 12 000 g, 4°C for 15 min and supernatant was discarded. Pellet was re-suspended in 500 μL 75% ethanol diluted with nuclease-free water. Tubes were centrifuged again at 7 500 g, 4°C for 3 min and supernatant was discarded. Pellet was air dried. For the cell lysate RNA, total RNA concentration was determined via Nanodrop (Thermo Fisher Scientific) and re-precipitated in ethanol to obtain 30 μg total RNA pellets. Viral supernatant RNA either underwent capture or direct RNA sequencing library preparation without re-precipitation.</p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Bait design</h3>
     <p id="p-11">Twenty-one DENV1 baits were designed using BaitMaker (PLoS Neglected Tropical Diseases, Kamaraj et al. 2019, in press). The bait stock in our experiments was previously used for hybridization-based target enrichment in a published study by The Singapore Zika Study Group (
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Ho et al. 2017</a>). Briefly, 120mer biotinylated DNA baits (Integrated DNA Technologies) specific to conserved regions across all serotypes of 
      <em>dengue</em> virus were used. Baits were tiled along the viral genome with intervals of 500nt. A total of 6 pmol of baits were added to each capture.
     </p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Viral RNA capture</h3>
     <p id="p-12">This protocol was modified from that for the use of xGEN Lockdown baits and reagents (Integrated DNA Technologies), optimized for our purposes of RNA capture. SeqCap Hybridization and Wash Kit (Roche) was used in place of Integrated DNA Technologies reagents. 
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> demonstrates the workflow for the capture protocol.
     </p>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/01/410282/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Capture-based purification workflow." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-21027053" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;em&gt;;Capture-based purification workflow&lt;;/em&gt;;.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/04/01/410282/F1.medium.gif" width="440" height="127"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2019/04/01/410282/F1.medium.gif" width="440" height="127"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/01/410282/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/01/410282/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 1:</span>
       <p id="p-13" class="first-child">
        <em>Capture-based purification workflow</em>.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="sec-7" class="subsection">
      <h4>Hybridization</h4>
      <p id="p-14">RNA Pellet from the extraction step was re-eluted with reagents shown in Table S1 and incubated at 65°C for 4 h in a thermocycler.</p>
     </div>
     <div id="sec-8" class="subsection">
      <h4>Streptavidin-Biotin Binding</h4>
      <p id="p-15">100 μL of Dynabeads M-270 Streptavidin beads (Thermo Fisher Scientific) was pipetted to a 1.5 mL Eppendorf tube and equilibrated at room temperature for 30 min. The tube was put on a magnetic rack (Invitrogen) until eluate was clear and supernatant was discarded. 200 μL 1x Bead Wash Buffer was added and mixture was vortexed for 10 s before placing it on a magnetic rack and discarding the supernatant for a total of two washes. Hybridization mixture was immediately added to the beads and incubated at 65°C for 45 min.</p>
     </div>
     <div id="sec-9" class="subsection">
      <h4>Stringent Washes</h4>
      <p id="p-16">All wash buffers were diluted to 1x prior to washing. Wash Buffer I and Stringent Wash Buffer was pre-heated for 1 hr to 65°C. 100 μL of Wash buffer I was added to the hybridization-bead mixture, agitated by hand and transferred to a new 1.5 mL Eppendorf tube. The new tube is placed on a magnetic rack. Once eluate is clear, supernatant was quickly discarded, taking caution not to remove any beads. 200 μL of Stringent Wash Buffer was added to the pellet and incubated at 65°C for 5 min, before discarding the supernatant as per the method above, for a total of two washes. Subsequent washes were performed as per the method above without the incubation step in the following sequence: Wash Buffer I, Wash Buffer II, and Wash Buffer III.</p>
     </div>
     <div id="sec-10" class="subsection">
      <h4>Dehybridization</h4>
      <p id="p-17">Viral RNA was released from the beads via DNase I treatment. Beads were re-suspended in 20 μL of nuclease-free water, incubated at 95°C for 3 min then cooled to 37°C. Subsequently, 2.7 μL of 10x reaction buffer and 4 μL of TURBO DNase (Thermo Fisher Scientific) was added to the reaction and mixed by pipetting up and down 5 times, followed by further incubation at 37°C for 40 min. Subsequently, the tube was placed on a magnetic rack to allow the beads to aggregate. The supernatant was then transferred to a clean 1.5 mL Eppendorf tube for a 2.2x Agencourt RNAClean XP beads clean-up (Beckman Coulter). Briefly, 58.7 μL of beads were added to the supernatant and incubated for 8 min at room temperature, and then 2 min on a magnetic rack. The beads were washed twice with 75% Ethanol and residual ethanol was air-dried. The beads were then re-eluted in 30 μL of nuclease-free water.</p>
     </div>
    </div>
    <div id="sec-11" class="subsection">
     <h3>RT-qPCR</h3>
     <p id="p-18">The proof of concept for our capture-based purification method was to demonstrate that host RNA background could be removed from target viral RNA. Absolute quantification of host and viral RNA was done using RT-qPCR. Two 1.5 ml Eppendorf tubes containing 30 μg of total RNA from cell lysate were prepared to represent pre and post-capture groups. 15 μL and 10 μL of nuclease-free water was added to the pre and post-capture group respectively. QuantiTect Probe RT-PCR Kit was used according to the manufacturer’s instructions. Primer targets were selected in the nonstructural 1 protein region of DENV1 and human GAPDH (glyceraldehyde 3-phosphate dehydrogenase) as a reference gene.</p>
     <p id="p-19">The volumes of reagents, shown in Table S2, were added to a 0.2 mL thin-walled PCR tube, and incubated in a ViiA 7 Real Time-PCR System (Applied Biosystems) with the program shown in Table S3 and analyzed using QuantStudio Real-Time PCR Software v1.1 (Applied Biosystems). Each plate included a non-template control for each pair of primers to check for PCR contamination. Samples were quantified in duplicate and non-template controls in triplicate.</p>
     <p id="p-20">A standard curve was constructed using a serial dilution of an oligo complementary to the primer target in accordance with MIQE Guidelines (
      <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Taylor et al. 2010</a>). Custom oligoes of primer targets were ordered from Integrated DNA Technologies. Serial dilution was performed with nuclease-free water to obtain concentrations of 2.41×10
      <sup>n</sup> copies/μL (where n=9,8,7,…,2). Each dilution was quantified in duplicate. The sequences of the oligoes, primers and probes used are shown in Table S4. Average C
      <sub>q</sub> values for the serial dilutions were linearly regressed on log
      <sub>2</sub>(copies/μL) and Pearson’s product-moment correlation coefficient was calculated. 
      <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Equation (1)</a> was used to determine primer efficiency: 
      <span class="disp-formula" id="disp-formula-1">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/04/01/410282/embed/graphic-2.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/04/01/410282/embed/graphic-2.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-21">Where m is the gradient of the regression line of &lt;;C
      <sub>q</sub>&gt;; on log
      <sub>2</sub>(copies/μl)
     </p>
     <p id="p-22">Absolute quantity (copies/μL) was calculated using the respective equations of the regression lines.</p>
     <p id="p-23">The absolute quantity of DENV1 RNA, normalized to human GAPDH reference gene, was then compared between pre and post-capture groups. The ratio of the absolute quantity of DENV1 and human RNA pre and post-capture was compared, and purification factor calculated using 
      <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">Equation (2)</a>.
     </p>
     <span class="disp-formula" id="disp-formula-2">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/04/01/410282/embed/graphic-3.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/04/01/410282/embed/graphic-3.gif"/>
       </noscript>
      </span>
     </span>
     <p id="p-24">In this study we defined purification factor as a measure of how well host RNA background is removed from viral RNA.</p>
    </div>
    <div id="sec-12" class="subsection">
     <h3>MinION Sequencing</h3>
     <p id="p-25">Pre and post-capture RNA extracted from supernatant were sequenced on the MinION. Separately, we pooled 6 captures to generate the concentrated post-cpature group. Six captures each consisiting of 30uL of purified RNA was added to the same RNAeasy MiniKit (Qiagen) column and final elution volume was 30uL. Prior to library preparation, all RNA samples were polyadenylated using E. Coli Poly(A) Polymerase (New England Biolabs). The volumes of reagents shown in Table S5 were added to a thin-walled 0.5 mL PCR tube.</p>
     <p id="p-26">The reaction was then incubated in a thermocycler with the following program: 37°C for 30 min then cooled to 4°C before proceeding to a 2.2x Agencourt RNAClean XP cleanup (Beckman Coulter). The RNA was eluted in 11.5 μL nuclease-free water. Subsequently, library preparation and sequencing was performed according to the protocol for Direct RNA Sequencing Kit (SQK-RNA001) using 9.5 μL of RNA. All ligation reactions were extended to 20 min. The RNA Library was loaded on a R9.4 MinION flow cell and sequencing was ran for 48 h using MinKNOW v1.14.1 (GUI 2.1.14) interface. Live basecalling was turned off.</p>
    </div>
    <div id="sec-13" class="subsection">
     <h3>Bioinformatic Analysis</h3>
     <p id="p-27">Raw fast5 files were basecalled and end adapters were trimmed using Poreplex v0.2 (
      <a href="https://github.com/hyeshik/poreplex">https://github.com/hyeshik/poreplex</a>) which internally calls Albacore v2.3.1 (Oxford Nanopore Technologies). Basecalled read metrics and plots were then generated using NanoPlot v1.0.0 (
      <a href="https://github.com/wdecoster/NanoPlot">https://github.com/wdecoster/NanoPlot</a>). Subsequently, mapping and alignment was performed using Graphmap v0.5.2 (
      <a href="https://github.com/isovic/graphmap">https://github.com/isovic/graphmap</a>) with default parameters. Mean coverage and mapping quality was calculated using Qualimap v0.2.0 (
      <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">García-Alcalde et al. 2012</a>) respectively. Viral and human genome references used were EU081230.1 and GRCh38 respectively.
     </p>
    </div>
   </div>
   <div class="section" id="sec-14">
    <h2 class="">RESULTS</h2>
    <div id="sec-15" class="subsection">
     <h3>RT-qPCR</h3>
     <p id="p-28">Based on the ratios of the absolute quantities of viral to host RNA shown in Table 1, we report a 561-791 purification factor, suggesting successful retrieval of viral RNA and substantial removal of host RNA background. Capture yield, which is defined as the percentage of viral RNA amount captured, was calculated to be 6.98%-40.4%.</p>
     <p id="p-29">Figure S1 shows the standard curves for DENV1 and GAPDH primers. The primer efficiency of DENV1 and GAPDH was 0.66 and 0.92 respectively. The DENV1 primer set used was a pre-optimized primer set that is able to detect all 4 
      <em>dengue</em> virus serotypes previously used and therefore contains ambiguous bases, explaining the lower primer efficiency. It was noted that relative quantification using the 2
      <sup>-ΔΔCT</sup> method (
      <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Schmittgen and Livak 2008</a>) was not appropriate as the primer efficiencies of DENV1 and GAPDH primer sets were not comparable. Absolute quantification and subsequent ratio comparison of viral and host RNA amounts is therefore more accurate in the determination of purification factor. All obtained values were within the dynamic linear range and limit of detection of the reaction conditions and primer sets used, suggesting reliability and accuracy of the reported RT-qPCR results.
     </p>
    </div>
    <div id="sec-16" class="subsection">
     <h3>MinION sequencing</h3>
     <p id="p-30">Based on the mapping rates to human and DENV1 reference genomes for pre and post-capture groups, shown in Table 2, purification factor was calculated to be 272 fold. We also report a 3.89 fold increase in percentage recovery of DENV1 genome over 15x coverage from a single pre-capture to a single post-capture RNA sample. On the other hand, comparing the concentrated post-capture group (containing a pool of 6 captures) to the pre-capture group yielded a purification factor of 1580 fold and a 14 fold increase in the percentage recovery of DENV1 genome above with at least 15x coverage from 7.05% to 99.91%. 
      <em>Supplementary figures 2, 3, and 4</em> show representative coverage plots for the pre-capture, post-capture and concentrated post-capture groups respectively.
     </p>
     <p id="p-31">In 
      <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">Equation (3)</a> and 
      <a id="xref-disp-formula-4-1" class="xref-disp-formula" href="#disp-formula-4">(4)</a>, a binomial distribution was used to determine the minimum read coverage, 
      <em>n</em>, required to have a 0.95 confidence that 
      <em>k</em> reads mapping to a particular base, 
      <em>X</em>, are correctly called. We used the average error read rate of our runs as the probability, 
      <em>p</em>, of correctly calling and mapping a single base.
     </p>
     <span class="disp-formula" id="disp-formula-3">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/04/01/410282/embed/graphic-4.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/04/01/410282/embed/graphic-4.gif"/>
       </noscript>
      </span>
     </span>
     <span class="disp-formula" id="disp-formula-4">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/04/01/410282/embed/graphic-5.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/04/01/410282/embed/graphic-5.gif"/>
       </noscript>
      </span>
     </span>
     <p id="p-32">Since the common heuristic for variant calling with Illumina reads requires 10x coverage (Illumina, 
      <a href="https://www.illumina.com/Documents/products/technotes/technote_snp_caller_sequencing.pdf">https://www.illumina.com/Documents/products/technotes/technote_snp_caller_sequencing.pdf</a>), we assumed the criterion that at least 10 reads were correctly called and mapped to a single base (
      <em>k</em>=10).
     </p>
     <span class="disp-formula" id="disp-formula-5">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/04/01/410282/embed/graphic-6.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/04/01/410282/embed/graphic-6.gif"/>
       </noscript>
      </span>
     </span>
     <p id="p-33">Based on our sequencing runs, the average error read rate of pre and post-capture sequencing runs was 17.1±0.8% (calculated from Table 2). With this error read rate, we substitute 1-
      <em>p</em>=0.171 into 
      <a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">Equation (5)</a> and determined that the minimum coverage, 
      <em>n</em>, required for variant calling to be at least 15x.
     </p>
    </div>
   </div>
   <div class="section" id="sec-17">
    <h2 class="">DISCUSSION</h2>
    <div id="sec-18" class="subsection">
     <h3>Capture-based purification removes high host background</h3>
     <p id="p-34">We demonstrated with RT-qPCR that our novel RNA capture technique can be used to retrieve and purify viral RNA in its native form for downstream assays. This was corroborated by our direct RNA sequencing results. Indeed, unpurified viral RNA represented by the pre-capture group did not produce sufficient throughput for any meaningful downstream analysis due to the indiscriminate nature of the sequencing of both host and viral RNA. However, after treatment with our capture method, the amount of host RNA background has decreased greatly as seen from the 272 fold increase in mapping rates to target viral RNA. In comparison, even with the incorporation of PCR amplification for shotgun sequencing of 
      <em>enterovirus</em>, 
      <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Jensen et al. (2015)</a> reported only a 2-13 fold increase in mapping rates to 
      <em>enterovirus</em>. Moreover, 
      <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Wongsurawat et al. (2019)</a> observed only a 36 fold increase in mapping rates to Zika virus (ZIKV). This suggests that our capture-based purification method is highly successful in the removal of host RNA background. The minimum coverage required for variant calling was, as described above, benchmarked to that of Illumina reads so that the effectiveness of our capture-based purification method could be more accurately evaluated based on the higher error read rates of direct RNA sequencing technology. Even with the higher adjusted heuristic for variant calling of 15x, the 3.89 fold increase in percentage of the genome above 15x coverage suggests that application of our capture-based purification method increases the amount of target genetic information retrieved.
     </p>
    </div>
    <div id="sec-19" class="subsection">
     <h3>Capture-based purification is robust</h3>
     <p id="p-35">Our method fundamentally allows for the capture of unknown viral strains and genotypes due to the flexibility of adding multiple baits complementary to different regions of the viral genome, or even multiple baits specific to different viruses in a single capture. Indeed, 
      <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Deviatkin et al. (2017)</a> has reported the successful use of genus-specific degenerate baits for hybridization-based enrichment, which can be seamlessly incorporated into our bait design. By extension, the baits used can be appropriately modified according to the intended research focus. The robustness of this capture-based purification method therefore not only broadens the applicability of direct RNA sequencing on studying unknown viruses but also the applicability of a diverse range of direct RNA probing technologies.
     </p>
     <p id="p-36">Another major advantage to our capture-based purification method is that large machinery like an ultracentrifuge is not required. To perform our capture method, only a thermocycler and a heat block is required. This is especially applicable to research environments where an ultracentrifuge is not readily available. More fundamentally, our capture-based purification method allows for the capture of both unassembled and assembled viruses, which any method leveraging on the differential physiochemical properties of encapsulated viral RNAs cannot. This would be important in studies of unassembled RNA segments, which have been demonstrated to be of great biological importance. Indeed, in the case of DENV1, subgenomic flaviviral particles have been shown to underlie mosquito immunity and viral transmission (
      <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Pompon et al. 2017</a>). If so, a method which can remove host background and capture unassembled viruses would be useful for characterising inherent RNA properties of unassembled viruses.
     </p>
    </div>
    <div id="sec-20" class="subsection">
     <h3>Limitations</h3>
     <p id="p-37">One point of concern for our protocol is the low capture yield. However, it must be noted that the capture yield is still significantly higher than that of conventional viral purification methods. 
      <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Hall et al. (2014)</a>, in a study evaluating conventional viral purification methods, reported a 100 fold decrease in viral copy number post-treatment, whereas our method results in, at most, a 14 fold decrease in viral copy number. In evaluating our capture-based purification method, a wide range in capture yield was observed, which was due to the high sensitivity of RNA to potential contamination during handling. This is a problem common to any viral RNA purification method.
     </p>
     <p id="p-38">Our capture-based purification method has a turnaround time of approximately 6 h 15 min (
      <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>), comparable to the time required for isopycnic gradient-based methods (
      <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Buclez et al. 2016</a>). While there is a significant loss of viral RNA post capture, we also note that our capture method is scalable. Multiple captures can be performed and purified RNA pooled together before re-concentrating in the final bead clean-up step. The concentrated post-capture group was a demonstration of how our capture-based purification method was appropriately scaled to suit the sensitivity of direct RNA sequencing. Indeed, after comparison of the post-capture and concentrated post-capture sequencing runs (Table 2), the 2.5 fold increase in the percentage of reads mapping to DENV1 suggests that scaling our method greatly improved the signal-to-noise ratio of this particular downstream RNA assay. While it was opined that this improvement would similarly be observed in other downstream RNA assays, further experimental verification is required.
     </p>
    </div>
    <div id="sec-21" class="subsection">
     <h3>Application of capture-based purification to direct RNA sequencing</h3>
     <p id="p-39">The use of direct RNA sequencing to study viral genomes is limited by its sensitivity. Conventionally, the sequencing adapter used in direct RNA sequencing was a poly(T) adapter which would hybridize with poly(A) tails, facilitating adapter ligation (
      <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Garalde et al. 2018</a>). This poly(T) adapter is also the sequencing adapter provided in the direct RNA sequencing library preparation kit (SQK-RNA001). Currently, four strategies may be employed for direct RNA sequencing of RNA viruses. Firstly, as described by 
      <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Keller et al. (2018)</a>, a custom adapter can be used during library preparation. The second strategy is the use of the conventional poly(T) adapter supplied in the kit by polyadenylation of RNA prior to library preparation and sequencing. The third strategy is to couple polyadenylation with capture-based purification. The fourth is to couple polyadenylation with host rRNA depletion (
      <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Wongsurawat et al. 2019</a>). A summary of the advantages and disadvantages for these strategies are presented in Table 3.
     </p>
     <p id="p-40">The custom adapter strategy employed by 
      <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Keller et al. (2018)</a> greatly improves sequencing specificity, and was demonstrated to yield 98.8% of reads mapping to 
      <em>influenza A</em>. This strategy allows sequencing of RNA viruses with or without an inherent poly(A) tail since the conventional poly(T) adapter was substituted with a custom sequencing adapter specific to the target. However, one drawback of this strategy is that foreknowledge of the 3’ end of RNA targets is required. This entails that sequencing of unknown, novel, or mutant viruses would be difficult. Another limitation is that this strategy may be ineffective in cases where the RNA sample is fragmented since the 3’ end of each fragment of viral RNA is distinct.
     </p>
     <p id="p-41">Degenerate adapters may be used to circumvent the first limitation, as with the degenerate adapters at the +4 position used by 
      <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Keller et al. (2018)</a>. However, using degenerate adapters would only be able to address the problem of mutant viruses but not that of novel or unknown viruses. Indeed, foreknowledge of target 3’ sequences is still required. On the other hand, attempts at using multiple adapters at different locations on the target genome cannot address the problem of RNA fragmentation. Custom adapters must be specific to the 3’ end of each fragment for ligation. In most laboratory samples, however, RNA is stochastically fragmented. As such, determination of the exact 3’ end of each fragment would be impossible and so such methods to circumvent the problem of RNA fragmentation are unviable.
     </p>
     <p id="p-42">In this study, we demonstrated the second and third strategies, which make use of the conventional poly(T) adapter. This was done using DENV1, a positive single stranded RNA virus lacking a poly(A) tail. The protocol used to generate the pre-capture group sequencing results employ the second strategy while that for the post-capture and concentrated post-capture results employ the third. The polyadenylation of RNA and subsequent use of the conventional poly(T) adapter has several advantages. First, the polyadenylation of RNA prior to library preparation and sequencing allows for sequencing of RNA viruses with or without an inherent poly(A) tail. Second, usage of the conventional poly(T) adapter removes the need for any foreknowledge of the target sequence. Third, all RNA fragments would be ligated with adapters since the adapters are not sequence specific, circumventing the problem of fragmented RNA. However, the main disadvantage strategies involving polyadenylation of RNA is that sequencing specificity is greatly reduced as host mRNA transcripts include a poly(A) tail and will also be sequenced. 
      <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Wongsurawat et al. (2019)</a> independently replicated our pre-capture group results but with ZIKV, which is also a positive single stranded RNA virus lacking a poly(A) tail in the family Flaviviridae. The authors observed 0.11% of reads mapping to ZIKV, which corroborates the low 0.63% of reads mapping to DENV1 in our pre-capture sequencing run. The authors also reported 3.89% of reads mapping to ZIKV whilst employing the fourth strategy. The low percentage of target reads underscores this main disadvantage of sequencing specificity.
     </p>
     <p id="p-43">The issue of low sequencing specificity is, however, less pronounced in the third strategy. Comparing the second strategy (pre-capture) and third strategy (post-capture), there is a 50 fold increase in percentage of reads mapping to DENV1. Furthermore, a comparison of pre-capture and concentrated post-capture results yields a 123 fold increase in mapping rates to DENV1. This suggests that while simply polyadenylating RNA samples leads to low sequencing specificity, applying our capture-based purification prior allows us to reap the advantages of using the conventional poly(T) adapter whilst having a significantly higher sequence specificity. Additionally, coupling capture-based purification with polyadenylation yields a higher target mapping rate of 31.31% with regard to the 3.98% observed when the fourth strategy was employed (
      <a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Wongsurawat et al. 2019</a>). This suggests that the third strategy is superior with respect to sequencing specificity when compared to the fourth strategy. It is noted, however, that the sequencing specificity of the third strategy was still lower than that of the first strategy. Moreover, applying capture-based purification in the third strategy significantly increases the turnaround time as compared to the first, second and fourth strategy. Therefore, it is suggested that the third strategy should only be employed where foreknowledge of the viral sequence at hand is limited, or where RNA integrity is low. It is also suggested that the second and fourth strategy be used in cases where detection rather than characterisation of RNA viruses is required due to significantly lower sequencing specificity.
     </p>
     <p id="p-44">In this study, we demonstrated a novel capture-based method for the purification of viral RNA from host RNA. In addition, we described a successful attempt of applying our capture method to the direct RNA sequencing of viral culture, using a DENV1 model, exemplifying how our capture method could significantly remove host background in downstream assays requiring viral RNA in its native form. Further improvements to the capture yield are underway and could potentially expand its application to clinical samples where starting RNA material is low. The fundamental advantages of using our capture-based purification method makes it a superior alternative to conventional viral purification methods and addresses limitations of current strategies for direct RNA sequencing, potentially paving a new path for the characterization and direct RNA sequencing of viral RNA.</p>
    </div>
   </div>
   <div class="section" id="sec-23">
    <h2 class="">DECLARATIONS</h2>
    <div class="section" id="sec-24">
     <h3>Authors’ contributions</h3>
     <p id="p-45">CSC Tan was the primary contributor to this study with guidance from his mentor PF de Sessions and helpful advice from OM Sessions, S Maurer-Stroh and Y Wan.</p>
    </div>
   </div>
   <div class="section COI-statement" id="sec-25">
    <h2 class="">Competing interests</h2>
    <p id="p-47">The authors declare that they have no competing interests.</p>
   </div>
   <div class="section" id="sec-26">
    <h2 class="">Availability of data and materials</h2>
    <p id="p-48">Data will be made available through publication and NCBI.</p>
   </div>
   <div class="section" id="sec-27">
    <h2 class="">Compliance with ethical standards</h2>
    <p id="p-49">This article does not contain any studies with human participants or animals performed by the author.</p>
   </div>
   <div class="section" id="sec-28">
    <h2 class="">Consent for publication</h2>
    <p id="p-50">All authors listed on this manuscript have read and agreed to the publication of this research.</p>
   </div>
   <div class="section" id="sec-29">
    <h2 class="">Funding</h2>
    <p id="p-51">This study was funded by the Biomedical Research Council, A*STAR, Singapore (Human Emerging Infectious Disease Initiative (HEIDI), Ref No: H16/99/b0/013).</p>
   </div>
   <div class="section" id="sec-30">
    <h2 class="">FIGURE LEGENDS</h2>
    <p id="p-52">Table 1: Comparison of absolute quantities of DENV1 and GAPDH used for calculation of purification factor.</p>
    <p id="p-53">Table 2: Summary of bioinformatic analysis for pre and post-capture MinION sequencing runs.</p>
    <p id="p-54">Table 3: Summary of advantages and disadvantages of strategies for direct RNA sequencing of RNA viruses.</p>
    <p id="p-55">Table S1: Reagents and volumes for hybridization step.</p>
    <p id="p-56">Table S2: Reagents and volumes for RT-qPCR.</p>
    <p id="p-57">Table S3: RT-qPCR program set on Applied Biosystems ViiA 7 Real Time-PCR System.</p>
    <p id="p-58">Table S4: Primers, probes and standards used with the respective modifications for RT-qPCR.</p>
    <p id="p-59">Table S5: Reagents and volumes used for polyadenylation of RNA samples.</p>
    <p id="p-60">Figure S1: DENV1 and GAPDH standard curves used for calculation of primer efficiency.</p>
    <p id="p-61">Figure S2: Coverage plot against nucleotide position on DENV1 reference for pre-capture MinION sequencing run.</p>
    <p id="p-62">Figure S3: Coverage plot against nucleotide position on DENV1 reference for post-capture MinION sequencing run.</p>
    <p id="p-63">Figure S4: Coverage plot against nucleotide position on DENV1 reference for concentrated post-capture MinION sequencing run.</p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgements</h2>
    <p id="p-46">We would like to acknowledge Ng Hui Qi Amanda and Niranjan Nagarajan from Computational Systems Biology 5, GIS; Aw Poh Kim Pauline (DENV1 primer design), Kwan Kam Weng Philip, Balamurugan Periaswamy, Png Eileen from GERMS Platform for their technical advice throughout the course of this study; Vithiagaran Gunalan and Dimitar Kenanov from Maurer-Stroh’s group for helpful conversations. We would also like to acknowledge members of Wan Yue’s team, in particular, Lim Wei Sheng Shaun, Aw Jong Ghut Ashley. Lastly, we would like to thank Andreas Wilm and Shih Chih Chuan from the GIS Bioinformatics Core.</p>
   </div>
   <div class="section fn-group" id="fn-group-1">
    <h2>Footnotes</h2>
    <ul>
     <li class="fn-other" id="fn-1">
      <p id="p-1">S Maurer-Stroh (
       <span class="em-link">
        <span class="em-addr">sebastianms{at}bii.a-star.edu.sg</span>
       </span>), Y Wan (
       <span class="em-link">
        <span class="em-addr">wany{at}gis.a-star.edu.sg</span>
       </span>), O M Sessions (
       <span class="em-link">
        <span class="em-addr">october.sessions{at}nus.edu.sg</span>
       </span>), P F de Sessions (
       <span class="em-link">
        <span class="em-addr">florezdesessions{at}gmail.com</span>
       </span>)
      </p>
     </li>
    </ul>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">REFERENCES</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-410282v3.1" data-doi="10.1128/AEM.71.12.8966-8969.2005">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Acinas</span>
          <span class="cit-name-given-names">SG</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sarma-Rupavtarm</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Klepac-Ceraj</span>
          <span class="cit-name-given-names">V</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Polz</span>
          <span class="cit-name-given-names">MF</span>
         </span> (
         <span class="cit-pub-date">2005</span>) 
         <span class="cit-article-title">PCR-induced sequence artifacts and bias: Insights from comparison of two 16s rRNA clone libraries constructed from the same sample</span>. 
         <abbr class="cit-jnl-abbrev">Applied and Environmental Microbiology</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1128/AEM.71.12.8966-8969.2005</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft.stitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft.aulast%253DAcinas%26rft.auinit1%253DS.%2BG.%26rft.volume%253D71%26rft.issue%253D12%26rft.spage%253D8966%26rft.epage%253D8969%26rft.atitle%253DPCR-Induced%2BSequence%2BArtifacts%2Band%2BBias%253A%2BInsights%2Bfrom%2BComparison%2Bof%2BTwo%2B16S%2BrRNA%2BClone%2BLibraries%2BConstructed%2Bfrom%2Bthe%2BSame%2BSample%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.71.12.8966-8969.2005%26rft_id%253Dinfo%253Apmid%252F16332901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI3MS8xMi84OTY2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDQvMDEvNDEwMjgyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-410282v3.2" data-doi="10.1038/mtm.2016.35">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Buclez</span>
          <span class="cit-name-given-names">PO</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dias Florencio</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Relizani</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Beley</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Garcia</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Benchaouir</span>
          <span class="cit-name-given-names">R</span>
         </span> (
         <span class="cit-pub-date">2016</span>) 
         <span class="cit-article-title">Rapid, scalable, and low-cost purification of recombinant adeno-associated virus produced by baculovirus expression vector system</span>. 
         <abbr class="cit-jnl-abbrev">Molecular Therapy - Methods and Clinical Development</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/mtm.2016.35</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMolecular%2BTherapy%2B-%2BMethods%2Band%2BClinical%2BDevelopment%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmtm.2016.35%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/mtm.2016.35&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-410282v3.3" data-doi="10.1016/j.virusres.2014.06.007">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Cantara</span>
          <span class="cit-name-given-names">WA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Olson</span>
          <span class="cit-name-given-names">ED</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Musier-Forsyth</span>
          <span class="cit-name-given-names">K</span>
         </span> (
         <span class="cit-pub-date">2014</span>) 
         <span class="cit-article-title">Progress and outlook in structural biology of large viral RNAs</span>. 
         <abbr class="cit-jnl-abbrev">Virus Research</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1016/j.virusres.2014.06.007</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVirus%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2014.06.007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.virusres.2014.06.007&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-410282v3.4" data-doi="10.1016/j.tibtech.2005.04.001">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Cowan</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Meyer</span>
          <span class="cit-name-given-names">Q</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Stafford</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Muyanga</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cameron</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wittwer</span>
          <span class="cit-name-given-names">P</span>
         </span> (
         <span class="cit-pub-date">2005</span>) 
         <span class="cit-article-title">Metagenomic gene discovery: Past, present and future</span>. 
         <abbr class="cit-jnl-abbrev">Trends in Biotechnology</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1016/j.tibtech.2005.04.001</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bbiotechnology%26rft.stitle%253DTrends%2BBiotechnol%26rft.aulast%253DCowan%26rft.auinit1%253DD.%26rft.volume%253D23%26rft.issue%253D6%26rft.spage%253D321%26rft.epage%253D329%26rft.atitle%253DMetagenomic%2Bgene%2Bdiscovery%253A%2Bpast%252C%2Bpresent%2Band%2Bfuture.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibtech.2005.04.001%26rft_id%253Dinfo%253Apmid%252F15922085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.tibtech.2005.04.001&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15922085&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F01%2F410282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000229886500009&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-410282v3.5" data-doi="10.1073/pnas.1716689115">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Dethoff</span>
          <span class="cit-name-given-names">EA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Boerneke</span>
          <span class="cit-name-given-names">MA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gokhale</span>
          <span class="cit-name-given-names">NS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Muhire</span>
          <span class="cit-name-given-names">BM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Martin</span>
          <span class="cit-name-given-names">DP</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sacco</span>
          <span class="cit-name-given-names">MT</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">McFadden</span>
          <span class="cit-name-given-names">MJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Weinstein</span>
          <span class="cit-name-given-names">JB</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Messer</span>
          <span class="cit-name-given-names">WB</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Horner</span>
          <span class="cit-name-given-names">SM</span>
         </span> (
         <span class="cit-pub-date">2018</span>) 
         <span class="cit-article-title">Pervasive tertiary structure in the dengue virus RNA genome</span>. 
         <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr>
         <span class="cit-vol">115</span>(
         <span class="cit-issue">45</span>)National Acad Sciences, : 
         <span class="cit-fpage">11513</span>–
         <span class="cit-lpage">11518</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1716689115%26rft_id%253Dinfo%253Apmid%252F30341219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE1LzQ1LzExNTEzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDQvMDEvNDEwMjgyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-410282v3.6" data-doi="10.1038/s41598-017-10342-w">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Deviatkin</span>
          <span class="cit-name-given-names">AA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lukashev</span>
          <span class="cit-name-given-names">AN</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Markelov</span>
          <span class="cit-name-given-names">MM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gmyl L</span>
          <span class="cit-name-given-names">V.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shipulin</span>
          <span class="cit-name-given-names">GA</span>
         </span> (
         <span class="cit-pub-date">2017</span>) 
         <span class="cit-article-title">Enrichment of Viral Nucleic Acids by Solution Hybrid Selection with Genus Specific Oligonucleotides</span>. 
         <abbr class="cit-jnl-abbrev">Scientific Reports</abbr>
         <span class="cit-vol">7</span>(
         <span class="cit-issue">1</span>)Springer US, : 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">7</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/s41598-017-10342-w</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScientific%2BReports%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-017-10342-w%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41598-017-10342-w&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-410282v3.7" data-doi="10.1038/nmeth.4577">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Garalde</span>
          <span class="cit-name-given-names">DR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Snell</span>
          <span class="cit-name-given-names">EA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jachimowicz</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sipos</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lloyd</span>
          <span class="cit-name-given-names">JH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bruce</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pantic</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Admassu</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">James</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Warland</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jordan</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ciccone</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Serra</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Keenan</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Martin</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">McNeill</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wallace</span>
          <span class="cit-name-given-names">EJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jayasinghe</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wright</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Blasco</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Young</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Brocklebank</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Juul</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Clarke</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Heron</span>
          <span class="cit-name-given-names">AJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Turner</span>
          <span class="cit-name-given-names">DJ</span>
         </span> (
         <span class="cit-pub-date">2018</span>) 
         <span class="cit-article-title">Highly parallel direct RN A sequencing on an array of nanopores</span>. 
         <abbr class="cit-jnl-abbrev">Nature Methods</abbr>
         <span class="cit-vol">15</span>(
         <span class="cit-issue">3</span>): 
         <span class="cit-fpage">201</span>–
         <span class="cit-lpage">206</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/nmeth.4577</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2BMethods%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.4577%26rft_id%253Dinfo%253Apmid%252F29334379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nmeth.4577&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=29334379&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F01%2F410282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-410282v3.8" data-doi="10.1093/bioinformatics/bts503">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">García-Alcalde</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Okonechnikov</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Carbonell</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cruz</span>
          <span class="cit-name-given-names">LM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Götz</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tarazona</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dopazo</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Meyer</span>
          <span class="cit-name-given-names">TF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Conesa</span>
          <span class="cit-name-given-names">A</span>
         </span> (
         <span class="cit-pub-date">2012</span>) 
         <span class="cit-article-title">Qualimap: Evaluating next-generation sequencing alignment data</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/bts503</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts503%26rft_id%253Dinfo%253Apmid%252F22914218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts503&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22914218&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F01%2F410282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000309881200016&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-410282v3.9" data-doi="10.1016/j.jviromet.2013.08.035">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hall</span>
          <span class="cit-name-given-names">RJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Todd</span>
          <span class="cit-name-given-names">AK</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bissielo</span>
          <span class="cit-name-given-names">AB</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yen</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Strydom</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Moore</span>
          <span class="cit-name-given-names">NE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ren</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>
          <span class="cit-name-given-names">QS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Carter</span>
          <span class="cit-name-given-names">PE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Peacey</span>
          <span class="cit-name-given-names">M</span>
         </span> (
         <span class="cit-pub-date">2014</span>) 
         <span class="cit-article-title">Evaluation of rapid and simple techniques for the enrichment of viruses prior to metagenomic virus discovery</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Virological Methods</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1016/j.jviromet.2013.08.035</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirological%2BMethods%26rft.volume%253D195%26rft.spage%253D194%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jviromet.2013.08.035%26rft_id%253Dinfo%253Apmid%252F24036074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.jviromet.2013.08.035&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24036074&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F01%2F410282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-410282v3.10" data-doi="10.1016/S1473-3099(17)30249-9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ho</span>
          <span class="cit-name-given-names">ZJM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hapuarachchi</span>
          <span class="cit-name-given-names">HC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Barkham</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chow</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ng</span>
          <span class="cit-name-given-names">LC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lee</span>
          <span class="cit-name-given-names">JMV</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Leo</span>
          <span class="cit-name-given-names">YS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Prem</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lim</span>
          <span class="cit-name-given-names">YHG</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">de Sessions</span>
          <span class="cit-name-given-names">PF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rabaa</span>
          <span class="cit-name-given-names">MA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chong</span>
          <span class="cit-name-given-names">CS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tan</span>
          <span class="cit-name-given-names">CH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rajarethinam</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tan</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Anderson</span>
          <span class="cit-name-given-names">DE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ong</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cook</span>
          <span class="cit-name-given-names">AR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chong</span>
          <span class="cit-name-given-names">CY</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hsu</span>
          <span class="cit-name-given-names">LY</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yap</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lai</span>
          <span class="cit-name-given-names">YL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chawla</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pan</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sim</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">ICM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Thoon</span>
          <span class="cit-name-given-names">KC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yung</span>
          <span class="cit-name-given-names">CF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">JH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ng</span>
          <span class="cit-name-given-names">HLD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nandar</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ooi</span>
          <span class="cit-name-given-names">PL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lin</span>
          <span class="cit-name-given-names">RTP</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Aw</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Uehara</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pratim De</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Soon</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hibberd</span>
          <span class="cit-name-given-names">ML</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ng</span>
          <span class="cit-name-given-names">HH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Maurer-Stroh</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sessions</span>
          <span class="cit-name-given-names">OM</span>
         </span> (
         <span class="cit-pub-date">2017</span>) 
         <span class="cit-article-title">Outbreak of Zika virus infection in Singapore: an epidemiological, entomological, virological, and clinical analysis</span>. 
         <abbr class="cit-jnl-abbrev">The Lancet Infectious Diseases</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1016/S1473-3099(17)30249-9</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BLancet%2BInfectious%2BDiseases%26rft.volume%253D17%26rft.spage%253D813%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1473-3099%252817%252930249-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S1473-3099(17)30249-9&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-410282v3.11" data-doi="10.1371/journal.pone.0131080">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jensen</span>
          <span class="cit-name-given-names">RH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mollerup</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mourier</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hansen</span>
          <span class="cit-name-given-names">TA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fridholm</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nielsen</span>
          <span class="cit-name-given-names">LP</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Willerslev</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hansen</span>
          <span class="cit-name-given-names">AJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vinner</span>
          <span class="cit-name-given-names">L</span>
         </span> (
         <span class="cit-pub-date">2015</span>) 
         <span class="cit-article-title">Target-dependent enrichment of virions determines the reduction of high-throughput sequencing in virus discovery</span>. 
         <abbr class="cit-jnl-abbrev">PLoS ONE</abbr>
         <span class="cit-vol">10</span>(
         <span class="cit-issue">4</span>): 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">18</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0122636</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D10%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0131080%26rft_id%253Dinfo%253Apmid%252F25919488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0131080&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25919488&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F01%2F410282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-410282v3.12" data-doi="10.1038/s41598-018-32615-8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Keller</span>
          <span class="cit-name-given-names">MW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rambo-Martin</span>
          <span class="cit-name-given-names">BL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wilson</span>
          <span class="cit-name-given-names">MM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ridenour</span>
          <span class="cit-name-given-names">CA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shepard</span>
          <span class="cit-name-given-names">SS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Stark</span>
          <span class="cit-name-given-names">TJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Neuhaus</span>
          <span class="cit-name-given-names">EB</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dugan</span>
          <span class="cit-name-given-names">VG</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wentworth</span>
          <span class="cit-name-given-names">DE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Barnes</span>
          <span class="cit-name-given-names">JR</span>
         </span> (
         <span class="cit-pub-date">2018</span>) 
         <span class="cit-article-title">Direct RNA Sequencing of the Coding Complete Influenza A Virus Genome</span>. 
         <abbr class="cit-jnl-abbrev">Scientific Reports</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/s41598-018-32615-8</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScientific%2BReports%26rft.volume%253D8%26rft.spage%253D14408%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-018-32615-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41598-018-32615-8&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-410282v3.13" data-doi="10.1371/journal.pone.0020005">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>
          <span class="cit-name-given-names">CC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Guo</span>
          <span class="cit-name-given-names">MS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lin</span>
          <span class="cit-name-given-names">FHY</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hsiao</span>
          <span class="cit-name-given-names">KN</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chang</span>
          <span class="cit-name-given-names">KHW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chou</span>
          <span class="cit-name-given-names">AH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">YC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">YC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">CS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chong</span>
          <span class="cit-name-given-names">PCS</span>
         </span> (
         <span class="cit-pub-date">2011</span>) 
         <span class="cit-article-title">Purification and characterization of enterovirus 71 viral particles produced from vero cells grown in a serum-free microcarrier bioreactor system</span>. 
         <abbr class="cit-jnl-abbrev">PLoS ONE</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0020005</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLiu%26rft.auinit1%253DC.%2BC.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De20005%26rft.epage%253De20005%26rft.atitle%253DPurification%2Band%2Bcharacterization%2Bof%2Benterovirus%2B71%2Bviral%2Bparticles%2Bproduced%2Bfrom%2Bvero%2Bcells%2Bgrown%2Bin%2Ba%2Bserum-free%2Bmicrocarrier%2Bbioreactor%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0020005%26rft_id%253Dinfo%253Apmid%252F21603631%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0020005&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21603631&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F01%2F410282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-410282v3.14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Low</span>
          <span class="cit-name-given-names">JGH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ooi</span>
          <span class="cit-name-given-names">EE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tolfvenstam</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Leo</span>
          <span class="cit-name-given-names">YS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hibberd</span>
          <span class="cit-name-given-names">ML</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ng</span>
          <span class="cit-name-given-names">LC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lai</span>
          <span class="cit-name-given-names">YL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yap</span>
          <span class="cit-name-given-names">GS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">CS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vasudevan</span>
          <span class="cit-name-given-names">SG</span>
         </span> (
         <span class="cit-pub-date">2006</span>) 
         <span class="cit-article-title">Early Dengue infection and outcome study (EDEN)-study design and preliminary findings</span>. 
         <abbr class="cit-jnl-abbrev">ANNALS-ACADEMY OF MEDICINE SINGAPORE</abbr>
         <span class="cit-vol">35</span>(
         <span class="cit-issue">11</span>)Academy of Medicine, Singapore; 1999, : 
         <span class="cit-fpage">783</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DANNALS-ACADEMY%2BOF%2BMEDICINE%2BSINGAPORE%26rft.volume%253D35%26rft.spage%253D783%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-410282v3.15" data-doi="10.1038/nmeth.1419">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Mamanova</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Coffey</span>
          <span class="cit-name-given-names">AJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Scott</span>
          <span class="cit-name-given-names">CE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kozarewa</span>
          <span class="cit-name-given-names">I</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Turner</span>
          <span class="cit-name-given-names">EH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kumar</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Howard</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shendure</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Turner</span>
          <span class="cit-name-given-names">DJ</span>
         </span> (
         <span class="cit-pub-date">2010</span>) 
         <span class="cit-article-title">Target-enrichment strategies for next-generation sequencing</span>. 
         <abbr class="cit-jnl-abbrev">Nature Methods</abbr>
         <span class="cit-vol">7</span>(
         <span class="cit-issue">2</span>)Nature Publishing Group, : 
         <span class="cit-fpage">111</span>–
         <span class="cit-lpage">118</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/nmeth.1419</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DMamanova%26rft.auinit1%253DL.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253D111%26rft.epage%253D118%26rft.atitle%253DTarget-enrichment%2Bstrategies%2Bfor%2Bnext-generation%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1419%26rft_id%253Dinfo%253Apmid%252F20111037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nmeth.1419&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20111037&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F01%2F410282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000274086200015&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-410282v3.16" data-doi="10.1007/BF03020102">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Manokaran</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Finol</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gunaratne</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bahl</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ong</span>
          <span class="cit-name-given-names">EZ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tan</span>
          <span class="cit-name-given-names">HC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sessions</span>
          <span class="cit-name-given-names">OM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ward</span>
          <span class="cit-name-given-names">AM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gubler</span>
          <span class="cit-name-given-names">DJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Harris</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Garcia-Blanco</span>
          <span class="cit-name-given-names">MA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ooi</span>
          <span class="cit-name-given-names">EE</span>
         </span> (
         <span class="cit-pub-date">2015</span>) 
         <span class="cit-article-title">Dengue subgenomic RNA binds TRIM25 to inhibit interferon expression for epidemiological fitness</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1007/BF03020102</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF03020102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1007/BF03020102&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-410282v3.17" data-doi="10.1186/1471-2164-14-444">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Marston</span>
          <span class="cit-name-given-names">DA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">McElhinney</span>
          <span class="cit-name-given-names">LM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ellis</span>
          <span class="cit-name-given-names">RJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Horton</span>
          <span class="cit-name-given-names">DL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wise</span>
          <span class="cit-name-given-names">EL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Leech</span>
          <span class="cit-name-given-names">SL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">David</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">de Lamballerie</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fooks</span>
          <span class="cit-name-given-names">AR</span>
         </span> (
         <span class="cit-pub-date">2013</span>) 
         <span class="cit-article-title">Next generation sequencing of viral RNA genomes</span>. 
         <abbr class="cit-jnl-abbrev">BMC Genomics</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1186/1471-2164-14-444</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D444%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-444%26rft_id%253Dinfo%253Apmid%252F23822119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-444&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23822119&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F01%2F410282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-410282v3.18" data-doi="10.1016/j.virol.2015.08.009">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Moon</span>
          <span class="cit-name-given-names">SL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dodd</span>
          <span class="cit-name-given-names">BJT</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Brackney</span>
          <span class="cit-name-given-names">DE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wilusz</span>
          <span class="cit-name-given-names">CJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ebel</span>
          <span class="cit-name-given-names">GD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wilusz</span>
          <span class="cit-name-given-names">J</span>
         </span> (
         <span class="cit-pub-date">2015</span>) 
         <span class="cit-article-title">Flavivirus sfRNA suppresses antiviral RNA interference in cultured cells and mosquitoes and directly interacts with the RNAi machinery</span>. 
         <abbr class="cit-jnl-abbrev">Virology</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1016/j.virol.2015.08.009</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D485%26rft.spage%253D322%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2015.08.009%26rft_id%253Dinfo%253Apmid%252F26331679%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.virol.2015.08.009&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26331679&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F01%2F410282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-410282v3.19" data-doi="10.1038/nrg2934">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ozsolak</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Milos</span>
          <span class="cit-name-given-names">PM</span>
         </span> (
         <span class="cit-pub-date">2011</span>) 
         <span class="cit-article-title">RNA sequencing: Advances, challenges and opportunities</span>. 
         <abbr class="cit-jnl-abbrev">Nature Reviews Genetics</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/nrg2934</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DOzsolak%26rft.auinit1%253DF.%26rft.volume%253D12%26rft.issue%253D2%26rft.spage%253D87%26rft.epage%253D98%26rft.atitle%253DRNA%2Bsequencing%253A%2Badvances%252C%2Bchallenges%2Band%2Bopportunities.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2934%26rft_id%253Dinfo%253Apmid%252F21191423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nrg2934&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21191423&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F01%2F410282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000286319600010&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-410282v3.20" data-doi="10.1038/nature08390">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ozsolak</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Platt</span>
          <span class="cit-name-given-names">AR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jones</span>
          <span class="cit-name-given-names">DR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Reifenberger</span>
          <span class="cit-name-given-names">JG</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sass</span>
          <span class="cit-name-given-names">LE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">McInerney</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Thompson</span>
          <span class="cit-name-given-names">JF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bowers</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jarosz</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Milos</span>
          <span class="cit-name-given-names">PM</span>
         </span> (
         <span class="cit-pub-date">2009</span>) 
         <span class="cit-article-title">Direct RNA sequencing</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/nature08390</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DOzsolak%26rft.auinit1%253DF.%26rft.volume%253D461%26rft.issue%253D7265%26rft.spage%253D814%26rft.epage%253D818%26rft.atitle%253DDirect%2BRNA%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08390%26rft_id%253Dinfo%253Apmid%252F19776739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature08390&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19776739&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F01%2F410282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000270547500040&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-410282v3.21" data-doi="10.1073/pnas.76.4.1760">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Peattie</span>
          <span class="cit-name-given-names">DA</span>
         </span> (
         <span class="cit-pub-date">1979</span>) 
         <span class="cit-article-title">Direct chemical method for sequencing RNA</span>. 
         <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.76.4.1760</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DPeattie%26rft.auinit1%253DD.%2BA.%26rft.volume%253D76%26rft.issue%253D4%26rft.spage%253D1760%26rft.epage%253D1764%26rft.atitle%253DDirect%2Bchemical%2Bmethod%2Bfor%2Bsequencing%2BRNA.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.76.4.1760%26rft_id%253Dinfo%253Apmid%252F377283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI3Ni80LzE3NjAiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNC8wMS80MTAyODIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-410282v3.22" data-doi="10.1371/journal.ppat.1006535">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pompon</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Manuel</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ng</span>
          <span class="cit-name-given-names">GK</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wong</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shan</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Manokaran</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Soto-Acosta</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bradrick</span>
          <span class="cit-name-given-names">SS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ooi</span>
          <span class="cit-name-given-names">EE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Missé</span>
          <span class="cit-name-given-names">D</span>
         </span> (
         <span class="cit-pub-date">2017</span>) 
         <span class="cit-article-title">Dengue subgenomic flaviviral RNA disrupts immunity in mosquito salivary glands to increase virus transmission</span>. 
         <abbr class="cit-jnl-abbrev">PLoS pathogens</abbr>
         <span class="cit-vol">13</span>(
         <span class="cit-issue">7</span>)Public Library of Science, : 
         <span class="cit-fpage">e1006535</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2Bpathogens%26rft.volume%253D13%26rft.spage%253De1006535%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1006535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1006535&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-410282v3.23" data-doi="10.1261/rna.1680809">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Romero-López</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Berzal-Herranz</span>
          <span class="cit-name-given-names">A</span>
         </span> (
         <span class="cit-pub-date">2009</span>) 
         <span class="cit-article-title">A long-range RNA-RNA interaction between the 51] and 31] ends of the HCV genome</span>. 
         <abbr class="cit-jnl-abbrev">RNA</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1261/rna.1680809</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.1680809%26rft_id%253Dinfo%253Apmid%252F19605533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjk6IjE1LzkvMTc0MCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA0LzAxLzQxMDI4Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-410282v3.24" data-doi="10.1038/nprot.2008.73">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Schmittgen</span>
          <span class="cit-name-given-names">TD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Livak</span>
          <span class="cit-name-given-names">KJ</span>
         </span> (
         <span class="cit-pub-date">2008</span>) 
         <span class="cit-article-title">Analyzing real-time PCR data by the comparative CTmethod</span>. 
         <abbr class="cit-jnl-abbrev">Nature Protocols</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/nprot.2008.73</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DSchmittgen%26rft.auinit1%253DT.%2BD.%26rft.volume%253D3%26rft.issue%253D6%26rft.spage%253D1101%26rft.epage%253D1108%26rft.atitle%253DAnalyzing%2Breal-time%2BPCR%2Bdata%2Bby%2Bthe%2Bcomparative%2BC%2528T%2529%2Bmethod.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2008.73%26rft_id%253Dinfo%253Apmid%252F18546601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nprot.2008.73&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18546601&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F01%2F410282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000258423700018&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-410282v3.25" data-doi="10.1016/j.ymeth.2010.01.005">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Taylor</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wakem</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dijkman</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Alsarraj</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nguyen</span>
          <span class="cit-name-given-names">M</span>
         </span> (
         <span class="cit-pub-date">2010</span>) 
         <span class="cit-article-title">A practical approach to RT-qPCR- Publishing data that conform to the MIQE guidelines</span>. 
         <abbr class="cit-jnl-abbrev">Methods</abbr>
         <span class="cit-vol">50</span>(
         <span class="cit-issue">4</span>)Elsevier Inc., : 
         <span class="cit-fpage">S1</span>–
         <span class="cit-lpage">S5</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1016/j.ymeth.2010.01.005</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMethods%2B%2528San%2BDiego%252C%2BCalif.%2529%26rft.stitle%253DMethods%26rft.aulast%253DTaylor%26rft.auinit1%253DS.%26rft.volume%253D50%26rft.issue%253D4%26rft.spage%253DS1%26rft.epage%253DS5%26rft.atitle%253DA%2Bpractical%2Bapproach%2Bto%2BRT-qPCR-Publishing%2Bdata%2Bthat%2Bconform%2Bto%2Bthe%2BMIQE%2Bguidelines.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ymeth.2010.01.005%26rft_id%253Dinfo%253Apmid%252F20215014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.ymeth.2010.01.005&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20215014&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F01%2F410282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000276262100018&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference in text" id="ref-26">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-410282v3.26" data-doi="10.1128/JCM.00330-16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Thomson</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ip</span>
          <span class="cit-name-given-names">CLC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Badhan</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Christiansen</span>
          <span class="cit-name-given-names">MT</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Adamson</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ansari</span>
          <span class="cit-name-given-names">MA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bibby</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Breuer</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Brown</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bowden</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bryant</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bonsall</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Silva Filipe</span>
          <span class="cit-name-given-names">A Da</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hinds</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hudson</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Klenerman</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lythgow</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mbisa</span>
          <span class="cit-name-given-names">JL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">McLauchlan</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Myers</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Piazza</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Roy</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Trebes</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sreenu</span>
          <span class="cit-name-given-names">VB</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Witteveldt</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Barnes</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Simmonds</span>
          <span class="cit-name-given-names">P</span>
         </span> (
         <span class="cit-pub-date">2016</span>) 
         <span class="cit-article-title">Comparison of next-generation sequencing technologies for comprehensive assessment of full-length hepatitis C viral genomes</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Clinical Microbiology</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1128/JCM.00330-16</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BMicrobiology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.00330-16%26rft_id%253Dinfo%253Apmid%252F27385709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjEwOiI1NC8xMC8yNDcwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDQvMDEvNDEwMjgyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference in text" id="ref-27">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-410282v3.27" data-doi="10.1093/nar/gkt1291">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Witteveldt</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Blundell</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Maarleveld</span>
          <span class="cit-name-given-names">JJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mcfadden</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Evans</span>
          <span class="cit-name-given-names">DJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Simmonds</span>
          <span class="cit-name-given-names">P</span>
         </span> (
         <span class="cit-pub-date">2014</span>) 
         <span class="cit-article-title">The influence of viral RNA secondary structure on interactions with innate host cell defences</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gkt1291</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1291%26rft_id%253Dinfo%253Apmid%252F24335283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkt1291&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24335283&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F01%2F410282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000333093600048&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference in text" id="ref-28">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-410282v3.28">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wongsurawat</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jenjaroenpun</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Taylor</span>
          <span class="cit-name-given-names">MK</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lee</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tolardo</span>
          <span class="cit-name-given-names">AL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Parvathareddy</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kandel</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wadley</span>
          <span class="cit-name-given-names">TD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kaewnapan</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Athipanyasilp</span>
          <span class="cit-name-given-names">N</span>
         </span> (
         <span class="cit-pub-date">2019</span>) 
         <span class="cit-article-title">Rapid Sequencing of Multiple RNA Viruses in their Native Form</span>. 
         <abbr class="cit-jnl-abbrev">Frontiers in microbiology 10Frontiers Media SA</abbr>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
