<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">eLife</span><span class="iso-abbrev" title="iso-abbrev">Elife</span><span class="publisher-id" title="publisher-id">eLife</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">eLife</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2050-084X</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">eLife Sciences Publications, Ltd</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7064339">7064339</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/32011232">32011232</a></span><span class="publisher-id" title="publisher-id">48401</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.7554/eLife.48401">10.7554/eLife.48401</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span>: <span class="subject" title="subject">Ecology</span>: <span class="subject" title="subject">Epidemiology and Global Health</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Accelerated viral dynamics in bat cell lines, with implications for zoonotic emergence</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brook</span><span class="given-names" tagx="given-names" title="given-names">Cara E</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0003-4276-073X</span><span class="email" tagx="email" title="email">cbrook@berkeley.edu</span><a href="#aff1">1</a><a href="#aff2">2</a><a href="#fund1" /><a href="#fund9" /><a href="#fund10" /><a href="#fund2" /><a href="#con1" /><a href="#conf1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boots</span><span class="given-names" tagx="given-names" title="given-names">Mike</span></span><a href="#aff1">1</a><a href="#con2" /><a href="#conf1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chandran</span><span class="given-names" tagx="given-names" title="given-names">Kartik</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-0232-7077</span><a href="#aff3">3</a><a href="#fund3" /><a href="#con3" /><a href="#conf1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dobson</span><span class="given-names" tagx="given-names" title="given-names">Andrew P</span></span><a href="#aff2">2</a><a href="#con4" /><a href="#conf1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drosten</span><span class="given-names" tagx="given-names" title="given-names">Christian</span></span><a href="#aff4">4</a><a href="#fund6" /><a href="#fund7" /><a href="#fund8" /><a href="#con5" /><a href="#conf1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graham</span><span class="given-names" tagx="given-names" title="given-names">Andrea L</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-6580-2755</span><a href="#aff2">2</a><a href="#con6" /><a href="#conf1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grenfell</span><span class="given-names" tagx="given-names" title="given-names">Bryan T</span></span><a href="#aff2">2</a><a href="#aff5">5</a><a href="#con7" /><a href="#conf1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Müller</span><span class="given-names" tagx="given-names" title="given-names">Marcel A</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-2242-5117</span><a href="#aff4">4</a><a href="#aff6">6</a><a href="#con8" /><a href="#conf1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ng</span><span class="given-names" tagx="given-names" title="given-names">Melinda</span></span><a href="#aff3">3</a><a href="#con9" /><a href="#conf1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Lin-Fa</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-2752-0535</span><a href="#aff7">7</a><a href="#fund4" /><a href="#fund5" /><a href="#con10" /><a href="#conf1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Leeuwen</span><span class="given-names" tagx="given-names" title="given-names">Anieke</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0003-1987-1458</span><a href="#aff2">2</a><a href="#aff8">8</a><a href="#con11" /><a href="#conf1" /></span><span class="citation_author_institution" id="aff1">[1], <span class="institution" title="institution">Department of Integrative Biology, University of California, Berkeley</span><span class="addr-line" title="addr-line">Berkeley</span><span class="country" tagx="country" title="country">United States</span></span><span class="citation_author_institution" id="aff2">[2], <span class="institution" title="institution">Department of Ecology and Evolutionary Biology, Princeton University</span><span class="addr-line" title="addr-line">Princeton</span><span class="country" tagx="country" title="country">United States</span></span><span class="citation_author_institution" id="aff3">[3], <span class="institution" title="institution">Department of Microbiology and Immunology, Albert Einstein College of Medicine</span><span class="addr-line" title="addr-line">Bronx</span><span class="country" tagx="country" title="country">United States</span></span><span class="citation_author_institution" id="aff4">[4], <span class="institution" title="institution">Institute of Virology, Charité-Universitätsmedizin Berlin, corporate member of Freie Universität Berlin, Humboldt-Universität zu Berlin, and Berlin Institute of Health</span><span class="addr-line" title="addr-line">Berlin</span><span class="country" tagx="country" title="country">Germany</span></span><span class="citation_author_institution" id="aff5">[5], <span class="institution" title="institution">Fogarty International Center, National Institutes of Health</span><span class="addr-line" title="addr-line">Bethesda</span><span class="country" tagx="country" title="country">United States</span></span><span class="citation_author_institution" id="aff6">[6], <span class="institution" title="institution">Martsinovsky Institute of Medical Parasitology, Tropical and Vector Borne Diseases, Sechenov University</span><span class="addr-line" title="addr-line">Moscow</span><span class="country" tagx="country" title="country">Russian Federation</span></span><span class="citation_author_institution" id="aff7">[7], <span class="institution" title="institution">Emerging Infectious Diseases Program, Duke-National University of Singapore Medical School</span><span class="addr-line" title="addr-line">Singapore</span><span class="country" tagx="country" title="country">Singapore</span></span><span class="citation_author_institution" id="aff8">[8], <span class="institution" title="institution">Royal Netherlands Institute for Sea Research, Department of Coastal Systems, and Utrecht University</span><span class="addr-line" title="addr-line">Den Burg</span><span class="country" tagx="country" title="country">Netherlands</span></span></div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haydon</span><span class="given-names" tagx="given-names" title="given-names">Dan</span></span><span class="role" tagx="role" title="role">Reviewing Editor</span><span class="citation_author_institution" id="">[], <span class="institution" title="institution">University of Glasgow</span><span class="country" tagx="country" title="country">United Kingdom</span></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferguson</span><span class="given-names" tagx="given-names" title="given-names">Neil M</span></span><span class="role" tagx="role" title="role">Senior Editor</span><span class="citation_author_institution" id="">[], <span class="institution" title="institution">Imperial College London</span><span class="country" tagx="country" title="country">United Kingdom</span></span></span></div><span class="pub-date-" title="pub-date-">: <span>2020-2-2</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="volume" tagx="volume" title="volume">9</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e48401</span><span class="history" title="history"><span class="received" title="received">received: 2019-5-13</span><span class="accepted" title="accepted">accepted: 2020-2-02</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Brook et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license">[license_ref] http://creativecommons.org/licenses/by/4.0/<span class="license-p" title="license-p">This article is distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use and redistribution provided that the original author and source are credited.</span></span></div><a class="self-uri" href="elife-48401.pdf" title="self-uri">elife-48401.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Bats host virulent zoonotic viruses without experiencing disease. A mechanistic understanding of the impact of bats’ virus hosting capacities, including uniquely constitutive immune pathways, on cellular-scale viral dynamics is needed to elucidate zoonotic emergence. We carried out virus infectivity assays on bat cell lines expressing induced and constitutive immune phenotypes, then developed a theoretical model of our <i>in vitro</i> system, which we fit to empirical data. Best fit models recapitulated expected immune phenotypes for representative cell lines, supporting robust antiviral defenses in bat cells that correlated with higher estimates for within-host viral propagation rates. In general, heightened immune responses limit pathogen-induced cellular morbidity, which can facilitate the establishment of rapidly-propagating persistent infections within-host. Rapidly-transmitting viruses that have evolved with bat immune systems will likely cause enhanced virulence following emergence into secondary hosts with immune systems that diverge from those unique to bats.</p> </div> <div class="abstract" title="abstract"> <div class="title" tagx="title" title="title">
eLife digest</div> <p>Bats can carry viruses that are deadly to other mammals without themselves showing serious symptoms. In fact, bats are natural reservoirs for viruses that have some of the highest fatality rates of any viruses that people acquire from wild animals – including rabies, Ebola and the SARS coronavirus.</p> <p>Bats have a suite of antiviral defenses that keep the amount of virus in check. For example, some bats have an antiviral immune response called the interferon pathway perpetually switched on. In most other mammals, having such a hyper-vigilant immune response would cause harmful inflammation. Bats, however, have adapted anti-inflammatory traits that protect them from such harm, include the loss of certain genes that normally promote inflammation. However, no one has previously explored how these unique antiviral defenses of bats impact the viruses themselves.</p> <p>Now, Brook et al. have studied this exact question using bat cells grown in the laboratory. The experiments made use of cells from one bat species – the black flying fox – in which the interferon pathway is always on, and another – the Egyptian fruit bat – in which this pathway is only activated during an infection. The bat cells were infected with three different viruses, and then Brook et al. observed how the interferon pathway helped keep the infections in check, before creating a computer model of this response.</p> <p>The experiments and model helped reveal that the bats’ defenses may have a potential downside for other animals, including humans. In both bat species, the strongest antiviral responses were countered by the virus spreading more quickly from cell to cell. This suggests that bat immune defenses may drive the evolution of faster transmitting viruses, and while bats are well protected from the harmful effects of their own prolific viruses, other creatures like humans are not.</p> <p>The findings may help to explain why bats are often the source for viruses that are deadly in humans. Learning more about bats' antiviral defenses and how they drive virus evolution may help scientists develop better ways to predict, prevent or limit the spread of viruses from bats to humans. More studies are needed in bats to help these efforts. In the meantime, the experiments highlight the importance of warning people to avoid direct contact with wild bats.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
bat</div> <div class="kwd" title="kwd">
innate immunity</div> <div class="kwd" title="kwd">
Chiroptera</div> <div class="kwd" title="kwd">
within-host model</div> <div class="kwd" title="kwd">
interferon</div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Research organism</div> <div class="kwd" title="kwd">
Other</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000001</span><span class="institution" title="institution">National Science Foundation</span></div> </div><span class="award-id" tagx="award-id" title="award-id">Graduate Research Fellowship</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brook</span><span class="given-names" tagx="given-names" title="given-names">Cara E</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100007247</span><span class="institution" title="institution">Adolph C. and Mary Sprague Miller Institute for Basic Research in Science, University of California Berkeley</span></div> </div><span class="award-id" tagx="award-id" title="award-id">Postdoctoral Fellowship</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brook</span><span class="given-names" tagx="given-names" title="given-names">Cara E</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000002</span><span class="institution" title="institution">National Institutes of Health</span></div> </div><span class="award-id" tagx="award-id" title="award-id">R01-AI134824</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chandran</span><span class="given-names" tagx="given-names" title="given-names">Kartik</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100001381</span><span class="institution" title="institution">Singapore National Research Foundation</span></div> </div><span class="award-id" tagx="award-id" title="award-id">NRF2012NRF-CRP001-056</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Lin-Fa</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100001381</span><span class="institution" title="institution">Singapore National Research Foundation</span></div> </div><span class="award-id" tagx="award-id" title="award-id">NRF2016NRF-NSFC002-013</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Lin-Fa</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100001659</span><span class="institution" title="institution">Deutsche Forschungsgemeinschaft</span></div> </div><span class="award-id" tagx="award-id" title="award-id">DR 772/10-2</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drosten</span><span class="given-names" tagx="given-names" title="given-names">Christian</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100002347</span><span class="institution" title="institution">Bundesministerium für Bildung und Forschung</span></div> </div><span class="award-id" tagx="award-id" title="award-id">RAPID #01KI1723A</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drosten</span><span class="given-names" tagx="given-names" title="given-names">Christian</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100007601</span><span class="institution" title="institution">Horizon 2020</span></div> </div><span class="award-id" tagx="award-id" title="award-id">#653316</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drosten</span><span class="given-names" tagx="given-names" title="given-names">Christian</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000185</span><span class="institution" title="institution">DARPA</span></div> </div><span class="award-id" tagx="award-id" title="award-id">PREEMPT Program - Cooperative Agreement no. D18AC00031</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brook</span><span class="given-names" tagx="given-names" title="given-names">Cara E</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000002</span><span class="institution" title="institution">National Institutes of Health</span></div> </div><span class="award-id" tagx="award-id" title="award-id">1R01AI129822-01</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brook</span><span class="given-names" tagx="given-names" title="given-names">Cara E</span></span></span></div> <div class="funding-statement" title="funding-statement">
The funders had no role in study design, data collection and interpretation, or the decision to submit the work for publication.</div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Author impact statement" value="Bats' uniquely robust innate antiviral immune defenses select for faster transmitting viruses likely to generate extreme virulence upon spillover to secondary hosts with immune systems divergent from those of bat." /> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Bats have received much attention in recent years for their role as reservoir hosts for emerging viral zoonoses, including rabies and related lyssaviruses, Hendra and Nipah henipaviruses, Ebola and Marburg filoviruses, and SARS coronavirus (<a href="#bib8">Calisher et al., 2006</a>; <a href="#bib57">Wang and Anderson, 2019</a>). In most non-Chiropteran mammals, henipaviruses, filoviruses, and coronaviruses induce substantial morbidity and mortality, display short durations of infection, and elicit robust, long-term immunity in hosts surviving infection (<a href="#bib35">Nicholls et al., 2003</a>; <a href="#bib22">Hooper et al., 2001</a>; <a href="#bib29">Mahanty and Bray, 2004</a>). Bats, by contrast, demonstrate no obvious disease symptoms upon infection with pathogens that are highly virulent in non-volant mammals (<a href="#bib53">Schountz et al., 2017</a>) but may, instead, support viruses as long-term persistent infections, rather than transient, immunizing pathologies (<a href="#bib46">Plowright et al., 2016</a>).</p> <p>Recent research advances are beginning to shed light on the molecular mechanisms by which bats avoid pathology from these otherwise virulent pathogens (<a href="#bib7">Brook and Dobson, 2015</a>). Bats leverage a suite of species-specific mechanisms to limit viral load, which include host receptor sequence incompatibilities for some bat-virus combinations (<a href="#bib34">Ng et al., 2015</a>; <a href="#bib56">Takadate et al., 2020</a>) and constitutive expression of the antiviral cytokine, IFN-α, for others (<a href="#bib62">Zhou et al., 2016</a>). Typically, the presence of viral RNA or DNA in the cytoplasm of mammalian cells will induce secretion of type I interferon proteins (IFN-α and IFN-β), which promote expression and translation of interferon-stimulated genes (ISGs) in neighboring cells and render them effectively antiviral (<a href="#bib55">Stetson and Medzhitov, 2006</a>). In some bat cells, the transcriptomic blueprints for this IFN response are expressed constitutively, even in the absence of stimulation by viral RNA or DNA (<a href="#bib62">Zhou et al., 2016</a>). In non-flying mammals, constitutive IFN expression would likely elicit widespread inflammation and concomitant immunopathology upon viral infection, but bats support unique adaptations to combat inflammation (<a href="#bib61">Zhang et al., 2013</a>; <a href="#bib1">Ahn et al., 2019</a>; <a href="#bib60">Xie et al., 2018</a>; <a href="#bib40">Pavlovich et al., 2018</a>) that may have evolved to mitigate metabolic damage induced during flight (<a href="#bib24">Kacprzyk et al., 2017</a>). The extent to which constitutive IFN-α expression signifies constitutive antiviral defense in the form of functional IFN-α protein remains unresolved. In bat cells constitutively expressing IFN-α, some protein-stimulated, downstream ISGs appear to be also constitutively expressed, but additional ISG induction is nonetheless possible following viral challenge and stimulation of IFN-β (<a href="#bib62">Zhou et al., 2016</a>; <a href="#bib60">Xie et al., 2018</a>). Despite recent advances in molecular understanding of bat viral tolerance, the consequences of this unique bat immunity on within-host virus dynamics—and its implications for understanding zoonotic emergence—have yet to be elucidated.</p> <p>The field of ‘virus dynamics’ was first developed to describe the mechanistic underpinnings of long-term patterns of steady-state viral load exhibited by patients in chronic phase infections with HIV, who appeared to produce and clear virus at equivalent rates (<a href="#bib38">Nowak and May, 2000</a>; <a href="#bib21">Ho et al., 1995</a>). Models of simple target cell depletion, in which viral load is dictated by a bottom-up resource supply of infection-susceptible host cells, were first developed for HIV (<a href="#bib45">Perelson, 2002</a>) but have since been applied to other chronic infections, including hepatitis-C virus (<a href="#bib33">Neumann et al., 1998</a>), hepatitis-B virus (<a href="#bib37">Nowak et al., 1996</a>) and cytomegalovirus (<a href="#bib18">Emery et al., 1999</a>). Recent work has adopted similar techniques to model the within-host dynamics of acute infections, such as influenza A and measles, inspiring debate over the extent to which explicit modeling of top-down immune control can improve inference beyond the basic resource limitation assumptions of the target cell model (<a href="#bib3">Baccam et al., 2006</a>; <a href="#bib41">Pawelek et al., 2012</a>; <a href="#bib51">Saenz et al., 2010</a>; <a href="#bib31">Morris et al., 2018</a>).</p> <p>To investigate the impact of unique bat immune processes on <i>in vitro</i> viral kinetics, we first undertook a series of virus infection experiments on bat cell lines expressing divergent interferon phenotypes, then developed a theoretical model elucidating the dynamics of within-host viral spread. We evaluated our theoretical model analytically independent of the data, then fit the model to data recovered from <i>in vitro</i> experimental trials in order to estimate rates of within-host virus transmission and cellular progression to antiviral status under diverse assumptions of absent, induced, and constitutive immunity. Finally, we confirmed our findings in spatially-explicit stochastic simulations of fitted time series from our mean field model. We hypothesized that top-down immune processes would overrule classical resource-limitation in bat cell lines described as constitutively antiviral in the literature, offering a testable prediction for models fit to empirical data. We further predicted that the most robust antiviral responses would be associated with the most rapid within-host virus propagation rates but also protect cells against virus-induced mortality to support the longest enduring infections in tissue culture.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="virusinfectionexperimentsinantiviralbatcellculturesyieldreducedcellmortalityandelongatedepidemics" title="sec"> <div class="title" tagx="title" title="title">
Virus infection experiments in antiviral bat cell cultures yield reduced cell mortality and elongated epidemics</div> <p>We first explored the influence of innate immune phenotype on within-host viral propagation in a series of infection experiments in cell culture. We conducted plaque assays on six-well plate monolayers of three immortalized mammalian kidney cell lines: [1] Vero (African green monkey) cells, which are IFN-defective and thus limited in antiviral capacity (<a href="#bib12">Desmyter et al., 1968</a>); [2] RoNi/7.1 (<i>Rousettus aegyptiacus</i>) cells which demonstrate idiosyncratic induced interferon responses upon viral challenge (<a href="#bib27">Kuzmin et al., 2017</a>; <a href="#bib2">Arnold et al., 2018</a>; <a href="#bib4">Biesold et al., 2011</a>; <a href="#bib40">Pavlovich et al., 2018</a>); and [3] PaKiT01 (<i>Pteropus alecto</i>) cells which constitutively express IFN-α (<a href="#bib62">Zhou et al., 2016</a>; <a href="#bib10">Crameri et al., 2009</a>). To intensify cell line-specific differences in constitutive immunity, we carried out infectivity assays with GFP-tagged, replication-competent vesicular stomatitis Indiana viruses: rVSV-G, rVSV-EBOV, and rVSV-MARV, which have been previously described (<a href="#bib30">Miller et al., 2012</a>; <a href="#bib59">Wong et al., 2010</a>). Two of these viruses, rVSV-EBOV and rVSV-MARV, are recombinants for which cell entry is mediated by the glycoprotein of the bat-evolved filoviruses, Ebola (EBOV) and Marburg (MARV), thus allowing us to modulate the extent of structural, as well as immunological, antiviral defense at play in each infection. Previous work in this lab has demonstrated incompatibilities in the NPC1 filovirus receptor which render PaKiT01 cells refractory to infection with rVSV-MARV (Ng and Chandrab, 2018, Unpublished results), making them structurally antiviral, over and above their constitutive expression of IFN-α. All three cell lines were challenged with all three viruses at two multiplicities of infection (MOI): 0.001 and 0.0001. Between 18 and 39 trials were run at each cell-virus-MOI combination, excepting rVSV-MARV infections on PaKiT01 cells at MOI = 0.001, for which only eight trials were run (see Materials and methods; <a href="#fig1s1">Figure 1—figure supplements 1</a>–<a href="#fig1s3">3</a>, <a href="#supp1">Supplementary file 1</a>).</p> <p>Because plaque assays restrict viral transmission neighbor-to-neighbor in two-dimensional cellular space (<a href="#bib23">Howat et al., 2006</a>), we were able to track the spread of GFP-expressing virus-infected cells across tissue monolayers via inverted fluorescence microscopy. For each infection trial, we monitored and re-imaged plates for up to 200 hr of observations or until total monolayer destruction, processed resulting images, and generated a time series of the proportion of infectious-cell occupied plate space across the duration of each trial (see Materials and methods). We used generalized additive models to infer the time course of all cell culture replicates and construct the multi-trial dataset to which we eventually fit our mechanistic transmission model for each cell line-virus-specific combination (<a href="#fig1">Figure 1</a>; <a href="#fig1s1">Figure 1—figure supplements 1</a>–<a href="#fig1s5">5</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1.</span></a></div>   <div class="title" tagx="title" title="title">
Fitted time series of infectious cell proportions from mean field model for rVSV-G, rVSV-EBOV, and rVSV-MARV infections (columns) on Vero, RoNi/7.1, and PaKiT01 cell lines (rows) at MOI = 0.001.</div> <p>Results are shown for the best fit immune absent model on Vero cells, induced immunity model on RoNi/7.1 cells, and constitutive (for rVSV-VSVG and rVSV-EBOV) and induced (for rVSV-MARV) immunity models on PaKiT01 cells. Raw data across all trials are shown as open circles (statistical smoothers from each trial used for fitting are available in <a href="#fig1s2">Figure 1—figure supplements 2</a>–<a href="#fig1s3">3</a>). Model output is shown as a solid crimson line (95% confidence intervals by standard error = red shading). Panel background corresponds to empirical outcome of the average stochastic cell culture trial (persistent infection = white; virus-induced epidemic extinction = gray; immune-mediated epidemic extinction = black). Parameter values are listed in <a href="#table1">Table 1</a> and <a href="#supp4">Supplementary file 4</a>. Results for absent/induced/constitutive fitted models across all cell lines are shown in <a href="#fig1s4">Figure 1—figure supplement 4</a> (MOI = 0.001) and <a href="#fig1s5">Figure 1—figure supplement 5</a> (MOI = 0.0001).</p>   </div> <p>All three recombinant vesicular stomatitis viruses (rVSV-G, rVSV-EBOV, and rVSV-MARV) infected Vero, RoNi/7.1, and PaKiT01 tissue cultures at both focal MOIs. Post-invasion, virus spread rapidly across most cell monolayers, resulting in virus-induced epidemic extinction. Epidemics were less severe in bat cell cultures, especially when infected with the recombinant filoviruses, rVSV-EBOV and rVSV-MARV. Monolayer destruction was avoided in the case of rVSV-EBOV and rVSV-MARV infections on PaKiT01 cells: in the former, persistent viral infection was maintained throughout the 200 hr duration of each experiment, while, in the latter, infection was eliminated early in the time series, preserving a large proportion of live, uninfectious cells across the duration of the experiment. We assumed this pattern to be the result of immune-mediated epidemic extinction (<a href="#fig1">Figure 1</a>). Patterns from MOI = 0.001 were largely recapitulated at MOI = 0.0001, though at somewhat reduced total proportions (<a href="#fig1s5">Figure 1—figure supplement 5</a>).</p> </div> <div class="atheoreticalmodelfittoinvitrodatarecapitulatesexpectedimmunephenotypesforbatcells" title="sec"> <div class="title" tagx="title" title="title">
A theoretical model fit to <i>in vitro</i> data recapitulates expected immune phenotypes for bat cells</div> <p>We next developed a within-host model to fit to these data to elucidate the effects of induced and constitutive immunity on the dynamics of viral spread in host tissue (<a href="#fig1">Figure 1</a>). The compartmental within-host system mimicked our two-dimensional cell culture monolayer, with cells occupying five distinct infection states: susceptible (S), antiviral (A), exposed (E), infectious (I), and dead (D). We modeled exposed cells as infected but not yet infectious, capturing the ‘eclipse phase’ of viral integration into a host cell which precedes viral replication. Antiviral cells were immune to viral infection, in accordance with the 'antiviral state' induced from interferon stimulation of ISGs in tissues adjacent to infection (<a href="#bib55">Stetson and Medzhitov, 2006</a>). Because we aimed to translate available data into modeled processes, we did not explicitly model interferon dynamics but instead scaled the rate of cell progression from susceptible to antiviral (�?) by the proportion of exposed cells (globally) in the system. In systems permitting constitutive immunity, a second rate of cellular acquisition of antiviral status (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div></div></span>) additionally scaled with the global proportion of susceptible cells in the model. Compared with virus, IFN particles are small and highly diffusive, justifying this global signaling assumption at the limited spatial extent of a six-well plate and maintaining consistency with previous modeling approximations of IFN signaling in plaque assay (<a href="#bib23">Howat et al., 2006</a>).</p> <p>To best represent our empirical monolayer system, we expressed our state variables as proportions (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> </div> </div> </div> </div></div></span>, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> </div> </div></div></span>, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> </div> </div></div></span>, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> </div> </div></div></span>, and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> </div> </div></div></span>), under assumptions of frequency-dependent transmission in a well-mixed population (<a href="#bib25">Keeling and Rohani, 2008</a>), though note that the inclusion of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> </div> </div></div></span> (representing the proportion of dead space in the modeled tissue) had the functional effect of varying transmission with infectious cell density. This resulted in the following system of ordinary differential equations: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(1)</span><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
b</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mtext_UNKNOWN"> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
β</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
μ</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
�?</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mtext_UNKNOWN"> </div> <div class="mi_UNKNOWN">
ε</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
c</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> </div> </div> </div> </div></div></div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(2)</span><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
�?</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mtext_UNKNOWN"> </div> <div class="mi_UNKNOWN">
ε</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
c</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
μ</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> </div> </div> </div> </div></div></div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(3)</span><div class="math_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
β</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> </div> </div> <div class="mo_UNKNOWN">
-</div> <div class="mi_UNKNOWN">
σ</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> </div> </div> <div class="mo_UNKNOWN">
-</div> <div class="mi_UNKNOWN">
μ</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> </div> </div></div></div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(4)</span><div class="math_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
σ</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> </div> </div> <div class="mo_UNKNOWN">
-</div> <div class="mi_UNKNOWN">
α</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> </div> </div> <div class="mo_UNKNOWN">
-</div> <div class="mi_UNKNOWN">
μ</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> </div> </div></div></div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(5)</span><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
μ</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mtext_UNKNOWN"> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mtext_UNKNOWN"> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
α</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
b</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mtext_UNKNOWN"> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div></div></div> <p /> <p>We defined 'induced immunity' as complete, modeling all cells as susceptible to viral invasion at disease-free equilibrium, with defenses induced subsequent to viral exposure through the term �?. By contrast, we allowed the extent of constitutive immunity to vary across the parameter range of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div></div></span> &amp;gt; 0, defining a 'constitutive' system as one containing <i>any</i> antiviral cells at disease-free equilibrium. In fitting this model to tissue culture data, we independently estimated both �? and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div> <div class="mo_UNKNOWN">
,</div></div></span> as well as the cell-to-cell transmission rate, β, for each cell-virus combination. Since the extent to which constitutively-expressed IFN-α is constitutively translated into functional protein is not yet known for bat hosts (<a href="#bib62">Zhou et al., 2016</a>), this approach permitted our tissue culture data to drive modeling inference: even in PaKiT01 cell lines known to constitutively express IFN-α, the true constitutive extent of the system (i.e. the quantity of antiviral cells present at disease-free equilibrium) was allowed to vary through estimation of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div> <div class="mo_UNKNOWN">
.</div></div></span> For the purposes of model-fitting, we fixed the value of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
c</div></div></span>, the return rate of antiviral cells to susceptible status, at 0. The small spatial scale and short time course (max 200 hours) of our experiments likely prohibited any return of antiviral cells to susceptible status in our empirical system; nonetheless, we retained the term <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
c</div></div></span> in analytical evaluations of our model because regression from antiviral to susceptible status is possible over long time periods <i>in vitro</i> and at the scale of a complete organism (<a href="#bib48">Radke et al., 1974</a>; <a href="#bib49">Rasmussen and Farley, 1975</a>; <a href="#bib52">Samuel and Knutson, 1982</a>).</p> <p>Before fitting to empirical time series, we undertook bifurcation analysis of our theoretical model and generated testable hypotheses on the basis of model outcomes. From our within-host model system (<a href="#equ1 equ2 equ3 equ4 equ5">Equation 1-5</a>), we derived the following expression for <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span>, the pathogen basic reproduction number (<a href="#supp2">Supplementary file 2</a>): </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(6)</span><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
σ</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
-</div> <div class="mi_UNKNOWN">
μ</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
c</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
μ</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
μ</div> </div> </div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
α</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
μ</div> </div> </div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
c</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
μ</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
ε</div> </div> </div> </div> </div></div></div> <p /> <p>Pathogens can invade a host tissue culture when <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="mo_UNKNOWN">
&amp;gt;</div> <div class="mn_UNKNOWN">
1</div> </div> </div></div></span>. Rapid rates of constitutive antiviral acquisition (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div></div></span>) will drive <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="mn_UNKNOWN">
1</div> </div> </div></div></span>: tissue cultures with highly constitutive antiviral immunity will be therefore resistant to virus invasion from the outset. Since, by definition, induced immunity is stimulated following initial virus invasion, the rate of induced antiviral acquisition (�?) is not incorporated into the equation for <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span>; while induced immune processes can control virus after initial invasion, they cannot prevent it from occurring to begin with. In cases of fully induced or absent immunity (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div></div></span>), the <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span> equation thus reduces to a form typical of the classic SEIR model: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(7)</span><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
σ</div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
-</div> <div class="mi_UNKNOWN">
μ</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
α</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
μ</div> </div> </div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
μ</div> </div> </div> </div> </div></div></div> <p /> <p>At equilibrium, the theoretical, mean field model demonstrates one of three infection states: endemic equilibrium, stable limit cycles, or no infection (<a href="#fig2">Figure 2</a>). Respectively, these states approximate the persistent infection, virus-induced epidemic extinction, and immune-mediated epidemic extinction phenotypes previously witnessed in tissue culture experiments (<a href="#fig1">Figure 1</a>). Theoretically, endemic equilibrium is maintained when new infections are generated at the same rate at which infections are lost, while limit cycles represent parameter space under which infectious and susceptible populations are locked in predictable oscillations. Endemic equilibria resulting from cellular regeneration (i.e. births) have been described <i>in vivo</i> for HIV (<a href="#bib9">Coffin, 1995</a>) and <i>in vitro</i> for herpesvirus plaque assays (<a href="#bib23">Howat et al., 2006</a>), but, because they so closely approach zero, true limit cycles likely only occur theoretically, instead yielding stochastic extinctions in empirical time series.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2.</span></a></div>   <div class="title" tagx="title" title="title">
Two parameter bifurcations of the mean field model, showing variation in the transmission rate, β, against variation in the pathogen-induced mortality rate, α, under diverse immune assumptions.</div> <p>Panel (<b>A</b>) depicts dynamics under variably constitutive immunity, ranging from absent (left: <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span>) to high (right: <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
.0025</div> </div> </div></div></span>). In all panel (<b>A</b>) plots, the rate of induced immune antiviral acquisition (�?) was fixed at 0.01. Panel (<b>B</b>) depicts dynamics under variably induced immunity, ranging from absent (left: �?=0) to high (right: �?=1). In all panel (<b>B</b>) plots, the rate of constitutive antiviral acquisition (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div> <div class="mo_UNKNOWN">
)</div></div></span>) was fixed at 0.0001 Branch point curves are represented as solid lines and Hopf curves as dashed lines. White space indicates endemic equilibrium (persistence), gray space indicates limit cycles, and black space indicates no infection (extinction). Other parameter values for equilibrium analysis were fixed at: <i>b</i> = .025, μ = .001, σ = 1/6, <i>c</i> = 0. Special points from bifurcations analyses are listed in <a href="#supp3">Supplementary file 3</a>.</p>   </div> <p>Bifurcation analysis of our mean field model revealed that regions of no infection (pathogen extinction) were bounded at lower threshold (Branch point) values for β, below which the pathogen was unable to invade. We found no upper threshold to invasion for β under any circumstances (i.e. β high enough to drive pathogen-induced extinction), but high β values resulted in Hopf bifurcations, which delineate regions of parameter space characterized by limit cycles. Since limit cycles so closely approach zero, high βs recovered in this range would likely produce virus-induced epidemic extinctions under experimental conditions. Under more robust representations of immunity, with higher values for either or both induced (�?) and constitutive (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div></div></span>) rates of antiviral acquisition, Hopf bifurcations occurred at increasingly higher values for β, meaning that persistent infections could establish at higher viral transmission rates (<a href="#fig2">Figure 2</a>). Consistent with our derivation for <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span>, we found that the Branch point threshold for viral invasion was independent of changes to the induced immune parameter (�?) but saturated at high values of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div></div></span> that characterize highly constitutive immunity (<a href="#fig3">Figure 3</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3.</span></a></div>   <div class="title" tagx="title" title="title">
Two parameter bifurcations of the mean field model, showing variation in the transmission rate, β, against variation in: (<b>A</b>) the induced immunity rate of antiviral acquisition (�?) and (<b>B</b>) the constitutive immunity rate of antiviral acquisition (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div></div></span>).</div> <p>Panels show variation in the extent of immunity, from absent (left) to high (right). Branch point curves are represented as solid lines and Hopf curves as dashed lines. White space indicates endemic equilibrium (persistence), gray space indicates limit cycling, and black space indicates no infection (extinction). Other parameter values for equilibrium analysis were fixed at: <i>b</i> = .025, μ = .001, σ = 1/6, α = 1/6, <i>c</i> = 0. Special points from bifurcations analyses are listed in <a href="#supp3">Supplementary file 3</a>.</p>   </div> <p>We next fit our theoretical model by least squares to each cell line-virus combination, under absent, induced, and constitutive assumptions of immunity. In general, best fit models recapitulated expected outcomes based on the immune phenotype of the cell line in question, as described in the general literature (<a href="#table1">Table 1</a>; <a href="#supp4">Supplementary file 4</a>). The absent immune model offered the most accurate approximation of IFN-deficient Vero cell time series, the induced immune model best recovered the RoNi/7.1 cell trials, and, in most cases, the constitutive immune model most closely recaptured infection dynamics across constitutively IFN-α-expressing PaKiT01 cell lines (<a href="#fig1">Figure 1</a>, <a href="#fig1s4">Figure 1—figure supplements 4</a>–<a href="#fig1s5">5</a>, <a href="#supp4">Supplementary file 4</a>). Ironically, the induced immune model offered a slightly better fit than the constitutive to rVSV-MARV infections on the PaKiT01 cell line (the one cell line-virus combination for which we know a constitutively antiviral cell-receptor incompatibility to be at play). Because constitutive immune assumptions can prohibit pathogen invasion (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="mn_UNKNOWN">
1</div> </div> </div></div></span>), model fits to this time series under constitutive assumptions were handicapped by overestimations of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div></div></span>, which prohibited pathogen invasion. Only by incorporating an exceedingly rapid rate of induced antiviral acquisition could the model guarantee that initial infection would be permitted and then rapidly controlled.</p> <div class="table-wrap_UNKNOWN" id="table1" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 1.</span> <div class="title" tagx="title" title="title">
Optimized parameters from best fit deterministic model and spatial approximation at MOI = 0.001</div>  <table frame="hsides" rules="groups"> <thead> <tr> <th rowspan="1" colspan="1">Cell line</th> <th rowspan="1" colspan="1">Virus</th> <th rowspan="1" colspan="1">Immune assumption</th> <th rowspan="1" colspan="1">AIC reduction <br /><br />from next-best model</th> <th rowspan="1" colspan="1">Antiviral rate</th> <th rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div> </div> </div></div></span><br /><br />[lci – uci] *</th> <th rowspan="1" colspan="1">�? <br /><br />[lci – uci] *</th> <th rowspan="1" colspan="1">β <br /><br />[lci – uci] *</th> <th rowspan="1" colspan="1">Mean field <br /><br />R<sub>0</sub></th> <th rowspan="1" colspan="1">Spatial <br /><br />β</th> </tr> </thead> <tbody> <tr> <td rowspan="3" colspan="1">Vero</td> <td rowspan="1" colspan="1">rVSV-G</td> <td rowspan="1" colspan="1">Absent</td> <td rowspan="1" colspan="1">2</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0 [0–0]</td> <td rowspan="1" colspan="1">0 [0–0]</td> <td rowspan="1" colspan="1">2.44 <br /><br />[1.52–3.36]</td> <td rowspan="1" colspan="1">8.73</td> <td rowspan="1" colspan="1">24.418</td> </tr> <tr> <td rowspan="1" colspan="1"> rVSV-EBOV</td> <td rowspan="1" colspan="1">Absent</td> <td rowspan="1" colspan="1">2</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0 [0–0]</td> <td rowspan="1" colspan="1">0 [0–0]</td> <td rowspan="1" colspan="1">1.5 <br /><br />[1.06–1.94]</td> <td rowspan="1" colspan="1">5.42</td> <td rowspan="1" colspan="1">14.996</td> </tr> <tr> <td rowspan="1" colspan="1"> rVSV-MARV</td> <td rowspan="1" colspan="1">Absent</td> <td rowspan="1" colspan="1">2</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0 [0–0]</td> <td rowspan="1" colspan="1">0 [0–0]</td> <td rowspan="1" colspan="1">0.975 <br /><br />[0.558–1.39]</td> <td rowspan="1" colspan="1">3.45</td> <td rowspan="1" colspan="1">9.752</td> </tr> <tr> <td rowspan="3" colspan="1">RoNi/7.1</td> <td rowspan="1" colspan="1">rVSV-G</td> <td rowspan="1" colspan="1">Induced</td> <td rowspan="1" colspan="1">2</td> <td rowspan="1" colspan="1">7.03 × 10<sup>−5</sup></td> <td rowspan="1" colspan="1">0 [0–0]</td> <td rowspan="1" colspan="1">0.089 <br /><br />[0–0.432]</td> <td rowspan="1" colspan="1">2.47 <br /><br />[1.49–3.45]</td> <td rowspan="1" colspan="1">10.91</td> <td rowspan="1" colspan="1">24.705</td> </tr> <tr> <td rowspan="1" colspan="1"> rVSV-EBOV</td> <td rowspan="1" colspan="1">Induced</td> <td rowspan="1" colspan="1">2.01</td> <td rowspan="1" colspan="1">2.87 × 10<sup>−5</sup></td> <td rowspan="1" colspan="1">0 [0–0]</td> <td rowspan="1" colspan="1">0.0363 <br /><br />[0–0.343]</td> <td rowspan="1" colspan="1">0.685 <br /><br />[0.451–0.919]</td> <td rowspan="1" colspan="1">3.04</td> <td rowspan="1" colspan="1">6.849</td> </tr> <tr> <td rowspan="1" colspan="1"> rVSV-MARV</td> <td rowspan="1" colspan="1">Induced</td> <td rowspan="1" colspan="1">2</td> <td rowspan="1" colspan="1">1.40 × 10<sup>−5</sup></td> <td rowspan="1" colspan="1">0 [0–0]</td> <td rowspan="1" colspan="1">0.0177 <br /><br />[0–0.257]</td> <td rowspan="1" colspan="1">1.23 <br /><br />[0.917–1.55]</td> <td rowspan="1" colspan="1">5.48</td> <td rowspan="1" colspan="1">12.324</td> </tr> <tr> <td rowspan="3" colspan="1">PaKiT01</td> <td rowspan="1" colspan="1">rVSV-G</td> <td rowspan="1" colspan="1">Constitutive</td> <td rowspan="1" colspan="1">29.9</td> <td rowspan="1" colspan="1">.00209</td> <td rowspan="1" colspan="1">0.00602 <br /><br />[0–0.019]</td> <td rowspan="1" colspan="1">8.26 × 10<sup>−8</sup><br /><br />[0–4.75 × 10<sup>−7</sup>]</td> <td rowspan="1" colspan="1">3.45 <br /><br />[1.07–5.84]</td> <td rowspan="1" colspan="1">6.20</td> <td rowspan="1" colspan="1">34.516</td> </tr> <tr> <td rowspan="1" colspan="1"> rVSV-EBOV</td> <td rowspan="1" colspan="1">Constitutive</td> <td rowspan="1" colspan="1">27.9</td> <td rowspan="1" colspan="1">.00499</td> <td rowspan="1" colspan="1">0.0478 <br /><br />[0–0.0958]</td> <td rowspan="1" colspan="1">4.46 × 10<sup>−8</sup><br /><br />[0–4.37 × 10<sup>−7</sup>]</td> <td rowspan="1" colspan="1">34.5 <br /><br />[28.7–40.2]</td> <td rowspan="1" colspan="1">18.82</td> <td rowspan="1" colspan="1">344.821</td> </tr> <tr> <td rowspan="1" colspan="1"> rVSV-MARV</td> <td rowspan="1" colspan="1">Induced</td> <td rowspan="1" colspan="1">2</td> <td rowspan="1" colspan="1">.00687</td> <td rowspan="1" colspan="1">0 [0–0]</td> <td rowspan="1" colspan="1">13.1 <br /><br />[0–37.9]</td> <td rowspan="1" colspan="1">3.25 <br /><br />[0–41.3]</td> <td rowspan="1" colspan="1">8.83</td> <td rowspan="1" colspan="1">32.452</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Improvement in AIC from next best model for same cell line-virus-MOI combination. All δ-AIC are reported in <a href="#supp4">Supplementary file 4</a>.</p> <p><sup>*</sup>lci = lower and uci = upper 95% confidence interval. No confidence interval is shown for spatial β which was fixed at 10 times the estimated mean for the mean field model fits when paired with equivalent values of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div> </div> </div></div></span> and �?.</p> </div> <div class="fn-type-" title=""> <p>All other parameters were fixed at: b = 0.025 (mean field), 0.15 (spatial); α = 1/6; c = 0; μ = 1/121 (Vero), 1/191 (RoNi/7.1), and 1/84 (PaKiT01).</p> </div> </div> </div> </div> <div class="robustimmunityislinkedtorapidwithin-hostvirustransmissionratesinfittedmodels" title="sec"> <div class="title" tagx="title" title="title">
Robust immunity is linked to rapid within-host virus transmission rates in fitted models</div> <p>In fitting our theoretical model to in vitro data, we estimated the within-host virus transmission rate (β) and the rate(s) of cellular acquisition to antiviral status (�? or �? + <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div></div></span>) (<a href="#table1">Table 1</a>; <a href="#supp4">Supplementary file 4</a>). Under absent immune assumptions, �? and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div></div></span> were fixed at 0 while β was estimated; under induced immune assumptions, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div></div></span> was fixed at 0 while �? and β were estimated; and under constitutive immune assumptions, all three parameters (�?, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div></div></span>, and β) were simultaneously estimated for each cell-virus combination. Best fit parameter estimates for MOI=0.001 data are visualized in conjunction with β – �? and β – <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div></div></span> bifurcations in <a href="#fig4">Figure 4</a>; all general patterns were recapitulated at lower values for β on MOI=0.0001 trials (<a href="#fig4s1">Figure 4—figure supplement 1</a>).</p> <p>As anticipated, the immune absent model (a simple target cell model) offered the best fit to IFN-deficient Vero cell infections (<a href="#fig4">Figure 4</a>; <a href="#table1">Table 1</a>; <a href="#supp4">Supplementary file 4</a>). Among Vero cell trials, infections with rVSV-G produced the highest β estimates, followed by infections with rVSV-EBOV and rVSV-MARV. Best fit parameter estimates on Vero cell lines localized in the region of parameter space corresponding to theoretical limit cycles, consistent with observed virus-induced epidemic extinctions in stochastic tissue cultures.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4.</span></a></div>   <div class="title" tagx="title" title="title">
Best fit parameter estimates for β and �? or <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div></div></span> from mean-field model fits to MOI=0.001 time series data, atop (<b>A,B</b>) β – �? and (<b>C</b>) β – <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div></div></span> bifurcation.</div> <p>Fits and bifurcations are grouped by immune phenotype: (<b>A</b>) absent; (<b>B</b>) induced; (<b>C</b>) constitutive immunity, with cell lines differentiated by shape (Vero=circles; RoNi/7.1 = triangles; PaKiT01=squares) and viral infections by color (rVSV-G = green, rVSV-EBOV = magenta, rVSV-MARV = blue). Note that y-axis values are ten-fold higher in panel (<b>C</b>). Branch point curves (solid lines) and Hopf curves (dashed lines) are reproduced from <a href="#fig3">Figure 3</a>. White space indicates endemic equilibrium (pathogen persistence), gray space indicates limit cycling (virus-induced epidemic extinction), and black space indicates no infection (immune-mediated pathogen extinction). In panel (<b>A</b>) and (<b>B</b>), <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div></div></span> is fixed at 0; in panel (<b>C</b>), �? is fixed at 5x10<sup>−8</sup> for bifurcation curves and estimated at 4x10<sup>−8</sup> and 8x10<sup>−8</sup> for rVSV-EBOV and rVSV-G parameter points, respectively. Other parameter values were fixed at: <i>b</i> = .025, μ = 0.001, σ = 1/6, α = 1/6, and c = 0 across all panels. Raw fitted values and corresponding 95% confidence intervals for β, �?, and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div> </div> </div></div></span>, background parameter values, and AIC recovered from model fit, are reported in <a href="#supp4">Supplementary file 4</a>. Parameter fits at MOI=0.0001 are visualized in <a href="#fig4s1">Figure 4—figure supplement 1</a>.</p>   </div> <p>In contrast to Vero cells, the induced immunity model offered the best fit to all RoNi/7.1 data, consistent with reported patterns in the literature and our own validation by qPCR (<a href="#table1">Table 1</a>; <a href="#fig1s6">Figure 1—figure supplement 6</a>; <a href="#bib2">Arnold et al., 2018</a>; <a href="#bib27">Kuzmin et al., 2017</a>; <a href="#bib4">Biesold et al., 2011</a>; <a href="#bib40">Pavlovich et al., 2018</a>). As in Vero cell trials, we estimated highest β values for rVSV-G infections on RoNi/7.1 cell lines but here recovered higher β estimates for rVSV-MARV than for rVSV-EBOV. This reversal was balanced by a higher estimated rate of acquisition to antiviral status (�?) for rVSV-EBOV versus rVSV-MARV. In general, we observed that more rapid rates of antiviral acquisition (either induced, �?, constitutive, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div></div></span>, or both) correlated with higher transmission rates (β). When offset by �?, β values estimated for RoNi/7.1 infections maintained the same amplitude as those estimated for immune-absent Vero cell lines but caused gentler epidemics and reduced cellular mortality (<a href="#fig1">Figure 1</a>). RoNi/7.1 parameter estimates localized in the region corresponding to endemic equilibrium for the deterministic, theoretical model (<a href="#fig4">Figure 4</a>), yielding less acute epidemics which nonetheless went extinct in stochastic experiments.</p> <p>Finally, rVSV-G and rVSV-EBOV trials on PaKiT01 cells were best fit by models assuming constitutive immunity, while rVSV-MARV infections on PaKiT01 were matched equivalently by models assuming either induced or constitutive immunity—with induced models favored over constitutive in AIC comparisons because one fewer parameter was estimated (<a href="#fig1s4">Figure 1—figure supplements 4</a>–<a href="#fig1s5">5</a>; <a href="#supp4">Supplementary file 4</a>). For all virus infections, PaKiT01 cell lines yielded β estimates a full order of magnitude higher than Vero or RoNi/7.1 cells, with each β balanced by an immune response (either �?, or �? combined with <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div></div></span>) also an order of magnitude higher than that recovered for the other cell lines (<a href="#fig4">Figure 4</a>; <a href="#table1">Table 1</a>). As in RoNi/7.1 cells, PaKiT01 parameter fits localized in the region corresponding to endemic equilibrium for the deterministic theoretical model. Because constitutive immune processes can actually prohibit initial pathogen invasion, constitutive immune fits to rVSV-MARV infections on PaKiT01 cell lines consistently localized at or below the Branch point threshold for virus invasion (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div></div></span>). During model fitting for optimization of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div></div></span>, any parameter tests of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div></div></span> values producing <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="mn_UNKNOWN">
1</div> </div> </div></div></span> resulted in no infection and, consequently, produced an exceedingly poor fit to infectious time series data. In all model fits assuming constitutive immunity, across all cell lines, parameter estimates for �? and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div></div></span> traded off, with one parameter optimized at values approximating zero, such that the immune response was modeled as almost entirely induced or entirely constitutive (<a href="#table1">Table 1</a>; <a href="#supp4">Supplementary file 4</a>). For RoNi/7.1 cells, even when constitutive immunity was allowed, the immune response was estimated as almost entirely induced, while for rVSV-G and rVSV-EBOV fits on PaKiT01 cells, the immune response optimized as almost entirely constitutive. For rVSV-MARV on PaKiT01 cells, however, estimation of �? was high under all assumptions, such that any additional antiviral contributions from <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div></div></span> prohibited virus from invading at all. The induced immune model thus produced a more parsimonious recapitulation of these data because virus invasion was always permitted, then rapidly controlled.</p> </div> <div class="antiviralcellssafeguardlivecellsagainstrapidcellmortalitytoelongateepidemicdurationinvitro" title="sec"> <div class="title" tagx="title" title="title">
Antiviral cells safeguard live cells against rapid cell mortality to elongate epidemic duration <i>in vitro</i></div> <p>In order to compare the relative contributions of each cell line’s disparate immune processes to epidemic dynamics, we next used our mean field parameter estimates to calculate the initial ‘antiviral rate’—the initial accumulation rate of antiviral cells upon virus invasion for each cell-virus-MOI combination—based on the following equation: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(8)</span><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mi_UNKNOWN">
n</div> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
l</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
R</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
e</div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
=</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
�?</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mi_UNKNOWN">
E</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mi_UNKNOWN">
s</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
ϵ</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mi_UNKNOWN">
s</div> </div> </div> </div></div></div>where P<sub>E</sub> was calculated from the initial infectious dose (MOI) of each infection experiment and P<sub>S</sub> was estimated at disease-free equilibrium: <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(9)</span><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN" /> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
-</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="mi_UNKNOWN">
M</div> <div class="mi_UNKNOWN">
O</div> <div class="mi_UNKNOWN">
I</div> </div> </div></div></div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(10)</span><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN" /> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
-</div> <div class="mi_UNKNOWN">
μ</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
c</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
μ</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
c</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
μ</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
ε</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div> <p /> <p>Because <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
�?</div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div></div></span> both contribute to this initial antiviral rate, induced and constitutive immune assumptions are capable of yielding equally rapid rates, depending on parameter fits. Indeed, under fully induced immune assumptions, the induced antiviral acquisition rate (�?) estimated for rVSV-MARV infection on PaKiT01 cells was so high that the initial antiviral rate exceeded even that estimated under constitutive assumptions for this cell-virus combination (<a href="#supp4">Supplementary file 4</a>). In reality, we know that NPC1 receptor incompatibilities make PaKiT01 cell lines constitutively refractory to rVSV-MARV infection (Ng and Chandrab, 2018, Unpublished results) and that PaKiT01 cells also constitutively express the antiviral cytokine, IFN-α. Model fitting results suggest that this constitutive expression of IFN-α may act more as a rapidly inducible immune response following virus invasion than as a constitutive secretion of functional IFN-α protein. Nonetheless, as hypothesized, PaKiT01 cell lines were by far the most antiviral of any in our study—with initial antiviral rates estimated several orders of magnitude higher than any others in our study, under either induced or constitutive assumptions (<a href="#table1">Table 1</a>; <a href="#supp4">Supplementary file 4</a>). RoNi/7.1 cells displayed the second-most-pronounced signature of immunity, followed by Vero cells, for which the initial antiviral rate was essentially zero even under forced assumptions of induced or constitutive immunity (<a href="#table1">Table 1</a>; <a href="#supp4">Supplementary file 4</a>).</p> <p>Using fitted parameters for <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div> </div> </div></div></span>, we additionally calculated R<sub>0</sub>, the basic reproduction number for the virus, for each cell line-virus-MOI combination (<a href="#table1">Table 1</a>; <a href="#supp4">Supplementary file 4</a>). We found that R<sub>0</sub> was essentially unchanged across differing immune assumptions for RoNi/7.1 and Vero cells, for which the initial antiviral rate was low. In the case of PaKiT01 cells, a high initial antiviral rate under either induced or constitutive immunity resulted in a correspondingly high estimation of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> </div> </div></div></span> (and, consequently, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div> </div> </div></div></span>) which still produced the same epidemic curve that resulted from the much lower estimates for <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> </div> </div></div></span> and R<sub>0</sub> paired with absent immunity. These findings suggest that antiviral immune responses protect host tissues against virus-induced cell mortality and may facilitate the establishment of more rapid within-host transmission rates.</p> <p>Total monolayer destruction occurred in all cell-virus combinations excepting rVSV-EBOV infections on RoNi/7.1 cells and rVSV-EBOV and rVSV-MARV infections on PaKiT01 cells. Monolayer destruction corresponded to susceptible cell depletion and epidemic turnover where R-effective (the product of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span> and the proportion susceptible) was reduced below one (<a href="#fig5">Figure 5</a>). For rVSV-EBOV infections on RoNi/7.1, induced antiviral cells safeguarded remnant live cells, which birthed new susceptible cells late in the time series. In rVSV-EBOV and rVSV-MARV infections on PaKiT01 cells, this antiviral protection halted the epidemic (<a href="#fig5">Figure 5</a>; R-effective &amp;lt;1) before susceptibles fully declined. In the case of rVSV-EBOV on PaKiT01, the birth of new susceptibles from remnant live cells protected by antiviral status maintained late-stage transmission to facilitate long-term epidemic persistence. Importantly, under fixed parameter values for the infection incubation rate (σ) and infection-induced mortality rate (α), models were unable to reproduce the longer-term infectious time series captured in data from rVSV-EBOV infections on PaKiT01 cell lines without incorporation of cell births, an assumption adopted in previous modeling representations of IFN-mediated viral dynamics in tissue culture (<a href="#bib23">Howat et al., 2006</a>). In our experiments, we observed that cellular reproduction took place as plaque assays achieved confluency.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5.</span></a></div>   <div class="title" tagx="title" title="title">
Fitted time series of susceptible (green shading) and antiviral (blue shading) cell proportions from the mean field model for rVSV-G, rVSV-EBOV, and rVSV-MARV infections (columns) on Vero, RoNi/7.1, and PaKiT01 cell lines (rows) at MOI = 0.001.</div> <p>Results are shown for the best fit immune absent model on Vero cells, induced immunity model on RoNi/7.1 cells and constitutive (rVSV-G and rVSV-EBOV) and induced (rVSV-MARV) immune models on PaKiT01 cells. Combined live, uninfectious cell populations (S + A + E) are shown in tan shading, with raw live, uninfectious cell data from Hoechst stains visualized as open circles. The right-hand y-axis corresponds to R-effective (pink solid line) across each time series; R-effective = 1 is a pink dashed, horizontal line. Panel background corresponds to empirical outcome of the average stochastic cell culture trial (persistent infection = white; virus-induced epidemic extinction = gray; immune-mediated epidemic extinction = black). Parameter values are listed in <a href="#supp4">Supplementary file 4</a> and results for absent/induced/constitutive fitted models across all cell lines in <a href="#fig5s1">Figure 5—figure supplement 1</a> (MOI = 0.001) and <a href="#fig5s2">Figure 5—figure supplement 2</a> (MOI = 0.0001).</p>   </div> <p>Finally, because the protective effect of antiviral cells is more clearly observable spatially, we confirmed our results by simulating fitted time series in a spatially-explicit, stochastic reconstruction of our mean field model. In spatial simulations, rates of antiviral acquisition were fixed at fitted values for �? and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div> </div> </div></div></span> derived from mean field estimates, while transmission rates (β) were fixed at values ten times greater than those estimated under mean field conditions, accounting for the intensification of parameter thresholds permitting pathogen invasion in local spatial interactions (see Materials and methods; <a href="#video1">Videos 1</a>–<a href="#video3">3</a>; <a href="#fig5s3">Figure 5—figure supplement 3</a>; <a href="#supp5">Supplementary file 5</a>; <a href="#bib58">Webb et al., 2007</a>). In immune capable time series, spatial antiviral cells acted as ‘refugia’ which protected live cells from infection as each initial epidemic wave ‘washed’ across a cell monolayer. Eventual birth of new susceptibles from these living refugia allowed for sustained epidemic transmission in cases where some infectious cells persisted at later timepoints in simulation (<a href="#video1">Videos 1</a>–<a href="#video3">3</a>; <a href="#fig5s3">Figure 5—figure supplement 3</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Video 1.</span></a></div>   <div class="title" tagx="title" title="title">
Two hundred hour time series of spatial stochastic model for rVSV-EBOV infection on 10,000 cell grid for PaKiT01, assuming conditions of <i>absent immunity</i>: (A) spatial spread of infection, (B) time series of state variables.</div> <p>Parameter values are listed in <a href="#supp4">Supplementary file 4</a>.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Video 2.</span></a></div>   <div class="title" tagx="title" title="title">
Two hundred hour time series of spatial stochastic model for rVSV-EBOV infection on 10,000 cell grid for PaKiT01, assuming conditions of <i>induced immunity</i>: (A) spatial spread of infection, (B) time series of state variables.</div> <p>Parameter values are listed in <a href="#supp4">Supplementary file 4</a>.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Video 3.</span></a></div>   <div class="title" tagx="title" title="title">
Two hundred hour time series of spatial stochastic model for rVSV-EBOV infection on 10,000 cell grid for PaKiT01, assuming conditions of <i>constitutive immunity</i>: (A) spatial spread of infection, (B) time series of state variables.</div> <p>Parameter values are listed in <a href="#supp4">Supplementary file 4</a>.</p>   </div> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Bats are reservoirs for several important emerging zoonoses but appear not to experience disease from otherwise virulent viral pathogens. Though the molecular biological literature has made great progress in elucidating the mechanisms by which bats tolerate viral infections (<a href="#bib62">Zhou et al., 2016</a>; <a href="#bib1">Ahn et al., 2019</a>; <a href="#bib60">Xie et al., 2018</a>; <a href="#bib40">Pavlovich et al., 2018</a>; <a href="#bib61">Zhang et al., 2013</a>), the impact of unique bat immunity on virus dynamics within-host has not been well-elucidated. We used an innovative combination of <i>in vitro</i> experimentation and within-host modeling to explore the impact of unique bat immunity on virus dynamics. Critically, we found that bat cell lines demonstrated a signature of enhanced interferon-mediated immune response, of either constitutive or induced form, which allowed for establishment of rapid within-host, cell-to-cell virus transmission rates (β). These results were supported by both data-independent bifurcation analysis of our mean field theoretical model, as well as fitting of this model to viral infection time series established in bat cell culture. Additionally, we demonstrated that the antiviral state induced by the interferon pathway protects live cells from mortality in tissue culture, resulting in <i>in vitro</i> epidemics of extended duration that enhance the probability of establishing a long-term persistent infection. Our findings suggest that viruses evolved in bat reservoirs possessing enhanced IFN capabilities could achieve more rapid within-host transmission rates without causing pathology to their hosts. Such rapidly-reproducing viruses would likely generate extreme virulence upon spillover to hosts lacking similar immune capacities to bats.</p> <p>To achieve these results, we first developed a novel, within-host, theoretical model elucidating the effects of unique bat immunity, then undertook bifurcation analysis of the model’s equilibrium properties under immune absent, induced, and constitutive assumptions. We considered a cell line to be constitutively immune if possessing any number of antiviral cells at disease-free equilibrium but allowed the extent of constitutive immunity to vary across the parameter range for <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div></div></span>, the constitutive rate of antiviral acquisition. In deriving the equation for <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div></div></span>, the basic reproduction number, which defines threshold conditions for virus invasion of a tissue (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="mo_UNKNOWN">
&amp;gt;</div> <div class="mn_UNKNOWN">
1</div> </div> </div></div></span>), we demonstrated how the invasion threshold is elevated at high values of constitutive antiviral acquisition, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div> </div> </div></div></span>. Constitutive immune processes can thus prohibit pathogen invasion, while induced responses, by definition, can only control infections <i>post-hoc</i>. Once thresholds for pathogen invasion have been met, assumptions of constitutive immunity will limit the cellular mortality (virulence) incurred at high transmission rates. Regardless of mechanism (induced or constitutive), interferon-stimulated antiviral cells appear to play a key role in maintaining longer term or persistent infections by safeguarding susceptible cells from rapid infection and concomitant cell death.</p> <p>Fitting of our model to <i>in vitro</i> data supported expected immune phenotypes for different bat cell lines as described in the literature. Simple target cell models that ignore the effects of immunity best recapitulated infectious time series derived from IFN-deficient Vero cells, while models assuming induced immune processes most accurately reproduced trials derived from RoNi/7.1 (<i>Rousettus aegyptiacus</i>) cells, which possess a standard virus-induced IFN-response. In most cases, models assuming constitutive immune processes best recreated virus epidemics produced on PaKiT01 (<i>Pteropus alecto</i>) cells, which are known to constitutively express the antiviral cytokine, IFN-α (<a href="#bib62">Zhou et al., 2016</a>). Model support for induced immune assumptions in fits to rVSV-MARV infections on PaKiT01 cells suggests that the constitutive IFN-α expression characteristic of <i>P. alecto</i> cells may represent more of a constitutive immune priming process than a perpetual, functional, antiviral defense. Results from mean field model fitting were additionally confirmed in spatially explicit stochastic simulations of each time series.</p> <p>As previously demonstrated in within-host models for HIV (<a href="#bib9">Coffin, 1995</a>; <a href="#bib44">Perelson et al., 1996</a>; <a href="#bib36">Nowak et al., 1995</a>; <a href="#bib5">Bonhoeffer et al., 1997</a>; <a href="#bib21">Ho et al., 1995</a>), assumptions of simple target-cell depletion can often provide satisfactory approximations of viral dynamics, especially those reproduced in simple <i>in vitro</i> systems. Critically, our model fitting emphasizes the need for incorporation of top-down effects of immune control in order to accurately reproduce infectious time series derived from bat cell tissue cultures, especially those resulting from the robustly antiviral PaKiT01 <i>P. alecto</i> cell line. These findings indicate that enhanced IFN-mediated immune pathways in bat reservoirs may promote elevated within-host virus replication rates prior to cross-species emergence. We nonetheless acknowledge the limitations imposed by <i>in vitro</i> experiments in tissue culture, especially involving recombinant viruses and immortalized cell lines. Future work should extend these cell culture studies to include measurements of multiple state variables (i.e. antiviral cells) to enhance epidemiological inference.</p> <p>The continued recurrence of Ebola epidemics across central Africa highlights the importance of understanding bats’ roles as reservoirs for virulent zoonotic disease. The past decade has born witness to emerging consensus regarding the unique pathways by which bats resist and tolerate highly virulent infections (<a href="#bib7">Brook and Dobson, 2015</a>; <a href="#bib60">Xie et al., 2018</a>; <a href="#bib61">Zhang et al., 2013</a>; <a href="#bib1">Ahn et al., 2019</a>; <a href="#bib62">Zhou et al., 2016</a>; <a href="#bib34">Ng et al., 2015</a>; <a href="#bib40">Pavlovich et al., 2018</a>). Nonetheless, an understanding of the mechanisms by which bats support endemic pathogens at the population level, or promote the evolution of virulent pathogens at the individual level, remains elusive. Endemic maintenance of infection is a defining characteristic of a pathogen reservoir (<a href="#bib19">Haydon et al., 2002</a>), and bats appear to merit such a title, supporting long-term persistence of highly transmissible viral infections in isolated island populations well below expected critical community sizes (<a href="#bib42">Peel et al., 2012</a>). Researchers debate the relative influence of population-level and within-host mechanisms which might explain these trends (<a href="#bib46">Plowright et al., 2016</a>), but increasingly, field data are difficult to reconcile without acknowledgement of a role for persistent infections (<a href="#bib43">Peel et al., 2018</a>; <a href="#bib6">Brook et al., 2019</a>). We present general methods to study cross-scale viral dynamics, which suggest that within-host persistence is supported by robust antiviral responses characteristic of bat immune processes. Viruses which evolve rapid replication rates under these robust antiviral defenses may pose the greatest hazard for cross-species pathogen emergence into spillover hosts with immune systems that differ from those unique to bats.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="table-wrap_UNKNOWN" id="keyresource" orientation="portrait" position="anchor"><span class="label" tagx="label" title="label">Key resources table</span><table frame="hsides" rules="groups"> <thead> <tr> <th valign="top" rowspan="1" colspan="1">Reagent type (species) <br /><br />or resource</th> <th valign="top" rowspan="1" colspan="1">Designation</th> <th valign="top" rowspan="1" colspan="1">Source or reference</th> <th valign="top" rowspan="1" colspan="1">Identifiers</th> <th valign="top" rowspan="1" colspan="1">Additional information</th> </tr> </thead> <tbody> <tr> <td valign="top" rowspan="1" colspan="1">Cell line (Vero)</td> <td valign="top" rowspan="1" colspan="1">Kidney (normal, epithelial, adult)</td> <td valign="top" rowspan="1" colspan="1">ATCC</td> <td valign="top" rowspan="1" colspan="1">CCL-81</td> <td valign="top" rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Cell line (<i>Rousettus aegyptiacus)</i></td> <td valign="top" rowspan="1" colspan="1">Kidney (normal, epithelial, adult)</td> <td valign="top" rowspan="1" colspan="1">(<a href="#bib4">Biesold et al., 2011</a>; <a href="#bib26">Kühl et al., 2011</a>)</td> <td valign="top" rowspan="1" colspan="1">RoNi/7.1</td> <td valign="top" rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Cell line (<i>Pteropus alecto</i>)</td> <td valign="top" rowspan="1" colspan="1">Kidney (normal, epithelial, adult)</td> <td valign="top" rowspan="1" colspan="1">(<a href="#bib10">Crameri et al., 2009</a>)</td> <td valign="top" rowspan="1" colspan="1">PaKiT01</td> <td valign="top" rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Virus strain</td> <td valign="top" rowspan="1" colspan="1">Replication competent, recombinant vesicular stomatitis Indiana virus expressing eGFP</td> <td valign="top" rowspan="1" colspan="1">(<a href="#bib30">Miller et al., 2012</a>; <a href="#bib59">Wong et al., 2010</a>)</td> <td valign="top" rowspan="1" colspan="1">rVSV-G</td> <td valign="top" rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Virus strain</td> <td valign="top" rowspan="1" colspan="1">Replication competent, recombinant vesicular stomatitis Indiana virus expressing eGFP and EBOV GP in place of VSV G</td> <td valign="top" rowspan="1" colspan="1">(<a href="#bib30">Miller et al., 2012</a>; <a href="#bib59">Wong et al., 2010</a>)</td> <td valign="top" rowspan="1" colspan="1">rVSV-EBOV</td> <td valign="top" rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Virus strain</td> <td valign="top" rowspan="1" colspan="1">Replication competent, recombinant vesicular stomatitis Indiana virus expressing eGFP and MARV GP in place of VSV G</td> <td valign="top" rowspan="1" colspan="1">(<a href="#bib30">Miller et al., 2012</a>; <a href="#bib59">Wong et al., 2010</a>)</td> <td valign="top" rowspan="1" colspan="1">rVSV-MARV</td> <td valign="top" rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Reagent</td> <td valign="top" rowspan="1" colspan="1">Hoechst 33342 Fluorescent Stain</td> <td valign="top" rowspan="1" colspan="1">ThermoFisher</td> <td valign="top" rowspan="1" colspan="1">cat #: 62249</td> <td valign="top" rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Reagent</td> <td valign="top" rowspan="1" colspan="1">L-Glutamine Solution</td> <td valign="top" rowspan="1" colspan="1">ThermoFisher</td> <td valign="top" rowspan="1" colspan="1">cat #: 25030081</td> <td valign="top" rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Reagent</td> <td valign="top" rowspan="1" colspan="1">Gibco HEPES</td> <td valign="top" rowspan="1" colspan="1">ThermoFisher</td> <td valign="top" rowspan="1" colspan="1">cat #: 15630080</td> <td valign="top" rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Reagent</td> <td valign="top" rowspan="1" colspan="1">iTaq Universal SYBR Green Supermix</td> <td valign="top" rowspan="1" colspan="1">BioRad</td> <td valign="top" rowspan="1" colspan="1">cat #: 1725120</td> <td valign="top" rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Commercial assay or kit</td> <td valign="top" rowspan="1" colspan="1">Quick RNA Mini Prep Kit</td> <td valign="top" rowspan="1" colspan="1">Zymo</td> <td valign="top" rowspan="1" colspan="1">cat #: <br /><br />R1054</td> <td valign="top" rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Commercial assay or kit</td> <td valign="top" rowspan="1" colspan="1">Invitrogen Superscript III cDNA Synthesis Kit</td> <td valign="top" rowspan="1" colspan="1">ThermoFisher</td> <td valign="top" rowspan="1" colspan="1">cat #: 18080051</td> <td valign="top" rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Software</td> <td valign="top" rowspan="1" colspan="1">MatCont (version 2.2)</td> <td valign="top" rowspan="1" colspan="1">(<a href="#bib13">Dhooge et al., 2008</a>)</td> <td valign="top" rowspan="1" colspan="1">MatCont</td> <td valign="top" rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" rowspan="1" colspan="1">R</td> <td valign="top" rowspan="1" colspan="1">R version 3.6.0</td> <td valign="top" rowspan="1" colspan="1">(<a href="#bib47">R Development Core Team, 2019</a>)</td> <td valign="top" rowspan="1" colspan="1">R</td> <td valign="top" rowspan="1" colspan="1" /> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p><sup>*</sup>Note that primers for <i>R. aegyptiacus</i> and <i>P. alecto</i> β-Actin, IFN-α, and IFN-β genes are listed in the <a href="#supp6">Supplementary file 6</a>.</p> </div> </div> </div> <div class="cellcultureexperiments" title="sec"> <div class="title" tagx="title" title="title">
Cell culture experiments</div> <div class="cells" title="sec"> <div class="title" tagx="title" title="title">
Cells</div> <p>All experiments were carried out on three immortalized mammalian kidney cell lines: Vero (African green monkey), RoNi/7.1 (<i>Rousettus aegyptiacus</i>) (<a href="#bib26">Kühl et al., 2011</a>; <a href="#bib4">Biesold et al., 2011</a>) and PaKiT01 (<i>Pteropus alecto</i>) (<a href="#bib10">Crameri et al., 2009</a>). The species identifications of all bat cell lines was confirmed morphologically and genetically in the publications in which they were originally described (<a href="#bib26">Kühl et al., 2011</a>; <a href="#bib4">Biesold et al., 2011</a>; <a href="#bib10">Crameri et al., 2009</a>). ﻿Vero cells were obtained from ATCC.</p> <p>Monolayers of each cell line were grown to 90% confluency (~9×10<sup>5</sup> cells) in 6-well plates. Cells were maintained in a humidified 37 °C, 5% CO<sub>2</sub> incubator and cultured in Dulbecco’s modified Eagle medium (DMEM) (Life Technologies, Grand Island, NY), supplemented with 2% fetal bovine serum (FBS) (Gemini Bio Products, West Sacramento, CA), and 1% penicillin-streptomycin (Life Technologies). Cells were tested monthly for mycoplasma contamination while experiments were taking place; all cells assayed negative for contamination at every testing.</p> <p>Previous work has demonstrated that all cell lines used are capable of mounting a type I IFN response upon viral challenge, with the exception of Vero cells, which possess an IFN-β deficiency (<a href="#bib12">Desmyter et al., 1968</a>; <a href="#bib50">Rhim et al., 1969</a>; <a href="#bib17">Emeny and Morgan, 1979</a>). RoNi/7.1 cells have been shown to mount idiosyncratic induced IFN defenses upon viral infection (<a href="#bib40">Pavlovich et al., 2018</a>; <a href="#bib27">Kuzmin et al., 2017</a>; <a href="#bib2">Arnold et al., 2018</a>; <a href="#bib26">Kühl et al., 2011</a>; <a href="#bib4">Biesold et al., 2011</a>), while PaKiT01 cells are known to constitutively express the antiviral cytokine, IFN-α (<a href="#bib62">Zhou et al., 2016</a>). This work is the first documentation of IFN signaling induced upon challenge with the particular recombinant VSVs outlined below. We verified known antiviral immune phenotypes via qPCR. Results were consistent with the literature, indicating a less pronounced role for interferon defense against viral infection in RoNi/7.1 versus PaKiT01 cells.</p> </div> <div class="viruses" title="sec"> <div class="title" tagx="title" title="title">
Viruses</div> <p>Replication-capable recombinant vesicular stomatitis Indiana viruses, expressing filovirus glycoproteins in place of wild type G (rVSV-G, rVSV-EBOV, and rVSV-MARV) have been previously described (<a href="#bib59">Wong et al., 2010</a>; <a href="#bib30">Miller et al., 2012</a>). Viruses were selected to represent a broad range of anticipated antiviral responses from host cells, based on a range of past evolutionary histories between the virus glycoprotein mediating cell entry and the host cell’s entry receptor. These interactions ranged from the total absence of evolutionary history in the case of rVSV-G infections on all cell lines to a known receptor-level cell entry incompatibility in the case of rVSV-MARV infections on PaKiT01 cell lines.</p> <p>To measure infectivities of rVSVs on each of the cell lines outlined above, so as to calculate the correct viral dose for each MOI, NH<sub>4</sub>Cl (20 mM) was added to infected cell cultures at 1–2 hr post-infection to block viral spread, and individual eGFP-positive cells were manually counted at 12–14 hr post-infection.</p> </div> <div class="innateimmunephenotypesviaqpcrofifngenes" title="sec"> <div class="title" tagx="title" title="title">
Innate immune phenotypes via qPCR of IFN genes</div> <p>Previously published work indicates that immortalized kidney cell lines of <i>Rousettus aegyptiacus</i> (RoNi/7.1) and <i>Pteropus alecto</i> (PaKiT01) exhibit different innate antiviral immune phenotypes through, respectively, induced (<a href="#bib4">Biesold et al., 2011</a>; <a href="#bib40">Pavlovich et al., 2018</a>; <a href="#bib26">Kühl et al., 2011</a>; <a href="#bib2">Arnold et al., 2018</a>) and constitutive (<a href="#bib62">Zhou et al., 2016</a>) expression of type I interferon genes. We verified these published phenotypes on our own cell lines infected with rVSV-G, rVSV-EBOV, and rVSV-MARV via qPCR of IFN-α and IFN-β genes across a longitudinal time series of infection.</p> <p>Specifically, we carried out multiple time series of infection of each cell line with each of the viruses described above, under mock infection conditions and at MOIs of 0.0001 and 0.001—with the exception of rVSV-MARV on PaKiT01 cell lines, for which infection was only performed at MOI = 0.0001 due to limited viral stocks and the extremely low infectivity of this virus on this cell line (thus requiring high viral loads for initial infection). All experiments were run in duplicate on 6-well plates, such that a typical plate for any of the three viruses had two control (mock) wells, two MOI = 0.0001 wells and two MOI = 0.001 wells, excepting PaKiT01 plates, which had two control and four MOI = 0.0001 wells at a given time. We justify this PaKiT01 exemption through the expectation that IFN-α expression is constitutive for these cells, and by the assumption that any expression exhibited at the lower MOI should also be present at the higher MOI.</p> <p>For these gene expression time series, four 6-well plates for each cell line–virus combination were incubated with virus for one hour at 37 °C. Following incubation, virus was aspirated off, and cell monolayers were washed in PBS, then covered with an agar plaque assay overlay to mimic conditions under which infection trials were run. Plates were then harvested sequentially at timepoints of roughly 5, 10, 15, and 20 hr post-infection (exact timing varied as multiple trials were running simultaneously). Upon harvest of each plate, agar overlay was removed, and virus was lysed and RNA extracted from cells using the Zymo Quick RNA Mini Prep kit, according to the manufacturer’s instructions and including the step for cellular DNA digestion. Post-extraction, RNA quality was verified via nanodrop, and RNA was converted to cDNA using the Invitrogen Superscript III cDNA synthesis kit, according to the manufacturer’s instructions. cDNA was then stored at 4 °C and as a frozen stock at −20 °C to await qPCR.</p> <p>We undertook qPCR of cDNA to assess expression of the type I interferon genes, IFN-α and IFN-β, and the housekeeping gene, β-Actin, using primers previously reported in the literature (<a href="#supp6">Supplementary file 6</a>). For qPCR, 2 µl of each cDNA sample was incubated with 7 µl of deionized water, 1 µl of 5 UM forward/reverse primer mix and 10 µl of iTaq Universal SYBR Green, then cycled on a QuantStudio3 Real-Time PCR machine under the following conditions: initial denaturation at 94 °C for 2 min followed by 40 cycles of: denaturation at 95 °C (5 s), annealing at 58 °C (15 s), and extension at 72 °C (10 s).</p> <p>We report simple δ-Ct values for each run, with raw Ct of the target gene of interest (IFN-α or IFN-β) subtracted from raw Ct of the β-Actin housekeeping gene in <a href="#fig1s6">Figure 1—figure supplement 6</a>. Calculation of fold change upon viral infection in comparison to mock using the δ-δ-Ct method (<a href="#bib28">Livak and Schmittgen, 2001</a>) was inappropriate in this case, as we wished to demonstrate constitutive expression of IFN-α in PaKiT01 cells, whereby data from mock cells was identical to that produced from infected cells.</p> </div> <div class="plaqueassaysandtimeseriesimaging" title="sec"> <div class="title" tagx="title" title="title">
Plaque assays and time series imaging</div> <p>After being grown to ~90% confluency, cells were incubated with pelleted rVSVs expressing eGFP (rVSV-G, rVSV-EBOV, rVSV-MARV). Cell lines were challenged with both a low (0.0001) and high (0.001) multiplicity of infection (MOI) for each virus. In a cell monolayer infected at a given MOI (m), the proportion of cells (P), infected by k viral particles can be described by the Poisson distribution: <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span>, such that the number of initially infected cells in an experiment equals: <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="mo_UNKNOWN">
&amp;gt;</div> <div class="mn_UNKNOWN">
1</div></div></span>. We assumed that a ~90% confluent culture at each trial’s origin was comprised of ~9x10<sup>5</sup> cells and conducted all experiments at MOIs of 0.0001 and 0.001, meaning that we began each trial by introducing virus to, respectively, ~81 or 810 cells, representing the state variable ‘E’ in our theoretical model. Low MOIs were selected to best approximate the dynamics of mean field infection and limit artifacts of spatial structuring, such as premature epidemic extinction when growing plaques collide with plate walls in cell culture.</p> <p>Six-well plates were prepared with each infection in duplicate or triplicate, such that a control well (no virus) and 2–3 wells each at MOI 0.001 and 0.0001 were incubated simultaneously on the same plate. In total, we ran between 18 and 39 trials at each cell-virus-MOI combination, excepting r-VSV-MARV infections on PaKiT01 cells at MOI = 0.001, for which we ran only eight trials due to the low infectivity of this virus on this cell line, which required high viral loads for initial infection. Cells were incubated with virus for one hour at 37 °C. Following incubation, virus was aspirated off, and cell monolayers were washed in PBS, then covered with a molten viscous overlay (50% 2X MEM/L-glutamine; 5% FBS; 3% HEPES; 42% agarose), cooled for 20 min, and re-incubated in their original humidified 37 °C, 5% CO<sub>2</sub> environment.</p> <p>After application of the overlay, plates were monitored periodically using an inverted fluorescence microscope until the first signs of GFP expression were witnessed (~6–9.5 hr post-infection, depending on the cell line and virus under investigation). From that time forward, a square subset of the center of each well (comprised of either 64- or 36-subframes and corresponding to roughly 60% and 40% of the entire well space) was imaged periodically, using a CellInsight CX5 High Content Screening (HCS) Platform with a 4X air objective (ThermoFisher, Inc, Waltham, MA). Microscope settings were held standard across all trials, with exposure time fixed at 0.0006 s for each image. One color channel was imaged, such that images produced show GFP-expressing cells in white and non-GFP-expressing cells in black (<a href="#fig1s1">Figure 1—figure supplement 1</a>).</p> <p>Wells were photographed in rotation, as frequently as possible, from the onset of GFP expression until the time that the majority of cells in the well were surmised to be dead, GFP expression could no longer be detected, or early termination was desired to permit Hoechst staining.</p> <p>In the case of PaKiT01 cells infected with rVSV-EBOV, where an apparently persistent infection established, the assay was terminated after 200+ hours (8+ days) of continuous observation. Upon termination of all trials, cells were fixed in formaldehyde (4% for 15 min), incubated with Hoechst stain (0.0005% for 15 min) (ThermoFisher, Inc, Waltham, MA), then imaged at 4X on the CellInsight CX5 High Content Screening (HCS) Platform. The machine was allowed to find optimal focus for each Hoechst stain image. One color channel was permitted such that images produced showed live nuclei in white and dead cells in black.</p> </div> <div class="hoechststaining" title="sec"> <div class="title" tagx="title" title="title">
Hoechst staining</div> <p>Hoechst stain colors cellular DNA, and viral infection is thought to interfere with the clarity of the stain (<a href="#bib11">Dembowski and DeLuca, 2015</a>). As such, infection termination, cell fixation, and Hoechst staining enables generation of a rough time series of uninfectious live cells (i.e. susceptible + antiviral cells) to complement the images which produced time series of proportions infectious. Due to uncertainty over the exact epidemic state of Hoechst-stained cells (i.e. exposed but not yet infectious cells may still stain), we elected to fit our models only to the infectious time series derived from GFP-expressing images and used Hoechst stain images as a <i>post hoc</i> visual check on our fit only (<a href="#fig5">Figure 5</a>; <a href="#fig5s1">Figure 5—figure supplements 1</a>–<a href="#fig5s2">2</a>).</p> </div> <div class="imageprocessing" title="sec"> <div class="title" tagx="title" title="title">
Image processing</div> <p>Images recovered from the time series above were processed into binary (‘infectious’ vs. ‘non-infectious’ or, for Hoechst-stained images, ‘live’ vs. ‘dead’) form using the EBImage package (<a href="#bib39">Pau et al., 2010</a>) in R version 3.6 for MacIntosh, after methods further detailed in <a href="#supp7">Supplementary file 7</a>. Binary images were then further processed into time series of infectious or, for Hoechst-stained images, live cells using a series of cell counting scripts. Because of logistical constraints (i.e. many plates of simultaneously running infection trials and only one available imaging microscope), the time course of imaging across the duration of each trial was quite variable. As such, we fitted a series of statistical models to our processed image data to reconstruct reliable values of the infectious proportion of each well per hour for each distinct trial in all cell line–virus-MOI combinations (<a href="#fig1s2">Figure 1—figure supplements 2</a>–<a href="#fig1s3">3</a>), as well as for declining live cell counts from control well data derived from the Hoestch time series (<a href="#supp1">Supplementary file 1</a>; <a href="#supp7">Supplementary file 7</a>; <a href="#fig1s7">Figure 1—figure supplement 7</a>). All original and processed images, image processing and counting code, and resulting time series data are freely available for download at the following FigShare repository: DOI: <a href="http://doi.org/10.6084/m9.figshare.8312807">10.6084/m9.figshare.8312807</a>.</p> </div> </div> <div class="meanfieldmodel" title="sec"> <div class="title" tagx="title" title="title">
Mean field model</div> <div class="theoreticalmodeldetails" title="sec"> <div class="title" tagx="title" title="title">
Theoretical model details</div> <p>To derive the expression for <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mn_UNKNOWN">
0</div> </div> </div> </div></div></span>, the basic pathogen reproductive number in vitro, we used Next Generation Matrix (NGM) techniques (<a href="#bib14">Diekmann et al., 1990</a>; <a href="#bib20">Heffernan et al., 2005</a>), employing Wolfram Mathematica (version 11.2) as an analytical tool. <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div> </div> </div></div></span> describes the number of new infections generated by an existing infection in a completely susceptible host population; a pathogen will invade a population when <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
&amp;gt;</div> <div class="mn_UNKNOWN">
1</div> </div> </div></div></span> (<a href="#supp2">Supplementary file 2</a>). We then analyzed stability properties of the system, exploring dynamics across a range of parameter spaces, using MatCont (version 2.2) (<a href="#bib13">Dhooge et al., 2008</a>) for Matlab (version R2018a) (<a href="#supp3">Supplementary file 3</a>).</p> </div> <div class="theoreticalmodelfitting" title="sec"> <div class="title" tagx="title" title="title">
Theoretical model fitting</div> <p>The birth rate, <i>b,</i> and natural mortality rate, μ, balance to yield a population-level growth rate, such that it is impossible to estimate both <i>b</i> and μ simultaneously from total population size data alone. As such, we fixed <i>b</i> at. 025 and estimated μ by fitting an infection-absent version of our mean field model to the susceptible time series derived via Hoechst staining of control wells for each of the three cell lines (<a href="#fig1s7">Figure 1—figure supplement 7</a>). This yielded a natural mortality rate, μ, corresponding to a lifespan of approximately 121, 191, and 84 hours, respectively, for Vero, RoNi/7.1, and PaKiT01 cell lines (<a href="#fig1s7">Figure 1—figure supplement 7</a>). We then fixed the virus incubation rate, σ, as the inverse of the shortest observed duration of time from initial infection to the observation of the first infectious cells via fluorescent microscope for all nine cell line – virus combinations (ranging 6 to 9.5 hours). We fixed α, the infection-induced mortality rate, at 1/6, an accepted standard for general viral kinetics (<a href="#bib23">Howat et al., 2006</a>), and held <i>c,</i> the rate of antiviral cell regression to susceptible status, at 0 for the timespan (&amp;lt;200 hours) of the experimental cell line infection trials.</p> <p>We estimated cell line–virus-MOI-specific values for β, �?, and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div> </div> </div></div></span> by fitting the deterministic output of infectious proportions in our mean field model to the full suite of statistical outputs of all trials for each infected cell culture time series (<a href="#fig1s2">Figure 1—figure supplements 2</a>–<a href="#fig1s3">3</a>). Fitting was performed by minimizing the sum of squared differences between the deterministic model output and cell line–virus-MOI-specific infectious proportion of the data at each timestep. We optimized parameters for MOI = 0.001 and 0.0001 simultaneously to leverage statistical power across the two datasets, estimating a different transmission rate, β, for trials run at each infectious dose but, where applicable, estimating the same rates of �? and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div> </div> </div></div></span> across the two time series. We used the differential equation solver lsoda() in the R package deSolve (<a href="#bib54">Soetaert et al., 2010</a>) to obtain numerical solutions for the mean field model and carried out minimization using the ‘Nelder-Mead’ algorithm of the optim() function in base R. All model fits were conducted using consistent starting guesses for the parameters, β (β = 3), and where applicable, �? (�? = 0.001) and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div> </div> </div></div></span> (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div> </div> </div></div></span> = 0.001). In the case of failed fits or indefinite hessians, we generated a series of random guesses around the starting conditions and continued estimation until successful fits were achieved.</p> <p>All eighteen cell line–virus-MOI combinations of data were fit by an immune absent (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div> </div> </div></div></span> = �? = 0) version of the theoretical model and, subsequently, an induced immunity (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div> </div> </div></div></span> = 0; �? &amp;gt;0) and constitutive immunity (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div> </div> </div></div></span> &amp;gt;0; �? &amp;gt;0) version of the model. Finally, we compared fits across each cell line–virus-MOI combination via AIC. In calculating AIC, the number of fitted parameters in each model (<i>k</i>) varied across the immune phenotypes, with one parameter (β) estimated for absent immune assumptions, two (β and �?) for induced immune assumptions, and three (β, �?, and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div> </div> </div></div></span>) for constitutive immune assumptions. The sample size (<i>n</i>) corresponded to the number of discrete time steps across all empirical infectious trials to which the model was fitted for each cell-line virus combination. All fitting and model comparison scripts are freely available for download at the following FigShare repository: DOI: <a href="http://doi.org/10.6084/m9.figshare.8312807">10.6084/m9.figshare.8312807</a>.</p> </div> <div class="spatialmodelsimulations" title="sec"> <div class="title" tagx="title" title="title">
Spatial model simulations</div> <p>Finally, we verified all mean field fits in a spatial context, in order to more thoroughly elucidate the role of antiviral cells in each time series. We constructed our spatial model in C++ implemented in R using the packages Rcpp and RcppArmadillo (<a href="#bib15">Eddelbuettel and Francois, 2011</a>; <a href="#bib16">Eddelbuettel and Sanderson, 2017</a>). Following <a href="#bib32">Nagai and Honda (2001)</a> and <a href="#bib23">Howat et al. (2006)</a>, we modeled this system on a two-dimensional hexagonal lattice, using a ten-minute epidemic timestep for cell state transitions. At the initialization of each simulation, we randomly assigned a duration of natural lifespan, incubation period, infectivity period, and time from antiviral to susceptible status to all cells in a theoretical monolayer. Parameter durations were drawn from a normal distribution centered at the inverse of the respective fixed rates of μ, σ, α, and <i>c</i>, as reported with our mean field model. Transitions involving the induced (�?) and constitutive (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div> </div> </div></div></span>) rates of antiviral acquisition were governed probabilistically and adjusted dynamically at each timestep based on the global environment. As such, we fixed these parameters at the same values estimated in the mean field model, and multiplied both �? and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ε</div> </div> </div></div></span> by the global proportion of, respectively, exposed and susceptible cells at a given timestep.</p> <p>In contrast to antiviral acquisition rates, transitions involving the birth rate (<i>b</i>) and the transmission rate (β) occurred probabilistically based on each cell’s local environment. The birth rate, <i>b</i>, was multiplied by the proportion of susceptible cells within a six-neighbor circumference of a focal dead cell, while β was multiplied by the proportion of infectious cells within a thirty-six neighbor vicinity of a focal susceptible cell, thus allowing viral transmission to extend beyond the immediate nearest-neighbor boundaries of an infectious cell. To compensate for higher thresholds to cellular persistence and virus invasion which occur under local spatial conditions (<a href="#bib58">Webb et al., 2007</a>), we increased the birth rate, <i>b,</i> and the cell-to-cell transmission rate, β, respectively, to six and ten times the values used in the mean field model (<a href="#supp4">Supplementary file 4</a>). We derived these increases based on the assumption that births took place exclusively based on pairwise nearest-neighbor interactions (the six immediately adjacent cells to a focal dead cell), while viral transmission was locally concentrated but included a small (7.5%) global contribution, representing the thirty-six cell surrounding vicinity of a focal susceptible. We justify these increases and derive their origins further in <a href="#supp5">Supplementary file 5</a>.</p> <p>We simulated ten stochastic spatial time series for all cell-virus combinations under all three immune assumptions at a population size of 10,000 cells and compared model output with data in <a href="#fig5s3">Figure 5—figure supplement 3</a>. Spatial model code is available for public access at the following FigShare repository: DOI: <a href="http://doi.org/10.6084/m9.figshare.8312807">10.6084/m9.figshare.8312807</a>.</p> </div> </div> </div> </div> <div class="back" title="back"> <div class="funding-information" title="funding-information"> <div class="title" tagx="title" title="title">
Funding Information</div> <p>This paper was supported by the following grants:</p> <ul> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000001</span><span class="institution" title="institution">National Science Foundation</span></div> </div><span class="award-id" tagx="award-id" title="award-id">Graduate Research Fellowship</span> to Cara E Brook.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100007247</span><span class="institution" title="institution">Adolph C. and Mary Sprague Miller Institute for Basic Research in Science, University of California Berkeley</span></div> </div><span class="award-id" tagx="award-id" title="award-id">Postdoctoral Fellowship</span> to Cara E Brook.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000002</span><span class="institution" title="institution">National Institutes of Health</span></div> </div><span class="award-id" tagx="award-id" title="award-id">R01-AI134824</span> to Kartik Chandran.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100001381</span><span class="institution" title="institution">Singapore National Research Foundation</span></div> </div><span class="award-id" tagx="award-id" title="award-id">NRF2012NRF-CRP001-056</span> to Lin-Fa Wang.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100001381</span><span class="institution" title="institution">Singapore National Research Foundation</span></div> </div><span class="award-id" tagx="award-id" title="award-id">NRF2016NRF-NSFC002-013</span> to Lin-Fa Wang.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100001659</span><span class="institution" title="institution">Deutsche Forschungsgemeinschaft</span></div> </div><span class="award-id" tagx="award-id" title="award-id">DR 772/10-2</span> to Christian Drosten.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100002347</span><span class="institution" title="institution">Bundesministerium für Bildung und Forschung</span></div> </div><span class="award-id" tagx="award-id" title="award-id">RAPID #01KI1723A</span> to Christian Drosten.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100007601</span><span class="institution" title="institution">Horizon 2020</span></div> </div><span class="award-id" tagx="award-id" title="award-id">#653316</span> to Christian Drosten.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000185</span><span class="institution" title="institution">DARPA</span></div> </div><span class="award-id" tagx="award-id" title="award-id">PREEMPT Program - Cooperative Agreement no. D18AC00031</span> to Cara E Brook.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000002</span><span class="institution" title="institution">National Institutes of Health</span></div> </div><span class="award-id" tagx="award-id" title="award-id">1R01AI129822-01</span> to Cara E Brook.<p /> </li> </ul> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>CEB was supported by a National Science Foundation Graduate Research Fellowship at Princeton University, a Miller Institute for Basic Research Fellowship at UC Berkeley, a DARPA PREEMPT program Cooperative Agreement grant D18AC00031, and an NIH grant 1R01AI129822-01. Tissue culture experiments were funded by an NIH grant R01 AI134824 to KC. Work in LFW’s lab was funded by the Singapore National Research Foundation grants (NRF2012NRF-CRP001-056 and NRF2016NRF-NSFC002-013). CD was supported by the German Research Council (DFG) grant DFG SPP1596 (DR 772/10–2), the Federal Ministry of Education and Research (BMBF) grant RAPID (#01KI1723A) and the EU Horizon 2020 grant EVAg (#653316). The authors thank the Chandran lab at Albert Einstein College of Medicine – in particular, Cecilia Harold, Megan Slough, Rohit Jangra, and Tanwee Alkutkar – for technical support during tissue culture experiments. The authors further thank Jessica Metcalf and the Graham lab at Princeton for conceptual guidance throughout the project’s development.</p> </div> <div class="additional-information" title="additional-information"> <div class="title" tagx="title" title="title">
Additional information</div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title"><b>Competing interests</b></div> <div class="fn-type-COI-statement" title="COI-statement"> <p>No competing interests declared.</p> </div> </div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title"><b>Author contributions</b></div> <div class="fn-type-con" title="con"> <p>Conceptualization, Data curation, Formal analysis, Investigation, Visualization, Methodology.</p> </div> <div class="fn-type-con" title="con"> <p>Conceptualization, Supervision.</p> </div> <div class="fn-type-con" title="con"> <p>Resources, Supervision, Funding acquisition.</p> </div> <div class="fn-type-con" title="con"> <p>Conceptualization, Supervision.</p> </div> <div class="fn-type-con" title="con"> <p>Resources, Methodology.</p> </div> <div class="fn-type-con" title="con"> <p>Conceptualization, Supervision, Investigation.</p> </div> <div class="fn-type-con" title="con"> <p>Conceptualization, Supervision, Investigation.</p> </div> <div class="fn-type-con" title="con"> <p>Resources, Methodology.</p> </div> <div class="fn-type-con" title="con"> <p>Conceptualization, Data curation, Validation, Investigation, Methodology.</p> </div> <div class="fn-type-con" title="con"> <p>Resources, Methodology.</p> </div> <div class="fn-type-con" title="con"> <p>Conceptualization, Formal analysis, Supervision, Investigation, Visualization.</p> </div> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Additional files</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary file 1.</span> <div class="title" tagx="title" title="title">
(A) Raw proportion infectious from cell culture images.</div> <p>Dataset gives raw proportion of infectious cells and time elapsed for all trials of all cell line/virus/MOI combinations, derived from image processing of binary images. (B) Raw proportion uninfectious from cell culture images. Dataset gives raw proportion of uninfectious cells and time elapsed for all trials of all cell line/virus/MOI combinations, derived from image processing of binary Hoechst-stained images. (C) Statistical mean of infectious time series for all trials of each cell line/virus/MOI experiment, from GAM fitted model incorporating random effects by trial. Data were smoothed to yield the proportion infectious per hourly timestep for each trial, and mean field mechanistic models were fit to the smoothed mean of all compiled trials for each cell line/virus/MOI combination. (D) Statistical mean of uninfectious time series for all eighteen cell line/virus/MOI experiments, from generalized linear model fit to Hoechst stain data reported on tab B. Note that these means were not used in epidemic model fitting but natural mortality rates for each cell line were derived from fitting an infection-absent model to the trajectory of susceptible decline for control trials for each cell line, as shown in <a href="#fig1s7">Figure 1—figure supplement 7</a>. All original raw image files, processed binary images, and image processing code are available freely for download at the following FigShare repository: DOI: <a href="http://doi.org/10.6084/m9.figshare.8312807">10.6084/m9.figshare.8312807</a>.</p>  <div class="media" title="media"><a href="elife-48401-supp1.xlsx">LINK</a></div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary file 2.</span> <div class="title" tagx="title" title="title">
Derivation of R<sub>0</sub>.</div>  <div class="media" title="media"><a href="elife-48401-supp2.docx">LINK</a></div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary file 3.</span> <div class="title" tagx="title" title="title">
Special points from bifurcation analysis.</div>  <div class="media" title="media"><a href="elife-48401-supp3.docx">LINK</a></div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary file 4.</span> <div class="title" tagx="title" title="title">
Optimized parameters from all deterministic model outputs and spatial approximations.</div>  <div class="media" title="media"><a href="elife-48401-supp4.docx">LINK</a></div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary file 5.</span> <div class="title" tagx="title" title="title">
Justification for parameter increase from mean field to spatial model.</div>  <div class="media" title="media"><a href="elife-48401-supp5.docx">LINK</a></div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary file 6.</span> <div class="title" tagx="title" title="title">
Primers for qPCR.</div>  <div class="media" title="media"><a href="elife-48401-supp6.docx">LINK</a></div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary file 7.</span> <div class="title" tagx="title" title="title">
Detailed methods for image and image data processing.</div>  <div class="media" title="media"><a href="elife-48401-supp7.docx">LINK</a></div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Transparent reporting form</span><div class="media" title="media"><a href="elife-48401-transrepform.pdf">LINK</a></div> </div> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
Data availability</div> <p>All data generated or analysed during this study are included in the manuscript and supporting files. All images and code used in this study have been made available for download at the following Figshare repository: <a href="https://doi.org/10.6084/m9.figshare.8312807">https://doi.org/10.6084/m9.figshare.8312807</a>.</p> <p>The following dataset was generated:</p> <p><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brook</span><span class="given-names" tagx="given-names" title="given-names">CE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ng</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boots</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dobson</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graham</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grenfell</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chandran</span><span class="given-names" tagx="given-names" title="given-names">KC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Leeuwen</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2019</span>[data-title] Data and Code from: Accelerated viral dynamics in bat cell lines, with implications for zoonotic emergence<span class="source" tagx="source" title="source">figshare</span><span class="pub-id"><a href="https://dx.doi.org/10.6084/m9.figshare.8312807">10.6084/m9.figshare.8312807</a></span></span></p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="bib1" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahn</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anderson</span><span class="given-names" tagx="given-names" title="given-names">DE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">CW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lim</span><span class="given-names" tagx="given-names" title="given-names">BL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luko</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wen</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chia</span><span class="given-names" tagx="given-names" title="given-names">WN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mani</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">LC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ng</span><span class="given-names" tagx="given-names" title="given-names">JHJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sobota</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dutertre</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ginhoux</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">ZL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Irving</span><span class="given-names" tagx="given-names" title="given-names">AT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">LF</span></span></span><span class="year" tagx="year" title="year">2019</span><span class="mixed-article-title" title="mixed-article-title">Dampened NLRP3-mediated inflammation in bats and implications for a special viral reservoir host</span><span class="source" tagx="source" title="source">Nature Microbiology</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">789</span><span class="lpage" tagx="lpage" title="lpage">799</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41564-019-0371-3">10.1038/s41564-019-0371-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30804542">30804542</a></span></span></li> <li tag="ref"><a name="bib2" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arnold</span><span class="given-names" tagx="given-names" title="given-names">CE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guito</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Altamura</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lovett</span><span class="given-names" tagx="given-names" title="given-names">SP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagle</span><span class="given-names" tagx="given-names" title="given-names">ER</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palacios</span><span class="given-names" tagx="given-names" title="given-names">GF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanchez-Lockhart</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Towner</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Transcriptomics reveal antiviral gene induction in the egyptian rousette bat is antagonized in vitro by marburg virus infection</span><span class="source" tagx="source" title="source">Viruses</span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">E607</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/v10110607">10.3390/v10110607</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30400182">30400182</a></span></span></li> <li tag="ref"><a name="bib3" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baccam</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beauchemin</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Macken</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hayden</span><span class="given-names" tagx="given-names" title="given-names">FG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perelson</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Kinetics of influenza A virus infection in humans</span><span class="source" tagx="source" title="source">Journal of Virology</span><span class="volume" tagx="volume" title="volume">80</span><span class="fpage" tagx="fpage" title="fpage">7590</span><span class="lpage" tagx="lpage" title="lpage">7599</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.01623-05">10.1128/JVI.01623-05</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16840338">16840338</a></span></span></li> <li tag="ref"><a name="bib4" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Biesold</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ritz</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gloza-Rausch</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wollny</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drexler</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Corman</span><span class="given-names" tagx="given-names" title="given-names">VM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kalko</span><span class="given-names" tagx="given-names" title="given-names">EK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oppong</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drosten</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Müller</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Type I interferon reaction to viral infection in interferon-competent, immortalized cell lines from the african fruit bat Eidolon helvum</span><span class="source" tagx="source" title="source">PLOS ONE</span><span class="volume" tagx="volume" title="volume">6</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e28131</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0028131">10.1371/journal.pone.0028131</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22140523">22140523</a></span></span></li> <li tag="ref"><a name="bib5" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonhoeffer</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">May</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shaw</span><span class="given-names" tagx="given-names" title="given-names">GM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nowak</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span></span><span class="year" tagx="year" title="year">1997</span><span class="mixed-article-title" title="mixed-article-title">Virus dynamics and drug therapy</span><span class="source" tagx="source" title="source">PNAS</span><span class="volume" tagx="volume" title="volume">94</span><span class="fpage" tagx="fpage" title="fpage">6971</span><span class="lpage" tagx="lpage" title="lpage">6976</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.94.13.6971">10.1073/pnas.94.13.6971</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9192676">9192676</a></span></span></li> <li tag="ref"><a name="bib6" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brook</span><span class="given-names" tagx="given-names" title="given-names">CE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ranaivoson</span><span class="given-names" tagx="given-names" title="given-names">HC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Broder</span><span class="given-names" tagx="given-names" title="given-names">CC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cunningham</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Héraud</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peel</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibson</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wood</span><span class="given-names" tagx="given-names" title="given-names">JLN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Metcalf</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dobson</span><span class="given-names" tagx="given-names" title="given-names">AP</span></span></span><span class="year" tagx="year" title="year">2019</span><span class="mixed-article-title" title="mixed-article-title">Disentangling serology to elucidate henipa- and Filovirus transmission in Madagascar fruit bats</span><span class="source" tagx="source" title="source">Journal of Animal Ecology</span><span class="volume" tagx="volume" title="volume">88</span><span class="fpage" tagx="fpage" title="fpage">1001</span><span class="lpage" tagx="lpage" title="lpage">1016</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/1365-2656.12985">10.1111/1365-2656.12985</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30908623">30908623</a></span></span></li> <li tag="ref"><a name="bib7" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brook</span><span class="given-names" tagx="given-names" title="given-names">CE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dobson</span><span class="given-names" tagx="given-names" title="given-names">AP</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Bats as 'special' reservoirs for emerging zoonotic pathogens</span><span class="source" tagx="source" title="source">Trends in Microbiology</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">172</span><span class="lpage" tagx="lpage" title="lpage">180</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tim.2014.12.004">10.1016/j.tim.2014.12.004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25572882">25572882</a></span></span></li> <li tag="ref"><a name="bib8" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Calisher</span><span class="given-names" tagx="given-names" title="given-names">CH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Childs</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Field</span><span class="given-names" tagx="given-names" title="given-names">HE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">KV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schountz</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Bats: important reservoir hosts of emerging viruses</span><span class="source" tagx="source" title="source">Clinical Microbiology Reviews</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">531</span><span class="lpage" tagx="lpage" title="lpage">545</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/CMR.00017-06">10.1128/CMR.00017-06</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16847084">16847084</a></span></span></li> <li tag="ref"><a name="bib9" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coffin</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span></span><span class="year" tagx="year" title="year">1995</span><span class="mixed-article-title" title="mixed-article-title">HIV population dynamics in vivo: implications for genetic variation, pathogenesis, and therapy</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">267</span><span class="fpage" tagx="fpage" title="fpage">483</span><span class="lpage" tagx="lpage" title="lpage">489</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.7824947">10.1126/science.7824947</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7824947">7824947</a></span></span></li> <li tag="ref"><a name="bib10" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crameri</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Todd</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grimley</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McEachern</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marsh</span><span class="given-names" tagx="given-names" title="given-names">GA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tachedjian</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Jong</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Virtue</span><span class="given-names" tagx="given-names" title="given-names">ER</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bulach</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Michalski</span><span class="given-names" tagx="given-names" title="given-names">WP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Middleton</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Field</span><span class="given-names" tagx="given-names" title="given-names">HE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">LF</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Establishment, immortalisation and characterisation of pteropid bat cell lines</span><span class="source" tagx="source" title="source">PLOS ONE</span><span class="volume" tagx="volume" title="volume">4</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e8266</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0008266">10.1371/journal.pone.0008266</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20011515">20011515</a></span></span></li> <li tag="ref"><a name="bib11" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dembowski</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DeLuca</span><span class="given-names" tagx="given-names" title="given-names">NA</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Selective recruitment of nuclear factors to productively replicating herpes simplex virus genomes</span><span class="source" tagx="source" title="source">PLOS Pathogens</span><span class="volume" tagx="volume" title="volume">11</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1004939</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1004939">10.1371/journal.ppat.1004939</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26018390">26018390</a></span></span></li> <li tag="ref"><a name="bib12" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Desmyter</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Melnick</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rawls</span><span class="given-names" tagx="given-names" title="given-names">WE</span></span></span><span class="year" tagx="year" title="year">1968</span><span class="mixed-article-title" title="mixed-article-title">Defectiveness of interferon production and of rubella virus interference in a line of african green monkey kidney cells (Vero)</span><span class="source" tagx="source" title="source">Journal of Virology</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">955</span><span class="lpage" tagx="lpage" title="lpage">961</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.2.10.955-961.1968">10.1128/JVI.2.10.955-961.1968</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4302013">4302013</a></span></span></li> <li tag="ref"><a name="bib13" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dhooge</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Govaerts</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuznetsov</span><span class="given-names" tagx="given-names" title="given-names">YA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meijer</span><span class="given-names" tagx="given-names" title="given-names">HGE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sautois</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">New features of the software MatCont for bifurcation analysis of dynamical systems</span><span class="source" tagx="source" title="source">Mathematical and Computer Modelling of Dynamical Systems</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">147</span><span class="lpage" tagx="lpage" title="lpage">175</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/13873950701742754">10.1080/13873950701742754</a></span></span></li> <li tag="ref"><a name="bib14" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diekmann</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heesterbeek</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Metz</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span></span><span class="year" tagx="year" title="year">1990</span><span class="mixed-article-title" title="mixed-article-title">On the definition and the computation of the basic reproduction ratio R0 in models for infectious diseases in heterogeneous populations</span><span class="source" tagx="source" title="source">Journal of Mathematical Biology</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">365</span><span class="lpage" tagx="lpage" title="lpage">382</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/BF00178324">10.1007/BF00178324</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2117040">2117040</a></span></span></li> <li tag="ref"><a name="bib15" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eddelbuettel</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Francois</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Rcpp: seamless R and C++ integration</span><span class="source" tagx="source" title="source">Journal of Statistical Software</span><span class="volume" tagx="volume" title="volume">40</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">18</span><span class="pub-id"><a href="https://dx.doi.org/10.18637/jss.v040.i08">10.18637/jss.v040.i08</a></span></span></li> <li tag="ref"><a name="bib16" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eddelbuettel</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanderson</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">RcppArmadillo: accelerating R with High-Performance C++ linear algebra</span><span class="source" tagx="source" title="source">Computational Statistics and Data Analysis</span><span class="volume" tagx="volume" title="volume">71</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">16</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.csda.2013.02.005">10.1016/j.csda.2013.02.005</a></span></span></li> <li tag="ref"><a name="bib17" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Emeny</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morgan</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span></span><span class="year" tagx="year" title="year">1979</span><span class="mixed-article-title" title="mixed-article-title">Regulation of the interferon system: evidence that vero cells have a genetic defect in interferon production</span><span class="source" tagx="source" title="source">Journal of General Virology</span><span class="volume" tagx="volume" title="volume">43</span><span class="fpage" tagx="fpage" title="fpage">247</span><span class="lpage" tagx="lpage" title="lpage">252</span><span class="pub-id"><a href="https://dx.doi.org/10.1099/0022-1317-43-1-247">10.1099/0022-1317-43-1-247</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/113494">113494</a></span></span></li> <li tag="ref"><a name="bib18" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Emery</span><span class="given-names" tagx="given-names" title="given-names">VC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cope</span><span class="given-names" tagx="given-names" title="given-names">AV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bowen</span><span class="given-names" tagx="given-names" title="given-names">EF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gor</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Griffiths</span><span class="given-names" tagx="given-names" title="given-names">PD</span></span></span><span class="year" tagx="year" title="year">1999</span><span class="mixed-article-title" title="mixed-article-title">The dynamics of human Cytomegalovirus replication in vivo</span><span class="source" tagx="source" title="source">The Journal of Experimental Medicine</span><span class="volume" tagx="volume" title="volume">190</span><span class="fpage" tagx="fpage" title="fpage">177</span><span class="lpage" tagx="lpage" title="lpage">182</span><span class="pub-id"><a href="https://dx.doi.org/10.1084/jem.190.2.177">10.1084/jem.190.2.177</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10432281">10432281</a></span></span></li> <li tag="ref"><a name="bib19" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haydon</span><span class="given-names" tagx="given-names" title="given-names">DT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cleaveland</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taylor</span><span class="given-names" tagx="given-names" title="given-names">LH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Laurenson</span><span class="given-names" tagx="given-names" title="given-names">MK</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">"Identifying Reservoirs of Infection: A Conceptual and Practical Challenge.�?</span><span class="source" tagx="source" title="source">Emerging Infectious Diseases</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">1468</span><span class="lpage" tagx="lpage" title="lpage">1473</span><span class="pub-id"><a href="https://dx.doi.org/10.3201/eid0812.010317">10.3201/eid0812.010317</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12498665">12498665</a></span></span></li> <li tag="ref"><a name="bib20" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heffernan</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wahl</span><span class="given-names" tagx="given-names" title="given-names">LM</span></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">Perspectives on the basic reproductive ratio</span><span class="source" tagx="source" title="source">Journal of the Royal Society Interface</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">281</span><span class="lpage" tagx="lpage" title="lpage">293</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rsif.2005.0042">10.1098/rsif.2005.0042</a></span></span></li> <li tag="ref"><a name="bib21" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ho</span><span class="given-names" tagx="given-names" title="given-names">DD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neumann</span><span class="given-names" tagx="given-names" title="given-names">AU</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perelson</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leonard</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Markowitz</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">1995</span><span class="mixed-article-title" title="mixed-article-title">Rapid turnover of plasma virions and CD4 lymphocytes in HIV-1 infection</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">373</span><span class="fpage" tagx="fpage" title="fpage">123</span><span class="lpage" tagx="lpage" title="lpage">126</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/373123a0">10.1038/373123a0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7816094">7816094</a></span></span></li> <li tag="ref"><a name="bib22" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hooper</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zaki</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daniels</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Middleton</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">Comparative pathology of the diseases caused by Hendra and nipah viruses</span><span class="source" tagx="source" title="source">Microbes and Infection</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">315</span><span class="lpage" tagx="lpage" title="lpage">322</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S1286-4579(01)01385-5">10.1016/S1286-4579(01)01385-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11334749">11334749</a></span></span></li> <li tag="ref"><a name="bib23" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Howat</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barreca</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Hare</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gog</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grenfell</span><span class="given-names" tagx="given-names" title="given-names">BT</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Modelling dynamics of the type I interferon response to <i>in vitro</i> viral infection</span><span class="source" tagx="source" title="source">Journal of the Royal Society Interface</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">699</span><span class="lpage" tagx="lpage" title="lpage">709</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rsif.2006.0136">10.1098/rsif.2006.0136</a></span></span></li> <li tag="ref"><a name="bib24" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kacprzyk</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hughes</span><span class="given-names" tagx="given-names" title="given-names">GM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palsson-McDermott</span><span class="given-names" tagx="given-names" title="given-names">EM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quinn</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Puechmaille</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Neill</span><span class="given-names" tagx="given-names" title="given-names">LAJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teeling</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">A potent Anti-Inflammatory response in bat macrophages may be linked to extended longevity and viral tolerance</span><span class="source" tagx="source" title="source">Acta Chiropterologica</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">219</span><span class="lpage" tagx="lpage" title="lpage">228</span><span class="pub-id"><a href="https://dx.doi.org/10.3161/15081109ACC2017.19.2.001">10.3161/15081109ACC2017.19.2.001</a></span></span></li> <li tag="ref"><a name="bib25" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keeling</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rohani</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="source" tagx="source" title="source">Modeling Infectious Diseases in Humans and Animals</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Princeton University Press</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Princeton, NJ</span></span></li> <li tag="ref"><a name="bib26" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kühl</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoffmann</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Müller</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Munster</span><span class="given-names" tagx="given-names" title="given-names">VJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gnirss</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kiene</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsegaye</span><span class="given-names" tagx="given-names" title="given-names">TS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Behrens</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herrler</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feldmann</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drosten</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pöhlmann</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Comparative analysis of ebola virus glycoprotein interactions with human and bat cells</span><span class="source" tagx="source" title="source">The Journal of Infectious Diseases</span><span class="volume" tagx="volume" title="volume">204</span><span class="fpage" tagx="fpage" title="fpage">S840</span><span class="lpage" tagx="lpage" title="lpage">S849</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/infdis/jir306">10.1093/infdis/jir306</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21987760">21987760</a></span></span></li> <li tag="ref"><a name="bib27" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuzmin</span><span class="given-names" tagx="given-names" title="given-names">IV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schwarz</span><span class="given-names" tagx="given-names" title="given-names">TM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ilinykh</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jordan</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ksiazek</span><span class="given-names" tagx="given-names" title="given-names">TG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sachidanandam</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Basler</span><span class="given-names" tagx="given-names" title="given-names">CF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bukreyev</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Innate immune responses of bat and human cells to filoviruses: commonalities and distinctions</span><span class="source" tagx="source" title="source">Journal of Virology</span><span class="volume" tagx="volume" title="volume">91</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e02471</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.02471-16">10.1128/JVI.02471-16</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28122983">28122983</a></span></span></li> <li tag="ref"><a name="bib28" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Livak</span><span class="given-names" tagx="given-names" title="given-names">KJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmittgen</span><span class="given-names" tagx="given-names" title="given-names">TD</span></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta delta C(T)) Method</span><span class="source" tagx="source" title="source">Methods</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">402</span><span class="lpage" tagx="lpage" title="lpage">408</span><span class="pub-id"><a href="https://dx.doi.org/10.1006/meth.2001.1262">10.1006/meth.2001.1262</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11846609">11846609</a></span></span></li> <li tag="ref"><a name="bib29" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mahanty</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bray</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Pathogenesis of filoviral haemorrhagic fevers</span><span class="source" tagx="source" title="source">The Lancet Infectious Diseases</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">487</span><span class="lpage" tagx="lpage" title="lpage">498</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S1473-3099(04)01103-X">10.1016/S1473-3099(04)01103-X</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15288821">15288821</a></span></span></li> <li tag="ref"><a name="bib30" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">EH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Obernosterer</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raaben</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herbert</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deffieu</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krishnan</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ndungo</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sandesara</span><span class="given-names" tagx="given-names" title="given-names">RG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carette</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuehne</span><span class="given-names" tagx="given-names" title="given-names">AI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruthel</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pfeffer</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dye</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Whelan</span><span class="given-names" tagx="given-names" title="given-names">SP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brummelkamp</span><span class="given-names" tagx="given-names" title="given-names">TR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chandran</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Ebola virus entry requires the host-programmed recognition of an intracellular receptor</span><span class="source" tagx="source" title="source">The EMBO Journal</span><span class="volume" tagx="volume" title="volume">31</span><span class="fpage" tagx="fpage" title="fpage">1947</span><span class="lpage" tagx="lpage" title="lpage">1960</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/emboj.2012.53">10.1038/emboj.2012.53</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22395071">22395071</a></span></span></li> <li tag="ref"><a name="bib31" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morris</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yates</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Swart</span><span class="given-names" tagx="given-names" title="given-names">RL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Vries</span><span class="given-names" tagx="given-names" title="given-names">RD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mina</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nelson</span><span class="given-names" tagx="given-names" title="given-names">AN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">WW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kouyos</span><span class="given-names" tagx="given-names" title="given-names">RD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Griffin</span><span class="given-names" tagx="given-names" title="given-names">DE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grenfell</span><span class="given-names" tagx="given-names" title="given-names">BT</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Modeling the measles paradox reveals the importance of cellular immunity in regulating viral clearance</span><span class="source" tagx="source" title="source">PLOS Pathogens</span><span class="volume" tagx="volume" title="volume">14</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1007493</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1007493">10.1371/journal.ppat.1007493</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30592772">30592772</a></span></span></li> <li tag="ref"><a name="bib32" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagai</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Honda</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">A dynamic cell model for the formation of epithelial tissues</span><span class="source" tagx="source" title="source">Philosophical Magazine B</span><span class="volume" tagx="volume" title="volume">81</span><span class="fpage" tagx="fpage" title="fpage">699</span><span class="lpage" tagx="lpage" title="lpage">719</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/13642810108205772">10.1080/13642810108205772</a></span></span></li> <li tag="ref"><a name="bib33" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neumann</span><span class="given-names" tagx="given-names" title="given-names">AU</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lam</span><span class="given-names" tagx="given-names" title="given-names">NP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dahari</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gretch</span><span class="given-names" tagx="given-names" title="given-names">DR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wiley</span><span class="given-names" tagx="given-names" title="given-names">TE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Layden</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perelson</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span></span><span class="year" tagx="year" title="year">1998</span><span class="mixed-article-title" title="mixed-article-title">Hepatitis C viral dynamics in vivo and the antiviral efficacy of interferon-alpha therapy</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">282</span><span class="fpage" tagx="fpage" title="fpage">103</span><span class="lpage" tagx="lpage" title="lpage">107</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.282.5386.103">10.1126/science.282.5386.103</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9756471">9756471</a></span></span></li> <li tag="ref"><a name="bib34" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ng</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ndungo</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaczmarek</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herbert</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Binger</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuehne</span><span class="given-names" tagx="given-names" title="given-names">AI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jangra</span><span class="given-names" tagx="given-names" title="given-names">RK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hawkins</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gifford</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Biswas</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Demogines</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">James</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brummelkamp</span><span class="given-names" tagx="given-names" title="given-names">TR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drosten</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">LF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuhn</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Müller</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dye</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sawyer</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chandran</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Filovirus receptor NPC1 contributes to species-specific patterns of ebolavirus susceptibility in bats</span><span class="source" tagx="source" title="source">eLife</span><span class="volume" tagx="volume" title="volume">4</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e11785</span><span class="pub-id"><a href="https://dx.doi.org/10.7554/eLife.11785">10.7554/eLife.11785</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26698106">26698106</a></span></span></li> <li tag="ref"><a name="bib35" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nicholls</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poon</span><span class="given-names" tagx="given-names" title="given-names">LLM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">KC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ng</span><span class="given-names" tagx="given-names" title="given-names">WF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lai</span><span class="given-names" tagx="given-names" title="given-names">ST</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leung</span><span class="given-names" tagx="given-names" title="given-names">CY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chu</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hui</span><span class="given-names" tagx="given-names" title="given-names">PK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mak</span><span class="given-names" tagx="given-names" title="given-names">KL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lim</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">KW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">KH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsang</span><span class="given-names" tagx="given-names" title="given-names">NC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guan</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuen</span><span class="given-names" tagx="given-names" title="given-names">KY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malik Peiris</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">Lung pathology of fatal severe acute respiratory syndrome</span><span class="source" tagx="source" title="source">The Lancet</span><span class="volume" tagx="volume" title="volume">361</span><span class="fpage" tagx="fpage" title="fpage">1773</span><span class="lpage" tagx="lpage" title="lpage">1778</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0140-6736(03)13413-7">10.1016/S0140-6736(03)13413-7</a></span></span></li> <li tag="ref"><a name="bib36" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nowak</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">May</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Phillips</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rowland-Jones</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lalloo</span><span class="given-names" tagx="given-names" title="given-names">DG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McAdam</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klenerman</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Köppe</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sigmund</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bangham</span><span class="given-names" tagx="given-names" title="given-names">CR</span></span></span><span class="year" tagx="year" title="year">1995</span><span class="mixed-article-title" title="mixed-article-title">Antigenic oscillations and shifting immunodominance in HIV-1 infections</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">375</span><span class="fpage" tagx="fpage" title="fpage">606</span><span class="lpage" tagx="lpage" title="lpage">611</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/375606a0">10.1038/375606a0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7791879">7791879</a></span></span></li> <li tag="ref"><a name="bib37" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nowak</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonhoeffer</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hill</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boehme</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thomas</span><span class="given-names" tagx="given-names" title="given-names">HC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McDade</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="year" tagx="year" title="year">1996</span><span class="mixed-article-title" title="mixed-article-title">Viral dynamics in hepatitis B virus infection</span><span class="source" tagx="source" title="source">PNAS</span><span class="volume" tagx="volume" title="volume">93</span><span class="fpage" tagx="fpage" title="fpage">4398</span><span class="lpage" tagx="lpage" title="lpage">4402</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.93.9.4398">10.1073/pnas.93.9.4398</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8633078">8633078</a></span></span></li> <li tag="ref"><a name="bib38" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nowak</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">May</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span></span><span class="year" tagx="year" title="year">2000</span><span class="source" tagx="source" title="source">Virus Dynamics: Mathematical Principles of Immunology and Virology</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Oxford</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford University Press</span></span></li> <li tag="ref"><a name="bib39" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pau</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fuchs</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sklyar</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boutros</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huber</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">EBImage--an R package for image processing with applications to cellular phenotypes</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">26</span><span class="fpage" tagx="fpage" title="fpage">979</span><span class="lpage" tagx="lpage" title="lpage">981</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btq046">10.1093/bioinformatics/btq046</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20338898">20338898</a></span></span></li> <li tag="ref"><a name="bib40" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pavlovich</span><span class="given-names" tagx="given-names" title="given-names">SS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lovett</span><span class="given-names" tagx="given-names" title="given-names">SP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koroleva</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guito</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arnold</span><span class="given-names" tagx="given-names" title="given-names">CE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagle</span><span class="given-names" tagx="given-names" title="given-names">ER</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kulcsar</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thibaud-Nissen</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hume</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mühlberger</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uebelhoer</span><span class="given-names" tagx="given-names" title="given-names">LS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Towner</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rabadan</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanchez-Lockhart</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kepler</span><span class="given-names" tagx="given-names" title="given-names">TB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palacios</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">The egyptian rousette genome reveals unexpected features of bat antiviral immunity</span><span class="source" tagx="source" title="source">Cell</span><span class="volume" tagx="volume" title="volume">173</span><span class="fpage" tagx="fpage" title="fpage">1098</span><span class="lpage" tagx="lpage" title="lpage">1110</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cell.2018.03.070">10.1016/j.cell.2018.03.070</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29706541">29706541</a></span></span></li> <li tag="ref"><a name="bib41" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pawelek</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huynh</span><span class="given-names" tagx="given-names" title="given-names">GT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quinlivan</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cullinane</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rong</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perelson</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Modeling within-host dynamics of influenza virus infection including immune responses</span><span class="source" tagx="source" title="source">PLOS Computational Biology</span><span class="volume" tagx="volume" title="volume">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1002588</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pcbi.1002588">10.1371/journal.pcbi.1002588</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22761567">22761567</a></span></span></li> <li tag="ref"><a name="bib42" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peel</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baker</span><span class="given-names" tagx="given-names" title="given-names">KS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crameri</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barr</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hayman</span><span class="given-names" tagx="given-names" title="given-names">DT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wright</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Broder</span><span class="given-names" tagx="given-names" title="given-names">CC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fernández-Loras</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fooks</span><span class="given-names" tagx="given-names" title="given-names">AR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">LF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cunningham</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wood</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Henipavirus neutralising antibodies in an isolated island population of african fruit bats</span><span class="source" tagx="source" title="source">PLOS ONE</span><span class="volume" tagx="volume" title="volume">7</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e30346</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0030346">10.1371/journal.pone.0030346</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22253928">22253928</a></span></span></li> <li tag="ref"><a name="bib43" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peel</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baker</span><span class="given-names" tagx="given-names" title="given-names">KS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hayman</span><span class="given-names" tagx="given-names" title="given-names">DTS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Broder</span><span class="given-names" tagx="given-names" title="given-names">CC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cunningham</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fooks</span><span class="given-names" tagx="given-names" title="given-names">AR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garnier</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wood</span><span class="given-names" tagx="given-names" title="given-names">JLN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Restif</span><span class="given-names" tagx="given-names" title="given-names">O</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Support for viral persistence in bats from age-specific serology and models of maternal immunity</span><span class="source" tagx="source" title="source">Scientific Reports</span><span class="volume" tagx="volume" title="volume">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">3859</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-018-22236-6">10.1038/s41598-018-22236-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29497106">29497106</a></span></span></li> <li tag="ref"><a name="bib44" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perelson</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neumann</span><span class="given-names" tagx="given-names" title="given-names">AU</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Markowitz</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leonard</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ho</span><span class="given-names" tagx="given-names" title="given-names">DD</span></span></span><span class="year" tagx="year" title="year">1996</span><span class="mixed-article-title" title="mixed-article-title">HIV-1 dynamics in vivo: virion clearance rate, infected cell life-span, and viral generation time</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">271</span><span class="fpage" tagx="fpage" title="fpage">1582</span><span class="lpage" tagx="lpage" title="lpage">1586</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.271.5255.1582">10.1126/science.271.5255.1582</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8599114">8599114</a></span></span></li> <li tag="ref"><a name="bib45" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perelson</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">Modelling viral and immune system dynamics</span><span class="source" tagx="source" title="source">Nature Reviews Immunology</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">28</span><span class="lpage" tagx="lpage" title="lpage">36</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nri700">10.1038/nri700</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11905835">11905835</a></span></span></li> <li tag="ref"><a name="bib46" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plowright</span><span class="given-names" tagx="given-names" title="given-names">RK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peel</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Streicker</span><span class="given-names" tagx="given-names" title="given-names">DG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gilbert</span><span class="given-names" tagx="given-names" title="given-names">AT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCallum</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wood</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baker</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Restif</span><span class="given-names" tagx="given-names" title="given-names">O</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Transmission or Within-Host dynamics driving pulses of zoonotic viruses in Reservoir-Host populations</span><span class="source" tagx="source" title="source">PLOS Neglected Tropical Diseases</span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e0004796</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pntd.0004796">10.1371/journal.pntd.0004796</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27489944">27489944</a></span></span></li> <li tag="ref"><a name="bib47" /><span class="element-citation'"><span class="person-group'"><span class="collab'">collab: R Development Core Team</span></span><span class="year" tagx="year" title="year">2019</span>[data-title] R: A Language and Environment for Statistical Computing<span class="source" tagx="source" title="source">R Foundation for Statistical Computing</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Vienna, Austria</span><a href="http://www.r-project.org">http://www.r-project.org</a></span></li> <li tag="ref"><a name="bib48" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Radke</span><span class="given-names" tagx="given-names" title="given-names">KL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colby</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kates</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krider</span><span class="given-names" tagx="given-names" title="given-names">HM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prescott</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span></span><span class="year" tagx="year" title="year">1974</span><span class="mixed-article-title" title="mixed-article-title">Establishment and maintenance of the interferon-induced antiviral state: studies in enucleated cells</span><span class="source" tagx="source" title="source">Journal of Virology</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">623</span><span class="lpage" tagx="lpage" title="lpage">630</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.13.3.623-630.1974">10.1128/JVI.13.3.623-630.1974</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4362865">4362865</a></span></span></li> <li tag="ref"><a name="bib49" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rasmussen</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farley</span><span class="given-names" tagx="given-names" title="given-names">LB</span></span></span><span class="year" tagx="year" title="year">1975</span><span class="mixed-article-title" title="mixed-article-title">Inhibition of <i>Herpesvirus</i> hominis replication by human interferon</span><span class="source" tagx="source" title="source">Infection and Immunity</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">104</span><span class="lpage" tagx="lpage" title="lpage">108</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/IAI.12.1.104-108.1975">10.1128/IAI.12.1.104-108.1975</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/166922">166922</a></span></span></li> <li tag="ref"><a name="bib50" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rhim</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schell</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Creasy</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Case</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="year" tagx="year" title="year">1969</span><span class="mixed-article-title" title="mixed-article-title">Biological characteristics and viral susceptibility of an african green monkey kidney cell line (Vero)</span><span class="source" tagx="source" title="source">Experimental Biology and Medicine</span><span class="volume" tagx="volume" title="volume">132</span><span class="fpage" tagx="fpage" title="fpage">670</span><span class="lpage" tagx="lpage" title="lpage">678</span><span class="pub-id"><a href="https://dx.doi.org/10.3181/00379727-132-34285">10.3181/00379727-132-34285</a></span></span></li> <li tag="ref"><a name="bib51" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saenz</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quinlivan</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elton</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Macrae</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blunden</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mumford</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daly</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Digard</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cullinane</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grenfell</span><span class="given-names" tagx="given-names" title="given-names">BT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCauley</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wood</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gog</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Dynamics of influenza virus infection and pathology</span><span class="source" tagx="source" title="source">Journal of Virology</span><span class="volume" tagx="volume" title="volume">84</span><span class="fpage" tagx="fpage" title="fpage">3974</span><span class="lpage" tagx="lpage" title="lpage">3983</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.02078-09">10.1128/JVI.02078-09</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20130053">20130053</a></span></span></li> <li tag="ref"><a name="bib52" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Samuel</span><span class="given-names" tagx="given-names" title="given-names">CE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knutson</span><span class="given-names" tagx="given-names" title="given-names">GS</span></span></span><span class="year" tagx="year" title="year">1982</span><span class="mixed-article-title" title="mixed-article-title">Mechanism of interferon action</span><span class="source" tagx="source" title="source">Journal of Biological Chemistry</span><span class="volume" tagx="volume" title="volume">257</span><span class="fpage" tagx="fpage" title="fpage">11791</span><span class="lpage" tagx="lpage" title="lpage">11795</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6181059">6181059</a></span></span></li> <li tag="ref"><a name="bib53" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schountz</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baker</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Butler</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Munster</span><span class="given-names" tagx="given-names" title="given-names">V</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Immunological control of viral infections in bats and the emergence of viruses highly pathogenic to humans</span><span class="source" tagx="source" title="source">Frontiers in Immunology</span><span class="volume" tagx="volume" title="volume">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">1098</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fimmu.2017.01098">10.3389/fimmu.2017.01098</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28959255">28959255</a></span></span></li> <li tag="ref"><a name="bib54" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Soetaert</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petzoldt</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woodrow Setzer</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Package DeSolve: solving initial value differential equations in R</span><span class="source" tagx="source" title="source">Journal of Statistical Software</span><span class="volume" tagx="volume" title="volume">33</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">25</span><span class="pub-id"><a href="https://dx.doi.org/10.18637/jss.v033.i09">10.18637/jss.v033.i09</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20808728">20808728</a></span></span></li> <li tag="ref"><a name="bib55" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stetson</span><span class="given-names" tagx="given-names" title="given-names">DB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Medzhitov</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Type I interferons in host defense</span><span class="source" tagx="source" title="source">Immunity</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">373</span><span class="lpage" tagx="lpage" title="lpage">381</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.immuni.2006.08.007">10.1016/j.immuni.2006.08.007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16979569">16979569</a></span></span></li> <li tag="ref"><a name="bib56" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takadate</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kondoh</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Igarashi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maruyama</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Manzoor</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ogawa</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kajihara</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Furuyama</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sato</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miyamoto</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yoshida</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hill</span><span class="given-names" tagx="given-names" title="given-names">TE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Freiberg</span><span class="given-names" tagx="given-names" title="given-names">AN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feldmann</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marzi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takada</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2020</span><span class="mixed-article-title" title="mixed-article-title">Niemann-Pick C1 Heterogeneity of Bat Cells Controls Filovirus Tropism</span><span class="source" tagx="source" title="source">Cell Reports</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">308</span><span class="lpage" tagx="lpage" title="lpage">319</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.celrep.2019.12.042">10.1016/j.celrep.2019.12.042</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31940478">31940478</a></span></span></li> <li tag="ref"><a name="bib57" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">LF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anderson</span><span class="given-names" tagx="given-names" title="given-names">DE</span></span></span><span class="year" tagx="year" title="year">2019</span><span class="mixed-article-title" title="mixed-article-title">Viruses in bats and potential spillover to animals and humans</span><span class="source" tagx="source" title="source">Current Opinion in Virology</span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">79</span><span class="lpage" tagx="lpage" title="lpage">89</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.coviro.2018.12.007">10.1016/j.coviro.2018.12.007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30665189">30665189</a></span></span></li> <li tag="ref"><a name="bib58" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webb</span><span class="given-names" tagx="given-names" title="given-names">SD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keeling</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boots</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Host-parasite interactions between the local and the mean-field: how and when does spatial population structure matter?</span><span class="source" tagx="source" title="source">Journal of Theoretical Biology</span><span class="volume" tagx="volume" title="volume">249</span><span class="fpage" tagx="fpage" title="fpage">140</span><span class="lpage" tagx="lpage" title="lpage">152</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jtbi.2007.06.013">10.1016/j.jtbi.2007.06.013</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17719608">17719608</a></span></span></li> <li tag="ref"><a name="bib59" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sandesara</span><span class="given-names" tagx="given-names" title="given-names">RG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mulherkar</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Whelan</span><span class="given-names" tagx="given-names" title="given-names">SP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chandran</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">A forward genetic strategy reveals destabilizing mutations in the ebolavirus glycoprotein that alter its protease dependence during cell entry</span><span class="source" tagx="source" title="source">Journal of Virology</span><span class="volume" tagx="volume" title="volume">84</span><span class="fpage" tagx="fpage" title="fpage">163</span><span class="lpage" tagx="lpage" title="lpage">175</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.01832-09">10.1128/JVI.01832-09</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19846533">19846533</a></span></span></li> <li tag="ref"><a name="bib60" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goh</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cui</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">L-F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">Z-L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Dampened STING-Dependent interferon activation in bats</span><span class="source" tagx="source" title="source">Cell Host &amp;amp; Microbe</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">297</span><span class="lpage" tagx="lpage" title="lpage">301</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.chom.2018.01.006">10.1016/j.chom.2018.01.006</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29478775">29478775</a></span></span></li> <li tag="ref"><a name="bib61" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cowled</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bishop-Lilly</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wynne</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiong</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baker</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tachedjian</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ng</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiao</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marsh</span><span class="given-names" tagx="given-names" title="given-names">GA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crameri</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Broder</span><span class="given-names" tagx="given-names" title="given-names">CC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frey</span><span class="given-names" tagx="given-names" title="given-names">KG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">LF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Comparative analysis of bat genomes provides insight into the evolution of flight and immunity</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">339</span><span class="fpage" tagx="fpage" title="fpage">456</span><span class="lpage" tagx="lpage" title="lpage">460</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1230835">10.1126/science.1230835</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23258410">23258410</a></span></span></li> <li tag="ref"><a name="bib62" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tachedjian</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wynne</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boyd</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cui</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cowled</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ng</span><span class="given-names" tagx="given-names" title="given-names">JHJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mok</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Michalski</span><span class="given-names" tagx="given-names" title="given-names">WP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mendenhall</span><span class="given-names" tagx="given-names" title="given-names">IH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tachedjian</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">L-F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baker</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Contraction of the type I IFN locus and unusual constitutive expression of <i>IFN-α</i> in bats</span><span class="source" tagx="source" title="source">PNAS</span><span class="volume" tagx="volume" title="volume">113</span><span class="fpage" tagx="fpage" title="fpage">2696</span><span class="lpage" tagx="lpage" title="lpage">2701</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1518240113">10.1073/pnas.1518240113</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26903655">26903655</a></span></span></li> </ul> </div> </div>  </body></html>