<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">J Biomol Struct Dyn</span><span class="iso-abbrev" title="iso-abbrev">J. Biomol. Struct. Dyn</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Journal of Biomolecular Structure &amp;amp; Dynamics</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">0739-1102</span><span class="issn-epub" tagx="issn" title="issn-epub">1538-0254</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Taylor &amp;amp; Francis</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7441777">7441777</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1080/07391102.2020.1796810">10.1080/07391102.2020.1796810</a></span><span class="publisher-id" title="publisher-id">1796810</span><div class="unknown" title="article-version" tag="article-version">
Version of Record</div> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span>: <span class="subject" title="subject">Research Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Plant-derived natural polyphenols as potential antiviral drugs against SARS-CoV-2 <i>via</i> RNA‐dependent RNA polymerase (RdRp) inhibition: an <i>in-silico</i> analysis</div> <div class="alt-title" title="alt-title">
S. Singh et al.</div> <div class="alt-title" title="alt-title">
Journal of Biomolecular Structure and Dynamics</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0003-2341-9544</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">Satyam</span></span><a href="#AF0001">a</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0003-4341-8370</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sk</span><span class="given-names" tagx="given-names" title="given-names">Md Fulbabu</span></span><a href="#AF0001">a</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sonawane</span><span class="given-names" tagx="given-names" title="given-names">Avinash</span></span><a href="#AF0001">a</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-8451-9739</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kar</span><span class="given-names" tagx="given-names" title="given-names">Parimal</span></span><a href="#AF0001">a</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-3302-5875</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sadhukhan</span><span class="given-names" tagx="given-names" title="given-names">Sushabhan</span></span><a href="#AF0002">b</a><a href="#AN0001" /></span><span class="citation_author_institution" id="AF0001">[a], <span class="institution" title="institution">Discipline of Biosciences and Biomedical Engineering, Indian Institute of Technology Indore</span><span class="city" tagx="city" title="city">Indore</span><span class="country" tagx="country" title="country">India</span></span><span class="citation_author_institution" id="AF0002">[b], <span class="institution" title="institution">Discipline of Chemistry, Indian Institute of Technology Palakkad</span><span class="city" tagx="city" title="city">Palakkad</span><span class="country" tagx="country" title="country">India</span></span></div> <div class="author-notes" title="author-notes"> <div class="fn-type-" title=""> <p>Supplemental data for this article can be accessed online at <a href="https://doi.org/10.1080/07391102.2020.1796810">https://doi.org/10.1080/07391102.2020.1796810</a>.</p> </div> <div class="corresp" title="corresp"><b>CONTACT</b> Sushabhan Sadhukhan <span class="email" tagx="email" title="email">sushabhan@iitpkd.ac.in</span><span class="institution" title="institution">Discipline of Chemistry, Indian Institute of Technology Palakkad</span>,<span class="state" tagx="state" title="state">Kerala</span><span class="postal-code" tagx="postal-code" title="postal-code">678 557</span>, <span class="country" tagx="country" title="country">India</span></div> </div><span class="pub-date-" title="pub-date-">: <span>2020-7-7</span></span><span class="pub-date-" title="pub-date-">: <span>2020</span></span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">16</span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Informa UK Limited, trading as Taylor &amp;amp; Francis Group</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This article is made available via the PMC Open Access Subset for unrestricted re-use and analyses in any form or by any means with acknowledgement of the original source. These permissions are granted for the duration of the COVID-19 pandemic or until permissions are revoked in writing. Upon expiration of these permissions, PMC is granted a perpetual license to make this article available via PMC and Europe PMC, consistent with existing copyright protections.</span></span></div><a class="self-uri" href="TBSD_0_1796810.pdf" title="self-uri">TBSD_0_1796810.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <p>The sudden outburst of Coronavirus disease (COVID-19) caused by the Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) poses a massive threat to global public health. Currently, no therapeutic drug or vaccine exists to treat COVID-19. Due to the time taking process of new drug development, drug repurposing might be the only viable solution to tackle COVID-19. RNA‐dependent RNA polymerase (RdRp) catalyzes SARS-CoV-2 RNA replication and hence, is an obvious target for antiviral drug design. Interestingly, several plant-derived polyphenols effectively inhibit the RdRp of other RNA viruses. More importantly, polyphenols have been used as dietary supplementations for a long time and played beneficial roles in immune homeostasis. We were curious to study the binding of polyphenols with SARS-CoV-2 RdRp and assess their potential to treat COVID-19. Herein, we made a library of polyphenols that have shown substantial therapeutic effects against various diseases. They were successfully docked in the catalytic pocket of RdRp. The investigation reveals that EGCG, theaflavin (TF1), theaflavin-3’-<i>O</i>-gallate (TF2a), theaflavin-3’-gallate (TF2b), theaflavin 3,3'-digallate (TF3), hesperidin, quercetagetin, and myricetin strongly bind to the active site of RdRp. Further, a 150-ns molecular dynamic simulation revealed that EGCG, TF2a, TF2b, TF3 result in highly stable bound conformations with RdRp. The binding free energy components calculated by the MM-PBSA also confirm the stability of the complexes. We also performed a detailed analysis of ADME prediction, toxicity prediction, and target analysis for their druggability. Overall, our results suggest that EGCG, TF2a, TF2b, TF3 can inhibit RdRp and represent an effective therapy for COVID-19.</p> <p>Communicated by Ramaswamy H. Sarma</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
SARS-CoV-2 and RNA‐dependent RNA polymerase (RdRp)</div> <div class="kwd" title="kwd">
molecular docking</div> <div class="kwd" title="kwd">
molecular dynamics</div> <div class="kwd" title="kwd">
MM-PBSA</div> <div class="kwd" title="kwd">
natural polyphenols</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span><span class="word-count" title="word-count">word-count: </span></div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"><span class="label" tagx="label" title="label">1.</span><div class="title" tagx="title" title="title">
Introduction</div> <p>An outbreak of Coronavirus disease (COVID-19) has caused a pandemic situation across the globe. Although the outburst was first observed at Wuhan city in China, at present more than 200 countries and territories around the world have witnessed the COVID-19 fatalities affecting all age groups. As of July 03, 2020, more than 11 million people have been affected by the disease, with a fatality of 5,25,410 across the globe as per the WHO report. Unfortunately, there is no clinically approved drug or vaccine for COVID-19 as of now. The disease is caused by the Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), a member of the Coronaviridae family of viruses and it belongs to the same family <i>Betacoronaviruses,</i> like Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) and Middle East Respiratory Syndrome Coronavirus (MERS-CoV) (Chan et al., <a href="#CIT0011">2015</a>; Elfiky &amp;amp; Azzam, <a href="#CIT0021">2020</a>; Ibrahim et al., <a href="#CIT0029">2020</a>). The symptoms of SARS-CoV-2 infection include fever, dry cough, shortness of breath, runny nose, and sore throat (Wu et al., <a href="#CIT0072">2020</a>). SARS-CoV-2 is a positive-sense single-stranded RNA virus, and its genome is around 29.7 kB long with twelve putative open reading frames (ORFs) that encode different viral structural and non-structural proteins. There are four structural proteins in SARS-CoV-2, namely spike (S), envelope (E), membrane (M), and nucleocapsid (N), and all of them can potentially serve as an antigen for neutralizing antibody preparation as potential therapeutics (Boopathi et al., <a href="#CIT0007">2020</a>).</p> <p>Another potential drug target for SARS-CoV-2 is RNA‐dependent RNA polymerase (RdRp) (<a href="#F0001">Figure 1</a>), which is a key component of the replication machinery of the virus to make multiple copies of the RNA genome (Elfiky <a href="#CIT0020">2020c</a>). RdRp in various coronaviruses are remarkably similar. For example, the RdRp of SARS-CoV exhibits ∼97% sequence similarity with that of SARS-CoV-2. More importantly, there is no human polymerase counterpart that resembles the sequence/structural homology with RdRp from coronaviruses, and hence, the development of RdRp inhibitors could be a potential therapeutic strategy without risk of crosstalk with human polymerases (Borgio et al., <a href="#CIT0008">2020</a>; Subissi et al., <a href="#CIT0065">2014</a>; Zhai et al., <a href="#CIT0077">2005</a>). Very recently, Yin et al. reported the crystal structure RdRp of SARS-CoV-2 complexed with an antiviral drug, Remdesivir highlighting how the template-primer RNA is recognized by the polymerase enzyme and the chain elongation is inhibited by Remdesivir providing a basis for developing a wide range of effective inhibitors to overcome from SARS-CoV-2 infection (Yin et al., <a href="#CIT0075">2020</a>). RdRp has been found to be an effective drug target for several other RNA viruses, spanning from the hepatitis C virus, zika virus to coronaviruses (Elfiky, <a href="#CIT0016">2017</a>, <a href="#CIT0017">2019</a>; Ganesan &amp;amp; Barakat, <a href="#CIT0024">2017</a>). The active site of RdRp is highly conserved, and the catalytic domains contain two consecutive aspartate residues in a beta-turn joining β15 and β16 (Elfiky <a href="#CIT0020">2020c</a>). The general structure of RdRp consists of 7 motifs (A to G) among them inner channel of catalytic sites represented by motif A to C, and they play a crucial role during the nucleotide addition cycle (Jia &amp;amp; Gong, <a href="#CIT0032">2019</a>; Wu &amp;amp; Gong, <a href="#CIT0073">2018</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1.</span></a></div>   <p>The 3-dimensional crystal structure of RNA‐dependent RNA polymerase (RdRp).</p>   </div> <p>Epidemiological studies repetitively suggested that consumption of bioactive compounds (<i>e.g.</i> vitamins, phytochemicals, polyphenols, flavonoids, flavonols, and carotenoids etc.) has beneficial activity on human health and could minimize the risk of various diseases starting from cancers to different viral infections (Khan et al., <a href="#CIT0043">2020</a>; Szajdek &amp;amp; Borowska, <a href="#CIT0066">2008</a>). Traditional natural compounds have been consumed since ancient times as they exhibit less toxicity, low-cost availability, minimum side-effects and are rich in therapeutic resources. Some of the previous findings also suggest that naturally occurring compounds possess a wide range of antiviral properties against RNA viruses, including polio-virus type 1, parainfluenza virus type 3, and respiratory syncytial virus by inhibiting their replication (Lin et al., <a href="#CIT0049">2014</a>). In that context, Ahmed-Belkacem et al. have screened more than forty potent natural flavonoids for their polymerase inhibition activity using HCV-NS5 strain and among the different flavonoids, quercetagetin showed strong HCV replication inhibitory activity <i>in vitro</i> (Ahmed-Belkacem et al., <a href="#CIT0003">2014</a>). Previously, song et al. reported that green tea catechins, by disrupting the membrane of the influenza virus, inhibited neuraminidase in the crude system (Song et al., <a href="#CIT0064">2005</a>). On a separate report, Takashi et al. reported that EGCG, a green tea polyphenol can inhibit the endonuclease activity of influenza A virus RNA polymerase. EGCG is also reported to interfere with viral replication <i>via</i> modulating the cellular redox environment (Ho et al., <a href="#CIT0028">2009</a>; Kuzuhara et al., <a href="#CIT0047">2009</a>). Therefore, the existing scientific evidence strongly suggests that natural flavonoids/polyphenol can act against SARS-CoV-2 (Aanouz et al., <a href="#CIT0001">2020</a>; Elfiky <a href="#CIT0019">2020b</a>; Elmezayen et al., <a href="#CIT0022">2020</a>; Enmozhi et al., <a href="#CIT0023">2020</a>). Because of the time-consuming process of new synthetic/semi-synthetic drug development, drug repurposing of phytomolecules is an ideal alternative in this urgent situation as the latter process is economical and scalable in a very short period of time (Adeoye et al., <a href="#CIT0002">2020</a>; Islam et al., <a href="#CIT0030">2020</a>; Muralidharan et al., <a href="#CIT0054">2020</a>; Sinha et al., <a href="#CIT0061">2020</a>). Hence, a comprehensive understanding of their binding to SARS-CoV-2 RdRp can yield interesting findings that can further be capitalized to develop COVID-19 drugs. Nevertheless, the apparent lack of cytotoxicity of polyphenols at even significantly high concentrations makes them potential antiviral drug candidates. In the present study, we selected a hundred natural polyphenols to assess their potential to act as SARS-CoV-2 RdRp inhibitors. The selected library was then explored to evaluate the binding affinity of individual polyphenols towards RdRp of the SARS-CoV-2 by molecular docking using AutoDock vina. Among the selected polyphenols, theaflavin (TF1), theaflavin-3′-O-gallate (TF2a), theaflavin-3′-gallate (TF2b), theaflavin 3,3′-digallate (TF3), hesperidin, EGCG, myricetin, and quercetagetin were found to be docked in the active site of RdRp of the SARS-CoV-2 with a highly favourable affinity for the binding pocket. Further to get a better understanding of the dynamics of the complexes, we performed a 150-nanoseconds molecular dynamic simulation with those eight polyphenols. The binding free energy components were calculated by the MM-PBSA. Remdesivir (Hendaus, <a href="#CIT0027">2020</a>) and GTP (a physiological nucleotide) were taken as positive controls to validate our results.</p> </div> <div class="materialsandmethods" title="sec"><span class="label" tagx="label" title="label">2.</span><div class="title" tagx="title" title="title">
Materials and methods</div> <div class="moleculardockingstudies" title="sec"><span class="label" tagx="label" title="label">2.1.</span><div class="title" tagx="title" title="title">
Molecular docking studies</div> <div class="proteinpreparations" title="sec"><span class="label" tagx="label" title="label">2.1.1.</span><div class="title" tagx="title" title="title">
Protein preparations</div> <p>The crystal structure of SARS-CoV-2 RdRp (PDB ID: 6M71) (Yan et al., <a href="#CIT0074">2020</a>) was retrieved from the protein databank (<a href="http://www.rcsb.org">www.rcsb.org</a>) (Berman et al., <a href="#CIT0005">2000</a>). The crystal structure was prepared individually by adding hydrogen atoms and computing the Gasteiger charge using the AutoDock v4.2 program (Morris et al., <a href="#CIT0053">2009</a>). Subsequently, the file was saved as .pdbqt format in preparation for molecular docking. Schematic representation of the work-flow for selecting potential natural polyphenolic inhibitors for the SARS-CoV-2 RdRp is shown in <a href="#F0002">Figure 2</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2.</span></a></div>   <p>Flow chart of the methodology for shortlisting the best natural polyphenolic inhibitor of the SARS-CoV-2 RdRp.</p>   </div> </div> <div class="ligandpreparations" title="sec"><span class="label" tagx="label" title="label">2.1.2.</span><div class="title" tagx="title" title="title">
Ligand preparations</div> <p>The SDF structures of GTP, remdesivir, and selected hundred polyphenols (see <a href="#t0001">Table S1</a> in <a href="https://doi.org/10.1080/07391102.2020.1796810">Supplementary Information</a>) were retrieved from the PubChem database (<a href="https://pubchem.ncbi.nlm.nih.gov/">https://pubchem.ncbi.nlm.nih.gov/</a>) (Kim et al., <a href="#CIT0044">2019</a>). The compounds were converted into PDB format, and conformational energies of all the compounds were minimized by using UCSF Chimera (Pettersen et al., <a href="#CIT0055">2004</a>).</p> <div class="table-wrap_UNKNOWN" id="t0001" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 1.</span> <p>Binding energy (kcal/mol) of the natural polyphenols along with the control compounds (GTP and remdesivir) against RdRp of the SARS-CoV-2 (PDB ID: 6M71) by molecular docking study.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th align="left">S. No.</th> <th align="center">Compound Name</th> <th align="center">Binding energy (kcal/mol)</th> <th align="center">S. No.</th> <th align="center">Compound Name</th> <th align="center">Binding energy (kcal/mol)</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">1</td> <td align="left"><b>TF3</b></td> <td align="char" char="."><b>−9.9</b></td> <td align="char" char=".">52</td> <td align="center">Cyanidin</td> <td align="char" char=".">−6.3</td> </tr> <tr> <td align="left">2</td> <td align="left"><b>TF2b</b></td> <td align="char" char="."><b>−9.6</b></td> <td align="char" char=".">53</td> <td align="center">Daidzein</td> <td align="char" char=".">−6.3</td> </tr> <tr> <td align="left">3</td> <td align="left"><b>TF1</b></td> <td align="char" char="."><b>−9.6</b></td> <td align="char" char=".">54</td> <td align="center">Glycitein</td> <td align="char" char=".">−6.3</td> </tr> <tr> <td align="left">4</td> <td align="left"><b>TF2a</b></td> <td align="char" char="."><b>−9.3</b></td> <td align="char" char=".">55</td> <td align="center">Wogonin</td> <td align="char" char=".">−6.3</td> </tr> <tr> <td align="left">5</td> <td align="left"><b>Hesperidin</b></td> <td align="char" char="."><b>−8.8</b></td> <td align="char" char=".">56</td> <td align="center">Phloretin</td> <td align="char" char=".">−6.3</td> </tr> <tr> <td align="left">6</td> <td align="left"><b>EGCG</b></td> <td align="char" char="."><b>−7.3</b></td> <td align="char" char=".">57</td> <td align="center">Catechin</td> <td align="char" char=".">−6.2</td> </tr> <tr> <td align="left">7</td> <td align="left"><b>Myricetin</b></td> <td align="char" char="."><b>−7.2</b></td> <td align="char" char=".">58</td> <td align="center">Urolithin B</td> <td align="char" char=".">−6.2</td> </tr> <tr> <td align="left">8</td> <td align="left"><b>Quercetagetin</b></td> <td align="char" char="."><b>−7.0</b></td> <td align="char" char=".">59</td> <td align="center">Angolensin</td> <td align="char" char=".">−6.2</td> </tr> <tr> <td align="left">9</td> <td align="left">Quercetin</td> <td align="char" char=".">−6.9</td> <td align="char" char=".">60</td> <td align="center">Pinosylvin</td> <td align="char" char=".">−6.2</td> </tr> <tr> <td align="left">10</td> <td align="left">Curcumin</td> <td align="char" char=".">−6.9</td> <td align="char" char=".">61</td> <td align="center">Formononetin</td> <td align="char" char=".">−6.2</td> </tr> <tr> <td align="left">11</td> <td align="left">Dihydrorobinetin</td> <td align="char" char=".">−6.8</td> <td align="char" char=".">62</td> <td align="center">Liquiritigenin</td> <td align="char" char=".">−6.2</td> </tr> <tr> <td align="left">12</td> <td align="left">Peonidin</td> <td align="char" char=".">−6.8</td> <td align="char" char=".">63</td> <td align="center">Prunetin</td> <td align="char" char=".">−6.2</td> </tr> <tr> <td align="left">13</td> <td align="left">Fisetin</td> <td align="char" char=".">−6.8</td> <td align="char" char=".">64</td> <td align="center">Alpinetin</td> <td align="char" char=".">−6.2</td> </tr> <tr> <td align="left">14</td> <td align="left">Robinetin</td> <td align="char" char=".">−6.7</td> <td align="char" char=".">65</td> <td align="center">Biochanin A</td> <td align="char" char=".">−6.2</td> </tr> <tr> <td align="left">15</td> <td align="left">5-Deoxygalangin</td> <td align="char" char=".">−6.7</td> <td align="char" char=".">66</td> <td align="center">Rhapontigenin</td> <td align="char" char=".">−6.1</td> </tr> <tr> <td align="left">16</td> <td align="left">Kaempferol</td> <td align="char" char=".">−6.7</td> <td align="char" char=".">67</td> <td align="center">Genistein</td> <td align="char" char=".">−6.1</td> </tr> <tr> <td align="left">17</td> <td align="left">Scutellarein</td> <td align="char" char=".">−6.7</td> <td align="char" char=".">68</td> <td align="center">Chrysin</td> <td align="char" char=".">−6.1</td> </tr> <tr> <td align="left">18</td> <td align="left"><i>(-)</i>-Epicatechin</td> <td align="char" char=".">−6.7</td> <td align="char" char=".">69</td> <td align="center">6-Hydroxyflavone</td> <td align="char" char=".">−6.1</td> </tr> <tr> <td align="left">19</td> <td align="left">Purpurin</td> <td align="char" char=".">−6.7</td> <td align="char" char=".">70</td> <td align="center">Equol</td> <td align="char" char=".">−6.1</td> </tr> <tr> <td align="left">20</td> <td align="left">Isorhamnetin</td> <td align="char" char=".">−6.7</td> <td align="char" char=".">71</td> <td align="center">Piceatannol</td> <td align="char" char=".">−6.1</td> </tr> <tr> <td align="left">21</td> <td align="left">Tricetin</td> <td align="char" char=".">−6.6</td> <td align="char" char=".">72</td> <td align="center">Isorhapontigenin</td> <td align="char" char=".">−6.0</td> </tr> <tr> <td align="left">22</td> <td align="left">Gossypetin</td> <td align="char" char=".">−6.6</td> <td align="char" char=".">73</td> <td align="center">Resveratrol</td> <td align="char" char=".">−5.8</td> </tr> <tr> <td align="left">23</td> <td align="left">Norathyriol</td> <td align="char" char=".">−6.6</td> <td align="char" char=".">74</td> <td align="center">Danshensu</td> <td align="char" char=".">−5.7</td> </tr> <tr> <td align="left">24</td> <td align="left">Coumestrol</td> <td align="char" char=".">−6.6</td> <td align="char" char=".">75</td> <td align="center">Eugenin</td> <td align="char" char=".">−5.6</td> </tr> <tr> <td align="left">25</td> <td align="left">Isosakuranetin</td> <td align="char" char=".">−6.6</td> <td align="char" char=".">76</td> <td align="center">Sinapic acid</td> <td align="char" char=".">−5.5</td> </tr> <tr> <td align="left">26</td> <td align="left">Pectolinarigenin</td> <td align="char" char=".">−6.6</td> <td align="char" char=".">77</td> <td align="center">Pterostilbene</td> <td align="char" char=".">−5.5</td> </tr> <tr> <td align="left">27</td> <td align="left">Tangeritin</td> <td align="char" char=".">−6.6</td> <td align="char" char=".">78</td> <td align="center">Ferulic acid</td> <td align="char" char=".">−5.4</td> </tr> <tr> <td align="left">28</td> <td align="left">Nobiletin</td> <td align="char" char=".">−6.6</td> <td align="char" char=".">79</td> <td align="center">Caffeic acid</td> <td align="char" char=".">−5.4</td> </tr> <tr> <td align="left">29</td> <td align="left">Pratensein</td> <td align="char" char=".">−6.6</td> <td align="char" char=".">80</td> <td align="center">Isoferulic acid</td> <td align="char" char=".">−5.4</td> </tr> <tr> <td align="left">30</td> <td align="left">Hispidulin</td> <td align="char" char=".">−6.6</td> <td align="char" char=".">81</td> <td align="center">Dihydrocaffeic acid</td> <td align="char" char=".">−5.4</td> </tr> <tr> <td align="left">31</td> <td align="left">Baicalein</td> <td align="char" char=".">−6.5</td> <td align="char" char=".">82</td> <td align="center">Gentisic acid</td> <td align="char" char=".">−5.3</td> </tr> <tr> <td align="left">32</td> <td align="left">Apigenin</td> <td align="char" char=".">−6.5</td> <td align="char" char=".">83</td> <td align="center">Pyrogallol</td> <td align="char" char=".">−5.3</td> </tr> <tr> <td align="left">33</td> <td align="left">Morin</td> <td align="char" char=".">−6.5</td> <td align="char" char=".">84</td> <td align="center">4-Hydroxycinnamic acid</td> <td align="char" char=".">−5.2</td> </tr> <tr> <td align="left">34</td> <td align="left">Urolithin A</td> <td align="char" char=".">−6.5</td> <td align="char" char=".">85</td> <td align="center">Resacetophenone</td> <td align="char" char=".">−5.2</td> </tr> <tr> <td align="left">35</td> <td align="left">Acacetin</td> <td align="char" char=".">−6.5</td> <td align="char" char=".">86</td> <td align="center">Salicyclic acid</td> <td align="char" char=".">−5.1</td> </tr> <tr> <td align="left">36</td> <td align="left">Pelargonidin</td> <td align="char" char=".">−6.5</td> <td align="char" char=".">87</td> <td align="center">Syringic acid</td> <td align="char" char=".">−5.1</td> </tr> <tr> <td align="left">37</td> <td align="left">Irilone</td> <td align="char" char=".">−6.5</td> <td align="char" char=".">88</td> <td align="center">2-Hydroxybenzoic acid</td> <td align="char" char=".">−5.1</td> </tr> <tr> <td align="left">38</td> <td align="left">Naringenin</td> <td align="char" char=".">−6.5</td> <td align="char" char=".">89</td> <td align="center">Gallic acid</td> <td align="char" char=".">−5.0</td> </tr> <tr> <td align="left">39</td> <td align="left">Pinocembrin</td> <td align="char" char=".">−6.5</td> <td align="char" char=".">90</td> <td align="center">3-Hydroxybenzoic acid</td> <td align="char" char=".">−5.0</td> </tr> <tr> <td align="left">40</td> <td align="left">Kaempferide</td> <td align="char" char=".">−6.5</td> <td align="char" char=".">91</td> <td align="center">4-Hydroxybenzoic acid</td> <td align="char" char=".">−5.0</td> </tr> <tr> <td align="left">41</td> <td align="left">Malvidin</td> <td align="char" char=".">−6.5</td> <td align="char" char=".">92</td> <td align="center">Vanillin</td> <td align="char" char=".">−5.0</td> </tr> <tr> <td align="left">42</td> <td align="left">Luteolin</td> <td align="char" char=".">−6.4</td> <td align="char" char=".">93</td> <td align="center"><i>p</i>-Coumeric acid</td> <td align="char" char=".">−4.9</td> </tr> <tr> <td align="left">43</td> <td align="left">Dalbergin</td> <td align="char" char=".">−6.4</td> <td align="char" char=".">94</td> <td align="center">Vanillic acid</td> <td align="char" char=".">−4.8</td> </tr> <tr> <td align="left">44</td> <td align="left">Butein</td> <td align="char" char=".">−6.4</td> <td align="char" char=".">95</td> <td align="center">Paeonol</td> <td align="char" char=".">−4.8</td> </tr> <tr> <td align="left">45</td> <td align="left">Biochanin A (1-)</td> <td align="char" char=".">−6.4</td> <td align="char" char=".">96</td> <td align="center">Cinnamic acid</td> <td align="char" char=".">−4.7</td> </tr> <tr> <td align="left">46</td> <td align="left">Fustin</td> <td align="char" char=".">−6.4</td> <td align="char" char=".">97</td> <td align="center">Protocatechuic acid</td> <td align="char" char=".">−4.6</td> </tr> <tr> <td align="left">47</td> <td align="left">5-Hydroxyflavone</td> <td align="char" char=".">−6.4</td> <td align="char" char=".">98</td> <td align="center">4-Ethylphenol</td> <td align="char" char=".">−4.5</td> </tr> <tr> <td align="left">48</td> <td align="left">Pinostrobin</td> <td align="char" char=".">−6.4</td> <td align="char" char=".">99</td> <td align="center">Catechol</td> <td align="char" char=".">−4.5</td> </tr> <tr> <td align="left">49</td> <td align="left">Pinobanksin</td> <td align="char" char=".">−6.4</td> <td align="char" char=".">100</td> <td align="center">Tyrosol</td> <td align="char" char=".">−4.5</td> </tr> <tr> <td align="left">50</td> <td align="left">Datiscetin</td> <td align="char" char=".">−6.3</td> <td align="char" char=".">101</td> <td align="center"><b>GTP</b></td> <td align="char" char="."><b>−7.9</b></td> </tr> <tr> <td align="left">51</td> <td align="left">Galangin</td> <td align="char" char=".">−6.3</td> <td align="char" char=".">102</td> <td align="center"><b>Remdesivir</b></td> <td align="char" char="."><b>−7.7</b></td> </tr> </tbody> </table> </div> </div> <div class="dockingstudiesusingautodockvina" title="sec"><span class="label" tagx="label" title="label">2.1.3.</span><div class="title" tagx="title" title="title">
Docking studies using AutoDock Vina</div> <p>The energy-minimized structure of all the natural polyphenols, remdesivir, and GTP were docked with the receptor (RdRp of SARS-CoV-2) using AutoDock Vina 1.1.2 (Trott &amp;amp; Olson, <a href="#CIT0067">2010</a>). The ligand files were further saved in PDBQT file format, a modified PDB format containing atomic charges, atom type definitions for ligands, and topological information (rotatable bonds). A grid box (30 Å × 30 Å × 30 Å) centered at (121, 120, 125) Å for the SARS-CoV-2 RdRp, was used in the docking experiments. After the receptor-ligand preparation, docking runs were started from the command prompt. The lowest binding energy and best-docked conformation were considered as the ligand molecule with maximum binding affinity.</p> </div> <div class="protein-ligandinteractions" title="sec"><span class="label" tagx="label" title="label">2.1.4.</span><div class="title" tagx="title" title="title">
Protein-ligand interactions</div> <p>LigPlot<sup>+</sup> was used to investigate protein-ligand interactions for a given .pdb file containing the docked conformation and also the final simulated conformation (Wallace et al., <a href="#CIT0068">1995</a>). The LigPlot<sup>+</sup> program self-generated schematic 2D representations of protein-ligand interaction. The output file represents the intermolecular interactions and their strengths, including hydrogen bonds, hydrophobic contacts, and atom accessibilities. H-bonds are shown in green dotted lines, whereas residues involved in hydrophobic interaction are represented in the red semicircle.</p> </div> </div> <div class="moleculardynamicssimulations" title="sec"><span class="label" tagx="label" title="label">2.2.</span><div class="title" tagx="title" title="title">
Molecular dynamics simulations</div> <p>All-atom molecular dynamics (MD) simulations were performed on the best eight selected plant-derived natural polyphenols obtained from the molecular docking study along with remdesivir, a well-known RdRp inhibitor, for studying thermodynamic stability of the docked structure. The <i>pmemd.cuda</i> module in AMBER18 (Case et al. <a href="#CIT0010">2018</a>) was used for conducting MD simulations, and all simulations were performed utilizing the graphics card, RTX 2080Ti. We adopted the same protocol that was used in our earlier studies (Jonniya et al., <a href="#CIT0034">2019</a>; Sk, Roy, Jonniya, et al., <a href="#CIT0062">2020</a>; Sk, Roy, &amp;amp; Kar, <a href="#CIT0063">2020</a>). The receptor and small molecules were described by the Amber ff14SB (Maier et al., <a href="#CIT0051">2015</a>) and GAFF2 (Wang et al., <a href="#CIT0071">2004</a>) force field, respectively. Ligands were assigned the AM1-BCC (Jakalian et al., <a href="#CIT0031">2002</a>) atomic charges calculated using the <i>antechamber</i> (Wang et al., <a href="#CIT0069">2001</a>) module. The complexes were then solvated using an explicit TIP3P (Price &amp;amp; Brooks III, <a href="#CIT0057">2004</a>) water model, and nearly 38124 water molecules were needed to solvate each system. Subsequently, all solvated systems were neutralized by adding an appropriate number of Na<sup>+</sup> ion. All bond lengths, including hydrogen atoms, were constrained by the SHAKE algorithm (Kräutler et al., <a href="#CIT0046">2001</a>). This allows the usage of a 2 fs time-step. The non-bonded cut-off was set to 8 Å and the long range electrostatic interactions were evaluated using the particle-mesh Ewald (PME) (Darden et al., <a href="#CIT0015">1993</a>) method. The temperature was kept at 300 K using the Langevin thermostat (Loncharich et al., <a href="#CIT0050">1992</a>) with a collision frequency of 2 ps<sup>−1</sup>. The system pressure was controlled by Berendsen’s Barostat (Berendsen et al., <a href="#CIT0004">1984</a>) and fixed at 1.0 bar. We used a time-step of 2.0 fs for all simulations.</p> <p>Briefly, we used two stages of minimization. Firstly, each complex was optimized by 500 steps of steepest descent followed by another 500 steps of conjugate gradient minimization, keeping all atoms of the complex restrained to their initial coordinate with a weak harmonic potential (force constant 2.0 kcal mol<sup>−1 </sup>Å<sup>−2</sup>). The second stage of minimization was carried out without any restraints by performing 100 steps of steepest descent, followed by another 900 steps of conjugate gradient minimization to remove any residual steric clashes. Subsequently, all systems were gradually heated from 0 K to 300 K at the NVT ensemble with a force constant of 2.0 kcal mol<sup>−1</sup>Å<sup>−2</sup> acting on all solute atoms. Next, 1.0 ns equilibration MD phase was carried out without any restraint. Finally, we performed 150 ns production simulations for all four systems at the NPT ensemble. Overall, we accumulated 15000 conformations for each simulation, and we used 500 snapshots from the last 50 ns trajectories for binding affinity estimation using the MM-PBSA scheme. The trajectory analysis was done by the AmberTools19 CPPTRAJ (Roe &amp;amp; Cheatham III, <a href="#CIT0059">2013</a>) module of Amber18.</p> </div> <div class="molecularmechanicspoisson-boltzmannsurfaceareacalculations" title="sec"><span class="label" tagx="label" title="label">2.3.</span><div class="title" tagx="title" title="title">
Molecular mechanics Poisson-Boltzmann surface area calculations</div> <p>The binding affinity of EGCG, TF1, TF3, TF2b, TF2a, hesperidin, myricetin, quercetagetin, and remdesivir against the SARS-CoV-2 RdRp, were calculated by the molecular mechanics Poisson-Boltzmann surface area (MM-PBSA) methodology (Jonniya &amp;amp; Kar, <a href="#CIT0033">2020</a>; Kar, Seel, et al., <a href="#CIT0041">2007</a>; Kar, Wei, et al., <a href="#CIT0042">2007</a>; Kar et al., <a href="#CIT0039">2011</a>; <a href="#CIT0040">2013</a>; Kar &amp;amp; Knecht, <a href="#CIT0035">2012a</a>, <a href="#CIT0036">2012b</a>, <a href="#CIT0037">2012c</a>, <a href="#CIT0038">2012d</a>; Kollman et al., <a href="#CIT0045">2000</a>; Roy et al., <a href="#CIT0060">2020</a>; Wang et al., <a href="#CIT0070">2006</a>). The MM-PBSA scheme can briefly be described as follows: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(1)</span><div class="alternatives" title="alternatives"> <div class="graphic" position="float" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Δ</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
bind</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
H</div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
≈</div> <div class="mi_UNKNOWN">
Δ</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
MM</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
Δ</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
solv</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
S</div></div></div> </div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(2)</span><div class="alternatives" title="alternatives"> <div class="graphic" position="float" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Δ</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
MM</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
Δ</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
internal</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
Δ</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
elec</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
Δ</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
vdW</div> </div> </div></div></div> </div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(3)</span><div class="alternatives" title="alternatives"> <div class="graphic" position="float" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Δ</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
solv</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
Δ</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
pol</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
Δ</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
np</div> </div> </div></div></div> </div> where <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"><span class="inline-graphic" title="inline-graphic" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Δ</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
MM</div> </div> </div> <div class="mo_UNKNOWN">
,</div></div></div></span><span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"><span class="inline-graphic" title="inline-graphic" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Δ</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
solv</div> </div> </div> <div class="mo_UNKNOWN">
,</div></div></div></span><span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"><span class="inline-graphic" title="inline-graphic" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
S</div></div></div></span> are the changes in molecular mechanical energy, solvation free energy, and conformational entropy, respectively. Further, molecular mechanical energy is composed of <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"><span class="inline-graphic" title="inline-graphic" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Δ</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
internal</div> </div> </div> <div class="mi_UNKNOWN"> </div></div></div></span>(bond, dihedral, and angle), <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"><span class="inline-graphic" title="inline-graphic" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Δ</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
elec</div> </div> </div></div></div></span> (electrostatic) and <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"><span class="inline-graphic" title="inline-graphic" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Δ</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
vdW</div> </div> </div></div></div></span> (van der Waals) and the change in desolvation free energy is composed of polar solvation (<span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"><span class="inline-graphic" title="inline-graphic" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Δ</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
pol</div> </div> </div></div></div></span>) and non-polar solvation free energy (<span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"><span class="inline-graphic" title="inline-graphic" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Δ</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
np</div> </div> </div></div></div></span>). The polar solvation free energy, <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"><span class="inline-graphic" title="inline-graphic" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
Δ</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
pol</div> </div> </div> <div class="mo_UNKNOWN">
,</div></div></div></span> was calculated by the <i>pbsa</i> module of AMBER18. Due to the high computational cost, we neglect the configurational entropy calculations. Further, to understand the polyphenol-protein interaction more closely, the interaction energy was decomposed into the contributions from each residue of the protein by using the molecular mechanics generalized Born surface area (MM-GBSA) scheme (Gohlke et al., <a href="#CIT0026">2003</a>).<p /> </div> <div class="admetstudies" title="sec"><span class="label" tagx="label" title="label">2.4.</span><div class="title" tagx="title" title="title">
ADMET studies</div> <p>The <i>in-silico</i> pharmacological studies of EGCG, TF2a, TF2b, TF3, and remdesivir were predicted based on their ADMET profile. The ADMET studies (absorption, distribution, metabolism, elimination, and toxicity) were predicted using the pkCSM tool (<a href="http://biosig.unimelb.edu.au/pkcsm/prediction">http://biosig.unimelb.edu.au/pkcsm/prediction</a>) (Pires et al., <a href="#CIT0056">2015</a>). The canonical SMILE molecular structures of the above-mentioned compounds were retrieved from the PubChem database (<a href="http://www.pubchem.ncbi.nlm.nih.gov">www.pubchem.ncbi.nlm.nih.gov</a>).</p> </div> <div class="moleculartargetprediction" title="sec"><span class="label" tagx="label" title="label">2.5.</span><div class="title" tagx="title" title="title">
Molecular target prediction</div> <p>Natural compounds interact with a large number of proteins, enzymes, lipids. This interaction plays a crucial role in elucidating the molecular mechanism of the small molecules. So, it is important to identify the molecular targets for new molecules (Gfeller et al., <a href="#CIT0025">2014</a>). Swiss Target Prediction website (<a href="http://www.swisstargetprediction.ch/index.php">http://www.swisstargetprediction.ch/index.php</a>) was logged on, and canonical SMILE molecular structures of remdesivir, EGCG, TF2a, TF2b, and TF3 were entered in the search bar option, and results were analyzed.</p> </div> </div> <div class="resultsanddiscussion" title="sec"><span class="label" tagx="label" title="label">3.</span><div class="title" tagx="title" title="title">
Results and discussion</div> <div class="moleculardockinganalysis" title="sec"><span class="label" tagx="label" title="label">3.1.</span><div class="title" tagx="title" title="title">
Molecular docking analysis</div> <div class="thebindingmodeanalysisandpredictedbindingaffinitycalculationsofnaturalpolyphenolsagainstthesars-cov-2rdrp" title="sec"><span class="label" tagx="label" title="label">3.1.1.</span><div class="title" tagx="title" title="title">
The binding mode analysis and predicted binding affinity calculations of natural polyphenols against the SARS-CoV-2 RdRp</div> <p>Herein, we investigated our natural polyphenol library against RdRp of the SARS-CoV-2 (PDB ID: 6M71) by molecular docking. The best conformation of the natural polyphenols was docked against the SARS-CoV-2 RdRp, and resulting binding energies are listed in <a href="#t0001">Table 1</a>.</p> <p>Polyphenols exhibiting binding energy of −7.0 kcal/mol or lower (eight polyphenols) against RdRp of the SARS-CoV-2 are listed in <a href="#t0002">Table 2</a> along with the ligand-amino acid interactions. Control compounds, GTP and remdesivir, exhibited the binding energy of −7.9 and −7.7 kcal/mol, respectively, against the SARS-CoV-2 RdRp. Eight polyphenols displayed significantly higher binding affinity among the selected hundred natural polyphenols docked against the SARS-CoV-2 RdRp, with binding energies of TF3, TF2b, TF1, TF2a, hesperidin, EGCG, myricetin and quercetagetin as −9.9, −9.6, −9.6, −9.3, −8.8, −7.3, −7.2 and −7.0 kcal/mol, respectively (highlighted in <a href="#t0001">Table 1</a>). Further, 2D LigPlot<sup>+</sup> representation of RdRp and the above-mentioned eight polyphenols reveal the stable network of molecular interactions (see <a href="#t0002">Table 2</a> and <a href="#F0001">Figure S1</a> in the <a href="https://doi.org/10.1080/07391102.2020.1796810">Supplementary Information</a>).</p> <div class="table-wrap_UNKNOWN" id="t0002" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 2.</span> <p>Ligand-amino acid interactions of top eight scoring natural polyphenols against the SARS-CoV-2 RdRp.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th align="left">S. No.</th> <th align="center">Compound name</th> <th align="center">Binding energy (kcal/mol)</th> <th align="center">No. of non-covalent interactions</th> <th align="center">Involved amino acids</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">1</td> <td align="center">TF3</td> <td align="char" char=".">−9.9</td> <td align="char" char=".">17</td> <td align="left">W617, K551, S549, D623, R836, S814, E811, F812, C813, D761, D618, S759, Y619, C622, R553, K621, D760</td> </tr> <tr> <td align="left">2</td> <td align="center">TF2b</td> <td align="char" char=".">−9.6</td> <td align="char" char=".">13</td> <td align="left">K551, Y619, D760, K798, W617, W800, D761, F812, C813, E811, D618, S549, A550</td> </tr> <tr> <td align="left">3</td> <td align="center">TF1</td> <td align="char" char=".">−9.6</td> <td align="char" char=".">12</td> <td align="left">W617, D761, D760, Y619, R553, K621, P620, F793, D164, S795, K798, D618</td> </tr> <tr> <td align="left">4</td> <td align="center">TF2a</td> <td align="char" char=".">−9.3</td> <td align="char" char=".">14</td> <td align="left">C813, F812, D761, D760, D618, K798, K551, A550, S549, K621, Y619, W800, W617, E811</td> </tr> <tr> <td align="left">5</td> <td align="center">Hesperidin</td> <td align="char" char=".">−8.8</td> <td align="char" char=".">13</td> <td align="left">Y619, D618, K798, S795, M794, P793, D164, V166, P620, K621, D623, R555, Y455</td> </tr> <tr> <td align="left">6</td> <td align="center">EGCG</td> <td align="char" char=".">−7.3</td> <td align="char" char=".">9</td> <td align="left">D623, Y619, K621, S795, C622, D618, M794, P620, K798</td> </tr> <tr> <td align="left">7</td> <td align="center">Myricetin</td> <td align="char" char=".">−7.2</td> <td align="char" char=".">10</td> <td align="left">W617, W800, D760, E811, K798, D618, Y619, C622, D761, F812</td> </tr> <tr> <td align="left">8</td> <td align="center">Quercetagetin</td> <td align="char" char=".">−7.0</td> <td align="char" char=".">8</td> <td align="left">R553, K545, K621, D623, C622, D760, P620, Y619</td> </tr> <tr> <td align="left">9</td> <td align="center">Remdesivir (Control)</td> <td align="char" char=".">−7.7</td> <td align="char" char=".">13</td> <td align="left">R553, K621, C622, D760, E811, W800, K798, P620, Y455, R624, Y619, D618, D761</td> </tr> <tr> <td align="left">10</td> <td align="center">GTP (Control)</td> <td align="char" char=".">−7.9</td> <td align="char" char=".">15</td> <td align="left">R624, T556, D623, D760, Y619, C622, K621, D452, A554, R553, Y455, R555, D761, D618, P620</td> </tr> </tbody> </table> </div> <p>In addition to remdesivir, here we observed that eight dietary polyphenols (TF1, TF2a, TF2b, TF3, hesperidin, EGCG, myricetin and quercetagetin) have significant potential to function as inhibitors of the SARS-CoV-2 RdRp. The top-eight scoring ligands for the SARS-CoV-2 RdRp (as highlighted in <a href="#t0001">Table 1</a>) suggest that these set of natural polyphenols can strongly bind to the catalytic site of the SARS-CoV-2 RdRp and are expected to inhibit the RdRp activity, and thus blocking the replication and preventing viral transcription. Many reports suggest polyphenols have low systemic toxicity, and they are highly beneficial for human health (Bhardwaj et al., <a href="#CIT0006">2020</a>; Cory et al., <a href="#CIT0014">2018</a>). TF1 and its gallate derivatives, collectively known as black tea polyphenols, previously have shown to exert antiviral activity against many viruses such as hepatitis virus and influenza A and B viruses (Chowdhury et al., <a href="#CIT0013">2018</a>). Hesperidin is also known to possess antiviral activity by altering the immune system mainly <i>via</i> regulating interferons in the influenza A virus (Randall &amp;amp; Goodbourn, <a href="#CIT0058">2008</a>). EGCG, a major green tea polyphenol, has several pharmacological properties, including antiviral activity (Carneiro et al., <a href="#CIT0009">2016</a>; Moon &amp;amp; Morris, <a href="#CIT0052">2007</a>). Similarly, myricetin has also been found to act as an inhibitor of the SARS coronavirus helicase (Yu et al., <a href="#CIT0076">2012</a>). Quercetagetin also showed strong hepatitis C virus replication inhibitory activity <i>in vitro</i> (Ahmed-Belkacem et al., <a href="#CIT0003">2014</a>). Thus, the set dietary polyphenols identified in the present study could be used as repurposed drugs for the treatment of the SARS-CoV-2 infection with further <i>in-vitro</i> and <i>in-vivo</i> validations.</p> </div> </div> <div class="moleculardynamics(md)simulation" title="sec"><span class="label" tagx="label" title="label">3.2.</span><div class="title" tagx="title" title="title">
Molecular dynamics (MD) simulation</div> <div class="thermodynamicstabilityandflexibilityanalysis" title="sec"><span class="label" tagx="label" title="label">3.2.1.</span><div class="title" tagx="title" title="title">
Thermodynamic stability and flexibility analysis</div> <p>The 150 ns production simulations carried out for nine systems (complex of remdesivir, EGCG, TF1, TF2a, TF2b, TF3, hesperidin, myricetin, and quercetagetin with the SARS-CoV-2 RdRp) were stable on the basis of the potential energy and total energy (data not shown) of those complexes. Subsequently, the root-mean-square deviations (RMSDs) of backbone atoms relative to their respective initial positions were calculated for each complex and are shown in <a href="#F0003">Figure 3(A)</a>. It is evident from <a href="#F0003">Figure 3(A)</a> that all the nine studied systems drifted from their initial positions during the first 50 ns, and after that, they reached equilibrium. The average RMSD values were estimated to be 2.30 Å, 2.45 Å, 1.87 Å, 2.28 Å, 1.68 Å, 2.47 Å, 1.90 Å, 2.03 Å and 1.88 Å for RdRp/remdesivir, RdRp/EGCG, RdRp/TF3, RdRp/TF2b, RdRp/TF2a, RdRp/myricetin, RdRp/quercetagetin, RdRp/hesperidin, and RdRp/TF1 complexes, respectively (<a href="#t0003">Table 3</a>). The least deviation was observed for RdRp/TF2a, while RdRp/myricetin displayed the highest deviation. We also investigated structural variations in the binding site, including all amino acids that fall within a radius of 5 Å from the inhibitor, and the same trend was observed (<a href="#F0002">Figure S2A</a> in the <a href="https://doi.org/10.1080/07391102.2020.1796810">Supplementary Information</a>). Overall, this suggests the convergence of our simulations.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3.</span></a></div>   <p>(A) Time evolution of root-mean-square deviations (RMSDs) of backbone atoms and (B) the root-mean-square fluctuations (RMSFs) of C<sub>α</sub> atoms of nine complexes relative to their respective energy minimized structure.</p>   </div> <div class="table-wrap_UNKNOWN" id="t0003" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 3.</span> <p>The average backbone RMSD, radius of gyration (RoG), and solvent accessible surface area (SASA) for all nine complexes. The data are reported as average ± standard error of the mean (SEM).</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th align="left">System</th> <th align="center">RMSD (Å)</th> <th align="center">RoG (Å)</th> <th align="center">SASA (Å<sup>2</sup>)</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">RdRp/Remdesivir</td> <td align="left">2.30 ± 0.03</td> <td align="char" char=".">29.96 ± 0.02</td> <td align="char" char=".">34973.20 ± 91.56</td> </tr> <tr> <td align="left">RdRp/EGCG</td> <td align="left">2.45 ± 0.05</td> <td align="char" char=".">29.52 ± 0.06</td> <td align="char" char=".">35026.03 ± 63.52</td> </tr> <tr> <td align="left">RdRp/TF3</td> <td align="left">1.87 ± 0.02</td> <td align="char" char=".">29.60 ± 0.01</td> <td align="char" char=".">34080.16 ± 53.41</td> </tr> <tr> <td align="left">RdRp/TF2b</td> <td align="left">2.28 ± 0.01</td> <td align="char" char=".">29.86 ± 0.02</td> <td align="char" char=".">35462.92 ± 50.40</td> </tr> <tr> <td align="left">RdRp/TF2a</td> <td align="left">1.68 ± 0.02</td> <td align="char" char=".">29.75 ± 0.02</td> <td align="char" char=".">34312.55 ± 112.89</td> </tr> <tr> <td align="left">RdRp/Myricetin</td> <td align="left">2.47 ± 0.03</td> <td align="char" char=".">29.88 ± 0.01</td> <td align="char" char=".">35395.35 ± 104.67</td> </tr> <tr> <td align="left">RdRp/Quercetagetin</td> <td align="left">1.90 ± 0.03</td> <td align="char" char=".">29.84 ± 0.01</td> <td align="char" char=".">34618.65 ± 51.25</td> </tr> <tr> <td align="left">RdRp/Hesperidin</td> <td align="left">2.03 ± 0.04</td> <td align="char" char=".">29.74 ± 0.02</td> <td align="char" char=".">34554.08 ± 47.25</td> </tr> <tr> <td align="left">RdRp/TF1</td> <td align="left">1.88 ± 0.03</td> <td align="char" char=".">29.86 ± 0.01</td> <td align="char" char=".">34420.69 ± 56.90</td> </tr> </tbody> </table> </div> <p>Next, we investigated the structural stability of remdesivir and eight polyphenols by estimating the temporal RMSDs of heavy atoms relative to their respective initial coordinates (see <a href="#F0002">Figure S2B</a> in the <a href="https://doi.org/10.1080/07391102.2020.1796810">Supplementary Information</a>). It is evident from <a href="#F0002">Figure S2B</a> that EGCG, myricetin, quercetagetin, TF1, TF3 displayed a rigid behavior in the bound form with an average RMSD of &amp;lt; 1.0 Å. However, remdesivir, TF2a, TF2b and hesperidin showed higher fluctuations as compared to the abovementioned polyphenols, and an average RMSD of &amp;gt; 2.0 Å was noted.</p> <p>To identify the regions which are flexible, the root-mean-square fluctuations (RMSFs) of C<sub>α</sub> atoms of each residue are calculated and shown in <a href="#F0003">Figure 3(B)</a>. From this analysis, we can get a better insight into what extent the binding of remdesivir and natural polyphenols affects the residual flexibility of RdRp (mainly Nsp12). <a href="#F0003">Figure 3(B)</a> indicates that RdRp/remdesivir and RdRp-polyphenols shared a similar RMSF pattern. Notable dynamic fluctuations were located in the non-active site domain, including both <i>N</i>-terminals and <i>C</i>-terminal. Regions around Asn150, Asp260, Arg305, Asn360, and Phe440 are found to be more flexible compared to the other area for all complexes. The binding pocket residues, such as Asp452, Lys545, Lys551, Tyr455, Arg553, Ala554, Arg555, Thr556, Asp618, Tyr619, pro620, Lys621, Cys622, Asp623 Arg624, Asn691, Asp760, Asp761, Phe793, Met794, Ser795, Lys798, Trp800, Glu811, Phe812, and Ser814 exhibited considerably low fluctuations for all the RdRp-inhibitor complexes. In the case of RdRp/TF3 and RdRp/TF2a, the binding site residues displayed lesser fluctuations compared to the other RdRp-polyphenol complexes. This suggests that TF3 and TF2a are likely to be bound to RdRp more strongly than the other polyphenols.</p> <p>Since the radius of gyration (RoG) helps us to understand the protein structural compactness, RoG of each complex was monitored and represented in <a href="#F0003">Figure S3A</a> in the <a href="https://doi.org/10.1080/07391102.2020.1796810">Supplementary Information</a>. The average values of RoG are 29.96 Å, 29.52 Å, 29.60 Å, 29.86 Å, 29.75 Å, 29.88 Å, 29.84 Å, 29.74 Å and 29.86 Å for RdRp complexed with remdesivir, EGCG, TF3, TF2b, TF2a, myricetin, quercetagetin, hesperidin and TF1 respectively (<a href="#t0003">Table 3</a>). This suggests that the structural compactness remained unchanged during simulations. Finally, the solvent-accessible surface area (SASA) was also explored, and the time evolution of SASA for four RdRp-polyphenol complexes are shown in <a href="#F0003">Figure S3B</a> in the <a href="https://doi.org/10.1080/07391102.2020.1796810">Supplementary Information</a>. The average values of SASA are reported in <a href="#t0003">Table 3</a>. Binding of an inhibitor to the substrate changes SASA and sometimes could greatly affect the protein structure. Here, a relatively higher SASA value was obtained for RdRp/TF2b (35462.9 Å<sup>2</sup>) compared to the other RdRp/inhibitor complexes. On the other hand, the lowest SASA value was noted for RdRp/TF3 (34080.2 Å<sup>2</sup>). Thus, it can be suggested that the binding of TF3 could potentially reduce protein expansion.</p> </div> <div class="bindingfreeenergyanalysis" title="sec"><span class="label" tagx="label" title="label">3.2.2.</span><div class="title" tagx="title" title="title">
Binding free energy analysis</div> <p>We predicted the binding free energy of all nine complexes by utilizing the MM-PBSA scheme, and four polyphenols, namely EGCG, TF3, TF2b, and TF2a, displayed a higher estimated affinity compared to remdesivir as depicted in <a href="#F0004">Figure 4</a>. Various components of the binding free energy of EGCG, TF3, TF2b, and TF2a are reported in <a href="#t0004">Table 4</a>. The remaining four polyphenols which showed lower estimated affinity compare to remdesivir are shown in <a href="#t0002">Table S2</a> in <a href="https://doi.org/10.1080/07391102.2020.1796810">Supplementary Information</a>. It can be noted from <a href="#F0004">Figure 4</a> that the intermolecular van der Waals (Δ<i>E</i><sub>vdW</sub>) and electrostatic (Δ<i>E</i><sub>elec</sub>) terms are favorable for the ligand binding, whereas the desolvation of polar groups (Δ<i>G</i><sub>pol</sub>) opposes the complex formation. Non-polar solvation free energy (Δ<i>G</i><sub>np</sub>) is favorable to the binding for all cases. A similar trend was observed in our earlier study (Sk, Roy, Jonniya, et al., <a href="#CIT0062">2020</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4.</span></a></div>   <p>Energy components (kcal/mol) for the binding of remdesivir and four polyphenols to RdRp receptor. Δ<i>E</i><sub>vdW</sub>, van der Waals interaction; Δ<i>E</i><sub>ele</sub>, electrostatic interaction in the gas phase; Δ<i>G</i><sub>pol</sub>, polar solvation energy; Δ<i>G</i><sub>np</sub>, non-polar solvation energy, and Δ<i>G</i><sub>bind</sub>, estimated binding affinity.</p>   </div> <div class="table-wrap_UNKNOWN" id="t0004" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 4.</span> <p>Energetic components of the binding free energy of RdRp and natural polyphenols along with remdesivir complexes in kcal/mol. Data are represented as average ± SEM.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th align="left">Components</th> <th align="center">Remdesivir</th> <th align="center">EGCG</th> <th align="center">TF3</th> <th align="center">TF2b</th> <th align="center">TF2a</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">Δ<i>E</i><b><sub>vdW</sub></b></td> <td align="char" char="±">−31.85 ± 0.15</td> <td align="char" char="±">−25.11 ± 0.18</td> <td align="char" char="±">−37.82 ± 0.21</td> <td align="char" char="±">−30.66 ± 0.23</td> <td align="char" char="±">−22.55 ± 0.19</td> </tr> <tr> <td align="left">Δ<i>E</i><b><sub>elec</sub></b></td> <td align="char" char="±">−98.40 ± 0.70</td> <td align="char" char="±">−69.38 ± 0.73</td> <td align="char" char="±">−123.63 ± 0.88</td> <td align="char" char="±">−47.18 ± 0.64</td> <td align="char" char="±">−95.28 ± 1.27</td> </tr> <tr> <td align="left">Δ<i>G</i><b><sub>pol</sub></b></td> <td align="char" char="±">109.97 ± 0.57</td> <td align="char" char="±">71.62 ± 0.48</td> <td align="char" char="±">124.47 ± 0.58</td> <td align="char" char="±">55.01 ± 0.49</td> <td align="char" char="±">94.94 ± 1.10</td> </tr> <tr> <td align="left">Δ<i>G</i><b><sub>np</sub></b></td> <td align="char" char="±">−4.29 ± 0.01</td> <td align="char" char="±">−4.15 ± 0.01</td> <td align="char" char="±">−5.29 ± 0.01</td> <td align="char" char="±">−3.91 ± 0.01</td> <td align="char" char="±">−4.28 ± 0.02</td> </tr> <tr> <td align="left"><a href="#TF1"><sup>a</sup></a>Δ<i>G</i><b><sub>solv</sub></b></td> <td align="char" char="±">105.68 ± 0.57</td> <td align="char" char="±">67.47 ± 0.48</td> <td align="char" char="±">119.18 ± 0.58</td> <td align="char" char="±">51.1 ± 0.49</td> <td align="char" char="±">90.66 ± 1.10</td> </tr> <tr> <td align="left"><a href="#TF2"><sup>b</sup></a>Δ<i>G</i><b><sub>pol + elec</sub></b></td> <td align="char" char="±">11.57 ± 0.90</td> <td align="char" char="±">2.24 ± 0.87</td> <td align="char" char="±">0.84 ± 1.05</td> <td align="char" char="±">7.83 ± 0.80</td> <td align="char" char="±">−0.34 ± 1.68</td> </tr> <tr> <td align="left"><a href="#TF3"><sup>c</sup></a>Δ<i>E</i><b><sub>MM</sub></b></td> <td align="char" char="±">−130.25 ± 0.71</td> <td align="char" char="±">−94.49 ± 0.75</td> <td align="char" char="±">−161.45 ± 0.90</td> <td align="char" char="±">−77.84 ± 0.68</td> <td align="char" char="±">−117.83 ± 1.28</td> </tr> <tr> <td align="left">Δ<i>G</i><b><sub>bind</sub><sup>Sim</sup></b></td> <td align="char" char="±">−24.57 ± 0.91</td> <td align="char" char="±">−27.02 ± 0.89</td> <td align="char" char="±">−42.27 ± 1.07</td> <td align="char" char="±">−26.74 ± 0.83</td> <td align="char" char="±">−27.17 ± 1.69</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""><span class="label" tagx="label" title="label"><sup>a</sup></span><p>Δ<i>G</i><sub>solv</sub> = Δ<i>G</i><sub>np</sub> + Δ<i>G</i><sub>pol</sub>,</p> </div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label"><sup>b</sup></span><p>Δ<i>G</i><sub>pol + elec</sub> = Δ<i>E</i><sub>elec</sub> + Δ<i>G</i><sub>pol</sub>,</p> </div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label"><sup>c</sup></span><p>Δ<i>E</i><sub>MM</sub> = Δ<i>E</i><sub>vdW</sub> + Δ<i>E</i><sub>elec</sub>.</p> </div> </div> </div> <p>It is evident from <a href="#t0004">Table 4</a> that for all complexes, Δ<i>E</i><sub>vdW</sub> varies between −22.55 kcal/mol and −37.82 kcal/mol while Δ<i>E</i><sub>elec</sub> ranges from −47.18 kcal/mol to −123.63 kcal/mol. Furthermore, in the cases of RdRp/remdesivir, RdRp/EGCG, RdRp/TF3, and RdRp/TF2b, Δ<i>E</i><sub>ele</sub> is over-compensated by the desolvation energy (Δ<i>G</i><sub>pol</sub>), indicating that the sum of these two components, Δ<i>G</i><sub>pol + elec,</sub> is unfavorable to the binding and varies between 0.84 kcal/mol and 11.57 kcal/mol (see <a href="#t0004">Table 4</a>) and similar results are found for RdRp/myricetin, RdRp/quercetagetin, RdRp/hesperidin, and RdRp/TF1 (see <a href="#t0002">Table S2</a> in <a href="https://doi.org/10.1080/07391102.2020.1796810">Supplementary Information</a>). In contrast, in the case of RdRp/TF2a, Δ<i>G</i><sub>pol + elec</sub>, is favorable to the complexation (Δ<i>G</i><sub>pol + elec</sub> = −0.34 kcal/mol). Overall, this suggests that the complex formation is mainly driven by the van der Waals interactions between polyphenols as well as remdesivir and RdRp. Therefore, hydrophobic residues in the binding pocket played a crucial role in the complexation process.</p> <p>The estimated binding free energy (ΔG<sub>bind</sub>) of remdesivir, EGCG, TF3, TF2b, and TF2a were −24.57, −27.02, −42.27, −26.74 and −27.17 kcal/mol, respectively (<a href="#t0004">Table 4</a>) and myricetin, quercetagetin, hesperidin and TF1 show lower binding affinity compared to that of remdesivir (<a href="#t0002">Table S2</a> in the <a href="https://doi.org/10.1080/07391102.2020.1796810">Supplementary Information</a>). This suggests that polyphenol TF3 binds most strongly to RdRp, followed by TF2a and EGCG. The potency of the five inhibitors decreases in the following order: TF3 &amp;gt; TF2a &amp;gt; EGCG &amp;gt; TF2b &amp;gt; remdesivir. TF3 binds most strongly to RdRp because both Δ<i>E</i><sub>vdW</sub> and Δ<i>E</i><sub>elec</sub> are more favorable to the binding compared to the other inhibitors. Similarly, TF2a binds more strongly to RdRp compared to EGCG or TF2b because Δ<i>G</i><sub>pol + elec</sub> is favorable for TF2a (Δ<i>G</i><sub>pol + elec</sub> = −0.34 kcal/mol) while it is found to be unfavorable for EGCG (Δ<i>G</i><sub>pol + elec</sub> = 2.24 kcal/mol) and TF2b (Δ<i>G</i><sub>pol + elec</sub> = 7.83 kcal/mol).</p> </div> <div class="essentialresiduesforpolyphenolsbinding" title="sec"><span class="label" tagx="label" title="label">3.2.3.</span><div class="title" tagx="title" title="title">
Essential residues for polyphenols binding</div> <p>Further, to gain a deeper insight into the best four RdRp/polyphenols and remdesivir interaction pattern, the total binding free energy was decomposed into polyphenols-residue pair based on the MM-GBSA scheme. The approach of per-residue based contributions is useful to determine the binding mechanisms of an inhibitor at an atomistic level, and it also reveals the individual residue contributions. The different energy contributions from the backbone and side-chain of each residue are shown in <a href="#F0005">Figure 5</a> and listed in <a href="#t0005">Table 5</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5.</span></a></div>   <p>Decomposition of the binding free energy into contributions from individual residues for RdRp complexed with remdesivir, EGCG, TF3, TF2b and TF2a.</p>   </div> <div class="table-wrap_UNKNOWN" id="t0005" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 5.</span> <p>Per-residue based decomposition of binding free energy for the complex of remdesivir, EGCG, TF3, TF2a and TF2b with the SARS-CoV-2 RdRp.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th align="left">Residue</th> <th align="center">T<sub>vdW</sub></th> <th align="center">T<sub>elec</sub></th> <th align="center">T<sub>pol</sub></th> <th align="center">T<sub>np</sub></th> <th align="center">T<sub>side_chain</sub></th> <th align="center">T<sub>backbone</sub></th> <th align="center">T<sub>total</sub></th> </tr> </thead> <tbody valign="top"> <tr> <td colspan="8" align="center">RdRp/Remdesivir</td> </tr> <tr> <td align="left">Asp761</td> <td align="char" char=".">1.05</td> <td align="char" char=".">−21.73</td> <td align="char" char=".">18.81</td> <td align="char" char=".">−0.14</td> <td align="char" char=".">−1.82</td> <td align="char" char=".">−0.19</td> <td align="char" char=".">−2.01</td> </tr> <tr> <td align="left">Lys798</td> <td align="char" char=".">−2.74</td> <td align="char" char=".">−4.28</td> <td align="char" char=".">5.74</td> <td align="char" char=".">−0.47</td> <td align="char" char=".">−1.76</td> <td align="char" char=".">0.01</td> <td align="char" char=".">−1.75</td> </tr> <tr> <td align="left">Pro620</td> <td align="char" char=".">−1.42</td> <td align="char" char=".">−1.15</td> <td align="char" char=".">1.16</td> <td align="char" char=".">−0.18</td> <td align="char" char=".">−1.26</td> <td align="char" char=".">−0.33</td> <td align="char" char=".">−1.59</td> </tr> <tr> <td align="left">Asp760</td> <td align="char" char=".">−0.57</td> <td align="char" char=".">−4.27</td> <td align="char" char=".">4.24</td> <td align="char" char=".">−0.12</td> <td align="char" char=".">−0.20</td> <td align="char" char=".">−0.52</td> <td align="char" char=".">−0.72</td> </tr> <tr> <td align="left">Arg553</td> <td align="char" char=".">−2.07</td> <td align="char" char=".">−3.71</td> <td align="char" char=".">5.61</td> <td align="char" char=".">−0.51</td> <td align="char" char=".">−0.76</td> <td align="char" char=".">0.08</td> <td align="char" char=".">−0.68</td> </tr> <tr> <td colspan="8" align="center">RdRp/EGCG</td> </tr> <tr> <td align="left">Asp452</td> <td align="char" char=".">1.65</td> <td align="char" char=".">−16.48</td> <td align="char" char=".">9.85</td> <td align="char" char=".">−0.08</td> <td align="char" char=".">−5.15</td> <td align="char" char=".">0.09</td> <td align="char" char=".">−5.06</td> </tr> <tr> <td align="left">Arg553</td> <td align="char" char=".">−3.83</td> <td align="char" char=".">−4.84</td> <td align="char" char=".">6.59</td> <td align="char" char=".">−0.59</td> <td align="char" char=".">−2.68</td> <td align="char" char=".">0.01</td> <td align="char" char=".">−2.67</td> </tr> <tr> <td align="left">Pro620</td> <td align="char" char=".">−1.64</td> <td align="char" char=".">−0.41</td> <td align="char" char=".">0.51</td> <td align="char" char=".">−0.33</td> <td align="char" char=".">−1.62</td> <td align="char" char=".">−0.25</td> <td align="char" char=".">−1.87</td> </tr> <tr> <td align="left">Asp618</td> <td align="char" char=".">0.49</td> <td align="char" char=".">−8.14</td> <td align="char" char=".">6.66</td> <td align="char" char=".">−0.09</td> <td align="char" char=".">−1.13</td> <td align="char" char=".">−0.05</td> <td align="char" char=".">−1.08</td> </tr> <tr> <td align="left">Lys621</td> <td align="char" char=".">−2.26</td> <td align="char" char=".">−5.12</td> <td align="char" char=".">6.92</td> <td align="char" char=".">−0.50</td> <td align="char" char=".">−0.87</td> <td align="char" char=".">−0.09</td> <td align="char" char=".">−0.96</td> </tr> <tr> <td colspan="8" align="center">RdRp/TF3</td> </tr> <tr> <td align="left">Asp761</td> <td align="char" char=".">2.52</td> <td align="char" char=".">−30.13</td> <td align="char" char=".">22.11</td> <td align="char" char=".">−0.25</td> <td align="char" char=".">−5.54</td> <td align="char" char=".">−0.21</td> <td align="char" char=".">−5.75</td> </tr> <tr> <td align="left">Arg836</td> <td align="char" char=".">−0.83</td> <td align="char" char=".">−14.95</td> <td align="char" char=".">12.29</td> <td align="char" char=".">−0.37</td> <td align="char" char=".">−3.85</td> <td align="char" char=".">−0.01</td> <td align="char" char=".">−3.86</td> </tr> <tr> <td align="left">Arg555</td> <td align="char" char=".">−5.80</td> <td align="char" char=".">−4.00</td> <td align="char" char=".">7.29</td> <td align="char" char=".">−0.70</td> <td align="char" char=".">−2.98</td> <td align="char" char=".">−0.23</td> <td align="char" char=".">−3.21</td> </tr> <tr> <td align="left">Thr556</td> <td align="char" char=".">−0.20</td> <td align="char" char=".">−3.96</td> <td align="char" char=".">2.44</td> <td align="char" char=".">−0.13</td> <td align="char" char=".">−0.21</td> <td align="char" char=".">−1.64</td> <td align="char" char=".">−1.85</td> </tr> <tr> <td align="left">Ile548</td> <td align="char" char=".">−1.03</td> <td align="char" char=".">−0.08</td> <td align="char" char=".">0.12</td> <td align="char" char=".">−0.13</td> <td align="char" char=".">−0.65</td> <td align="char" char=".">−0.47</td> <td align="char" char=".">−1.12</td> </tr> <tr> <td align="left">Ser814</td> <td align="char" char=".">−1.67</td> <td align="char" char=".">0.09</td> <td align="char" char=".">0.78</td> <td align="char" char=".">−0.08</td> <td align="char" char=".">−0.48</td> <td align="char" char=".">−0.40</td> <td align="char" char=".">−0.88</td> </tr> <tr> <td align="left">Val557</td> <td align="char" char=".">−0.65</td> <td align="char" char=".">−0.16</td> <td align="char" char=".">0.17</td> <td align="char" char=".">−0.24</td> <td align="char" char=".">−0.65</td> <td align="char" char=".">−0.23</td> <td align="char" char=".">−0.88</td> </tr> <tr> <td colspan="8" align="center">RdRp/TF2b</td> </tr> <tr> <td align="left">His816</td> <td align="char" char=".">−3.00</td> <td align="char" char=".">−0.81</td> <td align="char" char=".">1.69</td> <td align="char" char=".">−0.32</td> <td align="char" char=".">−1.63</td> <td align="char" char=".">−0.81</td> <td align="char" char=".">−2.44</td> </tr> <tr> <td align="left">Asp833</td> <td align="char" char=".">−1.29</td> <td align="char" char=".">−0.23</td> <td align="char" char=".">0.46</td> <td align="char" char=".">−0.18</td> <td align="char" char=".">−0.21</td> <td align="char" char=".">−1.03</td> <td align="char" char=".">−1.24</td> </tr> <tr> <td align="left">Tyr877</td> <td align="char" char=".">−0.92</td> <td align="char" char=".">−2.06</td> <td align="char" char=".">2.06</td> <td align="char" char=".">−0.27</td> <td align="char" char=".">−0.43</td> <td align="char" char=".">−0.76</td> <td align="char" char=".">−1.19</td> </tr> <tr> <td align="left">Glu811</td> <td align="char" char=".">0.22</td> <td align="char" char=".">−9.24</td> <td align="char" char=".">8.11</td> <td align="char" char=".">−0.13</td> <td align="char" char=".">−2.07</td> <td align="char" char=".">1.03</td> <td align="char" char=".">−1.04</td> </tr> <tr> <td align="left">His810</td> <td align="char" char=".">−1.88</td> <td align="char" char=".">−0.41</td> <td align="char" char=".">1.66</td> <td align="char" char=".">−0.34</td> <td align="char" char=".">−0.10</td> <td align="char" char=".">−0.85</td> <td align="char" char=".">−0.95</td> </tr> <tr> <td align="left">Tyr831</td> <td align="char" char=".">−1.51</td> <td align="char" char=".">−0.36</td> <td align="char" char=".">1.19</td> <td align="char" char=".">−0.13</td> <td align="char" char=".">−0.21</td> <td align="char" char=".">−0.60</td> <td align="char" char=".">−0.81</td> </tr> <tr> <td align="left">Asn815</td> <td align="char" char=".">−0.60</td> <td align="char" char=".">−0.27</td> <td align="char" char=".">0.09</td> <td align="char" char=".">−0.01</td> <td align="char" char=".">−0.25</td> <td align="char" char=".">−0.54</td> <td align="char" char=".">−0.79</td> </tr> <tr> <td colspan="8" align="center">RdRp/TF2a</td> </tr> <tr> <td align="left">Asp618</td> <td align="char" char=".">2.63</td> <td align="char" char=".">−20.71</td> <td align="char" char=".">13.40</td> <td align="char" char=".">−0.17</td> <td align="char" char=".">−4.84</td> <td align="char" char=".">−0.01</td> <td align="char" char=".">−4.85</td> </tr> <tr> <td align="left">Arg553</td> <td align="char" char=".">−3.80</td> <td align="char" char=".">−6.52</td> <td align="char" char=".">7.55</td> <td align="char" char=".">−0.69</td> <td align="char" char=".">−2.68</td> <td align="char" char=".">−0.78</td> <td align="char" char=".">−3.46</td> </tr> <tr> <td align="left">Lys551</td> <td align="char" char=".">−2.23</td> <td align="char" char=".">−0.80</td> <td align="char" char=".">2.23</td> <td align="char" char=".">−0.45</td> <td align="char" char=".">−0.84</td> <td align="char" char=".">−0.41</td> <td align="char" char=".">−1.25</td> </tr> <tr> <td align="left">Arg555</td> <td align="char" char=".">−1.33</td> <td align="char" char=".">−0.62</td> <td align="char" char=".">1.05</td> <td align="char" char=".">−0.27</td> <td align="char" char=".">−1.12</td> <td align="char" char=".">−0.05</td> <td align="char" char=".">−1.17</td> </tr> <tr> <td align="left">Glu167</td> <td align="char" char=".">0.42</td> <td align="char" char=".">−6.82</td> <td align="char" char=".">5.92</td> <td align="char" char=".">−0.12</td> <td align="char" char=".">−0.61</td> <td align="char" char=".">0.01</td> <td align="char" char=".">−0.60</td> </tr> </tbody> </table> </div> <p>As shown in <a href="#F0005">Figure 5</a>, it was observed that residues favoring the binding of the polyphenols with RdRp include Asp452, Arg553, Arg555, Val557, Asp618, Pro620, Lys621, Asp623, Arg624, Asp760, Asp761, and Glu811, Asp833, and Arg836. Most of these residues are located in the binding site of RdRp and can form direct contacts with polyphenols and remdesivir. <a href="#F0005">Figure 5</a> shows that amino acids Pro620, Asp761 and Lys798 for RdRp/remdesivir; Asp452, Arg553, Pro620 and Lys621 for RdRp/EGCG; Ile548, Arg555, Thr556, Asp761 and Arg836 for RdRp/TF3; Glu811, His816, Asp833 and Tyr877 for RdRp/TF2b; Lys551, Arg553, Arg555 and Asp618 for RdRp/TF2a contributed more favorably towards the binding by contributing more than −1.0 kcal/mol in size.</p> <p>To complement the energetic analysis, we performed MD trajectory-based hydrogen bond (h-bond) analysis for all five complexes, and the h-bonds with occupancy are listed in <a href="#t0006">Table 6</a>. The h-bonds were determined by setting the acceptor-donor distance of ≤ 3.5 Å, and the angle cut off ≥ 120<sup>0</sup>. Important h-bonds between RdRp-inhibitors are shown in <a href="#F0006">Figure 6</a>. In the case of RdRp/remdesivir, key residues involved in the hydrogen bonding are Asp761, Asp760, and Ser759, respectively. Asp760 is found to form two h-bonds with remdesivir (Asp760@OD2 - Lig@O7, Asp760@OD2 - Lig@O6) with an occupancy of more than 15% (see <a href="#t0006">Table 6</a> and <a href="#F0006">Figure 6</a>). In the case of RdRp/EGCG, both Asp618 and Asp760 form two h-bonds with the ligand with an occupancy in the range of 16.09 to 30.17%. On the other hand, Asp761 form an h-bond with TF3 (Asp761@OD1 - Lig@O11) with an occupancy of 69.84%, while Arg836 forms two h-bonds with the ligand (Arg836@NH2 - Lig@O14, Arg836@NE - Lig@O14) with an occupancy of 52.66%, and 48.70%, respectively. Glu811, Thr556 and Asp761 also formed h-bonds with the ligand during our simulations with an occupancy varying in the range of 44% to 58% (see <a href="#t0006">Table 6</a>). In the case of RdRp/TF2b, Glu811 is found to form two strong h–bonds with the ligand (Glu811@OE1 – Lig@O7, Glu811@OE2 – Lig@O7) with an occupancy of 22.45% and 18.89%, respectively. On the other hand, it can be observed from <a href="#t0006">Table 6</a> that Pro832 and Tyr877 form strong h-bonds (Pro832@O -Lig@O8 and Lig@O10 -Tyr877@OH) with increased occupancy (&amp;gt; 24%). Finally, in the case of RdRp/TF2a, Asp618 is found to form two strong h-bonds with the TF2a (Asp618@OD1 – Lig@O10, Asp618@OD1 – Lig@O11) with an occupancy of 38.68% and 38.38%, respectively. Asp760 also forms a h-bond (Asp760@O – Lig@O11) with an occupancy of 20.83%.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6.</span></a></div>   <p>Five main hydrogen bond interactions between ligands and RdRp.</p>   </div> <div class="table-wrap_UNKNOWN" id="t0006" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 6.</span> <p>Main hydrogen bond interactions formed by RdRp with remdesivir and polyphenols along with the corresponding average distance and percentage of occupancy determined using the trajectories of production simulations.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th align="left">Acceptor</th> <th align="center">Donor</th> <th align="center">Avg. Distance (Å)</th> <th align="center">Occupancy (%)</th> </tr> </thead> <tbody valign="top"> <tr> <td colspan="4" align="center">RdRp/Remdesivir</td> </tr> <tr> <td align="left">Asp760@OD2</td> <td align="left">Lig@O7</td> <td align="char" char=".">2.66</td> <td align="char" char=".">19.46</td> </tr> <tr> <td align="left">Asp761@OD1</td> <td align="left">Lig@O7</td> <td align="char" char=".">2.65</td> <td align="char" char=".">17.70</td> </tr> <tr> <td align="left">Asp761@OD2</td> <td align="left">Lig@O6</td> <td align="char" char=".">2.63</td> <td align="char" char=".">16.86</td> </tr> <tr> <td align="left">Asp760@OD2</td> <td align="left">Lig@O6</td> <td align="char" char=".">2.65</td> <td align="char" char=".">16.65</td> </tr> <tr> <td align="left">Lig@O6</td> <td align="left">Ser759@OG</td> <td align="char" char=".">2.80</td> <td align="char" char=".">11.63</td> </tr> <tr> <td align="left">Asp760@OD1</td> <td align="left">Lig@O7</td> <td align="char" char=".">2.66</td> <td align="char" char=".">10.59</td> </tr> <tr> <td colspan="4" align="center">RdRp/EGCG</td> </tr> <tr> <td align="left">Asp618@OD1</td> <td align="left">Lig@O5</td> <td align="char" char=".">2.61</td> <td align="char" char=".">30.13</td> </tr> <tr> <td align="left">Asp618@OD1</td> <td align="left">Lig@O6</td> <td align="char" char=".">2.61</td> <td align="char" char=".">29.28</td> </tr> <tr> <td align="left">Asp618@OD2</td> <td align="left">Lig@O5</td> <td align="char" char=".">2.61</td> <td align="char" char=".">18.25</td> </tr> <tr> <td align="left">Asp618@OD2</td> <td align="left">Lig@O6</td> <td align="char" char=".">2.61</td> <td align="char" char=".">17.38</td> </tr> <tr> <td align="left">Asp760@OD1</td> <td align="left">Lig@O5</td> <td align="char" char=".">2.63</td> <td align="char" char=".">16.09</td> </tr> <tr> <td align="left">Tyr455@OH</td> <td align="left">Lig@O11</td> <td align="char" char=".">2.83</td> <td align="char" char=".">10.97</td> </tr> <tr> <td colspan="4" align="center">RdRp/TF3</td> </tr> <tr> <td align="left">Asp761@OD1</td> <td align="left">Lig@O11</td> <td align="char" char=".">2.61</td> <td align="char" char=".">69.84</td> </tr> <tr> <td align="left">Glu811@O</td> <td align="left">Lig@O10</td> <td align="char" char=".">2.76</td> <td align="char" char=".">58.43</td> </tr> <tr> <td align="left">Thr556@O</td> <td align="left">Lig@O3</td> <td align="char" char=".">2.72</td> <td align="char" char=".">56.95</td> </tr> <tr> <td align="left">Lig@O14</td> <td align="left">Arg836@NH2</td> <td align="char" char=".">2.83</td> <td align="char" char=".">52.66</td> </tr> <tr> <td align="left">Lig@O14</td> <td align="left">Arg836@NE</td> <td align="char" char=".">2.86</td> <td align="char" char=".">48.70</td> </tr> <tr> <td align="left">Asp761@OD2</td> <td align="left">Lig@O20</td> <td align="char" char=".">2.62</td> <td align="char" char=".">44.51</td> </tr> <tr> <td colspan="4" align="center">RdRp/TF2b</td> </tr> <tr> <td align="left">Pro832@O</td> <td align="left">Lig@O8</td> <td align="char" char=".">2.77</td> <td align="char" char=".">26.31</td> </tr> <tr> <td align="left">Lig@O11</td> <td align="left">Tyr877@OH</td> <td align="char" char=".">2.75</td> <td align="char" char=".">24.39</td> </tr> <tr> <td align="left">Glu811@OE1</td> <td align="left">Lig@O7</td> <td align="char" char=".">2.65</td> <td align="char" char=".">22.45</td> </tr> <tr> <td align="left">Glu811@OE2</td> <td align="left">Lig@O7</td> <td align="char" char=".">2.65</td> <td align="char" char=".">18.89</td> </tr> <tr> <td align="left">Asp833@OD2</td> <td align="left">Lig@O8</td> <td align="char" char=".">2.65</td> <td align="char" char=".">12.01</td> </tr> <tr> <td align="left">Asn874@OD1</td> <td align="left">Lig@O11</td> <td align="char" char=".">2.68</td> <td align="char" char=".">8.58</td> </tr> <tr> <td colspan="4" align="center">RdRp<b>/</b>TF2a</td> </tr> <tr> <td align="left">Asp618@OD1</td> <td align="left">Lig@O10</td> <td align="char" char=".">2.59</td> <td align="char" char=".">38.68</td> </tr> <tr> <td align="left">Asp618@OD1</td> <td align="left">Lig@O11</td> <td align="char" char=".">2.62</td> <td align="char" char=".">38.38</td> </tr> <tr> <td align="left">Asp760@O</td> <td align="left">Lig@O11</td> <td align="char" char=".">2.70</td> <td align="char" char=".">20.83</td> </tr> <tr> <td align="left">Asp618@OD2</td> <td align="left">Lig@O11</td> <td align="char" char=".">2.62</td> <td align="char" char=".">16.59</td> </tr> <tr> <td align="left">Asp618@OD2</td> <td align="left">Lig@O10</td> <td align="char" char=".">2.58</td> <td align="char" char=".">16.49</td> </tr> <tr> <td align="left">Asp618@OD1</td> <td align="left">Lig@O15</td> <td align="char" char=".">2.67</td> <td align="char" char=".">16.37</td> </tr> </tbody> </table> </div> <p>Finally, we supplemented the above results by analyzing the final conformation of each production simulation with the help of 2D LigPlot<sup>+</sup> software, and different h-bonds and hydrophobic interactions were shown in <a href="#F0007">Figure 7</a>. Hydrogen bonds are depicted in green dotted lines, while red semicircle residues are involved in hydrophobic interactions. For the RdRp/remdesivir complex, <a href="#F0007">Figure 7(A)</a> displayed nine hydrophobic interactions with Lys545, Ala547, Ser549, Arg553, Val557, Asp684, Ser759, Ser814, and Arg836. This large number of interactions account for the high stability and good binding affinity of remdesivir to RdRp. EGCG formed hydrophobic interactions with Lys551, Ala554, Arg553, Arg624, Pro620, (<a href="#F0007">Figure 7(B)</a>). In the case of TF3, eight hydrophobic interactions with His439, Ile548, Ser814, Phe812, Val557, Ser549, Tyr619 and Arg555 were formed as revealed by <a href="#F0007">Figure 7(C)</a>. <a href="#F0007">Figure 7(D)</a> shows that seven hydrophobic interactions with Asp833, His816, Pro832, Gln815, His872, His810 and Ser434 were formed for RdRp/TF2b. Finally, <a href="#F0007">Figure 7(E)</a> shows that RdRp/TF2a formed hydrophobic interactions with Arg555, Ala554 and Lys551. Overall, TF3 has a higher binding affinity toward RdRp compared to the other polyphenols due to a larger number of stable hydrogen bonds and hydrophobic interactions.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 7.</span></a></div>   <p>The RdRp-ligands interaction profile for <b>(A)</b> RdRp/remdesivir, <b>(B)</b> RdRp/EGCG, <b>(C)</b> RdRp/TF3, <b>(D)</b> RdRp/TF2b and <b>(E)</b> RdRp/TF2a. The polyphenols and remdesivir are shown in balls and sticks. Hydrogen bonds are depicted in green dotted lines, and red semicircles residues are involved in hydrophobic interactions.</p>   </div> </div> </div> <div class="predictionoftheabsorption,distribution,metabolism,excretion,andtoxicity(admet)profile" title="sec"><span class="label" tagx="label" title="label">3.3.</span><div class="title" tagx="title" title="title">
Prediction of the absorption, distribution, metabolism, excretion, and toxicity (ADMET) profile</div> <p>In addition to testing the physiochemical efficiency of a given molecule to inhibit the target protein, other parameters such as absorption, distribution, metabolism, excretion, and toxicity (ADMET) of the inhibitor play a critical role on demonstrating the likelihood of success of a drug. Utilization of <i>in-silico</i> ADMET profiling, in combination with <i>in vivo</i> and <i>in vitro</i> predictions in the initial stage of the screening process, can significantly fasten the drug discovery process by minimizing the number of potential safety problems. Hence, we performed a detailed ADMET profiling to evaluate the drug likeliness of the four polyphenols: EGCG, TF2a, TF2b, TF3 that exhibited the highest score from the MD simulation and MM-PBSA study along with the positive control remdesivir.</p> <p>Human colon adenocarcinoma-2 cell line (Caco2) permeability and human intestinal absorption (HIA) are key parameters to decide the total bioavailability of a drug. All the five compounds (EGCG, TF2a, TF2b, TF3, and remdesivir) showed comparatively low Caco2 permeability potential (&amp;lt;8 × 10<sup>−6 </sup>cm/s) and could be absorbed <i>via</i> the human intestine (Larregieu &amp;amp; Benet, <a href="#CIT0048">2013</a>). EGCG, TF2a, TF2b, TF3, and remdesivir were predicted to be substrates of permeability glycoprotein (P-glycoprotein), which is an efflux membrane protein. However, remdesivir was predicted as a P-glycoprotein I inhibitor, and EGCG and TF3 as a P-glycoprotein II inhibitor, whereas TF2a and TF2b as both P-glycoprotein I and II inhibitor. Hence, above mentioned five compounds could regulate the physiological functions of P-glycoprotein (see <a href="#t0003">Table S3</a> in the <a href="https://doi.org/10.1080/07391102.2020.1796810">Supplementary Information</a>).</p> <p>The distribution of a drug is regulated by many parameters such as lipid-solubility, concentration in plasma and binding ability to plasma proteins, transport proteins, etc. The volume of distribution at steady-state (VDss) suggests that EGCG, TF2a, TF2b, TF3, and remdesivir had a lower theoretical dose required for uniform distribution in the plasma. Further, the degree of diffusion across the plasma membrane increases in the following order remdesivir &amp;lt; EGCG &amp;lt; TF2a &amp;lt; TF2b &amp;lt; TF3 (<a href="#t0004">Table S4</a> in the <a href="https://doi.org/10.1080/07391102.2020.1796810">Supplementary Information</a>) as measured by the fraction that is in the unbound state. The predictions through the distribution of the drugs <i>via</i> the central nervous system and blood-brain barrier suggest that these five compounds are poorly distributed to the brain and unable to penetrate the central nervous system. However, the medium level of the lipophilicity of the drugs suggests that they would have no negative impact on nervous system exposure.</p> <p>Cytochromes P450 (CYP) isozymes play crucial roles in drug metabolism. It has been observed that TF2a, TF3, and remdesivir are a substrate of CYP3A4 and hence, can be efficiently metabolized by CYP3A4. On the other hand, EGCG is a CYP3A4 inhibitor (<a href="#t0005">Table S5</a> in the <a href="https://doi.org/10.1080/07391102.2020.1796810">Supplementary Information</a>). On a separate note, EGCG is predominantly metabolized in the small intestine and liver by the conjugate formation of glucuronide, methyl sulfates in the urine and plasma (Chow et al. <a href="#CIT0012">2005</a>).</p> <p>Among the five compounds, TF2b and TF3 were predicted as the substrate of renal organic cation transporter-2 (Renal OCT2), as shown in <a href="#t0006">Table S6</a> in the <a href="https://doi.org/10.1080/07391102.2020.1796810">Supplementary Information</a>. While EGCG, TF2a, and remdesivir are possibly cleared through other available routes such as bile, breath, faces, and sweat. EGCG remains intact in the plasma and later excreted <i>via</i> bile and metabolized by colon microflora. It is also expected that all the compounds are absorbable <i>via</i> oral prescription.</p> <p>We have also analyzed the toxicity profiles for EGCG, TF2a, TF2b, TF3 as well as remdesivir (see <a href="#t0007">Table 7</a>). The toxicity prediction from the AMES test (<i>Salmonella typhimurium</i> reverse mutation assay) exhibited that all the compounds could be considered as non-mutagenic agents. High toxicity was observed for all the compounds in <i>Tetrahymena pyriformis</i>. EGCG, TF2a, TF2b, TF3, and remdesivir were shown to inhibit the human ether-a-go-go-related gene II (hERG II). However, Remdesivir has been shown to induce hepatotoxicity, whilst EGCG, TF2a, TF2b, TF3 are not likely to be associated with hepatotoxicity. The maximum recommended tolerated dose (MRTD) in human prediction shows that remdesivir violate MRTD whereas natural polyphenol EGCG, TF2a, TF2b, TF3 do not fall into this category. Remdesivir does not possess high acute toxicity whereas EGCG, TF2a, TF2b, and TF3 regarded as high acute toxic compound as it falls under minnow toxicity. Additionally, none of the compounds predicted to be associated with skin sensitization.</p> <div class="table-wrap_UNKNOWN" id="t0007" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 7.</span> <p>Predicted toxicity profile of EGCG, TF3, TF2b, TF2a, and remdesivir.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th align="left">S. No.</th> <th align="center">Compounds name</th> <th align="center">Toxicity prediction</th> <th align="center"> </th> </tr> <tr> <th align="left"> </th> <th align="center"> </th> <th align="center">Properties</th> <th align="center">Predicted values</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">1</td> <td align="left">EGCG</td> <td align="left">AMES toxicity</td> <td align="left">No</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">Maximum tolerated dose (Human)</td> <td align="left">0.441 (log mg/kg/day)</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">hERG I inhibitor</td> <td align="left">No</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">hERG II inhibitor</td> <td align="left">Yes</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">Oral rat acute toxicity (LD<sub>50</sub>)</td> <td align="left">2.522 (mol/kg)</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">Oral rat chronic toxicity (LOAEL)</td> <td align="left">3.065 (log mg/kg_bw/day)</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">Hepatotoxicity</td> <td align="left">No</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">Skin sensitivity</td> <td align="left">No</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left"><i>T. pyriformis</i> toxicity</td> <td align="left">0.285 (µg/L)</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">Minnow toxicity</td> <td align="left">7.713 log mM</td> </tr> <tr> <td align="left">2</td> <td align="left">TF3</td> <td align="left">AMES toxicity</td> <td align="left">No</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">Maximum tolerated dose (Human)</td> <td align="left">0.438 (log mg/kg/day)</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">hERG I inhibitor</td> <td align="left">No</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">hERG II inhibitor</td> <td align="left">Yes</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">Oral rat acute toxicity (LD<sub>50</sub>)</td> <td align="left">2.482 (mol/kg)</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">Oral rat chronic toxicity (LOAEL)</td> <td align="left">7.443 (log mg/kg_bw/day)</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">Hepatotoxicity</td> <td align="left">No</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">Skin sensitivity</td> <td align="left">No</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left"><i>T. pyriformis</i> toxicity</td> <td align="left">0.285 (µg/L)</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">Minnow toxicity</td> <td align="left">9.738 log mM</td> </tr> <tr> <td align="left">3</td> <td align="left">TF2b</td> <td align="left">AMES toxicity</td> <td align="left">No</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">Maximum tolerated dose (Human)</td> <td align="left">0.438 (log mg/kg/day)</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">hERG I inhibitor</td> <td align="left">No</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">hERG II inhibitor</td> <td align="left">Yes</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">Oral rat acute toxicity (LD<sub>50</sub>)</td> <td align="left">2.482 (mol/kg)</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">Oral rat chronic toxicity (LOAEL)</td> <td align="left">5.322 (log mg/kg_bw/day)</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">Hepatotoxicity</td> <td align="left">No</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">Skin sensitivity</td> <td align="left">No</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left"><i>T. pyriformis</i> toxicity</td> <td align="left">0.285 (µg/L)</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">Minnow toxicity</td> <td align="left">8.685 log mM</td> </tr> <tr> <td align="left">4</td> <td align="left">TF2a</td> <td align="left">AMES toxicity</td> <td align="left">No</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">Maximum tolerated dose (Human)</td> <td align="left">0.439 (log mg/kg/day)</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">hERG I inhibitor</td> <td align="left">No</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">hERG II inhibitor</td> <td align="left">Yes</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">Oral rat acute toxicity (LD<sub>50</sub>)</td> <td align="left">2.484 (mol/kg)</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">Oral rat chronic toxicity (LOAEL)</td> <td align="left">5.035 (log mg/kg_bw/day)</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">Hepatotoxicity</td> <td align="left">No</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">Skin sensitivity</td> <td align="left">No</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left"><i>T. pyriformis</i> toxicity</td> <td align="left">0.285 (µg/L)</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">Minnow toxicity</td> <td align="left">4.898 log mM</td> </tr> <tr> <td align="left">5</td> <td align="left">Remdesivir</td> <td align="left">AMES toxicity</td> <td align="left">No</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">Maximum tolerated dose (Human)</td> <td align="left">0.15 (log mg/kg/day)</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">hERG I inhibitor</td> <td align="left">No</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">hERG II inhibitor</td> <td align="left">Yes</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">Oral rat acute toxicity (LD<sub>50</sub>)</td> <td align="left">2.043 (mol/kg)</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">Oral rat chronic toxicity (LOAEL)</td> <td align="left">1.639 (log mg/kg_bw/day)</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">Hepatotoxicity</td> <td align="left">Yes</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">Skin sensitivity</td> <td align="left">No</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left"><i>T. pyriformis</i> toxicity</td> <td align="left">0.285 (µg/L)</td> </tr> <tr> <td align="left"> </td> <td align="left"> </td> <td align="left">Minnow toxicity</td> <td align="left">0.291 log mM</td> </tr> </tbody> </table> </div> </div> <div class="identificationoftargetclassfornaturalpolyphenolviatargetpredictionstudies" title="sec"><span class="label" tagx="label" title="label">3.4.</span><div class="title" tagx="title" title="title">
Identification of target class for natural polyphenol via target prediction studies</div> <p>The polyphenolic structural motif of dietary polyphenols allows them to serve as excellent hydrogen bond donors which in turn help them to strongly interact with various biomacromolecules such as proteins. This interaction is a critical step in the regulatory role of polyphenols on various key proteins involved in cellular physiology. The majority, if not all, of the beneficial effect of polyphenols, can be explained via the functional consequence of proteins it interacts with. Molecular target studies help us to predict therapeutic protein targets for a given small molecule. Herein, we analyzed the predicted interacting proteins/enzymes for EGCG, TF2a, TF2b, TF3, and remdesivir. This study is particularly important in the current context as we think that these polyphenols could target RdRp, an important enzyme that catalyzes the RNA replication in the SARS-CoV-2. Notably, the molecular target analysis suggests that all four polyphenols possess excellent properties of druggability, and they interact with a diverse class of proteins/enzymes. The top 25 target classes of EGCG, TF2a, TF2b, TF3, and remdesivir are represented in the pie-chart, as shown in <a href="#F0008">Figure 8</a>. The detailed output table with the target, common name, UniProt ID, ChEMBL ID, target class, probability, and known actives in 2D/3D are listed in Tables S7–S11 in the <a href="https://doi.org/10.1080/07391102.2020.1796810">Supplementary Information</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 8.</span></a></div>   <p>Molecular target predictions for <b>(A)</b> EGCG, <b>(B)</b> TF2a, <b>(C)</b> TF2b <b>(D)</b> TF3, and <b>(E)</b> remdesivir obtained from swiss target prediction report. The frequency of the target classes (top 25) is depicted in the pie chart.</p>   </div> </div> </div> <div class="conclusions" title="sec"><span class="label" tagx="label" title="label">4.</span><div class="title" tagx="title" title="title">
Conclusions</div> <p>Keeping in mind that RdRp inhibitors play a crucial role to combat the SARS-CoV-2 infection, in this work, we performed a comprehensive molecular docking study with a library of hundred natural polyphenols with potential antiviral properties that may inhibit the SARS-CoV-2 RdRp and prevent the RNA replication. We shortlisted eight natural polyphenols having binding energy −7.0 kcal/mol or less for molecular dynamics simulation. Further, we performed 150 ns molecular dynamics simulation of RdRp/EGCG, RdRp/TF1, RdRp/TF2a, RdRp/TF2b, RdRp/TF3, RdRp/hesperidin, RdRp/myricetin, RdRp/quercetagetin, along with RdRp/remdesivir complex and computed the binding energies by the molecular mechanics-Poisson-Boltzmann surface area (MM-PBSA) scheme from last 50 ns trajectories. Our study suggests that the complex formation of the SARS-CoV-2 RdRp and eight natural polyphenols is favoured by the intermolecular van der Waals and electrostatic interactions as well as nonpolar solvation free energy. We have also investigated the hotspot residues controlling the receptor-ligand binding. Finally, molecular dynamics simulation and MM-PBSA study reveals that EGCG, TF2a, TF2b, and TF3 possess a better binding affinity than the control drug remdesivir against the SARS-CoV-2 RdRp. Further, we also looked at the ADME prediction, toxicity prediction, and target analysis to assess their druggability of the five compounds. The obtained results strongly suggest that EGCG, TF2a, TF2b, and TF3 have a stable binding affinity towards RdRp of the SARS-CoV-2 with favourable pharmacokinetic properties. These bioactive compounds exhibit broad ranges of therapeutic properties. Therefore, we believe that these four natural polyphenols can act as potential inhibitors for the SARS-CoV-2 RdRp. However, further <i>in vitro</i> and <i>in vivo</i> studies need to be carried out to validate their efficacy against SARS-CoV-2 infection.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplemental Material</div>  <div class="media" title="media"><a href="TBSD_A_1796810_SM0875.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="COI-statement" title="COI-statement"> <div class="title" tagx="title" title="title">
Disclosure statement</div> <p>The authors declare that there is no conflict of interest.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CIT0001" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aanouz</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belhassan</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">El Khatabi</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lakhlifi</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">El Idrissi</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bouachrine</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Moroccan medicinal plants as inhibitors of COVID-19: Computational investigations</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1758790">10.1080/07391102.2020.1758790</a></span></span></li> <li tag="ref"><a name="CIT0002" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adeoye</span><span class="given-names" tagx="given-names" title="given-names">A. O.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oso</span><span class="given-names" tagx="given-names" title="given-names">B. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olaoye</span><span class="given-names" tagx="given-names" title="given-names">I. F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tijjani</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adebayo</span><span class="given-names" tagx="given-names" title="given-names">A. I.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Repurposing of chloroquine and some clinically approved antiviral drugs as effective therapeutics to prevent cellular entry and replication of coronavirus</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1765876">10.1080/07391102.2020.1765876</a></span></span></li> <li tag="ref"><a name="CIT0003" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahmed-Belkacem</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guichou</span><span class="given-names" tagx="given-names" title="given-names">J.-F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brillet</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahnou</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hernandez</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pallier</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pawlotsky</span><span class="given-names" tagx="given-names" title="given-names">J.-M.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Inhibition of RNA binding to hepatitis C virus RNA-dependent RNA polymerase: A new mechanism for antiviral intervention</span>. <span class="source" tagx="source" title="source"><i>Nucleic Acids Research</i></span>, (<span class="issue" tagx="issue" title="issue">14</span>), <span class="fpage" tagx="fpage" title="fpage">9399</span>–<span class="lpage" tagx="lpage" title="lpage">9409</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gku632">10.1093/nar/gku632</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25053847">25053847</a></span></span></li> <li tag="ref"><a name="CIT0004" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berendsen</span><span class="given-names" tagx="given-names" title="given-names">H. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Postma</span><span class="given-names" tagx="given-names" title="given-names">J. v.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Gunsteren</span><span class="given-names" tagx="given-names" title="given-names">W. F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DiNola</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haak</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span> (<span class="year" tagx="year" title="year">1984</span>). <span class="mixed-article-title" title="mixed-article-title">Molecular dynamics with coupling to an external bath</span>. <span class="source" tagx="source" title="source"><i>The Journal of Chemical Physics</i></span>, (<span class="issue" tagx="issue" title="issue">8</span>), <span class="fpage" tagx="fpage" title="fpage">3684</span>–<span class="lpage" tagx="lpage" title="lpage">3690</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1063/1.448118">10.1063/1.448118</a></span></span></li> <li tag="ref"><a name="CIT0005" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berman</span><span class="given-names" tagx="given-names" title="given-names">H. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhat</span><span class="given-names" tagx="given-names" title="given-names">T. N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bourne</span><span class="given-names" tagx="given-names" title="given-names">P. E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gilliland</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weissig</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Westbrook</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span> (<span class="year" tagx="year" title="year">2000</span>). <span class="mixed-article-title" title="mixed-article-title">The Protein Data Bank and the challenge of structural genomics</span>. <span class="source" tagx="source" title="source"><i>Nucleic Acids Research</i></span>, , <span class="fpage" tagx="fpage" title="fpage">957</span>–<span class="lpage" tagx="lpage" title="lpage">959</span>.</span></li> <li tag="ref"><a name="CIT0006" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhardwaj</span><span class="given-names" tagx="given-names" title="given-names">V. K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rajendran</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Purohit</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Identification of bioactive molecules from Tea plant as SARS-CoV-2 main protease inhibitors</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure Dynamics</i></span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1766572">10.1080/07391102.2020.1766572</a></span></span></li> <li tag="ref"><a name="CIT0007" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boopathi</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poma</span><span class="given-names" tagx="given-names" title="given-names">A. B.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kolandaivel</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Novel 2019 Coronavirus structure, mechanism of action, antiviral drug promises and rule out against its treatment</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1758788">10.1080/07391102.2020.1758788</a></span></span></li> <li tag="ref"><a name="CIT0008" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Borgio</span><span class="given-names" tagx="given-names" title="given-names">J. F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alsuwat</span><span class="given-names" tagx="given-names" title="given-names">H. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Al Otaibi</span><span class="given-names" tagx="given-names" title="given-names">W. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ibrahim</span><span class="given-names" tagx="given-names" title="given-names">A. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Almandil</span><span class="given-names" tagx="given-names" title="given-names">N. B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Al Asoom</span><span class="given-names" tagx="given-names" title="given-names">L. I.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salahuddin</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kamaraj</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">AbdulAzeez</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">State-of-the-art tools unveil potent drug targets amongst clinically approved drugs to inhibit helicase in SARS-CoV-2</span>. <span class="source" tagx="source" title="source"><i>Archives of Medical Science: AMS</i></span>, (<span class="issue" tagx="issue" title="issue">3</span>), <span class="fpage" tagx="fpage" title="fpage">508</span>–<span class="lpage" tagx="lpage" title="lpage">518</span>. <span class="pub-id"><a href="https://dx.doi.org/10.5114/aoms.2020.94567">10.5114/aoms.2020.94567</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32399096">32399096</a></span></span></li> <li tag="ref"><a name="CIT0009" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carneiro</span><span class="given-names" tagx="given-names" title="given-names">B. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Batista</span><span class="given-names" tagx="given-names" title="given-names">M. N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Braga</span><span class="given-names" tagx="given-names" title="given-names">A. C. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nogueira</span><span class="given-names" tagx="given-names" title="given-names">M. L.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rahal</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">The green tea molecule EGCG inhibits Zika virus entry</span>. <span class="source" tagx="source" title="source"><i>Virology</i></span>, , <span class="fpage" tagx="fpage" title="fpage">215</span>–<span class="lpage" tagx="lpage" title="lpage">218</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virol.2016.06.012">10.1016/j.virol.2016.06.012</a></span></span></li> <li tag="ref"><a name="CIT0010" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Case</span><span class="given-names" tagx="given-names" title="given-names">D. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ben-Shalom</span><span class="given-names" tagx="given-names" title="given-names">I. Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brozell</span><span class="given-names" tagx="given-names" title="given-names">S. R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cerutti</span><span class="given-names" tagx="given-names" title="given-names">D. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheatham</span><span class="given-names" tagx="given-names" title="given-names">T. E.</span><span class="suffix" tagx="suffix" title="suffix">III</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cruzeiro</span><span class="given-names" tagx="given-names" title="given-names">V. W. D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Darden</span><span class="given-names" tagx="given-names" title="given-names">T. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duke</span><span class="given-names" tagx="given-names" title="given-names">R. E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghoreishi</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gilson</span><span class="given-names" tagx="given-names" title="given-names">M. K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gohlke</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goetz</span><span class="given-names" tagx="given-names" title="given-names">A. W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greene</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Homeyer</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Izadi</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kovalenko</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kurtzman</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">T. S.</span></span>, … <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kollman</span><span class="given-names" tagx="given-names" title="given-names">P. A.</span></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="source" tagx="source" title="source"><i>AMBER 2018</i></span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">University of California</span>.</span></li> <li tag="ref"><a name="CIT0011" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">J. F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lau</span><span class="given-names" tagx="given-names" title="given-names">S. K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">To</span><span class="given-names" tagx="given-names" title="given-names">K. K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">V. C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woo</span><span class="given-names" tagx="given-names" title="given-names">P. C.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuen</span><span class="given-names" tagx="given-names" title="given-names">K.-Y.</span></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Middle East respiratory syndrome coronavirus: Another zoonotic betacoronavirus causing SARS-like disease</span>. <span class="source" tagx="source" title="source"><i>Clinical Microbiology Reviews</i></span>, (<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">465</span>–<span class="lpage" tagx="lpage" title="lpage">522</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/CMR.00102-14">10.1128/CMR.00102-14</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25810418">25810418</a></span></span></li> <li tag="ref"><a name="CIT0012" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chow</span><span class="given-names" tagx="given-names" title="given-names">H. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hakim</span><span class="given-names" tagx="given-names" title="given-names">I. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vining</span><span class="given-names" tagx="given-names" title="given-names">D. R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crowell</span><span class="given-names" tagx="given-names" title="given-names">J. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ranger-Moore</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chew</span><span class="given-names" tagx="given-names" title="given-names">W. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Celaya</span><span class="given-names" tagx="given-names" title="given-names">C. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodney</span><span class="given-names" tagx="given-names" title="given-names">S. R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hara</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alberts</span><span class="given-names" tagx="given-names" title="given-names">D. S.</span></span></span> (<span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">Effects of dosing condition on the oral bioavailability of green tea catechins after single-dose administration of Polyphenon E in healthy individuals</span>. <span class="source" tagx="source" title="source"><i>Clinical Cancer Research: An Official Journal of the American Association for Cancer Research</i></span>, (<span class="issue" tagx="issue" title="issue">12</span>), <span class="fpage" tagx="fpage" title="fpage">4627</span>–<span class="lpage" tagx="lpage" title="lpage">4633</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1158/1078-0432.CCR-04-2549">10.1158/1078-0432.CCR-04-2549</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15958649">15958649</a></span></span></li> <li tag="ref"><a name="CIT0013" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chowdhury</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sahuc</span><span class="given-names" tagx="given-names" title="given-names">M.-E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rouillé</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rivière</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonneau</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vandeputte</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brodin</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goswami</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bandyopadhyay</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dubuisson</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Séron</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Theaflavins, polyphenols of black tea, inhibit entry of hepatitis C virus in cell culture</span>. <span class="source" tagx="source" title="source"><i>PloS One</i></span>, (<span class="issue" tagx="issue" title="issue">11</span>), <span class="fpage" tagx="fpage" title="fpage">e0198226</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0198226">10.1371/journal.pone.0198226</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30485282">30485282</a></span></span></li> <li tag="ref"><a name="CIT0014" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cory</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Passarelli</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Szeto</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamez</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mattei</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">The role of polyphenols in human health and food systems: A mini-review</span>. <span class="source" tagx="source" title="source"><i>Frontiers in Nutrition</i></span>, , <span class="fpage" tagx="fpage" title="fpage">87</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fnut.2018.00087">10.3389/fnut.2018.00087</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30298133">30298133</a></span></span></li> <li tag="ref"><a name="CIT0015" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Darden</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">York</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pedersen</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span> (<span class="year" tagx="year" title="year">1993</span>). <span class="mixed-article-title" title="mixed-article-title">Particle mesh Ewald: An N⋅ log (N) method for Ewald sums in large systems</span>. <span class="source" tagx="source" title="source"><i>The Journal of Chemical Physics</i></span>, (<span class="issue" tagx="issue" title="issue">12</span>), <span class="fpage" tagx="fpage" title="fpage">10089</span>–<span class="lpage" tagx="lpage" title="lpage">10092</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1063/1.464397">10.1063/1.464397</a></span></span></li> <li tag="ref"><a name="CIT0016" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elfiky</span><span class="given-names" tagx="given-names" title="given-names">A. A.</span></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Zika virus: Novel guanosine derivatives revealed strong binding and possible inhibition of the polymerase</span>. <span class="source" tagx="source" title="source"><i>Future Virology</i></span>, (<span class="issue" tagx="issue" title="issue">12</span>), <span class="fpage" tagx="fpage" title="fpage">721</span>–<span class="lpage" tagx="lpage" title="lpage">728</span>. <span class="pub-id"><a href="https://dx.doi.org/10.2217/fvl-2017-0081">10.2217/fvl-2017-0081</a></span></span></li> <li tag="ref"><a name="CIT0017" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elfiky</span><span class="given-names" tagx="given-names" title="given-names">A. A.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Novel guanosine derivatives as anti-HCV NS5b polymerase: A QSAR and molecular docking Study</span>. <span class="source" tagx="source" title="source"><i>Medicinal Chemistry</i></span>, (<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">130</span>–<span class="lpage" tagx="lpage" title="lpage">137</span>. <span class="pub-id"><a href="https://dx.doi.org/10.2174/1573406414666181015152511">10.2174/1573406414666181015152511</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30324891">30324891</a></span></span></li> <li tag="ref"><a name="CIT0018" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elfiky</span><span class="given-names" tagx="given-names" title="given-names">A. A.</span></span></span> (<span class="year" tagx="year" title="year">2020a</span>). <span class="mixed-article-title" title="mixed-article-title">Anti-HCV, nucleotide inhibitors, repurposing against COVID-19</span>. <span class="source" tagx="source" title="source"><i>Life Sciences</i></span>, , <span class="fpage" tagx="fpage" title="fpage">117477</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.lfs.2020.117477">10.1016/j.lfs.2020.117477</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32119961">32119961</a></span></span></li> <li tag="ref"><a name="CIT0019" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elfiky</span><span class="given-names" tagx="given-names" title="given-names">A. A.</span></span></span> (<span class="year" tagx="year" title="year">2020b</span>). <span class="mixed-article-title" title="mixed-article-title">Natural products may interfere with SARS-CoV-2 attachment to the host cell</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure Dynamics</i></span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1761881">10.1080/07391102.2020.1761881</a></span></span></li> <li tag="ref"><a name="CIT0020" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elfiky</span><span class="given-names" tagx="given-names" title="given-names">A. A.</span></span></span> (<span class="year" tagx="year" title="year">2020c</span>). <span class="mixed-article-title" title="mixed-article-title">SARS-CoV-2 RNA dependent RNA polymerase (RdRp) targeting: An in silico perspective</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1761882">10.1080/07391102.2020.1761882</a></span></span></li> <li tag="ref"><a name="CIT0021" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elfiky</span><span class="given-names" tagx="given-names" title="given-names">A. A.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Azzam</span><span class="given-names" tagx="given-names" title="given-names">E. B.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Novel Guanosine Derivatives against MERS CoV polymerase: An in silico perspective</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1758789">10.1080/07391102.2020.1758789</a></span></span></li> <li tag="ref"><a name="CIT0022" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elmezayen</span><span class="given-names" tagx="given-names" title="given-names">A. D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Al-Obaidi</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Şahin</span><span class="given-names" tagx="given-names" title="given-names">A. T.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yelekçi</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Drug repurposing for coronavirus (COVID-19): In silico screening of known drugs against coronavirus 3CL hydrolase and protease enzymes</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure Dynamics</i></span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1758791">10.1080/07391102.2020.1758791</a></span></span></li> <li tag="ref"><a name="CIT0023" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Enmozhi</span><span class="given-names" tagx="given-names" title="given-names">S. K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raja</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sebastine</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joseph</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Andrographolide as a potential inhibitor of SARS-CoV-2 main protease: An in silico approach</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1760136">10.1080/07391102.2020.1760136</a></span></span></li> <li tag="ref"><a name="CIT0024" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ganesan</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barakat</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Applications of computer-aided approaches in the development of hepatitis C antiviral agents</span>. <span class="source" tagx="source" title="source"><i>Expert Opinion on Drug Discovery</i></span>, (<span class="issue" tagx="issue" title="issue">4</span>), <span class="fpage" tagx="fpage" title="fpage">407</span>–<span class="lpage" tagx="lpage" title="lpage">425</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/17460441.2017.1291628">10.1080/17460441.2017.1291628</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28164720">28164720</a></span></span></li> <li tag="ref"><a name="CIT0025" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gfeller</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grosdidier</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wirth</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daina</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Michielin</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zoete</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">SwissTargetPrediction: A web server for target prediction of bioactive small molecules</span>. <span class="source" tagx="source" title="source"><i>Nucleic Acids Research</i></span>, (<span class="issue" tagx="issue" title="issue">Web Server issue</span>), <span class="fpage" tagx="fpage" title="fpage">W32</span>–<span class="lpage" tagx="lpage" title="lpage">W38</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gku293">10.1093/nar/gku293</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24792161">24792161</a></span></span></li> <li tag="ref"><a name="CIT0026" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gohlke</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kiel</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Case</span><span class="given-names" tagx="given-names" title="given-names">D. A.</span></span></span> (<span class="year" tagx="year" title="year">2003</span>). <span class="mixed-article-title" title="mixed-article-title">Insights into protein–protein binding by binding free energy calculation and free energy decomposition for the Ras–Raf and Ras–RalGDS complexes</span>. <span class="source" tagx="source" title="source"><i>Journal of Molecular Biology</i></span>, (<span class="issue" tagx="issue" title="issue">4</span>), <span class="fpage" tagx="fpage" title="fpage">891</span>–<span class="lpage" tagx="lpage" title="lpage">913</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S0022-2836(03)00610-7">10.1016/S0022-2836(03)00610-7</a></span></span></li> <li tag="ref"><a name="CIT0027" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hendaus</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Remdesivir in the treatment of Coronavirus Disease 2019 (COVID-19): A simplified summary</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure Dynamics</i></span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1767691">10.1080/07391102.2020.1767691</a></span></span></li> <li tag="ref"><a name="CIT0028" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ho</span><span class="given-names" tagx="given-names" title="given-names">H.-Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">M.-L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weng</span><span class="given-names" tagx="given-names" title="given-names">S.-F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leu</span><span class="given-names" tagx="given-names" title="given-names">Y.-L.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chiu</span><span class="given-names" tagx="given-names" title="given-names">D. T.-Y.</span></span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Antiviral effect of epigallocatechin gallate on enterovirus 71</span>. <span class="source" tagx="source" title="source"><i>Journal of Agricultural and Food Chemistry</i></span>, (<span class="issue" tagx="issue" title="issue">14</span>), <span class="fpage" tagx="fpage" title="fpage">6140</span>–<span class="lpage" tagx="lpage" title="lpage">6147</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/jf901128u">10.1021/jf901128u</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19537794">19537794</a></span></span></li> <li tag="ref"><a name="CIT0029" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ibrahim</span><span class="given-names" tagx="given-names" title="given-names">I. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdelmalek</span><span class="given-names" tagx="given-names" title="given-names">D. H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elshahat</span><span class="given-names" tagx="given-names" title="given-names">M. E.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elfiky</span><span class="given-names" tagx="given-names" title="given-names">A. A.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">COVID-19 spike-host cell receptor GRP78 binding site prediction</span>. <span class="source" tagx="source" title="source"><i>The Journal of Infection</i></span>, (<span class="issue" tagx="issue" title="issue">5</span>), <span class="fpage" tagx="fpage" title="fpage">554</span>–<span class="lpage" tagx="lpage" title="lpage">562</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jinf.2020.02.026">10.1016/j.jinf.2020.02.026</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32169481">32169481</a></span></span></li> <li tag="ref"><a name="CIT0030" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Islam</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parves</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paul</span><span class="given-names" tagx="given-names" title="given-names">A. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uddin</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rahman</span><span class="given-names" tagx="given-names" title="given-names">M. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mamun</span><span class="given-names" tagx="given-names" title="given-names">A. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hossain</span><span class="given-names" tagx="given-names" title="given-names">M. N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ali</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halim</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">A molecular modeling approach to identify effective antiviral phytochemicals against the main protease of SARS-CoV-2</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure Dynamics</i></span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1761883">10.1080/07391102.2020.1761883</a></span></span></li> <li tag="ref"><a name="CIT0031" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jakalian</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jack</span><span class="given-names" tagx="given-names" title="given-names">D. B.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bayly</span><span class="given-names" tagx="given-names" title="given-names">C. I.</span></span></span> (<span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title">Fast, efficient generation of high‐quality atomic charges. AM1‐BCC model: II. Parameterization and validation</span>. <span class="source" tagx="source" title="source"><i>Journal of Computational Chemistry</i></span>, (<span class="issue" tagx="issue" title="issue">16</span>), <span class="fpage" tagx="fpage" title="fpage">1623</span>–<span class="lpage" tagx="lpage" title="lpage">1641</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/jcc.10128">10.1002/jcc.10128</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12395429">12395429</a></span></span></li> <li tag="ref"><a name="CIT0032" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jia</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gong</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">A structure-function diversity survey of the RNA-dependent RNA polymerases from the positive-strand RNA viruses</span>. <span class="source" tagx="source" title="source"><i>Frontiers in Microbiology</i></span>, , <span class="fpage" tagx="fpage" title="fpage">1945</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fmicb.2019.01945">10.3389/fmicb.2019.01945</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31507560">31507560</a></span></span></li> <li tag="ref"><a name="CIT0033" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jonniya</span><span class="given-names" tagx="given-names" title="given-names">N. A.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kar</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Investigating specificity of the anti-hypertensive inhibitor WNK463 against With-No-Lysine kinase family isoforms via multiscale simulations</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure &amp;amp; Dynamics</i></span>, (<span class="issue" tagx="issue" title="issue">5</span>), <span class="fpage" tagx="fpage" title="fpage">1306</span>–<span class="lpage" tagx="lpage" title="lpage">1321</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2019.1602079">10.1080/07391102.2019.1602079</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31017050">31017050</a></span></span></li> <li tag="ref"><a name="CIT0034" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jonniya</span><span class="given-names" tagx="given-names" title="given-names">N. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sk</span><span class="given-names" tagx="given-names" title="given-names">M. F.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kar</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Investigating phosphorylation-induced conformational changes in WNK1 kinase by molecular dynamics simulations</span>. <span class="source" tagx="source" title="source"><i>ACS Omega</i></span>, (<span class="issue" tagx="issue" title="issue">17</span>), <span class="fpage" tagx="fpage" title="fpage">17404</span>–<span class="lpage" tagx="lpage" title="lpage">17416</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/acsomega.9b02187">10.1021/acsomega.9b02187</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31656913">31656913</a></span></span></li> <li tag="ref"><a name="CIT0035" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kar</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knecht</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span></span> (<span class="year" tagx="year" title="year">2012</span>a). <span class="mixed-article-title" title="mixed-article-title">Energetic basis for drug resistance of HIV-1 protease mutants against amprenavir</span>. <span class="source" tagx="source" title="source"><i>Journal of Computer-Aided Molecular Design</i></span>, (<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">215</span>–<span class="lpage" tagx="lpage" title="lpage">232</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s10822-012-9550-5">10.1007/s10822-012-9550-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22350569">22350569</a></span></span></li> <li tag="ref"><a name="CIT0036" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kar</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knecht</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span></span> (<span class="year" tagx="year" title="year">2012</span>b). <span class="mixed-article-title" title="mixed-article-title">Energetics of mutation-induced changes in potency of lersivirine against HIV-1 reverse transcriptase</span>. <span class="source" tagx="source" title="source"><i>The Journal of Physical Chemistry B</i></span>, (<span class="issue" tagx="issue" title="issue">22</span>), <span class="fpage" tagx="fpage" title="fpage">6269</span>–<span class="lpage" tagx="lpage" title="lpage">6278</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/jp300818c">10.1021/jp300818c</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22574920">22574920</a></span></span></li> <li tag="ref"><a name="CIT0037" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kar</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knecht</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span></span> (<span class="year" tagx="year" title="year">2012</span>c). <span class="mixed-article-title" title="mixed-article-title">Mutation-induced loop opening and energetics for binding of tamiflu to influenza N8 neuraminidase</span>. <span class="source" tagx="source" title="source"><i>The Journal of Physical Chemistry B</i></span>, (<span class="issue" tagx="issue" title="issue">21</span>), <span class="fpage" tagx="fpage" title="fpage">6137</span>–<span class="lpage" tagx="lpage" title="lpage">6149</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/jp3022612">10.1021/jp3022612</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22553951">22553951</a></span></span></li> <li tag="ref"><a name="CIT0038" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kar</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knecht</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span></span> (<span class="year" tagx="year" title="year">2012</span>d). <span class="mixed-article-title" title="mixed-article-title">Origin of decrease in potency of darunavir and two related antiviral inhibitors against HIV-2 compared to HIV-1 protease</span>. <span class="source" tagx="source" title="source"><i>The Journal of Physical Chemistry B</i></span>, (<span class="issue" tagx="issue" title="issue">8</span>), <span class="fpage" tagx="fpage" title="fpage">2605</span>–<span class="lpage" tagx="lpage" title="lpage">2614</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/jp211768n">10.1021/jp211768n</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22280246">22280246</a></span></span></li> <li tag="ref"><a name="CIT0039" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kar</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lipowsky</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knecht</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span></span> (<span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Importance of polar solvation for cross-reactivity of antibody and its variants with steroids</span>. <span class="source" tagx="source" title="source"><i>The Journal of Physical Chemistry B</i></span>, (<span class="issue" tagx="issue" title="issue">23</span>), <span class="fpage" tagx="fpage" title="fpage">7661</span>–<span class="lpage" tagx="lpage" title="lpage">7669</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/jp201538t">10.1021/jp201538t</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21595427">21595427</a></span></span></li> <li tag="ref"><a name="CIT0040" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kar</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lipowsky</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knecht</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Importance of polar solvation and configurational entropy for design of antiretroviral drugs targeting HIV-1 protease</span>. <span class="source" tagx="source" title="source"><i>The Journal of Physical Chemistry B</i></span>, (<span class="issue" tagx="issue" title="issue">19</span>), <span class="fpage" tagx="fpage" title="fpage">5793</span>–<span class="lpage" tagx="lpage" title="lpage">5805</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/jp3085292">10.1021/jp3085292</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23614718">23614718</a></span></span></li> <li tag="ref"><a name="CIT0041" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kar</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seel</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hansmann</span><span class="given-names" tagx="given-names" title="given-names">U. H.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Höfinger</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Dispersion terms and analysis of size- and charge dependence in an enhanced Poisson-Boltzmann approach</span>. <span class="source" tagx="source" title="source"><i>The Journal of Physical Chemistry B</i></span>, (<span class="issue" tagx="issue" title="issue">30</span>), <span class="fpage" tagx="fpage" title="fpage">8910</span>–<span class="lpage" tagx="lpage" title="lpage">8918</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/jp072302u">10.1021/jp072302u</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17628098">17628098</a></span></span></li> <li tag="ref"><a name="CIT0042" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kar</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hansmann</span><span class="given-names" tagx="given-names" title="given-names">U. H.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Höfinger</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Systematic study of the boundary composition in Poisson Boltzmann calculations</span>. <span class="source" tagx="source" title="source"><i>Journal of Computational Chemistry</i></span>, (<span class="issue" tagx="issue" title="issue">16</span>), <span class="fpage" tagx="fpage" title="fpage">2538</span>–<span class="lpage" tagx="lpage" title="lpage">2544</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/jcc.20698">10.1002/jcc.20698</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17503456">17503456</a></span></span></li> <li tag="ref"><a name="CIT0043" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">M. T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ali</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Irfan</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zeb</span><span class="given-names" tagx="given-names" title="given-names">M. T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y.-J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chinnasamy</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">D.-Q.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Marine natural compounds as potents inhibitors against the main protease of SARS-CoV-2. A molecular dynamic study</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1769733">10.1080/07391102.2020.1769733</a></span></span></li> <li tag="ref"><a name="CIT0044" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gindulyte</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shoemaker</span><span class="given-names" tagx="given-names" title="given-names">B. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thiessen</span><span class="given-names" tagx="given-names" title="given-names">P. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zaslavsky</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bolton</span><span class="given-names" tagx="given-names" title="given-names">E. E.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">PubChem 2019 update: Improved access to chemical data</span>. <span class="source" tagx="source" title="source"><i>Nucleic Acids Research</i></span>, (<span class="issue" tagx="issue" title="issue">D1</span>), <span class="fpage" tagx="fpage" title="fpage">D1102</span>–<span class="lpage" tagx="lpage" title="lpage">D1109</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gky1033">10.1093/nar/gky1033</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30371825">30371825</a></span></span></li> <li tag="ref"><a name="CIT0045" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kollman</span><span class="given-names" tagx="given-names" title="given-names">P. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Massova</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reyes</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuhn</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huo</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chong</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duan</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Donini</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cieplak</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Srinivasan</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Case</span><span class="given-names" tagx="given-names" title="given-names">D. A.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheatham</span><span class="given-names" tagx="given-names" title="given-names">T. E.</span></span></span> (<span class="year" tagx="year" title="year">2000</span>). <span class="mixed-article-title" title="mixed-article-title">Calculating structures and free energies of complex molecules: Combining molecular mechanics and continuum models</span>. <span class="source" tagx="source" title="source"><i>Accounts of Chemical Research</i></span>, (<span class="issue" tagx="issue" title="issue">12</span>), <span class="fpage" tagx="fpage" title="fpage">889</span>–<span class="lpage" tagx="lpage" title="lpage">897</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/ar000033j">10.1021/ar000033j</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11123888">11123888</a></span></span></li> <li tag="ref"><a name="CIT0046" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kräutler</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Gunsteren</span><span class="given-names" tagx="given-names" title="given-names">W. F.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hünenberger</span><span class="given-names" tagx="given-names" title="given-names">P. H.</span></span></span> (<span class="year" tagx="year" title="year">2001</span>). <span class="mixed-article-title" title="mixed-article-title">A fast SHAKE algorithm to solve distance constraint equations for small molecules in molecular dynamics simulations</span>. <span class="source" tagx="source" title="source"><i>Journal of Computational Chemistry</i></span>, (<span class="issue" tagx="issue" title="issue">5</span>), <span class="fpage" tagx="fpage" title="fpage">501</span>–<span class="lpage" tagx="lpage" title="lpage">508</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/1096-987X(20010415)22:5&lt;501::AID-JCC1021&gt;3.0.CO;2-V">10.1002/1096-987X(20010415)22:5&amp;lt;501::AID-JCC1021&amp;gt;3.0.CO;2-V</a></span></span></li> <li tag="ref"><a name="CIT0047" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuzuhara</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iwai</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takahashi</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hatakeyama</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Echigo</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Green tea catechins inhibit the endonuclease activity of influenza A virus RNA polymerase</span>. <span class="source" tagx="source" title="source"><i>PLoS Currents</i></span>, , <span class="fpage" tagx="fpage" title="fpage">RRN1052</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/currents.RRN1052">10.1371/currents.RRN1052</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20025206">20025206</a></span></span></li> <li tag="ref"><a name="CIT0048" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larregieu</span><span class="given-names" tagx="given-names" title="given-names">C. A.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benet</span><span class="given-names" tagx="given-names" title="given-names">L. Z.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Drug discovery and regulatory considerations for improving in silico and in vitro predictions that use Caco-2 as a surrogate for human intestinal permeability measurements</span>. <span class="source" tagx="source" title="source"><i>The AAPS Journal</i></span>, (<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">483</span>–<span class="lpage" tagx="lpage" title="lpage">497</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1208/s12248-013-9456-8">10.1208/s12248-013-9456-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23344793">23344793</a></span></span></li> <li tag="ref"><a name="CIT0049" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">L.-T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hsu</span><span class="given-names" tagx="given-names" title="given-names">W.-C.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">C.-C.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Antiviral natural products and herbal medicines</span>. <span class="source" tagx="source" title="source"><i>Journal of Traditional and Complementary Medicine</i></span>, (<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">24</span>–<span class="lpage" tagx="lpage" title="lpage">35</span>. <span class="pub-id"><a href="https://dx.doi.org/10.4103/2225-4110.124335">10.4103/2225-4110.124335</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24872930">24872930</a></span></span></li> <li tag="ref"><a name="CIT0050" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Loncharich</span><span class="given-names" tagx="given-names" title="given-names">R. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brooks</span><span class="given-names" tagx="given-names" title="given-names">B. R.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pastor</span><span class="given-names" tagx="given-names" title="given-names">R. W.</span></span></span> (<span class="year" tagx="year" title="year">1992</span>). <span class="mixed-article-title" title="mixed-article-title">Langevin dynamics of peptides: The frictional dependence of isomerization rates of N-acetylalanyl-N'-methylamide</span>. <span class="source" tagx="source" title="source"><i>Biopolymers</i></span>, (<span class="issue" tagx="issue" title="issue">5</span>), <span class="fpage" tagx="fpage" title="fpage">523</span>–<span class="lpage" tagx="lpage" title="lpage">535</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/bip.360320508">10.1002/bip.360320508</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1515543">1515543</a></span></span></li> <li tag="ref"><a name="CIT0051" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maier</span><span class="given-names" tagx="given-names" title="given-names">J. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martinez</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kasavajhala</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wickstrom</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hauser</span><span class="given-names" tagx="given-names" title="given-names">K. E.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmerling</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">ff14SB: Improving the accuracy of protein side chain and backbone parameters from ff99SB</span>. <span class="source" tagx="source" title="source"><i>Journal of Chemical Theory and Computation</i></span>, (<span class="issue" tagx="issue" title="issue">8</span>), <span class="fpage" tagx="fpage" title="fpage">3696</span>–<span class="lpage" tagx="lpage" title="lpage">3713</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/acs.jctc.5b00255">10.1021/acs.jctc.5b00255</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26574453">26574453</a></span></span></li> <li tag="ref"><a name="CIT0052" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moon</span><span class="given-names" tagx="given-names" title="given-names">Y. J.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morris</span><span class="given-names" tagx="given-names" title="given-names">M. E.</span></span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Pharmacokinetics and bioavailability of the bioflavonoid biochanin A: Effects of quercetin and EGCG on biochanin A disposition in rats</span>. <span class="source" tagx="source" title="source"><i>Molecular Pharmaceutics</i></span>, (<span class="issue" tagx="issue" title="issue">6</span>), <span class="fpage" tagx="fpage" title="fpage">865</span>–<span class="lpage" tagx="lpage" title="lpage">872</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/mp7000928">10.1021/mp7000928</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17970592">17970592</a></span></span></li> <li tag="ref"><a name="CIT0053" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morris</span><span class="given-names" tagx="given-names" title="given-names">G. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huey</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lindstrom</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanner</span><span class="given-names" tagx="given-names" title="given-names">M. F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belew</span><span class="given-names" tagx="given-names" title="given-names">R. K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goodsell</span><span class="given-names" tagx="given-names" title="given-names">D. S.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olson</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">AutoDock4 and AutoDockTools4: Automated docking with selective receptor flexibility</span>. <span class="source" tagx="source" title="source"><i>Journal of Computational Chemistry</i></span>, (<span class="issue" tagx="issue" title="issue">16</span>), <span class="fpage" tagx="fpage" title="fpage">2785</span>–<span class="lpage" tagx="lpage" title="lpage">2791</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/jcc.21256">10.1002/jcc.21256</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19399780">19399780</a></span></span></li> <li tag="ref"><a name="CIT0054" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muralidharan</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sakthivel</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Velmurugan</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gromiha</span><span class="given-names" tagx="given-names" title="given-names">M. M.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Computational studies of drug repurposing and synergism of lopinavir, oseltamivir and ritonavir binding with SARS-CoV-2 Protease against COVID-19</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure Dynamics</i></span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1752802">10.1080/07391102.2020.1752802</a></span></span></li> <li tag="ref"><a name="CIT0055" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pettersen</span><span class="given-names" tagx="given-names" title="given-names">E. F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goddard</span><span class="given-names" tagx="given-names" title="given-names">T. D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">C. C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Couch</span><span class="given-names" tagx="given-names" title="given-names">G. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greenblatt</span><span class="given-names" tagx="given-names" title="given-names">D. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meng</span><span class="given-names" tagx="given-names" title="given-names">E. C.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferrin</span><span class="given-names" tagx="given-names" title="given-names">T. E.</span></span></span> (<span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">UCSF Chimera-a visualization system for exploratory research and analysis</span>. <span class="source" tagx="source" title="source"><i>Journal of Computational Chemistry</i></span>, (<span class="issue" tagx="issue" title="issue">13</span>), <span class="fpage" tagx="fpage" title="fpage">1605</span>–<span class="lpage" tagx="lpage" title="lpage">1612</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/jcc.20084">10.1002/jcc.20084</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15264254">15264254</a></span></span></li> <li tag="ref"><a name="CIT0056" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pires</span><span class="given-names" tagx="given-names" title="given-names">D. E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blundell</span><span class="given-names" tagx="given-names" title="given-names">T. L.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ascher</span><span class="given-names" tagx="given-names" title="given-names">D. B.</span></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">pkCSM: Predicting small-molecule pharmacokinetic and toxicity properties using graph-based signatures</span>. <span class="source" tagx="source" title="source"><i>Journal of Medicinal Chemistry</i></span>, (<span class="issue" tagx="issue" title="issue">9</span>), <span class="fpage" tagx="fpage" title="fpage">4066</span>–<span class="lpage" tagx="lpage" title="lpage">4072</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/acs.jmedchem.5b00104">10.1021/acs.jmedchem.5b00104</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25860834">25860834</a></span></span></li> <li tag="ref"><a name="CIT0057" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Price</span><span class="given-names" tagx="given-names" title="given-names">D. J.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brooks III</span><span class="given-names" tagx="given-names" title="given-names">C. L.</span></span></span> (<span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">A modified TIP3P water potential for simulation with Ewald summation</span>. <span class="source" tagx="source" title="source"><i>The Journal of Chemical Physics</i></span>, (<span class="issue" tagx="issue" title="issue">20</span>), <span class="fpage" tagx="fpage" title="fpage">10096</span>–<span class="lpage" tagx="lpage" title="lpage">10103</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1063/1.1808117">10.1063/1.1808117</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15549884">15549884</a></span></span></li> <li tag="ref"><a name="CIT0058" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Randall</span><span class="given-names" tagx="given-names" title="given-names">R. E.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goodbourn</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span> (<span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Interferons and viruses: An interplay between induction, signalling, antiviral responses and virus countermeasures</span>. <span class="source" tagx="source" title="source"><i>The Journal of General Virology</i></span>, (<span class="issue" tagx="issue" title="issue">Pt 1</span>), <span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">47</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1099/vir.0.83391-0">10.1099/vir.0.83391-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18089727">18089727</a></span></span></li> <li tag="ref"><a name="CIT0059" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roe</span><span class="given-names" tagx="given-names" title="given-names">D. R.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheatham III</span><span class="given-names" tagx="given-names" title="given-names">T. E.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">PTRAJ and CPPTRAJ: Software for processing and analysis of molecular dynamics trajectory data</span>. <span class="source" tagx="source" title="source"><i>Journal of Chemical Theory and Computation</i></span>, (<span class="issue" tagx="issue" title="issue">7</span>), <span class="fpage" tagx="fpage" title="fpage">3084</span>–<span class="lpage" tagx="lpage" title="lpage">3095</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/ct400341p">10.1021/ct400341p</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26583988">26583988</a></span></span></li> <li tag="ref"><a name="CIT0060" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roy</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghosh</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kar</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Investigating conformational dynamics of Lewis Y oligosaccharides and elucidating blood group dependency of cholera using molecular dynamics</span>. <span class="source" tagx="source" title="source"><i>ACS Omega</i></span>, (<span class="issue" tagx="issue" title="issue">8</span>), <span class="fpage" tagx="fpage" title="fpage">3932</span>–<span class="lpage" tagx="lpage" title="lpage">3942</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/acsomega.9b03398">10.1021/acsomega.9b03398</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32149220">32149220</a></span></span></li> <li tag="ref"><a name="CIT0061" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sinha</span><span class="given-names" tagx="given-names" title="given-names">S. K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shakya</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prasad</span><span class="given-names" tagx="given-names" title="given-names">S. K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gurav</span><span class="given-names" tagx="given-names" title="given-names">N. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prasad</span><span class="given-names" tagx="given-names" title="given-names">R. S.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gurav</span><span class="given-names" tagx="given-names" title="given-names">S. S.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">An in-silico evaluation of different Saikosaponins for their potency against SARS-CoV-2 using NSP15 and fusion spike glycoprotein as targets</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure Dynamics</i></span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1762741">10.1080/07391102.2020.1762741</a></span></span></li> <li tag="ref"><a name="CIT0062" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sk</span><span class="given-names" tagx="given-names" title="given-names">M. F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roy</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jonniya</span><span class="given-names" tagx="given-names" title="given-names">N. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poddar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kar</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Elucidating biophysical basis of binding of inhibitors to SARS-CoV-2 main protease by using molecular dynamics simulations and free energy calculations</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure Dynamics</i></span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.07392020.01768149">10.1080/07391102.07392020.01768149</a></span></span></li> <li tag="ref"><a name="CIT0063" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sk</span><span class="given-names" tagx="given-names" title="given-names">M. F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roy</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kar</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Exploring the potency of currently used drugs against HIV-1 protease of subtype D variant by using multiscale simulations</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure Dynamics</i></span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.07392020.01724196">10.1080/07391102.07392020.01724196</a></span></span></li> <li tag="ref"><a name="CIT0064" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">J.-M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">K.-H.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seong</span><span class="given-names" tagx="given-names" title="given-names">B.-L.</span></span></span> (<span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">Antiviral effect of catechins in green tea on influenza virus</span>. <span class="source" tagx="source" title="source"><i>Antiviral Research</i></span>, (<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">66</span>–<span class="lpage" tagx="lpage" title="lpage">74</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.antiviral.2005.06.010">10.1016/j.antiviral.2005.06.010</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16137775">16137775</a></span></span></li> <li tag="ref"><a name="CIT0065" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Subissi</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Posthuma</span><span class="given-names" tagx="given-names" title="given-names">C. C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Collet</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zevenhoven-Dobbe</span><span class="given-names" tagx="given-names" title="given-names">J. C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gorbalenya</span><span class="given-names" tagx="given-names" title="given-names">A. E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Decroly</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Snijder</span><span class="given-names" tagx="given-names" title="given-names">E. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Canard</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Imbert</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">One severe acute respiratory syndrome coronavirus protein complex integrates processive RNA polymerase and exonuclease activities</span>. <span class="source" tagx="source" title="source"><i>Proceedings of the National Academy of Sciences of the United States of America</i></span>, (<span class="issue" tagx="issue" title="issue">37</span>), <span class="fpage" tagx="fpage" title="fpage">E3900</span>–<span class="lpage" tagx="lpage" title="lpage">E3909</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1323705111">10.1073/pnas.1323705111</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25197083">25197083</a></span></span></li> <li tag="ref"><a name="CIT0066" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Szajdek</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Borowska</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span> (<span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Bioactive compounds and health-promoting properties of berry fruits: A review</span>. <span class="source" tagx="source" title="source"><i>Plant Foods for Human Nutrition (Dordrecht, Netherlands)</i></span>, (<span class="issue" tagx="issue" title="issue">4</span>), <span class="fpage" tagx="fpage" title="fpage">147</span>–<span class="lpage" tagx="lpage" title="lpage">156</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s11130-008-0097-5">10.1007/s11130-008-0097-5</a></span></span></li> <li tag="ref"><a name="CIT0067" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trott</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olson</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span></span> (<span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">AutoDock Vina: improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span>. <span class="source" tagx="source" title="source"><i>Journal of Computaional Chemistry</i></span>, , <span class="fpage" tagx="fpage" title="fpage">455</span>–<span class="lpage" tagx="lpage" title="lpage">461</span>.</span></li> <li tag="ref"><a name="CIT0068" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wallace</span><span class="given-names" tagx="given-names" title="given-names">A. C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Laskowski</span><span class="given-names" tagx="given-names" title="given-names">R. A.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thornton</span><span class="given-names" tagx="given-names" title="given-names">J. M.</span></span></span> (<span class="year" tagx="year" title="year">1995</span>). <span class="mixed-article-title" title="mixed-article-title">LIGPLOT: A program to generate schematic diagrams of protein-ligand interactions</span>. <span class="source" tagx="source" title="source"><i>Protein Engineering</i></span>, (<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">127</span>–<span class="lpage" tagx="lpage" title="lpage">134</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/protein/8.2.127">10.1093/protein/8.2.127</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7630882">7630882</a></span></span></li> <li tag="ref"><a name="CIT0069" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kollman</span><span class="given-names" tagx="given-names" title="given-names">P. A.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Case</span><span class="given-names" tagx="given-names" title="given-names">D. A.</span></span></span> (<span class="year" tagx="year" title="year">2001</span>). <span class="mixed-article-title" title="mixed-article-title">Antechamber: An accessory software package for molecular mechanical calculations</span>. <span class="source" tagx="source" title="source"><i>Journal of the American Chemical Society</i></span>, (<span class="issue" tagx="issue" title="issue">17</span>), <span class="fpage" tagx="fpage" title="fpage">3986</span>–<span class="lpage" tagx="lpage" title="lpage">3994</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/ja003164o">10.1021/ja003164o</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11457149">11457149</a></span></span></li> <li tag="ref"><a name="CIT0070" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kollman</span><span class="given-names" tagx="given-names" title="given-names">P. A.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Case</span><span class="given-names" tagx="given-names" title="given-names">D. A.</span></span></span> (<span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">Automatic atom type and bond type perception in molecular mechanical calculations</span>. <span class="source" tagx="source" title="source"><i>Journal of Molecular Graphics &amp;amp; Modelling</i></span>, (<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">247</span>–<span class="lpage" tagx="lpage" title="lpage">260</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jmgm.2005.12.005">10.1016/j.jmgm.2005.12.005</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16458552">16458552</a></span></span></li> <li tag="ref"><a name="CIT0071" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wolf</span><span class="given-names" tagx="given-names" title="given-names">R. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caldwell</span><span class="given-names" tagx="given-names" title="given-names">J. W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kollman</span><span class="given-names" tagx="given-names" title="given-names">P. A.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Case</span><span class="given-names" tagx="given-names" title="given-names">D. A.</span></span></span> (<span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Development and testing of a general amber force field</span>. <span class="source" tagx="source" title="source"><i>Journal of Computational Chemistry</i></span>, (<span class="issue" tagx="issue" title="issue">9</span>), <span class="fpage" tagx="fpage" title="fpage">1157</span>–<span class="lpage" tagx="lpage" title="lpage">1174</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/jcc.20035">10.1002/jcc.20035</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15116359">15116359</a></span></span></li> <li tag="ref"><a name="CIT0072" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Y.-M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">Z.-G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tao</span><span class="given-names" tagx="given-names" title="given-names">Z.-W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tian</span><span class="given-names" tagx="given-names" title="given-names">J.-H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pei</span><span class="given-names" tagx="given-names" title="given-names">Y.-Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">M.-L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y.-L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dai</span><span class="given-names" tagx="given-names" title="given-names">F.-H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Q.-M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">J.-J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">E. C.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y.-Z.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">A new coronavirus associated with human respiratory disease in China</span>. <span class="source" tagx="source" title="source"><i>Nature</i></span>, (<span class="issue" tagx="issue" title="issue">7798</span>), <span class="fpage" tagx="fpage" title="fpage">265</span>–<span class="lpage" tagx="lpage" title="lpage">269</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41586-020-2008-3">10.1038/s41586-020-2008-3</a></span></span></li> <li tag="ref"><a name="CIT0073" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gong</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Visualizing the nucleotide addition cycle of viral RNA-dependent RNA polymerase</span>. <span class="source" tagx="source" title="source"><i>Viruses</i></span>, (<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">24</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/v10010024">10.3390/v10010024</a></span></span></li> <li tag="ref"><a name="CIT0074" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Structural basis for the recognition of SARS-CoV-2 by full-length human ACE2</span>. <span class="source" tagx="source" title="source"><i>Science (New York, N.Y.)</i></span>, (<span class="issue" tagx="issue" title="issue">6485</span>), <span class="fpage" tagx="fpage" title="fpage">1444</span>–<span class="lpage" tagx="lpage" title="lpage">1448</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1126/science.abb2762">10.1126/science.abb2762</a></span></span></li> <li tag="ref"><a name="CIT0075" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yin</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mao</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luan</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">D.-D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">Y.-C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tian</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">H.-W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tao</span><span class="given-names" tagx="given-names" title="given-names">S.-C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, … <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">H. E.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Structural basis for inhibition of the RNA-dependent RNA polymerase from SARS-CoV-2 by remdesivir</span>. <span class="source" tagx="source" title="source"><i>Science (New York, N.Y.)</i></span>, (<span class="issue" tagx="issue" title="issue">6498</span>), <span class="fpage" tagx="fpage" title="fpage">1499</span>–<span class="lpage" tagx="lpage" title="lpage">1504</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1126/science.abc1560">10.1126/science.abc1560</a></span></span></li> <li tag="ref"><a name="CIT0076" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">M.-S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">J. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chin</span><span class="given-names" tagx="given-names" title="given-names">Y.-W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jee</span><span class="given-names" tagx="given-names" title="given-names">J.-G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keum</span><span class="given-names" tagx="given-names" title="given-names">Y.-S.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jeong</span><span class="given-names" tagx="given-names" title="given-names">Y.-J.</span></span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Identification of myricetin and scutellarein as novel chemical inhibitors of the SARS coronavirus helicase, nsP13</span>. <span class="source" tagx="source" title="source"><i>Bioorganic &amp;amp; Medicinal Chemistry Letters</i></span>, (<span class="issue" tagx="issue" title="issue">12</span>), <span class="fpage" tagx="fpage" title="fpage">4049</span>–<span class="lpage" tagx="lpage" title="lpage">4054</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bmcl.2012.04.081">10.1016/j.bmcl.2012.04.081</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22578462">22578462</a></span></span></li> <li tag="ref"><a name="CIT0077" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhai</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pang</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bartlam</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rao</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span></span> (<span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">Insights into SARS-CoV transcription and replication from the structure of the nsp7-nsp8 hexadecamer</span>. <span class="source" tagx="source" title="source"><i>Nature Structural &amp;amp; Molecular Biology</i></span>, (<span class="issue" tagx="issue" title="issue">11</span>), <span class="fpage" tagx="fpage" title="fpage">980</span>–<span class="lpage" tagx="lpage" title="lpage">986</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nsmb999">10.1038/nsmb999</a></span></span></li> </ul> </div> </div>  </body></html>