<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/02/16/2020.02.08.926006.atom" />
<meta name="HW.pisa" content="biorxiv;2020.02.08.926006v3" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The insert sequence in SARS-CoV-2 enhances spike protein cleavage by TMPRSS" />
<meta name="DC.Identifier" content="10.1101/2020.02.08.926006" />
<meta name="DC.Date" content="2020-02-16" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="At the end of 2019, the SARS-CoV-2 induces an ongoing outbreak of pneumonia in China[1][1], even more spread than SARS-CoV infection[2][2]. The entry of SARS-CoV into host cells mainly depends on the cell receptor (ACE2) recognition and spike protein cleavage-induced cell membrane fusion[3][3],[4][4]. The spike protein of SARS-CoV-2 also binds to ACE2 with a similar affinity, whereas its spike protein cleavage remains unclear[5][5],[6][6]. Here we show that an insertion sequence in the spike protein of SARS-CoV-2 enhances the cleavage efficiency, and besides pulmonary alveoli, intestinal and esophagus epithelium were also the target tissues of SARS-CoV-2. Compared with SARS-CoV, we found a SPRR insertion in the S1/S2 protease cleavage sites of SARS-CoV-2 spike protein increasing the cleavage efficiency by the protein sequence aligment and furin score calculation. Additionally, the insertion sequence facilitates the formation of an extended loop which was more suitable for protease recognition by the homology modeling and molicular docking. Furthermore, the single-cell transcriptomes identified that ACE2 and TMPRSSs are highly coexpressed in AT2 cells of lung, along with esophageal upper epithelial cells and absorptive enterocytes. Our results provide the bioinformatics evidence for the increased spike protein cleavage of SARS-CoV-2 and indicate its potential target cells.

 [1]: #ref-1
 [2]: #ref-2
 [3]: #ref-3
 [4]: #ref-4
 [5]: #ref-5
 [6]: #ref-6" />
<meta name="DC.Contributor" content="Tong Meng" />
<meta name="DC.Contributor" content="Hao Cao" />
<meta name="DC.Contributor" content="Hao Zhang" />
<meta name="DC.Contributor" content="Zijian Kang" />
<meta name="DC.Contributor" content="Da Xu" />
<meta name="DC.Contributor" content="Haiyi Gong" />
<meta name="DC.Contributor" content="Jing Wang" />
<meta name="DC.Contributor" content="Zifu Li" />
<meta name="DC.Contributor" content="Xingang Cui" />
<meta name="DC.Contributor" content="Huji Xu" />
<meta name="DC.Contributor" content="Haifeng Wei" />
<meta name="DC.Contributor" content="Xiuwu Pan" />
<meta name="DC.Contributor" content="Rongrong Zhu" />
<meta name="DC.Contributor" content="Jianru Xiao" />
<meta name="DC.Contributor" content="Wang Zhou" />
<meta name="DC.Contributor" content="Liming Cheng" />
<meta name="DC.Contributor" content="Jianmin Liu" />
<meta name="article:published_time" content="2020-02-16" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="The insert sequence in SARS-CoV-2 enhances spike protein cleavage by TMPRSS" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;At the end of 2019, the SARS-CoV-2 induces an ongoing outbreak of pneumonia in China&lt;sup&gt;1&lt;/sup&gt;, even more spread than SARS-CoV infection&lt;sup&gt;2&lt;/sup&gt;. The entry of SARS-CoV into host cells mainly depends on the cell receptor (ACE2) recognition and spike protein cleavage-induced cell membrane fusion&lt;sup&gt;3,4&lt;/sup&gt;. The spike protein of SARS-CoV-2 also binds to ACE2 with a similar affinity, whereas its spike protein cleavage remains unclear&lt;sup&gt;5,6&lt;/sup&gt;. Here we show that an insertion sequence in the spike protein of SARS-CoV-2 enhances the cleavage efficiency, and besides pulmonary alveoli, intestinal and esophagus epithelium were also the target tissues of SARS-CoV-2. Compared with SARS-CoV, we found a SPRR insertion in the S1/S2 protease cleavage sites of SARS-CoV-2 spike protein increasing the cleavage efficiency by the protein sequence aligment and furin score calculation. Additionally, the insertion sequence facilitates the formation of an extended loop which was more suitable for protease recognition by the homology modeling and molicular docking. Furthermore, the single-cell transcriptomes identified that ACE2 and TMPRSSs are highly coexpressed in AT2 cells of lung, along with esophageal upper epithelial cells and absorptive enterocytes. Our results provide the bioinformatics evidence for the increased spike protein cleavage of SARS-CoV-2 and indicate its potential target cells.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.02.08.926006v3" />
<meta name="citation_id" content="2020.02.08.926006v3" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.08.926006v3" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.08.926006v3.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.08.926006v3.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.02.08.926006" />
<meta name="citation_num_pages" content="36" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.02.08.926006" />
<meta name="citation_author" content="Tong Meng" />
<meta name="citation_author_institution" content="Division of Spine, Department of Orthopedics, Tongji Hospital affiliated to Tongji University School of Medicine" />
<meta name="citation_author_institution" content="Tongji University Cancer Center, School of Medicine, Tongji University" />
<meta name="citation_author_institution" content="Qiu-Jiang Bioinformatics Institute" />
<meta name="citation_author" content="Hao Cao" />
<meta name="citation_author_institution" content="School of Life Science and Biopharmaceutics, Shenyang Pharmaceutical University" />
<meta name="citation_author_institution" content="Peking-Tsinghua Center for Life Sciences, TsinghuaUniversity" />
<meta name="citation_author" content="Hao Zhang" />
<meta name="citation_author_institution" content="Department of Orthopaedic Oncology, Changzheng Hospital, Second Military Medical University" />
<meta name="citation_author_institution" content="Qiu-Jiang Bioinformatics Institute" />
<meta name="citation_author" content="Zijian Kang" />
<meta name="citation_author_institution" content="Department of Rheumatology and Immunology, Changzheng Hospital, Second Military Medical University" />
<meta name="citation_author_institution" content="Qiu-Jiang Bioinformatics Institute" />
<meta name="citation_author" content="Da Xu" />
<meta name="citation_author_institution" content="Depanrtment of Urology, The Third Affiliated Hospital of Second Military Medical University" />
<meta name="citation_author_institution" content="Qiu-Jiang Bioinformatics Institute" />
<meta name="citation_author" content="Haiyi Gong" />
<meta name="citation_author_institution" content="Department of Orthopaedic Oncology, Changzheng Hospital, Second Military Medical University" />
<meta name="citation_author_institution" content="Qiu-Jiang Bioinformatics Institute" />
<meta name="citation_author" content="Jing Wang" />
<meta name="citation_author_institution" content="Department of Neurosurgery, Changhai hospital, Second Military Medical University" />
<meta name="citation_author" content="Zifu Li" />
<meta name="citation_author_institution" content="Department of Neurosurgery, Changhai hospital, Second Military Medical University" />
<meta name="citation_author" content="Xingang Cui" />
<meta name="citation_author_institution" content="Depanrtment of Urology, The Third Affiliated Hospital of Second Military Medical University" />
<meta name="citation_author" content="Huji Xu" />
<meta name="citation_author_institution" content="Peking-Tsinghua Center for Life Sciences, TsinghuaUniversity" />
<meta name="citation_author_institution" content="Department of Rheumatology and Immunology, Changzheng Hospital, Second Military Medical University" />
<meta name="citation_author" content="Haifeng Wei" />
<meta name="citation_author_institution" content="Department of Orthopaedic Oncology, Changzheng Hospital, Second Military Medical University" />
<meta name="citation_author" content="Xiuwu Pan" />
<meta name="citation_author_institution" content="Depanrtment of Urology, The Third Affiliated Hospital of Second Military Medical University" />
<meta name="citation_author" content="Rongrong Zhu" />
<meta name="citation_author_institution" content="Key Laboratory of Spine and Spinal Cord Injury Repair and Regeneration of Ministry of Education, Orthopaedic Department of Tongji Hospital, School of Life Science and Technology, Tongji University" />
<meta name="citation_author" content="Jianru Xiao" />
<meta name="citation_author_institution" content="Department of Orthopaedic Oncology, Changzheng Hospital, Second Military Medical University" />
<meta name="citation_author" content="Wang Zhou" />
<meta name="citation_author_institution" content="Peking-Tsinghua Center for Life Sciences, TsinghuaUniversity" />
<meta name="citation_author_institution" content="Qiu-Jiang Bioinformatics Institute" />
<meta name="citation_author" content="Liming Cheng" />
<meta name="citation_author_institution" content="Division of Spine, Department of Orthopedics, Tongji Hospital affiliated to Tongji University School of Medicine" />
<meta name="citation_author" content="Jianmin Liu" />
<meta name="citation_author_institution" content="Department of Neurosurgery, Changhai hospital, Second Military Medical University" />
<meta name="citation_author_email" content="chstroke@163.com" />
<meta name="citation_reference" content="Gorbalenya AE, B. S., Baric RS, de Groot RJ, Drosten C, Gulyaeva AA, Haagmans BL, Lauber C, Leontovich AM, Neuman BW, Penzar D, Perlman S, Poon LL, Samborskiy D, Sidorov IA, Sola I, Ziebuhr J. Severe acute respiratory syndrome-related coronavirus: The species and its viruses – a statement of the Coronavirus Study Group. bioRxiv doi:doi: https://doi.org/10.1101/2020.02.07.937862 (2020)." />
<meta name="citation_reference" content="Zhong, N. S. et al. Epidemiology and cause of severe acute respiratory syndrome (SARS) in Guangdong, People’s Republic of China, in February, 2003. Lancet 362, 1353–1358, doi: 10.1016/s0140-6736(03)14630-2 (2003)." />
<meta name="citation_reference" content="Hofmann, H. &amp; Pohlmann, S. Cellular entry of the SARS coronavirus. Trends in microbiology 12, 466–472, doi: 10.1016/j.tim.2004.08.008 (2004)." />
<meta name="citation_reference" content="Chan, J. F. et al. A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster. Lancet, doi: 10.1016/s0140-6736(20)30154-9 (2020)." />
<meta name="citation_reference" content="P Zhou, X. Y., XG Wang, B Hu, L Zhang, W Zhang, HR Si, Y Zhu, B Li, CL Huang, HD Chen, J Chen, Y Luo, H Guo, RD Jiang, MQ Liu, Y Chen, XR Shen, X Wang, XS Zheng, K Zhao, QJ Chen, F Deng, LL Liu, B Yan, FX Zhan, YY Wang, GF Xiao, Zl Shi. A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature, doi:https://doi.org/10.1038/s41586-020-2012-7 (2020)." />
<meta name="citation_reference" content="Tian XL, L. C., Huang A, Xia S, Lu SC, Shi ZL, Lu L, Jiang SB, Yang ZL, Wu YL, Ying TL. Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody. bioRxiv doi:doi: https://doi.org/10.1101/2020.01.28.923011 (2020)." />
<meta name="citation_reference" content="Huang, C. et al. Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. Lancet, doi: 10.1016/s0140-6736(20)30183-5 (2020)." />
<meta name="citation_reference" content="Zhu, N. et al. A Novel Coronavirus from Patients with Pneumonia in China, 2019. The New England journal of medicine, doi:10.1056/NEJMoa2001017 (2020)." />
<meta name="citation_reference" content="Chen, J. Pathogenicity and Transmissibility of 2019-nCoV-A Quick Overview and Comparison with Other Emerging Viruses. Microbes and infection, doi:10.1016/j.micinf.2020.01.004 (2020)." />
<meta name="citation_reference" content="Walls, A. C. et al. Unexpected Receptor Functional Mimicry Elucidates Activation of Coronavirus Fusion. Cell 176, 1026–1039. e1015, doi:10.1016/j.cell.2018.12.028 (2019)." />
<meta name="citation_reference" content="Gallagher, T. M. &amp; Buchmeier, M. J. Coronavirus spike proteins in viral entry and pathogenesis. Virology 279, 371–374, doi:10.1006/viro.2000.0757 (2001)." />
<meta name="citation_reference" content="Gui, M. et al. Cryo-electron microscopy structures of the SARS-CoV spike glycoprotein reveal a prerequisite conformational state for receptor binding. Cell research 27, 119–129, doi:10.1038/cr.2016.152 (2017)." />
<meta name="citation_reference" content="Zhao S. R. J., Musa SS, Yang G, Lou Y, Gao D, Yang L, He D. Preliminary estimation of the basic reproduction number of novel coronavirus (2019-nCoV) in China, from 2019 to 2020: A data-driven analysis in the early phase of the outbreak. bioRxiv, 916395, doi:doi: https://doi.org/10.1101/2020.01.23.916395 (2020)." />
<meta name="citation_reference" content="Zhou, Y. et al. Protease inhibitors targeting coronavirus and filovirus entry. Antiviral research 116, 76–84, doi:10.1016/j.antiviral.2015.01.011 (2015)." />
<meta name="citation_reference" content="Millet, J. K. &amp; Whittaker, G. R. Host cell proteases: Critical determinants of coronavirus tropism and pathogenesis. Virus research 202, 120–134, doi:10.1016/j.virusres.2014.11.021 (2015)." />
<meta name="citation_reference" content="Shirato, K., Kawase, M. &amp; Matsuyama, S. Wild-type human coronaviruses prefer cell-surface TMPRSS2 to endosomal cathepsins for cell entry. Virology 517, 9–15, doi:10.1016/j.virol.2017.11.012 (2018)." />
<meta name="citation_reference" content="Heurich, A. et al. TMPRSS2 and ADAM17 cleave ACE2 differentially and only proteolysis by TMPRSS2 augments entry driven by the severe acute respiratory syndrome coronavirus spike protein. Journal of virology 88, 1293–1307, doi: 10.1128/jvi.02202-13 (2014)." />
<meta name="citation_reference" content="Herter, S. et al. Hepatocyte growth factor is a preferred in vitro substrate for human hepsin, a membrane-anchored serine protease implicated in prostate and ovarian cancers. The Biochemical journal 390, 125–136, doi:10.1042/bj20041955 (2005)." />
<meta name="citation_reference" content="Limburg, H. et al. TMPRSS2 Is the Major Activating Protease of Influenza A Virus in Primary Human Airway Cells and Influenza B Virus in Human Type II Pneumocytes. Journal of virology 93, doi:10.1128/jvi.00649-19 (2019)." />
<meta name="citation_reference" content="Holshue, M. L. et al. First Case of 2019 Novel Coronavirus in the United States. The New England journal of medicine, doi:10.1056/NEJMoa2001191 (2020)." />
<meta name="citation_reference" content="Guan WJ, N. Z., Hu Y, Liang WH, Ou CQ, He JX, Liu L, Shan H, Lei CL, Hui David S.C., Du B Clinical characteristics of 2019 novel coronavirus infection in China. medRxiv, doi:doi: http://dx.doi.org/10.1101/2020.02.06.20020974 (2020)." />
<meta name="citation_reference" content="Perlman, S. &amp; Netland, J. Coronaviruses post-SARS: update on replication and pathogenesis. Nature reviews. Microbiology 7, 439–450, doi:10.1038/nrmicro2147 (2009)." />
<meta name="citation_reference" content="de Wit, E., van Doremalen, N., Falzarano, D. &amp; Munster, V. J. SARS and MERS: recent insights into emerging coronaviruses. Nature reviews. Microbiology 14, 523–534, doi:10.1038/nrmicro.2016.81 (2016)." />
<meta name="citation_reference" content="Lee, P. I. &amp; Hsueh, P. R. Emerging threats from zoonotic coronaviruses-from SARS and MERS to 2019-nCoV. Journal of microbiology, immunology, and infection = Wei mian yu gan ran za zhi, doi:10.1016/j.jmii.2020.02.001 (2020)." />
<meta name="citation_reference" content="Li, W. et al. Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus. Nature 426, 450–454, doi:10.1038/nature02145 (2003)." />
<meta name="citation_reference" content="Kuiken, T. et al. Newly discovered coronavirus as the primary cause of severe acute respiratory syndrome. Lancet 362, 263–270, doi:10.1016/s0140-6736(03)13967-0 (2003)." />
<meta name="citation_reference" content="Nabhan, A. N., Brownfield, D. G., Harbury, P. B., Krasnow, M. A. &amp; Desai, T. J. Single-cell Wnt signaling niches maintain stemness of alveolar type 2 cells. Science (New York, N.Y.) 359, 1118–1123, doi: 10.1126/science.aam6603 (2018)." />
<meta name="citation_reference" content="Barkauskas, C. E. et al. Type 2 alveolar cells are stem cells in adult lung. The Journal of clinical investigation 123, 3025–3036, doi:10.1172/jci68782 (2013)." />
<meta name="citation_reference" content="Kroetz, D. N. et al. Type I Interferon Induced Epigenetic Regulation of Macrophages Suppresses Innate and Adaptive Immunity in Acute Respiratory Viral Infection. PLoS pathogens 11, e1005338, doi:10.1371/journal.ppat.1005338 (2015)." />
<meta name="citation_reference" content="Shen, L. W., Mao, H. J., Wu, Y. L., Tanaka, Y. &amp; Zhang, W. TMPRSS2: A potential target for treatment of influenza virus and coronavirus infections. Biochimie 142, 1–10, doi:10.1016/j.biochi.2017.07.016 (2017)." />
<meta name="citation_reference" content="Shin, W. J. &amp; Seong, B. L. Type II transmembrane serine proteases as potential target for anti-influenza drug discovery. Expert opinion on drug discovery 12, 1139–1152, doi: 10.1080/17460441.2017.1372417 (2017)." />
<meta name="citation_reference" content="Biasini, M. et al. SWISS-MODEL: modelling protein tertiary and quaternary structure using evolutionary information. Nucleic acids research 42, W252–258, doi:10.1093/nar/gku340 (2014)." />
<meta name="citation_reference" content="Pettersen, E. F. et al. UCSF Chimera--a visualization system for exploratory research and analysis. Journal of computational chemistry 25, 1605–1612, doi: 10.1002/jcc.20084 (2004)." />
<meta name="citation_reference" content="Wiehe, K. et al. ZDOCK and RDOCK performance in CAPRI rounds 3, 4, and 5. Proteins 60, 207–213, doi:10.1002/prot.20559 (2005)." />
<meta name="citation_reference" content="Duckert, P., Brunak, S. &amp; Blom, N. Prediction of proprotein convertase cleavage sites. Protein engineering, design &amp; selection: PEDS 17, 107–112, doi: 10.1093/protein/gzh013 (2004)." />
<meta name="citation_reference" content="Madissoon, E. et al. scRNA-seq assessment of the human lung, spleen, and esophagus tissue stability after cold preservation. Genome biology 21, 1, doi: 10.1186/S13059-019-1906-x (2019)." />
<meta name="citation_reference" content="Vieira Braga, F. A. et al. A cellular census of human lungs identifies novel cell states in health and in asthma. Nature medicine 25, 1153–1163, doi: 10.1038/S41591-019-0468-5 (2019)." />
<meta name="citation_reference" content="Reyfman, P. A. et al. Single-Cell Transcriptomic Analysis of Human Lung Provides Insights into the Pathobiology of Pulmonary Fibrosis. American journal of respiratory and critical care medicine 199, 1517–1536, doi: 10.1164/rccm.201712-2410OC (2019)." />
<meta name="citation_reference" content="Valenzi, E. et al. Single-cell analysis reveals fibroblast heterogeneity and myofibroblasts in systemic sclerosis-associated interstitial lung disease. Annals of the rheumatic diseases 78, 1379–1387, doi:10.1136/annrheumdis-2018-214865 (2019)." />
<meta name="citation_reference" content="Zhang, P. et al. Dissecting the Single-Cell Transcriptome Network Underlying Gastric Premalignant Lesions and Early Gastric Cancer. Cell reports 27, 1934–1947. e1935, doi:10.1016/j.celrep.2019.04.052 (2019)." />
<meta name="citation_reference" content="Martin, J. C. et al. Single-Cell Analysis of Crohn’s Disease Lesions Identifies a Pathogenic Cellular Module Associated with Resistance to Anti-TNF Therapy. Cell 178, 1493–1508. e1420, doi:10.1016/j.cell.2019.08.008 (2019)." />
<meta name="citation_reference" content="Smillie, C. S. et al. Intra- and Inter-cellular Rewiring of the Human Colon during Ulcerative Colitis. Cell 178, 714–730. e722, doi:10.1016/j.cell.2019.06.029 (2019)." />
<meta name="citation_reference" content="Stuart, T. et al. Comprehensive Integration of Single-Cell Data. Cell 177, 1888–1902. e1821, doi: 10.1016/j.cell.2019.05.031 (2019)." />
<meta name="citation_reference" content="Welch, J. D. et al. Single-Cell Multi-omic Integration Compares and Contrasts Features of Brain Cell Identity. Cell 177, 1873–1887. e1817, doi:10.1016/j.cell.2019.05.006 (2019)." />
<meta name="citation_reference" content="Rouillard, A. D. et al. The harmonizome: a collection of processed datasets gathered to serve and mine knowledge about genes and proteins. Database: the journal of biological databases and curation 2016, doi:10.1093/database/baw100 (2016)." />
<meta name="citation_reference" content="Human genomics. The Genotype-Tissue Expression (GTEx) pilot analysis: multitissue gene regulation in humans. Science (New York, N.Y.) 348, 648–660, doi: 10.1126/science.1262110 (2015)." />
<meta name="citation_reference" content="Uhlen, M. et al. Proteomics. Tissue-based map of the human proteome. Science (New York, N.Y.) 347, 1260419, doi:10.1126/science.1260419 (2015)." />
<meta name="twitter:title" content="The insert sequence in SARS-CoV-2 enhances spike protein cleavage by TMPRSS" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="At the end of 2019, the SARS-CoV-2 induces an ongoing outbreak of pneumonia in China[1][1], even more spread than SARS-CoV infection[2][2]. The entry of SARS-CoV into host cells mainly depends on the cell receptor (ACE2) recognition and spike protein cleavage-induced cell membrane fusion[3][3],[4][4]. The spike protein of SARS-CoV-2 also binds to ACE2 with a similar affinity, whereas its spike protein cleavage remains unclear[5][5],[6][6]. Here we show that an insertion sequence in the spike protein of SARS-CoV-2 enhances the cleavage efficiency, and besides pulmonary alveoli, intestinal and esophagus epithelium were also the target tissues of SARS-CoV-2. Compared with SARS-CoV, we found a SPRR insertion in the S1/S2 protease cleavage sites of SARS-CoV-2 spike protein increasing the cleavage efficiency by the protein sequence aligment and furin score calculation. Additionally, the insertion sequence facilitates the formation of an extended loop which was more suitable for protease recognition by the homology modeling and molicular docking. Furthermore, the single-cell transcriptomes identified that ACE2 and TMPRSSs are highly coexpressed in AT2 cells of lung, along with esophageal upper epithelial cells and absorptive enterocytes. Our results provide the bioinformatics evidence for the increased spike protein cleavage of SARS-CoV-2 and indicate its potential target cells.

 [1]: #ref-1
 [2]: #ref-2
 [3]: #ref-3
 [4]: #ref-4
 [5]: #ref-5
 [6]: #ref-6" />
<meta name="og-title" property="og:title" content="The insert sequence in SARS-CoV-2 enhances spike protein cleavage by TMPRSS" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.08.926006v3" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="At the end of 2019, the SARS-CoV-2 induces an ongoing outbreak of pneumonia in China[1][1], even more spread than SARS-CoV infection[2][2]. The entry of SARS-CoV into host cells mainly depends on the cell receptor (ACE2) recognition and spike protein cleavage-induced cell membrane fusion[3][3],[4][4]. The spike protein of SARS-CoV-2 also binds to ACE2 with a similar affinity, whereas its spike protein cleavage remains unclear[5][5],[6][6]. Here we show that an insertion sequence in the spike protein of SARS-CoV-2 enhances the cleavage efficiency, and besides pulmonary alveoli, intestinal and esophagus epithelium were also the target tissues of SARS-CoV-2. Compared with SARS-CoV, we found a SPRR insertion in the S1/S2 protease cleavage sites of SARS-CoV-2 spike protein increasing the cleavage efficiency by the protein sequence aligment and furin score calculation. Additionally, the insertion sequence facilitates the formation of an extended loop which was more suitable for protease recognition by the homology modeling and molicular docking. Furthermore, the single-cell transcriptomes identified that ACE2 and TMPRSSs are highly coexpressed in AT2 cells of lung, along with esophageal upper epithelial cells and absorptive enterocytes. Our results provide the bioinformatics evidence for the increased spike protein cleavage of SARS-CoV-2 and indicate its potential target cells.

 [1]: #ref-1
 [2]: #ref-2
 [3]: #ref-3
 [4]: #ref-4
 [5]: #ref-5
 [6]: #ref-6" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-02-16" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.02.08.926006v3.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.02.08.926006v3.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.02.08.926006v3.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.02.08.926006v3" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1153713" />
    <title>The insert sequence in SARS-CoV-2 enhances spike protein cleavage by TMPRSS | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ixQ2XJG6oVyN5Fhr_5aJZn2x6IKDMlFvVLKPzgr-WBU","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1153713","apath":"\/biorxiv\/early\/2020\/02\/16\/2020.02.08.926006.atom","pisa":"biorxiv;2020.02.08.926006v3","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.02.08.926006v3"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.02.08.926006v3","title":"The insert sequence in SARS-CoV-2 enhances spike protein cleavage by TMPRSS","identifier":"node\/1153713"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Tdecu+5+fG46zAwrcFN0y2sDTCZClyvBNYgFcbgMhQZF\\\\\\\/xmy7bP9IRzGPL+eM5nffrCjLLXk+Pq7SaF5S2IZ4Irw9cgpedAg7PoIEfo3V4eeIR5rASgZlfLJqRjBXNfdigQIEDOEJRb8D4W2F8D4gWzN8AuvDJXuUSuIEd9hcOZVOjQUuJ3y9c48miz0ivJqcrqn0UgNRrnAbEvuEfIxSAUY0EJvzHwbHs7Th3TLjHpKXIwyhDvq2SxPayTfTvYnHlmq0btI3rLC7Yh67af0EKtRNfhtRNb1MTd\\\\\\\/2OXGcNhTYlL7ks1xeHpC5sUXX2DqtpKD8I2SmrXniNommNfk1aujLkBW\\\\\\\/caHIWKd9\\\\\\\/yJ10SqLX+iQA4LQzQMlmL3F9rBzzyzeDIzfpCF6v8JulY81b3q7SGKD2yvC8HmHyHtnfyd9a+0Z3bM0YcJght1Z52guuSQAblN3mT6NqXqouzyyFulI9MnM2JquOEkytty6GCG4IiPTOIn4136mIlJAZGZwE6k3II+g05DdTRDiGMO50fn5K05yMxlPoA7DLWichw83P2R2elqd50m9tdvyViQ19PO6YVQHpoN8CWofQKQqYAd4HyXSrZPHN2dxR+wddtacM5Ptvblg10UMwlsdLnaxsA7Dq+0UeC2f5Dp\\\\\\\/eOgnIZNm\\\\\\\/gRfgZdfsF4AHTuLKDPsL0G0TxsHlAjeKJdrhBHFh6Z86OZyinySPqawrRv+XrnCNvbOIVco3D87wMbOMFMf29JYJQMzgCJs62aQsf4PkVwdo+TiuyrFCHkJRqGGWA9LUZx09WjwfmYTxSFkm+R0gwKYNgpexgHEEdYyz6jcpXF5EDHVbwq+cppC5Ipu7Gih27FHmg0cOUj8cvUd\\\\\\\/OaKTf7lJxUMkytsJ+xwMS\\\\\\\/BO5SM5wyPaaioA0L2l6jtNlbRE5BNHL7nx9qK2rCBVdkIpw7hgc\\\\\\\/\\\\\\\/GmteypzTj22MOk3QefJNI6Qhjwi4YuCh\\\\\\\/9K3k5eZI1K4eRj1RIsGgZD\\\\\\\/ecJm0DWqXvjPjp38+XAoDr8RqWC1snkzy4yAe3BTJV0gPROMK6zDV0JBXRvX6jaRsdh4GA5FmkIjak\\\\\\\/bdX3RJmx9HmKodhqxDH4pnRcEAjxvI96qnSdgSxooonwAf4n9z\\\\\\\/jPq\\\\\\\/8W8DjAtoS8N0e+5ixOdhw8mpQtOq\\\\\\\/iOMdsWtRFymbovOHvcApHu1j\\\\\\\/Rnd8P9LC+gHuEXwD7hClsJPaGdSZ+7WqVzL9jHL8mTZsbHTcPPI7eribMzD5hRO0sdgLqjK74SrmKt2Z\\\\\\\/QU5yzD5vQta5QrYfm2\\\\\\\/8CMKZ3FYFpgJV2CLsB9AD80kAoGBKa\\\\\\\/VMsSHsIXV4dMVLSVTBog\\\u0022,\\\u0022iv\\\u0022:\\\u0022Kjh2uCBH7vrfn2Ox5IP9iQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022d2dcdfa7f98c68c94288199ada737c4d9bec32256819b13ddfae4a4a8fa9079c\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.02.08.926006v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1153713 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.02.08.926006v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Mb6SFPwYB1ATKi9oepAcibd9_17WdGxiyHE5OfrPEdc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1153713" id="node1153713" data-pisa="biorxiv;2020.02.08.926006v3" data-pisa-master="biorxiv;2020.02.08.926006" data-apath="/biorxiv/early/2020/02/16/2020.02.08.926006.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">The insert sequence in SARS-CoV-2 enhances spike protein cleavage by TMPRSS</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Tong</span> <span class="nlm-surname">Meng</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Hao</span> <span class="nlm-surname">Cao</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Hao</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Zijian</span> <span class="nlm-surname">Kang</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Da</span> <span class="nlm-surname">Xu</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Haiyi</span> <span class="nlm-surname">Gong</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Jing</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Zifu</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Xingang</span> <span class="nlm-surname">Cui</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Huji</span> <span class="nlm-surname">Xu</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Haifeng</span> <span class="nlm-surname">Wei</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Xiuwu</span> <span class="nlm-surname">Pan</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Rongrong</span> <span class="nlm-surname">Zhu</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Jianru</span> <span class="nlm-surname">Xiao</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Wang</span> <span class="nlm-surname">Zhou</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Liming</span> <span class="nlm-surname">Cheng</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">Jianmin</span> <span class="nlm-surname">Liu</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.02.08.926006 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1153713" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Tong Meng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Division of Spine, Department of Orthopedics, Tongji Hospital affiliated to Tongji University School of Medicine</span>, 200065 Shanghai, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Tongji University Cancer Center, School of Medicine, Tongji University</span>, 200092 Shanghai, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>10</span><span class='nlm-institution'>Qiu-Jiang Bioinformatics Institute</span>, 200003 Shanghai, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tong%2BMeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meng%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATong%2BMeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Hao Cao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>School of Life Science and Biopharmaceutics, Shenyang Pharmaceutical University</span>, 103 Wenhua Road, 110016 Shenyang, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Peking-Tsinghua Center for Life Sciences, TsinghuaUniversity</span>, 100084 Beijing, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hao%2BCao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cao%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHao%2BCao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Hao Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Department of Orthopaedic Oncology, Changzheng Hospital, Second Military Medical University</span>, 200003 Shanghai, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>10</span><span class='nlm-institution'>Qiu-Jiang Bioinformatics Institute</span>, 200003 Shanghai, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hao%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHao%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Zijian Kang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Department of Rheumatology and Immunology, Changzheng Hospital, Second Military Medical University</span>, 200003 Shanghai, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>10</span><span class='nlm-institution'>Qiu-Jiang Bioinformatics Institute</span>, 200003 Shanghai, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zijian%2BKang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZijian%2BKang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Da Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>Depanrtment of Urology, The Third Affiliated Hospital of Second Military Medical University</span>, 201805 Shanghai, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>10</span><span class='nlm-institution'>Qiu-Jiang Bioinformatics Institute</span>, 200003 Shanghai, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Da%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADa%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Haiyi Gong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Department of Orthopaedic Oncology, Changzheng Hospital, Second Military Medical University</span>, 200003 Shanghai, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>10</span><span class='nlm-institution'>Qiu-Jiang Bioinformatics Institute</span>, 200003 Shanghai, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haiyi%2BGong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gong%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHaiyi%2BGong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jing Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>8</span><span class='nlm-institution'>Department of Neurosurgery, Changhai hospital, Second Military Medical University</span>, 200003 Shanghai, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jing%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJing%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Zifu Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>8</span><span class='nlm-institution'>Department of Neurosurgery, Changhai hospital, Second Military Medical University</span>, 200003 Shanghai, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zifu%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZifu%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Xingang Cui </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>Depanrtment of Urology, The Third Affiliated Hospital of Second Military Medical University</span>, 201805 Shanghai, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xingang%2BCui%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cui%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXingang%2BCui%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Huji Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Peking-Tsinghua Center for Life Sciences, TsinghuaUniversity</span>, 100084 Beijing, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Department of Rheumatology and Immunology, Changzheng Hospital, Second Military Medical University</span>, 200003 Shanghai, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Huji%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHuji%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Haifeng Wei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Department of Orthopaedic Oncology, Changzheng Hospital, Second Military Medical University</span>, 200003 Shanghai, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haifeng%2BWei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wei%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHaifeng%2BWei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Xiuwu Pan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>Depanrtment of Urology, The Third Affiliated Hospital of Second Military Medical University</span>, 201805 Shanghai, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiuwu%2BPan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pan%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiuwu%2BPan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Rongrong Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>9</span><span class='nlm-institution'>Key Laboratory of Spine and Spinal Cord Injury Repair and Regeneration of Ministry of Education, Orthopaedic Department of Tongji Hospital, School of Life Science and Technology, Tongji University</span>, 200092 Shanghai, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rongrong%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARongrong%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Jianru Xiao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Department of Orthopaedic Oncology, Changzheng Hospital, Second Military Medical University</span>, 200003 Shanghai, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jianru%2BXiao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xiao%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJianru%2BXiao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Wang Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Peking-Tsinghua Center for Life Sciences, TsinghuaUniversity</span>, 100084 Beijing, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>10</span><span class='nlm-institution'>Qiu-Jiang Bioinformatics Institute</span>, 200003 Shanghai, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wang%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWang%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Liming Cheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Division of Spine, Department of Orthopedics, Tongji Hospital affiliated to Tongji University School of Medicine</span>, 200065 Shanghai, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Liming%2BCheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cheng%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALiming%2BCheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-16"><div class="author-tooltip-name">Jianmin Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>8</span><span class='nlm-institution'>Department of Neurosurgery, Changhai hospital, Second Military Medical University</span>, 200003 Shanghai, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jianmin%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJianmin%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:chstroke@163.com" class="" data-icon-position="" data-hide-link-title="0">chstroke@163.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.02.08.926006v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1153713" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1153713/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.08.926006v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1153713" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1153713/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.08.926006v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1153713" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1153713/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.08.926006v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1153713" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1153713/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.02.08.926006v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1153713" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1153713/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-4">At the end of 2019, the SARS-CoV-2 induces an ongoing outbreak of pneumonia in China<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a></sup>, even more spread than SARS-CoV infection<sup><a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a></sup>. The entry of SARS-CoV into host cells mainly depends on the cell receptor (ACE2) recognition and spike protein cleavage-induced cell membrane fusion<sup><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>,<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></sup>. The spike protein of SARS-CoV-2 also binds to ACE2 with a similar affinity, whereas its spike protein cleavage remains unclear<sup><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>,<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a></sup>. Here we show that an insertion sequence in the spike protein of SARS-CoV-2 enhances the cleavage efficiency, and besides pulmonary alveoli, intestinal and esophagus epithelium were also the target tissues of SARS-CoV-2. Compared with SARS-CoV, we found a SPRR insertion in the S1/S2 protease cleavage sites of SARS-CoV-2 spike protein increasing the cleavage efficiency by the protein sequence aligment and furin score calculation. Additionally, the insertion sequence facilitates the formation of an extended loop which was more suitable for protease recognition by the homology modeling and molicular docking. Furthermore, the single-cell transcriptomes identified that ACE2 and TMPRSSs are highly coexpressed in AT2 cells of lung, along with esophageal upper epithelial cells and absorptive enterocytes. Our results provide the bioinformatics evidence for the increased spike protein cleavage of SARS-CoV-2 and indicate its potential target cells.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-6">At the end of 2019, a rising number of pneumonia patients with unknown pathogen emerged from Wuhan to nearly the entire China<sup><a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></sup>. A novel coronavirus was isolated and based on its phylogeny, taxonomy and established practice, the Coronavirus Study Group (CSG) recognized it as a sister to severe acute respiratory syndrome coronaviruses (SARS-CoVs) and labeled it as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)<sup><a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>,<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a></sup>. Although SARS-CoV-2 is generally less pathogenic than SARS-CoV and Middle East respiratory syndrome coronavirus (MERS-CoV), it has a relatively high transmissibility<sup><a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a></sup>.</p><p id="p-7">With regard to human coronavirus, the transmissibility and infectivity is largely controlled by the spike (S) surface envelope protein<sup><a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></sup>. Its surface unit (S1) mediates the entry into host cells by binding to cell receptor and the transmembrane unit (S2) subunit regulates the fusion of viral and cellular membranes<sup><a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a></sup>. Prior to membrane fusion, the S protein should be cleaved and activated to allow for the fusion peptide releasing onto host cell membranes (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1a</a>)<sup><a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a></sup>. SARS-CoV-2 uses the same cell receptor (angiotensin converting enzyme II, ACE2) as SARS-CoV, with a similar binding affinity, whereas their transmissibility and infectivity are different<sup><a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>,<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>,<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></sup> Thus, the different virus transmission and infectivity may be associated with the differentiated protease-induced S protein cleavage between SARS-CoV-2 and SARS-CoV.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The schematic diagram of the project. a. The entry of SARS-CoV into host cells: The spike protein of SARS-CoV binds to ACE2 through its S1 subunit for viral recognition. Then it is cleaved by TMPRSS2 at the S1/S2 boundary or within S2 subunit, which removes the structural constraint of S1 on S2, and releases the internal fusion peptide combined with the Spike TM domain for the fusion of viral and cellular membranes. Finally, the viral genomes enter into the host cells. b. ACE2 cleaving by TMPRSSs: TMPRSS2 can also cleave ACE2 amino acids 697 to 716, resulting in the shedding of 13kD ACE2 fragment in culture supernatants and augmented viral infectivity. c. The difference between SARS-CoV-2 and SARS-CoV in the Spike protein cleavage: The Spike protein of SARS involves two cleavage sites recognized by TMPRSSs, one at arginine 667 and the other at arginine 797 (right). Compared with SARS-CoV, the Spike protein of SARS-CoV-2 (left) has an insertion sequence 680-SPRR-683 at the S1/S2 cleavage site. We speculated that R682, R683 and R685 (red box) could be used as the most suitable substrates for TMPRSSs, which can increase the Spike protein cleavage efficiency of TMPRSSs, promote its activation and enhance SARS-CoV-2 infection." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1685871455" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;The schematic diagram of the project.&lt;/span&gt; a. The entry of SARS-CoV into host cells: The spike protein of SARS-CoV binds to ACE2 through its S1 subunit for viral recognition. Then it is cleaved by TMPRSS2 at the S1/S2 boundary or within S2 subunit, which removes the structural constraint of S1 on S2, and releases the internal fusion peptide combined with the Spike TM domain for the fusion of viral and cellular membranes. Finally, the viral genomes enter into the host cells. b. ACE2 cleaving by TMPRSSs: TMPRSS2 can also cleave ACE2 amino acids 697 to 716, resulting in the shedding of 13kD ACE2 fragment in culture supernatants and augmented viral infectivity. c. The difference between SARS-CoV-2 and SARS-CoV in the Spike protein cleavage: The Spike protein of SARS involves two cleavage sites recognized by TMPRSSs, one at arginine 667 and the other at arginine 797 (right). Compared with SARS-CoV, the Spike protein of SARS-CoV-2 (left) has an insertion sequence 680-SPRR-683 at the S1/S2 cleavage site. We speculated that R682, R683 and R685 (red box) could be used as the most suitable substrates for TMPRSSs, which can increase the Spike protein cleavage efficiency of TMPRSSs, promote its activation and enhance SARS-CoV-2 infection.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F1.medium.gif" width="393" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F1.medium.gif" width="393" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Fig. 1.</span> <span class="caption-title">The schematic diagram of the project.</span><p id="p-8" class="first-child">a. The entry of SARS-CoV into host cells: The spike protein of SARS-CoV binds to ACE2 through its S1 subunit for viral recognition. Then it is cleaved by TMPRSS2 at the S1/S2 boundary or within S2 subunit, which removes the structural constraint of S1 on S2, and releases the internal fusion peptide combined with the Spike TM domain for the fusion of viral and cellular membranes. Finally, the viral genomes enter into the host cells.</p><p id="p-9">b. ACE2 cleaving by TMPRSSs: TMPRSS2 can also cleave ACE2 amino acids 697 to 716, resulting in the shedding of 13kD ACE2 fragment in culture supernatants and augmented viral infectivity.</p><p id="p-10">c. The difference between SARS-CoV-2 and SARS-CoV in the Spike protein cleavage: The Spike protein of SARS involves two cleavage sites recognized by TMPRSSs, one at arginine 667 and the other at arginine 797 (right). Compared with SARS-CoV, the Spike protein of SARS-CoV-2 (left) has an insertion sequence 680-SPRR-683 at the S1/S2 cleavage site. We speculated that R682, R683 and R685 (red box) could be used as the most suitable substrates for TMPRSSs, which can increase the Spike protein cleavage efficiency of TMPRSSs, promote its activation and enhance SARS-CoV-2 infection.</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">The transmembrane serine proteases (TMPRSSs) were the main host cell proteases on the cell membrane<sup><a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></sup>. The substrate specificity of TMPRSSs are almost similar and revealing a strong preference for arginine or lysine residues in the P1 position. Nowadays, their hydrolytic effects of TMPRSSs have been widely reported in SARS-CoV and MERS-CoV pneumonia<sup><a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a></sup>. In the SARS-CoV-infected alveolar cells, TMPRSSs, especially the TMPRSS2 and TMPRSS11D, cleave the SARS-CoV S protein (SARS-S) at residue R667 (the S1/S2 cleavage site) and residue R797 (the S2’ cleavage site) (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1a</a>)<sup><a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>,<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a></sup>. Besides cleaving S protein, they can also promote viral spread in the host by cleaving ACE2 (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1b</a>)<sup><a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>,<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup>. Although SARS-CoV-2 and SARS-CoV share the same host cell receptor with a similar affinity, however, the SARS-CoV-2 S protein cleavage induced by TMPRSS remains unclear which may be associated with the viral infectivity<sup><a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>,<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">5</a></sup>.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3>The comparison of the S1/S2 and S2’ cleavage sites between SARS-CoV-2 and SARS-CoV</h3><p id="p-12">Generally, compared with SARS-CoV, the major differences in SARS-CoV-2 are the three short insertions in the N-terminal domain and four out of five key residues changes in the receptor-binding motif<sup><a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">5</a></sup>. Here we used the alignment, furin score and homology modeling to compare the sequence of the S1/S2 and S2’ cleavage sites between SARS-CoV-2 and SARS-CoV (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1c</a>). The amino acid sequence of the S1/S2 and S2’ cleavage sites among ten beta-coronavirus were then analyzed and we found that compared with SARS, there was an insertion sequence (SPRR) in the S1/S2 cleavage sites of SARS-CoV-2 (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2a</a>). The furin score was next used to identify the cleavage efficiency of the insertion sequence in SARS-CoV-2. Its furin score was 0.688, which was obviously higher than that of the corresponding sequence in SARS-CoV (0.139), indicating that the insertion sequence may increase the cleavage efficiency by proteases (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2b</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The two potential Spike protein cleavage sites of SARS-CoV and SARS-CoV-2 by TMPRSS2. a. Phylogenetic tree based on the protein sequences of Spike protein in SARS-CoV-2, SARS-CoV and other eight beta-coronaviruses are presented, along with the amino acid sequence alignment of two potential cleavage sites by TMPRSS2. b. The putative furin scores of the two potential cleavage sites of the ten coronaviruses. c. Structure comparison of the detailed Spike protein of the SARS-CoV and SARS-CoV-2. The insert 675-690 of SARS-CoV-2 Spike protein (yellow) and the corresponding loci to SARS-CoV Spike protein 661-672 (green). Three important residues, R682, R683, R685, are specially marked. d. The detail of c. The similarly SARS-CoV R797 with SARS-CoV-2 R815 are marked with forest green and orange, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1685871455" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;The two potential Spike protein cleavage sites of SARS-CoV and SARS-CoV-2 by TMPRSS2.&lt;/span&gt; a. Phylogenetic tree based on the protein sequences of Spike protein in SARS-CoV-2, SARS-CoV and other eight beta-coronaviruses are presented, along with the amino acid sequence alignment of two potential cleavage sites by TMPRSS2. b. The putative furin scores of the two potential cleavage sites of the ten coronaviruses. c. Structure comparison of the detailed Spike protein of the SARS-CoV and SARS-CoV-2. The insert 675-690 of SARS-CoV-2 Spike protein (yellow) and the corresponding loci to SARS-CoV Spike protein 661-672 (green). Three important residues, R682, R683, R685, are specially marked. d. The detail of c. The similarly SARS-CoV R797 with SARS-CoV-2 R815 are marked with forest green and orange, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F2.medium.gif" width="389" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F2.medium.gif" width="389" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 2.</span> <span class="caption-title">The two potential Spike protein cleavage sites of SARS-CoV and SARS-CoV-2 by TMPRSS2.</span><p id="p-13" class="first-child">a. Phylogenetic tree based on the protein sequences of Spike protein in SARS-CoV-2, SARS-CoV and other eight beta-coronaviruses are presented, along with the amino acid sequence alignment of two potential cleavage sites by TMPRSS2.</p><p id="p-14">b. The putative furin scores of the two potential cleavage sites of the ten coronaviruses.</p><p id="p-15">c. Structure comparison of the detailed Spike protein of the SARS-CoV and SARS-CoV-2. The insert 675-690 of SARS-CoV-2 Spike protein (yellow) and the corresponding loci to SARS-CoV Spike protein 661-672 (green). Three important residues, R682, R683, R685, are specially marked.</p><p id="p-16">d. The detail of c. The similarly SARS-CoV R797 with SARS-CoV-2 R815 are marked with forest green and orange, respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">The structures of SARS-S and SARS-CoV-2 S protein were presented in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Extended Data Fig. 1a</a> and <a id="xref-fig-5-2" class="xref-fig" href="#F5">1b</a>, along with their structural superimposition (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Extended Data Fig. 1c</a>). The structural comparison of homology modeling SARS-CoV-2 S protein with SARS-S protein (PDB: 5×5b) showed that a exposed loop was formed by the insertion which comprised R682 and R683 (S1/S2 site) on the surface of SARS-CoV-2 S protein, and no significant difference of them in S2’ site (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2c, d</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The overall structure of the Spike protein in SARS-CoV and SARS-CoV-2 homo-trimers a. The structure of the SARS-CoV Spike protein (from PDB: 5X5B). The insert aa675-690 to SARS-CoV Spike protein aa661-672 with the structural missed residues are marked with green. b. The structure of the SARS-CoV-2 Spike protein (Modelled by SWISS-MODEL). The insert aa675-690 of 2019-nCoV Spike protein that corresponds to the insert region of SARS-V Spike protein is marked with yellow. c. The structural superimpose of Spike protein in the SARS-CoV (yellow) and SARS-CoV-2 (blue)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1685871455" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;The overall structure of the Spike protein in SARS-CoV and SARS-CoV-2 homo-trimers&lt;/span&gt; a. The structure of the SARS-CoV Spike protein (from PDB: 5X5B). The insert aa675-690 to SARS-CoV Spike protein aa661-672 with the structural missed residues are marked with green. b. The structure of the SARS-CoV-2 Spike protein (Modelled by SWISS-MODEL). The insert aa675-690 of 2019-nCoV Spike protein that corresponds to the insert region of SARS-V Spike protein is marked with yellow. c. The structural superimpose of Spike protein in the SARS-CoV (yellow) and SARS-CoV-2 (blue).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Extended Data Fig. 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F5.medium.gif" width="440" height="413"/><noscript><img class="highwire-fragment fragment-image" alt="Extended Data Fig. 1" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F5.medium.gif" width="440" height="413"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Extended Data Fig. 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Extended Data Fig. 1</span> <span class="caption-title">The overall structure of the Spike protein in SARS-CoV and SARS-CoV-2 homo-trimers</span><p id="p-51" class="first-child">a. The structure of the SARS-CoV Spike protein (from PDB: 5X5B). The insert aa675-690 to SARS-CoV Spike protein aa661-672 with the structural missed residues are marked with green.</p><p id="p-52">b. The structure of the SARS-CoV-2 Spike protein (Modelled by SWISS-MODEL). The insert aa675-690 of 2019-nCoV Spike protein that corresponds to the insert region of SARS-V Spike protein is marked with yellow.</p><p id="p-53">c. The structural superimpose of Spike protein in the SARS-CoV (yellow) and SARS-CoV-2 (blue).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>The insertion sequence of SARS-CoV-2 facilitating the TMPRSS recognition and S protein cleavage</h3><p id="p-18">Structurally, TMPRSSs include extracellular domain, transmembrane domain and intracellular domain in which extracellular domain is the main catalytic domain. They show similar substrate-specificity and catalytic mechanism. Take TMPRSS2 as an example. The catalytic triad consisted of H296, D345 and S441 and the substrate binding residue D435, a conserved aspartate residue, was located in the bottom of pocket<sup><a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>,<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a></sup>. The substrate binding pocket is deeper than most of serine proteinase (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Extended Data Fig. 2a, b</a>). The bottom of the catalytic pocket has a negatively charged aspartic acid residue which can facilitate the binding and stabilization of arginine or lysine residues in the P1 position<sup><a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>,<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a></sup>.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The structure and catalytic mechanism of TMPRSS2 a-b. The overall structure and surface of TMPRSS2 (Modelled by SWISS-MODEL). The TMPRSS2, catalytic triad comprised of H296, D345 and S441 are marked with cyan, blue, cyan and green, respectively. The substrate binding residue D435 located in the bottom of pocket is marked with red. c. The polypeptide substrate analogue KQLR. The cleavage site Arg is marked with orange. Gln and Leu are marked with yellow. Lys is marked with pink. d. The state of substrate analogue binding in the catalytic pocket. The state of substrate analogue binding in the catalytic pocket. e. The detail of d. Arg of substrate analogue is strongly interacted with D435 f. The predicted state of SARS-CoV-2 Spike protein binding to the catalytic pocket of TMPRSS2. g. The detail of f. SARS-CoV-2 Spike protein and D345 of TMPRSS2 are marked with wheat and medium blue, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1685871455" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;The structure and catalytic mechanism of TMPRSS2&lt;/span&gt; a-b. The overall structure and surface of TMPRSS2 (Modelled by SWISS-MODEL). The TMPRSS2, catalytic triad comprised of H296, D345 and S441 are marked with cyan, blue, cyan and green, respectively. The substrate binding residue D435 located in the bottom of pocket is marked with red. c. The polypeptide substrate analogue KQLR. The cleavage site Arg is marked with orange. Gln and Leu are marked with yellow. Lys is marked with pink. d. The state of substrate analogue binding in the catalytic pocket. The state of substrate analogue binding in the catalytic pocket. e. The detail of d. Arg of substrate analogue is strongly interacted with D435 f. The predicted state of SARS-CoV-2 Spike protein binding to the catalytic pocket of TMPRSS2. g. The detail of f. SARS-CoV-2 Spike protein and D345 of TMPRSS2 are marked with wheat and medium blue, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Extended Data Fig. 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F6.medium.gif" width="436" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Extended Data Fig. 2" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F6.medium.gif" width="436" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Extended Data Fig. 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Extended Data Fig. 2</span> <span class="caption-title">The structure and catalytic mechanism of TMPRSS2</span><p id="p-54" class="first-child">a-b. The overall structure and surface of TMPRSS2 (Modelled by SWISS-MODEL). The TMPRSS2, catalytic triad comprised of H296, D345 and S441 are marked with cyan, blue, cyan and green, respectively. The substrate binding residue D435 located in the bottom of pocket is marked with red.</p><p id="p-55">c. The polypeptide substrate analogue KQLR. The cleavage site Arg is marked with orange. Gln and Leu are marked with yellow. Lys is marked with pink.</p><p id="p-56">d. The state of substrate analogue binding in the catalytic pocket. The state of substrate analogue binding in the catalytic pocket.</p><p id="p-57">e. The detail of d. Arg of substrate analogue is strongly interacted with D435</p><p id="p-58">f. The predicted state of SARS-CoV-2 Spike protein binding to the catalytic pocket of TMPRSS2.</p><p id="p-59">g. The detail of f. SARS-CoV-2 Spike protein and D345 of TMPRSS2 are marked with wheat and medium blue, respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">Polypeptide substrate analogue KQLR included arginine, glutamine, leucine and lysine (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Extended Data Fig. 2c</a>). The substrate analogue could bind to the catalytic pocket of TMPRSS2 (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Extended Data Fig. 2d, e</a>). The conformation of the insertion sequence in SARS-CoV-2 S protein and TMPRSS2 was next simulated by molecular docking. We found the insertion sequence formed a loop which was easily recognized by the catalytic pocket of TMPRSS2 (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Extended Data Fig. 2f, g</a>). Thus, both the furin score and molecular docking revealed that the insertion sequence of SARS-CoV-2 facilitates the TMPRSS2 recognition and S protein cleavage.</p></div><div id="sec-5" class="subsection"><h3>The potential target tissues of COVID-19</h3><p id="p-20">The entry of SARS-CoV-2 into host cells depends on the cell receptor recognition and cell proteases cleaving. Thus, the target cells should coexpress both the cell receptor ACE2 and cell proteases TMPRSSs. In order to identify the coexpressing cell composition and proportion, we utilized 3 datasets including 32 samples and built the largest single-cell transcriptome atlas of normal lung, the commonest infected organ of SARS-CoV-2.</p><p id="p-21">After initial quality controls, a total of 113,045 cells and 29 sub-clusters were identified in the lung (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3a</a>). The marker genes and dataset proportions of each sub-cluster were presented in <a id="xref-fig-7-1" class="xref-fig" href="#F7">Extended Data Fig. 3</a>–<a id="xref-fig-8-1" class="xref-fig" href="#F8">4</a>.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Subset-specific markers. a. The heatmap of marker genes (rows) across cell subsets (columns). The bubble diagram of marker genes in thirteen clusters (b) and the sub-clusters of T cells (c), B cells (d) and Myeloid cells (e)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1685871455" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Subset-specific markers.&lt;/span&gt; a. The heatmap of marker genes (rows) across cell subsets (columns). The bubble diagram of marker genes in thirteen clusters (b) and the sub-clusters of T cells (c), B cells (d) and Myeloid cells (e).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Extended Data Fig. 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F7.medium.gif" width="428" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Extended Data Fig. 3" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F7.medium.gif" width="428" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Extended Data Fig. 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Extended Data Fig. 3</span> <span class="caption-title">Subset-specific markers.</span><p id="p-60" class="first-child">a. The heatmap of marker genes (rows) across cell subsets (columns). The bubble diagram of marker genes in thirteen clusters (b) and the sub-clusters of T cells (c), B cells (d) and Myeloid cells (e).</p><div class="sb-div caption-clear"></div></div></div><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="All cell subset distributions across samples. The fractions of cells (y axis) in each cell subset (bars) that are derived from each sample in 3 databases (red, green and blue). The numbers of cells in each cluster are labeled above." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1685871455" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;All cell subset distributions across samples.&lt;/span&gt; The fractions of cells (y axis) in each cell subset (bars) that are derived from each sample in 3 databases (red, green and blue). The numbers of cells in each cluster are labeled above.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Extended Data Fig. 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F8.medium.gif" width="440" height="232"/><noscript><img class="highwire-fragment fragment-image" alt="Extended Data Fig. 4" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F8.medium.gif" width="440" height="232"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Extended Data Fig. 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Extended Data Fig. 4</span> <span class="caption-title">All cell subset distributions across samples.</span><p id="p-61" class="first-child">The fractions of cells (y axis) in each cell subset (bars) that are derived from each sample in 3 databases (red, green and blue). The numbers of cells in each cluster are labeled above.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Single-cell analysis of the normal lung tissue. a. The UMAP plots of the landscape of lung cells. Thirteen clusters are colored, distinctively labeled. T, B and myeloid cell subsets are further divided into finer cell subsets according to the heterogeneity within the cell population. b. The feature plots of the 17 TMPRSS genes, ACE2, TMPRSS1 and TMPRSS2. c. The expression of ACE2 across clusters in the violin plot. The expression is measured as the log2 (TP10K+1) value. d. The mean expression of TMPRSS family genes across clusters in the boxplot. The expression is measured as the mean log2 (TP10K+1) value." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1685871455" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Single-cell analysis of the normal lung tissue.&lt;/span&gt; a. The UMAP plots of the landscape of lung cells. Thirteen clusters are colored, distinctively labeled. T, B and myeloid cell subsets are further divided into finer cell subsets according to the heterogeneity within the cell population. b. The feature plots of the 17 TMPRSS genes, ACE2, TMPRSS1 and TMPRSS2. c. The expression of ACE2 across clusters in the violin plot. The expression is measured as the log2 (TP10K+1) value. d. The mean expression of TMPRSS family genes across clusters in the boxplot. The expression is measured as the mean log2 (TP10K+1) value.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F3.medium.gif" width="286" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F3.medium.gif" width="286" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 3.</span> <span class="caption-title">Single-cell analysis of the normal lung tissue.</span><p id="p-22" class="first-child">a. The UMAP plots of the landscape of lung cells. Thirteen clusters are colored, distinctively labeled. T, B and myeloid cell subsets are further divided into finer cell subsets according to the heterogeneity within the cell population.</p><p id="p-23">b. The feature plots of the 17 TMPRSS genes, ACE2, TMPRSS1 and TMPRSS2.</p><p id="p-24">c. The expression of ACE2 across clusters in the violin plot. The expression is measured as the log2 (TP10K+1) value.</p><p id="p-25">d. The mean expression of TMPRSS family genes across clusters in the boxplot. The expression is measured as the mean log2 (TP10K+1) value.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">We detected the expression of ACE2 and TMPRSSs in 29 cell groups, in which the expression of the whole 17 TMPRSS genes is in the form of total signature value. Pseudodyeing analysis was performed and we found that ACE2 was mainly expressed in AT2 cells and marked with red (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3b, c</a>). The total 17 TMPRSS genes was found in AT1, AT2, airway secretory and ciliated cells colored with blue (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3b, d</a>, <a id="xref-fig-9-1" class="xref-fig" href="#F9">Extended Data Fig. 5a</a>). Thus, we found an obvious coexpression between TMPRSSs and ACE2 in AT2. Among the whole TMPRSS genes, TMPRSS1 and TMPRSS2 were highly expressed in AT2 and AT1 cells, which were co-expressed with ACE2 in lung (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3b</a>, <a id="xref-fig-9-2" class="xref-fig" href="#F9">Extended Data Fig. 5b</a>). Due to the entry of virus into host cell is related to endocytosis, we also detected the endocytosis-related genes among different cells. We found that these genes had consistent distribution and highly expressed in AT1, AT2, airway secretory, ciliated cells and M2 macrophage (<a id="xref-fig-9-3" class="xref-fig" href="#F9">Extended Data Fig. 5c</a>).</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The expression levels of ACE2, TMPRSS genes and exocytosis-associated genes in lung subsets. a. The Violin plots of ACE2 and TMPRSS family genes across clusters. The expression is measured as the log2 (TP10K+1) value. b. The boxplot of exocytosis-associated across clusters. The expression is measured as the mean log2 (TP10K+1) value." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1685871455" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;The expression levels of ACE2, TMPRSS genes and exocytosis-associated genes in lung subsets.&lt;/span&gt; a. The Violin plots of ACE2 and TMPRSS family genes across clusters. The expression is measured as the log2 (TP10K+1) value. b. The boxplot of exocytosis-associated across clusters. The expression is measured as the mean log2 (TP10K+1) value.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Extended Data Fig. 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F9.medium.gif" width="436" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Extended Data Fig. 5" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F9.medium.gif" width="436" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Extended Data Fig. 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Extended Data Fig. 5</span> <span class="caption-title">The expression levels of ACE2, TMPRSS genes and exocytosis-associated genes in lung subsets.</span><p id="p-62" class="first-child">a. The Violin plots of ACE2 and TMPRSS family genes across clusters. The expression is measured as the log2 (TP10K+1) value.</p><p id="p-63">b. The boxplot of exocytosis-associated across clusters. The expression is measured as the mean log2 (TP10K+1) value.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">Due to the RNA of SARS-CoV-2 was also found in the stool specimen of the SARS-CoV-2-infected patient<sup><a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a></sup>, the digestive system may also be the potential route of COVID-19. Thus, in addition to lung, 4 datasets with the single-cell transcriptomes of the esophagus, gastric, small intestine and colon were analyzed to identify the expression of ACE2 and TMPRSSs in the digestive system. The co-expression of</p><p id="p-28">ACE2 and TMPRSS was analyzed in esophagus, stomach, small intestine and colon by 87947, 29678, 11218 and 47442 high-quality single cells, respectively (<a id="xref-fig-10-1" class="xref-fig" href="#F10">Extended Data Fig. 6a</a>). The coexpression of ACE2 and total TMPRSS genes were found in the upper epithelial cells of esophagus, the absorptive enterocytes of ileum epithelia and the enterocytes of colon epithelia (<a id="xref-fig-10-2" class="xref-fig" href="#F10">Extended Data Fig. 6b-e</a>, <a id="xref-fig-11-1" class="xref-fig" href="#F11">7a-d</a>).</p><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The single-cell analysis of esophageal cells, gastric mucosal cells, ileal epithelial cells and colonic epithelial cells. a. The UMAP plots of esophageal cells, gastric mucosal cells, ileal epithelial cells and colonic epithelial cells. The Feature plots show the expression of ACE2 (red) and TMPSS family genes (green). The plots were merged to reveal the co-expression of these genes (brown). c. The expression levels ACE2 and TMPRSS restriction signature across clusters in esophagus (b), stomach(c), ileum(d) and colon(d). The expression is measured as the mean log2 (TP10K+1) value." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1685871455" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;The single-cell analysis of esophageal cells, gastric mucosal cells, ileal epithelial cells and colonic epithelial cells.&lt;/span&gt; a. The UMAP plots of esophageal cells, gastric mucosal cells, ileal epithelial cells and colonic epithelial cells. The Feature plots show the expression of ACE2 (red) and TMPSS family genes (green). The plots were merged to reveal the co-expression of these genes (brown). c. The expression levels ACE2 and TMPRSS restriction signature across clusters in esophagus (b), stomach(c), ileum(d) and colon(d). The expression is measured as the mean log2 (TP10K+1) value.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Extended Data Fig. 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F10.medium.gif" width="440" height="253"/><noscript><img class="highwire-fragment fragment-image" alt="Extended Data Fig. 6" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F10.medium.gif" width="440" height="253"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Extended Data Fig. 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Extended Data Fig. 6</span> <span class="caption-title">The single-cell analysis of esophageal cells, gastric mucosal cells, ileal epithelial cells and colonic epithelial cells.</span><p id="p-64" class="first-child">a. The UMAP plots of esophageal cells, gastric mucosal cells, ileal epithelial cells and colonic epithelial cells. The Feature plots show the expression of ACE2 (red) and TMPSS family genes (green). The plots were merged to reveal the co-expression of these genes (brown).</p><p id="p-65">c. The expression levels ACE2 and TMPRSS restriction signature across clusters in esophagus (b), stomach(c), ileum(d) and colon(d). The expression is measured as the mean log2 (TP10K+1) value.</p><div class="sb-div caption-clear"></div></div></div><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The expression levels of ACE2 and TMPRSS family genes in lung and digestive tracts. The violin plots of ACE2 and TMPRSS family genes across clusters in esophagus (a), stomach (b), ileum (c) and colon (d). The expression is measured as the mean log2 (TP10K+1) value." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1685871455" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;The expression levels of ACE2 and TMPRSS family genes in lung and digestive tracts.&lt;/span&gt; The violin plots of ACE2 and TMPRSS family genes across clusters in esophagus (a), stomach (b), ileum (c) and colon (d). The expression is measured as the mean log2 (TP10K+1) value.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Extended Data Fig. 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F11.medium.gif" width="440" height="257"/><noscript><img class="highwire-fragment fragment-image" alt="Extended Data Fig. 7" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F11.medium.gif" width="440" height="257"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Extended Data Fig. 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Extended Data Fig. 7</span> <span class="caption-title">The expression levels of ACE2 and TMPRSS family genes in lung and digestive tracts.</span><p id="p-66" class="first-child">The violin plots of ACE2 and TMPRSS family genes across clusters in esophagus (a), stomach (b), ileum (c) and colon (d). The expression is measured as the mean log2 (TP10K+1) value.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">As both ACE2 and TMPRSSs are expressed in the lung and digestive system, we next compared their relative expression values in the ACE2-expressing cells. A similar distribution was found between ACE2 and TMPRSSs in all the 9 clusters with high expressions in the esophageal upper epithelial cells, the ileal absorptive enterocytes and the colonic enterocytes (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4a</a>). In addition, their expression of AT2 was relatively lower than that of epithelial cells in the digestive system. Among all the TMPRSSs, TMPRSS1 and TMPRSS2 were relatively highly expressed in AT2, and most TMPRSSs were highly found in the esophageal upper epithelial cells (<a id="xref-fig-12-1" class="xref-fig" href="#F12">Extended Data Fig. 8a</a>). The endocytosis- and exocytosis-associated genes which are related to the entry of virus into host cells and virus infection were also detected in all the 9 clusters. The endocytosis signature was more expressed in AT1 and AT2 cells, whereas the exocytosis signature was highly gathered in esophageal upper epithelial cells. It can explain that the commonest infected tissue in COVID-19 is pulmonary alveoli and SARS-CoV-2 can also be detected in the esophageal erosion (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4b</a>)<sup><a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a></sup>. The RNA-seq data of lung, esophagus, stomach, small intestine, colon-transverse and colon-sigmoid were obtained from GTEx database. The expressions of ACE2 and TMPRSS2 also had a similar tendency and were highly expressed in small intestine and colon, while the TMPRSS11D was mainly found in the esophagus (<a id="xref-fig-12-2" class="xref-fig" href="#F12">Extended Data Fig. 8b</a>).</p><div id="F12" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The expression levels of ASE2 and TMPRSS family genes in lung and digestive tracts a. The violin plots of TMPRSS family genes in lung and digestive tracts. The expression is measured as the mean log2 (TP10K+1) value. b. The expression levels of ACE2, TMPRSS1 and TMPRSS2 verified by RNA-seq data from the GTEx database." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1685871455" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;The expression levels of ASE2 and TMPRSS family genes in lung and digestive tracts&lt;/span&gt; a. The violin plots of TMPRSS family genes in lung and digestive tracts. The expression is measured as the mean log2 (TP10K+1) value. b. The expression levels of ACE2, TMPRSS1 and TMPRSS2 verified by RNA-seq data from the GTEx database.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Extended Data Fig. 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F12.medium.gif" width="440" height="366"/><noscript><img class="highwire-fragment fragment-image" alt="Extended Data Fig. 8" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F12.medium.gif" width="440" height="366"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Extended Data Fig. 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Extended Data Fig. 8</span> <span class="caption-title">The expression levels of ASE2 and TMPRSS family genes in lung and digestive tracts</span><p id="p-67" class="first-child">a. The violin plots of TMPRSS family genes in lung and digestive tracts. The expression is measured as the mean log2 (TP10K+1) value.</p><p id="p-68">b. The expression levels of ACE2, TMPRSS1 and TMPRSS2 verified by RNA-seq data from the GTEx database.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Expression levels of ACE2, TMPRSS restriction signature and functional gene sets in lung and digestive tracts. a. The expression levels of ACE2 and TMPRSS restriction signature in 2 lung clusters and 7 digestive tract clusters. The expression is measured as the log2 (TP10K+1) value. b. The expression levels of endocytosis and exocytosis-associated genes in 2 lung clusters and 7 digestive tract clusters. The expression is measured as the log2 (TP10K+1) value." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1685871455" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Expression levels of ACE2, TMPRSS restriction signature and functional gene sets in lung and digestive tracts.&lt;/span&gt; a. The expression levels of ACE2 and TMPRSS restriction signature in 2 lung clusters and 7 digestive tract clusters. The expression is measured as the log2 (TP10K+1) value. b. The expression levels of endocytosis and exocytosis-associated genes in 2 lung clusters and 7 digestive tract clusters. The expression is measured as the log2 (TP10K+1) value.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F4.medium.gif" width="440" height="239"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F4.medium.gif" width="440" height="239"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/16/2020.02.08.926006/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 4.</span> <span class="caption-title">Expression levels of ACE2, TMPRSS restriction signature and functional gene sets in lung and digestive tracts.</span><p id="p-30" class="first-child">a. The expression levels of ACE2 and TMPRSS restriction signature in 2 lung clusters and 7 digestive tract clusters. The expression is measured as the log2 (TP10K+1) value.</p><p id="p-31">b. The expression levels of endocytosis and exocytosis-associated genes in 2 lung clusters and 7 digestive tract clusters. The expression is measured as the log2 (TP10K+1) value.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-6"><h2 class="">Discussion</h2><p id="p-32">The coronaviruses is the common infection source of respiratory, enteric and central nervous system in humans and other mammals<sup><a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a></sup>. At the beginning of the twenty-first century, two betacoronaviruses, SARS-CoV and MERS-CoV, result in persistent public panics and became the most significant public health events<sup><a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></sup>. In December 2019, a novel identified coronavirus (SARS-CoV-2) induced an ongoing outbreak of pneumonia in Wuhan, Hubei, China<sup><a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a></sup>. The rapidly increasing number of SARS-CoV-2-infected cases suggests that SARS-CoV-2 may be transmitted effectively among humans and give rise to a high pandemic potential<sup><a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">7</a>,<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>,<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></sup> Previous studies identified that SARS-CoV mutated between 2002 and 2004 to better bind to its cell receptor, replicate in human cells and enhance the virulence<sup><a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a></sup>. Thus, it is important to explore whether SARS-CoV-2 behaves like SARS-CoV to adapt to the host cell. Notably, SARS-CoV and SARS-CoV-2 share the same receptor protein ACE2<sup><a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">5</a>,<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a></sup>. Besides, the receptor-binding domain (RBD) in S protein of SARS-CoV-2 binds to ACE2 with the similar affinity as SARS-CoV RBD does<sup><a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">6</a></sup>. Thus, during the process of viral and host cellular membrane fusion, whether the specific structure of SARS-CoV-2 S protein seems better suited to be activated by host cell proteases may be related to the different virus infectivities and transmissibilities between SARS-CoV and SARS-CoV-2<sup><a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">6</a></sup>.</p><p id="p-33">In this study, we found the furin score of the S1/S2 cleavage sites in SARS-CoV-2 was higher than that of SARS, implying a more degree of hydrolysis. Through the comparison of the two structures, R682, R683 and relative S680, P681 extended the original exposed loop combined with R685 of SARS-CoV-2, which was more suitable for hydrolysis by TMPRSSs. The substrate specificity of TMPRSSs are almost similar, revealing a strong preference for arginine or lysine residues in the P1 position represented by R. More R (R682, R683 and R685) in the S1/S2 cleavage sites of SARS-CoV-2 can enhance the cleavage of S1 with S2, which means that the structurally constrains of S1 on S2 is removed, and the fusion peptides in S2 are exposed and insert into the target host cell membrane, finally it increases the efficiency of fuse membranes<sup><a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">18</a>,<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">19</a></sup>.</p><p id="p-34">By the way, some researchers previously supposed the SARS-CoV-2 was artificial due to four inserts in the S protein of SARS-CoV-2 from HIV sequence. However, the results of protein sequence alignment revealed that the similar sequence of the reported fourth insertion site (680-SPRR-683) in SARS-CoV-2 was commonly found in many beta-coronavirus. Therefore, we supposed that based on the current evidence, it is not scientific to consider the insertion sequence in SARS-CoV-2 S protein being artificial.</p><p id="p-35">With the help of single cell sequencing, we found a strong co-expression between ACE2 and TMPRSSs, in especial TMPRSS1and TMPRSS2, in lung AT2 cells, which was also the main infected cell type in SARS-CoV pneumonia<sup><a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></sup>. Moreover, we also found the endocytosis-associated genes was highly expressed in AT2 cells, implying that endocytosis may also facilitate the entry of SARS-CoV-2 into host cells. As the alveolar stem-like cells, AT2 cells are in charge of surfactant biosynthesis, self-renewal and immunoregulation<sup><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a></sup>. Thus, SARS-CoV-2 not only damages the AT2 cells leading to the direct injury to alveoli, but also raises alveolar surface tension to induce dyspnea<sup><a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a></sup>. Additionally, the injuryed AT2 also damages the immunologic balance in alveoli and results in inflammatory cascade<sup><a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></sup>. In addition, they are also highly co-expressed in absorptive enterocytes and upper epithelial cells of esophagus, implying that intestinal epithelium and esophagus epithelium may also be the potential target tissues. This can explain the cases whose SARS-CoV-2 was detected in the esophageal erosion or stool specimen, implying that the digestive system is a potential route of COVID-19<sup><a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">7</a>,<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a>,<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a></sup>.</p><p id="p-36">Due to the critical role of TMPRSSs in influenza virus and coronavirus infections, serine protease inhibitors, such as camostat, nafamostat and leupeptin, have been used in the antiviral treatment targeting TMPRSSs with high antiviral activities<sup><a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">14</a>,<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>,<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a></sup>. Nowadays, Remdesivir (GS-5734) has been used in the treatment of SARS-CoV-2, however, the therapeutic effects are still unclear. Based on our results, we also supposed that TMPRSSs may also serve as candidate antiviral targets for SARS-CoV-2 infection and the clinical trials of serine protease inhibitors should also be performed for COVID-19.</p></div><div class="section" id="sec-7"><h2 class="">Methods</h2><div id="sec-8" class="subsection"><h3>Structure modelling</h3><p id="p-37">The structures of SARS-CoV-2 S protein and TMPRSS2 were generated by SWISS-MODEL online server<sup><a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup>. The structures were marked, superimposed and visualized by Chimera<sup><a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a></sup>. To further explore the possible catalytic mechanism of the SARS-CoV-2 S protein cleaved by TMPRSS2, ZDOCK program was used to predict their interaction<sup><a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a></sup> A total of 5000 models were generated and were set to 50 clusters, then the best scoring models from the 5 largest clusters were selected for further analysis.</p></div><div id="sec-9" class="subsection"><h3>Furin score</h3><p id="p-38">The fragmentation maps, scoring and residue coverage analysis were conducted using arginine and lysine propeptide cleavage sites prediction algorithms ProP 1.0 server<sup><a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup>.</p></div><div id="sec-10" class="subsection"><h3>Single cell transcriptome data sources</h3><p id="p-39">Single cell transcriptome data were obtained from Single Cell Portal (<a href="https://singlecell.broadinstitute.org/single_cell">https://singlecell.broadinstitute.org/single_cell</a>), Human Cell Atlas Data Protal (<a href="https://data.humancellatlas.org">https://data.humancellatlas.org</a>) and Gene Expression Omnibus (GEO; <a href="https://www.ncbi.nlm.nih.gov/">https://www.ncbi.nlm.nih.gov/</a>). Esophageal and lung data were obtained from the research of E Madissoon <em>et al</em> containing 21 esophageal and 19 lung tissue samples<sup><a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a></sup>. Two lung datasets were further obtained from GSE122960<sup><a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a></sup> and GSE128169<sup><a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a></sup>, including eight and five lung tissues respectively. GSE134520 included 6 gastric mucosal samples from 3 non-atrophic gastritis and 2 chronic atrophic gastritis patients<sup><a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a></sup>. GSE134809 comprises 11 noninflammatory ileal samples from Crohn’s disease patients<sup><a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a></sup>. The data from Christopher S <em>et al</em> consisted of 12 normal colon samples<sup><a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a></sup>.</p></div><div id="sec-11" class="subsection"><h3>Quality control</h3><p id="p-40">Cells would be identified as poor-quality once (1) the number of expressed genes fewer than 200 or greater than 5000, or (2) more than 20% of UMIs being mapped to mitochondrial or ribosomal genes.</p></div><div id="sec-12" class="subsection"><h3>Data Integration, Dimension Reduction and Cell Clustering</h3><p>Different methods were performed to process the downloaded data:
</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-42">Esophagus dataset. Rdata were obtained and dimension reduction and clustering had already been implemented by the authors<sup><a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">36</a></sup>.</p></li><li id="list-item-2"><p id="p-43">Lung, stomach and ileum datasets. We utilized functions in the Seurat package to normalize and scale the single-cell gene expression data<sup><a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a></sup>. Unique molecularidentifier (UMI) counts were normalized by the total number of UMIs per cell, multiplied by 10000 for normalization and log-transformed using the NormalizeData’’ function. Then, multiple sample data within each dataset were merged using the “FindIntegrationAnchors” and “Integratedata” functions. After identifying highly variable genes (HVGs) using the “FindVariableGenes” function a principal component analysis (PCA) was performed on the single-cell expression matrix using the ‘‘RunPCA’’ function. The ‘‘FindClusters’’ function in the Seurat package was next utilized to conduct the cell clustering analysis into a graph structure in PCA space after constructing a K-nearest-neighbor graph based on the Euclidean distance in PCA space. Uniform Manifold Approximation and Projection (UMAP) visualization was performed for obtaining the clusters of cells.</p></li><li id="list-item-3"><p id="p-44">Colon Dataset. The single cell data was processed with the R packages LIGER<sup><a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a></sup> and Seurat<sup><a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">43</a></sup>. The gene expression matrix was first normalized to remove differences in sequencing depth and capture efficiency among cells. Variable genes in each dataset were identified using the “selectGenes” function. Then we used the “optimizeALS” function in LIGER to perform the integrative nonnegative matrix factorization and selecte a k of 15 and lambda of 5.0 to obtain a plot of expected alignment. The “quantileAlignSNF” function was then performed to builds a shared factor neighborhood graph to jointly cluster cells, then quantile normalizes corresponding clusters. Next nonlinear dimensionality reduction was calculated using the “RunUMAP” function and the results were visualized with UMAP.</p></li></ol></div><div id="sec-13" class="subsection"><h3>Identification of cell types and Gene expression analysis</h3><p id="p-45">Clusters were annotated on the expression of known cell markers and the clustering information provided in the articles. Then, we utilized the “RunALRA” function to impute lost values in the gene expression matrix. The imputed gene expression was shown in Feature plots and violin plots. We used “Quantile normalization” in the R package preprocessCore (R package version 1.46.0. <a href="https://github.com/bmbolstad/preprocessCore">https://github.com/bmbolstad/preprocessCore</a>) to remove unwanted technical variability across different datasets. The data were further denoised to compare the gene expression levels of gene signature.</p><p id="p-46">Endocytosis or exocytosis associated genes were obtained from Harmonizome dataset<sup><a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a></sup>.Mean expressions of the genesets were calculated to compare the ability of endocytosis or exocytosis among clusters.</p><p id="p-47">To minimize bias, external databases of Genotype-Tissue Expression (GTEx)<sup><a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a></sup> was used to detect gene expression of ACE2, TMPRSS1 and TMPRSS2 at the tissue levels including normal lung and digestive system, such as esophagus, stomach, small intestine and colon.</p></div></div><div class="section" id="sec-14"><h2 class="">Author contributions</h2><p id="p-49">J.L., L.C., W.Z. and J.X. conceived the idea and directed the team. T.M., H.C., H.Z. and W.Z. designed and coordinated the analysis and characterization. H.Z., Z.K., D.X., H.G. performed single-cell sequencing and characterization under the guidance of X.C., H.X., and H.W.. Data collection and generation were performed by J.W., Z.L., R.Z. and X.P.. Data interpretation was performed by J.L., L.C., W.Z. and J.X.. The alignment and structure comparison was performed by H.C. under the guidance of W.Z. The manuscript was written by T.M., H.C., Z.K. and W.Z. All authors contributed to the analysis and discussion of the results leading to the manuscript.</p></div><div class="section" id="sec-15"><h2 class="">Competing interests</h2><p id="p-50">The authors declare no competing interests.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-48">This study was jointly supported by the National Natural Science Foundation of China (Grants 81702659 and 81572746) and National Key R&amp;D Program of China (Grants 2016YFA0100800).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-other" id="fn-2"><p id="p-2"><a class="rev-xref" href="#xref-fn-2-1">↵</a><span class="fn-label">*</span> e-mail: <span class="em-link"><span class="em-addr">chstroke{at}163.com</span></span>; <span class="em-link"><span class="em-addr">limingcheng{at}tongji.edu.cn</span></span>; <span class="em-link"><span class="em-addr">brilliant212{at}163.com</span></span>; <span class="em-link"><span class="em-addr">jianruxiao83{at}163.com</span></span></p></li><li class="fn-update fn-group-summary-of-updates" id="fn-3"><p id="p-5">Adding more data</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2>References</h2><ol class="cit-list"><li><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1 in text" id="ref-1">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.08.926006v3.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>, B. S., <span class="cit-auth"><span class="cit-name-surname">Baric</span>  <span class="cit-name-given-names">RS</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Groot</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drosten</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gulyaeva</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haagmans</span>  <span class="cit-name-given-names">BL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lauber</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leontovich</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neuman</span>  <span class="cit-name-given-names">BW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Penzar</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perlman</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poon</span>  <span class="cit-name-given-names">LL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Samborskiy</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sidorov</span>  <span class="cit-name-given-names">IA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sola</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Severe acute respiratory syndrome-related coronavirus: The species and its viruses – a statement of the Coronavirus Study Group</span>. <span class="cit-source">bioRxiv</span> doi:doi: <a href="https://doi.org/10.1101/2020.02.07.937862">https://doi.org/10.1101/2020.02.07.937862</a> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2 in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.2" data-doi="10.1016/S0140-6736(03)14630-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhong</span>,  <span class="cit-name-given-names">N. S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Epidemiology and cause of severe acute respiratory syndrome (SARS) in Guangdong, People’s Republic of China, in February, 2003</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-vol">362</span>, <span class="cit-fpage">1353</span>–<span class="cit-lpage">1358</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/s0140-6736(03)14630-2</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.stitle%253DLancet%26rft.aulast%253DZhong%26rft.auinit1%253DN.%2BS.%26rft.volume%253D362%26rft.issue%253D9393%26rft.spage%253D1353%26rft.epage%253D1358%26rft.atitle%253DEpidemiology%2Band%2Bcause%2Bof%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome%2B%2528SARS%2529%2Bin%2BGuangdong%252C%2BPeople%2527s%2BRepublic%2Bof%2BChina%252C%2Bin%2BFebruary%252C%2B2003.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252803%252914630-2%26rft_id%253Dinfo%253Apmid%252F14585636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(03)14630-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14585636&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F16%2F2020.02.08.926006.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186143300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3 in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.3" data-doi="10.1016/j.tim.2004.08.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hofmann</span>,  <span class="cit-name-given-names">H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Pohlmann</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Cellular entry of the SARS coronavirus</span>. <abbr class="cit-jnl-abbrev">Trends in microbiology</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">466</span>–<span class="cit-lpage">472</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.tim.2004.08.008</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bmicrobiology%26rft.stitle%253DTrends%2BMicrobiol%26rft.aulast%253DHofmann%26rft.auinit1%253DH.%26rft.volume%253D12%26rft.issue%253D10%26rft.spage%253D466%26rft.epage%253D472%26rft.atitle%253DCellular%2Bentry%2Bof%2Bthe%2BSARS%2Bcoronavirus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2004.08.008%26rft_id%253Dinfo%253Apmid%252F15381196%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2004.08.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15381196&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F16%2F2020.02.08.926006.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224474400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4 in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.4" data-doi="10.1016/s0140-6736(20)30154-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chan</span>,  <span class="cit-name-given-names">J. F.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/s0140-6736(20)30154-9</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fs0140-6736%252820%252930154-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/s0140-6736(20)30154-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5 in text" id="ref-5">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.08.926006v3.5"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">P</span> <span class="cit-name-surname">Zhou</span></span>, X. Y., <span class="cit-auth"> <span class="cit-name-given-names">XG</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B</span> <span class="cit-name-surname">Hu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">HR</span> <span class="cit-name-surname">Si</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y</span> <span class="cit-name-surname">Zhu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">CL</span> <span class="cit-name-surname">Huang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">HD</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y</span> <span class="cit-name-surname">Luo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H</span> <span class="cit-name-surname">Guo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">RD</span> <span class="cit-name-surname">Jiang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">MQ</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">XR</span> <span class="cit-name-surname">Shen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">XS</span> <span class="cit-name-surname">Zheng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K</span> <span class="cit-name-surname">Zhao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">QJ</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F</span> <span class="cit-name-surname">Deng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">LL</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B</span> <span class="cit-name-surname">Yan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">FX</span> <span class="cit-name-surname">Zhan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">YY</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">GF</span> <span class="cit-name-surname">Xiao</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zl</span>  <span class="cit-name-given-names">Shi</span></span>. <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. <span class="cit-source">Nature</span>, doi:<a href="https://doi.org/10.1038/s41586-020-2012-7">https://doi.org/10.1038/s41586-020-2012-7</a> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6 in text" id="ref-6">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.08.926006v3.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">XL</span></span>, L. C., <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">SC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">ZL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">SB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">ZL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">YL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ying</span>  <span class="cit-name-given-names">TL.</span></span> <span class="cit-article-title">Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody</span>. <span class="cit-source">bioRxiv</span> doi:doi: <a href="https://doi.org/10.1101/2020.01.28.923011">https://doi.org/10.1101/2020.01.28.923011</a> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7 in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.7" data-doi="10.1016/s0140-6736(20)30183-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/s0140-6736(20)30183-5</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fs0140-6736%252820%252930183-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/s0140-6736(20)30183-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8 in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.8" data-doi="10.1056/NEJMoa2001017"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">N.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A Novel Coronavirus from Patients with Pneumonia in China, 2019</span>. <abbr class="cit-jnl-abbrev">The New England journal of medicine</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1056/NEJMoa2001017</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BNew%2BEngland%2Bjournal%2Bof%2Bmedicine%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa2001017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa2001017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9 in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.9" data-doi="10.1016/j.micinf.2020.01.004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Pathogenicity and Transmissibility of 2019-nCoV-A Quick Overview and Comparison with Other Emerging Viruses</span>. <abbr class="cit-jnl-abbrev">Microbes and infection</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.micinf.2020.01.004</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobes%2Band%2Binfection%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.micinf.2020.01.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.micinf.2020.01.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10 in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.10" data-doi="10.1016/j.cell.2018.12.028"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Walls</span>,  <span class="cit-name-given-names">A. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Unexpected Receptor Functional Mimicry Elucidates Activation of Coronavirus Fusion</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">176</span>, <span class="cit-fpage">1026</span>–<span class="cit-lpage">1039</span>. <span class="cit-page-range">e1015</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cell.2018.12.028</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D176%26rft.spage%253D1026%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2018.12.028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2018.12.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11 in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.11" data-doi="10.1006/viro.2000.0757"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gallagher</span>,  <span class="cit-name-given-names">T. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Buchmeier</span>,  <span class="cit-name-given-names">M. J.</span></span> <span class="cit-article-title">Coronavirus spike proteins in viral entry and pathogenesis</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">279</span>, <span class="cit-fpage">371</span>–<span class="cit-lpage">374</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1006/viro.2000.0757</span> (<span class="cit-pub-date">2001</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DGallagher%26rft.auinit1%253DT.%2BM.%26rft.volume%253D279%26rft.issue%253D2%26rft.spage%253D371%26rft.epage%253D374%26rft.atitle%253DCoronavirus%2Bspike%2Bproteins%2Bin%2Bviral%2Bentry%2Band%2Bpathogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fviro.2000.0757%26rft_id%253Dinfo%253Apmid%252F11162792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/viro.2000.0757&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11162792&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F16%2F2020.02.08.926006.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166695000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12 in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.12" data-doi="10.1038/cr.2016.152"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gui</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Cryo-electron microscopy structures of the SARS-CoV spike glycoprotein reveal a prerequisite conformational state for receptor binding</span>. <abbr class="cit-jnl-abbrev">Cell research</abbr> <span class="cit-vol">27</span>, <span class="cit-fpage">119</span>–<span class="cit-lpage">129</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/cr.2016.152</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Bresearch%26rft.volume%253D27%26rft.spage%253D119%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2016.152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cr.2016.152&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13 in text" id="ref-13">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.08.926006v3.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">S. R. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Musa</span>  <span class="cit-name-given-names">SS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lou</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">Preliminary estimation of the basic reproduction number of novel coronavirus (2019-nCoV) in China, from 2019 to 2020: A data-driven analysis in the early phase of the outbreak</span>. <span class="cit-source">bioRxiv</span>, <span class="cit-fpage">916395</span>, doi:doi: <a href="https://doi.org/10.1101/2020.01.23.916395">https://doi.org/10.1101/2020.01.23.916395</a> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14 in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.14" data-doi="10.1016/j.antiviral.2015.01.011"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Protease inhibitors targeting coronavirus and filovirus entry</span>. <abbr class="cit-jnl-abbrev">Antiviral research</abbr> <span class="cit-vol">116</span>, <span class="cit-fpage">76</span>–<span class="cit-lpage">84</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.antiviral.2015.01.011</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntiviral%2Bresearch%26rft.volume%253D116%26rft.spage%253D76%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.antiviral.2015.01.011%26rft_id%253Dinfo%253Apmid%252F25666761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.antiviral.2015.01.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25666761&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F16%2F2020.02.08.926006.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15 in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.15" data-doi="10.1016/j.virusres.2014.11.021"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Millet</span>,  <span class="cit-name-given-names">J. K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Whittaker</span>,  <span class="cit-name-given-names">G. R.</span></span> <span class="cit-article-title">Host cell proteases: Critical determinants of coronavirus tropism and pathogenesis</span>. <abbr class="cit-jnl-abbrev">Virus research</abbr> <span class="cit-vol">202</span>, <span class="cit-fpage">120</span>–<span class="cit-lpage">134</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virusres.2014.11.021</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2014.11.021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2014.11.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16 in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.16" data-doi="10.1016/j.virol.2017.11.012"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shirato</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kawase</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Matsuyama</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Wild-type human coronaviruses prefer cell-surface TMPRSS2 to endosomal cathepsins for cell entry</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">517</span>, <span class="cit-fpage">9</span>–<span class="cit-lpage">15</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virol.2017.11.012</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D517%26rft.spage%253D9%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2017.11.012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2017.11.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17 in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.17" data-doi="10.1128/JVI.02202-13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Heurich</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">TMPRSS2 and ADAM17 cleave ACE2 differentially and only proteolysis by TMPRSS2 augments entry driven by the severe acute respiratory syndrome coronavirus spike protein</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr> <span class="cit-vol">88</span>, <span class="cit-fpage">1293</span>–<span class="cit-lpage">1307</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.02202-13</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02202-13%26rft_id%253Dinfo%253Apmid%252F24227843%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg4LzIvMTI5MyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzE2LzIwMjAuMDIuMDguOTI2MDA2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18 in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.18" data-doi="10.1042/BJ20041955"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Herter</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Hepatocyte growth factor is a preferred in vitro substrate for human hepsin, a membrane-anchored serine protease implicated in prostate and ovarian cancers</span>. <abbr class="cit-jnl-abbrev">The Biochemical journal</abbr> <span class="cit-vol">390</span>, <span class="cit-fpage">125</span>–<span class="cit-lpage">136</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1042/bj20041955</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BBiochemical%2Bjournal%26rft_id%253Dinfo%253Adoi%252F10.1042%252FBJ20041955%26rft_id%253Dinfo%253Apmid%252F15839837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InBwYmlvY2hlbWoiO3M6NToicmVzaWQiO3M6OToiMzkwLzEvMTI1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTYvMjAyMC4wMi4wOC45MjYwMDYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19 in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.19" data-doi="10.1128/jvi.00649-19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Limburg</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">TMPRSS2 Is the Major Activating Protease of Influenza A Virus in Primary Human Airway Cells and Influenza B Virus in Human Type II Pneumocytes</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr> <span class="cit-vol">93</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.00649-19</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjvi.00649-19%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/jvi.00649-19&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20 in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.20" data-doi="10.1056/NEJMoa2001191"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Holshue</span>,  <span class="cit-name-given-names">M. L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">First Case of 2019 Novel Coronavirus in the United States</span>. <abbr class="cit-jnl-abbrev">The New England journal of medicine</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1056/NEJMoa2001191</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BNew%2BEngland%2Bjournal%2Bof%2Bmedicine%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa2001191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa2001191&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21 in text" id="ref-21">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.08.926006v3.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">WJ</span></span>, N. Z., <span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">WH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ou</span>  <span class="cit-name-given-names">CQ</span></span>, <span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">JX</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shan</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lei</span>  <span class="cit-name-given-names">CL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hui David</span>  <span class="cit-name-given-names">S.C.</span></span>, <span class="cit-article-title">Du B Clinical characteristics of 2019 novel coronavirus infection in China</span>. <span class="cit-source">medRxiv</span>, doi:doi: <a href="http://dx.doi.org/10.1101/2020.02.06.20020974">http://dx.doi.org/10.1101/2020.02.06.20020974</a> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22 in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.22" data-doi="10.1038/nrmicro2147"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Perlman</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Netland</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Coronaviruses post-SARS: update on replication and pathogenesis</span>. <abbr class="cit-jnl-abbrev">Nature reviews. Microbiology</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">439</span>–<span class="cit-lpage">450</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nrmicro2147</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DPerlman%26rft.auinit1%253DS.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253D439%26rft.epage%253D450%26rft.atitle%253DCoronaviruses%2Bpost-SARS%253A%2Bupdate%2Bon%2Breplication%2Band%2Bpathogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro2147%26rft_id%253Dinfo%253Apmid%252F19430490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro2147&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19430490&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F16%2F2020.02.08.926006.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23 in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.23" data-doi="10.1038/nrmicro.2016.81"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">de Wit</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Doremalen</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Falzarano</span>,  <span class="cit-name-given-names">D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Munster</span>,  <span class="cit-name-given-names">V. J.</span></span> <span class="cit-article-title">SARS and MERS: recent insights into emerging coronaviruses</span>. <abbr class="cit-jnl-abbrev">Nature reviews. Microbiology</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">523</span>–<span class="cit-lpage">534</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nrmicro.2016.81</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.volume%253D14%26rft.spage%253D523%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro.2016.81%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro.2016.81&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24 in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.24" data-doi="10.1016/j.jmii.2020.02.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">P. I.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Hsueh</span>,  <span class="cit-name-given-names">P. R.</span></span> <span class="cit-article-title">Emerging threats from zoonotic coronaviruses-from SARS and MERS to 2019-nCoV</span>. <abbr class="cit-jnl-abbrev">Journal of microbiology, immunology, and infection = Wei mian yu gan ran za zhi</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.jmii.2020.02.001</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmicrobiology%252C%2Bimmunology%252C%2Band%2Binfection%2B%253D%2BWei%2Bmian%2Byu%2Bgan%2Bran%2Bza%2Bzhi%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmii.2020.02.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmii.2020.02.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25 in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.25" data-doi="10.1038/nature02145"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">426</span>, <span class="cit-fpage">450</span>–<span class="cit-lpage">454</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature02145</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLi%26rft.auinit1%253DW.%26rft.volume%253D426%26rft.issue%253D6965%26rft.spage%253D450%26rft.epage%253D454%26rft.atitle%253DAngiotensin-converting%2Benzyme%2B2%2Bis%2Ba%2Bfunctional%2Breceptor%2Bfor%2Bthe%2BSARS%2Bcoronavirus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02145%26rft_id%253Dinfo%253Apmid%252F14647384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02145&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14647384&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F16%2F2020.02.08.926006.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26 in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.26" data-doi="10.1016/S0140-6736(03)13967-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kuiken</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Newly discovered coronavirus as the primary cause of severe acute respiratory syndrome</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-vol">362</span>, <span class="cit-fpage">263</span>–<span class="cit-lpage">270</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/s0140-6736(03)13967-0</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.stitle%253DLancet%26rft.aulast%253DKuiken%26rft.auinit1%253DT.%26rft.volume%253D362%26rft.issue%253D9380%26rft.spage%253D263%26rft.epage%253D270%26rft.atitle%253DNewly%2Bdiscovered%2Bcoronavirus%2Bas%2Bthe%2Bprimary%2Bcause%2Bof%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252803%252913967-0%26rft_id%253Dinfo%253Apmid%252F12892955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(03)13967-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12892955&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F16%2F2020.02.08.926006.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184413400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27 in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.27" data-doi="10.1126/science.aam6603"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nabhan</span>,  <span class="cit-name-given-names">A. N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brownfield</span>,  <span class="cit-name-given-names">D. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harbury</span>,  <span class="cit-name-given-names">P. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krasnow</span>,  <span class="cit-name-given-names">M. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Desai</span>,  <span class="cit-name-given-names">T. J.</span></span> <span class="cit-article-title">Single-cell Wnt signaling niches maintain stemness of alveolar type 2 cells</span>. <abbr class="cit-jnl-abbrev">Science (New York, N.Y.)</abbr> <span class="cit-vol">359</span>, <span class="cit-fpage">1118</span>–<span class="cit-lpage">1123</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.aam6603</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2B%2528New%2BYork%252C%2BN.Y.%2529%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aam6603%26rft_id%253Dinfo%253Apmid%252F29420258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNTkvNjM4MC8xMTE4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTYvMjAyMC4wMi4wOC45MjYwMDYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28 in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.28" data-doi="10.1172/JCI68782"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Barkauskas</span>,  <span class="cit-name-given-names">C. E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Type 2 alveolar cells are stem cells in adult lung</span>. <abbr class="cit-jnl-abbrev">The Journal of clinical investigation</abbr> <span class="cit-vol">123</span>, <span class="cit-fpage">3025</span>–<span class="cit-lpage">3036</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1172/jci68782</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bclinical%2Binvestigation%26rft.volume%253D123%26rft.spage%253D3025%26rft_id%253Dinfo%253Adoi%252F10.1172%252FJCI68782%26rft_id%253Dinfo%253Apmid%252F23921127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1172/JCI68782&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23921127&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F16%2F2020.02.08.926006.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321316700030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29 in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.29" data-doi="10.1371/journal.ppat.1005338"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kroetz</span>,  <span class="cit-name-given-names">D. N.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Type I Interferon Induced Epigenetic Regulation of Macrophages Suppresses Innate and Adaptive Immunity in Acute Respiratory Viral Infection</span>. <abbr class="cit-jnl-abbrev">PLoS pathogens</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">e1005338</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.ppat.1005338</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bpathogens%26rft.volume%253D11%26rft.spage%253De1005338%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1005338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1005338&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30 in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.30" data-doi="10.1016/j.biochi.2017.07.016"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shen</span>,  <span class="cit-name-given-names">L. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mao</span>,  <span class="cit-name-given-names">H. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">Y. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tanaka</span>,  <span class="cit-name-given-names">Y.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">W.</span></span> <span class="cit-article-title">TMPRSS2: A potential target for treatment of influenza virus and coronavirus infections</span>. <abbr class="cit-jnl-abbrev">Biochimie</abbr> <span class="cit-vol">142</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.biochi.2017.07.016</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimie%26rft.volume%253D142%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.biochi.2017.07.016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.biochi.2017.07.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31 in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.31" data-doi="10.1080/17460441.2017.1372417"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shin</span>,  <span class="cit-name-given-names">W. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Seong</span>,  <span class="cit-name-given-names">B. L.</span></span> <span class="cit-article-title">Type II transmembrane serine proteases as potential target for anti-influenza drug discovery</span>. <abbr class="cit-jnl-abbrev">Expert opinion on drug discovery</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">1139</span>–<span class="cit-lpage">1152</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1080/17460441.2017.1372417</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExpert%2Bopinion%2Bon%2Bdrug%2Bdiscovery%26rft.volume%253D12%26rft.spage%253D1139%26rft_id%253Dinfo%253Adoi%252F10.1080%252F17460441.2017.1372417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/17460441.2017.1372417&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32 in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.32" data-doi="10.1093/nar/gku340"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Biasini</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">SWISS-MODEL: modelling protein tertiary and quaternary structure using evolutionary information</span>. <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr> <span class="cit-vol">42</span>, <span class="cit-fpage">W252</span>–<span class="cit-lpage">258</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gku340</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku340%26rft_id%253Dinfo%253Apmid%252F24782522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24782522&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F16%2F2020.02.08.926006.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000339715000042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33 in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.33" data-doi="10.1002/jcc.20084"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pettersen</span>,  <span class="cit-name-given-names">E. F.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">UCSF Chimera--a visualization system for exploratory research and analysis</span>. <abbr class="cit-jnl-abbrev">Journal of computational chemistry</abbr> <span class="cit-vol">25</span>, <span class="cit-fpage">1605</span>–<span class="cit-lpage">1612</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1002/jcc.20084</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bchemistry%26rft.stitle%253DJ%2BComput%2BChem%26rft.aulast%253DPettersen%26rft.auinit1%253DE.%2BF.%26rft.volume%253D25%26rft.issue%253D13%26rft.spage%253D1605%26rft.epage%253D1612%26rft.atitle%253DUCSF%2BChimera--a%2Bvisualization%2Bsystem%2Bfor%2Bexploratory%2Bresearch%2Band%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcc.20084%26rft_id%253Dinfo%253Apmid%252F15264254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jcc.20084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15264254&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F16%2F2020.02.08.926006.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223379100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34 in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.34" data-doi="10.1002/prot.20559"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wiehe</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">ZDOCK and RDOCK performance in CAPRI rounds 3, 4, and 5</span>. <abbr class="cit-jnl-abbrev">Proteins</abbr> <span class="cit-vol">60</span>, <span class="cit-fpage">207</span>–<span class="cit-lpage">213</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1002/prot.20559</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteins%26rft.stitle%253DProteins%26rft.aulast%253DWiehe%26rft.auinit1%253DK.%26rft.volume%253D60%26rft.issue%253D2%26rft.spage%253D207%26rft.epage%253D213%26rft.atitle%253DZDOCK%2Band%2BRDOCK%2Bperformance%2Bin%2BCAPRI%2Brounds%2B3%252C%2B4%252C%2Band%2B5.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fprot.20559%26rft_id%253Dinfo%253Apmid%252F15981263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/prot.20559&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15981263&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F16%2F2020.02.08.926006.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230249000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35 in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.35" data-doi="10.1093/protein/gzh013"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Duckert</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brunak</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Blom</span>,  <span class="cit-name-given-names">N.</span></span> <span class="cit-article-title">Prediction of proprotein convertase cleavage sites</span>. <abbr class="cit-jnl-abbrev">Protein engineering, design &amp; selection: PEDS</abbr> <span class="cit-vol">17</span>, <span class="cit-fpage">107</span>–<span class="cit-lpage">112</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/protein/gzh013</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtein%2BEngineering%2BDesign%2Band%2BSelection%26rft.stitle%253DProtein%2BEng%2BDes%2BSel%26rft.aulast%253DDuckert%26rft.auinit1%253DP.%26rft.volume%253D17%26rft.issue%253D1%26rft.spage%253D107%26rft.epage%253D112%26rft.atitle%253DPrediction%2Bof%2Bproprotein%2Bconvertase%2Bcleavage%2Bsites%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fprotein%252Fgzh013%26rft_id%253Dinfo%253Apmid%252F14985543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/protein/gzh013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14985543&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F16%2F2020.02.08.926006.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220905100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36 in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.36" data-doi="10.1186/S13059-019-1906-x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Madissoon</span>,  <span class="cit-name-given-names">E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">scRNA-seq assessment of the human lung, spleen, and esophagus tissue stability after cold preservation</span>. <abbr class="cit-jnl-abbrev">Genome biology</abbr> <span class="cit-vol">21</span>, <span class="cit-fpage">1</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1186/S13059-019-1906-x</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.volume%253D21%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252FS13059-019-1906-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/S13059-019-1906-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.08.926006v3.37" data-doi="10.1038/S41591-019-0468-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vieira Braga</span>,  <span class="cit-name-given-names">F. A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A cellular census of human lungs identifies novel cell states in health and in asthma</span>. <abbr class="cit-jnl-abbrev">Nature medicine</abbr> <span class="cit-vol">25</span>, <span class="cit-fpage">1153</span>–<span class="cit-lpage">1163</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/S41591-019-0468-5</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bmedicine%26rft.volume%253D25%26rft.spage%253D1153%26rft_id%253Dinfo%253Adoi%252F10.1038%252FS41591-019-0468-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/S41591-019-0468-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38 in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.38" data-doi="10.1164/rccm.201712-2410OC"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reyfman</span>,  <span class="cit-name-given-names">P. A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Single-Cell Transcriptomic Analysis of Human Lung Provides Insights into the Pathobiology of Pulmonary Fibrosis</span>. <abbr class="cit-jnl-abbrev">American journal of respiratory and critical care medicine</abbr> <span class="cit-vol">199</span>, <span class="cit-fpage">1517</span>–<span class="cit-lpage">1536</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1164/rccm.201712-2410OC</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Brespiratory%2Band%2Bcritical%2Bcare%2Bmedicine%26rft_id%253Dinfo%253Adoi%252F10.1164%252Frccm.201712-2410OC%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1164/rccm.201712-2410OC&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39 in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.39" data-doi="10.1136/annrheumdis-2018-214865"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Valenzi</span>,  <span class="cit-name-given-names">E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Single-cell analysis reveals fibroblast heterogeneity and myofibroblasts in systemic sclerosis-associated interstitial lung disease</span>. <abbr class="cit-jnl-abbrev">Annals of the rheumatic diseases</abbr> <span class="cit-vol">78</span>, <span class="cit-fpage">1379</span>–<span class="cit-lpage">1387</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1136/annrheumdis-2018-214865</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bthe%2Brheumatic%2Bdiseases%26rft_id%253Dinfo%253Adoi%252F10.1136%252Fannrheumdis-2018-214865%26rft_id%253Dinfo%253Apmid%252F31405848%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6ImFubnJoZXVtZGlzIjtzOjU6InJlc2lkIjtzOjEwOiI3OC8xMC8xMzc5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTYvMjAyMC4wMi4wOC45MjYwMDYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40 in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.40" data-doi="10.1016/j.celrep.2019.04.052"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Dissecting the Single-Cell Transcriptome Network Underlying Gastric Premalignant Lesions and Early Gastric Cancer</span>. <abbr class="cit-jnl-abbrev">Cell reports</abbr> <span class="cit-vol">27</span>, <span class="cit-fpage">1934</span>–<span class="cit-lpage">1947</span>. <span class="cit-page-range">e1935</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.celrep.2019.04.052</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Breports%26rft.volume%253D27%26rft.spage%253D1934%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2019.04.052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2019.04.052&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41 in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.41" data-doi="10.1016/j.cell.2019.08.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martin</span>,  <span class="cit-name-given-names">J. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Single-Cell Analysis of Crohn’s Disease Lesions Identifies a Pathogenic Cellular Module Associated with Resistance to Anti-TNF Therapy</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">178</span>, <span class="cit-fpage">1493</span>–<span class="cit-lpage">1508</span>. <span class="cit-page-range">e1420</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cell.2019.08.008</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D178%26rft.spage%253D1493%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2019.08.008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2019.08.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42 in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.42" data-doi="10.1016/j.cell.2019.06.029"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smillie</span>,  <span class="cit-name-given-names">C. S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Intra- and Inter-cellular Rewiring of the Human Colon during Ulcerative Colitis</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">178</span>, <span class="cit-fpage">714</span>–<span class="cit-lpage">730</span>. <span class="cit-page-range">e722</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cell.2019.06.029</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2019.06.029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2019.06.029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43 in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.43" data-doi="10.1016/j.cell.2019.05.031"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stuart</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Comprehensive Integration of Single-Cell Data</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">177</span>, <span class="cit-fpage">1888</span>–<span class="cit-lpage">1902</span>. <span class="cit-page-range">e1821</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cell.2019.05.031</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2019.05.031%26rft_id%253Dinfo%253Apmid%252F31178118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2019.05.031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=31178118&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F16%2F2020.02.08.926006.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44 in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.44" data-doi="10.1016/J.CELL.2019.05.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Welch</span>,  <span class="cit-name-given-names">J. D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Single-Cell Multi-omic Integration Compares and Contrasts Features of Brain Cell Identity</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">177</span>, <span class="cit-fpage">1873</span>–<span class="cit-lpage">1887</span>. <span class="cit-page-range">e1817</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cell.2019.05.006</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D177%26rft.spage%253D1873%26rft_id%253Dinfo%253Adoi%252F10.1016%252FJ.CELL.2019.05.006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/J.CELL.2019.05.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45 in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.45" data-doi="10.1093/database/baw100"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rouillard</span>,  <span class="cit-name-given-names">A. D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The harmonizome: a collection of processed datasets gathered to serve and mine knowledge about genes and proteins</span>. <abbr class="cit-jnl-abbrev">Database: the journal of biological databases and curation 2016</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/database/baw100</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDatabase%253A%2Bthe%2Bjournal%2Bof%2Bbiological%2Bdatabases%2Band%2Bcuration%2B2016%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdatabase%252Fbaw100%26rft_id%253Dinfo%253Apmid%252F27374120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/database/baw100&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27374120&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F16%2F2020.02.08.926006.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46 in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.08.926006v3.46" data-doi="10.1126/science.1262110"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Human genomics</span>. <span class="cit-article-title">The Genotype-Tissue Expression (GTEx) pilot analysis: multitissue gene regulation in humans</span>. <abbr class="cit-jnl-abbrev">Science (New York, N.Y.)</abbr> <span class="cit-vol">348</span>, <span class="cit-fpage">648</span>–<span class="cit-lpage">660</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1262110</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DThe%2BGTEx%2BConsortium%26rft.auinit1%253D%2B%26rft.volume%253D348%26rft.issue%253D6235%26rft.spage%253D648%26rft.epage%253D660%26rft.atitle%253DThe%2BGenotype-Tissue%2BExpression%2B%2528GTEx%2529%2Bpilot%2Banalysis%253A%2BMultitissue%2Bgene%2Bregulation%2Bin%2Bhumans%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1262110%26rft_id%253Dinfo%253Apmid%252F25954001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDgvNjIzNS82NDgiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8xNi8yMDIwLjAyLjA4LjkyNjAwNi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.08.926006v3.47" data-doi="10.1126/science.1260419"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Uhlen</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Proteomics. Tissue-based map of the human proteome</span>. <abbr class="cit-jnl-abbrev">Science (New York, N.Y.)</abbr> <span class="cit-vol">347</span>, <span class="cit-fpage">1260419</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1260419</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DUhlen%26rft.auinit1%253DM.%26rft.volume%253D347%26rft.issue%253D6220%26rft.spage%253D1260419%26rft.epage%253D1260419%26rft.atitle%253DTissue-based%2Bmap%2Bof%2Bthe%2Bhuman%2Bproteome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1260419%26rft_id%253Dinfo%253Apmid%252F25613900%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNDcvNjIyMC8xMjYwNDE5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTYvMjAyMC4wMi4wOC45MjYwMDYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.02.08.926006v3.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.02.08.926006v3" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/673459v2" title="Immune cells lacking Y chromosome have widespread dysregulation of autosomal genes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/069609v2" title="Frank-Starling law and Bowditch phenomenon may have the common theoretical grounds" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;February 16, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.02.08.926006v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.02.08.926006v3.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.02.08.926006v3" class="active" data-icon-position="" data-hide-link-title="0">The insert sequence in SARS-CoV-2 enhances spike protein cleavage by TMPRSS</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1153713" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-eeS9Eg5ekSPZPhmwQPsXqckBswdvK--jV8QhuGOCVtY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1153713" id="node1153713--2" data-pisa="biorxiv;2020.02.08.926006v3" data-pisa-master="biorxiv;2020.02.08.926006" data-seqnum="1153713" data-apath="/biorxiv/early/2020/02/16/2020.02.08.926006.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">The insert sequence in SARS-CoV-2 enhances spike protein cleavage by TMPRSS</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Tong</span> <span class="nlm-surname">Meng</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Hao</span> <span class="nlm-surname">Cao</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Hao</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Zijian</span> <span class="nlm-surname">Kang</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Da</span> <span class="nlm-surname">Xu</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Haiyi</span> <span class="nlm-surname">Gong</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Jing</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Zifu</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Xingang</span> <span class="nlm-surname">Cui</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Huji</span> <span class="nlm-surname">Xu</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Haifeng</span> <span class="nlm-surname">Wei</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Xiuwu</span> <span class="nlm-surname">Pan</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Rongrong</span> <span class="nlm-surname">Zhu</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Jianru</span> <span class="nlm-surname">Xiao</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Wang</span> <span class="nlm-surname">Zhou</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Liming</span> <span class="nlm-surname">Cheng</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">Jianmin</span> <span class="nlm-surname">Liu</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.02.08.926006; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.02.08.926006 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.02.08.926006v3" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.02.08.926006v3&amp;title=The%20insert%20sequence%20in%20SARS-CoV-2%20enhances%20spike%20protein%20cleavage%20by%20TMPRSS" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.08.926006v3&amp;title=The%20insert%20sequence%20in%20SARS-CoV-2%20enhances%20spike%20protein%20cleavage%20by%20TMPRSS" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.08.926006v3&amp;text=The%20insert%20sequence%20in%20SARS-CoV-2%20enhances%20spike%20protein%20cleavage%20by%20TMPRSS" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.08.926006v3&amp;title=The%20insert%20sequence%20in%20SARS-CoV-2%20enhances%20spike%20protein%20cleavage%20by%20TMPRSS" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.02.08.926006v3&amp;t=The%20insert%20sequence%20in%20SARS-CoV-2%20enhances%20spike%20protein%20cleavage%20by%20TMPRSS" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.02.08.926006v3&amp;title=The%20insert%20sequence%20in%20SARS-CoV-2%20enhances%20spike%20protein%20cleavage%20by%20TMPRSS" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.08.926006v3&amp;title=The%20insert%20sequence%20in%20SARS-CoV-2%20enhances%20spike%20protein%20cleavage%20by%20TMPRSS" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1153713" id="node1153713--3" data-pisa="biorxiv;2020.02.08.926006v3" data-pisa-master="biorxiv;2020.02.08.926006" data-seqnum="1153713" data-apath="/biorxiv/early/2020/02/16/2020.02.08.926006.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">The insert sequence in SARS-CoV-2 enhances spike protein cleavage by TMPRSS</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Tong</span> <span class="nlm-surname">Meng</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Hao</span> <span class="nlm-surname">Cao</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Hao</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Zijian</span> <span class="nlm-surname">Kang</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Da</span> <span class="nlm-surname">Xu</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Haiyi</span> <span class="nlm-surname">Gong</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Jing</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Zifu</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Xingang</span> <span class="nlm-surname">Cui</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Huji</span> <span class="nlm-surname">Xu</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Haifeng</span> <span class="nlm-surname">Wei</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Xiuwu</span> <span class="nlm-surname">Pan</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Rongrong</span> <span class="nlm-surname">Zhu</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Jianru</span> <span class="nlm-surname">Xiao</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Wang</span> <span class="nlm-surname">Zhou</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Liming</span> <span class="nlm-surname">Cheng</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">Jianmin</span> <span class="nlm-surname">Liu</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.02.08.926006; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.02.08.926006 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=The%20insert%20sequence%20in%20SARS-CoV-2%20enhances%20spike%20protein%20cleavage%20by%20TMPRSS&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Meng%2C+Tong&amp;rft.au=Cao%2C+Hao&amp;rft.au=Zhang%2C+Hao&amp;rft.au=Kang%2C+Zijian&amp;rft.au=Xu%2C+Da&amp;rft.au=Gong%2C+Haiyi&amp;rft.au=Wang%2C+Jing&amp;rft.au=Li%2C+Zifu&amp;rft.au=Cui%2C+Xingang&amp;rft.au=Xu%2C+Huji&amp;rft.au=Wei%2C+Haifeng&amp;rft.au=Pan%2C+Xiuwu&amp;rft.au=Zhu%2C+Rongrong&amp;rft.au=Xiao%2C+Jianru&amp;rft.au=Zhou%2C+Wang&amp;rft.au=Cheng%2C+Liming&amp;rft.au=Liu%2C+Jianmin"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1153713/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1153713/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1153713/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1153713/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1153713/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1153713/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1153713/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1153713/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1153713/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1153713/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1153713/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1153713/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.08.926006v3&amp;count=horizontal&amp;via=&amp;text=The%20insert%20sequence%20in%20SARS-CoV-2%20enhances%20spike%20protein%20cleavage%20by%20TMPRSS&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.02.08.926006v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.02.08.926006v3&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.02.08.926006v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">Microbiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
