<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/02/18/2020.02.13.945485.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.02.13.945485v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Identification of 2019-nCoV related coronaviruses in Malayan pangolins in southern China"/>
  <meta name="DC.Identifier" content="10.1101/2020.02.13.945485"/>
  <meta name="DC.Date" content="2020-02-18"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="The ongoing outbreak of viral pneumonia in China and beyond is associated with a novel coronavirus, provisionally termed 2019-nCoV. This outbreak has been tentatively associated with a seafood market in Wuhan, China, where the sale of wild animals may be the source of zoonotic infection. Although bats are likely reservoir hosts for 2019-nCoV, the identity of any intermediate host facilitating transfer to humans is unknown. Here, we report the identification of 2019-nCoV related coronaviruses in pangolins ( Manis javanica ) seized in anti-smuggling operations in southern China. Metagenomic sequencing identified pangolin associated CoVs that belong to two sub-lineages of 2019-nCoV related coronaviruses, including one very closely related to 2019-nCoV in the receptor-binding domain. The discovery of multiple lineages of pangolin coronavirus and their similarity to 2019-nCoV suggests that pangolins should be considered as possible intermediate hosts for this novel human virus and should be removed from wet markets to prevent zoonotic transmission."/>
  <meta name="DC.Contributor" content="Tommy Tsan-Yuk Lam"/>
  <meta name="DC.Contributor" content="Marcus Ho-Hin Shum"/>
  <meta name="DC.Contributor" content="Hua-Chen Zhu"/>
  <meta name="DC.Contributor" content="Yi-Gang Tong"/>
  <meta name="DC.Contributor" content="Xue-Bing Ni"/>
  <meta name="DC.Contributor" content="Yun-Shi Liao"/>
  <meta name="DC.Contributor" content="Wei Wei"/>
  <meta name="DC.Contributor" content="William Yiu-Man Cheung"/>
  <meta name="DC.Contributor" content="Wen-Juan Li"/>
  <meta name="DC.Contributor" content="Lian-Feng Li"/>
  <meta name="DC.Contributor" content="Gabriel M Leung"/>
  <meta name="DC.Contributor" content="Edward C. Holmes"/>
  <meta name="DC.Contributor" content="Yan-Ling Hu"/>
  <meta name="DC.Contributor" content="Yi Guan"/>
  <meta name="article:published_time" content="2020-02-18"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Identification of 2019-nCoV related coronaviruses in Malayan pangolins in southern China"/>
  <meta name="citation_abstract" lang="en" content="&lt;;h3&gt;;Abstract&lt;;/h3&gt;; &lt;;p&gt;;The ongoing outbreak of viral pneumonia in China and beyond is associated with a novel coronavirus, provisionally termed 2019-nCoV. This outbreak has been tentatively associated with a seafood market in Wuhan, China, where the sale of wild animals may be the source of zoonotic infection. Although bats are likely reservoir hosts for 2019-nCoV, the identity of any intermediate host facilitating transfer to humans is unknown. Here, we report the identification of 2019-nCoV related coronaviruses in pangolins (&lt;;i&gt;;Manis javanica&lt;;/i&gt;;) seized in anti-smuggling operations in southern China. Metagenomic sequencing identified pangolin associated CoVs that belong to two sub-lineages of 2019-nCoV related coronaviruses, including one very closely related to 2019-nCoV in the receptor-binding domain. The discovery of multiple lineages of pangolin coronavirus and their similarity to 2019-nCoV suggests that pangolins should be considered as possible intermediate hosts for this novel human virus and should be removed from wet markets to prevent zoonotic transmission.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.02.13.945485v1"/>
  <meta name="citation_id" content="2020.02.13.945485v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.13.945485v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.13.945485v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.13.945485v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/02/18/2020.02.13.945485.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.02.13.945485"/>
  <meta name="citation_num_pages" content="22"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.02.13.945485"/>
  <meta name="citation_author" content="Tommy Tsan-Yuk Lam"/>
  <meta name="citation_author_institution" content="Joint Institute of Virology (Shantou University / The University of Hong Kong) &amp;; Guangdong-Hongkong Joint Laboratory of Emerging Infectious Diseases, Shantou University"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Emerging Infectious Diseases, School of Public Health, The University of Hong Kong"/>
  <meta name="citation_author" content="Marcus Ho-Hin Shum"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Emerging Infectious Diseases, School of Public Health, The University of Hong Kong"/>
  <meta name="citation_author" content="Hua-Chen Zhu"/>
  <meta name="citation_author_institution" content="Joint Institute of Virology (Shantou University / The University of Hong Kong) &amp;; Guangdong-Hongkong Joint Laboratory of Emerging Infectious Diseases, Shantou University"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Emerging Infectious Diseases, School of Public Health, The University of Hong Kong"/>
  <meta name="citation_author" content="Yi-Gang Tong"/>
  <meta name="citation_author_institution" content="Beijing Advanced Innovation Center for Soft Matter Science and Engineering (BAIC-SM), College of Life Science and Technology, Beijing University of Chemical Technology"/>
  <meta name="citation_author" content="Xue-Bing Ni"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Emerging Infectious Diseases, School of Public Health, The University of Hong Kong"/>
  <meta name="citation_author" content="Yun-Shi Liao"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Emerging Infectious Diseases, School of Public Health, The University of Hong Kong"/>
  <meta name="citation_author" content="Wei Wei"/>
  <meta name="citation_author_institution" content="Life Sciences Institute, Guangxi Medical University"/>
  <meta name="citation_author" content="William Yiu-Man Cheung"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Emerging Infectious Diseases, School of Public Health, The University of Hong Kong"/>
  <meta name="citation_author" content="Wen-Juan Li"/>
  <meta name="citation_author_institution" content="Beijing Advanced Innovation Center for Soft Matter Science and Engineering (BAIC-SM), College of Life Science and Technology, Beijing University of Chemical Technology"/>
  <meta name="citation_author" content="Lian-Feng Li"/>
  <meta name="citation_author_institution" content="Life Sciences Institute, Guangxi Medical University"/>
  <meta name="citation_author" content="Gabriel M Leung"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Emerging Infectious Diseases, School of Public Health, The University of Hong Kong"/>
  <meta name="citation_author" content="Edward C. Holmes"/>
  <meta name="citation_author_institution" content="Marie Bashir Institute for Infectious Diseases and Biosecurity, School of Life and Environmental Sciences and School of Medical Sciences, The University of Sydney"/>
  <meta name="citation_author" content="Yan-Ling Hu"/>
  <meta name="citation_author_institution" content="Life Sciences Institute, Guangxi Medical University"/>
  <meta name="citation_author_email" content="yguan@hku.hk"/>
  <meta name="citation_author_email" content="huyanling@gxmu.edu.cn"/>
  <meta name="citation_author" content="Yi Guan"/>
  <meta name="citation_author_institution" content="Joint Institute of Virology (Shantou University / The University of Hong Kong) &amp;; Guangdong-Hongkong Joint Laboratory of Emerging Infectious Diseases, Shantou University"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Emerging Infectious Diseases, School of Public Health, The University of Hong Kong"/>
  <meta name="citation_author_email" content="yguan@hku.hk"/>
  <meta name="citation_author_email" content="huyanling@gxmu.edu.cn"/>
  <meta name="citation_reference" content="Wu, F. et al. A new coronavirus associated with human respiratory disease in China. Nature (2020). https://doi.org/10.1038/s41586-020-2008-3."/>
  <meta name="citation_reference" content="World Health Organization. WHO Novel Coronavirus (2019-nCoV) situation reports. https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports/"/>
  <meta name="citation_reference" content="Lu, R. et al. Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding. Lancet, (2020). doi:10.1016/S0140-6736(20)30251-8."/>
  <meta name="citation_reference" content="Cohen, J. Mining coronavirus genomes for clues to the outbreak’s origins. Science, (2020) https://www.sciencemag.org/news/2020/01/mining-coronavirus-genomes-clues-outbreak-s-origins."/>
  <meta name="citation_reference" content="Wang, M. et al. SARS-CoV infection in a restaurant from palm civet. Emerg Infect Dis 11:1860–5, (2005)."/>
  <meta name="citation_reference" content="Zhou, P. et al. A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature (2020). https://doi.org/10.1038/s41586-020-2012-7."/>
  <meta name="citation_reference" content="Liu, P. et al. Viral Metagenomics Revealed Sendai Virus and Coronavirus Infection of Malayan Pangolins (Manis javanica). Viruses 11, (2019)."/>
  <meta name="citation_reference" content="Hon, C.C. et al. Evidence of the recombinant origin of a bat severe acute respiratory syndrome (SARS)-like coronavirus and its implications on the direct ancestor of SARS coronavirus. J Virol 82, 1819–1826, (2008)."/>
  <meta name="citation_reference" content="Wan, Y. et al. Receptor recognition by novel coronavirus from Wuhan: An analysis based on decade-long structural studies of SARS. J Virol, (2020)."/>
  <meta name="citation_reference" content="Heinrich, S. et al. The global trafficking of pangolins: A comprehensive summary of seizures and trafficking routes from 2010–2015. Trafficking, (2017)."/>
  <meta name="citation_reference" content="Wang, W. et al. 2017. Discovery of a Highly Divergent Coronavirus in the Asian House Shrew from China Illuminates the Origin of the Alphacoronaviruses. J Virol 91, (2017). References associated in Methods only:"/>
  <meta name="citation_reference" content="Li, D. et al. MEGAHIT: an ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph. Bioinformatics 31, 1674–1676, (2015)."/>
  <meta name="citation_reference" content="Li, H. and Durbin, R. Fast and accurate long-read alignment with Burrows-Wheeler transform. Bioinformatics 26, 589–595, (2010)."/>
  <meta name="citation_reference" content="Rozewicki, J. et al. MAFFT-DASH: integrated protein sequence and structural alignment. Nucleic Acids Res 47, W5–W10, (2019)."/>
  <meta name="citation_reference" content="Guindon, S. et al. PHYML Online--a web server for fast maximum likelihood-based phylogenetic inference. Nucleic Acids Res 33, W557–559, (2010)."/>
  <meta name="twitter:title" content="Identification of 2019-nCoV related coronaviruses in Malayan pangolins in southern China"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="The ongoing outbreak of viral pneumonia in China and beyond is associated with a novel coronavirus, provisionally termed 2019-nCoV. This outbreak has been tentatively associated with a seafood market in Wuhan, China, where the sale of wild animals may be the source of zoonotic infection. Although bats are likely reservoir hosts for 2019-nCoV, the identity of any intermediate host facilitating transfer to humans is unknown. Here, we report the identification of 2019-nCoV related coronaviruses in pangolins ( Manis javanica ) seized in anti-smuggling operations in southern China. Metagenomic sequencing identified pangolin associated CoVs that belong to two sub-lineages of 2019-nCoV related coronaviruses, including one very closely related to 2019-nCoV in the receptor-binding domain. The discovery of multiple lineages of pangolin coronavirus and their similarity to 2019-nCoV suggests that pangolins should be considered as possible intermediate hosts for this novel human virus and should be removed from wet markets to prevent zoonotic transmission."/>
  <meta name="og-title" property="og:title" content="Identification of 2019-nCoV related coronaviruses in Malayan pangolins in southern China"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.13.945485v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="The ongoing outbreak of viral pneumonia in China and beyond is associated with a novel coronavirus, provisionally termed 2019-nCoV. This outbreak has been tentatively associated with a seafood market in Wuhan, China, where the sale of wild animals may be the source of zoonotic infection. Although bats are likely reservoir hosts for 2019-nCoV, the identity of any intermediate host facilitating transfer to humans is unknown. Here, we report the identification of 2019-nCoV related coronaviruses in pangolins ( Manis javanica ) seized in anti-smuggling operations in southern China. Metagenomic sequencing identified pangolin associated CoVs that belong to two sub-lineages of 2019-nCoV related coronaviruses, including one very closely related to 2019-nCoV in the receptor-binding domain. The discovery of multiple lineages of pangolin coronavirus and their similarity to 2019-nCoV suggests that pangolins should be considered as possible intermediate hosts for this novel human virus and should be removed from wet markets to prevent zoonotic transmission."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-02-18"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Identification of 2019-nCoV related coronaviruses in Malayan pangolins in southern China | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1156566 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-2">The ongoing outbreak of viral pneumonia in China and beyond is associated with a novel coronavirus, provisionally termed 2019-nCoV. This outbreak has been tentatively associated with a seafood market in Wuhan, China, where the sale of wild animals may be the source of zoonotic infection. Although bats are likely reservoir hosts for 2019-nCoV, the identity of any intermediate host facilitating transfer to humans is unknown. Here, we report the identification of 2019-nCoV related coronaviruses in pangolins (
     <em>Manis javanica</em>) seized in anti-smuggling operations in southern China. Metagenomic sequencing identified pangolin associated CoVs that belong to two sub-lineages of 2019-nCoV related coronaviruses, including one very closely related to 2019-nCoV in the receptor-binding domain. The discovery of multiple lineages of pangolin coronavirus and their similarity to 2019-nCoV suggests that pangolins should be considered as possible intermediate hosts for this novel human virus and should be removed from wet markets to prevent zoonotic transmission.
    </p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Text</h2>
    <p id="p-3">An outbreak of serious pneumonia disease was reported in Wuhan, China on 30 December 2019. The causative agent was soon identified as a novel coronavirus - 2019-nCoV
     <sup>
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>
     </sup>. Case numbers grew rapidly from 27 in December 2019 to 7,818 globally as of 30 January 2020
     <sup>
      <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>
     </sup>, leading to the WHO to declare a public health emergency. Many of the early cases were linked to Huanan seafood market in Wuhan city, Hubei province, from where the probable zoonotic source is speculated to originate
     <sup>
      <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>
     </sup>. Currently, only environmental samples taken from the market have been reported as positive for 2019-nCoV by the China CDC
     <sup>
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>
     </sup>. However, as similar wet markets were already implicated in the SARS outbreak of 2002-2003
     <sup>
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>
     </sup>, it seems likely that wild animals are also involved in the emergence of 2019-nCoV. Indeed, a number of non-aquatic mammals were available for purchase in the Huanan seafood market prior to the outbreak
     <sup>
      <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>
     </sup>. Unfortunately, because the market was cleared soon after the outbreak began, determining the source virus in the wild animals from the market is challenging. Although a coronavirus closely related to 2019-nCoV sampled from a 
     <em>Rhinolophus affinis</em> bat in Yunnan in 2013 has now been identified
     <sup>
      <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>
     </sup>, closely related viruses have not yet been detected in other wildlife species. Here, we present the identification of 2019-nCoV related viruses in pangolins smuggled into southern China.
    </p>
    <p id="p-4">We investigated the virome composition of pangolins (mammalian order Pholidota). These animals are of growing importance and interest because they are the most illegally trafficked of any group of mammal: they are used as both a food source and their scales are utilized in traditional Chinese medicine. A number of pangolin species now are regarded as critically endangered on the International Union for Conservation of Nature Red List of Threatened Species. We received frozen tissue (lungs, intestine, blood) samples that were collected from 18 Malayan pangolins (
     <em>Manis javanica</em>) during August 2017-January 2018. These pangolins were obtained during the anti-smuggling operations by Guangxi Customs. Strikingly, high-throughput sequencing of their RNA revealed the presence of coronaviruses in six (two lung, two intestine, one lung-intestine mix, one blood) of 43 samples. With the sequence read data, and by filling gaps with amplicon sequencing, we were able to obtain six full or nearly full genome sequences - denoted GX/P1E, GX/P2V, GX/P3B, GX/P4L, GX/P5E and GX/P5L - that fall into the 2019-nCoV lineage (within the genus 
     <em>Betacoronavirus</em>) in a phylogenetic analysis (
     <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1a</a>). These viruses also have similar genomic organization as 2019-nCoV, with nine predicted open reading frames (
     <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1b</a>; 
     <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Extended Data Table S5</a>). We were also able to successfully isolate the virus using the Vero E6 cell line (
     <a id="xref-fig-3-1" class="xref-fig" href="#F3">Extended Data Figure S1</a>).
    </p>
    <div id="F1" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/18/2020.02.13.945485/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Phylogenetic analyses depicting the evolutionary relationship between human 2019-nCoV, the pangolin coronavirus sequences obtained in this study, and the other reference coronaviruses. The phylogenies were estimated using a maximum likelihood approach employing the GTR+I+[[#x393]] nucleotide substitution model and 1,000 bootstrap replicates. (A) Phylogeny of the subgenus Sarbecovirus (genus Betacoronavirus) estimated from the concatenated ORF1ab-S-E-M-N genes. Red circles indicate the pangolin coronavirus sequences generated in this study. Note that GD/P1L is the consensus sequence re-assembled from the raw data previously published7. (B) Genome organization of coronaviruses including the pangolin coronaviruses, with the predicted ORFs shown in different colors. (C) Sliding window analysis of changing patterns of sequence similarity between 2019-nCoV, pangolin coronaviruses and bat coronavirus RaTG13. The name of the query sequences are shown vertically on the right of the analysis boxes. The similarities to different reference sequences are indicated by different colors shown in the legend box at the top. Guangdong pangolin coronaviruses GD/P1L and GD/P2S were merged for this analysis. The blue arrows at the top indicate the position of the ORFs in the alignment analyzed. The potential recombination breakpoints are shown in pink dash lines, which together slice the genomes into eight regions (regions with " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1849216001" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Phylogenetic analyses depicting the evolutionary relationship between human 2019-nCoV, the pangolin coronavirus sequences obtained in this study, and the other reference coronaviruses. The phylogenies were estimated using a maximum likelihood approach employing the GTR+I+[[#x393]] nucleotide substitution model and 1,000 bootstrap replicates. (A) Phylogeny of the subgenus &lt;;em&gt;;Sarbecovirus&lt;;/em&gt;; (genus &lt;;em&gt;;Betacoronavirus&lt;;/em&gt;;) estimated from the concatenated ORF1ab-S-E-M-N genes. Red circles indicate the pangolin coronavirus sequences generated in this study. Note that GD/P1L is the consensus sequence re-assembled from the raw data previously published&lt;;sup&gt;;7&lt;;/sup&gt;;. (B) Genome organization of coronaviruses including the pangolin coronaviruses, with the predicted ORFs shown in different colors. (C) Sliding window analysis of changing patterns of sequence similarity between 2019-nCoV, pangolin coronaviruses and bat coronavirus RaTG13. The name of the query sequences are shown vertically on the right of the analysis boxes. The similarities to different reference sequences are indicated by different colors shown in the legend box at the top. Guangdong pangolin coronaviruses GD/P1L and GD/P2S were merged for this analysis. The blue arrows at the top indicate the position of the ORFs in the alignment analyzed. The potential recombination breakpoints are shown in pink dash lines, which together slice the genomes into eight regions (regions with &lt;;200bp were omitted; indicated by the red line at bottom) for phylogenetic analysis. (D) Phylogenetic trees of different genomic regions. SARS-CoV and 2019-nCoV related lineages are shown in blue and red tree branches. Branch scale bars are 0.1 substitutions/site.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/18/2020.02.13.945485/F1.medium.gif" width="393" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/18/2020.02.13.945485/F1.medium.gif" width="393" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/18/2020.02.13.945485/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/18/2020.02.13.945485/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 1.</span>
      <p id="p-5" class="first-child">Phylogenetic analyses depicting the evolutionary relationship between human 2019-nCoV, the pangolin coronavirus sequences obtained in this study, and the other reference coronaviruses. The phylogenies were estimated using a maximum likelihood approach employing the GTR+I+Γ nucleotide substitution model and 1,000 bootstrap replicates. (A) Phylogeny of the subgenus 
       <em>Sarbecovirus</em> (genus 
       <em>Betacoronavirus</em>) estimated from the concatenated ORF1ab-S-E-M-N genes. Red circles indicate the pangolin coronavirus sequences generated in this study. Note that GD/P1L is the consensus sequence re-assembled from the raw data previously published
       <sup>
        <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>
       </sup>. (B) Genome organization of coronaviruses including the pangolin coronaviruses, with the predicted ORFs shown in different colors. (C) Sliding window analysis of changing patterns of sequence similarity between 2019-nCoV, pangolin coronaviruses and bat coronavirus RaTG13. The name of the query sequences are shown vertically on the right of the analysis boxes. The similarities to different reference sequences are indicated by different colors shown in the legend box at the top. Guangdong pangolin coronaviruses GD/P1L and GD/P2S were merged for this analysis. The blue arrows at the top indicate the position of the ORFs in the alignment analyzed. The potential recombination breakpoints are shown in pink dash lines, which together slice the genomes into eight regions (regions with &lt;;200bp were omitted; indicated by the red line at bottom) for phylogenetic analysis. (D) Phylogenetic trees of different genomic regions. SARS-CoV and 2019-nCoV related lineages are shown in blue and red tree branches. Branch scale bars are 0.1 substitutions/site.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-6">Based on these new genome sequences, we designed primers for qPCR detection to confirm that the raw samples were positive for the coronavirus. We conducted further qPCR testing on another batch of archived pangolin samples collected between May-July 2018. Among the 19 samples (nine intestine tissues, ten lung tissues) tested from 12 animals, three lung tissue samples were coronavirus positive.</p>
    <p id="p-7">In addition to the animals from Guangxi, after the start of the 2019-nCoV outbreak the Guangzhou Customs Technology Center re-examined their five archived pangolin samples (two skin swabs, one unknown tissue, one scale) obtained in previous anti-smuggling operations in March 2019. Following high-throughput sequencing the scale sample was found to contain coronavirus reads, and from these data we were able to assemble a partial genome sequence of 21,505bp (denoted as GD/P2S), representing approximately 72% of the 2019-nCoV genome. This virus sequence on the scale may in fact come from contaminants of other infected pangolin tissues. Notably, another study of diseased pangolins in Guangdong performed in 2019 also identified viral contigs from lung samples that were similarly related to 2019-nCoV
     <sup>
      <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>
     </sup>. Different assembly methods and manual curation were performed to generate a partial genome sequence that comprised about 86.3% of the full-length virus genome (denoted as GD/P1L in the 
     <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1a</a> tree).
    </p>
    <p id="p-8">These novel pangolin coronavirus genomes have approximately 85.5% to 92.4% similarity to 2019-nCoV, and represent two sub-lineages of 2019-nCoVs in the phylogenetic tree, one of which (GD/P1L and GDP2S) is extremely closely related to 2019-nCoV (
     <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>; red circles). It has previously been noted that members of the subgenus 
     <em>Sarbecovirus</em> have experienced widespread recombination
     <sup>
      <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>
     </sup>. In support of this, a recombination analysis performed here revealed that bat coronaviruses ZC45 and ZCS21 are likely recombinants, containing genome fragments from multiple SARS-CoV related lineages (genome regions 2, 5, 7) and 2019-nCoV related lineages including that from pangolins (regions 1, 3, 4, 6, 8).
    </p>
    <p id="p-9">More notable, however, was the observation of putative recombination signals between the pangolins coronaviruses, bat coronaviruses RaTG13, and human 2019-nCoV (
     <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1c, d</a>). In particular, 2019-nCoV exhibits very high sequence similarity to the Guangdong pangolin coronaviruses in the receptor-binding domain (RBD; 97.4% amino acid similarity; indicated by red arrow in 
     <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1c</a> and 
     <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2a</a>), even though it is most closely related to bat coronavirus RaTG13 in the remainder of the viral genome. Bat CoV RaTG and the human 2019-nCoV have only 89.2% amino acid similarity in RBD. Indeed, the Guangdong pangolin coronaviruses and 2019-nCoV possess identical amino acids at the five critical residues of the RBD, whereas RaTG13 only shares one amino acid with 2019-nCoV (residue 442, human SARS-CoV numbering
     <sup>
      <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>
     </sup>). Interestingly, a phylogenetic analysis of synonymous sites alone in the RBD revealed that the phylogenetic position of the Guangdong pangolin is consistent with that in the remainder of the viral genome, rather than being the closest relative of 2019-nCoV (
     <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2b</a>). Hence, it is possible that the amino acid similarity between the RBD of the Guangdong pangolin coronaviruses and 2019-nCoV is due to selectively-mediated convergent evolution rather than recombination, although it is difficult to choose between these scenarios on current data. Although the drivers of any convergent evolution are unknown, its possible occurrence, as well as that of recombination, would further highlight the role played by intermediate animal hosts in human virus emergence.
    </p>
    <div id="F2" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/18/2020.02.13.945485/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="(A) Sequence alignment showing the receptor-binding domain (RBD) in human, pangolin and bat coronaviruses. The five critical residues for binding between SARS-CoV RBD and human ACE2 protein are indicated in red boxes, and ACE2-contacting residues are indicated with yellow boxes, following Wan et al.9. Note that in Guangdong pangolin sequence, the codon positions coding for amino acid 337 proline, 420 aspartic acid, 499 proline and 519 asparagine have ambiguous nucleotide compositions, resulting to possibly alternative amino acids threonine, glycine, threonine and lysine respectively. GD: Guangdong, GX: Guangxi. (B) Phylogenetic trees of 2019-nCoV related lineage estimated from the whole RBD region (upper) and synonymous sites only (lower). Branch supports obtained from 1,000 bootstrap replicates are shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1849216001" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;(A) Sequence alignment showing the receptor-binding domain (RBD) in human, pangolin and bat coronaviruses. The five critical residues for binding between SARS-CoV RBD and human ACE2 protein are indicated in red boxes, and ACE2-contacting residues are indicated with yellow boxes, following Wan et al.&lt;;sup&gt;;9&lt;;/sup&gt;;. Note that in Guangdong pangolin sequence, the codon positions coding for amino acid 337 proline, 420 aspartic acid, 499 proline and 519 asparagine have ambiguous nucleotide compositions, resulting to possibly alternative amino acids threonine, glycine, threonine and lysine respectively. GD: Guangdong, GX: Guangxi. (B) Phylogenetic trees of 2019-nCoV related lineage estimated from the whole RBD region (upper) and synonymous sites only (lower). Branch supports obtained from 1,000 bootstrap replicates are shown.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/18/2020.02.13.945485/F2.medium.gif" width="440" height="191"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/18/2020.02.13.945485/F2.medium.gif" width="440" height="191"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/18/2020.02.13.945485/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/18/2020.02.13.945485/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 2.</span>
      <p id="p-10" class="first-child">(A) Sequence alignment showing the receptor-binding domain (RBD) in human, pangolin and bat coronaviruses. The five critical residues for binding between SARS-CoV RBD and human ACE2 protein are indicated in red boxes, and ACE2-contacting residues are indicated with yellow boxes, following Wan et al.
       <sup>
        <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>
       </sup>. Note that in Guangdong pangolin sequence, the codon positions coding for amino acid 337 proline, 420 aspartic acid, 499 proline and 519 asparagine have ambiguous nucleotide compositions, resulting to possibly alternative amino acids threonine, glycine, threonine and lysine respectively. GD: Guangdong, GX: Guangxi. (B) Phylogenetic trees of 2019-nCoV related lineage estimated from the whole RBD region (upper) and synonymous sites only (lower). Branch supports obtained from 1,000 bootstrap replicates are shown.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-11">To date, pangolins are the only mammals other than bats documented to be infected by a 2019-nCoV related coronavirus. It is striking that two related lineages of CoVs are found in pangolins and that both are also related to 2019-nCoV. This suggests that these animals may be long-term reservoir hosts for these viruses, which is surprising as pangolins are solitary animals with relatively small population sizes, reflecting their endangered status
     <sup>
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>
     </sup>. However, it cannot be excluded that pangolins acquired their 2019-nCoV related viruses independently from bats or another animal host. It is also notable that both lineages of pangolin coronaviruses were obtained from trafficked Malayan pangolins, likely originating from Southeast Asia, and there is a marked lack of knowledge of the viral diversity maintained by this animal in regions where it is indigenous. Undoubtedly, the extent of virus transmission in pangolin populations requires additional investigation, but the repeated occurrence of infections with 2019-nCoV related coronaviruses in Guangxi and Guangdong provinces suggests that this animal may be a potentially important host in coronavirus emergence.
    </p>
    <p id="p-12">Coronaviruses, including those related to 2019-nCoV, are clearly present in many wild mammals in Asia
     <sup>
      <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>,
      <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>,
      <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">7</a>,
      <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>
     </sup>. Although the epidemiology, pathogenicity, interspecies infectivity and transmissibility of coronaviruses in pangolins remains to be studied, the data presented here strongly suggests that handling these animals requires considerable caution, and that their sale in wet markets should be strictly prohibited. Further surveillance on pangolins in the natural environment in China and Southeast Asia are clearly needed to understand their role in the emergence of 2019-nCoV and the risk of future zoonotic transmission.
    </p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Methods</h2>
    <div id="sec-3" class="subsection">
     <h3>Ethics Statement</h3>
     <p id="p-13">The animals studied here were rescued and treated by the Guangxi Zhuang Autonomous Region Terrestrial Wildlife Medical-aid and Monitoring Epidemic Diseases Research Center under the ethics approval (wild animal treatment regulation No. [2011] 85). The samples were collected following the procedure guideline (Pangolins Rescue Procedure, November, 2016).</p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>Sample collection, viral detection and sequencing of pangolins in Guangxi</h3>
     <p id="p-14">We received frozen tissue samples of 18 pangolins (
      <em>Manis javanica</em>) from Guangxi Medical University, China, which were collected between August 2017 – January 2018. These pangolins were seized by the Guangxi Customs during their routine anti-smuggling operations. All animal individuals comprised samples from multiple organs including lungs, intestine and blood, with the exception of six individuals for which only lung tissues were available, five with mixed intestine and lung tissues only, one with intestine tissues only, and one comprising two blood samples. Using the intestine-lung mixed sample we were able to isolate a novel 
      <em>Betacoronavirus</em> using the Vero-E6 cell line (
      <a id="xref-fig-3-2" class="xref-fig" href="#F3">Extended Data Figure S1</a>). A High Pure Viral RNA Kit (Roche, Switzerland) was used for RNA extraction on all 43 samples. For RNA sequencing, a sequencing library was constructed using an Ion Total RNA-Seq Kit v2 (Thermo Fisher Scientific, MA, USA), and the library was subsequently sequenced using an Ion Torrent S5 sequencer (Thermo Fisher Scientific). Reverse Transcription was performed using an SuperScript III First-Strand Synthesis System for RT-PCR (Thermo Fisher Scientific, MA, USA). DNA libraries were constructed using the NEBNext Ultra II DNA Library Prep Kit and sequenced on a MiSeq sequencer. The NGS QC Toolkit V2.3.3 was used to remove low-quality and short reads. Both BLASTn and BLASTx were used to search against a local virus database, utilizing the data available at NCBI/GenBank. Genome sequences were assembled using the CLC Genomic Workbench v.9.0. To fill gaps in high throughput sequencing and obtain the whole viral genome sequence, amplicon primers based on the bat SARS-like coronavirus ZC45 ( GenBank accession number 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=MG772933&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F18%2F2020.02.13.945485.atom">MG772933</a>) sequence were designed for amplicon-based sequencing.
     </p>
     <p id="p-15">A total of six samples (including the virus isolate) contained reads that matched a 
      <em>Betacoronavirus</em> (
      <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Extended Data Table S1</a>). We obtained near complete genomes from these samples (98%, compared to 2019-nCoV), with the virus genomes denoted as GX/P1E, GX/P2V, GX/P3B, GX/P4L, GX/P5E and GX/P5L. Based on these genome sequences, we designed primers for qPCR to confirm the positivity of the original tissue samples (
      <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Extended Data Table S4</a>). This revealed an original lung tissue sample that was also qPCR positive, in addition to the six original samples with coronavirus reads. We further tested an addition 19 samples (nine intestine tissues, ten lung tissues), from 12 smuggled pangolins sampled between May-July 2018 by the group from Guangxi Medical University. The genome sequences of GX/P1E, GX/P2V, GX/P3B, GX/P4L, GX/P5E and GX/P5L were submitted to GenBank and GISAID databases, their accession numbers will be available as soon as it is generated.
     </p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Sample collection, viral detection and sequencing of pangolins in Guangdong</h3>
     <p id="p-16">After the start of the 2019-nCoV outbreak, the Guangzhou Customs Technology Center re-examined their five archived pangolin samples (two skin swabs, one unknown tissue, one scale) obtained in anti-smuggling operations undertaken in March 2019. RNA was extracted from all five samples (Qiagen, USA), and was subjected to high-throughput RNA sequencing on the Illumina HiSeq platform by Vision medicals, Guangdong, China. The scale sample was found to contain coronavirus reads using BLAST methods. These reads were quality assessed, cleaned and assembled into contigs by both 
      <em>de novo</em> (MEGAHIT v1.1.3
      <sup>
       <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>
      </sup>) and using reference (BWA v0.7.13
      <sup>
       <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>
      </sup>) assembly methods, with BetaCoV/Wuhan/WIV04/2019 as reference. The contigs were combined, and approximately 72% of the coronavirus genome (21,505bp) was obtained. This sequence was denoted as pangolin CoV GD/P2S.
     </p>
     <p id="p-17">Liu 
      <em>et al</em>. recently published a meta-transcriptomic study of pangolins
      <sup>
       <a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">7</a>
      </sup> and deposited 21 RNA-seq raw files on the SRA database (
      <a href="https://www.ncbi.nlm.nih.gov/sra">https://www.ncbi.nlm.nih.gov/sra</a>). We screened these raw read files using BLAST methods and found that five (SRR10168374, SRR10168376, SRR10168377, SRR10168378 and SRR10168392) contained reads that mapped to 2019-nCoV. These reads were subjected to quality assessment, cleaning and then 
      <em>de novo</em> assembly using MEGAHIT
      <sup>
       <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>
      </sup> and reference assembly using BWA
      <sup>
       <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>
      </sup>. These reads were then merged and curated in a pileup alignment file to obtain the consensus sequences. This combined consensus sequence is 25,753bp in length (about 86.3% of BetaCoV/Wuhan/WIV04/2019) and denoted pangolin CoV GD/P1L. Notably, it has 66.8% overlap and only 0.21% divergence (i.e. a sequence identify of 99.79%) with the GD/P2S sequence. Since their genetic difference is so low, for the recombination analysis we merged the GD/P1L and GD/P2S sequences into a single consensus sequence to minimize gap regions within any sequences.
     </p>
     <p id="p-18">The viral genome organizations of Guangxi and Guangdong pangolin coronaviruses were similar to 2019-nCoV. They had a total number of 9 open reading frames (ORFs) and shared the same gene order of ORF1ab replicase, envelope glycoprotein spike (S), envelope (E), membrane (M), nucleocapsid (N), plus other predicted ORFs. Detailed comparison of the ORF length and similarity with 2019-nCoV and bat coronavirus RaTG13 is provided in Extended 
      <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table S5</a>.
     </p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Sequence, phylogenetic and recombination analyses</h3>
     <p id="p-19">The human 2019-nCoV and bat RaTG13 coronavirus genome sequences were downloaded from 
      <a href="http://Virological.org">Virological.org</a> (
      <a href="http://virological.org">http://virological.org</a>) and the GISAID (
      <a href="https://www.gisaid.org">https://www.gisaid.org</a>) databases on 17 January 2020, with the data kindly shared by the submitters (
      <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Extended Data Table S2</a>). Other coronaviruses (subgenus 
      <em>Sarbecovirus</em>) were downloaded from GenBank (
      <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Extended Data Table S3</a>) and compared to those obtained here. We constructed a multiple sequence alignment of their complete genomes and individual genes using MAFFT v7.273
      <sup>
       <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>
      </sup>. Maximum likelihood phylogenies were estimated using PhyML v3.1
      <sup>
       <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>
      </sup>, utilizing the GTR+I+Γ model of nucleotide substitution with 1,000 bootstrap replicates. To investigate potential recombination events, we implemented a window sliding approach to determine the changing patterns of sequence similarity and phylogenetic clustering between the query and the reference sequences, as well as a scanning of phylogenetic clusters performed directly from the multiple sequence alignment. Maximum likelihood trees were estimated from each window extraction (i.e. genome regions 1 to 8) using PhyML as described above.
     </p>
    </div>
   </div>
   <div class="section" id="sec-7">
    <h2 class="">Author contributions</h2>
    <p id="p-22">Y.G. and Y.L.H. designed and supervised research. W.W., W.J.L. and L.F.L. collected samples and conducted genome sequencing. M.H.H.S, X.B.N. and T.T.Y.L. performed genome assembly and annotation. Y.G.T., T.T.Y.L., M.H.H.S, X.B.N, W.Y.M.C. and Y.S.L. performed genome analysis and interpretation. T.T.Y.L. and E.C.H.wrote the paper. H.C.Z., Y.L.H., G.M.L and Y.G. joined the data interpretation and edited the paper.</p>
   </div>
   <div class="section COI-statement" id="sec-8">
    <h2 class="">Competing interests</h2>
    <p id="p-23">No conflict of interest declared.</p>
   </div>
   <div class="section" id="sec-9">
    <h2 class="">Materials &amp;; correspondence</h2>
    <p id="p-24">Correspondence and requests for materials should be addressed to Prof. Yi Guan (
     <span class="em-link">
      <span class="em-addr">yguan{at}hku.hk</span>
     </span>)
    </p>
   </div>
   <div class="section" id="sec-10">
    <h2 class="">Data availability</h2>
    <p id="p-25">Data that support the findings of this study have been deposited in GenBank database with the xxxx-xxxx accession codes and SRA database with xxxx-xxxx accession codes.</p>
   </div>
   <div class="section" id="sec-11">
    <h2 class="">Extended Data</h2>
    <div id="F3" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/18/2020.02.13.945485/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Microscopic image of the cytopathic effect in virus isolation using Vero E6. (A) Negative control of Vero E6 cell line. (B) Cytopathic effect seen in viral culture (5 days post inoculation)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1849216001" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Microscopic image of the cytopathic effect in virus isolation using Vero E6. (A) Negative control of Vero E6 cell line. (B) Cytopathic effect seen in viral culture (5 days post inoculation).&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure S1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/18/2020.02.13.945485/F3.medium.gif" width="440" height="173"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure S1." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/18/2020.02.13.945485/F3.medium.gif" width="440" height="173"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/18/2020.02.13.945485/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/18/2020.02.13.945485/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure S1.</span>
      <p id="p-26" class="first-child">Microscopic image of the cytopathic effect in virus isolation using Vero E6. (A) Negative control of Vero E6 cell line. (B) Cytopathic effect seen in viral culture (5 days post inoculation).</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="T1" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1159548/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup">
         <a href="/highwire/markup/1159548/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
        <li class="download-ppt last">
         <a href="/highwire/powerpoint/1159548" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Table S1.</span>
      <p id="p-27" class="first-child">High-throughput sequencing results of the samples with coronavirus reads</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="T2" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1159556/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup last">
         <a href="/highwire/markup/1159556/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Table S2.</span>
      <p id="p-28" class="first-child">Acknowledgement of sharing of 2019-nCoV genome sequences from the 
       <a href="http://Virological.org">Virological.org</a> and the GISAID databases. We gratefully thank the authors listed below for sharing their genomic sequences of coronaviruses analyzed in this study.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="T3" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1159553/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup">
         <a href="/highwire/markup/1159553/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
        <li class="download-ppt last">
         <a href="/highwire/powerpoint/1159553" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Table S3.</span>
      <p id="p-29" class="first-child">GenBank accession numbers of coronavirus sequences used in this study.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="T4" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1159586/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup">
         <a href="/highwire/markup/1159586/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
        <li class="download-ppt last">
         <a href="/highwire/powerpoint/1159586" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Table S4.</span>
      <p id="p-30" class="first-child">Primers used for qPCR detection of pangolin associated coronavirus</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="T5" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1159585/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup">
         <a href="/highwire/markup/1159585/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
        <li class="download-ppt last">
         <a href="/highwire/powerpoint/1159585" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Table S5.</span>
      <p id="p-31" class="first-child">Genomic comparison of 2019-nCoV with Bat-Cov RaTG13, Guangdong pangolin CoV and Guangxi pangolin CoV.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgements</h2>
    <p id="p-20">We thank Prof. Wu-Chun Cao, Dr. Na Jia, Dr. Ya-Wei Zhang, Dr. Jia-Fu Jiang, Dr. Bao-Gui Jiang, and their team in State Key Laboratory of Pathogen and Biosecurity, Beijing Institute of Microbiology and Epidemiology, Beijing for their substantial contributions to this study, including coordinating among research parties, conducting virus isolation, qPCR and sequencing. We also thank Guangzhou Customs Technology Center, Guangzhou for re-examining their archived pangolin samples and providing the data. All the above persons agree on the content of the study, authorship arrangement and publication. We thank the staff of the Guangxi and Guangdong Custom Bureau for their laborious anti-smuggling operations. We thank all the scientists who kindly shared their genomic sequences of the coronaviruses used in this study.</p>
    <p id="p-21">This work was supported by research grants from National Key Plan for Scientific Research and Development of China (2016YFD0500302; 2017YFE0190800), funding for Guangdong-Hongkong-Macau Joint Laboratory (2019B121205009), The National Natural Science Foundation of China (NSFC) Excellent Young Scientists Fund (Hong Kong and Macau) (31922087), Li Ka Shing Foundation, National Institutes of Health (HHSN272201400006C), and the Australian Research Council (FL170100022).</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.13.945485v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A new coronavirus associated with human respiratory disease in China</span>. 
         <span class="cit-source">Nature</span> (
         <span class="cit-pub-date">2020</span>). 
         <a href="https://doi.org/10.1038/s41586-020-2008-3">https://doi.org/10.1038/s41586-020-2008-3</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.13.945485v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">World Health Organization</span>. 
         <span class="cit-source">WHO Novel Coronavirus (2019-nCoV) situation reports</span>. 
         <a href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports/">https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports/</a>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.13.945485v1.3" data-doi="10.1016/S0140-6736(20)30251-8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</span>. 
         <abbr class="cit-jnl-abbrev">Lancet</abbr>, (
         <span class="cit-pub-date">2020</span>).
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1016/S0140-6736(20)30251-8</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DLancet%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%252930251-8%26rft_id%253Dinfo%253Apmid%252F32007145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)30251-8&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=32007145&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F18%2F2020.02.13.945485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.13.945485v1.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Cohen</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>
         <span class="cit-article-title">Mining coronavirus genomes for clues to the outbreak’s origins</span>. 
         <span class="cit-source">Science</span>, (
         <span class="cit-pub-date">2020</span>) 
         <a href="https://www.sciencemag.org/news/2020/01/mining-coronavirus-genomes-clues-outbreak-s-origins">https://www.sciencemag.org/news/2020/01/mining-coronavirus-genomes-clues-outbreak-s-origins</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.13.945485v1.5" data-doi="10.3201/eid1112.041293">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">SARS-CoV infection in a restaurant from palm civet</span>. 
         <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr>
         <span class="cit-vol">11</span>:
         <span class="cit-fpage">1860</span>–
         <span class="cit-lpage">5</span>, (
         <span class="cit-pub-date">2005</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEmerging%2Binfectious%2Bdiseases%26rft.stitle%253DEmerg%2BInfect%2BDis%26rft.aulast%253DWang%26rft.auinit1%253DM.%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253D1860%26rft.epage%253D1865%26rft.atitle%253DSARS-CoV%2Binfection%2Bin%2Ba%2Brestaurant%2Bfrom%2Bpalm%2Bcivet.%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid1112.041293%26rft_id%253Dinfo%253Apmid%252F16485471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.3201/eid1112.041293&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16485471&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F18%2F2020.02.13.945485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000233768300009&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.13.945485v1.6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. 
         <span class="cit-source">Nature</span> (
         <span class="cit-pub-date">2020</span>). 
         <a href="https://doi.org/10.1038/s41586-020-2012-7">https://doi.org/10.1038/s41586-020-2012-7</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.13.945485v1.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Viral Metagenomics Revealed Sendai Virus and Coronavirus Infection of Malayan Pangolins (Manis javanica)</span>. 
         <abbr class="cit-jnl-abbrev">Viruses</abbr>
         <span class="cit-fpage">11</span>, (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.13.945485v1.8" data-doi="10.1128/JVI.01926-07">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hon</span>, 
          <span class="cit-name-given-names">C.C.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Evidence of the recombinant origin of a bat severe acute respiratory syndrome (SARS)-like coronavirus and its implications on the direct ancestor of SARS coronavirus</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>
         <span class="cit-vol">82</span>, 
         <span class="cit-fpage">1819</span>–
         <span class="cit-lpage">1826</span>, (
         <span class="cit-pub-date">2008</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01926-07%26rft_id%253Dinfo%253Apmid%252F18057240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgyLzQvMTgxOSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzE4LzIwMjAuMDIuMTMuOTQ1NDg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.13.945485v1.9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wan</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Receptor recognition by novel coronavirus from Wuhan: An analysis based on decade-long structural studies of SARS</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>, (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.13.945485v1.10">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Heinrich</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">The global trafficking of pangolins: A comprehensive summary of seizures and trafficking routes from 2010–2015</span>. 
         <abbr class="cit-jnl-abbrev">Trafficking</abbr>, (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.13.945485v1.11">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>
         <span class="cit-etal">et al.</span> 2017. 
         <span class="cit-article-title">Discovery of a Highly Divergent Coronavirus in the Asian House Shrew from China Illuminates the Origin of the Alphacoronaviruses</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>
         <span class="cit-fpage">91</span>, (
         <span class="cit-pub-date">2017</span>). 
         <em>References associated in Methods only:</em>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.13.945485v1.12" data-doi="10.1093/bioinformatics/btv033">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">MEGAHIT: an ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">31</span>, 
         <span class="cit-fpage">1674</span>–
         <span class="cit-lpage">1676</span>, (
         <span class="cit-pub-date">2015</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv033%26rft_id%253Dinfo%253Apmid%252F25609793%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv033&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25609793&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F18%2F2020.02.13.945485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.13.945485v1.13" data-doi="10.1093/bioinformatics/btp698">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">H.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Durbin</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>
         <span class="cit-article-title">Fast and accurate long-read alignment with Burrows-Wheeler transform</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">26</span>, 
         <span class="cit-fpage">589</span>–
         <span class="cit-lpage">595</span>, (
         <span class="cit-pub-date">2010</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp698%26rft_id%253Dinfo%253Apmid%252F20080505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp698&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20080505&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F18%2F2020.02.13.945485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000274973800001&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.13.945485v1.14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rozewicki</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">MAFFT-DASH: integrated protein sequence and structural alignment</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>
         <span class="cit-vol">47</span>, 
         <span class="cit-fpage">W5</span>–
         <span class="cit-lpage">W10</span>, (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D47%26rft.spage%253DW5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.13.945485v1.15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Guindon</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">PHYML Online--a web server for fast maximum likelihood-based phylogenetic inference</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>
         <span class="cit-vol">33</span>, 
         <span class="cit-fpage">W557</span>–
         <span class="cit-lpage">559</span>, (
         <span class="cit-pub-date">2010</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D33%26rft.spage%253DW557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
