<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2017/06/25/155242/embed/inline-graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2017/06/25/155242.atom" />
<meta name="HW.pisa" content="biorxiv;155242v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Mapping the drivers of within-host pathogen evolution using massive data sets" />
<meta name="DC.Identifier" content="10.1101/155242" />
<meta name="DC.Date" content="2017-06-25" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2017, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Differences among hosts, resulting from genetic variation in the immune system or heterogeneity in drug treatment, can impact within-host pathogen evolution. Identifying such interactions can potentially be achieved through genetic association studies. However, extensive and correlated genetic population structure in hosts and pathogens presents a substantial risk of confounding analyses. Moreover, the multiple testing burden of interaction scanning can potentially limit power. To address these problems, we have developed a Bayesian approach for detecting host influences on pathogen evolution that makes use of vast existing data sets of pathogen diversity to improve power and control for stratification. The approach models key processes, including recombination and selection, and identifies regions of the pathogen genome affected by host factors. Using simulations and empirical analysis of drug-induced selection on the HIV-1 genome we demonstrate the power of the method to recover known associations and show greatly improved precision-recall characteristics compared to other approaches. We build a high-resolution map of HLA-induced selection in the HIV-1 genome, identifying novel epitope-allele combinations." />
<meta name="DC.Contributor" content="Duncan S. Palmer" />
<meta name="DC.Contributor" content="Isaac Turner" />
<meta name="DC.Contributor" content="Sarah Fidler" />
<meta name="DC.Contributor" content="John Frater" />
<meta name="DC.Contributor" content="Philip Goulder" />
<meta name="DC.Contributor" content="Dominique Goedhals" />
<meta name="DC.Contributor" content="Kuan-Hsiang Gary Huang" />
<meta name="DC.Contributor" content="Annette Oxenius" />
<meta name="DC.Contributor" content="Rodney Phillips" />
<meta name="DC.Contributor" content="Roger Shapiro" />
<meta name="DC.Contributor" content="Cloete van Vuuren" />
<meta name="DC.Contributor" content="Angela R. McLean" />
<meta name="DC.Contributor" content="Gil McVean" />
<meta name="article:published_time" content="2017-06-25" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Mapping the drivers of within-host pathogen evolution using massive data sets" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Differences among hosts, resulting from genetic variation in the immune system or heterogeneity in drug treatment, can impact within-host pathogen evolution. Identifying such interactions can potentially be achieved through genetic association studies. However, extensive and correlated genetic population structure in hosts and pathogens presents a substantial risk of confounding analyses. Moreover, the multiple testing burden of interaction scanning can potentially limit power. To address these problems, we have developed a Bayesian approach for detecting host influences on pathogen evolution that makes use of vast existing data sets of pathogen diversity to improve power and control for stratification. The approach models key processes, including recombination and selection, and identifies regions of the pathogen genome affected by host factors. Using simulations and empirical analysis of drug-induced selection on the HIV-1 genome we demonstrate the power of the method to recover known associations and show greatly improved precision-recall characteristics compared to other approaches. We build a high-resolution map of HLA-induced selection in the HIV-1 genome, identifying novel epitope-allele combinations.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2017/01/01" />
<meta name="citation_mjid" content="biorxiv;155242v1" />
<meta name="citation_id" content="155242v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/155242v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/155242v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/155242v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2017/06/25/155242.full.pdf" />
<meta name="citation_doi" content="10.1101/155242" />
<meta name="citation_num_pages" content="23" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="155242" />
<meta name="citation_author" content="Duncan S. Palmer" />
<meta name="citation_author_institution" content="Department of Statistics, University of Oxford" />
<meta name="citation_author_institution" content="Wellcome Trust Centre for Human Genetics" />
<meta name="citation_author_institution" content="Institute for Emerging Infections, The Oxford Martin School" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0824-0047" />
<meta name="citation_author" content="Isaac Turner" />
<meta name="citation_author_institution" content="Department of Statistics, University of Oxford" />
<meta name="citation_author_institution" content="Wellcome Trust Centre for Human Genetics" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2378-4886" />
<meta name="citation_author" content="Sarah Fidler" />
<meta name="citation_author_institution" content="Division of Medicine, Wright Fleming Institute, Imperial College" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1676-7583" />
<meta name="citation_author" content="John Frater" />
<meta name="citation_author_institution" content="Institute for Emerging Infections, The Oxford Martin School" />
<meta name="citation_author_institution" content="Nuffield Department of Clinical Medicine, University of Oxford, Peter Medawar Building for Pathogen Research" />
<meta name="citation_author_institution" content="Oxford NIHR Biomedical Research Centre" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7163-7277" />
<meta name="citation_author" content="Philip Goulder" />
<meta name="citation_author_institution" content="Department of Paediatrics, University of Oxford, Peter Medawar Building for Pathogen Research" />
<meta name="citation_author_institution" content="HIV Pathogenesis Programme, Doris Duke Medical Research Institute, University of KwaZulu-Natal" />
<meta name="citation_author" content="Dominique Goedhals" />
<meta name="citation_author_institution" content="Department of Medical Microbiology and Virology, University of the Free State, and National Heath Laboratory Service" />
<meta name="citation_author" content="Kuan-Hsiang Gary Huang" />
<meta name="citation_author_institution" content="Nuffield Department of Clinical Medicine, University of Oxford, Peter Medawar Building for Pathogen Research" />
<meta name="citation_author_institution" content="Einstein Medical Center Philadelphia" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1910-9367" />
<meta name="citation_author" content="Annette Oxenius" />
<meta name="citation_author_institution" content="Institute of Microbiology, Swiss Federal Institute of Technology Zurich" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2079-2354" />
<meta name="citation_author" content="Rodney Phillips" />
<meta name="citation_author_institution" content="Institute for Emerging Infections, The Oxford Martin School" />
<meta name="citation_author_institution" content="Nuffield Department of Clinical Medicine, University of Oxford, Peter Medawar Building for Pathogen Research" />
<meta name="citation_author_institution" content="Oxford NIHR Biomedical Research Centre" />
<meta name="citation_author" content="Roger Shapiro" />
<meta name="citation_author_institution" content="Botswana Harvard AIDS Institute Partnership" />
<meta name="citation_author_institution" content="Department of Immunology and Infectious Diseases, Harvard TH Chan School of Public Health" />
<meta name="citation_author" content="Cloete van Vuuren" />
<meta name="citation_author_institution" content="Department of Medical Microbiology and Virology, University of the Free State, and National Heath Laboratory Service" />
<meta name="citation_author" content="Angela R. McLean" />
<meta name="citation_author_institution" content="Institute for Emerging Infections, The Oxford Martin School" />
<meta name="citation_author_institution" content="Zoology Department, South Parks Road, University of Oxford" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3386-7888" />
<meta name="citation_author" content="Gil McVean" />
<meta name="citation_author_institution" content="Department of Statistics, University of Oxford" />
<meta name="citation_author_institution" content="Wellcome Trust Centre for Human Genetics" />
<meta name="citation_author_institution" content="Big Data Institute, University of Oxford, Li Ka Shing Centre for Health Information and Discovery" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5012-4162" />
<meta name="citation_reference" content="Goulder, P. and B. Walker, HIV and HLA Class I: An Evolving Relationship. Immunity, 2012. 37(3): p. 426–440." />
<meta name="citation_reference" content="Woolthuis, R., et al., Long-term adaptation of the influenza A virus by escaping cytotoxic T-cell recognition. Scientific Reports, 2016. 6: p. 33334." />
<meta name="citation_reference" content="Yauch, L, et al., A Protective Role for Dengue Virus-Specific CD8+ T Cells. The Journal of Immunology, 2009. 182(8): p. 4865–4873." />
<meta name="citation_reference" content="Bowen, D. and C. Walker, Adaptive immune responses in acute and chronic hepatitis C virus infection. Nature, 2005. 436(7053): p. 946–952." />
<meta name="citation_reference" content="Phillips, R., et al., Human immunode ficiency virus genetic variation that can escape cytotoxic Tcell recognition. Nature, 1991. 354(6353): p. 453–459." />
<meta name="citation_reference" content="Butler, N., et al., Structural and Biological Basis of CTL Escape in Coronavirus-lnfected Mice. The Journal of Immunology, 2008. 180(6): p. 3926–3937." />
<meta name="citation_reference" content="Qing, M., et al., Characterization of dengue virus resistance to brequinar in cell culture. Antimicrobial agents and chemotherapy, 2010. 54(9): p. 3686–3695." />
<meta name="citation_reference" content="Sandgren, A., et al., Tuberculosis Drug Resistance Mutation Database. PLOS Medicine, 2009. 6(2): p. e1000002." />
<meta name="citation_reference" content="Shafer, R.W., Rationale and uses of a public HIV drug-resistance database. J Infect Dis, 2006. 194 Suppl 1: p. S51–8." />
<meta name="citation_reference" content="Shafer, R. and J. Schapiro, HIV-1 drug resistance mutations: an updated framework for the second decade of HAART. AIDS reviews, 2008.10(2): p. 67–84." />
<meta name="citation_reference" content="Chen, Z.-W., et al., Global prevalence of pre-existing HCV variants resistant to direct-acting antiviral agents (DAAs): mining the GenBank HCV genome data. Scientific reports, 2016. 6." />
<meta name="citation_reference" content="Martinez, J.L. and F. Baquero, Mutation Frequencies and Antibiotic Resistance. Antimicrobial Agents and Chemotherapy, 2000. 44(7): p. 1771–1777." />
<meta name="citation_reference" content="Tanwar, J., et al., Multidrug Resistance: An Emerging Crisis. Interdisciplinary Perspectives on Infectious Diseases, 2014. 2014: p. 1–7." />
<meta name="citation_reference" content="Zimmerman, P., et al., Red blood cell polymorphism and susceptibility to Plasmodium vivax. Advances in parasitology, 2013. 81: p. 27–76." />
<meta name="citation_reference" content="Lell, B., et al., The role of red blood cell polymorphisms in resistance and susceptibility to malaria. Clinical infectious diseases: an official publication of the Infectious Diseases Society of America, 1999. 28(4): p. 794–799." />
<meta name="citation_reference" content="Moore, C., et al., Evidence of HIV-1 adaptation to H LA-restricted immune responses at a population level. Science (New York, N.Y.), 2002. 296(5572): p. 1439–1443." />
<meta name="citation_reference" content="Palmer, D., et al., Integrating genealogical and dynamical modelling to infer escape and reversion rates in HIV epitopes. Proceedings of the Royal Society B: Biological Sciences, 2013. 280(1762): p. 20130696." />
<meta name="citation_reference" content="Fryer, H., et al., Modelling the Evolution and Spread of HIV Immune Escape Mutants. PLoS Pathog, 2010. 6(11): p. e1001196." />
<meta name="citation_reference" content="Apps, R., et al., Influence of HLA-C Expression Level on HIV Control. Science, 2013. 340(6128): p. 87–91." />
<meta name="citation_reference" content="Carlson, J., et al., Leveraging Hierarchical Population Structure in Discrete Association Studies. PLoS ONE, 2007. 2(7): p. e591." />
<meta name="citation_reference" content="Carlson, J., et al., Widespread Impact of HLA Restriction on Immune Control and Escape Pathways of HIV-1. Journal of Virology, 2012. 86(9): p. 5230–5243." />
<meta name="citation_reference" content="Bhattacharya, T., et al., Founder Effects in the Assessment of HIV Polymorphisms and HLA Allele Associations. Science, 2007. 315(5818): p. 1583–1586." />
<meta name="citation_reference" content="Palmer, D., et al., Integrating genealogical and dynamical modelling to infer escape and reversion rates in HIV epitopes. 2013." />
<meta name="citation_reference" content="Kessinger, T., A. Perelson, and R. Neher, Inferring HIV Escape Rates from Multi-Locus Genotype Data. Frontiers in immunology, 2013. 4." />
<meta name="citation_reference" content="Beerenwinkel, N., et al., Diversity and complexity of HIV-1 drug resistance: A bioinformatics approach to predicting phenotype from genotype. Proceedings of the National Academy of Sciences, 2002. 99(12): p. 8271–8276." />
<meta name="citation_reference" content="Carlson, J., et al., Phylogenetic dependency networks: inferring patterns of CTL escape and codon covariation in HIV-1 Gag. PLoS computational biology, 2008. 4(11): p. e1000225." />
<meta name="citation_reference" content="Carlson, J., et al., HIV-1 adaptation to HLA: a window into virus-host immune interactions. Trends in microbiology, 2015. 23(4): p. 212–224." />
<meta name="citation_reference" content="Chen, L. and C. Lee, Distinguishing HIV-1 drug resistance, accessory, and viral fitness mutations using conditional selection pressure analysis of treated versus untreated patient samples. Biology direct, 2006. 1(1): p. 14." />
<meta name="citation_reference" content="Chen, L., A. Perlina, and C. Lee, Positive selection detection in 40,000 human immunodeficiency virus (HIV) type 1 seguences automatically identifies drug resistance and positive fitness mutations in HIV protease and reverse transcriptase. Journal of virology, 2004. 78(7): p. 3722–3732." />
<meta name="citation_reference" content="Wilson, D. and G. McVean, Estimating diversifying selection and functional constraint in the presence of recombination. Genetics, 2006. 172(3): p. 1411–1425." />
<meta name="citation_reference" content="Nielsen, R. and Z. Yang, Likelihood Models for Detecting Positively Selected Amino Acid Sites and Applications to the HIV-1 Envelope Gene. Genetics, 1998. 148(3): p. 929–936." />
<meta name="citation_reference" content="van Westen, G., et al., Significantly improved HIV inhibitor efficacy prediction employing proteochemometric models generated from antivirogram data. PLoS computational biology, 2013. 9(2)." />
<meta name="citation_reference" content="Menéndez-Arias, L., A. Mas, and E. Domingo, Cytotoxic T-lymphocyte responses to HIV-1 reverse transcriptase (review). Viral immunology, 1998. 11(4): p. 167–181." />
<meta name="citation_reference" content="Liu, Y., et al., Selection on the human immunodeficiency virus type 1 proteome following primary infection. Journal of virology, 2006. 80(19): p. 9519–9529." />
<meta name="citation_reference" content="Liu, Y., et al., Dynamics of viral evolution and CTL responses in HIV-1 infection. PloS one, 2011. 6(1)." />
<meta name="citation_reference" content="Liu, Y., et al., Evolution of human immunodeficiency virus type 1 cytotoxic T-lymphocyte epitopes: fitness-balanced escape. Journal of virology, 2007. 81(22): p. 12179–12188." />
<meta name="citation_reference" content="Collection of B-list epitopes." />
<meta name="citation_reference" content="Gatanaga, H., et al., Naturally selected rilpivirine-resistant HIV-1 variants by host cellular immunity. Clinical infectious diseases: an official publication of the Infectious Diseases Society of America, 2013. 57(7): p. 1051–1055." />
<meta name="citation_reference" content="Ansari, A., et al., Genome-to-genome analysis highlights the effect of the human innate and adaptive immune systems on the hepatitis C virus. Nature Genetics, 2017. 49(5): p. 666–673." />
<meta name="citation_reference" content="Price, A., et al., Principal components analysis corrects for stratification in genome-wide association studies. Nature Genetics, 2006. 38(8): p. 904–909." />
<meta name="citation_reference" content="Li, N. and M. Stephens, Modeling Linkage Diseguilibrium and Identifying Recombination Hotspots Using Single-Nucleotide Polymorphism Data. Genetics, 2003. 165(4): p. 2213–2233." />
<meta name="citation_reference" content="Tenzer, S., et al., HIV-1 Adaptation to Antigen Processing Results in Population-Level Immune Evasion and Affects Subtype Diversification. Cell Reports, 2014. 7(2): p. 448–463." />
<meta name="citation_reference" content="Llano, A., et al., Best-Characterized HIV-1 CTL Epitopes: The 2013 Update. 2013." />
<meta name="citation_reference" content="Short-Course Antiretroviral Therapy in Primary HIV Infection. N Engl J Med, 2013. 368(3): p. 207–217." />
<meta name="citation_reference" content="Benson, D., et al., GenBank. Nucleic Acids Research, 2014. 42(D1): p. D32–D37." />
<meta name="citation_reference" content="Pan, C., et al., The HIV positive selection mutation database. Nucleic acids research, 2007. 35(Database issue): p. D371–D375." />
<meta name="citation_reference" content="Fagard, C., et al., A prospective trial of structured treatment interruptions in human immunodeficiency virus infection. Archives of internal medicine, 2003. 163(10): p. 12201226." />
<meta name="citation_reference" content="Frater, A.J., et al., Effective T-Cell Responses Select Human Immunodeficiency Virus Mutants and Slow Disease Progression. Journal of Virology, 2007. 81(12): p. 6742–6751." />
<meta name="citation_reference" content="Huang, K.-H., et al., Progression to AIDS in South Africa Is Associated with both Reverting and Compensatory Viral Mutations. PLoS ONE, 2011. 6(4): p. el9018." />
<meta name="citation_reference" content="Huang, K.-H.G., et al., Prevalence of HIV type-1 drug-associated mutations in pre-therapy patients in the Free State, South Africa. Antiviral therapy, 2009. 14(7): p. 975–984." />
<meta name="citation_reference" content="Leslie, A., et al., Additive Contribution of HLA Class I Alleles in the Immune Control of HIV-1 Infection. Journal of Virology, 2010. 84(19): p. 9879–9888." />
<meta name="citation_reference" content="Matthews, P., et al., Central Role of Reverting Mutations in HLA Associations with Human Immunodeficiency Virus Set Point. Journal of Virology, 2008. 82(17): p. 8548–8559." />
<meta name="citation_reference" content="Shapiro, R.L., et al., Antiretroviral Regimens in Pregnancy and Breast-Feeding in Botswana. N Engl J Med, 2010. 362(24): p. 2282–2294." />
<meta name="citation_reference" content="Los Alamos HIV Seguence Database." />
<meta name="twitter:title" content="Mapping the drivers of within-host pathogen evolution using massive data sets" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Differences among hosts, resulting from genetic variation in the immune system or heterogeneity in drug treatment, can impact within-host pathogen evolution. Identifying such interactions can potentially be achieved through genetic association studies. However, extensive and correlated genetic population structure in hosts and pathogens presents a substantial risk of confounding analyses. Moreover, the multiple testing burden of interaction scanning can potentially limit power. To address these problems, we have developed a Bayesian approach for detecting host influences on pathogen evolution that makes use of vast existing data sets of pathogen diversity to improve power and control for stratification. The approach models key processes, including recombination and selection, and identifies regions of the pathogen genome affected by host factors. Using simulations and empirical analysis of drug-induced selection on the HIV-1 genome we demonstrate the power of the method to recover known associations and show greatly improved precision-recall characteristics compared to other approaches. We build a high-resolution map of HLA-induced selection in the HIV-1 genome, identifying novel epitope-allele combinations." />
<meta name="og-title" property="og:title" content="Mapping the drivers of within-host pathogen evolution using massive data sets" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/155242v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Differences among hosts, resulting from genetic variation in the immune system or heterogeneity in drug treatment, can impact within-host pathogen evolution. Identifying such interactions can potentially be achieved through genetic association studies. However, extensive and correlated genetic population structure in hosts and pathogens presents a substantial risk of confounding analyses. Moreover, the multiple testing burden of interaction scanning can potentially limit power. To address these problems, we have developed a Bayesian approach for detecting host influences on pathogen evolution that makes use of vast existing data sets of pathogen diversity to improve power and control for stratification. The approach models key processes, including recombination and selection, and identifies regions of the pathogen genome affected by host factors. Using simulations and empirical analysis of drug-induced selection on the HIV-1 genome we demonstrate the power of the method to recover known associations and show greatly improved precision-recall characteristics compared to other approaches. We build a high-resolution map of HLA-induced selection in the HIV-1 genome, identifying novel epitope-allele combinations." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2017-06-25" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/155242v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/155242v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/155242v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/155242v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/46132" />
    <title>Mapping the drivers of within-host pathogen evolution using massive data sets | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__vup9YXfxWOBOGbthJCUtxdPS6xRY2OQ_vXqbojfxPAk__ziC21uj-cnpEf3LG0kjRMEQHSEtjX5ljVuosklz2DZ4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"yMCi57U5EEsbNYjwJNUSkrjoxHrZYZvVxMUTmZuh8e0","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"46132","apath":"\/biorxiv\/early\/2017\/06\/25\/155242.atom","pisa":"biorxiv;155242v1","markup":[{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;155242\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;155242\/DC1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;155242v1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;155242\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;155242\/DC1"}],"processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/155242v1","title":"Mapping the drivers of within-host pathogen evolution using massive data sets","identifier":"node\/46132"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ADChtHBHQ6S+lDT9iznHEAmRpJcKTuNwtf6yvS6WHUQfllEiFlFnhj57ZAjVWRnuy6Y6UfwfDiQatuRJqc78PLh0XB18aNjXR4xboXyyj8S3NrMEs90A8omcVASi\\\\\\\/i4luDi+mvNh3UPFRZjehXZPWZ90KMb6Dp62j1rLRedmkhI7PQLuZhAuO\\\\\\\/+Sjx5a0WlajPol3nITc5licCcPi0XqnaEbQ7n+9W+sMDnbxz5zLEX8Bbq3Jv6+Y9JYtSXZJkvqCl+sRWYIGc59Qjyv0M+uYiEb2qhWnMm7dujyNbYadVQD+kyQmXTx7eWwwjqq1ebBqZZAPtSNLhmdpBAO14U5Po3Qhi+rG4GbLgMsemOqu+UcpsLPERJ2JbziqAmhWrjrDup6k5JykTvGGZ5cQaafTT0ACIH+qohHSVz36OAHPs\\\\\\\/81dlnjoyJGQ3KnCDeZuxpq5sm+WzlyCagFgliEl+mT7CFlwZgIU2OB+fgYsYFAcr4Gyn81NDqWV5vXxP87rFfLR2tY179SEecoNuWFm9uSeVkkyITQ6XfQ4pGnhfwa4uJdb\\\\\\\/oCEVEGbNhptKqSSrEXm5\\\\\\\/mZ6RETqrUovRZqRYLz2zI6VZ2oWd+ITLUrPq+2STOC\\\\\\\/5ladacN+Io755xpbV2Gx8q62\\\\\\\/jvFIP\\\\\\\/q3Ywvx6Sv0CK7h7+uJiIscJjKEqMnay1kqvoIIMwh5\\\\\\\/WP5GBm+WmIcjzzNk6UCb\\\\\\\/3+ZK3GTAq1M3Lkx6\\\\\\\/02n01Asvtyc+i2aLqpkdswo6Z3jNYU0eU7Vz1eOY3fMF2ZVAJBBhq5dhytyXnnxdcICFiXZRd6QKV64d+Tj5\\\\\\\/nehgF8LyM4LfDXczk103yfyrJj41ER7KcFPlQEUGs8hmDUCmg8DaeYweUQkVqGPB5hoZyZCPSloQ95ExtnEqeqLys+C1JlKs93VoWq2MisGX1RvUcFlblf0zHaOKpwIlwO8fzlU6EfNCtrT+lwclX\\\\\\\/VT0A6wP5d+FyDJiif9i\\\\\\\/C6wXtzhHHmaJNsDrIOv9xHgeZVJLvnLqByfdkpWBLHnU1mQ8ZMYEcQ6DZWtK2mAeM673PJn4Mhx5h97hA0TbWD\\\\\\\/4xldpWp3Ql2spWEl+fb04koy8ZmlWuoHaG96JYy+WqNSAU5IUIzlG8GGKrKuReQn7fp1g1xTmT85mNC6zAsUirMoJvpsRlnb4dtM60Pt68zY7IfHc1J+\\\\\\\/poMzq5p8RP8T2g6vZcp18AtdISLYXCC6lxt0g\\\\\\\/lL65d74S9IZZANWzyWvjxLIZCCRrWRROmYqBt0pWUgw+CjM2FZ8t03b1soujx\\\\\\\/JIYeS+oyy9OcuZcpLzhkYZaZt\\\\\\\/Joo=\\\u0022,\\\u0022iv\\\u0022:\\\u0022a10VjQlDo2aHSzTU5GRxVA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022937002772ab17a0239a5dda53aec8843c2f51dbe68cd5357e19d18540ed0867d\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/155242v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-46132 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/155242v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-zSEkQPpICrBPLSvo7dMOZYHbDfHgT74kFYHzgaQTCfQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="46132" id="node46132" data-pisa="biorxiv;155242v1" data-pisa-master="biorxiv;155242" data-apath="/biorxiv/early/2017/06/25/155242.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Mapping the drivers of within-host pathogen evolution using massive data sets</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0824-0047" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Duncan S.</span> <span class="nlm-surname">Palmer</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2378-4886" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Isaac</span> <span class="nlm-surname">Turner</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-1676-7583" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Sarah</span> <span class="nlm-surname">Fidler</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-7163-7277" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">John</span> <span class="nlm-surname">Frater</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Philip</span> <span class="nlm-surname">Goulder</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Dominique</span> <span class="nlm-surname">Goedhals</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-1910-9367" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Kuan-Hsiang Gary</span> <span class="nlm-surname">Huang</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-2079-2354" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Annette</span> <span class="nlm-surname">Oxenius</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Rodney</span> <span class="nlm-surname">Phillips</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Roger</span> <span class="nlm-surname">Shapiro</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Cloete</span> <span class="nlm-surname">van Vuuren</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11"><a href="http://orcid.org/0000-0002-3386-7888" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Angela R.</span> <span class="nlm-surname">McLean</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0002-5012-4162" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Gil</span> <span class="nlm-surname">McVean</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/155242 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node46132" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Duncan S. Palmer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span> <span class='nlm-institution'>Department of Statistics, University of Oxford</span>, 24-29 St Giles’, Oxford OX1 3LB, <span class='nlm-country'>UK</span>.</div><div class='author-affiliation'><span class='nlm-sup'>2</span> <span class='nlm-institution'>Wellcome Trust Centre for Human Genetics</span>, Roosevelt Drive, Oxford OX3 7BN, <span class='nlm-country'>UK</span>.</div><div class='author-affiliation'><span class='nlm-sup'>3</span> <span class='nlm-institution'>Institute for Emerging Infections, The Oxford Martin School</span>, Oxford OX1 3BD, <span class='nlm-country'>UK</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Duncan%2BS.%2BPalmer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palmer%20DS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADuncan%2BS.%2BPalmer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0824-0047" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Duncan S. Palmer</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Isaac Turner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span> <span class='nlm-institution'>Department of Statistics, University of Oxford</span>, 24-29 St Giles’, Oxford OX1 3LB, <span class='nlm-country'>UK</span>.</div><div class='author-affiliation'><span class='nlm-sup'>2</span> <span class='nlm-institution'>Wellcome Trust Centre for Human Genetics</span>, Roosevelt Drive, Oxford OX3 7BN, <span class='nlm-country'>UK</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Isaac%2BTurner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Turner%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIsaac%2BTurner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2378-4886" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Isaac Turner</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sarah Fidler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span> <span class='nlm-institution'>Division of Medicine, Wright Fleming Institute, Imperial College</span>, London W2 1PG, <span class='nlm-country'>UK</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sarah%2BFidler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fidler%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASarah%2BFidler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1676-7583" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sarah Fidler</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">John Frater </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span> <span class='nlm-institution'>Institute for Emerging Infections, The Oxford Martin School</span>, Oxford OX1 3BD, <span class='nlm-country'>UK</span>.</div><div class='author-affiliation'><span class='nlm-sup'>5</span> <span class='nlm-institution'>Nuffield Department of Clinical Medicine, University of Oxford, Peter Medawar Building for Pathogen Research</span>, Oxford OX1 3SY, <span class='nlm-country'>UK</span>.</div><div class='author-affiliation'><span class='nlm-sup'>6</span> <span class='nlm-institution'>Oxford NIHR Biomedical Research Centre</span>, Oxford OX3 7LE, <span class='nlm-country'>UK</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BFrater%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Frater%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BFrater%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7163-7277" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for John Frater</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Philip Goulder </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>7</span> <span class='nlm-institution'>Department of Paediatrics, University of Oxford, Peter Medawar Building for Pathogen Research</span>, Oxford OX1 3SY, <span class='nlm-country'>UK</span>.</div><div class='author-affiliation'><span class='nlm-sup'>8</span> <span class='nlm-institution'>HIV Pathogenesis Programme, Doris Duke Medical Research Institute, University of KwaZulu-Natal</span>, Durban, <span class='nlm-country'>South Africa</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philip%2BGoulder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goulder%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APhilip%2BGoulder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Dominique Goedhals </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>9</span> <span class='nlm-institution'>Department of Medical Microbiology and Virology, University of the Free State, and National Heath Laboratory Service</span>, Bloemfontein, <span class='nlm-country'>South Africa</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dominique%2BGoedhals%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goedhals%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADominique%2BGoedhals%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Kuan-Hsiang Gary Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span> <span class='nlm-institution'>Nuffield Department of Clinical Medicine, University of Oxford, Peter Medawar Building for Pathogen Research</span>, Oxford OX1 3SY, <span class='nlm-country'>UK</span>.</div><div class='author-affiliation'><span class='nlm-sup'>10</span> <span class='nlm-institution'>Einstein Medical Center Philadelphia</span>, 5501 Old York Road, PA 19141, <span class='nlm-country'>USA</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kuan-Hsiang%2BGary%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20KG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKuan-Hsiang%2BGary%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1910-9367" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kuan-Hsiang Gary Huang</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Annette Oxenius </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>11</span> <span class='nlm-institution'>Institute of Microbiology, Swiss Federal Institute of Technology Zurich</span>, 8093 Zurich, <span class='nlm-country'>Switzerland</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Annette%2BOxenius%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oxenius%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnnette%2BOxenius%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2079-2354" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Annette Oxenius</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Rodney Phillips </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span> <span class='nlm-institution'>Institute for Emerging Infections, The Oxford Martin School</span>, Oxford OX1 3BD, <span class='nlm-country'>UK</span>.</div><div class='author-affiliation'><span class='nlm-sup'>5</span> <span class='nlm-institution'>Nuffield Department of Clinical Medicine, University of Oxford, Peter Medawar Building for Pathogen Research</span>, Oxford OX1 3SY, <span class='nlm-country'>UK</span>.</div><div class='author-affiliation'><span class='nlm-sup'>6</span> <span class='nlm-institution'>Oxford NIHR Biomedical Research Centre</span>, Oxford OX3 7LE, <span class='nlm-country'>UK</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rodney%2BPhillips%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Phillips%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARodney%2BPhillips%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Roger Shapiro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>12</span> <span class='nlm-institution'>Botswana Harvard AIDS Institute Partnership</span>, Gaborone BO 320, <span class='nlm-country'>Botswana</span>.</div><div class='author-affiliation'><span class='nlm-sup'>13</span> <span class='nlm-institution'>Department of Immunology and Infectious Diseases, Harvard TH Chan School of Public Health</span>, Boston, MA 02215, <span class='nlm-country'>USA</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Roger%2BShapiro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shapiro%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARoger%2BShapiro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Cloete van Vuuren </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>9</span> <span class='nlm-institution'>Department of Medical Microbiology and Virology, University of the Free State, and National Heath Laboratory Service</span>, Bloemfontein, <span class='nlm-country'>South Africa</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cloete%2Bvan%2BVuuren%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=van%20Vuuren%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACloete%2Bvan%2BVuuren%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Angela R. McLean </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span> <span class='nlm-institution'>Institute for Emerging Infections, The Oxford Martin School</span>, Oxford OX1 3BD, <span class='nlm-country'>UK</span>.</div><div class='author-affiliation'><span class='nlm-sup'>14</span> <span class='nlm-institution'>Zoology Department, South Parks Road, University of Oxford</span>, Oxford OX1 3PS, <span class='nlm-country'>UK</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Angela%2BR.%2BMcLean%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McLean%20AR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAngela%2BR.%2BMcLean%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3386-7888" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Angela R. McLean</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Gil McVean </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span> <span class='nlm-institution'>Department of Statistics, University of Oxford</span>, 24-29 St Giles’, Oxford OX1 3LB, <span class='nlm-country'>UK</span>.</div><div class='author-affiliation'><span class='nlm-sup'>2</span> <span class='nlm-institution'>Wellcome Trust Centre for Human Genetics</span>, Roosevelt Drive, Oxford OX3 7BN, <span class='nlm-country'>UK</span>.</div><div class='author-affiliation'><span class='nlm-sup'>15</span> <span class='nlm-institution'>Big Data Institute, University of Oxford, Li Ka Shing Centre for Health Information and Discovery</span>, Old Road Campus, Oxford OX3 7LF, <span class='nlm-country'>UK</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gil%2BMcVean%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McVean%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGil%2BMcVean%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5012-4162" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gil McVean</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/155242v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:46132" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:46132/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/155242v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:46132" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:46132/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/155242v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:46132" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:46132/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/155242v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:46132" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:46132/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/155242v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:46132" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:46132/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/155242v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:46132" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:46132/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Differences among hosts, resulting from genetic variation in the immune system or heterogeneity in drug treatment, can impact within-host pathogen evolution. Identifying such interactions can potentially be achieved through genetic association studies. However, extensive and correlated genetic population structure in hosts and pathogens presents a substantial risk of confounding analyses. Moreover, the multiple testing burden of interaction scanning can potentially limit power. To address these problems, we have developed a Bayesian approach for detecting host influences on pathogen evolution that makes use of vast existing data sets of pathogen diversity to improve power and control for stratification. The approach models key processes, including recombination and selection, and identifies regions of the pathogen genome affected by host factors. Using simulations and empirical analysis of drug-induced selection on the HIV-1 genome we demonstrate the power of the method to recover known associations and show greatly improved precision-recall characteristics compared to other approaches. We build a high-resolution map of HLA-induced selection in the HIV-1 genome, identifying novel epitope-allele combinations.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">Variation in multiple host factors, both genetic and non-genetic, can influence the genetic composition of infecting pathogens and their subsequent evolutionary trajectory within a host. Examples include human leukocyte antigen (HLA) restriction of epitopes and subsequent escape in HIV-1 and other viruses [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>-<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>], drug-induced selection pressure and appearance of drug-resistance mutations in viruses, bacteria and eukaryotic pathogens [<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>-<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>] and interactions between polymorphic red-blood cell types and malarial disease [<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>,<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>]. Consequently, the molecular mechanisms underlying diverse pathogen-related processes including infection, invasion, immune-response and drug resistance, can potentially be uncovered by studying the association between host factors and the genetic composition of pathogens [<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>-<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>],</p><p id="p-4">However, while it is now feasible to collect large-scale data on pathogen genomic variation and host parameters, reliable hypothesis-free detection of biologically-meaningful associations between host factors and pathogen diversity is challenging for several reasons; the greatest of which is population structure. Host genetic variation often has a strong spatial structure arising from historical patterns of isolation and gene flow. Because of their commensal nature and mode of transmission, most pathogens are likely to share some of this structure, leading to non-causal association between host and pathogen genetic variation. For the same reason, geographical heterogeneity among host factors that influence pathogens causally (e.g. local variation in treatment protocols) may also lead to indirect correlation between host and pathogen genetics.</p><p id="p-5">A second major challenge is statistical power. Consider searching genome-wide for associations between host genomic factors and pathogen genomic factors, in which the number of tests carried out could be in the billions. Naïve correction for multiple testing is likely to eliminate power for anything except the strongest associations. Consequently, there is a need for approaches to association testing that enable prior information about the likely structure of association to be used in the search for signal.</p><p id="p-6">To date, various approaches to testing for association between pathogen genetic variation and host factors, both genetic, as in the case of classical HLA loci, and non-genetic, as in the case of drug resistance, have been developed [<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>-<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>]. Standard association tests, which suffered the problems of stratification described above, were superseded by methods that utilise an inferred phylogeny of pathogen samples to correct for relatedness [<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>-<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>]. Moreover, additional power can be obtained by explicit modelling of the processes of escape and reversion in the context of HLA restriction of HIV-1 [<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>, <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">23</a>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>], However, such approaches have a number of limitations. For example, they do not consider recombination in pathogen genomes, they do not make use of all the data available, they typically do not infer the strength of host-induced selection or combine information across nearby sites in the context of epitope mapping and the more sophisticated approaches are often computationally prohibitive for very large samples.</p><p id="p-7">To address these limitations we have developed a model-based approach to inferring the effect of host factors on pathogen genomic variation. The approach is motivated by the presence of extremely large databases of pathogen genomic data. Given the size of the databases (e.g. the Los Alamos database on HIV-1 has over 150,000 sequences encoding a portion of reverse transcriptase), it is likely that there are sequences closely related to those that infected the individuals within a particular study in question. Our approach aims to infer the most likely ancestral infecting sequence (which may be a mosaic of those in the database) for each individual and therefore to identify the pathogen evolution that has resulted directly from exposure to the current host. Moreover, we model correlations in evolution among adjacent sites within the pathogen that arise through being a shared target of selection (e.g. within a restricted HLA epitope or within a region of a protein where drug resistance and consequent compensatory mutations can evolve). We show that the approach has substantially improved power to detect sites under selection and, through applications to the evolution of drug resistance and escape from HLA-drive immune-restriction in HIV-1, how the method can deliver new insight into important biological processes.</p></div><div class="section" id="sec-2"><h2 class="">Method overview</h2><p id="p-8">Our goal is to estimate the location and nature of host-associated selection upon the pathogen genome sequence through statistical analysis of association between host factor and pathogen genomic variation. We desire our method to have four properties: the ability to use all available pathogen sequence data, irrespective of whether host factors have been measured; the ability to measure the evidence for selection across the genome analysed in a hypothesis-free manner; the ability to combine information across neighbouring positions where appropriate; and the ability to account for confounding factors such as recombination and population structure. To meet these requirements we have developed a Bayesian model-based approach in which we use an approximation to the coalescent with recombination and codon-level selection [<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>]. In extension to the earlier work, we enable host-specific factors to influence patterns of variation and consider the case of two data sets. The first, <strong><em>D</em></strong>, represents the collection of viral sequences for which host factor data (e.g. HLA genotype or drug treatment) is also available. The second, <strong><em>D<sub>B</sub></em></strong>, represents a much larger dataset of viral sequences for which host factor data is not available. We use <strong><em>D<sub>B</sub></em></strong> to model the set of potential viral sequences that a host can be infected with (allowing for recombination) and assume that genetic differences between this reference panel and <strong><em>D</em></strong>, reflects evolution within the host. Thus host-induced selection will result in an association between host factors and the evolutionary changes observed in <strong><em>D</em></strong>. A cartoon of the underlying process and our model is shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>.
</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/06/25/155242/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The underlying process and our approximation. (a) The underlying process: Infected hosts are circles, coloured by host factor information. Coloured strips represent viral consensus within infected hosts, and arrows indicate the direction of infection. Sampled hosts with pathogen sequence information are outlined. A red outline indicates that host factor information is also available, (b) Our approximation: Pathogen sequences in D are generated from DB, modulated by host factors of D. As |DB| &#xBB; 0, we assume DB is the set of all possible sequences an individual can be infected with. Members of D arise from DB through recombination and mutation. We assume all selection along a lineage connecting each member of D to its closest neighbour in DB (subject to recombination) occurred within the host that the member of D was isolated from. Thus only host factor information for D is required. In (b), there is one green host factor, h. Recombination, shown by the arrow, results in the colouring of the sequence. Three mutations have occurred: one due to h- associated selection (red star), one reversion (green circle), and one synonymous transversion (black cross)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1129203249" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;The underlying process and our approximation.&lt;/span&gt; &lt;em&gt;(a)&lt;/em&gt; The underlying process: Infected hosts are circles, coloured by host factor information. Coloured strips represent viral consensus within infected hosts, and arrows indicate the direction of infection. Sampled hosts with pathogen sequence information are outlined. A red outline indicates that host factor information is also available, &lt;em&gt;(b)&lt;/em&gt; Our approximation: Pathogen sequences in &lt;strong&gt;&lt;em&gt;D&lt;/em&gt;&lt;/strong&gt; are generated from &lt;strong&gt;&lt;em&gt;D&lt;sub&gt;B&lt;/sub&gt;&lt;/em&gt;&lt;/strong&gt;, modulated by host factors of &lt;strong&gt;&lt;em&gt;D&lt;/em&gt;&lt;/strong&gt;. As |&lt;strong&gt;&lt;em&gt;D&lt;sub&gt;B&lt;/sub&gt;&lt;/em&gt;&lt;/strong&gt;| &#xBB; 0, we assume &lt;strong&gt;&lt;em&gt;D&lt;sub&gt;B&lt;/sub&gt;&lt;/em&gt;&lt;/strong&gt; is the set of all possible sequences an individual can be infected with. Members of &lt;strong&gt;&lt;em&gt;D&lt;/em&gt;&lt;/strong&gt; arise from &lt;strong&gt;&lt;em&gt;D&lt;sub&gt;B&lt;/sub&gt;&lt;/em&gt;&lt;/strong&gt; through recombination and mutation. We assume all selection along a lineage connecting each member of &lt;strong&gt;&lt;em&gt;D&lt;/em&gt;&lt;/strong&gt; to its closest neighbour in &lt;strong&gt;&lt;em&gt;D&lt;sub&gt;B&lt;/sub&gt;&lt;/em&gt;&lt;/strong&gt; (subject to recombination) occurred within the host that the member of &lt;strong&gt;&lt;em&gt;D&lt;/em&gt;&lt;/strong&gt; was isolated from. Thus only host factor information for &lt;strong&gt;&lt;em&gt;D&lt;/em&gt;&lt;/strong&gt; is required. In &lt;em&gt;(b)&lt;/em&gt;, there is one green host factor, &lt;em&gt;h.&lt;/em&gt; Recombination, shown by the arrow, results in the colouring of the sequence. Three mutations have occurred: one due to &lt;em&gt;h-&lt;/em&gt; associated selection (red star), one reversion (green circle), and one synonymous transversion (black cross).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/06/25/155242/F1.medium.gif" width="440" height="143"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2017/06/25/155242/F1.medium.gif" width="440" height="143"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/06/25/155242/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/06/25/155242/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">The underlying process and our approximation.</span><p id="p-9" class="first-child"><em>(a)</em> The underlying process: Infected hosts are circles, coloured by host factor information. Coloured strips represent viral consensus within infected hosts, and arrows indicate the direction of infection. Sampled hosts with pathogen sequence information are outlined. A red outline indicates that host factor information is also available, <em>(b)</em> Our approximation: Pathogen sequences in <strong><em>D</em></strong> are generated from <strong><em>D<sub>B</sub></em></strong>, modulated by host factors of <strong><em>D</em></strong>. As |<strong><em>D<sub>B</sub></em></strong>| » 0, we assume <strong><em>D<sub>B</sub></em></strong> is the set of all possible sequences an individual can be infected with. Members of <strong><em>D</em></strong> arise from <strong><em>D<sub>B</sub></em></strong> through recombination and mutation. We assume all selection along a lineage connecting each member of <strong><em>D</em></strong> to its closest neighbour in <strong><em>D<sub>B</sub></em></strong> (subject to recombination) occurred within the host that the member of <strong><em>D</em></strong> was isolated from. Thus only host factor information for <strong><em>D</em></strong> is required. In <em>(b)</em>, there is one green host factor, <em>h.</em> Recombination, shown by the arrow, results in the colouring of the sequence. Three mutations have occurred: one due to <em>h-</em> associated selection (red star), one reversion (green circle), and one synonymous transversion (black cross).</p><div class="sb-div caption-clear"></div></div></div><p id="p-10">The justification of the approach is that if |<strong><em>D<sub>B</sub></em></strong>| » |<strong><em>D</em></strong>| then members of <strong><em>D</em></strong> will typically coalesce very recently and approximately independently with members of <strong><em>D<sub>B</sub></em></strong>. Thus the prior on <strong><em>D</em></strong> can be modelled with the Li and Stephens imperfect mosaic hidden Markov model (HMM), utilising a modified NY98 codon model [<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>]. To incorporate host induced selection, we define a ‘consensus’ codon <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/inline-graphic-1.gif"/></noscript></span></span> at site <em>i</em> and model selection as a scaled increase in the rate of non-synonymous change away from <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/inline-graphic-2.gif"/></noscript></span></span>. Conversely, we model reversion as a scaled increase in the codon substitution rate towards <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/inline-graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/inline-graphic-3.gif"/></noscript></span></span>. To obtain emission probabilities we integrate over the distribution of coalescent time between a member of <strong><em>D</em></strong> and <strong><em>D<sub>B</sub></em></strong> assuming a coalescent model.</p><p id="p-11">The HMM formulation enables efficient computation of the probability of observing a given sequence in of <strong><em>D</em></strong> given current parameter values. The product over all members of <strong><em>D</em></strong> is used to approximate the joint probability <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/inline-graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/inline-graphic-4.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/inline-graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/inline-graphic-5.gif"/></noscript></span></span> is host factor data for members of <strong><em>D</em></strong> and <em>θ</em> represents the parameters of recombination and codon substitution. To speed up computation, we <em>a priori</em> identify a subset of <strong><em>D<sub>B</sub></em></strong> for each member of <strong><em>D</em></strong> that is used to model its ancestor. The model parameters we aim to infer are: the synonymous transition rate, <em>μ</em>, the <em>dN/dS</em> ratio at each site, <em>ω<sub>i</sub></em>, the recombination probabilities between neighbouring sites, <em>r<sub>i</sub></em>, ithe rate of reversion <em>ζ<sub>i</sub></em> and the host-factor-dependent scaling of escape rate at each site, <em>γ<sub>h,i</sub></em>. Other parameters (transition-transversion rate and fraction of codon substitutions more than one nucleotide change away) are estimated from external data. We use Markov chain Monte Carlo to sample from the posterior distribution for all parameters. Full details of the methodology are given in the Methods and Supplementary Methods: Supplementary Text 1.</p></div><div class="section" id="sec-3"><h2 class="">Simulations and methods comparison</h2><p id="p-12">To validate our method and compare its performance to alternative approaches for detecting host-induced selection we carried out two simulation studies. First, we simulated data under the fitted model to evaluate power and accuracy. We used HIV-1 protease sequence data from three major public databases as a reference (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>; <em>n</em> = 146,523), and simulated 100 replicates of a data set of 460 study sequences with six different HLA-induced selection profiles, four of which were common (n = 100 each) and two of which were rare (n = 30 each); Supplementary Figure 1. We performed simulations with recombination ranging between 0 and 0.01 (a rate of 0.01 meaning that on average the ancestor of each study sample copies 100 contiguous codons from a reference sample before recombination) and with the size of the selected sample-specific ancestral reference panel ranging from 10 to 100. In the absence of recombination, parameter estimates are largely unbiased and accurate, although the rate of reversion has high variance (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>; Supplementary Figure 2). Estimates for rare alleles perform similarly to those for common alleles (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). We find little impact of the size of the sample-specific reference data set and the posterior distributions are well calibrated (Supplementary Figure 3). Recombination leads to a downward bias in estimates of recombination and selection intensity and upward bias in the rate of reversion (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>; Supplementary Figure 4) and consequently poorer posterior calibration at sites under selection (Supplementary Figure 4). Nevertheless, the inferred profiles of host-dependent and host-independent selection pressures remain strongly correlated to the truth. We also considered the impact of error in choosing the sample-specific reference panel by forcing inclusion of the sequences actually copied. We find that forcing the true ancestors to be included improves accuracy by only a small margin (Supplementary Figure 5) indicating that 100 potential ancestors chosen through Hamming distance is sufficient.

</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/06/25/155242/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Simulation results summary. Inference results for Simulation studies 1 and 2 are shown in (a-d). (a) Simulation study 1; r = 0. &#x3B3; for a common HLA (n = 100), dN / dS ratio, recombination probability between adjacent sites (r), and reversion scaling (&#x3B6;). (b) Simulation study 1; r = 0, &#x3B3; for a rare HLA (n = 30). (c) Simulation study 1; r = 0.01, &#x3B3; for a common HLA (n = 100). (d) Simulation study 2, &#x3B3; for a common HLA (n = 100). In (a-d) (except &#x3C9; in (a)), averages are taken over 100 independent MCMC runs on independent simulated data with the same underlying parameters. The true value is shown in blue, mean and median estimates are in black and red respectively. White bands enclose 50% credible intervals, in turn enclosed by grey 95% credible intervals. In (c), the truth is rescaled by the average number of individuals between a randomly chosen pair of leaves in the sampled birth-death tree. For &#x3C9; in (a), averages are taken over a single MCMC run, chosen at random as &#x3C9; differs across simulation runs (sampled from the prior). See Figures S2-S5 for full results summaries, (e) ROC curves for existing method. Six methods used to identify HLA associated selection on viral sequence are applied to data simulated under the birth-death process used in simulation study 2. The Inset zooms in to the region enclosed by the black box. ROC curves for 100 independent birth-death simulations are lightly coloured, and averaged to generate the heavier lines. ROC curves for Fryer Approx, PhyloD and PhyloD OR do not extend to (1,1). For Fryer Approx this is because we stop our threshold for estimated rates at 0. For PhyloD and PhyloD OR, it is because many sites will not be included in leaf distribution or logistic regression respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1129203249" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Simulation results summary.&lt;/span&gt; Inference results for Simulation studies 1 and 2 are shown in (&lt;em&gt;a-d&lt;/em&gt;). (&lt;em&gt;a&lt;/em&gt;) Simulation study 1; &lt;em&gt;r&lt;/em&gt; = 0. &lt;em&gt;&#x3B3;&lt;/em&gt; for a common HLA (&lt;em&gt;n =&lt;/em&gt; 100), &lt;em&gt;dN / dS&lt;/em&gt; ratio, recombination probability between adjacent sites (&lt;em&gt;r&lt;/em&gt;), and reversion scaling (&#x3B6;). &lt;em&gt;(b)&lt;/em&gt; Simulation study 1; &lt;em&gt;r&lt;/em&gt; = 0, &lt;em&gt;&#x3B3;&lt;/em&gt; for a rare HLA (&lt;em&gt;n =&lt;/em&gt; 30). &lt;em&gt;(c)&lt;/em&gt; Simulation study 1; &lt;em&gt;r&lt;/em&gt; = 0.01, &lt;em&gt;&#x3B3;&lt;/em&gt; for a common HLA &lt;em&gt;(n =&lt;/em&gt; 100). &lt;em&gt;(d)&lt;/em&gt; Simulation study 2, &lt;em&gt;&#x3B3;&lt;/em&gt; for a common HLA &lt;em&gt;(n =&lt;/em&gt; 100). In &lt;em&gt;(a-d)&lt;/em&gt; (except &lt;em&gt;&#x3C9;&lt;/em&gt; in (&lt;em&gt;a&lt;/em&gt;)), averages are taken over 100 independent MCMC runs on independent simulated data with the same underlying parameters. The true value is shown in blue, mean and median estimates are in black and red respectively. White bands enclose 50% credible intervals, in turn enclosed by grey 95% credible intervals. In &lt;em&gt;(c)&lt;/em&gt;, the truth is rescaled by the average number of individuals between a randomly chosen pair of leaves in the sampled birth-death tree. For &lt;em&gt;&#x3C9;&lt;/em&gt; in &lt;em&gt;(a)&lt;/em&gt;, averages are taken over a single MCMC run, chosen at random as &lt;em&gt;&#x3C9;&lt;/em&gt; differs across simulation runs (sampled from the prior). See Figures S2-S5 for full results summaries, &lt;em&gt;(e)&lt;/em&gt; ROC curves for existing method. Six methods used to identify HLA associated selection on viral sequence are applied to data simulated under the birth-death process used in simulation study 2. The Inset zooms in to the region enclosed by the black box. ROC curves for 100 independent birth-death simulations are lightly coloured, and averaged to generate the heavier lines. ROC curves for Fryer Approx, PhyloD and PhyloD OR do not extend to (1,1). For Fryer Approx this is because we stop our threshold for estimated rates at 0. For PhyloD and PhyloD OR, it is because many sites will not be included in leaf distribution or logistic regression respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/06/25/155242/F2.medium.gif" width="336" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2017/06/25/155242/F2.medium.gif" width="336" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/06/25/155242/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/06/25/155242/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Simulation results summary.</span><p id="p-13" class="first-child">Inference results for Simulation studies 1 and 2 are shown in (<em>a-d</em>). (<em>a</em>) Simulation study 1; <em>r</em> = 0. <em>γ</em> for a common HLA (<em>n =</em> 100), <em>dN / dS</em> ratio, recombination probability between adjacent sites (<em>r</em>), and reversion scaling (ζ). <em>(b)</em> Simulation study 1; <em>r</em> = 0, <em>γ</em> for a rare HLA (<em>n =</em> 30). <em>(c)</em> Simulation study 1; <em>r</em> = 0.01, <em>γ</em> for a common HLA <em>(n =</em> 100). <em>(d)</em> Simulation study 2, <em>γ</em> for a common HLA <em>(n =</em> 100). In <em>(a-d)</em> (except <em>ω</em> in (<em>a</em>)), averages are taken over 100 independent MCMC runs on independent simulated data with the same underlying parameters. The true value is shown in blue, mean and median estimates are in black and red respectively. White bands enclose 50% credible intervals, in turn enclosed by grey 95% credible intervals. In <em>(c)</em>, the truth is rescaled by the average number of individuals between a randomly chosen pair of leaves in the sampled birth-death tree. For <em>ω</em> in <em>(a)</em>, averages are taken over a single MCMC run, chosen at random as <em>ω</em> differs across simulation runs (sampled from the prior). See Figures S2-S5 for full results summaries, <em>(e)</em> ROC curves for existing method. Six methods used to identify HLA associated selection on viral sequence are applied to data simulated under the birth-death process used in simulation study 2. The Inset zooms in to the region enclosed by the black box. ROC curves for 100 independent birth-death simulations are lightly coloured, and averaged to generate the heavier lines. ROC curves for Fryer Approx, PhyloD and PhyloD OR do not extend to (1,1). For Fryer Approx this is because we stop our threshold for estimated rates at 0. For PhyloD and PhyloD OR, it is because many sites will not be included in leaf distribution or logistic regression respectively.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/322962/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/322962/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/322962" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-14" class="first-child">Data used in this study.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">In the second set of simulations we assessed robustness by simulating data under a birth-death model (without recombination), setting the current infected population size at 1,000,000 and sampling proportion at 10% to define <strong><em>D<sub>B</sub></em></strong> (Supplementary Text 2). We find some attenuation of the selection signal (and over-estimation of the reversion parameter) but strong correlation between the inferred and estimated strengths of host-dependent and host-independent selection (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>; Supplementary Figure 6).</p><p id="p-16">The birth-death simulations also enables comparison of our approach with five alternative methods for identifying sites under host-factor specific selection (Supplementary Text 3): Fisher’s exact test (as in Moore <em>et al.</em> [<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">16</a>]); a phylogenetically corrected Fisher’s exact test (as in Bhattacharya <em>et al.</em> [<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">22</a>]); an approximate escape rate estimate (as in Fryer <em>et al.</em> [<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>]); a ‘Phylogenetic dependency networks’ approach - PhyloD (as in Carlson <em>et al.</em> [<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a>]); and PhyloD OR (as in Carlson <em>et al</em> [<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>]). Note that the phylogenetic approaches can only be applied if there is no recombination. In each case, we assume that the ‘wild-type’ strain is correctly defined and set any non-synonymous difference from consensus as a candidate for escape. For each method, we obtained p-values or parameter estimates that provide a metric for the strength of selection conditional on each HLA type and generated receiver operating characteristic (ROC) curves for each simulation (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2E</a>). We find that our approach dramatically increases sensitivity for a given false positive rate (FPR). For example at FPR = 0.01, our sensitivity is 0.65, compared to the second best performing method, PhyloD, at 0.13. We therefore conclude that augmenting study data with a large reference data set and modelling the escape process explicitly provides a substantial gain in the ability to identify sites under host-factor specific selection.</p></div><div class="section" id="sec-4"><h2 class="">Drug associated selection</h2><p id="p-17">To provide empirical validation of our approach for detecting host-factor dependent selection we analysed HIV-1 data from the Stanford drug resistance database [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>] in which viral sequences are linked to antiretroviral drug treatment history of the patient. We therefore aim to learn drug-treatment specific evolution, while other selective factors (e.g. selection due to CTL pressure, the antibody response, or the APOBEC3G response) will be captured by <em>ω.</em> We analysed protease and reverse transcriptase independently and excluded integrase due to lack of data [<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>]. Priors on parameters are given in Supplementary Table 1. We defined the collection of sequences in hosts not receiving any therapy at the time of sequencing as <strong><em>D<sub>B</sub></em></strong>, and viral sequences in hosts receiving any treatment (coupled with their drug regime data) as <strong><em>D</em></strong>. We then randomly assign 2000 sequences from <strong><em>D<sub>B</sub></em></strong> to <strong><em>D</em></strong>. This ensures that drug-associated selection common to all treatments is not absorbed into <em>ω</em>. Full details of data preparation are given in Supplementary Text 4.</p><p id="p-18">We estimated <em>γ<sub>i</sub></em><sub>,<em>h</em></sub> for protease and reverse transcriptase inhibitors prescribed to at least 10 individuals in <strong><em>D</em></strong> (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>; Supplementary Figure 7) and identified sites with very strong evidence for drug-induced escape (median estimate of selection factor &gt; 2 and at least 97.5% of the posterior &gt; 1; top-tier) and moderate evidence (median estimate of selection factor &gt; 1.5 and at least 90% of the posterior &gt; 1; second tier). We compared the enrichment of sites identified to known major drug resistance mutations (DRMs) [<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">9</a>] (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). For some drugs, DRM data was lacking because the drug is no longer commonly used (DDC and DLV), was an experimental drug that failed (<em>α</em>APA, and ADV) or is now used in small amounts with other drugs (RTV). We observed strong, statistically significant and consistent enrichment of DRMs at sites identified as selected; with elevated enrichment (as measured by odds-ratio) in the strongest DRMs and sites with strongest evidence for selection. For example, of the 31 strongest DRMs in RT, 11 are found in the top tier selected sites and a further 6 in the second tier. Of the 41 apparent false positive sites, 17 are described as being selected for in the literature (but not classified as a major drug resistance mutation in the Stanford database); for example mutation at codon 11 of RT is known to be associated with minor reductions in APV susceptibility [<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>], but is not classified as a major DRM [<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">9</a>], A further three apparent false positive are sites of DRMs for treatment cocktails including the drug, nine are sites of DRMs for drugs in the same class but where more specific information is not provided, seven are sites of DRMs for different drugs in the same class and only five appear to have no support in the literature (Supplementary Table 2). False negatives are caused by sites where the selected codon is more than one nucleotide change away from the consensus (five mutations), one is an insertion, eight have <em>in vitro</em> support for drug resistance but are not documented as being selected for <em>in vivo</em>, seven have no compelling evidence in the literature and three are genuinely missed (Supplementary Table 3) [<a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">9</a>]. Further evidence for the biological validity of the inferred profiles is given by the co-clustering of related drugs in trees constructed from the selection intensity profiles (Supplementary Figure 8). In summary, we find that the method can identify sites selected by specific drug treatments <em>in vivo</em>, either directly through resistance or indirectly through compensation for resistance mutations that reduce fitness.
</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/06/25/155242/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Drug associated selection analysis. (a) Protease results summary. Grey bars highlight the region analysed. Codon position is measured relative to the HXB2 from the start of protease. Rows summarise drug-associated selection - drugs are shown to the left. Purple bars show the number of individuals prescribed the drug at viral sampling. Red and orange indicate our median estimate is &gt; 2 and &gt; 1.5 respectively. Sites are outlined in Black and grey if the 2.5% and 10% quantile is &gt; 1 respectively. Green and blue lines at the top of the plots highlight differences between subtype B/C at the amino-acid level, and sites of DRMs [9] respectively. Classes of DRM are displayed: solid white lines tag sites of DRMs which &#x2018;confer the highest levels of resistance&#x2019; for that drug, dashed white lines tag &#x2018;major&#x2019; DRMs, and dotted white lines tag DRMs &#x2018;when combined with other mutations&#x2019; [9]. Drugs highlighted red do not have major DRMs in the Stanford drug resistance database. Median &#x3C9; is displayed at the foot of (a), according to the colourbar. Key: APV = Amprenavir; ATV = Atazanavir; IDV = Indinavir; LPV = Lopinavir; NFV = Nelfinavir; RTV = Ritonavir; SQV = Saquinavir, (b) Sensitivity of our inference. Proportions of the strongest major DRMs and major DRMs we identify when using our top-tier and second-tier candidate sites are provided, with associated p-values and odds-ratios (OR)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1129203249" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Drug associated selection analysis.&lt;/span&gt; &lt;em&gt;(a)&lt;/em&gt; Protease results summary. Grey bars highlight the region analysed. Codon position is measured relative to the HXB2 from the start of protease. Rows summarise drug-associated selection - drugs are shown to the left. Purple bars show the number of individuals prescribed the drug at viral sampling. Red and orange indicate our median estimate is &gt; 2 and &gt; 1.5 respectively. Sites are outlined in Black and grey if the 2.5% and 10% quantile is &gt; 1 respectively. Green and blue lines at the top of the plots highlight differences between subtype B/C at the amino-acid level, and sites of DRMs [9] respectively. Classes of DRM are displayed: solid white lines tag sites of DRMs which &#x2018;confer the highest levels of resistance&#x2019; for that drug, dashed white lines tag &#x2018;major&#x2019; DRMs, and dotted white lines tag DRMs &#x2018;when combined with other mutations&#x2019; [9]. Drugs highlighted red do not have major DRMs in the Stanford drug resistance database. Median &lt;em&gt;&#x3C9;&lt;/em&gt; is displayed at the foot of &lt;em&gt;(a)&lt;/em&gt;, according to the colourbar. Key: APV = Amprenavir; ATV = Atazanavir; IDV = Indinavir; LPV = Lopinavir; NFV = Nelfinavir; RTV = Ritonavir; SQV = Saquinavir, &lt;em&gt;(b)&lt;/em&gt; Sensitivity of our inference. Proportions of the strongest major DRMs and major DRMs we identify when using our top-tier and second-tier candidate sites are provided, with associated p-values and odds-ratios (OR).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/06/25/155242/F3.medium.gif" width="440" height="166"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2017/06/25/155242/F3.medium.gif" width="440" height="166"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/06/25/155242/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/06/25/155242/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Drug associated selection analysis.</span><p id="p-19" class="first-child"><em>(a)</em> Protease results summary. Grey bars highlight the region analysed. Codon position is measured relative to the HXB2 from the start of protease. Rows summarise drug-associated selection - drugs are shown to the left. Purple bars show the number of individuals prescribed the drug at viral sampling. Red and orange indicate our median estimate is &gt; 2 and &gt; 1.5 respectively. Sites are outlined in Black and grey if the 2.5% and 10% quantile is &gt; 1 respectively. Green and blue lines at the top of the plots highlight differences between subtype B/C at the amino-acid level, and sites of DRMs [<a id="xref-ref-9-6" class="xref-bibr" href="#ref-9">9</a>] respectively. Classes of DRM are displayed: solid white lines tag sites of DRMs which ‘confer the highest levels of resistance’ for that drug, dashed white lines tag ‘major’ DRMs, and dotted white lines tag DRMs ‘when combined with other mutations’ [<a id="xref-ref-9-7" class="xref-bibr" href="#ref-9">9</a>]. Drugs highlighted red do not have major DRMs in the Stanford drug resistance database. Median <em>ω</em> is displayed at the foot of <em>(a)</em>, according to the colourbar. Key: APV = Amprenavir; ATV = Atazanavir; IDV = Indinavir; LPV = Lopinavir; NFV = Nelfinavir; RTV = Ritonavir; SQV = Saquinavir, <em>(b)</em> Sensitivity of our inference. Proportions of the strongest major DRMs and major DRMs we identify when using our top-tier and second-tier candidate sites are provided, with associated p-values and odds-ratios (OR).</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-5"><h2 class="">HLA-associated selection</h2><p id="p-20">While the extent to which host HLA alleles can influence patterns of escape and reversion in HIV-1 has been studied extensively, the methods developed here provide additional power and accuracy as well as provide control against factors such as population stratification and recombination. Moreover, they also provide a framework in which to combine data from multiple previous studies. We have assembled nearly 3,000 HIV-1 sequences from patients with known HLA class I genotypes from six studies, of European and African ancestry representing a mixture of subtype B and C sequences (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Moreover, we augment the data with HIV-1 genome data taken from three sources, representing 250,000 − 280,000 sequences depending on the gene (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). We analysed the data in two ways, first by considering escape as deviation from subtype B and also be considering escape from subtype C. We expect these analyses to yield similar results, except at sites where subtypes differ systematically. We analysed data from protease and reverse transcriptase and considered all HLA loci jointly. Priors on parameters are given in Supplementary Table 1. Full details of data preparation are given in Supplementary Text 4. We report results for all alleles for which we have over 10 copies. We define ‘top-tier’ HLA associated candidate sites as those where the median <em>γ<sub>i,h</sub></em> &gt; 2 and the lower 2.5% quantile &gt; 1, and ‘second-tier’ candidate sites as those where the 10% quantile of <em>γ<sub>i,h</sub></em> &gt; 1.</p><p id="p-21">To illustrate the validity of our approach, we first considered the B*51 restricted epitope TAFTIPSI in reverse transcriptase (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). We find a strong signal of selection within the epitope, with the escape site (position 135) experiencing the strongest rate elevation. Interestingly a variant at site 129 which is known to abrogate CTL recognition <em>in vitro</em> [<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>], is highly conserved <em>in vivo</em> and shows no signal for selection, presumably due to other fitness consequences. In contrast, we also see strong evidence for B*51 associated escape around codons 173 and 195, which have not been reported previously. Other known allele-epitope combinations that we recover include, for example, B*18 associated selection around codon 138 [<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>-<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>].
</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/06/25/155242/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Selected results summaries. (a) B*51 associated selection away from across the analysed region of reverse transcriptase. Site numbering is relative to the HXB2, starting from beginning of reverse transcriptase. Plotting is as in Figure 2 (a-d). The B*51 epitope TAFTIPSI is highlighted by the blue box &#x2013; the site of the known escape variant is highlighted red. The black rectangle is zoomed in on in (b). (c) Epitopes predicted by Motif Scan and A-list/B-list epitopes for two example HLA types are displayed. Corresponding results from our inference are shown on the grey strips &#x2013; colouring is summarised by the key." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1129203249" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Selected results summaries.&lt;/span&gt; &lt;em&gt;(a) B&lt;/em&gt;*51 associated selection away from across the analysed region of reverse transcriptase. Site numbering is relative to the HXB2, starting from beginning of reverse transcriptase. Plotting is as in Figure 2 &lt;em&gt;(a-d).&lt;/em&gt; The &lt;em&gt;B&lt;/em&gt;*51 epitope TAFTIPSI is highlighted by the blue box &#x2013; the site of the known escape variant is highlighted red. The black rectangle is zoomed in on in &lt;em&gt;(b). (c)&lt;/em&gt; Epitopes predicted by Motif Scan and A-list/B-list epitopes for two example HLA types are displayed. Corresponding results from our inference are shown on the grey strips &#x2013; colouring is summarised by the key.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/06/25/155242/F4.medium.gif" width="440" height="265"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2017/06/25/155242/F4.medium.gif" width="440" height="265"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/06/25/155242/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/06/25/155242/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">Selected results summaries.</span><p id="p-22" class="first-child"><em>(a) B</em>*51 associated selection away from <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/F4/embed/inline-graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/F4/embed/inline-graphic-6.gif"/></noscript></span></span>across the analysed region of reverse transcriptase. Site numbering is relative to the HXB2, starting from beginning of reverse transcriptase. Plotting is as in <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a> <em>(a-d).</em> The <em>B</em>*51 epitope TAFTIPSI is highlighted by the blue box – the site of the known escape variant is highlighted red. The black rectangle is zoomed in on in <em>(b). (c)</em> Epitopes predicted by Motif Scan and A-list/B-list epitopes for two example HLA types are displayed. Corresponding results from our inference are shown on the grey strips – colouring is summarised by the key.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">To compare to experimental and previous work on epitope restriction we classify documented epitopes into an A-list, which represents the best-defined experimentally determined HIV-1 CTL epitopes, updated yearly [<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>] and a B-list, which refers to the entire collection of epitopes reported in the literature [<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>], Among the A-list epitopes, some have documented CTL escape variants, either <em>in vivo</em> or <em>in vitro.</em> See Supplementary Table 4-5 for details and Supplementary Text 5 for discussion of each case. We also consider an <em>in silico</em> set of predictions for ‘strongly-binding’ anchor residues generated by Motif-Scan [<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>]. We compare these sites to our estimates of HLA-associated selection across protease and reverse transcriptase (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, Supplementary Figures 9-13).
</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/06/25/155242/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="HLA-B associated selection in reverse transcriptase. (a) Motif scan compared to our inference. Inference of HLA associated selection from  is summarised on the grey strips: sites with median &#x3B3;i,h &gt; 2 and 1.5 are coloured red and orange respectively, sites with 2.5% and 10% quantile &gt; 1 are outlined in black and white respectively. Blue and green vertical lines tag sites of drug resistance mutations and sites that differentiate subtype B/C viruses at the amino acid level. We scan reverse transcriptase for putative epitopes matching to known binding motifs in the literature; see key. Anchor residues are highlighted in red. (b) A-list/B-list compared to our inference. Inference results on the grey strips are coloured as in (a). Two grey strips per HLA show selection away from and  respectively. Bar plots to the right show HLA frequencies. The two lowest strips summarise median &#x3C9;, see colour bar. A-list [43] and B-list [37] epitopes above the grey are coloured purple and pink respectively. Sites of known escape variants within the A-list epitopes are highlighted yellow." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1129203249" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;HLA-B associated selection in reverse transcriptase.&lt;/span&gt; &lt;em&gt;(a)&lt;/em&gt; Motif scan compared to our inference. Inference of HLA associated selection from  is summarised on the grey strips: sites with median &lt;em&gt;&#x3B3;&lt;sub&gt;i,h&lt;/sub&gt; &gt;&lt;/em&gt; 2 and 1.5 are coloured red and orange respectively, sites with 2.5% and 10% quantile &gt; 1 are outlined in black and white respectively. Blue and green vertical lines tag sites of drug resistance mutations and sites that differentiate subtype B/C viruses at the amino acid level. We scan reverse transcriptase for putative epitopes matching to known binding motifs in the literature; see key. Anchor residues are highlighted in red. &lt;em&gt;(b)&lt;/em&gt; A-list/B-list compared to our inference. Inference results on the grey strips are coloured as in &lt;em&gt;(a).&lt;/em&gt; Two grey strips per HLA show selection away from and  respectively. Bar plots to the right show HLA frequencies. The two lowest strips summarise median &lt;em&gt;&#x3C9;&lt;/em&gt;, see colour bar. A-list [43] and B-list [37] epitopes above the grey are coloured purple and pink respectively. Sites of known escape variants within the A-list epitopes are highlighted yellow.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/06/25/155242/F5.medium.gif" width="440" height="311"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2017/06/25/155242/F5.medium.gif" width="440" height="311"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/06/25/155242/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/06/25/155242/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <span class="caption-title">HLA-B associated selection in reverse transcriptase.</span><p id="p-24" class="first-child"><em>(a)</em> Motif scan compared to our inference. Inference of HLA associated selection from <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/F5/embed/inline-graphic-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/F5/embed/inline-graphic-7.gif"/></noscript></span></span> is summarised on the grey strips: sites with median <em>γ<sub>i,h</sub> &gt;</em> 2 and 1.5 are coloured red and orange respectively, sites with 2.5% and 10% quantile &gt; 1 are outlined in black and white respectively. Blue and green vertical lines tag sites of drug resistance mutations and sites that differentiate subtype B/C viruses at the amino acid level. We scan reverse transcriptase for putative epitopes matching to known binding motifs in the literature; see key. Anchor residues are highlighted in red. <em>(b)</em> A-list/B-list compared to our inference. Inference results on the grey strips are coloured as in <em>(a).</em> Two grey strips per HLA show selection away from <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/F5/embed/inline-graphic-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/F5/embed/inline-graphic-8.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/F5/embed/inline-graphic-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/F5/embed/inline-graphic-9.gif"/></noscript></span></span> respectively. Bar plots to the right show HLA frequencies. The two lowest strips summarise median <em>ω</em>, see colour bar. A-list [<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>] and B-list [<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">37</a>] epitopes above the grey are coloured purple and pink respectively. Sites of known escape variants within the A-list epitopes are highlighted yellow.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">To assess overlap between sites identified we measured the enrichment of sites identified as under selection and used a permutation strategy to assess significance (Supplementary Text 7). Across protease and reverse transcriptase we find strong evidence for enrichment of top-tier signals of selection at A-list epitopes for HLA-B (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). As the confidence in the epitope collection or the strength of selection decreases, so does the enrichment. We find no evidence for enrichment of selection at computationally predicted epitopes and little evidence for overlap of HLA-A associated sites, though in general we note that HLA-A alleles show much weaker evidence for selection in general (Supplementary Figures 9 and 12). We also find very little evidence for HLA-C driven selection at all (Supplementary Figures 10 and 13).

</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/322902/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/322902/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/322902" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2.</span> <p id="p-26" class="first-child">Overlap between sites under selection and known HLA epitopies</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/322927/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/322927/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/322927" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3.</span> <p id="p-27" class="first-child">Codons in reverse transcriptase showing evidence for HLA associated selection</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">Overall, only a small number of sites identified here as being HLA associated have been described previously; four in reverse transcriptase (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>),and one in protease (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). Conversely, not all previously reported escape mutation sites are identified here. In some cases this may be due to low sample size in this study. However, it is also possible that earlier studies failed to account adequately for linkage disequilibrium between HLA alleles. We also identified potentially novel signals of HLA-driven epitope escape. For example, B*45 shows multiple signals of selection around codon 200 of reverse transcriptase (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>), yet there are no reported epitopes. This may reflect a historical bias towards studies being carried out in European-ancestry populations with subtype B viruses where some HLA alleles and viral epitopes are rare. However, we also note that results between the subtype-B and subtype-C consensus analyses are highly concordant, with a few interesting exceptions. For example, B*58 shows evidence for inducing strong selection away from the subtype B consensus around codon 122-123 of reverse transcriptase but not away from the subtype C consensus. Interestingly, this is a position of divergence between subtype B and subtype C viruses and B*58 is typically more common in populations with HIV-1 subtype C viruses, suggesting that B*58-induced selection pressure may have driven the fixation of the difference between subtypes.
</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/323128/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/323128/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/323128" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4.</span> <p id="p-29" class="first-child">Codons in protease showing evidence for HLA associated selection</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">To assess the relationship between sequence similarity between HLA alleles and the inferred HLA-induced selection profiles, we measured the concordance between dendrograms inferred from pairwise differences of classical HLA alleles protein sequences and dendrograms inferred from the estimated selection profiles (Supplementary Text 8). We find that closely related HLA alleles have closely related selection profiles for HLA-A (odds-ratio = 2.1; permutation p-value = 0.017) and HLA-B (odds-ratio = 3.1; permutation p-value = 0.014), but not HLA-C (odds-ratio = 0.71; permutation p-value = 0.71). However, deeper structure within the inferred trees shows little concordance.</p><p id="p-31">Finally, as noted recently [<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>], a number of sites both show evidence of HLA-driven selection as well as being associated with evolution in response to drug treatment (for example, around codon 70 in reverse transcriptase for B*45). Depending on whether the two types of selection act in the same or different directions, such pressures could either speed up or delay the origin of drug resistance. These results suggest that HLA genotype could be a predictor of the response to drug treatment.</p></div><div class="section" id="sec-6"><h2 class="">Discussion</h2><p id="p-32">Differences between hosts, such as in their immune system, or treatment received, can lead to host-specific selection regimes and subsequent adaptation by the pathogen. If within-host adaptation is at a cost to intrinsic fitness, then, on further transmission, selection may favour reversion to the fitter, ancestral state. To learn about such forces, the ideal experiment is to compare the genetic composition of the original infecting pathogen strain and that present sometime after successful infection across a large number of hosts that differ only in the factors of interest. While only strictly possible in experimental systems, observational approaches aim to learn about the same processes, but, to do so, have to make assumptions about the (unobserved) pool of pathogens to which infected individuals were exposed and the distribution of potential confounding factors. If assumptions are not met, power to detect true associations will be reduced and there is a risk of false positive results.</p><p id="p-33">The introduction of phylogenetic methods for association testing in pathogen genomics [<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>-<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">23</a>, <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">26</a>, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>] provided a partial solution to many issues, by using genome-wide relatedness between pathogens as a proxy for correlation in unobserved confounding (such as population structure), similar to the use of principal components in the analysis of human genetic association [<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>]. Moreover, by developing explicit models of molecular evolution in response to host factors, it is possible to learn relative or even actual rates of escape and reversion [<a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">23</a>]. However, phylogenetic methods have limitations. For example, they can be computationally expensive, meaning they are hard to apply to huge data sets; assumptions of homogeneity in rates over time and space typically have to be made; and, more fundamentally, recombination is widespread among pathogens.</p><p id="p-34">To address these shortcomings, we have developed a model-based approach to association testing that exploits the availability of extremely large reference data sets on pathogen variation (but where there is no relevant metadata concerning the factors of interest), which are likely to contain genomes closely related to the samples of interest (with relevant metadata). Moreover, we argue that when identifying host-driven selection, most of the information lies near the tips of the trees as deeper comparisons have to integrate over many transmission events. By utilising the Li and Stephens [<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>] haplotype model, combined with previous work on modelling adaptive evolution and reversion [<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">18</a>, <a id="xref-ref-23-6" class="xref-bibr" href="#ref-23">23</a>, <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a>] we have developed an approach that both dramatically increases power to detect associations and scales to huge data sets.</p><p id="p-35">The analysis of <em>in vivo</em> patterns of evolution provides a complementary approach to the <em>in vitro</em> study of drug resistance and immune evasion. The evolutionary response reflects the combination of fitness gains of escape and the fitness loss through modification. Hence sites with strong resistance may have too strong a fitness cost to be typically selected for and weak resistance changes that have little fitness impact may be strongly selected for. Moreover, compensatory changes in the pathogen genome can potentially counteract fitness loss and hence appear as part of the resistance response. Our analysis of drug-associated evolution using data from the Los Alamos database identifies the majority of major DRMs and sites not identified typically result from failings of the model (variants that are more than one change away from the consensus codon, insertions and deletions). However, we also identify novel sites with compelling evidence for drug-associated selection, likely reflecting minor DRMs with small fitness loss and compensatory changes.</p><p id="p-36">The analysis of HLA-associated selection presented here is the most complete and comprehensive to date, covering six studies and nearly 3,000 HIV-1 sequences with linked patient HLA class I data. We show the method can recover well-known epitopes and associated escape mutations and that there is strong overlap between curated lists of epitopes and those identified here. However, we also found that simple <em>in silico</em> epitope prediction fails to provide a good predictor of the HIV-1 evolutionary response and multiple cases where curated epitopes do not appear to lead to a substantial selective pressure. We also showed a preponderance of signals of escape associated with HLA-B alleles, as opposed to HLA-A and HLA-C and identified a number of sites where differences between subtype consensus sequences collocate with sites selected by particular HLA alleles, raising the question of whether differences in HLA allele frequency between populations could have contributed to the differentiation of HIV-1 subtypes [<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>].</p><p id="p-37">As field-based sequencing of pathogens becomes feasible, so does the potential to accumulate vast data sets that can track spatial and temporal shifts in selection pressures affecting pathogens and their responses. Such studies will require new approaches to studying genetic association. While the method developed here is aimed at detecting within-host evolutionary responses, future extensions could include adaptation at a population level, for example through exploiting longitudinal sampling and/or spatial heterogeneity in treatment.</p></div><div class="section" id="sec-7"><h2 class="">Online Methods</h2><p>We perform inference via Markov chain Monte-Carlo (MCMC) using the Li and Stephens approximation to the coalescent with recombination [<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">41</a>], with one further key approximation. Consider two datasets:
</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-39"><strong><em>D</em></strong> <em>=</em> viral sequences for which host HLA data is also available.</p></li><li id="list-item-2"><p id="p-40"><strong><em>D<sub>B</sub></em></strong> <em>=</em> viral sequences for which host HLA data is not available.</p></li></ol><p id="p-41">As <strong><em>D<sub>B</sub></em></strong> is very large, we can assume that <strong><em>D<sub>B</sub></em></strong> represents the collection of viral sequences (subject to recombination) that a host can be infected with. A large number of sequences in <strong><em>D<sub>B</sub></em></strong> results in short coalescence times between any given member of <strong><em>D</em></strong> and <strong><em>D<sub>B</sub></em></strong>, increasing our power to estimate parameters. By making this approximation, we may then assume that selection along the lineage joining a given member <em>d</em> of <strong><em>D</em></strong> to its nearest neighbour in <strong><em>D<sub>B</sub></em></strong> occurred within the individual from whom the virus <em>d</em> was sampled. We have associated host HLA profiles for <strong><em>D</em></strong>, and so may then estimate HLA-associated selection conditional on this HLA information. A cartoon of this model is shown in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1b</a>. Full details of our approach and inference regime are provided in Supplementary Text 1.</p><p id="p-42">The Li and Stephens approximation allows us to evaluate the probability that a member of <strong><em>D</em></strong> arises as an imperfect mosaic of sequences in <strong><em>D<sub>B</sub></em></strong> using a hidden Markov model (HMM). To model this process, we must specify a recombination model, and model of sequence evolution. We assume that recombination is site dependent, and occurs with probability <em>r<sub>i</sub></em> between site <em>i</em> – 1 and <em>i</em> to any other member of <strong><em>D<sub>B</sub></em></strong> uniformly at random. We model sequence change using the NY98 codon model [<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a>] (which distinguishes transitions from transversions via the transition/transversion ratio; k, and nonsynonymous changes from synonymous changes via the <em>dN/dS</em> ratio <em>ω</em>), with some modifications. To incorporate HLA-associated selection for escape, we define a ‘consensus’ non-escaped strain <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/inline-graphic-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/inline-graphic-10.gif"/></noscript></span></span> for the region to be analysed. A scaling, Π<em><sub>h∈H</sub>γ<sub>i,h</sub></em>, dependent on the host’s HLA profile, <strong><em>H</em></strong>, then modifies any non-synonymous codon change from this consensus strain. We model reversion in a similar manner: a boost in codon substitution rate towards the consensus codon <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/inline-graphic-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/inline-graphic-11.gif"/></noscript></span></span> at each site, <em>ζ<sub>i</sub></em>. To switch from rates to probabilities we integrate over the length of a lineage connecting a member of <strong><em>D</em></strong> to <strong><em>D<sub>B</sub></em></strong> assuming a standard coalescent. For example for a non-synonymous transition from <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/inline-graphic-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/inline-graphic-12.gif"/></noscript></span></span> to some codon <em>C</em>,
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/graphic-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/graphic-10.gif"/></noscript></span></span>
where <em>ϕ</em> is the empirically estimated probability of observing a single codon change along the lineage joining <strong><em>D</em></strong> to <strong><em>D<sub>B</sub></em></strong><em>, A</em> is the total codon substitution rate out of <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/inline-graphic-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/inline-graphic-13.gif"/></noscript></span></span>, <em>α<sub>i</sub></em> is the number of nonsynonymous transitions from <em>C<sub>i</sub>,k</em> is the number of sequences in <strong><em>D<sub>B</sub></em></strong> and <em>π<sub>c</sub></em> is the empirical probability of observing <em>C</em> given a ≥2 step codon change. We approximate the probability in this manner to avoid computationally expensive matrix exponentiation (Supplementary Text 1). Given recombination probabilities and codon transition probabilities for all possible codon changes and the Li and Stephens approximation, we are armed with an HMM to describe an approximation to the process generating members of <strong><em>D</em></strong>. We can therefore use the forwards and backwards algorithms to integrate over all paths through <strong><em>D<sub>B</sub></em></strong> to generate each member of <strong><em>D</em></strong> in turn. We then take the product over all members of <strong><em>D</em></strong> to approximate the likelihood:
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/graphic-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/graphic-11.gif"/></noscript></span></span>
where <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/inline-graphic-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/inline-graphic-14.gif"/></noscript></span></span> HLA information (across members of <strong><em>D</em></strong> and <strong><em>D<sub>B</sub></em></strong>)<em>, <strong>H<sub>i</sub></strong></em> is the collection of HLA types of the host associated to viral sample <em>D<sub>i</sub></em> ∈ <strong><em>D</em></strong>, and <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/inline-graphic-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/inline-graphic-15.gif"/></noscript></span></span> is the function describing the Li and Stephens approximation with our model of recombination and codon evolution.</p><p id="p-43">Evaluating <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/inline-graphic-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/inline-graphic-16.gif"/></noscript></span></span> in practice becomes computationally difficult as <strong><em>D<sub>B</sub></em></strong> increases in size, as it involves the evaluation of arrays of dimension |<strong><em>D<sub>B</sub></em></strong>| × |<strong><em>D</em></strong>| × <em>|codon sequence</em>|. To increase computational efficiency, we restrict <strong><em>D<sub>B</sub></em></strong> to a subset <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/inline-graphic-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/inline-graphic-17.gif"/></noscript></span></span> for each member of <strong><em>D</em></strong>. In our simulations and analyses we use a simple Hamming distance restriction: the closest <em>n</em> sequences by Hamming distance to define each <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/inline-graphic-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/06/25/155242/embed/inline-graphic-18.gif"/></noscript></span></span>.</p><p>Now that we are able to evaluate approximate likelihoods rapidly, we can perform MCMC to obtain posterior distributions for our parameters of interest:
</p><ul class="list-unord " id="list-2"><li id="list-item-3"><p id="p-45"><em>dN/dS</em> at each site, <em>ω<sub>i</sub></em>.</p></li><li id="list-item-4"><p id="p-46">H LA-associated selection at each site, <em>γ<sub>h</sub>,<sub>i</sub></em>.</p></li><li id="list-item-5"><p id="p-47">Recombination probabilities between neighbouring sites, <em>r<sub>i</sub></em>.</p></li><li id="list-item-6"><p id="p-48">Synonymous transition rate, <em>μ.</em></p></li></ul><p id="p-49">Move proposals to explore that space of parameters and our inference model are provided in the Supplementary Text 1.</p></div><div class="section" id="sec-8"><h2 class="">Code and data availability</h2><p id="p-51">Software for performing analyses is available from <a href="http://https://github.com/astheeggeggs/mcqueen">https://github.com/astheeggeggs/mcqueen</a> under the MIT licence.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-50">Funded by an EPSRC studentship to DP, Wellcome Trust grant 100956/Z/13/Z to GM, Wellcome Trust grant WT104748MA to PJRG, National Institutes of Health grant R01AI46995 to PJRG, National Institute of Allergy and Infectious Diseases grant U01-AI066454 to RS, European Union grant SANTE/2007/147-790 to DG and CV. JF funded by the MRC. We thank Hannah Roberts for methods discussions.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.1" data-doi="10.1016/j.immuni.2012.09.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goulder</span>,  <span class="cit-name-given-names">P.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Walker</span></span>, <span class="cit-article-title">HIV and HLA Class I: An Evolving Relationship</span>. <abbr class="cit-jnl-abbrev">Immunity</abbr>, <span class="cit-pub-date">2012</span>. <span class="cit-vol">37</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">426</span>–<span class="cit-lpage">440</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunity%26rft.volume%253D37%26rft.spage%253D426%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.immuni.2012.09.005%26rft_id%253Dinfo%253Apmid%252F22999948%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.immuni.2012.09.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22999948&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F06%2F25%2F155242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309199000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">2.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-155242v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woolthuis</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Long-term adaptation of the influenza A virus by escaping cytotoxic T-cell recognition</span>. <abbr class="cit-jnl-abbrev">Scientific Reports</abbr>, <span class="cit-pub-date">2016</span>. <span class="cit-vol">6</span>: p. <span class="cit-fpage">33334</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScientific%2BReports%26rft.volume%253D6%26rft.spage%253D33334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">3.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-155242v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yauch</span>,  <span class="cit-name-given-names">L</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">A Protective Role for Dengue Virus-Specific CD8+ T Cells</span>. <abbr class="cit-jnl-abbrev">The Journal of Immunology</abbr>, <span class="cit-pub-date">2009</span>. <span class="cit-vol">182</span>(<span class="cit-issue">8</span>): p. <span class="cit-fpage">4865</span>–<span class="cit-lpage">4873</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BImmunology%26rft.volume%253D182%26rft.spage%253D4865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">4.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-155242v1.4" data-doi="10.1038/nature04079"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bowen</span>,  <span class="cit-name-given-names">D.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Walker</span></span>, <span class="cit-article-title">Adaptive immune responses in acute and chronic hepatitis C virus infection</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-pub-date">2005</span>. <span class="cit-vol">436</span>(<span class="cit-issue">7053</span>): p. <span class="cit-fpage">946</span>–<span class="cit-lpage">952</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBowen%26rft.auinit1%253DD.%2BG.%26rft.volume%253D436%26rft.issue%253D7053%26rft.spage%253D946%26rft.epage%253D952%26rft.atitle%253DAdaptive%2Bimmune%2Bresponses%2Bin%2Bacute%2Band%2Bchronic%2Bhepatitis%2BC%2Bvirus%2Binfection.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04079%26rft_id%253Dinfo%253Apmid%252F16107834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04079&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16107834&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F06%2F25%2F155242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231263900036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">5.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-155242v1.5" data-doi="10.1038/354453a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Phillips</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Human immunode ficiency virus genetic variation that can escape cytotoxic Tcell recognition</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-pub-date">1991</span>. <span class="cit-vol">354</span>(<span class="cit-issue">6353</span>): p. <span class="cit-fpage">453</span>–<span class="cit-lpage">459</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPhillips%26rft.auinit1%253DR.%2BE.%26rft.volume%253D354%26rft.issue%253D6353%26rft.spage%253D453%26rft.epage%253D459%26rft.atitle%253DHuman%2Bimmunodeficiency%2Bvirus%2Bgenetic%2Bvariation%2Bthat%2Bcan%2Bescape%2Bcytotoxic%2BT%2Bcell%2Brecognition.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F354453a0%26rft_id%253Dinfo%253Apmid%252F1721107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/354453a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1721107&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F06%2F25%2F155242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991GV07800051&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Butler</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Structural and Biological Basis of CTL Escape in Coronavirus-lnfected Mice</span>. <abbr class="cit-jnl-abbrev">The Journal of Immunology</abbr>, <span class="cit-pub-date">2008</span>. <span class="cit-vol">180</span>(<span class="cit-issue">6</span>): p. <span class="cit-fpage">3926</span>–<span class="cit-lpage">3937</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BImmunology%26rft.volume%253D180%26rft.spage%253D3926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.7" data-doi="10.1128/AAC.00561-10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Qing</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Characterization of dengue virus resistance to brequinar in cell culture</span>. <abbr class="cit-jnl-abbrev">Antimicrobial agents and chemotherapy</abbr>, <span class="cit-pub-date">2010</span>. <span class="cit-vol">54</span>(<span class="cit-issue">9</span>): p. <span class="cit-fpage">3686</span>–<span class="cit-lpage">3695</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntimicrobial%2Bagents%2Band%2Bchemotherapy%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAAC.00561-10%26rft_id%253Dinfo%253Apmid%252F20606073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWFjIjtzOjU6InJlc2lkIjtzOjk6IjU0LzkvMzY4NiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA2LzI1LzE1NTI0Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">8.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-155242v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sandgren</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Tuberculosis Drug Resistance Mutation Database</span>. <abbr class="cit-jnl-abbrev">PLOS Medicine</abbr>, <span class="cit-pub-date">2009</span>. <span class="cit-vol">6</span>(<span class="cit-issue">2</span>): p. <span class="cit-fpage">e1000002</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BMedicine%26rft.volume%253D6%26rft.spage%253D1000002e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.9" data-doi="10.1086/505356"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shafer</span>,  <span class="cit-name-given-names">R.W.</span></span>, <span class="cit-article-title">Rationale and uses of a public HIV drug-resistance database</span>. <abbr class="cit-jnl-abbrev">J Infect Dis</abbr>, <span class="cit-pub-date">2006</span>. <span class="cit-vol">194</span> <span class="cit-issue">Suppl 1</span>: p. <span class="cit-fpage">S51</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BInfect%2BDis%26rft_id%253Dinfo%253Adoi%252F10.1086%252F505356%26rft_id%253Dinfo%253Apmid%252F16921473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/505356&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16921473&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F06%2F25%2F155242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240317800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">10.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-155242v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shafer</span>,  <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Schapiro</span></span>, <span class="cit-article-title">HIV-1 drug resistance mutations: an updated framework for the second decade of HAART</span>. <abbr class="cit-jnl-abbrev">AIDS reviews</abbr>, <span class="cit-pub-date">2008</span>.<span class="cit-vol">10</span>(<span class="cit-issue">2</span>): p. <span class="cit-fpage">67</span>–<span class="cit-lpage">84</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAIDS%2Breviews%26rft.stitle%253DAIDS%2BRev%26rft.aulast%253DShafer%26rft.auinit1%253DR.%2BW.%26rft.volume%253D10%26rft.issue%253D2%26rft.spage%253D67%26rft.epage%253D84%26rft.atitle%253DHIV-1%2Bdrug%2Bresistance%2Bmutations%253A%2Ban%2Bupdated%2Bframework%2Bfor%2Bthe%2Bsecond%2Bdecade%2Bof%2BHAART.%26rft_id%253Dinfo%253Apmid%252F18615118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18615118&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F06%2F25%2F155242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257941100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">11.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-155242v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Z.-W.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Global prevalence of pre-existing HCV variants resistant to direct-acting antiviral agents (DAAs): mining the GenBank HCV genome data</span>. <abbr class="cit-jnl-abbrev">Scientific reports</abbr>, <span class="cit-pub-date">2016</span>. <span class="cit-vol">6</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">12.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-155242v1.12" data-doi="10.1128/AAC.44.7.1771-1777.2000"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martinez</span>,  <span class="cit-name-given-names">J.L.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Baquero</span></span>, <span class="cit-article-title">Mutation Frequencies and Antibiotic Resistance</span>. <abbr class="cit-jnl-abbrev">Antimicrobial Agents and Chemotherapy</abbr>, <span class="cit-pub-date">2000</span>. <span class="cit-vol">44</span>(<span class="cit-issue">7</span>): p. <span class="cit-fpage">1771</span>–<span class="cit-lpage">1777</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntimicrobial%2BAgents%2Band%2BChemotherapy%26rft.stitle%253DAntimicrob.%2BAgents%2BChemother.%26rft.aulast%253DMartinez%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D44%26rft.issue%253D7%26rft.spage%253D1771%26rft.epage%253D1777%26rft.atitle%253DMutation%2BFrequencies%2Band%2BAntibiotic%2BResistance%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAAC.44.7.1771-1777.2000%26rft_id%253Dinfo%253Apmid%252F10858329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWFjIjtzOjU6InJlc2lkIjtzOjk6IjQ0LzcvMTc3MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA2LzI1LzE1NTI0Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tanwar</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Multidrug Resistance: An Emerging Crisis</span>. <abbr class="cit-jnl-abbrev">Interdisciplinary Perspectives on Infectious Diseases</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">2014</span>: p. <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInterdisciplinary%2BPerspectives%2Bon%2BInfectious%2BDiseases%26rft.volume%253D2014%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.14" data-doi="10.1016/B978-0-12-407826-0.00002-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zimmerman</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Red blood cell polymorphism and susceptibility to Plasmodium vivax</span>. <abbr class="cit-jnl-abbrev">Advances in parasitology</abbr>, <span class="cit-pub-date">2013</span>. <span class="cit-vol">81</span>: p. <span class="cit-fpage">27</span>–<span class="cit-lpage">76</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bparasitology%26rft.volume%253D81%26rft.spage%253D27%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-407826-0.00002-3%26rft_id%253Dinfo%253Apmid%252F23384621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-407826-0.00002-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23384621&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F06%2F25%2F155242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.15" data-doi="10.1086/515193"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lell</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">The role of red blood cell polymorphisms in resistance and susceptibility to malaria</span>. <abbr class="cit-jnl-abbrev">Clinical infectious diseases: an official publication of the Infectious Diseases Society of America</abbr>, <span class="cit-pub-date">1999</span>. <span class="cit-vol">28</span>(<span class="cit-issue">4</span>): p. <span class="cit-fpage">794</span>–<span class="cit-lpage">799</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2Binfectious%2Bdiseases%253A%2Ban%2Bofficial%2Bpublication%2Bof%2Bthe%2BInfectious%2BDiseases%2BSociety%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1086%252F515193%26rft_id%253Dinfo%253Apmid%252F10825041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/515193&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10825041&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F06%2F25%2F155242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000079611600021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Moore</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Evidence of HIV-1 adaptation to H LA-restricted immune responses at a population level</span>. <abbr class="cit-jnl-abbrev">Science (New York, N.Y.)</abbr>, <span class="cit-pub-date">2002</span>. <span class="cit-vol">296</span>(<span class="cit-issue">5572</span>): p. <span class="cit-fpage">1439</span>–<span class="cit-lpage">1443</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2B%2528New%2BYork%252C%2BN.Y.%2529%26rft.volume%253D296%26rft.spage%253D1439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">17.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-155242v1.17" data-doi="10.1098/rspb.2013.0696"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Palmer</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Integrating genealogical and dynamical modelling to infer escape and reversion rates in HIV epitopes</span>. <abbr class="cit-jnl-abbrev">Proceedings of the Royal Society B: Biological Sciences</abbr>, <span class="cit-pub-date">2013</span>. <span class="cit-vol">280</span>(<span class="cit-issue">1762</span>): p. <span class="cit-fpage">20130696</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2013.0696%26rft_id%253Dinfo%253Apmid%252F23677344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2013.0696&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23677344&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F06%2F25%2F155242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.18" data-doi="10.1371/journal.ppat.1001196"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fryer</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Modelling the Evolution and Spread of HIV Immune Escape Mutants</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>, <span class="cit-pub-date">2010</span>. <span class="cit-vol">6</span>(<span class="cit-issue">11</span>): p. <span class="cit-fpage">e1001196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DFryer%26rft.auinit1%253DH.%2BR.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253De1001196%26rft.epage%253De1001196%26rft.atitle%253DModelling%2Bthe%2Bevolution%2Band%2Bspread%2Bof%2BHIV%2Bimmune%2Bescape%2Bmutants.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1001196%26rft_id%253Dinfo%253Apmid%252F21124991%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1001196&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21124991&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F06%2F25%2F155242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">19.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-155242v1.19" data-doi="10.1126/science.1232685"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Apps</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Influence of HLA-C Expression Level on HIV Control</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-pub-date">2013</span>. <span class="cit-vol">340</span>(<span class="cit-issue">6128</span>): p. <span class="cit-fpage">87</span>–<span class="cit-lpage">91</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DApps%26rft.auinit1%253DR.%26rft.volume%253D340%26rft.issue%253D6128%26rft.spage%253D87%26rft.epage%253D91%26rft.atitle%253DInfluence%2Bof%2BHLA-C%2BExpression%2BLevel%2Bon%2BHIV%2BControl%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1232685%26rft_id%253Dinfo%253Apmid%252F23559252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzNDAvNjEyOC84NyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA2LzI1LzE1NTI0Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.20" data-doi="10.1371/journal.pone.0000591"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carlson</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Leveraging Hierarchical Population Structure in Discrete Association Studies</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr>, <span class="cit-pub-date">2007</span>. <span class="cit-vol">2</span>(<span class="cit-issue">7</span>): p. <span class="cit-fpage">e591</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DCarlson%26rft.auinit1%253DJ.%26rft.volume%253D2%26rft.issue%253D7%26rft.spage%253De591%26rft.epage%253De591%26rft.atitle%253DLeveraging%2Bhierarchical%2Bpopulation%2Bstructure%2Bin%2Bdiscrete%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000591%26rft_id%253Dinfo%253Apmid%252F17611623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0000591&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17611623&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F06%2F25%2F155242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.21" data-doi="10.1128/JVI.06728-11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carlson</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Widespread Impact of HLA Restriction on Immune Control and Escape Pathways of HIV-1</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>, <span class="cit-pub-date">2012</span>. <span class="cit-vol">86</span>(<span class="cit-issue">9</span>): p. <span class="cit-fpage">5230</span>–<span class="cit-lpage">5243</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.06728-11%26rft_id%253Dinfo%253Apmid%252F22379086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg2LzkvNTIzMCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA2LzI1LzE1NTI0Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.22" data-doi="10.1126/science.1131528"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bhattacharya</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Founder Effects in the Assessment of HIV Polymorphisms and HLA Allele Associations</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-pub-date">2007</span>. <span class="cit-vol">315</span>(<span class="cit-issue">5818</span>): p. <span class="cit-fpage">1583</span>–<span class="cit-lpage">1586</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBhattacharya%26rft.auinit1%253DT.%26rft.volume%253D315%26rft.issue%253D5818%26rft.spage%253D1583%26rft.epage%253D1586%26rft.atitle%253DFounder%2BEffects%2Bin%2Bthe%2BAssessment%2Bof%2BHIV%2BPolymorphisms%2Band%2BHLA%2BAllele%2BAssociations%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1131528%26rft_id%253Dinfo%253Apmid%252F17363674%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTUvNTgxOC8xNTgzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDYvMjUvMTU1MjQyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Palmer</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-etal">et al.</span>, <abbr class="cit-jnl-abbrev">Integrating genealogical and dynamical modelling to infer escape and reversion rates in HIV epitopes</abbr>. <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kessinger</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Perelson</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Neher</span></span>, <span class="cit-article-title">Inferring HIV Escape Rates from Multi-Locus Genotype Data</span>. <abbr class="cit-jnl-abbrev">Frontiers in immunology</abbr>, <span class="cit-pub-date">2013</span>. <span class="cit-fpage">4</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.25" data-doi="10.1073/pnas.112177799"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Beerenwinkel</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Diversity and complexity of HIV-1 drug resistance: A bioinformatics approach to predicting phenotype from genotype</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr>, <span class="cit-pub-date">2002</span>. <span class="cit-vol">99</span>(<span class="cit-issue">12</span>): p. <span class="cit-fpage">8271</span>–<span class="cit-lpage">8276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DBeerenwinkel%26rft.auinit1%253DN.%26rft.volume%253D99%26rft.issue%253D12%26rft.spage%253D8271%26rft.epage%253D8276%26rft.atitle%253DDiversity%2Band%2Bcomplexity%2Bof%2BHIV-1%2Bdrug%2Bresistance%253A%2BA%2Bbioinformatics%2Bapproach%2Bto%2Bpredicting%2Bphenotype%2Bfrom%2Bgenotype%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.112177799%26rft_id%253Dinfo%253Apmid%252F12060770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTkvMTIvODI3MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA2LzI1LzE1NTI0Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carlson</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Phylogenetic dependency networks: inferring patterns of CTL escape and codon covariation in HIV-1 Gag</span>. <abbr class="cit-jnl-abbrev">PLoS computational biology</abbr>, <span class="cit-pub-date">2008</span>. <span class="cit-vol">4</span>(<span class="cit-issue">11</span>): p. <span class="cit-fpage">e1000225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bcomputational%2Bbiology%26rft.volume%253D4%26rft.spage%253De1000225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.27" data-doi="10.1016/j.tim.2014.12.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carlson</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">HIV-1 adaptation to HLA: a window into virus-host immune interactions</span>. <abbr class="cit-jnl-abbrev">Trends in microbiology</abbr>, <span class="cit-pub-date">2015</span>. <span class="cit-vol">23</span>(<span class="cit-issue">4</span>): p. <span class="cit-fpage">212</span>–<span class="cit-lpage">224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bmicrobiology%26rft.volume%253D23%26rft.spage%253D212%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2014.12.008%26rft_id%253Dinfo%253Apmid%252F25613992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2014.12.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25613992&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F06%2F25%2F155242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">28.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-155242v1.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">L.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Lee</span></span>, <span class="cit-article-title">Distinguishing HIV-1 drug resistance, accessory, and viral fitness mutations using conditional selection pressure analysis of treated versus untreated patient samples</span>. <abbr class="cit-jnl-abbrev">Biology direct</abbr>, <span class="cit-pub-date">2006</span>. <span class="cit-vol">1</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiology%2Bdirect%26rft.volume%253D1%26rft.spage%253D14%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.29" data-doi="10.1128/JVI.78.7.3722-3732.2004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Perlina</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Lee</span></span>, <span class="cit-article-title">Positive selection detection in 40,000 human immunodeficiency virus (HIV) type 1 seguences automatically identifies drug resistance and positive fitness mutations in HIV protease and reverse transcriptase</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr>, <span class="cit-pub-date">2004</span>. <span class="cit-vol">78</span>(<span class="cit-issue">7</span>): p. <span class="cit-fpage">3722</span>–<span class="cit-lpage">3732</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DChen%26rft.auinit1%253DL.%26rft.volume%253D78%26rft.issue%253D7%26rft.spage%253D3722%26rft.epage%253D3732%26rft.atitle%253DPositive%2BSelection%2BDetection%2Bin%2B40%252C000%2BHuman%2BImmunodeficiency%2BVirus%2B%2528HIV%2529%2BType%2B1%2BSequences%2BAutomatically%2BIdentifies%2BDrug%2BResistance%2Band%2BPositive%2BFitness%2BMutations%2Bin%2BHIV%2BProtease%2Band%2BReverse%2BTranscriptase%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.78.7.3722-3732.2004%26rft_id%253Dinfo%253Apmid%252F15016892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijc4LzcvMzcyMiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA2LzI1LzE1NTI0Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.30" data-doi="10.1534/genetics.105.044917"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wilson</span>,  <span class="cit-name-given-names">D.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">McVean</span></span>, <span class="cit-article-title">Estimating diversifying selection and functional constraint in the presence of recombination</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-pub-date">2006</span>. <span class="cit-vol">172</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">1411</span>–<span class="cit-lpage">1425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.044917%26rft_id%253Dinfo%253Apmid%252F16387887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi8zLzE0MTEiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNy8wNi8yNS8xNTUyNDIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>,  <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Yang</span></span>, <span class="cit-article-title">Likelihood Models for Detecting Positively Selected Amino Acid Sites and Applications to the HIV-1 Envelope Gene</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-pub-date">1998</span>. <span class="cit-vol">148</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">929</span>–<span class="cit-lpage">936</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DNielsen%26rft.auinit1%253DR.%26rft.volume%253D148%26rft.issue%253D3%26rft.spage%253D929%26rft.epage%253D936%26rft.atitle%253DLikelihood%2BModels%2Bfor%2BDetecting%2BPositively%2BSelected%2BAmino%2BAcid%2BSites%2Band%2BApplications%2Bto%2Bthe%2BHIV-1%2BEnvelope%2BGene%26rft_id%253Dinfo%253Apmid%252F9539414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ4LzMvOTI5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDYvMjUvMTU1MjQyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van Westen</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Significantly improved HIV inhibitor efficacy prediction employing proteochemometric models generated from antivirogram data</span>. <abbr class="cit-jnl-abbrev">PLoS computational biology</abbr>, <span class="cit-pub-date">2013</span>. <span class="cit-vol">9</span>(<span class="cit-issue">2</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.33" data-doi="10.1089/vim.1998.11.167"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Menéndez-Arias</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Mas</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Domingo</span></span>, <span class="cit-article-title">Cytotoxic T-lymphocyte responses to HIV-1 reverse transcriptase (review)</span>. <abbr class="cit-jnl-abbrev">Viral immunology</abbr>, <span class="cit-pub-date">1998</span>. <span class="cit-vol">11</span>(<span class="cit-issue">4</span>): p. <span class="cit-fpage">167</span>–<span class="cit-lpage">181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViral%2Bimmunology%26rft.stitle%253DViral%2BImmunol%26rft.aulast%253DMen%25C3%25A9ndez-Arias%26rft.auinit1%253DL.%26rft.volume%253D11%26rft.issue%253D4%26rft.spage%253D167%26rft.epage%253D181%26rft.atitle%253DCytotoxic%2BT-lymphocyte%2Bresponses%2Bto%2BHIV-1%2Breverse%2Btranscriptase%2B%2528review%2529.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fvim.1998.11.167%26rft_id%253Dinfo%253Apmid%252F10189185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/vim.1998.11.167&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10189185&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F06%2F25%2F155242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000079203500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.34" data-doi="10.1128/JVI.00575-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Selection on the human immunodeficiency virus type 1 proteome following primary infection</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr>, <span class="cit-pub-date">2006</span>. <span class="cit-vol">80</span>(<span class="cit-issue">19</span>): p. <span class="cit-fpage">9519</span>–<span class="cit-lpage">9529</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DLiu%26rft.auinit1%253DY.%26rft.volume%253D80%26rft.issue%253D19%26rft.spage%253D9519%26rft.epage%253D9529%26rft.atitle%253DSelection%2Bon%2Bthe%2BHuman%2BImmunodeficiency%2BVirus%2BType%2B1%2BProteome%2Bfollowing%2BPrimary%2BInfection%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00575-06%26rft_id%253Dinfo%253Apmid%252F16973556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4MC8xOS85NTE5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDYvMjUvMTU1MjQyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">35.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-155242v1.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Dynamics of viral evolution and CTL responses in HIV-1 infection</span>. <abbr class="cit-jnl-abbrev">PloS one</abbr>, <span class="cit-pub-date">2011</span>. <span class="cit-vol">6</span>(<span class="cit-issue">1</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.36" data-doi="10.1128/JVI.01277-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Evolution of human immunodeficiency virus type 1 cytotoxic T-lymphocyte epitopes: fitness-balanced escape</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr>, <span class="cit-pub-date">2007</span>. <span class="cit-vol">81</span>(<span class="cit-issue">22</span>): p. <span class="cit-fpage">12179</span>–<span class="cit-lpage">12188</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01277-07%26rft_id%253Dinfo%253Apmid%252F17728222%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4MS8yMi8xMjE3OSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA2LzI1LzE1NTI0Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.37"><div class="cit-metadata"><cite><abbr class="cit-jnl-abbrev">Collection of B-list epitopes</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.38" data-doi="10.1093/cid/cit430"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gatanaga</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Naturally selected rilpivirine-resistant HIV-1 variants by host cellular immunity</span>. <abbr class="cit-jnl-abbrev">Clinical infectious diseases: an official publication of the Infectious Diseases Society of America</abbr>, <span class="cit-pub-date">2013</span>. <span class="cit-vol">57</span>(<span class="cit-issue">7</span>): p. <span class="cit-fpage">1051</span>–<span class="cit-lpage">1055</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2Binfectious%2Bdiseases%253A%2Ban%2Bofficial%2Bpublication%2Bof%2Bthe%2BInfectious%2BDiseases%2BSociety%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fcid%252Fcit430%26rft_id%253Dinfo%253Apmid%252F23797286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/cid/cit430&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23797286&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F06%2F25%2F155242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.39" data-doi="10.1038/ng.3835"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ansari</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Genome-to-genome analysis highlights the effect of the human innate and adaptive immune systems on the hepatitis C virus</span>. <abbr class="cit-jnl-abbrev">Nature Genetics</abbr>, <span class="cit-pub-date">2017</span>. <span class="cit-vol">49</span>(<span class="cit-issue">5</span>): p. <span class="cit-fpage">666</span>–<span class="cit-lpage">673</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BGenetics%26rft.volume%253D49%26rft.spage%253D666%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3835&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.40" data-doi="10.1038/ng1847"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Price</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Principal components analysis corrects for stratification in genome-wide association studies</span>. <abbr class="cit-jnl-abbrev">Nature Genetics</abbr>, <span class="cit-pub-date">2006</span>. <span class="cit-vol">38</span>(<span class="cit-issue">8</span>): p. <span class="cit-fpage">904</span>–<span class="cit-lpage">909</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTo%26rft.auinit1%253DM.%2BD.%26rft.volume%253D38%26rft.issue%253D8%26rft.spage%253D904%26rft.epage%253D909%26rft.atitle%253DPrincipal%2Bcomponents%2Banalysis%2Bcorrects%2Bfor%2Bstratification%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1847%26rft_id%253Dinfo%253Apmid%252F16862161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16862161&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F06%2F25%2F155242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239325700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">N.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Stephens</span></span>, <span class="cit-article-title">Modeling Linkage Diseguilibrium and Identifying Recombination Hotspots Using Single-Nucleotide Polymorphism Data</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-pub-date">2003</span>. <span class="cit-vol">165</span>(<span class="cit-issue">4</span>): p. <span class="cit-fpage">2213</span>–<span class="cit-lpage">2233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLi%26rft.auinit1%253DN.%26rft.volume%253D165%26rft.issue%253D4%26rft.spage%253D2213%26rft.epage%253D2233%26rft.atitle%253DModeling%2BLinkage%2BDisequilibrium%2Band%2BIdentifying%2BRecombination%2BHotspots%2BUsing%2BSingle-Nucleotide%2BPolymorphism%2BData%26rft_id%253Dinfo%253Apmid%252F14704198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NS80LzIyMTMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNy8wNi8yNS8xNTUyNDIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tenzer</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">HIV-1 Adaptation to Antigen Processing Results in Population-Level Immune Evasion and Affects Subtype Diversification</span>. <abbr class="cit-jnl-abbrev">Cell Reports</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">7</span>(<span class="cit-issue">2</span>): p. <span class="cit-fpage">448</span>–<span class="cit-lpage">463</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BReports%26rft.volume%253D7%26rft.spage%253D448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-155242v1.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Llano</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-etal">et al.</span>, <abbr class="cit-jnl-abbrev">Best-Characterized HIV-1 CTL Epitopes: The 2013 Update</abbr>. <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">44.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-155242v1.44" data-doi="10.1056/NEJMoa1110039"><div class="cit-metadata"><cite><span class="cit-article-title">Short-Course Antiretroviral Therapy in Primary HIV Infection</span>. <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>, <span class="cit-pub-date">2013</span>. <span class="cit-vol">368</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">207</span>–<span class="cit-lpage">217</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft.volume%253D368%26rft.spage%253D207%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1110039%26rft_id%253Dinfo%253Apmid%252F23323897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa1110039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23323897&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F06%2F25%2F155242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000313549900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">45.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-155242v1.45" data-doi="10.1093/nar/gkt1030"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Benson</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">GenBank</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">42</span>(<span class="cit-issue">D1</span>): p. <span class="cit-fpage">D32</span>–<span class="cit-lpage">D37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1030%26rft_id%253Dinfo%253Apmid%252F24217914%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt1030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24217914&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F06%2F25%2F155242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000331139800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">46.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-155242v1.46" data-doi="10.1093/nar/gkl855"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pan</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">The HIV positive selection mutation database</span>. <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr>, <span class="cit-pub-date">2007</span>. <span class="cit-vol">35</span>(<span class="cit-issue">Database issue</span>): p. <span class="cit-fpage">D371</span>–<span class="cit-lpage">D375</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl855%26rft_id%253Dinfo%253Apmid%252F17108357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkl855&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17108357&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F06%2F25%2F155242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243494600077&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">47.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-155242v1.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fagard</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">A prospective trial of structured treatment interruptions in human immunodeficiency virus infection</span>. <abbr class="cit-jnl-abbrev">Archives of internal medicine</abbr>, <span class="cit-pub-date">2003</span>. <span class="cit-vol">163</span>(<span class="cit-issue">10</span>): p. <span class="cit-fpage">12201226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArchives%2Bof%2Binternal%2Bmedicine%26rft.volume%253D163%26rft.spage%253D12201226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">48.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-155242v1.48" data-doi="10.1128/JVI.00022-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Frater</span>,  <span class="cit-name-given-names">A.J.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Effective T-Cell Responses Select Human Immunodeficiency Virus Mutants and Slow Disease Progression</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>, <span class="cit-pub-date">2007</span>. <span class="cit-vol">81</span>(<span class="cit-issue">12</span>): p. <span class="cit-fpage">6742</span>–<span class="cit-lpage">6751</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00022-07%26rft_id%253Dinfo%253Apmid%252F17409157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4MS8xMi82NzQyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDYvMjUvMTU1MjQyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">49.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-155242v1.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">K.-H.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Progression to AIDS in South Africa Is Associated with both Reverting and Compensatory Viral Mutations</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr>, <span class="cit-pub-date">2011</span>. <span class="cit-vol">6</span>(<span class="cit-issue">4</span>): p. <span class="cit-fpage">el9018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D6%26rft.spage%253Del9018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">50.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-155242v1.50" data-doi="10.3851/IMP1416"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">K.-H.G.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Prevalence of HIV type-1 drug-associated mutations in pre-therapy patients in the Free State, South Africa</span>. <abbr class="cit-jnl-abbrev">Antiviral therapy</abbr>, <span class="cit-pub-date">2009</span>. <span class="cit-vol">14</span>(<span class="cit-issue">7</span>): p. <span class="cit-fpage">975</span>–<span class="cit-lpage">984</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntiviral%2Btherapy%26rft.stitle%253DAntivir%2BTher%26rft.aulast%253DHuang%26rft.auinit1%253DK.%2BH.%26rft.volume%253D14%26rft.issue%253D7%26rft.spage%253D975%26rft.epage%253D984%26rft.atitle%253DPrevalence%2Bof%2BHIV%2Btype-1%2Bdrug-associated%2Bmutations%2Bin%2Bpre-therapy%2Bpatients%2Bin%2Bthe%2BFree%2BState%252C%2BSouth%2BAfrica.%26rft_id%253Dinfo%253Adoi%252F10.3851%252FIMP1416%26rft_id%253Dinfo%253Apmid%252F19918101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3851/IMP1416&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19918101&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F06%2F25%2F155242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">51.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-155242v1.51" data-doi="10.1128/JVI.00320-10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Leslie</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Additive Contribution of HLA Class I Alleles in the Immune Control of HIV-1 Infection</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>, <span class="cit-pub-date">2010</span>. <span class="cit-vol">84</span>(<span class="cit-issue">19</span>): p. <span class="cit-fpage">9879</span>–<span class="cit-lpage">9888</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00320-10%26rft_id%253Dinfo%253Apmid%252F20660184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4NC8xOS85ODc5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDYvMjUvMTU1MjQyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">52.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-155242v1.52" data-doi="10.1128/JVI.00580-08"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Matthews</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Central Role of Reverting Mutations in HLA Associations with Human Immunodeficiency Virus Set Point</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>, <span class="cit-pub-date">2008</span>. <span class="cit-vol">82</span>(<span class="cit-issue">17</span>): p. <span class="cit-fpage">8548</span>–<span class="cit-lpage">8559</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00580-08%26rft_id%253Dinfo%253Apmid%252F18596105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Mi8xNy84NTQ4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDYvMjUvMTU1MjQyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">53.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-155242v1.53" data-doi="10.1056/NEJMoa0907736"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>,  <span class="cit-name-given-names">R.L.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Antiretroviral Regimens in Pregnancy and Breast-Feeding in Botswana</span>. <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>, <span class="cit-pub-date">2010</span>. <span class="cit-vol">362</span>(<span class="cit-issue">24</span>): p. <span class="cit-fpage">2282</span>–<span class="cit-lpage">2294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.stitle%253DNEJM%26rft.aulast%253DShapiro%26rft.auinit1%253DR.%2BL.%26rft.volume%253D362%26rft.issue%253D24%26rft.spage%253D2282%26rft.epage%253D2294%26rft.atitle%253DAntiretroviral%2Bregimens%2Bin%2Bpregnancy%2Band%2Bbreast-feeding%2Bin%2BBotswana.%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa0907736%26rft_id%253Dinfo%253Apmid%252F20554983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa0907736&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20554983&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F06%2F25%2F155242.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278816300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">54.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-155242v1.54"><div class="cit-metadata"><cite><abbr class="cit-jnl-abbrev">Los Alamos HIV Seguence Database</abbr>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/155242v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F155242v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/155291v1" title="Buffering Agent Induced Lactose Content Increases via Growth Hormone-Mediated Activation of Gluconeogenesis in Lactating Goats" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/155408v1" title="Epidermal stem cells self-renew upon neighboring differentiation" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;June 25, 2017.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/155242v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/155242v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/155242v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/155242v1" class="active" data-icon-position="" data-hide-link-title="0">Mapping the drivers of within-host pathogen evolution using massive data sets</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/46132" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-vY_ravSqMl1D0VZ-8FZ4e7bZuelqRH0swAcAqtURuUI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="46132" id="node46132--2" data-pisa="biorxiv;155242v1" data-pisa-master="biorxiv;155242" data-seqnum="46132" data-apath="/biorxiv/early/2017/06/25/155242.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Mapping the drivers of within-host pathogen evolution using massive data sets</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Duncan S.</span> <span class="nlm-surname">Palmer</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Isaac</span> <span class="nlm-surname">Turner</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Sarah</span> <span class="nlm-surname">Fidler</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">John</span> <span class="nlm-surname">Frater</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Philip</span> <span class="nlm-surname">Goulder</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Dominique</span> <span class="nlm-surname">Goedhals</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Kuan-Hsiang Gary</span> <span class="nlm-surname">Huang</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Annette</span> <span class="nlm-surname">Oxenius</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Rodney</span> <span class="nlm-surname">Phillips</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Roger</span> <span class="nlm-surname">Shapiro</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Cloete</span> <span class="nlm-surname">van Vuuren</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Angela R.</span> <span class="nlm-surname">McLean</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Gil</span> <span class="nlm-surname">McVean</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">155242; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/155242 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/155242v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/155242v1&amp;title=Mapping%20the%20drivers%20of%20within-host%20pathogen%20evolution%20using%20massive%20data%20sets" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/155242v1&amp;title=Mapping%20the%20drivers%20of%20within-host%20pathogen%20evolution%20using%20massive%20data%20sets" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/155242v1&amp;text=Mapping%20the%20drivers%20of%20within-host%20pathogen%20evolution%20using%20massive%20data%20sets" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/155242v1&amp;title=Mapping%20the%20drivers%20of%20within-host%20pathogen%20evolution%20using%20massive%20data%20sets" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/155242v1&amp;t=Mapping%20the%20drivers%20of%20within-host%20pathogen%20evolution%20using%20massive%20data%20sets" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/155242v1&amp;title=Mapping%20the%20drivers%20of%20within-host%20pathogen%20evolution%20using%20massive%20data%20sets" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/155242v1&amp;title=Mapping%20the%20drivers%20of%20within-host%20pathogen%20evolution%20using%20massive%20data%20sets" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="46132" id="node46132--3" data-pisa="biorxiv;155242v1" data-pisa-master="biorxiv;155242" data-seqnum="46132" data-apath="/biorxiv/early/2017/06/25/155242.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Mapping the drivers of within-host pathogen evolution using massive data sets</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Duncan S.</span> <span class="nlm-surname">Palmer</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Isaac</span> <span class="nlm-surname">Turner</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Sarah</span> <span class="nlm-surname">Fidler</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">John</span> <span class="nlm-surname">Frater</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Philip</span> <span class="nlm-surname">Goulder</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Dominique</span> <span class="nlm-surname">Goedhals</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Kuan-Hsiang Gary</span> <span class="nlm-surname">Huang</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Annette</span> <span class="nlm-surname">Oxenius</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Rodney</span> <span class="nlm-surname">Phillips</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Roger</span> <span class="nlm-surname">Shapiro</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Cloete</span> <span class="nlm-surname">van Vuuren</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Angela R.</span> <span class="nlm-surname">McLean</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Gil</span> <span class="nlm-surname">McVean</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">155242; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/155242 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Mapping%20the%20drivers%20of%20within-host%20pathogen%20evolution%20using%20massive%20data%20sets&amp;rft.volume&amp;rft.issue&amp;rft.date=2017-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Palmer%2C+Duncan+S.&amp;rft.au=Turner%2C+Isaac&amp;rft.au=Fidler%2C+Sarah&amp;rft.au=Frater%2C+John&amp;rft.au=Goulder%2C+Philip&amp;rft.au=Goedhals%2C+Dominique&amp;rft.au=Huang%2C+Kuan-Hsiang+Gary&amp;rft.au=Oxenius%2C+Annette&amp;rft.au=Phillips%2C+Rodney&amp;rft.au=Shapiro%2C+Roger&amp;rft.au=van+Vuuren%2C+Cloete&amp;rft.au=McLean%2C+Angela+R.&amp;rft.au=McVean%2C+Gil"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/46132/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/46132/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/46132/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/46132/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/46132/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/46132/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/46132/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/46132/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/46132/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/46132/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/46132/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/46132/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/155242v1&amp;count=horizontal&amp;via=&amp;text=Mapping%20the%20drivers%20of%20within-host%20pathogen%20evolution%20using%20massive%20data%20sets&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/155242v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/155242v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/155242v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/genetics" class="highlight" data-icon-position="" data-hide-link-title="0">Genetics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20277)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
