<?xml version="1.0" encoding="UTF-8"?><html xmlns:h="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
   <div class="abstract" title="abstract">
      <div class="abstract-title" title="abstract-title">Abstract</div>
      <p>A majority of emerging infectious diseases are of zoonotic origin. Metagenomic Next-Generation
         Sequencing (mNGS) has been employed to identify uncommon and novel infectious etiologies
         and characterize virus diversity in human, animal, and environmental samples. Here,
         we systematically reviewed studies that performed viral mNGS in common livestock (cattle,
         small ruminants, poultry, and pigs). We identified 2481 records and 120 records were
         ultimately included after a first and second screening. Pigs were the most frequently
         studied livestock and the virus diversity found in samples from poultry was the highest.
         Known animal viruses, zoonotic viruses, and novel viruses were reported in available
         literature, demonstrating the capacity of mNGS to identify both known and novel viruses.
         However, the coverage of metagenomic studies was patchy, with few data on the virome
         of small ruminants and respiratory virome of studied livestock. Essential metadata
         such as age of livestock and farm types were rarely mentioned in available literature,
         and only 10.8% of the datasets were publicly available. Developing a deeper understanding
         of livestock virome is crucial for detection of potential zoonotic and animal pathogens
         and One Health preparedness. Metagenomic studies can provide this background but only
         when combined with essential metadata and following the “FAIR” (Findable, Accessible,
         Interoperable, and Reusable) data principles.
      </p>
   </div>
</html>